<html lang="en">
<head>
<meta charset="utf-8"/>
<title offset="35">Adenosine triphosphate</title>
<script src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_SVG.js" type="text/javascript">
</script>
</head>
<body>
<h1>Adenosine triphosphate</h1>
<hr/>

<p><strong>Adenosine triphosphate</strong> (<strong>ATP</strong>) is a <a href="nucleoside_triphosphate" title="wikilink">nucleoside triphosphate</a> used in <a href="cell_(biology)" title="wikilink">cells</a> as a <a class="uri" href="coenzyme" title="wikilink">coenzyme</a> often called the "<a href="molecule" title="wikilink">molecular</a> unit of <a class="uri" href="currency" title="wikilink">currency</a>" of intracellular <a href="energy_transfer" title="wikilink">energy transfer</a>.<a class="footnoteRef" href="#fn1" id="fnref1"><sup>1</sup></a></p>

<p>ATP transports chemical energy within <a href="cell_(biology)" title="wikilink">cells</a> for <a class="uri" href="metabolism" title="wikilink">metabolism</a>. It is one of the end products of <a class="uri" href="photophosphorylation" title="wikilink">photophosphorylation</a>, <a href="cellular_respiration" title="wikilink">cellular respiration</a>, and <a href="fermentation_(biochemistry)" title="wikilink">fermentation</a> and used by <a href="enzyme" title="wikilink">enzymes</a> and <a href="Cytoskeleton" title="wikilink">structural proteins</a> in many cellular processes, including <a href="biosynthesis" title="wikilink">biosynthetic reactions</a>, <a href="Motor_protein" title="wikilink">motility</a>, and <a href="cell_division" title="wikilink">cell division</a>.<a class="footnoteRef" href="#fn2" id="fnref2"><sup>2</sup></a> One molecule of ATP contains three phosphate groups, and it is produced by a wide variety of enzymes, including <a href="ATP_synthase" title="wikilink">ATP synthase</a>, from <a href="adenosine_diphosphate" title="wikilink">adenosine diphosphate</a> (ADP) or <a href="adenosine_monophosphate" title="wikilink">adenosine monophosphate</a> (AMP) and various phosphate group donors. <a href="Substrate-level_phosphorylation" title="wikilink">Substrate-level phosphorylation</a>, <a href="oxidative_phosphorylation" title="wikilink">oxidative phosphorylation</a> in <a href="cellular_respiration" title="wikilink">cellular respiration</a>, and <a class="uri" href="photophosphorylation" title="wikilink">photophosphorylation</a> in <a class="uri" href="photosynthesis" title="wikilink">photosynthesis</a> are three major mechanisms of ATP biosynthesis.</p>

<p>Metabolic processes that use ATP as an energy source convert it back into its precursors. ATP is therefore continuously recycled in organisms: the human body, which on average contains only  of ATP,<a class="footnoteRef" href="#fn3" id="fnref3"><sup>3</sup></a> turns over its own body weight equivalent in ATP each day.<a class="footnoteRef" href="#fn4" id="fnref4"><sup>4</sup></a></p>

<p>ATP is used as a <a href="Substrate_(biochemistry)" title="wikilink">substrate</a> in <a href="signal_transduction" title="wikilink">signal transduction</a> pathways by <a href="kinase" title="wikilink">kinases</a> that <a href="phosphorylation" title="wikilink">phosphorylate</a> <a href="protein" title="wikilink">proteins</a> and <a href="lipid" title="wikilink">lipids</a>. It is also used by <a href="adenylate_cyclase" title="wikilink">adenylate cyclase</a>, which uses ATP to produce the <a href="second_messenger" title="wikilink">second messenger</a> molecule <a href="cyclic_adenosine_monophosphate" title="wikilink">cyclic AMP</a>. The ratio between ATP and AMP is used as a way for a cell to sense how much energy is available and control the <a href="metabolic_pathway" title="wikilink">metabolic pathways</a> that produce and consume ATP.<a class="footnoteRef" href="#fn5" id="fnref5"><sup>5</sup></a> Apart from its roles in signaling and energy metabolism, ATP is also incorporated into <a href="nucleic_acid" title="wikilink">nucleic acids</a> by <a href="polymerase" title="wikilink">polymerases</a> in the process of <a href="transcription_(genetics)" title="wikilink">transcription</a>. ATP is the neurotransmitter believed to signal the sense of taste.<a class="footnoteRef" href="#fn6" id="fnref6"><sup>6</sup></a></p>

<p>The structure of this molecule consists of a <a class="uri" href="purine" title="wikilink">purine</a> base (<a class="uri" href="adenine" title="wikilink">adenine</a>) attached to the 1' carbon atom of a <a class="uri" href="pentose" title="wikilink">pentose</a> sugar (<a class="uri" href="ribose" title="wikilink">ribose</a>). Three phosphate groups are attached at the 5' carbon atom of the pentose sugar. It is the addition and removal of these phosphate groups that inter-convert ATP, <a href="adenosine_diphosphate" title="wikilink">ADP</a> and AMP. When ATP is used in DNA synthesis, the ribose sugar is first converted to <a class="uri" href="deoxyribose" title="wikilink">deoxyribose</a> by <a href="ribonucleotide_reductase" title="wikilink">ribonucleotide reductase</a>.</p>

<p>ATP was discovered in 1929 by Karl Lohmann,<a class="footnoteRef" href="#fn7" id="fnref7"><sup>7</sup></a> and independently by Cyrus Fiske and <a href="Yellapragada_Subbarow" title="wikilink">Yellapragada Subbarow</a> of <a href="Harvard_Medical_School" title="wikilink">Harvard Medical School</a>,<a class="footnoteRef" href="#fn8" id="fnref8"><sup>8</sup></a> but its correct structure was not determined until some years later. It was proposed to be the intermediary molecule between energy-yielding and energy-requiring reactions in cells by <a href="Fritz_Albert_Lipmann" title="wikilink">Fritz Albert Lipmann</a> in 1941.<a class="footnoteRef" href="#fn9" id="fnref9"><sup>9</sup></a> It was first artificially synthesized by <a href="Alexander_R._Todd,_Baron_Todd" title="wikilink">Alexander Todd</a> in 1948.<a class="footnoteRef" href="#fn10" id="fnref10"><sup>10</sup></a></p>
<h2 id="physical-and-chemical-properties">Physical and chemical properties</h2>

<p>ATP consists of <a class="uri" href="adenosine" title="wikilink">adenosine</a> — composed of an <a class="uri" href="adenine" title="wikilink">adenine</a> ring and a <a class="uri" href="ribose" title="wikilink">ribose</a> sugar — and three <a class="uri" href="phosphate" title="wikilink">phosphate</a> groups (triphosphate). The phosphoryl groups, starting with the group closest to the ribose, are referred to as the alpha (α), beta (β), and gamma (γ) phosphates. Consequently, it is closely related to the adenosine nucleotide, a <a class="uri" href="monomer" title="wikilink">monomer</a> of <a class="uri" href="RNA" title="wikilink">RNA</a>. ATP is highly soluble in water and is quite stable in solutions between pH 6.8 and 7.4, but is rapidly <a href="hydrolysis" title="wikilink">hydrolysed</a> at extreme pH. Consequently, ATP is best stored as an anhydrous salt.<a class="footnoteRef" href="#fn11" id="fnref11"><sup>11</sup></a></p>

<p>ATP is an unstable molecule in <a href="Buffer_solution" title="wikilink">unbuffered</a> water, in which it <a href="Hydrolysis" title="wikilink">hydrolyses</a> to <a href="adenosine_diphosphate" title="wikilink">ADP</a> and phosphate. This is because the strength of the bonds between the phosphate groups in ATP is less than the strength of the <a href="hydrogen_bond" title="wikilink">hydrogen bonds</a> (hydration bonds), between its products (ADP + phosphate), and water. Thus, if ATP and ADP are in <a href="chemical_equilibrium" title="wikilink">chemical equilibrium</a> in water, almost all of the ATP will eventually be converted to ADP. A system that is far from equilibrium contains <a href="Gibbs_free_energy" title="wikilink">Gibbs free energy</a>, and is capable of doing <a href="work_(thermodynamics)" title="wikilink">work</a>. Living cells maintain the ratio of ATP to ADP at a point ten orders of magnitude from equilibrium, with ATP concentrations fivefold higher than the concentration of ADP. This displacement from equilibrium means that the hydrolysis of ATP in the cell releases a large amount of free energy.<a class="footnoteRef" href="#fn12" id="fnref12"><sup>12</sup></a></p>

<p>Two <a class="uri" href="phosphoanhydride" title="wikilink">phosphoanhydride</a> bonds (those that connect adjacent phosphates) in an ATP molecule are responsible for the high energy content of this molecule.<a class="footnoteRef" href="#fn13" id="fnref13"><sup>13</sup></a> In the context of biochemical reactions, these anhydride bonds are frequently—and sometimes controversially—referred to as <a href="high-energy_phosphate" title="wikilink"><em>high-energy bonds</em></a> (despite the fact it takes energy to break bonds).<a class="footnoteRef" href="#fn14" id="fnref14"><sup>14</sup></a> Energy stored in ATP may be released upon <a class="uri" href="hydrolysis" title="wikilink">hydrolysis</a> of the anhydride bonds.<a class="footnoteRef" href="#fn15" id="fnref15"><sup>15</sup></a> The primary phosphate group on the ATP molecule that is hydrolyzed when energy is needed to drive anabolic reactions is the γ-phosphate group. Located the farthest from the ribose sugar, it has a higher energy of hydrolysis than either the α- or β-phosphate. The bonds formed after hydrolysis—or the phosphorylation of a residue by ATP—are lower in energy than the phosphoanhydride bonds of ATP. During enzyme-catalyzed hydrolysis of ATP or phosphorylation by ATP, the available free energy can be harnessed by a living system to do work.<a class="footnoteRef" href="#fn16" id="fnref16"><sup>16</sup></a><a class="footnoteRef" href="#fn17" id="fnref17"><sup>17</sup></a></p>

<p>Any unstable system of potentially reactive molecules could potentially serve as a way of storing free energy, if the cell maintained their concentration far from the equilibrium point of the reaction.<a class="footnoteRef" href="#fn18" id="fnref18"><sup>18</sup></a> However, as is the case with most polymeric biomolecules, the breakdown of RNA, DNA, and ATP into simpler monomers is driven by both energy-release and entropy-increase considerations, in both standard concentrations, and also those concentrations encountered within the cell.</p>

<p>The standard amount of energy released from hydrolysis of ATP can be calculated from the changes in energy under non-natural (standard) conditions, then correcting to biological concentrations. The net change in heat energy (<a class="uri" href="enthalpy" title="wikilink">enthalpy</a>) at <a href="Standard_conditions_for_temperature_and_pressure" title="wikilink">standard temperature and pressure</a> of the decomposition of ATP into hydrated ADP and hydrated inorganic phosphate is −30.5 <a href="joule_per_mole" title="wikilink">kJ/mol</a>, with a change in <a href="Thermodynamic_free_energy" title="wikilink">free energy</a> of 3.4 kJ/mol.<a class="footnoteRef" href="#fn19" id="fnref19"><sup>19</sup></a> The energy released by cleaving either a phosphate (P<sub>i</sub>) or pyrophosphate (PP<sub>i</sub>) unit from ATP at <a href="standard_state" title="wikilink">standard state</a> of 1 M are:<a class="footnoteRef" href="#fn20" id="fnref20"><sup>20</sup></a></p>
<dl>
<dd>ATP +  → ADP + P<sub>i</sub>  ΔG˚ = −30.5 kJ/mol (−7.3 kcal/mol)
</dd>
<dd>ATP +  → AMP + PP<sub>i</sub>  ΔG˚ = −45.6 kJ/mol (−10.9 kcal/mol)
</dd>
</dl>

<p>These values can be used to calculate the change in energy under physiological conditions and the cellular ATP/ADP ratio. However, a more representative value (which takes AMP into consideration) called the <a href="Energy_charge" title="wikilink">Energy charge</a> is increasingly being employed. The values given for the <a href="Gibbs_free_energy" title="wikilink">Gibbs free energy</a> for this reaction are dependent on a number of factors, including overall ionic strength and the presence of <a href="alkaline_earth_metal" title="wikilink">alkaline earth metal</a> ions such as  and . Under typical cellular conditions, ΔG is approximately −57 kJ/mol (−14 kcal/mol).<a class="footnoteRef" href="#fn21" id="fnref21"><sup>21</sup></a></p>
<figure><b>(Figure)</b>
<figcaption>This image shows a full 360 degree rotation of a single, gas-phase Magnesium-ATP chelate with a charge of 2-. The molecule was optimized at the UB3LYP/6-311++G(d,p) theoretical level and the atomic connectivity modified by the human optimizer to reflect the probable electronic structure.</figcaption>
</figure>
<h3 id="ionization-in-biological-systems">Ionization in biological systems</h3>

<p>ATP (adenosine triphosphate) has multiple groups with different <a href="acid_dissociation_constant" title="wikilink">acid dissociation constants</a>. In neutral solution, ATP is ionized exists mostly as ATP<sup>4−</sup>, with a small proportion of ATP<sup>3−</sup>.<a class="footnoteRef" href="#fn22" id="fnref22"><sup>22</sup></a> As ATP has several negatively charged groups in neutral solution, it can <a href="chelation" title="wikilink">chelate</a> metals with very high affinity. The <a href="binding_constant" title="wikilink">binding constant</a> for various metal ions are (given as per mole) as  (9 554),  (13),  (3 722),  (8),  (1 381) and  (25).<a class="footnoteRef" href="#fn23" id="fnref23"><sup>23</sup></a> Due to the strength of these interactions, ATP exists in the cell mostly in a complex with .<a class="footnoteRef" href="#fn24" id="fnref24"><sup>24</sup></a><a class="footnoteRef" href="#fn25" id="fnref25"><sup>25</sup></a></p>
<h2 id="biosynthesis">Biosynthesis</h2>

<p>The ATP <a class="uri" href="concentration" title="wikilink">concentration</a> inside the cell is typically 1–10 <a href="molarity" title="wikilink">mM</a>.<a class="footnoteRef" href="#fn26" id="fnref26"><sup>26</sup></a> ATP can be produced by <a class="uri" href="redox" title="wikilink">redox</a> reactions using simple and complex sugars (<a href="carbohydrate" title="wikilink">carbohydrates</a>) or <a href="lipid" title="wikilink">lipids</a> as an energy source. For complex fuels to be synthesized into ATP, they first need to be broken down into smaller, more simple molecules. Carbohydrates are <a href="hydrolysis" title="wikilink">hydrolysed</a> into simple sugars, such as <a class="uri" href="glucose" title="wikilink">glucose</a> and <a class="uri" href="fructose" title="wikilink">fructose</a>. Fats (<a href="triglyceride" title="wikilink">triglycerides</a>) are metabolised to give <a href="fatty_acids" title="wikilink">fatty acids</a> and <a class="uri" href="glycerol" title="wikilink">glycerol</a>.</p>

<p>The overall process of oxidizing glucose to <a href="carbon_dioxide" title="wikilink">carbon dioxide</a> is known as <a href="cellular_respiration" title="wikilink">cellular respiration</a> and can produce about 30 molecules of ATP from a single molecule of glucose.<a class="footnoteRef" href="#fn27" id="fnref27"><sup>27</sup></a> ATP can be produced by a number of distinct cellular processes; the three main pathways used to generate energy in <a href="eukaryote" title="wikilink">eukaryotic</a> organisms are <a class="uri" href="glycolysis" title="wikilink">glycolysis</a> and the <a href="citric_acid_cycle" title="wikilink">citric acid cycle</a>/<a href="oxidative_phosphorylation" title="wikilink">oxidative phosphorylation</a>, both components of <a href="cellular_respiration" title="wikilink">cellular respiration</a>; and <a class="uri" href="beta-oxidation" title="wikilink">beta-oxidation</a>. The majority of this ATP production by a non-<a class="uri" href="photosynthetic" title="wikilink">photosynthetic</a> aerobic eukaryote takes place in the <a class="uri" href="mitochondria" title="wikilink">mitochondria</a>, which can make up nearly 25% of the total volume of a typical cell.<a class="footnoteRef" href="#fn28" id="fnref28"><sup>28</sup></a></p>
<h3 id="glycolysis">Glycolysis</h3>

<p>In glycolysis, glucose and glycerol are metabolized to <a class="uri" href="pyruvate" title="wikilink">pyruvate</a> via the glycolytic pathway. In most organisms, this process occurs in the <a class="uri" href="cytosol" title="wikilink">cytosol</a>, but, in some protozoa such as the <a href="Kinetoplastida" title="wikilink">kinetoplastids</a>, this is carried out in a specialized <a class="uri" href="organelle" title="wikilink">organelle</a> called the <a class="uri" href="glycosome" title="wikilink">glycosome</a>.<a class="footnoteRef" href="#fn29" id="fnref29"><sup>29</sup></a> Glycolysis generates a net two molecules of ATP through <a href="substrate-level_phosphorylation" title="wikilink">substrate phosphorylation</a> catalyzed by two enzymes: <a href="Phosphoglycerate_kinase" title="wikilink">PGK</a> and <a href="pyruvate_kinase" title="wikilink">pyruvate kinase</a>. Two molecules of <a class="uri" href="NADH" title="wikilink">NADH</a> are also produced, which can be oxidized via the <a href="electron_transport_chain" title="wikilink">electron transport chain</a> and result in the generation of additional ATP by <a href="ATP_synthase" title="wikilink">ATP synthase</a>. The pyruvate generated as an end-product of glycolysis is a substrate for the <a href="Krebs_Cycle" title="wikilink">Krebs Cycle</a>.<a class="footnoteRef" href="#fn30" id="fnref30"><sup>30</sup></a></p>
<h3 id="glucose">Glucose</h3>

<p>In the <a class="uri" href="mitochondrion" title="wikilink">mitochondrion</a>, pyruvate is oxidized by the <a href="pyruvate_dehydrogenase_complex" title="wikilink">pyruvate dehydrogenase complex</a> to Acetyl group, which is fully oxidized to carbon dioxide by the citric acid cycle (also known as the Krebs Cycle). Every "turn" of the citric acid cycle produces two molecules of <a href="carbon_dioxide" title="wikilink">carbon dioxide</a>, one molecule of the ATP equivalent <a href="guanosine_triphosphate" title="wikilink">guanosine triphosphate</a> (GTP) through <a href="substrate-level_phosphorylation" title="wikilink">substrate-level phosphorylation</a> catalyzed by <a href="succinyl-CoA_synthetase" title="wikilink">succinyl-CoA synthetase</a>, three molecules of the reduced <a class="uri" href="coenzyme" title="wikilink">coenzyme</a> <a class="uri" href="NADH" title="wikilink">NADH</a>, and one molecule of the reduced coenzyme <a href="Flavin_group" title="wikilink">FADH<sub>2</sub></a>. Both of these latter molecules are recycled to their oxidized states (NAD<sup>+</sup> and <a class="uri" href="FAD" title="wikilink">FAD</a>, respectively) via the <a href="electron_transport_chain" title="wikilink">electron transport chain</a>, which generates additional ATP by <a href="oxidative_phosphorylation" title="wikilink">oxidative phosphorylation</a>. The oxidation of an NADH molecule results in the synthesis of 2–3 ATP molecules, and the oxidation of one FADH<sub>2</sub> yields between 1–2 ATP molecules.<a class="footnoteRef" href="#fn31" id="fnref31"><sup>31</sup></a> The majority of cellular ATP is generated by this process. Although the citric acid cycle itself does not involve molecular <a class="uri" href="oxygen" title="wikilink">oxygen</a>, it is an obligately <a href="aerobic_glycolysis" title="wikilink">aerobic</a> process because  is needed to recycle the reduced NADH and FADH<sub>2</sub> to their oxidized states. In the absence of oxygen the citric acid cycle will cease to function due to the lack of available NAD<sup>+</sup> and FAD.<a class="footnoteRef" href="#fn32" id="fnref32"><sup>32</sup></a></p>

<p>The generation of ATP by the mitochondrion from cytosolic NADH relies on the <a href="malate-aspartate_shuttle" title="wikilink">malate-aspartate shuttle</a> (and to a lesser extent, the <a href="glycerol-phosphate_shuttle" title="wikilink">glycerol-phosphate shuttle</a>) because the inner mitochondrial membrane is impermeable to NADH and NAD<sup>+</sup>. Instead of transferring the generated NADH, a <a href="malate_dehydrogenase" title="wikilink">malate dehydrogenase</a> enzyme converts <a class="uri" href="oxaloacetate" title="wikilink">oxaloacetate</a> to <a class="uri" href="malate" title="wikilink">malate</a>, which is translocated to the mitochondrial matrix. Another malate dehydrogenase-catalyzed reaction occurs in the opposite direction, producing oxaloacetate and NADH from the newly transported malate and the mitochondrion's interior store of NAD<sup>+</sup>. A <a class="uri" href="transaminase" title="wikilink">transaminase</a> converts the oxaloacetate to <a class="uri" href="aspartate" title="wikilink">aspartate</a> for transport back across the membrane and into the intermembrane space.<a class="footnoteRef" href="#fn33" id="fnref33"><sup>33</sup></a></p>

<p>In oxidative phosphorylation, the passage of electrons from NADH and FADH<sub>2</sub> through the electron transport chain powers the pumping of <a href="proton" title="wikilink">protons</a> out of the mitochondrial matrix and into the intermembrane space. This creates a <a href="proton_motive_force" title="wikilink">proton motive force</a> that is the net effect of a <a class="uri" href="pH" title="wikilink">pH</a> gradient and an <a href="electric_potential" title="wikilink">electric potential</a> gradient across the inner mitochondrial membrane. Flow of protons down this potential gradient — that is, from the intermembrane space to the matrix — provides the driving force for ATP synthesis by <a href="ATP_synthase" title="wikilink">ATP synthase</a>. This <a class="uri" href="enzyme" title="wikilink">enzyme</a> contains a rotor subunit that physically rotates relative to the static portions of the protein during ATP synthesis.<a class="footnoteRef" href="#fn34" id="fnref34"><sup>34</sup></a></p>

<p>Most of the ATP synthesized in the mitochondria will be used for cellular processes in the cytosol; thus it must be exported from its site of synthesis in the mitochondrial matrix. The inner membrane contains an <a class="uri" href="antiporter" title="wikilink">antiporter</a>, the <a href="adenosine_diphosphate" title="wikilink">ADP</a>/ATP translocase, which is an <a href="integral_membrane_protein" title="wikilink">integral membrane protein</a> used to exchange newly synthesized ATP in the matrix for <a href="adenosine_diphosphate" title="wikilink">ADP</a> in the intermembrane space.<a class="footnoteRef" href="#fn35" id="fnref35"><sup>35</sup></a> This translocase is driven by the membrane potential, as it results in the movement of about 4 negative charges out of the mitochondrial membrane in exchange for 3 negative charges moved inside. However, it is also necessary to transport phosphate into the mitochondrion; the phosphate carrier moves a proton in with each phosphate, partially dissipating the proton gradient.</p>
<h3 id="beta-oxidation">Beta oxidation</h3>

<p>Fatty acids can also be broken down to <a class="uri" href="acetyl-CoA" title="wikilink">acetyl-CoA</a> by <a class="uri" href="beta-oxidation" title="wikilink">beta-oxidation</a>. Each round of this cycle reduces the length of the acyl chain by two carbon atoms and produces one NADH and one FADH<sub>2</sub> molecule, which are used to generate ATP by oxidative phosphorylation. Because NADH and FADH<sub>2</sub> are energy-rich molecules, dozens of ATP molecules can be generated by the beta-oxidation of a single long acyl chain. The high energy yield of this process and the compact storage of fat explain why it is the most dense source of dietary <a href="calorie" title="wikilink">calories</a>.<a class="footnoteRef" href="#fn36" id="fnref36"><sup>36</sup></a></p>
<h3 id="fermentation">Fermentation</h3>

<p><a href="fermentation_(biochemistry)" title="wikilink">Fermentation</a> entails the generation of energy via the process of <a href="substrate-level_phosphorylation" title="wikilink">substrate-level phosphorylation</a> in the absence of a respiratory <a href="electron_transport_chain" title="wikilink">electron transport chain</a>. In most eukaryotes, glucose is used as both an energy store and an electron donor. The equation for the oxidation of glucose to <a href="lactic_acid" title="wikilink">lactic acid</a> is:</p>
<dl>
<dd>

<p>

<math display="inline" id="Adenosine_triphosphate:0">
 <semantics>
  <mo>→</mo>
  <annotation-xml encoding="MathML-Content">
   <ci>normal-→</ci>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \to
  </annotation>
 </semantics>
</math>

 2 + 2 ATP</p>
</dd>
</dl>
<h3 id="anaerobic-respiration">Anaerobic respiration</h3>

<p>Anaerobic respiration is the process of respiration using an <a href="electron_acceptor" title="wikilink">electron acceptor</a> other than . In prokaryotes, multiple electron acceptors can be used in anaerobic respiration. These include <a class="uri" href="nitrate" title="wikilink">nitrate</a>, <a class="uri" href="sulfate" title="wikilink">sulfate</a> or carbon dioxide. These processes lead to the ecologically important processes of <a class="uri" href="denitrification" title="wikilink">denitrification</a>, sulfate reduction and <a class="uri" href="acetogenesis" title="wikilink">acetogenesis</a>, respectively.<a class="footnoteRef" href="#fn37" id="fnref37"><sup>37</sup></a><a class="footnoteRef" href="#fn38" id="fnref38"><sup>38</sup></a></p>
<h3 id="atp-replenishment-by-nucleoside-diphosphate-kinases">ATP replenishment by nucleoside diphosphate kinases</h3>

<p>ATP can also be synthesized through several so-called "replenishment" reactions catalyzed by the enzyme families of <a href="nucleoside_diphosphate_kinase" title="wikilink">nucleoside diphosphate kinases</a> (NDKs), which use other nucleoside triphosphates as a high-energy phosphate donor, and the <a href="ATP:guanido_phosphotransferase_family" title="wikilink">ATP:guanido-phosphotransferase</a> family,</p>
<h3 id="atp-production-during-photosynthesis">ATP production during photosynthesis</h3>

<p>In plants, ATP is synthesized in <a href="thylakoid_membrane" title="wikilink">thylakoid membrane</a> of the <a class="uri" href="chloroplast" title="wikilink">chloroplast</a> during the <a href="light-dependent_reaction" title="wikilink">light-dependent reactions</a> of <a class="uri" href="photosynthesis" title="wikilink">photosynthesis</a> in a process called photophosphorylation. Here, light energy is used to pump protons across the chloroplast membrane. This produces a proton-motive force and this drives the ATP synthase, exactly as in oxidative phosphorylation.<a class="footnoteRef" href="#fn39" id="fnref39"><sup>39</sup></a> Some of the ATP produced in the chloroplasts is consumed in the <a href="Calvin_cycle" title="wikilink">Calvin cycle</a>, which produces <a class="uri" href="triose" title="wikilink">triose</a> sugars.</p>
<h3 id="atp-recycling">ATP recycling</h3>

<p>The total quantity of ATP in the human body is about 0.2 <a href="Mole_(unit)" title="wikilink">mole</a>. The majority of ATP is not usually synthesised <em>de novo</em>, but is generated from <a href="adenosine_diphosphate" title="wikilink">ADP</a> by the aforementioned processes. Thus, at any given time, the total amount of ATP + <a href="adenosine_diphosphate" title="wikilink">ADP</a> remains fairly constant.</p>

<p>The energy used by human cells requires the <a class="uri" href="hydrolysis" title="wikilink">hydrolysis</a> of 100 to 150 moles of ATP daily, which is around 50 to 75 kg. A human will typically use up his or her body weight of ATP over the course of the day.<a class="footnoteRef" href="#fn40" id="fnref40"><sup>40</sup></a> This means that each ATP molecule is recycled 500 to 750 times during a single day (100 / 0.2 = 500). ATP cannot be stored, hence its consumption closely follows its synthesis. However a total of around 5g of ATP is used by cell processes at any time in the body.</p>
<h2 id="regulation-of-biosynthesis">Regulation of biosynthesis</h2>

<p>ATP production in an aerobic eukaryotic cell is tightly regulated by <a class="uri" href="allosteric" title="wikilink">allosteric</a> mechanisms, by <a class="uri" href="feedback" title="wikilink">feedback</a> effects, and by the substrate concentration dependence of individual enzymes within the glycolysis and oxidative phosphorylation pathways. Key control points occur in enzymatic reactions that are so energetically favorable that they are effectively irreversible under physiological conditions.</p>

<p>In glycolysis, <a class="uri" href="hexokinase" title="wikilink">hexokinase</a> is directly inhibited by its product, glucose-6-phosphate, and <a href="pyruvate_kinase" title="wikilink">pyruvate kinase</a> is inhibited by ATP itself. The main control point for the glycolytic pathway is <a class="uri" href="phosphofructokinase" title="wikilink">phosphofructokinase</a> (PFK), which is allosterically inhibited by high concentrations of ATP and activated by high concentrations of AMP. The inhibition of PFK by ATP is unusual, since ATP is also a substrate in the reaction catalyzed by PFK; the biologically active form of the enzyme is a <a href="tetramer_protein" title="wikilink">tetramer</a> that exists in two possible conformations, only one of which binds the second substrate fructose-6-phosphate (F6P). The protein has two <a href="binding_site" title="wikilink">binding sites</a> for ATP — the <a href="active_site" title="wikilink">active site</a> is accessible in either protein conformation, but ATP binding to the inhibitor site stabilizes the conformation that binds F6P poorly.<a class="footnoteRef" href="#fn41" id="fnref41"><sup>41</sup></a> A number of other small molecules can compensate for the ATP-induced shift in equilibrium conformation and reactivate PFK, including <a href="cyclic_AMP" title="wikilink">cyclic AMP</a>, <a class="uri" href="ammonium" title="wikilink">ammonium</a> ions, inorganic phosphate, and fructose 1,6 and 2,6 biphosphate.<a class="footnoteRef" href="#fn42" id="fnref42"><sup>42</sup></a></p>

<p>The citric acid cycle is regulated mainly by the availability of key substrates, particularly the ratio of NAD<sup>+</sup> to NADH and the concentrations of <a class="uri" href="calcium" title="wikilink">calcium</a>, inorganic phosphate, ATP, <a href="adenosine_diphosphate" title="wikilink">ADP</a>, and AMP. <a class="uri" href="Citrate" title="wikilink">Citrate</a> — the molecule that gives its name to the cycle — is a feedback inhibitor of <a href="citrate_synthase" title="wikilink">citrate synthase</a> and also inhibits PFK, providing a direct link between the regulation of the citric acid cycle and glycolysis.<a class="footnoteRef" href="#fn43" id="fnref43"><sup>43</sup></a></p>

<p>In oxidative phosphorylation, the key control point is the reaction catalyzed by <a href="cytochrome_c_oxidase" title="wikilink">cytochrome c oxidase</a>, which is regulated by the availability of its substrate—the reduced form of <a href="cytochrome_c" title="wikilink">cytochrome c</a>. The amount of reduced cytochrome c available is directly related to the amounts of other substrates:</p>

<p>

<math display="block" id="Adenosine_triphosphate:1">
 <semantics>
  <mrow>
   <mrow>
    <mrow>
     <mfrac>
      <mn>1</mn>
      <mn>2</mn>
     </mfrac>
     <mi>NADH</mi>
    </mrow>
    <mo>+</mo>
    <mrow>
     <mpadded width="+3.3pt">
      <mi>cyt</mi>
     </mpadded>
     <msub>
      <mi mathvariant="normal">c</mi>
      <mi>ox</mi>
     </msub>
    </mrow>
    <mo>+</mo>
    <mi>ADP</mi>
    <mo>+</mo>
    <msub>
     <mi>P</mi>
     <mi>i</mi>
    </msub>
   </mrow>
   <mo>⇔</mo>
   <mrow>
    <mrow>
     <mfrac>
      <mn>1</mn>
      <mn>2</mn>
     </mfrac>
     <msup>
      <mi>NAD</mi>
      <mo>+</mo>
     </msup>
    </mrow>
    <mo>+</mo>
    <mrow>
     <mpadded width="+3.3pt">
      <mi>cyt</mi>
     </mpadded>
     <msub>
      <mi mathvariant="normal">c</mi>
      <mi>red</mi>
     </msub>
    </mrow>
    <mo>+</mo>
    <mi>ATP</mi>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <csymbol cd="latexml">iff</csymbol>
    <apply>
     <plus></plus>
     <apply>
      <times></times>
      <apply>
       <divide></divide>
       <cn type="integer">1</cn>
       <cn type="integer">2</cn>
      </apply>
      <ci>NADH</ci>
     </apply>
     <apply>
      <times></times>
      <ci>cyt</ci>
      <apply>
       <csymbol cd="ambiguous">subscript</csymbol>
       <ci>normal-c</ci>
       <ci>ox</ci>
      </apply>
     </apply>
     <ci>ADP</ci>
     <apply>
      <csymbol cd="ambiguous">subscript</csymbol>
      <ci>P</ci>
      <ci>i</ci>
     </apply>
    </apply>
    <apply>
     <plus></plus>
     <apply>
      <times></times>
      <apply>
       <divide></divide>
       <cn type="integer">1</cn>
       <cn type="integer">2</cn>
      </apply>
      <apply>
       <csymbol cd="ambiguous">superscript</csymbol>
       <ci>NAD</ci>
       <plus></plus>
      </apply>
     </apply>
     <apply>
      <times></times>
      <ci>cyt</ci>
      <apply>
       <csymbol cd="ambiguous">subscript</csymbol>
       <ci>normal-c</ci>
       <ci>red</ci>
      </apply>
     </apply>
     <ci>ATP</ci>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \frac{1}{2}\mathrm{NADH}+\mathrm{cyt~{}c_{ox}}+\mathrm{ADP}+P_{i}\iff\frac{1}{%
2}\mathrm{NAD^{+}}+\mathrm{cyt~{}c_{red}}+\mathrm{ATP}
  </annotation>
 </semantics>
</math>

 which directly implies this equation:</p>

<p>

<math display="block" id="Adenosine_triphosphate:2">
 <semantics>
  <mrow>
   <mfrac>
    <mrow>
     <mpadded width="+3.3pt">
      <mi>cyt</mi>
     </mpadded>
     <msub>
      <mi mathvariant="normal">c</mi>
      <mi>red</mi>
     </msub>
    </mrow>
    <mrow>
     <mpadded width="+3.3pt">
      <mi>cyt</mi>
     </mpadded>
     <msub>
      <mi mathvariant="normal">c</mi>
      <mi>ox</mi>
     </msub>
    </mrow>
   </mfrac>
   <mo>=</mo>
   <mrow>
    <msup>
     <mrow>
      <mo>(</mo>
      <mfrac>
       <mrow>
        <mo stretchy="false">[</mo>
        <mi>NADH</mi>
        <mo stretchy="false">]</mo>
       </mrow>
       <msup>
        <mrow>
         <mo stretchy="false">[</mo>
         <mi>NAD</mi>
         <mo stretchy="false">]</mo>
        </mrow>
        <mo>+</mo>
       </msup>
      </mfrac>
      <mo>)</mo>
     </mrow>
     <mfrac>
      <mn>1</mn>
      <mn>2</mn>
     </mfrac>
    </msup>
    <mrow>
     <mo>(</mo>
     <mfrac>
      <mrow>
       <mrow>
        <mo stretchy="false">[</mo>
        <mi>ADP</mi>
        <mo stretchy="false">]</mo>
       </mrow>
       <mrow>
        <mo stretchy="false">[</mo>
        <msub>
         <mi>P</mi>
         <mi>i</mi>
        </msub>
        <mo stretchy="false">]</mo>
       </mrow>
      </mrow>
      <mrow>
       <mo stretchy="false">[</mo>
       <mi>ATP</mi>
       <mo stretchy="false">]</mo>
      </mrow>
     </mfrac>
     <mo>)</mo>
    </mrow>
    <msub>
     <mi>K</mi>
     <mrow>
      <mi>e</mi>
      <mi>q</mi>
     </mrow>
    </msub>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <eq></eq>
    <apply>
     <divide></divide>
     <apply>
      <times></times>
      <ci>cyt</ci>
      <apply>
       <csymbol cd="ambiguous">subscript</csymbol>
       <ci>normal-c</ci>
       <ci>red</ci>
      </apply>
     </apply>
     <apply>
      <times></times>
      <ci>cyt</ci>
      <apply>
       <csymbol cd="ambiguous">subscript</csymbol>
       <ci>normal-c</ci>
       <ci>ox</ci>
      </apply>
     </apply>
    </apply>
    <apply>
     <times></times>
     <apply>
      <csymbol cd="ambiguous">superscript</csymbol>
      <apply>
       <divide></divide>
       <apply>
        <csymbol cd="latexml">delimited-[]</csymbol>
        <ci>NADH</ci>
       </apply>
       <apply>
        <csymbol cd="ambiguous">superscript</csymbol>
        <apply>
         <csymbol cd="latexml">delimited-[]</csymbol>
         <ci>NAD</ci>
        </apply>
        <plus></plus>
       </apply>
      </apply>
      <apply>
       <divide></divide>
       <cn type="integer">1</cn>
       <cn type="integer">2</cn>
      </apply>
     </apply>
     <apply>
      <divide></divide>
      <apply>
       <times></times>
       <apply>
        <csymbol cd="latexml">delimited-[]</csymbol>
        <ci>ADP</ci>
       </apply>
       <apply>
        <csymbol cd="latexml">delimited-[]</csymbol>
        <apply>
         <csymbol cd="ambiguous">subscript</csymbol>
         <ci>P</ci>
         <ci>i</ci>
        </apply>
       </apply>
      </apply>
      <apply>
       <csymbol cd="latexml">delimited-[]</csymbol>
       <ci>ATP</ci>
      </apply>
     </apply>
     <apply>
      <csymbol cd="ambiguous">subscript</csymbol>
      <ci>K</ci>
      <apply>
       <times></times>
       <ci>e</ci>
       <ci>q</ci>
      </apply>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \frac{\mathrm{cyt~{}c_{red}}}{\mathrm{cyt~{}c_{ox}}}=\left(\frac{[\mathrm{NADH%
}]}{[\mathrm{NAD}]^{+}}\right)^{\frac{1}{2}}\left(\frac{[\mathrm{ADP}][P_{i}]}%
{[\mathrm{ATP}]}\right)K_{eq}
  </annotation>
 </semantics>
</math>

</p>

<p>Thus, a high ratio of [NADH] to [NAD<sup>+</sup>] or a high ratio of [ADP] [P<sub>i</sub>] to [ATP] imply a high amount of reduced cytochrome c and a high level of cytochrome c oxidase activity.<a class="footnoteRef" href="#fn44" id="fnref44"><sup>44</sup></a> An additional level of regulation is introduced by the transport rates of ATP and NADH between the mitochondrial matrix and the cytoplasm.<a class="footnoteRef" href="#fn45" id="fnref45"><sup>45</sup></a></p>
<h2 id="functions-in-cells">Functions in cells</h2>
<h3 id="metabolism-synthesis-and-active-transport">Metabolism, synthesis, and active transport</h3>

<p>ATP is consumed in the cell by energy-requiring (endergonic) processes and can be generated by energy-releasing (exergonic) processes. In this way ATP transfers energy between spatially separate <a href="metabolism" title="wikilink">metabolic reactions</a>. ATP is the main energy source for the majority of cellular functions. This includes the synthesis of macromolecules, including <a href="DNA_replication" title="wikilink">DNA</a> and <a href="Transcription_(genetics)" title="wikilink">RNA</a> (see below), and <a href="Translation_(biology)" title="wikilink">proteins</a>. ATP also plays a critical role in the <a href="active_transport" title="wikilink">transport</a> of macromolecules across cell membranes, e.g. <a class="uri" href="exocytosis" title="wikilink">exocytosis</a> and <a class="uri" href="endocytosis" title="wikilink">endocytosis</a>.</p>
<h3 id="roles-in-cell-structure-and-locomotion">Roles in cell structure and locomotion</h3>

<p>ATP is critically involved in maintaining cell structure by facilitating assembly and disassembly of elements of the <a class="uri" href="cytoskeleton" title="wikilink">cytoskeleton</a>. In a related process, ATP is required for the <a href="Sliding_filament_mechanism" title="wikilink">shortening of actin and myosin filament crossbridges</a> required for <a href="muscle_contraction" title="wikilink">muscle contraction</a>. This latter process is one of the main energy requirements of animals and is essential for <a href="Animal_locomotion" title="wikilink">locomotion</a> and <a href="Respiratory_system" title="wikilink">respiration</a>.</p>
<h3 id="cell-signalling">Cell signalling</h3>
<h4 id="extracellular-signalling">Extracellular signalling</h4>

<p>ATP is also a <a href="signalling_molecule" title="wikilink">signalling molecule</a>. ATP, ADP, or adenosine are recognised by <a href="purinergic_receptors" title="wikilink">purinergic receptors</a>. Purinoreceptors might be the most abundant receptors in mammalian tissues.<a class="footnoteRef" href="#fn46" id="fnref46"><sup>46</sup></a></p>

<p>In humans, this signalling role is important in both the central and peripheral nervous system.<a class="footnoteRef" href="#fn47" id="fnref47"><sup>47</sup></a> Activity-dependent release of ATP from synapses, axons and glia activates purinergic membrane receptors known as P2.<a class="footnoteRef" href="#fn48" id="fnref48"><sup>48</sup></a> The <em><a href="P2Y_receptors" title="wikilink">P2Y</a></em> receptors are <em>metabotropic</em>, i.e. <a href="G_protein–coupled_receptor" title="wikilink">G protein-coupled</a> and modulate mainly intracellular calcium and sometimes cyclic AMP levels. Though named between P2Y<sub>1</sub> and P2Y<sub>15</sub>, only nine members of the P2Y family have been cloned, and some are only related through weak homology and several (P2Y<sub>5</sub>, P2Y<sub>7</sub>, P2Y<sub>9</sub>, P2Y<sub>10</sub>) do not function as receptors that raise cytosolic calcium. The <em><a href="P2X_receptors" title="wikilink">P2X</a> ionotropic</em> receptor subgroup comprises seven members (P2X<sub>1</sub>–P2X<sub>7</sub>), which are ligand-gated -permeable ion channels that open when bound to an extracellular purine nucleotide. In contrast to P2 receptors (agonist order ATP &gt; <a href="adenosine_diphosphate" title="wikilink">ADP</a> &gt; AMP &gt; ADO), purinergic nucleoside triphosphates like ATP are not strong agonists of P1 receptors, which are strongly activated by <a class="uri" href="adenosine" title="wikilink">adenosine</a> and other <a href="nucleoside" title="wikilink">nucleosides</a> (ADO &gt; AMP &gt; <a href="adenosine_diphosphate" title="wikilink">ADP</a> &gt; ATP). P1 receptors have A1, A2a, A2b, and A3 subtypes ("A" as a remnant of old nomenclature of <em>adenosine receptor</em>), all of which are G protein-coupled receptors, A1 and A3 being coupled to Gi, and A2a and A2b being coupled to Gs.<a class="footnoteRef" href="#fn49" id="fnref49"><sup>49</sup></a></p>

<p>All adenosine receptors were shown to activate at least one subfamily of mitogen-activated protein kinases. The actions of adenosine are often antagonistic or synergistic to the actions of ATP. In the CNS, adenosine has multiple functions, such as modulation of neural development, neuron and glial signalling and the control of innate and adaptive immune systems.<a class="footnoteRef" href="#fn50" id="fnref50"><sup>50</sup></a></p>
<h4 id="intracellular-signaling">Intracellular signaling</h4>

<p>ATP is critical in <a href="signal_transduction" title="wikilink">signal transduction</a> processes. It is used by <a href="kinase" title="wikilink">kinases</a> as the source of phosphate groups in their phosphate transfer reactions. Kinase activity on substrates such as proteins or membrane lipids are a common form of signal transduction. <a class="uri" href="Phosphorylation" title="wikilink">Phosphorylation</a> of a protein by a kinase can activate this cascade such as the <a href="mitogen-activated_protein_kinase" title="wikilink">mitogen-activated protein kinase</a> cascade.<a class="footnoteRef" href="#fn51" id="fnref51"><sup>51</sup></a></p>

<p>ATP is also used by <a href="adenylate_cyclase" title="wikilink">adenylate cyclase</a> and is transformed to the <a href="second_messenger" title="wikilink">second messenger</a> molecule cyclic AMP, which is involved in triggering calcium signals by the release of calcium from intracellular stores.<a class="footnoteRef" href="#fn52" id="fnref52"><sup>52</sup></a> This form of signal transduction is particularly important in brain function, although it is involved in the regulation of a multitude of other cellular processes.<a class="footnoteRef" href="#fn53" id="fnref53"><sup>53</sup></a></p>
<h3 id="dna-and-rna-synthesis">DNA and RNA synthesis</h3>

<p>In all known organisms, the Deoxyribonucleotides that make up <a class="uri" href="DNA" title="wikilink">DNA</a> are synthesized by the action of <a href="ribonucleotide_reductase" title="wikilink">ribonucleotide reductase</a> (RNR) enzymes on their corresponding <a class="uri" href="ribonucleotides" title="wikilink">ribonucleotides</a>.<a class="footnoteRef" href="#fn54" id="fnref54"><sup>54</sup></a> These enzymes reduce the sugar residue from <a class="uri" href="ribose" title="wikilink">ribose</a> to <a class="uri" href="deoxyribose" title="wikilink">deoxyribose</a> by removing oxygen from the 2' <a class="uri" href="hydroxyl" title="wikilink">hydroxyl</a> group; the substrates are ribonucleoside diphosphates and the products deoxyribonucleoside diphosphates (the latter are denoted dADP, dCDP, dGDP, and dUDP respectively.) All ribonucleotide reductase enzymes use a common <a class="uri" href="sulfhydryl" title="wikilink">sulfhydryl</a> <a href="radical_(chemistry)" title="wikilink">radical</a> mechanism reliant on reactive <a class="uri" href="cysteine" title="wikilink">cysteine</a> residues that oxidize to form <a href="disulfide_bond" title="wikilink">disulfide bonds</a> in the course of the reaction.<a class="footnoteRef" href="#fn55" id="fnref55"><sup>55</sup></a> RNR enzymes are recycled by reaction with <a class="uri" href="thioredoxin" title="wikilink">thioredoxin</a> or <a class="uri" href="glutaredoxin" title="wikilink">glutaredoxin</a>.<a class="footnoteRef" href="#fn56" id="fnref56"><sup>56</sup></a></p>

<p>The regulation of RNR and related enzymes maintains a balance of dNTPs relative to each other and relative to NTPs in the cell. Very low dNTP concentration inhibits <a href="DNA_replication" title="wikilink">DNA synthesis</a> and <a href="DNA_repair" title="wikilink">DNA repair</a> and is lethal to the cell, while an abnormal ratio of dNTPs is <a href="mutagen" title="wikilink">mutagenic</a> due to the increased likelihood of the <a href="DNA_polymerase" title="wikilink">DNA polymerase</a> incorporating the wrong dNTP during DNA synthesis.<a class="footnoteRef" href="#fn57" id="fnref57"><sup>57</sup></a> Regulation of or differential specificity of RNR has been proposed as a mechanism for alterations in the relative sizes of intracellular dNTP pools under cellular stress such as <a href="Hypoxia_(medical)" title="wikilink">hypoxia</a>.<a class="footnoteRef" href="#fn58" id="fnref58"><sup>58</sup></a></p>

<p>In the synthesis of the <a href="nucleic_acid" title="wikilink">nucleic acid</a> <a class="uri" href="RNA" title="wikilink">RNA</a>, adenosine derived from ATP is one of the four nucleotides incorporated directly into RNA molecules by <a href="RNA_polymerase" title="wikilink">RNA polymerases</a>. The energy driving this polymerization comes from cleaving off a pyrophosphate (two phosphate groups).<a class="footnoteRef" href="#fn59" id="fnref59"><sup>59</sup></a> The process is similar in DNA biosynthesis, except that ATP is reduced to the <a class="uri" href="deoxyribonucleotide" title="wikilink">deoxyribonucleotide</a> dATP, before incorporation into DNA.</p>
<h2 id="amino-acid-activation-in-protein-synthesis">Amino acid activation in protein synthesis</h2>

<p>Aminoacyl-tRNA synthetase enzymes utilise ATP as an energy source to attach a tRNA molecule to its specific amino acid, forming an aminoacyl-tRNA complex, ready for <a href="Translation_(biology)" title="wikilink">translation</a> at <a href="ribosome" title="wikilink">ribosomes</a>. The energy is made available by ATP hydrolysis to <a href="adenosine_monophosphate" title="wikilink">adenosine monophosphate</a> (AMP) as two phosphate groups are removed. Amino acid activation refers to the attachment of an amino acid to its Transfer RNA (tRNA).</p>

<p><code>   Aminoacyl transferase binds Adenosine triphosphate (ATP) to amino acid, PP is released.</code><br/>
<code>   Aminoacyl transferase binds AMP-amino acid to tRNA. The AMP is used in this step.</code></p>
<h3 id="amino-acid-activation">Amino Acid Activation</h3>

<p>During amino acid activation the amino acids (aa) are attached to their corresponding tRNA. The coupling reactions are catalysed by a group of enzymes called aminoacyl-tRNA synthetases (named after the reaction product aminoacyl-tRNA or aa-tRNA). The coupling reaction proceeds in two steps:</p>

<p>1. aa + ATP aa-AMP + PP, (pyrophosphate) 2. aa-AMP + tRNA aa-tRNA + AMP</p>

<p>The amino acid is coupled to the penultimate nucleotide at the 3’-end of the tRNA (the A in the sequence CCA) via an ester bond (roll over in illustration). The formation of the ester bond conserves a considerable part of the energy from the activation reaction. This stored energy provides the majority of the energy needed for peptide bond formation during translation.</p>

<p>Each of the 20 amino acids are recognized by its specific aminoacyl-tRNA synthetase. The synthetases are usually composed of one to four protein subunits. The enzymes vary considerably in structure although they all perform the same type of reaction by binding ATP, one specific amino acid and its corresponding tRNA.</p>

<p>The specificity of the amino acid activation is as critical for the translational accuracy as the correct matching of the codon with the anticodon. The reason is that the ribosome only sees the anticodon of the tRNA during translation. Thus, the ribosome will not be able to discriminate between tRNAs with the same anticodon but linked to different amino acids.</p>

<p>The error frequency of the amino acid activation reaction is approximately 1 in 10 000 despite the small structural differences between some of the amino acids.<a class="footnoteRef" href="#fn60" id="fnref60"><sup>60</sup></a></p>
<h2 id="binding-to-proteins">Binding to proteins</h2>

<p>Some proteins that bind ATP do so in a characteristic <a href="tertiary_structure" title="wikilink">protein fold</a> known as the <a href="Rossmann_fold" title="wikilink">Rossmann fold</a>, which is a general <a class="uri" href="nucleotide" title="wikilink">nucleotide</a>-binding <a href="structural_domain" title="wikilink">structural domain</a> that can also bind the <a class="uri" href="coenzyme" title="wikilink">coenzyme</a> <a href="Nicotinamide_adenine_dinucleotide" title="wikilink">NAD</a>.<a class="footnoteRef" href="#fn61" id="fnref61"><sup>61</sup></a> The most common ATP-binding proteins, known as <a class="uri" href="kinases" title="wikilink">kinases</a>, share a small number of common folds; the <a href="protein_kinase" title="wikilink">protein kinases</a>, the largest kinase superfamily, all share common structural features specialized for ATP binding and phosphate transfer.<a class="footnoteRef" href="#fn62" id="fnref62"><sup>62</sup></a></p>

<p>ATP in complexes with proteins, in general, requires the presence of a <a class="uri" href="divalent" title="wikilink">divalent</a> <a class="uri" href="cation" title="wikilink">cation</a>, almost always <a class="uri" href="magnesium" title="wikilink">magnesium</a>, which binds to the ATP phosphate groups. The presence of magnesium greatly decreases the <a href="dissociation_constant" title="wikilink">dissociation constant</a> of ATP from its protein binding partner without affecting the ability of the enzyme to catalyze its reaction once the ATP has bound.<a class="footnoteRef" href="#fn63" id="fnref63"><sup>63</sup></a> The presence of magnesium ions can serve as a mechanism for kinase regulation.<a class="footnoteRef" href="#fn64" id="fnref64"><sup>64</sup></a> </p>
<h2 id="atp-analogues">ATP analogues</h2>

<p>Biochemistry laboratories often use <em><a href="in_vitro" title="wikilink">in vitro</a></em> studies to explore ATP-dependent molecular processes. <a href="Enzyme_inhibitor" title="wikilink">Enzyme inhibitors</a> of ATP-dependent enzymes such as <a href="kinase" title="wikilink">kinases</a> are needed to examine the <a href="binding_site" title="wikilink">binding sites</a> and <a href="transition_state" title="wikilink">transition states</a> involved in ATP-dependent reactions. ATP analogs are also used in <a href="X-ray_crystallography" title="wikilink">X-ray crystallography</a> to determine a <a href="protein_structure" title="wikilink">protein structure</a> in complex with ATP, often together with other substrates. Most useful ATP analogs cannot be hydrolyzed as ATP would be; instead they trap the enzyme in a structure closely related to the ATP-bound state. Adenosine 5'-(gamma-thiotriphosphate) is an extremely common ATP analog in which one of the gamma-phosphate oxygens is replaced by a <a class="uri" href="sulfur" title="wikilink">sulfur</a> atom; this molecule is hydrolyzed at a dramatically slower rate than ATP itself and functions as an inhibitor of ATP-dependent processes. In crystallographic studies, hydrolysis transition states are modeled by the bound <a class="uri" href="vanadate" title="wikilink">vanadate</a> ion. However, caution is warranted in interpreting the results of experiments using ATP analogs, since some enzymes can hydrolyze them at appreciable rates at high concentration.<a class="footnoteRef" href="#fn65" id="fnref65"><sup>65</sup></a> </p>
<h2 id="see-also">See also</h2>
<ul>
<li><a href="Adenosine_diphosphate" title="wikilink">Adenosine diphosphate</a> (ADP)</li>
<li><a href="Adenosine_monophosphate" title="wikilink">Adenosine monophosphate</a> (AMP)</li>
<li><a class="uri" href="Adenosine-tetraphosphatase" title="wikilink">Adenosine-tetraphosphatase</a></li>
<li><a class="uri" href="ATPases" title="wikilink">ATPases</a></li>
<li><a href="ATP_test" title="wikilink">ATP test</a></li>
<li><a href="ATP_hydrolysis" title="wikilink">ATP hydrolysis</a></li>
<li><a href="Citric_acid_cycle" title="wikilink">Citric acid cycle</a> (also called the Krebs cycle or TCA cycle)</li>
<li><a href="Cyclic_adenosine_monophosphate" title="wikilink">Cyclic adenosine monophosphate</a> (cAMP)</li>
<li><a href="Nucleotide_exchange_factor" title="wikilink">Nucleotide exchange factor</a></li>
<li><a class="uri" href="Phosphagen" title="wikilink">Phosphagen</a></li>
<li><a class="uri" href="Photophosphorylation" title="wikilink">Photophosphorylation</a></li>
</ul>
<h2 id="references">References</h2>
<h2 id="external-links">External links</h2>
<ul>
<li><a href="http://www.ebi.ac.uk/pdbe-srv/PDBeXplore/ligand/?ligand=ATP">ATP bound to proteins</a> in the <a href="Protein_Data_Bank" title="wikilink">PDB</a></li>
<li><a href="http://www.scienceaid.co.uk/biology/biochemistry/atp.html">ScienceAid: Energy ATP and Exercise</a></li>
<li><a href="http://pubchem.ncbi.nlm.nih.gov/summary/summary.cgi?cid=5957">PubChem entry for Adenosine Triphosphate</a></li>
<li><a href="http://www.genome.jp/dbget-bin/www_bget?cpd:C00002">KEGG entry for Adenosine Triphosphate</a></li>
</ul>

<p>"</p>

<p><a href="Category:Adenosine_receptor_agonists" title="wikilink">Category:Adenosine receptor agonists</a> <a class="uri" href="Category:Neurotransmitters" title="wikilink">Category:Neurotransmitters</a> <a href="Category:Purinergic_signalling" title="wikilink">Category:Purinergic signalling</a> <a href="Category:Cellular_respiration" title="wikilink">Category:Cellular respiration</a> <a href="Category:Exercise_physiology" title="wikilink">Category:Exercise physiology</a> <a class="uri" href="Category:Nucleotides" title="wikilink">Category:Nucleotides</a> <a class="uri" href="Category:Coenzymes" title="wikilink">Category:Coenzymes</a> <a class="uri" href="Category:Purines" title="wikilink">Category:Purines</a></p>
<section class="footnotes">
<hr/>
<ol>
<li id="fn1"><a href="#fnref1">↩</a></li>
<li id="fn2"><a href="#fnref2">↩</a></li>
<li id="fn3"><a href="#fnref3">↩</a></li>
<li id="fn4"><a href="#fnref4">↩</a></li>
<li id="fn5"><a href="#fnref5">↩</a></li>
<li id="fn6"><a href="#fnref6">↩</a></li>
<li id="fn7"><a href="#fnref7">↩</a></li>
<li id="fn8"><a href="#fnref8">↩</a></li>
<li id="fn9"><a href="#fnref9">↩</a></li>
<li id="fn10"><a href="#fnref10">↩</a></li>
<li id="fn11"><a href="#fnref11">↩</a></li>
<li id="fn12"><a href="#fnref12">↩</a></li>
<li id="fn13"><a href="#fnref13">↩</a></li>
<li id="fn14"><a href="#fnref14">↩</a></li>
<li id="fn15"></li>
<li id="fn16"><a href="#fnref16">↩</a></li>
<li id="fn17"><a href="#fnref17">↩</a></li>
<li id="fn18"></li>
<li id="fn19"><a href="#fnref19">↩</a></li>
<li id="fn20"><a href="#fnref20">↩</a></li>
<li id="fn21"><a href="#fnref21">↩</a></li>
<li id="fn22"><a href="#fnref22">↩</a></li>
<li id="fn23"><a href="#fnref23">↩</a></li>
<li id="fn24"></li>
<li id="fn25"><a href="#fnref25">↩</a></li>
<li id="fn26"><a href="#fnref26">↩</a></li>
<li id="fn27"><a href="#fnref27">↩</a></li>
<li id="fn28"><a href="#fnref28">↩</a></li>
<li id="fn29"><a href="#fnref29">↩</a></li>
<li id="fn30"><a href="#fnref30">↩</a></li>
<li id="fn31"></li>
<li id="fn32"></li>
<li id="fn33"></li>
<li id="fn34"><a href="#fnref34">↩</a></li>
<li id="fn35"><a href="#fnref35">↩</a></li>
<li id="fn36"><a href="#fnref36">↩</a></li>
<li id="fn37"><a href="#fnref37">↩</a></li>
<li id="fn38"><a href="#fnref38">↩</a></li>
<li id="fn39"><a href="#fnref39">↩</a></li>
<li id="fn40"><a href="#fnref40">↩</a></li>
<li id="fn41"></li>
<li id="fn42"></li>
<li id="fn43"></li>
<li id="fn44"></li>
<li id="fn45"></li>
<li id="fn46"><a href="#fnref46">↩</a></li>
<li id="fn47"><a href="#fnref47">↩</a></li>
<li id="fn48"><a href="#fnref48">↩</a></li>
<li id="fn49"><a href="#fnref49">↩</a></li>
<li id="fn50"></li>
<li id="fn51"><a href="#fnref51">↩</a></li>
<li id="fn52"><a href="#fnref52">↩</a></li>
<li id="fn53"><a href="#fnref53">↩</a></li>
<li id="fn54"><a href="#fnref54">↩</a></li>
<li id="fn55"></li>
<li id="fn56"></li>
<li id="fn57"></li>
<li id="fn58"><a href="#fnref58">↩</a></li>
<li id="fn59"><a href="#fnref59">↩</a></li>
<li id="fn60"><em>Eldra P. Solomon, Linda R. Berg, Diana W. Martin</em>. Biology (8th Ed). ISBN 978-0-495-30978-9<a href="#fnref60">↩</a></li>
<li id="fn61"><a href="#fnref61">↩</a></li>
<li id="fn62"><a href="#fnref62">↩</a></li>
<li id="fn63"><a href="#fnref63">↩</a></li>
<li id="fn64"><a href="#fnref64">↩</a></li>
<li id="fn65"><a href="#fnref65">↩</a></li>
</ol>
</section>
</body>
</html>
