<html lang="en">
<head>
<meta charset="utf-8"/>
<title offset="401">BLOSUM</title>
<script src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_SVG.js" type="text/javascript">
</script>
</head>
<body>
<h1>BLOSUM</h1>
<hr/>

<p> In <a class="uri" href="bioinformatics" title="wikilink">bioinformatics</a>, the <strong>BLOSUM</strong> (<strong>BLO</strong>cks <strong>SU</strong>bstitution <strong>M</strong>atrix) matrix is a <a href="substitution_matrix" title="wikilink">substitution matrix</a> used for <a href="sequence_alignment" title="wikilink">sequence alignment</a> of <a href="protein" title="wikilink">proteins</a>. BLOSUM matrices are used to score alignments between <a href="Evolutionary_divergence" title="wikilink">evolutionarily divergent</a> protein sequences. They are based on local alignments. BLOSUM matrices were first introduced in a paper by Henikoff and Henikoff.<a class="footnoteRef" href="#fn1" id="fnref1"><sup>1</sup></a> They scanned the <a href="BLOCKS_database" title="wikilink">BLOCKS database</a> for very <a href="Conserved_sequence" title="wikilink">conserved regions</a> of protein families (that do not have gaps in the sequence alignment) and then counted the relative frequencies of <a href="amino_acids" title="wikilink">amino acids</a> and their substitution probabilities. Then, they calculated a <a href="odds_ratio" title="wikilink">log-odds</a> score for each of the 210 possible substitution pairs of the 20 standard amino acids. All BLOSUM matrices are based on observed alignments; they are not extrapolated from comparisons of closely related proteins like the <a href="Point_accepted_mutation" title="wikilink">PAM Matrices</a>.</p>
<h2 id="biological-background">Biological background</h2>

<p>The genetic instructions of every replicating <a href="Cell_(biology)" title="wikilink">cell</a> in a living organism are contained within its DNA.<a class="footnoteRef" href="#fn2" id="fnref2"><sup>2</sup></a> Throughout the cell's lifetime, this information is transcribed and replicated by cellular mechanisms to produce proteins or to provide instructions for daughter cells during <a href="cell_division" title="wikilink">cell division</a>, and the possibility exists that the DNA may be altered during these processes.<a class="footnoteRef" href="#fn3" id="fnref3"><sup>3</sup></a><a class="footnoteRef" href="#fn4" id="fnref4"><sup>4</sup></a> This is known as a <a class="uri" href="mutation" title="wikilink">mutation</a>. At the molecular level, there are regulatory systems that correct most — but not all — of these changes to the DNA before it is replicated.<a class="footnoteRef" href="#fn5" id="fnref5"><sup>5</sup></a><a class="footnoteRef" href="#fn6" id="fnref6"><sup>6</sup></a></p>

<p>The functionality of a protein is highly dependent on its structure.<a class="footnoteRef" href="#fn7" id="fnref7"><sup>7</sup></a> Changing a single amino acid in a protein may reduce its ability to carry out this function, or the mutation may even change the function that the protein carries out.<a class="footnoteRef" href="#fn8" id="fnref8"><sup>8</sup></a> Changes like these may severely impact a crucial function in a cell, potentially causing the cell — and in extreme cases, the organism — to die.<a class="footnoteRef" href="#fn9" id="fnref9"><sup>9</sup></a> Conversely, the change may allow the cell to continue functioning albeit differently, and the mutation can be passed on to the organism's offspring. If this change does not result in any significant physical disadvantage to the offspring, the possibility exists that this mutation will persist within the population. The possibility also exists that the change in function becomes advantageous.</p>

<p>The 20 amino acids translated by the genetic code vary greatly by the physical and chemical properties of their side chains.<a class="footnoteRef" href="#fn10" id="fnref10"><sup>10</sup></a> However, these amino acids can be categorised into groups with similar physicochemical properties.<a class="footnoteRef" href="#fn11" id="fnref11"><sup>11</sup></a> Substituting an amino acid with another from the same category is more likely to have a smaller impact on the structure and function of a protein than replacement with an amino acid from a different category.</p>

<p>Sequence alignment is a fundamental research method for modern biology. The most common sequence alignment for protein is to look for the similarity between different sequences in order to understand the evolutionarily divergent protein sequences on the molecular level, so that researchers could predict the functions initiated by those mutated genes. Matrices are applied as algorithms to calculate the similarity of different sequences of proteins; however, the utility of Dayhoff Matrix which is a widely used method before is limited due to the requirement of sequences with a similarity more than 85%. In order to fill in this gap, Henikoff and Henikoff introduced BLOSUM (BLOcks SUbstitution Matrix) matrix which led to marked improvements in alignments and in searches using queries from each of the groups of related proteins.<a class="footnoteRef" href="#fn12" id="fnref12"><sup>12</sup></a></p>
<h2 id="terminology">Terminology</h2>

<p>BLOSUM: Blocks Substitution Matrix, a <a href="substitution_matrix" title="wikilink">substitution matrix</a> used for <a href="sequence_alignment" title="wikilink">sequence alignment</a> of <a href="protein" title="wikilink">proteins</a>.</p>

<p>Scoring metrics (statistical versus biological): When evaluating a sequence alignment, one would like to know how meaningful it is. This requires a scoring matrix, or a table of values that describes the probability of a biologically meaningful amino-acid or nucleotide residue-pair occurring in an alignment. Scores for each position are obtained frequencies of substitutions in blocks of local alignments of protein sequences.<a class="footnoteRef" href="#fn13" id="fnref13"><sup>13</sup></a></p>

<p>Several sets of BLOSUM matrices exist using different alignment databases, named with numbers. BLOSUM matrices with high numbers are designed for comparing closely related sequences, while those with low numbers are designed for comparing distant related sequences. For example, BLOSUM80 is used for less divergent alignments, and BLOSUM45 is used for more divergent alignments. The matrices were created by merging (clustering) all sequences that were more similar than a given percentage into one single sequence and then comparing those sequences (that were all more divergent than the given percentage value) only; thus reducing the contribution of closely related sequences. The percentage used was appended to the name, giving BLOSUM80 for example where sequences that were more than 80% identical were clustered.</p>

<p>BLOSUM r: the matrix built from blocks with no more than r% of similarity – E.g., BLOSUM62 is the matrix built using sequences with no more than 62% similarity. – Note: BLOSUM 62 is the default matrix for protein BLAST. Experimentation has shown that the BLOSUM-62 matrix is among the best for detecting most weak protein similarities.<a class="footnoteRef" href="#fn14" id="fnref14"><sup>14</sup></a></p>
<h2 id="construction-of-blosum-matrices">Construction of BLOSUM matrices</h2>

<p>BLOSUM matrices are obtained by using blocks of similar amino acid sequences as data, then applying statistical methods to the data to obtain the similarity scores. Statistical Methods Steps : <a class="footnoteRef" href="#fn15" id="fnref15"><sup>15</sup></a></p>
<h3 id="eliminating-sequences">Eliminating Sequences</h3>

<p>Eliminating the sequences that are more than r% identical. There are two ways to eliminate the sequences. It can be done either by removing sequences from the block or just by finding similar sequences and replace them by new sequences which could represent the cluster. Eliminating is done to avoid bias of the result in favor of a certain protein.</p>
<h3 id="calculating-frequency-probability">Calculating Frequency &amp; Probability</h3>

<p>A database storing the sequence alignments of the most conserved regions of protein families. These alignments are used to derive the BLOSUM matrices. Only the sequences with a percentage of identity higher are used. By using the block, counting the pairs of amino acids in each column of the multiple alignment.</p>
<h3 id="log-odd-ratio">Log odd ratio</h3>

<p>It gives the ratio of the occurrence each amino acid combination in the observed data to the expected value of occurrence of the pair. It is rounded off and used in the substitution matrix.</p>

<p>

<math display="inline" id="BLOSUM:0">
 <semantics>
  <mrow>
   <mrow>
    <mi>L</mi>
    <mi>o</mi>
    <mi>g</mi>
    <mi>O</mi>
    <mi>d</mi>
    <mi>d</mi>
    <mi>R</mi>
    <mi>a</mi>
    <mi>t</mi>
    <mi>i</mi>
    <mi>o</mi>
   </mrow>
   <mo>=</mo>
   <mrow>
    <mn>2</mn>
    <mrow>
     <msub>
      <mi>log</mi>
      <mn>2</mn>
     </msub>
     <mrow>
      <mo>(</mo>
      <mfrac>
       <mrow>
        <mi>P</mi>
        <mrow>
         <mo>(</mo>
         <mi>O</mi>
         <mo>)</mo>
        </mrow>
       </mrow>
       <mrow>
        <mi>P</mi>
        <mrow>
         <mo>(</mo>
         <mi>E</mi>
         <mo>)</mo>
        </mrow>
       </mrow>
      </mfrac>
      <mo>)</mo>
     </mrow>
    </mrow>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <eq></eq>
    <apply>
     <times></times>
     <ci>L</ci>
     <ci>o</ci>
     <ci>g</ci>
     <ci>O</ci>
     <ci>d</ci>
     <ci>d</ci>
     <ci>R</ci>
     <ci>a</ci>
     <ci>t</ci>
     <ci>i</ci>
     <ci>o</ci>
    </apply>
    <apply>
     <times></times>
     <cn type="integer">2</cn>
     <apply>
      <apply>
       <csymbol cd="ambiguous">subscript</csymbol>
       <log></log>
       <cn type="integer">2</cn>
      </apply>
      <apply>
       <divide></divide>
       <apply>
        <times></times>
        <ci>P</ci>
        <ci>O</ci>
       </apply>
       <apply>
        <times></times>
        <ci>P</ci>
        <ci>E</ci>
       </apply>
      </apply>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   LogOddRatio=2\log_{2}{\left(\frac{P\left(O\right)}{P\left(E\right)}\right)}
  </annotation>
 </semantics>
</math>

</p>

<p>In which 

<math display="inline" id="BLOSUM:1">
 <semantics>
  <mrow>
   <mi>P</mi>
   <mrow>
    <mo>(</mo>
    <mi>O</mi>
    <mo>)</mo>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <times></times>
    <ci>P</ci>
    <ci>O</ci>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   P\left(O\right)
  </annotation>
 </semantics>
</math>

 is the possibility of observed and 

<math display="inline" id="BLOSUM:2">
 <semantics>
  <mrow>
   <mi>P</mi>
   <mrow>
    <mo>(</mo>
    <mi>E</mi>
    <mo>)</mo>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <times></times>
    <ci>P</ci>
    <ci>E</ci>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   P\left(E\right)
  </annotation>
 </semantics>
</math>

 is the possibility of expected.</p>
<h3 id="blosum-matrices">BLOSUM Matrices</h3>

<p>The odds for relatedness are calculated from log odd ratio, which are then rounded off to get the substitution matrices BLOSUM matrices.</p>
<h3 id="score-of-the-blosum-matrices">Score of the BLOSUM matrices</h3>

<p>A scoring matrix or a table of values is required for evaluating the significance of a sequence alignment, such as describing the probability of a biologically meaningful amino-acid or nucleotide residue-pair occurring in an alignment. Typically, when two nucleotide sequences are being compared, all that is being scored is whether or not two bases are the same at one position. All matches and mismatches are respectively given the same score (typically +1 or +5 for matches, and -1 or -4 for mismatches).<a class="footnoteRef" href="#fn16" id="fnref16"><sup>16</sup></a> But it is different for proteins. Substitution matrices for amino acids are more complicated and implicitly take into account everything that might affect the frequency with which any amino acid is substituted for another. The objective is to provide a relatively heavy penalty for aligning two residues together if they have a low probability of being homologous (correctly aligned by evolutionary descent). Two major forces drive the amino-acid substitution rates away from uniformity: substitutions occur with the different frequencies, and lessen functionally tolerated than others. Thus, substitutions are selected against.<a class="footnoteRef" href="#fn17" id="fnref17"><sup>17</sup></a></p>

<p>Commonly used substitution matrices include the blocks substitution (BLOSUM) <a class="footnoteRef" href="#fn18" id="fnref18"><sup>18</sup></a> and point accepted mutation (PAM) <a class="footnoteRef" href="#fn19" id="fnref19"><sup>19</sup></a><a class="footnoteRef" href="#fn20" id="fnref20"><sup>20</sup></a> matrices. Both are based on taking sets of high-confidence alignments of many homologous proteins and assessing the frequencies of all substitutions, but they are computed using different methods.<a class="footnoteRef" href="#fn21" id="fnref21"><sup>21</sup></a></p>

<p>Scores within a BLOSUM are log-odds scores that measure, in an alignment, the logarithm for the ratio of the likelihood of two amino acids appearing with a biological sense and the likelihood of the same amino acids appearing by chance. The matrices are based on the minimum percentage identity of the aligned protein sequence used in calculating them.<a class="footnoteRef" href="#fn22" id="fnref22"><sup>22</sup></a> Every possible identity or substitution is assigned a score based on its observed frequencies in the alignment of related proteins.<a class="footnoteRef" href="#fn23" id="fnref23"><sup>23</sup></a> A positive score is given to the more likely substitutions while a negative score is given to the less likely substitutions.</p>

<p>To calculate a BLOSUM matrix, the following equation is used:</p>

<p>

<math display="block" id="BLOSUM:3">
 <semantics>
  <mrow>
   <msub>
    <mi>S</mi>
    <mrow>
     <mi>i</mi>
     <mi>j</mi>
    </mrow>
   </msub>
   <mo>=</mo>
   <mrow>
    <mrow>
     <mo>(</mo>
     <mfrac>
      <mn>1</mn>
      <mi>λ</mi>
     </mfrac>
     <mo>)</mo>
    </mrow>
    <mrow>
     <mi>log</mi>
     <mrow>
      <mo>(</mo>
      <mfrac>
       <msub>
        <mi>p</mi>
        <mrow>
         <mi>i</mi>
         <mi>j</mi>
        </mrow>
       </msub>
       <mrow>
        <msub>
         <mi>q</mi>
         <mi>i</mi>
        </msub>
        <mo>*</mo>
        <msub>
         <mi>q</mi>
         <mi>j</mi>
        </msub>
       </mrow>
      </mfrac>
      <mo>)</mo>
     </mrow>
    </mrow>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <eq></eq>
    <apply>
     <csymbol cd="ambiguous">subscript</csymbol>
     <ci>S</ci>
     <apply>
      <times></times>
      <ci>i</ci>
      <ci>j</ci>
     </apply>
    </apply>
    <apply>
     <times></times>
     <apply>
      <divide></divide>
      <cn type="integer">1</cn>
      <ci>λ</ci>
     </apply>
     <apply>
      <log></log>
      <apply>
       <divide></divide>
       <apply>
        <csymbol cd="ambiguous">subscript</csymbol>
        <ci>p</ci>
        <apply>
         <times></times>
         <ci>i</ci>
         <ci>j</ci>
        </apply>
       </apply>
       <apply>
        <times></times>
        <apply>
         <csymbol cd="ambiguous">subscript</csymbol>
         <ci>q</ci>
         <ci>i</ci>
        </apply>
        <apply>
         <csymbol cd="ambiguous">subscript</csymbol>
         <ci>q</ci>
         <ci>j</ci>
        </apply>
       </apply>
      </apply>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   S_{ij}=\left(\frac{1}{\lambda}\right)\log{\left(\frac{p_{ij}}{q_{i}*q_{j}}%
\right)}
  </annotation>
 </semantics>
</math>

</p>

<p>Here, 

<math display="inline" id="BLOSUM:4">
 <semantics>
  <msub>
   <mi>p</mi>
   <mrow>
    <mi>i</mi>
    <mi>j</mi>
   </mrow>
  </msub>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <csymbol cd="ambiguous">subscript</csymbol>
    <ci>p</ci>
    <apply>
     <times></times>
     <ci>i</ci>
     <ci>j</ci>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   p_{ij}
  </annotation>
 </semantics>
</math>

 is the probability of two amino acids 

<math display="inline" id="BLOSUM:5">
 <semantics>
  <mi>i</mi>
  <annotation-xml encoding="MathML-Content">
   <ci>i</ci>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   i
  </annotation>
 </semantics>
</math>

 and 

<math display="inline" id="BLOSUM:6">
 <semantics>
  <mi>j</mi>
  <annotation-xml encoding="MathML-Content">
   <ci>j</ci>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   j
  </annotation>
 </semantics>
</math>

 replacing each other in a homologous sequence, and 

<math display="inline" id="BLOSUM:7">
 <semantics>
  <msub>
   <mi>q</mi>
   <mi>i</mi>
  </msub>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <csymbol cd="ambiguous">subscript</csymbol>
    <ci>q</ci>
    <ci>i</ci>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   q_{i}
  </annotation>
 </semantics>
</math>

 and 

<math display="inline" id="BLOSUM:8">
 <semantics>
  <msub>
   <mi>q</mi>
   <mi>j</mi>
  </msub>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <csymbol cd="ambiguous">subscript</csymbol>
    <ci>q</ci>
    <ci>j</ci>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   q_{j}
  </annotation>
 </semantics>
</math>

 are the background probabilities of finding the amino acids 

<math display="inline" id="BLOSUM:9">
 <semantics>
  <mi>i</mi>
  <annotation-xml encoding="MathML-Content">
   <ci>i</ci>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   i
  </annotation>
 </semantics>
</math>

 and 

<math display="inline" id="BLOSUM:10">
 <semantics>
  <mi>j</mi>
  <annotation-xml encoding="MathML-Content">
   <ci>j</ci>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   j
  </annotation>
 </semantics>
</math>

 in any protein sequence. The factor 

<math display="inline" id="BLOSUM:11">
 <semantics>
  <mi>λ</mi>
  <annotation-xml encoding="MathML-Content">
   <ci>λ</ci>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \lambda
  </annotation>
 </semantics>
</math>

 is a scaling factor, set such that the matrix contains easily computable integer values.</p>
<h2 id="an-example---blosum62">An example - BLOSUM62</h2>

<p>BLOSUM62: midrange</p>

<p>BLOSUM80: more related proteins</p>

<p>BLOSUM45: distantly related proteins</p>

<p>An article in <a href="Nature_Biotechnology" title="wikilink">Nature Biotechnology</a><a class="footnoteRef" href="#fn24" id="fnref24"><sup>24</sup></a> revealed that the BLOSUM62 used for so many years as a standard is not exactly accurate according to the algorithm described by Henikoff and Henikoff.<a class="footnoteRef" href="#fn25" id="fnref25"><sup>25</sup></a> Surprisingly, the miscalculated BLOSUM62 improves search performance.<a class="footnoteRef" href="#fn26" id="fnref26"><sup>26</sup></a></p>

<p>The BLOSUM62 matrix with the amino acids in the table grouped according to the chemistry of the side chain, as in (a). Each value in the matrix is calculated by dividing the frequency of occurrence of the amino acid pair in the BLOCKS database, clustered at the 62% level, divided by the probability that the same two amino acids might align by chance. The ratio is then converted to a logarithm and expressed as a log odds score, as for PAM. BLOSUM matrices are usually scaled in half-bit units. A score of zero indicates that the frequency with which a given two amino acids were found aligned in the database was as expected by chance, while a positive score indicates that the alignment was found more often than by chance, and negative score indicates that the alignment was found less often than by chance.</p>
<h2 id="some-uses-in-bioinformatics">Some uses in bioinformatics</h2>
<h3 id="research-applications">Research applications</h3>

<p>BLOSUM scores was used to predict and understand the surface gene variants among hepatitis B virus carriers<a class="footnoteRef" href="#fn27" id="fnref27"><sup>27</sup></a> and T-cell epitopes.<a class="footnoteRef" href="#fn28" id="fnref28"><sup>28</sup></a></p>
<h4 id="surface-gene-variants-among-hepatitis-b-virus-carriers">Surface gene variants among hepatitis B virus carriers</h4>

<p>DNA sequences of HBsAg were obtained from 180 patients, in which 51 were chronic HBV carrier and 129 newly diagnosed patients, and compared with consensus sequences built with 168 HBV sequences imported from GenBank. Literature review and BLOSUM scores were used to define potentially altered antigenicity.<a class="footnoteRef" href="#fn29" id="fnref29"><sup>29</sup></a></p>
<h4 id="reliable-prediction-of-t-cell-epitopes">Reliable prediction of T-cell epitopes</h4>

<p>A novel input representation has been developed consisting of a combination of sparse encoding, Blosum encoding, and input derived from hidden Markov models. this method predicts T-cell epitopes for the genome of hepatitis C virus and discuss possible applications of the prediction method to guide the process of rational vaccine design.<a class="footnoteRef" href="#fn30" id="fnref30"><sup>30</sup></a></p>
<h3 id="use-in-blast">Use in BLAST</h3>

<p>BLOSUM matrices are also used as a scoring matrix when comparing DNA sequences or protein sequences to judge the quality of the alignment. This form of scoring system is utilized by a wide range of alignment software including <a class="uri" href="BLAST" title="wikilink">BLAST</a>.<a class="footnoteRef" href="#fn31" id="fnref31"><sup>31</sup></a></p>
<h4 id="comparing-pam-and-blosum">Comparing PAM and BLOSUM</h4>

<p>In addition to BLOSUM matrices, a previously developed scoring matrix can be used. This is known as a <a href="Point_accepted_mutation" title="wikilink">PAM</a>. The two result in the same scoring outcome, but use differing methodologies. BLOSUM directly look at mutations in motifs of related sequences while PAM's <a class="uri" href="extrapolate" title="wikilink">extrapolate</a> evolutionary information based on closely related sequences.<a class="footnoteRef" href="#fn32" id="fnref32"><sup>32</sup></a></p>

<p>Since both PAM and BLOSUM are different methods for showing the same scoring information, the two can be compared but due to the very different method of obtaining this score, a PAM100 does not equal a BLOSUM100.<a class="footnoteRef" href="#fn33" id="fnref33"><sup>33</sup></a></p>
<table>
<thead>
<tr class="header">
<th style="text-align: left;">
<p>PAM</p></th>
<th style="text-align: left;">
<p>BLOSUM</p></th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;">
<p>PAM100</p></td>
<td style="text-align: left;">
<p>BLOSUM90</p></td>
</tr>
<tr class="even">
<td style="text-align: left;">
<p>PAM120</p></td>
<td style="text-align: left;">
<p>BLOSUM80</p></td>
</tr>
<tr class="odd">
<td style="text-align: left;">
<p>PAM160</p></td>
<td style="text-align: left;">
<p>BLOSUM60</p></td>
</tr>
<tr class="even">
<td style="text-align: left;">
<p>PAM200</p></td>
<td style="text-align: left;">
<p>BLOSUM52</p></td>
</tr>
<tr class="odd">
<td style="text-align: left;">
<p>PAM250</p></td>
<td style="text-align: left;">
<p>BLOSUM45</p></td>
</tr>
</tbody>
</table>
<h5 id="the-relationship-between-pam-and-blosum">The relationship between PAM and BLOSUM</h5>
<table>
<thead>
<tr class="header">
<th style="text-align: left;">
<p>PAM</p></th>
<th style="text-align: left;">
<p>BLOSUM</p></th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;">
<p>To compare the closely related sequences, PAM matrices with lower numbers are created.</p></td>
<td style="text-align: left;">
<p>To compare the closely related sequences, BLOSUM matrices with higher numbers are created.</p></td>
</tr>
<tr class="even">
<td style="text-align: left;">
<p>To compare the distantly related proteins, PAM matrices with high numbers are created.</p></td>
<td style="text-align: left;">
<p>To compare the distantly related proteins, BLOSUM matrices with low numbers are created.</p></td>
</tr>
</tbody>
</table>
<h5 id="the-differences-between-pam-and-blosum">The differences between PAM and BLOSUM</h5>
<table>
<thead>
<tr class="header">
<th style="text-align: left;">
<p>PAM</p></th>
<th style="text-align: left;">
<p>BLOSUM</p></th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;">
<p>Based on global alignments of closely related proteins.</p></td>
<td style="text-align: left;">
<p>based on local alignments.</p></td>
</tr>
<tr class="even">
<td style="text-align: left;">
<p>PAM1 is the matrix calculated from comparisons of sequences with no more than 1% divergence.</p></td>
<td style="text-align: left;">
<p>BLOSUM 62 is a matrix calculated from comparisons of sequences with no more than 62% identical.</p></td>
</tr>
<tr class="odd">
<td style="text-align: left;">
<p>Other PAM matrices are extrapolated from PAM1.</p></td>
<td style="text-align: left;">
<p>Based on observed alignments; they are not extrapolated from comparisons of closely related proteins.</p></td>
</tr>
<tr class="even">
<td style="text-align: left;">
<p>Higher numbers in matrices naming scheme denote larger evolutionary distance.</p></td>
<td style="text-align: left;">
<p>Larger numbers in matrices naming scheme denote higher sequence similarity and therefore smaller evolutionary distance.<a class="footnoteRef" href="#fn34" id="fnref34"><sup>34</sup></a></p></td>
</tr>
</tbody>
</table>
<h2 id="see-also">See also</h2>
<ul>
<li><a href="Sequence_alignment" title="wikilink">Sequence alignment</a></li>
<li><a href="Point_accepted_mutation" title="wikilink">Point accepted mutation</a></li>
</ul>
<h2 id="references">References</h2>
<h2 id="external-links">External links</h2>
<ul>
<li></li>
<li><a href="http://blocks.fhcrc.org/">BLOCKS WWW server</a></li>
<li><a href="http://www.ncbi.nlm.nih.gov/BLAST/tutorial/Altschul-1.html">Scoring systems for BLAST at NCBI</a></li>
<li><a href="ftp://ftp.ncbi.nih.gov/blast/matrices/">Data files of BLOSUM on the NCBI FTP server</a>.</li>
<li><a href="http://ahmetrasit.com/blosum/">Interactive BLOSUM Network Visualization</a></li>
</ul>

<p><a class="uri" href="ko:블로섬" title="wikilink">ko:블로섬</a>"</p>

<p><a class="uri" href="Category:Genetics" title="wikilink">Category:Genetics</a> <a class="uri" href="Category:Bioinformatics" title="wikilink">Category:Bioinformatics</a> <a href="Category:Biochemistry_methods" title="wikilink">Category:Biochemistry methods</a> <a href="Category:Computational_phylogenetics" title="wikilink">Category:Computational phylogenetics</a> <a class="uri" href="Category:Matrices" title="wikilink">Category:Matrices</a></p>
<section class="footnotes">
<hr/>
<ol>
<li id="fn1"><a href="#fnref1">↩</a></li>
<li id="fn2"></li>
<li id="fn3"><a href="#fnref3">↩</a></li>
<li id="fn4"><a href="#fnref4">↩</a></li>
<li id="fn5"></li>
<li id="fn6"><a href="#fnref6">↩</a></li>
<li id="fn7"><a href="#fnref7">↩</a></li>
<li id="fn8"></li>
<li id="fn9"><a href="#fnref9">↩</a></li>
<li id="fn10"></li>
<li id="fn11"></li>
<li id="fn12"></li>
<li id="fn13"><a href="#fnref13">↩</a></li>
<li id="fn14"></li>
<li id="fn15"><a href="#fnref15">↩</a></li>
<li id="fn16"><a href="#fnref16">↩</a></li>
<li id="fn17"></li>
<li id="fn18"></li>
<li id="fn19"><a href="#fnref19">↩</a></li>
<li id="fn20"><a href="#fnref20">↩</a></li>
<li id="fn21"></li>
<li id="fn22">page 673<a href="#fnref22">↩</a></li>
<li id="fn23"><a href="http://www.ncbi.nlm.nih.gov/Education/BLASTinfo/Scoring2.html">NIH "Scoring Systems"</a><a href="#fnref23">↩</a></li>
<li id="fn24"><a href="#fnref24">↩</a></li>
<li id="fn25"></li>
<li id="fn26"></li>
<li id="fn27"></li>
<li id="fn28"></li>
<li id="fn29"><a href="#fnref29">↩</a></li>
<li id="fn30"><a href="#fnref30">↩</a></li>
<li id="fn31"><a href="#fnref31">↩</a></li>
<li id="fn32"></li>
<li id="fn33"><a href="#fnref33">↩</a></li>
<li id="fn34"><a href="#fnref34">↩</a></li>
</ol>
</section>
</body>
</html>
