<html lang="en">
<head>
<meta charset="utf-8"/>
<title offset="358">Coalescent theory</title>
<script src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_SVG.js" type="text/javascript">
</script>
</head>
<body>
<h1>Coalescent theory</h1>
<hr>'''Coalescent theory''' is a retrospective stochastic model of [[population genetics]] that relates genetic diversity in a sample to demographic history of the population from which it was taken. That is, it is a model of the effect of genetic drift, viewed backwards in time, on the [[genealogy]] of [[Antecedent (genealogy)|antecedents]].{{ref|Rosalind}} It comprises a probabilistic assessment of variation in time to common ancestry of alleles in a relatively small sample of individuals, from a much larger population.  This includes considerat
<p>ion of all pathways of inheritance through which sampled copies of a <a href="homology_(biology)" title="wikilink">homologous</a> DNA element are traced back to a single ancestral copy, known as the <a href="most_recent_common_ancestor" title="wikilink">most recent common ancestor</a> (MRCA; sometimes also termed the <em>coancestor</em> to emphasize the coalescent relationship). The inheritance relationships among alleles are typically represented as a <em>gene genealogy</em>, or <em>gene tree</em>, similar in form to a <a href="phylogenetic_tree" title="wikilink">phylogenetic tree</a>. The probabilistic expectation of this gene genealogy is also known as the <em>coalescent.</em> Understanding the statistical properties of the coalescent under different assumptions forms the basis of coalescent theory. Because of <a href="Genetic_recombination" title="wikilink">recombination</a>, different gene loci follow different pathways of ancestry, resulting in different gene genealogies. The coalescent is also relevant to <a class="uri" href="phylogenetics" title="wikilink">phylogenetics</a>, as <a href="incomplete_lineage_sorting" title="wikilink">incomplete lineage sorting</a> between speciation events results in conflict among gene-loci in phylogenetic relationships inferred among species.</p>

<p>The mathematical theory of the coalescent was originally developed in the early 1980s by <a href="John_Kingman" title="wikilink">John Kingman</a>. In the simplest case, coalescent theory assumes no <a href="genetic_recombination" title="wikilink">recombination</a>, no <a href="natural_selection" title="wikilink">natural selection</a>, and no <a href="gene_flow" title="wikilink">gene flow</a> or population structure. The gene genealogy is independent of the mutational process, such that changes in the DNA sequence do not affect inheritance and can be considered separately (even if all gene copies are identical in sequence they are not equally related in the gene tree). Under this model, the expected time between successive <em>coalescence</em> events, by which two gene copies arise from a single ancestral copy, increases almost <a href="exponential_growth" title="wikilink">exponentially</a> back in time (with wide <a class="uri" href="variance" title="wikilink">variance</a>). Advances in coalescent theory include recombination, selection, and virtually any arbitrarily complex evolutionary or demographic model in population genetic analysis.</p>
<h2 id="theory">Theory</h2>
<h3 id="time-to-coalescence">Time to coalescence</h3>

<p>Consider a single gene locus sampled from two <a class="uri" href="haploid" title="wikilink">haploid</a> individuals in a population. The ancestry of this sample is traced backwards in time to the point where these two lineages <em>coalesce</em> in their MRCA. Coalescent theory seeks to estimate the expectation of this time period and its variance.</p>

<p>The probability that two <a href="Lineage_(evolution)" title="wikilink">lineages</a> coalesce in the immediately preceding generation is the probability that they share a parental DNA sequence. In a population with a constant <a href="effective_population_size" title="wikilink">effective population size</a> with 2<em>N<sub>e</sub></em> copies of each locus, there are 2<em>N<sub>e</sub></em> "potential parents" in the previous generation. Under a random mating model, the probability that two alleles originate from the same parental copy is thus 1/(2<em>N<sub>e</sub></em>) and, correspondingly, the probability that they do <em>not</em> coalesce is 1 − 1/(2<em>N<sub>e</sub></em>).</p>

<p>At each successive preceding generation, the probability of coalescence is <a href="geometric_distribution" title="wikilink">geometrically distributed</a> — that is, it is the probability of <em>non</em>coalescence at the <em>t</em> − 1 preceding generations multiplied by the probability of coalescence at the generation of interest:</p>

<p>

<math display="block" id="Coalescent_theory:0">
 <semantics>
  <mrow>
   <mrow>
    <mrow>
     <msub>
      <mi>P</mi>
      <mi>c</mi>
     </msub>
     <mrow>
      <mo stretchy="false">(</mo>
      <mi>t</mi>
      <mo stretchy="false">)</mo>
     </mrow>
    </mrow>
    <mo>=</mo>
    <mrow>
     <msup>
      <mrow>
       <mo>(</mo>
       <mrow>
        <mn>1</mn>
        <mo>-</mo>
        <mfrac>
         <mn>1</mn>
         <mrow>
          <mn>2</mn>
          <msub>
           <mi>N</mi>
           <mi>e</mi>
          </msub>
         </mrow>
        </mfrac>
       </mrow>
       <mo>)</mo>
      </mrow>
      <mrow>
       <mi>t</mi>
       <mo>-</mo>
       <mn>1</mn>
      </mrow>
     </msup>
     <mrow>
      <mo>(</mo>
      <mfrac>
       <mn>1</mn>
       <mrow>
        <mn>2</mn>
        <msub>
         <mi>N</mi>
         <mi>e</mi>
        </msub>
       </mrow>
      </mfrac>
      <mo>)</mo>
     </mrow>
    </mrow>
   </mrow>
   <mo>.</mo>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <eq></eq>
    <apply>
     <times></times>
     <apply>
      <csymbol cd="ambiguous">subscript</csymbol>
      <ci>P</ci>
      <ci>c</ci>
     </apply>
     <ci>t</ci>
    </apply>
    <apply>
     <times></times>
     <apply>
      <csymbol cd="ambiguous">superscript</csymbol>
      <apply>
       <minus></minus>
       <cn type="integer">1</cn>
       <apply>
        <divide></divide>
        <cn type="integer">1</cn>
        <apply>
         <times></times>
         <cn type="integer">2</cn>
         <apply>
          <csymbol cd="ambiguous">subscript</csymbol>
          <ci>N</ci>
          <ci>e</ci>
         </apply>
        </apply>
       </apply>
      </apply>
      <apply>
       <minus></minus>
       <ci>t</ci>
       <cn type="integer">1</cn>
      </apply>
     </apply>
     <apply>
      <divide></divide>
      <cn type="integer">1</cn>
      <apply>
       <times></times>
       <cn type="integer">2</cn>
       <apply>
        <csymbol cd="ambiguous">subscript</csymbol>
        <ci>N</ci>
        <ci>e</ci>
       </apply>
      </apply>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   P_{c}(t)=\left(1-\frac{1}{2N_{e}}\right)^{t-1}\left(\frac{1}{2N_{e}}\right).
  </annotation>
 </semantics>
</math>

</p>

<p>For sufficiently large values of <em>N<sub>e</sub></em>, this distribution is well approximated by the continuously defined <a href="exponential_distribution" title="wikilink">exponential distribution</a></p>

<p>

<math display="block" id="Coalescent_theory:1">
 <semantics>
  <mrow>
   <mrow>
    <mrow>
     <msub>
      <mi>P</mi>
      <mi>c</mi>
     </msub>
     <mrow>
      <mo stretchy="false">(</mo>
      <mi>t</mi>
      <mo stretchy="false">)</mo>
     </mrow>
    </mrow>
    <mo>=</mo>
    <mrow>
     <mfrac>
      <mn>1</mn>
      <mrow>
       <mn>2</mn>
       <msub>
        <mi>N</mi>
        <mi>e</mi>
       </msub>
      </mrow>
     </mfrac>
     <msup>
      <mi>e</mi>
      <mrow>
       <mo>-</mo>
       <mfrac>
        <mrow>
         <mi>t</mi>
         <mo>-</mo>
         <mn>1</mn>
        </mrow>
        <mrow>
         <mn>2</mn>
         <msub>
          <mi>N</mi>
          <mi>e</mi>
         </msub>
        </mrow>
       </mfrac>
      </mrow>
     </msup>
    </mrow>
   </mrow>
   <mo>.</mo>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <eq></eq>
    <apply>
     <times></times>
     <apply>
      <csymbol cd="ambiguous">subscript</csymbol>
      <ci>P</ci>
      <ci>c</ci>
     </apply>
     <ci>t</ci>
    </apply>
    <apply>
     <times></times>
     <apply>
      <divide></divide>
      <cn type="integer">1</cn>
      <apply>
       <times></times>
       <cn type="integer">2</cn>
       <apply>
        <csymbol cd="ambiguous">subscript</csymbol>
        <ci>N</ci>
        <ci>e</ci>
       </apply>
      </apply>
     </apply>
     <apply>
      <csymbol cd="ambiguous">superscript</csymbol>
      <ci>e</ci>
      <apply>
       <minus></minus>
       <apply>
        <divide></divide>
        <apply>
         <minus></minus>
         <ci>t</ci>
         <cn type="integer">1</cn>
        </apply>
        <apply>
         <times></times>
         <cn type="integer">2</cn>
         <apply>
          <csymbol cd="ambiguous">subscript</csymbol>
          <ci>N</ci>
          <ci>e</ci>
         </apply>
        </apply>
       </apply>
      </apply>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   P_{c}(t)=\frac{1}{2N_{e}}e^{-\frac{t-1}{2N_{e}}}.
  </annotation>
 </semantics>
</math>

</p>

<p>This is mathematically convenient as the standard exponential distribution has both the <a href="expected_value" title="wikilink">expected value</a> and the <a href="standard_deviation" title="wikilink">standard deviation</a> equal to 2<em>N<sub>e</sub></em>; therefore, although the <em>expected</em> time to coalescence is 2<em>N<sub>e</sub></em>, actual coalescence times have a wide range of variation. Note that coalescent time is the number of preceding generations where the coalescence took place and not calendar time though an estimation of the latter can be made multiplying 2<em>N<sub>e</sub></em> with the average time between generations. The above calculations apply equally to a diploid population of effective size <em>N<sub>e</sub></em> (in other words, for a non-recombining segment of DNA, each chromosome can be treated as equivalent to an independent haploid individual; in the absence of inbreeding, sister chromosomes in a single individual are no more closely related than two chromosomes randomly sampled from the population). Some effectively haploid DNA elements, such as <a href="mitochondrial_DNA" title="wikilink">mitochondrial DNA</a>, however, are only carried by one sex and therefore have one quarter the effective size of the equivalent diploid population (<em>N<sub>e</sub>/2</em>)</p>
<h3 id="neutral-variation">Neutral variation</h3>

<p>Coalescent theory can also be used to model the amount of variation in <a class="uri" href="DNA" title="wikilink">DNA</a> sequences expected from genetic drift and mutation. This value is termed the mean <a href="heterozygote" title="wikilink">heterozygosity</a>, represented as 

<math display="inline" id="Coalescent_theory:2">
 <semantics>
  <mover accent="true">
   <mi>H</mi>
   <mo stretchy="false">¯</mo>
  </mover>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <ci>normal-¯</ci>
    <ci>H</ci>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \bar{H}
  </annotation>
 </semantics>
</math>

. Mean heterozygosity is calculated as the probability of a mutation occurring at a given generation divided by the probability of any "event" at that generation (either a mutation or a coalescence). The probability that the event is a mutation is the probability of a mutation in either of the two lineages

<math display="block" id="Coalescent_theory:3">
 <semantics>
  <mrow>
   <mn>2</mn>
   <mi>μ</mi>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <times></times>
    <cn type="integer">2</cn>
    <ci>μ</ci>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   2\mu
  </annotation>
 </semantics>
</math>

. Thus the mean heterozygosity is equal to</p>

<p>

<math display="inline" id="Coalescent_theory:4">
 <semantics>
  <mover accent="true">
   <mi>H</mi>
   <mo stretchy="false">¯</mo>
  </mover>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <ci>normal-¯</ci>
    <ci>H</ci>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \displaystyle\bar{H}
  </annotation>
 </semantics>
</math>


</p>

<p>For 

<math display="inline" id="Coalescent_theory:5">
 <semantics>
  <mrow>
   <mrow>
    <mn>4</mn>
    <msub>
     <mi>N</mi>
     <mi>e</mi>
    </msub>
    <mi>μ</mi>
   </mrow>
   <mo>≫</mo>
   <mn>1</mn>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <csymbol cd="latexml">much-greater-than</csymbol>
    <apply>
     <times></times>
     <cn type="integer">4</cn>
     <apply>
      <csymbol cd="ambiguous">subscript</csymbol>
      <ci>N</ci>
      <ci>e</ci>
     </apply>
     <ci>μ</ci>
    </apply>
    <cn type="integer">1</cn>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   4N_{e}\mu\gg 1
  </annotation>
 </semantics>
</math>

, the vast majority of allele pairs have at least one difference in <a class="uri" href="nucleotide" title="wikilink">nucleotide</a> sequence.</p>
<h2 id="graphical-representation">Graphical representation</h2>

<p>Coalescents can be visualised using <a href="dendrogram" title="wikilink">dendrograms</a> which show the relationship of branches of the population to each other. The point where two branches meet indicates a coalescent event.</p>
<h2 id="applications">Applications</h2>
<h3 id="disease-gene-mapping">Disease gene mapping</h3>

<p>The utility of coalescent theory in the mapping of disease is slowly gaining more appreciation; although the application of the theory is still in its infancy, there are a number of researchers who are actively developing algorithms for the analysis of human genetic data that utilise coalescent theory.</p>
<h3 id="the-genomic-distribution-of-heterozygosity">The genomic distribution of heterozygosity</h3>

<p>The human <a href="single-nucleotide_polymorphism" title="wikilink">single-nucleotide polymorphism</a> (SNP) map has revealed large regional variations in heterozygosity, more so than can be explained on the basis of (<a href="Poisson_distribution" title="wikilink">Poisson-distributed</a>) random chance. In part, these variations could be explained on the basis of assessment methods, the availability of genomic sequences, and possibly the standard coalescent population genetic model. Population genetic influences could have a major influence on this variation: some loci presumably would have comparatively recent common ancestors, others might have much older genealogies, and so the regional accumulation of SNPs over time could be quite different. The local density of SNPs along chromosomes appears to cluster in accordance with a <a href="Taylor's_law" title="wikilink">variance to mean power law</a> and to obey the <a href="Tweedie_distribution" title="wikilink">Tweedie compound Poisson distribution</a>. In this model the regional variations in the SNP map would be explained by the accumulation of multiple small genomic segments through recombination, where the mean number of SNPs per segment would be <a href="gamma_distribution" title="wikilink">gamma distributed</a> in proportion to a gamma distributed time to the most recent common ancestor for each segment.</p>
<h2 id="history">History</h2>

<p>Coalescent theory is a natural extension of the more classical <a href="population_genetics" title="wikilink">population genetics</a> concept of <a href="neutral_evolution" title="wikilink">neutral evolution</a> and is an approximation to the <a href="Genetic_drift#Wright–Fisher_model" title="wikilink">Fisher–Wright</a> (or Wright–Fisher) model for large populations. It was discovered independently by several researchers in the 1980s.</p>
<h2 id="software">Software</h2>

<p>A large body of software exists for both simulating data sets under the coalescent process as well as inferring parameters such as population size and migration rates from genetic data.</p>
<ul>
<li><a href="http://beast.bio.ed.ac.uk/">BEAST</a> – <a href="Bayesian_inference" title="wikilink">Bayesian</a> <a href="Markov_chain_Monte_Carlo" title="wikilink">MCMC</a> inference package with a wide range of coalescent models including the use of temporally sampled sequences.</li>
<li><a href="http://www.coaltree.net/">COAL</a> – Program for computing gene tree probabilities and simulating gene trees in species trees under the coalescent model .</li>
<li><a href="http://www.daimi.au.dk/~mailund/CoaSim/index.html">CoaSim</a> – software for simulating genetic data under the coalescent model.</li>
<li><a href="http://www1.montpellier.inra.fr/CBGP/diyabc/">DIYABC</a> - A user-friendly approach to <a href="Approximate_Bayesian_Computation" title="wikilink">ABC</a> for inference on population history using molecular markers </li>
<li><a href="http://www.daimi.au.dk/~mailund/GeneRecon/">GeneRecon</a> – software for the fine-scale mapping of <a href="linkage_disequilibrium" title="wikilink">linkage disequilibrium</a> mapping of disease genes using coalescent theory based on an <a href="Bayesian_inference" title="wikilink">Bayesian</a> <a href="Markov_chain_Monte_Carlo" title="wikilink">MCMC</a> framework.</li>
<li><a href="http://www.stats.ox.ac.uk/%7Egriff/software.html">genetree</a> software for estimation of <a href="population_genetics" title="wikilink">population genetics</a> parameters using coalescent theory and simulation (the <a href="R_(programming_language)" title="wikilink">R</a> package popgen). See also <a href="http://mathgen.stats.ox.ac.uk/software.html">Oxford Mathematical Genetics and Bioinformatics Group</a></li>
<li><a href="http://www.sph.umich.edu/csg/liang/genome/">GENOME</a> – rapid coalescent-based whole-genome simulation</li>
<li><a href="http://raphael.leblois.free.fr/#softwares">IBDSim</a> – A computer package for the simulation of genotypic data under general isolation by distance models .</li>
<li><a href="http://genfaculty.rutgers.edu/hey/software#IMa2">IMa</a> – IMa implements the same Isolation with Migration model, but does so using a new method that provides estimates of the joint posterior probability density of the model parameters. IMa also allows log likelihood ratio tests of nested demographic models. IMa is based on a method described in Hey and Nielsen (2007 PNAS 104:2785–2790). IMa is faster and better than IM (i.e. by virtue of providing access to the joint posterior density function), and it can be used for most (but not all) of the situations and options that IM can be used for.</li>
<li><a href="http://evolution.gs.washington.edu/lamarc">Lamarc</a> – software for estimation of rates of population growth, migration, and recombination.</li>
<li><a href="http://kimura.univ-montp2.fr/~rousset/Migraine.htm">Migraine</a> – A program which implements coalescent algorithms for a maximum likelihood analysis (using <a href="Importance_sampling" title="wikilink">Importance Sampling</a> algorithms) of genetic data with a focus on spatially structured populations .</li>
<li><a href="http://popgen.csit.fsu.edu/">Migrate</a> – <a href="Maximum_likelihood" title="wikilink">Maximum likelihood</a> and <a href="Bayesian_inference" title="wikilink">Bayesian inference</a> of migration rates under the <em>n</em>-coalescent. The inference is implemented using <a href="Markov_chain_Monte_Carlo" title="wikilink">MCMC</a></li>
<li><a href="https://code.google.com/p/macs/">MaCS</a> – Markovian Coalescent Simulator - Simulates genealogies spatially across chromosomes as a Markovian process. Similar to the SMC algorithm of McVean and Cardin, and supports all demographic scenarios found in Hudson's ms.</li>
<li><a href="http://home.uchicago.edu/~rhudson1/source/mksamples.html">ms &amp; msHOT</a> – Richard Hudson's original program for generating samples under neutral models  and an extension which allows <a href="recombination_hotspots" title="wikilink">recombination hotspots</a>.</li>
<li><a href="http://www.mabs.at/ewing/msms/">msms</a> – An extended version of ms that includes selective sweeps.</li>
<li><a href="http://darwin.uvigo.es/">Recodon and NetRecodon</a> – software to simulate coding sequences with inter/intracodon recombination, migration, growth rate and longitudinal sampling  .</li>
<li><a href="http://walnut.usc.edu/~magnus/software/">SARG</a> – Structure Ancestral Recombination Graph by Magnus Nordborg</li>
<li><a href="http://cmpg.unibe.ch/software/simcoal2/">simcoal2</a> – software to simulate genetic data under the coalescent model with complex demography and recombination</li>
<li><a href="http://staff.washington.edu/brendano/treesimj">TreesimJ</a> Forward simulation software allowing sampling of genealogies and data sets under diverse selective and demographic models.</li>
</ul>
<h2 id="references-and-notes">References and notes</h2>
<h3 id="articles">Articles</h3>
<ul>
<li>

<p>Arenas, M. and Posada, D. (2007) Recodon: Coalescent simulation of coding DNA sequences with recombination, migration and demography. <a href="http://www.biomedcentral.com/1471-2105/8/458"><em>BMC Bioinformatics</em> <strong>8</strong>: 458</a></p></li>
<li>

<p>Arenas, M. and Posada, D. (2010) Coalescent simulation of intracodon recombination. <a href="http://www.genetics.org/cgi/content/abstract/184/2/429"><em>Genetics</em> <em>184(2)</em>: 429–437</a></p></li>
<li>

<p>Browning, S.R. (2006) Multilocus association mapping using variable-length markov chains. [<a class="uri" href="http://www.pubmedcentral.nih.gov/picrender.fcgi?artid=1474089&amp;blobtype">http://www.pubmedcentral.nih.gov/picrender.fcgi?artid=1474089&amp;blobtype;</a>;=pdf <em>American Journal of Human Genetics</em> <strong>78</strong>:903–913]</p></li>
<li>

<p>Cornuet J.-M., Pudlo P., Veyssier J., Dehne-Garcia A., Gautier M., Leblois R., Marin J.-M., Estoup A. (2014) DIYABC v2.0: a software to make Approximate Bayesian Computation inferences about population history using Single Nucleotide Polymorphism, DNA sequence and microsatellite data. <a href="http://bioinformatics.oxfordjournals.org/content/30/8/1187"><em>Bioinformatics</em> <strong>'30</strong>': 1187–1189</a></p></li>
<li>

<p>Degnan, JH and LA Salter. 2005. Gene tree distributions under the coalescent process. Evolution 59(1): 24–37. <a href="http://www.coaltree.net/images/DegnanSalter.pdf">pdf from coaltree.net/</a></p></li>
<li>

<p>Donnelly, P., Tavaré, S. (1995) Coalescents and genealogical structure under neutrality. <em>Annual Review of Genetics</em> <strong>29</strong>:401–421</p></li>
<li>

<p>Ewing, G. and Hermisson J. (2010), MSMS: a coalescent simulation program including recombination, demographic structure and selection at a single locus, <a href="http://bioinformatics.oxfordjournals.org/content/26/16/2064.full"><em>Bioinformatics</em> <strong>26</strong>:15</a></p></li>
<li>

<p>Hellenthal, G., Stephens M. (2006) msHOT: modifying Hudson's ms simulator to incorporate crossover and gene conversion hotspots <a href="http://bioinformatics.oxfordjournals.org/cgi/content/abstract/btl622v1"><em>Bioinformatics</em> <strong>AOP</strong></a></p></li>
<li>

<p>Hudson RR (1983a) Testing the constant-rate neutral allele model with protein sequence data. <em>Evolution</em> <strong>37</strong>: 203–207 [<a class="uri" href="http://www.jstor.org/cgi-bin/jstor/printpage/00143820/di000260/00p01517/0.pdf?backcontext=table-of-contents&amp;dowhat">http://www.jstor.org/cgi-bin/jstor/printpage/00143820/di000260/00p01517/0.pdf?backcontext=table-of-contents&amp;dowhat;</a>;=Acrobat&amp;config;=jstor&amp;userID;=825fc2c5@uwa.edu.au/01cce4405d00501b61540&amp;0.pdf JSTOR copy]</p></li>
<li>

<p>Hudson RR (1983b) Properties of a neutral allele model with intragenic recombination. <em>Theoretical Population Biology</em> <strong>23</strong>:183–201.</p></li>
<li>

<p>Hudson RR (1991) <a href="http://home.uchicago.edu/~rhudson1/popgen356/OxfordSurveysEvolBiol7_1-44.pdf">Gene genealogies and the coalescent process.</a> <em>Oxford Surveys in Evolutionary Biology</em> <strong>7</strong>: 1–44</p></li>
<li>

<p>Hudson RR (2002) Generating samples under a Wright–Fisher neutral model. <a href="http://bioinformatics.oxfordjournals.org/cgi/content/abstract/18/2/337"><em>Bioinformatics</em> <strong>18</strong>:337–338</a></p></li>
<li>

<p>Kendal WS (2003) An exponential dispersion model for the distribution of human single nucleotide polymorphisms. <em>Mol Biol Evol</em> <strong>20</strong>: 579–590</p></li>
<li>Hein, J., Schierup, M., Wiuf C. (2004) <em>Gene Genealogies, Variation and Evolution: A Primer in Coalescent Theory</em> Oxford University Press ISBN 978-0-19-852996-5</li>
<li>

<p>Kaplan, N.L., Darden, T., Hudson, R.R. (1988) The coalescent process in models with selection. <em>Genetics</em> <strong>120</strong>:819–829</p></li>
<li>

<p>Kingman, J.F.C. (1982) On the Genealogy of Large Populations. <em>Journal of Applied Probability</em> <strong>19A</strong>:27–43 [<a class="uri" href="http://www.jstor.org/cgi-bin/jstor/printpage/00219002/sp050067/05x2498b/0?frame=noframe&amp;userID">http://www.jstor.org/cgi-bin/jstor/printpage/00219002/sp050067/05x2498b/0?frame=noframe&amp;userID;</a>;=825fc2c5@uwa.edu.au/01cce4405d00501b61540&amp;dpi;=3&amp;backcontext;=table-of-contents&amp;backurl;=/cgi-bin/jstor/listjournal/00219002/sp050067%3fframe%3dframe%26dpi%3d3%26userID%3d825fc2c5@uwa.edu.au/01cce4405d00501b61540%26config%3djstor&amp;config;=jstor JSTOR copy]</p></li>
<li>

<p>Kingman, J.F.C. (2000) Origins of the coalescent 1974–1982. <a href="http://www.genetics.org/cgi/content/full/156/4/1461"><em>Genetics</em> <strong>156</strong>:1461–1463</a></p></li>
<li>

<p>Leblois R., Estoup A. and Rousset F. (2009) IBDSim: a computer program to simulate genotypic data under isolation by distance <a href="http://raphael.leblois.free.fr/Papiers/LebloisEtAl.2009MolEcolRess_IBDSim.pdf"><em>Molecular Ecology Resources</em> <strong>9</strong>:107–109</a></p></li>
<li>

<p>Liang L., Zöllner S., Abecasis G.R. (2007) GENOME: a rapid coalescent-based whole genome simulator. <em>Bioinformatics</em> <a href="http://bioinformatics.oxfordjournals.org/cgi/content/abstract/23/12/1565"><strong>23</strong>: 1565–1567</a></p></li>
<li>

<p>Mailund, T., Schierup, M.H., Pedersen, C.N.S., Mechlenborg, P. J. M., Madsen, J.N., Schauser, L. (2005) CoaSim: A Flexible Environment for Simulating Genetic Data under Coalescent Models <a href="http://www.biomedcentral.com/1471-2105/6/252/abstract"><em>BMC Bioinformatics</em> <strong>6</strong>:252</a></p></li>
<li>

<p>Möhle, M., Sagitov, S. (2001) A classification of coalescent processes for haploid exchangeable population models <em>The Annals of Probability</em> <strong>29</strong>:1547–1562</p></li>
<li>

<p>Morris, A. P., Whittaker, J. C., Balding, D. J. (2002) Fine-scale mapping of disease loci via shattered coalescent modeling of genealogies [<a class="uri" href="http://www.pubmedcentral.nih.gov/picrender.fcgi?artid=384946&amp;blobtype">http://www.pubmedcentral.nih.gov/picrender.fcgi?artid=384946&amp;blobtype;</a>;=pdf <em>American Journal of Human Genetics</em> <strong>70</strong>:686–707]</p></li>
<li>

<p><a href="Claudia_Neuhauser" title="wikilink">Neuhauser, C.</a>, Krone, S.M. (1997) The genealogy of samples in models with selection <a href="http://www.genetics.org/cgi/reprint/145/2/519"><em>Genetics</em> <strong>145</strong> 519–534</a></p></li>
<li>

<p>Pitman, J. (1999) Coalescents with multiple collisions <em>The Annals of Probability</em> <strong>27</strong>:1870–1902</p></li>
<li>

<p>Harding, Rosalind, M. 1998. New phylogenies: an introductory look at the coalescent. pp. 15–22, in Harvey, P. H., Brown, A. J. L., Smith, J. M., Nee, S. New uses for new phylogenies. Oxford University Press (ISBN 0198549849)</p></li>
<li>

<p>Rosenberg, N.A., Nordborg, M. (2002) Genealogical Trees, Coalescent Theory and the Analysis of Genetic Polymorphisms. <em>Nature Reviews Genetics</em> <strong>3</strong>:380–390</p></li>
<li>

<p>Sagitov, S. (1999) The general coalescent with asynchronous mergers of ancestral lines <em>Journal of Applied Probability</em> <strong>36</strong>:1116–1125</p></li>
<li>

<p>Schweinsberg, J. (2000) Coalescents with simultaneous multiple collisions <em>Electronic Journal of Probability</em> <strong>5</strong>:1–50</p></li>
<li>

<p>Slatkin, M. (2001) Simulating genealogies of selected alleles in populations of variable size <em>Genetic Research</em> <strong>145</strong>:519–534</p></li>
<li>

<p>Tajima, F. (1983) Evolutionary Relationship of DNA Sequences in finite populations. <em>Genetics</em> <strong>105</strong>:437–460</p></li>
<li>

<p>Tavare S, Balding DJ, Griffiths RC &amp; Donnelly P. 1997. Inferring coalescent times from DNA sequence data. <em>Genetics</em> <strong>145</strong>: 505–518.</p></li>
<li>

<p>The international SNP map working group. 2001. A map of human genome variation containing 1.42 million single nucleotide polymorphisms. <em>Nature</em> <strong>409</strong>: 928–933.</p></li>
<li>

<p>Zöllner S. and <a href="Jonathan_K._Pritchard" title="wikilink">Pritchard J.K.</a> (2005) Coalescent-Based Association Mapping and Fine Mapping of Complex Trait Loci <a href="http://pritch.bsd.uchicago.edu/publications/ZollnerAndPritchard05.pdf"><em>Genetics</em> <strong>169</strong>:1071–1092</a></p></li>
<li>

<p>Rousset F. and Leblois R. (2007) Likelihood and Approximate Likelihood Analyses of Genetic Structure in a Linear Habitat: Performance and Robustness to Model Mis-Specification <a href="http://raphael.leblois.free.fr/Papiers/RoussetLeblois2007MBE.pdf"><em>Molecular Biology and Evolution</em> <strong>24</strong>:2730–2745</a></p></li>
</ul>
<h3 id="books">Books</h3>
<ul>
<li>Hein, J; Schierup, M. H., and Wiuf, C. <em>Gene Genealogies, Variation and Evolution – A Primer in Coalescent Theory</em>. <a href="Oxford_University_Press" title="wikilink">Oxford University Press</a>, 2005. ISBN 0-19-852996-1.</li>
<li>Nordborg, M. (2001) <a href="http://walnut.usc.edu/papers/references/copy_of_2002/wiley.pdf/view">Introduction to Coalescent Theory</a></li>
<li>Chapter 7 in Balding, D., Bishop, M., Cannings, C., editors, <em>Handbook of Statistical Genetics</em>. Wiley ISBN 978-0-471-86094-5</li>
<li>Wakeley J. (2006) <em>An Introduction to Coalescent Theory</em> Roberts &amp; Co ISBN 0-9747077-5-9 <a href="http://www.roberts-publishers.com/wakeley/">Accompanying website with sample chapters</a></li>
<li>

<p>Rice SH. (2004). <em>Evolutionary Theory: Mathematical and Conceptual Foundations</em>. Sinauer Associates: Sunderland, MA. See esp. ch. 3 for detailed derivations.</p></li>
<li>Berestycki N. "Recent progress in coalescent theory" 2009 ENSAIOS Matematicos vol.16</li>
<li>Bertoin J. "Random Fragmentation and Coagulation Processes"., 2006. Cambridge Studies in Advanced Mathematics, 102. <a href="Cambridge_University_Press" title="wikilink">Cambridge University Press</a>, Cambridge, 2006. ISBN 978-0-521-86728-3;</li>
<li>Pitman J. "Combinatorial stochastic processes" Springer (2003)</li>
</ul>
<h2 id="external-links">External links</h2>
<ul>
<li><a href="http://www.pandasthumb.org/archives/2004/05/evomath_3_genet.html">EvoMath 3: Genetic Drift and Coalescence, Briefly</a> — overview, with probability equations for genetic drift, and simulation graphs</li>
</ul>

<p>"</p>

<p><a href="Category:Population_genetics" title="wikilink">Category:Population genetics</a> <a href="Category:Statistical_genetics" title="wikilink">Category:Statistical genetics</a></p>
</hr></body>
</html>
