<html lang="en">
<head>
<meta charset="utf-8"/>
<title offset="902">CpG site</title>
<script src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_SVG.js" type="text/javascript">
</script>
</head>
<body>
<h1>CpG site</h1>
<hr/>

<p> The <strong>CpG sites</strong> or <strong>CG sites</strong> are regions of <a class="uri" href="DNA" title="wikilink">DNA</a> where a <a class="uri" href="cytosine" title="wikilink">cytosine</a> <a class="uri" href="nucleotide" title="wikilink">nucleotide</a> occurs next to a <a class="uri" href="guanine" title="wikilink">guanine</a> nucleotide in the linear <a href="DNA_sequence" title="wikilink">sequence</a> of <a href="Base_pair" title="wikilink">bases</a> along its length. "CpG" is shorthand for "—C—phosphate—G—", that is, cytosine and guanine separated by only one <a class="uri" href="phosphate" title="wikilink">phosphate</a>; phosphate links any two <a href="nucleoside" title="wikilink">nucleosides</a> together in DNA. The "CpG" notation is used to distinguish this linear sequence from the CG <a href="base_pair" title="wikilink">base-pairing</a> of cytosine and guanine. The CpG notation can also be interpreted as the cytosine being <a href="Directionality_(molecular_biology)#5′-end" title="wikilink">5 prime</a> to the guanine base.</p>

<p>Cytosines in CpG dinucleotides can be <a href="DNA_methylation" title="wikilink">methylated</a> to form <a class="uri" href="5-methylcytosine" title="wikilink">5-methylcytosine</a>. In <a href="mammal" title="wikilink">mammals</a>, methylating the cytosine within a gene can change its expression, a mechanism that is part of a larger field of science studying gene regulation that is called <a class="uri" href="epigenetics" title="wikilink">epigenetics</a>. <a href="Enzyme" title="wikilink">Enzymes</a> that add a methyl group are called <a href="DNA_methyltransferase" title="wikilink">DNA methyltransferases</a>.</p>

<p>In mammals, 70% to 80% of CpG cytosines are methylated.<a class="footnoteRef" href="#fn1" id="fnref1"><sup>1</sup></a></p>

<p>Unmethylated CpG dinucleotide sites can be detected by Toll-like receptor 9<a class="footnoteRef" href="#fn2" id="fnref2"><sup>2</sup></a> (<a href="TLR_9" title="wikilink">TLR 9</a>) on <a href="plasmacytoid_dendritic_cell" title="wikilink">plasmacytoid dendritic cells</a>, monocytes, natural killer (NK) cells, and <a href="B_cell" title="wikilink">B cells</a> in humans. This is used to detect intracellular viral, fungal, and bacterial pathogen DNA.</p>
<h2 id="frequency-in-vertebrates">Frequency in vertebrates</h2>

<p>CpG dinucleotides have long been observed to occur with a much lower frequency in the sequence of vertebrate genomes than would be expected due to random chance. For example, in the human genome, which has a 42% <a href="GC_content" title="wikilink">GC content</a>, a pair of <a href="nucleotide" title="wikilink">nucleotides</a> consisting of cytosine followed by guanine would be expected to occur 0.21 * 0.21 = 4.41% of the time. The frequency of CpG dinucleotides in human genomes is 1% — less than one-quarter of the expected frequency. Scarano <em>et al.</em> proposed that the CpG deficiency is due to an increased vulnerability of methylcytosines to spontaneously <a href="Deamination" title="wikilink">deaminate</a> to <a class="uri" href="thymine" title="wikilink">thymine</a> in genomes with CpG cytosine methylation.<a class="footnoteRef" href="#fn3" id="fnref3"><sup>3</sup></a></p>
<h2 id="cpg-islands">CpG islands</h2>
<figure><b>(Figure)</b>
<figcaption>How methylation of CpG sites followed by spontaneous deamination leads to a lack of CpG sites in methylated DNA. As a result residual CpG islands are created in areas where methylation is rare, and CpG sites stick (or where C to T mutation is highly detrimental).</figcaption>
</figure>

<p>CpG islands (or CG islands) are regions with a high frequency of CpG sites, though objective definitions for CpG islands are limited. The usual formal definition of a CpG island is a region with at least 200 <a href="base_pair" title="wikilink">bp</a>, and a GC percentage that is greater than 50 %, and with an observed-to-expected CpG ratio that is greater than 60 %. The "observed-to-expected CpG ratio" can be derived where the observed is calculated as:</p>

<p>

<math display="block" id="CpG_site:0">
 <semantics>
  <mrow>
   <mo stretchy="false">(</mo>
   <mrow>
    <mrow>
     <mtext>number of</mtext>
     <mi>C</mi>
     <mi>p</mi>
     <mi>G</mi>
     <mi>s</mi>
    </mrow>
    <mo>*</mo>
    <mtext>length of sequence</mtext>
   </mrow>
   <mo stretchy="false">)</mo>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <times></times>
    <apply>
     <times></times>
     <mtext>number of</mtext>
     <ci>C</ci>
     <ci>p</ci>
     <ci>G</ci>
     <ci>s</ci>
    </apply>
    <mtext>length of sequence</mtext>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   (\text{number of }CpGs*\text{length of sequence})
  </annotation>
 </semantics>
</math>

 and the expected as:</p>

<p>

<math display="block" id="CpG_site:1">
 <semantics>
  <mrow>
   <mo stretchy="false">(</mo>
   <mrow>
    <mrow>
     <mrow>
      <mtext>number of</mtext>
      <mi>C</mi>
     </mrow>
     <mo>*</mo>
     <mtext>number of</mtext>
    </mrow>
    <mi>G</mi>
   </mrow>
   <mo stretchy="false">)</mo>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <times></times>
    <apply>
     <times></times>
     <apply>
      <times></times>
      <mtext>number of</mtext>
      <ci>C</ci>
     </apply>
     <mtext>number of</mtext>
    </apply>
    <ci>G</ci>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   (\text{number of }C*\text{number of }G)
  </annotation>
 </semantics>
</math>

<a class="footnoteRef" href="#fn4" id="fnref4"><sup>4</sup></a> or</p>

<p>

<math display="block" id="CpG_site:2">
 <semantics>
  <msup>
   <mrow>
    <mo stretchy="false">(</mo>
    <mrow>
     <mrow>
      <mo stretchy="false">(</mo>
      <mrow>
       <mrow>
        <mtext>number of</mtext>
        <mi>C</mi>
       </mrow>
       <mo>+</mo>
       <mrow>
        <mtext>number of</mtext>
        <mi>G</mi>
       </mrow>
      </mrow>
      <mo stretchy="false">)</mo>
     </mrow>
     <mo>/</mo>
     <mn>2</mn>
    </mrow>
    <mo stretchy="false">)</mo>
   </mrow>
   <mn>2</mn>
  </msup>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <csymbol cd="ambiguous">superscript</csymbol>
    <apply>
     <divide></divide>
     <apply>
      <plus></plus>
      <apply>
       <times></times>
       <mtext>number of</mtext>
       <ci>C</ci>
      </apply>
      <apply>
       <times></times>
       <mtext>number of</mtext>
       <ci>G</ci>
      </apply>
     </apply>
     <cn type="integer">2</cn>
    </apply>
    <cn type="integer">2</cn>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   ((\text{number of }C+\text{number of }G)/2)^{2}
  </annotation>
 </semantics>
</math>

<a class="footnoteRef" href="#fn5" id="fnref5"><sup>5</sup></a></p>

<p>Many genes in mammalian genomes have CpG islands associated with the start of the gene<a class="footnoteRef" href="#fn6" id="fnref6"><sup>6</sup></a> (<a href="Promoter_(genetics)" title="wikilink">promoter regions</a>). Because of this, the presence of a CpG island is used to help in the prediction and annotation of genes.</p>

<p>In mammalian genomes, CpG islands are typically 300-3,000 base pairs in length, and have been found in or near approximately 40% of <a href="promoter_(biology)" title="wikilink">promoters</a> of mammalian genes.<a class="footnoteRef" href="#fn7" id="fnref7"><sup>7</sup></a> About 70% of human promoters have a high CpG content. Given the frequency of GC two-nucleotide sequences, the number of CpG dinucleotides is much lower than would be expected.<a class="footnoteRef" href="#fn8" id="fnref8"><sup>8</sup></a></p>

<p>A 2002 study revised the rules of CpG island prediction to exclude other GC-rich genomic sequences such as <a href="Alu_sequence" title="wikilink">Alu repeats</a>. Based on an extensive search on the complete sequences of human chromosomes 21 and 22, DNA regions greater than 500 bp were found more likely to be the "true" CpG islands associated with the 5' regions of genes if they had a GC content greater than 55%, and an observed-to-expected CpG ratio of 65%.<a class="footnoteRef" href="#fn9" id="fnref9"><sup>9</sup></a></p>

<p>CpG islands are characterized by CpG dinucleotide content of at least 60% of that which would be statistically expected (~4–6%), whereas the rest of the genome has much lower CpG frequency (~1%), a phenomenon called <a href="CG_suppression" title="wikilink">CG suppression</a>. Unlike CpG sites in the <a href="coding_region" title="wikilink">coding region</a> of a gene, in most instances the CpG sites in the CpG islands of promoters are unmethylated if the genes are expressed. This observation led to the speculation that <a class="uri" href="methylation" title="wikilink">methylation</a> of CpG sites in the promoter of a gene may inhibit gene expression. Methylation, along with <a class="uri" href="histone" title="wikilink">histone</a> modification, is central to <a href="Genomic_imprinting" title="wikilink">imprinting</a>.<a class="footnoteRef" href="#fn10" id="fnref10"><sup>10</sup></a> Most of the methylation differences between tissues, or between normal and cancer samples, occur a short distance from the CpG islands (at "CpG island shores") rather than in the islands themselves.<a class="footnoteRef" href="#fn11" id="fnref11"><sup>11</sup></a></p>

<p>CpG islands typically occur at or near the transcription start site of genes, particularly housekeeping genes, in vertebrates.<a class="footnoteRef" href="#fn12" id="fnref12"><sup>12</sup></a> A C (cytosine) base followed immediately by a G (guanine) base (a CpG) is rare in vertebrate DNA because the cytosines in such an arrangement tend to be methylated. This methylation helps distinguish the newly synthesized DNA strand from the parent strand, which aids in the final stages of DNA proofreading after duplication. However, over time methylated cytosines tend to turn into <a href="thymine" title="wikilink">thymines</a> because of spontaneous <a class="uri" href="deamination" title="wikilink">deamination</a>. There is a special enzyme in humans (<a href="Thymine-DNA_glycosylase" title="wikilink">Thymine-DNA glycosylase</a>, or TDG) that specifically replaces T's from T/G mismatches. However, due to the rarity of CpGs, it is theorised to be insufficiently effective in preventing a possibly rapid mutation of the dinucleotides. The existence of CpG islands is usually explained by the existence of selective forces for relatively high CpG content, or low levels of methylation in that genomic area, perhaps having to do with the regulation of gene expression. Recently a study showed that most CpG islands are a result of non-selective forces.<a class="footnoteRef" href="#fn13" id="fnref13"><sup>13</sup></a></p>
<h2 id="methylation-silencing-cancer-and-aging">Methylation, silencing, cancer, and aging</h2>

<p>  Methylation of CpG sites within the promoters of genes can lead to their <a href="Gene_silencing" title="wikilink">silencing</a>, a feature found in a number of human cancers (for example the silencing of <a href="tumor_suppressor_gene" title="wikilink">tumor suppressor genes</a>). In contrast, the hypomethylation of CpG sites has been associated with the over-expression of <a class="uri" href="oncogenes" title="wikilink">oncogenes</a> within cancer cells.<a class="footnoteRef" href="#fn14" id="fnref14"><sup>14</sup></a></p>

<p>Since age has a strong effect on DNA methylation levels on tens of thousands of CpG sites, one can define a highly accurate <a href="Biological_clock_(aging)" title="wikilink">biological clock</a> (referred to as epigenetic clock or <a href="Biological_clock_(aging)" title="wikilink">DNA methylation age</a>) in humans and chimpanzees.<a class="footnoteRef" href="#fn15" id="fnref15"><sup>15</sup></a></p>
<h2 id="see-also">See also</h2>
<ul>
<li><a class="uri" href="TLR9" title="wikilink">TLR9</a>, detector of unmethylated CpG sites</li>
<li><a href="Biological_clock_(aging)" title="wikilink">DNA methylation age</a></li>
</ul>
<h2 id="references">References</h2>

<p>"</p>

<p><a href="Category:Molecular_genetics" title="wikilink">Category:Molecular genetics</a> <a class="uri" href="Category:DNA" title="wikilink">Category:DNA</a></p>
<section class="footnotes">
<hr/>
<ol>
<li id="fn1"><a href="#fnref1">↩</a></li>
<li id="fn2"><a href="#fnref2">↩</a></li>
<li id="fn3"><a href="#fnref3">↩</a></li>
<li id="fn4"><a href="#fnref4">↩</a></li>
<li id="fn5"></li>
<li id="fn6"><a href="#fnref6">↩</a></li>
<li id="fn7"><a href="#fnref7">↩</a></li>
<li id="fn8"><a href="#fnref8">↩</a></li>
<li id="fn9"><a href="#fnref9">↩</a></li>
<li id="fn10"><a href="#fnref10">↩</a></li>
<li id="fn11"><a href="#fnref11">↩</a></li>
<li id="fn12"></li>
<li id="fn13"><a href="#fnref13">↩</a></li>
<li id="fn14"><a href="#fnref14">↩</a></li>
<li id="fn15"><a href="#fnref15">↩</a></li>
</ol>
</section>
</body>
</html>
