<html lang="en">
<head>
<meta charset="utf-8"/>
<title offset="662">Cupriavidus necator</title>
<script src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_SVG.js" type="text/javascript">
</script>
</head>
<body>
<h1>Cupriavidus necator</h1>
<hr/>

<p><strong><em>Cupriavidus necator</em></strong> is a <a class="uri" href="Gram-negative" title="wikilink">Gram-negative</a> <a class="uri" href="soil" title="wikilink">soil</a> <a class="uri" href="bacterium" title="wikilink">bacterium</a> of the <a class="uri" href="Betaproteobacteria" title="wikilink">Betaproteobacteria</a> class.<a class="footnoteRef" href="#fn1" id="fnref1"><sup>1</sup></a> </p>
<h2 id="taxonomy">Taxonomy</h2>

<p><em>C. necator</em> has gone through a series of name changes. In the first half of the 20th century, many microorganisms were isolated for their ability to use hydrogen. Hydrogen-metabolizing <a href="Lithotroph" title="wikilink">chemolithotrophic</a> organisms were clustered into the group <em>Hydrogenomonas</em>.<a class="footnoteRef" href="#fn2" id="fnref2"><sup>2</sup></a> <em>C. necator</em> was originally named <em>Hydrogenomonas eutrophus</em> because it fell under the <em>Hydrogenomonas</em> classification and was “well nourished and robust”.<a class="footnoteRef" href="#fn3" id="fnref3"><sup>3</sup></a> Some of the original <em>H. eutrophus</em> cultures isolated were by Bovell and Wilde.<a class="footnoteRef" href="#fn4" id="fnref4"><sup>4</sup></a><a class="footnoteRef" href="#fn5" id="fnref5"><sup>5</sup></a> After characterizing <a href="Morphology_(biology)" title="wikilink">cell morphology</a>, <a class="uri" href="metabolism" title="wikilink">metabolism</a> and <a href="GC-content" title="wikilink">GC content</a>, the <em>Hydrogenomonas</em> nomenclature was disbanded because it comprised many species of microorganisms.<a class="footnoteRef" href="#fn6" id="fnref6"><sup>6</sup></a> <em>H. eutrophus</em> was then renamed <em>Alcaligenes eutropha</em> because it was a micro-organism with degenerated peritrichous <a href="Flagellum" title="wikilink">flagellation</a>.<a class="footnoteRef" href="#fn7" id="fnref7"><sup>7</sup></a><a class="footnoteRef" href="#fn8" id="fnref8"><sup>8</sup></a> Investigating <a class="uri" href="phenotype" title="wikilink">phenotype</a>, <a class="uri" href="lipid" title="wikilink">lipid</a> composition, <a href="fatty_acid" title="wikilink">fatty acid</a> composition and <a href="16S_ribosomal_RNA" title="wikilink">16S rRNA</a> analysis, <em>A. eutropha</em> was found to belong to the genus <em>Ralstonia</em> and named <em>Ralstonia eutropha</em>.<a class="footnoteRef" href="#fn9" id="fnref9"><sup>9</sup></a> Upon further study of the <em>Ralstonia</em> genus, <em>Ralstonia</em> was found to comprise two phenotypically distinct clusters. The new genus <em>Wautersia</em> was created from one of these clusters which included <em>R. eutropha</em>. In turn <em>R. eutropha</em> was renamed <em>Wautersia eutropha</em>.<a class="footnoteRef" href="#fn10" id="fnref10"><sup>10</sup></a> Looking at <a href="DNA-DNA_hybridization" title="wikilink">DNA-DNA hybridization</a> and phenotype comparison with <em>Cupriavidus necator</em>, <em>W. eutropha</em> was found to be the same species as previously described <em>C. necator</em>. Because <em>C. necator</em> was named in 1987 far before the name change to <em>R. eutropha</em> and <em>W. eutropha</em>, the name <em>C. necator</em> was assigned to <em>R. eutropha</em> according to Rule 23a of the <a href="International_Code_of_Nomenclature_of_Bacteria" title="wikilink">International Code of Nomenclature of Bacteria</a>.<a class="footnoteRef" href="#fn11" id="fnref11"><sup>11</sup></a></p>
<h2 id="metabolism">Metabolism</h2>

<p><em>C. necator</em> is a <a href="hydrogen-oxidizing_bacterium" title="wikilink">hydrogen-oxidizing bacterium</a> (“knallgas” bacterium) capable of growing at the interface of anaerobic and aerobic environments. It can easily adapt between <a href="heterotroph" title="wikilink">heterotrophic</a> and <a href="autotroph" title="wikilink">autotrophic</a> lifestyles. Both organic compounds and hydrogen can be used as a source of energy<a class="footnoteRef" href="#fn12" id="fnref12"><sup>12</sup></a> <em>C. necator</em> can perform <a href="Cellular_respiration" title="wikilink">aerobic</a> or <a href="Cellular_respiration" title="wikilink">anaerobic respiration</a> by <a class="uri" href="denitrification" title="wikilink">denitrification</a> of nitrate and/or nitrite to nitrogen gas.<a class="footnoteRef" href="#fn13" id="fnref13"><sup>13</sup></a> When growing under autotrophic conditions, <em>C. necator</em> fixes carbon through the <a href="pentose_phosphate_pathway" title="wikilink">pentose phosphate pathway</a>.<a class="footnoteRef" href="#fn14" id="fnref14"><sup>14</sup></a> It is known to produce and sequester <a href="Polyhydroxyalkanoates" title="wikilink">polyhydroxyalkanoate</a> (PHA) plastics when exposed to excess amounts of sugar substrate. PHA can accumulate to levels around 90% of the cell's dry weight.<a class="footnoteRef" href="#fn15" id="fnref15"><sup>15</sup></a> To better characterize the lifestyle of <em>C. necator</em>, the <a href="genome" title="wikilink">genomes</a> of two strains have been <a href="DNA_sequencing" title="wikilink">sequenced</a>.<a class="footnoteRef" href="#fn16" id="fnref16"><sup>16</sup></a><a class="footnoteRef" href="#fn17" id="fnref17"><sup>17</sup></a></p>
<h2 id="hydrogenases">Hydrogenases</h2>

<p><em>C. necator</em> can use hydrogen gas as a source of energy when growing under autotrophic conditions. It contains three different <a href="hydrogenase" title="wikilink">hydrogenases</a> that have [[Nickel-dependent hydrogenase|[Ni-Fe] active sites]] and all perform this reaction:<a class="footnoteRef" href="#fn18" id="fnref18"><sup>18</sup></a></p>
<dl>
<dd>H<sub>2</sub> 

<math display="inline" id="Cupriavidus_necator:0">
 <semantics>
  <mo>⇌</mo>
  <annotation-xml encoding="MathML-Content">
   <ci>normal-⇌</ci>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \rightleftharpoons
  </annotation>
 </semantics>
</math>

 2H<sup>+</sup> + 2e<sup>−</sup>
</dd>
</dl>

<p>The hydrogenases of <em>C. necator</em> are like other typical [Ni-Fe] hydrogenases because they are made up of a large and a small subunit. The large subunit is where the [Ni-Fe] active site resides and the small subunit is composed of [[Iron-sulfur protein|[Fe-S] clusters]].<a class="footnoteRef" href="#fn19" id="fnref19"><sup>19</sup></a> However, the hydrogenases of <em>C. necator</em> are different from typical [Ni-Fe] hydrogenases because they are tolerant to oxygen and are not inhibited by <a href="Carbon_monoxide" title="wikilink">CO</a>.<a class="footnoteRef" href="#fn20" id="fnref20"><sup>20</sup></a> While the three hydrogenases perform the same reaction in the cell, each hydrogenase is linked to a different cellular process. The differences between the regulatory hydrogenase, membrane bound hydrogenase, and soluble hydrogeanse in <em>C. necator</em> are described below.</p>
<h3 id="regulatory-hydrogenase">Regulatory hydrogenase</h3>

<p>The first hydrogenase is a regulatory hydrogenase (RH) that signals to the cell hydrogen is present. The RH is a protein containing large and small [Ni-Fe] hydrogenase subunits attached to a <a href="Protein_kinase" title="wikilink">histidine protein kinase</a> subunit.<a class="footnoteRef" href="#fn21" id="fnref21"><sup>21</sup></a> The hydrogen gas is oxidized at the [Ni-Fe] center in the large subunit and in turn reduces the [Fe-S] clusters in the small subunit. It is unknown whether the electrons are transferred from the [Fe-S] clusters to the protein kinase domain.<a class="footnoteRef" href="#fn22" id="fnref22"><sup>22</sup></a> The histidine protein kinase activates a <a href="Two-component_regulatory_system" title="wikilink">response regulator</a>. The response regulator is active in the dephosphorylated form. The dephosphorylated response regulator promotes the transcription of the membrane bound hydrogenase and soluble hydrogenase.<a class="footnoteRef" href="#fn23" id="fnref23"><sup>23</sup></a></p>
<h3 id="membrane-bound-hydrogenase">Membrane-bound hydrogenase</h3>

<p>The membrane-bound hydrogenase (MBH) is linked to the <a href="Electron_transport_chain" title="wikilink">respiratory chain</a> through a specific <a href="cytochrome_b" title="wikilink">cytochrome b</a>-related protein in <em>C. necator</em>.<a class="footnoteRef" href="#fn24" id="fnref24"><sup>24</sup></a> Hydrogen gas is oxidized at the [Ni-Fe] active site in the large subunit and the electrons are shuttled through the [Fe-S] clusters in the small subunit to the cytochrome b-like protein.<a class="footnoteRef" href="#fn25" id="fnref25"><sup>25</sup></a> The MBH is located on the outer <a href="Cell_membrane" title="wikilink">cytoplasmic membrane</a>. It recovers energy for the cell by funneling electrons into the respiratory chain and by increasing the <a href="Electrochemical_gradient" title="wikilink">proton gradient</a>.<a class="footnoteRef" href="#fn26" id="fnref26"><sup>26</sup></a> The MBH in <em>C. necator</em> is not inhibited by CO and is tolerant to oxygen.<a class="footnoteRef" href="#fn27" id="fnref27"><sup>27</sup></a></p>
<h3 id="soluble-hydrogenase">Soluble hydrogenase</h3>

<p>The soluble hydrogenase (SH) creates a <a href="Nicotinamide_adenine_dinucleotide" title="wikilink">NADH</a>-reducing equivalence by oxidizing hydrogen gas. The SH is a <a href="Protein_dimer" title="wikilink">heterodimer protein</a><a class="footnoteRef" href="#fn28" id="fnref28"><sup>28</sup></a> with two subunits making up the large and small subunits of the [Ni-Fe] hydrogenase and the other two subunits comprising a protein similar to <a href="NADH_dehydrogenase" title="wikilink">Complex I</a> (as shown in the accompanying figure).<a class="footnoteRef" href="#fn29" id="fnref29"><sup>29</sup></a> The [Ni-Fe] active site oxidized hydrogen gas which transfers electrons to a <a href="Flavin_mononucleotide" title="wikilink">FMNa</a> cofactor, then to [Fe-S] clusters of the small hydrogenase subunit, then to another FMNb cofactor and finally to NAD<sup>+</sup>.<a class="footnoteRef" href="#fn30" id="fnref30"><sup>30</sup></a> The reducing equivalence provide a means for fixing carbon dioxide when <em>C. necator</em> is growing autotrophically.<a class="footnoteRef" href="#fn31" id="fnref31"><sup>31</sup></a></p>
<h4 id="soluble-hydrogenase-active-site">Soluble hydrogenase active site</h4>

<p>The active site of the soluble hydrogenase (SH) of <em>C. necator</em> H16 has been extensively studied because <em>C. necator</em> H16 can be produced in large amounts, can be genetically manipulated, and can be analyzed with <a href="Spectroscopy" title="wikilink">spectrographic</a> techniques. However, no crystal structure is currently available for the <em>C. necator</em> H16 soluble hydrogenase in the presence of oxygen to determine the interactions of the active site with the rest of the protein.<a class="footnoteRef" href="#fn32" id="fnref32"><sup>32</sup></a></p>
<h5 id="typical-anaerobic-ni-fe-hydrogenases">Typical anaerobic [Ni-Fe] hydrogenases</h5>

<p>The [Ni-Fe] hydrogenase from <em><a href="Desulfovibrio_vulgaris" title="wikilink">Desulfovibrio vulgaris</a></em> and <em>D. gigas</em> have similar protein structures to each other and represent typical [Ni-Fe] hydrogenases.<a class="footnoteRef" href="#fn33" id="fnref33"><sup>33</sup></a><a class="footnoteRef" href="#fn34" id="fnref34"><sup>34</sup></a><a class="footnoteRef" href="#fn35" id="fnref35"><sup>35</sup></a><a class="footnoteRef" href="#fn36" id="fnref36"><sup>36</sup></a> The large subunit contains the [Ni-Fe] active site buried deep in the core of the protein and the small subunit contains [Fe-S] clusters. The Ni atom is <a href="Coordination_complex" title="wikilink">coordinated</a> to the Desulfovibrio hydrogenase by 4 <a href="Cystine" title="wikilink">cysteine</a> <a href="ligand" title="wikilink">ligands</a>. Two of these same cysteine ligands also bridge the Fe of the [Ni-Fe] active site.<a class="footnoteRef" href="#fn37" id="fnref37"><sup>37</sup></a><a class="footnoteRef" href="#fn38" id="fnref38"><sup>38</sup></a> The Fe atom also contains three ligands, one CO and two <a href="Cyanide" title="wikilink">CN</a> that complete the active site.<a class="footnoteRef" href="#fn39" id="fnref39"><sup>39</sup></a> These additional ligands might contribute to the reactivity or help stabilize the Fe atom in the +2 oxidation state.<a class="footnoteRef" href="#fn40" id="fnref40"><sup>40</sup></a> Typical [NiFe] hydrogenases like those of <em>D. vulgaris</em> and <em>D. gigas</em> are poisoned by oxygen because an oxygen atom binds strongly to the NiFe active site.<a class="footnoteRef" href="#fn41" id="fnref41"><sup>41</sup></a></p>
<h5 id="c.-necator-oxygen-tolerant-soluble-hydrogenase"><em>C. necator</em> oxygen-tolerant soluble hydrogenase</h5>

<p>The soluble [Ni-Fe] hydrogenases in <em>C. necator</em> are unique for other organisms because it is oxygen tolerant.<a class="footnoteRef" href="#fn42" id="fnref42"><sup>42</sup></a> The active site of the SH has been studied to learn why this protein is tolerant to oxygen. A major difference of the [Ni-Fe] hydrogenases of <em>C. necator</em> is it has more coordinating ligands than in typical [Ni-Fe] hydrogenases. Two cysteine ligands bridge the Ni atom and Fe atom of the active site in <em>C. necator</em>. Two modified cysteine sulfenate ligands also are predicted to coordinate the Ni atom.<a class="footnoteRef" href="#fn43" id="fnref43"><sup>43</sup></a> An additional CN ligand is added to the Ni atom and removing this CN ligand makes the enzyme susceptible to oxygen. Also, the Fe has an additional CN ligand bound in addition to the two CN ligands and one CO ligand present in typical NiFe hydrogenases (see the above figure for a schematic of the active site).<a class="footnoteRef" href="#fn44" id="fnref44"><sup>44</sup></a> These additional ligands are predicted to help the protein be more oxygen tolerant because it stabilizes the Ni atom in the active, +2 oxidation state.<a class="footnoteRef" href="#fn45" id="fnref45"><sup>45</sup></a></p>
<h3 id="applications">Applications</h3>

<p>The oxygen-tolerant hydrogenases of <em>C. necator</em> have been studied for diverse purposes. <em>C. necator</em> was studied as an attractive organism to help support life in space. It can fix carbon dioxide as a carbon source, use the <a class="uri" href="urea" title="wikilink">urea</a> in urine as a nitrogen source and use hydrogen as an energy source to create dense cultures that could be used as a source of protein.<a class="footnoteRef" href="#fn46" id="fnref46"><sup>46</sup></a><a class="footnoteRef" href="#fn47" id="fnref47"><sup>47</sup></a></p>

<p><a href="Electrolysis_of_water" title="wikilink">Electrolysis of water</a> is one way of creating oxygenic atmosphere in space and <em>C. necator</em> was investigated to recycle the hydrogen produced during this process.<a class="footnoteRef" href="#fn48" id="fnref48"><sup>48</sup></a></p>

<p>Oxygen-tolerant hydrogenases are being used to investigate biofuels. Hydrogenases from <em>C. necator</em> have been used to coat electrode surfaces to create hydrogen <a href="fuel_cell" title="wikilink">fuel cells</a> tolerant to oxygen and carbon monoxide<a class="footnoteRef" href="#fn49" id="fnref49"><sup>49</sup></a> and to design hydrogen-producing <a href="Photosystem_I" title="wikilink">light complexes</a>.<a class="footnoteRef" href="#fn50" id="fnref50"><sup>50</sup></a> In addition, the hydrogenases from <em>C. necator</em> have been used to create hydrogen sensors.<a class="footnoteRef" href="#fn51" id="fnref51"><sup>51</sup></a> Genetically modified <em>C. necator</em> can produce <a class="uri" href="isobutanol" title="wikilink">isobutanol</a> from  that can directly substitute or blend with <a class="uri" href="gasoline" title="wikilink">gasoline</a>. The organism emits the isobutanol without having to be destroyed to obtain it.<a class="footnoteRef" href="#fn52" id="fnref52"><sup>52</sup></a></p>
<h2 id="references">References</h2>

<p>"</p>

<p><a class="uri" href="Category:Burkholderiales" title="wikilink">Category:Burkholderiales</a> <a href="Category:Bacteria_with_sequenced_genomes" title="wikilink">Category:Bacteria with sequenced genomes</a></p>
<section class="footnotes">
<hr/>
<ol>
<li id="fn1"><a href="#fnref1">↩</a></li>
<li id="fn2"><a href="#fnref2">↩</a></li>
<li id="fn3"><a href="#fnref3">↩</a></li>
<li id="fn4"><a href="#fnref4">↩</a></li>
<li id="fn5"><a href="#fnref5">↩</a></li>
<li id="fn6"></li>
<li id="fn7"></li>
<li id="fn8"><a href="#fnref8">↩</a></li>
<li id="fn9"></li>
<li id="fn10"><a href="#fnref10">↩</a></li>
<li id="fn11"><a href="#fnref11">↩</a></li>
<li id="fn12"><a href="#fnref12">↩</a></li>
<li id="fn13"><a href="#fnref13">↩</a></li>
<li id="fn14"><a href="#fnref14">↩</a></li>
<li id="fn15"><a href="#fnref15">↩</a></li>
<li id="fn16"></li>
<li id="fn17"><a href="#fnref17">↩</a></li>
<li id="fn18"><a href="#fnref18">↩</a></li>
<li id="fn19"><a href="#fnref19">↩</a></li>
<li id="fn20"></li>
<li id="fn21"><a href="#fnref21">↩</a></li>
<li id="fn22"></li>
<li id="fn23"><a href="#fnref23">↩</a></li>
<li id="fn24"><a href="#fnref24">↩</a></li>
<li id="fn25"></li>
<li id="fn26"></li>
<li id="fn27"><a href="#fnref27">↩</a></li>
<li id="fn28"><a href="#fnref28">↩</a></li>
<li id="fn29"><a href="#fnref29">↩</a></li>
<li id="fn30"></li>
<li id="fn31"><a href="#fnref31">↩</a></li>
<li id="fn32"></li>
<li id="fn33"></li>
<li id="fn34"><a href="#fnref34">↩</a></li>
<li id="fn35"><a href="#fnref35">↩</a></li>
<li id="fn36"><a href="#fnref36">↩</a></li>
<li id="fn37"></li>
<li id="fn38"></li>
<li id="fn39"><a href="#fnref39">↩</a></li>
<li id="fn40"></li>
<li id="fn41"></li>
<li id="fn42"><a href="#fnref42">↩</a></li>
<li id="fn43"></li>
<li id="fn44"><a href="#fnref44">↩</a></li>
<li id="fn45"></li>
<li id="fn46"><a href="#fnref46">↩</a></li>
<li id="fn47"><a href="#fnref47">↩</a></li>
<li id="fn48"><a href="#fnref48">↩</a></li>
<li id="fn49"></li>
<li id="fn50"><a href="#fnref50">↩</a></li>
<li id="fn51"><a href="#fnref51">↩</a></li>
<li id="fn52"><a href="#fnref52">↩</a></li>
</ol>
</section>
</body>
</html>
