<html lang="en">
<head>
<meta charset="utf-8"/>
<title offset="1857">Deep sequencing</title>
<script src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_SVG.js" type="text/javascript">
</script>
</head>
<body>
<h1>Deep sequencing</h1>
<hr/>

<p><strong>Depth</strong> (<strong>coverage</strong>) in <a href="DNA_sequencing" title="wikilink">DNA sequencing</a> refers to the number of times a <a class="uri" href="nucleotide" title="wikilink">nucleotide</a> is read during the sequencing process. <strong>Deep sequencing</strong> indicates that the total number of reads is many times larger than the length of the sequence under study. <a href="Shotgun_sequencing#Coverage" title="wikilink"><strong>Coverage</strong></a> is the average number of reads representing a given nucleotide in the reconstructed sequence.</p>

<p>Depth can be calculated from the length of the original genome (<em>G</em>), the number of reads(<em>N</em>), and the average read length(<em>L</em>) as 

<math display="inline" id="Deep_sequencing:0">
<semantics>
<mrow>
<mrow>
<mi>N</mi>
<mo>×</mo>
<mi>L</mi>
</mrow>
<mo>/</mo>
<mi>G</mi>
</mrow>
<annotation-xml encoding="MathML-Content">
<apply>
<divide></divide>
<apply>
<times></times>
<ci>N</ci>
<ci>L</ci>
</apply>
<ci>G</ci>
</apply>
</annotation-xml>
<annotation encoding="application/x-tex">
   N\times L/G
  </annotation>
</semantics>
</math>

. For example, a hypothetical genome with 2,000 base pairs reconstructed from 8 reads with an average length of 500 nucleotides will have 2x redundancy. This parameter also enables one to estimate other quantities, such as the percentage of the genome covered by reads (sometimes also called coverage). A high coverage in shotgun sequencing is desired because it can overcome errors in base calling and assembly. The subject of <a href="DNA_sequencing_theory" title="wikilink">DNA sequencing theory</a> addresses the relationships of such quantities.</p>

<p>Sometimes a distinction is made between sequence coverage and physical coverage. Sequence coverage is the average number of times a base is read (as described above). Physical coverage is the average number of times a base is read or spanned by mate paired reads.<a class="footnoteRef" href="#fn1" id="fnref1"><sup>1</sup></a></p>

<p>The term "deep" has been used for a wide range of depths (&gt;7×), and the newer term "ultra-deep" has appeared in the scientific literature to refer to even higher coverage (&gt;100×).<a class="footnoteRef" href="#fn2" id="fnref2"><sup>2</sup></a></p>

<p>Even though the sequencing accuracy for each individual nucleotide is very high, the very large number of nucleotides in the genome means that if an individual genome is only sequenced once, there will be a significant number of sequencing errors. Furthermore rare <a href="single-nucleotide_polymorphism" title="wikilink">single-nucleotide polymorphisms</a> (SNPs) are common. Hence to distinguish between sequencing errors and true SNPs, it is necessary to increase the sequencing accuracy even further by sequencing individual genomes a large number of times.</p>
<h2 id="deep-sequencing-of-transcriptome-or-rna">Deep sequencing of transcriptome or RNA</h2>

<p>Deep sequencing of <a class="uri" href="transcriptome" title="wikilink">transcriptome</a>, also known as <a class="uri" href="RNA-Seq" title="wikilink">RNA-Seq</a>, provides both the sequence and frequency of RNA molecules that are present at any particular time in a specific cell type, tissue or organ. Counting the number of mRNAs that are encoded by individual genes provides an indicator of protein-coding potential, a major contributor to <a class="uri" href="phenotype" title="wikilink">phenotype</a>.<a class="footnoteRef" href="#fn3" id="fnref3"><sup>3</sup></a> Improving methods for RNA sequencing is an active area of research both in terms of experimental <a class="footnoteRef" href="#fn4" id="fnref4"><sup>4</sup></a> and computational methods.</p>
<h2 id="references">References</h2>

<p>"</p>

<p><a href="Category:Molecular_biology" title="wikilink">Category:Molecular biology</a> <a href="Category:DNA_sequencing" title="wikilink">Category:DNA sequencing</a></p>
<section class="footnotes">
<hr/>
<ol>
<li id="fn1"><a href="#fnref1">↩</a></li>
<li id="fn2"><a href="#fnref2">↩</a></li>
<li id="fn3"><a href="#fnref3">↩</a></li>
<li id="fn4"><a href="#fnref4">↩</a></li>
</ol>
</section>
</body>
</html>
