<html lang="en">
<head>
<meta charset="utf-8"/>
<title offset="867">Denaturation mapping</title>
<script src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_SVG.js" type="text/javascript">
</script>
</head>
<body>
<h1>Denaturation mapping</h1>
<hr/>
<figure><b>(Figure)</b>
<figcaption><strong>Modern application of denaturation mapping</strong>: DNA is stained with a dye(green) that dissociates from single stranded DNA, changes in the environment like temperature increases that are not enough to completely dissociate the molecule causes parts of it to dissociate in a predictable manner from AT rich regions (red). fluorescence emissions from this partially denatured molecule is measured with photolithography. The graph shown is a simplified output of such an analysis.</figcaption>
</figure>

<p><strong>Denaturation Mapping</strong> is a form of <a href="optical_mapping" title="wikilink">optical mapping</a>, first described in 1966. It is used to characterize <a class="uri" href="DNA" title="wikilink">DNA</a> molecules without the need for amplification or <a class="uri" href="sequencing" title="wikilink">sequencing</a>. It is based on the differences between the <a href="Nucleic_acid_thermodynamics" title="wikilink">melting temperatures</a> of AT-rich and GC-rich regions.<a class="footnoteRef" href="#fn1" id="fnref1"><sup>1</sup></a> Even though modern <a class="uri" href="sequencing" title="wikilink">sequencing</a> methods reduced the need for denaturation mapping, it is still being used for specific purposes, such as detection of large scale <a href="structural_variation" title="wikilink">structural variants</a>.<a class="footnoteRef" href="#fn2" id="fnref2"><sup>2</sup></a></p>
<h2 id="methodology">Methodology</h2>

<p>When subjected to denaturing factors like increased heat or chemicals like <a class="uri" href="formamide" title="wikilink">formamide</a> in low levels, DNA is partially denatured in a predictable pattern based on its nucleotide content in different regions.<a class="footnoteRef" href="#fn3" id="fnref3"><sup>3</sup></a> This allows unique fingerprints or ‘barcodes' to be generated for molecules with different sequences not unlike restriction mapping. In the earliest forms of denaturation mapping, DNA was denatured by heating in presence of <a class="uri" href="formaldehyde" title="wikilink">formaldehyde</a> <a class="footnoteRef" href="#fn4" id="fnref4"><sup>4</sup></a> or <a class="uri" href="glyoxal" title="wikilink">glyoxal</a><a class="footnoteRef" href="#fn5" id="fnref5"><sup>5</sup></a> and visualized using electron microscopy. Dyes that selectively bind to double stranded DNA like <a href="ethidium_bromide" title="wikilink">ethidium bromide</a> could be used to monitor the extend of denaturation. But it was not possible to observe locations of denaturation based on this information. And requirement of electron microscopy made this method more strenuous to perform. More recently microfluidics were used for denaturation mapping of single molecules. In this method a reservoir that contains individual stained DNA molecules is placed next to nanochannels. Under pressure the molecules are pushed towards the nanochannels, and stretch out. With the application of heat in the presence of formamide the molecules denature and while in these channels, denaturation profile of these molecules can be observed by <a href="fluorescence_microscopy" title="wikilink">fluorescence microscopy</a>.<a class="footnoteRef" href="#fn6" id="fnref6"><sup>6</sup></a></p>
<h2 id="computational-prediction">Computational Prediction</h2>

<p>Because of the predictable nature of a denaturation map, given a sequence it is possible to computationally generate a candidate map with relatively high confidence based on the <a href="Poland-Sherga_model" title="wikilink">Poland-Sherga model</a>. This algorithm can predict the melting probability of a region for a certain temperature and salt concentration. Based on this probability, potential intensity of a region can be calculated as 

<math display="inline" id="Denaturation_mapping:0">
 <semantics>
  <mrow>
   <mrow>
    <mi>I</mi>
    <mrow>
     <mo stretchy="false">(</mo>
     <mi>s</mi>
     <mo stretchy="false">)</mo>
    </mrow>
   </mrow>
   <mo>=</mo>
   <mrow>
    <mrow>
     <msub>
      <mi>I</mi>
      <mrow>
       <mi>d</mi>
       <mi>s</mi>
      </mrow>
     </msub>
     <msub>
      <mi>p</mi>
      <mrow>
       <mi>d</mi>
       <mi>s</mi>
      </mrow>
     </msub>
     <mrow>
      <mo stretchy="false">(</mo>
      <mi>s</mi>
      <mo stretchy="false">)</mo>
     </mrow>
    </mrow>
    <mo>+</mo>
    <mrow>
     <msub>
      <mi>I</mi>
      <mrow>
       <mi>s</mi>
       <mi>s</mi>
      </mrow>
     </msub>
     <mo>*</mo>
     <mrow>
      <mo stretchy="false">[</mo>
      <mrow>
       <mn>1</mn>
       <mo>-</mo>
       <mrow>
        <msub>
         <mi>p</mi>
         <mrow>
          <mi>d</mi>
          <mi>s</mi>
         </mrow>
        </msub>
        <mrow>
         <mo stretchy="false">(</mo>
         <mi>s</mi>
         <mo stretchy="false">)</mo>
        </mrow>
       </mrow>
      </mrow>
      <mo stretchy="false">]</mo>
     </mrow>
    </mrow>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <eq></eq>
    <apply>
     <times></times>
     <ci>I</ci>
     <ci>s</ci>
    </apply>
    <apply>
     <plus></plus>
     <apply>
      <times></times>
      <apply>
       <csymbol cd="ambiguous">subscript</csymbol>
       <ci>I</ci>
       <apply>
        <times></times>
        <ci>d</ci>
        <ci>s</ci>
       </apply>
      </apply>
      <apply>
       <csymbol cd="ambiguous">subscript</csymbol>
       <ci>p</ci>
       <apply>
        <times></times>
        <ci>d</ci>
        <ci>s</ci>
       </apply>
      </apply>
      <ci>s</ci>
     </apply>
     <apply>
      <times></times>
      <apply>
       <csymbol cd="ambiguous">subscript</csymbol>
       <ci>I</ci>
       <apply>
        <times></times>
        <ci>s</ci>
        <ci>s</ci>
       </apply>
      </apply>
      <apply>
       <csymbol cd="latexml">delimited-[]</csymbol>
       <apply>
        <minus></minus>
        <cn type="integer">1</cn>
        <apply>
         <times></times>
         <apply>
          <csymbol cd="ambiguous">subscript</csymbol>
          <ci>p</ci>
          <apply>
           <times></times>
           <ci>d</ci>
           <ci>s</ci>
          </apply>
         </apply>
         <ci>s</ci>
        </apply>
       </apply>
      </apply>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   I(s)=I_{ds}p_{ds}(s)+I_{ss}*[1-p_{ds}(s)]
  </annotation>
 </semantics>
</math>

 where 

<math display="inline" id="Denaturation_mapping:1">
 <semantics>
  <msub>
   <mi>I</mi>
   <mrow>
    <mi>d</mi>
    <mi>s</mi>
   </mrow>
  </msub>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <csymbol cd="ambiguous">subscript</csymbol>
    <ci>I</ci>
    <apply>
     <times></times>
     <ci>d</ci>
     <ci>s</ci>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   I_{ds}
  </annotation>
 </semantics>
</math>

 and 

<math display="inline" id="Denaturation_mapping:2">
 <semantics>
  <msub>
   <mi>I</mi>
   <mrow>
    <mi>s</mi>
    <mi>s</mi>
   </mrow>
  </msub>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <csymbol cd="ambiguous">subscript</csymbol>
    <ci>I</ci>
    <apply>
     <times></times>
     <ci>s</ci>
     <ci>s</ci>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   I_{ss}
  </annotation>
 </semantics>
</math>

 are fluorescence intensities of double and single stranded regions respectively and 

<math display="inline" id="Denaturation_mapping:3">
 <semantics>
  <msub>
   <mi>p</mi>
   <mrow>
    <mi>d</mi>
    <mi>s</mi>
   </mrow>
  </msub>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <csymbol cd="ambiguous">subscript</csymbol>
    <ci>p</ci>
    <apply>
     <times></times>
     <ci>d</ci>
     <ci>s</ci>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   p_{ds}
  </annotation>
 </semantics>
</math>

 is the probability of DNA remaining double stranded.<a class="footnoteRef" href="#fn7" id="fnref7"><sup>7</sup></a></p>
<h2 id="uses">Uses</h2>

<p>Historically, this method was used to compare and analyze properties of a single sequenece or a group of sequences such as heterogeneity in the absence of sequencing. It was not uncommon to <a href="ribosomal_DNA" title="wikilink">ribosomal DNA</a> and compare the resulting maps of related organisms not unlike <a href="16S_ribosomal_RNA" title="wikilink">16s rRNA</a> identification method that is often used today.<a class="footnoteRef" href="#fn8" id="fnref8"><sup>8</sup></a> But development of <a href="DNA_sequencing" title="wikilink">DNA sequencing</a> techniques rendered this use of the method mostly obsolete.</p>

<p>Main advantage of denaturation mapping is that large-scale organization of the genome is left intact during the process. That means long-range structural variants can be detected easily.</p>

<p>For detection of large structural variants, it is possible to use this technique to make alignments on these barcodes that will enable one to determine the origins of fragments. For instance it was possible to demonstrate this on <a href="Enterobacteria_phage_T4" title="wikilink">T4GT7</a> DNA that circularly permutates in individual viruses. This was also shown fragments of human <a class="uri" href="chromosomes" title="wikilink">chromosomes</a>.<a class="footnoteRef" href="#fn9" id="fnref9"><sup>9</sup></a></p>

<p>Furthermore, as it is possible to profile and locate extremely long sequences (more than 100kb)<a class="footnoteRef" href="#fn10" id="fnref10"><sup>10</sup></a> in the genome in a consistent manner, fingerprints generated by denaturation sequencing can be used as a reference for de novo assembly of shotgun sequencing data which decreases the coverage steep depth requirements of de novo assembly algorithms. First round of de novo assembly produces hundreds to thousands of contigs, which can be computationally profiled in terms of their melting temperature. These profiles can be compared to results of an actual denaturation experiment to map the contigs.<a class="footnoteRef" href="#fn11" id="fnref11"><sup>11</sup></a> To this end, more recently it was shown that it is feasible to apply this method to large eukaryotic genomes with the mapping attempt on <a class="uri" href="yeast" title="wikilink">yeast</a><a class="footnoteRef" href="#fn12" id="fnref12"><sup>12</sup></a></p>

<p>Another recent application of denaturation of mapping is haplotype-phasing. Using a lab-on-a-chip approach, denaturated single molecules can be rescued,amplified and analyzed by FISH and conventional sequencing. This approach yields a complete picture of chromosome at both macro and micro levels, allowing researchers to associate large structural variations with haplotypes<a class="footnoteRef" href="#fn13" id="fnref13"><sup>13</sup></a></p>
<h2 id="references">References</h2>

<p>"</p>

<p><a class="uri" href="Category:DNA" title="wikilink">Category:DNA</a></p>
<section class="footnotes">
<hr/>
<ol>
<li id="fn1">Inman RB. (1966)."A denaturation map of the lamda phage DNA molecule determined by electron microscopy". Journal of Molecular Biology 18: pp. 464-476.<a href="#fnref1">↩</a></li>
<li id="fn2">Reisner W;Larsen N B; Silahtaroglu A; Kristensen A; Tommerup N; Tegenfeldt J O; Flyvbjerg H. (2010). "Single-molecule denaturation mapping of DNA in nanofluidic channels". PNAS. doi: 10.1073<a href="#fnref2">↩</a></li>
<li id="fn3"></li>
<li id="fn4"></li>
<li id="fn5">Johnson D. (1975)."A new method of DNA denaturation mapping". Nucleid Acids Research 2(11): pp. 2049-2053.<a href="#fnref5">↩</a></li>
<li id="fn6"></li>
<li id="fn7"></li>
<li id="fn8">Cramer J H; Rownd R H. (1980) “Denaturation mapping of ribosomal DNA of Saccharomyces cerevisiae”. Molecular Genetics and Genomics 177: p. 199-205<a href="#fnref8">↩</a></li>
<li id="fn9"></li>
<li id="fn10">Welch R L; Sladek R; Dewar K; Reisner W W. (2012) “Denaturation mapping of Saccharomyces cerevisiae”, Lab Chip 12:p. 3314-3321. DOI: 10.1039/c2lc40504k<a href="#fnref10">↩</a></li>
<li id="fn11"></li>
<li id="fn12"></li>
<li id="fn13">Marie R; Pedersen J N; Bauer D L V; Rasmussen K H; Yusuh M; Volpi E; Flyvjberg H; Kristensen A; Mir K U. (2013) “Integrated view of genome structure and sequence of a single DNA molecule in a nanofludic device". Proc Natl Acad Sci U S A. 110(13):p. 4893-4898<a href="#fnref13">↩</a></li>
</ol>
</section>
</body>
</html>
