<html lang="en">
<head>
<meta charset="utf-8"/>
<title offset="1068">DNA-Functionalized Quantum Dots</title>
<script src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_SVG.js" type="text/javascript">
</script>
</head>
<body>
<h1>DNA-Functionalized Quantum Dots</h1>
<hr>'''DNA-functionalization of [[quantum dot]]s''' is the attachment of strands of [[DNA]] to the surface of a quantum dot. Although quantum dots with Cd have some cytotoxic release, researchers have functionalized quantum dots for biocompatibility and bound them to DNA in order to combine the advantages of both materials. Quantum dots are commonly used for imaging biological systems ''in vitro'' and ''in vivo'' in animal studies due to their excellent optical properties when excited by light, while DNA has numerous bioengineering applications, including: genetic engineering, self-assembling nanostructures, protein bind
<p>ing, and <a href="Biomarker_(medicine)" title="wikilink">biomarkers</a>. The ability to visualize the chemical and biological processes of DNA allows feedback to optimize and learn about these small scale behaviors.<a class="footnoteRef" href="#fn1" id="fnref1"><sup>1</sup></a><a class="footnoteRef" href="#fn2" id="fnref2"><sup>2</sup></a> </p>
<h2 id="background">Background</h2>

<p>Quantum dots are inorganic nanocrystal <a href="semiconductor" title="wikilink">semiconductors</a> that behave exceptionally well as <a href="fluorophore" title="wikilink">fluorophores</a>. In the field of biology, fluorophores are one of the few tools that allow us to peer inside of a live biological system at a cellular level. As a fluorophore, the size of a quantum dot directly reflects the wavelength of light emitted, allowing for a highly tunable color spectrum. Since the size of quantum dots are controllable and an increased size produces an increased wavelength range of emission, researchers are able to paint pictures on the cellular and sub-cellular levels with this technology. The current problem with common CdSe-ZnS quantum dots is that Cd is toxic to cells <a class="footnoteRef" href="#fn3" id="fnref3"><sup>3</sup></a>  To prevent this problem researches are developing ways to modify the quantum dot surfaces for biocompatibility, in addition to the development of Cd-free quantum dots (“CFQDs”). After a <a href="surface_modification" title="wikilink">surface modification</a> has been made to limit toxicity, the particle can be further coated with a <a class="uri" href="hydrogel" title="wikilink">hydrogel</a> or <a class="uri" href="bioconjugate" title="wikilink">bioconjugate</a> layer to selectively bind to DNA, which may then be used for cellular or molecular level detection.<a class="footnoteRef" href="#fn4" id="fnref4"><sup>4</sup></a></p>
<figure><b>(Figure)</b>
<figcaption><strong>Figure 3:</strong> Schematic of a quantum dot coated with an organic protein bound to DNA</figcaption>
</figure>
<h2 id="surface-modification-methods">Surface Modification Methods</h2>
<h3 id="hydrogel-encapsulation-of-quantum-dots">Hydrogel encapsulation of quantum dots</h3>

<p>In order to coat the toxic Cadmium ions of the CdSe core, hydrogel layers may be used to coat quantum dots for biocompatibility. The purpose of the outer ZnS shell in this case is to interact with <a href="dangling_bond" title="wikilink">dangling bonds</a>, in addition to maintaining the fluorescent strength of a functional quantum dot fluorophore. Within a <a href="Hydrogel_encapsulation_of_quantum_dots" title="wikilink">hydrogel encapsulation</a>, the ZnSe shell surface may be charged to bind to the hydrophobic interior of a micelle, which then allows the hydrophilic exterior to remain in contact with an aqueous solution (i.e. the human body and most other biological systems). The hydrogel layer works as a simplified intermediary bond for DNA or other organic materials.</p>
<h3 id="bioconjugation-of-quantum-dots">Bioconjugation of Quantum Dots</h3>

<p>Another surface modification type is bioconjugation. This method uses two biomolecules which are covalently bonded to one another to form a protective shell around the quantum dot. Hydrophobic bioconjugation inhibits the breakdown of the quantum dot structure by sources within the body that may cause degradation. The bioconjugates can be further customized by attaching affinity ligands to the surface of the structure. These <a href="Ligand_(biochemistry)" title="wikilink">ligands</a> allow the quantum dot to bind to various antigens and can be used to specifically target certain cells. This is the driving mechanism for tumor targeting.</p>

<p>Core-shell CdSe-ZnS quantum dots can be protected through bioconjugation, using a coordinating ligand and an amphiphilic polymer. One study used <a href="tri-n-octylphosphine_oxide" title="wikilink">tri-n-octylphosphine oxide</a> (TOPO) as a ligand, and a triblock polymer structure consisting of two hydrophobic segments, and one hydrophilic segment, all with hydrophobic hydrocarbon side chains. The strong hydrophobic interactions between the TOPO and polymer hydrocarbon allow the two layers to “bond” to one another, forming a hydrophobic protection structure. This structure resists degradation via hydrolysis and enzymes, which are common methods of degradation <em>in vivo</em>. This bioconjugation layer protects quantum dot optical properties in a wide range of pH (1-14), salt conditions (0.01-1.0M), and even after 1.0M hydrochloric acid treatment.<a class="footnoteRef" href="#fn5" id="fnref5"><sup>5</sup></a></p>
<h3 id="carboxyl-attachments">Carboxyl attachments</h3>

<p> Carboxyl groups can be immobilized on the surface of a quantum dot coated in zinc sulfide. Single strands of DNA can then be modified with an added amino group in order to covalently bond to the carboxyl group, due to an amide bond formed between the carboxyl and amino groups in the presence of 1-ethyl-3-(3-dimethylaminopropyl)-carbodiimide (EDC).<a class="footnoteRef" href="#fn6" id="fnref6"><sup>6</sup></a> Factors that can influence the binding of single stranded DNA to the carboxyl group are pH and ionic strength. The pH determines how many protons are available to form covalent bonds, with less being present as the pH gets higher. This results in less strands of DNA binding to each quantum dot. Lower ionic strength results more stable quantum dots, but also causes DNA strands to repel each other. Optimum coupling conditions for over 10 DNA strands per quantum dot are at a pH of 7 and an ionic strength of 0M.<a class="footnoteRef" href="#fn7" id="fnref7"><sup>7</sup></a> A neutral pH of 7 allows enough protons from the carboxyl group to facilitate covalent bonding of amino modified DNA, but not enough protons to destabilize the colloids</p>
<h2 id="intermolecular-forces">Intermolecular Forces</h2>

<p>The addition of DNA to the surface of a quantum dot changes the intermolecular forces that occur between un-conjugated quantum dots. Changing the intermolecular forces between the quantum dots can alter many characteristics important for the use of quantum dots in aqueous conditions. As the surface of quantum dots are conjugated with DNA, the colloidal stability and solubility are affected.</p>
<h3 id="colloidal-stability">Colloidal stability</h3>

<p>Quantum dots conjugated with DNA are subject to electrostatic repulsion and Van der Waals forces which affect the colloidal stability of the quantum dot-DNA conjugates. Binding DNA to the surface of a quantum dot increases the stability of the quantum dots. The DNA chains provide more electrostatic repulsion than the surface of the quantum dots, which prevents them from aggregating and falling out of solution. The colloidal stability is estimated from the total interaction energy between two particles which calculated by the <a href="DLVO_theory" title="wikilink">DLVO</a> equation<a class="footnoteRef" href="#fn8" id="fnref8"><sup>8</sup></a> 

<math display="inline" id="DNA-Functionalized_Quantum_Dots:0">
 <semantics>
  <mrow>
   <msub>
    <mpadded lspace="5pt" width="+5pt">
     <mi>V</mi>
    </mpadded>
    <mi>t</mi>
   </msub>
   <mo>=</mo>
   <mrow>
    <msub>
     <mi>V</mi>
     <mrow>
      <mi>v</mi>
      <mi>d</mi>
      <mi>W</mi>
     </mrow>
    </msub>
    <mo>+</mo>
    <msub>
     <mi>V</mi>
     <mrow>
      <mi>e</mi>
      <mi>s</mi>
     </mrow>
    </msub>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <eq></eq>
    <apply>
     <csymbol cd="ambiguous">subscript</csymbol>
     <ci>V</ci>
     <ci>t</ci>
    </apply>
    <apply>
     <plus></plus>
     <apply>
      <csymbol cd="ambiguous">subscript</csymbol>
      <ci>V</ci>
      <apply>
       <times></times>
       <ci>v</ci>
       <ci>d</ci>
       <ci>W</ci>
      </apply>
     </apply>
     <apply>
      <csymbol cd="ambiguous">subscript</csymbol>
      <ci>V</ci>
      <apply>
       <times></times>
       <ci>e</ci>
       <ci>s</ci>
      </apply>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \ V_{t}=V_{vdW}+V_{es}
  </annotation>
 </semantics>
</math>

</p>

<p>V<sub>es</sub> is the electrostatic repulsion forces between two identical spherical particles the from the <a href="double_layer_(interfacial)#Electrical_double_layers" title="wikilink">electric double layer</a> of each particle. It is calculated with the equation<a class="footnoteRef" href="#fn9" id="fnref9"><sup>9</sup></a> 

<math display="inline" id="DNA-Functionalized_Quantum_Dots:1">
 <semantics>
  <mrow>
   <msub>
    <mpadded lspace="5pt" width="+5pt">
     <mi>V</mi>
    </mpadded>
    <mrow>
     <mi>e</mi>
     <mi>s</mi>
    </mrow>
   </msub>
   <mo>=</mo>
   <mrow>
    <mn>2</mn>
    <mi>π</mi>
    <msub>
     <mi>ϵ</mi>
     <mn>0</mn>
    </msub>
    <mi>ϵ</mi>
    <msub>
     <mi>R</mi>
     <mi>P</mi>
    </msub>
    <msup>
     <mi>ψ</mi>
     <mn>2</mn>
    </msup>
    <mi>l</mi>
    <mi>n</mi>
    <mrow>
     <mo stretchy="false">[</mo>
     <mrow>
      <mn>1</mn>
      <mo>+</mo>
      <mrow>
       <mi>e</mi>
       <mi>x</mi>
       <mi>p</mi>
       <mrow>
        <mo stretchy="false">(</mo>
        <mrow>
         <mi>κ</mi>
         <mi>h</mi>
        </mrow>
        <mo stretchy="false">)</mo>
       </mrow>
      </mrow>
     </mrow>
     <mo stretchy="false">]</mo>
    </mrow>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <eq></eq>
    <apply>
     <csymbol cd="ambiguous">subscript</csymbol>
     <ci>V</ci>
     <apply>
      <times></times>
      <ci>e</ci>
      <ci>s</ci>
     </apply>
    </apply>
    <apply>
     <times></times>
     <cn type="integer">2</cn>
     <ci>π</ci>
     <apply>
      <csymbol cd="ambiguous">subscript</csymbol>
      <ci>ϵ</ci>
      <cn type="integer">0</cn>
     </apply>
     <ci>ϵ</ci>
     <apply>
      <csymbol cd="ambiguous">subscript</csymbol>
      <ci>R</ci>
      <ci>P</ci>
     </apply>
     <apply>
      <csymbol cd="ambiguous">superscript</csymbol>
      <ci>ψ</ci>
      <cn type="integer">2</cn>
     </apply>
     <ci>l</ci>
     <ci>n</ci>
     <apply>
      <csymbol cd="latexml">delimited-[]</csymbol>
      <apply>
       <plus></plus>
       <cn type="integer">1</cn>
       <apply>
        <times></times>
        <ci>e</ci>
        <ci>x</ci>
        <ci>p</ci>
        <apply>
         <times></times>
         <ci>κ</ci>
         <ci>h</ci>
        </apply>
       </apply>
      </apply>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \ V_{es}=2\pi\epsilon_{0}\epsilon R_{P}\psi^{2}ln[1+exp(\kappa h)]
  </annotation>
 </semantics>
</math>

</p>

<p>Where:</p>
<ul>
<li>

<math display="inline" id="DNA-Functionalized_Quantum_Dots:2">
 <semantics>
  <mi>h</mi>
  <annotation-xml encoding="MathML-Content">
   <ci>h</ci>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   h
  </annotation>
 </semantics>
</math>

 is separation between two particles</li>
<li>

<math display="inline" id="DNA-Functionalized_Quantum_Dots:3">
 <semantics>
  <msub>
   <mi>R</mi>
   <mi>P</mi>
  </msub>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <csymbol cd="ambiguous">subscript</csymbol>
    <ci>R</ci>
    <ci>P</ci>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   R_{P}
  </annotation>
 </semantics>
</math>

 is the radius of the particles</li>
<li>

<math display="inline" id="DNA-Functionalized_Quantum_Dots:4">
 <semantics>
  <mrow>
   <mi>ϵ</mi>
   <msub>
    <mi>ϵ</mi>
    <mn>0</mn>
   </msub>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <times></times>
    <ci>ϵ</ci>
    <apply>
     <csymbol cd="ambiguous">subscript</csymbol>
     <ci>ϵ</ci>
     <cn type="integer">0</cn>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \epsilon\epsilon_{0}
  </annotation>
 </semantics>
</math>

 is the <a class="uri" href="permittivity" title="wikilink">permittivity</a> of water</li>
<li>

<math display="inline" id="DNA-Functionalized_Quantum_Dots:5">
 <semantics>
  <mi>ψ</mi>
  <annotation-xml encoding="MathML-Content">
   <ci>ψ</ci>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \psi
  </annotation>
 </semantics>
</math>

 is the surface potential</li>
<li>

<math display="inline" id="DNA-Functionalized_Quantum_Dots:6">
 <semantics>
  <mi>κ</mi>
  <annotation-xml encoding="MathML-Content">
   <ci>κ</ci>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \kappa
  </annotation>
 </semantics>
</math>

 is the inverse <a href="Debye_length" title="wikilink">Debye length</a></li>
</ul>

<p>V<sub>vdW</sub> is the attractive force between all particles. The Van der Waals forces are calculated with the equation<a class="footnoteRef" href="#fn10" id="fnref10"><sup>10</sup></a> 

<math display="inline" id="DNA-Functionalized_Quantum_Dots:7">
 <semantics>
  <mrow>
   <msub>
    <mpadded lspace="5pt" width="+5pt">
     <mi>V</mi>
    </mpadded>
    <mrow>
     <mi>v</mi>
     <mi>d</mi>
     <mi>W</mi>
    </mrow>
   </msub>
   <mo>=</mo>
   <mrow>
    <mfrac>
     <msub>
      <mi>A</mi>
      <mi>H</mi>
     </msub>
     <mn>6</mn>
    </mfrac>
    <mrow>
     <mo stretchy="false">[</mo>
     <mrow>
      <mrow>
       <mi>l</mi>
       <mi>n</mi>
       <mfrac>
        <mrow>
         <mi>h</mi>
         <mrow>
          <mo stretchy="false">(</mo>
          <mrow>
           <mrow>
            <mn>4</mn>
            <msub>
             <mi>R</mi>
             <mi>P</mi>
            </msub>
           </mrow>
           <mo>+</mo>
           <mi>h</mi>
          </mrow>
          <mo stretchy="false">)</mo>
         </mrow>
        </mrow>
        <msup>
         <mrow>
          <mo stretchy="false">(</mo>
          <mrow>
           <mrow>
            <mn>2</mn>
            <msub>
             <mi>R</mi>
             <mi>P</mi>
            </msub>
           </mrow>
           <mo>+</mo>
           <mi>h</mi>
          </mrow>
          <mo stretchy="false">)</mo>
         </mrow>
         <mn>2</mn>
        </msup>
       </mfrac>
      </mrow>
      <mo>+</mo>
      <mfrac>
       <mrow>
        <mo stretchy="false">(</mo>
        <mrow>
         <mn>2</mn>
         <msubsup>
          <mi>R</mi>
          <mi>P</mi>
          <mn>2</mn>
         </msubsup>
        </mrow>
        <mo stretchy="false">)</mo>
       </mrow>
       <mrow>
        <mi>h</mi>
        <mrow>
         <mo stretchy="false">(</mo>
         <mrow>
          <mrow>
           <mn>4</mn>
           <msub>
            <mi>R</mi>
            <mi>P</mi>
           </msub>
          </mrow>
          <mo>+</mo>
          <mi>h</mi>
         </mrow>
         <mo stretchy="false">)</mo>
        </mrow>
       </mrow>
      </mfrac>
      <mo>+</mo>
      <mfrac>
       <mrow>
        <mo stretchy="false">(</mo>
        <mrow>
         <mn>2</mn>
         <msubsup>
          <mi>R</mi>
          <mi>P</mi>
          <mn>2</mn>
         </msubsup>
        </mrow>
        <mo stretchy="false">)</mo>
       </mrow>
       <msup>
        <mrow>
         <mo stretchy="false">(</mo>
         <mrow>
          <mrow>
           <mn>2</mn>
           <msub>
            <mi>R</mi>
            <mi>P</mi>
           </msub>
          </mrow>
          <mo>+</mo>
          <mi>h</mi>
         </mrow>
         <mo stretchy="false">)</mo>
        </mrow>
        <mn>2</mn>
       </msup>
      </mfrac>
     </mrow>
     <mo stretchy="false">]</mo>
    </mrow>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <eq></eq>
    <apply>
     <csymbol cd="ambiguous">subscript</csymbol>
     <ci>V</ci>
     <apply>
      <times></times>
      <ci>v</ci>
      <ci>d</ci>
      <ci>W</ci>
     </apply>
    </apply>
    <apply>
     <times></times>
     <apply>
      <divide></divide>
      <apply>
       <csymbol cd="ambiguous">subscript</csymbol>
       <ci>A</ci>
       <ci>H</ci>
      </apply>
      <cn type="integer">6</cn>
     </apply>
     <apply>
      <csymbol cd="latexml">delimited-[]</csymbol>
      <apply>
       <plus></plus>
       <apply>
        <times></times>
        <ci>l</ci>
        <ci>n</ci>
        <apply>
         <divide></divide>
         <apply>
          <times></times>
          <ci>h</ci>
          <apply>
           <plus></plus>
           <apply>
            <times></times>
            <cn type="integer">4</cn>
            <apply>
             <csymbol cd="ambiguous">subscript</csymbol>
             <ci>R</ci>
             <ci>P</ci>
            </apply>
           </apply>
           <ci>h</ci>
          </apply>
         </apply>
         <apply>
          <csymbol cd="ambiguous">superscript</csymbol>
          <apply>
           <plus></plus>
           <apply>
            <times></times>
            <cn type="integer">2</cn>
            <apply>
             <csymbol cd="ambiguous">subscript</csymbol>
             <ci>R</ci>
             <ci>P</ci>
            </apply>
           </apply>
           <ci>h</ci>
          </apply>
          <cn type="integer">2</cn>
         </apply>
        </apply>
       </apply>
       <apply>
        <divide></divide>
        <apply>
         <times></times>
         <cn type="integer">2</cn>
         <apply>
          <csymbol cd="ambiguous">superscript</csymbol>
          <apply>
           <csymbol cd="ambiguous">subscript</csymbol>
           <ci>R</ci>
           <ci>P</ci>
          </apply>
          <cn type="integer">2</cn>
         </apply>
        </apply>
        <apply>
         <times></times>
         <ci>h</ci>
         <apply>
          <plus></plus>
          <apply>
           <times></times>
           <cn type="integer">4</cn>
           <apply>
            <csymbol cd="ambiguous">subscript</csymbol>
            <ci>R</ci>
            <ci>P</ci>
           </apply>
          </apply>
          <ci>h</ci>
         </apply>
        </apply>
       </apply>
       <apply>
        <divide></divide>
        <apply>
         <times></times>
         <cn type="integer">2</cn>
         <apply>
          <csymbol cd="ambiguous">superscript</csymbol>
          <apply>
           <csymbol cd="ambiguous">subscript</csymbol>
           <ci>R</ci>
           <ci>P</ci>
          </apply>
          <cn type="integer">2</cn>
         </apply>
        </apply>
        <apply>
         <csymbol cd="ambiguous">superscript</csymbol>
         <apply>
          <plus></plus>
          <apply>
           <times></times>
           <cn type="integer">2</cn>
           <apply>
            <csymbol cd="ambiguous">subscript</csymbol>
            <ci>R</ci>
            <ci>P</ci>
           </apply>
          </apply>
          <ci>h</ci>
         </apply>
         <cn type="integer">2</cn>
        </apply>
       </apply>
      </apply>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \ V_{vdW}={A_{H}\over 6}[ln{h(4R_{P}+h)\over(2R_{P}+h)^{2}}+{(2R_{P}^{2})\over
h%
(4R_{P}+h)}+{(2R_{P}^{2})\over(2R_{P}+h)^{2}}]
  </annotation>
 </semantics>
</math>

</p>

<p>Where</p>
<ul>
<li>

<math display="inline" id="DNA-Functionalized_Quantum_Dots:8">
 <semantics>
  <msub>
   <mi>A</mi>
   <mi>H</mi>
  </msub>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <csymbol cd="ambiguous">subscript</csymbol>
    <ci>A</ci>
    <ci>H</ci>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   A_{H}
  </annotation>
 </semantics>
</math>

 is the effective <a href="Hamaker_constant" title="wikilink">Hamaker constant</a></li>
</ul>

<p>The colloidal stability of the quantum dots can differ with changing pH and ionic strength. Overall, the DNA conjugation increases the stability of the quantum dots by providing electrostatic and steric repulsion, which prevents the particles from aggregating due to van der Waals forces.<a class="footnoteRef" href="#fn11" id="fnref11"><sup>11</sup></a></p>
<h3 id="solubility">Solubility</h3>

<p>In order to use quantum dots in many biology related applications, the quantum dots must be soluble in aqueous environments. For quantum dots to be solubilized in water, the amphiphilic ligands must be on the surface of quantum dots. DNA can be used as a solubilization ligand due to its amphiphilic nature.<a class="footnoteRef" href="#fn12" id="fnref12"><sup>12</sup></a> This allows quantum dots functionalized with DNA to be used in the aqueous conditions often found in biology and medical research. The increased solubility is necessary in order to allow quantum dots to be used as a DNA imaging probe in a biological system.</p>
<h2 id="applications">Applications</h2>

<p>Quantum dots have become powerful imaging tools and are continually evolving for biocompatibility in the hopes of successfully imaging humans and other live biological systems. By reducing the amount of Cd released around cells, researchers have been striving to create <em>in vitro</em> and <em>in vivo</em> testing methods for imaging nano- and micro-scaled structures. The high resolution within the nanometer range demonstrates the usefulness for imaging DNA behavior for both bioengineering feedback and biological and chemical observation and analysis. The ability to control the emission spectra by altering the size of the quantum dots allows researchers to code many different targets by color.<a class="footnoteRef" href="#fn13" id="fnref13"><sup>13</sup></a></p>
<table>
<tbody>
<tr class="odd">
<td style="text-align: left;">
<p>''' Size (nm) '''</p></td>
<td style="text-align: left;">
<p>''' Emission Peak (nm) '''</p></td>
<td style="text-align: left;">
<p>''' Color '''</p></td>
</tr>
<tr class="even">
<td style="text-align: left;">
<p>2.2 <a class="footnoteRef" href="#fn14" id="fnref14"><sup>14</sup></a></p></td>
<td style="text-align: left;">
<p>495</p></td>
<td style="text-align: left;">
<p>blue</p></td>
</tr>
<tr class="odd">
<td style="text-align: left;">
<p>2.9 <a class="footnoteRef" href="#fn15" id="fnref15"><sup>15</sup></a></p></td>
<td style="text-align: left;">
<p>550</p></td>
<td style="text-align: left;">
<p>green</p></td>
</tr>
<tr class="even">
<td style="text-align: left;">
<p>3.1 <a class="footnoteRef" href="#fn16" id="fnref16"><sup>16</sup></a></p></td>
<td style="text-align: left;">
<p>|576</p></td>
<td style="text-align: left;">
<p>yellow</p></td>
</tr>
<tr class="odd">
<td style="text-align: left;">
<p>4.1 <a class="footnoteRef" href="#fn17" id="fnref17"><sup>17</sup></a></p></td>
<td style="text-align: left;">
<p>595</p></td>
<td style="text-align: left;">
<p>orange</p></td>
</tr>
<tr class="even">
<td style="text-align: left;">
<p>4.4 <ref>{{cite journal | title= Luminescent quantum dots for multiplexed biological detection and imaging | journal = Current Opinion in Biotechnology</ref></p></td>
<td style="text-align: left;">
<p>year = 2002 | volume = 13| pages= 40–46 | doi =10.1016/S0958-1669(02)00282-3 | author1 = Warren C.W Chan | author2 = Dustin J Maxwell | author3 =Xiaohu Gao }}</p></td>
<td style="text-align: left;">
<p>611</p></td>
</tr>
<tr class="odd">
<td style="text-align: left;">
<p>4.8<a class="footnoteRef" href="#fn18" id="fnref18"><sup>18</sup></a></p></td>
<td style="text-align: left;">
<p>644</p></td>
<td style="text-align: left;">
<p>red</p></td>
</tr>
<tr class="even">
<td style="text-align: left;">
<p>7.3 <a class="footnoteRef" href="#fn19" id="fnref19"><sup>19</sup></a></p></td>
<td style="text-align: left;">
<p>655</p></td>
<td style="text-align: left;">
<p>dark red</p></td>
</tr>
</tbody>
</table>
<h3 id="quantifying-and-imaging-gene-expression">Quantifying and imaging gene expression</h3>

<p>Since quantum dots have high photostability and luminescence, researchers are using them to light up mRNA within cells to image gene expression. Amine-modified oligonucleotide probes attached to carboxyl groups on quantum dots show sequence-specific hybridization. These probes can also detect low expressing genes.<a class="footnoteRef" href="#fn20" id="fnref20"><sup>20</sup></a> This potentially allows researchers to understand when and where certain proteins are made.</p>
<h3 id="self-assembling-nanostructures">Self-assembling nanostructures</h3>

<p>Self-assembled quantum dots form spontaneously under specific conditions during molecular beam epitaxy or another form of atomic deposition. This spontaneous formation is a consequence of the lattice-mismatch between the deposited semiconductor material and the underlying substrate. The resulting structure formed on the substrate surface is a three-dimensional "island" nanostructure. The islands are formed into quantum dots by covering them with another semiconductor material, in a process called quantum confinement.<a class="footnoteRef" href="#fn21" id="fnref21"><sup>21</sup></a><a class="footnoteRef" href="#fn22" id="fnref22"><sup>22</sup></a> Self-Assembled quantum dots provide opportunities in technological applications such as quantum cryptography, quantum computing, optics and optoelectronics.<a class="footnoteRef" href="#fn23" id="fnref23"><sup>23</sup></a></p>
<h3 id="single-molecule-imaging">Single molecule imaging</h3>

<p>In the past, Green Fluorescent Protein (GFP) was used to track movement inside cells. However, GFP does not light up well and is unstable after application. Thus, GFP prevented long term studies of protein movement. By using quantum dots, which are more stable, researchers can now track proteins through cells undergoing different pathways.<a class="footnoteRef" href="#fn24" id="fnref24"><sup>24</sup></a> In order to overcome the inability for the cameras to capture depth, researchers have developed a 3D tracking apparatus that can accurately map the path of proteins inside cells.<a class="footnoteRef" href="#fn25" id="fnref25"><sup>25</sup></a></p>
<h3 id="live-time-protein-tracking">Live-time protein tracking</h3>

<p>Because quantum dots have fine-tuned wavelength spectrums along with high emission intensities and small size, quantum dots have become the norm for molecule tracking. However, quantum dots have two levels: bright and dark. For low quantities, this is a problem as researchers need to retrace where the molecule went during the dark stage, which can vary from several milliseconds to hours. The blinking phenomenon is not a problem when imaging larger objects (i.e. tumors) as there will be enough quantum dots in the bright state to image, even while several may remain in their dark stage.<a class="footnoteRef" href="#fn26" id="fnref26"><sup>26</sup></a></p>
<h2 id="see-also">See also</h2>
<ul>
<li><a class="uri" href="DNA" title="wikilink">DNA</a></li>
<li><a href="Quantum_Dots" title="wikilink">Quantum Dots</a></li>
<li><a class="uri" href="Hydrogel" title="wikilink">Hydrogel</a></li>
<li><a href="Hydrogel_encapsulation_of_quantum_dots" title="wikilink">Hydrogel encapsulation of quantum dots</a></li>
<li><a class="uri" href="Biocompatibility" title="wikilink">Biocompatibility</a></li>
</ul>
<h2 id="references">References</h2>

<p>"</p>

<p><a class="uri" href="Category:DNA" title="wikilink">Category:DNA</a> <a class="uri" href="Category:Nanomaterials" title="wikilink">Category:Nanomaterials</a></p>
<section class="footnotes">
<hr/>
<ol>
<li id="fn1"><a href="#fnref1">↩</a></li>
<li id="fn2"><a href="#fnref2">↩</a></li>
<li id="fn3"><a href="#fnref3">↩</a></li>
<li id="fn4"><a href="#fnref4">↩</a></li>
<li id="fn5"><a href="#fnref5">↩</a></li>
<li id="fn6"><a href="#fnref6">↩</a></li>
<li id="fn7"><a href="#fnref7">↩</a></li>
<li id="fn8"><a href="#fnref8">↩</a></li>
<li id="fn9"><a href="#fnref9">↩</a></li>
<li id="fn10"><a href="#fnref10">↩</a></li>
<li id="fn11"><a href="#fnref11">↩</a></li>
<li id="fn12"><a href="#fnref12">↩</a></li>
<li id="fn13"><a href="#fnref13">↩</a></li>
<li id="fn14"><a href="#fnref14">↩</a></li>
<li id="fn15"></li>
<li id="fn16"><a href="#fnref16">↩</a></li>
<li id="fn17"></li>
<li id="fn18"></li>
<li id="fn19"></li>
<li id="fn20"><a href="#fnref20">↩</a></li>
<li id="fn21"><a href="#fnref21">↩</a></li>
<li id="fn22"><a href="#fnref22">↩</a></li>
<li id="fn23"></li>
<li id="fn24"><a href="#fnref24">↩</a></li>
<li id="fn25"><a href="#fnref25">↩</a></li>
<li id="fn26"><a href="#fnref26">↩</a></li>
</ol>
</section>
</hr></body>
</html>
