<html lang="en">
<head>
<meta charset="utf-8"/>
<title offset="1171">DNA sequencing theory</title>
<script src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_SVG.js" type="text/javascript">
</script>
</head>
<body>
<h1>DNA sequencing theory</h1>
<hr>'''DNA sequencing theory''' is the broad body of work that attempts to lay analytical foundations for determining the order of specific [[nucleotide]]s in a sequence of [[DNA]], otherwise known as [[DNA sequencing]]. The practical aspects revolve around designing and optimizing sequencing projects (known as "strategic genomics"), predicting project performance, troubleshooting experimental results, characterizing factors such as sequence bias and the effects of software processing algorithms, and comparing various sequencing 
<p>methods to one another. In this sense, it could be considered a branch of <a href="systems_engineering" title="wikilink">systems engineering</a> or <a href="operations_research" title="wikilink">operations research</a>. The permanent archive of work is primarily mathematical, although numerical calculations are often conducted for particular problems too. DNA sequencing theory addresses <em>physical processes</em> related to sequencing DNA and should not be confused with theories of analyzing resultant DNA sequences, e.g. <a href="sequence_alignment" title="wikilink">sequence alignment</a>. Publications<a class="footnoteRef" href="#fn1" id="fnref1"><sup>1</sup></a> sometimes do not make a careful distinction, but the latter are primarily concerned with <a href="algorithm" title="wikilink">algorithmic</a> issues. Sequencing theory is based on elements of <a class="uri" href="mathematics" title="wikilink">mathematics</a>, <a class="uri" href="biology" title="wikilink">biology</a>, and <a href="systems_engineering" title="wikilink">systems engineering</a>, so it is highly interdisciplinary. The subject may be studied within the context of <a href="computational_biology" title="wikilink">computational biology</a>.</p>
<h2 id="theory-and-sequencing-strategies">Theory and sequencing strategies</h2>
<h3 id="sequencing-as-a-covering-problem">Sequencing as a covering problem</h3>

<p>All mainstream methods of <a href="DNA_sequencing" title="wikilink">DNA sequencing</a> rely on reading small fragments of DNA and subsequently reconstructing these data to infer the original DNA target, either via <a href="sequence_assembly" title="wikilink">assembly</a> or <a href="Sequence_alignment" title="wikilink">alignment</a> to a reference. The <a class="uri" href="abstraction" title="wikilink">abstraction</a> common to these methods is that of a mathematical <a href="Cover_(topology)" title="wikilink">covering problem</a>.<a class="footnoteRef" href="#fn2" id="fnref2"><sup>2</sup></a> For example, one can imagine a line segment representing the target and a subsequent process where smaller segments are "dropped" onto random locations of the target. The target is considered "sequenced" when adequate coverage accumulates (e.g., when no gaps remain).</p>

<p>The abstract properties of covering have been studied by mathematicians for over a century.<a class="footnoteRef" href="#fn3" id="fnref3"><sup>3</sup></a> However, direct application of these results has not generally been possible. Closed-form mathematical solutions, especially for probability distributions, often cannot be readily evaluated. That is, they involve inordinately large amounts of computer time for parameters characteristic of <a href="DNA_sequencing" title="wikilink">DNA sequencing</a>. Stevens' configuration is one such example.<a class="footnoteRef" href="#fn4" id="fnref4"><sup>4</sup></a> Results obtained from the perspective of <a href="pure_mathematics" title="wikilink">pure mathematics</a> also do not account for factors that are actually important in sequencing, for instance detectable overlap in sequencing fragments, double-stranding, edge-effects, and target multiplicity. Consequently, development of sequencing theory has proceeded more according to the philosophy of <a href="applied_mathematics" title="wikilink">applied mathematics</a>. In particular, it has been problem-focused and makes expedient use of approximations, simulations, etc.</p>
<h3 id="early-uses-derived-from-elementary-probability-theory">Early uses derived from elementary probability theory</h3>

<p>The earliest result was actually borrowed directly from elementary probability theory. If we model the above process and take 

<math display="inline" id="DNA_sequencing_theory:0">
 <semantics>
  <mi>L</mi>
  <annotation-xml encoding="MathML-Content">
   <ci>L</ci>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   L
  </annotation>
 </semantics>
</math>

 and 

<math display="inline" id="DNA_sequencing_theory:1">
 <semantics>
  <mi>G</mi>
  <annotation-xml encoding="MathML-Content">
   <ci>G</ci>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   G
  </annotation>
 </semantics>
</math>

 as the fragment length and target length, respectively, then the probability of "covering" any given location on the target <em>with one particular fragment</em> is 

<math display="inline" id="DNA_sequencing_theory:2">
 <semantics>
  <mrow>
   <mi>L</mi>
   <mo>/</mo>
   <mi>G</mi>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <divide></divide>
    <ci>L</ci>
    <ci>G</ci>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   L/G
  </annotation>
 </semantics>
</math>

. (Note that this presumes 

<math display="inline" id="DNA_sequencing_theory:3">
 <semantics>
  <mrow>
   <mi>L</mi>
   <mo>≪</mo>
   <mi>G</mi>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <csymbol cd="latexml">much-less-than</csymbol>
    <ci>L</ci>
    <ci>G</ci>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   L\ll G
  </annotation>
 </semantics>
</math>


, which is valid for many, though not all sequencing scenarios). The probability of <strong>not</strong> covering a given location on the target is therefore 

<math display="inline" id="DNA_sequencing_theory:4">
 <semantics>
  <mrow>
   <mn>1</mn>
   <mo>-</mo>
   <mrow>
    <mi>L</mi>
    <mo>/</mo>
    <mi>G</mi>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <minus></minus>
    <cn type="integer">1</cn>
    <apply>
     <divide></divide>
     <ci>L</ci>
     <ci>G</ci>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   1-L/G
  </annotation>
 </semantics>
</math>

 for a single fragment and 

<math display="inline" id="DNA_sequencing_theory:5">
 <semantics>
  <msup>
   <mrow>
    <mo>[</mo>
    <mrow>
     <mn>1</mn>
     <mo>-</mo>
     <mrow>
      <mi>L</mi>
      <mo>/</mo>
      <mi>G</mi>
     </mrow>
    </mrow>
    <mo>]</mo>
   </mrow>
   <mi>N</mi>
  </msup>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <csymbol cd="ambiguous">superscript</csymbol>
    <apply>
     <csymbol cd="latexml">delimited-[]</csymbol>
     <apply>
      <minus></minus>
      <cn type="integer">1</cn>
      <apply>
       <divide></divide>
       <ci>L</ci>
       <ci>G</ci>
      </apply>
     </apply>
    </apply>
    <ci>N</ci>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \left[1-L/G\right]^{N}
  </annotation>
 </semantics>
</math>

 for 

<math display="inline" id="DNA_sequencing_theory:6">
 <semantics>
  <mi>N</mi>
  <annotation-xml encoding="MathML-Content">
   <ci>N</ci>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   N
  </annotation>
 </semantics>
</math>

 fragments. The probability of covering a given location on the target with <em>at least one</em> fragment is therefore</p>

<p>

<math display="block" id="DNA_sequencing_theory:7">
 <semantics>
  <mrow>
   <mrow>
    <mi>P</mi>
    <mo>=</mo>
    <mrow>
     <mn>1</mn>
     <mo>-</mo>
     <msup>
      <mrow>
       <mo>[</mo>
       <mrow>
        <mn>1</mn>
        <mo>-</mo>
        <mfrac>
         <mi>L</mi>
         <mi>G</mi>
        </mfrac>
       </mrow>
       <mo>]</mo>
      </mrow>
      <mi>N</mi>
     </msup>
    </mrow>
   </mrow>
   <mo>.</mo>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <eq></eq>
    <ci>P</ci>
    <apply>
     <minus></minus>
     <cn type="integer">1</cn>
     <apply>
      <csymbol cd="ambiguous">superscript</csymbol>
      <apply>
       <csymbol cd="latexml">delimited-[]</csymbol>
       <apply>
        <minus></minus>
        <cn type="integer">1</cn>
        <apply>
         <divide></divide>
         <ci>L</ci>
         <ci>G</ci>
        </apply>
       </apply>
      </apply>
      <ci>N</ci>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   P=1-\left[1-\frac{L}{G}\right]^{N}.
  </annotation>
 </semantics>
</math>

</p>

<p>This equation was first used to characterize plasmid libraries,<a class="footnoteRef" href="#fn5" id="fnref5"><sup>5</sup></a> but is often more useful in a modified form. For most projects 

<math display="inline" id="DNA_sequencing_theory:8">
 <semantics>
  <mrow>
   <mi>N</mi>
   <mo>≫</mo>
   <mn>1</mn>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <csymbol cd="latexml">much-greater-than</csymbol>
    <ci>N</ci>
    <cn type="integer">1</cn>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   N\gg 1
  </annotation>
 </semantics>
</math>


, so that, to a good degree of approximation</p>

<p>

<math display="block" id="DNA_sequencing_theory:9">
 <semantics>
  <mrow>
   <mrow>
    <msup>
     <mrow>
      <mo>[</mo>
      <mrow>
       <mn>1</mn>
       <mo>-</mo>
       <mfrac>
        <mi>L</mi>
        <mi>G</mi>
       </mfrac>
      </mrow>
      <mo>]</mo>
     </mrow>
     <mi>N</mi>
    </msup>
    <mo>∼</mo>
    <mrow>
     <mi>exp</mi>
     <mrow>
      <mo stretchy="false">(</mo>
      <mrow>
       <mo>-</mo>
       <mrow>
        <mrow>
         <mi>N</mi>
         <mi>L</mi>
        </mrow>
        <mo>/</mo>
        <mi>G</mi>
       </mrow>
      </mrow>
      <mo stretchy="false">)</mo>
     </mrow>
    </mrow>
   </mrow>
   <mo>,</mo>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <csymbol cd="latexml">similar-to</csymbol>
    <apply>
     <csymbol cd="ambiguous">superscript</csymbol>
     <apply>
      <csymbol cd="latexml">delimited-[]</csymbol>
      <apply>
       <minus></minus>
       <cn type="integer">1</cn>
       <apply>
        <divide></divide>
        <ci>L</ci>
        <ci>G</ci>
       </apply>
      </apply>
     </apply>
     <ci>N</ci>
    </apply>
    <apply>
     <exp></exp>
     <apply>
      <minus></minus>
      <apply>
       <divide></divide>
       <apply>
        <times></times>
        <ci>N</ci>
        <ci>L</ci>
       </apply>
       <ci>G</ci>
      </apply>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \left[1-\frac{L}{G}\right]^{N}\sim\exp(-NL/G),
  </annotation>
 </semantics>
</math>

</p>

<p>where 

<math display="inline" id="DNA_sequencing_theory:10">
 <semantics>
  <mrow>
   <mi>R</mi>
   <mo>=</mo>
   <mrow>
    <mrow>
     <mi>N</mi>
     <mi>L</mi>
    </mrow>
    <mo>/</mo>
    <mi>G</mi>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <eq></eq>
    <ci>R</ci>
    <apply>
     <divide></divide>
     <apply>
      <times></times>
      <ci>N</ci>
      <ci>L</ci>
     </apply>
     <ci>G</ci>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   R=NL/G
  </annotation>
 </semantics>
</math>

 is called the <em>redundancy</em>. Note the significance of redundancy as representing the average number of times a position is covered with fragments. Note also that in considering the covering process over all positions in the target, this probability is identical to the <a href="expected_value" title="wikilink">expected value</a> of the random variable 

<math display="inline" id="DNA_sequencing_theory:11">
 <semantics>
  <mi>C</mi>
  <annotation-xml encoding="MathML-Content">
   <ci>C</ci>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   C
  </annotation>
 </semantics>
</math>

, which represents the fraction of the target coverage. The final result,</p>

<p>

<math display="block" id="DNA_sequencing_theory:12">
 <semantics>
  <mrow>
   <mrow>
    <mrow>
     <mi>E</mi>
     <mrow>
      <mo stretchy="false">⟨</mo>
      <mi>C</mi>
      <mo stretchy="false">⟩</mo>
     </mrow>
    </mrow>
    <mo>=</mo>
    <mrow>
     <mn>1</mn>
     <mo>-</mo>
     <msup>
      <mi>e</mi>
      <mrow>
       <mo>-</mo>
       <mi>R</mi>
      </mrow>
     </msup>
    </mrow>
   </mrow>
   <mo>,</mo>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <eq></eq>
    <apply>
     <times></times>
     <ci>E</ci>
     <apply>
      <csymbol cd="latexml">delimited-⟨⟩</csymbol>
      <ci>C</ci>
     </apply>
    </apply>
    <apply>
     <minus></minus>
     <cn type="integer">1</cn>
     <apply>
      <csymbol cd="ambiguous">superscript</csymbol>
      <ci>e</ci>
      <apply>
       <minus></minus>
       <ci>R</ci>
      </apply>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   E\langle C\rangle=1-e^{-R},
  </annotation>
 </semantics>
</math>

</p>

<p>remains in widespread use as a "<a href="Back-of-the-envelope_calculation" title="wikilink">back of the envelope</a>" estimator and predicts that coverage for all projects evolves along a universal curve that is a function only of the redundancy.</p>
<h3 id="lander-waterman-theory">Lander-Waterman theory</h3>

<p>In 1988, <a href="Eric_Lander" title="wikilink">Eric Lander</a> and <a href="Michael_Waterman" title="wikilink">Michael Waterman</a> published an important paper<a class="footnoteRef" href="#fn6" id="fnref6"><sup>6</sup></a> examining the covering problem from the standpoint of gaps. Although they focused on the so-called <a href="Gene_mapping" title="wikilink">mapping problem</a>, the abstraction to sequencing is much the same. They furnished a number of useful results that were adopted as the standard theory from the earliest days of "large-scale" genome sequencing.<a class="footnoteRef" href="#fn7" id="fnref7"><sup>7</sup></a> Their model was also used in designing the <a href="Human_Genome_Project" title="wikilink">Human Genome Project</a> and continues to play an important role in DNA sequencing.</p>

<p>Ultimately, the main goal of a sequencing project is to close all gaps, so the "gap perspective" was a logical basis of developing a sequencing model. One of the more frequently used results from this model is the expected number of <a href="contig" title="wikilink">contigs</a>, given the number of fragments sequenced. If one neglects the amount of sequence that is essentially "wasted" by having to detect overlaps, their theory yields</p>

<p>

<math display="block" id="DNA_sequencing_theory:13">
 <semantics>
  <mrow>
   <mrow>
    <mrow>
     <mi>E</mi>
     <mrow>
      <mo stretchy="false">⟨</mo>
      <mrow>
       <mi>c</mi>
       <mi>o</mi>
       <mi>n</mi>
       <mi>t</mi>
       <mi>i</mi>
       <mi>g</mi>
       <mi>s</mi>
      </mrow>
      <mo stretchy="false">⟩</mo>
     </mrow>
    </mrow>
    <mo>=</mo>
    <mrow>
     <mi>N</mi>
     <msup>
      <mi>e</mi>
      <mrow>
       <mo>-</mo>
       <mi>R</mi>
      </mrow>
     </msup>
    </mrow>
   </mrow>
   <mo>.</mo>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <eq></eq>
    <apply>
     <times></times>
     <ci>E</ci>
     <apply>
      <csymbol cd="latexml">delimited-⟨⟩</csymbol>
      <apply>
       <times></times>
       <ci>c</ci>
       <ci>o</ci>
       <ci>n</ci>
       <ci>t</ci>
       <ci>i</ci>
       <ci>g</ci>
       <ci>s</ci>
      </apply>
     </apply>
    </apply>
    <apply>
     <times></times>
     <ci>N</ci>
     <apply>
      <csymbol cd="ambiguous">superscript</csymbol>
      <ci>e</ci>
      <apply>
       <minus></minus>
       <ci>R</ci>
      </apply>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   E\langle contigs\rangle=Ne^{-R}.
  </annotation>
 </semantics>
</math>

</p>

<p>In 1995, Roach<a class="footnoteRef" href="#fn8" id="fnref8"><sup>8</sup></a> published improvements to this theory, enabling it to be applied to sequencing projects in which the goal was to completely sequence a target genome. <a href="Michael_Christopher_Wendl" title="wikilink">Michael Wendl</a> and <a href="Bob_Waterston" title="wikilink">Bob Waterston</a><a class="footnoteRef" href="#fn9" id="fnref9"><sup>9</sup></a> confirmed, based on Stevens' method,<a class="footnoteRef" href="#fn10" id="fnref10"><sup>10</sup></a> that both models produced similar results when the number of contigs was substantial, such as in low coverage mapping or sequencing projects. As sequencing projects ramped up in the 1990s, and projects approached completion, low coverage approximations became inadequate, and the exact model of Roach was necessary. However, as the cost of sequencing dropped, parameters of sequencing projects became easier to directly test empirically, and interest and funding for strategic genomics diminished</p>

<p>The basic ideas of Lander–Waterman theory led to a number of additional results for particular variations in mapping techniques.<a class="footnoteRef" href="#fn11" id="fnref11"><sup>11</sup></a><a class="footnoteRef" href="#fn12" id="fnref12"><sup>12</sup></a><a class="footnoteRef" href="#fn13" id="fnref13"><sup>13</sup></a> However, technological advancements have rendered mapping theories largely obsolete except in organisms other than highly studied model organisms (e.g., yeast, flies, mice, and humans).</p>
<h3 id="parking-strategy">Parking strategy</h3>

<p>The parking strategy for sequencing resembles the process of parking cars along a curb. Each car is a sequenced clone, and the curb is the genomic target.<a class="footnoteRef" href="#fn14" id="fnref14"><sup>14</sup></a> Each clone sequenced is screened to ensure that subsequently sequenced clones do not overlap any previously sequenced clone. No sequencing effort is redundant in this strategy. However, much like the gaps between parked cars, unsequenced gaps less than the length of a clone accumulate between sequenced clones. There can be considerable cost to close such gaps.</p>
<h3 id="pairwise-end-sequencing">Pairwise end-sequencing</h3>

<p>In 1995, Roach <em>et al.</em><a class="footnoteRef" href="#fn15" id="fnref15"><sup>15</sup></a> proposed and demonstrated through simulations a generalization of a set of strategies explored earlier by Edwards and Caskey.<a class="footnoteRef" href="#fn16" id="fnref16"><sup>16</sup></a> This <a href="Shotgun_sequencing#Whole_genome_shotgun_sequencing" title="wikilink">whole-genome sequencing</a> method became immensely popular as it was championed by Celera and used to sequenced several model organisms before Celera applied it to the human genome. Today, most sequencing projects employ this strategy, often called paired end sequencing.</p>
<h2 id="post-human-genome-project-advancements">Post Human Genome Project advancements</h2>

<p>The physical processes and protocols of DNA sequencing have continued to evolve, largely driven by advancements in bio-chemical methods, instrumentation, and automation techniques. There is now a wide range of problems that <a href="DNA_sequencing" title="wikilink">DNA sequencing</a> has made in-roads into, including <a class="uri" href="metagenomics" title="wikilink">metagenomics</a> and <a href="Cancer_Genome_Project" title="wikilink">medical (cancer) sequencing</a>. There are important factors in these scenarios that classical theory does not account for. Recent work has begun to focus on resolving the effects of some of these issues. The level of mathematics becomes commensurately more sophisticated.</p>
<h3 id="various-artifacts-of-large-insert-sequencing">Various artifacts of large-insert sequencing</h3>

<p>Biologists have developed methods to filter highly-repetitive, essentially un-sequenceable regions of genomes. These procedures are important for organisms whose genomes consist mostly of such DNA, for example corn. They yield multitudes of small islands of sequenceable DNA products. Wendl and Barbazuk<a class="footnoteRef" href="#fn17" id="fnref17"><sup>17</sup></a> proposed an extension to Lander–Waterman Theory to account for "gaps" in the target due to filtering and the so-called "edge-effect". The latter is a position-specific sampling bias, for example the terminal base position has only a 

<math display="inline" id="DNA_sequencing_theory:14">
 <semantics>
  <mrow>
   <mn>1</mn>
   <mo>/</mo>
   <mi>G</mi>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <divide></divide>
    <cn type="integer">1</cn>
    <ci>G</ci>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   1/G
  </annotation>
 </semantics>
</math>

 chance of being covered, as opposed to 

<math display="inline" id="DNA_sequencing_theory:15">
 <semantics>
  <mrow>
   <mi>L</mi>
   <mo>/</mo>
   <mi>G</mi>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <divide></divide>
    <ci>L</ci>
    <ci>G</ci>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   L/G
  </annotation>
 </semantics>
</math>

 for interior positions. For 

<math display="inline" id="DNA_sequencing_theory:16">
 <semantics>
  <mrow>
   <mi>R</mi>
   <mo><</mo>
   <mn>1</mn>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <lt></lt>
    <ci>R</ci>
    <cn type="integer">1</cn>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   R<1
  </annotation>
 </semantics>
</math>

, classical Lander–Waterman Theory still gives good predictions, but dynamics change for higher redundancies.</p>

<p>Modern sequencing methods usually sequence both ends of a larger fragment, which provides linking information for <em>de novo</em> assembly and improved probabilities for alignment to reference sequence. Researchers generally believe that longer lengths of data (read lengths) enhance performance for very large DNA targets, an idea consistent with predictions from distribution models.<a class="footnoteRef" href="#fn18" id="fnref18"><sup>18</sup></a> However, Wendl<a class="footnoteRef" href="#fn19" id="fnref19"><sup>19</sup></a> showed that smaller fragments provide better coverage on small, linear targets because they reduce the edge effect in linear molecules. These findings have implications for sequencing the products of DNA filtering procedures. Read-pairing and fragment size evidently have negligible influence for large, whole-genome class targets.</p>
<h3 id="individual-and-population-sequencing">Individual and population sequencing</h3>

<p>Sequencing is emerging as an important tool in medicine, for example in cancer research. Here, the ability to detect <a href="Loss_of_heterozygosity" title="wikilink">heterozygous mutations</a> is important and this can only be done if the sequence of the <a href="Diploid#Diploid" title="wikilink">diploid genome</a> is obtained. In the pioneering efforts to sequence individuals, Levy <em>et al.</em><a class="footnoteRef" href="#fn20" id="fnref20"><sup>20</sup></a> and Wheeler <em>et al.</em>,<a class="footnoteRef" href="#fn21" id="fnref21"><sup>21</sup></a> who sequenced <a href="Craig_Venter" title="wikilink">Craig Venter</a> and <a href="James_D._Watson" title="wikilink">Jim Watson</a>, respectively, outlined models for covering both alleles in a genome. Wendl and Wilson<a class="footnoteRef" href="#fn22" id="fnref22"><sup>22</sup></a> followed with a more general theory that allowed for an arbitrary number of coverings of each allele and arbitrary <a class="uri" href="ploidy" title="wikilink">ploidy</a>. These results point to the general conclusion that the amount of data needed for such projects is significantly higher than for traditional haploid projects. Generally, at least 30-fold redundancy, i.e. each nucleotide spanned by an average of 30 sequence reads, is now standard.<a class="footnoteRef" href="#fn23" id="fnref23"><sup>23</sup></a> However, requirements can be even greater, depending upon what kinds of genomic events are to be found. For example, in the so-called "discordant read pairs method", DNA insertions can be inferred if the distance between read pairs is larger than expected. Calculations show that around 50-fold redundancy is needed to avoid <a href="False_positive_rate" title="wikilink">false-positive errors</a> at 1% threshold.<a class="footnoteRef" href="#fn24" id="fnref24"><sup>24</sup></a></p>

<p>The advent of <a href="DNA_sequencing#Next-generation_methods" title="wikilink">next-generation sequencing</a> has also made large-scale population sequencing feasible, for example the <a href="1000_Genomes_Project" title="wikilink">1000 Genomes Project</a> to characterize variation in human population groups. While common variation is easily captured, rare variation poses a design challenge: too few samples with significant sequence redundancy risks not having a variant in the sample group, but large samples with light redundancy risk not capturing a variant in the read set that is actually in the sample group. Wendl and Wilson<a class="footnoteRef" href="#fn25" id="fnref25"><sup>25</sup></a> report a simple set of optimization rules that maximize the probability of discovery for a given set of parameters. For example, for observing a rare allele at least twice (to eliminate the possibility is unique to an individual) a little less than 4-fold redundancy should be used, regardless of the sample size.</p>
<h3 id="metagenomic-sequencing">Metagenomic sequencing</h3>

<p>Next-generation instruments are now also enabling the sequencing of whole uncultured metagenomic communities. The sequencing scenario is more complicated here and there are various ways of framing design theories for a given project. For example, Stanhope<a class="footnoteRef" href="#fn26" id="fnref26"><sup>26</sup></a> developed a probabilistic model for the amount of sequence needed to obtain at least one contig of a given size from each novel organism of the community, while Wendl et al. reported analysis for the average contig size or the probability of completely recovering a novel organism for a given rareness within the community.<a class="footnoteRef" href="#fn27" id="fnref27"><sup>27</sup></a> Conversely, Hooper et al. propose a semi-empirical model based on the <a href="Gamma_distribution" title="wikilink">Gamma distribution</a>.<a class="footnoteRef" href="#fn28" id="fnref28"><sup>28</sup></a></p>
<h2 id="limitations">Limitations</h2>

<p>DNA sequencing theories often invoke the assumption that certain random variables in a model are <a href="Independent_and_identically-distributed_random_variables" title="wikilink">independent and identically distributed</a>. For example, in Lander–Waterman Theory, a sequenced fragment is presumed to have the same probability of covering each region of a genome and all fragments are assumed to be independent of one another. In actuality, sequencing projects are subject to various types of bias, including differences of how well regions can be cloned, sequencing anomalies, biases in the target sequence (which is <em>not</em> random), and software-dependent errors and biases. In general, theory will agree well with observation up to the point that enough data have been generated to expose latent biases.<a class="footnoteRef" href="#fn29" id="fnref29"><sup>29</sup></a> The kinds of biases related to the underlying target sequence are particularly difficult to model, since the sequence itself may not be known <em>a priori</em>. This presents a type of <a href="Chicken_or_the_egg" title="wikilink">"chicken and egg"</a> closure problem.</p>
<h2 id="see-also">See also</h2>
<ul>
<li><a href="Computational_biology" title="wikilink">Computational biology</a></li>
<li><a class="uri" href="Bioinformatics" title="wikilink">Bioinformatics</a></li>
<li><a href="Mathematical_biology" title="wikilink">Mathematical biology</a></li>
<li><a href="Sulston_score" title="wikilink">Sulston score</a></li>
</ul>
<h2 id="references">References</h2>

<p>"</p>

<p><a class="uri" href="Category:Bioinformatics" title="wikilink">Category:Bioinformatics</a> <a href="Category:Mathematical_and_theoretical_biology" title="wikilink">Category:Mathematical and theoretical biology</a> <a href="Category:DNA_sequencing" title="wikilink">Category:DNA sequencing</a></p>
<section class="footnotes">
<hr/>
<ol>
<li id="fn1"><a href="#fnref1">↩</a></li>
<li id="fn2"><a href="#fnref2">↩</a></li>
<li id="fn3"><a href="#fnref3">↩</a></li>
<li id="fn4"><a href="#fnref4">↩</a></li>
<li id="fn5"><a href="#fnref5">↩</a></li>
<li id="fn6"><a href="#fnref6">↩</a></li>
<li id="fn7"><a href="#fnref7">↩</a></li>
<li id="fn8"><a href="#fnref8">↩</a></li>
<li id="fn9"><a href="#fnref9">↩</a></li>
<li id="fn10"></li>
<li id="fn11"><a href="#fnref11">↩</a></li>
<li id="fn12"><a href="#fnref12">↩</a></li>
<li id="fn13"><a href="#fnref13">↩</a></li>
<li id="fn14"><a href="#fnref14">↩</a></li>
<li id="fn15"><a href="#fnref15">↩</a></li>
<li id="fn16"><a href="#fnref16">↩</a></li>
<li id="fn17"><a href="#fnref17">↩</a></li>
<li id="fn18"><a href="#fnref18">↩</a></li>
<li id="fn19"><a href="#fnref19">↩</a></li>
<li id="fn20"><a href="#fnref20">↩</a></li>
<li id="fn21"><a href="#fnref21">↩</a></li>
<li id="fn22"><a href="#fnref22">↩</a></li>
<li id="fn23"><a href="#fnref23">↩</a></li>
<li id="fn24"><a href="#fnref24">↩</a></li>
<li id="fn25"><a href="#fnref25">↩</a></li>
<li id="fn26"><a href="#fnref26">↩</a></li>
<li id="fn27"><a href="#fnref27">↩</a></li>
<li id="fn28"><a href="#fnref28">↩</a></li>
<li id="fn29"></li>
</ol>
</section>
</hr></body>
</html>
