<html lang="en">
<head>
<meta charset="utf-8"/>
<title offset="317">Enzyme inhibitor</title>
<script src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_SVG.js" type="text/javascript">
</script>
</head>
<body>
<h1>Enzyme inhibitor</h1>
<hr/>
<figure><b>(Figure)</b>
<figcaption><a class="uri" href="HIV" title="wikilink">HIV</a> <a class="uri" href="protease" title="wikilink">protease</a> in a complex with the protease inhibitor <a class="uri" href="ritonavir" title="wikilink">ritonavir</a>. The structure of the protease is shown by the red, blue and yellow ribbons. The inhibitor is shown as the smaller ball-and-stick structure near the center. Created from PDB <a href="http://www.rcsb.org/pdb/explore.do?structureId=1HXW">1HXW</a>.</figcaption>
</figure>

<p>An <strong>enzyme inhibitor</strong> is a <a class="uri" href="molecule" title="wikilink">molecule</a> that binds to an <a class="uri" href="enzyme" title="wikilink">enzyme</a> and decreases its <a href="enzyme_activity" title="wikilink">activity</a>. Since blocking an enzyme's activity can kill a <a class="uri" href="pathogen" title="wikilink">pathogen</a> or correct a <a href="metabolism" title="wikilink">metabolic</a> imbalance, many drugs are enzyme inhibitors. They are also used in <a href="pesticide" title="wikilink">pesticides</a>. Not all molecules that bind to enzymes are inhibitors; <em><a href="enzyme_activator" title="wikilink">enzyme activators</a></em> bind to enzymes and increase their <a href="enzyme_assay" title="wikilink">enzymatic activity</a>, while enzyme substrates bind and are converted to products in the normal catalytic cycle of the enzyme.</p>

<p>The binding of an inhibitor can stop a <a href="substrate_(biochemistry)" title="wikilink">substrate</a> from entering the enzyme's <a href="active_site" title="wikilink">active site</a> and/or hinder the enzyme from <a href="catalysis" title="wikilink">catalyzing</a> its reaction. Inhibitor binding is either <a href="reversible_reaction" title="wikilink">reversible</a> or irreversible. Irreversible inhibitors usually react with the enzyme and change it chemically (e.g. via covalent bond formation). These inhibitors modify key <a href="amino_acid" title="wikilink">amino acid</a> residues needed for enzymatic activity. In contrast, reversible inhibitors bind <a href="Ligand_(biochemistry)" title="wikilink">non-covalently</a> and different types of inhibition are produced depending on whether these inhibitors bind to the <a class="uri" href="enzyme" title="wikilink">enzyme</a>, the enzyme-substrate complex, or both.</p>

<p>Many <a href="medication" title="wikilink">drug molecules</a> are enzyme inhibitors, so their discovery and improvement is an active area of research in <a class="uri" href="biochemistry" title="wikilink">biochemistry</a> and <a class="uri" href="pharmacology" title="wikilink">pharmacology</a>. A medicinal enzyme inhibitor is often judged by its <a href="specificity_(tests)" title="wikilink">specificity</a> (its lack of binding to other proteins) and its potency (its <a href="dissociation_constant" title="wikilink">dissociation constant</a>, which indicates the concentration needed to inhibit the enzyme). A high specificity and potency ensure that a drug will have few <a href="adverse_drug_reaction" title="wikilink">side effects</a> and thus low <a class="uri" href="toxicity" title="wikilink">toxicity</a>.</p>

<p>Enzyme inhibitors also occur naturally and are involved in the regulation of metabolism. For example, enzymes in a <a href="metabolic_pathway" title="wikilink">metabolic pathway</a> can be inhibited by downstream products. This type of <a href="negative_feedback" title="wikilink">negative feedback</a> slows the production line when products begin to build up and is an important way to maintain <a class="uri" href="homeostasis" title="wikilink">homeostasis</a> in a <a href="cell_(biology)" title="wikilink">cell</a>. Other cellular enzyme inhibitors are <a href="protein" title="wikilink">proteins</a> that specifically bind to and inhibit an enzyme target. This can help control enzymes that may be damaging to a cell, like <a href="protease" title="wikilink">proteases</a> or <a href="nuclease" title="wikilink">nucleases</a>. A well-characterised example of this is the <a href="ribonuclease_inhibitor" title="wikilink">ribonuclease inhibitor</a>, which binds to <a href="ribonuclease" title="wikilink">ribonucleases</a> in one of the tightest known <a href="protein-protein_interaction" title="wikilink">protein–protein interactions</a>.<a class="footnoteRef" href="#fn1" id="fnref1"><sup>1</sup></a> Natural enzyme inhibitors can also be poisons and are used as defences against predators or as ways of killing prey.</p>
<h2 id="reversible-inhibitors">Reversible inhibitors</h2>
<h3 id="types-of-reversible-inhibitors">Types of reversible inhibitors</h3>

<p>Reversible inhibitors attach to enzymes with non-covalent interactions such as <a href="hydrogen_bond" title="wikilink">hydrogen bonds</a>, <a href="hydrophobic_interaction" title="wikilink">hydrophobic interactions</a> and <a href="ionic_bond" title="wikilink">ionic bonds</a>. Multiple weak bonds between the inhibitor and the active site combine to produce strong and specific binding. In contrast to <a href="substrate_(biochemistry)" title="wikilink">substrates</a> and irreversible inhibitors, reversible inhibitors generally do not undergo chemical reactions when bound to the enzyme and can be easily removed by dilution or dialysis.</p>

<p>There are four kinds of reversible enzyme inhibitors. They are classified according to the effect of varying the concentration of the enzyme's substrate on the inhibitor.<a class="footnoteRef" href="#fn2" id="fnref2"><sup>2</sup></a></p>
<figure><b>(Figure)</b>
<figcaption>Types of inhibition. This classification was introduced by <a href="W.W._Cleland" title="wikilink">W.W. Cleland</a>.<a class="footnoteRef" href="#fn3" id="fnref3"><sup>3</sup></a></figcaption>
</figure>
<ul>
<li>In <strong><a href="competitive_inhibition" title="wikilink">competitive inhibition</a></strong>, the substrate and inhibitor cannot bind to the enzyme at the same time, as shown in the figure on the right. This usually results from the inhibitor having an affinity for the <a href="active_site" title="wikilink">active site</a> of an enzyme where the substrate also binds; the substrate and inhibitor <em>compete</em> for access to the enzyme's active site. This type of inhibition can be overcome by sufficiently high concentrations of substrate (<em>V<sub>max</sub></em> remains constant), i.e., by out-competing the inhibitor. However, the apparent <em>K<sub>m</sub></em> will increase as it takes a higher concentration of the substrate to reach the <em>K<sub>m</sub></em> point, or half the <em>V<sub>max</sub></em>. Competitive inhibitors are often similar in structure to the real substrate (see examples below).</li>
</ul>
<ul>
<li>In <strong><a href="uncompetitive_inhibition" title="wikilink">uncompetitive inhibition</a></strong>, the inhibitor binds only to the substrate-enzyme complex, it should not be confused with non-competitive inhibitors. This type of inhibition causes <em>V<sub>max</sub></em> to decrease (maximum velocity decreases as a result of removing activated complex) and <em>K<sub>m</sub></em> to decrease (due to better binding efficiency as a result of Le Chatelier's principle and the effective elimination of the ES complex thus decreasing the <em>K<sub>m</sub></em> which indicates a higher binding affinity).</li>
</ul>
<ul>
<li>In <strong><a href="non-competitive_inhibition" title="wikilink">non-competitive inhibition</a></strong>, the binding of the inhibitor to the enzyme reduces its <a href="enzyme_activity" title="wikilink">activity</a> but does not affect the binding of substrate. As a result, the extent of inhibition depends only on the concentration of the inhibitor. <em>V<sub>max</sub></em> will decrease due to the inability for the reaction to proceed as efficiently, but <em>K<sub>m</sub></em> will remain the same as the actual binding of the substrate, by definition, will still function properly.</li>
</ul>
<ul>
<li>In <strong><a href="mixed_inhibition" title="wikilink">mixed inhibition</a></strong>, the inhibitor can bind to the enzyme at the same time as the enzyme's substrate. However, the binding of the inhibitor affects the binding of the substrate, and vice versa. This type of inhibition can be reduced, but not overcome by increasing concentrations of substrate. Although it is possible for mixed-type inhibitors to bind in the active site, this type of inhibition generally results from an <a class="uri" href="allosteric" title="wikilink">allosteric</a> effect where the inhibitor binds to a different site on an enzyme. Inhibitor binding to this <a href="allosteric_site" title="wikilink">allosteric site</a> changes the <a href="Conformational_isomerism" title="wikilink">conformation</a> (i.e., <a href="tertiary_structure" title="wikilink">tertiary structure</a> or three-dimensional shape) of the enzyme so that the affinity of the substrate for the active site is reduced.</li>
</ul>
<h3 id="quantitative-description-of-reversible-inhibition">Quantitative description of reversible inhibition</h3>

<p>Reversible inhibition can be described quantitatively in terms of the inhibitor's <a href="dissociation_constant" title="wikilink">binding</a> to the enzyme and to the enzyme-substrate complex, and its effects on the <a href="enzyme_kinetics" title="wikilink">kinetic constants</a> of the enzyme. In the classic <a href="Michaelis-Menten_kinetics" title="wikilink">Michaelis-Menten scheme</a> below, an enzyme (E) binds to its substrate (S) to form the enzyme–substrate complex ES. Upon catalysis, this complex breaks down to release product P and free enzyme. The inhibitor (I) can bind to either E or ES with the <a href="dissociation_constant" title="wikilink">dissociation constants</a> <em>K</em><sub>i</sub> or <em>K</em><sub>i</sub>', respectively.</p>
<table>
<tbody>
<tr class="odd">
<td style="text-align: left;"><ul>
<li>Competitive inhibitors can bind to E, but not to ES. Competitive inhibition increases <em>K</em><sub>m</sub> (i.e., the inhibitor interferes with substrate binding), but does not affect <em>V</em><sub>max</sub> (the inhibitor does not hamper catalysis in ES because it cannot bind to ES).</li>
<li>Non-competitive inhibitors have identical affinities for E and ES (<em>K</em><sub>i</sub> = <em>K</em><sub>i</sub>'). Non-competitive inhibition does not change <em>K</em><sub>m</sub> (i.e., it does not affect substrate binding) but decreases <em>V</em><sub>max</sub> (i.e., inhibitor binding hampers catalysis).</li>
<li>Mixed-type inhibitors bind to both E and ES, but their affinities for these two forms of the enzyme are different (<em>K</em><sub>i</sub> ≠ <em>K</em><sub>i</sub>'). Thus, mixed-type inhibitors interfere with substrate binding (increase <em>K</em><sub>m</sub>) and hamper catalysis in the ES complex (decrease <em>V</em><sub>max</sub>).</li>
</ul></td>
<td style="text-align: left;"><figure><b>(Figure)</b>
<figcaption>Kinetic scheme for reversible enzyme inhibitors</figcaption>
</figure></td>
</tr>
</tbody>
</table>

<p>When an enzyme has multiple substrates, inhibitors can show different types of inhibition depending on which substrate is considered. This results from the active site containing two different binding sites within the active site, one for each substrate. For example, an inhibitor might compete with substrate A for the first binding site, but be a non-competitive inhibitor with respect to substrate B in the second binding site.<a class="footnoteRef" href="#fn4" id="fnref4"><sup>4</sup></a></p>
<h3 id="measuring-the-dissociation-constants-of-a-reversible-inhibitor">Measuring the dissociation constants of a reversible inhibitor</h3>
<figure><b>(Figure)</b>
<figcaption><a href="Lineweaver–Burk_plot" title="wikilink">Lineweaver–Burk plots</a> of different types of reversible enzyme inhibitors. The arrow shows the effect of increasing concentrations of inhibitor.</figcaption>
</figure>

<p>As noted above, an enzyme inhibitor is characterised by its two <a href="dissociation_constant" title="wikilink">dissociation constants</a>, <em>K</em><sub>i</sub> and <em>K</em><sub>i</sub>', to the enzyme and to the enzyme-substrate complex, respectively. The enzyme-inhibitor constant <em>K</em><sub>i</sub> can be measured directly by various methods; one extremely accurate method is <a href="Isothermal_Titration_Calorimetry" title="wikilink">isothermal titration calorimetry</a>, in which the inhibitor is titrated into a solution of enzyme and the heat released or absorbed is measured.<a class="footnoteRef" href="#fn5" id="fnref5"><sup>5</sup></a> However, the other dissociation constant <em>K</em><sub>i</sub>' is difficult to measure directly, since the enzyme-substrate complex is short-lived and undergoing a chemical reaction to form the product. Hence, <em>K</em><sub>i</sub>' is usually measured indirectly, by observing the <a href="enzyme_activity" title="wikilink">enzyme activity</a> under various substrate and inhibitor concentrations, and <a href="nonlinear_regression" title="wikilink">fitting</a> the data<a class="footnoteRef" href="#fn6" id="fnref6"><sup>6</sup></a> to a modified <a href="enzyme_kinetics" title="wikilink">Michaelis–Menten equation</a></p>

<p>

<math display="block" id="Enzyme_inhibitor:0">
 <semantics>
  <mrow>
   <mi>V</mi>
   <mo>=</mo>
   <mfrac>
    <mrow>
     <msub>
      <mi>V</mi>
      <mrow>
       <mi>m</mi>
       <mi>a</mi>
       <mi>x</mi>
      </mrow>
     </msub>
     <mrow>
      <mo stretchy="false">[</mo>
      <mi>S</mi>
      <mo stretchy="false">]</mo>
     </mrow>
    </mrow>
    <mrow>
     <mrow>
      <mi>α</mi>
      <msub>
       <mi>K</mi>
       <mi>m</mi>
      </msub>
     </mrow>
     <mo>+</mo>
     <mrow>
      <msup>
       <mi>α</mi>
       <mo>′</mo>
      </msup>
      <mrow>
       <mo stretchy="false">[</mo>
       <mi>S</mi>
       <mo stretchy="false">]</mo>
      </mrow>
     </mrow>
    </mrow>
   </mfrac>
   <mo>=</mo>
   <mfrac>
    <mrow>
     <mrow>
      <mo stretchy="false">(</mo>
      <mrow>
       <mn>1</mn>
       <mo>/</mo>
       <msup>
        <mi>α</mi>
        <mo>′</mo>
       </msup>
      </mrow>
      <mo stretchy="false">)</mo>
     </mrow>
     <msub>
      <mi>V</mi>
      <mrow>
       <mi>m</mi>
       <mi>a</mi>
       <mi>x</mi>
      </mrow>
     </msub>
     <mrow>
      <mo stretchy="false">[</mo>
      <mi>S</mi>
      <mo stretchy="false">]</mo>
     </mrow>
    </mrow>
    <mrow>
     <mrow>
      <mrow>
       <mo stretchy="false">(</mo>
       <mrow>
        <mi>α</mi>
        <mo>/</mo>
        <msup>
         <mi>α</mi>
         <mo>′</mo>
        </msup>
       </mrow>
       <mo stretchy="false">)</mo>
      </mrow>
      <msub>
       <mi>K</mi>
       <mi>m</mi>
      </msub>
     </mrow>
     <mo>+</mo>
     <mrow>
      <mo stretchy="false">[</mo>
      <mi>S</mi>
      <mo stretchy="false">]</mo>
     </mrow>
    </mrow>
   </mfrac>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <and></and>
    <apply>
     <eq></eq>
     <ci>V</ci>
     <apply>
      <divide></divide>
      <apply>
       <times></times>
       <apply>
        <csymbol cd="ambiguous">subscript</csymbol>
        <ci>V</ci>
        <apply>
         <times></times>
         <ci>m</ci>
         <ci>a</ci>
         <ci>x</ci>
        </apply>
       </apply>
       <apply>
        <csymbol cd="latexml">delimited-[]</csymbol>
        <ci>S</ci>
       </apply>
      </apply>
      <apply>
       <plus></plus>
       <apply>
        <times></times>
        <ci>α</ci>
        <apply>
         <csymbol cd="ambiguous">subscript</csymbol>
         <ci>K</ci>
         <ci>m</ci>
        </apply>
       </apply>
       <apply>
        <times></times>
        <apply>
         <csymbol cd="ambiguous">superscript</csymbol>
         <ci>α</ci>
         <ci>normal-′</ci>
        </apply>
        <apply>
         <csymbol cd="latexml">delimited-[]</csymbol>
         <ci>S</ci>
        </apply>
       </apply>
      </apply>
     </apply>
    </apply>
    <apply>
     <eq></eq>
     <share href="#.cmml">
     </share>
     <apply>
      <divide></divide>
      <apply>
       <times></times>
       <apply>
        <divide></divide>
        <cn type="integer">1</cn>
        <apply>
         <csymbol cd="ambiguous">superscript</csymbol>
         <ci>α</ci>
         <ci>normal-′</ci>
        </apply>
       </apply>
       <apply>
        <csymbol cd="ambiguous">subscript</csymbol>
        <ci>V</ci>
        <apply>
         <times></times>
         <ci>m</ci>
         <ci>a</ci>
         <ci>x</ci>
        </apply>
       </apply>
       <apply>
        <csymbol cd="latexml">delimited-[]</csymbol>
        <ci>S</ci>
       </apply>
      </apply>
      <apply>
       <plus></plus>
       <apply>
        <times></times>
        <apply>
         <divide></divide>
         <ci>α</ci>
         <apply>
          <csymbol cd="ambiguous">superscript</csymbol>
          <ci>α</ci>
          <ci>normal-′</ci>
         </apply>
        </apply>
        <apply>
         <csymbol cd="ambiguous">subscript</csymbol>
         <ci>K</ci>
         <ci>m</ci>
        </apply>
       </apply>
       <apply>
        <csymbol cd="latexml">delimited-[]</csymbol>
        <ci>S</ci>
       </apply>
      </apply>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   V=\frac{V_{max}[S]}{\alpha K_{m}+\alpha^{\prime}[S]}=\frac{(1/\alpha^{\prime})%
V_{max}[S]}{(\alpha/\alpha^{\prime})K_{m}+[S]}
  </annotation>
 </semantics>
</math>

</p>

<p>where the modifying factors α and α' are defined by the inhibitor concentration and its two dissociation constants</p>

<p>

<math display="block" id="Enzyme_inhibitor:1">
 <semantics>
  <mrow>
   <mi>α</mi>
   <mo>=</mo>
   <mrow>
    <mn>1</mn>
    <mo>+</mo>
    <mfrac>
     <mrow>
      <mo stretchy="false">[</mo>
      <mi>I</mi>
      <mo stretchy="false">]</mo>
     </mrow>
     <msub>
      <mi>K</mi>
      <mi>i</mi>
     </msub>
    </mfrac>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <eq></eq>
    <ci>α</ci>
    <apply>
     <plus></plus>
     <cn type="integer">1</cn>
     <apply>
      <divide></divide>
      <apply>
       <csymbol cd="latexml">delimited-[]</csymbol>
       <ci>I</ci>
      </apply>
      <apply>
       <csymbol cd="ambiguous">subscript</csymbol>
       <ci>K</ci>
       <ci>i</ci>
      </apply>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \alpha=1+\frac{[I]}{K_{i}}
  </annotation>
 </semantics>
</math>

</p>

<p>

<math display="block" id="Enzyme_inhibitor:2">
 <semantics>
  <mrow>
   <mrow>
    <msup>
     <mi>α</mi>
     <mo>′</mo>
    </msup>
    <mo>=</mo>
    <mrow>
     <mn>1</mn>
     <mo>+</mo>
     <mfrac>
      <mrow>
       <mo stretchy="false">[</mo>
       <mi>I</mi>
       <mo stretchy="false">]</mo>
      </mrow>
      <msubsup>
       <mi>K</mi>
       <mi>i</mi>
       <mo>′</mo>
      </msubsup>
     </mfrac>
    </mrow>
   </mrow>
   <mo>.</mo>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <eq></eq>
    <apply>
     <csymbol cd="ambiguous">superscript</csymbol>
     <ci>α</ci>
     <ci>normal-′</ci>
    </apply>
    <apply>
     <plus></plus>
     <cn type="integer">1</cn>
     <apply>
      <divide></divide>
      <apply>
       <csymbol cd="latexml">delimited-[]</csymbol>
       <ci>I</ci>
      </apply>
      <apply>
       <csymbol cd="ambiguous">superscript</csymbol>
       <apply>
        <csymbol cd="ambiguous">subscript</csymbol>
        <ci>K</ci>
        <ci>i</ci>
       </apply>
       <ci>normal-′</ci>
      </apply>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \alpha^{\prime}=1+\frac{[I]}{K_{i}^{\prime}}.
  </annotation>
 </semantics>
</math>

</p>

<p>Thus, in the presence of the inhibitor, the enzyme's effective <em>K</em><sub>m</sub> and <em>V</em><sub>max</sub> become (α/α')<em>K</em><sub>m</sub> and (1/α')<em>V</em><sub>max</sub>, respectively. However, the modified Michaelis-Menten equation assumes that binding of the inhibitor to the enzyme has reached equilibrium, which may be a very slow process for inhibitors with sub-nanomolar dissociation constants. In these cases, it is usually more practical to treat the tight-binding inhibitor as an irreversible inhibitor (see below); however, it can still be possible to estimate <em>K</em><sub>i</sub>' kinetically if <em>K</em><sub>i</sub> is measured independently.</p>

<p>The effects of different types of reversible enzyme inhibitors on enzymatic activity can be visualized using graphical representations of the Michaelis–Menten equation, such as <a href="Lineweaver–Burk_plot" title="wikilink">Lineweaver–Burk</a> and <a href="Eadie-Hofstee_diagram" title="wikilink">Eadie-Hofstee plots</a>. For example, in the Lineweaver–Burk plots at the right, the competitive inhibition lines intersect on the <em>y</em>-axis, illustrating that such inhibitors do not affect <em>V</em><sub>max</sub>. Similarly, the non-competitive inhibition lines intersect on the <em>x</em>-axis, showing these inhibitors do not affect <em>K</em><sub>m</sub>. However, it can be difficult to estimate <em>K</em><sub>i</sub> and <em>K</em><sub>i</sub>' accurately from such plots,<a class="footnoteRef" href="#fn7" id="fnref7"><sup>7</sup></a> so it is advisable to estimate these constants using more reliable <a href="nonlinear_regression" title="wikilink">nonlinear regression</a> methods, as described above.</p>
<h3 id="reversible-inhibitors-1">Reversible inhibitors</h3>

<p>Traditionally reversible enzyme inhibitors have been classified as competitive, uncompetitive, or non-competitive, according to their effects on <em>K</em><sub>m</sub> and <em>V</em><sub>max</sub>. These different effects result from the inhibitor binding to the enzyme E, to the enzyme–substrate complex ES, or to both, respectively. The division of these classes arises from a problem in their derivation and results in the need to use two different binding constants for one binding event. The binding of an inhibitor and its effect on the enzymatic activity are two distinctly different things, another problem the traditional equations fail to acknowledge. In noncompetitive inhibition the binding of the inhibitor results in 100% inhibition of the enzyme only, and fails to consider the possibility of anything in between.<a class="footnoteRef" href="#fn8" id="fnref8"><sup>8</sup></a> The common form of the inhibitory term also obscures the relationship between the inhibitor binding to the enzyme and its relationship to any other binding term be it the Michaelis–Menten equation or a dose response curve associated with ligand receptor binding. To demonstrate the relationship the following rearrangement can be made:</p>

<p>

<math display="block" id="Enzyme_inhibitor:3">
 <semantics>
  <mfrac>
   <msub>
    <mi>V</mi>
    <mi>max</mi>
   </msub>
   <mrow>
    <mn>1</mn>
    <mo>+</mo>
    <mstyle displaystyle="true">
     <mfrac>
      <mrow>
       <mo stretchy="false">[</mo>
       <mi>I</mi>
       <mo stretchy="false">]</mo>
      </mrow>
      <msub>
       <mi>K</mi>
       <mi>i</mi>
      </msub>
     </mfrac>
    </mstyle>
   </mrow>
  </mfrac>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <csymbol cd="latexml">continued-fraction</csymbol>
    <apply>
     <csymbol cd="ambiguous">subscript</csymbol>
     <ci>V</ci>
     <max></max>
    </apply>
    <apply>
     <plus></plus>
     <cn type="integer">1</cn>
     <apply>
      <csymbol cd="latexml">continued-fraction</csymbol>
      <apply>
       <csymbol cd="latexml">delimited-[]</csymbol>
       <ci>I</ci>
      </apply>
      <apply>
       <csymbol cd="ambiguous">subscript</csymbol>
       <ci>K</ci>
       <ci>i</ci>
      </apply>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \cfrac{V_{\max}}{1+\cfrac{[I]}{K_{i}}}
  </annotation>
 </semantics>
</math>

</p>

<p>

<math display="block" id="Enzyme_inhibitor:4">
 <semantics>
  <mfrac>
   <msub>
    <mi>V</mi>
    <mi>max</mi>
   </msub>
   <mstyle displaystyle="true">
    <mfrac>
     <mrow>
      <mrow>
       <mo stretchy="false">[</mo>
       <mi>I</mi>
       <mo stretchy="false">]</mo>
      </mrow>
      <mo>+</mo>
      <msub>
       <mi>K</mi>
       <mi>i</mi>
      </msub>
     </mrow>
     <msub>
      <mi>K</mi>
      <mi>i</mi>
     </msub>
    </mfrac>
   </mstyle>
  </mfrac>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <csymbol cd="latexml">continued-fraction</csymbol>
    <apply>
     <csymbol cd="ambiguous">subscript</csymbol>
     <ci>V</ci>
     <max></max>
    </apply>
    <apply>
     <csymbol cd="latexml">continued-fraction</csymbol>
     <apply>
      <plus></plus>
      <apply>
       <csymbol cd="latexml">delimited-[]</csymbol>
       <ci>I</ci>
      </apply>
      <apply>
       <csymbol cd="ambiguous">subscript</csymbol>
       <ci>K</ci>
       <ci>i</ci>
      </apply>
     </apply>
     <apply>
      <csymbol cd="ambiguous">subscript</csymbol>
      <ci>K</ci>
      <ci>i</ci>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \cfrac{V_{\max}}{\cfrac{[I]+K_{i}}{K_{i}}}
  </annotation>
 </semantics>
</math>

</p>

<p>Adding zero to the bottom ([I]-[I])</p>

<p>

<math display="block" id="Enzyme_inhibitor:5">
 <semantics>
  <mfrac>
   <msub>
    <mi>V</mi>
    <mi>max</mi>
   </msub>
   <mstyle displaystyle="true">
    <mfrac>
     <mrow>
      <mrow>
       <mo stretchy="false">[</mo>
       <mi>I</mi>
       <mo stretchy="false">]</mo>
      </mrow>
      <mo>+</mo>
      <msub>
       <mi>K</mi>
       <mi>i</mi>
      </msub>
     </mrow>
     <mrow>
      <mrow>
       <mrow>
        <mo stretchy="false">[</mo>
        <mi>I</mi>
        <mo stretchy="false">]</mo>
       </mrow>
       <mo>+</mo>
       <msub>
        <mi>K</mi>
        <mi>i</mi>
       </msub>
      </mrow>
      <mo>-</mo>
      <mrow>
       <mo stretchy="false">[</mo>
       <mi>I</mi>
       <mo stretchy="false">]</mo>
      </mrow>
     </mrow>
    </mfrac>
   </mstyle>
  </mfrac>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <csymbol cd="latexml">continued-fraction</csymbol>
    <apply>
     <csymbol cd="ambiguous">subscript</csymbol>
     <ci>V</ci>
     <max></max>
    </apply>
    <apply>
     <csymbol cd="latexml">continued-fraction</csymbol>
     <apply>
      <plus></plus>
      <apply>
       <csymbol cd="latexml">delimited-[]</csymbol>
       <ci>I</ci>
      </apply>
      <apply>
       <csymbol cd="ambiguous">subscript</csymbol>
       <ci>K</ci>
       <ci>i</ci>
      </apply>
     </apply>
     <apply>
      <minus></minus>
      <apply>
       <plus></plus>
       <apply>
        <csymbol cd="latexml">delimited-[]</csymbol>
        <ci>I</ci>
       </apply>
       <apply>
        <csymbol cd="ambiguous">subscript</csymbol>
        <ci>K</ci>
        <ci>i</ci>
       </apply>
      </apply>
      <apply>
       <csymbol cd="latexml">delimited-[]</csymbol>
       <ci>I</ci>
      </apply>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \cfrac{V_{\max}}{\cfrac{[I]+K_{i}}{[I]+K_{i}-[I]}}
  </annotation>
 </semantics>
</math>

</p>

<p>Dividing by [I]+K<sub>i</sub></p>

<p>

<math display="block" id="Enzyme_inhibitor:6">
 <semantics>
  <mfrac>
   <msub>
    <mi>V</mi>
    <mi>max</mi>
   </msub>
   <mstyle displaystyle="true">
    <mfrac>
     <mn>1</mn>
     <mrow>
      <mn>1</mn>
      <mo>-</mo>
      <mstyle displaystyle="true">
       <mfrac>
        <mrow>
         <mo stretchy="false">[</mo>
         <mi>I</mi>
         <mo stretchy="false">]</mo>
        </mrow>
        <mrow>
         <mrow>
          <mo stretchy="false">[</mo>
          <mi>I</mi>
          <mo stretchy="false">]</mo>
         </mrow>
         <mo>+</mo>
         <msub>
          <mi>K</mi>
          <mi>i</mi>
         </msub>
        </mrow>
       </mfrac>
      </mstyle>
     </mrow>
    </mfrac>
   </mstyle>
  </mfrac>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <csymbol cd="latexml">continued-fraction</csymbol>
    <apply>
     <csymbol cd="ambiguous">subscript</csymbol>
     <ci>V</ci>
     <max></max>
    </apply>
    <apply>
     <csymbol cd="latexml">continued-fraction</csymbol>
     <cn type="integer">1</cn>
     <apply>
      <minus></minus>
      <cn type="integer">1</cn>
      <apply>
       <csymbol cd="latexml">continued-fraction</csymbol>
       <apply>
        <csymbol cd="latexml">delimited-[]</csymbol>
        <ci>I</ci>
       </apply>
       <apply>
        <plus></plus>
        <apply>
         <csymbol cd="latexml">delimited-[]</csymbol>
         <ci>I</ci>
        </apply>
        <apply>
         <csymbol cd="ambiguous">subscript</csymbol>
         <ci>K</ci>
         <ci>i</ci>
        </apply>
       </apply>
      </apply>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \cfrac{V_{\max}}{\cfrac{1}{1-\cfrac{[I]}{[I]+K_{i}}}}
  </annotation>
 </semantics>
</math>

</p>

<p>

<math display="block" id="Enzyme_inhibitor:7">
 <semantics>
  <mrow>
   <msub>
    <mi>V</mi>
    <mi>max</mi>
   </msub>
   <mo>-</mo>
   <mrow>
    <msub>
     <mi>V</mi>
     <mi>max</mi>
    </msub>
    <mfrac>
     <mrow>
      <mo stretchy="false">[</mo>
      <mi>I</mi>
      <mo stretchy="false">]</mo>
     </mrow>
     <mrow>
      <mrow>
       <mo stretchy="false">[</mo>
       <mi>I</mi>
       <mo stretchy="false">]</mo>
      </mrow>
      <mo>+</mo>
      <msub>
       <mi>K</mi>
       <mi>i</mi>
      </msub>
     </mrow>
    </mfrac>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <minus></minus>
    <apply>
     <csymbol cd="ambiguous">subscript</csymbol>
     <ci>V</ci>
     <max></max>
    </apply>
    <apply>
     <times></times>
     <apply>
      <csymbol cd="ambiguous">subscript</csymbol>
      <ci>V</ci>
      <max></max>
     </apply>
     <apply>
      <csymbol cd="latexml">continued-fraction</csymbol>
      <apply>
       <csymbol cd="latexml">delimited-[]</csymbol>
       <ci>I</ci>
      </apply>
      <apply>
       <plus></plus>
       <apply>
        <csymbol cd="latexml">delimited-[]</csymbol>
        <ci>I</ci>
       </apply>
       <apply>
        <csymbol cd="ambiguous">subscript</csymbol>
        <ci>K</ci>
        <ci>i</ci>
       </apply>
      </apply>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   V_{\max}-V_{\max}\cfrac{[I]}{[I]+K_{i}}
  </annotation>
 </semantics>
</math>

</p>

<p>This notation demonstrates that similar to the Michaelis–Menten equation,where the rate of reaction depends on the percent of the enzyme population interacting with substrate.</p>

<p>fraction of the enzyme population bound by substrate</p>

<p>

<math display="block" id="Enzyme_inhibitor:8">
 <semantics>
  <mfrac>
   <mrow>
    <mo stretchy="false">[</mo>
    <mi>S</mi>
    <mo stretchy="false">]</mo>
   </mrow>
   <mrow>
    <mrow>
     <mo stretchy="false">[</mo>
     <mi>S</mi>
     <mo stretchy="false">]</mo>
    </mrow>
    <mo>+</mo>
    <msub>
     <mi>K</mi>
     <mi>m</mi>
    </msub>
   </mrow>
  </mfrac>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <csymbol cd="latexml">continued-fraction</csymbol>
    <apply>
     <csymbol cd="latexml">delimited-[]</csymbol>
     <ci>S</ci>
    </apply>
    <apply>
     <plus></plus>
     <apply>
      <csymbol cd="latexml">delimited-[]</csymbol>
      <ci>S</ci>
     </apply>
     <apply>
      <csymbol cd="ambiguous">subscript</csymbol>
      <ci>K</ci>
      <ci>m</ci>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \cfrac{[S]}{[S]+K_{m}}
  </annotation>
 </semantics>
</math>

</p>

<p>fraction of the enzyme population bound by inhibitor</p>

<p>

<math display="block" id="Enzyme_inhibitor:9">
 <semantics>
  <mfrac>
   <mrow>
    <mo stretchy="false">[</mo>
    <mi>I</mi>
    <mo stretchy="false">]</mo>
   </mrow>
   <mrow>
    <mrow>
     <mo stretchy="false">[</mo>
     <mi>I</mi>
     <mo stretchy="false">]</mo>
    </mrow>
    <mo>+</mo>
    <msub>
     <mi>K</mi>
     <mi>i</mi>
    </msub>
   </mrow>
  </mfrac>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <csymbol cd="latexml">continued-fraction</csymbol>
    <apply>
     <csymbol cd="latexml">delimited-[]</csymbol>
     <ci>I</ci>
    </apply>
    <apply>
     <plus></plus>
     <apply>
      <csymbol cd="latexml">delimited-[]</csymbol>
      <ci>I</ci>
     </apply>
     <apply>
      <csymbol cd="ambiguous">subscript</csymbol>
      <ci>K</ci>
      <ci>i</ci>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \cfrac{[I]}{[I]+K_{i}}
  </annotation>
 </semantics>
</math>

</p>

<p>the effect of the inhibitor is a result of the percent of the enzyme population interacting with inhibitor. The only problem with this equation in its present form is that it assumes absolute inhibition of the enzyme with inhibitor binding, when in fact there can be a wide range of effects anywhere from 100% inhibition of substrate turn over to just &gt;0%. To account for this the equation can be easily modified to allow for different degrees of inhibition by including a delta <em>V</em><sub>max</sub> term.</p>

<p>

<math display="block" id="Enzyme_inhibitor:10">
 <semantics>
  <mrow>
   <msub>
    <mi>V</mi>
    <mi>max</mi>
   </msub>
   <mo>-</mo>
   <mrow>
    <mi mathvariant="normal">Δ</mi>
    <msub>
     <mi>V</mi>
     <mi>max</mi>
    </msub>
    <mfrac>
     <mrow>
      <mo stretchy="false">[</mo>
      <mi>I</mi>
      <mo stretchy="false">]</mo>
     </mrow>
     <mrow>
      <mrow>
       <mo stretchy="false">[</mo>
       <mi>I</mi>
       <mo stretchy="false">]</mo>
      </mrow>
      <mo>+</mo>
      <msub>
       <mi>K</mi>
       <mi>i</mi>
      </msub>
     </mrow>
    </mfrac>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <minus></minus>
    <apply>
     <csymbol cd="ambiguous">subscript</csymbol>
     <ci>V</ci>
     <max></max>
    </apply>
    <apply>
     <times></times>
     <ci>normal-Δ</ci>
     <apply>
      <csymbol cd="ambiguous">subscript</csymbol>
      <ci>V</ci>
      <max></max>
     </apply>
     <apply>
      <csymbol cd="latexml">continued-fraction</csymbol>
      <apply>
       <csymbol cd="latexml">delimited-[]</csymbol>
       <ci>I</ci>
      </apply>
      <apply>
       <plus></plus>
       <apply>
        <csymbol cd="latexml">delimited-[]</csymbol>
        <ci>I</ci>
       </apply>
       <apply>
        <csymbol cd="ambiguous">subscript</csymbol>
        <ci>K</ci>
        <ci>i</ci>
       </apply>
      </apply>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   V_{\max}-\Delta V_{\max}\cfrac{[I]}{[I]+K_{i}}
  </annotation>
 </semantics>
</math>

</p>

<p>or</p>

<p>

<math display="block" id="Enzyme_inhibitor:11">
 <semantics>
  <mrow>
   <mrow>
    <msub>
     <mi>V</mi>
     <mi>max</mi>
    </msub>
    <mn>1</mn>
   </mrow>
   <mo>-</mo>
   <mrow>
    <mrow>
     <mo stretchy="false">(</mo>
     <mrow>
      <mrow>
       <msub>
        <mi>V</mi>
        <mi>max</mi>
       </msub>
       <mn>1</mn>
      </mrow>
      <mo>-</mo>
      <mrow>
       <msub>
        <mi>V</mi>
        <mi>max</mi>
       </msub>
       <mn>2</mn>
      </mrow>
     </mrow>
     <mo stretchy="false">)</mo>
    </mrow>
    <mfrac>
     <mrow>
      <mo stretchy="false">[</mo>
      <mi>I</mi>
      <mo stretchy="false">]</mo>
     </mrow>
     <mrow>
      <mrow>
       <mo stretchy="false">[</mo>
       <mi>I</mi>
       <mo stretchy="false">]</mo>
      </mrow>
      <mo>+</mo>
      <msub>
       <mi>K</mi>
       <mi>i</mi>
      </msub>
     </mrow>
    </mfrac>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <minus></minus>
    <apply>
     <times></times>
     <apply>
      <csymbol cd="ambiguous">subscript</csymbol>
      <ci>V</ci>
      <max></max>
     </apply>
     <cn type="integer">1</cn>
    </apply>
    <apply>
     <times></times>
     <apply>
      <minus></minus>
      <apply>
       <times></times>
       <apply>
        <csymbol cd="ambiguous">subscript</csymbol>
        <ci>V</ci>
        <max></max>
       </apply>
       <cn type="integer">1</cn>
      </apply>
      <apply>
       <times></times>
       <apply>
        <csymbol cd="ambiguous">subscript</csymbol>
        <ci>V</ci>
        <max></max>
       </apply>
       <cn type="integer">2</cn>
      </apply>
     </apply>
     <apply>
      <csymbol cd="latexml">continued-fraction</csymbol>
      <apply>
       <csymbol cd="latexml">delimited-[]</csymbol>
       <ci>I</ci>
      </apply>
      <apply>
       <plus></plus>
       <apply>
        <csymbol cd="latexml">delimited-[]</csymbol>
        <ci>I</ci>
       </apply>
       <apply>
        <csymbol cd="ambiguous">subscript</csymbol>
        <ci>K</ci>
        <ci>i</ci>
       </apply>
      </apply>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   V_{\max}1-(V_{\max}1-V_{\max}2)\cfrac{[I]}{[I]+K_{i}}
  </annotation>
 </semantics>
</math>

</p>

<p>This term can then define the residual enzymatic activity present when the inhibitor is interacting with individual enzymes in the population. However the inclusion of this term has the added value of allowing for the possibility of activation if the secondary <em>V</em><sub>max</sub> term turns out to be higher than the initial term. To account for the possibly of activation as well the notation can then be rewritten replacing the inhibitor "I" with a modifier term denoted here as "X".</p>

<p>

<math display="block" id="Enzyme_inhibitor:12">
 <semantics>
  <mrow>
   <mrow>
    <msub>
     <mi>V</mi>
     <mi>max</mi>
    </msub>
    <mn>1</mn>
   </mrow>
   <mo>-</mo>
   <mrow>
    <mrow>
     <mo stretchy="false">(</mo>
     <mrow>
      <mrow>
       <msub>
        <mi>V</mi>
        <mi>max</mi>
       </msub>
       <mn>1</mn>
      </mrow>
      <mo>-</mo>
      <mrow>
       <msub>
        <mi>V</mi>
        <mi>max</mi>
       </msub>
       <mn>2</mn>
      </mrow>
     </mrow>
     <mo stretchy="false">)</mo>
    </mrow>
    <mfrac>
     <mrow>
      <mo stretchy="false">[</mo>
      <mi>X</mi>
      <mo stretchy="false">]</mo>
     </mrow>
     <mrow>
      <mrow>
       <mo stretchy="false">[</mo>
       <mi>X</mi>
       <mo stretchy="false">]</mo>
      </mrow>
      <mo>+</mo>
      <msub>
       <mi>K</mi>
       <mi>x</mi>
      </msub>
     </mrow>
    </mfrac>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <minus></minus>
    <apply>
     <times></times>
     <apply>
      <csymbol cd="ambiguous">subscript</csymbol>
      <ci>V</ci>
      <max></max>
     </apply>
     <cn type="integer">1</cn>
    </apply>
    <apply>
     <times></times>
     <apply>
      <minus></minus>
      <apply>
       <times></times>
       <apply>
        <csymbol cd="ambiguous">subscript</csymbol>
        <ci>V</ci>
        <max></max>
       </apply>
       <cn type="integer">1</cn>
      </apply>
      <apply>
       <times></times>
       <apply>
        <csymbol cd="ambiguous">subscript</csymbol>
        <ci>V</ci>
        <max></max>
       </apply>
       <cn type="integer">2</cn>
      </apply>
     </apply>
     <apply>
      <csymbol cd="latexml">continued-fraction</csymbol>
      <apply>
       <csymbol cd="latexml">delimited-[]</csymbol>
       <ci>X</ci>
      </apply>
      <apply>
       <plus></plus>
       <apply>
        <csymbol cd="latexml">delimited-[]</csymbol>
        <ci>X</ci>
       </apply>
       <apply>
        <csymbol cd="ambiguous">subscript</csymbol>
        <ci>K</ci>
        <ci>x</ci>
       </apply>
      </apply>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   V_{\max}1-(V_{\max}1-V_{\max}2)\cfrac{[X]}{[X]+K_{x}}
  </annotation>
 </semantics>
</math>

</p>

<p>While this terminology results in a simplified way of dealing with kinetic effects relating to the maximum velocity of the Michaelis–Menten equation, it highlights potential problems with the term used to describe effects relating to the <em>K</em><sub>m</sub>. The <em>K</em><sub>m</sub> relating to the affinity of the enzyme for the substrate should in most cases relate to potential changes in the binding site of the enzyme which would directly result from enzyme inhibitor interactions. As such a term similar to the one proposed above to modulate <em>V</em><sub>max</sub> should be appropriate in most situations:<a class="footnoteRef" href="#fn9" id="fnref9"><sup>9</sup></a><a class="footnoteRef" href="#fn10" id="fnref10"><sup>10</sup></a></p>

<p>

<math display="block" id="Enzyme_inhibitor:13">
 <semantics>
  <mrow>
   <mrow>
    <msub>
     <mi>K</mi>
     <mi>m</mi>
    </msub>
    <mn>1</mn>
   </mrow>
   <mo>-</mo>
   <mrow>
    <mrow>
     <mo stretchy="false">(</mo>
     <mrow>
      <mrow>
       <msub>
        <mi>K</mi>
        <mi>m</mi>
       </msub>
       <mn>1</mn>
      </mrow>
      <mo>-</mo>
      <mrow>
       <msub>
        <mi>K</mi>
        <mi>m</mi>
       </msub>
       <mn>2</mn>
      </mrow>
     </mrow>
     <mo stretchy="false">)</mo>
    </mrow>
    <mfrac>
     <mrow>
      <mo stretchy="false">[</mo>
      <mi>X</mi>
      <mo stretchy="false">]</mo>
     </mrow>
     <mrow>
      <mrow>
       <mo stretchy="false">[</mo>
       <mi>X</mi>
       <mo stretchy="false">]</mo>
      </mrow>
      <mo>+</mo>
      <msub>
       <mi>K</mi>
       <mi>x</mi>
      </msub>
     </mrow>
    </mfrac>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <minus></minus>
    <apply>
     <times></times>
     <apply>
      <csymbol cd="ambiguous">subscript</csymbol>
      <ci>K</ci>
      <ci>m</ci>
     </apply>
     <cn type="integer">1</cn>
    </apply>
    <apply>
     <times></times>
     <apply>
      <minus></minus>
      <apply>
       <times></times>
       <apply>
        <csymbol cd="ambiguous">subscript</csymbol>
        <ci>K</ci>
        <ci>m</ci>
       </apply>
       <cn type="integer">1</cn>
      </apply>
      <apply>
       <times></times>
       <apply>
        <csymbol cd="ambiguous">subscript</csymbol>
        <ci>K</ci>
        <ci>m</ci>
       </apply>
       <cn type="integer">2</cn>
      </apply>
     </apply>
     <apply>
      <csymbol cd="latexml">continued-fraction</csymbol>
      <apply>
       <csymbol cd="latexml">delimited-[]</csymbol>
       <ci>X</ci>
      </apply>
      <apply>
       <plus></plus>
       <apply>
        <csymbol cd="latexml">delimited-[]</csymbol>
        <ci>X</ci>
       </apply>
       <apply>
        <csymbol cd="ambiguous">subscript</csymbol>
        <ci>K</ci>
        <ci>x</ci>
       </apply>
      </apply>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   K_{m}1-(K_{m}1-K_{m}2)\cfrac{[X]}{[X]+K_{x}}
  </annotation>
 </semantics>
</math>

</p>
<h3 id="special-cases">Special cases</h3>
<ul>
<li>The mechanism of <strong>partially competitive inhibition</strong> is similar to that of non-competitive, except that the EIS complex has catalytic activity, which may be lower or even higher (partially competitive activation) than that of the enzyme–substrate (ES) complex. This inhibition typically displays a lower <em>V</em><sub>max</sub>, but an unaffected <em>K</em><sub>m</sub> value.<a class="footnoteRef" href="#fn11" id="fnref11"><sup>11</sup></a></li>
</ul>
<ul>
<li><strong><a href="Uncompetitive_inhibitor" title="wikilink">Uncompetitive inhibition</a></strong> occurs when the inhibitor binds only to the enzyme–substrate complex, not to the free enzyme; the EIS complex is catalytically inactive. This mode of inhibition is rare and causes a decrease in both <em>V</em><sub>max</sub> and the <em>K</em><sub>m</sub> value.<a class="footnoteRef" href="#fn12" id="fnref12"><sup>12</sup></a></li>
</ul>
<ul>
<li><strong>Substrate and product inhibition</strong> is where either the substrate or product of an enzyme reaction inhibit the enzyme's activity. This inhibition may follow the competitive, uncompetitive or mixed patterns. In substrate inhibition there is a progressive decrease in activity at high substrate concentrations. This may indicate the existence of two substrate-binding sites in the enzyme. At low substrate, the high-affinity site is occupied and normal <a href="Enzyme_kinetics" title="wikilink">kinetics</a> are followed. However, at higher concentrations, the second inhibitory site becomes occupied, inhibiting the enzyme.<a class="footnoteRef" href="#fn13" id="fnref13"><sup>13</sup></a> Product inhibition is often a regulatory feature in <a class="uri" href="metabolism" title="wikilink">metabolism</a> and can be a form of <a href="negative_feedback" title="wikilink">negative feedback</a>.</li>
</ul>
<ul>
<li><strong>Slow-tight inhibition</strong> occurs when the initial enzyme–inhibitor complex EI undergoes isomerisation to a second more tightly held complex, EI*, but the overall inhibition process is reversible. This manifests itself as slowly increasing enzyme inhibition. Under these conditions, traditional Michaelis–Menten kinetics give a false value for <em>K</em><sub>i</sub>, which is time–dependent. The true value of <em>K</em><sub>i</sub> can be obtained through more complex analysis of the on (<em>k</em><sub>on</sub>) and off (<em>k</em><sub>off</sub>) rate constants for inhibitor association. See <a href="#Irreversible_inhibitors" title="wikilink">irreversible inhibition</a> below for more information.</li>
</ul>
<h3 id="examples-of-reversible-inhibitors">Examples of reversible inhibitors</h3>
<figure><b>(Figure)</b>
<figcaption>Peptide-based HIV-1 protease inhibitor <a class="uri" href="ritonavir" title="wikilink">ritonavir</a></figcaption>
</figure>

<p>As enzymes have evolved to bind their substrates tightly, and most reversible inhibitors bind in the active site of enzymes, it is unsurprising that some of these inhibitors are strikingly similar in structure to the substrates of their targets. An example of these substrate mimics are the <a href="Protease_inhibitor_(pharmacology)" title="wikilink">protease inhibitors</a>, a very successful class of <a href="antiretroviral_drug" title="wikilink">antiretroviral drugs</a> used to treat <a class="uri" href="HIV" title="wikilink">HIV</a>.<a class="footnoteRef" href="#fn14" id="fnref14"><sup>14</sup></a> The structure of <a class="uri" href="ritonavir" title="wikilink">ritonavir</a>, a protease inhibitor based on a peptide and containing three <a href="peptide_bond" title="wikilink">peptide bonds</a>, is shown on the right. As this drug resembles the protein that is the substrate of the HIV protease, it competes with this substrate in the enzyme's active site.</p>

<p>Enzyme inhibitors are often designed to mimic the <a href="transition_state" title="wikilink">transition state</a> or intermediate of an enzyme-catalyzed reaction. This ensures that the inhibitor exploits the transition state stabilising effect of the enzyme, resulting in a better binding affinity (lower <em>K</em><sub>i</sub>) than substrate-based designs. An example of such a transition state inhibitor is the antiviral drug <a class="uri" href="oseltamivir" title="wikilink">oseltamivir</a>; this drug mimics the planar nature of the ring <a href="oxonium_ion" title="wikilink">oxonium ion</a> in the reaction of the viral enzyme <a class="uri" href="neuraminidase" title="wikilink">neuraminidase</a>.<a class="footnoteRef" href="#fn15" id="fnref15"><sup>15</sup></a></p>
<figure><b>(Figure)</b>
<figcaption>Nonpeptidic HIV-1 protease inhibitor <a class="uri" href="tipranavir" title="wikilink">tipranavir</a></figcaption>
</figure>

<p>However, not all inhibitors are based on the structures of substrates. For example, the structure of another HIV protease inhibitor <a class="uri" href="tipranavir" title="wikilink">tipranavir</a> is shown on the left. This molecule is not based on a peptide and has no obvious structural similarity to a protein substrate. These non-peptide inhibitors can be more stable than inhibitors containing peptide bonds, because they will not be substrates for <a href="peptidase" title="wikilink">peptidases</a> and are less likely to be degraded.<a class="footnoteRef" href="#fn16" id="fnref16"><sup>16</sup></a></p>

<p>In drug design it is important to consider the concentrations of substrates to which the target enzymes are exposed. For example, some <a href="protein_kinase" title="wikilink">protein kinase</a> inhibitors have chemical structures that are similar to <a href="adenosine_triphosphate" title="wikilink">adenosine triphosphate</a>, one of the substrates of these enzymes. However, drugs that are simple competitive inhibitors will have to compete with the high concentrations of ATP in the cell. Protein kinases can also be inhibited by competition at the binding sites where the kinases interact with their substrate proteins, and most proteins are present inside cells at concentrations much lower than the concentration of ATP. As a consequence, if two protein kinase inhibitors both bind in the active site with similar affinity, but only one has to compete with ATP, then the competitive inhibitor at the protein-binding site will inhibit the enzyme more effectively.<a class="footnoteRef" href="#fn17" id="fnref17"><sup>17</sup></a> </p>
<h2 id="irreversible-inhibitors">Irreversible inhibitors</h2>
<h3 id="types-of-irreversible-inhibition">Types of irreversible inhibition</h3>
<figure><b>(Figure)</b>
<figcaption>Reaction of the irreversible inhibitor <a href="DIFP" title="wikilink">diisopropylfluorophosphate</a> (DFP) with a serine protease</figcaption>
</figure>

<p>Irreversible inhibitors usually <a href="covalent" title="wikilink">covalently</a> modify an enzyme, and inhibition can therefore not be reversed. Irreversible inhibitors often contain reactive functional groups such as <a href="nitrogen_mustard" title="wikilink">nitrogen mustards</a>, <a href="aldehyde" title="wikilink">aldehydes</a>, <a href="haloalkane" title="wikilink">haloalkanes</a>, <a href="alkene" title="wikilink">alkenes</a>, <a href="Michael_acceptor" title="wikilink">Michael acceptors</a>, <a href="sulfonate" title="wikilink">phenyl sulfonates</a>, or <a href="Methoxy_arachidonyl_fluorophosphonate" title="wikilink">fluorophosphonates</a>. These <a href="electrophile" title="wikilink">electrophilic</a> groups react with amino acid side chains to form <a href="covalent_adducts" title="wikilink">covalent adducts</a>. The residues modified are those with side chains containing <a href="nucleophile" title="wikilink">nucleophiles</a> such as <a class="uri" href="hydroxyl" title="wikilink">hydroxyl</a> or <a href="thiol" title="wikilink">sulfhydryl</a> groups; these include the amino acids <a class="uri" href="serine" title="wikilink">serine</a> (as in <a href="diisopropylfluorophosphate" title="wikilink">DFP</a>, right), <a class="uri" href="cysteine" title="wikilink">cysteine</a>, <a class="uri" href="threonine" title="wikilink">threonine</a>, or <a class="uri" href="tyrosine" title="wikilink">tyrosine</a>.<a class="footnoteRef" href="#fn18" id="fnref18"><sup>18</sup></a></p>

<p>Irreversible inhibition is different from irreversible enzyme inactivation. Irreversible inhibitors are generally specific for one class of enzyme and do not inactivate all proteins; they do not function by destroying <a href="protein_structure" title="wikilink">protein structure</a> but by specifically altering the active site of their target. For example, extremes of pH or temperature usually cause <a href="denaturation_(biochemistry)" title="wikilink">denaturation</a> of all <a href="protein_structure" title="wikilink">protein structure</a>, but this is a non-specific effect. Similarly, some non-specific chemical treatments destroy protein structure: for example, heating in concentrated <a href="hydrochloric_acid" title="wikilink">hydrochloric acid</a> will hydrolyse the <a href="peptide_bond" title="wikilink">peptide bonds</a> holding proteins together, releasing free amino acids.<a class="footnoteRef" href="#fn19" id="fnref19"><sup>19</sup></a></p>

<p>Irreversible inhibitors display time-dependent inhibition and their potency therefore cannot be characterised by an IC<sub>50</sub> value. This is because the amount of active enzyme at a given concentration of irreversible inhibitor will be different depending on how long the inhibitor is pre-incubated with the enzyme. Instead, <em>k</em><sub>obs</sub>/[<em>I</em>] values are used,<a class="footnoteRef" href="#fn20" id="fnref20"><sup>20</sup></a> where<em>k</em><sub>obs</sub> is the observed pseudo-first order rate of inactivation (obtained by plotting the log of % activity vs. time) and [<em>I</em>] is the concentration of inhibitor. The <em>k</em><sub>obs</sub>/[<em>I</em>] parameter is valid as long as the inhibitor does not saturate binding with the enzyme (in which case <em>k</em><sub>obs</sub> = <em>k</em><sub>inact</sub>).</p>
<h3 id="analysis-of-irreversible-inhibition">Analysis of irreversible inhibition</h3>
<figure><b>(Figure)</b>
<figcaption>Kinetic scheme for irreversible inhibitors</figcaption>
</figure>

<p>As shown in the figure to the left, irreversible inhibitors form a reversible non-covalent complex with the enzyme (EI or ESI) and this then reacts to produce the covalently modified "dead-end complex" EI*. The rate at which EI* is formed is called the inactivation rate or <em>k</em><sub>inact</sub>. Since formation of EI may compete with ES, binding of irreversible inhibitors can be prevented by competition either with substrate or with a second, reversible inhibitor. This protection effect is good evidence of a specific reaction of the irreversible inhibitor with the active site.</p>

<p>The binding and inactivation steps of this reaction are investigated by incubating the enzyme with inhibitor and assaying the amount of activity remaining over time. The activity will be decreased in a time-dependent manner, usually following <a href="exponential_decay" title="wikilink">exponential decay</a>. Fitting these data to a <a href="rate_equation" title="wikilink">rate equation</a> gives the rate of inactivation at this concentration of inhibitor. This is done at several different concentrations of inhibitor. If a reversible EI complex is involved the inactivation rate will be saturable and fitting this curve will give <em>k</em><sub>inact</sub> and <em>K</em><sub>i</sub>.<a class="footnoteRef" href="#fn21" id="fnref21"><sup>21</sup></a></p>

<p>Another method that is widely used in these analyses is <a href="mass_spectrometry" title="wikilink">mass spectrometry</a>. Here, accurate measurement of the mass of the unmodified native enzyme and the inactivated enzyme gives the increase in mass caused by reaction with the inhibitor and shows the stoichiometry of the reaction.<a class="footnoteRef" href="#fn22" id="fnref22"><sup>22</sup></a> This is usually done using a <a class="uri" href="MALDI-TOF" title="wikilink">MALDI-TOF</a> mass spectrometer. In a complementary technique, <a href="peptide_mass_fingerprinting" title="wikilink">peptide mass fingerprinting</a> involves digestion of the native and modified protein with a <a class="uri" href="protease" title="wikilink">protease</a> such as <a class="uri" href="trypsin" title="wikilink">trypsin</a>. This will produce a set of <a href="peptide" title="wikilink">peptides</a> that can be analysed using a mass spectrometer. The peptide that changes in mass after reaction with the inhibitor will be the one that contains the site of modification.</p>
<h3 id="special-cases-1">Special cases</h3>
<figure><b>(Figure)</b>
<figcaption>Chemical mechanism for irreversible inhibition of ornithine decarboxylase by DFMO. Pyridoxal 5'-phosphate (Py) and enzyme (E) are not shown. Adapted from<a class="footnoteRef" href="#fn23" id="fnref23"><sup>23</sup></a></figcaption>
</figure>

<p>Not all irreversible inhibitors form covalent adducts with their enzyme targets. Some reversible inhibitors bind so tightly to their target enzyme that they are essentially irreversible. These tight-binding inhibitors may show kinetics similar to covalent irreversible inhibitors. In these cases, some of these inhibitors rapidly bind to the enzyme in a low-affinity EI complex and this then undergoes a slower rearrangement to a very tightly bound EI* complex (see figure above). This kinetic behaviour is called slow-binding.<a class="footnoteRef" href="#fn24" id="fnref24"><sup>24</sup></a> This slow rearrangement after binding often involves a <a href="conformational_change" title="wikilink">conformational change</a> as the enzyme "clamps down" around the inhibitor molecule. Examples of slow-binding inhibitors include some important drugs, such <a class="uri" href="methotrexate" title="wikilink">methotrexate</a>,<a class="footnoteRef" href="#fn25" id="fnref25"><sup>25</sup></a> <a class="uri" href="allopurinol" title="wikilink">allopurinol</a>,<a class="footnoteRef" href="#fn26" id="fnref26"><sup>26</sup></a> and the activated form of <a class="uri" href="acyclovir" title="wikilink">acyclovir</a>.<a class="footnoteRef" href="#fn27" id="fnref27"><sup>27</sup></a></p>
<h3 id="examples-of-irreversible-inhibitors">Examples of irreversible inhibitors</h3>
<figure><b>(Figure)</b>
<figcaption><a href="trypanothione" title="wikilink">Trypanothione reductase</a> with the lower molecule of an inhibitor bound irreversibly and the upper one reversibly. Created from <a href="http://www.rcsb.org/pdb/explore.do?structureId=1GXF">PDB 1GXF</a>.</figcaption>
</figure>

<p><a class="uri" href="Diisopropylfluorophosphate" title="wikilink">Diisopropylfluorophosphate</a> (DFP) is shown as an example of an irreversible protease inhibitor in the figure <a href="#Irreversible_inhibitors" title="wikilink">above right</a>. The enzyme hydrolyses the phosphorus–fluorine bond, but the phosphate residue remains bound to the serine in the <a href="catalytic_triad" title="wikilink">active site</a>, deactivating it.<a class="footnoteRef" href="#fn28" id="fnref28"><sup>28</sup></a> Similarly, DFP also reacts with the active site of <a href="acetylcholine_esterase" title="wikilink">acetylcholine esterase</a> in the <a class="uri" href="synapses" title="wikilink">synapses</a> of neurons, and consequently is a potent neurotoxin, with a lethal dose of less than 100 mg.<a class="footnoteRef" href="#fn29" id="fnref29"><sup>29</sup></a></p>

<p><a href="Suicide_inhibition" title="wikilink">Suicide inhibition</a> is an unusual type of irreversible inhibition where the enzyme converts the inhibitor into a reactive form in its active site. An example is the inhibitor of <a class="uri" href="polyamine" title="wikilink">polyamine</a> biosynthesis, <a href="eflornithine" title="wikilink">α-difluoromethylornithine</a> or DFMO, which is an analogue of the amino acid <a class="uri" href="ornithine" title="wikilink">ornithine</a>, and is used to treat <a href="African_trypanosomiasis" title="wikilink">African trypanosomiasis</a> (sleeping sickness). <a href="Ornithine_decarboxylase" title="wikilink">Ornithine decarboxylase</a> can catalyse the decarboxylation of DFMO instead of ornithine, as shown above. However, this decarboxylation reaction is followed by the elimination of a fluorine atom, which converts this catalytic intermediate into a conjugated <a class="uri" href="imine" title="wikilink">imine</a>, a highly electrophilic species. This reactive form of DFMO then reacts with either a cysteine or lysine residue in the active site to irreversibly inactivate the enzyme.<a class="footnoteRef" href="#fn30" id="fnref30"><sup>30</sup></a></p>

<p>Since irreversible inhibition often involves the initial formation of a non-covalent EI complex, it is sometimes possible for an inhibitor to bind to an enzyme in more than one way. For example, in the figure showing <a href="trypanothione" title="wikilink">trypanothione reductase</a> from the human protozoan parasite <em><a href="Trypanosoma_cruzi" title="wikilink">Trypanosoma cruzi</a></em>, two molecules of an inhibitor called <em>quinacrine mustard</em> are bound in its active site. The top molecule is bound reversibly, but the lower one is bound covalently as it has reacted with an amino acid residue through its <a href="nitrogen_mustard" title="wikilink">nitrogen mustard</a> group.<a class="footnoteRef" href="#fn31" id="fnref31"><sup>31</sup></a></p>
<h2 id="discovery-and-design-of-inhibitors">Discovery and design of inhibitors</h2>
<figure><b>(Figure)</b>
<figcaption>Robots used for the high-throughput screening of chemical libraries to discover new enzyme inhibitors</figcaption>
</figure>

<p>New drugs are the products of a long <a href="drug_development" title="wikilink">drug development</a> process, the first step of which is often the discovery of a new enzyme inhibitor. In the past the only way to discover these new inhibitors was by trial and error: screening huge libraries of compounds against a target enzyme and hoping that some useful leads would emerge. This brute force approach is still successful and has even been extended by <a href="combinatorial_chemistry" title="wikilink">combinatorial chemistry</a> approaches that quickly produce large numbers of novel compounds and <a href="high-throughput_screening" title="wikilink">high-throughput screening</a> technology to rapidly screen these huge chemical libraries for useful inhibitors.<a class="footnoteRef" href="#fn32" id="fnref32"><sup>32</sup></a></p>

<p>More recently, an alternative approach has been applied: <a href="rational_drug_design" title="wikilink">rational drug design</a> uses the <a href="protein_structure" title="wikilink">three-dimensional structure</a> of an enzyme's active site to predict which molecules might be inhibitors.<a class="footnoteRef" href="#fn33" id="fnref33"><sup>33</sup></a> These predictions are then tested and one of these tested compounds may be a novel inhibitor. This new inhibitor is then used to try to obtain a structure of the enzyme in an inhibitor/enzyme complex to show how the molecule is binding to the active site, allowing changes to be made to the inhibitor to try to optimise binding. This test and improve cycle is then repeated until a sufficiently potent inhibitor is produced.<a class="footnoteRef" href="#fn34" id="fnref34"><sup>34</sup></a> <a href="Bioinformatics" title="wikilink">Computer-based methods</a> of predicting the affinity of an inhibitor for an enzyme are also being developed, such as <a href="Docking_(molecular)" title="wikilink">molecular docking</a><a class="footnoteRef" href="#fn35" id="fnref35"><sup>35</sup></a> and <a href="molecular_mechanics" title="wikilink">molecular mechanics</a>.</p>
<h2 id="uses-of-inhibitors">Uses of inhibitors</h2>

<p>Enzyme inhibitors are found in nature and are also designed and produced as part of <a class="uri" href="pharmacology" title="wikilink">pharmacology</a> and <a class="uri" href="biochemistry" title="wikilink">biochemistry</a>. Natural <a href="poison" title="wikilink">poisons</a> are often enzyme inhibitors that have evolved to defend a plant or animal against <a href="predation" title="wikilink">predators</a>. These natural toxins include some of the most poisonous compounds known. Artificial inhibitors are often used as drugs, but can also be <a href="insecticide" title="wikilink">insecticides</a> such as <a class="uri" href="malathion" title="wikilink">malathion</a>, <a href="herbicide" title="wikilink">herbicides</a> such as <a class="uri" href="glyphosate" title="wikilink">glyphosate</a>, or <a href="disinfection" title="wikilink">disinfectants</a> such as <a class="uri" href="triclosan" title="wikilink">triclosan</a>. Other artificial enzyme inhibitors block <a class="uri" href="acetylcholinesterase" title="wikilink">acetylcholinesterase</a>, an enzyme which breaks down <a class="uri" href="acetylcholine" title="wikilink">acetylcholine</a>, and are used as <a href="nerve_agent" title="wikilink">nerve agents</a> in <a href="chemical_warfare" title="wikilink">chemical warfare</a>.</p>
<h3 id="chemotherapy">Chemotherapy</h3>
<table>
<tbody>
<tr class="odd">
<td style="text-align: left;"></td>
<td style="text-align: left;"><figure><b>(Figure)</b>
<figcaption>The structure of <a class="uri" href="sildenafil" title="wikilink">sildenafil</a> (Viagra)</figcaption>
</figure></td>
</tr>
<tr class="even">
<td style="text-align: left;"></td>
<td style="text-align: left;">
<p>[[<a class="uri" href="File:Methotrexate">File:Methotrexate</a> and folic acid compared.png</p></td>
</tr>
<tr class="odd">
<td style="text-align: left;"></td>
<td style="text-align: left;"><figure><b>(Figure)</b>
<figcaption>The structure of a complex between penicillin G and the <em>Streptomyces</em> transpeptidase. Generated from <a href="http://www.rcsb.org/pdb/explore.do?structureId=1PWC">PDB 1PWC</a>.</figcaption>
</figure></td>
</tr>
</tbody>
</table>

<p>The most common uses for enzyme inhibitors are as drugs to treat disease. Many of these inhibitors target a human enzyme and aim to correct a pathological condition. However, not all drugs are enzyme inhibitors. Some, such as <a href="anticonvulsant" title="wikilink">anti-epileptic drugs</a>, alter enzyme activity by causing more or less of the enzyme to be produced. These effects are called <a href="enzyme_induction_and_inhibition" title="wikilink">enzyme induction and inhibition</a> and are alterations in <a href="gene_expression" title="wikilink">gene expression</a>, which is unrelated to the type of enzyme inhibition discussed here. Other drugs interact with cellular targets that are not enzymes, such as <a href="ion_channel" title="wikilink">ion channels</a> or <a href="receptor_(biochemistry)" title="wikilink">membrane receptors</a>.</p>

<p>An example of a medicinal enzyme inhibitor is <a class="uri" href="sildenafil" title="wikilink">sildenafil</a> (Viagra), a common treatment for male erectile dysfunction. This compound is a potent inhibitor of <a href="cGMP_specific_phosphodiesterase_type_5" title="wikilink">cGMP specific phosphodiesterase type 5</a>, the enzyme that degrades the <a href="cell_signaling" title="wikilink">signalling</a> molecule <a href="cyclic_guanosine_monophosphate" title="wikilink">cyclic guanosine monophosphate</a>.<a class="footnoteRef" href="#fn36" id="fnref36"><sup>36</sup></a> This signalling molecule triggers smooth muscle relaxation and allows blood flow into the <a href="corpus_cavernosum_penis" title="wikilink">corpus cavernosum</a>, which causes an erection. Since the drug decreases the activity of the enzyme that halts the signal, it makes this signal last for a longer period of time.</p>

<p>Another example of the structural similarity of some inhibitors to the substrates of the enzymes they target is seen in the figure comparing the drug <a class="uri" href="methotrexate" title="wikilink">methotrexate</a> to <a href="folic_acid" title="wikilink">folic acid</a>. Folic acid is a substrate of <a href="dihydrofolate_reductase" title="wikilink">dihydrofolate reductase</a>, an enzyme involved in making <a href="nucleotide" title="wikilink">nucleotides</a> that is potently inhibited by methotrexate. Methotrexate blocks the action of dihydrofolate reductase and thereby halts the production of nucleotides. This block of nucleotide biosynthesis is more toxic to rapidly growing cells than non-dividing cells, since a rapidly growing cell has to carry out <a href="DNA_replication" title="wikilink">DNA replication</a>, therefore methotrexate is often used in cancer <a class="uri" href="chemotherapy" title="wikilink">chemotherapy</a>.<a class="footnoteRef" href="#fn37" id="fnref37"><sup>37</sup></a></p>

<p>Drugs also are used to inhibit enzymes needed for the survival of <a href="pathogen" title="wikilink">pathogens</a>. For example, bacteria are surrounded by a thick <a href="bacterial_cell_structure" title="wikilink">cell wall</a> made of a net-like polymer called <a class="uri" href="peptidoglycan" title="wikilink">peptidoglycan</a>. Many antibiotics such as <a class="uri" href="penicillin" title="wikilink">penicillin</a> and <a class="uri" href="vancomycin" title="wikilink">vancomycin</a> inhibit the enzymes that produce and then cross-link the strands of this polymer together.<a class="footnoteRef" href="#fn38" id="fnref38"><sup>38</sup></a> This causes the cell wall to lose strength and the bacteria to burst. In the figure, a molecule of penicillin (shown in a ball-and-stick form) is shown bound to its target, the <a href="DD-transpeptidase" title="wikilink">transpeptidase</a> from the bacteria <em>Streptomyces</em> R61 (the protein is shown as a <a href="protein_structure" title="wikilink">ribbon-diagram</a>).</p>

<p><a href="Drug_design" title="wikilink">Drug design</a> is facilitated when an enzyme that is essential to the pathogen's survival is absent or very different in humans. In the example above, humans do not make peptidoglycan, therefore inhibitors of this process are selectively toxic to bacteria. Selective toxicity is also produced in antibiotics by exploiting differences in the structure of the <a href="ribosome" title="wikilink">ribosomes</a> in bacteria, or how they make <a href="fatty_acid" title="wikilink">fatty acids</a>.</p>
<h3 id="metabolic-control">Metabolic control</h3>

<p>Enzyme inhibitors are also important in metabolic control. Many <a href="metabolic_pathway" title="wikilink">metabolic pathways</a> in the cell are inhibited by <a href="metabolite" title="wikilink">metabolites</a> that control enzyme activity through <a href="allosteric_regulation" title="wikilink">allosteric regulation</a> or substrate inhibition. A good example is the allosteric regulation of the <a href="glycolysis" title="wikilink">glycolytic pathway</a>. This <a href="catabolism" title="wikilink">catabolic</a> pathway consumes <a class="uri" href="glucose" title="wikilink">glucose</a> and produces <a href="Adenosine_triphosphate" title="wikilink">ATP</a>, <a href="Nicotinamide_adenine_dinucleotide" title="wikilink">NADH</a> and <a class="uri" href="pyruvate" title="wikilink">pyruvate</a>. A key step for the regulation of glycolysis is an early reaction in the pathway catalysed by <a href="phosphofructokinase" title="wikilink">phosphofructokinase-1</a> (PFK1). When ATP levels rise, ATP binds an allosteric site in PFK1 to decrease the rate of the enzyme reaction; glycolysis is inhibited and ATP production falls. This <a href="feedback" title="wikilink">negative feedback</a> control helps maintain a steady concentration of ATP in the cell. However, metabolic pathways are not just regulated through inhibition since enzyme activation is equally important. With respect to PFK1, <a href="fructose_2,6-bisphosphate" title="wikilink">fructose 2,6-bisphosphate</a> and <a href="Adenosine_diphosphate" title="wikilink">ADP</a> are examples of metabolites that are allosteric activators.<a class="footnoteRef" href="#fn39" id="fnref39"><sup>39</sup></a></p>

<p>Physiological enzyme inhibition can also be produced by specific protein inhibitors. This mechanism occurs in the <a class="uri" href="pancreas" title="wikilink">pancreas</a>, which synthesises many digestive precursor enzymes known as <a href="zymogen" title="wikilink">zymogens</a>. Many of these are activated by the <a class="uri" href="trypsin" title="wikilink">trypsin</a> protease, so it is important to inhibit the activity of trypsin in the pancreas to prevent the organ from digesting itself. One way in which the activity of trypsin is controlled is the production of a specific and potent <a href="trypsin_inhibitor" title="wikilink">trypsin inhibitor</a> protein in the pancreas. This inhibitor binds tightly to trypsin, preventing the trypsin activity that would otherwise be detrimental to the organ.<a class="footnoteRef" href="#fn40" id="fnref40"><sup>40</sup></a> Although the trypsin inhibitor is a protein, it avoids being hydrolysed as a substrate by the protease by excluding water from trypsin's active site and destabilising the transition state.<a class="footnoteRef" href="#fn41" id="fnref41"><sup>41</sup></a> Other examples of physiological enzyme inhibitor proteins include the <a class="uri" href="barstar" title="wikilink">barstar</a> inhibitor of the bacterial ribonuclease <a class="uri" href="barnase" title="wikilink">barnase</a><a class="footnoteRef" href="#fn42" id="fnref42"><sup>42</sup></a> and the inhibitors of <a href="phosphatase" title="wikilink">protein phosphatases</a>.<a class="footnoteRef" href="#fn43" id="fnref43"><sup>43</sup></a></p>
<h3 id="pesticides">Pesticides</h3>

<p>Many <a href="pesticide" title="wikilink">pesticides</a> are enzyme inhibitors. <a class="uri" href="Acetylcholinesterase" title="wikilink">Acetylcholinesterase</a> (AChE) is an enzyme found in animals from insects to humans. It is essential to nerve cell function through its mechanism of breaking down the neurotransmitter <a class="uri" href="acetylcholine" title="wikilink">acetylcholine</a> into its constituents, <a class="uri" href="acetate" title="wikilink">acetate</a> and <a class="uri" href="choline" title="wikilink">choline</a>. This is somewhat unique among neurotransmitters as most, including <a class="uri" href="serotonin" title="wikilink">serotonin</a>, <a class="uri" href="dopamine" title="wikilink">dopamine</a>, and <a class="uri" href="norepinephrine" title="wikilink">norepinephrine</a>, are absorbed from the <a href="synaptic_cleft" title="wikilink">synaptic cleft</a> rather than cleaved. A large number of AChE inhibitors are used in both medicine and agriculture. Reversible competitive inhibitors, such as <a class="uri" href="edrophonium" title="wikilink">edrophonium</a>, <a class="uri" href="physostigmine" title="wikilink">physostigmine</a>, and <a class="uri" href="neostigmine" title="wikilink">neostigmine</a>, are used in the treatment of <a href="myasthenia_gravis" title="wikilink">myasthenia gravis</a> and in anaesthesia. The <a class="uri" href="carbamate" title="wikilink">carbamate</a> pesticides are also examples of reversible AChE inhibitors. The <a class="uri" href="organophosphate" title="wikilink">organophosphate</a> pesticides such as <a class="uri" href="malathion" title="wikilink">malathion</a>, <a class="uri" href="parathion" title="wikilink">parathion</a>, and <a class="uri" href="chlorpyrifos" title="wikilink">chlorpyrifos</a> irreversibly inhibit acetylcholinesterase.</p>

<p>The herbicide <a class="uri" href="glyphosate" title="wikilink">glyphosate</a> is an inhibitor of <a href="3-phosphoshikimate_1-carboxyvinyltransferase" title="wikilink">3-phosphoshikimate 1-carboxyvinyltransferase</a>,<a class="footnoteRef" href="#fn44" id="fnref44"><sup>44</sup></a> other herbicides, such as the sulfonylureas inhibit the enzyme <a href="acetolactate_synthase" title="wikilink">acetolactate synthase</a>. Both these enzymes are needed for plants to make branched-chain <a href="amino_acid" title="wikilink">amino acids</a>. Many other enzymess are inhibited by herbicides, including enzymes needed for the biosynthesis of <a href="lipid" title="wikilink">lipids</a> and <a href="carotenoid" title="wikilink">carotenoids</a> and the processes of <a class="uri" href="photosynthesis" title="wikilink">photosynthesis</a> and <a href="oxidative_phosphorylation" title="wikilink">oxidative phosphorylation</a>.<a class="footnoteRef" href="#fn45" id="fnref45"><sup>45</sup></a> </p>
<h3 id="natural-poisons">Natural poisons</h3>

<p>Animals and plants have evolved to synthesise a vast array of poisonous products including <a href="secondary_metabolite" title="wikilink">secondary metabolites</a>, peptides and proteins that can act as inhibitors. Natural toxins are usually <a href="Small_molecule" title="wikilink">small organic molecules</a> and are so diverse that there are probably natural inhibitors for most metabolic processes.<a class="footnoteRef" href="#fn46" id="fnref46"><sup>46</sup></a> The metabolic processes targeted by natural poisons encompass more than enzymes in metabolic pathways and can also include the inhibition of receptor, channel and structural protein functions in a cell. For example, <a class="uri" href="paclitaxel" title="wikilink">paclitaxel</a> (taxol), an organic molecule found in the <a href="Taxus" title="wikilink">Pacific yew tree</a>, binds tightly to <a class="uri" href="tubulin" title="wikilink">tubulin</a> dimers and inhibits their assembly into <a href="microtubule" title="wikilink">microtubules</a> in the <a class="uri" href="cytoskeleton" title="wikilink">cytoskeleton</a>.<a class="footnoteRef" href="#fn47" id="fnref47"><sup>47</sup></a></p>

<p>Many natural poisons act as <a href="neurotoxin" title="wikilink">neurotoxins</a> that can cause <a class="uri" href="paralysis" title="wikilink">paralysis</a> leading to death and have functions for defence against predators or in hunting and capturing prey. Some of these natural inhibitors, despite their toxic attributes, are valuable for therapeutic uses at lower doses.<a class="footnoteRef" href="#fn48" id="fnref48"><sup>48</sup></a> An example of a neurotoxin are the <a href="glycoalkaloid" title="wikilink">glycoalkaloids</a>, from the plant species in the <em><a class="uri" href="Solanaceae" title="wikilink">Solanaceae</a></em> family (includes <a class="uri" href="potato" title="wikilink">potato</a>, <a class="uri" href="tomato" title="wikilink">tomato</a> and <a class="uri" href="eggplant" title="wikilink">eggplant</a>), that are <a href="cholinesterase" title="wikilink">acetylcholinesterase</a> inhibitors. Inhibition of this enzyme causes an uncontrolled increase in the acetylcholine neurotransmitter, muscular paralysis and then death. Neurotoxicity can also result from the inhibition of receptors; for example, <a class="uri" href="atropine" title="wikilink">atropine</a> from deadly nightshade (<em><a href="Atropa_belladonna" title="wikilink">Atropa belladonna</a></em>) that functions as a <a href="competitive_antagonist" title="wikilink">competitive antagonist</a> of the <a href="acetylcholine_receptor" title="wikilink">muscarinic acetylcholine receptors</a>.<a class="footnoteRef" href="#fn49" id="fnref49"><sup>49</sup></a></p>

<p>Although many natural toxins are secondary metabolites, these poisons also include peptides and proteins. An example of a toxic peptide is <a class="uri" href="alpha-amanitin" title="wikilink">alpha-amanitin</a>, which is found in relatives of the <a href="death_cap" title="wikilink">death cap</a> mushroom. This is a potent enzyme inhibitor, in this case preventing the <a href="RNA_polymerase_II" title="wikilink">RNA polymerase II</a> enzyme from transcribing DNA.<a class="footnoteRef" href="#fn50" id="fnref50"><sup>50</sup></a> The algal toxin <a class="uri" href="microcystin" title="wikilink">microcystin</a> is also a peptide and is an inhibitor of <a href="phosphatase" title="wikilink">protein phosphatases</a>.<a class="footnoteRef" href="#fn51" id="fnref51"><sup>51</sup></a> This toxin can contaminate water supplies after <a href="algal_bloom" title="wikilink">algal blooms</a> and is a known carcinogen that can also cause acute liver hemorrhage and death at higher doses.<a class="footnoteRef" href="#fn52" id="fnref52"><sup>52</sup></a></p>

<p>Proteins can also be natural poisons or <a href="antinutrient" title="wikilink">antinutrients</a>, such as the <a href="trypsin_inhibitor" title="wikilink">trypsin inhibitors</a> (discussed above) that are found in some <a href="pulse_(legume)" title="wikilink">legumes</a>, as shown in the figure above. A less common class of toxins are toxic enzymes: these act as irreversible inhibitors of their target enzymes and work by chemically modifying their substrate enzymes. An example is <a class="uri" href="ricin" title="wikilink">ricin</a>, an extremely potent protein toxin found in <a href="castor_oil_plant" title="wikilink">castor oil beans</a>. This enzyme is a <a class="uri" href="glycosidase" title="wikilink">glycosidase</a> that inactivates ribosomes. Since ricin is a catalytic irreversible inhibitor, this allows just a single molecule of ricin to kill a cell.<a class="footnoteRef" href="#fn53" id="fnref53"><sup>53</sup></a></p>
<h2 id="see-also">See also</h2>
<ul>
<li><a href="Activity-based_proteomics" title="wikilink">Activity-based proteomics</a> – a branch of <a class="uri" href="proteomics" title="wikilink">proteomics</a> that uses covalent enzyme inhibitors as reporters to monitor enzyme activity.</li>
<li><a class="uri" href="Antimetabolite" title="wikilink">Antimetabolite</a></li>
<li><a class="uri" href="Pharmacophore" title="wikilink">Pharmacophore</a></li>
<li><a href="Transition_state_analog" title="wikilink">Transition state analog</a></li>
</ul>
<h2 id="references">References</h2>
<h2 id="external-links">External links</h2>
<ul>
<li><a href="http://web.archive.org/web/20070228044059/http://orion1.paisley.ac.uk/kinetics/Chapter_3/contents_chap3.html">Web tutorial on enzyme inhibition</a>, Tutorial by Dr Peter Birch of the University of Paisley, containing very clear animations</li>
<li><a href="http://www.chem.qmul.ac.uk/iubmb/kinetics/ek4t6.html#p6">Symbolism and Terminology in Enzyme Kinetics</a>, Recommendations of the Nomenclature Committee of the International Union of Biochemistry (NC-IUB) on enzyme inhibition terminology</li>
<li><a href="http://pubchem.ncbi.nlm.nih.gov/">PubChem from NCBI</a>, Database of drugs and enzyme inhibitors</li>
<li><a href="http://www.brenda.uni-koeln.de/">BRENDA</a>, Database of enzymes giving lists of known inhibitors for each entry</li>
<li><a href="http://web.indstate.edu/thcme/mwking/enzyme-kinetics.html">Enzymes, Kinetics and Diagnostic Use</a>, On-line lecture concentrating on medical applications of enzyme inhibitors: by Dr. Michael W. King of the IU School of Medicine</li>
<li><a href="http://www.bindingdb.org/">BindingDB</a>, a public database of measured protein-ligand binding affinities.</li>
<li><a href="http://www.wiley.com/college/pratt/0471393878/student/animations/enzyme_inhibition/index.html">Enzyme Inhibition Animated Exercise</a> (tutorial + quizzes).</li>
</ul>

<p>"</p>

<p><a href="Category:Medicinal_chemistry" title="wikilink">Category:Medicinal chemistry</a> <a class="uri" href="Category:Metabolism" title="wikilink">Category:Metabolism</a> <a href="Category:Enzyme_inhibitors" title="wikilink"> </a></p>
<section class="footnotes">
<hr/>
<ol>
<li id="fn1"><a href="#fnref1">↩</a></li>
<li id="fn2">Berg J., Tymoczko J. and Stryer L. (2002) [<a class="uri" href="http://www.ncbi.nlm.nih.gov/books/bv.fcgi?call=bv.View">http://www.ncbi.nlm.nih.gov/books/bv.fcgi?call=bv.View</a>..ShowTOC&amp;rid;=stryer.TOC&amp;depth;=2 <em>Biochemistry.</em>] W. H. Freeman and Company, ISBN 0-7167-4955-6.<a href="#fnref2">↩</a></li>
<li id="fn3"><a href="#fnref3">↩</a></li>
<li id="fn4">*Irwin H. Segel, <em>Enzyme Kinetics : Behavior and Analysis of Rapid Equilibrium and Steady-State Enzyme Systems</em>. Wiley–Interscience; New edition (1993), ISBN 0-471-30309-7<a href="#fnref4">↩</a></li>
<li id="fn5"><a href="#fnref5">↩</a></li>
<li id="fn6"><a href="#fnref6">↩</a></li>
<li id="fn7"><a href="#fnref7">↩</a></li>
<li id="fn8"><a href="#fnref8">↩</a></li>
<li id="fn9"><a href="#fnref9">↩</a></li>
<li id="fn10"><a href="#fnref10">↩</a></li>
<li id="fn11">Segel, Irwin H. (1993) <em>Enzyme Kinetics : Behavior and Analysis of Rapid Equilibrium and Steady-State Enzyme Systems</em>. Wiley-Interscience; New edition , ISBN 0-471-30309-7.<a href="#fnref11">↩</a></li>
<li id="fn12"></li>
<li id="fn13">Dixon, M. Webb, E.C., Thorne, C.J.R. and Tipton K.F., <em>Enzymes</em> (3rd edition) Longman, London (1979) p. 126<a href="#fnref13">↩</a></li>
<li id="fn14"><a href="#fnref14">↩</a></li>
<li id="fn15"><a href="#fnref15">↩</a></li>
<li id="fn16"><a href="#fnref16">↩</a></li>
<li id="fn17"><a href="#fnref17">↩</a></li>
<li id="fn18">Lundblad R. L. <em>Chemical Reagents for Protein Modification</em> CRC Press Inc (2004) ISBN 0-8493-1983-8<a href="#fnref18">↩</a></li>
<li id="fn19">Price, N.; Hames, B. and Rickwood, D. (eds.) (1996) <em>Proteins LabFax</em> Academic Press, ISBN 0-12-564710-7.<a href="#fnref19">↩</a></li>
<li id="fn20"><a href="#fnref20">↩</a></li>
<li id="fn21"><a href="#fnref21">↩</a></li>
<li id="fn22"><a href="#fnref22">↩</a></li>
<li id="fn23"><a href="#fnref23">↩</a></li>
<li id="fn24"><a href="#fnref24">↩</a></li>
<li id="fn25"><a href="#fnref25">↩</a></li>
<li id="fn26"><a href="#fnref26">↩</a></li>
<li id="fn27"><a href="#fnref27">↩</a></li>
<li id="fn28"><a href="#fnref28">↩</a></li>
<li id="fn29">Brenner, G. M. (2000): <em>Pharmacology.</em> Philadelphia, PA: W.B. Saunders Company. ISBN 0-7216-7757-6<a href="#fnref29">↩</a></li>
<li id="fn30"></li>
<li id="fn31"><a href="#fnref31">↩</a></li>
<li id="fn32"><a href="#fnref32">↩</a></li>
<li id="fn33"><a href="#fnref33">↩</a></li>
<li id="fn34"><a href="#fnref34">↩</a></li>
<li id="fn35"><a href="#fnref35">↩</a></li>
<li id="fn36"><a href="#fnref36">↩</a></li>
<li id="fn37"><a href="#fnref37">↩</a></li>
<li id="fn38"><a href="#fnref38">↩</a></li>
<li id="fn39"><a href="#fnref39">↩</a></li>
<li id="fn40">Price, Nicholas and Stevens, Lewis (1999) <em>Fundamentals of Enzymology</em>, Oxford University Press, ISBN 0-19-850229-X.<a href="#fnref40">↩</a></li>
<li id="fn41"><a href="#fnref41">↩</a></li>
<li id="fn42"><a href="#fnref42">↩</a></li>
<li id="fn43"><a href="#fnref43">↩</a></li>
<li id="fn44"><a href="#fnref44">↩</a></li>
<li id="fn45"><a href="#fnref45">↩</a></li>
<li id="fn46"><a href="#fnref46">↩</a></li>
<li id="fn47"><a href="#fnref47">↩</a></li>
<li id="fn48"><a href="#fnref48">↩</a></li>
<li id="fn49"><a href="#fnref49">↩</a></li>
<li id="fn50"><a href="#fnref50">↩</a></li>
<li id="fn51"><a href="#fnref51">↩</a></li>
<li id="fn52"><a href="#fnref52">↩</a></li>
<li id="fn53"><a href="#fnref53">↩</a></li>
</ol>
</section>
</body>
</html>
