<html lang="en">
<head>
<meta charset="utf-8"/>
<title offset="284">F-statistics</title>
<script src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_SVG.js" type="text/javascript">
</script>
</head>
<body>
<h1>F-statistics</h1>
<hr/>

<p>In <a href="population_genetics" title="wikilink">population genetics</a>, <strong><em>F</em>-statistics</strong> (also known as fixation indices) describe the statistically expected level of <a href="Zygosity" title="wikilink">heterozygosity</a> in a population; more specifically the expected degree of (usually) a reduction in heterozygosity when compared to <a href="Hardy–Weinberg_law" title="wikilink">Hardy–Weinberg expectation</a>.</p>

<p><em>F</em>-statistics can also be thought of as a measure of the correlation between genes drawn at different levels of a (hierarchically) subdivided population. This correlation is influenced by several evolutionary processes, such as mutation, migration, <a class="uri" href="inbreeding" title="wikilink">inbreeding</a>, <a href="natural_selection" title="wikilink">natural selection</a>, or the <a href="Wahlund_effect" title="wikilink">Wahlund effect</a>, but it was originally designed to measure the amount of allelic fixation owing to <a href="genetic_drift" title="wikilink">genetic drift</a>.</p>

<p>The concept of <em>F</em>-statistics was developed during the 1920s by the American geneticist <a href="Sewall_Wright" title="wikilink">Sewall Wright</a>,<a class="footnoteRef" href="#fn1" id="fnref1"><sup>1</sup></a><a class="footnoteRef" href="#fn2" id="fnref2"><sup>2</sup></a> who was interested in inbreeding in <a class="uri" href="cattle" title="wikilink">cattle</a>. However, because <a href="complete_dominance" title="wikilink">complete dominance</a> causes the <a href="phenotype" title="wikilink">phenotypes</a> of <a href="Zygosity" title="wikilink">homozygote</a> dominants and heterozygotes to be the same, it was not until the advent of <a href="molecular_genetics" title="wikilink">molecular genetics</a> from the 1960s onwards that heterozygosity in populations could be measured.</p>

<p><em>F</em> can be used to define <a href="effective_population_size" title="wikilink">effective population size</a>.</p>
<h2 id="definitions-and-equations">Definitions and equations</h2>

<p>The measures F<sub>IS</sub>, <a href="Fixation_index" title="wikilink">F<sub>ST</sub></a>, and F<sub>IT</sub> are related to the amounts of heterozygosity at various levels of population structure. Together, they are called <em>F</em>-statistics, and are derived from <em>F</em>, the inbreeding coefficient. In a simple two-allele system with inbreeding, the genotypic frequencies are:</p>

<p>

<math display="block" id="F-statistics:0">
 <semantics>
  <mrow>
   <mrow>
    <mrow>
     <mrow>
      <msup>
       <mi>p</mi>
       <mn>2</mn>
      </msup>
      <mrow>
       <mo stretchy="false">(</mo>
       <mrow>
        <mn>1</mn>
        <mo>-</mo>
        <mi>F</mi>
       </mrow>
       <mo stretchy="false">)</mo>
      </mrow>
     </mrow>
     <mo>+</mo>
     <mrow>
      <mi>p</mi>
      <mi>F</mi>
      <mtext>for</mtext>
      <mi>𝐀𝐀</mi>
     </mrow>
    </mrow>
    <mo>;</mo>
    <mrow>
     <mn>2</mn>
     <mi>p</mi>
     <mi>q</mi>
     <mrow>
      <mo stretchy="false">(</mo>
      <mrow>
       <mn>1</mn>
       <mo>-</mo>
       <mi>F</mi>
      </mrow>
      <mo stretchy="false">)</mo>
     </mrow>
     <mtext>for</mtext>
     <mi>𝐀𝐚</mi>
    </mrow>
    <mo>;</mo>
    <mrow>
     <mrow>
      <mtext>and</mtext>
      <msup>
       <mi>q</mi>
       <mn>2</mn>
      </msup>
      <mrow>
       <mo stretchy="false">(</mo>
       <mrow>
        <mn>1</mn>
        <mo>-</mo>
        <mi>F</mi>
       </mrow>
       <mo stretchy="false">)</mo>
      </mrow>
     </mrow>
     <mo>+</mo>
     <mrow>
      <mi>q</mi>
      <mi>F</mi>
      <mtext>for</mtext>
      <mi>𝐚𝐚</mi>
     </mrow>
    </mrow>
   </mrow>
   <mo>.</mo>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <list>
    <apply>
     <plus></plus>
     <apply>
      <times></times>
      <apply>
       <csymbol cd="ambiguous">superscript</csymbol>
       <ci>p</ci>
       <cn type="integer">2</cn>
      </apply>
      <apply>
       <minus></minus>
       <cn type="integer">1</cn>
       <ci>F</ci>
      </apply>
     </apply>
     <apply>
      <times></times>
      <ci>p</ci>
      <ci>F</ci>
      <mtext>for</mtext>
      <ci>𝐀𝐀</ci>
     </apply>
    </apply>
    <apply>
     <times></times>
     <cn type="float">2</cn>
     <ci>p</ci>
     <ci>q</ci>
     <apply>
      <minus></minus>
      <cn type="integer">1</cn>
      <ci>F</ci>
     </apply>
     <mtext>for</mtext>
     <ci>𝐀𝐚</ci>
    </apply>
    <apply>
     <plus></plus>
     <apply>
      <times></times>
      <mtext>and</mtext>
      <apply>
       <csymbol cd="ambiguous">superscript</csymbol>
       <ci>q</ci>
       <cn type="integer">2</cn>
      </apply>
      <apply>
       <minus></minus>
       <cn type="integer">1</cn>
       <ci>F</ci>
      </apply>
     </apply>
     <apply>
      <times></times>
      <ci>q</ci>
      <ci>F</ci>
      <mtext>for</mtext>
      <ci>𝐚𝐚</ci>
     </apply>
    </apply>
   </list>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   p^{2}(1-F)+pF\text{ for }\mathbf{AA};\ 2pq(1-F)\text{ for }\mathbf{Aa};\text{ %
and }q^{2}(1-F)+qF\text{ for }\mathbf{aa}.
  </annotation>
 </semantics>
</math>

</p>

<p>The value for <strong>F</strong> is found by solving the equation for <strong>F</strong> using heterozygotes in the above inbred population. This becomes one minus the <a href="observation" title="wikilink">observed</a> number of heterozygotes in a population divided by its <a href="expected_value" title="wikilink">expected</a> number of heterozygotes at <a href="Hardy–Weinberg_principle" title="wikilink">Hardy–Weinberg equilibrium</a>:</p>

<p>

<math display="block" id="F-statistics:1">
 <semantics>
  <mrow>
   <mrow>
    <mi>F</mi>
    <mo>=</mo>
    <mrow>
     <mn>1</mn>
     <mo>-</mo>
     <mfrac>
      <mrow>
       <mo>O</mo>
       <mrow>
        <mo stretchy="false">(</mo>
        <mrow>
         <mi>f</mi>
         <mrow>
          <mo stretchy="false">(</mo>
          <mi>𝐀𝐚</mi>
          <mo stretchy="false">)</mo>
         </mrow>
        </mrow>
        <mo stretchy="false">)</mo>
       </mrow>
      </mrow>
      <mrow>
       <mo>E</mo>
       <mrow>
        <mo stretchy="false">(</mo>
        <mrow>
         <mi>f</mi>
         <mrow>
          <mo stretchy="false">(</mo>
          <mi>𝐀𝐚</mi>
          <mo stretchy="false">)</mo>
         </mrow>
        </mrow>
        <mo stretchy="false">)</mo>
       </mrow>
      </mrow>
     </mfrac>
    </mrow>
    <mo>=</mo>
    <mrow>
     <mn>1</mn>
     <mo>-</mo>
     <mfrac>
      <mrow>
       <mo>ObservedNumber</mo>
       <mrow>
        <mo stretchy="false">(</mo>
        <mi>𝐀𝐚</mi>
        <mo stretchy="false">)</mo>
       </mrow>
      </mrow>
      <mrow>
       <mi>n</mi>
       <mrow>
        <mo>E</mo>
        <mrow>
         <mo stretchy="false">(</mo>
         <mrow>
          <mi>f</mi>
          <mrow>
           <mo stretchy="false">(</mo>
           <mi>𝐀𝐚</mi>
           <mo stretchy="false">)</mo>
          </mrow>
         </mrow>
         <mo stretchy="false">)</mo>
        </mrow>
       </mrow>
      </mrow>
     </mfrac>
    </mrow>
   </mrow>
   <mo>,</mo>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <and></and>
    <apply>
     <eq></eq>
     <ci>F</ci>
     <apply>
      <minus></minus>
      <cn type="integer">1</cn>
      <apply>
       <divide></divide>
       <apply>
        <ci>normal-O</ci>
        <apply>
         <times></times>
         <ci>f</ci>
         <ci>𝐀𝐚</ci>
        </apply>
       </apply>
       <apply>
        <ci>normal-E</ci>
        <apply>
         <times></times>
         <ci>f</ci>
         <ci>𝐀𝐚</ci>
        </apply>
       </apply>
      </apply>
     </apply>
    </apply>
    <apply>
     <eq></eq>
     <share href="#.cmml">
     </share>
     <apply>
      <minus></minus>
      <cn type="integer">1</cn>
      <apply>
       <divide></divide>
       <apply>
        <ci>ObservedNumber</ci>
        <ci>𝐀𝐚</ci>
       </apply>
       <apply>
        <times></times>
        <ci>n</ci>
        <apply>
         <ci>normal-E</ci>
         <apply>
          <times></times>
          <ci>f</ci>
          <ci>𝐀𝐚</ci>
         </apply>
        </apply>
       </apply>
      </apply>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   F=1-\frac{\operatorname{O}(f(\mathbf{Aa}))}{\operatorname{E}(f(\mathbf{Aa}))}=%
1-\frac{\operatorname{ObservedNumber}(\mathbf{Aa})}{n\operatorname{E}(f(%
\mathbf{Aa}))},\!
  </annotation>
 </semantics>
</math>

</p>

<p>where the expected value at Hardy–Weinberg equilibrium is given by</p>

<p>

<math display="block" id="F-statistics:2">
 <semantics>
  <mrow>
   <mrow>
    <mrow>
     <mo>E</mo>
     <mrow>
      <mo stretchy="false">(</mo>
      <mrow>
       <mi>f</mi>
       <mrow>
        <mo stretchy="false">(</mo>
        <mi>𝐀𝐚</mi>
        <mo stretchy="false">)</mo>
       </mrow>
      </mrow>
      <mo stretchy="false">)</mo>
     </mrow>
    </mrow>
    <mo>=</mo>
    <mrow>
     <mn>2</mn>
     <mi>p</mi>
     <mi>q</mi>
    </mrow>
   </mrow>
   <mo>,</mo>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <eq></eq>
    <apply>
     <ci>normal-E</ci>
     <apply>
      <times></times>
      <ci>f</ci>
      <ci>𝐀𝐚</ci>
     </apply>
    </apply>
    <apply>
     <times></times>
     <cn type="integer">2</cn>
     <ci>p</ci>
     <ci>q</ci>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \operatorname{E}(f(\mathbf{Aa}))=2pq,\!
  </annotation>
 </semantics>
</math>

</p>

<p>where <em>p</em> and <em>q</em> are the <a href="allele_frequencies" title="wikilink">allele frequencies</a> of <strong>A</strong> and <strong>a</strong>, respectively. It is also the probability that at any <a href="gene_locus" title="wikilink">locus</a>, two alleles from a random individuum of the population are <a href="identity_by_descent" title="wikilink">identical by descent</a>.</p>

<p>For example, consider the data from <a href="E.B._Ford" title="wikilink">E.B. Ford</a> (1971) on a single population of the <a href="scarlet_tiger_moth" title="wikilink">scarlet tiger moth</a>:</p>
<table>
<tbody>
<tr class="odd">
<td style="text-align: left;">
<p>'''Table 1: '''</p></td>
</tr>
<tr class="even">
<td style="text-align: left;">
<p>Genotype</p></td>
</tr>
<tr class="odd">
<td style="text-align: left;">
<p>Number</p></td>
</tr>
</tbody>
</table>

<p>From this, the <a href="allele_frequencies" title="wikilink">allele frequencies</a> can be calculated, and the expectation of <em>ƒ</em>(AA) derived:</p>

<p>

<math display="block" id="F-statistics:3">
 <semantics>
  <mrow>
   <mi>p</mi>
   <mo>=</mo>
   <mfrac>
    <mrow>
     <mrow>
      <mrow>
       <mn>2</mn>
       <mo>×</mo>
       <mi>obs</mi>
      </mrow>
      <mrow>
       <mo stretchy="false">(</mo>
       <mrow>
        <mi>A</mi>
        <mi>A</mi>
       </mrow>
       <mo stretchy="false">)</mo>
      </mrow>
     </mrow>
     <mo>+</mo>
     <mrow>
      <mi>obs</mi>
      <mrow>
       <mo stretchy="false">(</mo>
       <mrow>
        <mi>A</mi>
        <mi>a</mi>
       </mrow>
       <mo stretchy="false">)</mo>
      </mrow>
     </mrow>
    </mrow>
    <mrow>
     <mn>2</mn>
     <mo>×</mo>
     <mrow>
      <mo stretchy="false">(</mo>
      <mrow>
       <mrow>
        <mi>obs</mi>
        <mrow>
         <mo stretchy="false">(</mo>
         <mrow>
          <mi>A</mi>
          <mi>A</mi>
         </mrow>
         <mo stretchy="false">)</mo>
        </mrow>
       </mrow>
       <mo>+</mo>
       <mrow>
        <mi>obs</mi>
        <mrow>
         <mo stretchy="false">(</mo>
         <mrow>
          <mi>A</mi>
          <mi>a</mi>
         </mrow>
         <mo stretchy="false">)</mo>
        </mrow>
       </mrow>
       <mo>+</mo>
       <mrow>
        <mi>obs</mi>
        <mrow>
         <mo stretchy="false">(</mo>
         <mrow>
          <mi>a</mi>
          <mi>a</mi>
         </mrow>
         <mo stretchy="false">)</mo>
        </mrow>
       </mrow>
      </mrow>
      <mo stretchy="false">)</mo>
     </mrow>
    </mrow>
   </mfrac>
   <mo>=</mo>
   <mn>0.954</mn>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <and></and>
    <apply>
     <eq></eq>
     <ci>p</ci>
     <apply>
      <divide></divide>
      <apply>
       <plus></plus>
       <apply>
        <times></times>
        <apply>
         <times></times>
         <cn type="integer">2</cn>
         <ci>obs</ci>
        </apply>
        <apply>
         <times></times>
         <ci>A</ci>
         <ci>A</ci>
        </apply>
       </apply>
       <apply>
        <times></times>
        <ci>obs</ci>
        <apply>
         <times></times>
         <ci>A</ci>
         <ci>a</ci>
        </apply>
       </apply>
      </apply>
      <apply>
       <times></times>
       <cn type="integer">2</cn>
       <apply>
        <plus></plus>
        <apply>
         <times></times>
         <ci>obs</ci>
         <apply>
          <times></times>
          <ci>A</ci>
          <ci>A</ci>
         </apply>
        </apply>
        <apply>
         <times></times>
         <ci>obs</ci>
         <apply>
          <times></times>
          <ci>A</ci>
          <ci>a</ci>
         </apply>
        </apply>
        <apply>
         <times></times>
         <ci>obs</ci>
         <apply>
          <times></times>
          <ci>a</ci>
          <ci>a</ci>
         </apply>
        </apply>
       </apply>
      </apply>
     </apply>
    </apply>
    <apply>
     <eq></eq>
     <share href="#.cmml">
     </share>
     <cn type="float">0.954</cn>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   p={2\times\mathrm{obs}(AA)+\mathrm{obs}(Aa)\over 2\times(\mathrm{obs}(AA)+%
\mathrm{obs}(Aa)+\mathrm{obs}(aa))}=0.954
  </annotation>
 </semantics>
</math>

</p>

<p>

<math display="block" id="F-statistics:4">
 <semantics>
  <mrow>
   <mi>q</mi>
   <mo>=</mo>
   <mrow>
    <mn>1</mn>
    <mo>-</mo>
    <mi>p</mi>
   </mrow>
   <mo>=</mo>
   <mpadded width="+1.7pt">
    <mn>0.046</mn>
   </mpadded>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <and></and>
    <apply>
     <eq></eq>
     <ci>q</ci>
     <apply>
      <minus></minus>
      <cn type="integer">1</cn>
      <ci>p</ci>
     </apply>
    </apply>
    <apply>
     <eq></eq>
     <share href="#.cmml">
     </share>
     <cn type="float">0.046</cn>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   q=1-p=0.046\,
  </annotation>
 </semantics>
</math>

</p>

<p>

<math display="block" id="F-statistics:5">
 <semantics>
  <mrow>
   <mi>F</mi>
   <mo>=</mo>
   <mrow>
    <mn>1</mn>
    <mo>-</mo>
    <mfrac>
     <mrow>
      <mi>obs</mi>
      <mrow>
       <mo stretchy="false">(</mo>
       <mrow>
        <mi>A</mi>
        <mi>a</mi>
       </mrow>
       <mo stretchy="false">)</mo>
      </mrow>
     </mrow>
     <mrow>
      <mi>n</mi>
      <mn>2</mn>
      <mi>p</mi>
      <mi>q</mi>
     </mrow>
    </mfrac>
   </mrow>
   <mo>=</mo>
   <mrow>
    <mn>1</mn>
    <mo>-</mo>
    <mfrac>
     <mn>138</mn>
     <mrow>
      <mrow>
       <mn>1612</mn>
       <mo>*</mo>
       <mn>2</mn>
      </mrow>
      <mrow>
       <mo stretchy="false">(</mo>
       <mn>0.954</mn>
       <mo stretchy="false">)</mo>
      </mrow>
      <mrow>
       <mo stretchy="false">(</mo>
       <mn>0.046</mn>
       <mo stretchy="false">)</mo>
      </mrow>
     </mrow>
    </mfrac>
   </mrow>
   <mo>=</mo>
   <mn>0.023</mn>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <and></and>
    <apply>
     <eq></eq>
     <ci>F</ci>
     <apply>
      <minus></minus>
      <cn type="integer">1</cn>
      <apply>
       <divide></divide>
       <apply>
        <times></times>
        <ci>obs</ci>
        <apply>
         <times></times>
         <ci>A</ci>
         <ci>a</ci>
        </apply>
       </apply>
       <apply>
        <times></times>
        <ci>n</ci>
        <cn type="integer">2</cn>
        <ci>p</ci>
        <ci>q</ci>
       </apply>
      </apply>
     </apply>
    </apply>
    <apply>
     <eq></eq>
     <share href="#.cmml">
     </share>
     <apply>
      <minus></minus>
      <cn type="integer">1</cn>
      <apply>
       <divide></divide>
       <cn type="integer">138</cn>
       <apply>
        <times></times>
        <apply>
         <times></times>
         <cn type="integer">1612</cn>
         <cn type="integer">2</cn>
        </apply>
        <cn type="float">0.954</cn>
        <cn type="float">0.046</cn>
       </apply>
      </apply>
     </apply>
    </apply>
    <apply>
     <eq></eq>
     <share href="#.cmml">
     </share>
     <cn type="float">0.023</cn>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   F=1-\frac{\mathrm{obs}(Aa)}{n2pq}=1-{138\over 1612*2(0.954)(0.046)}=0.023
  </annotation>
 </semantics>
</math>

</p>

<p>The different <em>F</em>-statistics look at different levels of population structure. <strong>F<sub>IT</sub></strong> is the inbreeding coefficient of an individual (<strong>I</strong>) relative to the total (<strong>T</strong>) population, as above; <strong>F<sub>IS</sub></strong> is the inbreeding coefficient of an individual (<strong>I</strong>) relative to the subpopulation (<strong>S</strong>), using the above for subpopulations and averaging them; and <strong>F<sub>ST</sub></strong> is the effect of subpopulations (<strong>S</strong>) compared to the total population (<strong>T</strong>), and is calculated by solving the equation:</p>

<p>

<math display="block" id="F-statistics:6">
 <semantics>
  <mrow>
   <mrow>
    <mrow>
     <mrow>
      <mo stretchy="false">(</mo>
      <mrow>
       <mn>1</mn>
       <mo>-</mo>
       <msub>
        <mi>F</mi>
        <mrow>
         <mi>I</mi>
         <mi>S</mi>
        </mrow>
       </msub>
      </mrow>
      <mo stretchy="false">)</mo>
     </mrow>
     <mrow>
      <mo stretchy="false">(</mo>
      <mrow>
       <mn>1</mn>
       <mo>-</mo>
       <msub>
        <mi>F</mi>
        <mrow>
         <mi>S</mi>
         <mi>T</mi>
        </mrow>
       </msub>
      </mrow>
      <mo stretchy="false">)</mo>
     </mrow>
    </mrow>
    <mo>=</mo>
    <mrow>
     <mn>1</mn>
     <mo>-</mo>
     <msub>
      <mi>F</mi>
      <mrow>
       <mi>I</mi>
       <mi>T</mi>
      </mrow>
     </msub>
    </mrow>
   </mrow>
   <mo>,</mo>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <eq></eq>
    <apply>
     <times></times>
     <apply>
      <minus></minus>
      <cn type="integer">1</cn>
      <apply>
       <csymbol cd="ambiguous">subscript</csymbol>
       <ci>F</ci>
       <apply>
        <times></times>
        <ci>I</ci>
        <ci>S</ci>
       </apply>
      </apply>
     </apply>
     <apply>
      <minus></minus>
      <cn type="integer">1</cn>
      <apply>
       <csymbol cd="ambiguous">subscript</csymbol>
       <ci>F</ci>
       <apply>
        <times></times>
        <ci>S</ci>
        <ci>T</ci>
       </apply>
      </apply>
     </apply>
    </apply>
    <apply>
     <minus></minus>
     <cn type="integer">1</cn>
     <apply>
      <csymbol cd="ambiguous">subscript</csymbol>
      <ci>F</ci>
      <apply>
       <times></times>
       <ci>I</ci>
       <ci>T</ci>
      </apply>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   (1-F_{IS})(1-F_{ST})=1-F_{IT},\,
  </annotation>
 </semantics>
</math>

</p>

<p>as shown in the next section.</p>
<h2 id="partition-due-to-population-structure">Partition due to population structure</h2>
<figure><b>(Figure)</b>
<figcaption><em>F</em><sub><em>IT</em></sub> can be partitioned into <em>F</em><sub><em>ST</em></sub> due to the <a href="Wahlund_effect" title="wikilink">Wahlund effect</a> and <em>F</em><sub><em>IS</em></sub> due to <a class="uri" href="inbreeding" title="wikilink">inbreeding</a>.</figcaption>
</figure>

<p>Consider a population that has a <a href="population_structure" title="wikilink">population structure</a> of two levels; one from the individual (I) to the subpopulation (S) and one from the subpopulation to the total (T). Then the total <em>F</em>, known here as <em>F</em><sub><em>IT</em></sub>, can be <a href="Partition_of_the_sum_of_squares" title="wikilink">partitioned</a> into <em>F</em><sub><em>IS</em></sub> (or <em>f</em>) and <em>F</em><sub><em>ST</em></sub> (or <em>θ</em>):</p>

<p>

<math display="block" id="F-statistics:7">
 <semantics>
  <mrow>
   <mrow>
    <mrow>
     <mn>1</mn>
     <mo>-</mo>
     <msub>
      <mi>F</mi>
      <mrow>
       <mi>I</mi>
       <mi>T</mi>
      </mrow>
     </msub>
    </mrow>
    <mo>=</mo>
    <mrow>
     <mrow>
      <mo stretchy="false">(</mo>
      <mrow>
       <mn>1</mn>
       <mo>-</mo>
       <msub>
        <mi>F</mi>
        <mrow>
         <mi>I</mi>
         <mi>S</mi>
        </mrow>
       </msub>
      </mrow>
      <mo rspace="4.2pt" stretchy="false">)</mo>
     </mrow>
     <mrow>
      <mo stretchy="false">(</mo>
      <mrow>
       <mn>1</mn>
       <mo>-</mo>
       <msub>
        <mi>F</mi>
        <mrow>
         <mi>S</mi>
         <mi>T</mi>
        </mrow>
       </msub>
      </mrow>
      <mo stretchy="false">)</mo>
     </mrow>
    </mrow>
   </mrow>
   <mo>.</mo>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <eq></eq>
    <apply>
     <minus></minus>
     <cn type="integer">1</cn>
     <apply>
      <csymbol cd="ambiguous">subscript</csymbol>
      <ci>F</ci>
      <apply>
       <times></times>
       <ci>I</ci>
       <ci>T</ci>
      </apply>
     </apply>
    </apply>
    <apply>
     <times></times>
     <apply>
      <minus></minus>
      <cn type="integer">1</cn>
      <apply>
       <csymbol cd="ambiguous">subscript</csymbol>
       <ci>F</ci>
       <apply>
        <times></times>
        <ci>I</ci>
        <ci>S</ci>
       </apply>
      </apply>
     </apply>
     <apply>
      <minus></minus>
      <cn type="integer">1</cn>
      <apply>
       <csymbol cd="ambiguous">subscript</csymbol>
       <ci>F</ci>
       <apply>
        <times></times>
        <ci>S</ci>
        <ci>T</ci>
       </apply>
      </apply>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   1-F_{IT}=(1-F_{IS})\,(1-F_{ST}).\!
  </annotation>
 </semantics>
</math>

</p>

<p>This may be further partitioned for population substructure, and it expands according to the rules of <a href="binomial_expansion" title="wikilink">binomial expansion</a>, so that for <em>I</em> partitions:</p>

<p>

<math display="block" id="F-statistics:8">
 <semantics>
  <mrow>
   <mrow>
    <mn>1</mn>
    <mo>-</mo>
    <mi>F</mi>
   </mrow>
   <mo>=</mo>
   <mrow>
    <munderover>
     <mo largeop="true" movablelimits="false" symmetric="true">∏</mo>
     <mrow>
      <mi>i</mi>
      <mo>=</mo>
      <mn>0</mn>
     </mrow>
     <mrow>
      <mi>i</mi>
      <mo>=</mo>
      <mi>I</mi>
     </mrow>
    </munderover>
    <mrow>
     <mo stretchy="false">(</mo>
     <mrow>
      <mn>1</mn>
      <mo>-</mo>
      <msub>
       <mi>F</mi>
       <mrow>
        <mi>i</mi>
        <mo>,</mo>
        <mrow>
         <mi>i</mi>
         <mo>+</mo>
         <mn>1</mn>
        </mrow>
       </mrow>
      </msub>
     </mrow>
     <mo rspace="0.8pt" stretchy="false">)</mo>
    </mrow>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <eq></eq>
    <apply>
     <minus></minus>
     <cn type="integer">1</cn>
     <ci>F</ci>
    </apply>
    <apply>
     <apply>
      <csymbol cd="ambiguous">superscript</csymbol>
      <apply>
       <csymbol cd="ambiguous">subscript</csymbol>
       <csymbol cd="latexml">product</csymbol>
       <apply>
        <eq></eq>
        <ci>i</ci>
        <cn type="integer">0</cn>
       </apply>
      </apply>
      <apply>
       <eq></eq>
       <ci>i</ci>
       <ci>I</ci>
      </apply>
     </apply>
     <apply>
      <minus></minus>
      <cn type="integer">1</cn>
      <apply>
       <csymbol cd="ambiguous">subscript</csymbol>
       <ci>F</ci>
       <list>
        <ci>i</ci>
        <apply>
         <plus></plus>
         <ci>i</ci>
         <cn type="integer">1</cn>
        </apply>
       </list>
      </apply>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   1-F=\prod_{i=0}^{i=I}(1-F_{i,i+1})\!
  </annotation>
 </semantics>
</math>

</p>
<h2 id="fixation-index">Fixation index</h2>

<p>A reformulation of the definition of <strong>F</strong> would be the ratio of the average number of differences between pairs of chromosomes sampled within diploid individuals with the average number obtained when sampling chromosomes randomly from the population (excluding the grouping per individual). One can modify this definition and consider a grouping per sub-population instead of per individual. Population geneticists have used that idea to measure the degree of structure in a population.</p>

<p>Unfortunately, there is a large number of definitions for <a href="Fixation_index" title="wikilink"><em>F</em><sub><em>ST</em></sub></a>, causing some confusion in the scientific literature. A common definition is the following:</p>

<p>

<math display="block" id="F-statistics:9">
 <semantics>
  <mrow>
   <msub>
    <mi>F</mi>
    <mrow>
     <mi>S</mi>
     <mi>T</mi>
    </mrow>
   </msub>
   <mo>=</mo>
   <mpadded width="-1.7pt">
    <mfrac>
     <mrow>
      <mo>var</mo>
      <mrow>
       <mo stretchy="false">(</mo>
       <mi>𝐩</mi>
       <mo stretchy="false">)</mo>
      </mrow>
     </mrow>
     <mrow>
      <mpadded width="+1.7pt">
       <mi>p</mi>
      </mpadded>
      <mrow>
       <mo stretchy="false">(</mo>
       <mrow>
        <mn>1</mn>
        <mo>-</mo>
        <mi>p</mi>
       </mrow>
       <mo stretchy="false">)</mo>
      </mrow>
     </mrow>
    </mfrac>
   </mpadded>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <eq></eq>
    <apply>
     <csymbol cd="ambiguous">subscript</csymbol>
     <ci>F</ci>
     <apply>
      <times></times>
      <ci>S</ci>
      <ci>T</ci>
     </apply>
    </apply>
    <apply>
     <divide></divide>
     <apply>
      <ci>var</ci>
      <ci>𝐩</ci>
     </apply>
     <apply>
      <times></times>
      <ci>p</ci>
      <apply>
       <minus></minus>
       <cn type="integer">1</cn>
       <ci>p</ci>
      </apply>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   F_{ST}=\frac{\operatorname{var}(\mathbf{p})}{p\,(1-p)}\!
  </annotation>
 </semantics>
</math>

</p>

<p>where the variance of <strong>p</strong> is computed across sub-populations and <em>p</em>(1−<em>p</em>) is the expected frequency of heterozygotes.</p>
<h2 id="fixation-index-in-human-populations">Fixation index in human populations</h2>

<p>It is well established that the genetic diversity among human populations is low,<a class="footnoteRef" href="#fn3" id="fnref3"><sup>3</sup></a> although the distribution of the genetic diversity was only roughly estimated. Early studies argued that 85–90% of the genetic variation is found within individuals residing in the same populations within continents (intra-continental populations) and only an additional 10–15% is found between populations of different continents (continental populations).<a class="footnoteRef" href="#fn4" id="fnref4"><sup>4</sup></a><a class="footnoteRef" href="#fn5" id="fnref5"><sup>5</sup></a><a class="footnoteRef" href="#fn6" id="fnref6"><sup>6</sup></a><a class="footnoteRef" href="#fn7" id="fnref7"><sup>7</sup></a><a class="footnoteRef" href="#fn8" id="fnref8"><sup>8</sup></a> Later studies based on hundreds of thousands single-nucleotide polymorphism (SNPs) suggested that the genetic diversity between continental populations is even smaller and accounts for 3 to 7%<a class="footnoteRef" href="#fn9" id="fnref9"><sup>9</sup></a><a class="footnoteRef" href="#fn10" id="fnref10"><sup>10</sup></a><a class="footnoteRef" href="#fn11" id="fnref11"><sup>11</sup></a><a class="footnoteRef" href="#fn12" id="fnref12"><sup>12</sup></a><a class="footnoteRef" href="#fn13" id="fnref13"><sup>13</sup></a><a class="footnoteRef" href="#fn14" id="fnref14"><sup>14</sup></a> Most of these studies have used the <a href="Fixation_index" title="wikilink"><em>F</em><sub><em>ST</em></sub></a> statistics <a class="footnoteRef" href="#fn15" id="fnref15"><sup>15</sup></a> or closely related statistics.<a class="footnoteRef" href="#fn16" id="fnref16"><sup>16</sup></a><a class="footnoteRef" href="#fn17" id="fnref17"><sup>17</sup></a></p>
<h2 id="see-also">See also</h2>
<ul>
<li><a href="Malecot's_method_of_coancestry" title="wikilink">Malecot's method of coancestry</a></li>
<li><a class="uri" href="Heterozygosity" title="wikilink">Heterozygosity</a></li>
<li><a href="Fixation_index" title="wikilink">Fixation index</a></li>
</ul>
<h2 id="references">References</h2>
<h2 id="external-links">External links</h2>
<ul>
<li><a href="http://www.library.auckland.ac.nz/subjects/bio/pdfs/733Pop-g-stats2.pdf">Shane's Simple Guide to F-Statistics</a></li>
<li><a href="http://darwin.eeb.uconn.edu/eeb348/lecture-notes/genetic-structure.pdf">Analyzing the genetic structure of populations</a></li>
<li><a href="http://darwin.eeb.uconn.edu/eeb348/lecture-notes/wahlund/wahlund.html">Wahlund effect, Wright's F-statistics</a></li>
<li><a href="http://www.uwyo.edu/dbmcd/popecol/Maylects/FST.html">Worked example of calculating F-statistics from genotypic data</a></li>
<li><a href="http://helix.mcmaster.ca/brent/node10.html">IAM based F-statistics</a></li>
<li><a href="http://eco-tools.njit.edu/webMathematica/EcoTools/Fstats-1-1/Introduction.html">F-statistics for Population Genetics Eco-Tool</a></li>
<li><a href="http://www.stats.ox.ac.uk/~mcvean/slides7.pdf">Population Structure (slides)</a></li>
</ul>

<p>"</p>

<p><a href="Category:Population_genetics" title="wikilink">Category:Population genetics</a></p>
<section class="footnotes">
<hr/>
<ol>
<li id="fn1"><a href="#fnref1">↩</a></li>
<li id="fn2"><a href="#fnref2">↩</a></li>
<li id="fn3"><a href="#fnref3">↩</a></li>
<li id="fn4"><a href="#fnref4">↩</a></li>
<li id="fn5"><a href="#fnref5">↩</a></li>
<li id="fn6"><a href="#fnref6">↩</a></li>
<li id="fn7"><a href="#fnref7">↩</a></li>
<li id="fn8"><a href="#fnref8">↩</a></li>
<li id="fn9"><a href="#fnref9">↩</a></li>
<li id="fn10"><a href="#fnref10">↩</a></li>
<li id="fn11"><a href="#fnref11">↩</a></li>
<li id="fn12"><a href="#fnref12">↩</a></li>
<li id="fn13"><a href="#fnref13">↩</a></li>
<li id="fn14"><a href="#fnref14">↩</a></li>
<li id="fn15"><a href="#fnref15">↩</a></li>
<li id="fn16"><a href="#fnref16">↩</a></li>
<li id="fn17"><a href="#fnref17">↩</a></li>
</ol>
</section>
</body>
</html>
