<html lang="en">
<head>
<meta charset="utf-8"/>
<title offset="1192">Fixation index</title>
<script src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_SVG.js" type="text/javascript">
</script>
</head>
<body>
<h1>Fixation index</h1>
<hr/>

<p>The <strong>fixation index</strong> (<strong>F<sub>ST</sub></strong>) is a measure of <a href="population_differentiation" title="wikilink">population differentiation</a> due to <a href="genetic_structure" title="wikilink">genetic structure</a>. It is frequently estimated from <a href="Polymorphism_(biology)" title="wikilink">genetic polymorphism</a> data, such as <a href="single-nucleotide_polymorphism" title="wikilink">single-nucleotide polymorphisms</a> (SNP) or <a href="Microsatellite_(genetics)" title="wikilink">microsatellites</a>. Developed as a special case of <a href="Sewall_Wright" title="wikilink">Wright's</a> <a class="uri" href="F-statistics" title="wikilink">F-statistics</a>, it is one of the most commonly used statistics in <a href="population_genetics" title="wikilink">population genetics</a>.</p>
<h2 id="definition">Definition</h2>

<p>Two of the most commonly used definitions for F<sub>ST</sub> at a given locus are based on the variance of <a href="Allele_frequency" title="wikilink">allele frequencies</a> between populations, and on the probability of <a href="Identity_by_descent" title="wikilink">Identity by descent</a>.</p>

<p>If 

<math display="inline" id="Fixation_index:0">
 <semantics>
  <mover accent="true">
   <mi>p</mi>
   <mo stretchy="false">¯</mo>
  </mover>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <ci>normal-¯</ci>
    <ci>p</ci>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \bar{p}
  </annotation>
 </semantics>
</math>

 is the average frequency of an allele in the total population, 

<math display="inline" id="Fixation_index:1">
 <semantics>
  <msubsup>
   <mi>σ</mi>
   <mi>S</mi>
   <mn>2</mn>
  </msubsup>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <csymbol cd="ambiguous">subscript</csymbol>
    <apply>
     <csymbol cd="ambiguous">superscript</csymbol>
     <ci>σ</ci>
     <cn type="integer">2</cn>
    </apply>
    <ci>S</ci>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \sigma^{2}_{S}
  </annotation>
 </semantics>
</math>

 is the variance in the frequency of the allele between different subpopulations, weighted by the sizes of the subpopulations, and 

<math display="inline" id="Fixation_index:2">
 <semantics>
  <msubsup>
   <mi>σ</mi>
   <mi>T</mi>
   <mn>2</mn>
  </msubsup>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <csymbol cd="ambiguous">subscript</csymbol>
    <apply>
     <csymbol cd="ambiguous">superscript</csymbol>
     <ci>σ</ci>
     <cn type="integer">2</cn>
    </apply>
    <ci>T</ci>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \sigma^{2}_{T}
  </annotation>
 </semantics>
</math>

 is the variance of the allelic state in the total population, F<sub>ST</sub> is defined as <a class="footnoteRef" href="#fn1" id="fnref1"><sup>1</sup></a></p>

<p>

<math display="block" id="Fixation_index:3">
 <semantics>
  <mrow>
   <msub>
    <mi>F</mi>
    <mrow>
     <mi>S</mi>
     <mi>T</mi>
    </mrow>
   </msub>
   <mo>=</mo>
   <mfrac>
    <msubsup>
     <mi>σ</mi>
     <mi>S</mi>
     <mn>2</mn>
    </msubsup>
    <msubsup>
     <mi>σ</mi>
     <mi>T</mi>
     <mn>2</mn>
    </msubsup>
   </mfrac>
   <mo>=</mo>
   <mfrac>
    <msubsup>
     <mi>σ</mi>
     <mi>S</mi>
     <mn>2</mn>
    </msubsup>
    <mrow>
     <mover accent="true">
      <mi>p</mi>
      <mo stretchy="false">¯</mo>
     </mover>
     <mrow>
      <mo stretchy="false">(</mo>
      <mrow>
       <mn>1</mn>
       <mo>-</mo>
       <mover accent="true">
        <mi>p</mi>
        <mo stretchy="false">¯</mo>
       </mover>
      </mrow>
      <mo stretchy="false">)</mo>
     </mrow>
    </mrow>
   </mfrac>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <and></and>
    <apply>
     <eq></eq>
     <apply>
      <csymbol cd="ambiguous">subscript</csymbol>
      <ci>F</ci>
      <apply>
       <times></times>
       <ci>S</ci>
       <ci>T</ci>
      </apply>
     </apply>
     <apply>
      <divide></divide>
      <apply>
       <csymbol cd="ambiguous">subscript</csymbol>
       <apply>
        <csymbol cd="ambiguous">superscript</csymbol>
        <ci>σ</ci>
        <cn type="integer">2</cn>
       </apply>
       <ci>S</ci>
      </apply>
      <apply>
       <csymbol cd="ambiguous">subscript</csymbol>
       <apply>
        <csymbol cd="ambiguous">superscript</csymbol>
        <ci>σ</ci>
        <cn type="integer">2</cn>
       </apply>
       <ci>T</ci>
      </apply>
     </apply>
    </apply>
    <apply>
     <eq></eq>
     <share href="#.cmml">
     </share>
     <apply>
      <divide></divide>
      <apply>
       <csymbol cd="ambiguous">subscript</csymbol>
       <apply>
        <csymbol cd="ambiguous">superscript</csymbol>
        <ci>σ</ci>
        <cn type="integer">2</cn>
       </apply>
       <ci>S</ci>
      </apply>
      <apply>
       <times></times>
       <apply>
        <ci>normal-¯</ci>
        <ci>p</ci>
       </apply>
       <apply>
        <minus></minus>
        <cn type="integer">1</cn>
        <apply>
         <ci>normal-¯</ci>
         <ci>p</ci>
        </apply>
       </apply>
      </apply>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   F_{ST}=\frac{\sigma^{2}_{S}}{\sigma^{2}_{T}}=\frac{\sigma^{2}_{S}}{\bar{p}(1-%
\bar{p})}
  </annotation>
 </semantics>
</math>

</p>

<p>This definition, which is due to Wright, illustrates that F<sub>ST</sub> measures the amount of genetic variance that can be explained by population structure. This can also be thought of as the fraction of total diversity that is not a consequence of the average diversity within subpopulations, where diversity is measured by the probability that two randomly selected alleles are different, namely 

<math display="inline" id="Fixation_index:4">
 <semantics>
  <mrow>
   <mn>2</mn>
   <mi>p</mi>
   <mrow>
    <mo stretchy="false">(</mo>
    <mrow>
     <mn>1</mn>
     <mo>-</mo>
     <mi>p</mi>
    </mrow>
    <mo stretchy="false">)</mo>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <times></times>
    <cn type="integer">2</cn>
    <ci>p</ci>
    <apply>
     <minus></minus>
     <cn type="integer">1</cn>
     <ci>p</ci>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   2p(1-p)
  </annotation>
 </semantics>
</math>

. If the allele frequency in the 

<math display="inline" id="Fixation_index:5">
 <semantics>
  <mi>i</mi>
  <annotation-xml encoding="MathML-Content">
   <ci>i</ci>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   i
  </annotation>
 </semantics>
</math>

th population is 

<math display="inline" id="Fixation_index:6">
 <semantics>
  <msub>
   <mi>p</mi>
   <mi>i</mi>
  </msub>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <csymbol cd="ambiguous">subscript</csymbol>
    <ci>p</ci>
    <ci>i</ci>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   p_{i}
  </annotation>
 </semantics>
</math>

 and the relative size of the 

<math display="inline" id="Fixation_index:7">
 <semantics>
  <mi>i</mi>
  <annotation-xml encoding="MathML-Content">
   <ci>i</ci>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   i
  </annotation>
 </semantics>
</math>

th population is 

<math display="inline" id="Fixation_index:8">
 <semantics>
  <msub>
   <mi>c</mi>
   <mi>i</mi>
  </msub>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <csymbol cd="ambiguous">subscript</csymbol>
    <ci>c</ci>
    <ci>i</ci>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   c_{i}
  </annotation>
 </semantics>
</math>

, then</p>

<p>

<math display="block" id="Fixation_index:9">
 <semantics>
  <mrow>
   <msub>
    <mi>F</mi>
    <mrow>
     <mi>S</mi>
     <mi>T</mi>
    </mrow>
   </msub>
   <mo>=</mo>
   <mfrac>
    <mrow>
     <mrow>
      <mover accent="true">
       <mi>p</mi>
       <mo stretchy="false">¯</mo>
      </mover>
      <mrow>
       <mo stretchy="false">(</mo>
       <mrow>
        <mn>1</mn>
        <mo>-</mo>
        <mover accent="true">
         <mi>p</mi>
         <mo stretchy="false">¯</mo>
        </mover>
       </mrow>
       <mo stretchy="false">)</mo>
      </mrow>
     </mrow>
     <mo>-</mo>
     <mrow>
      <mo largeop="true" symmetric="true">∑</mo>
      <mrow>
       <msub>
        <mi>c</mi>
        <mi>i</mi>
       </msub>
       <msub>
        <mi>p</mi>
        <mi>i</mi>
       </msub>
       <mrow>
        <mo stretchy="false">(</mo>
        <mrow>
         <mn>1</mn>
         <mo>-</mo>
         <msub>
          <mi>p</mi>
          <mi>i</mi>
         </msub>
        </mrow>
        <mo stretchy="false">)</mo>
       </mrow>
      </mrow>
     </mrow>
    </mrow>
    <mrow>
     <mover accent="true">
      <mi>p</mi>
      <mo stretchy="false">¯</mo>
     </mover>
     <mrow>
      <mo stretchy="false">(</mo>
      <mrow>
       <mn>1</mn>
       <mo>-</mo>
       <mover accent="true">
        <mi>p</mi>
        <mo stretchy="false">¯</mo>
       </mover>
      </mrow>
      <mo stretchy="false">)</mo>
     </mrow>
    </mrow>
   </mfrac>
   <mo>=</mo>
   <mfrac>
    <mrow>
     <mrow>
      <mover accent="true">
       <mi>p</mi>
       <mo stretchy="false">¯</mo>
      </mover>
      <mrow>
       <mo stretchy="false">(</mo>
       <mrow>
        <mn>1</mn>
        <mo>-</mo>
        <mover accent="true">
         <mi>p</mi>
         <mo stretchy="false">¯</mo>
        </mover>
       </mrow>
       <mo stretchy="false">)</mo>
      </mrow>
     </mrow>
     <mo>-</mo>
     <mover accent="true">
      <mrow>
       <mi>p</mi>
       <mrow>
        <mo stretchy="false">(</mo>
        <mrow>
         <mn>1</mn>
         <mo>-</mo>
         <mi>p</mi>
        </mrow>
        <mo stretchy="false">)</mo>
       </mrow>
      </mrow>
      <mo>¯</mo>
     </mover>
    </mrow>
    <mrow>
     <mover accent="true">
      <mi>p</mi>
      <mo stretchy="false">¯</mo>
     </mover>
     <mrow>
      <mo stretchy="false">(</mo>
      <mrow>
       <mn>1</mn>
       <mo>-</mo>
       <mover accent="true">
        <mi>p</mi>
        <mo stretchy="false">¯</mo>
       </mover>
      </mrow>
      <mo stretchy="false">)</mo>
     </mrow>
    </mrow>
   </mfrac>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <and></and>
    <apply>
     <eq></eq>
     <apply>
      <csymbol cd="ambiguous">subscript</csymbol>
      <ci>F</ci>
      <apply>
       <times></times>
       <ci>S</ci>
       <ci>T</ci>
      </apply>
     </apply>
     <apply>
      <divide></divide>
      <apply>
       <minus></minus>
       <apply>
        <times></times>
        <apply>
         <ci>normal-¯</ci>
         <ci>p</ci>
        </apply>
        <apply>
         <minus></minus>
         <cn type="integer">1</cn>
         <apply>
          <ci>normal-¯</ci>
          <ci>p</ci>
         </apply>
        </apply>
       </apply>
       <apply>
        <sum></sum>
        <apply>
         <times></times>
         <apply>
          <csymbol cd="ambiguous">subscript</csymbol>
          <ci>c</ci>
          <ci>i</ci>
         </apply>
         <apply>
          <csymbol cd="ambiguous">subscript</csymbol>
          <ci>p</ci>
          <ci>i</ci>
         </apply>
         <apply>
          <minus></minus>
          <cn type="integer">1</cn>
          <apply>
           <csymbol cd="ambiguous">subscript</csymbol>
           <ci>p</ci>
           <ci>i</ci>
          </apply>
         </apply>
        </apply>
       </apply>
      </apply>
      <apply>
       <times></times>
       <apply>
        <ci>normal-¯</ci>
        <ci>p</ci>
       </apply>
       <apply>
        <minus></minus>
        <cn type="integer">1</cn>
        <apply>
         <ci>normal-¯</ci>
         <ci>p</ci>
        </apply>
       </apply>
      </apply>
     </apply>
    </apply>
    <apply>
     <eq></eq>
     <share href="#.cmml">
     </share>
     <apply>
      <divide></divide>
      <apply>
       <minus></minus>
       <apply>
        <times></times>
        <apply>
         <ci>normal-¯</ci>
         <ci>p</ci>
        </apply>
        <apply>
         <minus></minus>
         <cn type="integer">1</cn>
         <apply>
          <ci>normal-¯</ci>
          <ci>p</ci>
         </apply>
        </apply>
       </apply>
       <apply>
        <ci>normal-¯</ci>
        <apply>
         <times></times>
         <ci>p</ci>
         <apply>
          <minus></minus>
          <cn type="integer">1</cn>
          <ci>p</ci>
         </apply>
        </apply>
       </apply>
      </apply>
      <apply>
       <times></times>
       <apply>
        <ci>normal-¯</ci>
        <ci>p</ci>
       </apply>
       <apply>
        <minus></minus>
        <cn type="integer">1</cn>
        <apply>
         <ci>normal-¯</ci>
         <ci>p</ci>
        </apply>
       </apply>
      </apply>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   F_{ST}=\frac{\bar{p}(1-\bar{p})-\sum c_{i}p_{i}(1-p_{i})}{\bar{p}(1-\bar{p})}=%
\frac{\bar{p}(1-\bar{p})-\overline{p(1-p)}}{\bar{p}(1-\bar{p})}
  </annotation>
 </semantics>
</math>

</p>

<p>Alternatively,<a class="footnoteRef" href="#fn2" id="fnref2"><sup>2</sup></a></p>

<p>

<math display="block" id="Fixation_index:10">
 <semantics>
  <mrow>
   <msub>
    <mi>F</mi>
    <mrow>
     <mi>S</mi>
     <mi>T</mi>
    </mrow>
   </msub>
   <mo>=</mo>
   <mfrac>
    <mrow>
     <msub>
      <mi>f</mi>
      <mn>0</mn>
     </msub>
     <mo>-</mo>
     <mover accent="true">
      <mi>f</mi>
      <mo stretchy="false">¯</mo>
     </mover>
    </mrow>
    <mrow>
     <mn>1</mn>
     <mo>-</mo>
     <mover accent="true">
      <mi>f</mi>
      <mo stretchy="false">¯</mo>
     </mover>
    </mrow>
   </mfrac>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <eq></eq>
    <apply>
     <csymbol cd="ambiguous">subscript</csymbol>
     <ci>F</ci>
     <apply>
      <times></times>
      <ci>S</ci>
      <ci>T</ci>
     </apply>
    </apply>
    <apply>
     <divide></divide>
     <apply>
      <minus></minus>
      <apply>
       <csymbol cd="ambiguous">subscript</csymbol>
       <ci>f</ci>
       <cn type="integer">0</cn>
      </apply>
      <apply>
       <ci>normal-¯</ci>
       <ci>f</ci>
      </apply>
     </apply>
     <apply>
      <minus></minus>
      <cn type="integer">1</cn>
      <apply>
       <ci>normal-¯</ci>
       <ci>f</ci>
      </apply>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   F_{ST}=\frac{f_{0}-\bar{f}}{1-\bar{f}}
  </annotation>
 </semantics>
</math>

</p>

<p>where 

<math display="inline" id="Fixation_index:11">
 <semantics>
  <msub>
   <mi>f</mi>
   <mn>0</mn>
  </msub>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <csymbol cd="ambiguous">subscript</csymbol>
    <ci>f</ci>
    <cn type="integer">0</cn>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   f_{0}
  </annotation>
 </semantics>
</math>

 is the probability of identity by descent of two individuals given that the two individuals are in the same subpopulation, and 

<math display="inline" id="Fixation_index:12">
 <semantics>
  <mover accent="true">
   <mi>f</mi>
   <mo stretchy="false">¯</mo>
  </mover>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <ci>normal-¯</ci>
    <ci>f</ci>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \bar{f}
  </annotation>
 </semantics>
</math>

 is the probability that two individuals from the total population are identical by descent. Using this definition, F<sub>ST</sub> can be interpreted as measuring how much closer two individuals from the same subpopulation are, compared to the total population. If the <a href="mutation_rate" title="wikilink">mutation rate</a> is small, this interpretation can be made more explicit by linking the probability of identity by descent to <a href="coalescent_theory" title="wikilink">coalescent times</a>: Let T<sub>0</sub> and T denote the average time to coalescence for individuals from the same subpopulation and the total population, respectively. Then,</p>

<p>

<math display="block" id="Fixation_index:13">
 <semantics>
  <mrow>
   <msub>
    <mi>F</mi>
    <mrow>
     <mi>S</mi>
     <mi>T</mi>
    </mrow>
   </msub>
   <mo>≈</mo>
   <mrow>
    <mn>1</mn>
    <mo>-</mo>
    <mfrac>
     <msub>
      <mi>T</mi>
      <mn>0</mn>
     </msub>
     <mi>T</mi>
    </mfrac>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <approx></approx>
    <apply>
     <csymbol cd="ambiguous">subscript</csymbol>
     <ci>F</ci>
     <apply>
      <times></times>
      <ci>S</ci>
      <ci>T</ci>
     </apply>
    </apply>
    <apply>
     <minus></minus>
     <cn type="integer">1</cn>
     <apply>
      <divide></divide>
      <apply>
       <csymbol cd="ambiguous">subscript</csymbol>
       <ci>T</ci>
       <cn type="integer">0</cn>
      </apply>
      <ci>T</ci>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   F_{ST}\approx 1-\frac{T_{0}}{T}
  </annotation>
 </semantics>
</math>

</p>

<p>This formulation has the advantage that the expected time to coalescence can easily be estimated from genetic data, which led to the development of various estimators for F<sub>ST</sub>.</p>
<h2 id="estimation">Estimation</h2>

<p>In practice, none of the quantities used for the definitions can be easily measured. As a consequence, various estimators have been proposed. A particularly simple estimator applicable to DNA sequence data is:<a class="footnoteRef" href="#fn3" id="fnref3"><sup>3</sup></a></p>

<p>

<math display="block" id="Fixation_index:14">
 <semantics>
  <mrow>
   <msub>
    <mi>F</mi>
    <mrow>
     <mi>S</mi>
     <mi>T</mi>
    </mrow>
   </msub>
   <mo>=</mo>
   <mfrac>
    <mrow>
     <msub>
      <mi>π</mi>
      <mtext>Between</mtext>
     </msub>
     <mo>-</mo>
     <msub>
      <mi>π</mi>
      <mtext>Within</mtext>
     </msub>
    </mrow>
    <msub>
     <mi>π</mi>
     <mtext>Between</mtext>
    </msub>
   </mfrac>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <eq></eq>
    <apply>
     <csymbol cd="ambiguous">subscript</csymbol>
     <ci>F</ci>
     <apply>
      <times></times>
      <ci>S</ci>
      <ci>T</ci>
     </apply>
    </apply>
    <apply>
     <divide></divide>
     <apply>
      <minus></minus>
      <apply>
       <csymbol cd="ambiguous">subscript</csymbol>
       <ci>π</ci>
       <mtext>Between</mtext>
      </apply>
      <apply>
       <csymbol cd="ambiguous">subscript</csymbol>
       <ci>π</ci>
       <mtext>Within</mtext>
      </apply>
     </apply>
     <apply>
      <csymbol cd="ambiguous">subscript</csymbol>
      <ci>π</ci>
      <mtext>Between</mtext>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   F_{ST}=\frac{\pi_{\text{Between}}-\pi_{\text{Within}}}{\pi_{\text{Between}}}
  </annotation>
 </semantics>
</math>

</p>

<p>where 

<math display="inline" id="Fixation_index:15">
 <semantics>
  <msub>
   <mi>π</mi>
   <mtext>Between</mtext>
  </msub>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <csymbol cd="ambiguous">subscript</csymbol>
    <ci>π</ci>
    <mtext>Between</mtext>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \pi_{\text{Between}}
  </annotation>
 </semantics>
</math>

 and 

<math display="inline" id="Fixation_index:16">
 <semantics>
  <msub>
   <mi>π</mi>
   <mtext>Within</mtext>
  </msub>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <csymbol cd="ambiguous">subscript</csymbol>
    <ci>π</ci>
    <mtext>Within</mtext>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \pi_{\text{Within}}
  </annotation>
 </semantics>
</math>

 represent the <a href="Nucleotide_diversity" title="wikilink">average number of pairwise differences</a> between two individuals sampled from different sub-populations (

<math display="inline" id="Fixation_index:17">
 <semantics>
  <msub>
   <mi>π</mi>
   <mtext>Between</mtext>
  </msub>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <csymbol cd="ambiguous">subscript</csymbol>
    <ci>π</ci>
    <mtext>Between</mtext>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \pi_{\text{Between}}
  </annotation>
 </semantics>
</math>

) or from the same sub-population (

<math display="inline" id="Fixation_index:18">
 <semantics>
  <msub>
   <mi>π</mi>
   <mtext>Within</mtext>
  </msub>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <csymbol cd="ambiguous">subscript</csymbol>
    <ci>π</ci>
    <mtext>Within</mtext>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \pi_{\text{Within}}
  </annotation>
 </semantics>
</math>

). The average pairwise difference within a population can be calculated as the sum of the pairwise differences divided by the number of pairs. However, this estimator is biased when sample sizes are small or if they vary between populations. Therefore, more elaborate methods are used to compute F<sub>ST</sub> in practice. Two of the most widely used procedures are the estimator by Weir &amp; Cockerham (1984),<a class="footnoteRef" href="#fn4" id="fnref4"><sup>4</sup></a> or performing an <a href="Analysis_of_molecular_variance" title="wikilink">Analysis of molecular variance</a>. A list of implementations is available at the end of this article.</p>
<h2 id="interpretation">Interpretation</h2>

<p>This comparison of genetic variability within and between populations is frequently used in applied <a href="population_genetics" title="wikilink">population genetics</a>. The values range from 0 to 1. A zero value implies complete <a href="Panmixia" title="wikilink">panmixis</a>; that is, that the two populations are interbreeding freely. A value of one implies that all genetic variation is explained by the population structure, and that the two populations do not share any genetic diversity.</p>

<p>For idealized models such as Wright's finite island model, F<sub>ST</sub> can be used to estimate migration rates. Under that model, the migration rate is</p>

<p>

<math display="block" id="Fixation_index:19">
 <semantics>
  <mrow>
   <mover accent="true">
    <mi>M</mi>
    <mo stretchy="false">^</mo>
   </mover>
   <mo>≈</mo>
   <mrow>
    <mfrac>
     <mn>1</mn>
     <mn>2</mn>
    </mfrac>
    <mrow>
     <mo>(</mo>
     <mrow>
      <mfrac>
       <mn>1</mn>
       <msub>
        <mi>F</mi>
        <mrow>
         <mi>S</mi>
         <mi>T</mi>
        </mrow>
       </msub>
      </mfrac>
      <mo>-</mo>
      <mn>1</mn>
     </mrow>
     <mo>)</mo>
    </mrow>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <approx></approx>
    <apply>
     <ci>normal-^</ci>
     <ci>M</ci>
    </apply>
    <apply>
     <times></times>
     <apply>
      <divide></divide>
      <cn type="integer">1</cn>
      <cn type="integer">2</cn>
     </apply>
     <apply>
      <minus></minus>
      <apply>
       <divide></divide>
       <cn type="integer">1</cn>
       <apply>
        <csymbol cd="ambiguous">subscript</csymbol>
        <ci>F</ci>
        <apply>
         <times></times>
         <ci>S</ci>
         <ci>T</ci>
        </apply>
       </apply>
      </apply>
      <cn type="integer">1</cn>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \hat{M}\approx\frac{1}{2}\left(\frac{1}{F_{ST}}-1\right)
  </annotation>
 </semantics>
</math>

.</p>

<p>The interpretation of F<sub>ST</sub> can be difficult when the data analyzed are highly polymorphic. In this case, the probability of identity by descent is very low and F<sub>ST</sub> can have an arbitrarily low upper bound, which might lead to misinterpretation of the data. Also, strictly speaking F<sub>ST</sub> is not a genetic distance, as it does not satisfy the <a href="triangle_inequality" title="wikilink">triangle inequality</a>. As a consequence new tools for measuring genetic differentiation continue being developed.</p>
<h2 id="fst-in-humans">F<sub>ST</sub> in humans</h2>

<p>The <a href="International_HapMap_Project" title="wikilink">International HapMap Project</a> estimated F<sub>ST</sub> for three human populations using SNP data. Across the <a href="autosome" title="wikilink">autosomes</a>, F<sub>ST</sub> was estimated to be 0.12. The significance of this F<sub>ST</sub> value in humans is contentious. As an F<sub>ST</sub> of zero indicates no divergence between populations, whereas an F<sub>ST</sub> of one indicates complete isolation of populations, <a class="uri" href="Anthropologists" title="wikilink">Anthropologists</a> often cite <a href="Richard_Lewontin" title="wikilink">Lewontin's</a> 1972 work which came to a similar value and interpreted this number as meaning there was little biological differences between human races.<a class="footnoteRef" href="#fn5" id="fnref5"><sup>5</sup></a> On the other hand, while an F<sub>ST</sub> value of 0.12 is lower than that found between populations of many other species, <a href="Henry_Harpending" title="wikilink">Henry Harpending</a> argued that this value implies on a world scale a "kinship between two individuals of the same human population is equivalent to kinship between grandparent and grandchild or between half siblings".<a class="footnoteRef" href="#fn6" id="fnref6"><sup>6</sup></a></p>
<h3 id="autosomal-genetic-distances-based-on-snps">Autosomal genetic distances based on SNPs</h3>
<table>
<tbody>
<tr class="odd">
<td style="text-align: left;">
<p>Intercontinental autosomal genetic distances based on SNPs<ref name="nelis">{{Cite journal</ref></p></td>
<td style="text-align: left;">
<p>doi = 10.1371/journal.pone.0005472</p></td>
<td style="text-align: left;">
<p>volume = 4</p></td>
<td style="text-align: left;">
<p>issue = 5</p></td>
<td style="text-align: left;">
<p>pmc = 2675054</p></td>
<td style="text-align: left;">
<p>pages = e5472</p></td>
<td style="text-align: left;">
<p>last = Nelis</p></td>
<td style="text-align: left;">
<p>pmid = 19424496</p></td>
<td style="text-align: left;">
<p>first = Mari</p></td>
<td style="text-align: left;">
<p>title = Genetic Structure of Europeans: A View from the North–East</p></td>
<td style="text-align: left;">
<p>journal = PLoS ONE</p></td>
<td style="text-align: left;">
<p>date = 2009-05-08</p></td>
<td style="text-align: left;">
<p>bibcode=2009PLoSO...4.5472N</p></td>
<td style="text-align: left;">
<p>editor1-last = Fleischer</p></td>
<td style="text-align: left;">
<p>editor1-first = Robert C.</p></td>
<td style="text-align: left;">
<p>display-authors=etal}}, <a href="http://www.plosone.org/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0005472.s003">see table</a></p></td>
</tr>
<tr class="even">
<td style="text-align: left;"></td>
<td style="text-align: left;">
<p>Europe (CEU)</p></td>
<td style="text-align: left;">
<p>Sub-Saharan Africa (Yoruba)</p></td>
<td style="text-align: left;">
<p>East-Asia (Japanese)</p></td>
</tr>
<tr class="odd">
<td style="text-align: left;">
<p>Sub-Saharan Africa (Yoruba)</p></td>
<td style="text-align: left;">
<p>0.153</p></td>
<td style="text-align: left;"></td>
<td style="text-align: left;"></td>
</tr>
<tr class="even">
<td style="text-align: left;">
<p>East-Asia (Japanese)</p></td>
<td style="text-align: left;">
<p>0.111</p></td>
<td style="text-align: left;">
<p>0.190</p></td>
<td style="text-align: left;"></td>
</tr>
<tr class="odd">
<td style="text-align: left;">
<p>East-Asia (Chinese)</p></td>
<td style="text-align: left;">
<p>0.110</p></td>
<td style="text-align: left;">
<p>0.192</p></td>
<td style="text-align: left;">
<p>0.007</p></td>
</tr>
<tr class="even">
</tr>
</tbody>
</table>
<table>
<tbody>
<tr class="odd">
<td style="text-align: left;">
<p>Intra-European/mediterranean autosomal genetic distances based on SNPs<a class="footnoteRef" href="#fn7" id="fnref7"><sup>7</sup></a><ref>{{Cite journal</ref></p></td>
<td style="text-align: left;">
<p>doi = 10.2119/molmed.2009.00094</p></td>
<td style="text-align: left;">
<p>issn = 1076-1551</p></td>
<td style="text-align: left;">
<p>volume = 15</p></td>
<td style="text-align: left;">
<p>issue = 11-12</p></td>
<td style="text-align: left;">
<p>pages = 371–383</p></td>
<td style="text-align: left;">
<p>last = Tian</p></td>
<td style="text-align: left;">
<p>journal = Molecular Medicine</p></td>
<td style="text-align: left;">
<p>first = Chao</p></td>
<td style="text-align: left;">
<p>title = European Population Genetic Substructure: Further Definition of Ancestry Informative Markers for Distinguishing among Diverse European Ethnic Groups</p></td>
<td style="text-align: left;">
<p>date = November 2009</p></td>
<td style="text-align: left;">
<p>pmid=19707526</p></td>
<td style="text-align: left;">
<p>pmc=2730349</p></td>
<td style="text-align: left;">
<p>display-authors=etal}}, <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2730349/table/t1-09_94_tian/">see table</a></p></td>
</tr>
<tr class="even">
<td style="text-align: left;"></td>
<td style="text-align: left;">
<p>Italians</p></td>
<td style="text-align: left;">
<p>Palestinians</p></td>
<td style="text-align: left;">
<p>Swedish</p></td>
<td style="text-align: left;">
<p>Finns</p></td>
<td style="text-align: left;">
<p>Spanish</p></td>
<td style="text-align: left;">
<p>Germans</p></td>
<td style="text-align: left;">
<p>Russians</p></td>
<td style="text-align: left;">
<p>French</p></td>
<td style="text-align: left;">
<p>Greeks</p></td>
</tr>
<tr class="odd">
<td style="text-align: left;">
<p>Palestinians</p></td>
<td style="text-align: left;">
<p>0.0064</p></td>
<td style="text-align: left;"></td>
<td style="text-align: left;"></td>
<td style="text-align: left;"></td>
<td style="text-align: left;"></td>
<td style="text-align: left;"></td>
<td style="text-align: left;"></td>
<td style="text-align: left;"></td>
<td style="text-align: left;"></td>
</tr>
<tr class="even">
<td style="text-align: left;">
<p>Swedish</p></td>
<td style="text-align: left;">
<p>0.0064-0.0090</p></td>
<td style="text-align: left;">
<p>0.0191</p></td>
<td style="text-align: left;"></td>
<td style="text-align: left;"></td>
<td style="text-align: left;"></td>
<td style="text-align: left;"></td>
<td style="text-align: left;"></td>
<td style="text-align: left;"></td>
<td style="text-align: left;"></td>
</tr>
<tr class="odd">
<td style="text-align: left;">
<p>Finns</p></td>
<td style="text-align: left;">
<p>0.0130-0.0230</p></td>
<td style="text-align: left;"></td>
<td style="text-align: left;">
<p>0.0050-0.0110</p></td>
<td style="text-align: left;"></td>
<td style="text-align: left;"></td>
<td style="text-align: left;"></td>
<td style="text-align: left;"></td>
<td style="text-align: left;"></td>
<td style="text-align: left;"></td>
</tr>
<tr class="even">
<td style="text-align: left;">
<p>Spanish</p></td>
<td style="text-align: left;">
<p>0.0010-0.0050</p></td>
<td style="text-align: left;">
<p>0.0101</p></td>
<td style="text-align: left;">
<p>0.0040-0055</p></td>
<td style="text-align: left;">
<p>0.0110-0.0170</p></td>
<td style="text-align: left;"></td>
<td style="text-align: left;"></td>
<td style="text-align: left;"></td>
<td style="text-align: left;"></td>
<td style="text-align: left;"></td>
</tr>
<tr class="odd">
<td style="text-align: left;">
<p>Germans</p></td>
<td style="text-align: left;">
<p>0.0029-0.0080</p></td>
<td style="text-align: left;">
<p>0.0136</p></td>
<td style="text-align: left;">
<p>0.0007-0.0010</p></td>
<td style="text-align: left;">
<p>0.0060-0.0130</p></td>
<td style="text-align: left;">
<p>0.0015-0.0030</p></td>
<td style="text-align: left;"></td>
<td style="text-align: left;"></td>
<td style="text-align: left;"></td>
<td style="text-align: left;"></td>
</tr>
<tr class="even">
<td style="text-align: left;">
<p>Russians</p></td>
<td style="text-align: left;">
<p>0.0088-0.0120</p></td>
<td style="text-align: left;">
<p>0.0202</p></td>
<td style="text-align: left;">
<p>0.0030-0.0036</p></td>
<td style="text-align: left;">
<p>0.0060-0.0120</p></td>
<td style="text-align: left;">
<p>0.0070-0.0079</p></td>
<td style="text-align: left;">
<p>0.0030-0.0037</p></td>
<td style="text-align: left;"></td>
<td style="text-align: left;"></td>
<td style="text-align: left;"></td>
</tr>
<tr class="odd">
<td style="text-align: left;">
<p>French</p></td>
<td style="text-align: left;">
<p>0.0030-0.0050</p></td>
<td style="text-align: left;"></td>
<td style="text-align: left;">
<p>0.0020</p></td>
<td style="text-align: left;">
<p>0.0080-0.0150</p></td>
<td style="text-align: left;">
<p>0.0010</p></td>
<td style="text-align: left;">
<p>0.0010</p></td>
<td style="text-align: left;">
<p>0.0050</p></td>
<td style="text-align: left;"></td>
<td style="text-align: left;"></td>
</tr>
<tr class="even">
<td style="text-align: left;">
<p>Greeks</p></td>
<td style="text-align: left;">
<p>0.0000</p></td>
<td style="text-align: left;">
<p>0.0057</p></td>
<td style="text-align: left;">
<p>0.0084</p></td>
<td style="text-align: left;"></td>
<td style="text-align: left;">
<p>0.0035</p></td>
<td style="text-align: left;">
<p>0.0039</p></td>
<td style="text-align: left;">
<p>0.0108</p></td>
<td style="text-align: left;"></td>
<td style="text-align: left;"></td>
</tr>
<tr class="odd">
</tr>
</tbody>
</table>
<h2 id="programs-for-calculating-fst">Programs for calculating F<sub>ST</sub></h2>
<ul>
<li><a class="uri" href="Arlequin" title="wikilink">Arlequin</a></li>
<li><a href="FSTAT_(software)" title="wikilink">Fstat</a></li>
<li><a href="http://people.bu.edu/ngcrawfo/smogd/">SMOGD</a><a class="footnoteRef" href="#fn8" id="fnref8"><sup>8</sup></a></li>
<li><a href="http://cran.r-project.org/web/packages/diveRsity/index.html">diveRsity</a></li>
<li><a href="http://i122server.vu-wien.ac.at/MSA/info.html/MSA_info.html">Microsatellite Analyzer (MSA)</a></li>
<li><a href="http://vcftools.sourceforge.net/index.html">VCFtools</a></li>
</ul>
<h2 id="modules-for-calculating-fst">Modules for calculating F<sub>ST</sub></h2>
<ul>
<li><a href="#External_links" title="wikilink">BioPerl</a></li>
<li><a href="http://cran.r-project.org/web/packages/hierfstat/index.html">hierfstat (R package)</a></li>
</ul>
<h2 id="references">References</h2>
<h3 id="further-reading">Further reading</h3>
<ul>
<li>Evolution and the Genetics of Populations Volume 2: the Theory of Gene Frequencies, pg 294–295, S. Wright, Univ. of Chicago Press, Chicago, 1969</li>
<li>A haplotype map of the human genome, The International HapMap Consortium, Nature 2005</li>
</ul>
<h2 id="external-links">External links</h2>
<ul>
<li>[<a class="uri" href="http://www.bioperl.org/wiki/HOWTO:PopGen#Population_Statistics_using_Bio">http://www.bioperl.org/wiki/HOWTO:PopGen#Population_Statistics_using_Bio</a>::PopGen::PopStats BioPerl - Bio::PopGen::PopStats]</li>
</ul>

<p>"</p>

<p><a href="Category:Population_genetics" title="wikilink">Category:Population genetics</a></p>
<section class="footnotes">
<hr/>
<ol>
<li id="fn1"><a href="#fnref1">↩</a></li>
<li id="fn2"><a href="#fnref2">↩</a></li>
<li id="fn3"><a href="#fnref3">↩</a></li>
<li id="fn4"><a href="#fnref4">↩</a></li>
<li id="fn5"><a href="#fnref5">↩</a></li>
<li id="fn6"><a href="#fnref6">↩</a></li>
<li id="fn7"></li>
<li id="fn8"><a href="#fnref8">↩</a></li>
</ol>
</section>
</body>
</html>
