<html lang="en">
<head>
<meta charset="utf-8"/>
<title offset="1104">Flux balance analysis</title>
<script src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_SVG.js" type="text/javascript">
</script>
</head>
<body>
<h1>Flux balance analysis</h1>
<hr/>

<p><strong>Flux balance analysis</strong> (<strong>FBA</strong>) is a mathematical method for simulating metabolism in genome-scale reconstructions of <a href="metabolic_network" title="wikilink">metabolic networks</a>. In comparison to traditional methods of modeling, FBA is less intensive in terms of the input data required for constructing the model. Simulations performed using FBA are computationally inexpensive and can calculate steady-state metabolic fluxes for large models (over 2000 reactions) in a few seconds on modern personal computers.</p>
<figure><b>(Figure)</b>
<figcaption>The results of FBA on a prepared metabolic network of the top six reactions of <a class="uri" href="glycolysis" title="wikilink">glycolysis</a>. The predicted flux through each reaction is proportional to the width of the line. Objective function in red, constraints on alpha-D-glucose and beta-D-glucose import represented as red bars.</figcaption>
</figure>

<p>FBA finds applications in <a href="bioprocess_engineering" title="wikilink">bioprocess engineering</a> to systematically identify modifications to the metabolic networks of microbes used in fermentation processes that improve product yields of industrially important chemicals such as ethanol and succinic acid.<a class="footnoteRef" href="#fn1" id="fnref1"><sup>1</sup></a> It has also been used for the identification of putative drug targets in cancer <a class="footnoteRef" href="#fn2" id="fnref2"><sup>2</sup></a> and pathogens,<a class="footnoteRef" href="#fn3" id="fnref3"><sup>3</sup></a> rational design of culture media,<a class="footnoteRef" href="#fn4" id="fnref4"><sup>4</sup></a> and more recently <a href="Host-pathogen_interactions" title="wikilink">host–pathogen interactions</a>.<a class="footnoteRef" href="#fn5" id="fnref5"><sup>5</sup></a> The results of FBA can be visualized using flux maps similar to the image on the right, which illustrates the steady-state fluxes carried by reactions in <a class="uri" href="glycolysis" title="wikilink">glycolysis</a>. The thickness of the arrows is proportional to the flux through the reaction.</p>

<p>FBA formalizes the system of equations describing the concentration changes in a metabolic network as the dot product of a matrix of the stoichiometric coefficients (the stoichiometric matrix <strong>S</strong>) and the vector <strong>v</strong> of the unsolved fluxes. The right-hand side of the dot product is a vector of zeros representing the system at <a href="steady_state" title="wikilink">steady state</a>. <a href="Linear_programming" title="wikilink">Linear programming</a> is then used to calculate a solution of fluxes corresponding to the steady state.</p>
<h2 id="history">History</h2>

<p>Some of the earliest work in FBA dates back to the early 1980s. Papoutsakis<a class="footnoteRef" href="#fn6" id="fnref6"><sup>6</sup></a> demonstrated that it was possible to construct flux balance equations using a metabolic map. It was Watson,<a class="footnoteRef" href="#fn7" id="fnref7"><sup>7</sup></a> however, who first introduced the idea of using linear programming and an objective function to solve for the fluxes in a pathway. The first significant study was subsequently published by Fell and Small,<a class="footnoteRef" href="#fn8" id="fnref8"><sup>8</sup></a> who used flux balance analysis together with more elaborate objective functions to study the constraints in fat synthesis.</p>
<h2 id="simulations">Simulations</h2>
<figure><b>(Figure)</b>
<figcaption>An example of a non lethal gene deletion in a sample metabolic network with fluxes shown by the weight of the reaction lines as calculated by FBA. Here the flux through the objective function is halved but is still present.</figcaption>
</figure>

<p><a href="File:Lethal_Gene_Deletion.png" title="wikilink">right|thumb | An example of a lethal gene deletion in a sample metabolic network with fluxes shown by the weight of the reaction lines as calculated by FBA. Here there is no flux through the objective function, simulating that the pathway is no longer functional.</a></p>
<figure><b>(Figure)</b>
<figcaption> Reaction inhibition: Plot of FBA predicted growth rate (y-axis) to decreasing influx of Oxygen (x-axis) in an E.coli FBA model.</figcaption>
</figure>
<figure><b>(Figure)</b>
<figcaption>A 3-D Phenotypic Phase Plane showing the effect of varying glucose and glycerol input fluxes on the growth rate of Mycobacterium tuberculosis. The X axis represents glycerol influx and the Y axis represents glucose influx, the height of the surface (red) represents the value of the growth flux for each combination of the input fluxes.</figcaption>
</figure>
<figure><b>(Figure)</b>
<figcaption>A levelplot version of the Phenotypic Phase Plane showing the effect of varying glucose and glycerol input fluxes on the growth rate of Mycobacterium tuberculosis. The X axis represents glycerol influx and the Y axis represents glucose influx, the color represents the value of the growth flux.</figcaption>
</figure>
<figure><b>(Figure)</b>
<figcaption> Comparison of correlation plots of lethal Pairwise reaction deletions across different subsystems for E.coli(left) and M.tuberculosis(right).</figcaption>
</figure>

<p>FBA is not computationally intensive, taking on the order of seconds to calculate optimal fluxes for biomass production for a typical network (around 2000 reactions). This means that the effect of deleting reactions from the network and/or changing flux constraints can be sensibly modelled on a single computer.</p>
<h3 id="genereaction-deletion-and-perturbation-studies">Gene/reaction deletion and perturbation studies</h3>
<h4 id="single-reaction-deletion">Single reaction deletion</h4>

<p>A frequently used technique to search a metabolic network for reactions that are particularly critical to the production of biomass. By removing each reaction in a network in turn and measuring the predicted flux through the biomass function, each reaction can be classified as either essential (if the flux through the biomass function is substantially reduced) or non-essential (if the flux through the biomass function is unchanged or only slightly reduced).</p>
<h4 id="pairwise-reaction-deletion">Pairwise reaction deletion</h4>

<p>Pairwise reaction deletion of all possible pairs of reactions is useful when looking for drug targets, as it allows the simulation of multi-target treatments, either by a single drug with multiple targets or by drug combinations. Double deletion studies can also quantify the synthetic lethal interactions between different pathways providing a measure of the contribution of the pathway to overall network robustness.</p>
<h4 id="single-and-multiple-gene-deletions">Single and multiple gene deletions</h4>

<p>Genes are connected to enzyme-catalyzed reactions by <a href="Boolean_expression" title="wikilink">Boolean expressions</a> known as Gene-Protein-Reaction expressions (GPR). Typically a GPR takes the form (Gene A AND Gene B) to indicate that the products of genes A and B are protein sub-units that assemble to form the complete protein and therefore the absence of either would result in deletion of the reaction. On the other hand, if the GPR is (Gene A OR Gene B) it implies that the products of genes A and B are <a class="uri" href="isozymes" title="wikilink">isozymes</a>.</p>

<p>Therefore it is possible to evaluate the effect of single or multiple gene deletions by evaluation of the GPR as a Boolean expression. If the GPR evaluates to <strong>false</strong>, the reaction is constrained to zero in the model prior to performing FBA. Thus gene knockouts can be simulated using FBA.</p>
<h4 id="interpretation-of-gene-and-reaction-deletion-results">Interpretation of gene and reaction deletion results</h4>

<p>The utility of reaction inhibition and deletion analyses becomes most apparent if a gene-protein-reaction matrix has been assembled for the network being studied with FBA. The gene-protein-reaction matrix is a binary matrix connecting genes with the proteins made from them. Using this matrix reaction essentiality can be converted into gene essentiality indicating the gene defects which may cause a certain disease phenotype or the proteins/enzymes which are essential (and thus what enzymes are the most promising drug targets in pathogens). However the gene-protein-reaction matrix does not specify the Boolean relationship between genes with respect to the enzyme, instead it merely indicates an association between them. Therefore it should be used only if the Boolean GPR expression is unavailable.</p>
<h4 id="reaction-inhibition">Reaction inhibition</h4>

<p>The effect of inhibiting a reaction, rather than removing it entirely, can be simulated in FBA by restricting the allowed flux through it. The effect of an inhibition can be classified as lethal or non-lethal by applying the same criteria as in the case of a deletion where a suitable threshold is used to distinguish “substantially reduced” from “slightly reduced”. Generally the choice of threshold is arbitrary but a reasonable estimate can be obtained from growth experiments where the simulated inhibitions/deletions are actually performed and growth rate is measured.</p>
<h4 id="growth-media-optimization">Growth media optimization</h4>

<p>To design optimal growth media with respect to enhanced growth rates or useful by-product secretion, it is possible to use a method known as Phenotypic Phase Plane analysis. PhPP involves applying FBA repeatedly on the model while co-varying the nutrient uptake constraints and observing the value of the objective function (or by-product fluxes). PhPP makes it possible to find the optimal combination of nutrients that favor a particular phenotype or a mode of metabolism resulting in higher growth rates or secretion of industrially useful by-products. The predicted growth rates of bacteria in varying media have been shown to correlate well with experimental results.<a class="footnoteRef" href="#fn9" id="fnref9"><sup>9</sup></a> as well as to define precise minimal media for the culture of <em>Salmonella typhimurium</em>.<a class="footnoteRef" href="#fn10" id="fnref10"><sup>10</sup></a></p>
<h2 id="mathematical-description">Mathematical description</h2>

<p>In contrast to the traditionally followed approach of metabolic modeling using coupled <a href="ordinary_differential_equations" title="wikilink">ordinary differential equations</a>, flux balance analysis requires very little information in terms of the enzyme kinetic parameters and concentration of metabolites in the system. It achieves this by making two assumptions, <a href="steady_state" title="wikilink">steady state</a> and optimality. The first assumption is that the modeled system has entered a steady state, where the metabolite concentrations no longer change, i.e. in each metabolite node the producing and consuming fluxes cancel each other out. The second assumption is that the organism has been optimized through evolution for some biological goal, such as optimal growth or conservation of resources. The steady-state assumption reduces the system to a set of linear equations, which is then solved to find a flux distribution that satisfies the steady-state condition subject to the stoichiometry constraints while maximizing the value of a pseudo-reaction (the objective function) representing the conversion of biomass precursors into biomass.</p>

<p>The steady-state assumption dates to the ideas of <a href="material_balance" title="wikilink">material balance</a> developed to model the growth of microbial cells in fermenters in bioprocess engineering. During microbial growth, a substrate consisting of a complex mixture of carbon, hydrogen, oxygen and nitrogen sources along with trace elements are consumed to generate biomass. The material balance model for this process becomes:</p>

<p>

<math display="block" id="Flux_balance_analysis:0">
 <semantics>
  <mrow>
   <mi>Input</mi>
   <mo>=</mo>
   <mrow>
    <mi>Output</mi>
    <mo>+</mo>
    <mi>Accumulation</mi>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <eq></eq>
    <ci>Input</ci>
    <apply>
     <plus></plus>
     <ci>Output</ci>
     <ci>Accumulation</ci>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \mathrm{Input}=\mathrm{Output}+\mathrm{Accumulation}
  </annotation>
 </semantics>
</math>

</p>

<p>If we consider the system of microbial cells to be at steady state then we may set the accumulation term to zero and reduce the material balance equations to simple algebraic equations. In such a system, substrate becomes the input to the system which is consumed and biomass is produced becoming the output from the system. The material balance may then be represented as:</p>

<p>

<math display="block" id="Flux_balance_analysis:1">
 <semantics>
  <mrow>
   <mi>Input</mi>
   <mo>=</mo>
   <mi>Output</mi>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <eq></eq>
    <ci>Input</ci>
    <ci>Output</ci>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \mathrm{Input}=\mathrm{Output}
  </annotation>
 </semantics>
</math>

</p>

<p>

<math display="block" id="Flux_balance_analysis:2">
 <semantics>
  <mrow>
   <mrow>
    <mi>Input</mi>
    <mo>-</mo>
    <mi>Output</mi>
   </mrow>
   <mo>=</mo>
   <mn>0</mn>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <eq></eq>
    <apply>
     <minus></minus>
     <ci>Input</ci>
     <ci>Output</ci>
    </apply>
    <cn type="integer">0</cn>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \mathrm{Input}-\mathrm{Output}=\mathrm{0}
  </annotation>
 </semantics>
</math>

</p>

<p>Mathematically, the algebraic equations can be represented as a dot product of a matrix of coefficients and a vector of the unknowns. Since the steady-state assumption puts the accumulation term to zero. The system can be written as:</p>

<p>

<math display="block" id="Flux_balance_analysis:3">
 <semantics>
  <mrow>
   <mrow>
    <mpadded lspace="20pt" width="+20pt">
     <mi>A</mi>
    </mpadded>
    <mo>⋅</mo>
    <mi>𝐱</mi>
   </mrow>
   <mo>=</mo>
   <mn>𝟎</mn>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <eq></eq>
    <apply>
     <ci>normal-⋅</ci>
     <ci>A</ci>
     <ci>𝐱</ci>
    </apply>
    <cn type="integer">0</cn>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \qquad A\cdot\mathbf{x}=\mathbf{0}
  </annotation>
 </semantics>
</math>

</p>

<p>Extending this idea to metabolic networks, it is possible to represent a metabolic network as a stoichiometry balanced set of equations. Moving to the matrix formalism, we can represent the equations as the dot product of a matrix of stoichiometry coefficients (stoichiometric matrix 

<math display="inline" id="Flux_balance_analysis:4">
 <semantics>
  <mi>S</mi>
  <annotation-xml encoding="MathML-Content">
   <ci>S</ci>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   S
  </annotation>
 </semantics>
</math>

) and the vector of fluxes 

<math display="inline" id="Flux_balance_analysis:5">
 <semantics>
  <mi>𝐯</mi>
  <annotation-xml encoding="MathML-Content">
   <ci>𝐯</ci>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \mathbf{v}
  </annotation>
 </semantics>
</math>

 as the unknowns and set the right hand side to 0 implying the steady state.</p>

<p>

<math display="block" id="Flux_balance_analysis:6">
 <semantics>
  <mrow>
   <mrow>
    <mpadded lspace="20pt" width="+20pt">
     <mi>S</mi>
    </mpadded>
    <mo>⋅</mo>
    <mi>𝐯</mi>
   </mrow>
   <mo>=</mo>
   <mn>𝟎</mn>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <eq></eq>
    <apply>
     <ci>normal-⋅</ci>
     <ci>S</ci>
     <ci>𝐯</ci>
    </apply>
    <cn type="integer">0</cn>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \qquad S\cdot\mathbf{v}=\mathbf{0}
  </annotation>
 </semantics>
</math>

</p>

<p>Metabolic networks typically have more reactions than metabolites and this gives an under-determined system of linear equations containing more variables than equations. The standard approach to solve such under-determined systems is to apply <a href="linear_programming" title="wikilink">linear programming</a>.</p>

<p>Linear programs are problems that can be expressed in <a href="canonical_form" title="wikilink">canonical form</a>:</p>

<p>

<math display="inline" id="Flux_balance_analysis:7">
 <semantics>
  <mrow>
   <msup>
    <mi>𝐜</mi>
    <mi mathvariant="normal">T</mi>
   </msup>
   <mi>𝐱</mi>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <times></times>
    <apply>
     <csymbol cd="ambiguous">superscript</csymbol>
     <ci>𝐜</ci>
     <ci>normal-T</ci>
    </apply>
    <ci>𝐱</ci>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \displaystyle\mathbf{c}^{\mathrm{T}}\mathbf{x}
  </annotation>
 </semantics>
</math>


</p>

<p>where <strong>x</strong> represents the vector of variables (to be determined), <strong>c</strong> and <strong>b</strong> are <a href="vector_space" title="wikilink">vectors</a> of (known) coefficients, <em>A</em> is a (known) <a href="Matrix_(mathematics)" title="wikilink">matrix</a> of coefficients, and 

<math display="inline" id="Flux_balance_analysis:8">
 <semantics>
  <msup>
   <mrow>
    <mo stretchy="false">(</mo>
    <mo>⋅</mo>
    <mo stretchy="false">)</mo>
   </mrow>
   <mi mathvariant="normal">T</mi>
  </msup>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <csymbol cd="ambiguous">superscript</csymbol>
    <ci>normal-⋅</ci>
    <ci>normal-T</ci>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   (\cdot)^{\mathrm{T}}
  </annotation>
 </semantics>
</math>

 is the <a href="matrix_transpose" title="wikilink">matrix transpose</a>. The expression to be maximized or minimized is called the <em>objective function</em> (<strong>c</strong><sup>T</sup><strong>x</strong> in this case). The inequalities <em>A</em><strong>x</strong> ≤ <strong>b</strong> are the constraints which specify a <a href="convex_polytope" title="wikilink">convex polytope</a> over which the objective function is to be optimized.</p>

<p>Linear Programming requires the definition of an objective function. The optimal solution to the LP problem is considered to be the solution which maximizes or minimizes the value of the objective function depending on the case in point. In the case of flux balance analysis, the objective function <strong>Z</strong> for the LP is often defined as biomass production. Biomass production is simulated by an equation representing a lumped reaction that converts various biomass precursors into one unit of biomass.</p>

<p>Therefore the canonical form of a Flux Balance Analysis problem would be:</p>

<p>

<math display="inline" id="Flux_balance_analysis:9">
 <semantics>
  <mrow>
   <msup>
    <mi>𝐜</mi>
    <mi mathvariant="normal">T</mi>
   </msup>
   <mi>𝐯</mi>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <times></times>
    <apply>
     <csymbol cd="ambiguous">superscript</csymbol>
     <ci>𝐜</ci>
     <ci>normal-T</ci>
    </apply>
    <ci>𝐯</ci>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \displaystyle\mathbf{c}^{\mathrm{T}}\mathbf{v}
  </annotation>
 </semantics>
</math>


</p>

<p>where 

<math display="inline" id="Flux_balance_analysis:10">
 <semantics>
  <mi>𝐯</mi>
  <annotation-xml encoding="MathML-Content">
   <ci>𝐯</ci>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \mathbf{v}
  </annotation>
 </semantics>
</math>

 represents the vector of fluxes (to be determined), 

<math display="inline" id="Flux_balance_analysis:11">
 <semantics>
  <mi>S</mi>
  <annotation-xml encoding="MathML-Content">
   <ci>S</ci>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   S
  </annotation>
 </semantics>
</math>

 is a (known) <a href="Matrix_(mathematics)" title="wikilink">matrix</a> of coefficients. The expression to be maximized or minimized is called the <em>objective function</em> (

<math display="inline" id="Flux_balance_analysis:12">
 <semantics>
  <mrow>
   <msup>
    <mi>𝐜</mi>
    <mi mathvariant="normal">T</mi>
   </msup>
   <mi>𝐯</mi>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <times></times>
    <apply>
     <csymbol cd="ambiguous">superscript</csymbol>
     <ci>𝐜</ci>
     <ci>normal-T</ci>
    </apply>
    <ci>𝐯</ci>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \mathbf{c}^{\mathrm{T}}\mathbf{v}
  </annotation>
 </semantics>
</math>

 in this case). The inequalities 

<math display="inline" id="Flux_balance_analysis:13">
 <semantics>
  <mrow>
   <mi>𝐥𝐨𝐰𝐞𝐫𝐛𝐨𝐮𝐧𝐝</mi>
   <mo>≤</mo>
   <mi>𝐯</mi>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <leq></leq>
    <ci>𝐥𝐨𝐰𝐞𝐫𝐛𝐨𝐮𝐧𝐝</ci>
    <ci>𝐯</ci>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \mathbf{lowerbound}\leq\mathbf{v}
  </annotation>
 </semantics>
</math>


 and 

<math display="inline" id="Flux_balance_analysis:14">
 <semantics>
  <mrow>
   <mi>𝐯</mi>
   <mo>≤</mo>
   <mi>𝐮𝐩𝐩𝐞𝐫𝐛𝐨𝐮𝐧𝐝</mi>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <leq></leq>
    <ci>𝐯</ci>
    <ci>𝐮𝐩𝐩𝐞𝐫𝐛𝐨𝐮𝐧𝐝</ci>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \mathbf{v}\leq\mathbf{upperbound}
  </annotation>
 </semantics>
</math>

 define, respectively, the minimal and the maximal rates of flux for every reaction corresponding to the columns of the 

<math display="inline" id="Flux_balance_analysis:15">
 <semantics>
  <mi>S</mi>
  <annotation-xml encoding="MathML-Content">
   <ci>S</ci>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   S
  </annotation>
 </semantics>
</math>

 matrix. These rates can be experimentally determined to constrain and improve the predictive accuracy of the model even further or they can be specified to an arbitrarily high value indicating no constraint on the flux through the reaction.</p>

<p>The main advantage of the flux balance approach is that it does not require any knowledge of the metabolite concentrations, or more importantly, the <a href="enzyme_kinetics" title="wikilink">enzyme kinetics</a> of the system; the homeostasis assumption precludes the need for knowledge of metabolite concentrations at any time as long as that quantity remains constant, and additionally it removes the need for specific <a href="Rate_equation" title="wikilink">rate laws</a> since it assumes that at steady state, there is no change in the size of the metabolite pool in the system. The stoichiometric coefficients alone are sufficient for the mathematical maximization of a specific <a href="objective_function" title="wikilink">objective function</a>.</p>

<p>The objective function is essentially a measure of how each component in the system contributes to the production of the desired product. The product itself depends on the purpose of the model, but one of the most common examples is the study of total biomass. A notable example of the success of FBA is the ability to accurately predict the growth rate of the <a class="uri" href="prokaryote" title="wikilink">prokaryote</a> <em><a href="E._coli" title="wikilink">E. coli</a></em> when cultured in different conditions.<a class="footnoteRef" href="#fn11" id="fnref11"><sup>11</sup></a> In this case, the metabolic system was optimized to maximize the biomass objective function. However this model can be used to optimize the production of any product, and is often used to determine the output level of some <a href="Biotechnology" title="wikilink">biotechnologically</a> relevant product. The model itself can be experimentally verified by cultivating organisms using a <a class="uri" href="chemostat" title="wikilink">chemostat</a> or similar tools to ensure that nutrient concentrations are held constant. Measurements of the production of the desired objective can then be used to correct the model.</p>

<p>A good description of the basic concepts of FBA can be found in the freely available supplementary material to Edwards et al. 2001<a class="footnoteRef" href="#fn12" id="fnref12"><sup>12</sup></a> which can be found at the Nature website.<a class="footnoteRef" href="#fn13" id="fnref13"><sup>13</sup></a> Further sources include the book "Systems Biology" by B. Palsson dedicated to the subject<a class="footnoteRef" href="#fn14" id="fnref14"><sup>14</sup></a> and a useful tutorial and paper by J. Orth.<a class="footnoteRef" href="#fn15" id="fnref15"><sup>15</sup></a><a class="footnoteRef" href="#fn16" id="fnref16"><sup>16</sup></a> Many other sources of information on the technique exist in published scientific literature including Lee et al. 2006,<a class="footnoteRef" href="#fn17" id="fnref17"><sup>17</sup></a> Feist et al. 2008,<a class="footnoteRef" href="#fn18" id="fnref18"><sup>18</sup></a> and Lewis et al. 2012.<a class="footnoteRef" href="#fn19" id="fnref19"><sup>19</sup></a></p>
<h2 id="model-preparation-and-refinement">Model preparation and refinement</h2>

<p>A comprehensive guide to creating, preparing and analysing a metabolic model using FBA, in addition to other techniques, was published by Thiele and Palsson in 2010.<a class="footnoteRef" href="#fn20" id="fnref20"><sup>20</sup></a> The key parts of model preparation are: creating a metabolic network without gaps, adding constraints to the model, and finally adding an objective function (often called the Biomass function), usually to simulate the growth of the organism being modelled.</p>
<h3 id="metabolic-network-and-software-tools">Metabolic network and software tools</h3>
<figure><b>(Figure)</b>
<figcaption>The first six reactions in Glycolysis prepared for FBA through the addition of an objective function (red) and the import and export of nutrients (ATP, ADP, BDG, ADG) across the system boundary (dashed green line).</figcaption>
</figure>

<p>Metabolic networks can vary in scope from those describing a single <a href="Metabolic_pathway" title="wikilink">pathway</a>, up to the <a href="Cell_(biology)" title="wikilink">cell</a>, <a href="Tissue_(biology)" title="wikilink">tissue</a> or <a class="uri" href="organism" title="wikilink">organism</a>. The main requirement of a metabolic network that forms the basis of an FBA-ready network is that it contains no gaps. This typically means that extensive manual curation is required, making the preparation of a metabolic network for flux-balance analysis a process that can take months or years. However, recent advances such as so-called gap-filling methods can reduce the required time to weeks or months.</p>

<p>Software packages for creation of FBA models include <a href="http://bioinformatics.ai.sri.com/ptools/">Pathway Tools/MetaFlux</a>,<a class="footnoteRef" href="#fn21" id="fnref21"><sup>21</sup></a><a class="footnoteRef" href="#fn22" id="fnref22"><sup>22</sup></a> Simpheny,<a class="footnoteRef" href="#fn23" id="fnref23"><sup>23</sup></a><a class="footnoteRef" href="#fn24" id="fnref24"><sup>24</sup></a> and MetNetMaker.<a class="footnoteRef" href="#fn25" id="fnref25"><sup>25</sup></a></p>

<p>Generally models are created in <a class="uri" href="BioPAX" title="wikilink">BioPAX</a> or <a class="uri" href="SBML" title="wikilink">SBML</a> format so that further analysis or visualization can take place in other software although this is not a requirement.</p>
<h3 id="constraints">Constraints</h3>

<p>A key part of FBA is the ability to add constraints to the flux rates of reactions within networks, forcing them to stay within a range of selected values. This lets the model more accurately simulate real metabolism. The constraints belong to two subsets from a biological perspective; boundary constraints that limit nutrient uptake/excretion and internal constraints that limit the flux through reactions within the organism. In mathematical terms, the application of constraints can be considered to reduce the solution space of the FBA model. In addition to constraints applied at the edges of a metabolic network, constraints can be applied to reactions deep within the network. These constraints are usually simple; they may constrain the direction of a reaction due to energy considerations or constrain the maximum speed of a reaction due to the finite speed of all reactions in nature.</p>
<h4 id="growth-media-constraints">Growth media constraints</h4>

<p>Organisms, and all other metabolic systems, require some input of nutrients. Typically the rate of uptake of nutrients is dictated by their availability (a nutrient that is not present cannot be absorbed), their concentration and diffusion constants (higher concentrations of quickly-diffusing metabolites are absorbed more quickly) and the method of absorption (such as <a href="active_transport" title="wikilink">active transport</a> or <a href="facilitated_diffusion" title="wikilink">facilitated diffusion</a> versus simple diffusion).</p>

<p>If the rate of absorption (and/or excretion) of certain nutrients can be experimentally measured then this information can be added as a constraint on the flux rate at the edges of a metabolic model. This ensures that nutrients that are not present or not absorbed by the organism do not enter its metabolism (the flux rate is constrained to zero) and also means that known nutrient uptake rates are adhered to by the simulation. This provides a secondary method of making sure that the simulated metabolism has experimentally verified properties rather than just mathematically acceptable ones.</p>
<h4 id="thermodynamical-reaction-constraints">Thermodynamical reaction constraints</h4>

<p>In principle, all reactions are reversible however in practice reactions often effectively occur in only one direction. This may be due to significantly higher concentration of reactants compared to the concentration of the products of the reaction. But more often it happens because the products of a reaction have a much lower free energy than the reactants and therefore the forward direction of a reaction is favored more.</p>

<p>For ideal reactions,</p>

<p>

<math display="block" id="Flux_balance_analysis:16">
 <semantics>
  <mrow>
   <mrow>
    <mo>-</mo>
    <mi mathvariant="normal">∞</mi>
   </mrow>
   <mo><</mo>
   <msub>
    <mi>v</mi>
    <mi>i</mi>
   </msub>
   <mo><</mo>
   <mpadded width="+1.7pt">
    <mi mathvariant="normal">∞</mi>
   </mpadded>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <and></and>
    <apply>
     <lt></lt>
     <apply>
      <minus></minus>
      <infinity></infinity>
     </apply>
     <apply>
      <csymbol cd="ambiguous">subscript</csymbol>
      <ci>v</ci>
      <ci>i</ci>
     </apply>
    </apply>
    <apply>
     <lt></lt>
     <share href="#.cmml">
     </share>
     <infinity></infinity>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   -\infty<v_{i}<\infty\,
  </annotation>
 </semantics>
</math>

</p>

<p>For certain reactions a thermodynamic constraint can be applied implying direction (in this case forward)</p>

<p>

<math display="block" id="Flux_balance_analysis:17">
 <semantics>
  <mrow>
   <mn>0</mn>
   <mo><</mo>
   <msub>
    <mi>v</mi>
    <mi>i</mi>
   </msub>
   <mo><</mo>
   <mpadded width="+1.7pt">
    <mi mathvariant="normal">∞</mi>
   </mpadded>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <and></and>
    <apply>
     <lt></lt>
     <cn type="integer">0</cn>
     <apply>
      <csymbol cd="ambiguous">subscript</csymbol>
      <ci>v</ci>
      <ci>i</ci>
     </apply>
    </apply>
    <apply>
     <lt></lt>
     <share href="#.cmml">
     </share>
     <infinity></infinity>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   0<v_{i}<\infty\,
  </annotation>
 </semantics>
</math>

</p>

<p>Realistically the flux through a reaction cannot be infinite (given that enzymes in the real system are finite) which implies that,</p>

<p>

<math display="block" id="Flux_balance_analysis:18">
 <semantics>
  <mrow>
   <mn>0</mn>
   <mo><</mo>
   <msub>
    <mi>v</mi>
    <mi>i</mi>
   </msub>
   <mo><</mo>
   <mpadded width="+1.7pt">
    <msub>
     <mi>v</mi>
     <mi>max</mi>
    </msub>
   </mpadded>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <and></and>
    <apply>
     <lt></lt>
     <cn type="integer">0</cn>
     <apply>
      <csymbol cd="ambiguous">subscript</csymbol>
      <ci>v</ci>
      <ci>i</ci>
     </apply>
    </apply>
    <apply>
     <lt></lt>
     <share href="#.cmml">
     </share>
     <apply>
      <csymbol cd="ambiguous">subscript</csymbol>
      <ci>v</ci>
      <max></max>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   0<v_{i}<v_{\max}\,
  </annotation>
 </semantics>
</math>

</p>
<h4 id="experimentally-measured-flux-constraints">Experimentally measured flux Constraints</h4>

<p>Certain flux rates can be measured experimentally (

<math display="inline" id="Flux_balance_analysis:19">
 <semantics>
  <mpadded width="+1.7pt">
   <msub>
    <mi>v</mi>
    <mrow>
     <mi>i</mi>
     <mo>,</mo>
     <mi>m</mi>
    </mrow>
   </msub>
  </mpadded>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <csymbol cd="ambiguous">subscript</csymbol>
    <ci>v</ci>
    <list>
     <ci>i</ci>
     <ci>m</ci>
    </list>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   v_{i,m}\,
  </annotation>
 </semantics>
</math>

) and the fluxes within a metabolic model can be constrained, within some error (

<math display="inline" id="Flux_balance_analysis:20">
 <semantics>
  <mpadded width="+1.7pt">
   <mi>ε</mi>
  </mpadded>
  <annotation-xml encoding="MathML-Content">
   <ci>ε</ci>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \varepsilon\,
  </annotation>
 </semantics>
</math>

), to ensure these known flux rates are accurately reproduced in the simulation.</p>

<p>

<math display="inline" id="Flux_balance_analysis:21">
 <semantics>
  <mover accent="true">
   <mi>v</mi>
   <mo stretchy="false">→</mo>
  </mover>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <ci>normal-→</ci>
    <ci>v</ci>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \vec{v}
  </annotation>
 </semantics>
</math>

. However solutions of biological interest are the ones which produce the desired metabolites in the correct proportion. The objective function defines the proportion of these metabolites. For instance when modelling the growth of an organism the objective function is generally defined as biomass. Mathematically, it is a column in the stoichiometry matrix the entries of which place a "demand" or act as a "sink" for biosynthetic precursors such as fatty acids, amino acids and cell wall components which are present on the corresponding rows of the <strong>S</strong> matrix. These entries represent experimentally measured, dry weight proportions of cellular components. Therefore this column becomes a lumped reaction that simulates growth and reproduction. Therefore the accuracy of experimental measurements plays an essential role in the correct definition of the biomass function and makes the results of FBA biologically applicable by ensuring that the correct proportion of metabolites are produced by metabolism.</p>

<p>When modeling smaller networks the objective function can be changed accordingly. An example of this would be in the study of the <a href="Glycolysis" title="wikilink">carbohydrate metabolism</a> pathways where the objective function would probably be defined as a certain proportion of <a href="Adenosine_triphosphate" title="wikilink">ATP</a> and <a class="uri" href="NADH" title="wikilink">NADH</a> and thus simulate the production of high energy metabolites by this pathway.</p>
<h4 id="optimization-of-the-objectivebiomass-function">Optimization of the objective/biomass function</h4>

<p>Linear programming can be used to find a single optimal solution. The most common biological optimization goal for a whole organism metabolic network would be to choose the flux vector 

<math display="inline" id="Flux_balance_analysis:22">
 <semantics>
  <msub>
   <mi>v</mi>
   <mtext>biomass</mtext>
  </msub>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <csymbol cd="ambiguous">subscript</csymbol>
    <ci>v</ci>
    <mtext>biomass</mtext>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   v_{\textrm{biomass}}
  </annotation>
 </semantics>
</math>

 that maximises the flux through a biomass function composed of the constituent metabolites of the organism placed into the stoichiometric matrix and denoted 

<math display="inline" id="Flux_balance_analysis:23">
 <semantics>
  <msub>
   <mi>v</mi>
   <mi>b</mi>
  </msub>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <csymbol cd="ambiguous">subscript</csymbol>
    <ci>v</ci>
    <ci>b</ci>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   v_{b}
  </annotation>
 </semantics>
</math>


 or simply 

<math display="block" id="Flux_balance_analysis:24">
 <semantics>
  <mrow>
   <mrow>
    <mrow>
     <mpadded width="+5pt">
      <munder>
       <mi>max</mi>
       <mover accent="true">
        <mi>v</mi>
        <mo stretchy="false">→</mo>
       </mover>
      </munder>
     </mpadded>
     <msub>
      <mi>v</mi>
      <mi>b</mi>
     </msub>
    </mrow>
    <mtext>s. t.</mtext>
    <mrow>
     <mpadded width="+1.7pt">
      <mi>𝕊</mi>
     </mpadded>
     <mover accent="true">
      <mi>v</mi>
      <mo stretchy="false">→</mo>
     </mover>
    </mrow>
   </mrow>
   <mo>=</mo>
   <mn>0</mn>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <eq></eq>
    <list>
     <apply>
      <apply>
       <csymbol cd="ambiguous">subscript</csymbol>
       <max></max>
       <apply>
        <ci>normal-→</ci>
        <ci>v</ci>
       </apply>
      </apply>
      <apply>
       <csymbol cd="ambiguous">subscript</csymbol>
       <ci>v</ci>
       <ci>b</ci>
      </apply>
     </apply>
     <mtext>s. t.</mtext>
     <apply>
      <times></times>
      <ci>𝕊</ci>
      <apply>
       <ci>normal-→</ci>
       <ci>v</ci>
      </apply>
     </apply>
    </list>
    <cn type="integer">0</cn>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \max_{\vec{v}}\ v_{b}\qquad\textrm{s. t.}\qquad\mathbb{S}\,\vec{v}=0
  </annotation>
 </semantics>
</math>

</p>

<p>

<math display="inline" id="Flux_balance_analysis:25">
 <semantics>
  <mover accent="true">
   <mi>c</mi>
   <mo stretchy="false">→</mo>
  </mover>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <ci>normal-→</ci>
    <ci>c</ci>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \vec{c}
  </annotation>
 </semantics>
</math>

</p>

<p>In the more general case any reaction be defined and added defined as a biomass function with either the condition that it be maximised or minimised if a single “optimal” solution is desired. Alternatively, and in the most general case, a vector 

<math display="block" id="Flux_balance_analysis:26">
 <semantics>
  <mrow>
   <mrow>
    <mrow>
     <mpadded width="+5pt">
      <munder>
       <mi>max</mi>
       <mover accent="true">
        <mi>v</mi>
        <mo stretchy="false">→</mo>
       </mover>
      </munder>
     </mpadded>
     <mrow>
      <mover accent="true">
       <mi>v</mi>
       <mo stretchy="false">→</mo>
      </mover>
      <mo>⋅</mo>
      <mover accent="true">
       <mi>c</mi>
       <mo stretchy="false">→</mo>
      </mover>
     </mrow>
    </mrow>
    <mtext>s. t.</mtext>
    <mrow>
     <mpadded width="+1.7pt">
      <mi>𝕊</mi>
     </mpadded>
     <mover accent="true">
      <mi>v</mi>
      <mo stretchy="false">→</mo>
     </mover>
    </mrow>
   </mrow>
   <mo>=</mo>
   <mn>0</mn>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <eq></eq>
    <list>
     <apply>
      <apply>
       <csymbol cd="ambiguous">subscript</csymbol>
       <max></max>
       <apply>
        <ci>normal-→</ci>
        <ci>v</ci>
       </apply>
      </apply>
      <apply>
       <ci>normal-⋅</ci>
       <apply>
        <ci>normal-→</ci>
        <ci>v</ci>
       </apply>
       <apply>
        <ci>normal-→</ci>
        <ci>c</ci>
       </apply>
      </apply>
     </apply>
     <mtext>s. t.</mtext>
     <apply>
      <times></times>
      <ci>𝕊</ci>
      <apply>
       <ci>normal-→</ci>
       <ci>v</ci>
      </apply>
     </apply>
    </list>
    <cn type="integer">0</cn>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \max_{\vec{v}}\ \vec{v}\cdot\vec{c}\qquad\textrm{s. t.}\qquad\mathbb{S}\,\vec{%
v}=0
  </annotation>
 </semantics>
</math>

 can be defined which defines the weighted set of reactions that the linear programming model should aim to maximise or minimise,</p>

<p>

<math display="inline" id="Flux_balance_analysis:27">
 <semantics>
  <mover accent="true">
   <mi>c</mi>
   <mo stretchy="false">→</mo>
  </mover>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <ci>normal-→</ci>
    <ci>c</ci>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \vec{c}
  </annotation>
 </semantics>
</math>

</p>

<p>In the case of there being only a single separate biomass function/reaction within the stoichiometric matrix 

<math display="inline" id="Flux_balance_analysis:28">
 <semantics>
  <mover accent="true">
   <mi>c</mi>
   <mo stretchy="false">→</mo>
  </mover>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <ci>normal-→</ci>
    <ci>c</ci>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \vec{c}
  </annotation>
 </semantics>
</math>


 would simplify to all zeroes with a value of 1 (or any non-zero value) in the position corresponding to that biomass function. Where there were multiple separate objective functions 

<math display="inline" id="Flux_balance_analysis:29">
 <semantics>
  <mi>𝕊</mi>
  <annotation-xml encoding="MathML-Content">
   <ci>𝕊</ci>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \scriptstyle\mathbb{S}
  </annotation>
 </semantics>
</math>

 would simplify to all zeroes with weighted values in the positions corresponding to all objective functions.</p>
<h3 id="reducing-the-solution-space-biological-considerations-for-the-system">Reducing the solution space – biological considerations for the system</h3>

<p>The analysis of the null space of matrices is implemented in software packages specialized for matrix operations such as Matlab and Octave. Determination of the null space of 

<math display="inline" id="Flux_balance_analysis:30">
 <semantics>
  <msub>
   <mi>v</mi>
   <mn>1</mn>
  </msub>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <csymbol cd="ambiguous">subscript</csymbol>
    <ci>v</ci>
    <cn type="integer">1</cn>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   v_{1}
  </annotation>
 </semantics>
</math>

 tells us all the possible collections of flux vectors (or linear combinations thereof) that balance fluxes within the biological network. The advantage of this approach becomes evident in biological systems which are described by differential equation systems with many unknowns. The velocities in the differential equations above - 

<math display="inline" id="Flux_balance_analysis:31">
 <semantics>
  <msub>
   <mi>v</mi>
   <mn>2</mn>
  </msub>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <csymbol cd="ambiguous">subscript</csymbol>
    <ci>v</ci>
    <cn type="integer">2</cn>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   v_{2}
  </annotation>
 </semantics>
</math>

 and 

<math display="inline" id="Flux_balance_analysis:32">
 <semantics>
  <mrow>
   <mrow>
    <mpadded width="+1.7pt">
     <mi>𝕊</mi>
    </mpadded>
    <mover accent="true">
     <mi>v</mi>
     <mo stretchy="false">→</mo>
    </mover>
   </mrow>
   <mo>=</mo>
   <mn>0</mn>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <eq></eq>
    <apply>
     <times></times>
     <ci>𝕊</ci>
     <apply>
      <ci>normal-→</ci>
      <ci>v</ci>
     </apply>
    </apply>
    <cn type="integer">0</cn>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \scriptstyle\mathbb{S}\,\vec{v}=0
  </annotation>
 </semantics>
</math>

 - are dependent on the reaction rates of the underlying equations. The velocities are generally taken from the <a href="Michaelis-Menten_kinetics" title="wikilink">Michaelis–Menten kinetic theory</a>, which involves the kinetic parameters of the enzymes catalyzing the reactions and the concentration of the metabolites themselves. Isolating enzymes from living organisms and measuring their kinetic parameters is a difficult task, as is measuring the internal concentrations and diffusion constants of metabolites within an organism. Therefore the differential equation approach to metabolic modeling is beyond the current scope of science for all but the most studied organisms.<a class="footnoteRef" href="#fn26" id="fnref26"><sup>26</sup></a> FBA avoids this impediment by applying the homeostatic assumption, which is a reasonably approximate description of biological systems.</p>

<p>Although FBA avoids that biological obstacle, the mathematical issue of a large solution space remains. FBA has a two-fold purpose. Accurately representing the biological limits of the system and returning the flux distribution closest to the natural fluxes within the target system/organism. Certain biological principles can help overcome the mathematical difficulties. While the stoichiometric matrix is almost always under-determined initially (meaning that the solution space to 

<math display="inline" id="Flux_balance_analysis:33">
 <semantics>
  <mrow>
   <mpadded width="+5pt">
    <mi>min</mi>
   </mpadded>
   <mo stretchy="false">|</mo>
   <mo stretchy="false">|</mo>
   <msub>
    <mi>𝐯</mi>
    <mi>𝐰</mi>
   </msub>
   <mo>-</mo>
   <msub>
    <mi>𝐯</mi>
    <mi>𝐝</mi>
   </msub>
   <mo stretchy="false">|</mo>
   <msup>
    <mo stretchy="false">|</mo>
    <mn>2</mn>
   </msup>
   <mi>s</mi>
   <mo>.</mo>
   <mi>t</mi>
   <mo>.</mo>
   <mi>𝐒</mi>
   <mo>⋅</mo>
   <msub>
    <mi>𝐯</mi>
    <mi>𝐝</mi>
   </msub>
   <mo>=</mo>
   <mn>0</mn>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <cerror>
    <csymbol cd="ambiguous">fragments</csymbol>
    <min></min>
    <ci>normal-|</ci>
    <ci>normal-|</ci>
    <apply>
     <csymbol cd="ambiguous">subscript</csymbol>
     <ci>𝐯</ci>
     <ci>𝐰</ci>
    </apply>
    <minus></minus>
    <apply>
     <csymbol cd="ambiguous">subscript</csymbol>
     <ci>𝐯</ci>
     <ci>𝐝</ci>
    </apply>
    <ci>normal-|</ci>
    <apply>
     <csymbol cd="ambiguous">superscript</csymbol>
     <ci>normal-|</ci>
     <cn type="integer">2</cn>
    </apply>
    <ci>italic-</ci>
    <csymbol cd="unknown">s</csymbol>
    <ci>normal-.</ci>
    <csymbol cd="unknown">t</csymbol>
    <ci>normal-.</ci>
    <ci></ci>
    <csymbol cd="unknown">S</csymbol>
    <ci>normal-⋅</ci>
    <apply>
     <csymbol cd="ambiguous">subscript</csymbol>
     <ci>𝐯</ci>
     <ci>𝐝</ci>
    </apply>
    <eq></eq>
    <cn type="integer">0</cn>
   </cerror>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \min\ ||\mathbf{v_{w}}-\mathbf{v_{d}}||^{2}\qquad s.t.\quad\mathbf{S}\cdot%
\mathbf{v_{d}}=0
  </annotation>
 </semantics>
</math>


 is very large), the size of the solution space can be reduced and be made more reflective of the biology of the problem through the application of certain constraints on the solutions.</p>
<h2 id="extensions">Extensions</h2>

<p>The success of FBA and the realization of its limitations has led to extensions that attempt to mediate the limitations of the technique.</p>
<h3 id="flux-variability-analysis">Flux variability analysis</h3>
<figure><b>(Figure)</b>
<figcaption>Visual and numerical representation of FVA on a complete network.</figcaption>
</figure>

<p>The optimal solution to the flux-balance problem is rarely unique with many possible, and equally optimal, solutions existing. Flux variability analysis (FVA), built into virtually all current analysis software, returns the boundaries for the fluxes through each reaction that can, paired with the right combination of other fluxes, produce the optimal solution.</p>
<figure><b>(Figure)</b>
<figcaption>Visual and numerical representation of FVA on a network with non-lethal deletion.</figcaption>
</figure>

<p>Reactions which can support a low variability of fluxes through them are likely to be of a higher importance to an organism and FVA is a promising technique for the identification of reactions that are highly important.</p>
<h3 id="minimization-of-metabolic-adjustment-moma">Minimization of Metabolic Adjustment (MOMA)</h3>

<p>When simulating knockouts or growth on media, FBA gives the final steady-state flux distribution. This final steady state is reached in varying time-scales. For example, the predicted growth rate of <em>E. coli</em> on glycerol as the primary carbon source did not match the FBA predictions, however on sub-culturing for 40 days or 700 generations the growth rate adaptively evolved to match the FBA prediction.<a class="footnoteRef" href="#fn27" id="fnref27"><sup>27</sup></a></p>

<p>Sometimes it is of interest to find out what is the immediate effect of a perturbation or knockout, since it takes time for regulatory changes to occur and for the organism to re-organize fluxes to optimally utilize a different carbon source or circumvent the effect of the knockout. MOMA predicts the immediate sub-optimal flux distribution following the perturbation by minimizing the distance (Euclidean) between the wild-type FBA flux distribution and the mutant flux distribution using quadratic programming. This yields an optimization problem of the form.</p>

<p>

<math display="inline" id="Flux_balance_analysis:34">
 <semantics>
  <msub>
   <mi>𝐯</mi>
   <mi>𝐰</mi>
  </msub>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <csymbol cd="ambiguous">subscript</csymbol>
    <ci>𝐯</ci>
    <ci>𝐰</ci>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \mathbf{v_{w}}
  </annotation>
 </semantics>
</math>

</p>

<p>where 

<math display="inline" id="Flux_balance_analysis:35">
 <semantics>
  <msub>
   <mi>𝐯</mi>
   <mi>𝐝</mi>
  </msub>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <csymbol cd="ambiguous">subscript</csymbol>
    <ci>𝐯</ci>
    <ci>𝐝</ci>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \mathbf{v_{d}}
  </annotation>
 </semantics>
</math>

 represents the wild-type (or unperturbed state) flux distribution and 

<math display="inline" id="Flux_balance_analysis:36">
 <semantics>
  <mrow>
   <mpadded width="+5pt">
    <mi>min</mi>
   </mpadded>
   <mpadded width="+1.7pt">
    <mfrac>
     <mn>1</mn>
     <mn>2</mn>
    </mfrac>
   </mpadded>
   <mpadded width="+1.7pt">
    <mmultiscripts>
     <mi>𝐯</mi>
     <mi>𝐝</mi>
     <none></none>
     <none></none>
     <mi>T</mi>
    </mmultiscripts>
   </mpadded>
   <mpadded width="+1.7pt">
    <mi>𝐈</mi>
   </mpadded>
   <msub>
    <mi>𝐯</mi>
    <mi>𝐝</mi>
   </msub>
   <mo>+</mo>
   <mrow>
    <mo stretchy="false">(</mo>
    <mo>-</mo>
    <msub>
     <mi>𝐯</mi>
     <mi>𝐰</mi>
    </msub>
    <mo stretchy="false">)</mo>
   </mrow>
   <mo>⋅</mo>
   <msub>
    <mi>𝐯</mi>
    <mi>𝐝</mi>
   </msub>
   <mi>s</mi>
   <mo>.</mo>
   <mi>t</mi>
   <mo>.</mo>
   <mi>𝐒</mi>
   <mo>⋅</mo>
   <msub>
    <mi>𝐯</mi>
    <mi>𝐝</mi>
   </msub>
   <mo>=</mo>
   <mn>0</mn>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <cerror>
    <csymbol cd="ambiguous">fragments</csymbol>
    <min></min>
    <apply>
     <divide></divide>
     <cn type="integer">1</cn>
     <cn type="integer">2</cn>
    </apply>
    <apply>
     <csymbol cd="ambiguous">superscript</csymbol>
     <apply>
      <csymbol cd="ambiguous">subscript</csymbol>
      <ci>𝐯</ci>
      <ci>𝐝</ci>
     </apply>
     <ci>T</ci>
    </apply>
    <csymbol cd="unknown">I</csymbol>
    <apply>
     <csymbol cd="ambiguous">subscript</csymbol>
     <ci>𝐯</ci>
     <ci>𝐝</ci>
    </apply>
    <plus></plus>
    <cerror>
     <csymbol cd="ambiguous">fragments</csymbol>
     <ci>normal-(</ci>
     <minus></minus>
     <apply>
      <csymbol cd="ambiguous">subscript</csymbol>
      <ci>𝐯</ci>
      <ci>𝐰</ci>
     </apply>
     <ci>normal-)</ci>
    </cerror>
    <ci>normal-⋅</ci>
    <apply>
     <csymbol cd="ambiguous">subscript</csymbol>
     <ci>𝐯</ci>
     <ci>𝐝</ci>
    </apply>
    <ci>italic-</ci>
    <csymbol cd="unknown">s</csymbol>
    <ci>normal-.</ci>
    <csymbol cd="unknown">t</csymbol>
    <ci>normal-.</ci>
    <ci></ci>
    <csymbol cd="unknown">S</csymbol>
    <ci>normal-⋅</ci>
    <apply>
     <csymbol cd="ambiguous">subscript</csymbol>
     <ci>𝐯</ci>
     <ci>𝐝</ci>
    </apply>
    <eq></eq>
    <cn type="integer">0</cn>
   </cerror>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \min\ \frac{1}{2}\,{\mathbf{v_{d}}}^{T}\,\mathbf{I}\,\mathbf{v_{d}}+(\mathbf{-%
v_{w}})\cdot\mathbf{v_{d}}\qquad s.t.\quad\mathbf{S}\cdot\mathbf{v_{d}}=0
  </annotation>
 </semantics>
</math>

 represents the flux distribution on gene deletion that is to be solved for. This simplifies to:</p>

<p><span class="LaTeX">$\min\ \frac{1}{2}\,{\mathbf{v_d}}^T\,\mathbf{I}\,\mathbf{v_d} + (\mathbf{-v_w})\cdot\mathbf{v_d} \qquad s. t.\quad \mathbf{S}\cdot\mathbf{v_d}=0$</span></p>

<p>This is the MOMA solution which represents the flux distribution immediately post-perturbation.<a class="footnoteRef" href="#fn28" id="fnref28"><sup>28</sup></a></p>
<h3 id="regulatory-on-off-minimization-room">Regulatory On-Off Minimization (ROOM)</h3>

<p>ROOM attempts to improve the prediction of the metabolic state of an organism after a gene knockout. It follows the same premise as MOMA that an organism would try to restore a flux distribution as close as possible to the wild-type after a knockout. However it further hypothesizes that this steady state would be reached through a series of transient metabolic changes by the regulatory network and that the organism would try to minimize the number of regulatory changes required to reach the wild-type state. Instead of using a distance metric minimization however it uses a Mixed Integer Linear Programming method.<a class="footnoteRef" href="#fn29" id="fnref29"><sup>29</sup></a></p>
<h3 id="dynamic-fba">Dynamic FBA</h3>

<p>Dynamic FBA attempts to add the ability for models to change over time, thus in some ways avoiding the strict steady state condition of pure FBA. Typically the technique involves running an FBA simulation, changing the model based on the outputs of that simulation, and rerunning the simulation. By repeating this process an element of feedback is achieved over time.</p>
<h2 id="comparison-with-other-techniques">Comparison with other techniques</h2>

<p>FBA provides a less simplistic analysis than Choke Point Analysis while requiring far less information on reaction rates and a much less complete network reconstruction than a full dynamic simulation would require. In filling this niche, FBA has been shown to be a very useful technique for analysis of the metabolic capabilities of cellular systems.</p>
<h3 id="choke-point-analysis">Choke point analysis</h3>

<p>Unlike <a href="choke_point_analysis" title="wikilink">choke point analysis</a> which only considers points in the network where metabolites are produced but not consumed or vice versa, FBA is a true form of <a href="metabolic_network_modelling" title="wikilink">metabolic network modelling</a> because it considers the metabolic network as a single complete entity (<a href="Stoichiometry#Stoichiometry_matrix" title="wikilink">the stoichiometric matrix</a>) at all stages of analysis. This means that network effects, such as chemical reactions in distant pathways affecting each other, can be reproduced in the model. The upside to the inability of choke point analysis to simulate network effects is that it considers each reaction within a network in isolation and thus can suggest important reactions in a network even if a network is highly fragmented and contains many gaps.</p>
<h3 id="dynamic-metabolic-simulation">Dynamic metabolic simulation</h3>

<p>Unlike <a href="dynamic_metabolic_simulation" title="wikilink">dynamic metabolic simulation</a>, FBA assumes that the internal concentration of metabolites within a system stays constant over time and thus is unable to provide anything other than steady-state solutions. It is unlikely that FBA could, for example, simulate the functioning of a nerve cell. Since the internal concentration of metabolites is not considered within a model, it is possible that an FBA solution could contain metabolites at a concentration too high to be biologically acceptable. This is a problem that dynamic metabolic simulations would probably avoid. One advantage of the simplicity of FBA over dynamic simulations is that they are far less computationally expensive, allowing the simulation of large numbers of perturbations to the network. A second advantage is that the reconstructed model can be substantially simpler by avoiding the need to consider enzyme rates and the effect of complex interactions on enzyme kinetics.</p>
<h2 id="references">References</h2>

<p>"</p>

<p><a class="uri" href="Category:Bioinformatics" title="wikilink">Category:Bioinformatics</a> <a href="Category:Systems_biology" title="wikilink">Category:Systems biology</a> <a href="Category:Computational_biology" title="wikilink">Category:Computational biology</a></p>
<section class="footnotes">
<hr/>
<ol>
<li id="fn1">Ranganathan, Sridhar, Patrick F. Suthers, and Costas D. Maranas. "OptForce: An Optimization Procedure for Identifying All Genetic Manipulations Leading to Targeted Overproductions." <em>PLoS Comput Biol</em> 6, no. 4 (April 15, 2010): e1000744. <a href="#fnref1">↩</a></li>
<li id="fn2"><a href="#fnref2">↩</a></li>
<li id="fn3">Raman, Karthik, Kalidas Yeturu, and Nagasuma Chandra. "targetTB: A Target Identification Pipeline for Mycobacterium tuberculosis Through an Interactome, Reactome and Genome-scale Structural Analysis." BMC Systems Biology 2, no. 1 (December 19, 2008): 109. <a class="uri" href="doi:10.1186/1752-0509-2-109">doi:10.1186/1752-0509-2-109</a>.<a href="#fnref3">↩</a></li>
<li id="fn4">Hong Yang, Charles M. Roth, Marianthi G. Ierapetritou. "A rational design approach for amino acid supplementation in hepatocyte culture." Biotechnology and Bioengineering,2009, 103: 1176–1191:DOI: 10.1002/bit.22342<a href="#fnref4">↩</a></li>
<li id="fn5">Raghunathan, Anu, Sookil Shin, and Simon Daefler. "Systems Approach to Investigating Host-pathogen Interactions in Infections with the Biothreat Agent Francisella. Constraints-based Model of Francisella tularensis." BMC Systems Biology 4, no. 1 (August 23, 2010): 118. <a class="uri" href="doi:10.1186/1752-0509-4-118">doi:10.1186/1752-0509-4-118</a>.<a href="#fnref5">↩</a></li>
<li id="fn6"><a href="#fnref6">↩</a></li>
<li id="fn7">Watson MR (1984) Metabolic maps for the Apple II. 12, 1093-1094<a href="#fnref7">↩</a></li>
<li id="fn8"><a href="#fnref8">↩</a></li>
<li id="fn9">Edwards, J., Ibarra, R. &amp; Palsson, B. In silico predictions of <em>Escherichia coli</em> metabolic capabilities are consistent with experimental data. Nature Biotechnology 19, 125–130(2001).<a href="#fnref9">↩</a></li>
<li id="fn10">Raghunathan, A. et al. Constraint-based analysis of metabolic capacity of <em>Salmonella typhimurium</em> during host-pathogen interaction. BMC systems biology 3, 38(2009).<a href="#fnref10">↩</a></li>
<li id="fn11">Edwards, J., Ibarra, R. &amp; Palsson, B. <em>In silico</em> predictions of Escherichia coli metabolic capabilities are consistent with experimental data. Nature Biotechnology 19, 125–130(2001).<a href="#fnref11">↩</a></li>
<li id="fn12"></li>
<li id="fn13">(http://www.nature.com/nbt/web_extras/supp_info/nbt0201_125/info_frame.html)<a href="#fnref13">↩</a></li>
<li id="fn14">Palsson, B.O. Systems Biology: Properties of Reconstructed Networks. 334(Cambridge University Press: 2006).<a href="#fnref14">↩</a></li>
<li id="fn15"><a href="#fnref15">↩</a></li>
<li id="fn16"></li>
<li id="fn17"><a href="#fnref17">↩</a></li>
<li id="fn18"><a href="#fnref18">↩</a></li>
<li id="fn19"><a href="#fnref19">↩</a></li>
<li id="fn20">Thiele, I. &amp; Palsson, B.Ø. A protocol for generating a high-quality genome-scale metabolic reconstruction. Nature protocols 5, 93-121(2010).<a href="#fnref20">↩</a></li>
<li id="fn21"><a href="#fnref21">↩</a></li>
<li id="fn22"><a href="#fnref22">↩</a></li>
<li id="fn23">Schilling, C.H. et al. SimPheny: A Computational Infrastructure for Systems Biology. (2008).<a href="#fnref23">↩</a></li>
<li id="fn24"><a class="uri" href="http://www.genomatica.com/technology/technologySuite.html">http://www.genomatica.com/technology/technologySuite.html</a><a href="#fnref24">↩</a></li>
<li id="fn25"><a class="uri" href="http://www.metnetmaker.com">http://www.metnetmaker.com</a><a href="#fnref25">↩</a></li>
<li id="fn26"><a href="#fnref26">↩</a></li>
<li id="fn27">Ibarra, Rafael U., Jeremy S. Edwards, and Bernhard O. Palsson. "Escherichia Coli K-12 Undergoes Adaptive Evolution to Achieve in Silico Predicted Optimal Growth." <em>Nature</em> 420, no. 6912 (November 14, 2002): 186–189. <a href="#fnref27">↩</a></li>
<li id="fn28">Segrè, Daniel, Dennis Vitkup, and George M. Church. "Analysis of Optimality in Natural and Perturbed Metabolic Networks." <em>Proceedings of the National Academy of Sciences</em> 99, no. 23 (November 12, 2002): 15112–15117. <a href="#fnref28">↩</a></li>
<li id="fn29">Shlomi, Tomer, Omer Berkman, and Eytan Ruppin. "Regulatory On/off Minimization of Metabolic Flux Changes After Genetic Perturbations." <em>Proceedings of the National Academy of Sciences of the United States of America</em> 102, no. 21 (May 24, 2005): 7695–7700. <a class="uri" href="doi:10.1073/pnas.0406346102">doi:10.1073/pnas.0406346102</a>.<a href="#fnref29">↩</a></li>
</ol>
</section>
</body>
</html>
