<html lang="en">
<head>
<meta charset="utf-8"/>
<title offset="1613">Genetic linkage</title>
<script src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_SVG.js" type="text/javascript">
</script>
</head>
<body>
<h1>Genetic linkage</h1>
<hr/>

<p> <strong>Genetic linkage</strong> is the tendency of alleles that are located close together on a <a class="uri" href="chromosome" title="wikilink">chromosome</a> to be inherited together during <a class="uri" href="meiosis" title="wikilink">meiosis</a>. Genes whose <a href="Locus_(genetics)" title="wikilink">loci</a> are nearer to each other are less likely to be separated onto different <a href="chromatid" title="wikilink">chromatids</a> during <a href="chromosomal_crossover" title="wikilink">chromosomal crossover</a>, and are therefore said to be genetically <em>linked</em>. In other words, the nearer two genes are on a chromosome, the lower is the chance of a swap occurring between them, and the more likely they are to be inherited together.</p>
<h2 id="discovery">Discovery</h2>

<p>Genetic linkage was first discovered by the <a href="United_Kingdom" title="wikilink">British</a> geneticists <a href="William_Bateson" title="wikilink">William Bateson</a>, <a href="Edith_Rebecca_Saunders" title="wikilink">Edith Rebecca Saunders</a> and <a href="Reginald_Punnett" title="wikilink">Reginald Punnett</a> shortly after <a href="Gregor_Mendel#Rediscovery_of_Mendel.27s_work" title="wikilink">Mendel's laws were rediscovered</a>.<a class="footnoteRef" href="#fn1" id="fnref1"><sup>1</sup></a><a class="footnoteRef" href="#fn2" id="fnref2"><sup>2</sup></a> The understanding of genetic linkage was expanded by the work of <a href="Thomas_Hunt_Morgan" title="wikilink">Thomas Hunt Morgan</a>. Morgan's observation that the amount of crossing over between linked genes differs led to the idea that crossover frequency might indicate the distance separating genes on the <a class="uri" href="chromosome" title="wikilink">chromosome</a>.</p>

<p><a href="Alfred_Sturtevant" title="wikilink">Alfred Sturtevant</a>, a student of Morgan's, first developed genetic maps, also known as linkage maps. Sturtevant proposed that the greater the distance between linked genes, the greater the chance that non-sister chromatids would cross over in the region between the genes. By working out the number of <a href="genetic_recombination" title="wikilink">recombinants</a> it is possible to obtain a measure for the distance between the genes. This distance is expressed in terms of a <strong>genetic map unit</strong> <strong>(m.u.)</strong>, or a <strong><a class="uri" href="centimorgan" title="wikilink">centimorgan</a></strong> and is defined as the distance between genes for which one product of <a class="uri" href="meiosis" title="wikilink">meiosis</a> in 100 is recombinant. A <strong>recombinant frequency</strong> (RF) of 1% is equivalent to 1 m.u. But this equivalence is only a good approximation for small percentages; the largest percentage of recombinants cannot exceed 50%, which would be the situation where the two genes are at the extreme opposite ends of the same chromosomes. In this situation, any crossover events would result in an exchange of genes, but only an odd number of crossover events (a 50-50 chance between even and odd number of crossover events) would result in a recombinant product of meiotic crossover. A statistical interpretation of this is through the <a href="J._B._S._Haldane" title="wikilink">Haldane</a> mapping function or the <a href="Damodar_Dharmananda_Kosambi" title="wikilink">Kosambi</a> mapping function, among others. A linkage map is created by finding the map distances between a number of traits that are present on the same chromosome, ideally avoiding having significant gaps between traits to avoid the inaccuracies that will occur due to the possibility of multiple recombination events.</p>
<h2 id="linkage-map">Linkage map</h2>

<p>A linkage map is a genetic map of a species or experimental population that shows the position of its known <a class="uri" href="genes" title="wikilink">genes</a> or <a href="genetic_markers" title="wikilink">genetic markers</a> relative to each other in terms of recombination frequency, rather than a specific physical distance along each chromosome. Linkage mapping is critical for identifying the location of genes that cause genetic diseases.</p>

<p>A genetic map is a map based on the frequencies of <a href="homologous_recombination" title="wikilink">recombination</a> between markers during <a href="chromosomal_crossover" title="wikilink">crossover</a> of <a href="homologous_chromosome" title="wikilink">homologous chromosomes</a>. The greater the frequency of recombination (segregation) between two genetic markers, the further apart they are assumed to be. Conversely, the lower the frequency of recombination between the markers, the smaller the physical distance between them. Historically, the markers originally used were detectable <a href="phenotype" title="wikilink">phenotypes</a> (enzyme production, eye color) derived from <a href="coding_DNA" title="wikilink">coding DNA</a> sequences; eventually, confirmed or assumed <a href="noncoding_DNA" title="wikilink">noncoding DNA</a> sequences such as <a href="Microsatellite_(genetics)" title="wikilink">microsatellites</a> or those generating restriction fragment length polymorphisms (<a href="RFLP" title="wikilink">RFLPs</a>) have been used.</p>

<p>Genetic maps help researchers to locate other markers, such as other genes by testing for genetic linkage of the already known markers.</p>

<p>A genetic map is <strong>not</strong> a physical map (such as a <a href="radiation_reduced_hybrid" title="wikilink">radiation reduced hybrid</a> map) or <a href="gene_map" title="wikilink">gene map</a>.</p>
<h2 id="parametric-and-non-parametric-linkage-analysis">Parametric and non-parametric linkage analysis</h2>

<p>Linkage analysis may be either parametric (if we know the relationship between phenotypic and genetic similarity) or non-parametric. Parametric linkage analysis is the traditional approach, whereby the probability that a gene important for a disease is linked to a genetic marker is studied through the LOD score, which assesses how probably a given pedigree, where the disease and the marker are <a href="Cosegregation" title="wikilink">cosegregating</a>, is due to the existence of linkage (with a given linkage value) or to chance. Non-parametric linkage analysis, in turn, studies the probability of an allele being <a href="identical_by_descent" title="wikilink">identical by descent</a> with itself.</p>
<h3 id="parametric-linkage-analysis">Parametric linkage analysis</h3>

<p>The <strong>LOD score</strong> (logarithm (base 10) of odds), developed by <a href="Newton_E._Morton" title="wikilink">Newton E. Morton</a>,<a class="footnoteRef" href="#fn3" id="fnref3"><sup>3</sup></a> is a statistical test often used for <a href="linkage_analysis" title="wikilink">linkage analysis</a> in human, animal, and plant populations. The LOD score compares the likelihood of obtaining the test data if the two loci are indeed linked, to the likelihood of observing the same data purely by chance. Positive LOD scores favor the presence of linkage, whereas negative LOD scores indicate that linkage is less likely. Computerized LOD score analysis is a simple way to analyze complex family pedigrees in order to determine the linkage between Mendelian traits (or between a trait and a marker, or two markers).</p>

<p>The method is described in greater detail by Strachan and Read <a href="http://www.ncbi.nlm.nih.gov/books/bv.fcgi?rid=hmg.section.1412">1</a>. Briefly, it works as follows:</p>
<ol>
<li>Establish a <a href="Pedigree_chart" title="wikilink">pedigree</a></li>
<li>Make a number of estimates of recombination frequency</li>
<li>Calculate a LOD score for each estimate</li>
<li>The estimate with the highest LOD score will be considered the best estimate</li>
</ol>

<p>The LOD score is calculated as follows:</p>

<p>

<math display="inline" id="Genetic_linkage:0">
 <semantics>
  <mtable>
   <mtr>
    <mtd columnalign="right">
     <mrow>
      <mrow>
       <mi>L</mi>
       <mi>O</mi>
       <mi>D</mi>
      </mrow>
      <mo>=</mo>
      <mi>Z</mi>
     </mrow>
    </mtd>
    <mtd columnalign="left">
     <mrow>
      <mi></mi>
      <mo>=</mo>
      <mrow>
       <msub>
        <mi>log</mi>
        <mn>10</mn>
       </msub>
       <mstyle displaystyle="true">
        <mfrac>
         <mtext>probability of birth sequence with a given linkage value</mtext>
         <mtext>probability of birth sequence with no linkage</mtext>
        </mfrac>
       </mstyle>
      </mrow>
      <mo>=</mo>
      <mrow>
       <msub>
        <mi>log</mi>
        <mn>10</mn>
       </msub>
       <mstyle displaystyle="true">
        <mfrac>
         <mrow>
          <msup>
           <mrow>
            <mo stretchy="false">(</mo>
            <mrow>
             <mn>1</mn>
             <mo>-</mo>
             <mi>θ</mi>
            </mrow>
            <mo stretchy="false">)</mo>
           </mrow>
           <mrow>
            <mi>N</mi>
            <mi>R</mi>
           </mrow>
          </msup>
          <mo>×</mo>
          <msup>
           <mi>θ</mi>
           <mi>R</mi>
          </msup>
         </mrow>
         <msup>
          <mn>0.5</mn>
          <mrow>
           <mo stretchy="false">(</mo>
           <mrow>
            <mrow>
             <mi>N</mi>
             <mi>R</mi>
            </mrow>
            <mo>+</mo>
            <mi>R</mi>
           </mrow>
           <mo stretchy="false">)</mo>
          </mrow>
         </msup>
        </mfrac>
       </mstyle>
      </mrow>
     </mrow>
    </mtd>
   </mtr>
  </mtable>
  <annotation-xml encoding="MathML-Content">
   <matrix>
    <matrixrow>
     <apply>
      <eq></eq>
      <apply>
       <times></times>
       <ci>L</ci>
       <ci>O</ci>
       <ci>D</ci>
      </apply>
      <ci>Z</ci>
     </apply>
     <apply>
      <and></and>
      <apply>
       <eq></eq>
       <csymbol cd="latexml">absent</csymbol>
       <apply>
        <apply>
         <csymbol cd="ambiguous">subscript</csymbol>
         <log></log>
         <cn type="integer">10</cn>
        </apply>
        <apply>
         <divide></divide>
         <mtext>probability of birth sequence with a given linkage value</mtext>
         <mtext>probability of birth sequence with no linkage</mtext>
        </apply>
       </apply>
      </apply>
      <apply>
       <eq></eq>
       <share href="#.cmml">
       </share>
       <apply>
        <apply>
         <csymbol cd="ambiguous">subscript</csymbol>
         <log></log>
         <cn type="integer">10</cn>
        </apply>
        <apply>
         <divide></divide>
         <apply>
          <times></times>
          <apply>
           <csymbol cd="ambiguous">superscript</csymbol>
           <apply>
            <minus></minus>
            <cn type="integer">1</cn>
            <ci>θ</ci>
           </apply>
           <apply>
            <times></times>
            <ci>N</ci>
            <ci>R</ci>
           </apply>
          </apply>
          <apply>
           <csymbol cd="ambiguous">superscript</csymbol>
           <ci>θ</ci>
           <ci>R</ci>
          </apply>
         </apply>
         <apply>
          <csymbol cd="ambiguous">superscript</csymbol>
          <cn type="float">0.5</cn>
          <apply>
           <plus></plus>
           <apply>
            <times></times>
            <ci>N</ci>
            <ci>R</ci>
           </apply>
           <ci>R</ci>
          </apply>
         </apply>
        </apply>
       </apply>
      </apply>
     </apply>
    </matrixrow>
   </matrix>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \begin{aligned}\displaystyle LOD=Z&\displaystyle=\log_{10}\frac{\mbox{%
probability of birth sequence with a given linkage value}}{\mbox{probability %
of birth sequence with no linkage}}=\log_{10}\frac{(1-\theta)^{NR}\times\theta%
^{R}}{0.5^{(NR+R)}}\end{aligned}
  </annotation>
 </semantics>
</math>

</p>

<p>NR denotes the number of non-recombinant offspring, and R denotes the number of recombinant offspring. The reason 0.5 is used in the denominator is that any alleles that are completely unlinked (e.g. alleles on separate chromosomes) have a 50% chance of recombination, due to independent assortment. 'θ' is the recombinant fraction, i.e. the fraction of births in which recombination has happened between the studied genetic marker and the putative gene associated with the disease. Thus, it is equal to R / (NR + R)</p>

<p>By convention, a LOD score greater than 3.0 is considered evidence for linkage, as it indicates 1000 to 1 odds that the linkage being observed did not occur by chance. On the other hand, a LOD score less than -2.0 is considered evidence to exclude linkage. Although it is very unlikely that a LOD score of 3 would be obtained from a single pedigree, the mathematical properties of the test allow data from a number of pedigrees to be combined by summing their LOD scores. However, this traditional cut-off of LOD score &gt; +3 is arbitrary, and in certain types of linkage studies, such as analyses of complex genetic traits with hundreds of markers, this criterion should probably be modified to a higher cut-off (for example, by applying a <a href="Bonferroni_correction" title="wikilink">Bonferroni correction</a>).</p>
<h2 id="recombination-frequency">Recombination frequency</h2>

<p>Recombination frequency is a measure of genetic linkage and is used in the creation of a genetic linkage map. Recombination frequency (<em>θ</em>) is the frequency with which a single <a href="chromosomal_crossover" title="wikilink">chromosomal crossover</a> will take place between two <a href="gene" title="wikilink">genes</a> during <a class="uri" href="meiosis" title="wikilink">meiosis</a>. A <a class="uri" href="centimorgan" title="wikilink">centimorgan</a> (cM) is a unit that describes a recombination frequency of 1%. In this way we can measure the genetic distance between two loci, based upon their recombination frequency. This is a good estimate of the real distance. Double crossovers would turn into no recombination. In this case we cannot tell if crossovers took place. If the loci we're analysing are very close (less than 7 cM) a double crossover is very unlikely. When distances become higher, the likelihood of a double crossover increases. As the likelihood of a double crossover increases we systematically underestimate the genetic distance between two loci.</p>

<p>During meiosis, chromosomes assort randomly into <a href="gamete" title="wikilink">gametes</a>, such that the segregation of <a href="allele" title="wikilink">alleles</a> of one gene is independent of alleles of another gene. This is stated in <a href="Mendelian_inheritance" title="wikilink">Mendel's Second Law</a> and is known as <strong>the law of independent assortment</strong>. The law of independent assortment always holds true for genes that are located on different chromosomes, but for genes that are on the same chromosome, it does not always hold true.</p>

<p>As an example of independent assortment, consider the crossing of the pure-bred <a class="uri" href="homozygote" title="wikilink">homozygote</a> parental strain with <a class="uri" href="genotype" title="wikilink">genotype</a> <em>AABB</em> with a different pure-bred strain with genotype <em>aabb</em>. A and a and B and b represent the alleles of genes A and B. Crossing these homozygous parental strains will result in F1 generation offspring that are double <a class="uri" href="heterozygotes" title="wikilink">heterozygotes</a> with genotype AaBb. The F1 offspring AaBb produces gametes that are <em>AB</em>, <em>Ab</em>, <em>aB</em>, and <em>ab</em> with equal frequencies (25%) because the alleles of gene A assort independently of the alleles for gene B during meiosis. Note that 2 of the 4 gametes (50%)—<em>Ab</em> and <em>aB</em>—were not present in the parental generation. These gametes represent <strong>recombinant gametes</strong>. Recombinant gametes are those gametes that differ from both of the <a class="uri" href="haploid" title="wikilink">haploid</a> gametes that made up the original <a class="uri" href="diploid" title="wikilink">diploid</a> cell. In this example, the recombination frequency is 50% since 2 of the 4 gametes were recombinant gametes.</p>

<p>The recombination frequency will be 50% when two genes are located on different <a class="uri" href="chromosomes" title="wikilink">chromosomes</a> or when they are widely separated on the same chromosome. This is a consequence of independent assortment.</p>

<p>When two genes are close together on the same chromosome, they do not assort independently and are said to be linked. Whereas genes located on different chromosomes assort independently and have a recombination frequency of 50%, linked genes have a recombination frequency that is less than 50%.</p>

<p>As an example of linkage, consider the classic experiment by <a href="William_Bateson" title="wikilink">William Bateson</a> and <a href="Reginald_Punnett" title="wikilink">Reginald Punnett</a>. They were interested in trait inheritance in the sweet pea and were studying two genes—the gene for flower colour (<em>P</em>, purple, and <em>p</em>, red) and the gene affecting the shape of pollen grains (<em>L</em>, long, and <em>l</em>, round). They crossed the pure lines <em>PPLL</em> and <em>ppll</em> and then self-crossed the resulting <em>PpLl</em> lines. According to <a href="Mendelian_inheritance" title="wikilink">Mendelian genetics</a>, the expected <a href="phenotype" title="wikilink">phenotypes</a> would occur in a 9:3:3:1 ratio of PL:Pl:pL:pl. To their surprise, they observed an increased frequency of PL and pl and a decreased frequency of Pl and pL (see table below).</p>
<table>
<tbody>
<tr class="odd">
<td style="text-align: left;">
<p><strong>Bateson and Punnett experiment</strong></p></td>
</tr>
<tr class="even">
<td style="text-align: left;">
<p>Phenotype and genotype</p></td>
</tr>
<tr class="odd">
<td style="text-align: left;">
<p>Purple, long (<em>P_L_</em>)</p></td>
</tr>
<tr class="even">
<td style="text-align: left;">
<p>Purple, round (<em>P_ll</em>)</p></td>
</tr>
<tr class="odd">
<td style="text-align: left;">
<p>Red, long (<em>ppL_</em>)</p></td>
</tr>
<tr class="even">
<td style="text-align: left;">
<p>Red, round (<em>ppll</em>)</p></td>
</tr>
</tbody>
</table>

<p>Their experiment revealed <strong>linkage</strong> between the <em>P</em> and <em>L</em> alleles and the <em>p</em> and <em>l</em> alleles. The frequency of <em>P</em> occurring together with <em>L</em> and with <em>p</em> occurring together with <em>l</em> is greater than that of the recombinant <em>Pl</em> and <em>pL</em>. The recombination frequency is more difficult to compute in an F2 cross than a backcross,<a class="footnoteRef" href="#fn4" id="fnref4"><sup>4</sup></a> but the lack of fit between observed and expected numbers of progeny in the above table indicate it is less than 50%.</p>

<p>The progeny in this case received two dominant alleles linked on one chromosome (referred to as <strong>coupling</strong> or <strong>cis arrangement</strong>). However, after crossover, some progeny could have received one parental chromosome with a dominant allele for one trait (e.g. Purple) linked to a recessive allele for a second trait (e.g. round) with the opposite being true for the other parental chromosome (e.g. red and Long). This is referred to as <strong>repulsion</strong> or a <strong>trans arrangement</strong>. The <a class="uri" href="phenotype" title="wikilink">phenotype</a> here would still be purple and long but a test cross of this individual with the recessive parent would produce progeny with much greater proportion of the two crossover phenotypes. While such a problem may not seem likely from this example, unfavorable repulsion linkages do appear when breeding for disease resistance in some crops.</p>

<p>The two possible arrangements, cis and trans, of alleles in a double heterozygote are referred to as <a href="gametic_phase" title="wikilink">gametic phases</a>, and <em>phasing</em> is the process of determining which of the two is present in a given individual.</p>

<p>When two genes are located on the same chromosome, the chance of a <a href="chromosomal_crossover" title="wikilink">crossover</a> producing recombination between the genes is related to the distance between the two genes. Thus, the use of recombination frequencies has been used to develop <strong>linkage maps</strong> or <strong>genetic maps</strong>.</p>

<p>However, it is important to note that recombination frequency tends to underestimate the distance between two linked genes. This is because as the two genes are located farther apart, the chance of double or even number of crossovers between them also increases. Double or even number of crossovers between the two genes results in them being cosegregated to the same gamete, yielding a parental progeny instead of the expected recombinant progeny. As mentioned above, the Kosambi and Haldane transformations attempt to correct for multiple crossovers <a class="footnoteRef" href="#fn5" id="fnref5"><sup>5</sup></a> .<a class="footnoteRef" href="#fn6" id="fnref6"><sup>6</sup></a></p>
<h2 id="variation-of-recombination-frequency">Variation of recombination frequency</h2>

<p>While recombination of chromosomes is an essential process during meiosis, there is a large range of frequency of cross overs across organisms and within species. Sexually dimorphic rates of recombination are termed heterochiasmy, and is observed more often than a common rate between male and females. In mammals, females often have a higher rate of recombination compared to males. It is theorized that there are unique selections acting or meiotic drivers which influence the difference in rates. The difference in rates may also reflect the vastly different environments and conditions of meiosis in oogenesis and spermatogensis.</p>
<h2 id="meiosis-indicators">Meiosis indicators</h2>

<p>With very large pedigrees or with very dense genetic marker data, such as from whole-genome sequencing, it is possible to precisely locate recombinations. With this type of genetic analysis, a meiosis indicator is assigned to each position of the genome for each <a class="uri" href="meiosis" title="wikilink">meiosis</a> in a pedigree. The indicator indicates which copy of the parental chromosome contributes to the transmitted gamete at that position. For example, if the allele from the 'first' copy of the parental chromosome is transmitted, a '0' might be assigned to that meiosis. If the allele from the 'second' copy of the parental chromosome is transmitted, a '1' would be assigned to that meiosis. The two alleles in the parent came, one each, from two grandparents. These indicators are then used to determine identical-by-descent (IBD) states or inheritance states, which are in turn used to identify genes responsible for diseases .</p>
<h2 id="see-also">See also</h2>
<ul>
<li><a class="uri" href="Centimorgan" title="wikilink">Centimorgan</a></li>
<li><a href="Genetic_association" title="wikilink">Genetic association</a></li>
<li><a href="Genetic_epidemiology" title="wikilink">Genetic epidemiology</a></li>
<li><a href="Genome-wide_association_study" title="wikilink">Genome-wide association study</a></li>
<li><a href="Identity_by_descent" title="wikilink">Identity by descent</a></li>
<li><a href="Lander–Green_algorithm" title="wikilink">Lander–Green algorithm</a></li>
</ul>
<h2 id="references">References</h2>
<h2 id="notes">Notes</h2>
<ul>
<li></li>
<li></li>
</ul>
<h2 id="external-links">External links</h2>
<ul>
<li><a href="http://www.hopkinsmedicine.org/epigen/what_is_genetic_mapping.htm">Genetic Mapping</a></li>
</ul>

<p>"</p>

<p><a href="Category:Classical_genetics" title="wikilink">Category:Classical genetics</a></p>
<section class="footnotes">
<hr/>
<ol>
<li id="fn1"><a href="http://www.nature.com/scitable/topicpage/discovery-and-types-of-genetic-linkage-500">Discovery and Types of Genetic Linkage</a>, from Scitable<a href="#fnref1">↩</a></li>
<li id="fn2">William Bateson, E. R. Saunders, R. C. Punnett (1904) "Report II. Experimental studies in the physiology of heredity" Reports to the Evolution Committee of the Royal Society. <a class="uri" href="http://archive.org/details/RoyalSociety.ReportsToTheEvolutionCommittee.ReportIi.Experimental">http://archive.org/details/RoyalSociety.ReportsToTheEvolutionCommittee.ReportIi.Experimental</a><a href="#fnref2">↩</a></li>
<li id="fn3"><a href="#fnref3">↩</a></li>
<li id="fn4">Fisher, R.A., and Balmukand, B. 1928. The estimation of linkage from the offspring of selfed heterozygotes. Journal of Genetics 20:79-92.<a href="#fnref4">↩</a></li>
<li id="fn5">[<a class="uri" href="http://www.ncbi.nlm.nih.gov/bookshelf/br.fcgi?book=iga">http://www.ncbi.nlm.nih.gov/bookshelf/br.fcgi?book=iga</a>∂=A1110#A1115 Derivation of mapping function], from <em>Introduction to Genetic Analysis.</em> Griffiths, A. J. F.; Miller, J. H.; Suzuki, D. T.; Lewontin, R. C.; Gelbart, W. M. New York: W. H. Freeman &amp; Co.; 1999<a href="#fnref5">↩</a></li>
<li id="fn6">[<a class="uri" href="http://www.ncbi.nlm.nih.gov/bookshelf/br.fcgi?book=iga">http://www.ncbi.nlm.nih.gov/bookshelf/br.fcgi?book=iga</a>∂=A1110&amp;rendertype;=figure&amp;id;=A1117 Graph of mapping function] from compared to idealized 1-1 equivalence of recombination frequency percentage (RF%) to map units.<a href="#fnref6">↩</a></li>
</ol>
</section>
</body>
</html>
