<html lang="en">
<head>
<meta charset="utf-8"/>
<title offset="381">Genetic purging</title>
<script src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_SVG.js" type="text/javascript">
</script>
</head>
<body>
<h1>Genetic purging</h1>
<hr/>

<p><strong>Genetic purging</strong> is the reduction of the frequency of a deleterious <a class="uri" href="allele" title="wikilink">allele</a> occured by an increased efficiency of natural selction that is prompted by <a class="uri" href="inbreeding" title="wikilink">inbreeding</a>.<a class="footnoteRef" href="#fn1" id="fnref1"><sup>1</sup></a></p>

<p>Purging occurs because many deleterious alleles only express all their harmful effect in homozygosis. During inbreeding, as related individuals mate, they produce offspring that are more likely to be <a class="uri" href="homozygous" title="wikilink">homozygous</a>, so that these deleterious alleles appear more often in homozygous individuals that are less fit and that pass fewer copies of their genes to future generations. This allows <a href="natural_selection" title="wikilink">natural selection</a> to purge the deleterious alleles.</p>

<p>Purging reduces both the overall number of recessive deleterious alleles and the decline of mean fitness caused by inbreeding (the <a href="inbreeding_depression" title="wikilink">inbreeding depression</a> for fitness).</p>

<p>The term “purge” is sometimes used for selection against deleterious alleles in a general way. It would avoid ambiguity to use “purifying selection” in that general context, and to reserve purging to its more strict meaning defined above.</p>

<p>The mechanism of purging and its main consequences are explained below in more detail.</p>
<h2 id="the-mechanism">The mechanism</h2>

<p>Deleterious alleles segregating in populations of <a class="uri" href="diploid" title="wikilink">diploid</a> organisms have a remarkable trend to be, at least, partially recessive. This means that, when they occur in homozygosis (double dosage), they reduce <a href="Fitness_(biology)" title="wikilink">fitness</a> by more than twice than when they occur in heterozygosis (single dosage). In other words, part of their potential deleterious effect is hidden in heterozygosis but expressed in homozygosis, so that selection is more efficient against them when they occur in homozygosis. Since inbreeding increases the probability of being homozygous, it increases the fraction of the potential deleterious effect that is expressed and, therefore, exposed to selection. This causes some increase in the selective pressure against (partially) recessive deleterious alleles, which is known as purging. Of course, it also causes some reduction in fitness, which is known as <a href="inbreeding_depression" title="wikilink">inbreeding depression</a>.</p>

<p>Purging can reduce the average frequency of deleterious alleles across the genome below the value expected in a non-inbred population.<a class="footnoteRef" href="#fn2" id="fnref2"><sup>2</sup></a> Although this reduction usually does not compensate for all the negative effects of inbreeding,<a class="footnoteRef" href="#fn3" id="fnref3"><sup>3</sup></a> it has several beneficial consequences for fitness. A consequence is the reduction of the so-called inbreeding load. This means that, after purging, further inbreeding is expected to be less harmful. But the most immediate consequence is the reduction of the actual inbreeding depression of fitness: due to purging, mean fitness declines less than would be expected just from inbreeding and, after some initial decline, it can even rebound up to almost its value before inbreeding.<a class="footnoteRef" href="#fn4" id="fnref4"><sup>4</sup></a></p>
<h2 id="the-joint-effect-of-inbreeding-and-purging-on-fitness">The joint effect of inbreeding and purging on fitness</h2>

<p>Accounting for purging when predicting inbreeding depression is important in evolutionary genetics, because the fitness decline caused by inbreeding can be determinant in the evolution of <a class="uri" href="diploidy" title="wikilink">diploidy</a>, sexual reproduction and other main biological features. It is also important in animal breeding and, of course, in <a href="conservation_genetics" title="wikilink">conservation genetics</a>, because inbreeding depression may be a relevant factor determining the extinction risk of endangered populations, and because conservation programs can allow some breeding handling in order to control inbreeding.<a class="footnoteRef" href="#fn5" id="fnref5"><sup>5</sup></a></p>

<p>In brief, Due to purging, inbreeding depression is not proportional to the standard measure of inbreeding (Wright's inbreeding coefficient <mtpl></mtpl>), since this measure only applies to <a href="Neutral_evolution" title="wikilink">neutral</a> alleles. Instead, fitness decline is proportional to “purged inbreeding” <mtpl></mtpl>, which gives the probability of being homozygous for deleterious alleles due to inbreeding, taking into account how they are being purged.</p>

<p>Purging reduces inbreeding depression in two ways: first, it slows its progress; second, it reduces the overall inbreeding depression expected in the long term. The slower the progress of inbreeding, the more efficient is purging.</p>
<h3 id="a-more-detailed-explanation">A more detailed explanation</h3>

<p>In the absence of natural selection, mean fitness would be expected to decline exponentially as inbreeding increases, where inbreeding is measured using Wright’s inbreeding coefficient <mtpl></mtpl><a class="footnoteRef" href="#fn6" id="fnref6"><sup>6</sup></a> (the reason why decline is exponential on <mtpl></mtpl> instead of linear is just that fitness is usually considered a multiplicative trait). The rate at which fitness declines as <mtpl></mtpl> increases (the inbreeding depression rate <em>δ</em>) depends on the frequencies and deleterious effects of the alleles present in the population before inbreeding.</p>

<p>The above coefficient <mtpl></mtpl> is the standard measure of inbreeding, and gives the probability that, at any given neutral locus, an individual has inherited two copies of a same gene of a common ancestor (i.e. the probability of being homozygous “by descent”). In simple conditions, <mtpl></mtpl> can be easily computed in terms of population size or of genealogical information. <mtpl></mtpl> is often denoted using lowercase (<mtpl></mtpl>), but should not be confused with the coancestry coefficient.</p>

<p>However, the above prediction for the fitness decline rarely applies, since it was derived assuming no selection, and fitness is precisely the target trait of <a href="natural_selection" title="wikilink">natural selection</a>. Thus, Wright’s inbreeding coefficient <mtpl></mtpl> for neutral loci does not apply to deleterious alleles, unless inbreeding increases so fast that the change in gene frequency is governed just by random sampling (i.e., by <a href="genetic_drift" title="wikilink">genetic drift</a>).</p>

<p>Therefore, the decline of fitness should be predicted using, instead of the standard inbreeding coefficient <mtpl></mtpl>, a “purged inbreeding coefficient” (<mtpl></mtpl>) that gives the probability of being homozygous by descent for (partially) recessive deleterious alleles, taking into account how their frequency is reduced by purging.<a class="footnoteRef" href="#fn7" id="fnref7"><sup>7</sup></a> Due to purging, fitness declines at the same rate <mtpl></mtpl> than in the absence of selection, but as a function of <mtpl></mtpl> instead of <mtpl></mtpl>.</p>

<p>This purged inbreeding coefficient <mtpl></mtpl> can also be computed, to a good approximation, using simple expressions in terms of the population size or of the genealogy of individuals (see BOX 1). However this requires some information on the magnitude of the deleterious effects that are hidden in the heterozygous condition but become expressed in homozygosis. The larger this magnitude, denoted purging coefficient <em>d</em>, the more efficient is purging.</p>

<p>An interesting property of purging is that, during inbreeding, while <mtpl></mtpl> increases approaching a final value <mtpl> 1}}</mtpl>, <mtpl></mtpl> can approach a much smaller final value. Hence, it is not just that purging slows the fitness decline, but also that it reduces the overall fitness loss produced by inbreeding in the long term. This is illustrated in BOX 2 for the extreme case of inbreeding depression caused by recessive lethals, which are alleles that cause death before reproduction but only when they occur in homozygosis. Purging is less effective against mildly deleterious alleles than against lethal ones but, in general, the slower is the increase of inbreeding <mtpl></mtpl>, the smaller becomes the final value of the purged inbreeding coefficient <mtpl></mtpl> and, therefore, the final reduction in fitness. This implies that, if inbreeding progresses slowly enough, no relevant inbreeding depression is expected in the long term. In conservation genetics, it would be very useful to ascertain the maximum rate of increase of inbreeding that allows for such efficient purging.</p>
<h2 id="box-1-predictive-equations-when-inbreeding-is-due-to-small-population-size">BOX 1: Predictive equations when inbreeding is due to small population size</h2>

<p>Consider a large non-inbred population with mean fitness <mtpl></mtpl>. Then, the size of the population reduces to a new smaller value <mtpl></mtpl> (in fact, the <a href="effective_population_size" title="wikilink">effective population size</a> should be used here), leading to a progressive increase of inbreeding.</p>

<p>Then inbreeding depression occurs at a rate <mtpl></mtpl>, due to (partially) recessive deleterious alleles that were present at low frequencies at different loci. This means that, in the absence of selection, the expected value for mean fitness after <mtpl></mtpl> generations of inbreeding, would be:</p>

<p>

<math display="inline" id="Genetic_purging:0">
 <semantics>
  <mrow>
   <msub>
    <mi>W</mi>
    <mi>t</mi>
   </msub>
   <mo>=</mo>
   <mrow>
    <mi>W</mi>
    <msup>
     <mi>e</mi>
     <mrow>
      <mo>-</mo>
      <mrow>
       <mi>δ</mi>
       <msub>
        <mi>F</mi>
        <mi>t</mi>
       </msub>
      </mrow>
     </mrow>
    </msup>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <eq></eq>
    <apply>
     <csymbol cd="ambiguous">subscript</csymbol>
     <ci>W</ci>
     <ci>t</ci>
    </apply>
    <apply>
     <times></times>
     <ci>W</ci>
     <apply>
      <csymbol cd="ambiguous">superscript</csymbol>
      <ci>e</ci>
      <apply>
       <minus></minus>
       <apply>
        <times></times>
        <ci>δ</ci>
        <apply>
         <csymbol cd="ambiguous">subscript</csymbol>
         <ci>F</ci>
         <ci>t</ci>
        </apply>
       </apply>
      </apply>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   W_{t}=We^{-\delta F_{t}}
  </annotation>
 </semantics>
</math>

</p>

<p>where 

<math display="inline" id="Genetic_purging:1">
 <semantics>
  <msub>
   <mi>F</mi>
   <mi>t</mi>
  </msub>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <csymbol cd="ambiguous">subscript</csymbol>
    <ci>F</ci>
    <ci>t</ci>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   F_{t}
  </annotation>
 </semantics>
</math>

 is the population mean for Wright’s inbreeding coefficient after <mtpl></mtpl> generations of inbreeding.<a class="footnoteRef" href="#fn8" id="fnref8"><sup>8</sup></a></p>

<p>However, since selection operates upon fitness, mean fitness should be predicted taking into account both inbreeding and purging, as</p>

<p>

<math display="inline" id="Genetic_purging:2">
 <semantics>
  <mrow>
   <msub>
    <mi>W</mi>
    <mi>t</mi>
   </msub>
   <mo>=</mo>
   <mrow>
    <mi>W</mi>
    <msup>
     <mi>e</mi>
     <mrow>
      <mo>-</mo>
      <mrow>
       <mi>δ</mi>
       <msub>
        <mi>g</mi>
        <mi>t</mi>
       </msub>
      </mrow>
     </mrow>
    </msup>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <eq></eq>
    <apply>
     <csymbol cd="ambiguous">subscript</csymbol>
     <ci>W</ci>
     <ci>t</ci>
    </apply>
    <apply>
     <times></times>
     <ci>W</ci>
     <apply>
      <csymbol cd="ambiguous">superscript</csymbol>
      <ci>e</ci>
      <apply>
       <minus></minus>
       <apply>
        <times></times>
        <ci>δ</ci>
        <apply>
         <csymbol cd="ambiguous">subscript</csymbol>
         <ci>g</ci>
         <ci>t</ci>
        </apply>
       </apply>
      </apply>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   W_{t}=We^{-\delta g_{t}}
  </annotation>
 </semantics>
</math>

</p>

<p>In the above equation, 

<math display="inline" id="Genetic_purging:3">
 <semantics>
  <msub>
   <mi>g</mi>
   <mi>t</mi>
  </msub>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <csymbol cd="ambiguous">subscript</csymbol>
    <ci>g</ci>
    <ci>t</ci>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   g_{t}
  </annotation>
 </semantics>
</math>


 is the average “purged inbreeding coefficient” after <mtpl></mtpl> generations of inbreeding.<a class="footnoteRef" href="#fn9" id="fnref9"><sup>9</sup></a> It depends upon the “purging coefficient” <mtpl></mtpl>, which represents the deleterious effects that are hidden in heterozygosis but exposed in homozygosis.</p>

<p>The average “purged inbreeding coefficient” can be approximated using the recurrent expression</p>

<p>

<math display="inline" id="Genetic_purging:4">
 <semantics>
  <mrow>
   <msub>
    <mi>g</mi>
    <mi>t</mi>
   </msub>
   <mo>=</mo>
   <mrow>
    <mrow>
     <mo>[</mo>
     <mrow>
      <mrow>
       <mrow>
        <mo>(</mo>
        <mrow>
         <mn>1</mn>
         <mo>-</mo>
         <mfrac>
          <mn>1</mn>
          <mrow>
           <mn>2</mn>
           <mi>N</mi>
          </mrow>
         </mfrac>
        </mrow>
        <mo>)</mo>
       </mrow>
       <msub>
        <mi>g</mi>
        <mrow>
         <mi>t</mi>
         <mo>-</mo>
         <mn>1</mn>
        </mrow>
       </msub>
      </mrow>
      <mo>+</mo>
      <mfrac>
       <mn>1</mn>
       <mrow>
        <mn>2</mn>
        <mi>N</mi>
       </mrow>
      </mfrac>
     </mrow>
     <mo>]</mo>
    </mrow>
    <mrow>
     <mo>[</mo>
     <mrow>
      <mn>1</mn>
      <mo>-</mo>
      <mrow>
       <mn>2</mn>
       <mi>d</mi>
       <msub>
        <mi>F</mi>
        <mrow>
         <mi>t</mi>
         <mo>-</mo>
         <mn>1</mn>
        </mrow>
       </msub>
      </mrow>
     </mrow>
     <mo>]</mo>
    </mrow>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <eq></eq>
    <apply>
     <csymbol cd="ambiguous">subscript</csymbol>
     <ci>g</ci>
     <ci>t</ci>
    </apply>
    <apply>
     <times></times>
     <apply>
      <csymbol cd="latexml">delimited-[]</csymbol>
      <apply>
       <plus></plus>
       <apply>
        <times></times>
        <apply>
         <minus></minus>
         <cn type="integer">1</cn>
         <apply>
          <divide></divide>
          <cn type="integer">1</cn>
          <apply>
           <times></times>
           <cn type="integer">2</cn>
           <ci>N</ci>
          </apply>
         </apply>
        </apply>
        <apply>
         <csymbol cd="ambiguous">subscript</csymbol>
         <ci>g</ci>
         <apply>
          <minus></minus>
          <ci>t</ci>
          <cn type="integer">1</cn>
         </apply>
        </apply>
       </apply>
       <apply>
        <divide></divide>
        <cn type="integer">1</cn>
        <apply>
         <times></times>
         <cn type="integer">2</cn>
         <ci>N</ci>
        </apply>
       </apply>
      </apply>
     </apply>
     <apply>
      <csymbol cd="latexml">delimited-[]</csymbol>
      <apply>
       <minus></minus>
       <cn type="integer">1</cn>
       <apply>
        <times></times>
        <cn type="integer">2</cn>
        <ci>d</ci>
        <apply>
         <csymbol cd="ambiguous">subscript</csymbol>
         <ci>F</ci>
         <apply>
          <minus></minus>
          <ci>t</ci>
          <cn type="integer">1</cn>
         </apply>
        </apply>
       </apply>
      </apply>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   g_{t}=\left[\left(1-\frac{1}{2N}\right)g_{t-1}+\frac{1}{2N}\right]\left[1-2dF_%
{t-1}\right]
  </annotation>
 </semantics>
</math>

</p>

<p>There are also predictive equations to be used with genealogical information.</p>
<h2 id="box-2-the-example-of-inbreeding-depression-due-to-recessive-lethals">BOX 2: The example of inbreeding depression due to recessive lethals</h2>

<p><embed src="Fitness decline from lethals vs generations of inbreeding.tif" title="fig:Average population fitness () against number of generations () of maintenance with population size N=10 when, in the original population, individuals carried on the average one rare recessive lethal. Evolution of  expected only from inbreeding (red line) or from inbreeding and purging (blue line)."></embed> As an example of genetic purging, consider a large population where there are recessive lethal alleles segregating at very low frequency in many loci, so that each individual carries on the average one of these alleles. Although about 63% of the individuals carry at least one of these lethal alleles, almost none carry two copies of the same lethal. Therefore, since lethals are considered completely recessive (i.e., they are harmless in heterozygosis), they cause almost no deaths. Now assume that population size reduces to a small value (say <em>N</em>=10), and remains that small for many generations. As inbreeding increases, the probability of being homozygous for one (or more) of these lethal alleles also increases, causing fitness to decline. However, as those lethals begin to occur in homozygosis, natural selection begins purging them. The figure to the right gives the expected decline of fitness against the number of generations, taking into account just the increase in inbreeding <mtpl></mtpl> (red line), or both inbreeding and purging (blue line, computed using the purged inbreeding coefficient <mtpl></mtpl>). This example shows that purging can be very efficient preventing inbreeding depression. However, for non-lethal deleterious alleles, the efficiency of purging would be smaller, and it can require larger populations to overcome genetic drift.</p>
<h2 id="evidence-and-problems">Evidence and problems</h2>

<p>When a previously stable population undergoes inbreeding, if nothing else changes, natural selection should consist mainly of purging. The joint consequences of inbreeding and purging on fitness vary depending on many factors: the previous history of the population, the rate of increase of inbreeding, the harshness of the environment or of the competitive conditions, etc. The effects of purging were first noted by Darwin<a class="footnoteRef" href="#fn10" id="fnref10"><sup>10</sup></a> in plants, and have been detected in laboratory experiments and in vertebrate populations undergoing inbreeding in zoos or in the wild, as well as in humans.<a class="footnoteRef" href="#fn11" id="fnref11"><sup>11</sup></a> The detection of purging is often obscured by many factors, but there is consistent evidence that, in agreement with the predictions explained above, slow inbreeding results in more efficient purging, so that a given inbreeding F leads to less threat to population viability if it has been produced more slowly.<a class="footnoteRef" href="#fn12" id="fnref12"><sup>12</sup></a></p>

<p>Nevertheless, in practical situations, the genetic change in fitness also depends on many other factors, besides inbreeding and purging. For example, adaptation to changing environmental conditions often causes relevant genetic changes during inbreeding. Furthermore, if inbreeding is due to a reduction in population size, selection against new deleterious <a href="mutation" title="wikilink">mutations</a> can become less efficient, and this can induce additional fitness decline in the medium-long term.</p>

<p>In addition, part of the inbreeding depression could not be due to deleterious alleles, but to an intrinsic advantage of being heterozygous compared to being homozygous for any available allele, which is known as overdominance. Inbreeding depression caused by overdominance cannot be purged, but seems to be a minor cause of overall inbreeding depression, although its actual importance is still a matter of debate.<a class="footnoteRef" href="#fn13" id="fnref13"><sup>13</sup></a></p>

<p>Therefore, predicting the actual evolution of fitness during inbreeding is highly elusive. However, the component of fitness decline expected from inbreeding and purging on deleterious alleles could be predicted using <mtpl></mtpl>.</p>
<h2 id="references">References</h2>

<p>"</p>

<p><a href="Category:Population_genetics" title="wikilink"> </a> <a class="uri" href="Category:Breeding" title="wikilink">Category:Breeding</a> <a href="Category:Evolutionary_biology" title="wikilink"> </a> <a class="uri" href="Category:Conservation" title="wikilink">Category:Conservation</a> <a href="Category:Applied_genetics" title="wikilink">Category:Applied genetics</a> <a href="Category:Rare_breed_conservation" title="wikilink">Category:Rare breed conservation</a></p>
<section class="footnotes">
<hr/>
<ol>
<li id="fn1"><a href="#fnref1">↩</a></li>
<li id="fn2"><a href="#fnref2">↩</a></li>
<li id="fn3"><a href="#fnref3">↩</a></li>
<li id="fn4"><a href="#fnref4">↩</a></li>
<li id="fn5"><a href="#fnref5">↩</a></li>
<li id="fn6"><a href="#fnref6">↩</a></li>
<li id="fn7"></li>
<li id="fn8"></li>
<li id="fn9"></li>
<li id="fn10"><a href="#fnref10">↩</a></li>
<li id="fn11"><a href="#fnref11">↩</a></li>
<li id="fn12"><a href="#fnref12">↩</a></li>
<li id="fn13"><a href="#fnref13">↩</a></li>
</ol>
</section>
</body>
</html>
