<html lang="en">
<head>
<meta charset="utf-8"/>
<title offset="1178">Genetic distance</title>
<script src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_SVG.js" type="text/javascript">
</script>
</head>
<body>
<h1>Genetic distance</h1>
<hr/>

<p><strong>Genetic distance</strong> is a measure of the <a href="genetics" title="wikilink">genetic</a> divergence between <a class="uri" href="species" title="wikilink">species</a> or between <a href="population#genetics" title="wikilink">populations</a> within a species.<a class="footnoteRef" href="#fn1" id="fnref1"><sup>1</sup></a> Populations with many similar genes have small genetic distances. This indicates that they are closely related and have a recent <a href="common_ancestor" title="wikilink">common ancestor</a>.</p>

<p>Genetic distance is useful for reconstructing the history of populations. For example, evidence from genetic distance suggests that African and Eurasian people diverged about 100,000 years ago.<a class="footnoteRef" href="#fn2" id="fnref2"><sup>2</sup></a> Genetic distance is also used for understanding the origin of <a class="uri" href="biodiversity" title="wikilink">biodiversity</a>. For example, the genetic distances between different breeds of domesticated animals are often investigated in order to determine which breeds should be protected to maintain genetic diversity.<a class="footnoteRef" href="#fn3" id="fnref3"><sup>3</sup></a></p>
<h2 id="biological-foundation">Biological foundation</h2>

<p>In the genome of an <a class="uri" href="organism" title="wikilink">organism</a>, each <a class="uri" href="gene" title="wikilink">gene</a> is located at a specific place called the <a href="locus_(genetics)" title="wikilink">locus</a> for that gene. Allelic variations at these loci cause phenotypic variation within species (e.g. hair colour, eye colour). However, most <a class="uri" href="alleles" title="wikilink">alleles</a> do not have an observable impact on the phenotype. Within a population new alleles generated by mutation either die out or spread throughout the population. When a population is split into different isolated populations (by either geographical or ecological factors), mutations that occur after the split will be present only in the isolated population. Random fluctuation of allele frequencies also produces genetic differentiation between populations. This process is known as <a href="genetic_drift" title="wikilink">genetic drift</a>. By examining the differences between <a href="allele_frequencies" title="wikilink">allele frequencies</a> between the populations and computing genetic distance, we can estimate how long ago the two populations were separated.<a class="footnoteRef" href="#fn4" id="fnref4"><sup>4</sup></a></p>
<h2 id="measures-of-genetic-distance">Measures of genetic distance</h2>

<p>Although it is simple to define genetic distance as a measure of genetic divergence, there are several different statistical measures that have been proposed. This has happened because different authors considered different evolutionary models. The most commonly used are Nei's genetic distance,<a class="footnoteRef" href="#fn5" id="fnref5"><sup>5</sup></a> Cavalli-Sforza and Edwards measure,<a class="footnoteRef" href="#fn6" id="fnref6"><sup>6</sup></a> and Reynolds, Weir and Cockerham's genetic distance,<a class="footnoteRef" href="#fn7" id="fnref7"><sup>7</sup></a> listed below.</p>

<p>In all the formulae in this section, 

<math display="inline" id="Genetic_distance:0">
 <semantics>
  <mi>X</mi>
  <annotation-xml encoding="MathML-Content">
   <ci>X</ci>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   X
  </annotation>
 </semantics>
</math>

 and 

<math display="inline" id="Genetic_distance:1">
 <semantics>
  <mi>Y</mi>
  <annotation-xml encoding="MathML-Content">
   <ci>Y</ci>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   Y
  </annotation>
 </semantics>
</math>

 represent two different populations for which 

<math display="inline" id="Genetic_distance:2">
 <semantics>
  <mi>L</mi>
  <annotation-xml encoding="MathML-Content">
   <ci>L</ci>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   L
  </annotation>
 </semantics>
</math>

 loci have been studied. Let 

<math display="inline" id="Genetic_distance:3">
 <semantics>
  <msub>
   <mi>X</mi>
   <mi>u</mi>
  </msub>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <csymbol cd="ambiguous">subscript</csymbol>
    <ci>X</ci>
    <ci>u</ci>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   X_{u}
  </annotation>
 </semantics>
</math>

 represent the 

<math display="inline" id="Genetic_distance:4">
 <semantics>
  <mi>u</mi>
  <annotation-xml encoding="MathML-Content">
   <ci>u</ci>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   u
  </annotation>
 </semantics>
</math>

<sup>th</sup> allele at the 

<math display="inline" id="Genetic_distance:5">
 <semantics>
  <mi>l</mi>
  <annotation-xml encoding="MathML-Content">
   <ci>l</ci>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   l
  </annotation>
 </semantics>
</math>

<sup>th</sup> locus.</p>
<h3 id="neis-standard-genetic-distance">Nei's standard genetic distance</h3>

<p>In 1972, <a href="Masatoshi_Nei" title="wikilink">Masatoshi Nei</a> published what came to be known as Nei's standard genetic distance. This distance has the nice property that if the rate of genetic change (amino acid substitution) is constant per year or generation then Nei's standard genetic distance (<em>D</em>) increases in proportion to divergence time. This measure assumes that genetic differences are caused by <a class="uri" href="mutation" title="wikilink">mutation</a> and <a href="genetic_drift" title="wikilink">genetic drift</a>.<a class="footnoteRef" href="#fn8" id="fnref8"><sup>8</sup></a></p>

<p>

<math display="inline" id="Genetic_distance:6">
 <semantics>
  <mrow>
   <mi>D</mi>
   <mo>=</mo>
   <mrow>
    <mo>-</mo>
    <mrow>
     <mi>ln</mi>
     <mstyle displaystyle="true">
      <mfrac>
       <mrow>
        <munder>
         <mo largeop="true" movablelimits="false" symmetric="true">∑</mo>
         <mi>l</mi>
        </munder>
        <mrow>
         <munder>
          <mo largeop="true" movablelimits="false" symmetric="true">∑</mo>
          <mi>u</mi>
         </munder>
         <mrow>
          <msub>
           <mi>X</mi>
           <mi>u</mi>
          </msub>
          <msub>
           <mi>Y</mi>
           <mi>u</mi>
          </msub>
         </mrow>
        </mrow>
       </mrow>
       <msqrt>
        <mrow>
         <mrow>
          <mo stretchy="false">(</mo>
          <mrow>
           <munder>
            <mo largeop="true" movablelimits="false" symmetric="true">∑</mo>
            <mi>l</mi>
           </munder>
           <mrow>
            <munder>
             <mo largeop="true" movablelimits="false" symmetric="true">∑</mo>
             <mi>u</mi>
            </munder>
            <msubsup>
             <mi>X</mi>
             <mi>u</mi>
             <mn>2</mn>
            </msubsup>
           </mrow>
          </mrow>
          <mo stretchy="false">)</mo>
         </mrow>
         <mrow>
          <mo stretchy="false">(</mo>
          <mrow>
           <munder>
            <mo largeop="true" movablelimits="false" symmetric="true">∑</mo>
            <mi>l</mi>
           </munder>
           <mrow>
            <munder>
             <mo largeop="true" movablelimits="false" symmetric="true">∑</mo>
             <mi>u</mi>
            </munder>
            <msubsup>
             <mi>Y</mi>
             <mi>u</mi>
             <mn>2</mn>
            </msubsup>
           </mrow>
          </mrow>
          <mo stretchy="false">)</mo>
         </mrow>
        </mrow>
       </msqrt>
      </mfrac>
     </mstyle>
    </mrow>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <eq></eq>
    <ci>D</ci>
    <apply>
     <minus></minus>
     <apply>
      <ln></ln>
      <apply>
       <divide></divide>
       <apply>
        <apply>
         <csymbol cd="ambiguous">subscript</csymbol>
         <sum></sum>
         <ci>l</ci>
        </apply>
        <apply>
         <apply>
          <csymbol cd="ambiguous">subscript</csymbol>
          <sum></sum>
          <ci>u</ci>
         </apply>
         <apply>
          <times></times>
          <apply>
           <csymbol cd="ambiguous">subscript</csymbol>
           <ci>X</ci>
           <ci>u</ci>
          </apply>
          <apply>
           <csymbol cd="ambiguous">subscript</csymbol>
           <ci>Y</ci>
           <ci>u</ci>
          </apply>
         </apply>
        </apply>
       </apply>
       <apply>
        <root></root>
        <apply>
         <times></times>
         <apply>
          <apply>
           <csymbol cd="ambiguous">subscript</csymbol>
           <sum></sum>
           <ci>l</ci>
          </apply>
          <apply>
           <apply>
            <csymbol cd="ambiguous">subscript</csymbol>
            <sum></sum>
            <ci>u</ci>
           </apply>
           <apply>
            <csymbol cd="ambiguous">superscript</csymbol>
            <apply>
             <csymbol cd="ambiguous">subscript</csymbol>
             <ci>X</ci>
             <ci>u</ci>
            </apply>
            <cn type="integer">2</cn>
           </apply>
          </apply>
         </apply>
         <apply>
          <apply>
           <csymbol cd="ambiguous">subscript</csymbol>
           <sum></sum>
           <ci>l</ci>
          </apply>
          <apply>
           <apply>
            <csymbol cd="ambiguous">subscript</csymbol>
            <sum></sum>
            <ci>u</ci>
           </apply>
           <apply>
            <csymbol cd="ambiguous">superscript</csymbol>
            <apply>
             <csymbol cd="ambiguous">subscript</csymbol>
             <ci>Y</ci>
             <ci>u</ci>
            </apply>
            <cn type="integer">2</cn>
           </apply>
          </apply>
         </apply>
        </apply>
       </apply>
      </apply>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \displaystyle D=-\ln\frac{\sum\limits_{l}\sum\limits_{u}X_{u}Y_{u}}{\sqrt{(%
\sum\limits_{l}\sum\limits_{u}X_{u}^{2})(\sum\limits_{l}\sum\limits_{u}Y_{u}^{%
2})}}
  </annotation>
 </semantics>
</math>


</p>

<p>This distance can also be expressed in terms of the arithmetic mean of gene identity. Let 

<math display="inline" id="Genetic_distance:7">
 <semantics>
  <msub>
   <mi>j</mi>
   <mi>X</mi>
  </msub>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <csymbol cd="ambiguous">subscript</csymbol>
    <ci>j</ci>
    <ci>X</ci>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   j_{X}
  </annotation>
 </semantics>
</math>

 be the probability for the two members of population 

<math display="inline" id="Genetic_distance:8">
 <semantics>
  <mi>X</mi>
  <annotation-xml encoding="MathML-Content">
   <ci>X</ci>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   X
  </annotation>
 </semantics>
</math>

 having the same allele at a particular locus and 

<math display="inline" id="Genetic_distance:9">
 <semantics>
  <msub>
   <mi>j</mi>
   <mi>Y</mi>
  </msub>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <csymbol cd="ambiguous">subscript</csymbol>
    <ci>j</ci>
    <ci>Y</ci>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   j_{Y}
  </annotation>
 </semantics>
</math>

 be the corresponding probability in population 

<math display="inline" id="Genetic_distance:10">
 <semantics>
  <mi>Y</mi>
  <annotation-xml encoding="MathML-Content">
   <ci>Y</ci>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   Y
  </annotation>
 </semantics>
</math>

. Also, let 

<math display="inline" id="Genetic_distance:11">
 <semantics>
  <msub>
   <mi>j</mi>
   <mrow>
    <mi>X</mi>
    <mi>Y</mi>
   </mrow>
  </msub>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <csymbol cd="ambiguous">subscript</csymbol>
    <ci>j</ci>
    <apply>
     <times></times>
     <ci>X</ci>
     <ci>Y</ci>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   j_{XY}
  </annotation>
 </semantics>
</math>

 be the probability for a member of 

<math display="inline" id="Genetic_distance:12">
 <semantics>
  <mi>X</mi>
  <annotation-xml encoding="MathML-Content">
   <ci>X</ci>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   X
  </annotation>
 </semantics>
</math>

 and a member of 

<math display="inline" id="Genetic_distance:13">
 <semantics>
  <mi>Y</mi>
  <annotation-xml encoding="MathML-Content">
   <ci>Y</ci>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   Y
  </annotation>
 </semantics>
</math>

 having the same allele. Now let 

<math display="inline" id="Genetic_distance:14">
 <semantics>
  <msub>
   <mi>J</mi>
   <mi>X</mi>
  </msub>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <csymbol cd="ambiguous">subscript</csymbol>
    <ci>J</ci>
    <ci>X</ci>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   J_{X}
  </annotation>
 </semantics>
</math>

, 

<math display="inline" id="Genetic_distance:15">
 <semantics>
  <msub>
   <mi>J</mi>
   <mi>Y</mi>
  </msub>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <csymbol cd="ambiguous">subscript</csymbol>
    <ci>J</ci>
    <ci>Y</ci>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   J_{Y}
  </annotation>
 </semantics>
</math>

 and 

<math display="inline" id="Genetic_distance:16">
 <semantics>
  <msub>
   <mi>J</mi>
   <mrow>
    <mi>X</mi>
    <mi>Y</mi>
   </mrow>
  </msub>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <csymbol cd="ambiguous">subscript</csymbol>
    <ci>J</ci>
    <apply>
     <times></times>
     <ci>X</ci>
     <ci>Y</ci>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   J_{XY}
  </annotation>
 </semantics>
</math>

 represent the <a href="arithmetic_mean" title="wikilink">arithmetic mean</a> of 

<math display="inline" id="Genetic_distance:17">
 <semantics>
  <msub>
   <mi>j</mi>
   <mi>X</mi>
  </msub>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <csymbol cd="ambiguous">subscript</csymbol>
    <ci>j</ci>
    <ci>X</ci>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   j_{X}
  </annotation>
 </semantics>
</math>

, 

<math display="inline" id="Genetic_distance:18">
 <semantics>
  <msub>
   <mi>j</mi>
   <mi>Y</mi>
  </msub>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <csymbol cd="ambiguous">subscript</csymbol>
    <ci>j</ci>
    <ci>Y</ci>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   j_{Y}
  </annotation>
 </semantics>
</math>

 and 

<math display="inline" id="Genetic_distance:19">
 <semantics>
  <msub>
   <mi>j</mi>
   <mrow>
    <mi>X</mi>
    <mi>Y</mi>
   </mrow>
  </msub>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <csymbol cd="ambiguous">subscript</csymbol>
    <ci>j</ci>
    <apply>
     <times></times>
     <ci>X</ci>
     <ci>Y</ci>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   j_{XY}
  </annotation>
 </semantics>
</math>

 over all loci, respectively. In other words</p>

<p>

<math display="inline" id="Genetic_distance:20">
 <semantics>
  <mrow>
   <msub>
    <mi>J</mi>
    <mi>X</mi>
   </msub>
   <mo>=</mo>
   <mrow>
    <mstyle displaystyle="true">
     <munder>
      <mo largeop="true" movablelimits="false" symmetric="true">∑</mo>
      <mi>l</mi>
     </munder>
    </mstyle>
    <mrow>
     <mstyle displaystyle="true">
      <munder>
       <mo largeop="true" movablelimits="false" symmetric="true">∑</mo>
       <mi>u</mi>
      </munder>
     </mstyle>
     <mstyle displaystyle="true">
      <mfrac>
       <mmultiscripts>
        <mi>X</mi>
        <mi>u</mi>
        <none></none>
        <none></none>
        <mn>2</mn>
       </mmultiscripts>
       <mi>L</mi>
      </mfrac>
     </mstyle>
    </mrow>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <eq></eq>
    <apply>
     <csymbol cd="ambiguous">subscript</csymbol>
     <ci>J</ci>
     <ci>X</ci>
    </apply>
    <apply>
     <apply>
      <csymbol cd="ambiguous">subscript</csymbol>
      <sum></sum>
      <ci>l</ci>
     </apply>
     <apply>
      <apply>
       <csymbol cd="ambiguous">subscript</csymbol>
       <sum></sum>
       <ci>u</ci>
      </apply>
      <apply>
       <divide></divide>
       <apply>
        <csymbol cd="ambiguous">superscript</csymbol>
        <apply>
         <csymbol cd="ambiguous">subscript</csymbol>
         <ci>X</ci>
         <ci>u</ci>
        </apply>
        <cn type="integer">2</cn>
       </apply>
       <ci>L</ci>
      </apply>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \displaystyle J_{X}=\sum\limits_{l}\sum\limits_{u}\frac{{X_{u}}^{2}}{L}
  </annotation>
 </semantics>
</math>


</p>

<p>

<math display="inline" id="Genetic_distance:21">
 <semantics>
  <mrow>
   <msub>
    <mi>J</mi>
    <mi>Y</mi>
   </msub>
   <mo>=</mo>
   <mrow>
    <mstyle displaystyle="true">
     <munder>
      <mo largeop="true" movablelimits="false" symmetric="true">∑</mo>
      <mi>l</mi>
     </munder>
    </mstyle>
    <mrow>
     <mstyle displaystyle="true">
      <munder>
       <mo largeop="true" movablelimits="false" symmetric="true">∑</mo>
       <mi>u</mi>
      </munder>
     </mstyle>
     <mstyle displaystyle="true">
      <mfrac>
       <mmultiscripts>
        <mi>Y</mi>
        <mi>u</mi>
        <none></none>
        <none></none>
        <mn>2</mn>
       </mmultiscripts>
       <mi>L</mi>
      </mfrac>
     </mstyle>
    </mrow>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <eq></eq>
    <apply>
     <csymbol cd="ambiguous">subscript</csymbol>
     <ci>J</ci>
     <ci>Y</ci>
    </apply>
    <apply>
     <apply>
      <csymbol cd="ambiguous">subscript</csymbol>
      <sum></sum>
      <ci>l</ci>
     </apply>
     <apply>
      <apply>
       <csymbol cd="ambiguous">subscript</csymbol>
       <sum></sum>
       <ci>u</ci>
      </apply>
      <apply>
       <divide></divide>
       <apply>
        <csymbol cd="ambiguous">superscript</csymbol>
        <apply>
         <csymbol cd="ambiguous">subscript</csymbol>
         <ci>Y</ci>
         <ci>u</ci>
        </apply>
        <cn type="integer">2</cn>
       </apply>
       <ci>L</ci>
      </apply>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \displaystyle J_{Y}=\sum\limits_{l}\sum\limits_{u}\frac{{Y_{u}}^{2}}{L}
  </annotation>
 </semantics>
</math>


</p>

<p>

<math display="inline" id="Genetic_distance:22">
 <semantics>
  <mrow>
   <msub>
    <mi>J</mi>
    <mrow>
     <mi>X</mi>
     <mi>Y</mi>
    </mrow>
   </msub>
   <mo>=</mo>
   <mrow>
    <mstyle displaystyle="true">
     <munder>
      <mo largeop="true" movablelimits="false" symmetric="true">∑</mo>
      <mi>l</mi>
     </munder>
    </mstyle>
    <mrow>
     <mstyle displaystyle="true">
      <munder>
       <mo largeop="true" movablelimits="false" symmetric="true">∑</mo>
       <mi>u</mi>
      </munder>
     </mstyle>
     <mstyle displaystyle="true">
      <mfrac>
       <mrow>
        <msub>
         <mi>X</mi>
         <mi>u</mi>
        </msub>
        <msub>
         <mi>Y</mi>
         <mi>u</mi>
        </msub>
       </mrow>
       <mi>L</mi>
      </mfrac>
     </mstyle>
    </mrow>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <eq></eq>
    <apply>
     <csymbol cd="ambiguous">subscript</csymbol>
     <ci>J</ci>
     <apply>
      <times></times>
      <ci>X</ci>
      <ci>Y</ci>
     </apply>
    </apply>
    <apply>
     <apply>
      <csymbol cd="ambiguous">subscript</csymbol>
      <sum></sum>
      <ci>l</ci>
     </apply>
     <apply>
      <apply>
       <csymbol cd="ambiguous">subscript</csymbol>
       <sum></sum>
       <ci>u</ci>
      </apply>
      <apply>
       <divide></divide>
       <apply>
        <times></times>
        <apply>
         <csymbol cd="ambiguous">subscript</csymbol>
         <ci>X</ci>
         <ci>u</ci>
        </apply>
        <apply>
         <csymbol cd="ambiguous">subscript</csymbol>
         <ci>Y</ci>
         <ci>u</ci>
        </apply>
       </apply>
       <ci>L</ci>
      </apply>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \displaystyle J_{XY}=\sum\limits_{l}\sum\limits_{u}\frac{X_{u}Y_{u}}{L}
  </annotation>
 </semantics>
</math>


</p>

<p>where 

<math display="inline" id="Genetic_distance:23">
 <semantics>
  <mi>L</mi>
  <annotation-xml encoding="MathML-Content">
   <ci>L</ci>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   L
  </annotation>
 </semantics>
</math>

 is the total number of loci examined.<a class="footnoteRef" href="#fn9" id="fnref9"><sup>9</sup></a></p>

<p>Nei's standard distance can then be written as<a class="footnoteRef" href="#fn10" id="fnref10"><sup>10</sup></a></p>

<p>

<math display="inline" id="Genetic_distance:24">
 <semantics>
  <mrow>
   <mi>D</mi>
   <mo>=</mo>
   <mrow>
    <mo>-</mo>
    <mrow>
     <mi>ln</mi>
     <mstyle displaystyle="true">
      <mfrac>
       <msub>
        <mi>J</mi>
        <mrow>
         <mi>X</mi>
         <mi>Y</mi>
        </mrow>
       </msub>
       <msqrt>
        <mrow>
         <msub>
          <mi>J</mi>
          <mi>X</mi>
         </msub>
         <msub>
          <mi>J</mi>
          <mi>Y</mi>
         </msub>
        </mrow>
       </msqrt>
      </mfrac>
     </mstyle>
    </mrow>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <eq></eq>
    <ci>D</ci>
    <apply>
     <minus></minus>
     <apply>
      <ln></ln>
      <apply>
       <divide></divide>
       <apply>
        <csymbol cd="ambiguous">subscript</csymbol>
        <ci>J</ci>
        <apply>
         <times></times>
         <ci>X</ci>
         <ci>Y</ci>
        </apply>
       </apply>
       <apply>
        <root></root>
        <apply>
         <times></times>
         <apply>
          <csymbol cd="ambiguous">subscript</csymbol>
          <ci>J</ci>
          <ci>X</ci>
         </apply>
         <apply>
          <csymbol cd="ambiguous">subscript</csymbol>
          <ci>J</ci>
          <ci>Y</ci>
         </apply>
        </apply>
       </apply>
      </apply>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \displaystyle D=-\ln{\frac{J_{XY}}{\sqrt{J_{X}J_{Y}}}}
  </annotation>
 </semantics>
</math>


</p>
<h3 id="cavalli-sforza-chord-distance">Cavalli-Sforza chord distance</h3>

<p>In 1967 <a href="Luigi_Luca_Cavalli-Sforza" title="wikilink">Luigi Luca Cavalli-Sforza</a> and <a href="A._W._F._Edwards" title="wikilink">A. W. F. Edwards</a> published this measure. It assumes that genetic differences arise due to <a href="genetic_drift" title="wikilink">genetic drift</a> only. One major advantage of this measure is that the populations are represented in a hypersphere, the scale of which is one unit per gene substitution. The chord distance in the hyperdimensional sphere is given by<a class="footnoteRef" href="#fn11" id="fnref11"><sup>11</sup></a><a class="footnoteRef" href="#fn12" id="fnref12"><sup>12</sup></a></p>

<p>

<math display="inline" id="Genetic_distance:25">
 <semantics>
  <mrow>
   <msub>
    <mi>D</mi>
    <mrow>
     <mi>C</mi>
     <mi>H</mi>
    </mrow>
   </msub>
   <mo>=</mo>
   <mrow>
    <mstyle displaystyle="true">
     <mfrac>
      <mn>2</mn>
      <mi>π</mi>
     </mfrac>
    </mstyle>
    <msqrt>
     <mrow>
      <mn>2</mn>
      <mrow>
       <mo stretchy="false">(</mo>
       <mn>1</mn>
       <mo>-</mo>
       <mstyle displaystyle="true">
        <munder>
         <mo largeop="true" movablelimits="false" symmetric="true">∑</mo>
         <mi>l</mi>
        </munder>
       </mstyle>
       <mstyle displaystyle="true">
        <munder>
         <mo largeop="true" movablelimits="false" symmetric="true">∑</mo>
         <mi>u</mi>
        </munder>
       </mstyle>
       <msqrt>
        <mrow>
         <msub>
          <mi>X</mi>
          <mi>u</mi>
         </msub>
         <msub>
          <mi>Y</mi>
          <mi>u</mi>
         </msub>
         <mo stretchy="false">)</mo>
        </mrow>
       </msqrt>
      </mrow>
     </mrow>
    </msqrt>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <eq></eq>
    <apply>
     <csymbol cd="ambiguous">subscript</csymbol>
     <ci>D</ci>
     <apply>
      <times></times>
      <ci>C</ci>
      <ci>H</ci>
     </apply>
    </apply>
    <apply>
     <times></times>
     <apply>
      <divide></divide>
      <cn type="integer">2</cn>
      <ci>π</ci>
     </apply>
     <apply>
      <root></root>
      <cerror>
       <csymbol cd="ambiguous">fragments</csymbol>
       <cn type="integer">2</cn>
       <cerror>
        <csymbol cd="ambiguous">fragments</csymbol>
        <ci>normal-(</ci>
        <cn type="integer">1</cn>
        <minus></minus>
        <apply>
         <csymbol cd="ambiguous">subscript</csymbol>
         <sum></sum>
         <ci>l</ci>
        </apply>
        <apply>
         <csymbol cd="ambiguous">subscript</csymbol>
         <sum></sum>
         <ci>u</ci>
        </apply>
        <apply>
         <root></root>
         <cerror>
          <csymbol cd="ambiguous">fragments</csymbol>
          <apply>
           <csymbol cd="ambiguous">subscript</csymbol>
           <ci>X</ci>
           <ci>u</ci>
          </apply>
          <apply>
           <csymbol cd="ambiguous">subscript</csymbol>
           <ci>Y</ci>
           <ci>u</ci>
          </apply>
          <ci>normal-)</ci>
         </cerror>
        </apply>
       </cerror>
      </cerror>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \displaystyle D_{CH}=\frac{2}{\pi}\sqrt{2(1-\sum\limits_{l}\sum\limits_{u}%
\sqrt{X_{u}Y_{u})}}
  </annotation>
 </semantics>
</math>


</p>

<p>Some authors drop the factor 

<math display="inline" id="Genetic_distance:26">
 <semantics>
  <mfrac>
   <mn>2</mn>
   <mi>π</mi>
  </mfrac>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <divide></divide>
    <cn type="integer">2</cn>
    <ci>π</ci>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \frac{2}{\pi}
  </annotation>
 </semantics>
</math>

 to simplify the formula at the cost of losing the property that the scale is one unit per gene substitution.</p>
<h3 id="reynolds-weir-and-cockerhams-genetic-distance">Reynolds, Weir, and Cockerham's genetic distance</h3>

<p>In 1983, this measure was published by John Reynolds, B.S. Weir and C. Clark Cockerham. This measure assumes that genetic differentiation occurs only by <a href="genetic_drift" title="wikilink">genetic drift</a> without mutations. It estimates the <a href="Malecot's_method_of_coancestry" title="wikilink">coancestry coefficient</a> 

<math display="inline" id="Genetic_distance:27">
 <semantics>
  <mi mathvariant="normal">Θ</mi>
  <annotation-xml encoding="MathML-Content">
   <ci>normal-Θ</ci>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \Theta
  </annotation>
 </semantics>
</math>

 which provides a measure of the genetic divergence by:<a class="footnoteRef" href="#fn13" id="fnref13"><sup>13</sup></a></p>

<p>

<math display="inline" id="Genetic_distance:28">
 <semantics>
  <mrow>
   <msub>
    <mi mathvariant="normal">Θ</mi>
    <mi>w</mi>
   </msub>
   <mo>=</mo>
   <msqrt>
    <mstyle displaystyle="true">
     <mfrac>
      <mrow>
       <munder>
        <mo largeop="true" movablelimits="false" symmetric="true">∑</mo>
        <mi>l</mi>
       </munder>
       <mrow>
        <munder>
         <mo largeop="true" movablelimits="false" symmetric="true">∑</mo>
         <mi>u</mi>
        </munder>
        <msup>
         <mrow>
          <mo stretchy="false">(</mo>
          <mrow>
           <msub>
            <mi>X</mi>
            <mi>u</mi>
           </msub>
           <mo>-</mo>
           <msub>
            <mi>Y</mi>
            <mi>u</mi>
           </msub>
          </mrow>
          <mo stretchy="false">)</mo>
         </mrow>
         <mn>2</mn>
        </msup>
       </mrow>
      </mrow>
      <mrow>
       <mn>2</mn>
       <mrow>
        <munder>
         <mo largeop="true" movablelimits="false" symmetric="true">∑</mo>
         <mi>l</mi>
        </munder>
        <mrow>
         <mo stretchy="false">(</mo>
         <mrow>
          <mn>1</mn>
          <mo>-</mo>
          <mrow>
           <munder>
            <mo largeop="true" movablelimits="false" symmetric="true">∑</mo>
            <mi>u</mi>
           </munder>
           <mrow>
            <msub>
             <mi>X</mi>
             <mi>u</mi>
            </msub>
            <msub>
             <mi>Y</mi>
             <mi>u</mi>
            </msub>
           </mrow>
          </mrow>
         </mrow>
         <mo stretchy="false">)</mo>
        </mrow>
       </mrow>
      </mrow>
     </mfrac>
    </mstyle>
   </msqrt>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <eq></eq>
    <apply>
     <csymbol cd="ambiguous">subscript</csymbol>
     <ci>normal-Θ</ci>
     <ci>w</ci>
    </apply>
    <apply>
     <root></root>
     <apply>
      <divide></divide>
      <apply>
       <apply>
        <csymbol cd="ambiguous">subscript</csymbol>
        <sum></sum>
        <ci>l</ci>
       </apply>
       <apply>
        <apply>
         <csymbol cd="ambiguous">subscript</csymbol>
         <sum></sum>
         <ci>u</ci>
        </apply>
        <apply>
         <csymbol cd="ambiguous">superscript</csymbol>
         <apply>
          <minus></minus>
          <apply>
           <csymbol cd="ambiguous">subscript</csymbol>
           <ci>X</ci>
           <ci>u</ci>
          </apply>
          <apply>
           <csymbol cd="ambiguous">subscript</csymbol>
           <ci>Y</ci>
           <ci>u</ci>
          </apply>
         </apply>
         <cn type="integer">2</cn>
        </apply>
       </apply>
      </apply>
      <apply>
       <times></times>
       <cn type="integer">2</cn>
       <apply>
        <apply>
         <csymbol cd="ambiguous">subscript</csymbol>
         <sum></sum>
         <ci>l</ci>
        </apply>
        <apply>
         <minus></minus>
         <cn type="integer">1</cn>
         <apply>
          <apply>
           <csymbol cd="ambiguous">subscript</csymbol>
           <sum></sum>
           <ci>u</ci>
          </apply>
          <apply>
           <times></times>
           <apply>
            <csymbol cd="ambiguous">subscript</csymbol>
            <ci>X</ci>
            <ci>u</ci>
           </apply>
           <apply>
            <csymbol cd="ambiguous">subscript</csymbol>
            <ci>Y</ci>
            <ci>u</ci>
           </apply>
          </apply>
         </apply>
        </apply>
       </apply>
      </apply>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \displaystyle\Theta_{w}=\sqrt{\frac{\sum\limits_{l}\sum\limits_{u}(X_{u}-Y_{u}%
)^{2}}{2\sum\limits_{l}(1-\sum\limits_{u}X_{u}Y_{u})}}
  </annotation>
 </semantics>
</math>


</p>
<h3 id="other-measures-of-genetic-distance">Other measures of genetic distance</h3>

<p>Many other measures of genetic distance have been proposed with varying success.</p>
<h4 id="neis-da-distance-1983">Nei's D<sub>A</sub> distance 1983</h4>

<p>This distance assumes that genetic differences arise due to <a class="uri" href="mutation" title="wikilink">mutation</a> and <a href="genetic_drift" title="wikilink">genetic drift</a>, but this distance measure is known to give more reliable population trees than other distances particularly for microsatellite DNA data.<a class="footnoteRef" href="#fn14" id="fnref14"><sup>14</sup></a><a class="footnoteRef" href="#fn15" id="fnref15"><sup>15</sup></a></p>

<p>

<math display="inline" id="Genetic_distance:29">
 <semantics>
  <mrow>
   <msub>
    <mi>D</mi>
    <mi>A</mi>
   </msub>
   <mo>=</mo>
   <mrow>
    <mn>1</mn>
    <mo>-</mo>
    <mrow>
     <mstyle displaystyle="true">
      <munder>
       <mo largeop="true" movablelimits="false" symmetric="true">∑</mo>
       <mi>l</mi>
      </munder>
     </mstyle>
     <mrow>
      <mstyle displaystyle="true">
       <munder>
        <mo largeop="true" movablelimits="false" symmetric="true">∑</mo>
        <mi>u</mi>
       </munder>
      </mstyle>
      <mrow>
       <msqrt>
        <mrow>
         <msub>
          <mi>X</mi>
          <mi>u</mi>
         </msub>
         <msub>
          <mi>Y</mi>
          <mi>u</mi>
         </msub>
        </mrow>
       </msqrt>
       <mo>/</mo>
       <mi>L</mi>
      </mrow>
     </mrow>
    </mrow>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <eq></eq>
    <apply>
     <csymbol cd="ambiguous">subscript</csymbol>
     <ci>D</ci>
     <ci>A</ci>
    </apply>
    <apply>
     <minus></minus>
     <cn type="integer">1</cn>
     <apply>
      <apply>
       <csymbol cd="ambiguous">subscript</csymbol>
       <sum></sum>
       <ci>l</ci>
      </apply>
      <apply>
       <apply>
        <csymbol cd="ambiguous">subscript</csymbol>
        <sum></sum>
        <ci>u</ci>
       </apply>
       <apply>
        <divide></divide>
        <apply>
         <root></root>
         <apply>
          <times></times>
          <apply>
           <csymbol cd="ambiguous">subscript</csymbol>
           <ci>X</ci>
           <ci>u</ci>
          </apply>
          <apply>
           <csymbol cd="ambiguous">subscript</csymbol>
           <ci>Y</ci>
           <ci>u</ci>
          </apply>
         </apply>
        </apply>
        <ci>L</ci>
       </apply>
      </apply>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \displaystyle D_{A}=1-\sum\limits_{l}\sum\limits_{u}\sqrt{X_{u}Y_{u}}/{L}
  </annotation>
 </semantics>
</math>


</p>
<h4 id="euclidean-distance">Euclidean distance</h4>

<p>

<math display="inline" id="Genetic_distance:30">
 <semantics>
  <mrow>
   <msub>
    <mi>D</mi>
    <mrow>
     <mi>E</mi>
     <mi>U</mi>
    </mrow>
   </msub>
   <mo>=</mo>
   <msqrt>
    <mrow>
     <mstyle displaystyle="true">
      <munder>
       <mo largeop="true" movablelimits="false" symmetric="true">∑</mo>
       <mi>u</mi>
      </munder>
     </mstyle>
     <msup>
      <mrow>
       <mo stretchy="false">(</mo>
       <mrow>
        <msub>
         <mi>X</mi>
         <mi>u</mi>
        </msub>
        <mo>-</mo>
        <msub>
         <mi>Y</mi>
         <mi>u</mi>
        </msub>
       </mrow>
       <mo stretchy="false">)</mo>
      </mrow>
      <mn>2</mn>
     </msup>
    </mrow>
   </msqrt>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <eq></eq>
    <apply>
     <csymbol cd="ambiguous">subscript</csymbol>
     <ci>D</ci>
     <apply>
      <times></times>
      <ci>E</ci>
      <ci>U</ci>
     </apply>
    </apply>
    <apply>
     <root></root>
     <apply>
      <apply>
       <csymbol cd="ambiguous">subscript</csymbol>
       <sum></sum>
       <ci>u</ci>
      </apply>
      <apply>
       <csymbol cd="ambiguous">superscript</csymbol>
       <apply>
        <minus></minus>
        <apply>
         <csymbol cd="ambiguous">subscript</csymbol>
         <ci>X</ci>
         <ci>u</ci>
        </apply>
        <apply>
         <csymbol cd="ambiguous">subscript</csymbol>
         <ci>Y</ci>
         <ci>u</ci>
        </apply>
       </apply>
       <cn type="integer">2</cn>
      </apply>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \displaystyle D_{EU}=\sqrt{\sum\limits_{u}(X_{u}-Y_{u})^{2}}
  </annotation>
 </semantics>
</math>


 <a class="footnoteRef" href="#fn16" id="fnref16"><sup>16</sup></a></p>
<h4 id="goldstein-distance-1995">Goldstein distance 1995</h4>

<p>It was specifically developed for microsatellite markers and is based on the stepwise-mutation model (SMM). 

<math display="inline" id="Genetic_distance:31">
 <semantics>
  <msub>
   <mi>μ</mi>
   <mi>X</mi>
  </msub>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <csymbol cd="ambiguous">subscript</csymbol>
    <ci>μ</ci>
    <ci>X</ci>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \mu_{X}
  </annotation>
 </semantics>
</math>

 and 

<math display="inline" id="Genetic_distance:32">
 <semantics>
  <msub>
   <mi>μ</mi>
   <mi>Y</mi>
  </msub>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <csymbol cd="ambiguous">subscript</csymbol>
    <ci>μ</ci>
    <ci>Y</ci>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \mu_{Y}
  </annotation>
 </semantics>
</math>

 are the means of the allele frequencies in population X and Y.<a class="footnoteRef" href="#fn17" id="fnref17"><sup>17</sup></a></p>

<p>

<math display="inline" id="Genetic_distance:33">
 <semantics>
  <mrow>
   <msup>
    <mrow>
     <mo stretchy="false">(</mo>
     <mrow>
      <mi>δ</mi>
      <mi>μ</mi>
     </mrow>
     <mo stretchy="false">)</mo>
    </mrow>
    <mn>2</mn>
   </msup>
   <mo>=</mo>
   <mrow>
    <mstyle displaystyle="true">
     <munder>
      <mo largeop="true" movablelimits="false" symmetric="true">∑</mo>
      <mi>l</mi>
     </munder>
    </mstyle>
    <mrow>
     <msup>
      <mrow>
       <mo stretchy="false">(</mo>
       <mrow>
        <msub>
         <mi>μ</mi>
         <mi>X</mi>
        </msub>
        <mo>-</mo>
        <msub>
         <mi>μ</mi>
         <mi>Y</mi>
        </msub>
       </mrow>
       <mo stretchy="false">)</mo>
      </mrow>
      <mn>2</mn>
     </msup>
     <mo>/</mo>
     <mi>L</mi>
    </mrow>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <eq></eq>
    <apply>
     <csymbol cd="ambiguous">superscript</csymbol>
     <apply>
      <times></times>
      <ci>δ</ci>
      <ci>μ</ci>
     </apply>
     <cn type="integer">2</cn>
    </apply>
    <apply>
     <apply>
      <csymbol cd="ambiguous">subscript</csymbol>
      <sum></sum>
      <ci>l</ci>
     </apply>
     <apply>
      <divide></divide>
      <apply>
       <csymbol cd="ambiguous">superscript</csymbol>
       <apply>
        <minus></minus>
        <apply>
         <csymbol cd="ambiguous">subscript</csymbol>
         <ci>μ</ci>
         <ci>X</ci>
        </apply>
        <apply>
         <csymbol cd="ambiguous">subscript</csymbol>
         <ci>μ</ci>
         <ci>Y</ci>
        </apply>
       </apply>
       <cn type="integer">2</cn>
      </apply>
      <ci>L</ci>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \displaystyle(\delta\mu)^{2}=\sum\limits_{l}(\mu_{X}-\mu_{Y})^{2}/{L}
  </annotation>
 </semantics>
</math>


</p>
<h4 id="neis-minimum-genetic-distance-1973">Nei's minimum genetic distance 1973</h4>

<p>This measure assumes that genetic differences arise due to <a class="uri" href="mutation" title="wikilink">mutation</a> and <a href="genetic_drift" title="wikilink">genetic drift</a>.<a class="footnoteRef" href="#fn18" id="fnref18"><sup>18</sup></a></p>

<p>

<math display="inline" id="Genetic_distance:34">
 <semantics>
  <mrow>
   <msub>
    <mi>D</mi>
    <mi>m</mi>
   </msub>
   <mo>=</mo>
   <mrow>
    <mstyle displaystyle="true">
     <mfrac>
      <mrow>
       <mo stretchy="false">(</mo>
       <mrow>
        <msub>
         <mi>J</mi>
         <mi>X</mi>
        </msub>
        <mo>+</mo>
        <msub>
         <mi>J</mi>
         <mi>Y</mi>
        </msub>
       </mrow>
       <mo stretchy="false">)</mo>
      </mrow>
      <mn>2</mn>
     </mfrac>
    </mstyle>
    <mo>-</mo>
    <msub>
     <mi>J</mi>
     <mrow>
      <mi>X</mi>
      <mi>Y</mi>
     </mrow>
    </msub>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <eq></eq>
    <apply>
     <csymbol cd="ambiguous">subscript</csymbol>
     <ci>D</ci>
     <ci>m</ci>
    </apply>
    <apply>
     <minus></minus>
     <apply>
      <divide></divide>
      <apply>
       <plus></plus>
       <apply>
        <csymbol cd="ambiguous">subscript</csymbol>
        <ci>J</ci>
        <ci>X</ci>
       </apply>
       <apply>
        <csymbol cd="ambiguous">subscript</csymbol>
        <ci>J</ci>
        <ci>Y</ci>
       </apply>
      </apply>
      <cn type="integer">2</cn>
     </apply>
     <apply>
      <csymbol cd="ambiguous">subscript</csymbol>
      <ci>J</ci>
      <apply>
       <times></times>
       <ci>X</ci>
       <ci>Y</ci>
      </apply>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \displaystyle D_{m}=\frac{(J_{X}+J_{Y})}{2}-J_{XY}
  </annotation>
 </semantics>
</math>


</p>
<h4 id="rogers-distance-1972">Roger's distance 1972</h4>

<p>

<math display="inline" id="Genetic_distance:35">
 <semantics>
  <mrow>
   <msub>
    <mi>D</mi>
    <mi>R</mi>
   </msub>
   <mo>=</mo>
   <mrow>
    <mstyle displaystyle="true">
     <mfrac>
      <mn>1</mn>
      <mi>L</mi>
     </mfrac>
    </mstyle>
    <msqrt>
     <mstyle displaystyle="true">
      <mfrac>
       <mrow>
        <munder>
         <mo largeop="true" movablelimits="false" symmetric="true">∑</mo>
         <mi>u</mi>
        </munder>
        <msup>
         <mrow>
          <mo stretchy="false">(</mo>
          <mrow>
           <msub>
            <mi>X</mi>
            <mi>u</mi>
           </msub>
           <mo>-</mo>
           <msub>
            <mi>Y</mi>
            <mi>u</mi>
           </msub>
          </mrow>
          <mo stretchy="false">)</mo>
         </mrow>
         <mn>2</mn>
        </msup>
       </mrow>
       <mn>2</mn>
      </mfrac>
     </mstyle>
    </msqrt>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <eq></eq>
    <apply>
     <csymbol cd="ambiguous">subscript</csymbol>
     <ci>D</ci>
     <ci>R</ci>
    </apply>
    <apply>
     <times></times>
     <apply>
      <divide></divide>
      <cn type="integer">1</cn>
      <ci>L</ci>
     </apply>
     <apply>
      <root></root>
      <apply>
       <divide></divide>
       <apply>
        <apply>
         <csymbol cd="ambiguous">subscript</csymbol>
         <sum></sum>
         <ci>u</ci>
        </apply>
        <apply>
         <csymbol cd="ambiguous">superscript</csymbol>
         <apply>
          <minus></minus>
          <apply>
           <csymbol cd="ambiguous">subscript</csymbol>
           <ci>X</ci>
           <ci>u</ci>
          </apply>
          <apply>
           <csymbol cd="ambiguous">subscript</csymbol>
           <ci>Y</ci>
           <ci>u</ci>
          </apply>
         </apply>
         <cn type="integer">2</cn>
        </apply>
       </apply>
       <cn type="integer">2</cn>
      </apply>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \displaystyle D_{R}=\frac{1}{L}\sqrt{\frac{\sum\limits_{u}(X_{u}-Y_{u})^{2}}{2}}
  </annotation>
 </semantics>
</math>


 <a class="footnoteRef" href="#fn19" id="fnref19"><sup>19</sup></a></p>
<h4 id="fixation-index">Fixation index</h4>

<p>A commonly used measure of genetic distance is the <a href="fixation_index" title="wikilink">fixation index</a> which varies between 0 and 1. A value of 0 indicates that two populations are genetically identical whereas a value of 1 indicates that two populations are different species. No mutation is assumed.</p>
<h2 id="software">Software</h2>
<ul>
<li><a class="uri" href="PHYLIP" title="wikilink">PHYLIP</a> uses <a href="http://evolution.genetics.washington.edu/phylip/doc/gendist.html">GENDIST</a>
<ul>
<li>Nei's standard genetic distance 1972</li>
<li>Cavalli-Sforza and Edwards 1967</li>
<li>Reynolds, Weir, and Cockerham's 1983</li>
</ul></li>
<li><a href="http://tomato.biol.trinity.edu/programs/index.php/TFPGA">TFPGA</a>
<ul>
<li>Nei's standard genetic distance (original and unbiased)</li>
<li>Nei's minimum genetic distance (original and unbiased)</li>
<li>Wright's (1978) modification of Roger's (1972) distance</li>
<li>Reynolds, Weir, and Cockerham's 1983</li>
</ul></li>
<li><a href="http://tomato.biol.trinity.edu/programs/index.php/Genetic_Data_Analysis">GDA</a></li>
<li><a href="http://tomato.biol.trinity.edu/programs/index.php/POPGENE">POPGENE</a></li>
<li><a href="http://www.med.kagawa-u.ac.jp/~genomelb/takezaki/poptree2/index.html">POPTREE2 Takezaki, Nei, and Tamura (2010, 2014)</a>
<ul>
<li>Commonly used genetic distances and gene diversity analysis</li>
</ul></li>
<li><a href="http://www.personal.psu.edu/nxm2/dispan2.htm">DISPAN</a>
<ul>
<li>Nei's standard genetic distance 1972</li>
<li>Nei's D<sub>A</sub> distance between populations 1983</li>
</ul></li>
</ul>
<h2 id="see-also">See also</h2>
<ul>
<li><a href="Coefficient_of_relationship" title="wikilink">Coefficient of relationship</a></li>
<li><a href="Degree_of_consanguinity" title="wikilink">Degree of consanguinity</a></li>
<li><a href="Human_genetic_variation" title="wikilink">Human genetic variation</a></li>
<li><a href="Human_genetic_clustering" title="wikilink">Human genetic clustering</a></li>
<li><a class="uri" href="Phylogenetics" title="wikilink">Phylogenetics</a></li>
<li><a href="Allele_frequency" title="wikilink">Allele frequency</a></li>
</ul>
<h2 id="references">References</h2>
<references>
</references>
<h2 id="external-links">External links</h2>
<ul>
<li><a href="http://helix.biology.mcmaster.ca/brent/node7.html"><em>The Estimation of Genetic Distance and Population Substructure from Microsatellite allele frequency data.</em>, Brent W. Murray (May 1996), McMaster University website on genetic distance</a></li>
<li><a href="http://nitro.biosci.arizona.edu/ftDNA/Distance.html">Computing distance by stepwise genetic distance model, web pages of Bruce Walsh at the Department of Ecology and Evolutionary Biology at the University of Arizona</a></li>
</ul>

<p><br/>
 "</p>

<p><a class="uri" href="Category:Phylogenetics" title="wikilink">Category:Phylogenetics</a></p>
<section class="footnotes">
<hr/>
<ol>
<li id="fn1"><a href="#fnref1">↩</a></li>
<li id="fn2"><a href="#fnref2">↩</a></li>
<li id="fn3">Ruane, J. (1999). A critical review of the value of genetic distance studies in conservation of animal genetic resources. Journal of Animal Breeding and Genetics, 116(5), 317-323. Chicago.<a href="#fnref3">↩</a></li>
<li id="fn4"><a href="#fnref4">↩</a></li>
<li id="fn5"></li>
<li id="fn6"><a href="#fnref6">↩</a></li>
<li id="fn7"><a href="#fnref7">↩</a></li>
<li id="fn8"></li>
<li id="fn9">Nei, M. (1987) Genetic distance and molecular phylogeny. In: <em>Population Genetics and Fishery Management</em> (N. Ryman and F. Utter, eds.), University of Washington Press, Seattle, WA, pp. 193-223.<a href="#fnref9">↩</a></li>
<li id="fn10"></li>
<li id="fn11"></li>
<li id="fn12"></li>
<li id="fn13"></li>
<li id="fn14">Nei, M., F. Tajima, &amp; Y. Tateno (1983) Accuracy of estimated phylogenetic trees from molecular data. II. Gene frequency data. <em>J. Mol. Evol.</em> 19:153-170.<a href="#fnref14">↩</a></li>
<li id="fn15">Takezaki, N. and <a href="Masatoshi_Nei" title="wikilink">Nei, M.</a> (1996) Genetic distances and reconstruction of phylogenetic trees from microsatellite DNA. <em>Genetics</em> 144:389-399.<a href="#fnref15">↩</a></li>
<li id="fn16"></li>
<li id="fn17"><a href="#fnref17">↩</a></li>
<li id="fn18"></li>
<li id="fn19">Rogers, J. S. (1972). Measures of similarity and genetic distance. In <em>Studies in Genetics</em> VII. pp. 145−153. University of Texas Publication 7213. Austin, Texas.<a href="#fnref19">↩</a></li>
</ol>
</section>
</body>
</html>
