<html lang="en">
<head>
<meta charset="utf-8"/>
<title offset="1744">Genic capture</title>
<script src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_SVG.js" type="text/javascript">
</script>
</head>
<body>
<h1>Genic capture</h1>
<hr/>

<p><strong>Genic capture</strong> is a hypothesis explaining the maintenance of <a href="genetic_variance" title="wikilink">genetic variance</a> in traits under <a href="sexual_selection" title="wikilink">sexual selection</a>. A classic problem in sexual selection is the fixation of <a class="uri" href="alleles" title="wikilink">alleles</a> that are beneficial under strong selection, thereby eliminating the benefits of <a href="mate_choice" title="wikilink">mate choice</a>. Genic capture resolves this <a href="lek_paradox" title="wikilink">paradox</a> by suggesting that <a href="Quantitative_genetics" title="wikilink">additive genetic variance</a> of <a href="Sexual_selection" title="wikilink">sexually selected traits</a> reflects the genetic variance in total condition.<a class="footnoteRef" href="#fn1" id="fnref1"><sup>1</sup></a> A deleterious <a class="uri" href="mutation" title="wikilink">mutation</a> anywhere in the <a class="uri" href="genome" title="wikilink">genome</a> will adversely affect condition, and thereby adversely affect a condition-dependent sexually selected trait. Genic capture therefore resolves the <a href="lek_paradox" title="wikilink">lek paradox</a> by proposing that recurrent deleterious mutation maintains additive genetic variance in fitness by incorporating the entire <a href="mutation_load" title="wikilink">mutation load</a> of an individual. Thus any condition-dependent trait "captures" the overall genetic variance in condition. Rowe and Houle argued that genic capture ensures that <a href="Handicap_principle" title="wikilink">good genes</a> will become a central feature of the evolution of any sexually selected trait.</p>
<h2 id="condition">Condition</h2>

<p>The key quantity for genic capture is vaguely defined as "condition." The hypothesis only defines condition as a quantity that correlates tightly with overall <a href="fitness_(biology)" title="wikilink">fitness</a>, such that <a href="directional_selection" title="wikilink">directional selection</a> will always increase average condition over time. Condition should, in general, reflect overall energy acquisition, such that life-history variation reflects differential allocation to survival and sexual signalling. Genetic variation in condition should be very broadly affected by any changes in the genome. Close to <a href="genetic_equilibrium" title="wikilink">equilibrium</a> any mutation should be <a href="Fisher's_geometric_model" title="wikilink">deleterious</a>, thereby leading to non-zero overall mutation rate, maintaining variance in fitness.</p>
<h2 id="rowe-and-houles-quantitative-genetic-model">Rowe and Houle's Quantitative Genetic Model</h2>

<p>Rowe and Houle's simple model defines a trait as the result of three heritable components, a condition-independent component 

<math display="inline" id="Genic_capture:0">
 <semantics>
  <mi>a</mi>
  <annotation-xml encoding="MathML-Content">
   <ci>a</ci>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   a
  </annotation>
 </semantics>
</math>

, epistatic modification 

<math display="inline" id="Genic_capture:1">
 <semantics>
  <mi>b</mi>
  <annotation-xml encoding="MathML-Content">
   <ci>b</ci>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   b
  </annotation>
 </semantics>
</math>

 and condition, suggesting the following function for a trait:</p>

<p>

<math display="inline" id="Genic_capture:2">
 <semantics>
  <mrow>
   <mi>T</mi>
   <mo>=</mo>
   <mrow>
    <mi>a</mi>
    <mo>+</mo>
    <mrow>
     <mi>b</mi>
     <mi>C</mi>
    </mrow>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <eq></eq>
    <ci>T</ci>
    <apply>
     <plus></plus>
     <ci>a</ci>
     <apply>
      <times></times>
      <ci>b</ci>
      <ci>C</ci>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   T=a+bC
  </annotation>
 </semantics>
</math>

</p>

<p>where 

<math display="inline" id="Genic_capture:3">
 <semantics>
  <mi>C</mi>
  <annotation-xml encoding="MathML-Content">
   <ci>C</ci>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   C
  </annotation>
 </semantics>
</math>

 is the condition of an individual. Loci contributing to 

<math display="inline" id="Genic_capture:4">
 <semantics>
  <mi>C</mi>
  <annotation-xml encoding="MathML-Content">
   <ci>C</ci>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   C
  </annotation>
 </semantics>
</math>

 are loosely linked and independent of loci contributing to 

<math display="inline" id="Genic_capture:5">
 <semantics>
  <mi>a</mi>
  <annotation-xml encoding="MathML-Content">
   <ci>a</ci>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   a
  </annotation>
 </semantics>
</math>

 and 

<math display="inline" id="Genic_capture:6">
 <semantics>
  <mi>b</mi>
  <annotation-xml encoding="MathML-Content">
   <ci>b</ci>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   b
  </annotation>
 </semantics>
</math>

. Rowe and Houle then find the expected variance of 

<math display="inline" id="Genic_capture:7">
 <semantics>
  <mi>T</mi>
  <annotation-xml encoding="MathML-Content">
   <ci>T</ci>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   T
  </annotation>
 </semantics>
</math>

 and ignored higher-order terms (i.e. products of variances):</p>

<p>

<math display="inline" id="Genic_capture:8">
 <semantics>
  <mrow>
   <msub>
    <mi>G</mi>
    <mi>T</mi>
   </msub>
   <mo>≈</mo>
   <mrow>
    <msub>
     <mi>G</mi>
     <mi>a</mi>
    </msub>
    <mo>+</mo>
    <mrow>
     <msup>
      <mover accent="true">
       <mi>b</mi>
       <mo stretchy="false">¯</mo>
      </mover>
      <mn>2</mn>
     </msup>
     <msub>
      <mi>G</mi>
      <mi>C</mi>
     </msub>
    </mrow>
    <mo>+</mo>
    <mrow>
     <msup>
      <mover accent="true">
       <mi>C</mi>
       <mo stretchy="false">¯</mo>
      </mover>
      <mn>2</mn>
     </msup>
     <msub>
      <mi>G</mi>
      <mi>b</mi>
     </msub>
    </mrow>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <approx></approx>
    <apply>
     <csymbol cd="ambiguous">subscript</csymbol>
     <ci>G</ci>
     <ci>T</ci>
    </apply>
    <apply>
     <plus></plus>
     <apply>
      <csymbol cd="ambiguous">subscript</csymbol>
      <ci>G</ci>
      <ci>a</ci>
     </apply>
     <apply>
      <times></times>
      <apply>
       <csymbol cd="ambiguous">superscript</csymbol>
       <apply>
        <ci>normal-¯</ci>
        <ci>b</ci>
       </apply>
       <cn type="integer">2</cn>
      </apply>
      <apply>
       <csymbol cd="ambiguous">subscript</csymbol>
       <ci>G</ci>
       <ci>C</ci>
      </apply>
     </apply>
     <apply>
      <times></times>
      <apply>
       <csymbol cd="ambiguous">superscript</csymbol>
       <apply>
        <ci>normal-¯</ci>
        <ci>C</ci>
       </apply>
       <cn type="integer">2</cn>
      </apply>
      <apply>
       <csymbol cd="ambiguous">subscript</csymbol>
       <ci>G</ci>
       <ci>b</ci>
      </apply>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   G_{T}\approx G_{a}+\bar{b}^{2}G_{C}+\bar{C}^{2}G_{b}
  </annotation>
 </semantics>
</math>

</p>

<p>where 

<math display="inline" id="Genic_capture:9">
 <semantics>
  <msub>
   <mi>G</mi>
   <mi>T</mi>
  </msub>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <csymbol cd="ambiguous">subscript</csymbol>
    <ci>G</ci>
    <ci>T</ci>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   G_{T}
  </annotation>
 </semantics>
</math>

 represents the <a href="genetic_variance" title="wikilink">genetic variance</a> in the signal and analogously for other traits. Under <a href="directional_selection" title="wikilink">directional selection</a> on 

<math display="inline" id="Genic_capture:10">
 <semantics>
  <mi>T</mi>
  <annotation-xml encoding="MathML-Content">
   <ci>T</ci>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   T
  </annotation>
 </semantics>
</math>

, the loci underlying 

<math display="inline" id="Genic_capture:11">
 <semantics>
  <mi>a</mi>
  <annotation-xml encoding="MathML-Content">
   <ci>a</ci>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   a
  </annotation>
 </semantics>
</math>

 and 

<math display="inline" id="Genic_capture:12">
 <semantics>
  <mi>b</mi>
  <annotation-xml encoding="MathML-Content">
   <ci>b</ci>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   b
  </annotation>
 </semantics>
</math>

 may lose all genetic variance. However, there is no qualitative difference in directional selection on 

<math display="inline" id="Genic_capture:13">
 <semantics>
  <mi>C</mi>
  <annotation-xml encoding="MathML-Content">
   <ci>C</ci>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   C
  </annotation>
 </semantics>
</math>

 between <a href="stabilizing_selection" title="wikilink">stabilizing selection</a> (i.e. no sexual selection) and directional selection on 

<math display="inline" id="Genic_capture:14">
 <semantics>
  <mi>T</mi>
  <annotation-xml encoding="MathML-Content">
   <ci>T</ci>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   T
  </annotation>
 </semantics>
</math>

. Therefore the second term 

<math display="inline" id="Genic_capture:15">
 <semantics>
  <mrow>
   <msup>
    <mover accent="true">
     <mi>b</mi>
     <mo stretchy="false">¯</mo>
    </mover>
    <mn>2</mn>
   </msup>
   <msub>
    <mi>G</mi>
    <mi>C</mi>
   </msub>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <times></times>
    <apply>
     <csymbol cd="ambiguous">superscript</csymbol>
     <apply>
      <ci>normal-¯</ci>
      <ci>b</ci>
     </apply>
     <cn type="integer">2</cn>
    </apply>
    <apply>
     <csymbol cd="ambiguous">subscript</csymbol>
     <ci>G</ci>
     <ci>C</ci>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \bar{b}^{2}G_{C}
  </annotation>
 </semantics>
</math>

 will remain positive (due to biased mutation) and dominate 

<math display="inline" id="Genic_capture:16">
 <semantics>
  <msub>
   <mi>G</mi>
   <mi>T</mi>
  </msub>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <csymbol cd="ambiguous">subscript</csymbol>
    <ci>G</ci>
    <ci>T</ci>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   G_{T}
  </annotation>
 </semantics>
</math>

 under sexual selection.</p>
<h2 id="other-applications">Other Applications</h2>

<p>Genic capture can also play a role in accelerating adaptation to new environments.<a class="footnoteRef" href="#fn2" id="fnref2"><sup>2</sup></a></p>
<h2 id="comparisons">Comparisons</h2>

<p>Genic capture was proposed as a simpler alternative to another theory explaining the lek paradox<a class="footnoteRef" href="#fn3" id="fnref3"><sup>3</sup></a> that proposed that sexual selection creates <a href="disruptive_selection" title="wikilink">disruptive selection</a>, i.e. positive selection for genetic variance. Genic capture does not require any particular fitness function.</p>
<h2 id="references">References</h2>

<p>"</p>

<p><a href="Category:Evolutionary_biology" title="wikilink">Category:Evolutionary biology</a> <a href="Category:Sexual_selection" title="wikilink">Category:Sexual selection</a></p>
<section class="footnotes">
<hr/>
<ol>
<li id="fn1"><a href="#fnref1">↩</a></li>
<li id="fn2"><a href="#fnref2">↩</a></li>
<li id="fn3"><a href="#fnref3">↩</a></li>
</ol>
</section>
</body>
</html>
