<html lang="en">
<head>
<meta charset="utf-8"/>
<title offset="1949">Genomic library</title>
<script src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_SVG.js" type="text/javascript">
</script>
</head>
<body>
<h1>Genomic library</h1>
<hr>A '''genomic library''' is a collection of the total genomic [[DNA]] from a single [[organism]]. The DNA is stored in a population of identical [[Vector (molecular biology)|vectors]], each containing a different [[Insert (molecular biology)|insert]] of DNA. In order to construct a genomic library, the organism's DNA is [[Dna extraction|extracted]] from [[Cell (biology)|cells]] and then digested with a [[restriction enzyme]] to cut the DNA into fragments of a specific size. The fragments are then inserted
<p><code>into the vector using </code><a href="DNA_ligase" title="wikilink"><code>DNA</code> <code>ligase</code></a><code>.</code><a class="footnoteRef" href="#fn1" id="fnref1"><sup>1</sup></a><code> Next, the vector DNA can be taken up by a host organism - commonly a population of </code><a href="Escherichia_coli" title="wikilink"><code>Escherichia</code> <code>coli</code></a><code> or </code><a href="yeast" title="wikilink"><code>yeast</code></a><code> - with each cell containing only one vector molecule. Using a host cell to carry the vector allows for easy </code><a href="Molecular_genetics#Amplification" title="wikilink"><code>amplification</code></a><code> and retrieval of specific </code><a href="Molecular_cloning" title="wikilink"><code>clones</code></a><code> from the </code><a href="Library_(biology)" title="wikilink"><code>library</code></a><code> for analysis.</code><a class="footnoteRef" href="#fn2" id="fnref2"><sup>2</sup></a></p>

<p>There are several kinds of vectors available with various insert capacities. Generally, libraries made from organisms with larger <a class="uri" href="genomes" title="wikilink">genomes</a> require vectors featuring larger inserts, thereby fewer vector molecules are needed to make the library. Researchers can choose a vector also considering the ideal insert size to find a desired number of clones necessary for full genome coverage.<a class="footnoteRef" href="#fn3" id="fnref3"><sup>3</sup></a></p>

<p>Genomic libraries are commonly used for <a class="uri" href="sequencing" title="wikilink">sequencing</a> applications. They have played an important role in the whole genome sequencing of several organisms, including the human genome and several <a href="model_organism" title="wikilink">model organisms</a>.<a class="footnoteRef" href="#fn4" id="fnref4"><sup>4</sup></a> <a class="footnoteRef" href="#fn5" id="fnref5"><sup>5</sup></a></p>
<h2 id="history">History</h2>

<p>The first DNA-based <a class="uri" href="genome" title="wikilink">genome</a> ever fully sequenced was achieved by two-time Nobel Prize winner, <a href="Frederick_Sanger" title="wikilink">Frederick Sanger</a>, in 1977. Sanger and his team of scientists created a library of the <a class="uri" href="bacteriophage" title="wikilink">bacteriophage</a>, <a href="phi_X_174" title="wikilink">phi X 174</a>, for use in DNA <a class="uri" href="sequencing" title="wikilink">sequencing</a>.<a class="footnoteRef" href="#fn6" id="fnref6"><sup>6</sup></a> The importance of this success contributed to the ever-increasing demand for sequencing genomes to research <a href="gene_therapy" title="wikilink">gene therapy</a>. Teams are now able to catalog <a href="Polymorphism_(biology)" title="wikilink">polymorphisms</a> in genomes and investigate those candidate genes contributing to maladies such as <a href="Parkinson's_disease" title="wikilink">Parkinson's disease</a>, <a href="Alzheimer's_disease" title="wikilink">Alzheimer's disease</a>, <a href="multiple_sclerosis" title="wikilink">multiple sclerosis</a>, <a href="rheumatoid_arthritis" title="wikilink">rheumatoid arthritis</a>, and <a href="Type_1_diabetes" title="wikilink">Type 1 diabetes</a>.<a class="footnoteRef" href="#fn7" id="fnref7"><sup>7</sup></a> These are due to the advance of <a href="Genome-wide_association_study" title="wikilink">genome-wide association studies</a> from the ability to create and sequence genomic libraries. Prior, linkage and candidate-gene studies were some of the only approaches.<a class="footnoteRef" href="#fn8" id="fnref8"><sup>8</sup></a></p>
<h2 id="genomic-library-construction">Genomic library construction</h2>

<p>Construction of a genomic library involves creating many <a href="recombinant_DNA" title="wikilink">recombinant DNA</a> molecules. An organism's genomic <a class="uri" href="DNA" title="wikilink">DNA</a> is extracted and then digested with a <a href="restriction_enzyme" title="wikilink">restriction enzyme</a>. For organisms with very small genomes <em>(~10 kb)</em>, the digested fragments can be separated by <a href="gel_electrophoresis" title="wikilink">gel electrophoresis</a>. The separated fragments can then be excised and cloned into the vector separately. However, when a large genome is digested with a restriction enzyme, there are far too many fragments to excise individually. The entire set of fragments must be cloned together with the vector, and separation of clones can occur after. In either case, the fragments are ligated into a vector that has been digested with the same restriction enzyme. The vector containing the inserted fragments of genomic DNA can then be introduced into a host organism.<a class="footnoteRef" href="#fn9" id="fnref9"><sup>9</sup></a></p>

<p>Below are the steps for creating a genomic library from a large genome.</p>
<ol>
<li><a href="Dna_extraction" title="wikilink">Extract</a> and purify DNA.</li>
<li>Digest the DNA with a restriction enzyme. This creates fragments that are similar in size, each containing one or more genes.</li>
<li>Insert the fragments of DNA into vectors that were cut with the same restriction enzyme. Use the enzyme DNA ligase to seal the DNA fragments into the vector. This creates a large pool of recombinant molecules.</li>
<li>These recombinant molecules are taken up by a host bacterium by <a href="Transformation_(genetics)" title="wikilink">transformation</a>, creating a DNA library.<a class="footnoteRef" href="#fn10" id="fnref10"><sup>10</sup></a><a class="footnoteRef" href="#fn11" id="fnref11"><sup>11</sup></a></li>
</ol>

<p>Below is a diagram of the above outlined steps.</p>
<figure><b>(Figure)</b>
<figcaption>Genomic Library Construction</figcaption>
</figure>
<h3 id="determining-titer-of-library">Determining titer of library</h3>

<p>After a genomic library is constructed with a viral vector, such as <a href="lambda_phage" title="wikilink">lambda phage</a>, the <a class="uri" href="titer" title="wikilink">titer</a> of the library can be determined. Calculating the titer allows researchers to approximate how many infectious viral particles were successfully created in the library. To do this, dilutions of the library are used to <a class="uri" href="transfect" title="wikilink">transfect</a> <a href="Cell_culture" title="wikilink">cultures</a> of E. coli of known concentrations. The cultures are then plated on <a href="agar_plate" title="wikilink">agar plates</a> and incubated overnight. The number of <a href="viral_plaque" title="wikilink">viral plaques</a> are counted and can be used to calculate the total number of infectious viral particles in the library. Most viral vectors also carry a marker that allows clones containing an insert to be distinguished from those that do not have an insert. This allows researchers to also determine the percentage of infectious viral particles actually carrying a fragment of the library.<a class="footnoteRef" href="#fn12" id="fnref12"><sup>12</sup></a></p>

<p>A similar method can be used to titer genomic libraries made with non-viral vectors, such as <a href="plasmid" title="wikilink">plasmids</a> and <a href="Bacterial_Artificial_Chromosome" title="wikilink">BACs</a>. A test <a href="DNA_ligase" title="wikilink">ligation</a> of the library can be used to transform E. coli. The transformation is then spread on agar plates and incubated overnight. The titer of the transformation is determined by counting the number of colonies present on the plates. These vectors generally have a <a href="selectable_marker" title="wikilink">selectable marker</a> allowing the differentiation of clones containing an insert from those that do not. By doing this test, researchers can also determine the efficiency of the ligation and make adjustments as needed to ensure they get the desired number of clones for the library.<a class="footnoteRef" href="#fn13" id="fnref13"><sup>13</sup></a></p>
<h3 id="screening-library">Screening library</h3>
<figure><b>(Figure)</b>
<figcaption>Colony Blot Hybridization</figcaption>
</figure>

<p>In order to isolate clones that contain regions of interest from a library, the library must first be <a href="Marker_gene#Marker_for_screening" title="wikilink">screened</a>. One method of screening is <a href="Nucleic_acid_hybridisation#Hybridization" title="wikilink">hybridization</a>. Each transformed host cell of a library will contain only one vector with one insert of DNA. The whole library can be plated onto a filter over <a href="Growth_medium" title="wikilink">media</a>. The filter and <a href="Colony_(biology)" title="wikilink">colonies</a> are prepared for hybridization and then labeled with a <a href="Hybridization_probe" title="wikilink">probe</a>.<a class="footnoteRef" href="#fn14" id="fnref14"><sup>14</sup></a> The target DNA- insert of interest- can be identified by detection such as <a class="uri" href="autoradiography" title="wikilink">autoradiography</a> because of the <a href="Nucleic_acid_hybridisation#Hybridization" title="wikilink">hybridization</a> with the probe as seen below.</p>

<p>Another method of screening is with <a href="polymerase_chain_reaction" title="wikilink">polymerase chain reaction</a> (PCR). Some libraries are stored as pools of clones and screening by PCR is an efficient way to identify pools containing specific clones.<a class="footnoteRef" href="#fn15" id="fnref15"><sup>15</sup></a></p>
<h2 id="types-of-vectors">Types of vectors</h2>

<p><a class="uri" href="Genome" title="wikilink">Genome</a> size varies among different organisms and the <a href="cloning_vector" title="wikilink">cloning vector</a> must be selected accordingly. For a large genome, a vector with a large capacity should be chosen so that a relatively small number of <a class="uri" href="clones" title="wikilink">clones</a> are sufficient for coverage of the entire genome. However, it is often more difficult to characterize an <a href="Insert_(molecular_biology)" title="wikilink">insert</a> contained in a higher capacity vector.<a class="footnoteRef" href="#fn16" id="fnref16"><sup>16</sup></a></p>

<p>Below is a table of several kinds of vectors commonly used for genomic libraries and the insert size that each generally holds.</p>
<table>
<thead>
<tr class="header">
<th style="text-align: left;">
<p><a href="Vector_(molecular_biology)" title="wikilink">Vector</a> type</p></th>
<th style="text-align: left;">
<p><a href="Insert_(molecular_biology)" title="wikilink">Insert</a> size (thousands of <a href="Basepairs" title="wikilink">bases</a>)</p></th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;">
<p><a href="Plasmid" title="wikilink">Plasmids</a></p></td>
<td style="text-align: left;">
<p>up to 15</p></td>
</tr>
<tr class="even">
<td style="text-align: left;">
<p><a href="Lambda_phage" title="wikilink">Phage lambda (λ)</a></p></td>
<td style="text-align: left;">
<p>up to 25</p></td>
</tr>
<tr class="odd">
<td style="text-align: left;">
<p><a href="Cosmid" title="wikilink">Cosmids</a></p></td>
<td style="text-align: left;">
<p>up to 45</p></td>
</tr>
<tr class="even">
<td style="text-align: left;">
<p><a href="P1_phage" title="wikilink">Bacteriophage P1</a></p></td>
<td style="text-align: left;">
<p>70 to 100</p></td>
</tr>
<tr class="odd">
<td style="text-align: left;">
<p><a href="P1-derived_artificial_chromosome" title="wikilink">P1 artificial chromosomes</a> (PACs)</p></td>
<td style="text-align: left;">
<p>130 to 150</p></td>
</tr>
<tr class="even">
<td style="text-align: left;">
<p><a href="Bacterial_artificial_chromosome" title="wikilink">Bacterial artificial chromosomes</a> (BACs)</p></td>
<td style="text-align: left;">
<p>120 to 300</p></td>
</tr>
<tr class="odd">
<td style="text-align: left;">
<p><a href="Yeast_artificial_chromosome" title="wikilink">Yeast artificial chromosomes</a> (YACs)</p></td>
<td style="text-align: left;">
<p>250 to 2000</p></td>
</tr>
</tbody>
</table>
<h3 id="plasmids">Plasmids</h3>

<p>A <a class="uri" href="plasmid" title="wikilink">plasmid</a> is a double stranded <a href="circular_DNA" title="wikilink">circular DNA</a> molecule commonly used for <a href="molecular_cloning" title="wikilink">molecular cloning</a>. Plasmids are generally 2 to 4 <a href="Basepairs" title="wikilink">kilobase-pairs</a> (kb) in length and are capable of carrying inserts up to 15kb. Plasmids contain an <a href="origin_of_replication" title="wikilink">origin of replication</a> allowing them to replicate inside a bacterium independently of the host <a class="uri" href="chromosome" title="wikilink">chromosome</a>. Plasmids commonly carry a gene for <a href="antibiotic_resistance" title="wikilink">antibiotic resistance</a> that allows for the selection of bacterial cells containing the plasmid. Many plasmids also carry a <a href="reporter_gene" title="wikilink">reporter gene</a> that allows researchers to distinguish clones containing an insert from those that do not.<a class="footnoteRef" href="#fn17" id="fnref17"><sup>17</sup></a></p>
<h3 id="phage-lambda-λ">Phage lambda (λ)</h3>

<p><a href="Lambda_phage" title="wikilink">Phage λ</a> is a <a href="DNA_virus" title="wikilink">double-stranded DNA virus</a> that infects <em><a href="Escherichia_coli" title="wikilink">E. coli</a></em>. The λ chromosome is 48.5kb long and can carry inserts up to 25kb. These inserts replace non-essential viral sequences in the λ chromosome, while the genes required for formation of <a href="Virus" title="wikilink">viral particles</a> and <a class="uri" href="infection" title="wikilink">infection</a> remain intact. The insert DNA is <a href="Dna_replication" title="wikilink">replicated</a> with the viral DNA; thus, together they are packaged into viral particles. These particles are very efficient at infection and multiplication leading to a higher production of the recombinant λ chromosomes.<a class="footnoteRef" href="#fn18" id="fnref18"><sup>18</sup></a> However, due to the smaller insert size, libraries made with λ phage may require many clones for full genome coverage.<a class="footnoteRef" href="#fn19" id="fnref19"><sup>19</sup></a></p>
<h3 id="cosmids">Cosmids</h3>

<p><a class="uri" href="Cosmid" title="wikilink">Cosmid</a> vectors are plasmids that contain a small region of bacteriophage λ DNA called the cos sequence. This sequence allows the cosmid to be packaged into bacteriophage λ particles. These particles- containing a linearized cosmid- are introduced into the host cell by <a href="Transduction_(genetics)" title="wikilink">transduction</a>. Once inside the host, the cosmids circularize with the aid of the host's <a href="DNA_ligase" title="wikilink">DNA ligase</a> and then function as plasmids. Cosmids are capable of carrying inserts up to 45kb in size.<a class="footnoteRef" href="#fn20" id="fnref20"><sup>20</sup></a></p>
<h3 id="bacteriophage-p1-vectors">Bacteriophage P1 vectors</h3>

<p><a href="P1_phage" title="wikilink">Bacteriophage P1</a> vectors can hold inserts 70 – 100kb in size. They begin as linear DNA molecules packaged into bacteriophage P1 particles. These particles are injected into an E. coli strain expressing <a href="Cre_recombinase" title="wikilink">Cre recombinase</a>. The linear P1 vector becomes circularized by <a href="Cre-Lox_recombination" title="wikilink">recombination</a> between two loxP sites in the vector. P1 vectors generally contain a gene for antibiotic resistance and a positive selection marker to distinguish clones containing an insert from those that do not. P1 vectors also contain a P1 plasmid <a href="Replicon_(genetics)" title="wikilink">replicon</a>, which ensures only one copy of the vector is present in a cell. However, there is a second P1 replicon- called the P1 lytic replicon- that is controlled by an inducible <a href="Promoter_(genetics)" title="wikilink">promoter</a>. This promoter allows the amplification of more than one copy of the vector per cell prior to <a href="Dna_extraction" title="wikilink">DNA extraction</a>.<a class="footnoteRef" href="#fn21" id="fnref21"><sup>21</sup></a></p>
<figure><b>(Figure)</b>
<figcaption>bac vector</figcaption>
</figure>
<h3 id="p1-artificial-chromosomes">P1 artificial chromosomes</h3>

<p><a href="P1-derived_artificial_chromosome" title="wikilink">P1 artificial chromosomes</a> (PACs) have features of both P1 vectors and Bacterial Artificial Chromosomes (BACs). Similar to P1 vectors, they contain a plasmid and a lytic replicon as described above. Unlike P1 vectors, they do not need to be packaged into bacteriophage particles for transduction. Instead they are introduced into E. coli as circular DNA molecules through <a class="uri" href="electroporation" title="wikilink">electroporation</a> just as BACs are.<a class="footnoteRef" href="#fn22" id="fnref22"><sup>22</sup></a> Also similar to BACs, these are relatively harder to prepare due to a single origin of replication.<a class="footnoteRef" href="#fn23" id="fnref23"><sup>23</sup></a></p>
<h3 id="bacterial-artificial-chromosomes">Bacterial artificial chromosomes</h3>

<p><a href="Bacterial_artificial_chromosome" title="wikilink">Bacterial artificial chromosomes</a> (BACs) are circular DNA molecules, usually about 7kb in length, that are capable of holding inserts up to 300kb in size. BAC vectors contain a replicon derived from E. coli <a href="Fertility_factor" title="wikilink">F factor</a>, which ensures they are maintained at one copy per cell.<a class="footnoteRef" href="#fn24" id="fnref24"><sup>24</sup></a> Once an insert is ligated into a BAC, the BAC is introduced into <a href="Homologous_recombination" title="wikilink">recombination</a> deficient strains of E. coli by electroporation. Most BAC vectors contain a gene for antibiotic resistance and also a positive selection marker.<a class="footnoteRef" href="#fn25" id="fnref25"><sup>25</sup></a> The figure to the right depicts a BAC vector being cut with a restriction enzyme, followed by the insertion of foreign DNA that is re-annealed by a ligase. Overall, this is a very stable vector, but they may be hard to prepare due to a single origin of replication just like PACs.<a class="footnoteRef" href="#fn26" id="fnref26"><sup>26</sup></a></p>
<h3 id="yeast-artificial-chromosomes">Yeast artificial chromosomes</h3>

<p><a href="Yeast_artificial_chromosome" title="wikilink">Yeast artificial chromosomes</a> (YACs) are linear DNA molecules containing the necessary features of an authentic <a class="uri" href="yeast" title="wikilink">yeast</a> chromosome, including <a href="telomere" title="wikilink">telomeres</a>, a <a class="uri" href="centromere" title="wikilink">centromere</a>, and an <a href="origin_of_replication" title="wikilink">origin of replication</a>. Large inserts of DNA can be ligated into the middle of the YAC so that there is an “arm” of the YAC on either side of the insert. The recombinant YAC is introduced into yeast by transformation; <a href="selectable_markers" title="wikilink">selectable markers</a> present in the YAC allow for the identification of successful transformants. YACs can hold inserts up to 2000kb, but most YAC libraries contain inserts 250-400kb in size. Theoretically there is no upper limit on the size of insert a YAC can hold. It is the quality in the preparation of DNA used for inserts that determines the size limit.<a class="footnoteRef" href="#fn27" id="fnref27"><sup>27</sup></a> The most challenging aspect of using YAC is the fact they are prone to <a href="Chromosomal_rearrangement" title="wikilink">rearrangement</a>.<a class="footnoteRef" href="#fn28" id="fnref28"><sup>28</sup></a></p>
<h2 id="how-to-select-a-vector">How to select a vector</h2>

<p>Vector selection requires one to ensure the library made is representative of the entire genome. Any insert of the genome derived from a restriction enzyme should have an equal chance of being in the library compared to any other insert. Furthermore, recombinant molecules should contain large enough inserts ensuring the library size is able to be handled conveniently.<a class="footnoteRef" href="#fn29" id="fnref29"><sup>29</sup></a> This is particularly determined by the amount of clones needed to have in a library. The amount of clones to get a sampling of all the genes is determined by the size of the organism's genome as well as the average insert size. This is represented by the formula (also known as the Carbon and Clarke formula):<a class="footnoteRef" href="#fn30" id="fnref30"><sup>30</sup></a> 

<math display="inline" id="Genomic_library:0">
 <semantics>
  <mrow>
   <mi>N</mi>
   <mo>=</mo>
   <mfrac>
    <mrow>
     <mi>l</mi>
     <mi>n</mi>
     <mrow>
      <mo stretchy="false">(</mo>
      <mrow>
       <mn>1</mn>
       <mo>-</mo>
       <mi>P</mi>
      </mrow>
      <mo stretchy="false">)</mo>
     </mrow>
    </mrow>
    <mrow>
     <mi>l</mi>
     <mi>n</mi>
     <mrow>
      <mo stretchy="false">(</mo>
      <mrow>
       <mn>1</mn>
       <mo>-</mo>
       <mi>f</mi>
      </mrow>
      <mo stretchy="false">)</mo>
     </mrow>
    </mrow>
   </mfrac>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <eq></eq>
    <ci>N</ci>
    <apply>
     <divide></divide>
     <apply>
      <times></times>
      <ci>l</ci>
      <ci>n</ci>
      <apply>
       <minus></minus>
       <cn type="integer">1</cn>
       <ci>P</ci>
      </apply>
     </apply>
     <apply>
      <times></times>
      <ci>l</ci>
      <ci>n</ci>
      <apply>
       <minus></minus>
       <cn type="integer">1</cn>
       <ci>f</ci>
      </apply>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   N=\frac{ln(1-P)}{ln(1-f)}
  </annotation>
 </semantics>
</math>

</p>

<p>where,</p>

<p>

<math display="inline" id="Genomic_library:1">
 <semantics>
  <mi>N</mi>
  <annotation-xml encoding="MathML-Content">
   <ci>N</ci>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   N
  </annotation>
 </semantics>
</math>

 is the necessary number of recombinants<a class="footnoteRef" href="#fn31" id="fnref31"><sup>31</sup></a> 

<math display="inline" id="Genomic_library:2">
 <semantics>
  <mi>P</mi>
  <annotation-xml encoding="MathML-Content">
   <ci>P</ci>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   P
  </annotation>
 </semantics>
</math>

 is the desired probability that any fragment in the genome will occur at least once in the library created</p>

<p>

<math display="inline" id="Genomic_library:3">
 <semantics>
  <mi>f</mi>
  <annotation-xml encoding="MathML-Content">
   <ci>f</ci>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   f
  </annotation>
 </semantics>
</math>


 is the fractional proportion of the genome in a single recombinant</p>

<p>

<math display="inline" id="Genomic_library:4">
 <semantics>
  <mi>f</mi>
  <annotation-xml encoding="MathML-Content">
   <ci>f</ci>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   f
  </annotation>
 </semantics>
</math>

 can be further shown to be:</p>

<p>

<math display="inline" id="Genomic_library:5">
 <semantics>
  <mrow>
   <mi>f</mi>
   <mo>=</mo>
   <mfrac>
    <mi>i</mi>
    <mi>g</mi>
   </mfrac>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <eq></eq>
    <ci>f</ci>
    <apply>
     <divide></divide>
     <ci>i</ci>
     <ci>g</ci>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   f=\frac{i}{g}
  </annotation>
 </semantics>
</math>

</p>

<p>where,</p>

<p>

<math display="inline" id="Genomic_library:6">
 <semantics>
  <mi>i</mi>
  <annotation-xml encoding="MathML-Content">
   <ci>i</ci>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   i
  </annotation>
 </semantics>
</math>

 is the insert size</p>

<p>

<math display="inline" id="Genomic_library:7">
 <semantics>
  <mi>g</mi>
  <annotation-xml encoding="MathML-Content">
   <ci>g</ci>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   g
  </annotation>
 </semantics>
</math>

 is the genome size</p>

<p>Thus, increasing the insert size (by choice of vector) would allow for fewer clones needed to represent a genome. The proportion of the insert size versus the genome size represents the proportion of the respective genome in a single clone.<a class="footnoteRef" href="#fn32" id="fnref32"><sup>32</sup></a> Here is the equation with all parts considered:</p>

<p>

<math display="inline" id="Genomic_library:8">
 <semantics>
  <mrow>
   <mi>N</mi>
   <mo>=</mo>
   <mfrac>
    <mrow>
     <mi>l</mi>
     <mi>n</mi>
     <mrow>
      <mo stretchy="false">(</mo>
      <mrow>
       <mn>1</mn>
       <mo>-</mo>
       <mi>P</mi>
      </mrow>
      <mo stretchy="false">)</mo>
     </mrow>
    </mrow>
    <mrow>
     <mi>l</mi>
     <mi>n</mi>
     <mrow>
      <mo stretchy="false">(</mo>
      <mrow>
       <mn>1</mn>
       <mo>-</mo>
       <mfrac>
        <mi>i</mi>
        <mi>g</mi>
       </mfrac>
      </mrow>
      <mo stretchy="false">)</mo>
     </mrow>
    </mrow>
   </mfrac>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <eq></eq>
    <ci>N</ci>
    <apply>
     <divide></divide>
     <apply>
      <times></times>
      <ci>l</ci>
      <ci>n</ci>
      <apply>
       <minus></minus>
       <cn type="integer">1</cn>
       <ci>P</ci>
      </apply>
     </apply>
     <apply>
      <times></times>
      <ci>l</ci>
      <ci>n</ci>
      <apply>
       <minus></minus>
       <cn type="integer">1</cn>
       <apply>
        <divide></divide>
        <ci>i</ci>
        <ci>g</ci>
       </apply>
      </apply>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   N=\frac{ln(1-P)}{ln(1-\frac{i}{g})}
  </annotation>
 </semantics>
</math>


</p>
<h3 id="vector-selection-example">Vector selection example</h3>

<p>The above formula can be used to determine the 99% confidence level that all sequences in a genome are represented by using a vector with an insert size of twenty thousand basepairs (such as the phage lambda vector). The genome size of the organism is three billion basepairs in this example.</p>

<p>

<math display="inline" id="Genomic_library:9">
 <semantics>
  <mrow>
   <mi>N</mi>
   <mo>=</mo>
   <mfrac>
    <mrow>
     <mi>l</mi>
     <mi>n</mi>
     <mrow>
      <mo stretchy="false">(</mo>
      <mrow>
       <mn>1</mn>
       <mo>-</mo>
       <mn>0.99</mn>
      </mrow>
      <mo stretchy="false">)</mo>
     </mrow>
    </mrow>
    <mrow>
     <mi>l</mi>
     <mi>n</mi>
     <mrow>
      <mo stretchy="false">[</mo>
      <mrow>
       <mn>1</mn>
       <mo>-</mo>
       <mfrac>
        <mrow>
         <mrow>
          <mn>2.0</mn>
          <mo>×</mo>
          <msup>
           <mn>10</mn>
           <mn>4</mn>
          </msup>
         </mrow>
         <mi>b</mi>
         <mi>a</mi>
         <mi>s</mi>
         <mi>e</mi>
         <mi>p</mi>
         <mi>a</mi>
         <mi>i</mi>
         <mi>r</mi>
         <mi>s</mi>
        </mrow>
        <mrow>
         <mrow>
          <mn>3.0</mn>
          <mo>×</mo>
          <msup>
           <mn>10</mn>
           <mn>9</mn>
          </msup>
         </mrow>
         <mi>b</mi>
         <mi>a</mi>
         <mi>s</mi>
         <mi>e</mi>
         <mi>p</mi>
         <mi>a</mi>
         <mi>i</mi>
         <mi>r</mi>
         <mi>s</mi>
        </mrow>
       </mfrac>
      </mrow>
      <mo stretchy="false">]</mo>
     </mrow>
    </mrow>
   </mfrac>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <eq></eq>
    <ci>N</ci>
    <apply>
     <divide></divide>
     <apply>
      <times></times>
      <ci>l</ci>
      <ci>n</ci>
      <apply>
       <minus></minus>
       <cn type="integer">1</cn>
       <cn type="float">0.99</cn>
      </apply>
     </apply>
     <apply>
      <times></times>
      <ci>l</ci>
      <ci>n</ci>
      <apply>
       <csymbol cd="latexml">delimited-[]</csymbol>
       <apply>
        <minus></minus>
        <cn type="integer">1</cn>
        <apply>
         <divide></divide>
         <apply>
          <times></times>
          <apply>
           <times></times>
           <cn type="float">2.0</cn>
           <apply>
            <csymbol cd="ambiguous">superscript</csymbol>
            <cn type="integer">10</cn>
            <cn type="integer">4</cn>
           </apply>
          </apply>
          <ci>b</ci>
          <ci>a</ci>
          <ci>s</ci>
          <ci>e</ci>
          <ci>p</ci>
          <ci>a</ci>
          <ci>i</ci>
          <ci>r</ci>
          <ci>s</ci>
         </apply>
         <apply>
          <times></times>
          <apply>
           <times></times>
           <cn type="float">3.0</cn>
           <apply>
            <csymbol cd="ambiguous">superscript</csymbol>
            <cn type="integer">10</cn>
            <cn type="integer">9</cn>
           </apply>
          </apply>
          <ci>b</ci>
          <ci>a</ci>
          <ci>s</ci>
          <ci>e</ci>
          <ci>p</ci>
          <ci>a</ci>
          <ci>i</ci>
          <ci>r</ci>
          <ci>s</ci>
         </apply>
        </apply>
       </apply>
      </apply>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   N=\frac{ln(1-0.99)}{ln[1-\frac{2.0\times 10^{4}basepairs}{3.0\times 10^{9}%
basepairs}]}
  </annotation>
 </semantics>
</math>

 

<math display="inline" id="Genomic_library:10">
 <semantics>
  <mrow>
   <mi>N</mi>
   <mo>=</mo>
   <mfrac>
    <mrow>
     <mo>-</mo>
     <mn>4.61</mn>
    </mrow>
    <mrow>
     <mo>-</mo>
     <mrow>
      <mn>6.7</mn>
      <mo>×</mo>
      <msup>
       <mn>10</mn>
       <mrow>
        <mo>-</mo>
        <mn>6</mn>
       </mrow>
      </msup>
     </mrow>
    </mrow>
   </mfrac>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <eq></eq>
    <ci>N</ci>
    <apply>
     <divide></divide>
     <apply>
      <minus></minus>
      <cn type="float">4.61</cn>
     </apply>
     <apply>
      <minus></minus>
      <apply>
       <times></times>
       <cn type="float">6.7</cn>
       <apply>
        <csymbol cd="ambiguous">superscript</csymbol>
        <cn type="integer">10</cn>
        <apply>
         <minus></minus>
         <cn type="integer">6</cn>
        </apply>
       </apply>
      </apply>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   N=\frac{-4.61}{-6.7\times 10^{-6}}
  </annotation>
 </semantics>
</math>

 

<math display="inline" id="Genomic_library:11">
 <semantics>
  <mrow>
   <mi>N</mi>
   <mo>=</mo>
   <mrow>
    <mn>688</mn>
    <mo>,</mo>
    <mn>060</mn>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <eq></eq>
    <ci>N</ci>
    <list>
     <cn type="integer">688</cn>
     <cn type="integer">060</cn>
    </list>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   N=688,060
  </annotation>
 </semantics>
</math>

 clones</p>

<p>Thus, approximately 688,060 clones are required to ensure a 99% probability that a given DNA sequence from this three billion basepair genome will be present in a library using a vector with an insert size of twenty thousand basepairs.</p>
<h2 id="applications">Applications</h2>

<p>After a library is created, the genome of an organism can be <a href="Sequencing#DNA_sequencing" title="wikilink">sequenced</a> to elucidate how genes affect an organism or to compare similar organisms at the genome-level. The aforementioned <a href="Genome-wide_association_study" title="wikilink">genome-wide association studies</a> can identify candidate genes stemming from many functional traits. Genes can be isolated through genomic libraries and used on human cell lines or animal models to further research.<a class="footnoteRef" href="#fn33" id="fnref33"><sup>33</sup></a> Furthermore, creating high-fidelity clones with accurate genome representation- and no stability issues- would contribute well as intermediates for <a href="shotgun_sequencing" title="wikilink">shotgun sequencing</a> or the study of complete genes in functional analysis.<a class="footnoteRef" href="#fn34" id="fnref34"><sup>34</sup></a></p>
<h3 id="hierarchical-sequencing">Hierarchical sequencing</h3>

<p> One major use of genomic libraries is <a href="Shotgun_sequencing#Hierarchical_Shotgun_sequencing" title="wikilink">hierarchichal shotgun sequencing</a>, which is also called top-down, map-based or clone-by-clone sequencing. This strategy was developed in the 1980s for sequencing whole genomes before high throughput techniques for sequencing were available. Individual clones from genomic libraries can be sheared into smaller fragments, usually 500bp to 1000bp, which are more manageable for sequencing.<a class="footnoteRef" href="#fn35" id="fnref35"><sup>35</sup></a> Once a clone from a genomic library is sequenced, the sequence can be used to screen the library for other clones containing inserts which overlap with the sequenced clone. Any new overlapping clones can then be sequenced forming a <a class="uri" href="contig" title="wikilink">contig</a>. This technique, called <a href="Primer_walking" title="wikilink">chromosome walking</a>, can be exploited to sequence entire chromosomes.<a class="footnoteRef" href="#fn36" id="fnref36"><sup>36</sup></a></p>

<p><a href="Shotgun_sequencing#Whole_genome_shotgun_sequencing" title="wikilink">Whole genome shotgun sequencing</a> is another method of genome sequencing that does not require a library of high-capacity vectors. Rather, it uses computer algorithms to assemble short sequence reads to cover the entire genome. Genomic libraries are often used in combination with whole genome shotgun sequencing for this reason. A high resolution map can be created by sequencing both ends of inserts from several clones in a genomic library. This map provides sequences of known distances apart, which can be used to help with the assembly of sequence reads acquired through shotgun sequencing.<a class="footnoteRef" href="#fn37" id="fnref37"><sup>37</sup></a> The human genome sequence, which was declared complete in 2003, was assembled using both a BAC library and shotgun sequencing.<a class="footnoteRef" href="#fn38" id="fnref38"><sup>38</sup></a><a class="footnoteRef" href="#fn39" id="fnref39"><sup>39</sup></a></p>
<h3 id="genome-wide-association-studies">Genome-wide association studies</h3>

<p><a href="Genome-wide_association_study" title="wikilink">Genome-wide association studies</a> are general applications to find specific gene targets and polymorphisms within the human race. In fact, the International HapMap project was created through a partnership of scientists and agencies from several countries to catalog and utilize this data.<a class="footnoteRef" href="#fn40" id="fnref40"><sup>40</sup></a> The goal of this project is to compare genetic sequences of different individuals to elucidate similarities and differences within chromosomal regions.<a class="footnoteRef" href="#fn41" id="fnref41"><sup>41</sup></a> Scientists from all of the participating nations are cataloging these attributes with data from populations of African, Asian, and European ancestry. Such genome-wide assessments may lead to further diagnostic and drug therapies while also helping future teams focus on orchestrating therapeutics with genetic features in mind. These concepts are already being exploited in <a href="genetic_engineering" title="wikilink">genetic engineering</a>.<a class="footnoteRef" href="#fn42" id="fnref42"><sup>42</sup></a> For example, a research team has actually constructed a PAC shuttle vector that creates a library representing two-fold coverage of the human genome.<a class="footnoteRef" href="#fn43" id="fnref43"><sup>43</sup></a> This could serve as an incredible resource to identify genes, or sets of genes, causing disease. Moreover, these studies can serve as a powerful way to investigate transcriptional regulation as it has been seen in the study of baculoviruses.<a class="footnoteRef" href="#fn44" id="fnref44"><sup>44</sup></a> Overall, advances in genome library construction and DNA sequencing has allowed for efficient discovery of different molecular targets.<a class="footnoteRef" href="#fn45" id="fnref45"><sup>45</sup></a> Assimilation of these features through such efficient methods can hasten the employment of novel drug candidates.</p>
<h2 id="references">References</h2>
<h2 id="further-reading">Further reading</h2>
<h2 id="external-links">External links</h2>
<ul>
<li><a href="http://www.cd-genomics.com/customer/shotgun.htm">Custom genomic DNA library construction</a></li>
<li><a href="http://www.biost.com/page/Arrayed.aspx">Genomic BAC library construction</a></li>
</ul>

<p><a href="it:Libreria_di_DNA_complementare" title="wikilink">it:Libreria di DNA complementare</a> <a href="pl:Biblioteka_cDNA" title="wikilink">pl:Biblioteka cDNA</a> <a class="uri" href="zh:CDNA文庫" title="wikilink">zh:CDNA文庫</a>"</p>

<p><a href="Category:Molecular_biology" title="wikilink">Category:Molecular biology</a> <a class="uri" href="Category:DNA" title="wikilink">Category:DNA</a> <a class="uri" href="Category:Genetics" title="wikilink">Category:Genetics</a></p>
<section class="footnotes">
<hr/>
<ol>
<li id="fn1"><a href="#fnref1">↩</a></li>
<li id="fn2"><a href="#fnref2">↩</a></li>
<li id="fn3"><a href="#fnref3">↩</a></li>
<li id="fn4"><a href="#fnref4">↩</a></li>
<li id="fn5"><a href="#fnref5">↩</a></li>
<li id="fn6"><a href="#fnref6">↩</a></li>
<li id="fn7"><a href="#fnref7">↩</a></li>
<li id="fn8"><a href="#fnref8">↩</a></li>
<li id="fn9"></li>
<li id="fn10"><a href="#fnref10">↩</a></li>
<li id="fn11"><a href="#fnref11">↩</a></li>
<li id="fn12"><a href="#fnref12">↩</a></li>
<li id="fn13"><a href="#fnref13">↩</a></li>
<li id="fn14"><a href="#fnref14">↩</a></li>
<li id="fn15"></li>
<li id="fn16"></li>
<li id="fn17"></li>
<li id="fn18"></li>
<li id="fn19"><a href="#fnref19">↩</a></li>
<li id="fn20"></li>
<li id="fn21"></li>
<li id="fn22"></li>
<li id="fn23"></li>
<li id="fn24"></li>
<li id="fn25"></li>
<li id="fn26"></li>
<li id="fn27"></li>
<li id="fn28"></li>
<li id="fn29"></li>
<li id="fn30"><a class="uri" href="http://users.wmin.ac.uk/~redwayk/lectures/gene_libraries.htm">http://users.wmin.ac.uk/~redwayk/lectures/gene_libraries.htm</a><a href="#fnref30">↩</a></li>
<li id="fn31"><a href="#fnref31">↩</a></li>
<li id="fn32"></li>
<li id="fn33"><a href="#fnref33">↩</a></li>
<li id="fn34"></li>
<li id="fn35"></li>
<li id="fn36"></li>
<li id="fn37"></li>
<li id="fn38"><a href="#fnref38">↩</a></li>
<li id="fn39"><a href="#fnref39">↩</a></li>
<li id="fn40"><a href="#fnref40">↩</a></li>
<li id="fn41"></li>
<li id="fn42"></li>
<li id="fn43"></li>
<li id="fn44"><a href="#fnref44">↩</a></li>
<li id="fn45"></li>
</ol>
</section>
</hr></body>
</html>
