<html lang="en">
<head>
<meta charset="utf-8"/>
<title offset="570">Genome size</title>
<script src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_SVG.js" type="text/javascript">
</script>
</head>
<body>
<h1>Genome size</h1>
<hr/>

<p> <strong>Genome size</strong> is the total amount of <a class="uri" href="DNA" title="wikilink">DNA</a> contained within one copy of a single <a class="uri" href="genome" title="wikilink">genome</a>. It is typically measured in terms of <a class="uri" href="mass" title="wikilink">mass</a> in picograms (trillionths (10<sup>−12</sup>) of a <a class="uri" href="gram" title="wikilink">gram</a>, abbreviated pg) or less frequently in <a href="Dalton_(unit)" title="wikilink">Daltons</a> or as the total number of <a class="uri" href="nucleotide" title="wikilink">nucleotide</a> <a href="base_pair" title="wikilink">base pairs</a> typically in megabases (millions of base pairs, abbreviated Mb or Mbp). One picogram equals 978 megabases.<a class="footnoteRef" href="#fn1" id="fnref1"><sup>1</sup></a> In <a class="uri" href="diploid" title="wikilink">diploid</a> <a href="organism" title="wikilink">organisms</a>, genome size is used interchangeably with the term <a class="uri" href="C-value" title="wikilink">C-value</a>. An organism's complexity is not directly proportional to its genome size; some single cell organisms have much more DNA than humans (see <a href="Junk_DNA" title="wikilink">Junk DNA</a> and <a href="C-value_enigma" title="wikilink">C-value enigma</a>).</p>
<h2 id="origin-of-the-term">Origin of the term</h2>

<p> The term "genome size" is often erroneously attributed to Hinegardner,<a class="footnoteRef" href="#fn2" id="fnref2"><sup>2</sup></a> even in discussions dealing specifically with terminology in this area of research (e.g., Greilhuber, 2005<a class="footnoteRef" href="#fn3" id="fnref3"><sup>3</sup></a>). Notably, Hinegardner<a class="footnoteRef" href="#fn4" id="fnref4"><sup>4</sup></a> used the term only once: in the title. The term actually seems to have first appeared in 1968 when Hinegardner wondered, in the last paragraph of his article, whether "<a href="Cell_(biology)" title="wikilink">cellular</a> DNA content does, in fact, reflect genome size".<a class="footnoteRef" href="#fn5" id="fnref5"><sup>5</sup></a> In this context, "genome size" was being used in the sense of <a class="uri" href="genotype" title="wikilink">genotype</a> to mean the number of <a href="gene" title="wikilink">genes</a>. In a paper submitted only two months later (in February 1969), Wolf et al. (1969)<a class="footnoteRef" href="#fn6" id="fnref6"><sup>6</sup></a> used the term "genome size" throughout and in its present usage; therefore these authors should probably be credited with originating the term in its modern sense. By the early 1970s, "genome size" was in common usage with its present definition, probably as a result of its inclusion in <a href="Susumu_Ohno" title="wikilink">Susumu Ohno</a>'s influential book <em>Evolution by Gene Duplication</em>, published in 1970.<a class="footnoteRef" href="#fn7" id="fnref7"><sup>7</sup></a></p>
<h2 id="variation-in-genome-size-and-gene-content">Variation in genome size and gene content</h2>

<p>The genome sizes of thousands of <a href="eukaryote" title="wikilink">eukaryotes</a> have been analyzed over the past 50 years, and these data are available in online databases for animals, plants, and fungi (see external links). Nuclear genome size is typically measured in eukaryotes using either <a href="Densitometry" title="wikilink">densitometric</a> measurements of <a class="uri" href="Feulgen" title="wikilink">Feulgen</a>-stained nuclei (previously using specialized densitometers, now more commonly using computerized <a href="image_analysis" title="wikilink">image analysis</a><a class="footnoteRef" href="#fn8" id="fnref8"><sup>8</sup></a>) or <a href="flow_cytometry" title="wikilink">flow cytometry</a>. In <a href="prokaryote" title="wikilink">prokaryotes</a>, <a href="pulsed_field_gel_electrophoresis" title="wikilink">pulsed field gel electrophoresis</a> and complete <a href="genome_sequencing" title="wikilink">genome sequencing</a> are the predominant methods of genome size determination. Nuclear genome sizes are well known to vary enormously among eukaryotic species. In animals they range more than 3,300-fold, and in land plants they differ by a factor of about 1,000.<a class="footnoteRef" href="#fn9" id="fnref9"><sup>9</sup></a><a class="footnoteRef" href="#fn10" id="fnref10"><sup>10</sup></a> <a class="uri" href="Protist" title="wikilink">Protist</a> genomes have been reported to vary more than 300,000-fold in size, but the high end of this range (<em><a href="Amoeba_(genus)" title="wikilink">Amoeba</a></em>) has been called into question. In eukaryotes (but not prokaryotes), variation in genome size is not proportional to the number of <a href="gene" title="wikilink">genes</a>, an observation that was deemed wholly counterintuitive before the discovery of <a href="non-coding_DNA" title="wikilink">non-coding DNA</a> and which became known as the <a href="C-value_paradox" title="wikilink">C-value paradox</a> as a result. However, although there is no longer any <a href="paradox" title="wikilink">paradoxical</a> aspect to the discrepancy between genome size and gene number, this term remains in common usage. For reasons of conceptual clarification, the various puzzles that remain with regard to genome size variation instead have been suggested by one author to more accurately comprise a puzzle or an enigma (the <a href="C-value_enigma" title="wikilink">C-value enigma</a>). Genome size correlates with a range of features at the <a href="Cell_(biology)" title="wikilink">cell</a> and organism levels, including cell size, <a href="cell_division" title="wikilink">cell division</a> rate, and, depending on the <a class="uri" href="taxon" title="wikilink">taxon</a>, body size, <a href="metabolic_rate" title="wikilink">metabolic rate</a>, <a href="developmental_rate" title="wikilink">developmental rate</a>, <a href="Organ_(anatomy)" title="wikilink">organ</a> complexity, geographical distribution, or <a class="uri" href="extinction" title="wikilink">extinction</a> risk (for recent reviews, see Bennett and Leitch 2005;<a class="footnoteRef" href="#fn11" id="fnref11"><sup>11</sup></a> Gregory 2005<a class="footnoteRef" href="#fn12" id="fnref12"><sup>12</sup></a>). Based on completely sequenced genome data currently (as of April 2009) available, log-transformed gene number forms a linear correlation with log-transformed genome size in bacteria, archea, viruses, and organelles combined whereas a nonlinear (semi-natural log) correlation in eukaryotes (Hou and Lin 2009 <a class="footnoteRef" href="#fn13" id="fnref13"><sup>13</sup></a>). The nonlinear correlation for eukaryotes, although claim of its existence contrasts the previous view that no correlation exists for this group of organisms, reflects disproportionately fast increasing noncoding DNA in increasingly large eukaryotic genomes. Although sequenced genome data are practically biased toward small genomes, which may compromise the accuracy of the empirically derived correlation, and the ultimate proof of the correlation remains to be obtained by sequencing some of the largest eukaryotic genomes, current data do not seem to rule out a correlation.</p>
<h2 id="genome-reduction">Genome reduction</h2>

<p> <strong>Genome reduction</strong>, also known as <strong><a href="bacterial_genome_size" title="wikilink">Genome degradation</a></strong>, is the process by which a <a class="uri" href="genome" title="wikilink">genome</a> shrinks relative to its ancestor. Genomes fluctuate in size regularly, however, <a href="bacterial_genome_size" title="wikilink">genome size reduction</a> is most significant in bacteria.</p>

<p>The most <a href="evolution" title="wikilink">evolutionary</a> significant cases of genome reduction may be the <a class="uri" href="eukaryotic" title="wikilink">eukaryotic</a> <a href="organelle" title="wikilink">organelles</a> that are derived from bacteria: the <a class="uri" href="mitochondrion" title="wikilink">mitochondrion</a> and <a class="uri" href="plastid" title="wikilink">plastid</a>. These organelles are descended from <a href="endosymbiont" title="wikilink">endosymbionts</a>, which can only survive within the host cell and which the host cell likewise needs for survival. Many mitochondria have less than 20 genes in their entire genome, whereas a free-living bacterium generally has at least 1000 genes. Many genes have been transferred to the host <a href="cell_nucleus" title="wikilink">nucleus</a>, while others have simply been lost and their function replaced by host processes.</p>

<p>Other bacteria have become endosymbionts or obligate intracellular <a href="pathogen" title="wikilink">pathogens</a> and experienced extensive genome reduction as a result. This process seems to be dominated by <a href="genetic_drift" title="wikilink">genetic drift</a> resulting from small <a href="population_genetics" title="wikilink">population</a> size, low <a href="genetic_recombination" title="wikilink">recombination</a> rates, and high <a class="uri" href="mutation" title="wikilink">mutation</a> rates, as opposed to <a href="natural_selection" title="wikilink">selection</a> for smaller genomes.</p>

<p>Some free-living marine bacterioplanktons also shows signs of genome reduction, which are hypothesized to be driven by natural selection.<a class="footnoteRef" href="#fn14" id="fnref14"><sup>14</sup></a><a class="footnoteRef" href="#fn15" id="fnref15"><sup>15</sup></a><a class="footnoteRef" href="#fn16" id="fnref16"><sup>16</sup></a></p>
<h3 id="genome-reduction-in-obligate-endosymbiotic-species">Genome reduction in obligate endosymbiotic species</h3>

<p><strong>Obligate endosymbiotic species</strong> are characterized by a complete inability to survive external to their <strong><a href="Host_(biology)" title="wikilink">host</a></strong> environment. These species have become a considerable threat to human health, as they are often highly capable of evading human immune systems and manipulating the host environment to acquire nutrients. A common explanation for these keen manipulative abilities is the <strong>compact</strong> and <strong>efficient</strong> genomic structure consistently found in obligate endosymbionts. This compact genome structure is the result of massive losses of extraneous DNA - an occurrence that is exclusively associated with the loss of a free-living stage. In fact, as much as 90% of the genetic material can be lost when a species makes the evolutionary transition from a <strong>free-living</strong> to <strong>obligate intracellular</strong> lifestyle. Common examples of species with reduced genomes include: <em><a href="Buchnera_aphidicola" title="wikilink">Buchnera aphidicola</a></em>, <em><a class="uri" href="Rickettsia" title="wikilink">Rickettsia</a> prowazekii</em> and <em><a href="Mycobacterium_leprae" title="wikilink">Mycobacterium leprae</a></em>. One obligate endosymbiont of <a class="uri" href="leafhoppers" title="wikilink">leafhoppers</a>, <em><a href="Nasuia_deltocephalinicola" title="wikilink">Nasuia deltocephalinicola</a></em>, has the smallest genome currently known among cellular organisms at 112kb.<a class="footnoteRef" href="#fn17" id="fnref17"><sup>17</sup></a> It is important to note, however, that some obligate intracellular species have positive fitness effects on their hosts. (See also <a href="mutualist_(biology)" title="wikilink">mutualists</a> and <a href="parasite" title="wikilink">parasites</a>.)</p>

<p>The <strong>reductive evolution model</strong> has been proposed as an effort to define the genomic commonalities seen in all obligate endosymbionts.<a class="footnoteRef" href="#fn18" id="fnref18"><sup>18</sup></a> This model illustrates four general features of reduced genomes and obligate intracellular species:</p>
<ol>
<li>‘genome streamlining’ resulting from relaxed selection on genes that are superfluous in the intracellular environment;</li>
<li>a bias towards deletions (rather than insertions), which heavily affects genes that have been disrupted by accumulation of mutations (<a href="pseudogene" title="wikilink">pseudogenes</a>);<a class="footnoteRef" href="#fn19" id="fnref19"><sup>19</sup></a></li>
<li>very little or no capability for acquiring new DNA; and</li>
<li>considerable reduction of <a href="effective_population_size" title="wikilink">effective population size</a> in endosymbiotic populations, particularly in species that rely on vertical transmission.</li>
</ol>

<p>Based on this model, it is clear that endosymbionts face different adaptive challenges than free-living species.</p>
<h2 id="conversion-from-picograms-pg-to-base-pairs-bp">Conversion from picograms (pg) to base pairs (bp)</h2>

<p>

<math display="block" id="Genome_size:0">
 <semantics>
  <mrow>
   <mtext>number of base pairs</mtext>
   <mo>=</mo>
   <mrow>
    <mtext>mass in pg</mtext>
    <mo>×</mo>
    <mn>0.978</mn>
    <mo>×</mo>
    <msup>
     <mn>10</mn>
     <mn>9</mn>
    </msup>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <eq></eq>
    <mtext>number of base pairs</mtext>
    <apply>
     <times></times>
     <mtext>mass in pg</mtext>
     <cn type="float">0.978</cn>
     <apply>
      <csymbol cd="ambiguous">superscript</csymbol>
      <cn type="integer">10</cn>
      <cn type="integer">9</cn>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \text{number of base pairs}=\text{mass in pg}\times 0.978\times 10^{9}
  </annotation>
 </semantics>
</math>

</p>

<p>or simply:</p>

<p>

<math display="block" id="Genome_size:1">
 <semantics>
  <mrow>
   <mrow>
    <mn>1</mn>
    <mtext>pg</mtext>
   </mrow>
   <mo>=</mo>
   <mrow>
    <mn>978</mn>
    <mtext>Mb</mtext>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <eq></eq>
    <apply>
     <times></times>
     <cn type="integer">1</cn>
     <mtext>pg</mtext>
    </apply>
    <apply>
     <times></times>
     <cn type="integer">978</cn>
     <mtext>Mb</mtext>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   1\text{pg}=978\text{Mb}
  </annotation>
 </semantics>
</math>

<a class="footnoteRef" href="#fn20" id="fnref20"><sup>20</sup></a></p>
<h2 id="drakes-rule">Drake's rule</h2>

<p>In 1991 Drake proposed a rule: that the mutation rate within a genome and its size were inversely correlated.<a class="footnoteRef" href="#fn21" id="fnref21"><sup>21</sup></a> This rule has been found to be approximately correct for DNA viruses and unicellular organisms. Its basis is unknown.</p>

<p>The small size of RNA viruses has been proposed to be locked into a three part relation between replication fidelity, genome size and genetic complexity. The majority of RNA viruses lack an RNA proofreading facility which limits their replication fidelity and hence the genome size. This has also been described as the Eigen paradox.<a class="footnoteRef" href="#fn22" id="fnref22"><sup>22</sup></a></p>

<p>An exception to the rule of small genome sizes in RNA viruses is found the <a href="Nidovirales" title="wikilink">Nidoviruses</a>. These viruses appear to have acquired a 3′-to-5′ exoribonuclease (ExoN) which has allowed for an increase in genome size.<a class="footnoteRef" href="#fn23" id="fnref23"><sup>23</sup></a></p>
<h2 id="see-also">See also</h2>
<ul>
<li><a href="Genome#Genome_size" title="wikilink">Comparison of different genome sizes</a></li>
<li><a href="Animal_Genome_Size_Database" title="wikilink">Animal Genome Size Database</a></li>
<li><a href="Cell_nucleus" title="wikilink">Cell nucleus</a></li>
<li><a href="Comparative_genomics" title="wikilink">Comparative genomics</a></li>
<li><a class="uri" href="C-value" title="wikilink">C-value</a></li>
<li><a href="C-value_enigma" title="wikilink">C-value enigma</a></li>
<li><a class="uri" href="Genome" title="wikilink">Genome</a></li>
<li><a href="Human_genome" title="wikilink">Human genome</a></li>
<li><a href="Junk_DNA" title="wikilink">Junk DNA</a></li>
<li><a href="List_of_sequenced_eukaryotic_genomes" title="wikilink">List of sequenced eukaryotic genomes</a></li>
<li><a href="Noncoding_DNA" title="wikilink">Noncoding DNA</a></li>
<li><a href="Plant_DNA_C-values_Database" title="wikilink">Plant DNA C-values Database</a></li>
<li><a href="Selfish_DNA" title="wikilink">Selfish DNA</a></li>
<li><a href="Transposable_elements" title="wikilink">Transposable elements</a></li>
<li><a href="Bacterial_Genome_Size" title="wikilink">Bacterial Genome Size</a></li>
</ul>
<h2 id="references">References</h2>
<h3 id="further-reading">Further reading</h3>
<ul>
<li><a href="http://www.chlamydiae.com/docs/Chlamydiales/ev_genomedegradn.asp">Evolution of Chlamydiaceae</a></li>
<li></li>
</ul>
<h2 id="external-links">External links</h2>
<ul>
<li><a href="http://www.genomesize.com">Animal Genome Size Database</a></li>
<li><a href="http://www.rbgkew.org.uk/cval/homepage.html">Plant DNA C-values Database</a></li>
<li><a href="http://www.zbi.ee/fungal-genomesize/index.php">Fungal Genome Size Database</a></li>
<li><a href="http://www.cbs.dtu.dk/services/FD/">Fungal Database</a> — by CBS</li>
</ul>

<p>"</p>

<p><a class="uri" href="Category:Genomics" title="wikilink">Category:Genomics</a></p>
<section class="footnotes">
<hr/>
<ol>
<li id="fn1"><a href="#fnref1">↩</a></li>
<li id="fn2"><a href="#fnref2">↩</a></li>
<li id="fn3"><a href="#fnref3">↩</a></li>
<li id="fn4"></li>
<li id="fn5"><a href="#fnref5">↩</a></li>
<li id="fn6"><a href="#fnref6">↩</a></li>
<li id="fn7"><a href="#fnref7">↩</a></li>
<li id="fn8"><a href="#fnref8">↩</a></li>
<li id="fn9"><a href="#fnref9">↩</a></li>
<li id="fn10"><a href="#fnref10">↩</a></li>
<li id="fn11"></li>
<li id="fn12"></li>
<li id="fn13"><a href="#fnref13">↩</a></li>
<li id="fn14"><a href="#fnref14">↩</a></li>
<li id="fn15"><a href="#fnref15">↩</a></li>
<li id="fn16"><a href="#fnref16">↩</a></li>
<li id="fn17"><a href="http://phenomena.nationalgeographic.com/2013/08/23/and-the-genomes-keep-shrinking/">And the Genomes Keep Shrinking…</a><a href="#fnref17">↩</a></li>
<li id="fn18"><a href="#fnref18">↩</a></li>
<li id="fn19"><a href="#fnref19">↩</a></li>
<li id="fn20"></li>
<li id="fn21"><a href="#fnref21">↩</a></li>
<li id="fn22"><a href="#fnref22">↩</a></li>
<li id="fn23"><a href="#fnref23">↩</a></li>
</ol>
</section>
</body>
</html>
