<html lang="en">
<head>
<meta charset="utf-8"/>
<title offset="1650">Glutamate racemase</title>
<script src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_SVG.js" type="text/javascript">
</script>
</head>
<body>
<h1>Glutamate racemase</h1>
<hr/>

<p>In enzymology, <strong>glutamate racemase</strong> (<strong>MurI</strong> with a capital <em>i</em>) () is an <a class="uri" href="enzyme" title="wikilink">enzyme</a> that <a href="catalysis" title="wikilink">catalyzes</a> the <a href="chemical_reaction" title="wikilink">chemical reaction</a></p>
<dl>
<dd>L-glutamate 

<math display="inline" id="Glutamate_racemase:0">
 <semantics>
  <mo>⇌</mo>
  <annotation-xml encoding="MathML-Content">
   <ci>normal-⇌</ci>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \rightleftharpoons
  </annotation>
 </semantics>
</math>

 D-glutamate
</dd>
</dl>

<p>Hence, this enzyme RacE has one <a href="substrate_(biochemistry)" title="wikilink">substrate</a>, <a href="glutamic_acid" title="wikilink">L-glutamate</a>, and one <a href="product_(chemistry)" title="wikilink">product</a>, D-glutamate.</p>

<p>This enzyme belongs to the family of <a href="isomerase" title="wikilink">isomerases</a>, specifically those <a href="racemase" title="wikilink">racemases</a> and <a href="epimerase" title="wikilink">epimerases</a> acting on <a href="amino_acid" title="wikilink">amino acids</a> and derivatives, including proline racemase, aspartate racemase, and diaminopimelate epimerase.<a class="footnoteRef" href="#fn1" id="fnref1"><sup>1</sup></a> This enzyme participates in <a href="Glutamic_acid#Metabolism" title="wikilink">glutamate metabolism</a> that is essential for <a href="cell_wall" title="wikilink">cell wall</a> <a class="uri" href="biosynthesis" title="wikilink">biosynthesis</a> in bacteria.<a class="footnoteRef" href="#fn2" id="fnref2"><sup>2</sup></a> Glutamate racemase performs the additional function of <a class="uri" href="gyrase" title="wikilink">gyrase</a> inhibition, preventing gyrase from binding to DNA.<a class="footnoteRef" href="#fn3" id="fnref3"><sup>3</sup></a></p>

<p>Glutamate racemase (MurI) serves two distinct metabolic functions: primarily, it is a critical enzyme in cell wall biosynthesis,<a class="footnoteRef" href="#fn4" id="fnref4"><sup>4</sup></a> but also plays a role in <a class="uri" href="gyrase" title="wikilink">gyrase</a> inhibition.<a class="footnoteRef" href="#fn5" id="fnref5"><sup>5</sup></a> The ability of glutamate racemase and other proteins to serve two distinct functions is known as "<a href="protein_moonlighting" title="wikilink">moonlighting</a>".</p>
<h2 id="moonlighting-background">Moonlighting background</h2>

<p>Before the discovery of <a href="protein_moonlighting" title="wikilink">moonlighting proteins</a>, it was generally believed by scientists that an <a class="uri" href="enzyme" title="wikilink">enzyme</a> only had one function which lead to the concept of "one gene, one enzyme". However, this concept no longer applies in science after the discovery that some proteins consist of both major and minor functions. This led to numerous studies attempting to relate the two functions to each other. The minor functions of these unique enzymes are called moonlighting functions, in which a protein can have a secondary functions not dependent upon the main function. These two functions of the moonlighting protein are found in a single <a href="polypeptide_chain" title="wikilink">polypeptide chain</a>. Proteins that are multifunctional are not included due to gene fusion, families of homologous proteins, splice variants or promiscuous enzyme activities. The enzyme glutamate racemase (MurI) is an example of a moonlighting protein, functioning both in bacterial <a href="cell_wall" title="wikilink">cell wall</a> biosynthesis as well as in gyrase inhibition.</p>
<h2 id="structure">Structure</h2>

<p>The dimensions of MurI is approximately 35 Å × 40 Å × 45 Å and consists of two compact domains of <a href="α/β_proteins" title="wikilink">α/β structure</a>.<a class="footnoteRef" href="#fn6" id="fnref6"><sup>6</sup></a><a class="footnoteRef" href="#fn7" id="fnref7"><sup>7</sup></a> With the active site in between the two domains, the <a href="N-terminus" title="wikilink">N-terminal</a> domain contains residues 1-97 and 207-264 while the <a href="C-terminus" title="wikilink">C-terminal</a> domain includes residues 98-206.<a class="footnoteRef" href="#fn8" id="fnref8"><sup>8</sup></a> This allows the enzyme to produce L-isomer from D-glutamate. Also, the N-domain is composed of five-stranded <a href="beta_sheet" title="wikilink">β-sheets</a> compared to four-stranded β-sheets of C-domain.<a class="footnoteRef" href="#fn9" id="fnref9"><sup>9</sup></a> These structural specifications are not identical between MurI of different species; <em><a href="Streptococcus_pyogenes" title="wikilink">S. pyogenes</a></em> and <em><a href="Bacillus_subtilis" title="wikilink">B. subtilis</a></em> actually possess the most structurally similar MurI enzymes yet found. It is also not rare to find MurI as a <a href="protein_dimer" title="wikilink">dimer</a>.</p>

<p>The active site, as it is evenly between the N-domain and C-domain, is also between the two <a class="uri" href="cysteine" title="wikilink">cysteine</a> residues. It is accessible to solvents, as several water molecules, such as W1, are found in the active site. In some species, the active site also incorporates <a class="uri" href="sulfate" title="wikilink">sulfate</a> ions to undergo hydrogen bonding on the <a href="peptide_bond" title="wikilink">amide backbone</a> and the <a href="side_chain" title="wikilink">side chains</a>.<a class="footnoteRef" href="#fn10" id="fnref10"><sup>10</sup></a><a class="footnoteRef" href="#fn11" id="fnref11"><sup>11</sup></a></p>
<h2 id="function">Function</h2>
<h3 id="bacterial-wall-synthesis">Bacterial wall synthesis</h3>

<p>Glutamate racemase is a bacterial enzyme that is encoded by the <em>murI</em> <a class="uri" href="gene" title="wikilink">gene</a>. This enzyme is most commonly known as being responsible for the synthesis of bacterial cell walls. Through experimentation it was found that this enzyme is able to construct these cell walls by synthesizing D-glutamate from L-glutamate through <a class="uri" href="racemization" title="wikilink">racemization</a>.<a class="footnoteRef" href="#fn12" id="fnref12"><sup>12</sup></a> D-glutamate is a monomer of the <a class="uri" href="peptidoglycan" title="wikilink">peptidoglycan</a> layer in prokaryotic cell walls. Peptidoglycan is an essential structural component of the bacterial cell wall. The peptidoglycan layer is also responsible for the rigidity of the cell wall.<a class="footnoteRef" href="#fn13" id="fnref13"><sup>13</sup></a> This process, in which MurI helps catalyze the interconversion of glutamate enantiomers, like L-Glutamate, into the essential D-glutamate, is also cofactor independent. As such it can proceed without needing an additional source, which would bind to an allosteric site, altering the enzyme shape to assist in catalyzing the reaction.<a class="footnoteRef" href="#fn14" id="fnref14"><sup>14</sup></a> Murl involves a two-step process to catalyze the glutamate enantiomers to D-glutamate. The first step is a deprotonation of the substrate to form an anion.<a class="footnoteRef" href="#fn15" id="fnref15"><sup>15</sup></a> Subsequently the substrate gets reprotonated. Once the glutamate is in the active site of the enzyme it undergoes a very large conformational change of its domains. This change helps superimpose the two catalytic cysteine residues, Cys73 and Cys184, located on either sides of the substrate at equal positions. Those domains mentioned earlier are symmetric and this symmetry suggests that this racemase activity of the protein may have evolved from gene duplication.<a class="footnoteRef" href="#fn16" id="fnref16"><sup>16</sup></a> Due to this main function of biosynthesis of bacterial cell walls MurI has been targeted as an antibacterial in drug discovery.<a class="footnoteRef" href="#fn17" id="fnref17"><sup>17</sup></a></p>
<h3 id="gyrase-inhibition">Gyrase inhibition</h3>

<p>Along with its main function of cell wall biosynthesis, the moonlighting protein glutamate racemase also functions independently as a gyrase inhibitor.<a class="footnoteRef" href="#fn18" id="fnref18"><sup>18</sup></a> Present in certain forms of bacteria, MurI reduces the activity of DNA gyrase by preventing gyrase from binding to DNA.<a class="footnoteRef" href="#fn19" id="fnref19"><sup>19</sup></a> When gyrase binds to DNA, the enzyme decreases the tension in the DNA strands as they are unwound and causes the strands to become supercoiled.<a class="footnoteRef" href="#fn20" id="fnref20"><sup>20</sup></a> This is a critical step in DNA replication in these cells which results in the reproduction of bacterial cells.<a class="footnoteRef" href="#fn21" id="fnref21"><sup>21</sup></a> The presence of glutamate racemase in the process inhibits gyrase from effectively binding to DNA by deforming the shape of the enzyme’s active site. It essentially disallows gyrase from catalyzing the reaction that coils unwinding DNA strands.<a class="footnoteRef" href="#fn22" id="fnref22"><sup>22</sup></a></p>

<p>This function of MurI was discovered experimentally. DNA gyrase was incubated with the MurI enzyme and then added to a sample of DNA; the results of this experiment showed inhibition of supercoiling activity when MurI was present.<a class="footnoteRef" href="#fn23" id="fnref23"><sup>23</sup></a> The cell wall biosynthesis function of MurI is not directly related to its moonlighting function. MurI’s ability to inhibit gyrase binding can proceed independently of its main function.<a class="footnoteRef" href="#fn24" id="fnref24"><sup>24</sup></a> This means that DNA gyrase, in turn, will not have any effect on MurI's racemization, which was confirmed in a study of the racemization with and without the presence of DNA gyrase.<a class="footnoteRef" href="#fn25" id="fnref25"><sup>25</sup></a> In an experimental analysis, it was determined that MurI employs the use of two different enzymatic active sites for its two functions. This was shown by the inclusion of the racemase substrate L-glutamate in an assay with the separated gyrase inhibition site. The gyrase inhibition occurs in both supercoiling and relaxing activities of the DNA gyrase, and the study concluded that the inhibition activity was able to proceed, unchanged, in the presence of the racemase substrate.<a class="footnoteRef" href="#fn26" id="fnref26"><sup>26</sup></a> This dictates that the two functions can be carried out independently of each other, on non-overlapping sites, making MurI a true moonlighting protein.<a class="footnoteRef" href="#fn27" id="fnref27"><sup>27</sup></a> Mutant forms of MurI that are unable to exhibit their racemase function, no matter how compromised their racemase abilities were, were still proven through a study to be able to perform the DNA gyrase inhibition, with comparable results to a non-mutated form of MurI.<a class="footnoteRef" href="#fn28" id="fnref28"><sup>28</sup></a></p>
<h3 id="relationship-between-main-and-moonlighting-functions">Relationship between main and moonlighting functions</h3>

<p>Glutamate racemase (MurI) provides multiple functions for bacterial cells. MurI is an enzyme which is primarily known for its role in synthesizing bacterial cell walls. While performing the function of cell wall synthesis, MurI also acts as a gyrase inhibitor, preventing gyrase from binding to DNA. The two processes have been shown two be unrelated.<a class="footnoteRef" href="#fn29" id="fnref29"><sup>29</sup></a> In order to ascertain the effects of gyrase inhibition on cell wall synthesis, the efficiency of the conversion of D-glutamate to L-glutamate was measured while varying the concentration of DNA gyrase. Conversely, the effects of cell wall production on gyrase inhibition were discovered by varying the concentration of the racemization substrate.<a class="footnoteRef" href="#fn30" id="fnref30"><sup>30</sup></a> The results of these experiments conclude that there is no significant effect of racemization on gyrase inhibition or vice versa.<a class="footnoteRef" href="#fn31" id="fnref31"><sup>31</sup></a> The two functions of MurI act independently of each other reaffirming the fact that MurI is a moonlighting protein.</p>
<h3 id="relationship-to-active-site">Relationship to active site</h3>
<figure><b>(Figure)</b>
<figcaption>|thumb|300px|<a href="X-ray_crystallography#Biological_macromolecular_crystallography" title="wikilink">Crystallographic structure</a> of glutamate racemase from the bacteria <em><a href="Streptococcus_pyogenes" title="wikilink">S. pyogenes</a></em> (rainbow colored cartoon, <a class="uri" href="N-terminus" title="wikilink">N-terminus</a> = blue, <a class="uri" href="C-terminus" title="wikilink">C-terminus</a> = red) complexed with the inhibitor gamma-2-naphthylmethyl-D-glutamate (magenta <a href="space-filling_model" title="wikilink">spheres</a>; carbon = white, oxygen = red, nitrogen = blue) that occupies the active site.<a class="footnoteRef" href="#fn32" id="fnref32"><sup>32</sup></a> </figcaption>
</figure>

<p>Glutamate racemase is known to use its active site to undergo racemization and participate in the cell wall biosynthesis pathway of bacteria.<a class="footnoteRef" href="#fn33" id="fnref33"><sup>33</sup></a> Based on homology to other racemases and epimerases, glutamate racemase is thought to employ two active site cysteine <a href="Residue_(chemistry)" title="wikilink">residues</a> as acid/base catalysts.<a class="footnoteRef" href="#fn34" id="fnref34"><sup>34</sup></a> Surprisingly however, substituting either of the two residues with serine did not appreciable change the rate of the reaction significantly; the k<sub>cat</sub> value remained within .3% to 3% compared to the <a class="uri" href="wild-type" title="wikilink">wild-type</a> enzyme.<a class="footnoteRef" href="#fn35" id="fnref35"><sup>35</sup></a> From previous studies, it is most likely that the active site of MurI that performs racemization is not the same active site that undergoes gyrase inhibition. In order to ascertain the effects of gyrase inhibition on cell wall synthesis, the efficiency of the conversion of D-glutamate to L-glutamate was measured while varying the concentration of DNA gyrase. Conversely, the effects of cell wall production on gyrase inhibition were discovered by varying the concentration of the racemization substrate. It has been shown that the two functions are neutral to each other.<a class="footnoteRef" href="#fn36" id="fnref36"><sup>36</sup></a> In other words, racemization substrates are neutral to gyrase inhibition, and DNA gyrase has no effect on racemization. This explains how glutamate racemase in certain bacteria, such as Glr from <em>B. subtilis</em>, do not inhibit gyrase;<a class="footnoteRef" href="#fn37" id="fnref37"><sup>37</sup></a> if one active site is involved with both functions, this independence would not be possible. Consequently, a different site of MurI, distant from its active site, is involved in interacting with gyrase.<a class="footnoteRef" href="#fn38" id="fnref38"><sup>38</sup></a></p>
<h2 id="enzyme-regulation">Enzyme regulation</h2>

<p>This protein may use the <a class="uri" href="morpheein" title="wikilink">morpheein</a> model of <a href="allosteric_regulation" title="wikilink">allosteric regulation</a>. <a class="footnoteRef" href="#fn39" id="fnref39"><sup>39</sup></a></p>
<h2 id="application">Application</h2>

<p>Glutamate racemase has emerged as a potential antibacterial target since the product of this enzyme, D-glutamate, is an essential component of bacterial walls. Inhibiting the enzyme will prevent bacterial wall formation and ultimately result in lysis of the bacteria cell by osmotic pressure. Furthermore glutamate racemase is not expressed nor is the product of this enzyme, D-glutamate is normally found in mammals, hence inhibiting this enzyme should not result in toxicity to the mammalian host organism.<a class="footnoteRef" href="#fn40" id="fnref40"><sup>40</sup></a> Possible inhibitors to MurI includes aziridino-glutamate that would alkylate the catalytic cysteines; N-hydroxy glutamate that by mimicking Wat2 (the bound water molecule that interacts with glutamate amino group) would prevent binding of the substrate;<a class="footnoteRef" href="#fn41" id="fnref41"><sup>41</sup></a> or 4-substituted D-glutamic acid analogs bearing aryl-, heteroaryl-, cinnamyl-, or biaryl-methyl substituents that would also prevent binding of substrate.<a class="footnoteRef" href="#fn42" id="fnref42"><sup>42</sup></a></p>
<h2 id="references">References</h2>
<h2 id="further-reading">Further reading</h2>
<ul>
<li></li>
</ul>

<p>"</p>

<p><a href="Category:EC_5.1.1" title="wikilink">Category:EC 5.1.1</a> <a href="Category:Moonlighting_proteins" title="wikilink">Category:Moonlighting proteins</a></p>
<section class="footnotes">
<hr/>
<ol>
<li id="fn1"></li>
<li id="fn2"><a href="#fnref2">↩</a></li>
<li id="fn3"><a href="#fnref3">↩</a></li>
<li id="fn4"></li>
<li id="fn5"></li>
<li id="fn6"><a href="#fnref6">↩</a></li>
<li id="fn7"></li>
<li id="fn8"></li>
<li id="fn9"></li>
<li id="fn10"></li>
<li id="fn11"><a href="#fnref11">↩</a></li>
<li id="fn12"><a href="#fnref12">↩</a></li>
<li id="fn13"><a href="#fnref13">↩</a></li>
<li id="fn14"><a href="#fnref14">↩</a></li>
<li id="fn15"></li>
<li id="fn16"></li>
<li id="fn17"><a href="#fnref17">↩</a></li>
<li id="fn18"></li>
<li id="fn19"></li>
<li id="fn20"><a href="#fnref20">↩</a></li>
<li id="fn21"></li>
<li id="fn22"><a href="#fnref22">↩</a></li>
<li id="fn23"></li>
<li id="fn24"></li>
<li id="fn25"></li>
<li id="fn26"></li>
<li id="fn27"></li>
<li id="fn28"></li>
<li id="fn29"></li>
<li id="fn30"></li>
<li id="fn31"></li>
<li id="fn32">; <a href="#fnref32">↩</a></li>
<li id="fn33"></li>
<li id="fn34"></li>
<li id="fn35"></li>
<li id="fn36"></li>
<li id="fn37"><a href="#fnref37">↩</a></li>
<li id="fn38"></li>
<li id="fn39"><a href="#fnref39">↩</a></li>
<li id="fn40"></li>
<li id="fn41"></li>
<li id="fn42"></li>
</ol>
</section>
</body>
</html>
