<html lang="en">
<head>
<meta charset="utf-8"/>
<title offset="1687">Graphlets</title>
<script src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_SVG.js" type="text/javascript">
</script>
</head>
<body>
<h1>Graphlets</h1>
<hr/>

<p><strong>Graphlets</strong> are small connected non-isomorphic <em>induced</em> subgraphs of a large network.<a class="footnoteRef" href="#fn1" id="fnref1"><sup>1</sup></a><a class="footnoteRef" href="#fn2" id="fnref2"><sup>2</sup></a> Graphlets differ from <a href="network_motif" title="wikilink">network motifs</a>, since they must be <em>induced</em> subgraphs, whereas motifs are <em>partial</em> subgraphs. An <a href="Induced_subgraph#Subgraphs" title="wikilink">induced subgraph</a> must contain all edges between its nodes that are present in the large network, while a partial subgraph may contain only some of these edges. Moreover, graphlets do not need to be over-represented in the data when compared with randomized networks, while motifs do.<a class="footnoteRef" href="#fn3" id="fnref3"><sup>3</sup></a></p>

<p>Graphlets were first introduced by Nataša Pržulj, when they were used as a basis for designing two new highly sensitive measures of network local structural similarities: the relative graphlet frequency distance (RGF-distance)<a class="footnoteRef" href="#fn4" id="fnref4"><sup>4</sup></a> and the graphlet degree distribution agreement (GDD-agreement).<a class="footnoteRef" href="#fn5" id="fnref5"><sup>5</sup></a> Additionally, Pržulj group developed a novel measure of network topological similarity that generalizes the <a href="Degree_(graph_theory)" title="wikilink">degree</a> of a node in the network to its graphlet degree vector (GDV) or graphlet degree signature.<a class="footnoteRef" href="#fn6" id="fnref6"><sup>6</sup></a></p>
<h2 id="graphlet-based-network-properties">Graphlet-based network properties</h2>
<h3 id="relative-graphlet-frequency-distance">Relative graphlet frequency distance</h3>

<p>RGF-distance compares the frequencies of the appearance of all 3-5-node graphlets in two networks.<a class="footnoteRef" href="#fn7" id="fnref7"><sup>7</sup></a> Let <em>N<sub>i</sub>(G)</em> be the number of graphlets of type 

<math display="inline" id="Graphlets:0">
 <semantics>
  <mi>i</mi>
  <annotation-xml encoding="MathML-Content">
   <ci>i</ci>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   i
  </annotation>
 </semantics>
</math>

 (

<math display="inline" id="Graphlets:1">
 <semantics>
  <mrow>
   <mi>i</mi>
   <mo>∈</mo>
   <mrow>
    <mo stretchy="false">{</mo>
    <mn>1</mn>
    <mo>,</mo>
    <mi mathvariant="normal">…</mi>
    <mo>,</mo>
    <mn>29</mn>
    <mo stretchy="false">}</mo>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <in></in>
    <ci>i</ci>
    <set>
     <cn type="integer">1</cn>
     <ci>normal-…</ci>
     <cn type="integer">29</cn>
    </set>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   i\in\{1,\ldots,29\}
  </annotation>
 </semantics>
</math>

) in network <em>G</em>, and let 

<math display="inline" id="Graphlets:2">
 <semantics>
  <mrow>
   <mrow>
    <mi>T</mi>
    <mrow>
     <mo stretchy="false">(</mo>
     <mi>G</mi>
     <mo stretchy="false">)</mo>
    </mrow>
   </mrow>
   <mo>=</mo>
   <mrow>
    <msubsup>
     <mo largeop="true" symmetric="true">∑</mo>
     <mrow>
      <mi>i</mi>
      <mo>=</mo>
      <mn>1</mn>
     </mrow>
     <mn>29</mn>
    </msubsup>
    <mrow>
     <msub>
      <mi>N</mi>
      <mi>i</mi>
     </msub>
     <mrow>
      <mo stretchy="false">(</mo>
      <mi>G</mi>
      <mo stretchy="false">)</mo>
     </mrow>
    </mrow>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <eq></eq>
    <apply>
     <times></times>
     <ci>T</ci>
     <ci>G</ci>
    </apply>
    <apply>
     <apply>
      <csymbol cd="ambiguous">superscript</csymbol>
      <apply>
       <csymbol cd="ambiguous">subscript</csymbol>
       <sum></sum>
       <apply>
        <eq></eq>
        <ci>i</ci>
        <cn type="integer">1</cn>
       </apply>
      </apply>
      <cn type="integer">29</cn>
     </apply>
     <apply>
      <times></times>
      <apply>
       <csymbol cd="ambiguous">subscript</csymbol>
       <ci>N</ci>
       <ci>i</ci>
      </apply>
      <ci>G</ci>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   T(G)=\sum_{i=1}^{29}N_{i}(G)
  </annotation>
 </semantics>
</math>

 be the total number of graphlets of <em>G</em>. The "similarity" between two graphs should be independent of the total number of nodes or edges, and should depend only upon the differences between relative frequencies of graphlets. Thus, <em>relative graphlet frequency distance</em> <em>D(G,H)</em> between two graphs <em>G</em> and <em>H</em> is defined as:<br/>


<math display="inline" id="Graphlets:3">
 <semantics>
  <mrow>
   <mrow>
    <mi>D</mi>
    <mrow>
     <mo stretchy="false">(</mo>
     <mi>G</mi>
     <mo>,</mo>
     <mi>H</mi>
     <mo stretchy="false">)</mo>
    </mrow>
   </mrow>
   <mo>=</mo>
   <mrow>
    <msubsup>
     <mo largeop="true" symmetric="true">∑</mo>
     <mrow>
      <mi>i</mi>
      <mo>=</mo>
      <mn>1</mn>
     </mrow>
     <mn>29</mn>
    </msubsup>
    <mrow>
     <mo stretchy="false">|</mo>
     <mrow>
      <mrow>
       <msub>
        <mi>F</mi>
        <mi>i</mi>
       </msub>
       <mrow>
        <mo stretchy="false">(</mo>
        <mi>G</mi>
        <mo stretchy="false">)</mo>
       </mrow>
      </mrow>
      <mo>-</mo>
      <mrow>
       <msub>
        <mi>F</mi>
        <mi>i</mi>
       </msub>
       <mrow>
        <mo stretchy="false">(</mo>
        <mi>H</mi>
        <mo stretchy="false">)</mo>
       </mrow>
      </mrow>
     </mrow>
     <mo stretchy="false">|</mo>
    </mrow>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <eq></eq>
    <apply>
     <times></times>
     <ci>D</ci>
     <interval closure="open">
      <ci>G</ci>
      <ci>H</ci>
     </interval>
    </apply>
    <apply>
     <apply>
      <csymbol cd="ambiguous">superscript</csymbol>
      <apply>
       <csymbol cd="ambiguous">subscript</csymbol>
       <sum></sum>
       <apply>
        <eq></eq>
        <ci>i</ci>
        <cn type="integer">1</cn>
       </apply>
      </apply>
      <cn type="integer">29</cn>
     </apply>
     <apply>
      <abs></abs>
      <apply>
       <minus></minus>
       <apply>
        <times></times>
        <apply>
         <csymbol cd="ambiguous">subscript</csymbol>
         <ci>F</ci>
         <ci>i</ci>
        </apply>
        <ci>G</ci>
       </apply>
       <apply>
        <times></times>
        <apply>
         <csymbol cd="ambiguous">subscript</csymbol>
         <ci>F</ci>
         <ci>i</ci>
        </apply>
        <ci>H</ci>
       </apply>
      </apply>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   D(G,H)=\sum_{i=1}^{29}|F_{i}(G)-F_{i}(H)|
  </annotation>
 </semantics>
</math>

,<br/>
where 

<math display="inline" id="Graphlets:4">
 <semantics>
  <mrow>
   <mrow>
    <msub>
     <mi>F</mi>
     <mi>i</mi>
    </msub>
    <mrow>
     <mo stretchy="false">(</mo>
     <mi>G</mi>
     <mo stretchy="false">)</mo>
    </mrow>
   </mrow>
   <mo>=</mo>
   <mrow>
    <mo>-</mo>
    <mrow>
     <mi>log</mi>
     <mrow>
      <mo stretchy="false">(</mo>
      <mrow>
       <mrow>
        <mrow>
         <msub>
          <mi>N</mi>
          <mi>i</mi>
         </msub>
         <mrow>
          <mo stretchy="false">(</mo>
          <mi>G</mi>
          <mo stretchy="false">)</mo>
         </mrow>
        </mrow>
        <mo>/</mo>
        <mi>T</mi>
       </mrow>
       <mrow>
        <mo stretchy="false">(</mo>
        <mi>G</mi>
        <mo stretchy="false">)</mo>
       </mrow>
      </mrow>
      <mo stretchy="false">)</mo>
     </mrow>
    </mrow>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <eq></eq>
    <apply>
     <times></times>
     <apply>
      <csymbol cd="ambiguous">subscript</csymbol>
      <ci>F</ci>
      <ci>i</ci>
     </apply>
     <ci>G</ci>
    </apply>
    <apply>
     <minus></minus>
     <apply>
      <log></log>
      <apply>
       <times></times>
       <apply>
        <divide></divide>
        <apply>
         <times></times>
         <apply>
          <csymbol cd="ambiguous">subscript</csymbol>
          <ci>N</ci>
          <ci>i</ci>
         </apply>
         <ci>G</ci>
        </apply>
        <ci>T</ci>
       </apply>
       <ci>G</ci>
      </apply>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   F_{i}(G)=-\log(N_{i}(G)/T(G))
  </annotation>
 </semantics>
</math>

. The logarithm of the graphlet frequency is used because frequencies of different graphlets can differ by several orders of magnitude and the distance measure should not be entirely dominated by the most frequent graphlets.</p>
<h3 id="graphlet-degree-distribution-agreement">Graphlet degree distribution agreement</h3>

<p>GDD-agreement generalizes the notion of the <a href="degree_distribution" title="wikilink">degree distribution</a> to the spectrum of graphlet degree distributions (GDDs) in the following way.<a class="footnoteRef" href="#fn8" id="fnref8"><sup>8</sup></a> The degree distribution measures the number of nodes of degree <em>k</em> in graph <em>G</em>, i.e., the number of nodes "touching" <em>k</em> edges, for each value of <em>k</em>. Note that an edge is the only graphlet with two nodes. GDDs generalize the degree distribution to other graphlets: they measure for each 2-5-node graphlet <em>G<sub>i</sub></em>, 

<math display="inline" id="Graphlets:5">
 <semantics>
  <mrow>
   <mi>i</mi>
   <mo>=</mo>
   <mrow>
    <mn>0</mn>
    <mo>,</mo>
    <mn>1</mn>
    <mo>,</mo>
    <mi mathvariant="normal">…</mi>
    <mo>,</mo>
    <mn>29</mn>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <eq></eq>
    <ci>i</ci>
    <list>
     <cn type="integer">0</cn>
     <cn type="integer">1</cn>
     <ci>normal-…</ci>
     <cn type="integer">29</cn>
    </list>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   i=0,1,...,29
  </annotation>
 </semantics>
</math>

, such as a triangle or a square, the number of nodes "touching" <em>k</em> graphlets <em>G<sub>i</sub></em> at a particular node. A node at which a graphlet is "touched" is topologically relevant, since it allows us to distinguish between nodes "touching", for example, a three node path at an end node or at the middle node. This is summarized by <a class="uri" href="automorphism" title="wikilink">automorphism</a> orbits (or just orbits, for brevity): by taking into account the "symmetries" between nodes of a graphlet, there are 73 different orbits across all 2-5-node graphlets (see [Pržulj, 2007]<a class="footnoteRef" href="#fn9" id="fnref9"><sup>9</sup></a> for details).</p>

<p>For each orbit <em>j</em>, one needs to measure the <em>j<sup>th</sup></em> GDD, <em>d<sub>G</sub><sup>j</sup>(k)</em>, i.e., the distribution of the number of nodes in <em>G</em> "touching" the corresponding graphlet at orbit <em>j</em> <em>k</em> times. Clearly, the degree distribution is the <em>0th</em> GDD. <em>d<sub>G</sub><sup>j</sup>(k)</em> is scaled as 

<math display="inline" id="Graphlets:6">
 <semantics>
  <mrow>
   <mrow>
    <msubsup>
     <mi>S</mi>
     <mi>G</mi>
     <mi>j</mi>
    </msubsup>
    <mrow>
     <mo stretchy="false">(</mo>
     <mi>k</mi>
     <mo stretchy="false">)</mo>
    </mrow>
   </mrow>
   <mo>=</mo>
   <mfrac>
    <mrow>
     <msubsup>
      <mi>d</mi>
      <mi>G</mi>
      <mi>j</mi>
     </msubsup>
     <mrow>
      <mo stretchy="false">(</mo>
      <mi>k</mi>
      <mo stretchy="false">)</mo>
     </mrow>
    </mrow>
    <mi>k</mi>
   </mfrac>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <eq></eq>
    <apply>
     <times></times>
     <apply>
      <csymbol cd="ambiguous">superscript</csymbol>
      <apply>
       <csymbol cd="ambiguous">subscript</csymbol>
       <ci>S</ci>
       <ci>G</ci>
      </apply>
      <ci>j</ci>
     </apply>
     <ci>k</ci>
    </apply>
    <apply>
     <divide></divide>
     <apply>
      <times></times>
      <apply>
       <csymbol cd="ambiguous">superscript</csymbol>
       <apply>
        <csymbol cd="ambiguous">subscript</csymbol>
        <ci>d</ci>
        <ci>G</ci>
       </apply>
       <ci>j</ci>
      </apply>
      <ci>k</ci>
     </apply>
     <ci>k</ci>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   S_{G}^{j}(k)=\frac{d_{G}^{j}(k)}{k}
  </annotation>
 </semantics>
</math>

 to decrease the contribution of larger degrees in a GDD and then normalized with respect to its total area 

<math display="inline" id="Graphlets:7">
 <semantics>
  <mrow>
   <msubsup>
    <mi>T</mi>
    <mi>G</mi>
    <mi>j</mi>
   </msubsup>
   <mo>=</mo>
   <mrow>
    <msubsup>
     <mo largeop="true" symmetric="true">∑</mo>
     <mrow>
      <mi>k</mi>
      <mo>=</mo>
      <mn>1</mn>
     </mrow>
     <mi mathvariant="normal">∞</mi>
    </msubsup>
    <mrow>
     <msubsup>
      <mi>S</mi>
      <mi>G</mi>
      <mi>j</mi>
     </msubsup>
     <mrow>
      <mo stretchy="false">(</mo>
      <mi>k</mi>
      <mo stretchy="false">)</mo>
     </mrow>
    </mrow>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <eq></eq>
    <apply>
     <csymbol cd="ambiguous">superscript</csymbol>
     <apply>
      <csymbol cd="ambiguous">subscript</csymbol>
      <ci>T</ci>
      <ci>G</ci>
     </apply>
     <ci>j</ci>
    </apply>
    <apply>
     <apply>
      <csymbol cd="ambiguous">superscript</csymbol>
      <apply>
       <csymbol cd="ambiguous">subscript</csymbol>
       <sum></sum>
       <apply>
        <eq></eq>
        <ci>k</ci>
        <cn type="integer">1</cn>
       </apply>
      </apply>
      <infinity></infinity>
     </apply>
     <apply>
      <times></times>
      <apply>
       <csymbol cd="ambiguous">superscript</csymbol>
       <apply>
        <csymbol cd="ambiguous">subscript</csymbol>
        <ci>S</ci>
        <ci>G</ci>
       </apply>
       <ci>j</ci>
      </apply>
      <ci>k</ci>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   T_{G}^{j}=\sum_{k=1}^{\infty}S_{G}^{j}(k)
  </annotation>
 </semantics>
</math>

 giving the "normalized distribution" 

<math display="inline" id="Graphlets:8">
 <semantics>
  <mrow>
   <mrow>
    <msubsup>
     <mi>N</mi>
     <mi>G</mi>
     <mi>j</mi>
    </msubsup>
    <mrow>
     <mo stretchy="false">(</mo>
     <mi>k</mi>
     <mo stretchy="false">)</mo>
    </mrow>
   </mrow>
   <mo>=</mo>
   <mfrac>
    <mrow>
     <msubsup>
      <mi>S</mi>
      <mi>G</mi>
      <mi>j</mi>
     </msubsup>
     <mrow>
      <mo stretchy="false">(</mo>
      <mi>k</mi>
      <mo stretchy="false">)</mo>
     </mrow>
    </mrow>
    <msubsup>
     <mi>T</mi>
     <mi>G</mi>
     <mi>j</mi>
    </msubsup>
   </mfrac>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <eq></eq>
    <apply>
     <times></times>
     <apply>
      <csymbol cd="ambiguous">superscript</csymbol>
      <apply>
       <csymbol cd="ambiguous">subscript</csymbol>
       <ci>N</ci>
       <ci>G</ci>
      </apply>
      <ci>j</ci>
     </apply>
     <ci>k</ci>
    </apply>
    <apply>
     <divide></divide>
     <apply>
      <times></times>
      <apply>
       <csymbol cd="ambiguous">superscript</csymbol>
       <apply>
        <csymbol cd="ambiguous">subscript</csymbol>
        <ci>S</ci>
        <ci>G</ci>
       </apply>
       <ci>j</ci>
      </apply>
      <ci>k</ci>
     </apply>
     <apply>
      <csymbol cd="ambiguous">superscript</csymbol>
      <apply>
       <csymbol cd="ambiguous">subscript</csymbol>
       <ci>T</ci>
       <ci>G</ci>
      </apply>
      <ci>j</ci>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   N_{G}^{j}(k)=\frac{S_{G}^{j}(k)}{T_{G}^{j}}
  </annotation>
 </semantics>
</math>

.</p>

<p>The <em>j<sup>th</sup></em> GDD-agreement compares the <em>j<sup>th</sup></em> GDDs of two networks. For two networks <em>G</em> and <em>H</em> and a particular orbit <em>j</em>, the "distance" <em>D<sup>j</sup>(G,H)</em> between their normalized <em>j<sup>th</sup></em> GDDs is:<br/>


<math display="inline" id="Graphlets:9">
 <semantics>
  <mrow>
   <mrow>
    <msup>
     <mi>D</mi>
     <mi>j</mi>
    </msup>
    <mrow>
     <mo stretchy="false">(</mo>
     <mi>G</mi>
     <mo>,</mo>
     <mi>H</mi>
     <mo stretchy="false">)</mo>
    </mrow>
   </mrow>
   <mo>=</mo>
   <mrow>
    <mfrac>
     <mn>1</mn>
     <msqrt>
      <mn>2</mn>
     </msqrt>
    </mfrac>
    <msup>
     <mrow>
      <mo stretchy="false">(</mo>
      <mrow>
       <msubsup>
        <mo largeop="true" symmetric="true">∑</mo>
        <mrow>
         <mi>k</mi>
         <mo>=</mo>
         <mn>1</mn>
        </mrow>
        <mi mathvariant="normal">∞</mi>
       </msubsup>
       <msup>
        <mrow>
         <mo stretchy="false">[</mo>
         <mrow>
          <mrow>
           <msubsup>
            <mi>N</mi>
            <mi>G</mi>
            <mi>j</mi>
           </msubsup>
           <mrow>
            <mo stretchy="false">(</mo>
            <mi>k</mi>
            <mo stretchy="false">)</mo>
           </mrow>
          </mrow>
          <mo>-</mo>
          <mrow>
           <msubsup>
            <mi>N</mi>
            <mi>H</mi>
            <mi>j</mi>
           </msubsup>
           <mrow>
            <mo stretchy="false">(</mo>
            <mi>k</mi>
            <mo stretchy="false">)</mo>
           </mrow>
          </mrow>
         </mrow>
         <mo stretchy="false">]</mo>
        </mrow>
        <mn>2</mn>
       </msup>
      </mrow>
      <mo stretchy="false">)</mo>
     </mrow>
     <mfrac>
      <mn>1</mn>
      <mn>2</mn>
     </mfrac>
    </msup>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <eq></eq>
    <apply>
     <times></times>
     <apply>
      <csymbol cd="ambiguous">superscript</csymbol>
      <ci>D</ci>
      <ci>j</ci>
     </apply>
     <interval closure="open">
      <ci>G</ci>
      <ci>H</ci>
     </interval>
    </apply>
    <apply>
     <times></times>
     <apply>
      <divide></divide>
      <cn type="integer">1</cn>
      <apply>
       <root></root>
       <cn type="integer">2</cn>
      </apply>
     </apply>
     <apply>
      <csymbol cd="ambiguous">superscript</csymbol>
      <apply>
       <apply>
        <csymbol cd="ambiguous">superscript</csymbol>
        <apply>
         <csymbol cd="ambiguous">subscript</csymbol>
         <sum></sum>
         <apply>
          <eq></eq>
          <ci>k</ci>
          <cn type="integer">1</cn>
         </apply>
        </apply>
        <infinity></infinity>
       </apply>
       <apply>
        <csymbol cd="ambiguous">superscript</csymbol>
        <apply>
         <csymbol cd="latexml">delimited-[]</csymbol>
         <apply>
          <minus></minus>
          <apply>
           <times></times>
           <apply>
            <csymbol cd="ambiguous">superscript</csymbol>
            <apply>
             <csymbol cd="ambiguous">subscript</csymbol>
             <ci>N</ci>
             <ci>G</ci>
            </apply>
            <ci>j</ci>
           </apply>
           <ci>k</ci>
          </apply>
          <apply>
           <times></times>
           <apply>
            <csymbol cd="ambiguous">superscript</csymbol>
            <apply>
             <csymbol cd="ambiguous">subscript</csymbol>
             <ci>N</ci>
             <ci>H</ci>
            </apply>
            <ci>j</ci>
           </apply>
           <ci>k</ci>
          </apply>
         </apply>
        </apply>
        <cn type="integer">2</cn>
       </apply>
      </apply>
      <apply>
       <divide></divide>
       <cn type="integer">1</cn>
       <cn type="integer">2</cn>
      </apply>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   D^{j}(G,H)=\frac{1}{\sqrt{2}}(\sum_{k=1}^{\infty}[N_{G}^{j}(k)-N_{H}^{j}(k)]^{%
2})^{\frac{1}{2}}
  </annotation>
 </semantics>
</math>

.</p>

<p>The distance is between 0 and 1, where 0 means that <em>G</em> and <em>H</em> have identical <em>j<sup>th</sup></em> GDDs, and 1 means that their <em>j<sup>th</sup></em> GDDs are far away. Next, <em>D<sup>j</sup>(G,H)</em> is reversed to obtain the <em>j<sup>th</sup></em> <em>GDD-agreement</em>:<br/>


<math display="inline" id="Graphlets:10">
 <semantics>
  <mrow>
   <mrow>
    <msup>
     <mi>A</mi>
     <mi>j</mi>
    </msup>
    <mrow>
     <mo stretchy="false">(</mo>
     <mi>G</mi>
     <mo>,</mo>
     <mi>H</mi>
     <mo stretchy="false">)</mo>
    </mrow>
   </mrow>
   <mo>=</mo>
   <mrow>
    <mn>1</mn>
    <mo>-</mo>
    <mrow>
     <msup>
      <mi>D</mi>
      <mi>j</mi>
     </msup>
     <mrow>
      <mo stretchy="false">(</mo>
      <mi>G</mi>
      <mo>,</mo>
      <mi>H</mi>
      <mo stretchy="false">)</mo>
     </mrow>
    </mrow>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <eq></eq>
    <apply>
     <times></times>
     <apply>
      <csymbol cd="ambiguous">superscript</csymbol>
      <ci>A</ci>
      <ci>j</ci>
     </apply>
     <interval closure="open">
      <ci>G</ci>
      <ci>H</ci>
     </interval>
    </apply>
    <apply>
     <minus></minus>
     <cn type="integer">1</cn>
     <apply>
      <times></times>
      <apply>
       <csymbol cd="ambiguous">superscript</csymbol>
       <ci>D</ci>
       <ci>j</ci>
      </apply>
      <interval closure="open">
       <ci>G</ci>
       <ci>H</ci>
      </interval>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   A^{j}(G,H)=1-D^{j}(G,H)
  </annotation>
 </semantics>
</math>

, for 

<math display="inline" id="Graphlets:11">
 <semantics>
  <mrow>
   <mi>j</mi>
   <mo>∈</mo>
   <mrow>
    <mo stretchy="false">{</mo>
    <mn>0</mn>
    <mo>,</mo>
    <mn>1</mn>
    <mo>,</mo>
    <mi mathvariant="normal">…</mi>
    <mo>,</mo>
    <mn>72</mn>
    <mo stretchy="false">}</mo>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <in></in>
    <ci>j</ci>
    <set>
     <cn type="integer">0</cn>
     <cn type="integer">1</cn>
     <ci>normal-…</ci>
     <cn type="integer">72</cn>
    </set>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   j\in\{0,1,\ldots,72\}
  </annotation>
 </semantics>
</math>

.</p>

<p>The total GDD-agreement between two networks <em>G</em> and <em>H</em> is the arithmetic or the geometric average of the <em>j<sup>th</sup></em> GDD-agreements over all <em>j</em>, i.e.,<br/>


<math display="inline" id="Graphlets:12">
 <semantics>
  <mrow>
   <mrow>
    <msub>
     <mi>A</mi>
     <mrow>
      <mi>a</mi>
      <mi>r</mi>
      <mi>i</mi>
      <mi>t</mi>
      <mi>h</mi>
     </mrow>
    </msub>
    <mrow>
     <mo stretchy="false">(</mo>
     <mi>G</mi>
     <mo>,</mo>
     <mi>H</mi>
     <mo stretchy="false">)</mo>
    </mrow>
   </mrow>
   <mo>=</mo>
   <mrow>
    <mfrac>
     <mn>1</mn>
     <mn>73</mn>
    </mfrac>
    <mrow>
     <msubsup>
      <mo largeop="true" symmetric="true">∑</mo>
      <mrow>
       <mi>j</mi>
       <mo>=</mo>
       <mn>0</mn>
      </mrow>
      <mn>72</mn>
     </msubsup>
     <mrow>
      <msup>
       <mi>A</mi>
       <mi>j</mi>
      </msup>
      <mrow>
       <mo stretchy="false">(</mo>
       <mi>G</mi>
       <mo>,</mo>
       <mi>H</mi>
       <mo stretchy="false">)</mo>
      </mrow>
     </mrow>
    </mrow>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <eq></eq>
    <apply>
     <times></times>
     <apply>
      <csymbol cd="ambiguous">subscript</csymbol>
      <ci>A</ci>
      <apply>
       <times></times>
       <ci>a</ci>
       <ci>r</ci>
       <ci>i</ci>
       <ci>t</ci>
       <ci>h</ci>
      </apply>
     </apply>
     <interval closure="open">
      <ci>G</ci>
      <ci>H</ci>
     </interval>
    </apply>
    <apply>
     <times></times>
     <apply>
      <divide></divide>
      <cn type="integer">1</cn>
      <cn type="integer">73</cn>
     </apply>
     <apply>
      <apply>
       <csymbol cd="ambiguous">superscript</csymbol>
       <apply>
        <csymbol cd="ambiguous">subscript</csymbol>
        <sum></sum>
        <apply>
         <eq></eq>
         <ci>j</ci>
         <cn type="integer">0</cn>
        </apply>
       </apply>
       <cn type="integer">72</cn>
      </apply>
      <apply>
       <times></times>
       <apply>
        <csymbol cd="ambiguous">superscript</csymbol>
        <ci>A</ci>
        <ci>j</ci>
       </apply>
       <interval closure="open">
        <ci>G</ci>
        <ci>H</ci>
       </interval>
      </apply>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   A_{arith}(G,H)=\frac{1}{73}\sum_{j=0}^{72}A^{j}(G,H)
  </annotation>
 </semantics>
</math>

,<br/>
and<br/>


<math display="inline" id="Graphlets:13">
 <semantics>
  <mrow>
   <mrow>
    <msub>
     <mi>A</mi>
     <mrow>
      <mi>g</mi>
      <mi>e</mi>
      <mi>o</mi>
     </mrow>
    </msub>
    <mrow>
     <mo stretchy="false">(</mo>
     <mi>G</mi>
     <mo>,</mo>
     <mi>H</mi>
     <mo stretchy="false">)</mo>
    </mrow>
   </mrow>
   <mo>=</mo>
   <msup>
    <mrow>
     <mo>(</mo>
     <mrow>
      <msubsup>
       <mo largeop="true" symmetric="true">∏</mo>
       <mrow>
        <mi>j</mi>
        <mo>=</mo>
        <mn>0</mn>
       </mrow>
       <mn>72</mn>
      </msubsup>
      <mrow>
       <msup>
        <mi>A</mi>
        <mi>j</mi>
       </msup>
       <mrow>
        <mo stretchy="false">(</mo>
        <mi>G</mi>
        <mo>,</mo>
        <mi>H</mi>
        <mo stretchy="false">)</mo>
       </mrow>
      </mrow>
     </mrow>
     <mo>)</mo>
    </mrow>
    <mfrac>
     <mn>1</mn>
     <mn>73</mn>
    </mfrac>
   </msup>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <eq></eq>
    <apply>
     <times></times>
     <apply>
      <csymbol cd="ambiguous">subscript</csymbol>
      <ci>A</ci>
      <apply>
       <times></times>
       <ci>g</ci>
       <ci>e</ci>
       <ci>o</ci>
      </apply>
     </apply>
     <interval closure="open">
      <ci>G</ci>
      <ci>H</ci>
     </interval>
    </apply>
    <apply>
     <csymbol cd="ambiguous">superscript</csymbol>
     <apply>
      <apply>
       <csymbol cd="ambiguous">superscript</csymbol>
       <apply>
        <csymbol cd="ambiguous">subscript</csymbol>
        <csymbol cd="latexml">product</csymbol>
        <apply>
         <eq></eq>
         <ci>j</ci>
         <cn type="integer">0</cn>
        </apply>
       </apply>
       <cn type="integer">72</cn>
      </apply>
      <apply>
       <times></times>
       <apply>
        <csymbol cd="ambiguous">superscript</csymbol>
        <ci>A</ci>
        <ci>j</ci>
       </apply>
       <interval closure="open">
        <ci>G</ci>
        <ci>H</ci>
       </interval>
      </apply>
     </apply>
     <apply>
      <divide></divide>
      <cn type="integer">1</cn>
      <cn type="integer">73</cn>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   A_{geo}(G,H)=\left(\prod_{j=0}^{72}A^{j}(G,H)\right)^{\frac{1}{73}}
  </annotation>
 </semantics>
</math>

,<br/>
respectively. GDD-agreement is scaled to always be between 0 and 1, where 1 means that two networks are identical with respect to this property. (See [Pržulj, 2007]<a class="footnoteRef" href="#fn10" id="fnref10"><sup>10</sup></a> for details.)</p>
<h3 id="graphlet-degree-vectors-signatures-and-signature-similarities">Graphlet degree vectors (signatures) and signature similarities</h3>

<p>This method generalizes the degree of a node, which counts the number of edges that the node touches, into the vector of graphlet degrees, or graphlet degree signature, counting the number of graphlets that the node touches at a particular orbit, for all graphlets on 2 to 5 nodes.<a class="footnoteRef" href="#fn11" id="fnref11"><sup>11</sup></a> The resulting vector of 73 coordinates is the signature of a node that describes the topology of node's neighborhood and captures its interconnectivities out to a distance of 4 (see [Milenković and Pržulj, 2008]<a class="footnoteRef" href="#fn12" id="fnref12"><sup>12</sup></a> for details). The graphlet degree signature of a node provides a highly constraining measure of local topology in its vicinity and comparing the signatures of two nodes provides a highly constraining measure of local topological similarity between them.</p>

<p>The <em>signature similarity</em><a class="footnoteRef" href="#fn13" id="fnref13"><sup>13</sup></a> is computed as follows. For a node <em>u</em> in graph <em>G</em>, <em>u<sub>i</sub></em> denotes the <em>i<sup>th</sup></em> coordinate of its signature vector, i.e., <em>u<sub>i</sub></em> is the number of times node <em>u</em> is touched by an orbit <em>i</em> in <em>G</em>. The distance <em>D<sub>i</sub>(u,v)</em> between the <em>i<sup>th</sup></em> orbits of nodes <em>u</em> and <em>v</em> is defined as:<br/>


<math display="inline" id="Graphlets:14">
 <semantics>
  <mrow>
   <mrow>
    <msub>
     <mi>D</mi>
     <mi>i</mi>
    </msub>
    <mrow>
     <mo stretchy="false">(</mo>
     <mi>u</mi>
     <mo>,</mo>
     <mi>v</mi>
     <mo stretchy="false">)</mo>
    </mrow>
   </mrow>
   <mo>=</mo>
   <mrow>
    <msub>
     <mi>w</mi>
     <mi>i</mi>
    </msub>
    <mo>×</mo>
    <mfrac>
     <mrow>
      <mo stretchy="false">|</mo>
      <mrow>
       <mrow>
        <mi>l</mi>
        <mi>o</mi>
        <mi>g</mi>
        <mrow>
         <mo stretchy="false">(</mo>
         <mrow>
          <msub>
           <mi>u</mi>
           <mi>i</mi>
          </msub>
          <mo>+</mo>
          <mn>1</mn>
         </mrow>
         <mo stretchy="false">)</mo>
        </mrow>
       </mrow>
       <mo>-</mo>
       <mrow>
        <mi>l</mi>
        <mi>o</mi>
        <mi>g</mi>
        <mrow>
         <mo stretchy="false">(</mo>
         <mrow>
          <msub>
           <mi>v</mi>
           <mi>i</mi>
          </msub>
          <mo>+</mo>
          <mn>1</mn>
         </mrow>
         <mo stretchy="false">)</mo>
        </mrow>
       </mrow>
      </mrow>
      <mo stretchy="false">|</mo>
     </mrow>
     <mrow>
      <mi>l</mi>
      <mi>o</mi>
      <mi>g</mi>
      <mrow>
       <mo stretchy="false">(</mo>
       <mrow>
        <mrow>
         <mi>m</mi>
         <mi>a</mi>
         <mi>x</mi>
         <mrow>
          <mo stretchy="false">{</mo>
          <msub>
           <mi>u</mi>
           <mi>i</mi>
          </msub>
          <mo>,</mo>
          <msub>
           <mi>v</mi>
           <mi>i</mi>
          </msub>
          <mo stretchy="false">}</mo>
         </mrow>
        </mrow>
        <mo>+</mo>
        <mn>2</mn>
       </mrow>
       <mo stretchy="false">)</mo>
      </mrow>
     </mrow>
    </mfrac>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <eq></eq>
    <apply>
     <times></times>
     <apply>
      <csymbol cd="ambiguous">subscript</csymbol>
      <ci>D</ci>
      <ci>i</ci>
     </apply>
     <interval closure="open">
      <ci>u</ci>
      <ci>v</ci>
     </interval>
    </apply>
    <apply>
     <times></times>
     <apply>
      <csymbol cd="ambiguous">subscript</csymbol>
      <ci>w</ci>
      <ci>i</ci>
     </apply>
     <apply>
      <divide></divide>
      <apply>
       <abs></abs>
       <apply>
        <minus></minus>
        <apply>
         <times></times>
         <ci>l</ci>
         <ci>o</ci>
         <ci>g</ci>
         <apply>
          <plus></plus>
          <apply>
           <csymbol cd="ambiguous">subscript</csymbol>
           <ci>u</ci>
           <ci>i</ci>
          </apply>
          <cn type="integer">1</cn>
         </apply>
        </apply>
        <apply>
         <times></times>
         <ci>l</ci>
         <ci>o</ci>
         <ci>g</ci>
         <apply>
          <plus></plus>
          <apply>
           <csymbol cd="ambiguous">subscript</csymbol>
           <ci>v</ci>
           <ci>i</ci>
          </apply>
          <cn type="integer">1</cn>
         </apply>
        </apply>
       </apply>
      </apply>
      <apply>
       <times></times>
       <ci>l</ci>
       <ci>o</ci>
       <ci>g</ci>
       <apply>
        <plus></plus>
        <apply>
         <times></times>
         <ci>m</ci>
         <ci>a</ci>
         <ci>x</ci>
         <set>
          <apply>
           <csymbol cd="ambiguous">subscript</csymbol>
           <ci>u</ci>
           <ci>i</ci>
          </apply>
          <apply>
           <csymbol cd="ambiguous">subscript</csymbol>
           <ci>v</ci>
           <ci>i</ci>
          </apply>
         </set>
        </apply>
        <cn type="integer">2</cn>
       </apply>
      </apply>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   D_{i}(u,v)=w_{i}\times\frac{|log(u_{i}+1)-log(v_{i}+1)|}{log(max\{u_{i},v_{i}%
\}+2)}
  </annotation>
 </semantics>
</math>

,<br/>
where <em>w<sub>i</sub></em> is the weight of orbit <em>i</em> that accounts for dependencies between orbits (see [Milenković and Pržulj, 2008]<a class="footnoteRef" href="#fn14" id="fnref14"><sup>14</sup></a> for details). The total distance <em>D(u,v)</em> between nodes <em>u</em> and <em>v</em> is defined as:<br/>


<math display="inline" id="Graphlets:15">
 <semantics>
  <mrow>
   <mrow>
    <mi>D</mi>
    <mrow>
     <mo stretchy="false">(</mo>
     <mi>u</mi>
     <mo>,</mo>
     <mi>v</mi>
     <mo stretchy="false">)</mo>
    </mrow>
   </mrow>
   <mo>=</mo>
   <mfrac>
    <mrow>
     <mstyle displaystyle="false">
      <msubsup>
       <mo largeop="true" symmetric="true">∑</mo>
       <mrow>
        <mi>i</mi>
        <mo>=</mo>
        <mn>0</mn>
       </mrow>
       <mn>72</mn>
      </msubsup>
     </mstyle>
     <msub>
      <mi>D</mi>
      <mi>i</mi>
     </msub>
    </mrow>
    <mrow>
     <mstyle displaystyle="false">
      <msubsup>
       <mo largeop="true" symmetric="true">∑</mo>
       <mrow>
        <mi>i</mi>
        <mo>=</mo>
        <mn>0</mn>
       </mrow>
       <mn>72</mn>
      </msubsup>
     </mstyle>
     <msub>
      <mi>w</mi>
      <mi>i</mi>
     </msub>
    </mrow>
   </mfrac>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <eq></eq>
    <apply>
     <times></times>
     <ci>D</ci>
     <interval closure="open">
      <ci>u</ci>
      <ci>v</ci>
     </interval>
    </apply>
    <apply>
     <divide></divide>
     <apply>
      <apply>
       <csymbol cd="ambiguous">superscript</csymbol>
       <apply>
        <csymbol cd="ambiguous">subscript</csymbol>
        <sum></sum>
        <apply>
         <eq></eq>
         <ci>i</ci>
         <cn type="integer">0</cn>
        </apply>
       </apply>
       <cn type="integer">72</cn>
      </apply>
      <apply>
       <csymbol cd="ambiguous">subscript</csymbol>
       <ci>D</ci>
       <ci>i</ci>
      </apply>
     </apply>
     <apply>
      <apply>
       <csymbol cd="ambiguous">superscript</csymbol>
       <apply>
        <csymbol cd="ambiguous">subscript</csymbol>
        <sum></sum>
        <apply>
         <eq></eq>
         <ci>i</ci>
         <cn type="integer">0</cn>
        </apply>
       </apply>
       <cn type="integer">72</cn>
      </apply>
      <apply>
       <csymbol cd="ambiguous">subscript</csymbol>
       <ci>w</ci>
       <ci>i</ci>
      </apply>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   D(u,v)=\frac{\sum_{i=0}^{72}D_{i}}{\sum_{i=0}^{72}w_{i}}
  </annotation>
 </semantics>
</math>

.<br/>
The distance <em>D(u,v)</em> is in [0, 1), where distance 0 means that signatures of nodes <em>u</em> and <em>v</em> are identical. Finally, the signature similarity, <em>S(u,v)</em>, between nodes <em>u</em> and <em>v</em> is:<br/>


<math display="inline" id="Graphlets:16">
 <semantics>
  <mrow>
   <mrow>
    <mi>S</mi>
    <mrow>
     <mo stretchy="false">(</mo>
     <mi>u</mi>
     <mo>,</mo>
     <mi>v</mi>
     <mo stretchy="false">)</mo>
    </mrow>
   </mrow>
   <mo>=</mo>
   <mrow>
    <mn>1</mn>
    <mo>-</mo>
    <mrow>
     <mi>D</mi>
     <mrow>
      <mo stretchy="false">(</mo>
      <mi>u</mi>
      <mo>,</mo>
      <mi>v</mi>
      <mo stretchy="false">)</mo>
     </mrow>
    </mrow>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <eq></eq>
    <apply>
     <times></times>
     <ci>S</ci>
     <interval closure="open">
      <ci>u</ci>
      <ci>v</ci>
     </interval>
    </apply>
    <apply>
     <minus></minus>
     <cn type="integer">1</cn>
     <apply>
      <times></times>
      <ci>D</ci>
      <interval closure="open">
       <ci>u</ci>
       <ci>v</ci>
      </interval>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   S(u,v)=1-D(u,v)
  </annotation>
 </semantics>
</math>

.<br/>
Clearly, a higher signature similarity between two nodes corresponds to a higher topological similarity between their extended neighborhoods (out to distance 4).</p>
<h2 id="application-of-graphlet-based-network-properties">Application of graphlet-based network properties</h2>

<p>RGF-distance and GDD-agreement were used to evaluate the fit of various network models to real-world networks and to discover a new, well-fitting, <a href="Random_geometric_graph" title="wikilink">geometric random graph model</a> for <a href="Protein–protein_interaction" title="wikilink">protein-protein interaction</a> networks,<a class="footnoteRef" href="#fn15" id="fnref15"><sup>15</sup></a><a class="footnoteRef" href="#fn16" id="fnref16"><sup>16</sup></a> as well as other types of <a href="Network_biology" title="wikilink">biological networks</a>, such as <a href="protein_structure" title="wikilink">protein structure</a> networks, also called residue interaction graphs.<a class="footnoteRef" href="#fn17" id="fnref17"><sup>17</sup></a> These graphlet-based network properties are implemented in GraphCrunch, a software tool for large network analyses and modeling,<a class="footnoteRef" href="#fn18" id="fnref18"><sup>18</sup></a></p>

<p>Graphlet degree vectors (signatures) and signature similarities were applied to biological networks to identify groups (or <a href="Cluster_analysis" title="wikilink">clusters</a>) of topologically similar nodes in a network and predict biological properties of yet uncharacterized nodes based on known biological properties of characterized nodes. Specifically, they were applied to <a href="protein_function_prediction" title="wikilink">protein function prediction</a>,<a class="footnoteRef" href="#fn19" id="fnref19"><sup>19</sup></a> cancer gene identification,<a class="footnoteRef" href="#fn20" id="fnref20"><sup>20</sup></a> and discovery of pathways underlying certain biological processes, such as melanogenesis<a class="footnoteRef" href="#fn21" id="fnref21"><sup>21</sup></a> or protein degradation.<a class="footnoteRef" href="#fn22" id="fnref22"><sup>22</sup></a><a class="footnoteRef" href="#fn23" id="fnref23"><sup>23</sup></a> Additionally, <a href="GRAph_ALigner_(GRAAL)" title="wikilink">GRAph ALigner (GRAAL)</a>,<a class="footnoteRef" href="#fn24" id="fnref24"><sup>24</sup></a> a global network alignment method, used graphlet degree vectors and signature similarities to produce <em>topological</em> alignments of biological networks, without using any information external to network topology.</p>
<h2 id="references">References</h2>
<h2 id="external-links">External links</h2>
<ul>
<li><a href="http://bio-nets.doc.ic.ac.uk/graphcrunch2/">1</a></li>
</ul>

<p>"</p>

<p><a class="uri" href="Category:Networks" title="wikilink">Category:Networks</a></p>
<section class="footnotes">
<hr/>
<ol>
<li id="fn1">Pržulj N, <a href="Derek_Corneil" title="wikilink">Corneil DG</a>, Jurisica I: Modeling Interactome, Scale-Free or Geometric?, Bioinformatics 2004, 20(18):3508-3515.<a href="#fnref1">↩</a></li>
<li id="fn2">Pržulj N, Biological Network Comparison Using Graphlet Degree Distribution, Bioinformatics 2007, 23:e177-e183.<a href="#fnref2">↩</a></li>
<li id="fn3">R. Milo, S. Shen-Orr, S. Itzkovitz, N. Kashtan, D. Chklovskii, and U. Alon, Network motifs, simple building blocks of complex networks, <em>Science</em> 2002, 298(5594): p. 824-7.<a href="#fnref3">↩</a></li>
<li id="fn4"></li>
<li id="fn5"></li>
<li id="fn6">Tijana Milenković and Nataša Pržulj, Uncovering Biological Network Function via Graphlet Degree Signatures, Cancer Informatics 2008, 6:257–273.<a href="#fnref6">↩</a></li>
<li id="fn7"></li>
<li id="fn8"></li>
<li id="fn9"></li>
<li id="fn10"></li>
<li id="fn11"></li>
<li id="fn12"></li>
<li id="fn13"></li>
<li id="fn14"></li>
<li id="fn15"></li>
<li id="fn16"></li>
<li id="fn17">Tijana Milenković, Ioannis Filippis, Michael Lappe, and Nataša Pržulj, Optimized Null Model for Protein Structure Networks, 2009, PLoS ONE 4(6): e5967.<a href="#fnref17">↩</a></li>
<li id="fn18">Tijana Milenković, Jason Lai, and Nataša Pržulj, GraphCrunch: a tool for large network analyses, BMC Bioinformatics 2008, 9:70. Highly accessed.<a href="#fnref18">↩</a></li>
<li id="fn19"></li>
<li id="fn20">Tijana Milenković, Vesna Memisević, Anand K. Ganesan, and Nataša Pržulj, Systems-level Cancer Gene Identification from Protein Interaction Network Topology Applied to Melanogenesis-related Interaction Networks, Journal of the Royal Society Interface 2009, .<a href="#fnref20">↩</a></li>
<li id="fn21"></li>
<li id="fn22">Cortnie Guerrero, Tijana Milenković, Nataša Pržulj, Peter Kaiser, Lan Huang, Characterization of the Yeast Proteasome Interaction Network by QTAX-Based Tag-Team Mass Spectrometry and Protein Interaction Network Analysis, PNAS 2008, 105(36): 13333–13338.<a href="#fnref22">↩</a></li>
<li id="fn23">Robyn Kaake, Tijana Milenković, Nataša Pržulj, Peter Kaiser, and Lan Huang, Quantifying Cell Cycle Dependent Changes in Protein Interacting Network of the Yeast 26S Proteasome, Journal of Proteome Research 2010, to appear.<a href="#fnref23">↩</a></li>
<li id="fn24">Oleksii Kuchaiev, Tijana Milenković, Vesna Memisević, Wayne Hayes, and Nataša Pržulj, Topological network alignment uncovers biological function and phylogeny, Journal of the Royal Society Interface 2010, to appear.<a href="#fnref24">↩</a></li>
</ol>
</section>
</body>
</html>
