<html lang="en">
<head>
<meta charset="utf-8"/>
<title offset="1430">Haplotype estimation</title>
<script src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_SVG.js" type="text/javascript">
</script>
</head>
<body>
<h1>Haplotype estimation</h1>
<hr/>

<p>In <a class="uri" href="genetics" title="wikilink">genetics</a>, <a class="uri" href="haplotype" title="wikilink">haplotype</a> estimation (also known as, phasing) refers to the process of statistical estimation of haplotypes from genotype data. The most common situation arises when genotypes are collected at a set of polymorphic sites from a group of individuals. For example, in human genetics genome-wide association studies collect genotypes in thousands of individuals at between 200,000-5,000,000 SNPs using microarrays. Haplotype estimation methods are used in the analysis of these datasets and allow genotype imputation <a class="footnoteRef" href="#fn1" id="fnref1"><sup>1</sup></a><a class="footnoteRef" href="#fn2" id="fnref2"><sup>2</sup></a> of alleles from reference databases such as the <a href="HapMap_Project" title="wikilink">HapMap Project</a> and <a href="the_1000_Genomes_Project" title="wikilink">the 1000 Genomes Project</a>. Haplotype estimation is sometimes referred to as phasing.</p>
<h2 id="genotypes-and-haplotypes">Genotypes and haplotypes</h2>
<figure><b>(Figure)</b>
<figcaption>Heterozygote genotypes at 3 sites together with the 4 pairs of haplotypes that are consistent with the genotypes.</figcaption>
</figure>

<p>Genotypes measure the unordered combination of alleles at each site, whereas haplotypes are the two sequence of alleles that have been inherited together from the individuals parents. When there are 

<math display="inline" id="Haplotype_estimation:0">
 <semantics>
  <mi>N</mi>
  <annotation-xml encoding="MathML-Content">
   <ci>N</ci>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   N
  </annotation>
 </semantics>
</math>

 <a class="uri" href="heterozygous" title="wikilink">heterozygous</a> genotypes present an individuals set of genotypes there will be 

<math display="inline" id="Haplotype_estimation:1">
 <semantics>
  <msup>
   <mn>2</mn>
   <mrow>
    <mi>N</mi>
    <mo>-</mo>
    <mn>1</mn>
   </mrow>
  </msup>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <csymbol cd="ambiguous">superscript</csymbol>
    <cn type="integer">2</cn>
    <apply>
     <minus></minus>
     <ci>N</ci>
     <cn type="integer">1</cn>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   2^{N-1}
  </annotation>
 </semantics>
</math>

 possible pairs of haplotypes that could underlie the genotypes. If there are missing genotypes then the number of possible haplotype pairs increases.</p>
<h2 id="haplotype-estimation-methods">Haplotype estimation methods</h2>

<p>There is a large literature of statistical methods that have been proposed for estimation of haplotypes. Some of the earliest approaches used a simple multinomial model in which each possible haplotype consistent with the sample was given an unknown frequency parameter and these parameters were estimated with an EM algorithm. These approaches were only able to handle small numbers of sites at once, although sequential versions were later developed, specifically the SNPHAP method.</p>

<p>The most accurate and widely used methods for haplotype estimation utilize some form of hidden Markov model (HMM) to carry out inference. For a long time the method PHASE<a class="footnoteRef" href="#fn3" id="fnref3"><sup>3</sup></a> was the most accurate method. PHASE was the first method to utilize ideas from coalescent theory concerning the joint distribution of haplotypes. This method used a Gibbs sampling approach in which each individuals haplotypes were updated conditional upon the current estimates of haplotypes from all other samples. Approximations to the distribution of a haplotype conditional upon a set of other haplotypes were used for the conditional distributions of the Gibbs sampler. PHASE was used to estimate the haplotypes from the <a href="HapMap_Project" title="wikilink">HapMap Project</a>. PHASE was limited by its speed and was not applicable to datasets from genome-wide association studies.</p>

<p>The fastPHASE <a class="footnoteRef" href="#fn4" id="fnref4"><sup>4</sup></a> and BEAGLE methods <a class="footnoteRef" href="#fn5" id="fnref5"><sup>5</sup></a> introduced haplotype cluster models applicable to <a href="Genome-wide_association_study" title="wikilink">GWAS</a>-sized datasets. Subsequently the IMPUTE2<a class="footnoteRef" href="#fn6" id="fnref6"><sup>6</sup></a> and MaCH<a class="footnoteRef" href="#fn7" id="fnref7"><sup>7</sup></a> methods were introduced that were similar to the PHASE approach but much faster. These methods iteratively update the haplotype estimates of each sample conditional upon a subset of K haplotype estimates of other samples. IMPUTE2 introduced the idea of carefully choosing which subset of haplotypes to condition on to improve accuracy. Accuracy increases with K but with quadratic 

<math display="inline" id="Haplotype_estimation:2">
 <semantics>
  <mrow>
   <mi>O</mi>
   <mrow>
    <mo stretchy="false">(</mo>
    <msup>
     <mi>K</mi>
     <mn>2</mn>
    </msup>
    <mo stretchy="false">)</mo>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <times></times>
    <ci>O</ci>
    <apply>
     <csymbol cd="ambiguous">superscript</csymbol>
     <ci>K</ci>
     <cn type="integer">2</cn>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   O(K^{2})
  </annotation>
 </semantics>
</math>

 computational complexity.</p>

<p>The SHAPEIT1 method made a major advance by introducing a linear 

<math display="inline" id="Haplotype_estimation:3">
 <semantics>
  <mrow>
   <mi>O</mi>
   <mrow>
    <mo stretchy="false">(</mo>
    <mi>K</mi>
    <mo stretchy="false">)</mo>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <times></times>
    <ci>O</ci>
    <ci>K</ci>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   O(K)
  </annotation>
 </semantics>
</math>

 complexity method that operates only on the space of haplotypes consistent with an individual’s genotypes.<a class="footnoteRef" href="#fn8" id="fnref8"><sup>8</sup></a> The HAPI-UR method subsequently proposed a very similar method.<a class="footnoteRef" href="#fn9" id="fnref9"><sup>9</sup></a> SHAPEIT2 <a class="footnoteRef" href="#fn10" id="fnref10"><sup>10</sup></a> combines the best features of SHAPEIT1 and IMPUTE2 to improve efficiency and accuracy.</p>
<h2 id="software">Software</h2>
<ul>
<li><a href="https://www-gene.cimr.cam.ac.uk/staff/clayton/software/">SNPHAP</a></li>
<li><a href="http://stephenslab.uchicago.edu/software.html#phase">PHASE</a></li>
<li><a href="http://stephenslab.uchicago.edu/software.html#fastphase">fastPHASE</a></li>
<li><a href="http://faculty.washington.edu/browning/beagle/beagle.html">Beagle</a></li>
<li><a href="http://mathgen.stats.ox.ac.uk/impute/impute_v2.html">IMPUTE2</a></li>
<li><a href="http://www.sph.umich.edu/csg/abecasis/MACH/download/">MaCH</a></li>
<li><a href="https://code.google.com/p/hapi-ur/">HAPI-UR</a></li>
<li><a href="http://www.shapeit.fr/">SHAPEIT</a></li>
</ul>
<h2 id="see-also">See also</h2>
<ul>
<li><a href="Imputation_(genetics)" title="wikilink">imputation</a>: predict missing genotypes using known haplotypes</li>
</ul>
<h2 id="references">References</h2>

<p>"</p>

<p><a class="uri" href="Category:Genetics" title="wikilink">Category:Genetics</a></p>
<section class="footnotes">
<hr/>
<ol>
<li id="fn1"><a href="#fnref1">↩</a></li>
<li id="fn2"><a href="#fnref2">↩</a></li>
<li id="fn3"><a href="#fnref3">↩</a></li>
<li id="fn4"><a href="#fnref4">↩</a></li>
<li id="fn5"><a href="#fnref5">↩</a></li>
<li id="fn6"><a href="#fnref6">↩</a></li>
<li id="fn7"><a href="#fnref7">↩</a></li>
<li id="fn8"><a href="#fnref8">↩</a></li>
<li id="fn9"><a href="#fnref9">↩</a></li>
<li id="fn10"><a href="#fnref10">↩</a></li>
</ol>
</section>
</body>
</html>
