<html lang="en">
<head>
<meta charset="utf-8"/>
<title offset="1340">Haploid-relative-risk</title>
<script src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_SVG.js" type="text/javascript">
</script>
</head>
<body>
<h1>Haploid-relative-risk</h1>
<hr/>

<p><strong>Haploid-relative-risk</strong> (HRR) is a family based method for determining gene <a class="uri" href="allele" title="wikilink">allele</a> <a href="genetic_association" title="wikilink">association</a> to a disease. <a href="nuclear_family" title="wikilink">Nuclear families</a> with one affected child are sampled using the parental <a href="haplotype" title="wikilink">haplotypes</a> not transmitted as a control. While similar to <a class="uri" href="genotype" title="wikilink">genotype</a> <a href="relative_risk" title="wikilink">relative risk</a> (RR), HRR provides a solution to the problem of <a href="population_stratification" title="wikilink">population stratification</a> by only sampling within family trios. HRR method was first proposed by Rubinstein in 1981 then detailed in 1987 by Rubinstein and Falk <a class="footnoteRef" href="#fn1" id="fnref1"><sup>1</sup></a> and is an important tool in <a href="genetic_association" title="wikilink">genetic association</a> studies.</p>

<p>The original method proposed by Falk and Rubinstien fell under scrutiny in 1989, when Ott showed the equivalence of HRR to the classical RR method<a class="footnoteRef" href="#fn2" id="fnref2"><sup>2</sup></a> demonstrating the HRR holds only when there is zero chance of <a href="Genetic_recombination" title="wikilink">recombination</a> between disease <a href="Locus_(genetics)" title="wikilink">locus</a> and its markers.<a class="footnoteRef" href="#fn3" id="fnref3"><sup>3</sup></a> This led to the adoption of the <a href="transmission_disequilibrium_test" title="wikilink">transmission disequilibrium test</a> as the more commonly used method for association <a href="Genetic_linkage" title="wikilink">linkage</a>; Yet, when the recombination factor for a gene marker and its locus are &gt;0 there is no tendency for <a href="false_positive" title="wikilink">false positives</a> seen.<a class="footnoteRef" href="#fn4" id="fnref4"><sup>4</sup></a></p>

<p>While HRR method has been shown as an effective means of avoiding population stratification another family based associations test known as the transmission disequilibrium test<a class="footnoteRef" href="#fn5" id="fnref5"><sup>5</sup></a> or, TDT is more commonly used because of its more simplified algorithms. Some research uses both HRR and TDT for their ability to complement each other since one result my give no association while the other will. A positive association result from both TDT and HRR means there is strong evidence that a link exists and vice versa. For example, both HRR and TDT methods were used in a study looking for polymorphism in D2 and D3 <a href="dopamine_receptor" title="wikilink">dopamine receptor</a> in association with <a class="uri" href="schizophrenia" title="wikilink">schizophrenia</a> and found no linkage in both tests;<a class="footnoteRef" href="#fn6" id="fnref6"><sup>6</sup></a> showing absolutely no possible association.</p>
<h2 id="calculation">Calculation</h2>

<p>This model represents a case which there is a single locus where all genotypes may lead to expression of the allele in its most simplified definition. Under these parameters a <a href="linkage_disequilibrium" title="wikilink">linkage disequilibrium</a> of more than 50% means there is a possible link to the gene allele and inheritance.</p>

<p>

<math display="inline" id="Haploid-relative-risk:0">
 <semantics>
  <mrow>
   <mrow>
    <mi>H</mi>
    <mi>R</mi>
    <mi>R</mi>
   </mrow>
   <mo>=</mo>
   <mrow>
    <mfrac>
     <msub>
      <mi>P</mi>
      <mn>1</mn>
     </msub>
     <mrow>
      <mn>1</mn>
      <mo>-</mo>
      <msub>
       <mi>P</mi>
       <mn>1</mn>
      </msub>
     </mrow>
    </mfrac>
    <mo>*</mo>
    <mfrac>
     <mrow>
      <mn>1</mn>
      <mo>-</mo>
      <msub>
       <mi>P</mi>
       <mn>2</mn>
      </msub>
     </mrow>
     <msub>
      <mi>P</mi>
      <mn>2</mn>
     </msub>
    </mfrac>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <eq></eq>
    <apply>
     <times></times>
     <ci>H</ci>
     <ci>R</ci>
     <ci>R</ci>
    </apply>
    <apply>
     <times></times>
     <apply>
      <divide></divide>
      <apply>
       <csymbol cd="ambiguous">subscript</csymbol>
       <ci>P</ci>
       <cn type="integer">1</cn>
      </apply>
      <apply>
       <minus></minus>
       <cn type="integer">1</cn>
       <apply>
        <csymbol cd="ambiguous">subscript</csymbol>
        <ci>P</ci>
        <cn type="integer">1</cn>
       </apply>
      </apply>
     </apply>
     <apply>
      <divide></divide>
      <apply>
       <minus></minus>
       <cn type="integer">1</cn>
       <apply>
        <csymbol cd="ambiguous">subscript</csymbol>
        <ci>P</ci>
        <cn type="integer">2</cn>
       </apply>
      </apply>
      <apply>
       <csymbol cd="ambiguous">subscript</csymbol>
       <ci>P</ci>
       <cn type="integer">2</cn>
      </apply>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   HRR=\frac{P_{1}}{1-P_{1}}*\frac{1-P_{2}}{P_{2}}
  </annotation>
 </semantics>
</math>

</p>

<p>Gives the HHR which can be estimated by</p>

<p>

<math display="inline" id="Haploid-relative-risk:1">
 <semantics>
  <mrow>
   <mrow>
    <mi>H</mi>
    <mi>R</mi>
    <mi>R</mi>
   </mrow>
   <mo>=</mo>
   <mrow>
    <mfrac>
     <msup>
      <mi>a</mi>
      <msup>
       <mi></mi>
       <mo>′</mo>
      </msup>
     </msup>
     <msup>
      <mi>b</mi>
      <msup>
       <mi></mi>
       <mo>′</mo>
      </msup>
     </msup>
    </mfrac>
    <mo>*</mo>
    <mfrac>
     <msup>
      <mi>d</mi>
      <msup>
       <mi></mi>
       <mo>′</mo>
      </msup>
     </msup>
     <msup>
      <mi>c</mi>
      <msup>
       <mi></mi>
       <mo>′</mo>
      </msup>
     </msup>
    </mfrac>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <eq></eq>
    <apply>
     <times></times>
     <ci>H</ci>
     <ci>R</ci>
     <ci>R</ci>
    </apply>
    <apply>
     <times></times>
     <apply>
      <divide></divide>
      <apply>
       <csymbol cd="ambiguous">superscript</csymbol>
       <ci>a</ci>
       <apply>
        <ci>normal-′</ci>
       </apply>
      </apply>
      <apply>
       <csymbol cd="ambiguous">superscript</csymbol>
       <ci>b</ci>
       <apply>
        <ci>normal-′</ci>
       </apply>
      </apply>
     </apply>
     <apply>
      <divide></divide>
      <apply>
       <csymbol cd="ambiguous">superscript</csymbol>
       <ci>d</ci>
       <apply>
        <ci>normal-′</ci>
       </apply>
      </apply>
      <apply>
       <csymbol cd="ambiguous">superscript</csymbol>
       <ci>c</ci>
       <apply>
        <ci>normal-′</ci>
       </apply>
      </apply>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   HRR=\frac{a^{^{\prime}}}{b^{^{\prime}}}*\frac{d^{^{\prime}}}{c^{^{\prime}}}
  </annotation>
 </semantics>
</math>

</p>

<p>a<sup>'</sup> denotes the observed frequency of children who are positive for the gene allele H.</p>

<p>b<sup>'</sup> denotes the observed frequency of children who are negative for the gene allele H.</p>

<p>c<sup>'</sup> is the observed frequency of families with at least one transmitted parental marker allele H.</p>

<p>d<sup>'</sup> is the observed frequency of families with no transmitted parental marker allele H.</p>

<p>P<sub>1</sub> is the probability this child is positive for the allele of interest H.</p>

<p>P<sub>2</sub> is the probability that at least one of the nontransmitted parental marker alleles equals the allele of interest H.</p>

<p>H is the allele of interest. <a class="footnoteRef" href="#fn7" id="fnref7"><sup>7</sup></a></p>
<h2 id="references">References</h2>
<references>
</references>

<p>"</p>

<p><a class="uri" href="Category:Genetics" title="wikilink">Category:Genetics</a></p>
<section class="footnotes">
<hr/>
<ol>
<li id="fn1">Falk CT, Rubinstein P (1987) "Haplotype relative risks: and easy reliable way to construct a proper control sample for risk calculations". Ann Hum Genet 51:227-233<a href="#fnref1">↩</a></li>
<li id="fn2">Woolf B (1955) "On estimating the relation between blood group and disease". Ann Human Genet 19:251-253<a href="#fnref2">↩</a></li>
<li id="fn3">Ott J (1989) statistical properties of the haplotype relative risk. Genetic Epidemiol 6:127-130<a href="#fnref3">↩</a></li>
<li id="fn4">Knapp,Seuchter,Baur (1987) "The haplotype-relative-risk" (HRR) method for analysis of association in nuclear families. Am J Hum Genet 52:1085-10093,1993<a href="#fnref4">↩</a></li>
<li id="fn5">a b c Spielman RS, McGinnis RE, Ewens WJ (Mar 1993). "Transmission test for linkage disequilibrium: the insulin gene region and insulin-dependent diabetes mellitus (IDDM)". Am J Hum Genet. 52 (3): 506–16. PMID 8447318.<a href="#fnref5">↩</a></li>
<li id="fn6">Ambròsio, Kennedy, Macciardi, macoedo, Valente, Dourado, oliveria, Carlos, Pato (2004)"Family association study between DRD2 and DRD3 gene polymorphisms and schizophrenia in a portuguese population". Psychiatry Research 125:185-191<a href="#fnref6">↩</a></li>
<li id="fn7">Knapp,Seuchter,Baur (1987) "The haplotype-relative-risk (HRR) method for analysis of association in nuclear families". Am J Hum Genet 52:1085-10093,1993<a href="#fnref7">↩</a></li>
</ol>
</section>
</body>
</html>
