<html lang="en">
<head>
<meta charset="utf-8"/>
<title offset="1353">Human mitochondrial molecular clock</title>
<script src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_SVG.js" type="text/javascript">
</script>
</head>
<body>
<h1>Human mitochondrial molecular clock</h1>
<hr/>

<p>The <strong>human mitochondrial molecular clock</strong> is the rate at which mutations have been accumulating in the <a class="uri" href="mitochondrial" title="wikilink">mitochondrial</a> <a class="uri" href="genome" title="wikilink">genome</a> of hominids during the course of <a href="human_evolution" title="wikilink">human evolution</a>. The archeological record of human activity from early periods in human prehistory is relatively limited and its interpretation has been controversial. Because of the uncertainties from the archeological record, scientists have turned to molecular dating techniques in order to refine the timeline of human evolution. A major goal of scientists in the field is to develop an accurate hominid mitochondrial molecular clock which could then be used to confidently date events that occurred during the course of human evolution.</p>

<p>Estimates of the mutation rate of human mitochondrial DNA (mtDNA) vary greatly depending on the available data and the method used for estimation. The two main methods of estimation, phylogeny based methods and pedigree based methods, have produced mutation rates that differ by almost an order of magnitude. Current research has been focused on resolving the high variability obtained from different rate estimates.</p>
<h2 id="rate-variability">Rate variability</h2>

<p>A major assumption of the molecular clock theory is that mutations within a particular genetic system occur at a statistically uniform rate and this uniform rate can be used for dating genetic events. In practice the assumption of a single uniform rate is an oversimplification. Though a single mutation rate is often applied, it is often a composite or an average of several different mutation rates.<a class="footnoteRef" href="#fn1" id="fnref1"><sup>1</sup></a> Many factors influence observed <a href="mutation_rate" title="wikilink">mutation rates</a> and these factors include the type of samples, the region of the genome studied and the time period covered.</p>
<h3 id="actual-vs.-observed-rates">Actual vs. observed rates</h3>

<p>The rate at which mutations occur during reproduction, the <a href="germline_mutation" title="wikilink">germline mutation</a> rate, is thought to be higher than all observed mutation rates, because not all mutations are successfully passed down to subsequent generations.<a class="footnoteRef" href="#fn2" id="fnref2"><sup>2</sup></a> MtDNA is only passed down along the matrilineal line, and therefore mutations passed down to sons are lost. Random genetic drift may also cause the loss of mutations. For these reasons, the actual mutation rate will not be equivalent to the mutation rate observed from a population sample.<a class="footnoteRef" href="#fn3" id="fnref3"><sup>3</sup></a></p>
<h3 id="population-size">Population size</h3>

<p>Population dynamics are believed to influence observed mutation rates. When a population is expanding, more <a href="germline_mutation" title="wikilink">germline mutations</a> are preserved in the population. As a result, observed mutation rates tend to increase in an expanding population. When populations contract, as in a <a href="population_bottleneck" title="wikilink">population bottleneck</a>, more germline mutations are lost. Population bottlenecks thus tend to slow down observed mutation rates. Since the emergence of the species homo sapiens about 200,000 years ago, human population have expanded from a few thousand individuals living in Africa to over 6 billion all over the world. However the expansion has not been uniform, the history of human populations may have consisted of both bottlenecks and expansions.<a class="footnoteRef" href="#fn4" id="fnref4"><sup>4</sup></a></p>
<h3 id="structural-variability">Structural variability</h3>

<p>The mutation rate across the mitochondrial genome is not uniformly distributed. Certain regions of the genome are known to mutate more rapidly than others. The <a href="Hypervariable_region" title="wikilink">Hypervariable regions</a> are known to be highly polymorphic relative to other parts of the genome.</p>

<p>The rate at which mutations accumulate in coding and <a href="non-coding_region" title="wikilink">non-coding regions</a> of the genome also differs as mutations in the <a href="coding_region" title="wikilink">coding region</a> are subject to <a href="natural_selection" title="wikilink">purifying selection</a>. For this reason, some studies avoid coding region or <a href="synonymous_mutations" title="wikilink">synonymous mutations</a> when calibrating the molecular clock.  only consider synonymous mutations, they have recalibrated the molecular clock of human mtDNA as 7990 years per synonymous mutation over the mitochondrial genome.<a class="footnoteRef" href="#fn5" id="fnref5"><sup>5</sup></a>  consider both coding and non-coding region mutations to arrive at a single mutation rate, but apply a correction factor to account for selection in the coding region.</p>
<h3 id="temporal-variability">Temporal variability</h3>

<p>The mutation rate has been observed to vary with time. Mutation rates within the human species are faster than those observed along the human-ape lineage. The mutation rate is also thought to be faster in recent times, since the beginning of the <a class="uri" href="Holocene" title="wikilink">Holocene</a> 11,000 years ago.<a class="footnoteRef" href="#fn6" id="fnref6"><sup>6</sup></a><a class="footnoteRef" href="#fn7" id="fnref7"><sup>7</sup></a><a class="footnoteRef" href="#fn8" id="fnref8"><sup>8</sup></a></p>
<h3 id="parallel-mutations-and-saturation">Parallel mutations and saturation</h3>

<p>Parallel mutation (sometimes referred to as Homoplasy) or <a href="convergent_evolution" title="wikilink">convergent evolution</a> occurs when separate lineages have the same mutation independently occur at the same site in the genome. <a href="Saturation_(genetic)" title="wikilink">Saturation</a> occurs when a single site experiences multiple mutations. Parallel mutations and saturation result in the underestimation of the mutation rate because they are likely to be overlooked.<a class="footnoteRef" href="#fn9" id="fnref9"><sup>9</sup></a></p>
<h3 id="heteroplasmy">Heteroplasmy</h3>

<p>Individuals affected by <a class="uri" href="heteroplasmy" title="wikilink">heteroplasmy</a> have a mixture of mtDNA types, some with new mutations and some without. The new mutations may or may not be passed down to subsequent generations. Thus the presence of heteroplasmic individuals in a sample may complicate the calculation of mutation rates.<a class="footnoteRef" href="#fn10" id="fnref10"><sup>10</sup></a><a class="footnoteRef" href="#fn11" id="fnref11"><sup>11</sup></a></p>
<h2 id="methods">Methods</h2>
<h3 id="pedigree-based">Pedigree based</h3>

<p>Pedigree methods estimate the mutation rate by comparing the mtDNA sequences of a sample of parent/offspring pairs or analyzing mtDNA sequences of individuals from a deep-rooted genealogy. The number of new mutations in the sample is counted and divided by the total number of parent-to-child DNA transmission events to arrive at a mutation rate.<a class="footnoteRef" href="#fn12" id="fnref12"><sup>12</sup></a><a class="footnoteRef" href="#fn13" id="fnref13"><sup>13</sup></a></p>
<h3 id="phylogeny-based">Phylogeny based</h3>

<p>Phylogeny based methods are estimated by first reconstructing the haplotype of the most recent common ancestor (MRCA) of a sample of two or more genetic lineages. A requirement is that the time to the most recent common ancestor (<a href="Most_recent_common_ancestor#Time_to_MRCA_estimates" title="wikilink">TMRCA</a>) of the sample of lineages must already be known from other independent sources, usually the archeological record. The average number of mutations that have accumulated since the <a class="uri" href="MRCA" title="wikilink">MRCA</a> is then computed and divided by the TMRCA to arrive at the mutation rate. The human mutation rate is usually estimated by comparing the sequences of modern humans and chimpanzees and then reconstructing the ancestral haplotype of the chimpanzee-human common ancestor. According to the paleontological record the last common ancestor of humans may have lived around 6 million years ago.<a class="footnoteRef" href="#fn14" id="fnref14"><sup>14</sup></a></p>
<h3 id="pedigree-vs.-phylogeny-comparison">Pedigree vs. Phylogeny comparison</h3>

<p>Rates obtained by pedigree methods are about 10 times faster than those obtained by phylogenetic methods. Several factors acting together may be responsible for this difference. As pedigree methods record mutations in living subjects, the mutation rates from pedigree studies are closer to the germline mutation rate. Pedigree studies use genealogies that are only a few generations deep whereas phylogeny based methods use timescales that are thousands or millions of years deep. According to Henn et al. 2009, phylogeny based methods take into account events that occur over long time scales and are thus less affected by stochastic fluctuations. Howell et al. 2003 suggests that selection, saturation, parallel mutations and genetic drift are responsible for the differences observed between pedigree based methods and phylogeny based methods.</p>
<h2 id="estimating-based-on-amh-archaeology">Estimating based on AMH archaeology</h2>
<table>
<tbody>
<tr class="odd">
<td style="text-align: left;">
<p>Methods/parameters for archaeologically estimated dates of mitochondrial Eve</p></td>
</tr>
<tr class="even">
<td style="text-align: left;">
<p>Study</p></td>
</tr>
<tr class="odd">
<td style="text-align: left;">
<p>style= "background:#efefef" | </p></td>
</tr>
<tr class="even">
<td style="text-align: left;">
<p>style= "background:#efefef" | </p></td>
</tr>
</tbody>
</table>

<p>Anatomical modern humans (AMH) spread out of Africa and over a large area of Eurasia and left artifacts along the northern coast of the Southwest, South, Southeast and East Asia.  did not rely on a predicted T<sub>CHLCA</sub> to estimate <a href="single-nucleotide_polymorphism" title="wikilink">single-nucleotide polymorphism</a> (SNP) rates. Instead, they used evidence of colonization in Southeast Asia and Oceania to estimate mutation rates. In addition they used RFLP technology (<em><a href="Restriction_fragment_length_polymorphism" title="wikilink">Restriction fragment length polymorphism</a></em>) to examine differences between DNA. Using these techniques this group came up with a T<sub>MRCA</sub> of 140,000 to 290,000 years. It should be noted however that Cann et al. (1987) estimated the TMRCA of humans to be approximately 210 ky and the most recent estimates Soares et al. 2009 (using 7 million year chimpanzee human mtDNA MRCA) differ by only 9%, which is relatively close considering the wide confidence range for both estimates and calls for more ancient T<sub>CHLCA</sub>.</p>

<p>have reevaluated the predicted migrations globally and compared those to the actual evidence. This group used the coding regions of sequences. They postulate that the molecular clock based on chimp-human comparisons is not reliable, particularly in predicting recent migrations, such as founding migrations into Europe, Australia, and the Americans. With this technique this group came up with a T<sub>MRCA</sub> of 82,000 to 134,000 years.</p>
<h2 id="estimating-based-on-chlca">Estimating based on CHLCA</h2>

<p>Because chimps and humans share a matrilineal ancestor, establishing the geological age of that last ancestor allows the estimation of the mutation rate. The <a href="chimp-human_last_common_ancestor" title="wikilink">chimp-human last common ancestor</a> (CHLCA) is frequently applied as an anchor for mt-T<sub>MRCA</sub> studies with ranges between 4 and 13 million years cited in the literature.<a class="footnoteRef" href="#fn15" id="fnref15"><sup>15</sup></a> This is one source of variation in the time estimates. The other weakness is the non-clocklike accumulation of SNPs, would tend to make more recent branches look older than they actually are.<a class="footnoteRef" href="#fn16" id="fnref16"><sup>16</sup></a></p>
<table>
<tbody>
<tr class="odd">
<td style="text-align: left;">
<p>SNP rates as described by Soares et al. (2009)</p></td>
</tr>
<tr class="even">
<td style="text-align: left;">
<p>Regions(s)</p></td>
</tr>
<tr class="odd">
<td style="text-align: left;">
<p>style= "background:#efefef" Control<br/>
region</p></td>
</tr>
<tr class="even">
<td style="text-align: left;">
<p>style= "background:#efefef" | HVR II</p></td>
</tr>
<tr class="odd">
<td style="text-align: left;">
<p>style= "background:#efefef" | remaining</p></td>
</tr>
<tr class="even">
<td style="text-align: left;">
<p>style= "background:#efefef" Protein-<br/>
coding</p></td>
</tr>
<tr class="odd">
<td style="text-align: left;">
<p>style= "background:#efefef" | (<a href="wobble_base_pair" title="wikilink">3rd</a>)</p></td>
</tr>
<tr class="even">
<td style="text-align: left;">
<p>style= "background:#efefef" DNA encoding <a class="uri" href="rRNA" title="wikilink">rRNA</a> (rDNA)</p></td>
</tr>
<tr class="odd">
<td style="text-align: left;">
<p>style= "background:#efefef" DNA encoding <a class="uri" href="tRNA" title="wikilink">tRNA</a> (tDNA)</p></td>
</tr>
<tr class="even">
<td style="text-align: left;">
<p>style= "background:#efefef" other</p></td>
</tr>
<tr class="odd">
<td style="text-align: left;">
<p>style= "background:#dfdfdf" Colspan = 3 | T<sub>CHLCA</sub> assumed 6.5 Ma, relative rate to 1st &amp; 2nd codons</p></td>
</tr>
</tbody>
</table>

<p>These two sources may balance each other or amplify each other depending on the direction of the T<sub>CHLCA</sub> error. There are two major reasons why this method is widely employed. First the pedigree based rates are inappropriate for estimates for very long periods of time. Second, while the archaeology anchored rates represent the intermediate range, archaeological evidence for human colonization often occurs well after colonization. For example, colonization of Eurasia from west to east is believed to have occurred along the Indian Ocean. However, the oldest archaeological sites that also demonstrate anatomically modern humans (AMH) are in China and Australia, greater than 42,000 years in age. However the oldest Indian site with AMH remains is from 34,000 years, and another site with AMH compatible archaeology is in excess of 76,000 years in age.<a class="footnoteRef" href="#fn17" id="fnref17"><sup>17</sup></a> Therefore application of the anchor is a subjective interpretation of when humans were first present.</p>

<p>A simple measure the <a href="Genetic_divergence" title="wikilink">sequence divergence</a> between humans and chimps by observing the SNPs. Given that the mitogenome is about 16553 base pairs in length (each base-pair which can be aligned with known references is called a site).<a class="footnoteRef" href="#fn18" id="fnref18"><sup>18</sup></a> The formula is:<br/>
::

<math display="block" id="Human_mitochondrial_molecular_clock:0">
 <semantics>
  <mrow>
   <mrow>
    <mi>r</mi>
    <mi>a</mi>
    <mi>t</mi>
    <mi>e</mi>
   </mrow>
   <mo>=</mo>
   <mfrac>
    <mrow>
     <mi>S</mi>
     <mi>N</mi>
     <mi>P</mi>
     <mi>s</mi>
    </mrow>
    <mrow>
     <mo stretchy="false">(</mo>
     <mrow>
      <mn>2</mn>
      <msub>
       <mi>T</mi>
       <mrow>
        <mi>C</mi>
        <mi>H</mi>
        <mi>L</mi>
        <mi>C</mi>
        <mi>A</mi>
       </mrow>
      </msub>
      <mn>16553</mn>
     </mrow>
     <mo stretchy="false">)</mo>
    </mrow>
   </mfrac>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <eq></eq>
    <apply>
     <times></times>
     <ci>r</ci>
     <ci>a</ci>
     <ci>t</ci>
     <ci>e</ci>
    </apply>
    <apply>
     <divide></divide>
     <apply>
      <times></times>
      <ci>S</ci>
      <ci>N</ci>
      <ci>P</ci>
      <ci>s</ci>
     </apply>
     <apply>
      <times></times>
      <cn type="integer">2</cn>
      <apply>
       <csymbol cd="ambiguous">subscript</csymbol>
       <ci>T</ci>
       <apply>
        <times></times>
        <ci>C</ci>
        <ci>H</ci>
        <ci>L</ci>
        <ci>C</ci>
        <ci>A</ci>
       </apply>
      </apply>
      <cn type="integer">16553</cn>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   rate=\frac{SNPs}{(2T_{CHLCA}16553)}
  </annotation>
 </semantics>
</math>

 The '2' in the <a class="uri" href="denominator" title="wikilink">denominator</a> is derived from the 2 lineages, human and chimpanzee, that split from the CHLCA. Ideally it represents the accumulation of mutations on both lineages but in different positions (SNPs). As long as the number of SNP observed approximates the number of mutations this formula works well. However, at rapidly evolving sites mutations are obscured by saturation affects. Sorting positions within the mitogenome by rate and compensating for saturation are alternative approaches.<a class="footnoteRef" href="#fn19" id="fnref19"><sup>19</sup></a></p>

<p>Because the T<sub>CHLCA</sub> is subject to change with more paleontological information, the equation described above allows the comparison of TMRCA from different studies.</p>
<table>
<tbody>
<tr class="odd">
<td style="text-align: left;">
<p>Methods/parameters for estimating date of mitochondrial Eve</p></td>
</tr>
<tr class="even">
<td style="text-align: left;">
<p>Study</p></td>
</tr>
<tr class="odd">
<td style="text-align: left;">
<p>style= "background:#efefef" | </p></td>
</tr>
<tr class="even">
<td style="text-align: left;">
<p>style= "background:#efefef" | </p></td>
</tr>
<tr class="odd">
<td style="text-align: left;">
<p>style= "background:#efefef" | </p></td>
</tr>
<tr class="even">
<td style="text-align: left;">
<p>style= "background:#efefef" | </p></td>
</tr>
<tr class="odd">
<td style="text-align: left;">
<p>style= "background:#efefef" | </p></td>
</tr>
<tr class="even">
<td style="text-align: left;">
<p>style= "background:#efefef" | </p></td>
</tr>
<tr class="odd">
<td style="text-align: left;">
<p>style= "background:#dfdfdf" <small>Chimpanzee to Human = CH, LCA = last common ancestor</small></p></td>
</tr>
</tbody>
</table>
<h2 id="early-hvr-sequence-based-methods">Early, HVR, sequence-based methods</h2>

<p>To overcome the effects of <a href="saturation_(genetic)" title="wikilink">saturation</a>, HVR analysis relied on the <a href="transversion" title="wikilink">transversional</a> distance between humans and chimpanzees.<a class="footnoteRef" href="#fn20" id="fnref20"><sup>20</sup></a> A <a href="Transition_(genetics)" title="wikilink">transition</a> to transversion ratio was applied to this distance to estimate sequence divergence in the HVR between chimpanzees and humans, and divided by an assumed T<sub>CHLCA</sub> of 4 to 6 million years.<a class="footnoteRef" href="#fn21" id="fnref21"><sup>21</sup></a> Based on 26.4 substitutions between chimpanzee and human and 15:1 ratio, the estimated 396 transitions over 610 base-pairs demonstrated sequence divergence of 69.2% (rate * T<sub>CHLCA</sub> of 0.369), producing divergence rates of <em>roughly 11.5% to 17.3% per million years</em>.   also estimated the sequence divergence rate for the sites in the rapidly evolving HVR I and HVR II regions. As noted in the table above, the rate of evolution is so high that site saturation occurs in direct chimpanzee and human comparisons. Consequently this study used transversions, which evolve at a slower rate than the more common transition polymorphisms. Comparing chimp and human mitogenomes, they noted 26.4 transversions within the HVR regions, however they made no correction for saturation. As more HVR sequence was obtained following this study, it was noted that the dinucleotide site CRS:16181-16182 experienced numerous transversions in parsimony analysis, many of these were considered to be sequencing errors. However the sequencing of <a href="Neanderthal_genome_project" title="wikilink">Feldhofer I Neanderthal</a> revealed that there was also a transversion between humans and Neanderthals at this site.<a class="footnoteRef" href="#fn22" id="fnref22"><sup>22</sup></a> In addition,  noted three sites in which recurrent transversions had occurred in human lineages, two of which are in HVR I, 16265 (12 occurrences) and 16318(8 occurrences).<a class="footnoteRef" href="#fn23" id="fnref23"><sup>23</sup></a> Therefore, 26.4 transversions was an underestimate of the likely number of transversion events. The year 1991 study also used a transition-to-transversion ratio from the study of old world monkeys of 15:1. However, examination of chimp and gorilla HVR reveals a rate that is lower, and the examination of humans places the rate at 34:1.<a class="footnoteRef" href="#fn24" id="fnref24"><sup>24</sup></a> Therefore this study underestimated that level of sequence divergence between chimpanzee and human. The estimated sequence divergence 0.738/site (includes transversions) is significantly lower than the ~2.5 per site suggested by Soares et al. (2009). These two errors would result in an overestimate of the human mitochondrial TMRCA. However, they failed to detect the basal L0 lineage in the analysis and also failed to detect recurrent transitions in many lineages, which also underestimate the TMRCA. Also, Vigilant et al. (1991) used a more recent CHLCA anchor of 4 to 6 million years.</p>
<h2 id="coding-region-sequence-based-methods">Coding region sequence based methods</h2>

<p>Partial coding region sequence originally supplemented HVR studies because complete coding region sequence was uncommon. There were suspicions that the HVR studies had missed major branches based on some earlier RFLP and coding region studies.  was the first study to compare genomic sequences for coalescence analysis. Coding region sequence discriminated <a href="Haplogroup_M_(mtDNA)" title="wikilink">M</a> and <a href="Haplogroup_N_(mtDNA)" title="wikilink">N</a> haplogroups and <a href="Haplogroup_L0" title="wikilink">L0</a> and <a href="Haplogroup_L1" title="wikilink">L1</a> macrohaplogroups. Because the genomic DNA sequencing resolved the two deepest branches it improved some aspects estimating TMRCA over HVR sequence alone. Excluding the D-loop and using a 5-million-year T<sub>CHLCA</sub>,  estimated the mutation rate <em>to be 1.70 × 10<sup>−8</sup> per site per year</em> (rate * T<sub>CHLCA</sub> = 0.085, 15,435 sites).</p>

<p>However, coding region DNA has come under question because coding sequences are either under purifying selection to maintain structure and function, or under regional selection to evolve new capacities.<a class="footnoteRef" href="#fn25" id="fnref25"><sup>25</sup></a> The problem with mutations in the coding region has been described as such: mutations occurring in the coding region that are not <a href="Mutation#By_effect_on_fitness" title="wikilink">lethal</a> to the mitochondria can persist but are negatively <a href="natural_selection" title="wikilink">selective</a> to the host; over a few generations these will persist, but over thousands of generations these slowly are pruned from the population, leaving SNPs.<a class="footnoteRef" href="#fn26" id="fnref26"><sup>26</sup></a> However, over thousands of generations regionally selective mutations may not be discriminated from these transient coding region mutations. The problem with rare mutations in the human mitogenomes is significant enough to prompt a half-dozen recent studies on the matter.</p>

<p>estimated the <em>non-D loop region</em> evolution 1.7 × 10<sup>−8</sup> per year per site based on 53 non-identical genomic sequence overrepresenting Africa in a global sample. Despite this over-representation, the resolution of the L0 subbranches was lacking and one other deep L1 branches has been found. Despite these limitations that sampling was adequate for the hallmark study. Today, L0 is restricted to African populations, whereas L1 is the ancestral haplogroup of all non-Africans, as well as most Africans. Mitochondrial Eve's sequence can be approximated by comparing a sequence from L0 with a sequence from L1. By reconciling the mutations in L0 and L1. The mtDNA sequences of contemporary human populations will generally differ from Mitochondrial Eve's sequence by about 50 mutations.<a class="footnoteRef" href="#fn27" id="fnref27"><sup>27</sup></a><a class="footnoteRef" href="#fn28" id="fnref28"><sup>28</sup></a> Mutation rates were not classified according to site (other than excluding the HVR reigons). The T<sub>CHLCA</sub> used in the year 2000 study of 5 Ma was also lower than values used in the most recent studies.</p>
<h2 id="inter-comparing-rates-and-studies">Inter-comparing rates and studies</h2>

<p>Molecular clocking of mitochondrial DNA has been criticized because of its inconsistent molecular clock.<a class="footnoteRef" href="#fn29" id="fnref29"><sup>29</sup></a><a class="footnoteRef" href="#fn30" id="fnref30"><sup>30</sup></a><a class="footnoteRef" href="#fn31" id="fnref31"><sup>31</sup></a> A retrospective analysis of any pioneering process will reveal inadequacies. With mitochondrial the inadequacies are the <a href="argument_from_ignorance" title="wikilink">argument from ignorance</a> of rate variation and overconfidence concerning the T<sub>CHLCA</sub> of 5 Ma. Lack of historical perspective might explain the second issue, the problem of rate variation is something that could only be resolved by the massive study of mitochondria that followed. The number of HVR sequences that have accumulated from 1987 to 2000 increased by magnitudes.  used 2196 mitogenomic sequences and uncovered 10,683 substitution events within these sequences. Eleven of 16560 sites in the mitogenome produced greater than 11% of all the substitutions with statistically significant rate variation within the 11 sites.<a class="footnoteRef" href="#fn32" id="fnref32"><sup>32</sup></a> They argue that there is a neutral-site mutation rate which is a magnitude slower than rate observed for the fastest site, CRS 16519. Consequently, purifying selection aside, the rate of mutation itself varies between sites, with a few sites much more likely to undergo new mutations relative to others.<a class="footnoteRef" href="#fn33" id="fnref33"><sup>33</sup></a> Soares et al. (2009) noted two spans of DNA, CRS 2651-2700 and 3028-3082, that had no SNPs within the 2196 mitogenomic sequences.</p>
<h2 id="notes">Notes</h2>
<h2 id="footnotes">Footnotes</h2>
<h2 id="references">References</h2>
<ul>
<li></li>
<li></li>
<li></li>
<li></li>
<li></li>
<li></li>
<li></li>
<li></li>
<li></li>
<li></li>
<li></li>
<li></li>
<li></li>
<li></li>
<li></li>
<li></li>
<li></li>
<li></li>
<li></li>
<li></li>
<li></li>
<li></li>
<li></li>
<li></li>
<li></li>
<li></li>
<li></li>
</ul>

<p>"</p>

<p><a href="Category:Biology_timelines" title="wikilink">Category:Biology timelines</a> <a class="uri" href="Category:DNA" title="wikilink">Category:DNA</a> <a href="Category:Evolutionary_biology" title="wikilink">Category:Evolutionary biology</a> <a href="Category:Genetic_genealogy" title="wikilink">Category:Genetic genealogy</a> <a href="Category:Human_evolution" title="wikilink">Category:Human evolution</a> <a href="Category:Human_mtDNA_haplogroups" title="wikilink">*</a> <a class="uri" href="Category:Phylogenetics" title="wikilink">Category:Phylogenetics</a> <a href="Category:Human_mitochondrial_genetics" title="wikilink">Category:Human mitochondrial genetics</a> <a href="Category:Prehistoric_Africa" title="wikilink">Category:Prehistoric Africa</a></p>
<section class="footnotes">
<hr/>
<ol>
<li id="fn1"></li>
<li id="fn2"><a href="#fnref2">↩</a></li>
<li id="fn3"></li>
<li id="fn4"></li>
<li id="fn5"><a href="#fnref5">↩</a></li>
<li id="fn6"></li>
<li id="fn7"><a href="#fnref7">↩</a></li>
<li id="fn8"><a href="#fnref8">↩</a></li>
<li id="fn9"></li>
<li id="fn10"></li>
<li id="fn11"><a href="#fnref11">↩</a></li>
<li id="fn12"></li>
<li id="fn13"></li>
<li id="fn14"></li>
<li id="fn15"></li>
<li id="fn16">see: <a href="#fnref16">↩</a></li>
<li id="fn17"></li>
<li id="fn18"><a href="#fnref18">↩</a></li>
<li id="fn19">See: ,<a href="#fnref19">↩</a></li>
<li id="fn20"><a href="#fnref20">↩</a></li>
<li id="fn21"><a href="#fnref21">↩</a></li>
<li id="fn22"><a href="#fnref22">↩</a></li>
<li id="fn23">Soares et al excluded 16182 and 16183 from their analysis<a href="#fnref23">↩</a></li>
<li id="fn24"><a href="#fnref24">↩</a></li>
<li id="fn25">see: , <a href="#fnref25">↩</a></li>
<li id="fn26"></li>
<li id="fn27"><a href="#fnref27">↩</a></li>
<li id="fn28"><a href="#fnref28">↩</a></li>
<li id="fn29"><a href="#fnref29">↩</a></li>
<li id="fn30"><a href="#fnref30">↩</a></li>
<li id="fn31"><a href="#fnref31">↩</a></li>
<li id="fn32">(CRS sites 16519, 152, 16311, 145, 195, 16189, 16129, 16083, 16362, 160, 709, 16129, 16083, 16362, 150, and 709)<a href="#fnref32">↩</a></li>
<li id="fn33"><a href="#fnref33">↩</a></li>
</ol>
</section>
</body>
</html>
