<html lang="en">
<head>
<meta charset="utf-8"/>
<title offset="1949">Linguistic sequence complexity</title>
<script src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_SVG.js" type="text/javascript">
</script>
</head>
<body>
<h1>Linguistic sequence complexity</h1>
<hr/>

<p><strong>Linguistic sequence complexity</strong> (LC) is a measure of the 'vocabulary richness' of a genetic text in <a href="gene_sequence" title="wikilink">gene sequences</a>.<a class="footnoteRef" href="#fn1" id="fnref1"><sup>1</sup></a> When a <a class="uri" href="nucleotide" title="wikilink">nucleotide</a> sequence is written as text using a four-letter alphabet, the repetitiveness of the text, that is, the repetition of its <a href="N-gram" title="wikilink">N-grams</a> (words), can be calculated and serves as a measure of sequence complexity. Thus, the more complex a <a href="DNA_sequence" title="wikilink">DNA sequence</a>, the richer its <a class="uri" href="oligonucleotide" title="wikilink">oligonucleotide</a> vocabulary, whereas repetitious sequences have relatively lower complexities. Subsequent work improved the original algorithm described in <a href="Edward_Trifonov" title="wikilink">Trifonov</a> (1990),<a class="footnoteRef" href="#fn2" id="fnref2"><sup>2</sup></a> without changing the essence of the linguistic complexity approach.<a class="footnoteRef" href="#fn3" id="fnref3"><sup>3</sup></a><a class="footnoteRef" href="#fn4" id="fnref4"><sup>4</sup></a><a class="footnoteRef" href="#fn5" id="fnref5"><sup>5</sup></a></p>

<p>The meaning of LC may be better understood by regarding the presentation of a sequence as a <a href="Tree_(data_structure)" title="wikilink">tree</a> of all subsequences of the given sequence. The most complex sequences have maximally balanced trees, while the measure of imbalance or tree asymmetry serves as a complexity measure. The number of nodes at the tree level <mtpl></mtpl> is equal to the actual vocabulary size of words with the length <mtpl></mtpl> in a given sequence; the number of nodes in the most balanced tree, which corresponds to the most complex sequence of length N, at the tree level <mtpl></mtpl> is either 4<sup>i</sup> or N-j+1, whichever is smaller. Complexity (<mtpl></mtpl>) of a sequence fragment (with a length RW) can be directly calculated as the product of vocabulary-usage measures (U<sub>i</sub>):<a class="footnoteRef" href="#fn6" id="fnref6"><sup>6</sup></a> 

<math display="inline" id="Linguistic_sequence_complexity:0">
 <semantics>
  <mrow>
   <mrow>
    <mi>C</mi>
    <mo>=</mo>
    <mrow>
     <msub>
      <mi>U</mi>
      <mn>1</mn>
     </msub>
     <msub>
      <mi>U</mi>
      <mn>2</mn>
     </msub>
     <mi mathvariant="normal">…</mi>
     <msub>
      <mi>U</mi>
      <mi>i</mi>
     </msub>
     <mi mathvariant="normal">…</mi>
    </mrow>
   </mrow>
   <mo>.</mo>
   <msub>
    <mi>U</mi>
    <mi>w</mi>
   </msub>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <csymbol cd="ambiguous">formulae-sequence</csymbol>
    <apply>
     <eq></eq>
     <ci>C</ci>
     <apply>
      <times></times>
      <apply>
       <csymbol cd="ambiguous">subscript</csymbol>
       <ci>U</ci>
       <cn type="integer">1</cn>
      </apply>
      <apply>
       <csymbol cd="ambiguous">subscript</csymbol>
       <ci>U</ci>
       <cn type="integer">2</cn>
      </apply>
      <ci>normal-…</ci>
      <apply>
       <csymbol cd="ambiguous">subscript</csymbol>
       <ci>U</ci>
       <ci>i</ci>
      </apply>
      <ci>normal-…</ci>
     </apply>
    </apply>
    <apply>
     <csymbol cd="ambiguous">subscript</csymbol>
     <ci>U</ci>
     <ci>w</ci>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   C=U_{1}U_{2}...U_{i}....U_{w}
  </annotation>
 </semantics>
</math>

</p>

<p>Vocabulary usage for <a class="uri" href="oligomers" title="wikilink">oligomers</a> of a given size <mtpl></mtpl> can be defined as the ratio of the actual vocabulary size of a given sequence to the maximal possible vocabulary size for a sequence of that length. For example, U<sub>2</sub> for the sequence ACGGGAAGCTGATTCCA = 14/16, as it contains 14 of 16 possible different dinucleotides; U<sub>3</sub> for the same sequence = 15/15, and U<sub>4</sub>=14/14. For the sequence ACACACACACACACACA, U<sub>1</sub>=1/2; U<sub>2</sub>=2/16=0.125, as it has a simple vocabulary of only two dinucleotides; U<sub>3</sub> for this sequence = 2/15. k-tuples with k from two to W considered, while W depends on RW. For RW values less than 18, W is equal to 3; for RW less than 67, W is equal to 4; for RW provides a measure of sequence complexity in the range 0<c class="footnoteRef" different="" formula="" from="" href="#fn7" id="fnref7" is="" lc="" measure<a="" original="" the="" this=""><sup>7</sup></c></p></body></html> in two respects: in the way vocabulary usage U<sub>i</sub> is calculated, and because <mtpl></mtpl> is not in the range of 2 to N-1 but only up to W. This limitation on the range of U<sub>i</sub> makes the algorithm substantially more efficient without loss of power.<a class="footnoteRef" href="#fn8" id="fnref8"><sup>8</sup></a> In <a class="footnoteRef" href="#fn9" id="fnref9"><sup>9</sup></a>was used another modified version, wherein linguistic complexity (LC) is defined as the ratio of the number of substrings of any length present in the string to the maximum possible number of substrings. Maximum vocabulary over word sizes 1 to m can be calculated according to the simple formula .<a class="footnoteRef" href="#fn10" id="fnref10"><sup>10</sup></a> This sequence analysis complexity calculation can be used to search for conserved regions between compared sequences for the detection of low-complexity regions including simple sequence repeats, imperfect <a href="Direct_repeat" title="wikilink">direct</a> or <a href="inverted_repeat" title="wikilink">inverted repeats</a>, polypurine and polypyrimidine <a href="Triple-stranded_DNA" title="wikilink">triple-stranded DNA structures</a>, and four-stranded structures (such as <a href="G-quadruplex" title="wikilink">G-quadruplexes</a>).<a class="footnoteRef" href="#fn11" id="fnref11"><sup>11</sup></a>
<h2 id="references">References</h2>

<p>"</p>

<p><a href="Category:Nucleic_acids" title="wikilink">Category:Nucleic acids</a> <a class="uri" href="Category:Bioinformatics" title="wikilink">Category:Bioinformatics</a></p>
<section class="footnotes">
<hr/>
<ol>
<li id="fn1"><a href="#fnref1">↩</a></li>
<li id="fn2"></li>
<li id="fn3"><a href="#fnref3">↩</a></li>
<li id="fn4"><a href="#fnref4">↩</a></li>
<li id="fn5"><a href="#fnref5">↩</a></li>
<li id="fn6"></li>
<li id="fn7"></li>
<li id="fn8"></li>
<li id="fn9"><a href="#fnref9">↩</a></li>
<li id="fn10"></li>
<li id="fn11"><a href="#fnref11">↩</a></li>
</ol>
</section>


