<html lang="en">
<head>
<meta charset="utf-8"/>
<title offset="1125">MA plot</title>
<script src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_SVG.js" type="text/javascript">
</script>
</head>
<body>
<h1>MA plot</h1>
<hr/>

<p>An <strong>MA plot</strong> is an application of a <a href="Bland–Altman_plot" title="wikilink">Bland–Altman plot</a> for visual representation of two channel <a href="DNA_microarray" title="wikilink">DNA microarray</a> gene expression data which has been transformed onto the M (log ratios) and A (<a href="arithmetic_mean" title="wikilink">mean average</a>) scale.</p>
<h2 id="explanation">Explanation</h2>

<p><a href="DNA_microarray" title="wikilink">Microarray</a> data is often normalized within arrays to control for systematic biases in dye coupling and hybridization efficiencies, as well as other technical biases in the DNA probes and the print tip used to spot the array.<a class="footnoteRef" href="#fn1" id="fnref1"><sup>1</sup></a> By minimizing these systematic variations, true biological differences can be found. To determine whether normalization is needed, one can plot <a href="cyanine#Cy3_and_Cy5" title="wikilink">Cy5</a> (R) intensities against <a href="cyanine#Cy3_and_Cy5" title="wikilink">Cy3</a> (G) intensities and see whether the slope of the line is around 1. An improved method, which is basically a scaled, 45 degree rotation of the R vs. G plot is an MA-plot.<a class="footnoteRef" href="#fn2" id="fnref2"><sup>2</sup></a> The MA-plot is a plot of the distribution of the red/green intensity ratio ('M') plotted by the average intensity ('A'). M and A are defined by the following equations.</p>

<p>

<math display="block" id="MA_plot:0">
 <semantics>
  <mrow>
   <mi>M</mi>
   <mo>=</mo>
   <mrow>
    <msub>
     <mi>log</mi>
     <mn>2</mn>
    </msub>
    <mrow>
     <mo stretchy="false">(</mo>
     <mrow>
      <mi>R</mi>
      <mo>/</mo>
      <mi>G</mi>
     </mrow>
     <mo stretchy="false">)</mo>
    </mrow>
   </mrow>
   <mo>=</mo>
   <mrow>
    <mrow>
     <msub>
      <mi>log</mi>
      <mn>2</mn>
     </msub>
     <mrow>
      <mo stretchy="false">(</mo>
      <mi>R</mi>
      <mo stretchy="false">)</mo>
     </mrow>
    </mrow>
    <mo>-</mo>
    <mrow>
     <msub>
      <mi>log</mi>
      <mn>2</mn>
     </msub>
     <mrow>
      <mo stretchy="false">(</mo>
      <mi>G</mi>
      <mo stretchy="false">)</mo>
     </mrow>
    </mrow>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <and></and>
    <apply>
     <eq></eq>
     <ci>M</ci>
     <apply>
      <apply>
       <csymbol cd="ambiguous">subscript</csymbol>
       <log></log>
       <cn type="integer">2</cn>
      </apply>
      <apply>
       <divide></divide>
       <ci>R</ci>
       <ci>G</ci>
      </apply>
     </apply>
    </apply>
    <apply>
     <eq></eq>
     <share href="#.cmml">
     </share>
     <apply>
      <minus></minus>
      <apply>
       <apply>
        <csymbol cd="ambiguous">subscript</csymbol>
        <log></log>
        <cn type="integer">2</cn>
       </apply>
       <ci>R</ci>
      </apply>
      <apply>
       <apply>
        <csymbol cd="ambiguous">subscript</csymbol>
        <log></log>
        <cn type="integer">2</cn>
       </apply>
       <ci>G</ci>
      </apply>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   M=\log_{2}(R/G)=\log_{2}(R)-\log_{2}(G)
  </annotation>
 </semantics>
</math>

</p>

<p>

<math display="block" id="MA_plot:1">
 <semantics>
  <mrow>
   <mi>A</mi>
   <mo>=</mo>
   <mrow>
    <mfrac>
     <mn>1</mn>
     <mn>2</mn>
    </mfrac>
    <mrow>
     <msub>
      <mi>log</mi>
      <mn>2</mn>
     </msub>
     <mrow>
      <mo stretchy="false">(</mo>
      <mrow>
       <mi>R</mi>
       <mi>G</mi>
      </mrow>
      <mo stretchy="false">)</mo>
     </mrow>
    </mrow>
   </mrow>
   <mo>=</mo>
   <mrow>
    <mfrac>
     <mn>1</mn>
     <mn>2</mn>
    </mfrac>
    <mrow>
     <mo stretchy="false">(</mo>
     <mrow>
      <mrow>
       <msub>
        <mi>log</mi>
        <mn>2</mn>
       </msub>
       <mrow>
        <mo stretchy="false">(</mo>
        <mi>R</mi>
        <mo stretchy="false">)</mo>
       </mrow>
      </mrow>
      <mo>+</mo>
      <mrow>
       <msub>
        <mi>log</mi>
        <mn>2</mn>
       </msub>
       <mrow>
        <mo stretchy="false">(</mo>
        <mi>G</mi>
        <mo stretchy="false">)</mo>
       </mrow>
      </mrow>
     </mrow>
     <mo stretchy="false">)</mo>
    </mrow>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <and></and>
    <apply>
     <eq></eq>
     <ci>A</ci>
     <apply>
      <times></times>
      <apply>
       <divide></divide>
       <cn type="integer">1</cn>
       <cn type="integer">2</cn>
      </apply>
      <apply>
       <apply>
        <csymbol cd="ambiguous">subscript</csymbol>
        <log></log>
        <cn type="integer">2</cn>
       </apply>
       <apply>
        <times></times>
        <ci>R</ci>
        <ci>G</ci>
       </apply>
      </apply>
     </apply>
    </apply>
    <apply>
     <eq></eq>
     <share href="#.cmml">
     </share>
     <apply>
      <times></times>
      <apply>
       <divide></divide>
       <cn type="integer">1</cn>
       <cn type="integer">2</cn>
      </apply>
      <apply>
       <plus></plus>
       <apply>
        <apply>
         <csymbol cd="ambiguous">subscript</csymbol>
         <log></log>
         <cn type="integer">2</cn>
        </apply>
        <ci>R</ci>
       </apply>
       <apply>
        <apply>
         <csymbol cd="ambiguous">subscript</csymbol>
         <log></log>
         <cn type="integer">2</cn>
        </apply>
        <ci>G</ci>
       </apply>
      </apply>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   A=\frac{1}{2}\log_{2}(RG)=\frac{1}{2}(\log_{2}(R)+\log_{2}(G))
  </annotation>
 </semantics>
</math>

</p>

<p>M is, therefore, the <a href="binary_logarithm" title="wikilink">binary logarithm</a> of the intensity ratio (or difference between log intensities) and A is the average log intensity for a dot in the plot. MA plots are then used to visualize intensity-dependent ratio of raw microarray data (microarrays typically show a bias here, with higher A resulting in higher |M|, i.e. the brighter the spot the more likely an observed difference between sample and control). The MA plot uses M as the y-axis and A as the x-axis and gives a quick overview of the <a href="frequency_distribution" title="wikilink">distribution</a> of the data.</p>

<p>In many microarray gene expression experiments, an underlying assumption is that most of the genes would not see any change in their expression therefore the majority of the points on the y-axis (M) would be located at 0, since Log(1) is 0. If this is not the case, then a <a href="normalization_(statistics)" title="wikilink">normalization</a> method such as <a class="uri" href="LOESS" title="wikilink">LOESS</a> should be applied to the data before statistical analysis. (On the diagram below see the red line running below the zero mark before normalization, it should be straight. Since it is not straight, the data should be normalized. After being normalized, the red line is straight on the zero line and shows as pink/black.)</p>
<h2 id="packages">Packages</h2>

<p>Several <a class="uri" href="Bioconductor" title="wikilink">Bioconductor</a> packages, for the <a href="R_(programming_language)" title="wikilink">R software</a>, provide the facility for creating MA plots, these include affy (ma.plot, mva.pairs), limma (plotMA), marray (maPlot), and edgeR(maPlot)</p>

<p>Similar <a href="RA_plot" title="wikilink">"RA" plots</a> can be generated using the raPlot function in the caroline <a href="http://cran.r-project.org/">CRAN</a> <a href="R_(programming_language)" title="wikilink">R</a> package.</p>
<h3 id="example-in-the-r-programming-language">Example in the R programming language</h3>
<pre class="rsplus"><code>library(affy)

if (require(affydata)) 
{
     data(Dilution)
}

y &lt;- (exprs(Dilution)[, c("20B", "10A")])

x11()

ma.plot( rowMeans(log2(y)), log2(y[, 1])-log2(y[, 2]), cex=1 ) 

title("Dilutions Dataset (array 20B v 10A)")

library(preprocessCore)

#do a quantile normalization
x &lt;- normalize.quantiles(y)

x11()

ma.plot( rowMeans(log2(x)), log2(x[, 1])-log2(x[, 2]), cex=1 ) 
title("Post Norm: Dilutions Dataset (array 20B v 10A)")</code></pre>

<p>{{-}}</p>
<h2 id="see-also">See also</h2>
<ul>
<li><a href="RA_plot" title="wikilink">RA plot</a></li>
<li><a href="Bland-Altman_plot" title="wikilink">Bland-Altman plot</a></li>
</ul>
<h2 id="references">References</h2>
<references>
</references>

<p>"</p>

<p><a href="Category:Gene_expression" title="wikilink">Category:Gene expression</a> <a class="uri" href="Category:Microarrays" title="wikilink">Category:Microarrays</a> <a href="Category:Plots_(graphics)" title="wikilink">Category:Plots (graphics)</a></p>
<section class="footnotes">
<hr/>
<ol>
<li id="fn1">YH Yang, S Dudoit, P Luu, DM Lin, V Peng, J Ngai, TP Speed. (2002). Normalization for cDNA microarray data: a robust composite method addressing single and multiple slide systematic variation. <em>Nucleic Acids Research</em> vol. 30 (4) pp. e15.<a href="#fnref1">↩</a></li>
<li id="fn2">Dudoit, S, Yang, YH, Callow, MJ, Speed, TP. (2002). Statistical methods for identifying differentially expressed genes in replicated cDNA microarray experiments. <em>Stat. Sin.</em> 12:1 111-139<a href="#fnref2">↩</a></li>
</ol>
</section>
</body>
</html>
