<html lang="en">
<head>
<meta charset="utf-8"/>
<title offset="712">McDonald–Kreitman test</title>
<script src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_SVG.js" type="text/javascript">
</script>
</head>
<body>
<h1>McDonald–Kreitman test</h1><hr/>

<p>The <strong>McDonald–Kreitman test</strong> is a statistical test often used by evolution and population biologists to detect and measure the amount of <a href="adaptive_evolution" title="wikilink">adaptive evolution</a> within a species by determining whether adaptive evolution has occurred, and the proportion of substitutions that resulted from positive selection (also known as <a href="directional_selection" title="wikilink">directional selection</a>). To do this, the McDonald–Kreitman test compares the amount of variation within a species (<a href="Polymorphic_code" title="wikilink">polymorphism</a>) to the divergence between species (substitutions) at two types of sites, neutral and nonneutral. A substitution refers to a nucleotide that is fixed within one species, but a different nucleotide is fixed within a second species at the same base pair of homologous DNA sequences.<a class="footnoteRef" href="#fn1" id="fnref1"><sup>1</sup></a> A site is nonneutral if it is either advantageous or deleterious.<a class="footnoteRef" href="#fn2" id="fnref2"><sup>2</sup></a> The two types of sites can be either synonymous or nonsynonymous within a protein-coding region. In a protein-coding sequence of DNA, a site is synonymous if a <a href="point_mutation" title="wikilink">point mutation</a> at that site would not change the amino acid, also known as a <a href="silent_mutation" title="wikilink">silent mutation</a>. Because the mutation did not result in a change in the amino acid that was originally coded for by the protein-coding sequence, the phenotype, or the observable trait, of the organism is generally unchanged by the silent mutation.<a class="footnoteRef" href="#fn3" id="fnref3"><sup>3</sup></a> A site in a protein-coding sequence of DNA is nonsynonymous if a point mutation at that site results in a change in the amino acid, resulting in a change in the organism's phenotype.<a class="footnoteRef" href="#fn4" id="fnref4"><sup>4</sup></a> Typically, silent mutations in protein-coding regions are used as the "control" in the McDonald–Kreitman test.</p>

<p>In 1991, John H. McDonald and <a href="Martin_Kreitman" title="wikilink">Martin Kreitman</a> derived the McDonald–Kreitman test while performing an experiment with <a class="uri" href="Drosophila" title="wikilink">Drosophila</a> (fruit flies) and their differences in amino acid sequence of the alcohol dehydrogenase gene. McDonald and Kreitman proposed this method to estimate the proportion of substitutions that are fixed by positive selection rather than by genetic drift.<a class="footnoteRef" href="#fn5" id="fnref5"><sup>5</sup></a></p>

<p>In order to set up the McDonald–Kreitman test, we must first set up a two-way <a href="contingency_table" title="wikilink">contingency table</a> of our data on the species being investigated as shown below:</p>
<table>
<thead>
<tr class="header">
<th style="text-align: left;"></th>
<th style="text-align: left;">
<p>Fixed</p></th>
<th style="text-align: left;">
<p>Polymorphic</p></th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;">
<p><a href="Synonymous_substitution" title="wikilink">Synonymous</a></p></td>
<td style="text-align: left;">
<p><em>D</em><sub>s</sub></p></td>
<td style="text-align: left;">
<p><em>P</em><sub>s</sub></p></td>
</tr>
<tr class="even">
<td style="text-align: left;">
<p>Nonsynonymous</p></td>
<td style="text-align: left;">
<p><em>D</em><sub>n</sub></p></td>
<td style="text-align: left;">
<p><em>P</em><sub>n</sub></p></td>
</tr>
</tbody>
</table>
<ul>
<li><em>D</em><sub>s</sub>: the number of synonymous substitutions per gene</li>
<li><em>D</em><sub>n</sub>: the number of non-synonymous substitutions per gene</li>
<li><em>P</em><sub>s</sub>: the number of synonymous polymorphisms per gene</li>
<li><em>P</em><sub>n</sub>: the number of non-synonymous polymorphisms per gene</li>
</ul>

<p>To quantify the values for <em>D</em><sub>s</sub>, <em>D</em><sub>n</sub>, <em>P</em><sub>s</sub>, and <em>P</em><sub>n</sub>, you count the number of differences in the protein-coding region for each type of variable in the contingency table.</p>

<p>The <a href="null_hypothesis" title="wikilink">null hypothesis</a> of the McDonald–Kreitman test is that the ratio of nonsynonymous to synonymous variation within a species is going to equal the ratio of nonsynonymous to synonymous variation between species (i.e. <em>D</em><sub>n</sub>/<em>D</em><sub>s</sub> = <em>P</em><sub>n</sub>/<em>P</em><sub>s</sub>). When positive or <a href="negative_selection_(natural_selection)" title="wikilink">negative selection (natural selection)</a> influences nonsynonymous variation, the ratios will no longer equal. The ratio of nonsynonymous to synonymous variation between species is going to be lower than the ratio of nonsynonymous to synonymous variation within species (i.e. <em>D</em><sub>n</sub>/<em>D</em><sub>s</sub> n/<em>P</em><sub>s</sub>) when negative selection is at work, and deleterious mutations strongly affect polymorphism. The ratio of nonsynonymous to synonymous variation within species is lower than the ratio of nonsynonymous to synonymous variation between species (i.e. <em>D</em><sub>n</sub>/<em>D</em><sub>s</sub> &gt; <em>P</em><sub>n</sub>/<em>P</em><sub>s</sub>) when we observe positive selection. Since mutations under positive selection spread through a population rapidly, they don't contribute to polymorphism but do have an effect on divergence.<a class="footnoteRef" href="#fn6" id="fnref6"><sup>6</sup></a></p>

<p>Using an equation derived by Smith and Eyre-Walker, we can estimate the proportion of base substitutions fixed by natural selection, α,<a class="footnoteRef" href="#fn7" id="fnref7"><sup>7</sup></a> using the following formula:</p>

<p>

<math display="block" id="McDonald–Kreitman_test:0">
 <semantics>
  <mrow>
   <mi>α</mi>
   <mo>=</mo>
   <mrow>
    <mn>1</mn>
    <mo>-</mo>
    <mfrac>
     <mrow>
      <msub>
       <mi>D</mi>
       <mi>s</mi>
      </msub>
      <msub>
       <mi>P</mi>
       <mi>n</mi>
      </msub>
     </mrow>
     <mrow>
      <msub>
       <mi>D</mi>
       <mi>n</mi>
      </msub>
      <msub>
       <mi>P</mi>
       <mi>s</mi>
      </msub>
     </mrow>
    </mfrac>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <eq></eq>
    <ci>α</ci>
    <apply>
     <minus></minus>
     <cn type="integer">1</cn>
     <apply>
      <divide></divide>
      <apply>
       <times></times>
       <apply>
        <csymbol cd="ambiguous">subscript</csymbol>
        <ci>D</ci>
        <ci>s</ci>
       </apply>
       <apply>
        <csymbol cd="ambiguous">subscript</csymbol>
        <ci>P</ci>
        <ci>n</ci>
       </apply>
      </apply>
      <apply>
       <times></times>
       <apply>
        <csymbol cd="ambiguous">subscript</csymbol>
        <ci>D</ci>
        <ci>n</ci>
       </apply>
       <apply>
        <csymbol cd="ambiguous">subscript</csymbol>
        <ci>P</ci>
        <ci>s</ci>
       </apply>
      </apply>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \alpha=1-\frac{D_{s}P_{n}}{D_{n}P_{s}}
  </annotation>
 </semantics>
</math>

</p>

<p>Alpha represents the proportion of substitutions driven by positive selection. Alpha can be equal to any number between -∞ and 1. Negative values of alpha are produced by sampling error or violations of the model, such as the segregation of slightly deleterious amino acid mutations.<a class="footnoteRef" href="#fn8" id="fnref8"><sup>8</sup></a> Similar to above, our null hypothesis here is that α=0, and we expect <em>D</em><sub>n</sub>/<em>D</em><sub>s</sub> to equal <em>P</em><sub>n</sub>/<em>P</em><sub>s</sub>.<a class="footnoteRef" href="#fn9" id="fnref9"><sup>9</sup></a></p>
<h2 id="the-neutrality-index">The Neutrality Index</h2>

<p>The neutrality index (NI) quantifies the direction and degree of departure from neutrality (where <em>P</em><sub>n</sub>/<em>P</em><sub>s</sub> and <em>D</em><sub>n</sub>/<em>D</em><sub>s</sub> ratios equal). When assuming that silent mutations are neutral, a neutrality index greater than 1 (i.e. NI &gt; 1) indicates negative selection is at work, resulting in an excess of amino acid polymorphism. This occurs because natural selection is favoring the purifying selection, and the weeding out of deleterious alleles.<a class="footnoteRef" href="#fn10" id="fnref10"><sup>10</sup></a> Because silent mutations are neutral, a neutrality index lower than 1 (i.e. NI <a href="http://mbe.oxfordjournals.org/content/28/1/63">7</a>, Stoletzki, N. Eyre-Walker, A. 2010. Estimation of the Neutrality Index. Molecular Biology and Evolution 28 (1): 63-70. To find the neutrality index, we can use the following equation:</p>

<p>

<math display="block" id="McDonald–Kreitman_test:1">
 <semantics>
  <mrow>
   <mrow>
    <mi>N</mi>
    <mi>I</mi>
   </mrow>
   <mo>=</mo>
   <mfrac>
    <mrow>
     <msub>
      <mi>P</mi>
      <mi>n</mi>
     </msub>
     <mo>/</mo>
     <msub>
      <mi>P</mi>
      <mi>s</mi>
     </msub>
    </mrow>
    <mrow>
     <msub>
      <mi>D</mi>
      <mi>n</mi>
     </msub>
     <mo>/</mo>
     <msub>
      <mi>D</mi>
      <mi>s</mi>
     </msub>
    </mrow>
   </mfrac>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <eq></eq>
    <apply>
     <times></times>
     <ci>N</ci>
     <ci>I</ci>
    </apply>
    <apply>
     <divide></divide>
     <apply>
      <divide></divide>
      <apply>
       <csymbol cd="ambiguous">subscript</csymbol>
       <ci>P</ci>
       <ci>n</ci>
      </apply>
      <apply>
       <csymbol cd="ambiguous">subscript</csymbol>
       <ci>P</ci>
       <ci>s</ci>
      </apply>
     </apply>
     <apply>
      <divide></divide>
      <apply>
       <csymbol cd="ambiguous">subscript</csymbol>
       <ci>D</ci>
       <ci>n</ci>
      </apply>
      <apply>
       <csymbol cd="ambiguous">subscript</csymbol>
       <ci>D</ci>
       <ci>s</ci>
      </apply>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   NI=\frac{P_{n}/P_{s}}{D_{n}/D_{s}}
  </annotation>
 </semantics>
</math>

</p>
<h2 id="sources-of-error-with-the-mcdonaldkreitman-test">Sources of Error with the McDonald–Kreitman test</h2>

<p>One drawback of performing a McDonald–Kreitman test is that the test is vulnerable to error, as with every other statistical test. Many factors can contribute to errors in estimates of the level of adaptive evolution, including presence of slightly deleterious mutations, variation of mutation rates across the genome, variation in coalescent histories across the genome, and changes in the effective population size. All these factors result in α being underestimated.<a class="footnoteRef" href="#fn11" id="fnref11"><sup>11</sup></a> However, according to research done by Charlesworth (2008),<a class="footnoteRef" href="#fn12" id="fnref12"><sup>12</sup></a> Andolfatto(2008),<a class="footnoteRef" href="#fn13" id="fnref13"><sup>13</sup></a> and Eyre-Walker(2006),<a class="footnoteRef" href="#fn14" id="fnref14"><sup>14</sup></a> none of these factors are significant enough to make scientists believe the McDonald–Kreitman test is unreliable, except for the presence of slightly deleterious mutations in species.</p>

<p>In general, the McDonald-Kreitman test is often thought to be unreliable because of how significantly the test tends to underestimate the degree of adaptive evolution in the presence of slightly deleterious mutations. A slightly deleterious mutation can be defined as a mutation that negative selection acts on only very weakly so that its fate is determined by both selection and random genetic drift.<a class="footnoteRef" href="#fn15" id="fnref15"><sup>15</sup></a> If slightly deleterious mutations are segregating in the population, then it becomes difficult to detect positive selection and the degree of positive selection is underestimated. Weakly deleterious mutations have a larger chance of contributing to polymorphism than strongly deleterious mutations, but still have low probabilities of fixation. This creates bias in the McDonald–Kreitman test's estimate of the degree of adaptive evolution, resulting in a dramatically lower estimate of α. By contrast, since strongly deleterious mutations contribute to neither polymorphism nor divergence, strongly deleterious mutations do not bias estimates of α.<a class="footnoteRef" href="#fn16" id="fnref16"><sup>16</sup></a> The presence of slightly deleterious mutations is strongly linked to genes that have experienced the greatest reduction in effective population size.<a class="footnoteRef" href="#fn17" id="fnref17"><sup>17</sup></a> This means that soon after a recent reduction in effective population size in a species has occurred, such as a bottleneck, we observe a larger presence of slightly deleterious mutations in the protein-coding regions.<a class="footnoteRef" href="#fn18" id="fnref18"><sup>18</sup></a> We can make a direct connection with the increase in number of slightly deleterious mutations and the recent decrease in effective population size.<a class="footnoteRef" href="#fn19" id="fnref19"><sup>19</sup></a> For more information on why population size affects the tendency of slightly deleterious mutations to increase in frequency, refer to the article <a href="Nearly_neutral_theory_of_molecular_evolution" title="wikilink">Nearly neutral theory of molecular evolution</a>.</p>

<p>Additionally, as with every statistical test, there is always the chance of having <a href="type_I_error" title="wikilink">type I error</a> and <a href="type_II_error" title="wikilink">type II error</a> in the McDonald Kreitman test. With statistical tests, we must strive more try to avoid making type I errors, to avoid rejecting the null hypothesis, when it is in fact, true.<a class="footnoteRef" href="#fn20" id="fnref20"><sup>20</sup></a> However, the McDonald Kreitman test is very vulnerable to type I error, because of the many factors that can lead to the accidental rejection of the true null hypothesis. Such factors include variation in recombination rate, nonequilibrium demography, small sample sizes, and in comparisons involving more recently diverged species.<a class="footnoteRef" href="#fn21" id="fnref21"><sup>21</sup></a> All of these factors have the ability to influence the ability of the McDonald-Kreitman test to detect positive selection, as well as the level of positive selection acting on a species. This inability to correctly determine the level of positive selection acting on a species often leads to a false positive, and the incorrect rejection of the null hypothesis.</p>

<p>While performing the McDonald–Kreitman test, scientists also have to avoid making too numerous type II errors. Otherwise, a test's results may be too flawed and its results will be termed useless.</p>
<h2 id="error-correcting-mechanisms-of-the-mcdonaldkreitman-test">Error-correcting Mechanisms of the McDonald–Kreitman test</h2>

<p>There continues to be more experimentation with the McDonald–Kreitman test and how to improve the accuracy of the test. The most important error to correct for is the error that α is severely underestimated in the presence of slightly deleterious mutations, as discussed in the previous section "Sources of Error with the McDonald-Kreitman Test." This possible adjustment of the McDonald–Kreitman test includes removing polymorphisms below a specific value from the data set to improve and increase the number of substitutions that occurred due to adaptive evolution.<a class="footnoteRef" href="#fn22" id="fnref22"><sup>22</sup></a> To minimize the impact of slightly deleterious mutations, it has been proposed to exclude polymorphisms that are below a certain cutoff frequency, such as </p>

<p>One adjustment necessary is to control for the <a href="type_I_error" title="wikilink">type I error</a> in the McDonald–Kreitman test, refer to the discussion of this in previous section "Sources of Error with the McDonald Kreitman Test." One method to avoid type I errors is to avoid using populations that have undergone a recent bottleneck, meaning they have recently undergone a recent decrease in effective population size.<a class="footnoteRef" href="#fn23" id="fnref23"><sup>23</sup></a> To make the analysis as accurate as possible in the McDonald–Kreitman test, it is best to use large sample sizes, but there is still debate and how large "large" is.<a class="footnoteRef" href="#fn24" id="fnref24"><sup>24</sup></a> Another method of controlling for type I error, Peter Andolfatto(2008) suggests, is to establish significance levels by coalescent simulation with recombination in genomewide scans for selection on noncoding DNA. By doing this, you will be able to improve the accuracy of your statistical test and avoid any false positive tests.  With all these possible ways to avoid making type I errors, scientists should cautiously choose which populations they are analyzing, to avoid analyzing populations that will lead to inaccurate results.</p>
<h2 id="see-also">See also</h2>
<ul>
<li><a href="Adaptive_Evolution" title="wikilink">Adaptive Evolution</a></li>
<li><a href="Directional_selection" title="wikilink">Directional selection</a></li>
<li><a href="Negative_selection_(natural_selection)" title="wikilink">Negative selection (natural selection)</a></li>
<li><a href="Ka/Ks_ratio" title="wikilink">K<sub>a</sub>/K<sub>s</sub> ratio</a></li>
</ul>
<h2 id="references">References</h2>

<p>"</p>

<p><a class="uri" href="Category:Phylogenetics" title="wikilink">Category:Phylogenetics</a> <a href="Category:Molecular_evolution" title="wikilink">Category:Molecular evolution</a> <a href="Category:Statistical_genetics" title="wikilink">Category:Statistical genetics</a></p>
<section class="footnotes">
<hr/>
<ol>
<li id="fn1">Futuyma, D. J. 2013. Evolution. Sinauer Associates, Inc.: Sunderland.<a href="#fnref1">↩</a></li>
<li id="fn2"><a href="http://mbe.oxfordjournals.org/content/25/6/1007.short">1</a>, Charlesworth, J. Eyre-Walker, A. 2008. The McDonald–Kreitman Test and Slightly Deleterious Mutations. Molecular Biology and Evolution 25 (6): 1007-1015.<a href="#fnref2">↩</a></li>
<li id="fn3"><a href="http://www.sciencemag.org/content/315/5811/525">2</a>, Kimchi-Sarfaty, C. Oh, J. M. Kim, I. Sauna, Z. E. Calcagno, A. M. Ambudkar, S. V. Gottesman, M. M. 2007. A "Silent" Polymorphism in the MDR1 Gene Changes Substrate Specificity. Science 315 (5811): 525-528.<a href="#fnref3">↩</a></li>
<li id="fn4"></li>
<li id="fn5"><a href="http://www.zoology.wisc.edu/courses/611/Part2/Readings/3eyre-walker_tree2006.pdf">3</a>, Eyre-Walker, A. 2006. The genomic rate of adaptive evolution. Trends in Ecology and Evolution 21 (10): 569-575.<a href="#fnref5">↩</a></li>
<li id="fn6"><a href="http://nar.oxfordjournals.org/content/36/suppl_2/W157.full.pdf+html?sid=6001f1bd-4744-4bce-8837-293e14e69a3c">4</a>, Barbadilla, A. Casillas, S. Egea, R. 2008. Standard and generalized McDonald-Kreitman test: a website to detect selection by comparing different classes of DNA sites. Nucleic Acids Research 36: 157-162.<a href="#fnref6">↩</a></li>
<li id="fn7"><a href="#fnref7">↩</a></li>
<li id="fn8"><a href="http://www.genetics.org/content/162/4/2017.full.pdf">5</a>, Eyre-Walker, A. 2002. Changing Effective Population Size and the McDonald-Kreitman Test. Genetics Society of America 162: 2017-2024.<a href="#fnref8">↩</a></li>
<li id="fn9"></li>
<li id="fn10"><a href="http://www.sciencedirect.com/science/article/pii/S0168952507001102#">6</a>, Meiklejohns, C. D. Montooth, K. L. Rand, D. M. 2007. Positive and negative selection on the mitochondrial genome. Trends in Genetics 23 (6): 259-263.<a href="#fnref10">↩</a></li>
<li id="fn11"><a href="http://mbe.oxfordjournals.org/content/26/3/691">8</a>, Baines, J. F. Parsch, J. Zhang, Z. 2008. The Influence of Demography and Weak Selection on the McDonald–Kreitman Test: An Empirical Study in Drosophila. Molecular Biology and Evolution 26: 691-698.<a href="#fnref11">↩</a></li>
<li id="fn12"></li>
<li id="fn13"><a href="http://www.genetics.org/content/180/3/1767">9</a>, Andolfatto, P. 2008. Controlling Type-I Error of the McDonald-Kreitman Test in Genomewide Scans for Selection on Noncoding DNA. Genetics Society of America 180 (3): 1767-1771.<a href="#fnref13">↩</a></li>
<li id="fn14"></li>
<li id="fn15"></li>
<li id="fn16"><a href="http://0-www.pnas.org.books.redlands.edu/content/110/21/8615">10</a>, Messer, P. W. Petrov, D. A. 2013. Frequent adaptation and the McDonald–Kreitman test. Proceedings of the National Academy of Sciences of the United States of America 110 (21): 8615-8620.<a href="#fnref16">↩</a></li>
<li id="fn17"><a href="http://mbe.oxfordjournals.org/content/26/3/691.full.pdf+html">11</a>, Parsch, J. Zhang, Z. Baines, J. 2009. The Influence of Demography and Weak Selection on the McDonald–Kreitman Test: An Empirical Study in Drosophila. Molecular Biology and Evolution 26 (3): 691-698.<a href="#fnref17">↩</a></li>
<li id="fn18">[<a class="uri" href="http://onlinelibrary.wiley.com/store/10.1111/j.1558-5646.2008.00560.x/asset/j.1558-5646.2008.00560.x.pdf?v=1&amp;t">http://onlinelibrary.wiley.com/store/10.1111/j.1558-5646.2008.00560.x/asset/j.1558-5646.2008.00560.x.pdf?v=1&amp;t;</a>;=hofet08x&amp;s;=7aedab873cf04e4d96cf75954e43c2cd7730e158], Ellegren, Hans. 2009. A Selection Model of Molecular Evolution Incorporating the Effective Population Size. Evolution 63(2): 301-305.<a href="#fnref18">↩</a></li>
<li id="fn19"></li>
<li id="fn20">Rossman, A. J. Chance, B. L. 2012. Workshop Statistics Discovery with Data. John Wiley &amp; Sons: Danvers.<a href="#fnref20">↩</a></li>
<li id="fn21"></li>
<li id="fn22"></li>
<li id="fn23"></li>
<li id="fn24"></li>
</ol>
</section>
</body>
</html>
