<html lang="en">
<head>
<meta charset="utf-8"/>
<title offset="1000">Metabolic engineering</title>
<script src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_SVG.js" type="text/javascript">
</script>
</head>
<body>
<h1>Metabolic engineering</h1>
<hr/>

<p> <strong>Metabolic engineering</strong> is the practice of optimizing <a href="genetics" title="wikilink">genetic</a> and regulatory processes within <a href="Cell_(biology)" title="wikilink">cells</a> to increase the cells' production of a certain substance. These processes are chemical networks that use a series of biochemical reactions and <a class="uri" href="enzymes" title="wikilink">enzymes</a> that allow cells to convert raw materials into molecules necessary for the cell’s survival. Metabolic engineering specifically seeks to mathematically model these networks, calculate a yield of useful products, and pin point parts of the network that constrain the production of these products.<a class="footnoteRef" href="#fn1" id="fnref1"><sup>1</sup></a> <a href="Genetic_engineering" title="wikilink">Genetic engineering</a> techniques can then be used to modify the network in order to relieve these constraints. Once again this modified network can be modeled to calculate the new product yield.</p>

<p>The ultimate goal of metabolic engineering is to be able to use these organisms to produce valuable substances on an industrial scale in a cost effective manner. Current examples include producing <a class="uri" href="beer" title="wikilink">beer</a>, <a class="uri" href="wine" title="wikilink">wine</a>, <a class="uri" href="cheese" title="wikilink">cheese</a>, <a class="uri" href="pharmaceuticals" title="wikilink">pharmaceuticals</a>, and other <a class="uri" href="biotechnology" title="wikilink">biotechnology</a> products.</p>

<p>Since cells use these metabolic networks for their survival, changes can have drastic effects on the cells' viability. Therefore, trade-offs in metabolic engineering arise between the cells ability to produce the desired substance and its natural survival needs. Therefore, instead of directly deleting and/or overexpressing the genes that encode for metabolic enzymes, the current focus is to target the regulatory networks in a cell to efficiently engineer the metabolism.<a class="footnoteRef" href="#fn2" id="fnref2"><sup>2</sup></a></p>
<h2 id="history-and-applications-of-metabolic-engineering">History and applications of metabolic engineering</h2>

<p>In the past, to increase the productivity of a desired <a class="uri" href="metabolite" title="wikilink">metabolite</a>, a <a class="uri" href="microorganism" title="wikilink">microorganism</a> was genetically modified by chemically induced <a class="uri" href="mutation" title="wikilink">mutation</a>, and the mutant <a href="strain_(biology)" title="wikilink">strain</a> that overexpressed the desired metabolite was then chosen.<a class="footnoteRef" href="#fn3" id="fnref3"><sup>3</sup></a> However, one of the main problems with this technique was that the metabolic pathway for the production of that metabolite was not analyzed, and as a result, the constraints to production and relevant pathway enzymes to be modified were unknown.<a class="footnoteRef" href="#fn4" id="fnref4"><sup>4</sup></a> In 1990s, a new technique called metabolic engineering emerged. This technique analyzes the metabolic pathway of a <a class="uri" href="microorganism" title="wikilink">microorganism</a>, and determines the constraints and their effects on the production of desired compounds. It then uses genetic engineering to relieve these constraints. Some examples of successful metabolic engineering are the following: (i) Identification of constraints to lysine production in <a class="uri" href="corynebacterium" title="wikilink">corynebacterium</a> <em>glutamicum</em> and insertion of new genes to relieve these constraints to improve production<a class="footnoteRef" href="#fn5" id="fnref5"><sup>5</sup></a> (ii) Engineering of a new <a href="fatty_acid_biosynthesis" title="wikilink">fatty acid biosynthesis</a> pathway, called reversed <a href="beta_oxidation" title="wikilink">beta oxidation</a> pathway, that is more efficient than the native pathway in producing fatty acids and alcohols which can potentially be catalytically converted to chemicals and fuels<a class="footnoteRef" href="#fn6" id="fnref6"><sup>6</sup></a> (iii) Improved production of <a href="3-deoxy-D-arabino-heptulosonate_7-phosphate" title="wikilink">DAHP</a> an aromatic metabolite produced by <em>E.coli</em> that is an intermediate in the production of aromatic amino acids.<a class="footnoteRef" href="#fn7" id="fnref7"><sup>7</sup></a> It was determined through metabolic flux analysis that the theoretical maximal yield of DAHP per glucose molecule utilized, was 3/7. This is because some of the carbon from glucose is lost as carbon dioxide, instead of being utilized to produce DAHP. Also, one of the metabolites (PEP, or <a class="uri" href="phosphoenolpyruvate" title="wikilink">phosphoenolpyruvate</a>) that are used to produce DAHP, was being converted to <a class="uri" href="pyruvate" title="wikilink">pyruvate</a> (PYR) to transport glucose into the cell, and therefore, was no longer available to produce DAHP. In order to relieve the shortage of PEP and increase yield, Patnaik et al. used genetic engineering on <em>E.coli</em> to introduce a reaction that converts PYR back to PEP. Thus, the PEP used to transport glucose into the cell is regenerated, and can be used to make DAHP. This resulted in a new theoretical maximal yield of 6/7 - double that of the native <em>E.coli</em> system.</p>

<p>At the industrial scale, metabolic engineering is becoming more convenient and cost effective. According to the <a href="Biotechnology_Industry_Organization" title="wikilink">Biotechnology Industry Organization</a>, " more than 50 <a class="uri" href="biorefinery" title="wikilink">biorefinery</a> facilities are being built across North America to apply metabolic engineering to produce biofuels and chemicals from renewable <a class="uri" href="biomass" title="wikilink">biomass</a> which can help reduce greenhouse gas emissions ". Potential biofuels include short-chain <a class="uri" href="alcohols" title="wikilink">alcohols</a> and alkanes (to replace <a class="uri" href="gasoline" title="wikilink">gasoline</a>), <a href="fatty_acid_methyl_esters" title="wikilink">fatty acid methyl esters</a> and <a href="fatty_alcohols" title="wikilink">fatty alcohols</a> (to replace <a href="diesel_fuel" title="wikilink">diesel</a>), and <a href="fatty_acid" title="wikilink">fatty acid</a>-and <a class="uri" href="isoprenoid" title="wikilink">isoprenoid</a>-based biofuels (to replace <a href="diesel_fuel" title="wikilink">diesel</a>).<a class="footnoteRef" href="#fn8" id="fnref8"><sup>8</sup></a></p>
<h2 id="metabolic-flux-analysis">Metabolic flux analysis</h2>

<p>An analysis of metabolic flux can be found at <em><a href="Flux_balance_analysis" title="wikilink">Flux balance analysis</a></em></p>
<h3 id="setting-up-a-metabolic-pathway-for-analysis">Setting up a metabolic pathway for analysis</h3>

<p>The first step in the process is to identify a desired goal to achieve through the improvement or modification of an organism's metabolism. Reference books and online databases are used to research reactions and metabolic pathways that are able to produce this product or result. These databases contain copious genomic and chemical information including pathways for metabolism and other cellular processes. Using this research, an organism is chosen that will be used to create the desired product or result. Considerations that are taken into account when making this decision are how close the organism's metabolic pathway is to the desired pathway, the maintenance costs associated with the organism, and how easy it is to modify the pathway of the organism. Escherichia coli (E. coli) is widely used in metabolic engineering to synthesize a wide variety of products such as amino acids because it is relatively easy to maintain and modify.<a class="footnoteRef" href="#fn9" id="fnref9"><sup>9</sup></a> If the organism does not contain the complete pathway for the desired product or result, then genes that produce the missing enzymes must be incorporated into the organism.</p>
<h3 id="analyzing-a-metabolic-pathway">Analyzing a metabolic pathway</h3>

<p>The completed metabolic pathway is modeled mathematically to find the theoretical yield of the product or the reaction fluxes in the cell. A flux is the rate at which a given reaction in the network occurs. Simple metabolic pathway analysis can be done by hand, but most require the use of software to perform the computations.<a class="footnoteRef" href="#fn10" id="fnref10"><sup>10</sup></a> These programs use complex linear algebra algorithms to solve these models. To solve a network using the equation for determined systems shown below, one must input the necessary information about the relevant reactions and their fluxes. Information about the reaction (such as the reactants and stoichiometry) are contained in the matrices G<sub>x</sub> and G<sub>m</sub>. Matrices V<sub>m</sub> and V<sub>x</sub> contain the fluxes of the relevant reactions. When solved, the equation yields the values of all the unknown fluxes (contained in V<sub>x</sub>).</p>

<p>

<math display="block" id="Metabolic_engineering:0">
 <semantics>
  <mrow>
   <msub>
    <mi>V</mi>
    <mi>x</mi>
   </msub>
   <mo>=</mo>
   <mrow>
    <mo>-</mo>
    <mrow>
     <msup>
      <mrow>
       <mo stretchy="false">(</mo>
       <msub>
        <mi>G</mi>
        <mi>x</mi>
       </msub>
       <mo stretchy="false">)</mo>
      </mrow>
      <mrow>
       <mo>-</mo>
       <mn>1</mn>
      </mrow>
     </msup>
     <mo>*</mo>
     <mrow>
      <mo stretchy="false">(</mo>
      <mrow>
       <msub>
        <mi>G</mi>
        <mi>m</mi>
       </msub>
       <mo>*</mo>
       <msub>
        <mi>V</mi>
        <mi>m</mi>
       </msub>
      </mrow>
      <mo stretchy="false">)</mo>
     </mrow>
    </mrow>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <eq></eq>
    <apply>
     <csymbol cd="ambiguous">subscript</csymbol>
     <ci>V</ci>
     <ci>x</ci>
    </apply>
    <apply>
     <minus></minus>
     <apply>
      <times></times>
      <apply>
       <csymbol cd="ambiguous">superscript</csymbol>
       <apply>
        <csymbol cd="ambiguous">subscript</csymbol>
        <ci>G</ci>
        <ci>x</ci>
       </apply>
       <apply>
        <minus></minus>
        <cn type="integer">1</cn>
       </apply>
      </apply>
      <apply>
       <times></times>
       <apply>
        <csymbol cd="ambiguous">subscript</csymbol>
        <ci>G</ci>
        <ci>m</ci>
       </apply>
       <apply>
        <csymbol cd="ambiguous">subscript</csymbol>
        <ci>V</ci>
        <ci>m</ci>
       </apply>
      </apply>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   V_{x}=-(G_{x})^{-1}*(G_{m}*V_{m})
  </annotation>
 </semantics>
</math>

</p>
<h3 id="determining-the-optimal-genetic-manipulations">Determining the optimal genetic manipulations</h3>

<p>After solving for the fluxes of reactions in the network, it is necessary to determine which reactions may be altered in order to maximize the yield of the desired product. To determine what specific genetic manipulations to perform, it is necessary to use computational algorithms, such as OptGene or OptFlux.<a class="footnoteRef" href="#fn11" id="fnref11"><sup>11</sup></a> They provide recommendations for which genes should be overexpressed, knocked out, or introduced in a cell to allow increased production of the desired product. For example, if a given reaction has particularly low flux and is limiting the amount of product, the software may recommend that the enzyme catalyzing this reaction should be overexpressed in the cell to increase the reaction flux. The necessary genetic manipulations can be performed using standard molecular biology techniques. Genes may be overexpressed or knocked out from an organism, depending on their effect on the pathway and the ultimate goal.<a class="footnoteRef" href="#fn12" id="fnref12"><sup>12</sup></a></p>
<h3 id="experimental-measurements">Experimental measurements</h3>

<p>In order to create a solvable model, it is often necessary to have certain fluxes already known or experimentally measured. In addition, in order to verify the effect of genetic manipulations on the metabolic network (to ensure they align with the model), it is necessary to experimentally measure the fluxes in the network. To measure reaction fluxes, carbon flux measurements are made using <a href="isotopic_labeling" title="wikilink">carbon-13 isotopic labeling</a>.<a class="footnoteRef" href="#fn13" id="fnref13"><sup>13</sup></a> The organism is fed a mixture that contains molecules where specific carbons are engineered to be carbon-13 atoms, instead of carbon-12. After these molecules are used in the network, downstream metabolites also become labeled with carbon-13, as they incorporate those atoms in their structures. The specific labeling pattern of the various metabolites is determined by the reaction fluxes in the network. Labeling patterns may be measured using techniques such as <a href="Gas_chromatography-mass_spectrometry" title="wikilink">Gas chromatography-mass spectrometry</a> (GC-MS) along with computational algorithms to determine reaction fluxes.</p>
<h2 id="see-also">See also</h2>
<ul>
<li><a class="uri" href="Bioreactor" title="wikilink">Bioreactor</a></li>
<li><a href="Bacterial_transformation" title="wikilink">Bacterial transformation</a></li>
<li><a href="Genetic_engineering" title="wikilink">Genetic engineering</a></li>
</ul>
<h2 id="references">References</h2>
<references>
</references>
<h2 id="external-links">External links</h2>

<p>Biotechnology Industry Organization(BIO) website:</p>
<ul>
<li><a href="http://www.bio.org/">BIO Website</a></li>
</ul>

<p>"</p>

<p><a href="Category:Biological_engineering" title="wikilink">Category:Biological engineering</a></p>
<section class="footnotes">
<hr/>
<ol>
<li id="fn1">Yang, Y.T., Bennet, G. N., San, K.Y., (1998) <em>Genetic and Metabolic Engineering</em>, Electronic Journal of Biotechnology, ISSN 07117-3458<a href="#fnref1">↩</a></li>
<li id="fn2">Vemuri, G.M, Aristidou, A.A, (2005) <em>Metabolic Engineering in the -omics Era: Elucidating and Modulating Regulatory Networks</em>, Microbial Mol Biology Review vol. 69: 197-216<a href="#fnref2">↩</a></li>
<li id="fn3">Voit,Eberhard.,Torres,Nestor V.(2002)." Pathways Analysis and Optimization in Metabolic Engineering." Cambridge:University Press,p.ix-x<a href="#fnref3">↩</a></li>
<li id="fn4"></li>
<li id="fn5">Stephanopoulos, G. N., Aristidou, A. A., Nielsen, J. (1998). " Metabolic Engineering: Principles and Methodologies ". San Diego: Academic Press<a href="#fnref5">↩</a></li>
<li id="fn6">Dellomonaco, Clementina.(2011). '' Engineered Reversal of the beta oxidation cycle for the Synthesis of Fuels and Chemicals.'' Nature 476,355-359<a href="#fnref6">↩</a></li>
<li id="fn7">Patnaik, R. and Liao, J. (1994). "Engineering of Escherichia coli central metabolism for aromatic metabolite production with near theoretical yield". <em>Appl. Environ. Microbiol.</em> 60(11):3903-3908<a href="#fnref7">↩</a></li>
<li id="fn8">Keasling D.,Jay(2010).'' Advanced Biofuel production in microbes.'' Biotechnol.J.,5,147-162<a href="#fnref8">↩</a></li>
<li id="fn9">University of California - Los Angeles (2008, December 18). "Genetic Modification Turns E. Coli Bacteria Into High Density Biofuel". <em>ScienceDaily.</em> Retrieved December 7, 2011, from <a class="uri" href="http://www.sciencedaily.com/releases/2008/12/081218151652.htm">http://www.sciencedaily.com/releases/2008/12/081218151652.htm</a><a href="#fnref9">↩</a></li>
<li id="fn10">Schellenberger, J., Que, R., Fleming, R., et al. (2011). "Quantitative prediction of cellular metabolism with constraint-based models: the COBRA Toolbox v2.0". <em>Nature Protocols.</em> 6(9):1290-1307<a href="#fnref10">↩</a></li>
<li id="fn11">Rocha, I., Maia, P., Evangelista, P., et al. (2010). "OptFlux: an open-source software platform for in silico metabolic engineering". BMC Sys Biol. 45(4)<a href="#fnref11">↩</a></li>
<li id="fn12">Work, T.S., Hinton, R., Work, E., Dobrota, M., Chard, T. (1980). "Laboratory Techniques in Biochemistry and Molecular Biology". v.8<a href="#fnref12">↩</a></li>
<li id="fn13">Wiechert, W. and de Graaf, A.A. (2000). "Bidirectional Reaction Steps in Metabolic Networks: Modeling and Simulation of Carbon Isotope Labeling Experiments". <em>Biotechnol. Bioeng.</em> 55(1):101-117<a href="#fnref13">↩</a></li>
</ol>
</section>
</body>
</html>
