<html lang="en">
<head>
<meta charset="utf-8"/>
<title offset="1581">METAP2</title>
<script src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_SVG.js" type="text/javascript">
</script>
</head>
<body>
<h1>METAP2</h1>
<hr/>

<p><strong>Methionine aminopeptidase 2</strong> is an <a class="uri" href="enzyme" title="wikilink">enzyme</a> that in humans is encoded by the <em>METAP2</em> <a class="uri" href="gene" title="wikilink">gene</a>.<a class="footnoteRef" href="#fn1" id="fnref1"><sup>1</sup></a><a class="footnoteRef" href="#fn2" id="fnref2"><sup>2</sup></a></p>

<p>Methionine aminopeptidase 2, a member of the dimetallohydrolase family, is a <a class="uri" href="cytosolic" title="wikilink">cytosolic</a> metalloenzyme that catalyzes the hydrolytic removal of N-terminal <a class="uri" href="methionine" title="wikilink">methionine</a> residues from nascent <a class="uri" href="proteins" title="wikilink">proteins</a>.<a class="footnoteRef" href="#fn3" id="fnref3"><sup>3</sup></a><a class="footnoteRef" href="#fn4" id="fnref4"><sup>4</sup></a><a class="footnoteRef" href="#fn5" id="fnref5"><sup>5</sup></a></p>
<ul>
<li>peptide-methionine 

<math display="inline" id="METAP2:0">
 <semantics>
  <mo>⇌</mo>
  <annotation-xml encoding="MathML-Content">
   <ci>normal-⇌</ci>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \rightleftharpoons
  </annotation>
 </semantics>
</math>

 peptide + methionine</li>
</ul>

<p>MetAP2 is found in all organisms and is especially important because of its critical role in tissue repair and protein degradation.<a class="footnoteRef" href="#fn6" id="fnref6"><sup>6</sup></a> Furthermore, MetAP2 is of particular interest because the enzyme plays a key role in <a class="uri" href="angiogenesis" title="wikilink">angiogenesis</a>, the growth of new blood vessels, which is necessary for the progression of diseases including solid tumor <a class="uri" href="cancers" title="wikilink">cancers</a> and <a href="rheumatoid_arthritis" title="wikilink">rheumatoid arthritis</a>.<a class="footnoteRef" href="#fn7" id="fnref7"><sup>7</sup></a> MetAP2 is also the target of two groups of anti-angiogenic natural products, <a class="uri" href="ovalicin" title="wikilink">ovalicin</a> and <a class="uri" href="fumagillin" title="wikilink">fumagillin</a>, and their analogs.<a class="footnoteRef" href="#fn8" id="fnref8"><sup>8</sup></a><a class="footnoteRef" href="#fn9" id="fnref9"><sup>9</sup></a><a class="footnoteRef" href="#fn10" id="fnref10"><sup>10</sup></a><a class="footnoteRef" href="#fn11" id="fnref11"><sup>11</sup></a></p>
<h2 id="structure">Structure</h2>

<p>In living organisms, the start <a class="uri" href="codon" title="wikilink">codon</a> that initiates protein <a href="Biosynthesis" title="wikilink">synthesis</a> codes for either methionine (<a class="uri" href="eukaryotes" title="wikilink">eukaryotes</a>) or <a class="uri" href="formylmethionine" title="wikilink">formylmethionine</a> (<a class="uri" href="prokaryotes" title="wikilink">prokaryotes</a>). In <a href="E._coli" title="wikilink">E. coli</a> (prokaryote), an <a class="uri" href="enzyme" title="wikilink">enzyme</a> called <a href="formylmethionine_deformylase" title="wikilink">formylmethionine deformylase</a> can cleave the formyl group, leaving just the N-terminal methionine residue. For proteins with small, uncharged penultimate N-terminal residues, a methionine aminopeptidase can cleave the methionine residue.<a class="footnoteRef" href="#fn12" id="fnref12"><sup>12</sup></a> The number of <a class="uri" href="genes" title="wikilink">genes</a> encoding for a methionine aminopeptidase varies between organisms. In E. coli, there is only one known MetAP, a 29,333 Da <a class="uri" href="monomeric" title="wikilink">monomeric</a> enzyme coded for by a gene consisting of 264 <a class="uri" href="codons" title="wikilink">codons</a>.<a class="footnoteRef" href="#fn13" id="fnref13"><sup>13</sup></a> The knockout of this gene in E. coli leads to cell inviability.<a class="footnoteRef" href="#fn14" id="fnref14"><sup>14</sup></a> In humans, there are two genes encoding MetAP, <a class="uri" href="MetAP1" title="wikilink">MetAP1</a> and MetAP2. MetAP1 codes for a 42 kDa enzyme, while MetAP2 codes for a 67 kDa enzyme. <a class="uri" href="Yeast" title="wikilink">Yeast</a> MetAP1 is 40 percent homologous to E. coli MetAP; within <a href="S._cerevisiae" title="wikilink">S. cerevisiae</a>, MetAP2 is 22 percent <a href="Homology_(biology)" title="wikilink">homologous</a> with the sequence of MetAP1; MetAP2 is highly conserved between <em>S. cerevisiae</em> and humans.<a class="footnoteRef" href="#fn15" id="fnref15"><sup>15</sup></a> In contrast to prokaryotes, eukaryotic S. cerevisiae strains lacking the gene for either MetAP1 or MetAP2 are viable, but exhibit a slower growth rate than a control strain expressing both genes.</p>

<p><a href="Image:MetAP2_active_site.ogv" title="wikilink">thumb|upright=2.4|left|Figure 1. Active site structure of MetAP2. Generated using PDB:1BOA in PyMol. Click to view rotatable structure</a></p>
<h3 id="active-site">Active site</h3>

<p>The active site of MetAP2 has a structural motif characteristic of many metalloenzymes—including the dioxygen carrier protein, <a class="uri" href="hemerythrin" title="wikilink">hemerythrin</a>; the dinuclear non-heme iron protein, <a href="ribonucleotide_reductase" title="wikilink">ribonucleotide reductase</a>; <a href="leucine_aminopeptidase" title="wikilink">leucine aminopeptidase</a>; <a class="uri" href="urease" title="wikilink">urease</a>; <a class="uri" href="arginase" title="wikilink">arginase</a>; several <a class="uri" href="phosphatases" title="wikilink">phosphatases</a> and <a class="uri" href="phosphoesterases" title="wikilink">phosphoesterases</a>—that includes two bridging <a class="uri" href="carboxylate" title="wikilink">carboxylate</a> <a class="uri" href="ligands" title="wikilink">ligands</a> and a bridging <a class="uri" href="water" title="wikilink">water</a> or <a class="uri" href="hydroxide" title="wikilink">hydroxide</a> <a class="uri" href="ligand" title="wikilink">ligand</a>.<a class="footnoteRef" href="#fn16" id="fnref16"><sup>16</sup></a><a class="footnoteRef" href="#fn17" id="fnref17"><sup>17</sup></a><a class="footnoteRef" href="#fn18" id="fnref18"><sup>18</sup></a><a class="footnoteRef" href="#fn19" id="fnref19"><sup>19</sup></a><a class="footnoteRef" href="#fn20" id="fnref20"><sup>20</sup></a><a class="footnoteRef" href="#fn21" id="fnref21"><sup>21</sup></a><a class="footnoteRef" href="#fn22" id="fnref22"><sup>22</sup></a> Specifically in human MetAP2 (PDB: 1BOA), one of the catalytic metal ions is bound to His331, Glu364, Glu459, Asp263, and a bridging water or hydroxide, while the other metal ion is bound to Asp251 (<a class="uri" href="bidentate" title="wikilink">bidentate</a>), App262 (bidentate), Glu459, and the same bridging water or hydroxide. Here, the two bridging carboxylates are Asp262 and Glu459.</p>
<h3 id="dimetal-center">Dimetal center</h3>

<p>The identity of the <a href="active_site" title="wikilink">active site</a> metal <a class="uri" href="ions" title="wikilink">ions</a> under physiological conditions has not been successfully established, and remains a controversial issue. MetAP2 shows activity in the presence of Zn(II), Co(II), Mn(II), and Fe(II) ions, and various authors have argued any given metal ion is the physiological one: some in the presence of <a class="uri" href="iron" title="wikilink">iron</a>,<a class="footnoteRef" href="#fn23" id="fnref23"><sup>23</sup></a> others in <a class="uri" href="cobalt" title="wikilink">cobalt</a>,<a class="footnoteRef" href="#fn24" id="fnref24"><sup>24</sup></a><a class="footnoteRef" href="#fn25" id="fnref25"><sup>25</sup></a> others in <a class="uri" href="manganese" title="wikilink">manganese</a>,<a class="footnoteRef" href="#fn26" id="fnref26"><sup>26</sup></a> and yet others in the presence of <a class="uri" href="zinc" title="wikilink">zinc</a>.<a class="footnoteRef" href="#fn27" id="fnref27"><sup>27</sup></a> Nonetheless, the majority of crystallographers have <a class="uri" href="crystallized" title="wikilink">crystallized</a> MetAP2 either in the presence of Zn(II) or Co(II) (see PDB database).</p>
<h2 id="mechanism">Mechanism</h2>

<p><a href="File:Mechanism_of_MetAP2.jpg" title="wikilink">thumb|upright=3|right|Figure 2. Two proposed reaction mechanisms for MetAP in E. coli. (A) Tetrahedral intermediate stabilized by Glu204 and metal center. (B) Tetrahedral intermediate stabilized His178 and metal center.</a><a class="footnoteRef" href="#fn28" id="fnref28"><sup>28</sup></a></p>

<p>The bridging water or hydroxide ligand acts as a nucleophile during the hydrolysis reaction, but the exact mechanism of catalysis is not yet known.<a class="footnoteRef" href="#fn29" id="fnref29"><sup>29</sup></a><a class="footnoteRef" href="#fn30" id="fnref30"><sup>30</sup></a><a class="footnoteRef" href="#fn31" id="fnref31"><sup>31</sup></a> The catalytic mechanisms of hydrolase enzymes depend greatly on the identity of the bridging ligand,<a class="footnoteRef" href="#fn32" id="fnref32"><sup>32</sup></a> which can be challenging to determine due to the difficulty of studying hydrogen atoms via <a href="x-ray_crystallography" title="wikilink">x-ray crystallography</a>.</p>

<p>The histidine residues shown in the mechanism to the right, H178 and H79, are conserved in all MetAPs (MetAP1s and MetAP2s) sequenced to date, suggesting their presence is important to catalytic activity.<a class="footnoteRef" href="#fn33" id="fnref33"><sup>33</sup></a> Based upon X-ray crystallographic data, histidine 79 (H79) has been proposed to help position the methionine residue in the active site and transfer a proton to the newly exposed N-terminal amine.<a class="footnoteRef" href="#fn34" id="fnref34"><sup>34</sup></a> Lowther and Colleagues have proposed two possible mechanisms for MetAP2 in E. coli, shown at the right.<a class="footnoteRef" href="#fn35" id="fnref35"><sup>35</sup></a></p>
<h2 id="function">Function</h2>

<p>While previous studies have indicated MetAP2 catalyzes the removal of N-terminal methionine residues in vitro, the function of this enzyme in vivo may be more complex. For example, a significant correlation exists between the inhibition of the enzymatic activity of MetAP2 and inhibition of cell growth, thus implicating the enzyme in <a href="endothelial_cell" title="wikilink">endothelial cell</a> <a href="Cell_proliferation" title="wikilink">proliferation</a>.<a class="footnoteRef" href="#fn36" id="fnref36"><sup>36</sup></a> For this reason, scientists have singled out MetAP2 as a potential target for the inhibition of angiogenesis. Moreover, studies have demonstrated that MetAP2 copurifies and interacts with the α subunit of eukaryotic initiation factor 2 (<a class="uri" href="eIF2" title="wikilink">eIF2</a>), a protein that is necessary for protein synthesis in vivo.<a class="footnoteRef" href="#fn37" id="fnref37"><sup>37</sup></a> Specifically, MetAP2 protects eIF-2α from inhibitory <a class="uri" href="phosphorylation" title="wikilink">phosphorylation</a> from the enzyme eIF-2α <a class="uri" href="kinase" title="wikilink">kinase</a>, inhibits RNA-dependent protein kinase (PKR)-catalyzed eIF-2 R-subunit phosphorylation, and also reverses PKR-mediated inhibition of protein synthesis in intact cells.</p>
<h2 id="clinical-significance">Clinical significance</h2>

<p><a href="File:MetAP2_bound_to_fumagillin.png" title="wikilink">thumb|upright=2.2|left|Figure 3. Fumagillin (green and red) bound to human MetAP2 active site (multicolored, with cyan, purple, and pink corresponding to helices, sheets, and loops, respectively), with dimetal ions (blue) shown.</a></p>

<p>Numerous studies implicate MetAP2 in angiogenesis.<a class="footnoteRef" href="#fn38" id="fnref38"><sup>38</sup></a><a class="footnoteRef" href="#fn39" id="fnref39"><sup>39</sup></a><a class="footnoteRef" href="#fn40" id="fnref40"><sup>40</sup></a><a class="footnoteRef" href="#fn41" id="fnref41"><sup>41</sup></a><a class="footnoteRef" href="#fn42" id="fnref42"><sup>42</sup></a> Specifically, the <a class="uri" href="covalent" title="wikilink">covalent</a> binding of either the ovalicin or fumagillin <a class="uri" href="epoxide" title="wikilink">epoxide</a> moiety to the active site histidine residue of MetAP2 has been shown to inactivate the enzyme, thereby inhibiting angiogenesis. The way in which MetAP2 regulates angiogenesis has yet to be established, however, such that further study is required to validate that antiangiogenic activity results directly from MetAP2 inhibition. Nevertheless, with both the growth and <a class="uri" href="metastasis" title="wikilink">metastasis</a> of solid tumors depending heavily on angiogenesis, fumagillin and its analogs—including <a class="uri" href="TNP-470" title="wikilink">TNP-470</a>, <a class="uri" href="caplostatin" title="wikilink">caplostatin</a>, and <a class="uri" href="beloranib" title="wikilink">beloranib</a>—as well as ovalicin represent potential anticancer agents.<a class="footnoteRef" href="#fn43" id="fnref43"><sup>43</sup></a><a class="footnoteRef" href="#fn44" id="fnref44"><sup>44</sup></a> Moreover, the ability of MetAP2 to decrease cell viability in prokaryotic and small eukaryotic organisms has made it a target for antibacterial agents.<a class="footnoteRef" href="#fn45" id="fnref45"><sup>45</sup></a> Thus far, both fumagillin and TNP-470 have been shown to possess antimalarial activity both in vitro and in vivo, and <a class="uri" href="fumarranol" title="wikilink">fumarranol</a>, another fumagillin analog, represents a promising lead.<a class="footnoteRef" href="#fn46" id="fnref46"><sup>46</sup></a></p>

<p>Another METAP2 inhibitor <a class="uri" href="beloranib" title="wikilink">beloranib</a> (ZGN-433) has shown efficacy in reducing weight in severely obese subjects.<a class="footnoteRef" href="#fn47" id="fnref47"><sup>47</sup></a> MetAP2 inhibitors work by re-establishing balance to the ways the body metabolizes fat, leading to substantial loss of body weight.</p>
<h2 id="interactions">Interactions</h2>

<p>METAP2 has been shown to <a href="Protein-protein_interaction" title="wikilink">interact</a> with <a href="Protein_kinase_R" title="wikilink">Protein kinase R</a>.<a class="footnoteRef" href="#fn48" id="fnref48"><sup>48</sup></a> {{-}}</p>
<h2 id="references">References</h2>
<h2 id="further-reading">Further reading</h2>
<ul>
<li></li>
<li></li>
<li></li>
<li></li>
<li></li>
<li></li>
<li></li>
<li></li>
<li></li>
<li></li>
<li></li>
<li></li>
<li></li>
<li></li>
</ul>

<p>"</p>

<p><a href="Category:EC_3.4.11" title="wikilink">Category:EC 3.4.11</a> <a href="Category:Human_proteins" title="wikilink">Category:Human proteins</a> <a href="Category:Articles_containing_video_clips" title="wikilink">Category:Articles containing video clips</a></p>
<section class="footnotes">
<hr/>
<ol>
<li id="fn1"><a href="#fnref1">↩</a></li>
<li id="fn2"><a href="#fnref2">↩</a></li>
<li id="fn3"><a href="#fnref3">↩</a></li>
<li id="fn4"><a href="#fnref4">↩</a></li>
<li id="fn5"><a href="#fnref5">↩</a></li>
<li id="fn6"></li>
<li id="fn7"><a href="#fnref7">↩</a></li>
<li id="fn8"><a href="#fnref8">↩</a></li>
<li id="fn9"><a href="#fnref9">↩</a></li>
<li id="fn10"><a href="#fnref10">↩</a></li>
<li id="fn11"><a href="#fnref11">↩</a></li>
<li id="fn12"></li>
<li id="fn13"></li>
<li id="fn14"><a href="#fnref14">↩</a></li>
<li id="fn15"><a href="#fnref15">↩</a></li>
<li id="fn16"></li>
<li id="fn17"></li>
<li id="fn18"><a href="#fnref18">↩</a></li>
<li id="fn19"><a href="#fnref19">↩</a></li>
<li id="fn20"><a href="#fnref20">↩</a></li>
<li id="fn21"><a href="#fnref21">↩</a></li>
<li id="fn22"><a href="#fnref22">↩</a></li>
<li id="fn23"><a href="#fnref23">↩</a></li>
<li id="fn24"><a href="#fnref24">↩</a></li>
<li id="fn25"><a href="#fnref25">↩</a></li>
<li id="fn26"><a href="#fnref26">↩</a></li>
<li id="fn27"><a href="#fnref27">↩</a></li>
<li id="fn28"><a href="#fnref28">↩</a></li>
<li id="fn29"></li>
<li id="fn30"></li>
<li id="fn31"><a href="#fnref31">↩</a></li>
<li id="fn32"><a href="#fnref32">↩</a></li>
<li id="fn33"><a href="#fnref33">↩</a></li>
<li id="fn34"></li>
<li id="fn35"></li>
<li id="fn36"></li>
<li id="fn37"><a href="#fnref37">↩</a></li>
<li id="fn38"></li>
<li id="fn39"></li>
<li id="fn40"><a href="#fnref40">↩</a></li>
<li id="fn41"><a href="#fnref41">↩</a></li>
<li id="fn42"><a href="#fnref42">↩</a></li>
<li id="fn43"></li>
<li id="fn44"></li>
<li id="fn45"></li>
<li id="fn46"></li>
<li id="fn47"><a href="#fnref47">↩</a></li>
<li id="fn48"><a href="#fnref48">↩</a></li>
</ol>
</section>
</body>
</html>
