<html lang="en">
<head>
<meta charset="utf-8"/>
<title offset="1454">Molecular clock</title>
<script src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_SVG.js" type="text/javascript">
</script>
</head>
<body>
<h1>Molecular clock</h1>
<hr/>

<p>The '''molecular clock ''' (based on the <strong>molecular clock hypothesis</strong> (<strong>MCH</strong>)) is a technique in <a href="molecular_evolution" title="wikilink">molecular evolution</a> that uses fossil constraints and rates of molecular change to deduce the time in <a href="Geologic_time_scale" title="wikilink">geologic history</a> when two <a class="uri" href="species" title="wikilink">species</a> or other <a href="Taxon" title="wikilink">taxa</a> <a href="Genetic_divergence" title="wikilink">diverged</a>. It is used to estimate the time of occurrence of events called <a class="uri" href="speciation" title="wikilink">speciation</a> or <a href="Evolutionary_radiation" title="wikilink">radiation</a>. The molecular data used for such calculations are usually <a class="uri" href="nucleotide" title="wikilink">nucleotide</a> <a href="DNA_sequence" title="wikilink">sequences</a> for <a class="uri" href="DNA" title="wikilink">DNA</a> or <a href="amino_acid" title="wikilink">amino acid</a> sequences for <a href="protein" title="wikilink">proteins</a>. It is sometimes called a <strong>gene clock</strong> or <strong>evolutionary clock</strong>.</p>
<h2 id="early-discovery-and-genetic-equidistance">Early discovery and genetic equidistance</h2>

<p>The notion of the existence of a so-called "molecular clock" was first attributed to <a href="Emile_Zuckerkandl" title="wikilink">Emile Zuckerkandl</a> and <a href="Linus_Pauling" title="wikilink">Linus Pauling</a> who, in 1962, noticed that the number of <a href="amino_acid" title="wikilink">amino acid</a> differences in <a class="uri" href="hemoglobin" title="wikilink">hemoglobin</a> between different lineages changes roughly <a href="Linear_function" title="wikilink">linearly</a> with time, as estimated from fossil evidence.<a class="footnoteRef" href="#fn1" id="fnref1"><sup>1</sup></a> They generalized this observation to assert that the rate of <a href="evolution" title="wikilink">evolutionary</a> change of any specified <a class="uri" href="protein" title="wikilink">protein</a> was approximately constant over time and over different lineages.</p>

<p>The <strong>genetic equidistance</strong> phenomenon was first noted in 1963 by <a href="Emanuel_Margoliash" title="wikilink">Emanuel Margoliash</a>, who wrote: "It appears that the number of residue differences between <a href="cytochrome_C" title="wikilink">cytochrome C</a> of any two species is mostly conditioned by the time elapsed since the lines of evolution leading to these two species originally diverged. If this is correct, the cytochrome c of all mammals should be equally different from the cytochrome c of all birds. Since fish diverges from the main stem of vertebrate evolution earlier than either birds or mammals, the cytochrome c of both mammals and birds should be equally different from the cytochrome c of fish. Similarly, all vertebrate cytochrome c should be equally different from the yeast protein."<a class="footnoteRef" href="#fn2" id="fnref2"><sup>2</sup></a> For example, the difference between the cytochrome C of a carp and a frog, turtle, chicken, rabbit, and horse is a very constant 13% to 14%. Similarly, the difference between the cytochrome C of a bacterium and yeast, wheat, moth, tuna, pigeon, and horse ranges from 64% to 69%. Together with the work of Emile Zuckerkandl and Linus Pauling, the genetic equidistance result directly led to the formal postulation of the molecular clock hypothesis in the early 1960s.<a class="footnoteRef" href="#fn3" id="fnref3"><sup>3</sup></a> Genetic equidistance has often been used to infer equal time of separation of different sister species from an outgroup.<a class="footnoteRef" href="#fn4" id="fnref4"><sup>4</sup></a><a class="footnoteRef" href="#fn5" id="fnref5"><sup>5</sup></a></p>

<p>Later <a href="Allan_Wilson" title="wikilink">Allan Wilson</a> and <a href="Vincent_Sarich" title="wikilink">Vincent Sarich</a> built upon this work.</p>
<h2 id="relationship-with-neutral-theory">Relationship with neutral theory</h2>

<p>The observation of a clock-like rate of molecular change was originally purely <a href="Phenomenology_(science)" title="wikilink">phenomenological</a>. Later, the work of <a href="Motoo_Kimura" title="wikilink">Motoo Kimura</a><a class="footnoteRef" href="#fn6" id="fnref6"><sup>6</sup></a> developed the <a href="neutral_theory_of_molecular_evolution" title="wikilink">neutral theory of molecular evolution</a>, which predicted a molecular clock. Let there be N individuals, and to keep this calculation simple, let the individuals be <a href="Ploidy" title="wikilink">haploid</a> (i.e. have one copy of each gene). Let the rate of neutral <a href="mutation" title="wikilink">mutations</a> (i.e. mutations with no effect on <a href="Fitness_(biology)" title="wikilink">fitness</a>) in a new individual be 

<math display="inline" id="Molecular_clock:0">
 <semantics>
  <mi>μ</mi>
  <annotation-xml encoding="MathML-Content">
   <ci>μ</ci>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \mu
  </annotation>
 </semantics>
</math>

. The probability that this new mutation will become <a href="Fixation_(population_genetics)" title="wikilink">fixed</a> in the population is then 1/N, since each copy of the gene is as good as any other. Every generation, each individual can have new mutations, so there are 

<math display="inline" id="Molecular_clock:1">
 <semantics>
  <mi>μ</mi>
  <annotation-xml encoding="MathML-Content">
   <ci>μ</ci>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \mu
  </annotation>
 </semantics>
</math>

N new neutral mutations in the population as a whole. That means that each generation, 

<math display="inline" id="Molecular_clock:2">
 <semantics>
  <mi>μ</mi>
  <annotation-xml encoding="MathML-Content">
   <ci>μ</ci>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \mu
  </annotation>
 </semantics>
</math>

 new neutral mutations will become fixed. If most changes seen during <a href="molecular_evolution" title="wikilink">molecular evolution</a> are neutral, then <a href="Fixation_(population_genetics)" title="wikilink">fixations</a> in a population will accumulate at a clock-rate that is equal to the rate of neutral <a href="mutation" title="wikilink">mutations</a> in an individual.</p>
<h2 id="calibration">Calibration</h2>

<p>The molecular clock alone can only say that one time period is twice as long as another: it cannot assign concrete dates. To achieve this, the molecular clock must first be <a href="calibration" title="wikilink">calibrated</a> against independent evidence about dates, such as the <a class="uri" href="fossil" title="wikilink">fossil</a> record.<a class="footnoteRef" href="#fn7" id="fnref7"><sup>7</sup></a> For viral phylogenetics and <a href="ancient_DNA" title="wikilink">ancient DNA</a> studies – two areas of evolutionary biology where it is possible to sample sequences over an evolutionary timescale – the dates of the intermediate samples can be used to more precisely calibrate the molecular clock.</p>
<h2 id="non-constant-rate-of-molecular-clock">Non-constant rate of molecular clock</h2>

<p>Sometimes only a single divergence date can be estimated from fossils, with all other dates inferred from that. Other sets of species have abundant fossils available, allowing the MCH of constant divergence rates to be tested. DNA sequences experiencing low levels of <a href="negative_selection_(natural_selection)" title="wikilink">negative selection</a> showed divergence rates of 0.7–0.8% per <a class="uri" href="Myr" title="wikilink">Myr</a> in bacteria, mammals, invertebrates, and plants.<a class="footnoteRef" href="#fn8" id="fnref8"><sup>8</sup></a> In the same study, genomic regions experiencing very high negative or purifying selection (encoding rRNA) were considerably slower (1% per 50 Myr).</p>

<p>In addition to such variation in rate with genomic position, since the early 1990s, variation among taxa has proven fertile ground for research too,<a class="footnoteRef" href="#fn9" id="fnref9"><sup>9</sup></a> even over comparatively short periods of evolutionary time (for example <a href="mockingbird" title="wikilink">mockingbirds</a><a class="footnoteRef" href="#fn10" id="fnref10"><sup>10</sup></a>). <a href="Procellariiformes" title="wikilink">Tube-nosed seabirds</a> have molecular clocks that on average run at half speed of many other birds,<a class="footnoteRef" href="#fn11" id="fnref11"><sup>11</sup></a> possibly due to long <a class="uri" href="generation" title="wikilink">generation</a> times, and many turtles have a molecular clock running at one-eighth the speed it does in small mammals or even slower.<a class="footnoteRef" href="#fn12" id="fnref12"><sup>12</sup></a> Effects of <a href="small_population_size" title="wikilink">small population size</a> are also likely to confound molecular clock analyses. Researchers such as Francisco Ayala have more fundamentally challenged the molecular clock hypothesis.<a class="footnoteRef" href="#fn13" id="fnref13"><sup>13</sup></a><a class="footnoteRef" href="#fn14" id="fnref14"><sup>14</sup></a> According to Ayala's 1999 study, five factors combine to limit the application of molecular clock models:</p>
<ul>
<li>Changing generation times (If the rate of new mutations depends at least partly on the number of generations rather than the number of years)</li>
<li>Population size (<a href="Genetic_drift" title="wikilink">Genetic drift</a> is stronger in small populations, and so more mutations are effectively neutral)</li>
<li>Species-specific differences (due to differing metabolism, ecology, evolutionary history,...)</li>
<li>Change in function of the protein studied (can be avoided in closely related species by utilizing <a href="non-coding_DNA" title="wikilink">non-coding DNA</a> sequences or emphasizing <a href="silent_mutation" title="wikilink">silent mutations</a>)</li>
<li>Changes in the intensity of natural selection.</li>
</ul>

<p>Molecular clock users have developed workaround solutions using a number of statistical approaches including <a href="maximum_likelihood" title="wikilink">maximum likelihood</a> techniques and later <a href="Bayesian_statistics" title="wikilink">Bayesian modeling</a>. In particular, models that take into account rate variation across lineages have been proposed in order to obtain better estimates of divergence times. These models are called <strong>relaxed molecular clocks</strong><a class="footnoteRef" href="#fn15" id="fnref15"><sup>15</sup></a> because they represent an intermediate position between the 'strict' molecular clock hypothesis and Joseph Felsenstein's many-rates model and are made possible through <a href="Markov_chain_Monte_Carlo" title="wikilink">MCMC</a> techniques that explore a weighted range of tree topologies and simultaneously estimate parameters of the chosen substitution model. It must be remembered that divergence dates inferred using a molecular clock are based on statistical <a class="uri" href="inference" title="wikilink">inference</a> and not on direct <a class="uri" href="evidence" title="wikilink">evidence</a>.</p>

<p>The molecular clock runs into particular challenges at very short and very long timescales. At long timescales, the problem is <a href="Saturation_(genetic)" title="wikilink">saturation</a>. When enough time has passed, many sites have undergone more than one change, but it is impossible to detect more than one. This means that the observed number of changes is no longer <a href="Linear_function" title="wikilink">linear</a> with time, but instead flattens out.</p>

<p>At very short time scales, many differences between samples do not represent <a href="Fixation_(population_genetics)" title="wikilink">fixation</a> of different sequences in the different populations. Instead, they represent alternative <a class="uri" href="alleles" title="wikilink">alleles</a> that were both present as part of a polymorphism in the common ancestor. The inclusion of differences that have not yet become <a href="Fixation_(population_genetics)" title="wikilink">fixed</a> leads to a potentially dramatic inflation of the apparent rate of the molecular clock at very short timescales.<a class="footnoteRef" href="#fn16" id="fnref16"><sup>16</sup></a><a class="footnoteRef" href="#fn17" id="fnref17"><sup>17</sup></a></p>
<h2 id="methods">Methods</h2>

<p>Bayesian methods can provide more appropriate estimates of divergence times, especially if large datasets—such as those yielded by <a class="uri" href="phylogenomics" title="wikilink">phylogenomics</a>—are employed.<a class="footnoteRef" href="#fn18" id="fnref18"><sup>18</sup></a></p>
<h2 id="uses">Uses</h2>

<p>The molecular clock technique is an important tool in <a href="molecular_systematics" title="wikilink">molecular systematics</a>, the use of <a href="molecular_genetics" title="wikilink">molecular genetics</a> information to determine the correct <a href="scientific_classification" title="wikilink">scientific classification</a> of organisms or to study variation in selective forces.</p>

<p>Knowledge of approximately constant rate of molecular evolution in particular sets of lineages also facilitates establishing the dates of <a href="phylogeny" title="wikilink">phylogenetic</a> events, including those not documented by <a class="uri" href="fossils" title="wikilink">fossils</a>, such as the divergence of living <a href="taxon" title="wikilink">taxa</a> and the formation of the <a href="phylogenetic_tree" title="wikilink">phylogenetic tree</a>. In these cases—especially over long stretches of time—the limitations of MCH (above) must be considered; such estimates may be off by 50% or more.</p>
<h2 id="see-also">See also</h2>
<ul>
<li><a href="Charles_Darwin" title="wikilink">Charles Darwin</a></li>
<li><a href="Gene_orders" title="wikilink">Gene orders</a></li>
<li><a href="Human_mitochondrial_molecular_clock" title="wikilink">Human mitochondrial molecular clock</a></li>
<li><a href="Mitochondrial_Eve" title="wikilink">Mitochondrial Eve</a> and <a href="Y-chromosomal_Adam" title="wikilink">Y-chromosomal Adam</a></li>
<li><a href="Models_of_DNA_evolution" title="wikilink">Models of DNA evolution</a></li>
<li><a href="Molecular_evolution" title="wikilink">Molecular evolution</a></li>
<li><a href="Neutral_theory_of_molecular_evolution" title="wikilink">Neutral theory of molecular evolution</a></li>
</ul>
<h2 id="references">References</h2>
<h2 id="further-reading">Further reading</h2>
<ul>
<li></li>
<li></li>
<li></li>
</ul>
<h2 id="external-links">External links</h2>
<ul>
<li><a href="http://web.archive.org/web/20100525101859/http://awcmee.massey.ac.nz/aw.htm">Allan Wilson and the molecular clock</a></li>
<li><a href="http://web.archive.org/web/20090213150149/http://rtis.com/nat/user/elsberry/evobio/evc/argresp/sequence.html">Molecular clock explanation of the molecular equidistance phenomenon</a></li>
<li><a href="http://www.fossilrecord.net/dateaclade/index.html">Date-a-Clade service for the molecular tree of life</a></li>
</ul>

<p>"</p>

<p><a href="Category:Molecular_evolution" title="wikilink">Category:Molecular evolution</a> <a href="Category:Molecular_genetics" title="wikilink">Category:Molecular genetics</a> <a class="uri" href="Category:Phylogenetics" title="wikilink">Category:Phylogenetics</a></p>
<section class="footnotes">
<hr/>
<ol>
<li id="fn1"><a href="#fnref1">↩</a></li>
<li id="fn2"><a href="#fnref2">↩</a></li>
<li id="fn3"><a href="#fnref3">↩</a></li>
<li id="fn4"><a href="#fnref4">↩</a></li>
<li id="fn5">Huang, S. (2008) The genetic equidistance result of molecular evolution is independent of mutation rates. J. Comp. Sci. Syst. Biol., 1: 92-102. <a class="uri" href="http://omicsonline.com/ArchiveJCSB/Ab01/JCSB1.092.html">http://omicsonline.com/ArchiveJCSB/Ab01/JCSB1.092.html</a><a href="#fnref5">↩</a></li>
<li id="fn6"><a href="#fnref6">↩</a></li>
<li id="fn7"><a href="#fnref7">↩</a></li>
<li id="fn8"><a href="#fnref8">↩</a></li>
<li id="fn9"><a href="#fnref9">↩</a></li>
<li id="fn10"><a href="#fnref10">↩</a></li>
<li id="fn11"><a href="#fnref11">↩</a></li>
<li id="fn12"><a href="#fnref12">↩</a></li>
<li id="fn13"><a href="#fnref13">↩</a></li>
<li id="fn14"><a href="#fnref14">↩</a></li>
<li id="fn15"><a href="#fnref15">↩</a></li>
<li id="fn16"><a href="#fnref16">↩</a></li>
<li id="fn17"><a href="#fnref17">↩</a></li>
<li id="fn18"><a href="#fnref18">↩</a></li>
</ol>
</section>
</body>
</html>
