<html lang="en">
<head>
<meta charset="utf-8"/>
<title offset="1883">Natural computing</title>
<script src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_SVG.js" type="text/javascript">
</script>
</head>
<body>
<h1>Natural computing</h1>
<hr/>

<p><strong>Natural computing</strong>,<a class="footnoteRef" href="#fn1" id="fnref1"><sup>1</sup></a> also called <strong>natural computation</strong>, is a terminology introduced to encompass three classes of methods: 1) those that take inspiration from nature for the development of novel problem-solving techniques; 2) those that are based on the use of computers to synthesize natural phenomena; and 3) those that employ natural materials (e.g., molecules) to compute. The main fields of research that compose these three branches are <a href="artificial_neural_networks" title="wikilink">artificial neural networks</a>, <a href="evolutionary_algorithms" title="wikilink">evolutionary algorithms</a>, <a href="swarm_intelligence" title="wikilink">swarm intelligence</a>, <a href="artificial_immune_systems" title="wikilink">artificial immune systems</a>, fractal geometry, <a href="artificial_life" title="wikilink">artificial life</a>, <a href="DNA_computing" title="wikilink">DNA computing</a>, and <a href="quantum_computing" title="wikilink">quantum computing</a>, among others.</p>

<p>Computational paradigms studied by natural computing are abstracted from natural phenomena as diverse as <a class="uri" href="self-replication" title="wikilink">self-replication</a>, the functioning of the <a class="uri" href="brain" title="wikilink">brain</a>, <a href="Darwinian_evolution" title="wikilink">Darwinian evolution</a>, <a href="group_behavior" title="wikilink">group behavior</a>, the <a href="immune_system" title="wikilink">immune system</a>, the defining properties of life forms, <a href="cell_membranes" title="wikilink">cell membranes</a>, and <a class="uri" href="morphogenesis" title="wikilink">morphogenesis</a>. Besides traditional <a href="electronic_hardware" title="wikilink">electronic hardware</a>, these computational paradigms can be implemented on alternative physical media such as biomolecules (DNA, RNA), or trapped-ion <a href="#Quantum_computing" title="wikilink">quantum computing</a> devices.</p>

<p>Dually, one can view processes occurring in nature as information processing. Such processes include <a class="uri" href="self-assembly" title="wikilink">self-assembly</a>, <a href="developmental_process" title="wikilink">developmental processes</a>, <a href="gene_regulation" title="wikilink">gene regulation</a> networks, <a href="protein-protein_interaction" title="wikilink">protein-protein interaction</a> networks, biological transport (<a href="active_transport" title="wikilink">active transport</a>, <a href="passive_transport" title="wikilink">passive transport</a>) networks, and <a href="gene_assembly" title="wikilink">gene assembly</a> in <a href="unicellular_organism" title="wikilink">unicellular organisms</a>. Efforts to understand biological systems also include engineering of semi-synthetic organisms, and understanding the universe itself from the point of view of information processing. Indeed, the idea was even advanced that information is more fundamental than matter or energy. The Zuse-Fredkin thesis, dating back to the 1960s, states that the entire universe is a huge <a href="cellular_automaton" title="wikilink">cellular automaton</a> which continuously updates its rules.<a class="footnoteRef" href="#fn2" id="fnref2"><sup>2</sup></a><a class="footnoteRef" href="#fn3" id="fnref3"><sup>3</sup></a> Recently it has been suggested that the whole universe is a <a href="quantum_computer" title="wikilink">quantum computer</a> that computes its own behaviour.<a class="footnoteRef" href="#fn4" id="fnref4"><sup>4</sup></a></p>
<h2 id="nature-inspired-models-of-computation">Nature-inspired models of computation</h2>

<p>The most established "classical" nature-inspired models of computation are cellular automata, neural computation, and evolutionary computation. More recent computational systems abstracted from natural processes include swarm intelligence, artificial immune systems, membrane computing, and amorphous computing. In fact, all major methods and algorithms are nature-inspired <a class="uri" href="metaheuristic" title="wikilink">metaheuristic</a> algorithms<a class="footnoteRef" href="#fn5" id="fnref5"><sup>5</sup></a> including cellular automata, evolutionary computation, swarm intelligence and others. The detailed review can be found in many books <a class="footnoteRef" href="#fn6" id="fnref6"><sup>6</sup></a><a class="footnoteRef" href="#fn7" id="fnref7"><sup>7</sup></a></p>
<h3 id="cellular-automata">Cellular automata</h3>
<dl>
<dd><em>Further information: <a href="Cellular_automaton" title="wikilink">Cellular automaton</a></em>
</dd>
</dl>

<p>A cellular automaton is a <a href="dynamical_system" title="wikilink">dynamical system</a> consisting of an array of cells. Space and time are discrete and each of the cells can be in a finite number of <a href="state_(computer_science)" title="wikilink">states</a>. The cellular automaton updates the states of its cells synchronously according to the transition rules given <a href="a_priori_probability" title="wikilink"><em>a priori</em></a>. The next state of a cell is computed by a transition rule and it depends only on its current state and the states of its neighbors.</p>

<p><a href="Conway's_game_of_life" title="wikilink">Conway's game of life</a> is one of the best-known examples of cellular automata, shown to be <a href="Turing_completeness" title="wikilink">computationally universal</a>. Cellular automata have been applied to modelling a variety of phenomena such as communication, growth, reproduction, competition, evolution and other physical and biological processes.</p>
<h3 id="neural-computation">Neural computation</h3>
<dl>
<dd><em>Further information: <a href="Neural_computation" title="wikilink">Neural computation</a></em>
</dd>
</dl>

<p>Neural computation is the field of research that emerged from the comparison between <a href="computer" title="wikilink">computing machines</a> and the human <a href="nervous_system" title="wikilink">nervous system</a>.<a class="footnoteRef" href="#fn8" id="fnref8"><sup>8</sup></a> This field aims both to understand how the <strong><a class="uri" href="brain" title="wikilink">brain</a></strong> of <a href="living_organisms" title="wikilink">living organisms</a> works (<a href="brain_theory" title="wikilink">brain theory</a> or <a href="computational_neuroscience" title="wikilink">computational neuroscience</a>), and to design efficient algorithms based on the principles of how the human brain processes information (Artificial Neural Networks, ANN <a class="footnoteRef" href="#fn9" id="fnref9"><sup>9</sup></a>).</p>

<p>An <a href="artificial_neural_network" title="wikilink">artificial neural network</a> is a network of <a href="artificial_neurons" title="wikilink">artificial neurons</a>.<a class="footnoteRef" href="#fn10" id="fnref10"><sup>10</sup></a> An artificial neuron <em>A</em> is equipped with a function 

<math display="inline" id="Natural_computing:0">
 <semantics>
  <msub>
   <mi>f</mi>
   <mi>A</mi>
  </msub>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <csymbol cd="ambiguous">subscript</csymbol>
    <ci>f</ci>
    <ci>A</ci>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   f_{A}
  </annotation>
 </semantics>
</math>

, receives <em>n</em> <a href="real_number" title="wikilink">real-valued</a> inputs 

<math display="inline" id="Natural_computing:1">
 <semantics>
  <mrow>
   <msub>
    <mi>x</mi>
    <mn>1</mn>
   </msub>
   <mo>,</mo>
   <msub>
    <mi>x</mi>
    <mn>2</mn>
   </msub>
   <mo>,</mo>
   <mi mathvariant="normal">…</mi>
   <mo>,</mo>
   <msub>
    <mi>x</mi>
    <mi>n</mi>
   </msub>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <list>
    <apply>
     <csymbol cd="ambiguous">subscript</csymbol>
     <ci>x</ci>
     <cn type="integer">1</cn>
    </apply>
    <apply>
     <csymbol cd="ambiguous">subscript</csymbol>
     <ci>x</ci>
     <cn type="integer">2</cn>
    </apply>
    <ci>normal-…</ci>
    <apply>
     <csymbol cd="ambiguous">subscript</csymbol>
     <ci>x</ci>
     <ci>n</ci>
    </apply>
   </list>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   x_{1},x_{2},\ldots,x_{n}
  </annotation>
 </semantics>
</math>

 with respective <strong><a href="weight_function" title="wikilink">weights</a></strong> 

<math display="inline" id="Natural_computing:2">
 <semantics>
  <mrow>
   <msub>
    <mi>w</mi>
    <mn>1</mn>
   </msub>
   <mo>,</mo>
   <msub>
    <mi>w</mi>
    <mn>2</mn>
   </msub>
   <mo>,</mo>
   <mi mathvariant="normal">…</mi>
   <mo>,</mo>
   <msub>
    <mi>w</mi>
    <mi>n</mi>
   </msub>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <list>
    <apply>
     <csymbol cd="ambiguous">subscript</csymbol>
     <ci>w</ci>
     <cn type="integer">1</cn>
    </apply>
    <apply>
     <csymbol cd="ambiguous">subscript</csymbol>
     <ci>w</ci>
     <cn type="integer">2</cn>
    </apply>
    <ci>normal-…</ci>
    <apply>
     <csymbol cd="ambiguous">subscript</csymbol>
     <ci>w</ci>
     <ci>n</ci>
    </apply>
   </list>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   w_{1},w_{2},\ldots,w_{n}
  </annotation>
 </semantics>
</math>

, and it outputs 

<math display="inline" id="Natural_computing:3">
 <semantics>
  <mrow>
   <msub>
    <mi>f</mi>
    <mi>A</mi>
   </msub>
   <mrow>
    <mo stretchy="false">(</mo>
    <mrow>
     <mrow>
      <msub>
       <mi>w</mi>
       <mn>1</mn>
      </msub>
      <msub>
       <mi>x</mi>
       <mn>1</mn>
      </msub>
     </mrow>
     <mo>+</mo>
     <mrow>
      <msub>
       <mi>w</mi>
       <mn>2</mn>
      </msub>
      <msub>
       <mi>x</mi>
       <mn>2</mn>
      </msub>
     </mrow>
     <mo>+</mo>
     <mi mathvariant="normal">…</mi>
     <mo>+</mo>
     <mrow>
      <msub>
       <mi>w</mi>
       <mi>n</mi>
      </msub>
      <msub>
       <mi>x</mi>
       <mi>n</mi>
      </msub>
     </mrow>
    </mrow>
    <mo stretchy="false">)</mo>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <times></times>
    <apply>
     <csymbol cd="ambiguous">subscript</csymbol>
     <ci>f</ci>
     <ci>A</ci>
    </apply>
    <apply>
     <plus></plus>
     <apply>
      <times></times>
      <apply>
       <csymbol cd="ambiguous">subscript</csymbol>
       <ci>w</ci>
       <cn type="integer">1</cn>
      </apply>
      <apply>
       <csymbol cd="ambiguous">subscript</csymbol>
       <ci>x</ci>
       <cn type="integer">1</cn>
      </apply>
     </apply>
     <apply>
      <times></times>
      <apply>
       <csymbol cd="ambiguous">subscript</csymbol>
       <ci>w</ci>
       <cn type="integer">2</cn>
      </apply>
      <apply>
       <csymbol cd="ambiguous">subscript</csymbol>
       <ci>x</ci>
       <cn type="integer">2</cn>
      </apply>
     </apply>
     <ci>normal-…</ci>
     <apply>
      <times></times>
      <apply>
       <csymbol cd="ambiguous">subscript</csymbol>
       <ci>w</ci>
       <ci>n</ci>
      </apply>
      <apply>
       <csymbol cd="ambiguous">subscript</csymbol>
       <ci>x</ci>
       <ci>n</ci>
      </apply>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   f_{A}(w_{1}x_{1}+w_{2}x_{2}+\ldots+w_{n}x_{n})
  </annotation>
 </semantics>
</math>


. Some neurons are selected to be the output neurons, and the network function is the vectorial function that associates to the <em>n</em> input values, the outputs of the <em>m</em> selected output neurons. Note that different choices of weights produce different network functions for the same inputs. Back-propagation is a <a href="supervised_learning" title="wikilink">supervised learning method</a> by which the weights of the connections in the network are repeatedly adjusted so as to minimize the difference between the vector of actual outputs and that of desired outputs. <a href="machine_learning" title="wikilink">Learning algorithms</a> based on <a href="backpropagation" title="wikilink">backwards propagation of errors</a> can be used to find optimal weights for given <a href="network_topology" title="wikilink">topology of the network</a> and input-output pairs.</p>
<h3 id="evolutionary-computation">Evolutionary computation</h3>
<dl>
<dd><em>Further information: <a href="Evolutionary_computation" title="wikilink">Evolutionary computation</a></em>
</dd>
</dl>

<p>Evolutionary computation<a class="footnoteRef" href="#fn11" id="fnref11"><sup>11</sup></a> is a computational paradigm inspired by <a href="Darwinian_evolution" title="wikilink">Darwinian evolution</a>.</p>

<p>An artificial evolutionary system is a computational system based on the notion of simulated evolution. It comprises a constant- or variable-size population of individuals, a <a href="fitness_(biology)" title="wikilink">fitness criterion</a>, and genetically inspired operators that produce the next <strong><a class="uri" href="generation" title="wikilink">generation</a></strong> from the current one. The initial population is typically generated randomly or heuristically, and typical operators are <a class="uri" href="mutation" title="wikilink">mutation</a> and <a href="genetic_recombination" title="wikilink">recombination</a>. At each step, the individuals are evaluated according to the given fitness function (<a href="survival_of_the_fittest" title="wikilink">survival of the fittest</a>). The next generation is obtained from selected individuals (parents) by using genetically inspired operators. The choice of parents can be guided by a selection operator which reflects the biological principle of <a href="mate_selection" title="wikilink">mate selection</a>. This process of simulated <a class="uri" href="evolution" title="wikilink">evolution</a> eventually converges towards a nearly optimal population of individuals, from the point of view of the fitness function.</p>

<p>The study of evolutionary systems has historically evolved along three main branches: <a href="Evolution_strategies" title="wikilink">Evolution strategies</a> provide a solution to <a href="optimization_(mathematics)" title="wikilink">parameter optimization problems</a> for real-valued as well as discrete and mixed types of parameters. <a href="Evolutionary_programming" title="wikilink">Evolutionary programming</a> originally aimed at creating optimal "intelligent agents" modelled, e.g., as finite state machines. <a href="Genetic_algorithms" title="wikilink">Genetic algorithms</a><a class="footnoteRef" href="#fn12" id="fnref12"><sup>12</sup></a> applied the idea of evolutionary computation to the problem of finding a (nearly-)optimal solution to a given problem. Genetic algorithms initially consisted of an input population of individuals encoded as fixed-length bit strings, the genetic operators mutation (bit flips) and recombination (combination of a prefix of a parent with the suffix of the other), and a problem-dependent fitness function. Genetic algorithms have been used to optimize computer programs, called <a href="genetic_programming" title="wikilink">genetic programming</a>, and today they are also applied to real-valued parameter optimization problems as well as to many types of <a href="combinatorial_tasks" title="wikilink">combinatorial tasks</a>.</p>
<h3 id="swarm-intelligence">Swarm intelligence</h3>

<p><a href="Swarm_intelligence" title="wikilink">Swarm intelligence</a>,<a class="footnoteRef" href="#fn13" id="fnref13"><sup>13</sup></a> sometimes referred to as <a href="collective_intelligence" title="wikilink">collective intelligence</a>, is defined as the problem solving behavior that emerges from the interaction of <a href="intelligence_agent" title="wikilink">individual agents</a> (e.g., <a class="uri" href="bacteria" title="wikilink">bacteria</a>, <a class="uri" href="ants" title="wikilink">ants</a>, <a class="uri" href="termites" title="wikilink">termites</a>, <a class="uri" href="bees" title="wikilink">bees</a>, <a class="uri" href="spiders" title="wikilink">spiders</a>, <a class="uri" href="fish" title="wikilink">fish</a>, <a class="uri" href="birds" title="wikilink">birds</a>) which communicate with other agents by acting on their <a href="neighborhood_(mathematics)" title="wikilink">local environments</a>.</p>

<p><a href="Particle_swarm_optimization" title="wikilink">Particle swarm optimization</a> applies this idea to the problem of finding an optimal solution to a given problem by a search through a (multi-dimensional) <a href="solution_space" title="wikilink">solution space</a>. The initial set-up is a swarm of <em>particles</em>, each representing a possible solution to the problem. Each particle has its own <a class="uri" href="velocity" title="wikilink">velocity</a> which depends on its previous velocity (the inertia component), the tendency towards the past personal best position (the nostalgia component), and its tendency towards a global neighborhood optimum or local neighborhood optimum (the social component). Particles thus move through a multidimensional space and eventually converge towards a point between the <a href="maxima_and_minima" title="wikilink">global best</a> and their personal best. Particle swarm optimization algorithms have been applied to various optimization problems, and to <a href="unsupervised_learning" title="wikilink">unsupervised learning</a>, <a href="game_learning" title="wikilink">game learning</a>, and <a href="scheduling_(computing)" title="wikilink">scheduling</a> applications.</p>

<p>In the same vein, <a href="ant_colony_optimization" title="wikilink">ant algorithms</a> model the foraging behaviour of ant colonies. To find the best path between the nest and a source of food, ants rely on indirect communication by laying a <a class="uri" href="pheromone" title="wikilink">pheromone</a> trail on the way back to the nest if they found food, respectively following the concentration of pheromones if they are looking for food. Ant algorithms have been successfully applied to a variety of combinatorial optimization problems over discrete search spaces.</p>
<h3 id="artificial-immune-systems">Artificial immune systems</h3>

<p>Artificial immune systems (a.k.a. immunological computation or <a class="uri" href="immunocomputing" title="wikilink">immunocomputing</a>) are computational systems inspired by the natural immune systems of biological organisms.</p>

<p>Viewed as an information processing system, the <a href="immune_system" title="wikilink">natural immune system</a> of organisms performs many complex tasks in <a href="parallel_computation" title="wikilink">parallel</a> and <a href="distributed_computing" title="wikilink">distributed computing</a> fashion.<a class="footnoteRef" href="#fn14" id="fnref14"><sup>14</sup></a> These include distinguishing between self and <a href="exogenous_antigen" title="wikilink">nonself</a>,<a class="footnoteRef" href="#fn15" id="fnref15"><sup>15</sup></a> <a href="Neutralisation_(immunology)" title="wikilink">neutralization</a> of nonself <a class="uri" href="pathogens" title="wikilink">pathogens</a> (<a class="uri" href="viruses" title="wikilink">viruses</a>, bacteria, <a class="uri" href="fungi" title="wikilink">fungi</a>, and <a href="parasitism" title="wikilink">parasites</a>), <a class="uri" href="learning" title="wikilink">learning</a>, <a class="uri" href="memory" title="wikilink">memory</a>, <a href="associative_retrieval" title="wikilink">associative retrieval</a>, <a href="homeostasis" title="wikilink">self-regulation</a>, and <a class="uri" href="fault-tolerance" title="wikilink">fault-tolerance</a>. <a href="Artificial_immune_systems" title="wikilink">Artificial immune systems</a> are abstractions of the natural immune system, emphasizing these computational aspects. Their applications include <a href="antivirus_software" title="wikilink">computer virus detection</a>, <a href="anomaly_detection" title="wikilink">anomaly detection</a> in a time series of data, <a href="fault_diagnosis" title="wikilink">fault diagnosis</a>, <a href="pattern_recognition" title="wikilink">pattern recognition</a>, machine learning, <a class="uri" href="bioinformatics" title="wikilink">bioinformatics</a>, optimization, <a class="uri" href="robotics" title="wikilink">robotics</a> and <a href="Control_theory" title="wikilink">control</a>.</p>
<h3 id="membrane-computing">Membrane computing</h3>

<p><a href="Membrane_computing" title="wikilink">Membrane computing</a> investigates computing models abstracted from the <a href="cell_compartment" title="wikilink">compartmentalized structure</a> of living cells effected by <a href="cell_membrane" title="wikilink">membranes</a>.<a class="footnoteRef" href="#fn16" id="fnref16"><sup>16</sup></a> A generic membrane system (P-system) consists of cell-like compartments (regions) delimited by <em>membranes</em>, that are placed in a <a href="nested_hierarchy" title="wikilink">nested hierarchical</a> structure. Each membrane-enveloped region contains objects, transformation rules which modify these objects, as well as transfer rules, which specify whether the objects will be transferred outside or stay inside the region. Regions communicate with each other via the transfer of objects. The computation by a membrane system starts with an initial configuration, where the number (<a href="Multiplicity_(mathematics)" title="wikilink">multiplicity</a>) of each object is set to some value for each region (<a href="multiset" title="wikilink">multiset of objects</a>). It proceeds by choosing, <a href="nondeterminism" title="wikilink">nondeterministically</a> and in a <a href="parallelism_(computing)" title="wikilink">maximally parallel manner</a>, which rules are applied to which objects. The output of the computation is collected from an <em>a priori</em> determined output region.</p>

<p>Applications of membrane systems include machine learning, modelling of biological processes (<a class="uri" href="photosynthesis" title="wikilink">photosynthesis</a>, certain <a href="signaling_pathways" title="wikilink">signaling pathways</a>, <a href="quorum_sensing" title="wikilink">quorum sensing</a> in bacteria, cell-mediated <a href="Immunity_(medical)" title="wikilink">immunity</a>), as well as computer science applications such as <a href="computer_graphics" title="wikilink">computer graphics</a>, <a href="public-key_cryptography" title="wikilink">public-key cryptography</a>, <a href="approximation_algorithm" title="wikilink">approximation</a> and <a href="sorting_algorithms" title="wikilink">sorting algorithms</a>, as well as analysis of various <a href="computationally_hard_problems" title="wikilink">computationally hard problems</a>.</p>
<h3 id="amorphous-computing">Amorphous computing</h3>

<p>In biological organisms, <a class="uri" href="morphogenesis" title="wikilink">morphogenesis</a> (the development of well-defined shapes and functional structures) is achieved by the interactions between cells guided by the genetic <em>program</em> encoded in the organism's DNA.</p>

<p>Inspired by this idea, <a href="amorphous_computing" title="wikilink">amorphous computing</a> aims at engineering well-defined shapes and patterns, or coherent computational behaviours, from the local interactions of a multitude of simple unreliable, irregularly placed, asynchronous, identically programmed computing elements (particles).<a class="footnoteRef" href="#fn17" id="fnref17"><sup>17</sup></a> As a programming paradigm, the aim is to find new <a href="abstraction_(computer_science)" title="wikilink">programming techniques</a> that would work well for amorphous computing environments. Amorphous computing also plays an important role as the basis for "<a href="#Cellular_computing" title="wikilink">cellular computing</a>" (see the topics <a href="synthetic_biology" title="wikilink">synthetic biology</a> and <a href="cellular_computing" title="wikilink">cellular computing</a>, below).</p>
<h2 id="synthesizing-nature-by-means-of-computing">Synthesizing nature by means of computing</h2>
<h3 id="artificial-life">Artificial life</h3>

<p><a href="Artificial_life" title="wikilink">Artificial life</a> (ALife) is a research field whose ultimate goal is to understand the essential properties of life organisms <a class="footnoteRef" href="#fn18" id="fnref18"><sup>18</sup></a> by building, within electronic computers or other artificial media, <em><a href="ab_initio" title="wikilink">ab initio</a></em> systems that exhibit properties normally associated only with living organisms. Early examples include <a href="Lindenmayer_systems" title="wikilink">Lindenmayer systems</a> (L-systems), that have been used to model plant growth and development. An L-system is a parallel rewriting system that starts with an initial word, and applies its rewriting rules in parallel to all letters of the word.<a class="footnoteRef" href="#fn19" id="fnref19"><sup>19</sup></a></p>

<p>Pioneering experiments in artificial life included the design of evolving "virtual block creatures" acting in simulated environments with realistic features such as <a href="kinetics_(physics)" title="wikilink">kinetics</a>, <a href="Dynamics_(mechanics)" title="wikilink">dynamics</a>, <a class="uri" href="gravity" title="wikilink">gravity</a>, <a class="uri" href="collision" title="wikilink">collision</a>, and <a class="uri" href="friction" title="wikilink">friction</a>.<a class="footnoteRef" href="#fn20" id="fnref20"><sup>20</sup></a> These artificial creatures were selected for their abilities endowed to swim, or walk, or jump, and they competed for a common limited resource (controlling a cube). The simulation resulted in the evolution of creatures exhibiting surprising behaviour: some developed hands to grab the cube, others developed legs to move towards the cube. This computational approach was further combined with rapid manufacturing technology to actually build the physical robots that virtually evolved.<a class="footnoteRef" href="#fn21" id="fnref21"><sup>21</sup></a> This marked the emergence of the field of <strong>mechanical artificial life</strong>.</p>

<p>The field of <a href="#Synthetic_biology" title="wikilink">synthetic biology</a> explores a biological implementation of similar ideas. Other research directions within the field of artificial life include <a href="artificial_chemistry" title="wikilink">artificial chemistry</a> as well as traditionally biological phenomena explored in artificial systems, ranging from computational processes such as <a href="coevolution" title="wikilink">co-evolutionary</a> adaptation and development, to physical processes such as growth, <a class="uri" href="self-replication" title="wikilink">self-replication</a>, and <a class="uri" href="self-repair" title="wikilink">self-repair</a>.</p>
<h2 id="nature-inspired-novel-hardware">Nature-inspired novel hardware</h2>

<p>All of the computational techniques mentioned above, while inspired by nature, have been implemented until now mostly on traditional <a href="electronic_hardware" title="wikilink">electronic hardware</a>. In contrast, the two paradigms introduced here, <a href="#Molecular_computing" title="wikilink">molecular computing</a> and <a href="#Quantum_computing" title="wikilink">quantum computing</a>, employ radically different types of hardware.</p>
<h3 id="molecular-computing">Molecular computing</h3>

<p><a href="Molecular_computer" title="wikilink">Molecular computing</a> (a.k.a. biomolecular computing, biocomputing, biochemical computing, <a href="DNA_computing" title="wikilink">DNA computing</a>) is a computational paradigm in which data is encoded as <a class="uri" href="biomolecules" title="wikilink">biomolecules</a> such as <a href="DNA_sequence" title="wikilink">DNA strands</a>, and molecular biology tools act on the data to perform various operations (e.g., <a href="arithmetic_operation" title="wikilink">arithmetic</a> or <a href="logical_operations" title="wikilink">logical operations</a>). The first experimental realization of special-purpose molecular computer was the 1994 breakthrough experiment by <a href="Leonard_Adleman" title="wikilink">Leonard Adleman</a> who solved a 7-node instance of the <a href="Hamiltonian_Path_Problem" title="wikilink">Hamiltonian Path Problem</a> solely by manipulating DNA strands in test tubes.<a class="footnoteRef" href="#fn22" id="fnref22"><sup>22</sup></a> DNA computations start from an initial input encoded as a DNA sequence (essentially a sequence over the four-letter alphabet {A, C, G, T}), and proceed by a succession of bio-operations such as cut-and-paste (by <a href="restriction_enzymes" title="wikilink">restriction enzymes</a> and <a class="uri" href="ligases" title="wikilink">ligases</a>), extraction of strands containing a certain subsequence (by using Watson-Crick complementarity), copy (by using <a href="polymerase_chain_reaction" title="wikilink">polymerase chain reaction</a> that employs the polymerase enzyme), and read-out.<a class="footnoteRef" href="#fn23" id="fnref23"><sup>23</sup></a> Recent experimental research succeeded in solving more complex instances of <a class="uri" href="NP-complete" title="wikilink">NP-complete</a> problems such as a 20-variable instance of <a class="uri" href="3SAT" title="wikilink">3SAT</a>, and wet DNA implementations of finite state machines with potential applications to the design of <a href="Targeted_drug_delivery" title="wikilink">smart drugs</a>.</p>

<p> One of the most notable contributions of research in this field is to the understanding of <a class="uri" href="self-assembly" title="wikilink">self-assembly</a>.<a class="footnoteRef" href="#fn24" id="fnref24"><sup>24</sup></a> Self-assembly is the <a href=":wikt:bottom-up" title="wikilink">bottom-up</a> process by which objects autonomously come together to form complex structures. Instances in nature abound, and include <a class="uri" href="atoms" title="wikilink">atoms</a> binding by chemical bonds to form <a class="uri" href="molecules" title="wikilink">molecules</a>, and molecules forming <a class="uri" href="crystals" title="wikilink">crystals</a> or <a class="uri" href="macromolecules" title="wikilink">macromolecules</a>. Examples of self-assembly research topics include self-assembled DNA nanostructures<a class="footnoteRef" href="#fn25" id="fnref25"><sup>25</sup></a> such as <a href="Sierpinski_triangle" title="wikilink">Sierpinski triangles</a><a class="footnoteRef" href="#fn26" id="fnref26"><sup>26</sup></a> or arbitrary nanoshapes obtained using the <a href="DNA_origami" title="wikilink">DNA origami</a><a class="footnoteRef" href="#fn27" id="fnref27"><sup>27</sup></a> technique, and DNA nanomachines<a class="footnoteRef" href="#fn28" id="fnref28"><sup>28</sup></a> such as DNA-based circuits (<a href="binary_counter" title="wikilink">binary counter</a>, <a href="bit-wise_cumulative_XOR" title="wikilink">bit-wise cumulative XOR</a>), ribozymes for logic operations, molecular switches (<a href="DNA_tweezers" title="wikilink">DNA tweezers</a>), and autonomous molecular motors (<a href="DNA_walkers" title="wikilink">DNA walkers</a>).</p>

<p>Theoretical research in molecular computing has yielded several novel models of DNA computing (e.g. <a href="splicing_systems" title="wikilink">splicing systems</a> introduced by Tom Head already in 1987) and their computational power has been investigated.<a class="footnoteRef" href="#fn29" id="fnref29"><sup>29</sup></a> Various subsets of bio-operations are now known to be able to achieve the computational power of <a href="Turing_machines" title="wikilink">Turing machines</a> .</p>
<h3 id="quantum-computing">Quantum computing</h3>
<dl>
<dd><em>Further information: <a href="Quantum_computing" title="wikilink">Quantum computing</a></em>
</dd>
</dl>

<p>A quantum computer<a class="footnoteRef" href="#fn30" id="fnref30"><sup>30</sup></a> processes data stored as quantum bits (<a class="uri" href="qubits" title="wikilink">qubits</a>), and uses quantum mechanical phenomena such as <a href="quantum_superposition" title="wikilink">superposition</a> and <a href="quantum_entanglement" title="wikilink">entanglement</a> to perform computations. A qubit can hold a "0", a "1", or a quantum superposition of these. A quantum computer operates on qubits with <a href="quantum_gate" title="wikilink">quantum logic gates</a>. Through <a href="Shor's_algorithm" title="wikilink">Shor's polynomial algorithm</a> for factoring integers, and <a href="Grover's_algorithm" title="wikilink">Grover's algorithm</a> for quantum database search that has a quadratic time advantage, quantum computers were shown to potentially possess a significant benefit relative to electronic computers.</p>

<p><a href="Quantum_cryptography" title="wikilink">Quantum cryptography</a> is not based on the <a href="Computational_complexity_theory" title="wikilink">complexity of the computation</a>, but on the special properties of <a href="quantum_information" title="wikilink">quantum information</a>, such as the fact that quantum information cannot be measured reliably and any attempt at measuring it results in an unavoidable and irreversible disturbance. A successful open air experiment in quantum cryptography was reported in 2007, where data was transmitted securely over a distance of 144 km.<a class="footnoteRef" href="#fn31" id="fnref31"><sup>31</sup></a> <a href="Quantum_teleportation" title="wikilink">Quantum teleportation</a> is another promising application, in which a quantum state (not matter or energy) is transferred to an arbitrary distant location. Implementations of practical quantum computers are based on various substrates such as <a href="ion_trap" title="wikilink">ion-traps</a>, <a class="uri" href="superconductors" title="wikilink">superconductors</a>, <a href="nuclear_magnetic_resonance" title="wikilink">nuclear magnetic resonance</a>, etc. As of 2006, the largest quantum computing experiment used liquid state nuclear magnetic resonance quantum information processors, and could operate on up to 12 qubits.<a class="footnoteRef" href="#fn32" id="fnref32"><sup>32</sup></a></p>
<h2 id="nature-as-information-processing">Nature as information processing</h2>

<p>The dual aspect of natural computation is that it aims to understand nature by regarding natural phenomena as information processing. The two main directions of research in this area are <a href="#Systems_biology" title="wikilink">systems biology</a> and <a href="#Synthetic_biology" title="wikilink">synthetic biology</a>.</p>
<h3 id="systems-biology">Systems biology</h3>

<p><em>Further information: <a href="systems_biology" title="wikilink">systems biology</a></em></p>

<p>Computational systems biology (or simply systems biology) is an integrative and qualitative approach that investigates the complex communications and interactions taking place in biological systems. Thus, in systems biology, the focus of the study is the <a href="interaction_network" title="wikilink">interaction networks</a> themselves and the properties of biological systems that arise due to these networks, rather than the individual components of functional processes in an organism. This type of research on organic components has focused strongly on four different interdependent interaction networks:<a class="footnoteRef" href="#fn33" id="fnref33"><sup>33</sup></a> gene-regulatory networks, biochemical networks, transport networks, and carbohydrate networks.</p>

<p><a href="Gene_regulatory_networks" title="wikilink">Gene regulatory networks</a> comprise gene-gene interactions, as well as interactions between genes and other substances in the cell. <a href="Gene" title="wikilink">Genes</a> are transcribed into <a href="messenger_RNA" title="wikilink">messenger RNA</a> (mRNA), and then translated into <a href="protein" title="wikilink">proteins</a> according to the <a href="genetic_code" title="wikilink">genetic code</a>. Each gene is associated with other DNA segments (<a href="promoter_(biology)" title="wikilink">promoters</a>, <a href="enhancer_(genetics)" title="wikilink">enhancers</a>, or <a href="silencer_(DNA)" title="wikilink">silencers</a>) that act as <a href="binding_site" title="wikilink">binding sites</a> for <a href="activator_(genetics)" title="wikilink">activators</a> or <a href="repressor" title="wikilink">repressors</a> for <a href="gene_transcription" title="wikilink">gene transcription</a>. Genes interact with each other either through their gene products (mRNA, proteins) which can regulate gene transcription, or through small <a href="RNA_species" title="wikilink">RNA species</a> that can directly regulate genes. These <a href="gene-gene_interactions" title="wikilink">gene-gene interactions</a>, together with genes' interactions with other substances in the cell, form the most basic interaction network: the <a href="gene_regulatory_network" title="wikilink">gene regulatory networks</a>. They perform information processing tasks within the cell, including the assembly and maintenance of other networks. Models of gene regulatory networks include random and probabilistic <a href="Boolean_network" title="wikilink">Boolean networks</a>, <a href="asynchronous_automata" title="wikilink">asynchronous automata</a>, and <a href="network_motif" title="wikilink">network motifs</a>.</p>

<p>Another viewpoint is that the entire genomic regulatory system is a computational system, a <em>genomic computer</em>. This interpretation allows one to compare human-made electronic computation with computation as it occurs in nature.<a class="footnoteRef" href="#fn34" id="fnref34"><sup>34</sup></a></p>
<table>
<tbody>
<tr class="odd">
<td style="text-align: left;">
<p>A comparison between genomic and electronic computers</p></td>
<td style="text-align: left;"></td>
<td style="text-align: left;">
<p>Genomic computer</p></td>
<td style="text-align: left;">
<p>Electronic computer</p></td>
</tr>
<tr class="even">
<td style="text-align: left;">
<p>Architecture</p></td>
<td style="text-align: left;">
<p>changeable</p></td>
<td style="text-align: left;">
<p>rigid</p></td>
</tr>
<tr class="odd">
<td style="text-align: left;">
<p>Components construction</p></td>
<td style="text-align: left;">
<p>as-needed basis</p></td>
<td style="text-align: left;">
<p>from the start</p></td>
</tr>
<tr class="even">
<td style="text-align: left;">
<p>Coordination</p></td>
<td style="text-align: left;">
<p>causal coordination</p></td>
<td style="text-align: left;">
<p>temporal synchrony</p></td>
</tr>
<tr class="odd">
<td style="text-align: left;">
<p>Distinction between hardware and software</p></td>
<td style="text-align: left;">
<p>No</p></td>
<td style="text-align: left;">
<p>Yes</p></td>
</tr>
<tr class="even">
<td style="text-align: left;">
<p>Transport media</p></td>
<td style="text-align: left;">
<p>molecules and ions</p></td>
<td style="text-align: left;">
<p>wires</p></td>
</tr>
</tbody>
</table>

<p>In addition, unlike a conventional computer, robustness in a genomic computer is achieved by various <a href="feedback_mechanism" title="wikilink">feedback mechanisms</a> by which poorly functional processes are rapidly degraded, poorly functional cells are killed by <a class="uri" href="apoptosis" title="wikilink">apoptosis</a>, and poorly functional organisms are out-competed by more fit species.</p>

<p><a href="Biochemical_networks" title="wikilink">Biochemical networks</a> refer to the interactions between proteins, and they perform various mechanical and metabolic tasks inside a cell. Two or more proteins may bind to each other via binding of their interactions sites, and form a dynamic protein complex (<a class="uri" href="complexation" title="wikilink">complexation</a>). These protein complexes may act as <a class="uri" href="catalysts" title="wikilink">catalysts</a> for other chemical reactions, or may chemically modify each other. Such modifications cause changes to available binding sites of proteins. There are tens of thousands of proteins in a cell, and they interact with each other. To describe such a massive scale interactions, <a href="Kohn_maps" title="wikilink">Kohn maps</a><a class="footnoteRef" href="#fn35" id="fnref35"><sup>35</sup></a> were introduced as a graphical notation to depict molecular interactions in succinct pictures. Other approaches to describing accurately and succinctly protein-protein interactions include the use of <a href="textual_bio-calculus" title="wikilink">textual bio-calculus</a><a class="footnoteRef" href="#fn36" id="fnref36"><sup>36</sup></a> or <a class="uri" href="pi-calculus" title="wikilink">pi-calculus</a> enriched with stochastic features.<a class="footnoteRef" href="#fn37" id="fnref37"><sup>37</sup></a></p>

<p><a href="Transport_network_(biology)" title="wikilink">Transport networks</a> refer to the separation and transport of substances mediated by lipid membranes. Some lipids can self-assemble into biological membranes. A lipid membrane consists of a <a href="lipid_bilayer" title="wikilink">lipid bilayer</a> in which proteins and other molecules are embedded, being able to travel along this layer. Through lipid bilayers, substances are transported between the inside and outside of membranes to interact with other molecules. Formalisms depicting transport networks include membrane systems and <a href="brane_calculi" title="wikilink">brane calculi</a>.<a class="footnoteRef" href="#fn38" id="fnref38"><sup>38</sup></a></p>
<h3 id="synthetic-biology">Synthetic biology</h3>
<dl>
<dd><em>Further information: <a href="synthetic_biology" title="wikilink">synthetic biology</a></em>
</dd>
</dl>

<p>Synthetic biology aims at engineering synthetic biological components, with the ultimate goal of assembling whole biological systems from their constituent components. The history of synthetic biology can be traced back to the 1960s, when <a href="Francois_Jacob" title="wikilink">Francois Jacob</a> and <a href="Jacques_Monod" title="wikilink">Jacques Monod</a> discovered the mathematical logic in gene regulation. Genetic engineering techniques, based on <a href="recombinant_DNA" title="wikilink">recombinant DNA</a> technology, are a precursor of today's synthetic biology which extends these techniques to entire systems of genes and gene products.</p>

<p>Along with the possibility of synthesizing longer and longer DNA strands, the prospect of creating synthetic genomes with the purpose of building entirely artificial <a href="synthetic_organism" title="wikilink">synthetic organisms</a> became a reality. Indeed, rapid assembly of chemically synthesized short DNA strands made it possible to generate a 5386bp synthetic genome of a virus.<a class="footnoteRef" href="#fn39" id="fnref39"><sup>39</sup></a></p>

<p>Alternatively, Smith et al. found about 100 genes that can be removed individually from the genome of <em><a href="Mycoplasma_Genitalium" title="wikilink">Mycoplasma Genitalium</a></em>. This discovery paves the way to the assembly of a minimal but still viable artificial genome consisting of the essential genes only.</p>

<p>A third approach to engineering semi-synthetic cells is the construction of a single type of RNA-like molecule with the ability of self-replication.<a class="footnoteRef" href="#fn40" id="fnref40"><sup>40</sup></a> Such a molecule could be obtained by guiding the rapid evolution of an initial population of RNA-like molecules, by selection for the desired traits.</p>

<p>Another effort in this field is towards engineering multi-cellular systems by designing, e.g., <a href="cell_signaling" title="wikilink">cell-to-cell communication modules</a> used to coordinate living bacterial cell populations.<a class="footnoteRef" href="#fn41" id="fnref41"><sup>41</sup></a></p>
<h3 id="cellular-computing">Cellular computing</h3>

<p>Computation in living cells (a.k.a. <a href="cellular_computing" title="wikilink">cellular computing</a>, or <a href="in-vivo_computing" title="wikilink">in-vivo computing</a>) is another approach to understand nature as computation. One particular study in this area is that of the computational nature of gene assembly in unicellular organisms called <a href="ciliate" title="wikilink">ciliates</a>. Ciliates store a copy of their DNA containing functional genes in the <a class="uri" href="macronucleus" title="wikilink">macronucleus</a>, and another "encrypted" copy in the <a class="uri" href="micronucleus" title="wikilink">micronucleus</a>. Conjugation of two ciliates consists of the exchange of their micronuclear genetic information, leading to the formation of two new micronuclei, followed by each ciliate re-assembling the information from its new micronucleus to construct a new functional macronucleus. The latter process is called <a href="gene_assembly" title="wikilink">gene assembly</a>, or gene re-arrangement. It involves re-ordering some fragments of DNA (<a href="permutation" title="wikilink">permutations</a> and possibly <a href="Chromosomal_inversion" title="wikilink">inversion</a>) and deleting other fragments from the micronuclear copy. From the computational point of view, the study of this gene assembly process led to many challenging research themes and results, such as the Turing universality of various models of this process.<a class="footnoteRef" href="#fn42" id="fnref42"><sup>42</sup></a> From the biological point of view, a plausible hypothesis about the "bioware" that implements the gene-assembly process was proposed, based on <a href="template_guided_recombination" title="wikilink">template guided recombination</a>.<a class="footnoteRef" href="#fn43" id="fnref43"><sup>43</sup></a><a class="footnoteRef" href="#fn44" id="fnref44"><sup>44</sup></a></p>

<p>Other approaches to cellular computing include developing an <em><a href="in_vivo" title="wikilink">in vivo</a></em> programmable and autonomous finite-state automaton with <em><a href="Escherichia_coli" title="wikilink">E. coli</a></em>,<a class="footnoteRef" href="#fn45" id="fnref45"><sup>45</sup></a> and designing and constructing <em>in vivo</em> cellular logic gates and genetic circuits that harness the cell's existing biochemical processes (see for example <a class="footnoteRef" href="#fn46" id="fnref46"><sup>46</sup></a><a class="footnoteRef" href="#fn47" id="fnref47"><sup>47</sup></a>).</p>
<h2 id="see-also">See also</h2>
<ul>
<li><a href="DNA_computing" title="wikilink">DNA computing</a></li>
<li><a href="Quantum_computing" title="wikilink">Quantum computing</a></li>
<li><a href="Synthetic_biology" title="wikilink">Synthetic biology</a></li>
<li><a href="Turing_machine" title="wikilink">Turing machine</a></li>
<li><a href="Natural_Computing_(journal)" title="wikilink">Natural Computing Journal</a></li>
</ul>
<h2 id="further-reading">Further reading</h2>

<p>This article was written based on the following references with the kind permission of their authors:</p>
<ul>
<li></li>
<li></li>
</ul>

<p>Many of the constituent research areas of natural computing have their own specialized journals and books series. Journals and book series dedicated to the broad field of Natural Computing include the journals <a href="http://www.igi-global.com/ijncr">International Journal of Natural Computing Research</a> (IGI Global),<a href="http://www.springer.com/computer/foundations/journal/11047">Natural Computing</a> (Springer Verlag), <a href="http://www.elsevier.com/wps/find/journaldescription.cws_home/505625/description#description">Theoretical Computer Science, Series C: Theory of Natural Computing</a> (Elsevier), <a href="http://www.springer.com/series/4190">the Natural Computing book series</a> (Springer Verlag), and the <a href="http://www.springer.com/computer/foundations/book/978-3-540-92911-6">Handbook of Natural Computing</a> (G.Rozenberg, T.Back, J.Kok, Editors, Springer Verlag).</p>
<ul>
<li></li>
<li><em>Swarms and Swarm Intelligence</em> by Michael G. Hinchey, Roy Sterritt, and Chris Rouff,</li>
</ul>
<h2 id="references">References</h2>

<p>"</p>

<p><a href="Category:Natural_computation" title="wikilink"> </a> <a href="Category:Theoretical_computer_science" title="wikilink">Category:Theoretical computer science</a></p>
<section class="footnotes">
<hr/>
<ol>
<li id="fn1">G.Rozenberg, T.Back, J.Kok, Editors, Handbook of Natural Computing, Springer Verlag, 2012<a href="#fnref1">↩</a></li>
<li id="fn2">Fredkin, F. Digital mechanics: An informational process based on reversible universal CA. Physica D 45 (1990) 254-270<a href="#fnref2">↩</a></li>
<li id="fn3">Zuse, K. Rechnender Raum. Elektronische Datenverarbeitung 8 (1967) 336-344<a href="#fnref3">↩</a></li>
<li id="fn4">Lloyd, S. Programming the Universe: A Quantum Computer Scientist Takes on the Cosmos. Knopf, 2006<a href="#fnref4">↩</a></li>
<li id="fn5">Yang, X.-S., Nature-inspired metaheuristic algorithms, Luniver Press, (2008).<a href="#fnref5">↩</a></li>
<li id="fn6">Olarius S., Zomaya A. Y., Handbook of Bioinspired Algorithms and Applications, Chapman &amp; Hall/CRC, 2005.<a href="#fnref6">↩</a></li>
<li id="fn7">de Castro, L. N., Fundamentals of Natural Computing: Basic Concepts, Algorithms, and Applications, CRC Press, 2006.<a href="#fnref7">↩</a></li>
<li id="fn8">von Neumann, J. The Computer and the Brain. Yale University Press, 1958<a href="#fnref8">↩</a></li>
<li id="fn9">Arbib, M., editor. The Handbook of Brain Theory and Neural Networks. MIT Press, 2003.<a href="#fnref9">↩</a></li>
<li id="fn10">Rojas, R. Neural Networks: A Systematic Introduction. Springer, 1996<a href="#fnref10">↩</a></li>
<li id="fn11">Bäck, T., Fogel, D., Michalewicz, Z., editors. Handbook of Evolutionary Computation. IOP Publishing, U.K., 1997<a href="#fnref11">↩</a></li>
<li id="fn12">Koza, J. <a href="http://www.ru.lv/~peter/zinatne/ebooks/MIT%20-%20Genetic%20Programming.pdf">Genetic Programming: On the Programming of Computers by Means of Natural Selection</a>. MIT Press, 1992<a href="#fnref12">↩</a></li>
<li id="fn13">Engelbrecht, A. Fundamentals of Computational Swarm Intelligence. Wiley and Sons, 2005.<a href="#fnref13">↩</a></li>
<li id="fn14">Dasgupta, D. editor. Artificial Immune Systems and Their Applications. Springer, 1998<a href="#fnref14">↩</a></li>
<li id="fn15">de Castro, L., Timmis, J. <a href="http://www.springer.com/computer/artificial/book/978-1-85233-594-6">Artificial Immune Systems: A New Computational Intelligence Approach</a>. Springer, 2002.<a href="#fnref15">↩</a></li>
<li id="fn16">Paun, G. Membrane Computing: An Introduction. Springer, 2002<a href="#fnref16">↩</a></li>
<li id="fn17">Abelson, H., Allen, D., Coore, D., Hanson, C., Homsy, G., Knight Jr., T., Nagpal, R., Rauch, E., Sussman, G., Weiss, R. <a href="http://portal.acm.org/citation.cfm?id=332842">Amorphous computing</a>. Communications of the ACM 43, 5 (May 2000), 74-82<a href="#fnref17">↩</a></li>
<li id="fn18">Langton, C., editor. Artificial Life. Addison-Wesley Longman, 1990<a href="#fnref18">↩</a></li>
<li id="fn19">Rozenberg, G. and Salomaa, A. The Mathematical Theory of L Systems. Academic Press, 1980<a href="#fnref19">↩</a></li>
<li id="fn20">Brooks. R. <a href="http://www.nature.com/doifinder/10.1038/35023200">Artificial life: from robot dreams to reality</a>. <em>Nature</em> 406 (2000), 945-947<a href="#fnref20">↩</a></li>
<li id="fn21">Lipson, P., Pollack, J. <a href="http://www.nature.com/nature/journal/v406/n6799/full/406974a0.html">Automatic design and manufacture of robotic lifeforms</a>. <em>Nature</em> 406 (2000), 974-978<a href="#fnref21">↩</a></li>
<li id="fn22">Adleman, L. <a href="http://www.usc.edu/dept/molecular-science/papers/fp-sci94.pdf">Molecular computation of solutions to combinatorial problems</a>. <em>Science</em> 266 (1994), 1021-1024<a href="#fnref22">↩</a></li>
<li id="fn23">Kari, L. <a href="http://www.csd.uwo.ca/~lila/intelh.pdf">DNA computing - the arrival of biological mathematics</a>. The Mathematical Intelligencer 19, 2 (1997) 9-22<a href="#fnref23">↩</a></li>
<li id="fn24">Reif, J. and LaBean, T. <a href="http://portal.acm.org/citation.cfm?id=1284647">Autonomous programmable biomolecular devices using self-assembled DNA nanostructures</a>. Communications of the ACM 50, 9 (Spet. 2007), 46-53<a href="#fnref24">↩</a></li>
<li id="fn25">Seeman, N. <a href="http://www.scientificamerican.com/article.cfm?id=nanotechnology-and-the-do">Nanotechnology and the double helix</a>. Scientific American Reports, 17. 3 (2007), 30-39<a href="#fnref25">↩</a></li>
<li id="fn26">Rothemund, P., Papadakis, N., Winfree, E. <a href="http://www.plosbiology.org/article/info:doi/10.1371/journal.pbio.0020424">Algorithmic self-assembly of DNA Sierpinski triangles</a>. <em>PLoS Biology</em> 2, 12 (December 2004)<a href="#fnref26">↩</a></li>
<li id="fn27">Rothemund, P. <a href="http://www.nature.com/nature/journal/v440/n7082/abs/nature04586.html">Folding DNA to create nanoscale shapes and patterns</a>. <em>Nature</em> 440 (2006) 297-302.<a href="#fnref27">↩</a></li>
<li id="fn28">Bath, J., Turberfield, A. <a href="http://www.nature.com/nnano/journal/v2/n5/full/nnano.2007.104.html">DNA nanomachines</a>. Nature Nanotechnology 2 (May 2007), 275-284<a href="#fnref28">↩</a></li>
<li id="fn29">Paun, G., Rozenberg, G., Salomaa, A. DNA Computing: New Computing Paradigms. Springer, 1998<a href="#fnref29">↩</a></li>
<li id="fn30">Hirvensalo, M. Quantum Computing, 2nd Ed. Springer, 2004<a href="#fnref30">↩</a></li>
<li id="fn31">Ursin, R. et al. <a href="http://www.nature.com/nphys/journal/v3/n7/abs/nphys629.html">Entanglemen-based quantum communication over 144km</a>. Nature Physics 3 (2007) 481-486<a href="#fnref31">↩</a></li>
<li id="fn32">Negrevergne, C. et al. [<a class="uri" href="http://scitation.aip.org/getabs/servlet/GetabsServlet?prog=normal&amp;id">http://scitation.aip.org/getabs/servlet/GetabsServlet?prog=normal&amp;id;</a>;=PRLTAO000096000017170501000001&amp;idtype;=cvips&amp;gifs;=yes Benchmarking quantum control methods on a 12-qubit system]. <em>Physical Review Letters</em> 96:art170501, 2006<a href="#fnref32">↩</a></li>
<li id="fn33">Cardelli, L. <a href="http://lucacardelli.name/Bibliography.htm">Abstract machines of systems biology</a> Bulletin of the EATCS 93 (2007), 176-204<a href="#fnref33">↩</a></li>
<li id="fn34">Istrail, S., De-Leon, B-T., Davidson, E. <a href="http://www.cs.brown.edu/research/pubs/pdfs/2007/Istrail-2007-RGC.pdf">The regulatory genome and the computer</a>. Developmental Biology 310 (2007), 187-195<a href="#fnref34">↩</a></li>
<li id="fn35">Kohn, K. <a href="http://www.molbiolcell.org/cgi/content/full/10/8/2703">Molecular interaction map of the mammalian cell cycle control and DNA repair systems</a>. Molecular Biology of the Cell 10(8) (1999) 2703-2734.<a href="#fnref35">↩</a></li>
<li id="fn36">Nagasaki, M., Onami, S., Miyano, S., Kitano, H. <a href="http://recomb2000.ims.u-tokyo.ac.jp/Posters/pdf/15.pdf">Bio-calculus: its concept and molecular interaction</a>. Genome Informatics 10 (1999) 133-143.<a href="#fnref36">↩</a></li>
<li id="fn37">Regev, A., Shapiro, E. <a href="http://www.nature.com/nature/journal/v419/n6905/full/419343a.html">Cellular abstractions: Cells as computation</a>. <em>Nature</em> 419 (2002) 343<a href="#fnref37">↩</a></li>
<li id="fn38">Cardelli, L. <a href="http://lucacardelli.name/Papers/Brane%20Calculi.pdf">Brane calculi: Interactions of biological membranes</a>. In LNCS 3082, pages 257-280. Springer, 2005.<a href="#fnref38">↩</a></li>
<li id="fn39">Smith, H., Hutchison III, C., Pfannkoch, C., and Venter, C. <a href="http://www.pnas.org/content/100/26/15440.full">Generating a synthetic genome by whole genome assembly: {phi}X174 bacteriophage from synthetic oligonucleotides</a>. <em>PNAS 100</em>, 26 (2003), 15440-15445.<a href="#fnref39">↩</a></li>
<li id="fn40">Sazani, P., Larralde, R., Szostak, J. <a href="http://genetics.mgh.harvard.edu/szostakweb/publications/Szostak_pdfs/Sazani_et_al_2004_JACS_SI.pdf">A small aptamer with strong and specific recognition of the triphosphate of ATP</a>. <em>Journal of the American Chemical Society</em>, 126(27) (2004) 8370-8371<a href="#fnref40">↩</a></li>
<li id="fn41">Weiss, R., Knight, Jr., T. <a href="http://www.springerlink.com/content/tavjkk4hymen6glj/">Engineered communications for microbial robotics</a>. In LNCS 2054, pages 1-16, Springer, 2001<a href="#fnref41">↩</a></li>
<li id="fn42">Landweber, L. and Kari, L. <a href="http://scholarsportal.info/pdflinks/08110515452920340.pdf">The evolution of cellular computing: Nature's solution to a computational problem</a>. <em>Biosystems</em>, 52, 1/3 (1999) 3-13.<a href="#fnref42">↩</a></li>
<li id="fn43"><a href="#fnref43">↩</a></li>
<li id="fn44">Prescott, D., Ehrenfeucht, A., and Rozenberg, G. [<a href="http://www.sciencedirect.com/science?_ob=ArticleURL&amp;_udi=B6WMD-484VCX1-1&amp;_user=940030&amp;_coverDate=06%2F07%2F2003&amp;_rdoc=6&amp;_fmt=high&amp;_orig=browse&amp;_srch=doc-info(%23toc%236932%232003%23997779996%23424670%23FLA%23display%23Volume)&amp;_cdi=6932&amp;_sort=d&amp;_docanchor">http://www.sciencedirect.com/science?_ob=ArticleURL&amp;_udi=B6WMD-484VCX1-1&amp;_user=940030&amp;_coverDate=06%2F07%2F2003&amp;_rdoc=6&amp;_fmt=high&amp;_orig=browse&amp;_srch=doc-info(%23toc%236932%232003%23997779996%23424670%23FLA%23display%23Volume)&amp;_cdi=6932&amp;_sort=d&amp;_docanchor</a>=&amp;_ct=13&amp;_acct=C000048763&amp;_version=1&amp;_urlVersion=0&amp;_userid=940030&amp;md5;=36f664865c20a4521bb61f931a19449b Template-guided recombination for IES elimination and unscrambling of genes in stichotrichous ciliates]. <em>J. Theoretical Biology</em> 222, 3 (2003), 323-330.<a href="#fnref44">↩</a></li>
<li id="fn45">Nakagawa, H., Sakamoto, K., Sakakibara, Y. <a href="http://www.springerlink.com/content/x3j1x0573681g1t4/">Development of an <em>in vivo</em> computer based on Escherichia Coli</a>. In LNCS 3892, pages 203-212, Springer, 2006<a href="#fnref45">↩</a></li>
<li id="fn46">Zabet NR, Hone ANW, Chu DF [<a class="uri" href="http://mitpress.mit.edu/catalog/item/default.asp?ttype=2&amp;tid">http://mitpress.mit.edu/catalog/item/default.asp?ttype=2&amp;tid;</a>;=12433 Design principles of transcriptional logic circuits]. In Artificial Life XII Proceedings of the Twelfth International Conference on the Synthesis and Simulation of Living Systems, pages 186-193. MIT Press, August 2010.<a href="#fnref46">↩</a></li>
<li id="fn47">Zabet NR <a href="http://www.cs.kent.ac.uk/pubs/2010/3098/index.html">Towards Modular, Scalable and Optimal Design of Transcriptional Logic Systems</a>.2010.<a href="#fnref47">↩</a></li>
</ol>
</section>
</body>
</html>
