<html lang="en">
<head>
<meta charset="utf-8"/>
<title offset="103">Neofunctionalization</title>
<script src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_SVG.js" type="text/javascript">
</script>
</head>
<body>
<h1>Neofunctionalization</h1>
<hr/>

<p> <strong>Neofunctionalization</strong>, one of the possible outcomes of <a href="functional_divergence" title="wikilink">functional divergence</a>, occurs when one gene copy, or <a class="uri" href="paralog" title="wikilink">paralog</a>, takes on a totally new function after a <a href="gene_duplication" title="wikilink">gene duplication</a> event. Neofunctionalization is an adaptive mutation process; meaning one of the gene copies must mutate to develop a function that was not present in the ancestral gene.<a class="footnoteRef" href="#fn1" id="fnref1"><sup>1</sup></a><a class="footnoteRef" href="#fn2" id="fnref2"><sup>2</sup></a><a class="footnoteRef" href="#fn3" id="fnref3"><sup>3</sup></a> In other words, one of the duplicates retains its original function, while the other accumulates molecular changes such that, in time, it can perform a different task.<a class="footnoteRef" href="#fn4" id="fnref4"><sup>4</sup></a> This process is thought to be free of selective pressure because one gene copy can mutate without adversely affecting the fitness of the organism since ancestral function is retained in the other copy.<a class="footnoteRef" href="#fn5" id="fnref5"><sup>5</sup></a><a class="footnoteRef" href="#fn6" id="fnref6"><sup>6</sup></a><a class="footnoteRef" href="#fn7" id="fnref7"><sup>7</sup></a><a class="footnoteRef" href="#fn8" id="fnref8"><sup>8</sup></a></p>
<h2 id="the-process">The process</h2>

<p>The process of Neofunctionalization begins with a gene duplication event, which is thought to occur as a defense mechanism against the accumulation of deleterious mutations.<a class="footnoteRef" href="#fn9" id="fnref9"><sup>9</sup></a><a class="footnoteRef" href="#fn10" id="fnref10"><sup>10</sup></a><a class="footnoteRef" href="#fn11" id="fnref11"><sup>11</sup></a> Following the gene duplication event there are two identical copies of the ancestral gene performing exactly the same function. This redundancy allows one the copies to take on a new function. In the event that the new function is advantageous, natural selection positively selects for it and the new mutation becomes fixed in the population.<a class="footnoteRef" href="#fn12" id="fnref12"><sup>12</sup></a><a class="footnoteRef" href="#fn13" id="fnref13"><sup>13</sup></a> The occurrence of Neofunctionalization can most often be attributed to changes in the coding region or changes in the regulatory elements of a gene.<a class="footnoteRef" href="#fn14" id="fnref14"><sup>14</sup></a> It is much more rare to see major changes in protein function, such as subunit structure or substrate and ligand affinity, as a result of Neofunctionalization.<a class="footnoteRef" href="#fn15" id="fnref15"><sup>15</sup></a></p>
<h2 id="selective-constraints">Selective Constraints</h2>

<p>Neofunctionalization is also commonly referred to as "mutation during non-functionality” or “mutation during redundancy”.<a class="footnoteRef" href="#fn16" id="fnref16"><sup>16</sup></a> Regardless of if the mutation arises after non-functionality of a gene or due to redundant gene copies, the important aspect is that in both scenarios one copy of the duplicated gene is freed from selective constraints and by chance acquires a new function which is then improved by natural selection.<a class="footnoteRef" href="#fn17" id="fnref17"><sup>17</sup></a> This process is thought to occur very rarely in evolution for two major reasons. The first reason is that functional changes typically require a large number of amino acid changes; which has a low probability of occurrence. Secondly, because deleterious mutations occur much more frequently than advantageous mutations in evolution.<a class="footnoteRef" href="#fn18" id="fnref18"><sup>18</sup></a> This makes the likelihood that gene function is lost over time (i.e. pseudogenization) far greater than the likelihood of the emergence of a new gene function.<a class="footnoteRef" href="#fn19" id="fnref19"><sup>19</sup></a> Walsh discovered that the relative probability of Neofunctionalization is determined by the selective advantage and the relative rate of advantageous mutations.<a class="footnoteRef" href="#fn20" id="fnref20"><sup>20</sup></a> This was proven in his derivation of the relative probability of Neofunctionalization to pseudogenization, which is given by

<math display="block" id="Neofunctionalization:0">
 <semantics>
  <mfrac>
   <mrow>
    <mrow>
     <mi>ρ</mi>
     <mi>S</mi>
    </mrow>
    <mo>-</mo>
    <mn>1</mn>
   </mrow>
   <mrow>
    <mn>1</mn>
    <mo>-</mo>
    <msup>
     <mi>e</mi>
     <mi>s</mi>
    </msup>
   </mrow>
  </mfrac>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <divide></divide>
    <apply>
     <minus></minus>
     <apply>
      <times></times>
      <ci>ρ</ci>
      <ci>S</ci>
     </apply>
     <cn type="integer">1</cn>
    </apply>
    <apply>
     <minus></minus>
     <cn type="integer">1</cn>
     <apply>
      <csymbol cd="ambiguous">superscript</csymbol>
      <ci>e</ci>
      <ci>s</ci>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \frac{\rho\,\!S-1}{1-e^{s}}
  </annotation>
 </semantics>
</math>

 where ρ is the ratio of advantageous mutation rate to null mutation rate and S is the population selection 4NeS (Ne: effective population size S: selection intensity).<a class="footnoteRef" href="#fn21" id="fnref21"><sup>21</sup></a></p>
<h2 id="classical-model">Classical Model</h2>

<p>In 1935, originally proposed Neofunctionalization as a possible outcome of a gene duplication event. In 1970, Ohno suggested that Neofunctionalization was the only evolutionary mechanism that gave rise to new gene functions in a population.<a class="footnoteRef" href="#fn22" id="fnref22"><sup>22</sup></a> He also believed that Neofunctionalization was the only alternative to pseudogenization.<a class="footnoteRef" href="#fn23" id="fnref23"><sup>23</sup></a> Ohta (1987) was among the first to suggest that other mechanisms may exist for the preservation of duplicated genes in the population.<a class="footnoteRef" href="#fn24" id="fnref24"><sup>24</sup></a> Today, subfunctionalization is a widely excepted alternative fixation process for gene duplicates in the population and is currently the only other possible outcome of functional divergence.<a class="footnoteRef" href="#fn25" id="fnref25"><sup>25</sup></a></p>
<h2 id="neosubfunctionalization">Neosubfunctionalization</h2>

<p>Neosubfunctionalization occurs when Neofunctionalization is the end result of <a class="uri" href="subfunctionalization" title="wikilink">subfunctionalization</a>. In other words, once a gene duplication event occurs forming parologs that after an evolutionary period subfunctionalize, one gene copy continues on this evolutionary journey and accumulates mutations that give rise to a new function.<a class="footnoteRef" href="#fn26" id="fnref26"><sup>26</sup></a><a class="footnoteRef" href="#fn27" id="fnref27"><sup>27</sup></a> Some believe that Neofunctionalization is the end stage for all subfunctionalized genes. For instance, according to Rastogi and Liberles “Neofunctionalization is the terminal fate of all duplicate gene copies retained in the genome and subfuctionlization merely exist as a transient sate to preserve the duplicate gene copy.”<a class="footnoteRef" href="#fn28" id="fnref28"><sup>28</sup></a> The results of their study become punctuated as population size increases.</p>
<h2 id="antifreeze-protein">Antifreeze Protein</h2>

<p>The evolution of the antifreeze protein in the Antarctic zoarcid fish provides a prime example of Neofunctionalization after gene duplication. In the case of the Antarctic zoarcid fish type III antifreeze protein gene diverged from a parologus copy of sialic acid synthase (SAS) gene.<a class="footnoteRef" href="#fn29" id="fnref29"><sup>29</sup></a> The ancestral SAS gene was found to have both sialic acid synthase and rudimentary ice-binding functionalities. After duplication one of the paralogs began to accumulate mutations that lead to the replacement of SAS domains of the gene allowing for further development and optimization of the antifreeze functionality.<a class="footnoteRef" href="#fn30" id="fnref30"><sup>30</sup></a> The new gene is now capable of noncolligative freezing-point depression, and thus is neofunctionalized.<a class="footnoteRef" href="#fn31" id="fnref31"><sup>31</sup></a> This specialization allows Antarctic zoarcid fish to survive in the frigid temperatures of the Antarctic Seas.</p>
<h2 id="model-limitations">Model Limitations</h2>

<p>Limitations exist in Neofunctionalization as model for functional divergence primarily because: (1) the amount of nucleotide changes giving rise to a new function must be very minimal; making the probability for <a href="Pseudogene" title="wikilink">pseudogenization</a> much higher than neofunctionalization after a gene duplication event.<a class="footnoteRef" href="#fn32" id="fnref32"><sup>32</sup></a> (2) After a gene duplication event both copies may be subjected to selective pressure equivalent to that constraining the ancestral gene; meaning that neither copy is available for Neofunctionalization.<a class="footnoteRef" href="#fn33" id="fnref33"><sup>33</sup></a> (3) In many cases positive Darwinian selection presents a more parsimonious explanation for the divergence of multigene families.<a class="footnoteRef" href="#fn34" id="fnref34"><sup>34</sup></a></p>
<h2 id="see-also">See also</h2>
<ul>
<li><a class="uri" href="Subfunctionalization" title="wikilink">Subfunctionalization</a></li>
</ul>
<h2 id="references">References</h2>
<references>
</references>

<p>"</p>

<p><a class="uri" href="Category:Genetics" title="wikilink">Category:Genetics</a></p>
<section class="footnotes">
<hr/>
<ol>
<li id="fn1"><a href="#fnref1">↩</a></li>
<li id="fn2">S. Rastogi and D. A. Liberles, “Subfunctionalization of duplicated genes as a transition state to neofunctionalization,” BMC Evolutionary Biology, vol. 5, no. 1, p. 28, 2005<a href="#fnref2">↩</a></li>
<li id="fn3">B. Conrad and S. E. Antonarakis, “Gene duplication: a drive for phenotypic diversity and cause of human disease.,” Annual review of genomics and human genetics, vol. 8, pp. 17-35, Jan. 2007<a href="#fnref3">↩</a></li>
<li id="fn4">S. Ohno, Evolution by Gene Duplication. New York, Heidelberg, Berlin: Springer-Verlag, 1970, pp. 59-87<a href="#fnref4">↩</a></li>
<li id="fn5">M. Sémon and K. H. Wolfe, “Preferential subfunctionalization of slow-evolving genes after allopolyploidization in Xenopus laevis.,” Proceedings of the National Academy of Sciences of the United States of America, vol. 105, no. 24, pp. 8333-8, Jun. 2008<a href="#fnref5">↩</a></li>
<li id="fn6">R. De Smet and Y. Van de Peer, “Redundancy and rewiring of genetic networks following genome-wide duplication events.,” Current opinion in plant biology, pp. 1-9, Feb. 2012<a href="#fnref6">↩</a></li>
<li id="fn7">J. G. Ruby, A. Stark, W. K. Johnston, M. Kellis, D. P. Bartel, and E. C. Lai, “Evolution, biogenesis, expression, and target predictions of a substantially expanded set of Drosophila microRNAs.,” Genome research, vol. 17, no. 12, pp. 1850-64, Dec. 2007<a href="#fnref7">↩</a></li>
<li id="fn8">D. (University of H. Graur and W.-H. (University of C. Li, Fundamentals of Molecular Evolution, Second. Sinauer Associates, Inc.,, 2000<a href="#fnref8">↩</a></li>
<li id="fn9">G. D. Amoutzias, Y. He, J. Gordon, D. Mossialos, S. G. Oliver, and Y. Van de Peer, “Posttranslational regulation impacts the fate of duplicated genes.,” Proceedings of the National Academy of Sciences of the United States of America, vol. 107, no. 7, pp. 2967-71, Feb. 2010<a href="#fnref9">↩</a></li>
<li id="fn10">R. De Smet and Y. Van de Peer, “Redundancy and rewiring of genetic networks following genome-wide duplication events.,” Current opinion in plant biology, pp. 1-9, Feb. 2012<a href="#fnref10">↩</a></li>
<li id="fn11">D. (University of H. Graur and W.-H. (University of C. Li, Fundamentals of Molecular Evolution, Second. Sinauer Associates, Inc.,, 2000<a href="#fnref11">↩</a></li>
<li id="fn12">B. Conrad and S. E. Antonarakis, “Gene duplication: a drive for phenotypic diversity and cause of human disease.,” Annual review of genomics and human genetics, vol. 8, pp. 17-35, Jan. 2007<a href="#fnref12">↩</a></li>
<li id="fn13">H. Innan, “Population genetic models of duplicated genes.,” Genetica, vol. 137, no. 1, pp. 19-37, Sep. 2009<a href="#fnref13">↩</a></li>
<li id="fn14">D. (University of H. Graur and W.-H. (University of C. Li, Fundamentals of Molecular Evolution, Second. Sinauer Associates, Inc.,, 2000<a href="#fnref14">↩</a></li>
<li id="fn15">D. (University of H. Graur and W.-H. (University of C. Li, Fundamentals of Molecular Evolution, Second. Sinauer Associates, Inc.,, 2000<a href="#fnref15">↩</a></li>
<li id="fn16">A. Hughes, Adaptive Evolution of Genes and Genomes. New York: Oxford University Press, 1999<a href="#fnref16">↩</a></li>
<li id="fn17">D. (University of H. Graur and W.-H. (University of C. Li, Fundamentals of Molecular Evolution, Second. Sinauer Associates, Inc.,, 2000<a href="#fnref17">↩</a></li>
<li id="fn18">D. (University of H. Graur and W.-H. (University of C. Li, Fundamentals of Molecular Evolution, Second. Sinauer Associates, Inc.,, 2000<a href="#fnref18">↩</a></li>
<li id="fn19">H. Innan, “Population genetic models of duplicated genes.,” Genetica, vol. 137, no. 1, pp. 19-37, Sep. 2009<a href="#fnref19">↩</a></li>
<li id="fn20">M. Lynch and a Force, “The probability of duplicate gene preservation by subfunctionalization.,” Genetics, vol. 154, no. 1, pp. 459-73, Jan. 2000<a href="#fnref20">↩</a></li>
<li id="fn21">M. Lynch and a Force, “The probability of duplicate gene preservation by subfunctionalization.,” Genetics, vol. 154, no. 1, pp. 459-73, Jan. 2000<a href="#fnref21">↩</a></li>
<li id="fn22">[8D. (University of H. Graur and W.-H. (University of C. Li, Fundamentals of Molecular Evolution, Second. Sinauer Associates, Inc.,, 2000<a href="#fnref22">↩</a></li>
<li id="fn23">S. Rastogi and D. A. Liberles, “Subfunctionalization of duplicated genes as a transition state to neofunctionalization,” BMC Evolutionary Biology, vol. 5, no. 1, p. 28, 2005<a href="#fnref23">↩</a></li>
<li id="fn24">D. (University of H. Graur and W.-H. (University of C. Li, Fundamentals of Molecular Evolution, Second. Sinauer Associates, Inc.,, 2000<a href="#fnref24">↩</a></li>
<li id="fn25">S. Rastogi and D. A. Liberles, “Subfunctionalization of duplicated genes as a transition state to neofunctionalization,” BMC Evolutionary Biology, vol. 5, no. 1, p. 28, 2005<a href="#fnref25">↩</a></li>
<li id="fn26">X. He and J. Zhang, “Rapid subfunctionalization accompanied by prolonged and substantial neofunctionalization in duplicate gene evolution.,” Genetics, vol. 169, no. 2, pp. 1157-1164, 2005<a href="#fnref26">↩</a></li>
<li id="fn27">D. (University of H. Graur and W.-H. (University of C. Li, Fundamentals of Molecular Evolution, Second. Sinauer Associates, Inc.,, 2000<a href="#fnref27">↩</a></li>
<li id="fn28">S. Rastogi and D. A. Liberles, “Subfunctionalization of duplicated genes as a transition state to neofunctionalization,” BMC Evolutionary Biology, vol. 5, no. 1, p. 28, 2005<a href="#fnref28">↩</a></li>
<li id="fn29">C. Deng, C.-H. C. Cheng, H. Ye, X. He, and L. Chen, “Evolution of an antifreeze protein by neofunctionalization under escape from adaptive conflict.,” Proceedings of the National Academy of Sciences of the United States of America, vol. 107, no. 50, pp. 21593-8, Dec. 2010<a href="#fnref29">↩</a></li>
<li id="fn30">C. Deng, C.-H. C. Cheng, H. Ye, X. He, and L. Chen, “Evolution of an antifreeze protein by neofunctionalization under escape from adaptive conflict.,” Proceedings of the National Academy of Sciences of the United States of America, vol. 107, no. 50, pp. 21593-8, Dec. 2010<a href="#fnref30">↩</a></li>
<li id="fn31">C. Deng, C.-H. C. Cheng, H. Ye, X. He, and L. Chen, “Evolution of an antifreeze protein by neofunctionalization under escape from adaptive conflict.,” Proceedings of the National Academy of Sciences of the United States of America, vol. 107, no. 50, pp. 21593-8, Dec. 2010<a href="#fnref31">↩</a></li>
<li id="fn32">D. (University of H. Graur and W.-H. (University of C. Li, Fundamentals of Molecular Evolution, Second. Sinauer Associates, Inc.,, 2000<a href="#fnref32">↩</a></li>
<li id="fn33">D. (University of H. Graur and W.-H. (University of C. Li, Fundamentals of Molecular Evolution, Second. Sinauer Associates, Inc.,, 2000<a href="#fnref33">↩</a></li>
<li id="fn34">D. (University of H. Graur and W.-H. (University of C. Li, Fundamentals of Molecular Evolution, Second. Sinauer Associates, Inc.,, 2000<a href="#fnref34">↩</a></li>
</ol>
</section>
</body>
</html>
