<html lang="en">
<head>
<meta charset="utf-8"/>
<title offset="1087">Phosphopentose epimerase</title>
<script src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_SVG.js" type="text/javascript">
</script>
</head>
<body>
<h1>Phosphopentose epimerase</h1>
<hr/>

<p><strong>Phosphopentose epimerase</strong> (also known as <strong>ribulose-phosphate 3-epimerase</strong> and <strong>ribulose 5-phosphate 3-epimerase</strong>)() is a <a class="uri" href="metalloprotein" title="wikilink">metalloprotein</a> that catalyzes the interconversion between D-ribulose 5-phosphate and D-xylulose 5-phosphate.<a class="footnoteRef" href="#fn1" id="fnref1"><sup>1</sup></a></p>
<dl>
<dd>D-ribulose 5-phosphate 

<math display="inline" id="Phosphopentose_epimerase:0">
 <semantics>
  <mo>⇌</mo>
  <annotation-xml encoding="MathML-Content">
   <ci>normal-⇌</ci>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \rightleftharpoons
  </annotation>
 </semantics>
</math>

 D-xylulose 5-phosphate
</dd>
</dl>

<p>This reversible conversion is required for <a href="carbon_fixation" title="wikilink">carbon fixation</a> in plants – through the <a href="Calvin_cycle" title="wikilink">Calvin cycle</a> – and for the nonoxidative phase of the <a href="pentose_phosphate_pathway" title="wikilink">pentose phosphate pathway</a>.<a class="footnoteRef" href="#fn2" id="fnref2"><sup>2</sup></a><a class="footnoteRef" href="#fn3" id="fnref3"><sup>3</sup></a> This enzyme has also been implicated in additional <a class="uri" href="pentose" title="wikilink">pentose</a> and glucuronate interconversions.</p>

<p>In <em><a href="Alcaligenes_eutrophus" title="wikilink">Alcaligenes eutrophus</a></em> two copies of the gene coding for PPE are known,<a class="footnoteRef" href="#fn4" id="fnref4"><sup>4</sup></a> one is chromosomally encoded , the other one is on a plasmid . PPE has been found in a wide range of bacteria, archaebacteria, fungi and plants. All the proteins have from 209 to 241 amino acid residues. The enzyme has a <a href="TIM_barrel" title="wikilink">TIM barrel</a> structure.</p>

<p>The systematic name of this enzyme class is <strong>D-ribulose-5-phosphate 3-epimerase</strong>. Other names in common use include <strong>phosphoribulose epimerase</strong>, <strong>erythrose-4-phosphate isomerase</strong>, <strong>phosphoketopentose 3-epimerase</strong>, <strong>xylulose phosphate 3-epimerase</strong>, <strong>phosphoketopentose epimerase</strong>, <strong>ribulose 5-phosphate 3-epimerase</strong>, <strong>D-ribulose phosphate-3-epimerase</strong>, <strong>D-ribulose 5-phosphate epimerase</strong>, <strong>D-ribulose-5-P 3-epimerase</strong>, <strong>D-xylulose-5-phosphate 3-epimerase</strong>, and <strong>pentose-5-phosphate 3-epimerase</strong>.</p>

<p>This enzyme participates in 3 <a href="metabolism" title="wikilink">metabolic pathways</a>: <a href="pentose_phosphate_pathway" title="wikilink">pentose phosphate pathway</a>, <a href="pentose_and_glucuronate_interconversions" title="wikilink">pentose and glucuronate interconversions</a>, and <a href="carbon_fixation" title="wikilink">carbon fixation</a>.</p>

<p>The human protein containing this domain is the <a href="RPE_(gene)" title="wikilink">RPE (gene)</a>.</p>
<h2 id="family">Family</h2>

<p>Phosphopentose epimerase belongs to two protein families of increasing hierarchy. This enzyme belongs to the <a class="uri" href="isomerase" title="wikilink">isomerase</a> family, specifically those <a href="racemase" title="wikilink">racemases</a> and <a href="epimerase" title="wikilink">epimerases</a> which act on <a class="uri" href="carbohydrates" title="wikilink">carbohydrates</a> and their derivatives.<a class="footnoteRef" href="#fn5" id="fnref5"><sup>5</sup></a> In addition, the <a href="Structural_Classification_of_Proteins_database" title="wikilink">Structural Classification of Proteins database</a> has defined the “ribulose phosphate binding” superfamily for which this <a class="uri" href="epimerase" title="wikilink">epimerase</a> is a member.<a class="footnoteRef" href="#fn6" id="fnref6"><sup>6</sup></a> Other proteins included in this superfamily are 5‘-monophosphate decarboxylase (OMPDC), and 3-keto-l-gulonate 6-phosphate decarboxylase (KGPDC).</p>
<h2 id="structure">Structure</h2>

<p>As of late 2007, 4 <a href="tertiary_structure" title="wikilink">structures</a> have been solved for this class of enzymes, with <a href="Protein_Data_Bank" title="wikilink">PDB</a> accession codes , , , and .</p>
<h3 id="overall">Overall</h3>

<p>Crystallographic studies have helped elucidate the <a class="uri" href="apoenzyme" title="wikilink">apoenzyme</a> structure of phosphopentose epimerase. Results of these studies have shown that this enzyme exists as a <a class="uri" href="homodimer" title="wikilink">homodimer</a> in solution.<a class="footnoteRef" href="#fn7" id="fnref7"><sup>7</sup></a><a class="footnoteRef" href="#fn8" id="fnref8"><sup>8</sup></a> Furthermore, Phosphopentose epimerase folds into a (β/α)8 <a href="triosephosphate_isomerase" title="wikilink">triosephosphate isomerase</a> (TIM) barrel that includes loops.<a class="footnoteRef" href="#fn9" id="fnref9"><sup>9</sup></a> The core barrel is composed of 8 parallel strands that make up the central <a href="beta_sheet" title="wikilink">beta sheet</a>, with <a class="uri" href="helices" title="wikilink">helices</a> located in between consecutive strands. The loops in this structure have been known to regulate <a href="substrate_(biology)" title="wikilink">substrate</a> specificities. Specifically, the loop that connects <a class="uri" href="helix" title="wikilink">helix</a> α6 with strand β6 caps the <a href="active_site" title="wikilink">active site</a> upon binding of the substrate.<a class="footnoteRef" href="#fn10" id="fnref10"><sup>10</sup></a></p>

<p>As previously mentioned, Phosphopentose epimerase is a metalloenzyme. It requires a <a href="cofactor_(biochemistry)" title="wikilink">cofactor</a> for functionality and binds one <a class="uri" href="divalent" title="wikilink">divalent</a> metal <a class="uri" href="cation" title="wikilink">cation</a> per subunit.<a class="footnoteRef" href="#fn11" id="fnref11"><sup>11</sup></a> This enzyme has been shown to use Zn2+ predominantly for <a class="uri" href="catalysis" title="wikilink">catalysis</a>, along with Co2+ and Mn2+.<a class="footnoteRef" href="#fn12" id="fnref12"><sup>12</sup></a> However, human phosphopentose epimerase – which is encoded by the <a href="RPE_(gene)" title="wikilink">RPE</a> gene - differs in that it binds Fe2+ predominantly in catalysis. Fe2+ is octahedrally coordinated and stabilizes the 2,3-enediolate reaction intermediate observed in the figure.<a class="footnoteRef" href="#fn13" id="fnref13"><sup>13</sup></a></p>
<h3 id="active-site">Active site</h3>

<p>The β6/α6 loop region interacts with the substrate and regulates access to the active site. Phe147, Gly148, and Ala149 of this region cap the active site once binding has occurred. In addition, the Fe2+ ion is coordinated to His35, His70, Asp37, Asp175, and oxygens O2 and O3 of the substrate.<a class="footnoteRef" href="#fn14" id="fnref14"><sup>14</sup></a> The binding of substrate <a class="uri" href="atoms" title="wikilink">atoms</a> to the <a class="uri" href="iron" title="wikilink">iron</a> <a class="uri" href="cation" title="wikilink">cation</a> helps stabilize the complex during catalysis. <a class="uri" href="Mutagenesis" title="wikilink">Mutagenesis</a> studies have also indicated that two aspartic <a class="uri" href="acids" title="wikilink">acids</a> are located within the active site and help mediate catalysis through a 1,1-proton transfer reaction.<a class="footnoteRef" href="#fn15" id="fnref15"><sup>15</sup></a> The aspartic acids are the acid/base catalysts. Lastly, once the <a class="uri" href="ligand" title="wikilink">ligand</a> is attached to the active site, a series of methionines (Met39, Met72, and Met141) restrict further movement through constriction.<a class="footnoteRef" href="#fn16" id="fnref16"><sup>16</sup></a></p>
<h2 id="mechanism">Mechanism</h2>
<figure><b>(Figure)</b>
<embed src="Phosphopentose Epimerase Mechanism.pdf" title="This is a mechanism by which phosphopentose epimerase converts ribulose 5-phosphate to xylulose 5-phosphate. The intermediate in 2,3-trans-enediolate."></embed><figcaption>This is a mechanism by which phosphopentose epimerase converts ribulose 5-phosphate to xylulose 5-phosphate. The intermediate in 2,3-trans-enediolate.</figcaption>
</figure>

<p>Phosphopentose utilizes an acid/base type of catalytic mechanism. The reaction proceeds in such a way that trans-2,3-enediol phosphate is the intermediate.<a class="footnoteRef" href="#fn17" id="fnref17"><sup>17</sup></a><a class="footnoteRef" href="#fn18" id="fnref18"><sup>18</sup></a> The two aspartic acids mentioned above act as proton donors and acceptors. Asp37 and Asp175 are both <a class="uri" href="hydrogen" title="wikilink">hydrogen</a> bonded to the iron cation in the active site.<a class="footnoteRef" href="#fn19" id="fnref19"><sup>19</sup></a> When Asp37 is deprotonated, it attacks a proton on the third carbon of D-ribulose 5-phosphate, which forms the intermediate.<a class="footnoteRef" href="#fn20" id="fnref20"><sup>20</sup></a> In a concerted step, as Asp37 grabs a proton, the <a class="uri" href="carbonyl" title="wikilink">carbonyl</a> bond on the substrate grabs a second proton from Asp175 to form a <a class="uri" href="hydroxyl" title="wikilink">hydroxyl</a> group. The iron complex helps stabilize any additional charges. It is C3 of D-ribulose 5-phosphate which undergoes this epimerization, forming D-xylulose 5-phosphate.<a class="footnoteRef" href="#fn21" id="fnref21"><sup>21</sup></a> The mechanism is clearly demonstrated in the figure.</p>
<h2 id="biological-function">Biological function</h2>
<h3 id="calvin-cycle">Calvin Cycle</h3>

<p><a href="Electron_microscopy" title="wikilink">Electron microscopy</a> experiments in plants have shown that phosphopentose epimerase localizes to the <a class="uri" href="thykaloid" title="wikilink">thykaloid</a> membrane of <a class="uri" href="chloroplasts" title="wikilink">chloroplasts</a>.<a class="footnoteRef" href="#fn22" id="fnref22"><sup>22</sup></a> This epimerase participates in the third phase of the <a href="Calvin_cycle" title="wikilink">Calvin cycle</a>, which involves the regeneration of <a href="ribulose_1,5-bisphosphate" title="wikilink">ribulose 1,5-bisphosphate</a>. RuBP is the acceptor of the <a class="uri" href="CO2" title="wikilink">CO2</a> in the first step of the pathway, which suggests that phosphopentose epimerase regulates flux through the Calvin cycle. Without the regeneration of ribulose 1,5-bisphosphate, the cycle will be unable to continue. Therefore, <a href="xylulose_5-phosphate" title="wikilink">xylulose 5-phosphate</a> is reversibly converted into <a href="ribulose_5-phosphate" title="wikilink">ribulose 5-phosphate</a> by this epimerase. Subsequently, phosphoribulose kinase converts ribulose 5-phosphate into ribulose 1,5-bisphosphate.<a class="footnoteRef" href="#fn23" id="fnref23"><sup>23</sup></a></p>
<h3 id="pentose-phosphate-pathway">Pentose Phosphate Pathway</h3>

<p>The reactions of the <a href="pentose_phosphate_pathway" title="wikilink">pentose phosphate pathway</a> (PPP) take place in the <a class="uri" href="cytoplasm" title="wikilink">cytoplasm</a>. Phosphopentose epimerase specifically affects the nonoxidative portion of the pathway, which involves the production of various sugars and precursors.<a class="footnoteRef" href="#fn24" id="fnref24"><sup>24</sup></a> This enzyme converts <a href="ribulose_5-phosphate" title="wikilink">ribulose 5-phosphate</a> into the appropriate epimer for the <a class="uri" href="transketolase" title="wikilink">transketolase</a> reaction, <a href="xylulose_5-phosphate" title="wikilink">xylulose 5-phosphate</a>.<a class="footnoteRef" href="#fn25" id="fnref25"><sup>25</sup></a> Therefore, the reaction that occurs in the pentose phosphate pathway is exactly the reverse of the reaction which occurs in the Calvin cycle. The mechanism remains the same and involves the formation of an enediolate intermediate.</p>

<p>Due to its involvement in this pathway, phosphopentose epimerase is an important enzyme for the cellular response to oxidative stress.<a class="footnoteRef" href="#fn26" id="fnref26"><sup>26</sup></a> The generation of <a class="uri" href="NADPH" title="wikilink">NADPH</a> by the pentose phosphate pathway helps protect cells against <a href="reactive_oxygen_species" title="wikilink">reactive oxygen species</a>. NADPH is able to reduce <a class="uri" href="glutathione" title="wikilink">glutathione</a>, which detoxifies the body by producing <a class="uri" href="H2O" title="wikilink">H2O</a> from <a class="uri" href="H2O2" title="wikilink">H2O2</a>.<a class="footnoteRef" href="#fn27" id="fnref27"><sup>27</sup></a> Therefore, not only does phosphopentose epimerase alter flux through the PPP, but it also prevents buildup of peroxides.</p>
<h2 id="evolution">Evolution</h2>

<p>The structures of many phosphopentose epimerase analogs have been discovered through crystallographic studies.<a class="footnoteRef" href="#fn28" id="fnref28"><sup>28</sup></a><a class="footnoteRef" href="#fn29" id="fnref29"><sup>29</sup></a> Due to its role in the Calvin cycle and the pentose phosphate pathway, the overall structure is conserved. When the sequences of evolutionarily-distant organisms were compared, greater than 50% similarity was observed.<a class="footnoteRef" href="#fn30" id="fnref30"><sup>30</sup></a> However, amino acids positioned at the <a href="Protein_dimer" title="wikilink">dimer</a> interface – which are involved in many intermolecular interactions – are not necessarily conserved. It is important to note that the members of the “ribulose phosphate binding” superfamily resulted from <a href="divergent_evolution" title="wikilink">divergent evolution</a> from a (β/α)8 - barrel ancestor.<a class="footnoteRef" href="#fn31" id="fnref31"><sup>31</sup></a></p>
<h2 id="drug-targeting-and-malaria">Drug Targeting and Malaria</h2>

<p>The <a class="uri" href="protozoan" title="wikilink">protozoan</a> <a class="uri" href="organism" title="wikilink">organism</a> <a href="Plasmodium_falciparum" title="wikilink">Plasmodium falciparum</a> is a major causative agent of <a class="uri" href="malaria" title="wikilink">malaria</a>. Phosphopentose epimerase has been implicated in the shikimate pathway, an essential pathway for the propagation of malaria.<a class="footnoteRef" href="#fn32" id="fnref32"><sup>32</sup></a> As the enzyme converts ribulose 5-phosphate into xylulose 5-phosphate, the latter is further metabolized into <a href="erythrose_4-phosphate" title="wikilink">erythrose 4-phosphate</a>. The shikimate pathway then converts erythrose 4-phosphate into chorismate.<a class="footnoteRef" href="#fn33" id="fnref33"><sup>33</sup></a> It is phosphopentose epimerase which allows Plasmodium falciparum to use erythorse 4-phosphate as a substrate. Due to this enzyme’s involvement in the shikimate pathway, phosphopentose epimerase is a potential drug target for developing antimalarials.</p>
<h2 id="see-also">See also</h2>
<ul>
<li>Phosphopentose Isomerase</li>
<li>Phosphoribulose Kinase</li>
<li>Pentose Phosphate Pathway</li>
<li>TIM barrel</li>
<li>RPE</li>
</ul>
<h2 id="references">References</h2>
<h2 id="external-links">External links</h2>
<ul>
<li><a href="http://www.ncbi.nlm.nih.gov/books/bv.fcgi?rid=stryer.figgrp.2802">Diagram at nlm.nih.gov</a></li>
<li><a href="http://cmgm.stanford.edu/biochem200/OVERHEADS_PPP.html">Diagram at stanford.edu</a></li>
<li><a href="http://www.fasebj.org/content/25/2/497.long#">1</a></li>
</ul>

<p>"</p>

<p><a href="Category:EC_5.1.3" title="wikilink">Category:EC 5.1.3</a> <a href="Category:Enzymes_of_known_structure" title="wikilink">Category:Enzymes of known structure</a></p>
<section class="footnotes">
<hr/>
<ol>
<li id="fn1"></li>
<li id="fn2"><a href="#fnref2">↩</a></li>
<li id="fn3"><a href="#fnref3">↩</a></li>
<li id="fn4"><a href="#fnref4">↩</a></li>
<li id="fn5"><a href="#fnref5">↩</a></li>
<li id="fn6"></li>
<li id="fn7"><a href="#fnref7">↩</a></li>
<li id="fn8"><a href="#fnref8">↩</a></li>
<li id="fn9"></li>
<li id="fn10"></li>
<li id="fn11"><a href="#fnref11">↩</a></li>
<li id="fn12"></li>
<li id="fn13"></li>
<li id="fn14"></li>
<li id="fn15"></li>
<li id="fn16"><a href="#fnref16">↩</a></li>
<li id="fn17"><a href="#fnref17">↩</a></li>
<li id="fn18"><a href="#fnref18">↩</a></li>
<li id="fn19"></li>
<li id="fn20"><a href="#fnref20">↩</a></li>
<li id="fn21"></li>
<li id="fn22"><a href="#fnref22">↩</a></li>
<li id="fn23"></li>
<li id="fn24"></li>
<li id="fn25"></li>
<li id="fn26"></li>
<li id="fn27"></li>
<li id="fn28"><a href="#fnref28">↩</a></li>
<li id="fn29"><a href="#fnref29">↩</a></li>
<li id="fn30"><a href="#fnref30">↩</a></li>
<li id="fn31"></li>
<li id="fn32"><a href="#fnref32">↩</a></li>
<li id="fn33"></li>
</ol>
</section>
</body>
</html>
