<html lang="en">
<head>
<meta charset="utf-8"/>
<title offset="396">Phylogenetic tree</title>
<script src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_SVG.js" type="text/javascript">
</script>
</head>
<body>
<h1>Phylogenetic tree</h1>
<hr/>

<p>  A <strong>phylogenetic tree</strong> or <strong>evolutionary tree</strong> is a branching <a class="uri" href="diagram" title="wikilink">diagram</a> or "<a href="tree_(graph_theory)" title="wikilink">tree</a>" showing the inferred <a href="evolution" title="wikilink">evolutionary</a> relationships among various biological <a class="uri" href="species" title="wikilink">species</a> or other entities—their <a class="uri" href="phylogeny" title="wikilink">phylogeny</a>—based upon similarities and differences in their physical or genetic characteristics. The taxa joined together in the tree are implied to have descended from a <a href="common_descent" title="wikilink">common ancestor</a>.</p>

<p>In a  phylogenetic tree, each node with descendants represents the inferred <a href="most_recent_common_ancestor" title="wikilink">most recent common ancestor</a> of the descendants, and the edge lengths in some trees may be interpreted as <a class="uri" href="time" title="wikilink">time</a> estimates. Each node is called a taxonomic unit. Internal nodes are generally called hypothetical taxonomic units, as they cannot be directly observed. Trees are useful in fields of biology such as <a class="uri" href="bioinformatics" title="wikilink">bioinformatics</a>, <a class="uri" href="systematics" title="wikilink">systematics</a>, and <a href="comparative_phylogenetics" title="wikilink">comparative phylogenetics</a>.</p>

<p><em>Unrooted</em> trees illustrate only the relatedness of the <a href="leaf_nodes" title="wikilink">leaf nodes</a> and do not require the ancestral root to be known or inferred.</p>
<h2 id="history">History</h2>

<p>The idea of a "<a href="tree_of_life_(science)" title="wikilink">tree of life</a>" arose from ancient notions of a ladder-like progression from lower to higher forms of <a class="uri" href="life" title="wikilink">life</a> (such as in the <a href="Great_Chain_of_Being" title="wikilink">Great Chain of Being</a>). Early representations of "branching" phylogenetic trees include a "paleontological chart" showing the geological relationships among plants and animals in the book <em>Elementary Geology</em>, by Edward Hitchcock (first edition: 1840).</p>

<p><a href="Charles_Darwin" title="wikilink">Charles Darwin</a> (1859) also produced one of the first illustrations and crucially popularized the notion of an <a href="Natural_selection" title="wikilink">evolutionary "tree"</a> in his seminal book <em><a href="The_Origin_of_Species" title="wikilink">The Origin of Species</a></em>. Over a century later, <a href="Evolutionary_biology" title="wikilink">evolutionary biologists</a> still use <a href="Tree_structure" title="wikilink">tree diagrams</a> to depict <a class="uri" href="evolution" title="wikilink">evolution</a> because such diagrams effectively convey the concept that <a class="uri" href="speciation" title="wikilink">speciation</a> occurs through the <a href="Adaptation" title="wikilink">adaptive</a> and semi<a class="uri" href="random" title="wikilink">random</a> splitting of lineages. Over time, species classification has become less static and more dynamic.</p>
<h2 id="types">Types</h2>
<h3 id="rooted-tree">Rooted tree</h3>

<p>A rooted phylogenetic tree (see two graphics at top) is a <a href="directed_graph" title="wikilink">directed</a> <a href="tree_(data_structure)" title="wikilink">tree</a> with a unique node corresponding to the (usually <a href="imputation_(statistics)" title="wikilink">imputed</a>) most recent common ancestor of all the entities at the <a href="leaf_node" title="wikilink">leaves</a> of the tree. The most common method for rooting trees is the use of an uncontroversial <a href="outgroup_(cladistics)" title="wikilink">outgroup</a>—close enough to allow inference from trait data or molecular sequencing, but far enough to be a clear outgroup.</p>
<h3 id="unrooted-tree">Unrooted tree</h3>
<figure><b>(Figure)</b>
<figcaption>An unrooted phylogenetic tree for <a class="uri" href="myosin" title="wikilink">myosin</a>, a <a href="gene_family" title="wikilink">superfamily</a> of <a href="protein" title="wikilink">proteins</a>. <a class="footnoteRef" href="#fn1" id="fnref1"><sup>1</sup></a></figcaption>
</figure>

<p>Unrooted trees illustrate the relatedness of the leaf nodes without making assumptions about ancestry. They do not require the ancestral root to be known or inferred.<a class="footnoteRef" href="#fn2" id="fnref2"><sup>2</sup></a> Unrooted trees can always be generated from rooted ones by simply omitting the root. By contrast, inferring the root of an unrooted tree requires some means of identifying ancestry. This is normally done by including an outgroup in the input data so that the root is necessarily between the outgroup and the rest of the taxa in the tree, or by introducing additional assumptions about the relative rates of evolution on each branch, such as an application of the <a href="molecular_clock" title="wikilink">molecular clock</a> <a class="uri" href="hypothesis" title="wikilink">hypothesis</a>. <a class="footnoteRef" href="#fn3" id="fnref3"><sup>3</sup></a></p>
<h3 id="bifurcating-tree">Bifurcating tree</h3>

<p>Both rooted and unrooted phylogenetic trees can be either <a href="bifurcation_theory" title="wikilink">bifurcating</a> or multifurcating, and either labeled or unlabeled. A rooted bifurcating tree has exactly two descendants arising from each <a href="interior_node" title="wikilink">interior node</a> (that is, it forms a <a href="binary_tree" title="wikilink">binary tree</a>), and an unrooted bifurcating tree takes the form of an <a href="unrooted_binary_tree" title="wikilink">unrooted binary tree</a>, a <a href="free_tree" title="wikilink">free tree</a> with exactly three neighbors at each internal node. In contrast, a rooted multifurcating tree may have more than two children at some nodes and an unrooted multifurcating tree may have more than three neighbors at some nodes. A labeled tree has specific values assigned to its leaves, while an unlabeled tree, sometimes called a tree shape, defines a topology only. The number of possible trees for a given number of leaf nodes depends on the specific type of tree, but there are always more multifurcating than bifurcating trees, more labeled than unlabeled trees, and more rooted than unrooted trees. The last distinction is the most biologically relevant; it arises because there are many places on an unrooted tree to put the root. For labeled bifurcating trees, there are:</p>

<p>

<math display="block" id="Phylogenetic_tree:0">
 <semantics>
  <mrow>
   <mrow>
    <mrow>
     <mrow>
      <mrow>
       <mo stretchy="false">(</mo>
       <mrow>
        <mrow>
         <mn>2</mn>
         <mi>n</mi>
        </mrow>
        <mo>-</mo>
        <mn>3</mn>
       </mrow>
       <mo stretchy="false">)</mo>
      </mrow>
      <mo lspace="0pt" rspace="3.5pt">!</mo>
     </mrow>
     <mo lspace="0pt" rspace="3.5pt">!</mo>
    </mrow>
    <mo>=</mo>
    <mpadded width="+1.7pt">
     <mfrac>
      <mrow>
       <mrow>
        <mo stretchy="false">(</mo>
        <mrow>
         <mrow>
          <mn>2</mn>
          <mi>n</mi>
         </mrow>
         <mo>-</mo>
         <mn>3</mn>
        </mrow>
        <mo stretchy="false">)</mo>
       </mrow>
       <mo lspace="0pt" rspace="3.5pt">!</mo>
      </mrow>
      <mrow>
       <msup>
        <mn>2</mn>
        <mrow>
         <mi>n</mi>
         <mo>-</mo>
         <mn>2</mn>
        </mrow>
       </msup>
       <mrow>
        <mrow>
         <mo stretchy="false">(</mo>
         <mrow>
          <mi>n</mi>
          <mo>-</mo>
          <mn>2</mn>
         </mrow>
         <mo stretchy="false">)</mo>
        </mrow>
        <mo lspace="0pt" rspace="3.5pt">!</mo>
       </mrow>
      </mrow>
     </mfrac>
    </mpadded>
   </mrow>
   <mo rspace="4.2pt">,</mo>
   <mrow>
    <mrow>
     <mpadded width="+1.7pt">
      <mtext>for</mtext>
     </mpadded>
     <mi>n</mi>
    </mrow>
    <mo>≥</mo>
    <mn>2</mn>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <csymbol cd="ambiguous">formulae-sequence</csymbol>
    <apply>
     <eq></eq>
     <apply>
      <factorial></factorial>
      <apply>
       <factorial></factorial>
       <apply>
        <minus></minus>
        <apply>
         <times></times>
         <cn type="integer">2</cn>
         <ci>n</ci>
        </apply>
        <cn type="integer">3</cn>
       </apply>
      </apply>
     </apply>
     <apply>
      <divide></divide>
      <apply>
       <factorial></factorial>
       <apply>
        <minus></minus>
        <apply>
         <times></times>
         <cn type="integer">2</cn>
         <ci>n</ci>
        </apply>
        <cn type="integer">3</cn>
       </apply>
      </apply>
      <apply>
       <times></times>
       <apply>
        <csymbol cd="ambiguous">superscript</csymbol>
        <cn type="integer">2</cn>
        <apply>
         <minus></minus>
         <ci>n</ci>
         <cn type="integer">2</cn>
        </apply>
       </apply>
       <apply>
        <factorial></factorial>
        <apply>
         <minus></minus>
         <ci>n</ci>
         <cn type="integer">2</cn>
        </apply>
       </apply>
      </apply>
     </apply>
    </apply>
    <apply>
     <geq></geq>
     <apply>
      <times></times>
      <mtext>for</mtext>
      <ci>n</ci>
     </apply>
     <cn type="integer">2</cn>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   (2n-3)!!=\frac{(2n-3)!}{2^{n-2}(n-2)!}\,,\,\text{for}\,n\geq 2
  </annotation>
 </semantics>
</math>

 total rooted trees and</p>

<p>

<math display="block" id="Phylogenetic_tree:1">
 <semantics>
  <mrow>
   <mrow>
    <mrow>
     <mrow>
      <mrow>
       <mo stretchy="false">(</mo>
       <mrow>
        <mrow>
         <mn>2</mn>
         <mi>n</mi>
        </mrow>
        <mo>-</mo>
        <mn>5</mn>
       </mrow>
       <mo stretchy="false">)</mo>
      </mrow>
      <mo lspace="0pt" rspace="3.5pt">!</mo>
     </mrow>
     <mo lspace="0pt" rspace="3.5pt">!</mo>
    </mrow>
    <mo>=</mo>
    <mpadded width="+1.7pt">
     <mfrac>
      <mrow>
       <mrow>
        <mo stretchy="false">(</mo>
        <mrow>
         <mrow>
          <mn>2</mn>
          <mi>n</mi>
         </mrow>
         <mo>-</mo>
         <mn>5</mn>
        </mrow>
        <mo stretchy="false">)</mo>
       </mrow>
       <mo lspace="0pt" rspace="3.5pt">!</mo>
      </mrow>
      <mrow>
       <msup>
        <mn>2</mn>
        <mrow>
         <mi>n</mi>
         <mo>-</mo>
         <mn>3</mn>
        </mrow>
       </msup>
       <mrow>
        <mrow>
         <mo stretchy="false">(</mo>
         <mrow>
          <mi>n</mi>
          <mo>-</mo>
          <mn>3</mn>
         </mrow>
         <mo stretchy="false">)</mo>
        </mrow>
        <mo lspace="0pt" rspace="3.5pt">!</mo>
       </mrow>
      </mrow>
     </mfrac>
    </mpadded>
   </mrow>
   <mo rspace="4.2pt">,</mo>
   <mrow>
    <mrow>
     <mpadded width="+1.7pt">
      <mtext>for</mtext>
     </mpadded>
     <mi>n</mi>
    </mrow>
    <mo>≥</mo>
    <mn>3</mn>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <csymbol cd="ambiguous">formulae-sequence</csymbol>
    <apply>
     <eq></eq>
     <apply>
      <factorial></factorial>
      <apply>
       <factorial></factorial>
       <apply>
        <minus></minus>
        <apply>
         <times></times>
         <cn type="integer">2</cn>
         <ci>n</ci>
        </apply>
        <cn type="integer">5</cn>
       </apply>
      </apply>
     </apply>
     <apply>
      <divide></divide>
      <apply>
       <factorial></factorial>
       <apply>
        <minus></minus>
        <apply>
         <times></times>
         <cn type="integer">2</cn>
         <ci>n</ci>
        </apply>
        <cn type="integer">5</cn>
       </apply>
      </apply>
      <apply>
       <times></times>
       <apply>
        <csymbol cd="ambiguous">superscript</csymbol>
        <cn type="integer">2</cn>
        <apply>
         <minus></minus>
         <ci>n</ci>
         <cn type="integer">3</cn>
        </apply>
       </apply>
       <apply>
        <factorial></factorial>
        <apply>
         <minus></minus>
         <ci>n</ci>
         <cn type="integer">3</cn>
        </apply>
       </apply>
      </apply>
     </apply>
    </apply>
    <apply>
     <geq></geq>
     <apply>
      <times></times>
      <mtext>for</mtext>
      <ci>n</ci>
     </apply>
     <cn type="integer">3</cn>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   (2n-5)!!=\frac{(2n-5)!}{2^{n-3}(n-3)!}\,,\,\text{for}\,n\geq 3
  </annotation>
 </semantics>
</math>

 total unrooted trees, where 

<math display="inline" id="Phylogenetic_tree:2">
 <semantics>
  <mi>n</mi>
  <annotation-xml encoding="MathML-Content">
   <ci>n</ci>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   n
  </annotation>
 </semantics>
</math>

 represents the number of leaf nodes. Among labeled bifurcating trees, the number of unrooted trees with 

<math display="inline" id="Phylogenetic_tree:3">
 <semantics>
  <mi>n</mi>
  <annotation-xml encoding="MathML-Content">
   <ci>n</ci>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   n
  </annotation>
 </semantics>
</math>

 leaves is equal to the number of rooted trees with 

<math display="inline" id="Phylogenetic_tree:4">
 <semantics>
  <mrow>
   <mi>n</mi>
   <mo>-</mo>
   <mn>1</mn>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <minus></minus>
    <ci>n</ci>
    <cn type="integer">1</cn>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   n-1
  </annotation>
 </semantics>
</math>

 leaves.<a class="footnoteRef" href="#fn4" id="fnref4"><sup>4</sup></a></p>
<h3 id="special-tree-types">Special tree types</h3>

<p> </p>
<ul>
<li>A <a class="uri" href="dendrogram" title="wikilink">dendrogram</a> is a broad term for the diagrammatic representation of a phylogenetic tree.</li>
<li>A <a class="uri" href="cladogram" title="wikilink">cladogram</a> is a phylogenetic tree formed using <a href="cladistics" title="wikilink">cladistic</a> methods. This type of tree only represents a branching pattern; i.e., its branch spans do not represent time or relative amount of character change.</li>
<li>A phylogram is a phylogenetic tree that has branch spans proportional to the amount of character change.</li>
<li>A chronogram is a phylogenetic tree that explicitly represents evolutionary time through its branch spans.</li>
<li>A spindle diagram (often called a Romerogram after the American palaeontologist <a href="Alfred_Romer" title="wikilink">Alfred Romer</a>) is the representation of the evolution and abundance of the various taxa through time.</li>
<li>A <a href="Rolf_Dahlgren" title="wikilink">Dahlgrenogram</a> is a diagram representing a cross section of a phylogenetic tree</li>
<li>A <a href="phylogenetic_network" title="wikilink">phylogenetic network</a> is not strictly speaking a tree, but rather a more general <a href="Graph_(mathematics)" title="wikilink">graph</a>, or a <a href="directed_acyclic_graph" title="wikilink">directed acyclic graph</a> in the case of rooted networks. They are used to overcome some of the <a href="Phylogenetic_tree#Limitations" title="wikilink">limitations</a> inherent to trees.</li>
</ul>
<h2 id="construction">Construction</h2>

<p>Phylogenetic trees composed with a nontrivial number of input sequences are constructed using <a href="computational_phylogenetics" title="wikilink">computational phylogenetics</a> methods. Distance-matrix methods such as <a class="uri" href="neighbor-joining" title="wikilink">neighbor-joining</a> or <a class="uri" href="UPGMA" title="wikilink">UPGMA</a>, which calculate <a href="genetic_distance" title="wikilink">genetic distance</a> from <a href="multiple_sequence_alignment" title="wikilink">multiple sequence alignments</a>, are simplest to implement, but do not invoke an evolutionary model. Many sequence alignment methods such as <a class="uri" href="ClustalW" title="wikilink">ClustalW</a> also create trees by using the simpler algorithms (i.e. those based on distance) of tree construction. <a href="Maximum_parsimony" title="wikilink">Maximum parsimony</a> is another simple method of estimating phylogenetic trees, but implies an implicit model of evolution (i.e. parsimony). More advanced methods use the <a href="optimality_criterion" title="wikilink">optimality criterion</a> of <a href="maximum_likelihood" title="wikilink">maximum likelihood</a>, often within a <a href="Bayesian_inference" title="wikilink">Bayesian Framework</a>, and apply an explicit model of evolution to phylogenetic tree estimation.<a class="footnoteRef" href="#fn5" id="fnref5"><sup>5</sup></a> Identifying the optimal tree using many of these techniques is <a class="uri" href="NP-hard" title="wikilink">NP-hard</a>,<a class="footnoteRef" href="#fn6" id="fnref6"><sup>6</sup></a> so <a class="uri" href="heuristic" title="wikilink">heuristic</a> search and <a href="Optimization_(mathematics)" title="wikilink">optimization</a> methods are used in combination with tree-scoring functions to identify a reasonably good tree that fits the data.</p>

<p>Tree-building methods can be assessed on the basis of several criteria:<a class="footnoteRef" href="#fn7" id="fnref7"><sup>7</sup></a></p>
<ul>
<li>efficiency (how long does it take to compute the answer, how much memory does it need?)</li>
<li>power (does it make good use of the data, or is information being wasted?)</li>
<li>consistency (will it converge on the same answer repeatedly, if each time given different data for the same model problem?)</li>
<li>robustness (does it cope well with violations of the assumptions of the underlying model?)</li>
<li>falsifiability (does it alert us when it is not good to use, i.e. when assumptions are violated?)</li>
</ul>

<p>Tree-building techniques have also gained the attention of mathematicians. Trees can also be built using <a class="uri" href="T-theory" title="wikilink">T-theory</a>.<a class="footnoteRef" href="#fn8" id="fnref8"><sup>8</sup></a></p>
<h2 id="limitations">Limitations</h2>

<p>Although phylogenetic trees produced on the basis of sequenced <a href="gene" title="wikilink">genes</a> or <a href="genome" title="wikilink">genomic</a> data in different species can provide evolutionary insight, they have important limitations. Most importantly, they do not necessarily accurately represent the evolutionary history of the included taxa. In fact, they are literally scientific <a class="uri" href="hypotheses" title="wikilink">hypotheses</a>, subject to <a class="uri" href="falsification" title="wikilink">falsification</a> by further study (e.g., gathering of additional data, analyzing the existing data with improved methods). The data on which they are based is <a href="signal_noise" title="wikilink">noisy</a>; the analysis can be confounded by <a href="genetic_recombination" title="wikilink">genetic recombination</a>,<a class="footnoteRef" href="#fn9" id="fnref9"><sup>9</sup></a> <a href="horizontal_gene_transfer" title="wikilink">horizontal gene transfer</a>,<a class="footnoteRef" href="#fn10" id="fnref10"><sup>10</sup></a> <a href="Hybrid_(biology)" title="wikilink">hybridisation</a> between species that were not nearest neighbors on the tree before hybridisation takes place, <a href="convergent_evolution" title="wikilink">convergent evolution</a>, and <a href="conserved_sequence" title="wikilink">conserved sequences</a>.</p>

<p>Also, there are problems in basing the analysis on a single type of character, such as a single <a class="uri" href="gene" title="wikilink">gene</a> or <a class="uri" href="protein" title="wikilink">protein</a> or only on morphological analysis, because such trees constructed from another unrelated data source often differ from the first, and therefore great care is needed in inferring phylogenetic relationships among species. This is most true of genetic material that is subject to lateral gene transfer and <a href="Genetic_recombination" title="wikilink">recombination</a>, where different <a class="uri" href="haplotype" title="wikilink">haplotype</a> blocks can have different histories. In general, the output tree of a phylogenetic analysis is an estimate of the <em>character</em>'s phylogeny (i.e. a gene tree) and not the phylogeny of the <a class="uri" href="taxa" title="wikilink">taxa</a> (i.e. species tree) from which these characters were sampled, though ideally, both should be very close. For this reason, serious phylogenetic studies generally use a combination of genes that come from different genomic sources (e.g., from mitochondrial or plastid vs. nuclear genomes), or genes that would be expected to evolve under different selective regimes, so that homoplasy (false <a href="homology_(biology)" title="wikilink">homology</a>) would be unlikely to result from natural selection.</p>

<p>When extinct species are included in a tree, they are <a href="leaf_node" title="wikilink">terminal nodes</a>, as it is unlikely that they are direct ancestors of any extant species. Skepticism might be applied when extinct species are included in trees that are wholly or partly based on DNA sequence data, due to the fact that little useful "<a href="ancient_DNA" title="wikilink">ancient DNA</a>" is preserved for longer than 100,000 years, and except in the most unusual circumstances no DNA sequences long enough for use in phylogenetic analyses have yet been recovered from material over 1 million years old.</p>

<p>The range of useful DNA materials has expanded with advances in extraction and sequencing technologies. Development of technologies able to infer sequences from smaller fragments, or from spatial patterns of DNA degradation products, would further expand the range of DNA considered useful.</p>

<p>In some organisms, <a href="endosymbiont" title="wikilink">endosymbionts</a> have an independent genetic history from the host.</p>

<p><a href="Phylogenetic_network" title="wikilink">Phylogenetic networks</a> are used when bifurcating trees are not suitable, due to these complications which suggest a more <a class="uri" href="reticulate" title="wikilink">reticulate</a> evolutionary history of the organisms sampled.</p>
<h2 id="see-also">See also</h2>
<h3 id="the-tree-of-life">The "tree of life"</h3>
<ul>
<li><a href="Evolutionary_history_of_life" title="wikilink">Evolutionary history of life</a>, an overview of the major time periods of life on earth</li>
<li><a class="uri" href="Life" title="wikilink">Life</a>, the top level for Wikipedia articles on living species, reflecting a diversity of classification systems.</li>
<li><a href="Three-domain_system" title="wikilink">Three-domain system</a> (cell types)</li>
<li><a class="uri" href="Wikispecies" title="wikilink">Wikispecies</a>, an external Wikimedia Foundation project to construct a "tree of life" appropriate for use by scientists</li>
</ul>
<h3 id="fields-of-study">Fields of study</h3>
<ul>
<li><a class="uri" href="Cladistics" title="wikilink">Cladistics</a></li>
<li><a href="Comparative_phylogenetics" title="wikilink">Comparative phylogenetics</a></li>
<li><a href="Computational_phylogenetics" title="wikilink">Computational phylogenetics</a></li>
<li><a href="Evolutionary_taxonomy" title="wikilink">Evolutionary taxonomy</a></li>
<li><a href="Evolutionary_biology" title="wikilink">Evolutionary biology</a></li>
<li><a href="Generalized_tree_alignment" title="wikilink">Generalized tree alignment</a></li>
<li><a href="Phylogenetic_comparative_methods" title="wikilink">Phylogenetic comparative methods</a></li>
<li><a class="uri" href="Phylogenetics" title="wikilink">Phylogenetics</a></li>
</ul>
<h3 id="software">Software</h3>
<ul>
<li><a href="List_of_phylogenetics_software" title="wikilink">List of phylogenetics software</a></li>
<li><a href="List_of_phylogenetic_tree_visualization_software" title="wikilink">List of phylogenetic tree visualization software</a></li>
</ul>
<h2 id="references">References</h2>
<h2 id="further-reading">Further reading</h2>
<ul>
<li>Schuh, R. T. and A. V. Z. Brower. 2009. <em>Biological Systematics: principles and applications (2nd edn.)</em> ISBN 978-0-8014-4799-0</li>
<li><a href="Manuel_Lima" title="wikilink">Manuel Lima</a>, <em><a href="The_Book_of_Trees:_Visualizing_Branches_of_Knowledge" title="wikilink">The Book of Trees: Visualizing Branches of Knowledge</a></em>, 2014, Princeton Architectural Press, New York.</li>
<li><a href="MEGA,_Molecular_Evolutionary_Genetics_Analysis" title="wikilink">MEGA</a>, a free software to draw phylogenetic trees.</li>
</ul>
<h2 id="external-links">External links</h2>
<h3 id="images">Images</h3>
<ul>
<li><a href="http://www.tellapallet.com/tree_of_life.htm">Poster-sized tree of life illustration</a></li>
<li><a href="http://www.webcitation.org/66QTvQVdZ">Human Y-Chromosome 2002 Phylogenetic Tree</a></li>
<li>In 2003, the <a href="Science_(journal)" title="wikilink"><em>Science</em></a> journal dedicated a special issue to the tree of life, including an <a href="http://www.sciencemag.org/feature/data/tol/">online version of a tree of life</a>.</li>
<li><a href="http://itol.embl.de/">iTOL: Interactive Tree Of Life</a></li>
<li><a href="http://picbreeder.com/tol.php">Phylogenetic Tree of Artificial Organisms Evolved on Computers</a></li>
<li><a href="http://phylogram.org/">Miyamoto and Goodman's Phylogram of Eutherian Mammals</a></li>
</ul>
<h3 id="general">General</h3>
<ul>
<li>An overview of different methods of tree visualization is available at </li>
<li><a href="http://www.discoverlife.org/tree">Discover Life</a> An interactive tree based on the U.S. National Science Foundation's Assembling the Tree of Life Project</li>
<li><a href="http://www.ohiou.edu/phylocode/index.html">PhyloCode</a></li>
<li><a href="http://www.mrc-lmb.cam.ac.uk/myosin/trees/trees.html">A Multiple Alignment of 139 Myosin Sequences and a Phylogenetic Tree</a></li>
<li><a href="http://tolweb.org/tree">Tree of Life Web Project</a></li>
<li><a href="http://www.trex.uqam.ca">Phylogenetic inferring on the T-REX server</a></li>
<li><a href="http://www.ncbi.nlm.nih.gov/Taxonomy/">NCBI's Taxonomy Database</a><a href="http://www.ncbi.nlm.nih.gov/Taxonomy/">1</a></li>
<li><a href="http://ete.cgenomics.org">ETE: A Python Environment for Tree Exploration</a> This is a programming library to analyze, manipulate and visualize phylogenetic trees. <a href="http://www.biomedcentral.com/1471-2105/11/24">Ref.</a></li>
<li><a href="http://supfam.org/SUPERFAMILY/sTOL">A daily-updated tree of (sequenced) life</a> </li>
</ul>

<p>"</p>

<p><a class="uri" href="Category:Phylogenetics" title="wikilink">Category:Phylogenetics</a> <a href="Category:Tree_of_life_(biology)" title="wikilink"> </a> <a href="Category:Trees_(data_structures)" title="wikilink">Category:Trees (data structures)</a></p>
<section class="footnotes">
<hr/>
<ol>
<li id="fn1"><a href="#fnref1">↩</a></li>
<li id="fn2"><a class="uri" href="http://www.ncbi.nlm.nih.gov/Class/NAWBIS/Modules/Phylogenetics/phylo9.html">http://www.ncbi.nlm.nih.gov/Class/NAWBIS/Modules/Phylogenetics/phylo9.html</a><a href="#fnref2">↩</a></li>
<li id="fn3"><a href="#fnref3">↩</a></li>
<li id="fn4">Felsenstein J. (2004). <em>Inferring Phylogenies</em> Sinauer Associates: Sunderland, MA.<a href="#fnref4">↩</a></li>
<li id="fn5"></li>
<li id="fn6"></li>
<li id="fn7"><a href="#fnref7">↩</a></li>
<li id="fn8">A. Dress, K. T. Huber, and V. Moulton. 2001. Metric Spaces in Pure and Applied Mathematics. <em>Documenta Mathematica</em> <em>LSU 2001</em>: 121-139<a href="#fnref8">↩</a></li>
<li id="fn9"><a href="#fnref9">↩</a></li>
<li id="fn10"><a href="#fnref10">↩</a></li>
</ol>
</section>
</body>
</html>
