<html lang="en">
<head>
<meta charset="utf-8"/>
<title offset="1128">Population genetics</title>
<script src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_SVG.js" type="text/javascript">
</script>
</head>
<body>
<h1>Population genetics</h1>
<hr/>

<p><strong>Population genetics</strong> is the study of the distributions and changes of <a href="allele_frequency" title="wikilink">allele frequency</a> in a <a class="uri" href="population" title="wikilink">population</a>, as it is subject to the four main evolutionary processes: <a href="natural_selection" title="wikilink">natural selection</a>, <a href="genetic_drift" title="wikilink">genetic drift</a>, <a class="uri" href="mutation" title="wikilink">mutation</a> and <a href="Genetic_recombination" title="wikilink">recombination</a>. It also takes into account the factors of <a href="gene_flow" title="wikilink">gene flow</a>, environmental change, population subdivision and <a href="population_structure" title="wikilink">population structure</a>. Studies in this branch of biology examine such phenomena as <a href="Adaptation_(biology)" title="wikilink">adaptation</a> and <a class="uri" href="speciation" title="wikilink">speciation</a>.</p>

<p>Population genetics was a vital ingredient in the emergence of the <a href="modern_evolutionary_synthesis" title="wikilink">modern evolutionary synthesis</a>. Its primary founders were <a href="Sewall_Wright" title="wikilink">Sewall Wright</a>, <a href="J._B._S._Haldane" title="wikilink">J. B. S. Haldane</a> and <a href="Ronald_Fisher" title="wikilink">R. A. Fisher</a>, who also laid the foundations for the related discipline of <a href="quantitative_genetics" title="wikilink">quantitative genetics</a>.</p>

<p>Traditionally a highly mathematical discipline, modern population genetics encompasses theoretical, lab and field work. Computational approaches, often utilising <a href="coalescent_theory" title="wikilink">coalescent theory</a>, have played a central role since the 1980s.</p>
<h2 id="history">History</h2>

<p> </p>

<p><em>Population genetics</em> began as a reconciliation of <a href="Mendelian_inheritance" title="wikilink">Mendelian inheritance</a> and <a class="uri" href="biostatistics" title="wikilink">biostatistics</a> models. A key step was the work of the British biologist and statistician <a href="Ronald_Fisher" title="wikilink">Ronald Fisher</a>. In a series of papers starting in 1918 and culminating in his 1930 book <em><a href="The_Genetical_Theory_of_Natural_Selection" title="wikilink">The Genetical Theory of Natural Selection</a></em>, Fisher showed that the continuous variation measured by the biometricians could be produced by the combined action of many discrete genes, and that <a href="natural_selection" title="wikilink">natural selection</a> could change <a href="Allele_frequency" title="wikilink">allele frequencies</a> in a population, resulting in evolution. In a series of papers beginning in 1924, another British geneticist, <a href="J._B._S._Haldane" title="wikilink">J.B.S. Haldane</a> worked out the mathematics of allele frequency change at a single gene locus under a broad range of conditions. Haldane also applied statistical analysis to real-world examples of natural selection, such as the <a href="Peppered_moth_evolution" title="wikilink">evolution of industrial melanism in peppered moths</a>, and showed that <a href="selection_coefficient" title="wikilink">selection coefficients</a> could be larger than Fisher assumed, leading to more rapid adaptive evolution.<a class="footnoteRef" href="#fn1" id="fnref1"><sup>1</sup></a><a class="footnoteRef" href="#fn2" id="fnref2"><sup>2</sup></a></p>

<p>The American biologist <a href="Sewall_Wright" title="wikilink">Sewall Wright</a>, who had a background in <a href="animal_breeding" title="wikilink">animal breeding</a> experiments, focused on combinations of interacting genes, and the effects of inbreeding on small, relatively isolated populations that exhibited <a href="genetic_drift" title="wikilink">genetic drift</a>. In 1932, Wright introduced the concept of an <a href="Fitness_landscape" title="wikilink">adaptive landscape</a> and argued that genetic drift and inbreeding could drive a small, isolated sub-population away from an <a href="adaptive_peak" title="wikilink">adaptive peak</a>, allowing natural selection to drive it towards different adaptive peaks.</p>

<p>The work of Fisher, Haldane and Wright founded the discipline of <em>population genetics</em>. This integrated natural selection with Mendelian genetics, which was the critical first step in developing a unified theory of how evolution worked.<a class="footnoteRef" href="#fn3" id="fnref3"><sup>3</sup></a><a class="footnoteRef" href="#fn4" id="fnref4"><sup>4</sup></a> <a href="John_Maynard_Smith" title="wikilink">John Maynard Smith</a> was Haldane's pupil, whilst <a href="W.D._Hamilton" title="wikilink">W.D. Hamilton</a> was heavily influenced by the writings of Fisher. The American <a href="George_R._Price" title="wikilink">George R. Price</a> worked with both Hamilton and Maynard Smith. American <a href="Richard_Lewontin" title="wikilink">Richard Lewontin</a> and Japanese <a href="Motoo_Kimura" title="wikilink">Motoo Kimura</a> were heavily influenced by Wright.</p>
<h3 id="modern-evolutionary-synthesis">Modern evolutionary synthesis</h3>

<p>The mathematics of population genetics were originally developed as the beginning of the <a href="modern_evolutionary_synthesis" title="wikilink">modern evolutionary synthesis</a>. According to Beatty (1986), population genetics defines the core of the modern synthesis. In the first few decades of the 20th century, most field naturalists continued to believe that Lamarckian and orthogenic mechanisms of evolution provided the best explanation for the complexity they observed in the living world. However, as the field of genetics continued to develop, those views became less tenable.<a class="footnoteRef" href="#fn5" id="fnref5"><sup>5</sup></a> During the modern evolutionary synthesis, these ideas were purged, and only evolutionary causes that could be expressed in the mathematical framework of population genetics were retained.<a class="footnoteRef" href="#fn6" id="fnref6"><sup>6</sup></a> Consensus was reached as to which evolutionary factors might influence evolution, but not as to the relative importance of the various factors.<a class="footnoteRef" href="#fn7" id="fnref7"><sup>7</sup></a></p>

<p><a href="Theodosius_Dobzhansky" title="wikilink">Theodosius Dobzhansky</a>, a postdoctoral worker in T. H. Morgan's lab, had been influenced by the work on genetic diversity by <a href="Russia" title="wikilink">Russian</a> geneticists such as <a href="Sergei_Chetverikov" title="wikilink">Sergei Chetverikov</a>. He helped to bridge the divide between the foundations of <a class="uri" href="microevolution" title="wikilink">microevolution</a> developed by the population geneticists and the patterns of <a class="uri" href="macroevolution" title="wikilink">macroevolution</a> observed by field biologists, with his 1937 book <em><a href="Genetics_and_the_Origin_of_Species" title="wikilink">Genetics and the Origin of Species</a></em>. Dobzhansky examined the genetic diversity of wild populations and showed that, contrary to the assumptions of the population geneticists, these populations had large amounts of genetic diversity, with marked differences between sub-populations. The book also took the highly mathematical work of the population geneticists and put it into a more accessible form. Many more biologists were influenced by population genetics via Dobzhansky than were able to read the highly mathematical works in the original.<a class="footnoteRef" href="#fn8" id="fnref8"><sup>8</sup></a></p>
<h3 id="selection-vs.-genetic-drift">Selection vs. genetic drift</h3>

<p>Fisher and Wright had some fundamental disagreements and a controversy about the relative roles of selection and drift continued for much of the century between the Americans and the British.</p>

<p>In Great Britain <a href="E.B._Ford" title="wikilink">E.B. Ford</a>, the pioneer of <a href="ecological_genetics" title="wikilink">ecological genetics</a>, continued throughout the 1930s and 1940s to demonstrate the power of selection due to ecological factors including the ability to maintain genetic diversity through <a href="polymorphism_(biology)" title="wikilink">genetic polymorphisms</a> such as human <a href="blood_type" title="wikilink">blood types</a>. Ford's work, in collaboration with Fisher, contributed to a shift in emphasis during the course of the modern synthesis towards <a href="natural_selection" title="wikilink">natural selection</a> over <a href="genetic_drift" title="wikilink">genetic drift</a>.<a class="footnoteRef" href="#fn9" id="fnref9"><sup>9</sup></a><a class="footnoteRef" href="#fn10" id="fnref10"><sup>10</sup></a><a class="footnoteRef" href="#fn11" id="fnref11"><sup>11</sup></a><a class="footnoteRef" href="#fn12" id="fnref12"><sup>12</sup></a></p>

<p>Recent studies of eukaryotic <a href="transposable_element" title="wikilink">transposable elements</a>, and of their impact on <a class="uri" href="speciation" title="wikilink">speciation</a>, point again to a major role of nonadaptive processes such as <a class="uri" href="mutation" title="wikilink">mutation</a> and <a href="genetic_drift" title="wikilink">genetic drift</a>.<a class="footnoteRef" href="#fn13" id="fnref13"><sup>13</sup></a> Mutation and genetic drift are also viewed as major factors in the evolution of genome complexity<a class="footnoteRef" href="#fn14" id="fnref14"><sup>14</sup></a></p>
<h2 id="fundamentals">Fundamentals</h2>

<p>Population genetics is the study of the frequency and interaction of alleles and genes in populations.<a class="footnoteRef" href="#fn15" id="fnref15"><sup>15</sup></a> A <a href="sexual_reproduction" title="wikilink">sexual</a> population is a set of organisms in which any pair of members can <a href="Breeding_in_the_wild" title="wikilink">breed freely</a> together. This implies that all members belong to the same <a class="uri" href="species" title="wikilink">species</a> and live near each other.<a class="footnoteRef" href="#fn16" id="fnref16"><sup>16</sup></a></p>

<p>For example, all of the moths of the same species living in an isolated forest are a population. A gene in this population may have several alternate forms, which account for variations between the <a href="phenotype" title="wikilink">phenotypes</a> of the organisms. An example might be a gene for coloration in moths that has two <a href="allele" title="wikilink">alleles</a>: black and white. A <a href="gene_pool" title="wikilink">gene pool</a> is the complete set of alleles for a gene in a single population; the <a href="allele_frequency" title="wikilink">allele frequency</a> for an allele is the fraction of the genes in the pool that is composed of that allele (for example, what fraction of moth coloration genes are the black allele). <a class="uri" href="Evolution" title="wikilink">Evolution</a> occurs when there are changes in the frequencies of alleles within a population; for example, the allele for black color in a population of moths becoming more common.</p>

<p> </p>
<h2 id="four-processes">Four processes</h2>
<h3 id="natural-selection">Natural selection</h3>

<p><em>Natural selection</em> is the fact that some <a href="trait_(biology)" title="wikilink">traits</a> make it more likely for an <a class="uri" href="organism" title="wikilink">organism</a> to survive and <a href="reproduction" title="wikilink">reproduce</a>. Population genetics describes natural selection by defining <a href="fitness_(biology)" title="wikilink">fitness</a> as a <a href="propensity_probability" title="wikilink">propensity or probability</a> of survival and reproduction in a particular environment. The fitness is normally given by the symbol <strong>w</strong>=1-<strong>s</strong> where <strong>s</strong> is the <a href="selection_coefficient" title="wikilink">selection coefficient</a>. Natural selection acts on <a href="phenotype" title="wikilink">phenotypes</a>, or the observable characteristics of organisms, but the <a href="heritability" title="wikilink">genetically heritable</a> basis of any phenotype which gives a reproductive advantage will become more common in a population (see <a href="allele_frequency" title="wikilink">allele frequency</a>). In this way, natural selection converts differences in fitness into changes in allele frequency in a <a class="uri" href="population" title="wikilink">population</a> over successive generations.</p>

<p>Before the advent of population genetics, many biologists doubted that small differences in fitness were sufficient to make a large difference to evolution.<a class="footnoteRef" href="#fn17" id="fnref17"><sup>17</sup></a> Population geneticists addressed this concern in part by comparing selection to <a href="genetic_drift" title="wikilink">genetic drift</a>. Selection can overcome genetic drift when <strong>s</strong> is greater than 1 divided by the <a href="effective_population_size" title="wikilink">effective population size</a>. When this criterion is met, the probability that a new advantageous mutant becomes <a href="fixation_(population_genetics)" title="wikilink">fixed</a> is approximately equal to <strong>2s</strong>.<a class="footnoteRef" href="#fn18" id="fnref18"><sup>18</sup></a><a class="footnoteRef" href="#fn19" id="fnref19"><sup>19</sup></a> The time until fixation of such an allele depends little on genetic drift, and is approximately proportional to log(sN)/s.<a class="footnoteRef" href="#fn20" id="fnref20"><sup>20</sup></a></p>
<h4 id="hardyweinberg-principle">Hardy–Weinberg principle</h4>

<p><a href="Natural_selection" title="wikilink">Natural selection</a> will only cause evolution if there is enough <a href="genetic_variation" title="wikilink">genetic variation</a> in a population. Before the discovery of <a href="Mendelian_genetics" title="wikilink">Mendelian genetics</a>, one common hypothesis was <a href="blending_inheritance" title="wikilink">blending inheritance</a>. But with blending inheritance, genetic variance would be rapidly lost, making evolution by natural selection implausible. The <em><a href="Hardy–Weinberg_principle" title="wikilink">Hardy–Weinberg principle</a></em> provides the solution to how variation is maintained in a population with <a href="Mendelian_inheritance" title="wikilink">Mendelian inheritance</a>. According to this principle, the frequencies of alleles (variations in a gene) will remain constant in the absence of selection, mutation, migration and genetic drift.<a class="footnoteRef" href="#fn21" id="fnref21"><sup>21</sup></a> The Hardy–Weinberg "equilibrium" refers to this stability of allele frequencies over time.</p>

<p>A second component of the Hardy–Weinberg principle concerns the effects of a single generation of random mating. In this case, the genotype frequencies can be predicted from the allele frequencies. For example, in the simplest case of a single locus with two <a href="allele" title="wikilink">alleles</a>: the <a href="dominant_allele" title="wikilink">dominant allele</a> is denoted <strong>A</strong> and the <a class="uri" href="recessive" title="wikilink">recessive</a> <strong>a</strong> and their frequencies are denoted by <em>p</em> and <em>q</em>; freq(<strong>A</strong>) = <em>p</em>; freq(<strong>a</strong>) = <em>q</em>; <em>p</em> + <em>q</em> = 1. If the genotype frequencies are in Hardy–Weinberg proportions resulting from random mating, then we will have freq(<strong>AA</strong>) = <em>p</em><sup>2</sup> for the <strong>AA</strong> <a href="homozygote" title="wikilink">homozygotes</a> in the population, freq(<strong>aa</strong>) = <em>q</em><sup>2</sup> for the <strong>aa</strong> homozygotes, and freq(<strong>Aa</strong>) = 2<em>pq</em> for the <a href="heterozygote" title="wikilink">heterozygotes</a>.</p>
<h3 id="genetic-drift">Genetic drift</h3>

<p><em>Genetic drift</em> is a change in <a href="Allele_frequency" title="wikilink">allele frequencies</a> caused by <a href="Sampling_(statistics)" title="wikilink">random sampling</a>.<a class="footnoteRef" href="#fn22" id="fnref22"><sup>22</sup></a> That is, the alleles in the offspring are a random sample of those in the parents.<a class="footnoteRef" href="#fn23" id="fnref23"><sup>23</sup></a> Genetic drift may cause gene variants to disappear completely, and thereby reduce genetic variability. In contrast to <a href="natural_selection" title="wikilink">natural selection</a>, which makes gene variants more common or less common depending on their reproductive success,<a class="footnoteRef" href="#fn24" id="fnref24"><sup>24</sup></a> the changes due to genetic drift are not driven by environmental or adaptive pressures, and may be beneficial, neutral, or detrimental to reproductive success.</p>

<p>The effect of genetic drift is larger for alleles present in few copies than when an allele is present in many copies. Scientists wage vigorous debates over the relative importance of genetic drift compared with natural selection. <a href="Ronald_Fisher" title="wikilink">Ronald Fisher</a> held the view that genetic drift plays at the most a minor role in evolution, and this remained the dominant view for several decades. In 1968 <a href="Motoo_Kimura" title="wikilink">Motoo Kimura</a> rekindled the debate with his <a href="neutral_theory_of_molecular_evolution" title="wikilink">neutral theory of molecular evolution</a> which claims that most of the changes in the genetic material are caused by neutral mutations and genetic drift.<a class="footnoteRef" href="#fn25" id="fnref25"><sup>25</sup></a> The role of genetic drift by means of sampling error in evolution has been criticized by <a href="John_H_Gillespie" title="wikilink">John H Gillespie</a><a class="footnoteRef" href="#fn26" id="fnref26"><sup>26</sup></a> and <a href="Will_Provine" title="wikilink">Will Provine</a>, who argue that selection on linked sites is a more important stochastic force.</p>

<p>The population genetics of genetic drift are described using either <a href="branching_process" title="wikilink">branching processes</a> or a <a href="diffusion_equation" title="wikilink">diffusion equation</a> describing changes in allele frequency.<a class="footnoteRef" href="#fn27" id="fnref27"><sup>27</sup></a> These approaches are usually applied to the Wright-Fisher and Moran models of population genetics. Assuming genetic drift is the only evolutionary force acting on an allele, after t generations in many replicated populations, starting with allele frequencies of p and q, the variance in allele frequency across those populations is</p>

<p>

<math display="block" id="Population_genetics:0">
 <semantics>
  <mrow>
   <mrow>
    <msub>
     <mi>V</mi>
     <mi>t</mi>
    </msub>
    <mo>≈</mo>
    <mrow>
     <mi>p</mi>
     <mi>q</mi>
     <mrow>
      <mo>(</mo>
      <mrow>
       <mn>1</mn>
       <mo>-</mo>
       <mrow>
        <mi>exp</mi>
        <mrow>
         <mo>{</mo>
         <mrow>
          <mo>-</mo>
          <mfrac>
           <mi>t</mi>
           <mrow>
            <mn>2</mn>
            <msub>
             <mi>N</mi>
             <mi>e</mi>
            </msub>
           </mrow>
          </mfrac>
         </mrow>
         <mo>}</mo>
        </mrow>
       </mrow>
      </mrow>
      <mo>)</mo>
     </mrow>
    </mrow>
   </mrow>
   <mo>.</mo>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <approx></approx>
    <apply>
     <csymbol cd="ambiguous">subscript</csymbol>
     <ci>V</ci>
     <ci>t</ci>
    </apply>
    <apply>
     <times></times>
     <ci>p</ci>
     <ci>q</ci>
     <apply>
      <minus></minus>
      <cn type="integer">1</cn>
      <apply>
       <exp></exp>
       <apply>
        <minus></minus>
        <apply>
         <divide></divide>
         <ci>t</ci>
         <apply>
          <times></times>
          <cn type="integer">2</cn>
          <apply>
           <csymbol cd="ambiguous">subscript</csymbol>
           <ci>N</ci>
           <ci>e</ci>
          </apply>
         </apply>
        </apply>
       </apply>
      </apply>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   V_{t}\approx pq\left(1-\exp\left\{-\frac{t}{2N_{e}}\right\}\right).
  </annotation>
 </semantics>
</math>

<a class="footnoteRef" href="#fn28" id="fnref28"><sup>28</sup></a></p>
<h3 id="mutation">Mutation</h3>

<p>Mutation is the ultimate source of <a href="genetic_variation" title="wikilink">genetic variation</a> in the form of new alleles. Mutation can result in several different types of change in DNA sequences; these can either have no effect, alter the <a href="gene_product" title="wikilink">product of a gene</a>, or prevent the gene from functioning. Studies in the fly <em><a href="Drosophila_melanogaster" title="wikilink">Drosophila melanogaster</a></em> suggest that if a mutation changes a protein produced by a gene, this will probably be harmful, with about 70 percent of these mutations having damaging effects, and the remainder being either neutral or weakly beneficial.<a class="footnoteRef" href="#fn29" id="fnref29"><sup>29</sup></a></p>

<p>Mutations can involve large sections of DNA becoming <a href="gene_duplication" title="wikilink">duplicated</a>, usually through <a href="genetic_recombination" title="wikilink">genetic recombination</a>.<a class="footnoteRef" href="#fn30" id="fnref30"><sup>30</sup></a> These duplications are a major source of raw material for evolving new genes, with tens to hundreds of genes duplicated in animal genomes every million years.<a class="footnoteRef" href="#fn31" id="fnref31"><sup>31</sup></a> Most genes belong to larger <a href="gene_family" title="wikilink">families of genes</a> of <a href="homology_(biology)" title="wikilink">shared ancestry</a>.<a class="footnoteRef" href="#fn32" id="fnref32"><sup>32</sup></a> Novel genes are produced by several methods, commonly through the duplication and mutation of an ancestral gene, or by recombining parts of different genes to form new combinations with new functions.<a class="footnoteRef" href="#fn33" id="fnref33"><sup>33</sup></a><a class="footnoteRef" href="#fn34" id="fnref34"><sup>34</sup></a> Here, <a href="protein_domain" title="wikilink">domains</a> act as modules, each with a particular and independent function, that can be mixed together to produce genes encoding new proteins with novel properties.<a class="footnoteRef" href="#fn35" id="fnref35"><sup>35</sup></a> For example, the human eye uses four genes to make structures that sense light: three for <a href="Cone_cell" title="wikilink">color vision</a> and one for <a href="Rod_cell" title="wikilink">night vision</a>; all four arose from a single ancestral gene.<a class="footnoteRef" href="#fn36" id="fnref36"><sup>36</sup></a> Another advantage of duplicating a gene (or even an <a href="Polyploidy" title="wikilink">entire genome</a>) is that this increases <a href="Redundancy_(engineering)" title="wikilink">redundancy</a>; this allows one gene in the pair to acquire a new function while the other copy performs the original function.<a class="footnoteRef" href="#fn37" id="fnref37"><sup>37</sup></a><a class="footnoteRef" href="#fn38" id="fnref38"><sup>38</sup></a> Other types of mutation occasionally create new genes from previously noncoding DNA.<a class="footnoteRef" href="#fn39" id="fnref39"><sup>39</sup></a><a class="footnoteRef" href="#fn40" id="fnref40"><sup>40</sup></a></p>

<p>In addition to being a major source of variation, mutation may also function as a mechanism of evolution when there are different probabilities at the molecular level for different mutations to occur, a process known as mutation bias.<a class="footnoteRef" href="#fn41" id="fnref41"><sup>41</sup></a> If two genotypes, for example one with the nucleotide G and another with the nucleotide A in the same position, have the same fitness, but mutation from G to A happens more often than mutation from A to G, then genotypes with A will tend to evolve.<a class="footnoteRef" href="#fn42" id="fnref42"><sup>42</sup></a> Different insertion vs. deletion mutation biases in different taxa can lead to the evolution of different genome sizes.<a class="footnoteRef" href="#fn43" id="fnref43"><sup>43</sup></a><a class="footnoteRef" href="#fn44" id="fnref44"><sup>44</sup></a> Developmental or mutational biases have also been observed in <a href="Morphology_(biology)" title="wikilink">morphological</a> evolution.<a class="footnoteRef" href="#fn45" id="fnref45"><sup>45</sup></a><a class="footnoteRef" href="#fn46" id="fnref46"><sup>46</sup></a> For example, according to the <a href="Baldwin_effect" title="wikilink">phenotype-first theory of evolution</a>, mutations can eventually cause the <a href="genetic_assimilation" title="wikilink">genetic assimilation</a> of traits that were previously <a href="phenotypic_plasticity" title="wikilink">induced by the environment</a>.<a class="footnoteRef" href="#fn47" id="fnref47"><sup>47</sup></a><a class="footnoteRef" href="#fn48" id="fnref48"><sup>48</sup></a></p>

<p>Mutation bias effects are superimposed on other processes. If selection would favor either one out of two mutations, but there is no extra advantage to having both, then the mutation that occurs the most frequently is the one that is most likely to become fixed in a population.<a class="footnoteRef" href="#fn49" id="fnref49"><sup>49</sup></a><a class="footnoteRef" href="#fn50" id="fnref50"><sup>50</sup></a> Mutations leading to the loss of function of a gene are much more common than mutations that produce a new, fully functional gene. Most loss of function mutations are selected against. But when selection is weak, mutation bias towards loss of function can affect evolution.<a class="footnoteRef" href="#fn51" id="fnref51"><sup>51</sup></a> For example, <a href="pigment" title="wikilink">pigments</a> are no longer useful when animals live in the darkness of caves, and tend to be lost.<a class="footnoteRef" href="#fn52" id="fnref52"><sup>52</sup></a> This kind of loss of function can occur because of mutation bias, and/or because the function had a cost, and once the benefit of the function disappeared, natural selection leads to the loss. Loss of <a href="endospore" title="wikilink">sporulation</a> ability in a <a href="Bacillus_subtilis" title="wikilink">bacterium</a> during laboratory evolution appears to have been caused by mutation bias, rather than natural selection against the cost of maintaining sporulation ability.<a class="footnoteRef" href="#fn53" id="fnref53"><sup>53</sup></a> When there is no selection for loss of function, the speed at which loss evolves depends more on the mutation rate than it does on the <a href="effective_population_size" title="wikilink">effective population size</a>,<a class="footnoteRef" href="#fn54" id="fnref54"><sup>54</sup></a> indicating that it is driven more by mutation bias than by genetic drift.</p>
<h4 id="evolution-of-mutation-rate">Evolution of mutation rate</h4>

<p>Due to the damaging effects that mutations can have on cells, organisms have evolved mechanisms such as <a href="DNA_repair" title="wikilink">DNA repair</a> to remove mutations.<a class="footnoteRef" href="#fn55" id="fnref55"><sup>55</sup></a> Therefore, the optimal mutation rate for a species is a trade-off between costs of a high mutation rate, such as deleterious mutations, and the <a href="metabolism" title="wikilink">metabolic</a> costs of maintaining systems to reduce the mutation rate, such as DNA repair enzymes.<a class="footnoteRef" href="#fn56" id="fnref56"><sup>56</sup></a> Viruses that use RNA as their genetic material have rapid mutation rates,<a class="footnoteRef" href="#fn57" id="fnref57"><sup>57</sup></a> which can be an advantage since these viruses will evolve constantly and rapidly, and thus evade the defensive responses of e.g. the human <a href="immune_system" title="wikilink">immune system</a>.<a class="footnoteRef" href="#fn58" id="fnref58"><sup>58</sup></a></p>
<h3 id="gene-flow-and-transfer">Gene flow and transfer</h3>

<p> </p>

<p><a href="Gene_flow" title="wikilink">Gene flow</a> is the exchange of genes between populations, which are usually of the same species.<a class="footnoteRef" href="#fn59" id="fnref59"><sup>59</sup></a> Examples of gene flow within a species include the migration and then breeding of organisms, or the exchange of <a class="uri" href="pollen" title="wikilink">pollen</a>. Gene transfer between species includes the formation of <a href="Hybrid_(biology)" title="wikilink">hybrid</a> organisms and <a href="horizontal_gene_transfer" title="wikilink">horizontal gene transfer</a>.</p>

<p>Migration into or out of a population can change allele frequencies, as well as introducing genetic variation into a population. Immigration may add new genetic material to the established <a href="gene_pool" title="wikilink">gene pool</a> of a population. Conversely, emigration may remove genetic material. Population genetic models can be used to reconstruct the history of gene flow between populations.<a class="footnoteRef" href="#fn60" id="fnref60"><sup>60</sup></a></p>
<h4 id="reproductive-isolation">Reproductive isolation</h4>

<p>As <a href="reproductive_isolation" title="wikilink">barriers to reproduction</a> between two diverging populations are required for the populations to <a href="speciation" title="wikilink">become new species</a>, gene flow may slow this process by spreading genetic differences between the populations. Gene flow is hindered by mountain ranges, oceans and deserts or even man-made structures such as the <a href="Great_Wall_of_China" title="wikilink">Great Wall of China</a>, which has hindered the flow of plant genes.<a class="footnoteRef" href="#fn61" id="fnref61"><sup>61</sup></a></p>

<p>Depending on how far two species have diverged since their <a href="most_recent_common_ancestor" title="wikilink">most recent common ancestor</a>, it may still be possible for them to produce offspring, as with <a href="horse" title="wikilink">horses</a> and <a href="donkey" title="wikilink">donkeys</a> mating to produce <a href="mule" title="wikilink">mules</a>.<a class="footnoteRef" href="#fn62" id="fnref62"><sup>62</sup></a> Such <a href="Hybrid_(biology)" title="wikilink">hybrids</a> are generally <a href="infertility" title="wikilink">infertile</a>, due to the two different sets of chromosomes being unable to pair up during <a class="uri" href="meiosis" title="wikilink">meiosis</a>. In this case, closely related species may regularly interbreed, but hybrids will be selected against and the species will remain distinct. However, viable hybrids are occasionally formed and these new species can either have properties intermediate between their parent species, or possess a totally new phenotype.<a class="footnoteRef" href="#fn63" id="fnref63"><sup>63</sup></a> The importance of hybridization in creating <a href="hybrid_speciation" title="wikilink">new species</a> of animals is unclear, although cases have been seen in many types of animals,<a class="footnoteRef" href="#fn64" id="fnref64"><sup>64</sup></a> with the <a href="gray_tree_frog" title="wikilink">gray tree frog</a> being a particularly well-studied example.<a class="footnoteRef" href="#fn65" id="fnref65"><sup>65</sup></a></p>

<p>Hybridization is, however, an important means of speciation in plants, since <a class="uri" href="polyploidy" title="wikilink">polyploidy</a> (having more than two copies of each chromosome) is tolerated in plants more readily than in animals.<a class="footnoteRef" href="#fn66" id="fnref66"><sup>66</sup></a><a class="footnoteRef" href="#fn67" id="fnref67"><sup>67</sup></a> Polyploidy is important in hybrids as it allows reproduction, with the two different sets of chromosomes each being able to pair with an identical partner during meiosis.<a class="footnoteRef" href="#fn68" id="fnref68"><sup>68</sup></a> Polyploids also have more genetic diversity, which allows them to avoid <a href="inbreeding_depression" title="wikilink">inbreeding depression</a> in small populations.<a class="footnoteRef" href="#fn69" id="fnref69"><sup>69</sup></a></p>
<h4 id="genetic-structure">Genetic structure</h4>

<p>Because of physical barriers to migration, along with limited tendency for individuals to move or spread (<a href="wiktionary:vagility" title="wikilink">vagility</a>), and tendency to remain or come back to natal place (<a class="uri" href="philopatry" title="wikilink">philopatry</a>), natural populations rarely all interbreed as convenient in theoretical random models (<a class="uri" href="panmixy" title="wikilink">panmixy</a>) (Buston <em>et al.</em>, 2007). There is usually a geographic range within which individuals are more closely <a href="coefficient_of_relatedness" title="wikilink">related</a> to one another than those randomly selected from the general population. This is described as the extent to which a population is genetically structured (Repaci <em>et al.</em>, 2007). Genetic structuring can be caused by migration due to historical <a href="climate_change" title="wikilink">climate change</a>, species <a href="range_expansion" title="wikilink">range expansion</a> or current availability of <a class="uri" href="habitat" title="wikilink">habitat</a>.</p>
<h4 id="horizontal-gene-transfer">Horizontal Gene Transfer</h4>

<p> </p>

<p>Horizontal gene transfer is the transfer of genetic material from one organism to another organism that is not its offspring; this is most common among <a class="uri" href="bacteria" title="wikilink">bacteria</a>.<a class="footnoteRef" href="#fn70" id="fnref70"><sup>70</sup></a> In medicine, this contributes to the spread of <a href="antibiotic_resistance" title="wikilink">antibiotic resistance</a>, as when one bacteria acquires resistance genes it can rapidly transfer them to other species.<a class="footnoteRef" href="#fn71" id="fnref71"><sup>71</sup></a> Horizontal transfer of genes from bacteria to eukaryotes such as the yeast <em><a href="Saccharomyces_cerevisiae" title="wikilink">Saccharomyces cerevisiae</a></em> and the adzuki bean beetle <em>Callosobruchus chinensis</em> may also have occurred.<a class="footnoteRef" href="#fn72" id="fnref72"><sup>72</sup></a><a class="footnoteRef" href="#fn73" id="fnref73"><sup>73</sup></a> An example of larger-scale transfers are the eukaryotic <a href="Bdelloidea" title="wikilink">bdelloid rotifers</a>, which appear to have received a range of genes from bacteria, fungi, and plants.<a class="footnoteRef" href="#fn74" id="fnref74"><sup>74</sup></a> <a href="Virus" title="wikilink">Viruses</a> can also carry DNA between organisms, allowing transfer of genes even across <a href="domain_(biology)" title="wikilink">biological domains</a>.<a class="footnoteRef" href="#fn75" id="fnref75"><sup>75</sup></a> Large-scale gene transfer has also occurred between the ancestors of <a href="eukaryote" title="wikilink">eukaryotic cells</a> and prokaryotes, during the acquisition of <a href="chloroplast" title="wikilink">chloroplasts</a> and <a href="Mitochondrion" title="wikilink">mitochondria</a>.<a class="footnoteRef" href="#fn76" id="fnref76"><sup>76</sup></a></p>
<h2 id="complications">Complications</h2>

<p>Basic models of population genetics consider only one gene locus at a time. In practice, <a href="epistasis" title="wikilink">epistatic</a> and <a href="Genetic_linkage" title="wikilink">linkage</a> relationships between loci may also be important.</p>
<h3 id="epistasis">Epistasis</h3>

<p>Because of <a class="uri" href="epistasis" title="wikilink">epistasis</a>, the phenotypic effect of an allele at one locus may depend on which alleles are present at many other loci. Selection does not act on a single locus, but on a phenotype that arises through development from a complete genotype.</p>

<p>According to <a href="Richard_Lewontin" title="wikilink">Lewontin</a> (1974), the theoretical task for population genetics is a process in two spaces: a "genotypic space" and a "phenotypic space". The challenge of a <em>complete</em> theory of population genetics is to provide a set of laws that predictably map a population of <a href="genotype" title="wikilink">genotypes</a> (<em>G</em><sub>1</sub>) to a <a class="uri" href="phenotype" title="wikilink">phenotype</a> space (<em>P</em><sub>1</sub>), where <a href="natural_selection" title="wikilink">selection</a> takes place, and another set of laws that map the resulting population (<em>P</em><sub>2</sub>) back to genotype space (<em>G</em><sub>2</sub>) where <a href="Mendelism" title="wikilink">Mendelian</a> genetics can predict the next generation of genotypes, thus completing the cycle. Even leaving aside for the moment the non-Mendelian aspects of <a href="molecular_genetics" title="wikilink">molecular genetics</a>, this is clearly a gargantuan task. Visualizing this transformation schematically:</p>

<p>

<math display="block" id="Population_genetics:1">
 <semantics>
  <mrow>
   <mpadded width="+2.8pt">
    <msub>
     <mi>G</mi>
     <mn>1</mn>
    </msub>
   </mpadded>
   <mpadded width="+2.8pt">
    <mover>
     <mo movablelimits="false">→</mo>
     <msub>
      <mi>T</mi>
      <mn>1</mn>
     </msub>
    </mover>
   </mpadded>
   <mpadded width="+2.8pt">
    <msub>
     <mi>P</mi>
     <mn>1</mn>
    </msub>
   </mpadded>
   <mpadded width="+2.8pt">
    <mover>
     <mo movablelimits="false">→</mo>
     <msub>
      <mi>T</mi>
      <mn>2</mn>
     </msub>
    </mover>
   </mpadded>
   <mpadded width="+2.8pt">
    <msub>
     <mi>P</mi>
     <mn>2</mn>
    </msub>
   </mpadded>
   <mpadded width="+2.8pt">
    <mover>
     <mo movablelimits="false">→</mo>
     <msub>
      <mi>T</mi>
      <mn>3</mn>
     </msub>
    </mover>
   </mpadded>
   <mpadded width="+2.8pt">
    <msub>
     <mi>G</mi>
     <mn>2</mn>
    </msub>
   </mpadded>
   <mpadded width="+2.8pt">
    <mover>
     <mo movablelimits="false">→</mo>
     <msub>
      <mi>T</mi>
      <mn>4</mn>
     </msub>
    </mover>
   </mpadded>
   <mpadded width="+2.8pt">
    <msubsup>
     <mi>G</mi>
     <mn>1</mn>
     <mo>′</mo>
    </msubsup>
   </mpadded>
   <mo>→</mo>
   <mi mathvariant="normal">⋯</mi>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <and></and>
    <apply>
     <apply>
      <csymbol cd="ambiguous">superscript</csymbol>
      <ci>normal-→</ci>
      <apply>
       <csymbol cd="ambiguous">subscript</csymbol>
       <ci>T</ci>
       <cn type="integer">1</cn>
      </apply>
     </apply>
     <apply>
      <csymbol cd="ambiguous">subscript</csymbol>
      <ci>G</ci>
      <cn type="integer">1</cn>
     </apply>
     <apply>
      <csymbol cd="ambiguous">subscript</csymbol>
      <ci>P</ci>
      <cn type="integer">1</cn>
     </apply>
    </apply>
    <apply>
     <apply>
      <csymbol cd="ambiguous">superscript</csymbol>
      <ci>normal-→</ci>
      <apply>
       <csymbol cd="ambiguous">subscript</csymbol>
       <ci>T</ci>
       <cn type="integer">2</cn>
      </apply>
     </apply>
     <share href="#.cmml">
     </share>
     <apply>
      <csymbol cd="ambiguous">subscript</csymbol>
      <ci>P</ci>
      <cn type="integer">2</cn>
     </apply>
    </apply>
    <apply>
     <apply>
      <csymbol cd="ambiguous">superscript</csymbol>
      <ci>normal-→</ci>
      <apply>
       <csymbol cd="ambiguous">subscript</csymbol>
       <ci>T</ci>
       <cn type="integer">3</cn>
      </apply>
     </apply>
     <share href="#.cmml">
     </share>
     <apply>
      <csymbol cd="ambiguous">subscript</csymbol>
      <ci>G</ci>
      <cn type="integer">2</cn>
     </apply>
    </apply>
    <apply>
     <apply>
      <csymbol cd="ambiguous">superscript</csymbol>
      <ci>normal-→</ci>
      <apply>
       <csymbol cd="ambiguous">subscript</csymbol>
       <ci>T</ci>
       <cn type="integer">4</cn>
      </apply>
     </apply>
     <share href="#.cmml">
     </share>
     <apply>
      <csymbol cd="ambiguous">superscript</csymbol>
      <apply>
       <csymbol cd="ambiguous">subscript</csymbol>
       <ci>G</ci>
       <cn type="integer">1</cn>
      </apply>
      <ci>normal-′</ci>
     </apply>
    </apply>
    <apply>
     <ci>normal-→</ci>
     <share href="#.cmml">
     </share>
     <ci>normal-⋯</ci>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   G_{1}\;\stackrel{T_{1}}{\rightarrow}\;P_{1}\;\stackrel{T_{2}}{\rightarrow}\;P_%
{2}\;\stackrel{T_{3}}{\rightarrow}\;G_{2}\;\stackrel{T_{4}}{\rightarrow}\;G_{1%
}^{\prime}\;\rightarrow\cdots
  </annotation>
 </semantics>
</math>

</p>

<p>(adapted from Lewontin 1974, p. 12). XD</p>

<p><em>T</em><sub>1</sub> represents the genetic and <a class="uri" href="epigenetic" title="wikilink">epigenetic</a> laws, the aspects of functional biology, or <a href="developmental_biology" title="wikilink">development</a>, that transform a genotype into phenotype. We will refer to this as the "<a href="genotype-phenotype_map" title="wikilink">genotype-phenotype map</a>". <em>T</em><sub>2</sub> is the transformation due to natural selection, <em>T</em><sub>3</sub> are epigenetic relations that predict genotypes based on the selected phenotypes and finally <em>T</em><sub>4</sub> the rules of Mendelian genetics.</p>

<p>In practice, there are two bodies of evolutionary theory that exist in parallel, traditional population genetics operating in the genotype space and the <a href="biometry" title="wikilink">biometric</a> theory used in <a href="plant_breeding" title="wikilink">plant</a> and <a href="animal_breeding" title="wikilink">animal breeding</a>, operating in phenotype space. The missing part is the mapping between the genotype and phenotype space. This leads to a "sleight of hand" (as Lewontin terms it) whereby variables in the equations of one domain, are considered parameters or <em>constants</em>, where, in a full-treatment they would be transformed themselves by the evolutionary process and are in reality <em>functions</em> of the state variables in the other domain. The "sleight of hand" is assuming that we know this mapping. Proceeding as if we do understand it is enough to analyze many cases of interest. For example, if the phenotype is almost one-to-one with genotype (<a href="sickle-cell_disease" title="wikilink">sickle-cell disease</a>) or the time-scale is sufficiently short, the "constants" can be treated as such; however, there are many situations where it is inaccurate.</p>
<h3 id="linkage">Linkage</h3>

<p>If all genes are in <a href="linkage_equilibrium" title="wikilink">linkage equilibrium</a>, the effect of an allele at one locus can be averaged across the <a href="gene_pool" title="wikilink">gene pool</a> at other loci. In reality, one allele is frequently found in <a href="linkage_disequilibrium" title="wikilink">linkage disequilibrium</a> with genes at other loci, especially with genes located nearby on the same chromosome. <a href="Recombination_(biology)" title="wikilink">Recombination</a> breaks up this linkage disequilibrium too slowly to avoid <a href="genetic_hitchhiking" title="wikilink">genetic hitchhiking</a>, where an allele at one locus rises to high frequency because it is <a href="genetic_linkage" title="wikilink">linked</a> to an allele under selection at a nearby locus. This is a problem for population genetic models that treat one gene locus at a time. It can, however, be exploited as a method for detecting the action of <a href="natural_selection" title="wikilink">natural selection</a> via <a href="selective_sweep" title="wikilink">selective sweeps</a>.</p>

<p>In the extreme case of primarily <a href="asexual_reproduction" title="wikilink">asexual populations</a>, linkage is complete, and different population genetic equations can be derived and solved, which behave quite differently from the sexual case.<a class="footnoteRef" href="#fn77" id="fnref77"><sup>77</sup></a> Most <a href="microorganisms" title="wikilink">microbes</a>, such as <a class="uri" href="bacteria" title="wikilink">bacteria</a>, are asexual. The population genetics of <a href="microorganism" title="wikilink">microorganisms</a> lays the foundations for tracking the origin and evolution of <a href="antibiotic_resistance" title="wikilink">antibiotic resistance</a> and deadly infectious <a href="pathogen" title="wikilink">pathogens</a>. Population genetics of microorganisms is also an essential factor for devising strategies for the conservation and better utilization of beneficial microbes (Xu, 2010).</p>
<h2 id="see-also">See also</h2>
<h2 id="notes-and-references">Notes and references</h2>
<h2 id="bibliography">Bibliography</h2>
<ul>
<li>J. Beatty. "The synthesis and the synthetic theory" in Integrating Scientific Disciplines, edited by W. Bechtel and Nijhoff. Dordrecht, 1986.</li>
<li></li>
<li></li>
<li><a href="Luigi_Luca_Cavalli-Sforza" title="wikilink">Luigi Luca Cavalli-Sforza</a>. Genes, Peoples, and Languages. North Point Press, 2000.</li>
<li><a href="Luigi_Luca_Cavalli-Sforza" title="wikilink">Luigi Luca Cavalli-Sforza</a> et al. The History and Geography of Human Genes. Princeton University Press, 1994.</li>
<li><a href="James_F._Crow" title="wikilink">James F. Crow</a> and <a href="Motoo_Kimura" title="wikilink">Motoo Kimura</a>. Introduction to Population Genetics Theory. Harper &amp; Row, 1972.</li>
<li><a href="Warren_Ewens" title="wikilink">Warren J Ewens</a>. Mathematical Population Genetics. Springer-Verlag New York, Inc., 2004. ISBN 0-387-20191-2</li>
<li><a href="John_H._Gillespie" title="wikilink">John H. Gillespie</a> Population Genetics: A Concise Guide, Johns Hopkins Press, 1998. ISBN 0-8018-5755-4.</li>
<li>Richard Halliburton. Introduction to Population Genetics. Prentice Hall, 2004</li>
<li>Daniel Hartl. Primer of Population Genetics, 3rd edition. Sinauer, 2000. ISBN 0-87893-304-2</li>
<li>Daniel Hartl and Andrew Clark. Principles of Population Genetics, 3rd edition. Sinauer, 1997. ISBN 0-87893-306-9.</li>
<li></li>
<li><a href="Richard_Lewontin" title="wikilink">Richard C. Lewontin</a>. The Genetic Basis of Evolutionary Change. <a href="Columbia_University_Press" title="wikilink">Columbia University Press</a>, 1974.</li>
<li><a href="William_B._Provine" title="wikilink">William B. Provine</a>. The Origins of Theoretical Population Genetics. <a href="University_of_Chicago_Press" title="wikilink">University of Chicago Press</a>. 1971. ISBN 0-226-68464-4.</li>
<li></li>
<li><a href="Spencer_Wells" title="wikilink">Spencer Wells</a>. The Journey of Man. Random House, 2002.</li>
<li><a href="Spencer_Wells" title="wikilink">Spencer Wells</a>. Deep Ancestry: Inside the Genographic Project. National Geographic Society, 2006.</li>
<li></li>
<li>Xu, J. Microbial Population Genetics. <a href="Caister_Academic_Press" title="wikilink">Caister Academic Press</a>, 2010. ISBN 978-1-904455-59-2</li>
</ul>
<h2 id="external-links">External links</h2>
<ul>
<li><a href="http://www.extension.org/pages/68167/population-development-and-genetics">Population Genetics Tutorials</a></li>
<li><a href="http://alfred.med.yale.edu/alfred/">The ALlele FREquency Database</a> at <a href="Yale_University" title="wikilink">Yale University</a></li>
<li><a href="http://www.ehstrafd.org">EHSTRAFD.org - Earth Human STR Allele Frequencies Database</a></li>
<li><a href="http://www.esp.org/books/sturt/history/contents/sturt-history-ch-17.pdf">History of population genetics</a></li>
<li><a href="http://www.cosmolearning.com/video-lectures/how-selection-changes-the-genetic-composition-of-population-6688/">How Selection Changes the Genetic Composition of Population</a>, video of lecture by <a href="Stephen_C._Stearns" title="wikilink">Stephen C. Stearns</a> (<a href="Yale_University" title="wikilink">Yale University</a>)</li>
<li><a href="National_Geographic_Society" title="wikilink">National Geographic</a>: <a href="https://www5.nationalgeographic.com/genographic/atlas.html">Atlas of the Human Journey</a> (<a class="uri" href="Haplogroup" title="wikilink">Haplogroup</a>-based human migration maps)</li>
<li><a href="http://vlab.infotech.monash.edu.au/simulations/cellular-automata/population-genetics/">Monash Virtual Laboratory</a> - Simulations of habitat fragmentation and population genetics online at Monash University's Virtual Laboratory.</li>
</ul>

<p>"</p>

<p><a class="uri" href="Category:Genetics" title="wikilink">Category:Genetics</a> <a href="Category:Population_genetics" title="wikilink">*</a> <a href="Category:Evolutionary_biology" title="wikilink">Category:Evolutionary biology</a> <a href="Category:Statistical_genetics" title="wikilink">Category:Statistical genetics</a></p>
<section class="footnotes">
<hr/>
<ol>
<li id="fn1"><a href="#fnref1">↩</a></li>
<li id="fn2"><a href="#fnref2">↩</a></li>
<li id="fn3"></li>
<li id="fn4"></li>
<li id="fn5"><a href="#fnref5">↩</a></li>
<li id="fn6"><a href="#fnref6">↩</a></li>
<li id="fn7"></li>
<li id="fn8"><a href="#fnref8">↩</a></li>
<li id="fn9"></li>
<li id="fn10"></li>
<li id="fn11"><a href="#fnref11">↩</a></li>
<li id="fn12"><a href="#fnref12">↩</a></li>
<li id="fn13"><a href="#fnref13">↩</a></li>
<li id="fn14"><a href="#fnref14">↩</a></li>
<li id="fn15"><a href="#fnref15">↩</a></li>
<li id="fn16"><a href="#fnref16">↩</a></li>
<li id="fn17"><a href="#fnref17">↩</a></li>
<li id="fn18"><a href="#fnref18">↩</a></li>
<li id="fn19"><a href="#fnref19">↩</a></li>
<li id="fn20"><a href="#fnref20">↩</a></li>
<li id="fn21"><a href="#fnref21">↩</a></li>
<li id="fn22"><a href="#fnref22">↩</a></li>
<li id="fn23"><a href="#fnref23">↩</a></li>
<li id="fn24"><a href="#fnref24">↩</a></li>
<li id="fn25"><a href="#fnref25">↩</a></li>
<li id="fn26"><a href="#fnref26">↩</a></li>
<li id="fn27"><a href="#fnref27">↩</a></li>
<li id="fn28"><a href="#fnref28">↩</a></li>
<li id="fn29"><a href="#fnref29">↩</a></li>
<li id="fn30"><a href="#fnref30">↩</a></li>
<li id="fn31"><a href="#fnref31">↩</a></li>
<li id="fn32"><a href="#fnref32">↩</a></li>
<li id="fn33"><a href="#fnref33">↩</a></li>
<li id="fn34"><a href="#fnref34">↩</a></li>
<li id="fn35"><a href="#fnref35">↩</a></li>
<li id="fn36"><a href="#fnref36">↩</a></li>
<li id="fn37"><a href="#fnref37">↩</a></li>
<li id="fn38"><a href="#fnref38">↩</a></li>
<li id="fn39"><a href="#fnref39">↩</a></li>
<li id="fn40"><a href="#fnref40">↩</a></li>
<li id="fn41"><a href="#fnref41">↩</a></li>
<li id="fn42"><a href="#fnref42">↩</a></li>
<li id="fn43"><a href="#fnref43">↩</a></li>
<li id="fn44"><a href="#fnref44">↩</a></li>
<li id="fn45"><a href="#fnref45">↩</a></li>
<li id="fn46"><a href="#fnref46">↩</a></li>
<li id="fn47"><a href="#fnref47">↩</a></li>
<li id="fn48"><a href="#fnref48">↩</a></li>
<li id="fn49"><a href="#fnref49">↩</a></li>
<li id="fn50"><a href="#fnref50">↩</a></li>
<li id="fn51"><a href="#fnref51">↩</a></li>
<li id="fn52"><a href="#fnref52">↩</a></li>
<li id="fn53"><a href="#fnref53">↩</a></li>
<li id="fn54"><a href="#fnref54">↩</a></li>
<li id="fn55"><a href="#fnref55">↩</a></li>
<li id="fn56"><a href="#fnref56">↩</a></li>
<li id="fn57"><a href="#fnref57">↩</a></li>
<li id="fn58"><a href="#fnref58">↩</a></li>
<li id="fn59"><a href="#fnref59">↩</a></li>
<li id="fn60"><a href="#fnref60">↩</a></li>
<li id="fn61"><a href="#fnref61">↩</a></li>
<li id="fn62"><a href="#fnref62">↩</a></li>
<li id="fn63"><a href="#fnref63">↩</a></li>
<li id="fn64"><a href="#fnref64">↩</a></li>
<li id="fn65"><a href="#fnref65">↩</a></li>
<li id="fn66"><a href="#fnref66">↩</a></li>
<li id="fn67"><a href="#fnref67">↩</a></li>
<li id="fn68"><a href="#fnref68">↩</a></li>
<li id="fn69"><a href="#fnref69">↩</a></li>
<li id="fn70"><a href="#fnref70">↩</a></li>
<li id="fn71"><a href="#fnref71">↩</a></li>
<li id="fn72"><a href="#fnref72">↩</a></li>
<li id="fn73"><a href="#fnref73">↩</a></li>
<li id="fn74"><a href="#fnref74">↩</a></li>
<li id="fn75"><a href="#fnref75">↩</a></li>
<li id="fn76"><a href="#fnref76">↩</a></li>
<li id="fn77"><a href="#fnref77">↩</a></li>
</ol>
</section>
</body>
</html>
