<html lang="en">
<head>
<meta charset="utf-8"/>
<title offset="1246">Predictive genomics</title>
<script src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_SVG.js" type="text/javascript">
</script>
</head>
<body>
<h1>Predictive genomics</h1>
<hr/>

<p><strong>Predictive genomics</strong> is at the intersection of multiple disciplines: <a href="predictive_medicine" title="wikilink">predictive medicine</a>, <a href="personal_genomics" title="wikilink">personal genomics</a> and <a href="translational_bioinformatics" title="wikilink">translational bioinformatics</a>. Specifically, predictive genomics deals with the future <a class="uri" href="phenotypic" title="wikilink">phenotypic</a> outcomes via <a class="uri" href="prediction" title="wikilink">prediction</a> in areas such as complex multifactorial <a href="disease" title="wikilink">diseases</a> in humans.<a class="footnoteRef" href="#fn1" id="fnref1"><sup>1</sup></a> To date, the success of predictive genomics has been dependent on the genetic framework underlying these applications, typically explored in <a href="genome-wide_association_study" title="wikilink">genome-wide association</a> (GWA) studies.<a class="footnoteRef" href="#fn2" id="fnref2"><sup>2</sup></a> The identification of associated <a href="single-nucleotide_polymorphism" title="wikilink">single-nucleotide polymorphisms</a> (variation of a DNA sequence in a population) underpin GWA studies in complex diseases that have ranged from <a href="Diabetes_mellitus_type_2" title="wikilink">Type 2 Diabetes</a> (T2D), <a href="Macular_degeneration" title="wikilink">Age-related macular degeneration</a> (AMD) and <a href="Crohn’s_Disease" title="wikilink">Crohn’s Disease</a>.</p>

<p>Although the <a href="Human_Genome_Project" title="wikilink">Human Genome Project</a> has progressively improved the fidelity of sequence determination, the overbearing complexity of the genome hinders the identification of associated or ultimately causal variants.<a class="footnoteRef" href="#fn3" id="fnref3"><sup>3</sup></a> In particular, there are likely to be a large number of implicated <a href="Locus_(genetics)" title="wikilink">genetic loci</a> which exhibit small marginal effects.<a class="footnoteRef" href="#fn4" id="fnref4"><sup>4</sup></a></p>
<h2 id="objectives">Objectives</h2>

<p>A number of short- and long-term goals exist for predictive genomics. The identification of associated variants underpin all other downstream endeavors that point toward better data-cum-knowledge outcomes. In particular, those outcomes that facilitate clinical improvement and individualised healthcare further lead to actionable measures in <a href="Medical_diagnosis" title="wikilink">diagnosis</a>, <a class="uri" href="prognosis" title="wikilink">prognosis</a> and <a href="Preventive_healthcare" title="wikilink">prevention</a>. </p>
<h3 id="identify-associated-variants-to-disease">Identify associated variants to disease</h3>

<p>Whilst the single-gene, single-disease hypothesis holds for Mendelian disorders such as <a href="Huntington's_disease" title="wikilink">Huntington's disease</a> and <a href="Cystic_Fibrosis" title="wikilink">Cystic Fibrosis</a>, complex diseases and traits are affected by a number of gene loci and genetic variants with varying risk.<a class="footnoteRef" href="#fn5" id="fnref5"><sup>5</sup></a> A precursor to the development of preventative, prognostic and diagnostic tools in these diseases requires mapping genetic loci in disease etiology and discovering causal mutations.<a class="footnoteRef" href="#fn6" id="fnref6"><sup>6</sup></a> Creating a ‘genomic profile’ of individuals with the number of variants at the genome-wide level facilitates not only the prediction of disease prior to onset, but also serves as a primer to increasing the knowledge of causal variants.<a class="footnoteRef" href="#fn7" id="fnref7"><sup>7</sup></a></p>

<p>The foremost difficulty in achieving this goal is to understand the functionality of these variants with respect to areas of physiological and molecular importance in conjunction with phenotype.<a class="footnoteRef" href="#fn8" id="fnref8"><sup>8</sup></a> If associated variants are mapped to sequences with unknown function, then this restricts the ability for specific targeting in areas of interest. Therefore the ability for predictive genomics to succeed also depends upon other related areas such as the functional annotation of the genome (<a class="uri" href="ENCODE" title="wikilink">ENCODE</a>).<a class="footnoteRef" href="#fn9" id="fnref9"><sup>9</sup></a></p>
<h3 id="translation-research-to-clinical">Translation: research to clinical</h3>

<p>The identification of causal variants, genes and pathways leads to opportunities that bridge the divide between research and clinical usage. If successful, the subsequent discovery of therapeutic targets within implicated biological pathways have consequences for both treatment and prevention.<a class="footnoteRef" href="#fn10" id="fnref10"><sup>10</sup></a> Furthermore, the downstream effect of identifying disease-relevant biomarkers allow for improvements to monitoring disease progression and response-to-treatment, where the implementation of these results into <a href="clinical_decision_support_system" title="wikilink">clinical decision support systems</a> (CDSS) facilitate personalised medicine and outcomes.<a class="footnoteRef" href="#fn11" id="fnref11"><sup>11</sup></a> Even if only marginally effectual, the repeated replication of associated variants can offer significant translational value.<a class="footnoteRef" href="#fn12" id="fnref12"><sup>12</sup></a></p>
<h3 id="individualising-healthcare">Individualising healthcare</h3>

<p>The significance of translation from research to clinical usage relates to use of the complete knowledge of an individual to develop personalised approaches to disease management. The caveat with this is that there have been difficulties in both prediction and inference for complex diseases.<a class="footnoteRef" href="#fn13" id="fnref13"><sup>13</sup></a> Therefore, unless individuals have an overwhelming high or low number of risk <a class="uri" href="alleles" title="wikilink">alleles</a>, there is a limit to the predictive accuracy of their ‘genomic profiles’. However, preliminary examples of predictive genomics for personalising healthcare include: using an individuals <a href="gene_expression" title="wikilink">gene expression</a> data to monitor progress to treatment,<a class="footnoteRef" href="#fn14" id="fnref14"><sup>14</sup></a> or using the genomic profile of the P450 drug metabolising system of individuals to assist dosage and selection.<a class="footnoteRef" href="#fn15" id="fnref15"><sup>15</sup></a></p>
<h2 id="applications-in-complex-human-diseases">Applications in complex human diseases</h2>

<p>In the table below is a performance comparison of diseases selected on disease frequency and known heritability estimates, with use of <a href="single-nucleotide_polymorphism" title="wikilink">single-nucleotide polymorphism</a> (SNP) based models reflecting known genetic factors for a European population (subject to change as more associations are discovered).<a class="footnoteRef" href="#fn16" id="fnref16"><sup>16</sup></a> 

<math display="inline" id="Predictive_genomics:0">
 <semantics>
  <mi>K</mi>
  <annotation-xml encoding="MathML-Content">
   <ci>K</ci>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   K
  </annotation>
 </semantics>
</math>

 denotes lifetime morbid risk, 

<math display="inline" id="Predictive_genomics:1">
 <semantics>
  <msubsup>
   <mi>h</mi>
   <mi>L</mi>
   <mn>2</mn>
  </msubsup>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <csymbol cd="ambiguous">subscript</csymbol>
    <apply>
     <csymbol cd="ambiguous">superscript</csymbol>
     <ci>h</ci>
     <cn type="integer">2</cn>
    </apply>
    <ci>L</ci>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   h^{2}_{L}
  </annotation>
 </semantics>
</math>

 denotes heritability of liability, 

<math display="inline" id="Predictive_genomics:2">
 <semantics>
  <mrow>
   <mi>A</mi>
   <mi>U</mi>
   <mi>C</mi>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <times></times>
    <ci>A</ci>
    <ci>U</ci>
    <ci>C</ci>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   AUC
  </annotation>
 </semantics>
</math>

 denotes area under the ROC curve.</p>
<table>
<thead>
<tr class="header">
<th style="text-align: left;">
<p>Disease</p></th>
<th style="text-align: left;">
<p>

<math display="inline" id="Predictive_genomics:3">
 <semantics>
  <mi>K</mi>
  <annotation-xml encoding="MathML-Content">
   <ci>K</ci>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   K
  </annotation>
 </semantics>
</math>

</p></th>
<th style="text-align: left;">
<p>

<math display="inline" id="Predictive_genomics:4">
 <semantics>
  <msubsup>
   <mi>h</mi>
   <mi>L</mi>
   <mn>2</mn>
  </msubsup>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <csymbol cd="ambiguous">subscript</csymbol>
    <apply>
     <csymbol cd="ambiguous">superscript</csymbol>
     <ci>h</ci>
     <cn type="integer">2</cn>
    </apply>
    <ci>L</ci>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   h^{2}_{L}
  </annotation>
 </semantics>
</math>

</p></th>
<th style="text-align: left;">
<p>

<math display="inline" id="Predictive_genomics:5">
 <semantics>
  <mrow>
   <mi>A</mi>
   <mi>U</mi>
   <mi>C</mi>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <times></times>
    <ci>A</ci>
    <ci>U</ci>
    <ci>C</ci>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   AUC
  </annotation>
 </semantics>
</math>

</p></th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;">
<p>Coronary artery disease</p></td>
<td style="text-align: left;">
<p>0.402</p></td>
<td style="text-align: left;">
<p>0.49</p></td>
<td style="text-align: left;">
<p>0.584</p></td>
</tr>
<tr class="even">
<td style="text-align: left;">
<p>Type 2 diabetes</p></td>
<td style="text-align: left;">
<p>0.339</p></td>
<td style="text-align: left;">
<p>0.30</p></td>
<td style="text-align: left;">
<p>0.592</p></td>
</tr>
<tr class="odd">
<td style="text-align: left;">
<p>Atrial fibrillation</p></td>
<td style="text-align: left;">
<p>0.245</p></td>
<td style="text-align: left;">
<p>0.62</p></td>
<td style="text-align: left;">
<p>0.593</p></td>
</tr>
<tr class="even">
<td style="text-align: left;">
<p>Stroke</p></td>
<td style="text-align: left;">
<p>0.190</p></td>
<td style="text-align: left;">
<p>0.17</p></td>
<td style="text-align: left;">
<p>0.528</p></td>
</tr>
<tr class="odd">
<td style="text-align: left;">
<p>Prostate cancer</p></td>
<td style="text-align: left;">
<p>0.165</p></td>
<td style="text-align: left;">
<p>0.42</p></td>
<td style="text-align: left;">
<p>0.614</p></td>
</tr>
<tr class="even">
<td style="text-align: left;">
<p>Alzheimer disease</p></td>
<td style="text-align: left;">
<p>0.132</p></td>
<td style="text-align: left;">
<p>0.79</p></td>
<td style="text-align: left;">
<p>0.648</p></td>
</tr>
<tr class="odd">
<td style="text-align: left;">
<p>Breast cancer</p></td>
<td style="text-align: left;">
<p>0.123</p></td>
<td style="text-align: left;">
<p>0.25</p></td>
<td style="text-align: left;">
<p>0.586</p></td>
</tr>
<tr class="even">
<td style="text-align: left;">
<p>Lung cancer</p></td>
<td style="text-align: left;">
<p>0.069</p></td>
<td style="text-align: left;">
<p>0.08</p></td>
<td style="text-align: left;">
<p>0.525</p></td>
</tr>
<tr class="odd">
<td style="text-align: left;">
<p>Bipolar disorder</p></td>
<td style="text-align: left;">
<p>0.051</p></td>
<td style="text-align: left;">
<p>0.60</p></td>
<td style="text-align: left;">
<p>0.550</p></td>
</tr>
<tr class="even">
<td style="text-align: left;">
<p>Colorectal cancer</p></td>
<td style="text-align: left;">
<p>0.051</p></td>
<td style="text-align: left;">
<p>0.13</p></td>
<td style="text-align: left;">
<p>0.564</p></td>
</tr>
<tr class="odd">
<td style="text-align: left;">
<p>Age-related macular degeneration</p></td>
<td style="text-align: left;">
<p>0.047</p></td>
<td style="text-align: left;">
<p>0.71</p></td>
<td style="text-align: left;">
<p>0.758</p></td>
</tr>
<tr class="even">
<td style="text-align: left;">
<p>Bladder cancer</p></td>
<td style="text-align: left;">
<p>0.024</p></td>
<td style="text-align: left;">
<p>0.08</p></td>
<td style="text-align: left;">
<p>0.577</p></td>
</tr>
<tr class="odd">
<td style="text-align: left;">
<p>Multiple sclerosis</p></td>
<td style="text-align: left;">
<p>0.020</p></td>
<td style="text-align: left;">
<p>0.51</p></td>
<td style="text-align: left;">
<p>0.622</p></td>
</tr>
<tr class="even">
<td style="text-align: left;">
<p>Melanoma</p></td>
<td style="text-align: left;">
<p>0.020</p></td>
<td style="text-align: left;">
<p>0.21</p></td>
<td style="text-align: left;">
<p>0.640</p></td>
</tr>
<tr class="odd">
<td style="text-align: left;">
<p>Type 1 diabetes</p></td>
<td style="text-align: left;">
<p>0.018</p></td>
<td style="text-align: left;">
<p>0.87</p></td>
<td style="text-align: left;">
<p>0.638</p></td>
</tr>
<tr class="even">
<td style="text-align: left;">
<p>Parkinson disease</p></td>
<td style="text-align: left;">
<p>0.016</p></td>
<td style="text-align: left;">
<p>0.27</p></td>
<td style="text-align: left;">
<p>0.592</p></td>
</tr>
<tr class="odd">
<td style="text-align: left;">
<p>Pancreatic cancer</p></td>
<td style="text-align: left;">
<p>0.015</p></td>
<td style="text-align: left;">
<p>0.36</p></td>
<td style="text-align: left;">
<p>0.557</p></td>
</tr>
<tr class="even">
<td style="text-align: left;">
<p>Ovarian cancer</p></td>
<td style="text-align: left;">
<p>0.014</p></td>
<td style="text-align: left;">
<p>0.22</p></td>
<td style="text-align: left;">
<p>0.548</p></td>
</tr>
<tr class="odd">
<td style="text-align: left;">
<p>Thyroid cancer</p></td>
<td style="text-align: left;">
<p>0.010</p></td>
<td style="text-align: left;">
<p>0.53</p></td>
<td style="text-align: left;">
<p>0.614</p></td>
</tr>
<tr class="even">
<td style="text-align: left;">
<p>Ulcerative colitis</p></td>
<td style="text-align: left;">
<p>0.009</p></td>
<td style="text-align: left;">
<p>0.53</p></td>
<td style="text-align: left;">
<p>0.666</p></td>
</tr>
<tr class="odd">
<td style="text-align: left;">
<p>Schizophrenia</p></td>
<td style="text-align: left;">
<p>0.007</p></td>
<td style="text-align: left;">
<p>0.66</p></td>
<td style="text-align: left;">
<p>0.540</p></td>
</tr>
<tr class="even">
<td style="text-align: left;">
<p>Celiac disease</p></td>
<td style="text-align: left;">
<p>0.007</p></td>
<td style="text-align: left;">
<p>0.75</p></td>
<td style="text-align: left;">
<p>0.733</p></td>
</tr>
<tr class="odd">
<td style="text-align: left;">
<p>Crohn's disease</p></td>
<td style="text-align: left;">
<p>0.005</p></td>
<td style="text-align: left;">
<p>0.56</p></td>
<td style="text-align: left;">
<p>0.717</p></td>
</tr>
</tbody>
</table>

<p>In the applications of predictive genomics below, these complex diseases either lack or are lacking reliable diagnostics for disease. Given the medical consequences of these diseases, the economic impact is also significant. However, none of the use cases below has been translated into the clinic.</p>
<h3 id="age-related-macular-degeneration">Age-related macular degeneration</h3>

<p><a href="Macular_degeneration" title="wikilink">Age-related macular degeneration</a> (AMD) is one of the flagship complex diseases from the genomic revolution with over 19 associated genetic loci replicated in GWA studies.<a class="footnoteRef" href="#fn17" id="fnref17"><sup>17</sup></a> In particular, the first significant genetic risk variant was identified in the complement factor H(<em>CFH</em>) gene in 2005 motivating the search for more genetic variants in the disease. Over the past decade, a number of models have been proposed to assess individual risk to AMD.<a class="footnoteRef" href="#fn18" id="fnref18"><sup>18</sup></a> The genetic predisposition of AMD risk varies from 45% to 71% where highly effectual <a href="odds_ratio" title="wikilink">odds ratios</a> (OR) have been reported (greater than 2.0 per allele in some cases).<a class="footnoteRef" href="#fn19" id="fnref19"><sup>19</sup></a> In 2013, a comprehensive case-control GWA study with approximately 77,000 observations involving 18 international research groups from the International AMD Genetics Consortium implicated 19 gene loci and 9 <a href="biological_pathway" title="wikilink">biological pathways</a> including the regulation of complement, <a href="lipid_metabolism" title="wikilink">lipid metabolism</a> and <a href="Angiogenesis" title="wikilink">angiogenic</a> activity. The predictive performance of the full model including all 19 loci exhibited 0.74 <a href="Area_under_the_curve" title="wikilink">AUC</a> - according to Jakobsdottir et al., 0.75 AUC is sufficient to distinguish between extreme cases and controls.<a class="footnoteRef" href="#fn20" id="fnref20"><sup>20</sup></a> In particular, of the 19 associated gene loci, there were 7 newly discovered loci, which the authors point to as additional entry points into AMD etiology and <a href="Biological_target#Drug_targets" title="wikilink">drug targets</a>.<a class="footnoteRef" href="#fn21" id="fnref21"><sup>21</sup></a></p>
<h3 id="type-2-diabetes">Type 2 diabetes</h3>

<p><a href="Diabetes_mellitus_type_2" title="wikilink">Type 2 diabetes</a> (T2D), an extremely common metabolic disorder, has demonstrated interplay between many environmental and genetic risk factors leading to disease onset.<a class="footnoteRef" href="#fn22" id="fnref22"><sup>22</sup></a> A number of risk assessment models incorporating a number of demographic, environmental and clinical risk factors are already shown to elicit reasonable discrimination in case-control studies; it has been proposed that identifying genetic variants that contribute to T2D as for standalone prediction or in conjunction with current risk models can improve prediction of T2D risk, if current models lack sufficient coverage of the full effect of an individual's genotype.<a class="footnoteRef" href="#fn23" id="fnref23"><sup>23</sup></a> Approximately 20 associated SNPs have been replicated in T2D; however, their effect sizes do not seem to be substantial: OR 1.37 for SNPs in the <em>TCF7L2</em> gene purported to give highest genetic risk.<a class="footnoteRef" href="#fn24" id="fnref24"><sup>24</sup></a></p>

<p>In 2009, a study was conducted on the WTCCC (GWA study involving 7 cohorts with 7 diseases: including <a href="bipolar_disorder" title="wikilink">bipolar disorder</a>, <a href="Crohn’s_disease" title="wikilink">Crohn’s disease</a>, <a class="uri" href="hypertension" title="wikilink">hypertension</a>, <a href="rheumatoid_arthritis" title="wikilink">rheumatoid arthritis</a>, <a href="Diabetes_mellitus_type_1" title="wikilink">Type I Diabetes</a> (T1D) and <a href="Diabetes_mellitus_type_2" title="wikilink">Type II Diabetes</a> (T2D)).<a class="footnoteRef" href="#fn25" id="fnref25"><sup>25</sup></a> With particular attention to T2D, Evans et al. were able to discern a marginal increase in AUC (+0.04) based on genome-wide information with respect to known susceptible variants.<a class="footnoteRef" href="#fn26" id="fnref26"><sup>26</sup></a> However, non-genetic based tests such as the Cambridge and Framingham offspring risk scores have been purported to perform better than genetic-risk models with 20 loci.<a class="footnoteRef" href="#fn27" id="fnref27"><sup>27</sup></a> Moreover, the addition of genetic risk with these phenotypical models did not produce statistically significant AUC results.</p>
<h3 id="celiac-disease">Celiac disease</h3>

<p><a href="Celiac_disease" title="wikilink">Celiac disease</a> (CD) is a complex immune disorder that has been found to have strong genetic links in disease. In particular, <a href="human_leukocyte_antigen" title="wikilink">human leukocyte antigen</a> (HLA) genes are strongly implicated in CD development and HLA testing is undertaken in clinical practice. However, although there are serological and histological tests available for CD, these clinical screenings have been found to generate false positives.<a class="footnoteRef" href="#fn28" id="fnref28"><sup>28</sup></a> In 2014, Abraham et al. used a genomic risk score (GRS) generated over 6 cohorts with an AUC of 0.86 to 0.90.<a class="footnoteRef" href="#fn29" id="fnref29"><sup>29</sup></a></p>
<h2 id="limitations">Limitations</h2>

<p>For predictive genomics to address their objectives, there must be an improvement in the accuracy of prediction through added methods or improvements to current techniques and to demonstrate that there is bonafide improvement in patient outcomes. Currently, although <a href="Receiver_operating_characteristic#Area_under_the_curve" title="wikilink">AUC</a> (Area Under the ROC) is the de facto metric in comparing and evaluating the performance of predictive models, there is no consensus as to what kind of score is sufficient for clinical use. Jakobsdottir et al. states that 0.75 AUC is sufficient for discriminating between clear cases and controls; however, this is still arbitrary.<a class="footnoteRef" href="#fn30" id="fnref30"><sup>30</sup></a> The <a href="Positive_and_negative_predictive_values#Positive_predictive_value" title="wikilink">positive predictive value</a> (PPV) must be high enough to avoid a higher prevalence of false-positives.</p>
<h3 id="variants-in-prediction-snps-and-alternatives">Variants in prediction: SNPs and alternatives</h3>

<p>SNPs identified in GWA studies are considered to give better predictive performance if they have high effect sizes of Odds Ratios (OR). A case study involving 5 use cases of genomic prediction demonstrate that SNPs with extremely small p-values, and by implication extreme OR do not give extreme differences in discrimination.<a class="footnoteRef" href="#fn31" id="fnref31"><sup>31</sup></a> They point out that use of significantly associated genetic variants does not necessarily lead to better classification. Alternatively, <a href="Copy-number_variation" title="wikilink">CNV</a> (copy number variants) have been proposed to usurp SNPs as better candidates for prediction with <a href="Body_mass_index" title="wikilink">BMI</a> stratified across different ethnicities demonstrating better, although marginal, improvement of CNVs over SNPs for prediction.<a class="footnoteRef" href="#fn32" id="fnref32"><sup>32</sup></a> Furthermore, a comparison of over 10 complex disorders in prediction with respect to family history and SNPs for prediction did not suggest better discrimination with SNPs.<a class="footnoteRef" href="#fn33" id="fnref33"><sup>33</sup></a></p>
<h3 id="interacting-variants-higher-order-analysis">Interacting variants: higher order analysis</h3>

<p>Currently, the prevailing standard of risk models focus on univariate analysis rather than focusing upon interactions of higher order. Therefore although typical GWA studies are able to detect a number of statistically significant loci, they have not been sufficient to fully explain the estimates of theoretical genetic heritability.<a class="footnoteRef" href="#fn34" id="fnref34"><sup>34</sup></a> It has been demonstrated by Goudey et al. that both 2-way and 3-way interactions between SNPs are able to explain trait variance relative to single SNPs.<a class="footnoteRef" href="#fn35" id="fnref35"><sup>35</sup></a> Goudey et al. also states that the barrier to expansion of higher order interactions has been limited by the intractability of exhaustive search techniques (see <a class="uri" href="NP-complete" title="wikilink">NP-complete</a>).</p>
<h3 id="population-size-and-scope">Population: size and scope</h3>

<p>The issues surrounding sample size and number of variants become exacerbated particularly when GWA studies consider variants of volume in the order of millions. Therefore, due to the current constraints in the curse of dimensionality, prior screening methods that decrease the number of loci to below the number of observations may be used before modelling disease risk.<a class="footnoteRef" href="#fn36" id="fnref36"><sup>36</sup></a> Hayes et al. states that population size must be &gt;100,000 in order to achieve high accuracy under their model assumptions; the exception is the case where there is a small effective population size.<a class="footnoteRef" href="#fn37" id="fnref37"><sup>37</sup></a> Furthermore, ethnic specific GWA studies show that each group has varied detectability of variants in terms of: frequency, <a href="linkage_disequilibrium" title="wikilink">linkage disequilibrium</a> – the co-inheritance of SNPs through generations – and the actual loci themselves.<a class="footnoteRef" href="#fn38" id="fnref38"><sup>38</sup></a></p>
<h2 id="other-applications">Other applications</h2>

<p>Predictive genomics has not been constrained to prediction of complex diseases. For instance, Hayes et al. uses genomic prediction for livestock, crop and forage species selection, where predicted results are currently in use.<a class="footnoteRef" href="#fn39" id="fnref39"><sup>39</sup></a> Furthermore, Kambouris et al. discusses the use of ‘genomic profiles’ for the performance of elite athletes noting individualised and personalised training regimens for both dietary and physical aspects.<a class="footnoteRef" href="#fn40" id="fnref40"><sup>40</sup></a> Additionally, Kayser et al. point to <a href="DNA_profiling" title="wikilink">DNA profiling</a> in forensics as a beneficiary of the genomic revolution.<a class="footnoteRef" href="#fn41" id="fnref41"><sup>41</sup></a></p>
<h2 id="see-also">See also</h2>
<ul>
<li><a href="Predictive_medicine" title="wikilink">Predictive medicine</a></li>
<li><a class="uri" href="Pharmacogenomics" title="wikilink">Pharmacogenomics</a></li>
<li><a href="Clinical_decision_support_system" title="wikilink">Clinical decision support system</a></li>
<li><a href="Personal_genomics" title="wikilink">Personal genomics</a></li>
<li><a href="Human_Genome_Project" title="wikilink">Human Genome Project</a></li>
<li><a class="uri" href="Statistics" title="wikilink">Statistics</a></li>
<li><a href="Machine_learning" title="wikilink">Machine learning</a></li>
<li><a href="Receiver_operating_characteristic" title="wikilink">Receiver operating characteristic</a></li>
<li><a href="Odds_ratio" title="wikilink">Odds ratio</a></li>
<li><a href="Logistic_regression" title="wikilink">Logistic regression</a></li>
<li><a href="Single-nucleotide_polymorphism" title="wikilink">Single-nucleotide polymorphism</a></li>
<li><a href="Copy-number_variation" title="wikilink">Copy-number variation</a></li>
<li><a href="Translational_bioinformatics" title="wikilink">Translational bioinformatics</a></li>
</ul>
<h2 id="references">References</h2>

<p>"</p>

<p><a class="uri" href="Category:Genomics" title="wikilink">Category:Genomics</a></p>
<section class="footnotes">
<hr/>
<ol>
<li id="fn1"><a href="#fnref1">↩</a></li>
<li id="fn2"><a href="#fnref2">↩</a></li>
<li id="fn3"><a href="#fnref3">↩</a></li>
<li id="fn4"><a href="#fnref4">↩</a></li>
<li id="fn5"></li>
<li id="fn6"><a href="#fnref6">↩</a></li>
<li id="fn7"><a href="#fnref7">↩</a></li>
<li id="fn8"></li>
<li id="fn9"><a href="#fnref9">↩</a></li>
<li id="fn10"><a href="#fnref10">↩</a></li>
<li id="fn11"><a href="#fnref11">↩</a></li>
<li id="fn12"></li>
<li id="fn13"><a href="#fnref13">↩</a></li>
<li id="fn14"><a href="#fnref14">↩</a></li>
<li id="fn15"><a href="#fnref15">↩</a></li>
<li id="fn16"></li>
<li id="fn17"><a href="#fnref17">↩</a></li>
<li id="fn18"><a href="#fnref18">↩</a></li>
<li id="fn19"></li>
<li id="fn20"><a href="#fnref20">↩</a></li>
<li id="fn21"></li>
<li id="fn22"><a href="#fnref22">↩</a></li>
<li id="fn23"><a href="#fnref23">↩</a></li>
<li id="fn24"><a href="#fnref24">↩</a></li>
<li id="fn25"><a href="#fnref25">↩</a></li>
<li id="fn26"></li>
<li id="fn27"></li>
<li id="fn28"><a href="#fnref28">↩</a></li>
<li id="fn29"></li>
<li id="fn30"></li>
<li id="fn31"></li>
<li id="fn32"><a href="#fnref32">↩</a></li>
<li id="fn33"><a href="#fnref33">↩</a></li>
<li id="fn34"></li>
<li id="fn35"><a href="#fnref35">↩</a></li>
<li id="fn36"><a href="#fnref36">↩</a></li>
<li id="fn37"><a href="#fnref37">↩</a></li>
<li id="fn38"><a href="#fnref38">↩</a></li>
<li id="fn39"></li>
<li id="fn40"><a href="#fnref40">↩</a></li>
<li id="fn41"><a href="#fnref41">↩</a></li>
</ol>
</section>
</body>
</html>
