<html lang="en">
<head>
<meta charset="utf-8"/>
<title offset="987">Protein secondary structure</title>
<script src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_SVG.js" type="text/javascript">
</script>
</head>
<body>
<h1>Protein secondary structure</h1>
<hr/>

<p> In <a class="uri" href="biochemistry" title="wikilink">biochemistry</a> and <a href="structural_biology" title="wikilink">structural biology</a>, <strong>protein secondary structure</strong> is the general three-dimensional <a href="conformational_isomerism" title="wikilink">form</a> of <em>local segments</em> of <a href="protein" title="wikilink">proteins</a>. Secondary structure can be formally defined by the pattern of <a href="hydrogen_bond" title="wikilink">hydrogen bonds</a> of the protein (such as <a href="alpha_helice" title="wikilink">alpha helices</a> and <a href="beta_sheet" title="wikilink">beta sheets</a>) that are observed in an atomic-resolution structure. More specifically, the secondary structure is defined by the patterns of hydrogen bonds formed between <a class="uri" href="amine" title="wikilink">amine</a> hydrogen and <a class="uri" href="carbonyl" title="wikilink">carbonyl</a> oxygen atoms contained in the backbone <a href="peptide_bond" title="wikilink">peptide bonds</a> of the protein. The secondary structure may alternatively be defined based on the regular pattern of backbone <a href="dihedral_angle" title="wikilink">dihedral angles</a> in a particular region of the <a href="Ramachandran_plot" title="wikilink">Ramachandran plot</a>; thus, a segment of residues with such dihedral angles may be called a <a class="uri" href="helix" title="wikilink">helix</a>, regardless of whether it has the correct hydrogen bonds. The secondary structure may be provided by crystallographers in the corresponding <a href="Protein_Data_Bank_(file_format)" title="wikilink">PDB</a> file.</p>

<p>Secondary structure does not describe the specific identity of amino acids in the protein which are defined as the <a href="protein_primary_structure" title="wikilink">primary structure</a>, nor the <em>global</em> <a href="atom" title="wikilink">atomic</a> positions in three-dimensional space, which are considered to be <a href="protein_tertiary_structure" title="wikilink">tertiary structure</a>. Other types of <a href="biopolymer" title="wikilink">biopolymers</a> such as <a href="nucleic_acid" title="wikilink">nucleic acids</a> also possess characteristic <a href="nucleic_acid_secondary_structure" title="wikilink">secondary structures</a>.</p>

<p>The concept of secondary structure was first introduced by <a href="Kaj_Ulrik_Linderstrøm-Lang" title="wikilink">Kaj Ulrik Linderstrøm-Lang</a> at <a class="uri" href="Stanford" title="wikilink">Stanford</a> in 1952.<a class="footnoteRef" href="#fn1" id="fnref1"><sup>1</sup></a><a class="footnoteRef" href="#fn2" id="fnref2"><sup>2</sup></a></p>
<h2 id="types">Types</h2>
<table>
<tbody>
<tr class="odd">
<td style="text-align: left;">
<p>Structural features of the three major forms of protein helices<a class="footnoteRef" href="#fn3" id="fnref3"><sup>3</sup></a></p></td>
<td style="text-align: left;">
<p>Geometry attribute</p></td>
<td style="text-align: left;">
<p>α-helix</p></td>
<td style="text-align: left;">
<p>3<sub>10</sub> helix</p></td>
<td style="text-align: left;">
<p>π-helix</p></td>
</tr>
<tr class="even">
<td style="text-align: left;">
<p>Residues per turn</p></td>
<td style="text-align: left;">
<p>3.6</p></td>
<td style="text-align: left;">
<p>3.0</p></td>
<td style="text-align: left;">
<p>4.4</p></td>
</tr>
<tr class="odd">
<td style="text-align: left;">
<p>Translation per residue</p></td>
<td style="text-align: left;"></td>
<td style="text-align: left;"></td>
<td style="text-align: left;"></td>
</tr>
<tr class="even">
<td style="text-align: left;">
<p>Radius of helix</p></td>
<td style="text-align: left;"></td>
<td style="text-align: left;"></td>
<td style="text-align: left;"></td>
</tr>
<tr class="odd">
<td style="text-align: left;">
<p>Pitch</p></td>
<td style="text-align: left;"></td>
<td style="text-align: left;"></td>
<td style="text-align: left;"></td>
</tr>
</tbody>
</table>
<figure><b>(Figure)</b>
<figcaption>Hydrogen bonds (yellow dots) stabilizing an alpha-helix|120px</figcaption>
</figure>

<p>The most common secondary structures are <a href="alpha_helix" title="wikilink">alpha helices</a> and <a href="beta_sheet" title="wikilink">beta sheets</a>. Other helices, such as the <a href="310_helix" title="wikilink">3<sub>10</sub> helix</a> and <a href="pi_helix" title="wikilink">π helix</a>, are calculated to have energetically favorable hydrogen-bonding patterns but are rarely observed in natural proteins except at the ends of α helices due to unfavorable backbone packing in the center of the helix. Other extended structures such as the <a href="polyproline_helix" title="wikilink">polyproline helix</a> and <a href="alpha_sheet" title="wikilink">alpha sheet</a> are rare in <a href="native_state" title="wikilink">native state</a> proteins but are often hypothesized as important <a href="protein_folding" title="wikilink">protein folding</a> intermediates. Tight <a href="turn_(biochemistry)" title="wikilink">turns</a> and loose, flexible loops link the more "regular" secondary structure elements. The <a href="random_coil" title="wikilink">random coil</a> is not a true secondary structure, but is the class of conformations that indicate an absence of regular secondary structure.</p>

<p><a href="Amino_acid" title="wikilink">Amino acids</a> vary in their ability to form the various secondary structure elements. <a class="uri" href="Proline" title="wikilink">Proline</a> and <a class="uri" href="glycine" title="wikilink">glycine</a> are sometimes known as "helix breakers" because they disrupt the regularity of the α helical backbone conformation; however, both have unusual conformational abilities and are commonly found in <a href="turn_(biochemistry)" title="wikilink">turns</a>. Amino acids that prefer to adopt <a href="alpha_helix" title="wikilink">helical</a> conformations in proteins include <a class="uri" href="methionine" title="wikilink">methionine</a>, <a class="uri" href="alanine" title="wikilink">alanine</a>, <a class="uri" href="leucine" title="wikilink">leucine</a>, <a class="uri" href="glutamate" title="wikilink">glutamate</a> and <a class="uri" href="lysine" title="wikilink">lysine</a> ("MALEK" in <a href="amino_acid" title="wikilink">amino-acid</a> 1-letter codes); by contrast, the large aromatic residues (<a class="uri" href="tryptophan" title="wikilink">tryptophan</a>, <a class="uri" href="tyrosine" title="wikilink">tyrosine</a> and <a class="uri" href="phenylalanine" title="wikilink">phenylalanine</a>) and C<sup>β</sup>-branched amino acids (<a class="uri" href="isoleucine" title="wikilink">isoleucine</a>, <a class="uri" href="valine" title="wikilink">valine</a>, and <a class="uri" href="threonine" title="wikilink">threonine</a>) prefer to adopt <a href="beta_sheet" title="wikilink">β-strand</a> conformations. However, these preferences are not strong enough to produce a reliable method of predicting secondary structure from sequence alone.</p>

<p>Low frequency collective vibrations are thought to be sensitive to local rigidity within proteins, revealing beta structures to be generically more rigid than alpha or disordered proteins.<a class="footnoteRef" href="#fn4" id="fnref4"><sup>4</sup></a><a class="footnoteRef" href="#fn5" id="fnref5"><sup>5</sup></a> Neutron scattering measurements have directly connected the spectral feature at ~1 THz to collective motions of the secondary structure of beta-barrel protein GFP.<a class="footnoteRef" href="#fn6" id="fnref6"><sup>6</sup></a></p>

<p>Hydrogen bonding patterns in secondary structures may be significantly distorted, which makes <a href="DSSP_(protein)" title="wikilink">an automatic determination of secondary structure</a> difficult. There are several methods for formally defining protein secondary structure (e.g., DEFINE,<a class="footnoteRef" href="#fn7" id="fnref7"><sup>7</sup></a> DSSP,<a class="footnoteRef" href="#fn8" id="fnref8"><sup>8</sup></a> <a href="STRIDE_(protein)" title="wikilink">STRIDE</a>,<a class="footnoteRef" href="#fn9" id="fnref9"><sup>9</sup></a> <a href="http://lcb.infotech.monash.edu.au/sstweb">SST</a><a class="footnoteRef" href="#fn10" id="fnref10"><sup>10</sup></a>).</p>
<h3 id="dssp-classification">DSSP classification</h3>
<figure><b>(Figure)</b>
<figcaption>Distribution obtained from non-redundant pdb_select dataset (March 2006); Secondary structure assigned by DSSP; 8 conformational states reduced to 3 states: H=HGI, E=EB, C=STC. Visible are mixtures of (gaussian) distributions, resulting also from the reduction of DSSP states.</figcaption>
</figure>

<p>The Dictionary of Protein Secondary Structure, in short DSSP, is commonly used to describe the protein secondary structure with single letter codes. The secondary structure is assigned based on hydrogen bonding patterns as those initially proposed by Pauling et al. in 1951 (before any <a href="protein_structure" title="wikilink">protein structure</a> had ever been experimentally determined). There are eight types of secondary structure that DSSP defines:</p>
<ul>
<li>G = 3-turn helix (<a href="3_10_helix" title="wikilink">3<sub>10</sub> helix</a>). Min length 3 residues.</li>
<li>H = 4-turn helix (<a href="α_helix" title="wikilink">α helix</a>). Min length 4 residues.</li>
<li>I = 5-turn helix (<a href="π_helix" title="wikilink">π helix</a>). Min length 5 residues.</li>
<li>T = hydrogen bonded turn (3, 4 or 5 turn)</li>
<li>E = extended strand in parallel and/or anti-parallel <a class="uri" href="β-sheet" title="wikilink">β-sheet</a> conformation. Min length 2 residues.</li>
<li>B = residue in isolated β-bridge (single pair β-sheet hydrogen bond formation)</li>
<li>S = bend (the only non-hydrogen-bond based assignment).</li>
<li>C = coil (residues which are not in any of the above conformations).</li>
</ul>

<p>'Coil' is often codified as ' ' (space), C (coil) or '-' (dash). The helices (G,H and I) and sheet conformations are all required to have a reasonable length. This means that 2 adjacent residues in the primary structure must form the same hydrogen bonding pattern. If the helix or sheet hydrogen bonding pattern is too short they are designated as T or B, respectively. Other protein secondary structure assignment categories exist (sharp turns, Omega loops etc.), but they are less frequently used.</p>

<p>Secondary structure is defined by <a href="hydrogen_bond" title="wikilink">hydrogen bonding</a>, so the exact definition of a hydrogen bond is critical. The standard H-bond definition for secondary structure is that of <a href="DSSP_(protein)" title="wikilink">DSSP</a>, which is a purely electrostatic model. It assigns charges of 

<math display="inline" id="Protein_secondary_structure:0">
 <semantics>
  <mrow>
   <mrow>
    <mo>±</mo>
    <msub>
     <mi>q</mi>
     <mn>1</mn>
    </msub>
   </mrow>
   <mo>≡</mo>
   <mrow>
    <mn>0.42</mn>
    <mi>e</mi>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <equivalent></equivalent>
    <apply>
     <csymbol cd="latexml">plus-or-minus</csymbol>
     <apply>
      <csymbol cd="ambiguous">subscript</csymbol>
      <ci>q</ci>
      <cn type="integer">1</cn>
     </apply>
    </apply>
    <apply>
     <times></times>
     <cn type="float">0.42</cn>
     <ci>e</ci>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \pm q_{1}\equiv 0.42e
  </annotation>
 </semantics>
</math>

 to the carbonyl carbon and oxygen, respectively, and charges of 

<math display="inline" id="Protein_secondary_structure:1">
 <semantics>
  <mrow>
   <mrow>
    <mo>±</mo>
    <msub>
     <mi>q</mi>
     <mn>2</mn>
    </msub>
   </mrow>
   <mo>≡</mo>
   <mrow>
    <mn>0.20</mn>
    <mi>e</mi>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <equivalent></equivalent>
    <apply>
     <csymbol cd="latexml">plus-or-minus</csymbol>
     <apply>
      <csymbol cd="ambiguous">subscript</csymbol>
      <ci>q</ci>
      <cn type="integer">2</cn>
     </apply>
    </apply>
    <apply>
     <times></times>
     <cn type="float">0.20</cn>
     <ci>e</ci>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \pm q_{2}\equiv 0.20e
  </annotation>
 </semantics>
</math>

 to the amide nitrogen and hydrogen, respectively. The electrostatic energy is</p>

<p>

<math display="block" id="Protein_secondary_structure:2">
 <semantics>
  <mrow>
   <mrow>
    <mi>E</mi>
    <mo>=</mo>
    <mrow>
     <mrow>
      <mrow>
       <mrow>
        <msub>
         <mi>q</mi>
         <mn>1</mn>
        </msub>
        <msub>
         <mi>q</mi>
         <mn>2</mn>
        </msub>
        <mrow>
         <mo>[</mo>
         <mrow>
          <mrow>
           <mfrac>
            <mn>1</mn>
            <msub>
             <mi>r</mi>
             <mrow>
              <mi>O</mi>
              <mi>N</mi>
             </mrow>
            </msub>
           </mfrac>
           <mo>+</mo>
           <mfrac>
            <mn>1</mn>
            <msub>
             <mi>r</mi>
             <mrow>
              <mi>C</mi>
              <mi>H</mi>
             </mrow>
            </msub>
           </mfrac>
          </mrow>
          <mo>-</mo>
          <mfrac>
           <mn>1</mn>
           <msub>
            <mi>r</mi>
            <mrow>
             <mi>O</mi>
             <mi>H</mi>
            </mrow>
           </msub>
          </mfrac>
          <mo>-</mo>
          <mfrac>
           <mn>1</mn>
           <msub>
            <mi>r</mi>
            <mrow>
             <mi>C</mi>
             <mi>N</mi>
            </mrow>
           </msub>
          </mfrac>
         </mrow>
         <mo>]</mo>
        </mrow>
       </mrow>
       <mo>⋅</mo>
       <mpadded width="+5pt">
        <mn>332</mn>
       </mpadded>
      </mrow>
      <mi>kcal</mi>
     </mrow>
     <mo>/</mo>
     <mi>mol</mi>
    </mrow>
   </mrow>
   <mo>.</mo>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <eq></eq>
    <ci>E</ci>
    <apply>
     <divide></divide>
     <apply>
      <times></times>
      <apply>
       <ci>normal-⋅</ci>
       <apply>
        <times></times>
        <apply>
         <csymbol cd="ambiguous">subscript</csymbol>
         <ci>q</ci>
         <cn type="integer">1</cn>
        </apply>
        <apply>
         <csymbol cd="ambiguous">subscript</csymbol>
         <ci>q</ci>
         <cn type="integer">2</cn>
        </apply>
        <apply>
         <csymbol cd="latexml">delimited-[]</csymbol>
         <apply>
          <minus></minus>
          <apply>
           <plus></plus>
           <apply>
            <divide></divide>
            <cn type="integer">1</cn>
            <apply>
             <csymbol cd="ambiguous">subscript</csymbol>
             <ci>r</ci>
             <apply>
              <times></times>
              <ci>O</ci>
              <ci>N</ci>
             </apply>
            </apply>
           </apply>
           <apply>
            <divide></divide>
            <cn type="integer">1</cn>
            <apply>
             <csymbol cd="ambiguous">subscript</csymbol>
             <ci>r</ci>
             <apply>
              <times></times>
              <ci>C</ci>
              <ci>H</ci>
             </apply>
            </apply>
           </apply>
          </apply>
          <apply>
           <divide></divide>
           <cn type="integer">1</cn>
           <apply>
            <csymbol cd="ambiguous">subscript</csymbol>
            <ci>r</ci>
            <apply>
             <times></times>
             <ci>O</ci>
             <ci>H</ci>
            </apply>
           </apply>
          </apply>
          <apply>
           <divide></divide>
           <cn type="integer">1</cn>
           <apply>
            <csymbol cd="ambiguous">subscript</csymbol>
            <ci>r</ci>
            <apply>
             <times></times>
             <ci>C</ci>
             <ci>N</ci>
            </apply>
           </apply>
          </apply>
         </apply>
        </apply>
       </apply>
       <cn type="integer">332</cn>
      </apply>
      <ci>kcal</ci>
     </apply>
     <ci>mol</ci>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   E=q_{1}q_{2}\left[\frac{1}{r_{ON}}+\frac{1}{r_{CH}}-\frac{1}{r_{OH}}-\frac{1}{%
r_{CN}}\right]\cdot 332\ \mathrm{kcal/mol}.
  </annotation>
 </semantics>
</math>

 According to DSSP, an H-bond exists if and only if 

<math display="inline" id="Protein_secondary_structure:3">
 <semantics>
  <mi>E</mi>
  <annotation-xml encoding="MathML-Content">
   <ci>E</ci>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   E
  </annotation>
 </semantics>
</math>

 is less than −0.5 kcal/mol. Although the DSSP formula is a relatively crude approximation of the <em>physical</em> H-bond energy, it is generally accepted as a tool for defining secondary structure.</p>
<h2 id="experimental-determination">Experimental determination</h2>

<p>The rough secondary-structure content of a biopolymer (e.g., "this protein is 40% <a href="alpha_helix" title="wikilink">α-helix</a> and 20% <a href="beta_sheet" title="wikilink">β-sheet</a>.") can be estimated <a href="spectroscopy" title="wikilink">spectroscopically</a>.<a class="footnoteRef" href="#fn11" id="fnref11"><sup>11</sup></a> For proteins, a common method is far-ultraviolet (far-UV, 170–250 nm) <a href="circular_dichroism" title="wikilink">circular dichroism</a>. A pronounced double minimum at 208 and 222 nm indicate α-helical structure, whereas a single minimum at 204 nm or 217 nm reflects random-coil or β-sheet structure, respectively. A less common method is <a href="infrared_spectroscopy" title="wikilink">infrared spectroscopy</a>, which detects differences in the bond oscillations of amide groups due to hydrogen-bonding. Finally, secondary-structure contents may be estimated accurately using the <a href="chemical_shift" title="wikilink">chemical shifts</a> of an initially unassigned <a href="nuclear_magnetic_resonance" title="wikilink">NMR</a> spectrum.<a class="footnoteRef" href="#fn12" id="fnref12"><sup>12</sup></a></p>
<h2 id="prediction">Prediction</h2>

<p>Predicting protein tertiary structure from only its amino acid sequence is a very challenging problem (see <a href="protein_structure_prediction" title="wikilink">protein structure prediction</a>), but using the simpler secondary structure definitions is more tractable.</p>

<p>Early methods of secondary-structure prediction were restricted to predicting the three predominate states: helix, sheet, or random coil. These methods were based on the helix- or sheet-forming propensities of individual amino acids, sometimes coupled with rules for estimating the free energy of forming secondary structure elements. Such methods were typically ~60% accurate in predicting which of the three states (helix/sheet/coil) a residue adopts. The first widely used technique to predict protein secondary structure from the amino acid sequence was the <a href="Chou–Fasman_method" title="wikilink">Chou–Fasman method</a>.<a class="footnoteRef" href="#fn13" id="fnref13"><sup>13</sup></a><a class="footnoteRef" href="#fn14" id="fnref14"><sup>14</sup></a><a class="footnoteRef" href="#fn15" id="fnref15"><sup>15</sup></a></p>

<p>A significant increase in accuracy (to nearly ~80%) was made by exploiting <a href="multiple_sequence_alignment" title="wikilink">multiple sequence alignment</a>; knowing the full distribution of amino acids that occur at a position (and in its vicinity, typically ~7 residues on either side) throughout <a class="uri" href="evolution" title="wikilink">evolution</a> provides a much better picture of the structural tendencies near that position.<a class="footnoteRef" href="#fn16" id="fnref16"><sup>16</sup></a><a class="footnoteRef" href="#fn17" id="fnref17"><sup>17</sup></a> For illustration, a given protein might have a <a class="uri" href="glycine" title="wikilink">glycine</a> at a given position, which by itself might suggest a random coil there. However, multiple sequence alignment might reveal that helix-favoring amino acids occur at that position (and nearby positions) in 95% of homologous proteins spanning nearly a billion years of evolution. Moreover, by examining the average <a class="uri" href="hydrophobicity" title="wikilink">hydrophobicity</a> at that and nearby positions, the same alignment might also suggest a pattern of residue <a href="accessible_surface_area" title="wikilink">solvent accessibility</a> consistent with an α-helix. Taken together, these factors would suggest that the glycine of the original protein adopts α-helical structure, rather than random coil. Several types of methods are used to combine all the available data to form a 3-state prediction, including <a href="neural_network" title="wikilink">neural networks</a>, <a href="hidden_Markov_model" title="wikilink">hidden Markov models</a> and <a href="support_vector_machine" title="wikilink">support vector machines</a>. Modern prediction methods also provide a confidence score for their predictions at every position.</p>

<p>Secondary-structure prediction methods are continuously benchmarked, e.g., <a href="EVA_(benchmark)" title="wikilink">EVA (benchmark)</a>. Based on these tests, the most accurate methods at present are <a class="uri" href="Psipred" title="wikilink">Psipred</a>, SAM,<a class="footnoteRef" href="#fn18" id="fnref18"><sup>18</sup></a> PORTER,<a class="footnoteRef" href="#fn19" id="fnref19"><sup>19</sup></a> PROF,<a class="footnoteRef" href="#fn20" id="fnref20"><sup>20</sup></a> and SABLE.<a class="footnoteRef" href="#fn21" id="fnref21"><sup>21</sup></a> The chief area for improvement appears to be the prediction of β-strands; residues confidently predicted as β-strand are likely to be so, but the methods are apt to overlook some β-strand segments (false negatives). There is likely an upper limit of ~90% prediction accuracy overall, due to the idiosyncrasies of the standard method (<a href="DSSP_(protein)" title="wikilink">DSSP</a>) for assigning secondary-structure classes (helix/strand/coil) to PDB structures, against which the predictions are benchmarked.<a class="footnoteRef" href="#fn22" id="fnref22"><sup>22</sup></a></p>

<p>Accurate secondary-structure prediction is a key element in the prediction of <a href="tertiary_structure" title="wikilink">tertiary structure</a>, in all but the simplest (<a href="protein_structure_prediction" title="wikilink">homology modeling</a>) cases. For example, a confidently predicted pattern of six secondary structure elements βαββαβ is the signature of a <a class="uri" href="ferredoxin" title="wikilink">ferredoxin</a> fold.<a class="footnoteRef" href="#fn23" id="fnref23"><sup>23</sup></a></p>
<h2 id="applications">Applications</h2>

<p>Both protein and nucleic acid secondary structures can be used to aid in <a href="multiple_sequence_alignment" title="wikilink">multiple sequence alignment</a>. These alignments can be made more accurate by the inclusion of secondary structure information in addition to simple sequence information. This is sometimes less useful in RNA because base pairing is much more highly conserved than sequence. Distant relationships between proteins whose primary structures are unalignable can sometimes be found by secondary structure.<ref name="&lt;ref name=" simossis_2004"=""></ref></p>
<h2 id="see-also">See also</h2>
<ul>
<li><a href="Protein_structure" title="wikilink">Protein structure</a>
<ul>
<li><a href="Protein_primary_structure" title="wikilink">Primary structure</a></li>
<li>Secondary structure</li>
<li><a href="Protein_tertiary_structure" title="wikilink">Tertiary structure</a></li>
<li><a href="Protein_quaternary_structure" title="wikilink">Quaternary structure</a></li>
</ul></li>
<li><a href="Folding_(chemistry)" title="wikilink">Folding (chemistry)</a></li>
<li><a href="Nucleic_acid_secondary_structure" title="wikilink">Nucleic acid secondary structure</a></li>
<li><a href="Translation_(biology)" title="wikilink">Translation</a></li>
<li><a href="Structural_motif" title="wikilink">Structural motif</a></li>
<li><a href="Protein_circular_dichroism_data_bank" title="wikilink">Protein circular dichroism data bank</a></li>
</ul>
<h2 id="references">References</h2>
<h2 id="further-reading">Further reading</h2>
<ul>
<li></li>
<li>

<p>(The original beta-sheet conformation article.)</p></li>
<li>

<p>(alpha- and pi-helix conformations, since they predicted that 

<math display="inline" id="Protein_secondary_structure:4">
 <semantics>
  <msub>
   <mn>3</mn>
   <mn>10</mn>
  </msub>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <csymbol cd="ambiguous">subscript</csymbol>
    <cn type="integer">3</cn>
    <cn type="integer">10</cn>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   3_{10}
  </annotation>
 </semantics>
</math>

 helices would not be possible.)</p></li>
</ul>
<h2 id="external-links">External links</h2>
<ul>
<li><a href="http://www.cbs.dtu.dk/services/NetSurfP/">NetSurfP – Secondary Structure and Surface Accessibility predictor</a></li>
<li><a href="http://www.predictprotein.org">PROF</a></li>
<li><a href="http://www.compbio.dundee.ac.uk/~www-jpred/">Jpred</a></li>
<li><a href="http://bioinf.cs.ucl.ac.uk/psipred/">PSIPRED</a></li>
<li><a href="http://swift.cmbi.ru.nl/gv/dssp/">DSSP</a></li>
<li><a href="http://swift.cmbi.ru.nl/whatif/">WhatIf</a></li>
<li><a href="http://bioweb.pasteur.fr/seqanal/interfaces/mfold-simple.html">Mfold</a></li>
<li><a href="http://dirac.cnrs-orleans.fr/ScrewFit/">ScrewFit</a></li>
<li><a href="STRIDE_(protein)" title="wikilink">STRIDE</a></li>
<li><a href="http://zhanglab.ccmb.med.umich.edu/PSSpred/">PSSpred</a> A multiple neural network training program for protein secondary structure prediction</li>
<li><a href="https://genesilico.pl/meta2/">Genesilico metaserver</a> Metaserver which allows to run over 20 different secondary structure predictors by one click</li>
<li><a href="http://lcb.infotech.monash.edu.au/sstweb">SST</a> webserver: An information-theoretic (compression-based) secondary structural assignment.</li>
</ul>

<p><a href="eo:Sekundara_strukturo" title="wikilink">eo:Sekundara strukturo</a> <a href="fr:Structure_secondaire" title="wikilink">fr:Structure secondaire</a> <a href="it:Struttura_secondaria" title="wikilink">it:Struttura secondaria</a> <a href="he:מבנה_שניוני" title="wikilink">he:מבנה שניוני</a> <a class="uri" href="sv:Sekundärstruktur" title="wikilink">sv:Sekundärstruktur</a>"</p>

<p><a href="Category:Protein_structure" title="wikilink">Protein structure 2</a> <a class="uri" href="Category:Stereochemistry" title="wikilink">Category:Stereochemistry</a></p>
<section class="footnotes">
<hr/>
<ol>
<li id="fn1"><a href="#fnref1">↩</a></li>
<li id="fn2"><a href="#fnref2">↩</a></li>
<li id="fn3"><a href="#fnref3">↩</a></li>
<li id="fn4"><a href="#fnref4">↩</a></li>
<li id="fn5"><a href="#fnref5">↩</a></li>
<li id="fn6"><a href="#fnref6">↩</a></li>
<li id="fn7"><a href="#fnref7">↩</a></li>
<li id="fn8"><a href="#fnref8">↩</a></li>
<li id="fn9"><a href="#fnref9">↩</a></li>
<li id="fn10"><a href="#fnref10">↩</a></li>
<li id="fn11"><a href="#fnref11">↩</a></li>
<li id="fn12"><a href="#fnref12">↩</a></li>
<li id="fn13"><a href="#fnref13">↩</a></li>
<li id="fn14"><a href="#fnref14">↩</a></li>
<li id="fn15"><a href="#fnref15">↩</a></li>
<li id="fn16"><a href="#fnref16">↩</a></li>
<li id="fn17"><a href="#fnref17">↩</a></li>
<li id="fn18"><a href="#fnref18">↩</a></li>
<li id="fn19"><a href="#fnref19">↩</a></li>
<li id="fn20"><a href="#fnref20">↩</a></li>
<li id="fn21"><a href="#fnref21">↩</a></li>
<li id="fn22"><a href="#fnref22">↩</a></li>
<li id="fn23"><a href="#fnref23">↩</a></li>
</ol>
</section>
</body>
</html>
