<html lang="en">
<head>
<meta charset="utf-8"/>
<title offset="631">Proton ATPase</title>
<script src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_SVG.js" type="text/javascript">
</script>
</head>
<body>
<h1>Proton ATPase</h1>
<hr/>
<dl>
<dd><em>This article is about the P-type -ATPase found in plants and fungi. For the gastric / ATPase (involved in the acidification of the stomach in mammals), see <a href="Hydrogen_potassium_ATPase" title="wikilink">Hydrogen potassium ATPase</a>.</em>
</dd>
</dl>

<p>In the field of <a class="uri" href="enzymology" title="wikilink">enzymology</a>, the <strong>proton-ATPase</strong> () is an <a class="uri" href="enzyme" title="wikilink">enzyme</a> that <a href="catalysis" title="wikilink">catalyzes</a> the following <a href="chemical_reaction" title="wikilink">chemical reaction</a>:</p>
<dl>
<dd>ATP +  + <sub>in</sub> 

<math display="inline" id="Proton_ATPase:0">
 <semantics>
  <mo>⇌</mo>
  <annotation-xml encoding="MathML-Content">
   <ci>normal-⇌</ci>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \rightleftharpoons
  </annotation>
 </semantics>
</math>

 ADP + phosphate + <sub>out</sub>
</dd>
</dl>

<p>The 3 <a href="substrate_(biochemistry)" title="wikilink">substrates</a> of this enzyme are <a href="adenosine_triphosphate" title="wikilink">ATP</a>, , and , whereas its 3 <a href="product_(chemistry)" title="wikilink">products</a> are <a href="adenosine_diphosphate" title="wikilink">ADP</a>, <a class="uri" href="phosphate" title="wikilink">phosphate</a>, and .</p>

<p>This enzyme belongs to the family of <a href="hydrolase" title="wikilink">hydrolases</a>, specifically those acting on acid anhydrides to catalyse transmembrane movement of substances. To be specific, the protein is a part of the <a href="P-type_ATPase" title="wikilink">P-type ATPase</a> family. The systematic name of this enzyme class is <strong>ATP phosphohydrolase (-exporting)</strong>.</p>

<p><strong>-exporting ATPase</strong> is also known as <strong>proton ATPase</strong> or more simply <strong>proton pump</strong>. Other names in common use include <strong>proton-translocating ATPase</strong>, <strong>yeast plasma membrane -ATPase</strong>, <strong>yeast plasma membrane ATPase</strong>, and <strong>ATP phosphohydrolase</strong>.</p>
<h2 id="function-and-location">Function and location</h2>

<p>The -<a class="uri" href="ATPase" title="wikilink">ATPase</a> or proton pump creates the <a href="electrochemical_gradients" title="wikilink">electrochemical gradients</a> in the <a href="plasma_membrane" title="wikilink">plasma membrane</a> of <a class="uri" href="plants" title="wikilink">plants</a>, <a class="uri" href="fungi" title="wikilink">fungi</a>, <a class="uri" href="protists" title="wikilink">protists</a>, and many <a class="uri" href="prokaryotes" title="wikilink">prokaryotes</a>. Here, proton gradients are used to drive <a href="Secondary_active_transport" title="wikilink">secondary transport</a> processes. As such, it is essential for the uptake of most <a class="uri" href="metabolites" title="wikilink">metabolites</a>, and also for plant responses to the environment (e.g., movement of leaves).</p>

<p>It is interesting to note that -ATPases are specific for <a class="uri" href="plants" title="wikilink">plants</a>, <a class="uri" href="fungi" title="wikilink">fungi</a>, and <a class="uri" href="protists" title="wikilink">protists</a>; and <a href="sodium_potassium_pump" title="wikilink">-ATPases</a> are specific for <a class="uri" href="animal" title="wikilink">animal</a> cells. These two groups of <a href="P-type_ATPase" title="wikilink">P-type ATPases</a>, although not from the same subfamily, seem to perform a complementary function in plants/fungi/protists and animal cells, namely the creation of an <a href="electrochemical_gradient" title="wikilink">electrochemical gradient</a> used as an energy source for <a href="Secondary_active_transport" title="wikilink">secondary transport</a>.</p>
<h2 id="structural-studies">Structural studies</h2>
<figure><b>(Figure)</b>
<figcaption>300|right|Proton ATPase AHA2 (3b8c)</figcaption>
</figure>

<p>Structural information on P-type proton ATPases are scarce compared to that obtained for <a class="uri" href="SERCA1a" title="wikilink">SERCA1a</a>. A low resolution structure from 2D crystals of the plasma membrane (PM) -ATPase from <em><a href="Neurospora_crassa" title="wikilink">Neurospora crassa</a></em> is, as of medio 2011, the only structural information on the fungal -ATPase.<a class="footnoteRef" href="#fn1" id="fnref1"><sup>1</sup></a> For the plant counterpart, a crystal structure of the AHA2 PM -ATPase from <em><a href="Arabidopsis_thaliana" title="wikilink">Arabidopsis thaliana</a></em> has been obtained from 3D crystals with a resolution of 3.6 Å.<a class="footnoteRef" href="#fn2" id="fnref2"><sup>2</sup></a> The structure of AHA2 clearly identifies three cytosolic domains corresponding to the N (nucleotide binding), P (phosphorylation), and A (actuator) domains, similar to those observed in the <a href="SERCA" title="wikilink">SR -ATPase</a> and also verifies the presence of ten transmembrane helices. The 3D crystal structure shows the AHA2 PM -ATPase in a so-called quasi-occluded <em>E</em>1 state with the non-hydrolysable ATP analogue AMPPCP bound, and the overall fold of the catalytic unit reveals a high degree of structural similarity to the <a href="SERCA" title="wikilink">SR -ATPase</a> and the <a href="sodium_potassium_pump" title="wikilink">,-ATPase</a>. The overall arrangement of the domains is similar to that observed for the occluded <em>E</em>1 conformation of the <a href="SERCA" title="wikilink">SR -ATPase</a>, and based on comparison with structural data for the other conformations of the <a href="SERCA" title="wikilink">SR -ATPase</a>, it was suggested that the structure of the AHA2 PM -ATPase represents a novel <em>E</em>1 intermediate.<a class="footnoteRef" href="#fn3" id="fnref3"><sup>3</sup></a> A distinct feature of the PM -ATPase not observed in other P-type ATPases is the presence of a large cavity in the transmembrane domain formed by M4, M5 and M6.</p>
<h2 id="regulation">Regulation</h2>

<p>Precise regulation of PM -ATPase activity is crucial to the plant. Over-expression of the PM -ATPase is compensated by a down-regulation of activity,<a class="footnoteRef" href="#fn4" id="fnref4"><sup>4</sup></a> whereas deletion of an isoform is compensated by redundancy as well as augmented activity of other isoforms by increased level of post-translational modifications.<a class="footnoteRef" href="#fn5" id="fnref5"><sup>5</sup></a> The PM -ATPase is subject to autoinhibition, which negatively regulates the activity of the pump and keeps the enzyme in a low activity state where ATP hydrolytic activity is partly uncoupled from ATP hydrolysis,.<a class="footnoteRef" href="#fn6" id="fnref6"><sup>6</sup></a><a class="footnoteRef" href="#fn7" id="fnref7"><sup>7</sup></a> Release from the autoinhibitory restraints requires posttranslational modifications such as phosphorylation and interacting proteins. Autoinhibition is achieved by the N- and C-termini of the protein - communication between the two termini facilitates the necessary precise control of pump activity.<a class="footnoteRef" href="#fn8" id="fnref8"><sup>8</sup></a> The autoinhibitory C-terminal domain can be displaced by phosphorylation of the penultimate Thr residue and the subsequent binding of 14-3-3 proteins.<a class="footnoteRef" href="#fn9" id="fnref9"><sup>9</sup></a> The PM -ATPase is the first P-type ATPase for which both termini have been demonstrated to take part in the regulation of protein activity.<a class="footnoteRef" href="#fn10" id="fnref10"><sup>10</sup></a></p>
<h2 id="physiological-roles-in-plants">Physiological roles in plants</h2>

<p>Plasma membrane -ATPases are found throughout the plant in all cell types investigated, but some cell types have much higher concentrations of -ATPase than others. In general, these cell types are specialised for intensive <a href="active_transport" title="wikilink">active transport</a> and accumulate solutes from their surroundings. Most studies of these roles come from genetic studies on <em>Arabidopsis thaliana</em>.<a class="footnoteRef" href="#fn11" id="fnref11"><sup>11</sup></a> -ATPases in plants are expressed from a multigene subfamily, and <em>Arabidopsis thaliana</em> for instance, have 12 different -ATPase genes.</p>

<p>Some important physiological processes the plant -ATPase is involved in are:</p>
<ul>
<li><strong>Phloem loading</strong>. The <a class="uri" href="phloem" title="wikilink">phloem</a> is a tissue specialised for long-distance transport of organic compounds, and is well known for its involvement in the transport of sugar from leaves or other source areas. Here the -ATPase powers the sucrose/ cotransporters and is found to be essential for the loading of <a class="uri" href="sucrose" title="wikilink">sucrose</a> into the <a class="uri" href="phloem" title="wikilink">phloem</a>.</li>
<li><strong>Solute uptake in roots</strong>. -ATPases energize the uptake of nutrients from the soil into the <a href="root" title="wikilink">roots</a>, and is also involved in the further loading of these solutes into the <a class="uri" href="xylem" title="wikilink">xylem</a>, a tissue specialised for long-distance transport of water and <a class="uri" href="micronutrients" title="wikilink">micronutrients</a>.</li>
<li><strong>Tip-growing systems</strong>. <a href="Pollen_tubes" title="wikilink">Pollen tubes</a> and <a href="root_hairs" title="wikilink">root hairs</a> are examples of plant <a href="Tip_growth" title="wikilink">tip-growing systems</a>, where a single cell expands in one direction only. The direction of growth is controlled by an asymmetrical proton gradient, where protons enter at the extreme tip and are pumped out just below the tip.</li>
<li><strong>Size of stomatal aperture</strong>. The <a href="stomata" title="wikilink">somatal pore</a> controls the diffusion of  into the leaves to be utilized for <a class="uri" href="photosynthesis" title="wikilink">photosynthesis</a>. The pore is formed by two <a href="guard_cells" title="wikilink">guard cells</a>, which control the size of the pore by swelling in response to the activity of the -ATPase. Opening and closure of the pore is partly controlled by regulation of the -ATPase.</li>
<li><strong>Plant movements</strong>. Like the <a href="stomata" title="wikilink">somatal pore</a>, other movements of <a href="plant_organs" title="wikilink">plant organs</a> are controlled by motor cells' changing cell <a class="uri" href="turgor" title="wikilink">turgor</a>. These cells control phenomena such as <a href="solar_tracking" title="wikilink">solar tracking</a> by the plant to optimize orientation of <a class="uri" href="photosynthetic" title="wikilink">photosynthetic</a> leaves, and the swift and spectacular reactions to touch found in some plant species (e.g., <a href="carnivorous_plants" title="wikilink">carnivorous plants</a>). All of these swelling and shrinking processes take place by massive water and ion fluxes through channels. Here, activation of the -ATPase leads to <a href="plasma_membrane" title="wikilink">plasma membrane</a> <a href="Hyperpolarization_(biology)" title="wikilink">hyperpolarization</a> and the opening of voltage sensitive <a href="potassium_channels" title="wikilink">potassium channels</a>. The  influx leads to water uptake and turgor increase in the cell.</li>
<li><strong>Salt and osmotolerance</strong>. Salinity imposes two stresses on the cell: one is the loss of <a class="uri" href="turgor" title="wikilink">turgor</a> due to the <a class="uri" href="hypertonicity" title="wikilink">hypertonicity</a> of the extracellular medium, and the other is a direct effect of toxic ions on <a class="uri" href="metabolism" title="wikilink">metabolism</a>. Therefore plants have developed several defence mechanisms. The Na/ <a class="uri" href="antiporter" title="wikilink">antiporter</a> is heavily involved and is powered by the action of the -ATPase, which is highly expressed in leaves and roots during salt stress.</li>
<li><strong>Intracellular pH regulation</strong>. <a href="Intracellular_pH" title="wikilink">Intracellular pH</a> remains constant during cell growth, it is presumed, to ensure optimal activity of the <a class="uri" href="cytoplasmic" title="wikilink">cytoplasmic</a> <a class="uri" href="enzymes" title="wikilink">enzymes</a>. This is controlled by the proton pump.</li>
<li><strong>Acid growth</strong>. Acidification of the external medium caused by activation of the plasma membrane -ATPase initiates cellular expansion. It is believed that the <a href="plant_hormone" title="wikilink">plant hormone</a> <a class="uri" href="auxin" title="wikilink">auxin</a> activates the proton pump. The <a href="apoplast" title="wikilink">apoplastic</a> acidification leads to loosening of the <a href="cell_wall" title="wikilink">cell wall</a> and <a href="Hyperpolarization_(biology)" title="wikilink">hyperpolarization</a> of the plasma membrane inducing  uptake and swelling.</li>
</ul>
<h2 id="references">References</h2>
<ul>
<li></li>
<li></li>
<li></li>
</ul>

<p>"</p>

<p><a href="Category:EC_3.6.3" title="wikilink">Category:EC 3.6.3</a> <a href="Category:Enzymes_of_known_structure" title="wikilink">Category:Enzymes of known structure</a> <a href="Category:Transmembrane_proteins" title="wikilink">Category:Transmembrane proteins</a></p>
<section class="footnotes">
<hr/>
<ol>
<li id="fn1"><a href="#fnref1">↩</a></li>
<li id="fn2"><a href="#fnref2">↩</a></li>
<li id="fn3"></li>
<li id="fn4"><a href="#fnref4">↩</a></li>
<li id="fn5"><a href="#fnref5">↩</a></li>
<li id="fn6"><a href="#fnref6">↩</a></li>
<li id="fn7"><a href="#fnref7">↩</a></li>
<li id="fn8"><a href="#fnref8">↩</a></li>
<li id="fn9"><a href="#fnref9">↩</a></li>
<li id="fn10"></li>
<li id="fn11"><a href="#fnref11">↩</a></li>
</ol>
</section>
</body>
</html>
