<html lang="en">
<head>
<meta charset="utf-8"/>
<title offset="199">Pseudo amino acid composition</title>
<script src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_SVG.js" type="text/javascript">
</script>
</head>
<body>
<h1>Pseudo amino acid composition</h1>
<hr/>

<p><strong>Pseudo amino acid composition</strong>, or <strong>PseAA composition</strong>, or <strong>Chou's PseAAC</strong>, <a class="footnoteRef" href="#fn1" id="fnref1"><sup>1</sup></a> was originally introduced by Kuo-Chen Chou in 2001 to represent <a class="uri" href="protein" title="wikilink">protein</a> samples for improving <a href="protein_subcellular_localization_prediction" title="wikilink">protein subcellular localization prediction</a> and <a href="membrane_protein" title="wikilink">membrane protein</a> type prediction.<a class="footnoteRef" href="#fn2" id="fnref2"><sup>2</sup></a></p>
<h2 id="background">Background</h2>

<p>To predict the <a href="subcellular_localization" title="wikilink">subcellular localization</a> of proteins and other attributes based on their sequence, two kinds of models are generally used to represent protein samples: (1) the sequential model, and (2) the non-sequential model or discrete model.</p>

<p>The most typical sequential representation for a protein sample is its entire <a href="amino_acid" title="wikilink">amino acid</a> (AA) sequence, which can contain its most complete information. This is an obvious advantage of the sequential model. To get the desired results, the sequence-similarity-search-based tools are usually utilized to conduct the prediction. However, this kind of approach fails when a query protein does not have significant homology to the known protein(s). Thus, various discrete models were proposed which do not rely on sequence-order.</p>

<p>The simplest discrete model is using the amino acid composition (AAC) to represent protein samples, formulated as follows. Given a protein sequence <strong>P</strong> with 

<math display="inline" id="Pseudo_amino_acid_composition:0">
 <semantics>
  <mi>L</mi>
  <annotation-xml encoding="MathML-Content">
   <ci>L</ci>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   L
  </annotation>
 </semantics>
</math>

 amino acid residues, i.e.,</p>
<ul>
<li>

<math display="inline" id="Pseudo_amino_acid_composition:1">
 <semantics>
  <mrow>
   <mi>ùêè</mi>
   <mo>=</mo>
   <mrow>
    <mrow>
     <mo>[</mo>
     <mtable>
      <mtr>
       <mtd columnalign="center">
        <mrow>
         <msub>
          <mi mathvariant="normal">R</mi>
          <mn>1</mn>
         </msub>
         <msub>
          <mi mathvariant="normal">R</mi>
          <mn>2</mn>
         </msub>
         <msub>
          <mi mathvariant="normal">R</mi>
          <mn>3</mn>
         </msub>
         <msub>
          <mi mathvariant="normal">R</mi>
          <mn>4</mn>
         </msub>
         <msub>
          <mi mathvariant="normal">R</mi>
          <mn>5</mn>
         </msub>
         <msub>
          <mi mathvariant="normal">R</mi>
          <mn>6</mn>
         </msub>
         <msub>
          <mi mathvariant="normal">R</mi>
          <mn>7</mn>
         </msub>
         <mi mathvariant="normal">‚ãØ</mi>
         <msub>
          <mi mathvariant="normal">R</mi>
          <mi>L</mi>
         </msub>
        </mrow>
       </mtd>
      </mtr>
     </mtable>
     <mo>]</mo>
    </mrow>
    <mtext>(1)</mtext>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <eq></eq>
    <ci>ùêè</ci>
    <list>
     <matrix>
      <matrixrow>
       <apply>
        <times></times>
        <apply>
         <csymbol cd="ambiguous">subscript</csymbol>
         <ci>normal-R</ci>
         <cn type="integer">1</cn>
        </apply>
        <apply>
         <csymbol cd="ambiguous">subscript</csymbol>
         <ci>normal-R</ci>
         <cn type="integer">2</cn>
        </apply>
        <apply>
         <csymbol cd="ambiguous">subscript</csymbol>
         <ci>normal-R</ci>
         <cn type="integer">3</cn>
        </apply>
        <apply>
         <csymbol cd="ambiguous">subscript</csymbol>
         <ci>normal-R</ci>
         <cn type="integer">4</cn>
        </apply>
        <apply>
         <csymbol cd="ambiguous">subscript</csymbol>
         <ci>normal-R</ci>
         <cn type="integer">5</cn>
        </apply>
        <apply>
         <csymbol cd="ambiguous">subscript</csymbol>
         <ci>normal-R</ci>
         <cn type="integer">6</cn>
        </apply>
        <apply>
         <csymbol cd="ambiguous">subscript</csymbol>
         <ci>normal-R</ci>
         <cn type="integer">7</cn>
        </apply>
        <ci>normal-‚ãØ</ci>
        <apply>
         <csymbol cd="ambiguous">subscript</csymbol>
         <ci>normal-R</ci>
         <ci>L</ci>
        </apply>
       </apply>
      </matrixrow>
     </matrix>
     <mtext>(1)</mtext>
    </list>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \mathbf{P}={\begin{bmatrix}\mathrm{R}_{1}\mathrm{R}_{2}\mathrm{R}_{3}\mathrm{R%
}_{4}\mathrm{R}_{5}\mathrm{R}_{6}\mathrm{R}_{7}\cdots\mathrm{R}_{L}\end{%
bmatrix}}\qquad\text{(1)}
  </annotation>
 </semantics>
</math>

</li>
</ul>

<p>where R<sub>1</sub> represents the 1st residue of the protein <strong>P</strong>, R<sub>2</sub> the 2nd residue, and so forth, according to the amino acic composition (AAC) model, the protein <strong>P</strong> of <strong>Eq.1</strong> can be expressed by</p>
<ul>
<li>

<math display="inline" id="Pseudo_amino_acid_composition:2">
 <semantics>
  <mrow>
   <mi>ùêè</mi>
   <mo>=</mo>
   <mrow>
    <msup>
     <mrow>
      <mo>[</mo>
      <mtable>
       <mtr>
        <mtd columnalign="center">
         <msub>
          <mi>f</mi>
          <mn>1</mn>
         </msub>
        </mtd>
        <mtd columnalign="center">
         <msub>
          <mi>f</mi>
          <mn>2</mn>
         </msub>
        </mtd>
        <mtd columnalign="center">
         <mi mathvariant="normal">‚ãØ</mi>
        </mtd>
        <mtd columnalign="center">
         <msub>
          <mi>f</mi>
          <mn>20</mn>
         </msub>
        </mtd>
       </mtr>
      </mtable>
      <mo>]</mo>
     </mrow>
     <mi>ùêì</mi>
    </msup>
    <mtext>(2)</mtext>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <eq></eq>
    <ci>ùêè</ci>
    <list>
     <apply>
      <csymbol cd="ambiguous">superscript</csymbol>
      <matrix>
       <matrixrow>
        <apply>
         <csymbol cd="ambiguous">subscript</csymbol>
         <ci>f</ci>
         <cn type="integer">1</cn>
        </apply>
        <apply>
         <csymbol cd="ambiguous">subscript</csymbol>
         <ci>f</ci>
         <cn type="integer">2</cn>
        </apply>
        <ci>normal-‚ãØ</ci>
        <apply>
         <csymbol cd="ambiguous">subscript</csymbol>
         <ci>f</ci>
         <cn type="integer">20</cn>
        </apply>
       </matrixrow>
      </matrix>
      <ci>ùêì</ci>
     </apply>
     <mtext>(2)</mtext>
    </list>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \mathbf{P}={\begin{bmatrix}f_{1}&f_{2}&\cdots&f_{20}\end{bmatrix}}^{\mathbf{T}%
}\qquad\text{(2)}
  </annotation>
 </semantics>
</math>

</li>
</ul>

<p>where 

<math display="inline" id="Pseudo_amino_acid_composition:3">
 <semantics>
  <mrow>
   <mpadded width="+1.7pt">
    <msub>
     <mpadded lspace="1.7pt" width="+1.7pt">
      <mi>f</mi>
     </mpadded>
     <mi>u</mi>
    </msub>
   </mpadded>
   <mrow>
    <mo stretchy="false">(</mo>
    <mi>u</mi>
    <mo>=</mo>
    <mn>1</mn>
    <mo>,</mo>
    <mn>2</mn>
    <mo>,</mo>
    <mi mathvariant="normal">‚ãØ</mi>
    <mo>,</mo>
    <mn>20</mn>
    <mo stretchy="false">)</mo>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <cerror>
    <csymbol cd="ambiguous">fragments</csymbol>
    <apply>
     <csymbol cd="ambiguous">subscript</csymbol>
     <ci>f</ci>
     <ci>u</ci>
    </apply>
    <cerror>
     <csymbol cd="ambiguous">fragments</csymbol>
     <ci>normal-(</ci>
     <csymbol cd="unknown">u</csymbol>
     <eq></eq>
     <cn type="integer">1</cn>
     <ci>normal-,</ci>
     <cn type="integer">2</cn>
     <ci>normal-,</ci>
     <ci>normal-‚ãØ</ci>
     <ci>normal-,</ci>
     <cn type="integer">20</cn>
     <ci>normal-)</ci>
    </cerror>
   </cerror>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \,f_{u}\,(u=1,2,\cdots,20)
  </annotation>
 </semantics>
</math>

 are the normalized occurrence frequencies of the 20 native amino acids in <strong>P</strong>, and <strong>T</strong> the transposing operator. Accordingly, the amino acid composition of a protein can be easily derived once the <a href="protein_sequencing" title="wikilink">protein sequencing</a> information is known.</p>

<p>Owing to its simplicity, the amino acid composition (AAC) model was widely used in many earlier statistical methods for predicting protein attributes. However, all the sequence-order information is lost. This is its main shortcoming.</p>
<h2 id="concept">Concept</h2>

<p>To avoid completely losing the sequence-order information, the concept of PseAA (<strong>pse</strong>udo <strong>a</strong>mino <strong>a</strong>cid) composition was proposed.<a class="footnoteRef" href="#fn3" id="fnref3"><sup>3</sup></a> In contrast with the conventional amino acid composition that contains 20 components with each reflecting the occurrence frequency for one of the 20 native amino acids in a protein, the PseAA composition contains a set of greater than 20 discrete factors, where the first 20 represent the components of its conventional AA composition while the additional factors incorporate some sequence-order information via various modes.</p>

<p>The additional factors are a series of rank-different correlation factors along a protein chain, but they can also be any combinations of other factors so long as they can reflect some sorts of sequence-order effects one way or the other. Therefore, the essence of PseAA composition is that on one hand it covers the AA composition, but on the other hand it contains the information beyond the AA composition and hence can better reflect the feature of a protein sequence through a discrete model.</p>

<p>Meanwhile, various modes to formulate the PseAA composition have also been developed, as summarized in a review.<a class="footnoteRef" href="#fn4" id="fnref4"><sup>4</sup></a></p>
<h2 id="algorithm">Algorithm</h2>
<figure><b>(Figure)</b>
<figcaption><strong>Figure 1</strong>. A schematic drawing to show (a) the 1st-tier, (b) the 2nd-tier, and (3) the 3rd-tier sequence-order-correlation mode along a protein sequence, where R<sub>1</sub> represents the amino acid residue at the sequence position 1, R<sub>2</sub> at position 2, and so forth (cf. <strong>Eq.1</strong>), and the coupling factors 

<math display="inline" id="Pseudo_amino_acid_composition:4">
 <semantics>
  <msub>
   <mi>J</mi>
   <mrow>
    <mi>i</mi>
    <mo>,</mo>
    <mi>j</mi>
   </mrow>
  </msub>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <csymbol cd="ambiguous">subscript</csymbol>
    <ci>J</ci>
    <list>
     <ci>i</ci>
     <ci>j</ci>
    </list>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   J_{i,j}
  </annotation>
 </semantics>
</math>

 are given by <strong>Eq.6</strong>. Panel (a) reflects the correlation mode between all the most contiguous residues, panel (b) that between all the 2nd most contiguous residues, and panel (c) that between all the 3rd most contiguous residues.</figcaption>
</figure>

<p>According to the PseAA composition model, the protein <strong>P</strong> of <strong>Eq.1</strong> can be formulated as</p>
<ul>
<li>

<math display="inline" id="Pseudo_amino_acid_composition:5">
 <semantics>
  <mrow>
   <mi>ùêè</mi>
   <mo>=</mo>
   <msup>
    <mrow>
     <mo>[</mo>
     <mtable>
      <mtr>
       <mtd columnalign="center">
        <mrow>
         <msub>
          <mi>p</mi>
          <mn>1</mn>
         </msub>
         <mo rspace="4.2pt">,</mo>
         <msub>
          <mi>p</mi>
          <mn>2</mn>
         </msub>
         <mo rspace="4.2pt">,</mo>
         <mi mathvariant="normal">‚Ä¶</mi>
         <mo rspace="4.2pt">,</mo>
         <msub>
          <mi>p</mi>
          <mn>20</mn>
         </msub>
         <mo rspace="4.2pt">,</mo>
         <msub>
          <mi>p</mi>
          <mrow>
           <mn>20</mn>
           <mo>+</mo>
           <mn>1</mn>
          </mrow>
         </msub>
         <mo rspace="4.2pt">,</mo>
         <mi mathvariant="normal">‚Ä¶</mi>
         <mo rspace="4.2pt">,</mo>
         <msub>
          <mi>p</mi>
          <mrow>
           <mn>20</mn>
           <mo>+</mo>
           <mi>Œª</mi>
          </mrow>
         </msub>
        </mrow>
       </mtd>
      </mtr>
     </mtable>
     <mo>]</mo>
    </mrow>
    <mi>ùêì</mi>
   </msup>
   <mo rspace="7.6pt">,</mo>
   <mrow>
    <mo stretchy="false">(</mo>
    <mi>Œª</mi>
    <mo><</mo>
    <mi>L</mi>
    <mo stretchy="false">)</mo>
   </mrow>
   <mtext>(3)</mtext>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <cerror>
    <csymbol cd="ambiguous">fragments</csymbol>
    <csymbol cd="unknown">P</csymbol>
    <eq></eq>
    <apply>
     <csymbol cd="ambiguous">superscript</csymbol>
     <matrix>
      <matrixrow>
       <list>
        <apply>
         <csymbol cd="ambiguous">subscript</csymbol>
         <ci>p</ci>
         <cn type="integer">1</cn>
        </apply>
        <apply>
         <csymbol cd="ambiguous">subscript</csymbol>
         <ci>p</ci>
         <cn type="integer">2</cn>
        </apply>
        <ci>normal-‚Ä¶</ci>
        <apply>
         <csymbol cd="ambiguous">subscript</csymbol>
         <ci>p</ci>
         <cn type="integer">20</cn>
        </apply>
        <apply>
         <csymbol cd="ambiguous">subscript</csymbol>
         <ci>p</ci>
         <apply>
          <plus></plus>
          <cn type="integer">20</cn>
          <cn type="integer">1</cn>
         </apply>
        </apply>
        <ci>normal-‚Ä¶</ci>
        <apply>
         <csymbol cd="ambiguous">subscript</csymbol>
         <ci>p</ci>
         <apply>
          <plus></plus>
          <cn type="integer">20</cn>
          <ci>Œª</ci>
         </apply>
        </apply>
       </list>
      </matrixrow>
     </matrix>
     <ci>ùêì</ci>
    </apply>
    <ci>normal-,</ci>
    <cerror>
     <csymbol cd="ambiguous">fragments</csymbol>
     <ci>normal-(</ci>
     <csymbol cd="unknown">Œª</csymbol>
     <lt></lt>
     <csymbol cd="unknown">L</csymbol>
     <ci>normal-)</ci>
    </cerror>
    <ci>italic-</ci>
    <mtext>(3)</mtext>
   </cerror>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \mathbf{P}={\begin{bmatrix}p_{1},\,p_{2},\,\ldots,\,p_{20},\,p_{20+1},\,\ldots%
,\,p_{20+\lambda}\end{bmatrix}}^{\mathbf{T}},\,\,\,(\lambda<L)\qquad\text{(3)}
  </annotation>
 </semantics>
</math>

</li>
</ul>

<p>where the (

<math display="inline" id="Pseudo_amino_acid_composition:6">
 <semantics>
  <mrow>
   <mn>20</mn>
   <mo>+</mo>
   <mi>Œª</mi>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <plus></plus>
    <cn type="integer">20</cn>
    <ci>Œª</ci>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   20+\lambda
  </annotation>
 </semantics>
</math>

) components are given by</p>
<ul>
<li><math> p_u = \begin{cases}</math></li>
</ul>

<p>\dfrac {f_u} {\sum_{i=1}^{20}f_i \, + \, w\sum_{k=1}^{\lambda} \tau_k}, &amp; (1 \le u \le 20) \\[10pt] \dfrac {w \tau_{u-20}} {\sum_{i=1}^{20} f_i \, + \, w\sum_{k=1}^{\lambda} \tau_k}, &amp; (20+1 \le u \le 20+\lambda) \end{cases} \qquad \text{(4)} </p>

<p>where 

<math display="inline" id="Pseudo_amino_acid_composition:7">
 <semantics>
  <mi>w</mi>
  <annotation-xml encoding="MathML-Content">
   <ci>w</ci>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   w
  </annotation>
 </semantics>
</math>

 is the weight factor, and 

<math display="inline" id="Pseudo_amino_acid_composition:8">
 <semantics>
  <msub>
   <mi>œÑ</mi>
   <mi>k</mi>
  </msub>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <csymbol cd="ambiguous">subscript</csymbol>
    <ci>œÑ</ci>
    <ci>k</ci>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \tau_{k}
  </annotation>
 </semantics>
</math>

 the 

<math display="inline" id="Pseudo_amino_acid_composition:9">
 <semantics>
  <mi>k</mi>
  <annotation-xml encoding="MathML-Content">
   <ci>k</ci>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   k
  </annotation>
 </semantics>
</math>

-th tier correlation factor that reflects the sequence order correlation between all the 

<math display="inline" id="Pseudo_amino_acid_composition:10">
 <semantics>
  <mi>k</mi>
  <annotation-xml encoding="MathML-Content">
   <ci>k</ci>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   k
  </annotation>
 </semantics>
</math>

-th most contiguous residues as formulated by</p>
<ul>
<li><math></math></li>
</ul>

<p>\tau_k = \frac {1}{L-k} \sum_{i=1}^{L-k} \, \mathrm{J}_{i, i+k}, \,\,\, (k </p>

<p>with</p>
<ul>
<li><math></math></li>
</ul>

<p>\mathrm{J}_{i, i+k} = \frac{1}{\Gamma} \sum_{q=1}^{\Gamma} \left[\Phi_{q}\left(\mathrm{R}_{i+k}\right) - \Phi_{q}\left(\mathrm{R}_{i}\right ) \right]^2 \qquad \text{(6)} </p>

<p>where 

<math display="inline" id="Pseudo_amino_acid_composition:11">
 <semantics>
  <mrow>
   <msub>
    <mi mathvariant="normal">Œ¶</mi>
    <mi>q</mi>
   </msub>
   <mrow>
    <mo>(</mo>
    <msub>
     <mi mathvariant="normal">R</mi>
     <mi>i</mi>
    </msub>
    <mo>)</mo>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <times></times>
    <apply>
     <csymbol cd="ambiguous">subscript</csymbol>
     <ci>normal-Œ¶</ci>
     <ci>q</ci>
    </apply>
    <apply>
     <csymbol cd="ambiguous">subscript</csymbol>
     <ci>normal-R</ci>
     <ci>i</ci>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \Phi_{q}\left(\mathrm{R}_{i}\right)
  </annotation>
 </semantics>
</math>

 is the 

<math display="inline" id="Pseudo_amino_acid_composition:12">
 <semantics>
  <mi>q</mi>
  <annotation-xml encoding="MathML-Content">
   <ci>q</ci>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   {q}
  </annotation>
 </semantics>
</math>

-th function of the amino acid 

<math display="inline" id="Pseudo_amino_acid_composition:13">
 <semantics>
  <mpadded width="+1.7pt">
   <msub>
    <mi mathvariant="normal">R</mi>
    <mi>i</mi>
   </msub>
  </mpadded>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <csymbol cd="ambiguous">subscript</csymbol>
    <ci>normal-R</ci>
    <ci>i</ci>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \mathrm{R}_{i}\,
  </annotation>
 </semantics>
</math>

, and 

<math display="inline" id="Pseudo_amino_acid_composition:14">
 <semantics>
  <mpadded width="+1.7pt">
   <mi mathvariant="normal">Œì</mi>
  </mpadded>
  <annotation-xml encoding="MathML-Content">
   <ci>normal-Œì</ci>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \Gamma\,
  </annotation>
 </semantics>
</math>

 the total number of the functions considered. For example, in the original paper by Chou,<a class="footnoteRef" href="#fn5" id="fnref5"><sup>5</sup></a> 

<math display="inline" id="Pseudo_amino_acid_composition:15">
 <semantics>
  <mrow>
   <msub>
    <mi mathvariant="normal">Œ¶</mi>
    <mn>1</mn>
   </msub>
   <mrow>
    <mo>(</mo>
    <msub>
     <mi mathvariant="normal">R</mi>
     <mi>i</mi>
    </msub>
    <mo>)</mo>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <times></times>
    <apply>
     <csymbol cd="ambiguous">subscript</csymbol>
     <ci>normal-Œ¶</ci>
     <cn type="integer">1</cn>
    </apply>
    <apply>
     <csymbol cd="ambiguous">subscript</csymbol>
     <ci>normal-R</ci>
     <ci>i</ci>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \Phi_{1}\left(\mathrm{R}_{i}\right)
  </annotation>
 </semantics>
</math>

, 

<math display="inline" id="Pseudo_amino_acid_composition:16">
 <semantics>
  <mrow>
   <msub>
    <mi mathvariant="normal">Œ®</mi>
    <mn>2</mn>
   </msub>
   <mrow>
    <mo>(</mo>
    <msub>
     <mi mathvariant="normal">R</mi>
     <mi>i</mi>
    </msub>
    <mo>)</mo>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <times></times>
    <apply>
     <csymbol cd="ambiguous">subscript</csymbol>
     <ci>normal-Œ®</ci>
     <cn type="integer">2</cn>
    </apply>
    <apply>
     <csymbol cd="ambiguous">subscript</csymbol>
     <ci>normal-R</ci>
     <ci>i</ci>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \Psi_{2}\left(\mathrm{R}_{i}\right)
  </annotation>
 </semantics>
</math>

 and 

<math display="inline" id="Pseudo_amino_acid_composition:17">
 <semantics>
  <mrow>
   <msub>
    <mi mathvariant="normal">Œ®</mi>
    <mn>3</mn>
   </msub>
   <mrow>
    <mo>(</mo>
    <msub>
     <mi mathvariant="normal">R</mi>
     <mi>i</mi>
    </msub>
    <mo>)</mo>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <times></times>
    <apply>
     <csymbol cd="ambiguous">subscript</csymbol>
     <ci>normal-Œ®</ci>
     <cn type="integer">3</cn>
    </apply>
    <apply>
     <csymbol cd="ambiguous">subscript</csymbol>
     <ci>normal-R</ci>
     <ci>i</ci>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \Psi_{3}\left(\mathrm{R}_{i}\right)
  </annotation>
 </semantics>
</math>

 are respectively the hydrophobicity value, hydrophilicity value, and side chain mass of amino acid 

<math display="inline" id="Pseudo_amino_acid_composition:18">
 <semantics>
  <mpadded width="+1.7pt">
   <msub>
    <mi mathvariant="normal">R</mi>
    <mi>i</mi>
   </msub>
  </mpadded>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <csymbol cd="ambiguous">subscript</csymbol>
    <ci>normal-R</ci>
    <ci>i</ci>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \mathrm{R}_{i}\,
  </annotation>
 </semantics>
</math>

; while 

<math display="inline" id="Pseudo_amino_acid_composition:19">
 <semantics>
  <mrow>
   <msub>
    <mi mathvariant="normal">Œ¶</mi>
    <mn>1</mn>
   </msub>
   <mrow>
    <mo>(</mo>
    <msub>
     <mi mathvariant="normal">R</mi>
     <mrow>
      <mi>i</mi>
      <mo>+</mo>
      <mn>1</mn>
     </mrow>
    </msub>
    <mo>)</mo>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <times></times>
    <apply>
     <csymbol cd="ambiguous">subscript</csymbol>
     <ci>normal-Œ¶</ci>
     <cn type="integer">1</cn>
    </apply>
    <apply>
     <csymbol cd="ambiguous">subscript</csymbol>
     <ci>normal-R</ci>
     <apply>
      <plus></plus>
      <ci>i</ci>
      <cn type="integer">1</cn>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \Phi_{1}\left(\mathrm{R}_{i+1}\right)
  </annotation>
 </semantics>
</math>

, 

<math display="inline" id="Pseudo_amino_acid_composition:20">
 <semantics>
  <mrow>
   <msub>
    <mi mathvariant="normal">Œ¶</mi>
    <mn>2</mn>
   </msub>
   <mrow>
    <mo>(</mo>
    <msub>
     <mi mathvariant="normal">R</mi>
     <mrow>
      <mi>i</mi>
      <mo>+</mo>
      <mn>1</mn>
     </mrow>
    </msub>
    <mo>)</mo>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <times></times>
    <apply>
     <csymbol cd="ambiguous">subscript</csymbol>
     <ci>normal-Œ¶</ci>
     <cn type="integer">2</cn>
    </apply>
    <apply>
     <csymbol cd="ambiguous">subscript</csymbol>
     <ci>normal-R</ci>
     <apply>
      <plus></plus>
      <ci>i</ci>
      <cn type="integer">1</cn>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \Phi_{2}\left(\mathrm{R}_{i+1}\right)
  </annotation>
 </semantics>
</math>

 and 

<math display="inline" id="Pseudo_amino_acid_composition:21">
 <semantics>
  <mrow>
   <msub>
    <mi mathvariant="normal">Œ¶</mi>
    <mn>3</mn>
   </msub>
   <mrow>
    <mo>(</mo>
    <msub>
     <mi mathvariant="normal">R</mi>
     <mrow>
      <mi>i</mi>
      <mo>+</mo>
      <mn>1</mn>
     </mrow>
    </msub>
    <mo>)</mo>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <times></times>
    <apply>
     <csymbol cd="ambiguous">subscript</csymbol>
     <ci>normal-Œ¶</ci>
     <cn type="integer">3</cn>
    </apply>
    <apply>
     <csymbol cd="ambiguous">subscript</csymbol>
     <ci>normal-R</ci>
     <apply>
      <plus></plus>
      <ci>i</ci>
      <cn type="integer">1</cn>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \Phi_{3}\left(\mathrm{R}_{i+1}\right)
  </annotation>
 </semantics>
</math>

 the corresponding values for the amino acid 

<math display="inline" id="Pseudo_amino_acid_composition:22">
 <semantics>
  <mpadded width="+1.7pt">
   <msub>
    <mi mathvariant="normal">R</mi>
    <mrow>
     <mi>i</mi>
     <mo>+</mo>
     <mn>1</mn>
    </mrow>
   </msub>
  </mpadded>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <csymbol cd="ambiguous">subscript</csymbol>
    <ci>normal-R</ci>
    <apply>
     <plus></plus>
     <ci>i</ci>
     <cn type="integer">1</cn>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \mathrm{R}_{i+1}\,
  </annotation>
 </semantics>
</math>

. Therefore, the total number of functions considered there is 

<math display="inline" id="Pseudo_amino_acid_composition:23">
 <semantics>
  <mrow>
   <mi mathvariant="normal">Œì</mi>
   <mo>=</mo>
   <mpadded width="+1.7pt">
    <mn>3</mn>
   </mpadded>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <eq></eq>
    <ci>normal-Œì</ci>
    <cn type="integer">3</cn>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \Gamma=3\,
  </annotation>
 </semantics>
</math>

. It can be seen from <strong>Eq.3</strong> that the first 20 components, i.e. 

<math display="inline" id="Pseudo_amino_acid_composition:24">
 <semantics>
  <mrow>
   <msub>
    <mi>p</mi>
    <mn>1</mn>
   </msub>
   <mo rspace="4.2pt">,</mo>
   <msub>
    <mi>p</mi>
    <mn>2</mn>
   </msub>
   <mo rspace="4.2pt">,</mo>
   <mi mathvariant="normal">‚ãØ</mi>
   <mo rspace="4.2pt">,</mo>
   <msub>
    <mi>p</mi>
    <mn>20</mn>
   </msub>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <list>
    <apply>
     <csymbol cd="ambiguous">subscript</csymbol>
     <ci>p</ci>
     <cn type="integer">1</cn>
    </apply>
    <apply>
     <csymbol cd="ambiguous">subscript</csymbol>
     <ci>p</ci>
     <cn type="integer">2</cn>
    </apply>
    <ci>normal-‚ãØ</ci>
    <apply>
     <csymbol cd="ambiguous">subscript</csymbol>
     <ci>p</ci>
     <cn type="integer">20</cn>
    </apply>
   </list>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   p_{1},\,p_{2},\,\cdots,\,p_{20}
  </annotation>
 </semantics>
</math>

 are associated with the conventional AA composition of protein, while the remaining components 

<math display="inline" id="Pseudo_amino_acid_composition:25">
 <semantics>
  <mrow>
   <msub>
    <mi>p</mi>
    <mrow>
     <mn>20</mn>
     <mo>+</mo>
     <mn>1</mn>
    </mrow>
   </msub>
   <mo rspace="4.2pt">,</mo>
   <mi mathvariant="normal">‚ãØ</mi>
   <mo rspace="4.2pt">,</mo>
   <msub>
    <mi>p</mi>
    <mrow>
     <mn>20</mn>
     <mo>+</mo>
     <mi>Œª</mi>
    </mrow>
   </msub>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <list>
    <apply>
     <csymbol cd="ambiguous">subscript</csymbol>
     <ci>p</ci>
     <apply>
      <plus></plus>
      <cn type="integer">20</cn>
      <cn type="integer">1</cn>
     </apply>
    </apply>
    <ci>normal-‚ãØ</ci>
    <apply>
     <csymbol cd="ambiguous">subscript</csymbol>
     <ci>p</ci>
     <apply>
      <plus></plus>
      <cn type="integer">20</cn>
      <ci>Œª</ci>
     </apply>
    </apply>
   </list>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   p_{20+1},\,\cdots,\,p_{20+\lambda}
  </annotation>
 </semantics>
</math>

 are the correlation factors that reflect the 1st tier, 2nd tier, ‚Ä¶, and the 

<math display="inline" id="Pseudo_amino_acid_composition:26">
 <semantics>
  <mpadded width="+1.7pt">
   <mi>Œª</mi>
  </mpadded>
  <annotation-xml encoding="MathML-Content">
   <ci>Œª</ci>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \lambda\,
  </annotation>
 </semantics>
</math>

-th tier sequence order correlation patterns (<strong>Figure 1</strong>). It is through these additional 

<math display="inline" id="Pseudo_amino_acid_composition:27">
 <semantics>
  <mpadded width="+1.7pt">
   <mi>Œª</mi>
  </mpadded>
  <annotation-xml encoding="MathML-Content">
   <ci>Œª</ci>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \lambda\,
  </annotation>
 </semantics>
</math>

 factors that some important sequence-order effects are incorporated.</p>

<p>

<math display="inline" id="Pseudo_amino_acid_composition:28">
 <semantics>
  <mpadded width="+1.7pt">
   <mi>Œª</mi>
  </mpadded>
  <annotation-xml encoding="MathML-Content">
   <ci>Œª</ci>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \lambda\,
  </annotation>
 </semantics>
</math>

 in <strong>Eq.3</strong> is a parameter of integer and that choosing a different integer for 

<math display="inline" id="Pseudo_amino_acid_composition:29">
 <semantics>
  <mpadded width="+1.7pt">
   <mi>Œª</mi>
  </mpadded>
  <annotation-xml encoding="MathML-Content">
   <ci>Œª</ci>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \lambda\,
  </annotation>
 </semantics>
</math>

 will lead to a dimension-different PseAA composition.<a class="footnoteRef" href="#fn6" id="fnref6"><sup>6</sup></a></p>

<p>Using <strong>Eq.6</strong> is just one of the modes for deriving the correlation factors or PseAA components. The others, such as the physicochemical distance mode<a class="footnoteRef" href="#fn7" id="fnref7"><sup>7</sup></a> and amphiphilic pattern mode,<a class="footnoteRef" href="#fn8" id="fnref8"><sup>8</sup></a> can also be used to derive different types of PseAA composition, as summarized in a review paper.<a class="footnoteRef" href="#fn9" id="fnref9"><sup>9</sup></a></p>
<h2 id="applications">Applications</h2>

<p>Since <strong>PseAA composition</strong> was introduced, it has been widely used to predict various attributes of proteins, such as structural classes of proteins,<a class="footnoteRef" href="#fn10" id="fnref10"><sup>10</sup></a><a class="footnoteRef" href="#fn11" id="fnref11"><sup>11</sup></a> enzyme family classes and subfamily classes,<a class="footnoteRef" href="#fn12" id="fnref12"><sup>12</sup></a> GABA(A) receptor proteins,<a class="footnoteRef" href="#fn13" id="fnref13"><sup>13</sup></a> protein folding rates,<a class="footnoteRef" href="#fn14" id="fnref14"><sup>14</sup></a> cyclin proteins,<a class="footnoteRef" href="#fn15" id="fnref15"><sup>15</sup></a> supersecondary structure,<a class="footnoteRef" href="#fn16" id="fnref16"><sup>16</sup></a> subcellular location of proteins,<a class="footnoteRef" href="#fn17" id="fnref17"><sup>17</sup></a><a class="footnoteRef" href="#fn18" id="fnref18"><sup>18</sup></a> subnuclear location of proteins,<a class="footnoteRef" href="#fn19" id="fnref19"><sup>19</sup></a> apoptosis protein subcellular localization,<a class="footnoteRef" href="#fn20" id="fnref20"><sup>20</sup></a> submitochondria localization,<a class="footnoteRef" href="#fn21" id="fnref21"><sup>21</sup></a> protein quaternary structure,<a class="footnoteRef" href="#fn22" id="fnref22"><sup>22</sup></a><a class="footnoteRef" href="#fn23" id="fnref23"><sup>23</sup></a> bacterial secreted proteins,<a class="footnoteRef" href="#fn24" id="fnref24"><sup>24</sup></a> conotoxin superfamily and family classification,<a class="footnoteRef" href="#fn25" id="fnref25"><sup>25</sup></a> protease types,<a class="footnoteRef" href="#fn26" id="fnref26"><sup>26</sup></a> <a href="G_protein-coupled_receptor" title="wikilink">GPCR</a> types,<a class="footnoteRef" href="#fn27" id="fnref27"><sup>27</sup></a><a class="footnoteRef" href="#fn28" id="fnref28"><sup>28</sup></a> human papillomaviruses, <a class="footnoteRef" href="#fn29" id="fnref29"><sup>29</sup></a> outer membrane proteins,<a class="footnoteRef" href="#fn30" id="fnref30"><sup>30</sup></a> membrane protein types,<a class="footnoteRef" href="#fn31" id="fnref31"><sup>31</sup></a> protein secondary structural contents,<a class="footnoteRef" href="#fn32" id="fnref32"><sup>32</sup></a> metalloproteinase family <a class="footnoteRef" href="#fn33" id="fnref33"><sup>33</sup></a> subcellular localization of mycobacterial proteins,<a class="footnoteRef" href="#fn34" id="fnref34"><sup>34</sup></a> antibacterial peptides <a class="footnoteRef" href="#fn35" id="fnref35"><sup>35</sup></a> lipase types,<a class="footnoteRef" href="#fn36" id="fnref36"><sup>36</sup></a> allergenic proteins <a class="footnoteRef" href="#fn37" id="fnref37"><sup>37</sup></a> DNA-binding proteins,<a class="footnoteRef" href="#fn38" id="fnref38"><sup>38</sup></a> essential proteins,<a class="footnoteRef" href="#fn39" id="fnref39"><sup>39</sup></a> cell wall lytic enzymes,<a class="footnoteRef" href="#fn40" id="fnref40"><sup>40</sup></a> cofactors of oxidoreductases,<a class="footnoteRef" href="#fn41" id="fnref41"><sup>41</sup></a> among many other protein attributes and protein-related features (see, e.g., the review paper by Gonzalez-Diaz et al.<a class="footnoteRef" href="#fn42" id="fnref42"><sup>42</sup></a> as well as the relevant references cited therein).</p>

<p>Ever since the concept of PseAA composition was introduced, it has been widely utilized to predict various protein attributes. It has also been used to incorporate the <a href="protein_domain" title="wikilink">protein domain</a> or FunD (functional domain) information and GO (<a href="gene_ontology" title="wikilink">gene ontology</a>) information for improving the prediction quality for the subcellular localization of proteins.<a class="footnoteRef" href="#fn43" id="fnref43"><sup>43</sup></a> as well as their other attributes.</p>

<p>Meanwhile, the concept of PseAA composition has also stimulated the generation of pseudo-folding topological indices and pseudo-folding lattice network.<a class="footnoteRef" href="#fn44" id="fnref44"><sup>44</sup></a><a class="footnoteRef" href="#fn45" id="fnref45"><sup>45</sup></a><a class="footnoteRef" href="#fn46" id="fnref46"><sup>46</sup></a></p>

<p>Recently, two open accessible tools were established to generate various modes of Chou‚Äôs pseudo amino acid composition.<a class="footnoteRef" href="#fn47" id="fnref47"><sup>47</sup></a><a class="footnoteRef" href="#fn48" id="fnref48"><sup>48</sup></a></p>
<h2 id="references">References</h2>
<h2 id="external-links">External links</h2>
<ul>
<li><a href="http://chou.med.harvard.edu/bioinf/PseAAC/">PseAAC web server</a><a class="footnoteRef" href="#fn49" id="fnref49"><sup>49</sup></a></li>
</ul>

<p>"</p>

<p><a href="Category:Amino_acids" title="wikilink">Category:Amino acids</a> <a href="Category:Bioinformatics_algorithms" title="wikilink">Category:Bioinformatics algorithms</a></p>
<section class="footnotes">
<hr/>
<ol>
<li id="fn1"><a href="#fnref1">‚Ü©</a></li>
<li id="fn2"><a href="#fnref2">‚Ü©</a></li>
<li id="fn3"></li>
<li id="fn4"><a href="#fnref4">‚Ü©</a></li>
<li id="fn5"></li>
<li id="fn6"><a href="#fnref6">‚Ü©</a></li>
<li id="fn7"><a href="#fnref7">‚Ü©</a></li>
<li id="fn8"><a href="#fnref8">‚Ü©</a></li>
<li id="fn9"></li>
<li id="fn10"><a href="#fnref10">‚Ü©</a></li>
<li id="fn11"><a href="#fnref11">‚Ü©</a></li>
<li id="fn12"><a href="#fnref12">‚Ü©</a></li>
<li id="fn13"><a href="#fnref13">‚Ü©</a></li>
<li id="fn14"><a href="#fnref14">‚Ü©</a></li>
<li id="fn15"><a href="#fnref15">‚Ü©</a></li>
<li id="fn16"><a href="#fnref16">‚Ü©</a></li>
<li id="fn17"><a href="#fnref17">‚Ü©</a></li>
<li id="fn18"><a href="#fnref18">‚Ü©</a></li>
<li id="fn19"><a href="#fnref19">‚Ü©</a></li>
<li id="fn20"><a href="#fnref20">‚Ü©</a></li>
<li id="fn21"><a href="#fnref21">‚Ü©</a></li>
<li id="fn22"><a href="#fnref22">‚Ü©</a></li>
<li id="fn23"><a href="#fnref23">‚Ü©</a></li>
<li id="fn24"><a href="#fnref24">‚Ü©</a></li>
<li id="fn25"><a href="#fnref25">‚Ü©</a></li>
<li id="fn26"><a href="#fnref26">‚Ü©</a></li>
<li id="fn27"><a href="#fnref27">‚Ü©</a></li>
<li id="fn28"><a href="#fnref28">‚Ü©</a></li>
<li id="fn29"><a href="#fnref29">‚Ü©</a></li>
<li id="fn30"><a href="#fnref30">‚Ü©</a></li>
<li id="fn31"><a href="#fnref31">‚Ü©</a></li>
<li id="fn32"><a href="#fnref32">‚Ü©</a></li>
<li id="fn33"><a href="#fnref33">‚Ü©</a></li>
<li id="fn34"><a href="#fnref34">‚Ü©</a></li>
<li id="fn35"><a href="#fnref35">‚Ü©</a></li>
<li id="fn36"><a href="#fnref36">‚Ü©</a></li>
<li id="fn37"><a href="#fnref37">‚Ü©</a></li>
<li id="fn38"><a href="#fnref38">‚Ü©</a></li>
<li id="fn39"><a href="#fnref39">‚Ü©</a></li>
<li id="fn40"><a href="#fnref40">‚Ü©</a></li>
<li id="fn41"><a href="#fnref41">‚Ü©</a></li>
<li id="fn42"><a href="#fnref42">‚Ü©</a></li>
<li id="fn43"><a href="#fnref43">‚Ü©</a></li>
<li id="fn44"><a href="#fnref44">‚Ü©</a></li>
<li id="fn45"><a href="#fnref45">‚Ü©</a></li>
<li id="fn46"><a href="#fnref46">‚Ü©</a></li>
<li id="fn47"><a href="#fnref47">‚Ü©</a></li>
<li id="fn48"><a href="#fnref48">‚Ü©</a></li>
<li id="fn49"><a href="#fnref49">‚Ü©</a></li>
</ol>
</section>
</body>
</html>
