<html lang="en">
<head>
<meta charset="utf-8"/>
<title offset="1595">Riboflavin kinase</title>
<script src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_SVG.js" type="text/javascript">
</script>
</head>
<body>
<h1>Riboflavin kinase</h1>
<hr/>

<p>In <a class="uri" href="enzymology" title="wikilink">enzymology</a>, a <strong>riboflavin kinase</strong> () is an <a class="uri" href="enzyme" title="wikilink">enzyme</a> that <a href="catalysis" title="wikilink">catalyzes</a> the <a href="chemical_reaction" title="wikilink">chemical reaction</a></p>
<dl>
<dd>ATP + riboflavin 

<math display="inline" id="Riboflavin_kinase:0">
 <semantics>
  <mo>⇌</mo>
  <annotation-xml encoding="MathML-Content">
   <ci>normal-⇌</ci>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \rightleftharpoons
  </annotation>
 </semantics>
</math>

 ADP + FMN
</dd>
</dl>

<p>Thus, the two <a href="substrate_(biochemistry)" title="wikilink">substrates</a> of this enzyme are <a href="adenosine_triphosphate" title="wikilink">ATP</a> and <a class="uri" href="riboflavin" title="wikilink">riboflavin</a>, whereas its two <a href="product_(chemistry)" title="wikilink">products</a> are <a href="adenosine_diphosphate" title="wikilink">ADP</a> and <a href="flavin_mononucleotide" title="wikilink">FMN</a>.</p>

<p>Riboflavin is converted into catalytically active cofactors (FAD and FMN) by the actions of riboflavin kinase (), which converts it into FMN, and FAD synthetase (), which adenylates FMN to FAD. Eukaryotes usually have two separate enzymes, while most prokaryotes have a single bifunctional protein that can carry out both catalyses, although exceptions occur in both cases. While eukaryotic monofunctional riboflavin kinase is orthologous to the <a href="Prokaryotic_riboflavin_biosynthesis_protein" title="wikilink">bifunctional prokaryotic enzyme</a>,<a class="footnoteRef" href="#fn1" id="fnref1"><sup>1</sup></a> the monofunctional FAD synthetase differs from its prokaryotic counterpart, and is instead related to the PAPS-reductase family.<a class="footnoteRef" href="#fn2" id="fnref2"><sup>2</sup></a> The bacterial FAD synthetase that is part of the bifunctional enzyme has remote similarity to nucleotidyl transferases and, hence, it may be involved in the adenylylation reaction of FAD synthetases.<a class="footnoteRef" href="#fn3" id="fnref3"><sup>3</sup></a></p>

<p>This enzyme belongs to the family of <a href="transferase" title="wikilink">transferases</a>, to be specific, those transferring phosphorus-containing groups (<a href="phosphotransferase" title="wikilink">phosphotransferases</a>) with an alcohol group as acceptor. The systematic name of this enzyme class is <strong>ATP:riboflavin 5'-phosphotransferase</strong>. This enzyme is also called <strong>flavokinase</strong>. This enzyme participates in <a href="riboflavin_metabolism" title="wikilink">riboflavin metabolism</a>.</p>

<p>However, <a href="CTP-dependent_riboflavin_kinase" title="wikilink">archaeal riboflavin kinases</a> () in general utilize <a href="Cytidine_triphosphate" title="wikilink">CTP</a> rather than ATP as the donor nucleotide, catalyzing the reaction</p>
<dl>
<dd>CTP + riboflavin 

<math display="inline" id="Riboflavin_kinase:1">
 <semantics>
  <mo>⇌</mo>
  <annotation-xml encoding="MathML-Content">
   <ci>normal-⇌</ci>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \rightleftharpoons
  </annotation>
 </semantics>
</math>

 CDP + FMN <a class="footnoteRef" href="#fn4" id="fnref4"><sup>4</sup></a>
</dd>
</dl>

<p>Riboflavin kinase can also be isolated from other types of bacteria, all with similar function but a different number of amino acids.</p>
<h2 id="structure">Structure</h2>

<p>  The complete enzyme arrangement can be observed with <a href="X-ray_crystallography" title="wikilink">X-ray crystallography</a> and with <a class="uri" href="NMR" title="wikilink">NMR</a>. The riboflavin kinase enzyme isolated from <em><a href="Thermoplasma" title="wikilink">Thermoplasma acidophilum</a></em> contains 220 amino acids. The structure of this enzyme has been determined <a href="X-ray_crystallography" title="wikilink">X-ray crystallography</a> at a resolution of 2.20 Å. Its secondary structure contains 69 residues (30%) in <a href="alpha_helix" title="wikilink">alpha helix</a> form, and 60 residues (26%) a <a href="beta_sheet" title="wikilink">beta sheet</a> conformation. The enzyme contains a <a class="uri" href="magnesium" title="wikilink">magnesium</a> binding site at amino acids 131 and 133, and a <a href="Flavin_mononucleotide" title="wikilink">Flavin mononucleotide</a> binding site at amino acids 188 and 195.</p>

<p>As of late 2007, 14 <a href="tertiary_structure" title="wikilink">structures</a> have been solved for this class of enzymes, with <a href="Protein_Data_Bank" title="wikilink">PDB</a> accession codes , , , , , , , , , , , , , and .</p>
<h2 id="references">References</h2>
<h2 id="further-reading">Further reading</h2>
<ul>
<li></li>
<li></li>
<li></li>
<li></li>
<li></li>
<li></li>
<li></li>
</ul>

<p>"</p>

<p><a href="Category:EC_2.7.1" title="wikilink">Category:EC 2.7.1</a> <a href="Category:Enzymes_of_known_structure" title="wikilink">Category:Enzymes of known structure</a></p>
<section class="footnotes">
<hr/>
<ol>
<li id="fn1"><a href="#fnref1">↩</a></li>
<li id="fn2"><a href="#fnref2">↩</a></li>
<li id="fn3"><a href="#fnref3">↩</a></li>
<li id="fn4"><a href="#fnref4">↩</a></li>
</ol>
</section>
</body>
</html>
