<html lang="en">
<head>
<meta charset="utf-8"/>
<title offset="175">Selected reaction monitoring</title>
<script src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_SVG.js" type="text/javascript">
</script>
</head>
<body>
<h1>Selected reaction monitoring</h1>
<hr/>

<p> <strong>Selected reaction monitoring (SRM)</strong> is a method used in <a href="tandem_mass_spectrometry" title="wikilink">tandem mass spectrometry</a> in which an <a class="uri" href="ion" title="wikilink">ion</a> of a particular mass is selected in the first stage of a tandem <a href="mass_spectrometer" title="wikilink">mass spectrometer</a> and an ion product of a fragmentation reaction of the precursor ion is selected in the second mass spectrometer stage for detection.<a class="footnoteRef" href="#fn1" id="fnref1"><sup>1</sup></a></p>
<h2 id="variants">Variants</h2>

<p>A general case of SRM can be represented by</p>

<p>

<math display="block" id="Selected_reaction_monitoring:0">
 <semantics>
  <mrow>
   <mrow>
    <mi>A</mi>
    <mi>B</mi>
    <mi>C</mi>
    <msup>
     <mi>D</mi>
     <mo>+</mo>
    </msup>
   </mrow>
   <mo>→</mo>
   <mrow>
    <mrow>
     <mi>A</mi>
     <mi>B</mi>
    </mrow>
    <mo>+</mo>
    <mrow>
     <mi>C</mi>
     <msup>
      <mi>D</mi>
      <mo>+</mo>
     </msup>
    </mrow>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <ci>normal-→</ci>
    <apply>
     <times></times>
     <ci>A</ci>
     <ci>B</ci>
     <ci>C</ci>
     <apply>
      <csymbol cd="ambiguous">superscript</csymbol>
      <ci>D</ci>
      <plus></plus>
     </apply>
    </apply>
    <apply>
     <plus></plus>
     <apply>
      <times></times>
      <ci>A</ci>
      <ci>B</ci>
     </apply>
     <apply>
      <times></times>
      <ci>C</ci>
      <apply>
       <csymbol cd="ambiguous">superscript</csymbol>
       <ci>D</ci>
       <plus></plus>
      </apply>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   ABCD^{+}\to AB+CD^{+}
  </annotation>
 </semantics>
</math>

</p>

<p>where the precursor ion ABCD<sup>+</sup> is selected by the first stage of mass spectrometry (MS1), dissociates into molecule AB and product ion CD<sup>+</sup>, and the latter is selected by the second stage of mass spectrometry (MS2) and detected. The precursor and product ion pair is called a SRM "transition." <a class="footnoteRef" href="#fn2" id="fnref2"><sup>2</sup></a> <strong>Consecutive reaction monitoring</strong> (<strong>CRM</strong>) is the serial application of three or more stages of mass spectrometry to SRM, represented in a simple case by</p>

<p>

<math display="block" id="Selected_reaction_monitoring:1">
 <semantics>
  <mrow>
   <mrow>
    <mi>A</mi>
    <mi>B</mi>
    <mi>C</mi>
    <msup>
     <mi>D</mi>
     <mo>+</mo>
    </msup>
   </mrow>
   <mo>→</mo>
   <mrow>
    <mrow>
     <mi>A</mi>
     <mi>B</mi>
    </mrow>
    <mo>+</mo>
    <mrow>
     <mi>C</mi>
     <msup>
      <mi>D</mi>
      <mo>+</mo>
     </msup>
    </mrow>
   </mrow>
   <mo>→</mo>
   <mrow>
    <mi>C</mi>
    <mo>+</mo>
    <msup>
     <mi>D</mi>
     <mo>+</mo>
    </msup>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <and></and>
    <apply>
     <ci>normal-→</ci>
     <apply>
      <times></times>
      <ci>A</ci>
      <ci>B</ci>
      <ci>C</ci>
      <apply>
       <csymbol cd="ambiguous">superscript</csymbol>
       <ci>D</ci>
       <plus></plus>
      </apply>
     </apply>
     <apply>
      <plus></plus>
      <apply>
       <times></times>
       <ci>A</ci>
       <ci>B</ci>
      </apply>
      <apply>
       <times></times>
       <ci>C</ci>
       <apply>
        <csymbol cd="ambiguous">superscript</csymbol>
        <ci>D</ci>
        <plus></plus>
       </apply>
      </apply>
     </apply>
    </apply>
    <apply>
     <ci>normal-→</ci>
     <share href="#.cmml">
     </share>
     <apply>
      <plus></plus>
      <ci>C</ci>
      <apply>
       <csymbol cd="ambiguous">superscript</csymbol>
       <ci>D</ci>
       <plus></plus>
      </apply>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   ABCD^{+}\to AB+CD^{+}\to C+D^{+}
  </annotation>
 </semantics>
</math>

</p>

<p>where ABCD<sup>+</sup> is selected by MS1, dissociates into molecule AB and ion CD<sup>+</sup>.<a class="footnoteRef" href="#fn3" id="fnref3"><sup>3</sup></a> The ion is selected in the second mass spectrometry stage MS2 then undergoes further fragmentation to form ion D<sup>+</sup> which is selected in the third mass spectrometry stage MS3 and detected. <strong>Multiple reaction monitoring</strong> (<strong>MRM</strong>) is the application of selected reaction monitoring to multiple product ions from one or more precursor ions,<a class="footnoteRef" href="#fn4" id="fnref4"><sup>4</sup></a><a class="footnoteRef" href="#fn5" id="fnref5"><sup>5</sup></a> for example</p>

<p>

<math display="block" id="Selected_reaction_monitoring:2">
 <semantics>
  <mrow>
   <mrow>
    <mi>A</mi>
    <mi>B</mi>
    <mi>C</mi>
    <msup>
     <mi>D</mi>
     <mo>+</mo>
    </msup>
   </mrow>
   <mo>→</mo>
   <mrow>
    <mrow>
     <mi>A</mi>
     <mi>B</mi>
    </mrow>
    <mo>+</mo>
    <mrow>
     <mi>C</mi>
     <msup>
      <mi>D</mi>
      <mo>+</mo>
     </msup>
    </mrow>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <ci>normal-→</ci>
    <apply>
     <times></times>
     <ci>A</ci>
     <ci>B</ci>
     <ci>C</ci>
     <apply>
      <csymbol cd="ambiguous">superscript</csymbol>
      <ci>D</ci>
      <plus></plus>
     </apply>
    </apply>
    <apply>
     <plus></plus>
     <apply>
      <times></times>
      <ci>A</ci>
      <ci>B</ci>
     </apply>
     <apply>
      <times></times>
      <ci>C</ci>
      <apply>
       <csymbol cd="ambiguous">superscript</csymbol>
       <ci>D</ci>
       <plus></plus>
      </apply>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   ABCD^{+}\to AB+CD^{+}
  </annotation>
 </semantics>
</math>

</p>

<p>

<math display="block" id="Selected_reaction_monitoring:3">
 <semantics>
  <mrow>
   <mrow>
    <mi>A</mi>
    <mi>B</mi>
    <mi>C</mi>
    <msup>
     <mi>D</mi>
     <mo>+</mo>
    </msup>
   </mrow>
   <mo>→</mo>
   <mrow>
    <mrow>
     <mi>A</mi>
     <msup>
      <mi>B</mi>
      <mo>+</mo>
     </msup>
    </mrow>
    <mo>+</mo>
    <mrow>
     <mi>C</mi>
     <mi>D</mi>
    </mrow>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <ci>normal-→</ci>
    <apply>
     <times></times>
     <ci>A</ci>
     <ci>B</ci>
     <ci>C</ci>
     <apply>
      <csymbol cd="ambiguous">superscript</csymbol>
      <ci>D</ci>
      <plus></plus>
     </apply>
    </apply>
    <apply>
     <plus></plus>
     <apply>
      <times></times>
      <ci>A</ci>
      <apply>
       <csymbol cd="ambiguous">superscript</csymbol>
       <ci>B</ci>
       <plus></plus>
      </apply>
     </apply>
     <apply>
      <times></times>
      <ci>C</ci>
      <ci>D</ci>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   ABCD^{+}\to AB^{+}+CD
  </annotation>
 </semantics>
</math>

</p>

<p>where ABCD<sup>+</sup> is selected by MS1 and dissociates by two pathways, forming either AB<sup>+</sup> or CD<sup>+</sup>. The ions are selected sequentially by MS2 and detected. <strong>Parallel reaction monitoring</strong> (<strong>PRM</strong>) is the application of SRM with parallel detection of all transitions in a single analysis using a high resolution mass spectrometer.<a class="footnoteRef" href="#fn6" id="fnref6"><sup>6</sup></a></p>
<h2 id="proteomics">Proteomics</h2>

<p>SRM can be used for targeted <a href="quantitative_proteomics" title="wikilink">quantitative proteomics</a> by <a href="mass_spectrometry" title="wikilink">mass spectrometry</a>.<a class="footnoteRef" href="#fn7" id="fnref7"><sup>7</sup></a> Following <a class="uri" href="ionization" title="wikilink">ionization</a> in, for example, an <a href="electrospray_ionization" title="wikilink">electrospray</a> <a href="ion_source" title="wikilink">source</a>, a <a class="uri" href="peptide" title="wikilink">peptide</a> precursor is first isolated to obtain a substantial <a class="uri" href="ion" title="wikilink">ion</a> population of mostly the intended species. This population is then <a href="fragmentation_(chemistry)" title="wikilink">fragmented</a> to yield product ions whose signal abundances are indicative of the abundance of the peptide in the sample. This experiment can be performed on <a href="triple_quadrupole_mass_spectrometers" title="wikilink">triple quadrupole mass spectrometers</a>, where mass-resolving Q<sub>1</sub> isolates the precursor, q<sub>2</sub> acts as a collision cell, and mass-resolving Q<sub>3</sub> is cycled through the product ions which are detected upon exiting the last <a href="quadrupole_mass_analyzer" title="wikilink">quadrupole</a> by an <a href="electron_multiplier" title="wikilink">electron multiplier</a>. A precursor/product pair is often referred to as a <em>transition</em>. Much work goes into ensuring that transitions are selected that have maximum specificity.</p>

<p>Using <a href="isotopic_labeling" title="wikilink">isotopic labeling</a> with heavy-labeled (e.g., <a href="Deuterium" title="wikilink">D</a>, <a href="Carbon-13" title="wikilink"><sup>13</sup>C</a>, or <a href="Nitrogen-15" title="wikilink"><sup>15</sup>N</a>) peptides to a complex matrix as <a href="standard_solution" title="wikilink">concentration standards</a>, SRM can be used to construct a <a href="calibration_curve" title="wikilink">calibration curve</a> that can provide the absolute quantification (i.e., copy number per <a href="cell_(biology)" title="wikilink">cell</a>) of the native, light peptide, and by extension, its parent <a class="uri" href="protein" title="wikilink">protein</a>.<a class="footnoteRef" href="#fn8" id="fnref8"><sup>8</sup></a></p>

<p>SRM has been used to identify the proteins encoded by wild-type and mutant genes (<a href="mutant_proteins" title="wikilink">mutant proteins</a>) and quantify their absolute copy numbers in tumors and biological fluids, thus answering the basic questions about the absolute copy number of proteins in a single cell, which will be essential in digital modelling of mammalian cells and human body, and the relative levels of genetically abnormal proteins in tumors, and proving useful for diagnostic applications.<a class="footnoteRef" href="#fn9" id="fnref9"><sup>9</sup></a> SRM has also been used as a method of triggering full product ion scans of peptides to either a) confirm the specificity of the SRM transition, or b) detect specific <a href="post-translational_modifications" title="wikilink">post-translational modifications</a> which are below the limit of detection of standard MS analyses.<a class="footnoteRef" href="#fn10" id="fnref10"><sup>10</sup></a></p>
<h2 id="see-also">See also</h2>
<ul>
<li><a href="Protein_mass_spectrometry" title="wikilink">Protein mass spectrometry</a></li>
</ul>
<h2 id="references">References</h2>

<p>"</p>

<p><a href="Category:Mass_spectrometry" title="wikilink">Category:Mass spectrometry</a></p>
<section class="footnotes">
<hr/>
<ol>
<li id="fn1"><a href="#fnref1">↩</a></li>
<li id="fn2"><a href="#fnref2">↩</a></li>
<li id="fn3"><a href="#fnref3">↩</a></li>
<li id="fn4"><a href="#fnref4">↩</a></li>
<li id="fn5"></li>
<li id="fn6"><a href="#fnref6">↩</a></li>
<li id="fn7"><a href="#fnref7">↩</a></li>
<li id="fn8"><a href="#fnref8">↩</a></li>
<li id="fn9"><a href="#fnref9">↩</a></li>
<li id="fn10"><a href="#fnref10">↩</a></li>
</ol>
</section>
</body>
</html>
