<html lang="en">
<head>
<meta charset="utf-8"/>
<title offset="542">SNV calling from NGS data</title>
<script src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_SVG.js" type="text/javascript">
</script>
</head>
<body>
<h1>SNV calling from NGS data</h1>
<hr/>

<p><strong>SNV calling from NGS data</strong> refers to a range of methods for identifying the existence of <a href="single_nucleotide_polymorphism" title="wikilink">single nucleotide variants</a> (SNVs) from the results of <a href="next_generation_sequencing" title="wikilink">next generation sequencing</a> (NGS) experiments. These are computational techniques, and are in contrast to special experimental methods based on known population-wide single nucleotide polymorphisms (see <a href="SNP_genotyping" title="wikilink">SNP genotyping</a>). Due to the increasing abundance of NGS data, these techniques are becoming increasingly popular for performing SNP genotyping, with a wide variety of algorithms designed for specific experimental designs and applications.<a class="footnoteRef" href="#fn1" id="fnref1"><sup>1</sup></a> In addition to the usual application domain of SNP genotyping, these techniques have been successfully adapted to identify rare SNPs within a population,<a class="footnoteRef" href="#fn2" id="fnref2"><sup>2</sup></a> as well as detecting <a href="Somatic_(biology)" title="wikilink">somatic</a> SNVs within an individual using multiple tissue samples.<a class="footnoteRef" href="#fn3" id="fnref3"><sup>3</sup></a></p>
<h2 id="methods-for-detecting-germline-variants">Methods for detecting germline variants</h2>

<p>Most NGS based methods for SNV detection are designed to detect <a class="uri" href="germline" title="wikilink">germline</a> variations in the individual's genome. These are the mutations that an individual biologically inherits from their parents, and are the usual type of variants searched for when performing such analysis (except for certain specific applications where <a href="#Methods_for_detecting_somatic_variants" title="wikilink">somatic mutations</a> are sought). Very often, the searched for variants occur with some (possibly rare) frequency, throughout the population, in which case they may be referred to as <a href="single_nucleotide_polymorphisms" title="wikilink">single nucleotide polymorphisms</a> (SNPs). Technically the term SNP only refers to these kinds of variations, however in practice they are often used synonymously with SNV in the literature on variant calling. In addition, since the detection of germline SNVs requires determining the individual's genotype at each locus, the phrase "SNP genotyping" may also be used to refer to this process. However this phrase may also refer to wet-lab experimental procedures for classifying genotypes at a set of known SNP locations.</p>

<p>The usual process of such techniques are based around:<a class="footnoteRef" href="#fn4" id="fnref4"><sup>4</sup></a></p>
<ol>
<li>Filtering the set of NGS reads to remove sources of error/bias</li>
<li>Aligning the reads to a reference genome</li>
<li>Using an algorithm, either based on a statistical model or some heuristics, to predict the likelihood of variation at each locus, based on the quality scores and allele counts of the aligned reads at that locus</li>
<li>Filtering the predicted results, often based on metrics relevant to the application</li>
<li><a href="SNP_annotation" title="wikilink">SNP annotation</a> to predict the functional effect of each variation.</li>
</ol>

<p>The usual output of these procedures is a <a href="Variant_Call_Format" title="wikilink">VCF</a> file.</p>
<h3 id="probabilistic-methods">Probabilistic methods</h3>
<figure><b>(Figure)</b>
<figcaption>A set of hypothetical NGS reads are shown, aligned against a reference sequence. At the annotated locus, the reads contain a mixture of A/G nucleotides, against the A reference allele. Depending on the prior genotype probabilities, and the chosen error model, this may be called as a heterozygous SNV (genotype AG predicted), the G nucleotides may be classified as errors and no variant called (genotype AA predicted), or alternatively the A nucleotides may be classified as errors and a homozygous SNV valled (genotype GG predicted).</figcaption>
</figure>

<p>In an ideal error free world with high read coverage, the task of variant calling from the results of a NGS data alignment would be simple; at each <a href="Locus_(genetics)" title="wikilink">locus</a> (position on the genome) the number of occurrences of each distinct nucleotide among the reads aligned at that position can be counted, and the true genotype would be obvious; either <strong>AA</strong> if all nucleotides match allele <strong>A</strong>, <strong>BB</strong> if they match allele <strong>B</strong>, or <strong>AB</strong> if there is a mixture. However when working with real NGS data this sort of naive approach is not used, as it cannot account for the noise in the input data.<a class="footnoteRef" href="#fn5" id="fnref5"><sup>5</sup></a> The nucleotide counts used for base calling contain errors and bias, both due do the sequenced reads themselves, and the alignment process. This issue can be mitigated to some extent by sequencing to a greater depth of read coverage, however this is often expensive, and many practical studies require making inferences on low coverage data.<a class="footnoteRef" href="#fn6" id="fnref6"><sup>6</sup></a></p>

<p>Probabilistic methods aim to overcome the above issue, by producing robust estimates of the probabilities of each of the possible genotypes, taking into account noise, as well as other available prior information that can be used to improve estimates. A genotype can then be predicted based on these probabilities, often according to the <a href="Maximum_a_posteriori_estimation" title="wikilink">MAP</a> estimate.</p>

<p>Probabilistic methods for variant calling are based on <a href="Bayes_Theorem" title="wikilink">Bayes' Theorem</a>. In the context of variant calling, Bayes' Theorem defines the probability of each genotype being the true genotype given the observed data, in terms of the prior probabilities of each possible genotype, and the probability distribution of the data given each possible genotype. The formula is:</p>

<p>

<math display="inline" id="SNV_calling_from_NGS_data:0">
 <semantics>
  <mrow>
   <mi>P</mi>
   <mrow>
    <mo stretchy="false">(</mo>
    <mi>G</mi>
    <mo stretchy="false">|</mo>
    <mi>D</mi>
    <mo stretchy="false">)</mo>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <cerror>
    <csymbol cd="ambiguous">fragments</csymbol>
    <csymbol cd="unknown">P</csymbol>
    <cerror>
     <csymbol cd="ambiguous">fragments</csymbol>
     <ci>normal-(</ci>
     <csymbol cd="unknown">G</csymbol>
     <ci>normal-|</ci>
     <csymbol cd="unknown">D</csymbol>
     <ci>normal-)</ci>
    </cerror>
   </cerror>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \displaystyle P(G|D)
  </annotation>
 </semantics>
</math>


 In the above equation:</p>
<ul>
<li>

<math display="inline" id="SNV_calling_from_NGS_data:1">
 <semantics>
  <mi>D</mi>
  <annotation-xml encoding="MathML-Content">
   <ci>D</ci>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   D
  </annotation>
 </semantics>
</math>


 refers to the observed data; that is, the aligned reads</li>
<li>

<math display="inline" id="SNV_calling_from_NGS_data:2">
 <semantics>
  <mi>G</mi>
  <annotation-xml encoding="MathML-Content">
   <ci>G</ci>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   G
  </annotation>
 </semantics>
</math>

 is the genotype whose probability is being calculated</li>
<li>

<math display="inline" id="SNV_calling_from_NGS_data:3">
 <semantics>
  <msub>
   <mi>G</mi>
   <mi>i</mi>
  </msub>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <csymbol cd="ambiguous">subscript</csymbol>
    <ci>G</ci>
    <ci>i</ci>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   G_{i}
  </annotation>
 </semantics>
</math>

 refers to the <em>i</em>th possible genotype, out of <em>n</em> possibilities</li>
</ul>

<p>Given the above framework, different software solutions for detecting SNVs vary based on how they calculate the prior probabilities 

<math display="inline" id="SNV_calling_from_NGS_data:4">
 <semantics>
  <mrow>
   <mi>P</mi>
   <mrow>
    <mo stretchy="false">(</mo>
    <mi>G</mi>
    <mo stretchy="false">)</mo>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <times></times>
    <ci>P</ci>
    <ci>G</ci>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   P(G)
  </annotation>
 </semantics>
</math>

, the error model used to model the probabilities 

<math display="inline" id="SNV_calling_from_NGS_data:5">
 <semantics>
  <mrow>
   <mi>P</mi>
   <mrow>
    <mo stretchy="false">(</mo>
    <mi>D</mi>
    <mo stretchy="false">|</mo>
    <mi>G</mi>
    <mo stretchy="false">)</mo>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <cerror>
    <csymbol cd="ambiguous">fragments</csymbol>
    <csymbol cd="unknown">P</csymbol>
    <cerror>
     <csymbol cd="ambiguous">fragments</csymbol>
     <ci>normal-(</ci>
     <csymbol cd="unknown">D</csymbol>
     <ci>normal-|</ci>
     <csymbol cd="unknown">G</csymbol>
     <ci>normal-)</ci>
    </cerror>
   </cerror>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   P(D|G)
  </annotation>
 </semantics>
</math>

, and the partitioning of the overall genotypes into separate sub-genotypes, whose probabilities can be individually estimated in this framework.<a class="footnoteRef" href="#fn7" id="fnref7"><sup>7</sup></a></p>
<h4 id="prior-genotype-probability-estimation">Prior genotype probability estimation</h4>

<p>The calculation of prior probabilities depends on available data from the genome being studied, and the type of analysis being performed. For studies where good reference data containing frequencies of known mutations is available (for example, in studying human genome data), these known frequencies of genotypes in the population can be used to estimate priors. Given population wide allele frequencies, prior genotype probabilities can be calculated at each locus according to the <a href="Hardy_Weinberg_Equilibrium" title="wikilink">Hardy Weinberg Equilibrium</a>.<a class="footnoteRef" href="#fn8" id="fnref8"><sup>8</sup></a> In the absence of such data, constant priors can be used, independent of the locus. These can be set using heuristically chosen values, possibly informed by the kind of variations being sought by the study. Alternatively, supervised machine-learning procedures have been investigated that seek to learn optimal prior values for individuals in a sample, using supplied NGS data from these individuals.<a class="footnoteRef" href="#fn9" id="fnref9"><sup>9</sup></a></p>
<h4 id="error-models-for-data-observations">Error models for data observations</h4>

<p>The error model used in creating a probabilistic method for variant calling is the basis for calculating the 

<math display="inline" id="SNV_calling_from_NGS_data:6">
 <semantics>
  <mrow>
   <mi>P</mi>
   <mrow>
    <mo stretchy="false">(</mo>
    <mi>D</mi>
    <mo stretchy="false">|</mo>
    <mi>G</mi>
    <mo stretchy="false">)</mo>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <cerror>
    <csymbol cd="ambiguous">fragments</csymbol>
    <csymbol cd="unknown">P</csymbol>
    <cerror>
     <csymbol cd="ambiguous">fragments</csymbol>
     <ci>normal-(</ci>
     <csymbol cd="unknown">D</csymbol>
     <ci>normal-|</ci>
     <csymbol cd="unknown">G</csymbol>
     <ci>normal-)</ci>
    </cerror>
   </cerror>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   P(D|G)
  </annotation>
 </semantics>
</math>


 term used in Bayes' theorem. If the data was assumed to be error free, then the distribution of observed nucleotide counts at each locus would follow a <a href="Binomial_Distribution" title="wikilink">Binomial Distribution</a>, with 100% of nucleotides matching the A or B allele respectively in the <strong>AA</strong> and <strong>BB</strong> cases, and a 50% chance of each nucleotide matching either <strong>A</strong> or <strong>B</strong> in the <strong>AB</strong> case. However in presence of noise in the read data this assumption is violated, and the 

<math display="inline" id="SNV_calling_from_NGS_data:7">
 <semantics>
  <mrow>
   <mi>P</mi>
   <mrow>
    <mo stretchy="false">(</mo>
    <mi>D</mi>
    <mo stretchy="false">|</mo>
    <mi>G</mi>
    <mo stretchy="false">)</mo>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <cerror>
    <csymbol cd="ambiguous">fragments</csymbol>
    <csymbol cd="unknown">P</csymbol>
    <cerror>
     <csymbol cd="ambiguous">fragments</csymbol>
     <ci>normal-(</ci>
     <csymbol cd="unknown">D</csymbol>
     <ci>normal-|</ci>
     <csymbol cd="unknown">G</csymbol>
     <ci>normal-)</ci>
    </cerror>
   </cerror>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   P(D|G)
  </annotation>
 </semantics>
</math>

 values need to account for the possibility that erroneous nucleotides are present in the aligned reads at each locus.</p>

<p>A simple error model is to introduce a small error to the data probability term in the homozygous cases, allowing a small constant probability that nucleotides which don't match the <strong>A</strong> allele are observed in the <strong>AA</strong> case, and respectively a small constant probability that nucleotides not matching the <strong>B</strong> allele are observed in the <strong>BB</strong> case. However more sophisticated procedures are available which attempt to more realistically replicate the actual error patterns observed in real data in calculating the conditional data probabilities. For instance, estimations of read quality (measured as <a href="Phred_quality_score" title="wikilink">Phred</a> quality scores) have been incorporated in these calculations, taking into account the expected error rate in each individual read at a locus.<a class="footnoteRef" href="#fn10" id="fnref10"><sup>10</sup></a> Another technique that has successfully been incorporated into error models is base quality recalibration, where separate error rates are calculated - based on prior known information about error patterns - for each possible nucleotide substitution. Research shows that each possible nucleotide substitution is not equally likely to show up as an error in sequencing data, and so base quality recalibration has been applied to improve error probability estimates.<a class="footnoteRef" href="#fn11" id="fnref11"><sup>11</sup></a></p>
<h4 id="partitioning-of-the-genotype">Partitioning of the genotype</h4>

<p>In the above discussion, it has been assumed that the genotype probabilities at each locus are calculated independently; that is, the entire genotype is partitioned into independent genotypes at each locus, whose probabilities are calculated independently. However due to <a href="linkage_disequilibrium" title="wikilink">linkage disequilibrium</a> the genotypes of nearby loci are in general not independent. As a result, partitioning the overall genotype instead into a sequence of overlapping <a class="uri" href="haplotypes" title="wikilink">haplotypes</a> allows these correlations to be modelled, resulting in more precise probability estimates through the incorporation of population-wide haplotype frequencies in the prior. The use of haplotypes to improve variant detection accuracy has been applied successfully, for instance in the <a href="1000_Genomes_Project" title="wikilink">1000 Genomes Project</a>.<a class="footnoteRef" href="#fn12" id="fnref12"><sup>12</sup></a></p>
<h3 id="heuristic-based-algorithms">Heuristic based algorithms</h3>

<p>As an alternative to probabilistic methods, <a href="Heuristic_(computer_science)" title="wikilink">heuristic</a> methods exist for performing variant calling on NGS data. Instead of modelling the distribution of the observed data and using Bayesian statistics to calculate genotype probabilities, variant calls are made based on a variety of heuristic factors, such as minimum allele counts, read quality cut-offs, bounds on read depth, etc. Although they have been relatively unpopular in practice in comparison to probabilistic methods, in practice due to their use of bounds and cut-offs they can be robust to outlying data that violate the assumptions of probabilistic models.<a class="footnoteRef" href="#fn13" id="fnref13"><sup>13</sup></a></p>
<h3 id="reference-genome-used-for-alignment">Reference genome used for alignment</h3>

<p>An important part of the design of variant calling methods using NGS data is the DNA sequence used as a reference to align the NGS reads to. In human genetics studies, high quality references are available, from sources such as the HapMap project,<a class="footnoteRef" href="#fn14" id="fnref14"><sup>14</sup></a> which can substantially improve the accuracy of the variant calls made by variant calling algorithms. As a bonus, such references can be a source of prior genotype probabilities for Bayesian based analysis. However in the absence of such a high quality reference, experimentally obtained reads can first be <a href="sequence_assembly" title="wikilink">assembled</a> in order to create a reference sequence for alignment.<a class="footnoteRef" href="#fn15" id="fnref15"><sup>15</sup></a></p>
<h3 id="pre-processing-and-filtering-of-results">Pre-processing and filtering of results</h3>

<p>Various methods exist for filtering data in variant calling experiments, in order to remove sources of error/bias. This can involve the removal of suspicious reads before performing alignment and/or filtering of the list of variants returned by the variant calling algorithm.</p>

<p>Depending on the sequencing platform used, various biases may exist within the set of sequenced reads. For instance, strand bias can occur, where there is a highly unequal distribution of forward vs reverse directions in the reads aligned in some neighborhood. Additionally, there may occur an unusually high duplication of some reads (for instance due to bias in <a href="Polymerase_chain_reaction" title="wikilink">PCR</a>). Such biases can result in dubious variant calls - for instance if a fragment containing a PCR error at some locus is over amplified due to PCR bias, that locus will have a high count of the false allele, and may be called as a SNV - and so analysis pipelines frequently filter calls based on these biases.<a class="footnoteRef" href="#fn16" id="fnref16"><sup>16</sup></a></p>
<h2 id="methods-for-detecting-somatic-variants">Methods for detecting somatic variants</h2>

<p>In addition to methods that align reads from individual sample(s) to a reference genome in order to detect <a class="uri" href="germline" title="wikilink">germline</a> genetic variants, reads from multiple tissue samples within a single individual can be aligned and compared in order to detect somatic variants. These variants correspond to <a href="genetic_mutation" title="wikilink">mutations</a> that have occurred <a href="Mutation" title="wikilink">de novo</a> within groups of <a href="somatic_cell" title="wikilink">somatic cells</a> within an individual (that is, they are not present within the individual's germline cells). This form of analysis has been frequently applied to the study of <a class="uri" href="cancer" title="wikilink">cancer</a>, where many studies are designed around investigating the profile of somatic mutations within cancerous tissues. Such investigations have resulted in diagnostic tools that have seen clinical application, and are used to improve scientific understanding of the disease, for instance by the discovery of new cancer-related genes, identification of involved <a href="gene_regulatory_networks" title="wikilink">gene regulatory networks</a> and <a href="metabolic_pathways" title="wikilink">metabolic pathways</a>, and by informing models of how tumors grow and evolve.<a class="footnoteRef" href="#fn17" id="fnref17"><sup>17</sup></a></p>
<h3 id="recent-developments">Recent developments</h3>

<p>Until recently, software tools for carrying out this form of analysis have been heavily underdeveloped, and were based on the same algorithms used to detect germline variations. Such procedures are not optimized for this task, because they do not adequately model the statistical correlation between the genotypes present in multiple tissue samples from the same individual.<a class="footnoteRef" href="#fn18" id="fnref18"><sup>18</sup></a></p>

<p>More recent investigations have resulted in the development of software tools especially optimized for the detection of somatic mutations from multiple tissue samples. Probabilistic techniques have been developed that pool allele counts from all tissue samples at each locus, and using statistical models for the likelihoods of joint-genotypes for all the tissues, and the distribution of allele counts given the genotype, are able to calculate relatively robust probabilities of somatic mutations at each locus using all available data.<a class="footnoteRef" href="#fn19" id="fnref19"><sup>19</sup></a><a class="footnoteRef" href="#fn20" id="fnref20"><sup>20</sup></a> In addition there has recently been some investigation in <a href="machine_learning" title="wikilink">machine learning</a> based techniques for performing this analysis.<a class="footnoteRef" href="#fn21" id="fnref21"><sup>21</sup></a></p>
<h2 id="list-of-available-software">List of available software</h2>
<ul>
<li><a href="http://soap.genomics.org.cn/index.html">SOAPsnp</a></li>
<li><a href="http://128.32.118.212/thorfinn/realSFS">realSFS</a></li>
<li><a href="http://samtools.sourceforge.net">Samtools</a></li>
<li><a href="http://www.broadinstitute.org/gsa/wiki/index.php/The_Genome_Analysis_Toolkit">GATK</a></li>
<li><a href="http://faculty.washington.edu/browning/beagle/beagle.html">Beagle</a></li>
<li><a href="http://mathgen.stats.ox.ac.uk/impute/impute_v2.html">IMPUTE2</a></li>
<li><a href="http://genome.sph.umich.edu/wiki/Thunder">MaCH</a></li>
<li><a href="http://compbio.bccrc.ca/software/snvmix">SNVmix</a></li>
<li><a href="http://varscan.sourceforge.net">VarScan</a></li>
<li><a href="http://gmt.genome.wustl.edu/somatic-sniper">Somaticsniper</a></li>
<li><a href="http://compbio.bccrc.ca/software/jointsnvmix">JointSNVMix</a></li>
</ul>
<h2 id="references">References</h2>

<p>"</p>

<p><a href="Category:DNA_sequencing" title="wikilink">Category:DNA sequencing</a></p>
<section class="footnotes">
<hr/>
<ol>
<li id="fn1"><a href="#fnref1">↩</a></li>
<li id="fn2"><a href="#fnref2">↩</a></li>
<li id="fn3"><a href="#fnref3">↩</a></li>
<li id="fn4"></li>
<li id="fn5"><a href="#fnref5">↩</a></li>
<li id="fn6"></li>
<li id="fn7"><a href="#fnref7">↩</a></li>
<li id="fn8"><a href="#fnref8">↩</a></li>
<li id="fn9"><a href="#fnref9">↩</a></li>
<li id="fn10"><a href="#fnref10">↩</a></li>
<li id="fn11"></li>
<li id="fn12"><a href="#fnref12">↩</a></li>
<li id="fn13"><a href="#fnref13">↩</a></li>
<li id="fn14"><a href="#fnref14">↩</a></li>
<li id="fn15"></li>
<li id="fn16"></li>
<li id="fn17"><a href="#fnref17">↩</a></li>
<li id="fn18"></li>
<li id="fn19"></li>
<li id="fn20"><a href="#fnref20">↩</a></li>
<li id="fn21"><a href="#fnref21">↩</a></li>
</ol>
</section>
</body>
</html>
