<html lang="en">
<head>
<meta charset="utf-8"/>
<title offset="746">Statistical coupling analysis</title>
<script src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_SVG.js" type="text/javascript">
</script>
</head>
<body>
<h1>Statistical coupling analysis</h1>
<hr/>

<p><strong>Statistical coupling analysis</strong> or <strong>SCA</strong> is a technique used in <a class="uri" href="bioinformatics" title="wikilink">bioinformatics</a> to measure <a class="uri" href="covariation" title="wikilink">covariation</a> between pairs of <a href="amino_acids" title="wikilink">amino acids</a> in a protein <a href="multiple_sequence_alignment" title="wikilink">multiple sequence alignment</a> (MSA). More specifically, it quantifies how much the amino acid distribution at some position <em>i</em> changes upon a perturbation of the amino acid distribution at another position <em>j</em>. The resulting <strong>statistical coupling energy</strong> indicates the degree of evolutionary dependence between the residues, with higher coupling energy corresponding to increased dependence.<a class="footnoteRef" href="#fn1" id="fnref1"><sup>1</sup></a></p>
<h2 id="definition-of-statistical-coupling-energy">Definition of statistical coupling energy</h2>

<p>Statistical coupling energy measures how a perturbation of amino acid distribution at one site in an MSA affects the amino acid distribution at another site. For example, consider a multiple sequence alignment with sites (or columns) <em>a</em> through <em>z</em>, where each site has some distribution of amino acids. At position <em>i</em>, 60% of the sequences have a <a class="uri" href="valine" title="wikilink">valine</a> and the remaining 40% of sequences have a <a class="uri" href="leucine" title="wikilink">leucine</a>, at position <em>j</em> the distribution is 40% <a class="uri" href="isoleucine" title="wikilink">isoleucine</a>, 40% <a class="uri" href="histidine" title="wikilink">histidine</a> and 20% <a class="uri" href="methionine" title="wikilink">methionine</a>, <em>k</em> has an average distribution (the 20 amino acids are present at roughly the same frequencies seen in all proteins), and <em>l</em> has 80% histidine, 20% valine. Since positions <em>i</em>, <em>j</em> and <em>l</em> have an amino acid distribution different from the mean distribution observed in all proteins, they are said to have some degree of <strong>conservation</strong>.</p>

<p>In statistical coupling analysis, the conservation (ΔG<sup>stat</sup>) at each site (<em>i</em>) is defined as

<math display="block" id="Statistical_coupling_analysis:0">
 <semantics>
  <mrow>
   <mrow>
    <mi mathvariant="normal">Δ</mi>
    <msubsup>
     <mi>G</mi>
     <mi>i</mi>
     <mrow>
      <mi>s</mi>
      <mi>t</mi>
      <mi>a</mi>
      <mi>t</mi>
     </mrow>
    </msubsup>
   </mrow>
   <mo>=</mo>
   <msqrt>
    <mrow>
     <munder>
      <mo largeop="true" movablelimits="false" symmetric="true">∑</mo>
      <mi>x</mi>
     </munder>
     <msup>
      <mrow>
       <mo stretchy="false">(</mo>
       <mrow>
        <mi>ln</mi>
        <msubsup>
         <mi>P</mi>
         <mi>i</mi>
         <mi>x</mi>
        </msubsup>
       </mrow>
       <mo stretchy="false">)</mo>
      </mrow>
      <mn>2</mn>
     </msup>
    </mrow>
   </msqrt>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <eq></eq>
    <apply>
     <times></times>
     <ci>normal-Δ</ci>
     <apply>
      <csymbol cd="ambiguous">superscript</csymbol>
      <apply>
       <csymbol cd="ambiguous">subscript</csymbol>
       <ci>G</ci>
       <ci>i</ci>
      </apply>
      <apply>
       <times></times>
       <ci>s</ci>
       <ci>t</ci>
       <ci>a</ci>
       <ci>t</ci>
      </apply>
     </apply>
    </apply>
    <apply>
     <root></root>
     <apply>
      <apply>
       <csymbol cd="ambiguous">subscript</csymbol>
       <sum></sum>
       <ci>x</ci>
      </apply>
      <apply>
       <csymbol cd="ambiguous">superscript</csymbol>
       <apply>
        <ln></ln>
        <apply>
         <csymbol cd="ambiguous">superscript</csymbol>
         <apply>
          <csymbol cd="ambiguous">subscript</csymbol>
          <ci>P</ci>
          <ci>i</ci>
         </apply>
         <ci>x</ci>
        </apply>
       </apply>
       <cn type="integer">2</cn>
      </apply>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \Delta G_{i}^{stat}=\sqrt{\sum_{x}(\ln P_{i}^{x})^{2}}
  </annotation>
 </semantics>
</math>

.<a class="footnoteRef" href="#fn2" id="fnref2"><sup>2</sup></a></p>

<p>Here, P<sub>i</sub><sup>x</sup> describes the probability of finding amino acid <em>x</em> at position <em>i</em>, and is defined by a function in <a href="Binomial_distribution#Mean" title="wikilink">binomial form</a> as follows:</p>
<div align="center">

<p>

<math display="inline" id="Statistical_coupling_analysis:1">
 <semantics>
  <mrow>
   <msubsup>
    <mi>P</mi>
    <mi>i</mi>
    <mi>x</mi>
   </msubsup>
   <mo>=</mo>
   <mrow>
    <mfrac>
     <mrow>
      <mi>N</mi>
      <mo lspace="0pt" rspace="3.5pt">!</mo>
     </mrow>
     <mrow>
      <mrow>
       <msub>
        <mi>n</mi>
        <mi>x</mi>
       </msub>
       <mo lspace="0pt" rspace="3.5pt">!</mo>
      </mrow>
      <mrow>
       <mrow>
        <mo stretchy="false">(</mo>
        <mrow>
         <mi>N</mi>
         <mo>-</mo>
         <msub>
          <mi>n</mi>
          <mi>x</mi>
         </msub>
        </mrow>
        <mo stretchy="false">)</mo>
       </mrow>
       <mo lspace="0pt" rspace="3.5pt">!</mo>
      </mrow>
     </mrow>
    </mfrac>
    <msubsup>
     <mi>p</mi>
     <mi>x</mi>
     <msub>
      <mi>n</mi>
      <mi>x</mi>
     </msub>
    </msubsup>
    <msup>
     <mrow>
      <mo stretchy="false">(</mo>
      <mrow>
       <mn>1</mn>
       <mo>-</mo>
       <msub>
        <mi>p</mi>
        <mi>x</mi>
       </msub>
      </mrow>
      <mo stretchy="false">)</mo>
     </mrow>
     <mrow>
      <mi>N</mi>
      <mo>-</mo>
      <msub>
       <mi>n</mi>
       <mi>x</mi>
      </msub>
     </mrow>
    </msup>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <eq></eq>
    <apply>
     <csymbol cd="ambiguous">superscript</csymbol>
     <apply>
      <csymbol cd="ambiguous">subscript</csymbol>
      <ci>P</ci>
      <ci>i</ci>
     </apply>
     <ci>x</ci>
    </apply>
    <apply>
     <times></times>
     <apply>
      <divide></divide>
      <apply>
       <factorial></factorial>
       <ci>N</ci>
      </apply>
      <apply>
       <times></times>
       <apply>
        <factorial></factorial>
        <apply>
         <csymbol cd="ambiguous">subscript</csymbol>
         <ci>n</ci>
         <ci>x</ci>
        </apply>
       </apply>
       <apply>
        <factorial></factorial>
        <apply>
         <minus></minus>
         <ci>N</ci>
         <apply>
          <csymbol cd="ambiguous">subscript</csymbol>
          <ci>n</ci>
          <ci>x</ci>
         </apply>
        </apply>
       </apply>
      </apply>
     </apply>
     <apply>
      <csymbol cd="ambiguous">superscript</csymbol>
      <apply>
       <csymbol cd="ambiguous">subscript</csymbol>
       <ci>p</ci>
       <ci>x</ci>
      </apply>
      <apply>
       <csymbol cd="ambiguous">subscript</csymbol>
       <ci>n</ci>
       <ci>x</ci>
      </apply>
     </apply>
     <apply>
      <csymbol cd="ambiguous">superscript</csymbol>
      <apply>
       <minus></minus>
       <cn type="integer">1</cn>
       <apply>
        <csymbol cd="ambiguous">subscript</csymbol>
        <ci>p</ci>
        <ci>x</ci>
       </apply>
      </apply>
      <apply>
       <minus></minus>
       <ci>N</ci>
       <apply>
        <csymbol cd="ambiguous">subscript</csymbol>
        <ci>n</ci>
        <ci>x</ci>
       </apply>
      </apply>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   P_{i}^{x}=\frac{N!}{n_{x}!(N-n_{x})!}p_{x}^{n_{x}}(1-p_{x})^{N-n_{x}}
  </annotation>
 </semantics>
</math>

,</p>
</div>

<p>where N is 100, n<sub>x</sub> is the percentage of sequences with residue <em>x</em> (e.g. methionine) at position <em>i</em>, and p<sub>x</sub> corresponds to the approximate distribution of amino acid <em>x</em> in all positions among all sequenced proteins. The summation runs over all 20 amino acids. After ΔG<sub>i</sub><sup>stat</sup> is computed, the conservation for position <em>i</em> in a subalignment produced after a perturbation of amino acid distribution at <em>j</em> (ΔG<sub>i | δj</sub><sup>stat</sup>) is taken. Statistical coupling energy, denoted ΔΔG<sub>i, j</sub><sup>stat</sup>, is simply the difference between these two values. That is:</p>

<p><div -="" <span="" \delta="" class="LaTeX" commonly,="" g_i^{stat},="" more="" or,="">$\Delta\Delta G_{i, j}^{stat} = \sqrt{\sum_x (\ln P_{i|\delta j}^x - \ln P_i^x)^2}$</div></p></body></html>

<p>Statistical coupling energy is often systematically calculated between a fixed, perturbated position, and all other positions in an MSA. Continuing with the example MSA from the beginning of the section, consider a perturbation at position <em>j</em> where the amino distribution changes from 40% I, 40% H, 20% M to 100% I. If, in a subsequent subalignment, this changes the distribution at <em>i</em> from 60% V, 40% L to 90% V, 10% L, but does not change the distribution at position <em>l</em>, then there would be some amount of statistical coupling energy between <em>i</em> and <em>j</em> but none between <em>l</em> and <em>j</em>.</p>
<h2 id="applications">Applications</h2>

<p>Ranganathan and Lockless originally developed SCA to examine thermodynamic (energetic) coupling of residue pairs in proteins.<a class="footnoteRef" href="#fn3" id="fnref3"><sup>3</sup></a> Using the <a href="PDZ_domain" title="wikilink">PDZ domain</a> family, they were able to identify a small network of residues that were energetically coupled to a binding site residue. The network consisted of both residues spatially close to the binding site in the tertiary fold, called contact pairs, and more distant residues that participate in longer-range energetic interactions. Later applications of SCA by the <a href="http://www.hhmi.swmed.edu/Labs/rr/world/people.html">Ranganathan group</a> on the <a class="uri" href="GPCR" title="wikilink">GPCR</a>, <a href="serine_protease" title="wikilink">serine protease</a> and <a class="uri" href="hemoglobin" title="wikilink">hemoglobin</a> families also showed energetic coupling in sparse networks of residues that cooperate in <a href="Allosteric_enzyme" title="wikilink">allosteric communication</a>.<a class="footnoteRef" href="#fn4" id="fnref4"><sup>4</sup></a></p>

<p>Statistical coupling analysis has also been used as a basis for computational protein design. In 2005, Russ et al.<a class="footnoteRef" href="#fn5" id="fnref5"><sup>5</sup></a> used an SCA for the <a href="WW_domain" title="wikilink">WW domain</a> to create artificial proteins with similar <a href="Equilibrium_unfolding#Thermal_denaturation" title="wikilink">thermodynamic stability</a> and <a href="Protein_structure" title="wikilink">structure</a> to natural WW domains. The fact that 12 out of the 43 designed proteins with the same SCA profile as natural WW domains properly folded provided strong evidence that little information—only coupling information—was required for specifying the protein fold. This support for the SCA hypothesis was made more compelling considering that a) the successfully folded proteins had only 36% average <a href="Sequence_alignment" title="wikilink">sequence identity</a> to natural WW folds, and b) none of the artificial proteins designed without coupling information folded properly. An accompanying study showed that the artificial WW domains were functionally similar to natural WW domains in <a href="Ligand_(biochemistry)#Receptor/ligand_binding_affinity" title="wikilink">ligand binding affinity and specificity</a>.<a class="footnoteRef" href="#fn6" id="fnref6"><sup>6</sup></a></p>

<p>In <a href="Protein_structure_prediction#Ab_initio_protein_modelling" title="wikilink"><em>de novo</em> protein structure prediction</a>, it has been shown that, when combined with a simple residue-residue distance metric, SCA-based scoring can fairly accurately distinguish native from non-native protein folds.<a class="footnoteRef" href="#fn7" id="fnref7"><sup>7</sup></a></p>
<h2 id="see-also">See also</h2>

<p><a href="Mutual_information" title="wikilink">Mutual information</a></p>
<h2 id="external-links">External links</h2>
<ul>
<li><a href="http://www.bork.embl-heidelberg.de/Modules/ww_summary.html">What is a WW domain?</a></li>
<li><a href="http://esmane.physics.lsa.umich.edu/wl/external/ICSB/2005/20051021-umwlap001-02-ranganathan-movies/realaudio/f001.htm">Ranganathan lecture on statistical coupling analysis (audio included)</a></li>
<li><a href="http://www.pandasthumb.org/archives/2005/10/protein-folding.html">Protein folding — a step closer?</a> - A summary of the Ranganathan lab's SCA-based design of artificial yet functional WW domains.</li>
</ul>
<h2 id="references">References</h2>

<p>"</p>

<p><a class="uri" href="Category:Bioinformatics" title="wikilink">Category:Bioinformatics</a></p>
<section class="footnotes">
<hr/>
<ol>
<li id="fn1"><a href="#fnref1">↩</a></li>
<li id="fn2"><a href="#fnref2">↩</a></li>
<li id="fn3"><a href="#fnref3">↩</a></li>
<li id="fn4"><a href="#fnref4">↩</a></li>
<li id="fn5"><a href="#fnref5">↩</a></li>
<li id="fn6"><a href="#fnref6">↩</a></li>
<li id="fn7"><a href="#fnref7">↩</a></li>
</ol>
</section>


