<html lang="en">
<head>
<meta charset="utf-8"/>
<title offset="644">Tandem mass spectrometry</title>
<script src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_SVG.js" type="text/javascript">
</script>
</head>
<body>
<h1>Tandem mass spectrometry</h1>
<hr/>

<p> <strong>Tandem mass spectrometry</strong>, also known as <strong>MS/MS</strong> or <strong>MS<sup>2</sup></strong>, involves multiple steps of <a href="mass_spectrometry" title="wikilink">mass spectrometry</a> selection, with some form of fragmentation occurring in between the stages.<a class="footnoteRef" href="#fn1" id="fnref1"><sup>1</sup></a></p>
<h2 id="instrumentation">Instrumentation</h2>

<p> For tandem mass spectrometry in space, the different elements are often noted in shorthand. Multiple stages of mass analysis separation can be accomplished with individual mass spectrometer elements separated in space or using a single mass spectrometer with the MS steps separated in time.</p>
<h3 id="tandem-in-space">Tandem in space</h3>

<p> In tandem mass spectrometry <em>in space</em>, the separation elements are physically separated and distinct, although there is a physical connection between the elements to maintain <a href="vacuum" title="wikilink">high vacuum</a>. These elements can be <a href="sector_mass_spectrometer" title="wikilink">sectors</a>, <a href="Quadrupole_mass_analyzer" title="wikilink">transmission quadrupole</a>, or <a href="Time-of-flight_mass_spectrometry" title="wikilink">time-of-flight</a>. When using multiple <a href="Quadrupole_mass_analyzer" title="wikilink">quadrupoles</a>, they can act as both <a href="mass_analyzer" title="wikilink">mass analyzers</a> and collision chambers.</p>
<dl>
<dd><strong>Q</strong> – <a href="Quadrupole_mass_analyzer" title="wikilink">Quadrupole mass analyzer</a>
</dd>
<dd><strong>q</strong> – <a href="Radio_frequency" title="wikilink">Radio frequency</a> collision quadrupole
</dd>
<dd><strong>TOF</strong> – <a href="Time-of-flight_mass_spectrometry" title="wikilink">Time-of-flight</a> mass analyzer
</dd>
<dd><strong>B</strong> – Magnetic <a href="sector_mass_spectrometer" title="wikilink">sector</a>
</dd>
<dd><strong>E</strong> – Electric sector
</dd>
</dl>

<p>The notation can be combined to indicate various hybrid instrument, for example</p>
<dl>
<dd><strong>QqQ</strong> – <a href="Triple_quadrupole_mass_spectrometer" title="wikilink">Triple quadrupole mass spectrometer</a>
</dd>
<dd><strong>QTOF</strong> – Quadrupole time-of-flight mass spectrometer (also QqTOF)
</dd>
<dd><strong>BEBE</strong> – Four-sector (reverse geometry) mass spectrometer
</dd>
</dl>
<h3 id="tandem-in-time">Tandem in time</h3>

<p> By doing tandem mass spectrometry <em>in time</em>, the separation is accomplished with ions trapped in the same place, with multiple separation steps taking place over time. A <a href="quadrupole_ion_trap" title="wikilink">quadrupole ion trap</a> or <a href="Fourier_transform_ion_cyclotron_resonance" title="wikilink">Fourier transform ion cyclotron resonance</a> (FTICR) instrument can be used for such an analysis. Trapping instruments can perform multiple steps of analysis, which is sometimes referred to as MS<sup>n</sup> (MS to the n). Often the number of steps, n, is not indicated, but occasionally the value is specified; for example MS<sup>3</sup> indicates three stages of separation. Tandem in time MS instruments do not use the modes described next, but typically collect all of the information from a precursor ion scan and a parent ion scan of the entire spectrum. Each instrumental configuration utilizes a unique mode of mass identification.</p>
<h3 id="tandem-in-space-msms-modes">Tandem in space MS/MS modes</h3>

<p>When tandem MS is performed with an in space design, the instrument must operate in one of a variety of modes. There are a number of different tandem MS/MS experimental setups and each mode has its own applications and provides different information. Tandem MS in space uses the coupling of two instrument components which measure the same mass spectrum range but with a controlled fractionation between them in space, while tandem MS in time involves the use of an <a href="ion_trap" title="wikilink">ion trap</a>.</p>

<p>There are four main scan experiments possible using MS/MS: precursor ion scan, product ion scan, neutral loss scan, and selected reaction monitoring.</p>

<p>For a precursor ion scan, the product ion is selected in the second mass analyzer, and the precursor masses are scanned in the first mass analyzer. Note that precursor ion<a class="footnoteRef" href="#fn2" id="fnref2"><sup>2</sup></a> is synonymous with parent ion<a class="footnoteRef" href="#fn3" id="fnref3"><sup>3</sup></a> and product ion<a class="footnoteRef" href="#fn4" id="fnref4"><sup>4</sup></a> with daughter ion;<a class="footnoteRef" href="#fn5" id="fnref5"><sup>5</sup></a> however the use of these anthropomorphic terms is discouraged.<a class="footnoteRef" href="#fn6" id="fnref6"><sup>6</sup></a><a class="footnoteRef" href="#fn7" id="fnref7"><sup>7</sup></a></p>

<p>In a product ion scan, a precursor ion is selected in the first stage, allowed to fragment and then all resultant masses are scanned in the second mass analyzer and detected in the detector that is positioned after the second mass analyzer. This experiment is commonly performed to identify transitions used for quantification by tandem MS.</p>

<p>In a neutral loss scan, the first mass analyzer scans all the masses. The second mass analyzer also scans, but at a set offset from the first mass analyzer.<a class="footnoteRef" href="#fn8" id="fnref8"><sup>8</sup></a> This offset corresponds to a neutral loss that is commonly observed for the class of compounds. In a constant-neutral-loss scan, all precursors that undergo the loss of a specified common neutral are monitored. To obtain this information, both mass analyzers are scanned simultaneously, but with a mass offset that correlates with the mass of the specified neutral. Similar to the precursor-ion scan, this technique is also useful in the selective identification of closely related class of compounds in a mixture.</p>

<p>In selected reaction monitoring, both mass analyzers are set to a selected mass. This mode is analogous to selected ion monitoring for MS experiments. A selective analysis mode, which can increase sensitivity.<a class="footnoteRef" href="#fn9" id="fnref9"><sup>9</sup></a></p>
<h2 id="fragmentation">Fragmentation</h2>

<p>Fragmentation of gas-phase ions is essential to tandem mass spectrometry and occurs between different stages of mass analysis. There are many methods used to fragment the ions and these can result in different types of fragmentation and thus different information about the structure and composition of the molecule.</p>
<h3 id="in-source-fragmentation">In-source fragmentation</h3>

<p>Often, the <a class="uri" href="ionization" title="wikilink">ionization</a> process is sufficiently violent to leave the resulting ions with sufficient <a href="internal_energy" title="wikilink">internal energy</a> to fragment within the mass spectrometer. If the product ions persist in their non-equilibrium state for a moderate amount of time before auto-dissociation this process is called <a class="uri" href="metastable" title="wikilink">metastable</a> fragmentation.<a class="footnoteRef" href="#fn10" id="fnref10"><sup>10</sup></a><a class="footnoteRef" href="#fn11" id="fnref11"><sup>11</sup></a> Nozzle-skimmer fragmentation refers to the purposeful induction of in-source fragmentation by increasing the nozzle-skimmer potential on usually <a href="electrospray_ionization" title="wikilink">electrospray</a> based instruments. Although in-source fragmentation allows for fragmentation analysis, it is not technically tandem mass spectrometry unless metastable ions are mass analyzed or selected before auto-dissociation and a second stage of analysis is performed on the resulting fragments. In-source fragmentation is often used in addition to tandem mass spectrometry (with post-source fragmentation) to allow for two steps of fragmentation in a pseudo MS<sup>3</sup>-type of experiment.<a class="footnoteRef" href="#fn12" id="fnref12"><sup>12</sup></a></p>
<h3 id="collision-induced-dissociation">Collision-induced dissociation</h3>

<p>Post-source fragmentation is most often what is being used in a tandem mass spectrometry experiment. Energy can also be added to the ions, which are usually already vibrationally excited, through post-source collisions with neutral atoms or molecules, the absorption of radiation, or the transfer or capture of an electron by a multiply charged ion. <a href="Collision-induced_dissociation" title="wikilink">Collision-induced dissociation</a> (CID), also called collisionally activated dissociation (CAD), involves the collision of an ion with a neutral atom or molecule in the gas phase and subsequent dissociation of the ion.<a class="footnoteRef" href="#fn13" id="fnref13"><sup>13</sup></a><a class="footnoteRef" href="#fn14" id="fnref14"><sup>14</sup></a> For example, consider</p>

<p>
<math display="block" id="Tandem_mass_spectrometry:0">
<semantics>
<mrow>
<mrow>
<mrow>
<mi>A</mi>
<msup>
<mi>B</mi>
<mo>+</mo>
</msup>
</mrow>
<mo>+</mo>
<mi>M</mi>
</mrow>
<mo>→</mo>
<mrow>
<mi>A</mi>
<mo>+</mo>
<msup>
<mi>B</mi>
<mo>+</mo>
</msup>
<mo>+</mo>
<mi>M</mi>
</mrow>
</mrow>
<annotation-xml encoding="MathML-Content">
<apply>
<ci>normal-→</ci>
<apply>
<plus></plus>
<apply>
<times></times>
<ci>A</ci>
<apply>
<csymbol cd="ambiguous">superscript</csymbol>
<ci>B</ci>
<plus></plus>
</apply>
</apply>
<ci>M</ci>
</apply>
<apply>
<plus></plus>
<ci>A</ci>
<apply>
<csymbol cd="ambiguous">superscript</csymbol>
<ci>B</ci>
<plus></plus>
</apply>
<ci>M</ci>
</apply>
</apply>
</annotation-xml>
<annotation encoding="application/x-tex">
   AB^{+}+M\to A+B^{+}+M
  </annotation>
</semantics>
</math>
</p>

<p>where the ion AB<sup>+</sup> collides with the neutral species M and subsequently breaks apart. The details of this process are described by <a href="collision_theory" title="wikilink">collision theory</a>.</p>

<p><a href="Higher-energy_collisional_dissociation" title="wikilink">Higher-energy collisional dissociation</a> (HCD) is a CID technique specific to <a class="uri" href="orbitrap" title="wikilink">orbitrap</a> mass spectrometers in which fragmentation takes place external to the trap.<a class="footnoteRef" href="#fn15" id="fnref15"><sup>15</sup></a></p>
<h3 id="electron-capture-and-transfer-methods">Electron capture and transfer methods</h3>

<p>The energy released when an electron is transferred to or captured by a multiply charged ion can induce fragmentation.</p>
<h4 id="electron-capture-dissociation">Electron capture dissociation</h4>

<p>If an <a class="uri" href="electron" title="wikilink">electron</a> is added to a multiply charged positive ion, the <a href="Coulomb's_law" title="wikilink">Coulomb energy</a> is liberated. Adding a free electron is called <a href="electron_capture_dissociation" title="wikilink">electron capture dissociation</a> (ECD),<a class="footnoteRef" href="#fn16" id="fnref16"><sup>16</sup></a> and is represented by</p>

<p>
<math display="block" id="Tandem_mass_spectrometry:1">
<semantics>
<mrow>
<mrow>
<msup>
<mrow>
<mo stretchy="false">[</mo>
<mrow>
<mi>M</mi>
<mo>+</mo>
<mrow>
<mi>n</mi>
<mi>H</mi>
</mrow>
</mrow>
<mo stretchy="false">]</mo>
</mrow>
<mrow>
<mi>n</mi>
<mo>+</mo>
</mrow>
</msup>
<mo>+</mo>
<msup>
<mi>e</mi>
<mo>-</mo>
</msup>
</mrow>
<mo>→</mo>
<msup>
<mrow>
<mo maxsize="210%" minsize="210%">[</mo>
<msup>
<mrow>
<mo stretchy="false">[</mo>
<mrow>
<mi>M</mi>
<mo>+</mo>
<mrow>
<mrow>
<mo stretchy="false">(</mo>
<mrow>
<mi>n</mi>
<mo>-</mo>
<mn>1</mn>
</mrow>
<mo stretchy="false">)</mo>
</mrow>
<mi>H</mi>
</mrow>
</mrow>
<mo stretchy="false">]</mo>
</mrow>
<mrow>
<mrow>
<mo stretchy="false">(</mo>
<mrow>
<mi>n</mi>
<mo>-</mo>
<mn>1</mn>
</mrow>
<mo stretchy="false">)</mo>
</mrow>
<mo>+</mo>
</mrow>
</msup>
<mo maxsize="210%" minsize="210%">]</mo>
</mrow>
<mo>*</mo>
</msup>
<mo>→</mo>
<mrow>
<mi>f</mi>
<mi>r</mi>
<mi>a</mi>
<mi>g</mi>
<mi>m</mi>
<mi>e</mi>
<mi>n</mi>
<mi>t</mi>
<mi>s</mi>
</mrow>
</mrow>
<annotation-xml encoding="MathML-Content">
<apply>
<and></and>
<apply>
<ci>normal-→</ci>
<apply>
<plus></plus>
<apply>
<csymbol cd="ambiguous">superscript</csymbol>
<apply>
<csymbol cd="latexml">delimited-[]</csymbol>
<apply>
<plus></plus>
<ci>M</ci>
<apply>
<times></times>
<ci>n</ci>
<ci>H</ci>
</apply>
</apply>
</apply>
<apply>
<csymbol cd="latexml">limit-from</csymbol>
<ci>n</ci>
<plus></plus>
</apply>
</apply>
<apply>
<csymbol cd="ambiguous">superscript</csymbol>
<ci>e</ci>
<minus></minus>
</apply>
</apply>
<apply>
<csymbol cd="ambiguous">superscript</csymbol>
<apply>
<csymbol cd="latexml">delimited-[]</csymbol>
<apply>
<csymbol cd="ambiguous">superscript</csymbol>
<apply>
<csymbol cd="latexml">delimited-[]</csymbol>
<apply>
<plus></plus>
<ci>M</ci>
<apply>
<times></times>
<apply>
<minus></minus>
<ci>n</ci>
<cn type="integer">1</cn>
</apply>
<ci>H</ci>
</apply>
</apply>
</apply>
<apply>
<csymbol cd="latexml">limit-from</csymbol>
<apply>
<minus></minus>
<ci>n</ci>
<cn type="integer">1</cn>
</apply>
<plus></plus>
</apply>
</apply>
</apply>
<times></times>
</apply>
</apply>
<apply>
<ci>normal-→</ci>
<share href="#.cmml">
</share>
<apply>
<times></times>
<ci>f</ci>
<ci>r</ci>
<ci>a</ci>
<ci>g</ci>
<ci>m</ci>
<ci>e</ci>
<ci>n</ci>
<ci>t</ci>
<ci>s</ci>
</apply>
</apply>
</apply>
</annotation-xml>
<annotation encoding="application/x-tex">
   [M+nH]^{n+}+e^{-}\to\bigg[[M+(n-1)H]^{(n-1)+}\bigg]^{*}\to fragments
  </annotation>
</semantics>
</math>
</p>

<p>for a multiply protonated molecule M.</p>
<h4 id="electron-transfer-dissociation">Electron transfer dissociation</h4>

<p>Adding an electron through an ion-ion reaction is called <a href="electron_transfer_dissociation" title="wikilink">electron transfer dissociation</a> (ETD).<a class="footnoteRef" href="#fn17" id="fnref17"><sup>17</sup></a><a class="footnoteRef" href="#fn18" id="fnref18"><sup>18</sup></a> Similar to electron-capture dissociation, ETD induces fragmentation of cations (e.g. <a href="peptide" title="wikilink">peptides</a> or <a href="protein" title="wikilink">proteins</a>) by transferring <a href="electron" title="wikilink">electrons</a> to them. It was invented by <a href="Donald_F._Hunt" title="wikilink">Donald F. Hunt</a>, <a href="Joshua_Coon" title="wikilink">Joshua Coon</a>, John E. P. Syka and Jarrod Marto at the <a href="University_of_Virginia" title="wikilink">University of Virginia</a>.<a class="footnoteRef" href="#fn19" id="fnref19"><sup>19</sup></a></p>

<p>ETD does not use free electrons but employs radical anions (e.g. <a class="uri" href="anthracene" title="wikilink">anthracene</a> or <a class="uri" href="azobenzene" title="wikilink">azobenzene</a>) for this purpose:</p>

<p>
<math display="block" id="Tandem_mass_spectrometry:2">
<semantics>
<mrow>
<mrow>
<msup>
<mrow>
<mo stretchy="false">[</mo>
<mrow>
<mi>M</mi>
<mo>+</mo>
<mrow>
<mi>n</mi>
<mi>H</mi>
</mrow>
</mrow>
<mo stretchy="false">]</mo>
</mrow>
<mrow>
<mi>n</mi>
<mo>+</mo>
</mrow>
</msup>
<mo>+</mo>
<msup>
<mi>A</mi>
<mo>-</mo>
</msup>
</mrow>
<mo>→</mo>
<mrow>
<msup>
<mrow>
<mo maxsize="210%" minsize="210%">[</mo>
<msup>
<mrow>
<mo stretchy="false">[</mo>
<mrow>
<mi>M</mi>
<mo>+</mo>
<mrow>
<mrow>
<mo stretchy="false">(</mo>
<mrow>
<mi>n</mi>
<mo>-</mo>
<mn>1</mn>
</mrow>
<mo stretchy="false">)</mo>
</mrow>
<mi>H</mi>
</mrow>
</mrow>
<mo stretchy="false">]</mo>
</mrow>
<mrow>
<mrow>
<mo stretchy="false">(</mo>
<mrow>
<mi>n</mi>
<mo>-</mo>
<mn>1</mn>
</mrow>
<mo stretchy="false">)</mo>
</mrow>
<mo>+</mo>
</mrow>
</msup>
<mo maxsize="210%" minsize="210%">]</mo>
</mrow>
<mo>*</mo>
</msup>
<mo>+</mo>
<mi>A</mi>
</mrow>
<mo>→</mo>
<mrow>
<mi>f</mi>
<mi>r</mi>
<mi>a</mi>
<mi>g</mi>
<mi>m</mi>
<mi>e</mi>
<mi>n</mi>
<mi>t</mi>
<mi>s</mi>
</mrow>
</mrow>
<annotation-xml encoding="MathML-Content">
<apply>
<and></and>
<apply>
<ci>normal-→</ci>
<apply>
<plus></plus>
<apply>
<csymbol cd="ambiguous">superscript</csymbol>
<apply>
<csymbol cd="latexml">delimited-[]</csymbol>
<apply>
<plus></plus>
<ci>M</ci>
<apply>
<times></times>
<ci>n</ci>
<ci>H</ci>
</apply>
</apply>
</apply>
<apply>
<csymbol cd="latexml">limit-from</csymbol>
<ci>n</ci>
<plus></plus>
</apply>
</apply>
<apply>
<csymbol cd="ambiguous">superscript</csymbol>
<ci>A</ci>
<minus></minus>
</apply>
</apply>
<apply>
<plus></plus>
<apply>
<csymbol cd="ambiguous">superscript</csymbol>
<apply>
<csymbol cd="latexml">delimited-[]</csymbol>
<apply>
<csymbol cd="ambiguous">superscript</csymbol>
<apply>
<csymbol cd="latexml">delimited-[]</csymbol>
<apply>
<plus></plus>
<ci>M</ci>
<apply>
<times></times>
<apply>
<minus></minus>
<ci>n</ci>
<cn type="integer">1</cn>
</apply>
<ci>H</ci>
</apply>
</apply>
</apply>
<apply>
<csymbol cd="latexml">limit-from</csymbol>
<apply>
<minus></minus>
<ci>n</ci>
<cn type="integer">1</cn>
</apply>
<plus></plus>
</apply>
</apply>
</apply>
<times></times>
</apply>
<ci>A</ci>
</apply>
</apply>
<apply>
<ci>normal-→</ci>
<share href="#.cmml">
</share>
<apply>
<times></times>
<ci>f</ci>
<ci>r</ci>
<ci>a</ci>
<ci>g</ci>
<ci>m</ci>
<ci>e</ci>
<ci>n</ci>
<ci>t</ci>
<ci>s</ci>
</apply>
</apply>
</apply>
</annotation-xml>
<annotation encoding="application/x-tex">
   [M+nH]^{n+}+A^{-}\to\bigg[[M+(n-1)H]^{(n-1)+}\bigg]^{*}+A\to fragments
  </annotation>
</semantics>
</math>
</p>

<p>where A is the anion.<a class="footnoteRef" href="#fn20" id="fnref20"><sup>20</sup></a></p>

<p>ETD cleaves randomly along the peptide backbone (c and z ions) while side chains and modifications such as phosphorylation are left intact. The technique only works well for higher charge state ions (z&gt;2), however relative to <a href="collision-induced_dissociation" title="wikilink">collision-induced dissociation</a> (CID), ETD is advantageous for the fragmentation of longer peptides or even entire proteins. This makes the technique important for <a href="top-down_proteomics" title="wikilink">top-down proteomics</a>. Much like ECD, ETD is effective for peptides with <a href="posttranslational_modification" title="wikilink">modifications</a> such as phosphorylation.<a class="footnoteRef" href="#fn21" id="fnref21"><sup>21</sup></a></p>

<p>Electron-transfer and higher-energy collision dissociation (EThcD) is a combination ETD and HCD where the peptide precursor is initially subjected to an ion/ion reaction with <a class="uri" href="fluoranthene" title="wikilink">fluoranthene</a> anions in a <a href="linear_ion_trap" title="wikilink">linear ion trap</a>, which generates c- and z-ions.<a class="footnoteRef" href="#fn22" id="fnref22"><sup>22</sup></a><a class="footnoteRef" href="#fn23" id="fnref23"><sup>23</sup></a> In the second step HCD all-ion fragmentation is applied to all ETD derived ions to generate b- and y- ions prior to final analysis in the orbitrap analyzer.<a class="footnoteRef" href="#fn24" id="fnref24"><sup>24</sup></a> This method employs dual fragmentation to generate ion- and thus data-rich MS/MS spectra for peptide sequencing and <a href="Post-translational_modification" title="wikilink">PTM</a> localization.<a class="footnoteRef" href="#fn25" id="fnref25"><sup>25</sup></a></p>
<h4 id="negative-electron-transfer-dissociation">Negative electron transfer dissociation</h4>

<p>Fragmentation can also occur with a deprotonated species, in which an electron is transferred from the specie to an cationic reagent in a <a href="negative_electron_transfer_dissociation" title="wikilink">negative electron transfer dissociation</a> (NETD):<a class="footnoteRef" href="#fn26" id="fnref26"><sup>26</sup></a></p>

<p>
<math display="block" id="Tandem_mass_spectrometry:3">
<semantics>
<mrow>
<mrow>
<msup>
<mrow>
<mo stretchy="false">[</mo>
<mrow>
<mi>M</mi>
<mo>-</mo>
<mrow>
<mi>n</mi>
<mi>H</mi>
</mrow>
</mrow>
<mo stretchy="false">]</mo>
</mrow>
<mrow>
<mi>n</mi>
<mo>-</mo>
</mrow>
</msup>
<mo>+</mo>
<msup>
<mi>A</mi>
<mo>+</mo>
</msup>
</mrow>
<mo>→</mo>
<mrow>
<msup>
<mrow>
<mo maxsize="210%" minsize="210%">[</mo>
<msup>
<mrow>
<mo stretchy="false">[</mo>
<mrow>
<mi>M</mi>
<mo>-</mo>
<mrow>
<mi>n</mi>
<mi>H</mi>
</mrow>
</mrow>
<mo stretchy="false">]</mo>
</mrow>
<mrow>
<mrow>
<mo stretchy="false">(</mo>
<mrow>
<mi>n</mi>
<mo>+</mo>
<mn>1</mn>
</mrow>
<mo stretchy="false">)</mo>
</mrow>
<mo>-</mo>
</mrow>
</msup>
<mo maxsize="210%" minsize="210%">]</mo>
</mrow>
<mo>*</mo>
</msup>
<mo>+</mo>
<mi>A</mi>
</mrow>
<mo>→</mo>
<mrow>
<mi>f</mi>
<mi>r</mi>
<mi>a</mi>
<mi>g</mi>
<mi>m</mi>
<mi>e</mi>
<mi>n</mi>
<mi>t</mi>
<mi>s</mi>
</mrow>
</mrow>
<annotation-xml encoding="MathML-Content">
<apply>
<and></and>
<apply>
<ci>normal-→</ci>
<apply>
<plus></plus>
<apply>
<csymbol cd="ambiguous">superscript</csymbol>
<apply>
<csymbol cd="latexml">delimited-[]</csymbol>
<apply>
<minus></minus>
<ci>M</ci>
<apply>
<times></times>
<ci>n</ci>
<ci>H</ci>
</apply>
</apply>
</apply>
<apply>
<csymbol cd="latexml">limit-from</csymbol>
<ci>n</ci>
<minus></minus>
</apply>
</apply>
<apply>
<csymbol cd="ambiguous">superscript</csymbol>
<ci>A</ci>
<plus></plus>
</apply>
</apply>
<apply>
<plus></plus>
<apply>
<csymbol cd="ambiguous">superscript</csymbol>
<apply>
<csymbol cd="latexml">delimited-[]</csymbol>
<apply>
<csymbol cd="ambiguous">superscript</csymbol>
<apply>
<csymbol cd="latexml">delimited-[]</csymbol>
<apply>
<minus></minus>
<ci>M</ci>
<apply>
<times></times>
<ci>n</ci>
<ci>H</ci>
</apply>
</apply>
</apply>
<apply>
<csymbol cd="latexml">limit-from</csymbol>
<apply>
<plus></plus>
<ci>n</ci>
<cn type="integer">1</cn>
</apply>
<minus></minus>
</apply>
</apply>
</apply>
<times></times>
</apply>
<ci>A</ci>
</apply>
</apply>
<apply>
<ci>normal-→</ci>
<share href="#.cmml">
</share>
<apply>
<times></times>
<ci>f</ci>
<ci>r</ci>
<ci>a</ci>
<ci>g</ci>
<ci>m</ci>
<ci>e</ci>
<ci>n</ci>
<ci>t</ci>
<ci>s</ci>
</apply>
</apply>
</apply>
</annotation-xml>
<annotation encoding="application/x-tex">
   [M-nH]^{n-}+A^{+}\to\bigg[[M-nH]^{(n+1)-}\bigg]^{*}+A\to fragments
  </annotation>
</semantics>
</math>
</p>

<p>Following this transfer event, the electron deficient anion undergoes internal rearrangement and <a href="fragmentation_(mass_spectrometry)" title="wikilink">fragments</a>. NETD is the ion/ion analogue of <a href="electron-detachment_dissociation" title="wikilink">electron-detachment dissociation</a> (EDD).</p>

<p>NETD is compatible with fragmenting <a class="uri" href="peptide" title="wikilink">peptide</a> and <a class="uri" href="proteins" title="wikilink">proteins</a> along the backbone at the C<sub>α</sub>-C bond. The resulting fragments are usually a<sup>•</sup>- and x-type product ions.</p>
<h4 id="electron-detachment-dissociation">Electron-detachment dissociation</h4>

<p>Electron-detachment dissociation (EDD) is a method for fragmenting anionic species in mass spectrometry.<a class="footnoteRef" href="#fn27" id="fnref27"><sup>27</sup></a> It serves as a negative counter mode to electron capture dissociation. Negatively charged ions are activated by irradiation with <a class="uri" href="electrons" title="wikilink">electrons</a> of moderate kinetic energy. The result is ejection of electrons from the parent <a href="Ionic_compound" title="wikilink">ionic</a> molecule, which causes dissociation via recombination.</p>
<h4 id="charge-transfer-dissociation">Charge transfer dissociation</h4>

<p>Reaction between positively charged peptides and cationic reagents,<a class="footnoteRef" href="#fn28" id="fnref28"><sup>28</sup></a> also known as charge transfer dissociation (CTD),<a class="footnoteRef" href="#fn29" id="fnref29"><sup>29</sup></a> has recently been demonstrated as an alternative high-energy fragmentation pathway for low-charge state (1+ or 2+) peptides. The proposed mechanism of CTD using helium cations as the reagent is:</p>

<p>
<math display="block" id="Tandem_mass_spectrometry:4">
<semantics>
<mrow>
<mrow>
<msup>
<mrow>
<mo stretchy="false">[</mo>
<mrow>
<mi>M</mi>
<mo>+</mo>
<mi>H</mi>
</mrow>
<mo stretchy="false">]</mo>
</mrow>
<mrow>
<mn>1</mn>
<mo>+</mo>
</mrow>
</msup>
<mo>+</mo>
<mrow>
<mi>H</mi>
<msup>
<mi>e</mi>
<mo>+</mo>
</msup>
</mrow>
</mrow>
<mo>→</mo>
<mrow>
<msup>
<mrow>
<mo maxsize="210%" minsize="210%">[</mo>
<msup>
<mrow>
<mo stretchy="false">[</mo>
<mrow>
<mi>M</mi>
<mo>+</mo>
<mi>H</mi>
</mrow>
<mo stretchy="false">]</mo>
</mrow>
<mrow>
<mn>2</mn>
<mo>+</mo>
</mrow>
</msup>
<mo maxsize="210%" minsize="210%">]</mo>
</mrow>
<mo>*</mo>
</msup>
<mo>+</mo>
<mrow>
<mi>H</mi>
<msup>
<mi>e</mi>
<mn>0</mn>
</msup>
</mrow>
</mrow>
<mo>→</mo>
<mrow>
<mi>f</mi>
<mi>r</mi>
<mi>a</mi>
<mi>g</mi>
<mi>m</mi>
<mi>e</mi>
<mi>n</mi>
<mi>t</mi>
<mi>s</mi>
</mrow>
</mrow>
<annotation-xml encoding="MathML-Content">
<apply>
<and></and>
<apply>
<ci>normal-→</ci>
<apply>
<plus></plus>
<apply>
<csymbol cd="ambiguous">superscript</csymbol>
<apply>
<csymbol cd="latexml">delimited-[]</csymbol>
<apply>
<plus></plus>
<ci>M</ci>
<ci>H</ci>
</apply>
</apply>
<apply>
<csymbol cd="latexml">limit-from</csymbol>
<cn type="integer">1</cn>
<plus></plus>
</apply>
</apply>
<apply>
<times></times>
<ci>H</ci>
<apply>
<csymbol cd="ambiguous">superscript</csymbol>
<ci>e</ci>
<plus></plus>
</apply>
</apply>
</apply>
<apply>
<plus></plus>
<apply>
<csymbol cd="ambiguous">superscript</csymbol>
<apply>
<csymbol cd="latexml">delimited-[]</csymbol>
<apply>
<csymbol cd="ambiguous">superscript</csymbol>
<apply>
<csymbol cd="latexml">delimited-[]</csymbol>
<apply>
<plus></plus>
<ci>M</ci>
<ci>H</ci>
</apply>
</apply>
<apply>
<csymbol cd="latexml">limit-from</csymbol>
<cn type="integer">2</cn>
<plus></plus>
</apply>
</apply>
</apply>
<times></times>
</apply>
<apply>
<times></times>
<ci>H</ci>
<apply>
<csymbol cd="ambiguous">superscript</csymbol>
<ci>e</ci>
<cn type="integer">0</cn>
</apply>
</apply>
</apply>
</apply>
<apply>
<ci>normal-→</ci>
<share href="#.cmml">
</share>
<apply>
<times></times>
<ci>f</ci>
<ci>r</ci>
<ci>a</ci>
<ci>g</ci>
<ci>m</ci>
<ci>e</ci>
<ci>n</ci>
<ci>t</ci>
<ci>s</ci>
</apply>
</apply>
</apply>
</annotation-xml>
<annotation encoding="application/x-tex">
   [M+H]^{1+}+He^{+}\to\bigg[[M+H]^{2+}\bigg]^{*}+He^{0}\to fragments
  </annotation>
</semantics>
</math>

 Initial reports are that CTD causes backbone C<sub>α</sub>-C bond cleavage of peptides and provides a<sup>•</sup>- and x-type product ions.</p>
<h3 id="photodissociation">Photodissociation</h3>

<p>The energy required for dissociation can be added by <a class="uri" href="photon" title="wikilink">photon</a> absorption, resulting in ion <a class="uri" href="photodissociation" title="wikilink">photodissociation</a> and represented by</p>

<p>
<math display="block" id="Tandem_mass_spectrometry:5">
<semantics>
<mrow>
<mrow>
<mrow>
<mi>A</mi>
<msup>
<mi>B</mi>
<mo>+</mo>
</msup>
</mrow>
<mo>+</mo>
<mrow>
<mi>h</mi>
<mi>ν</mi>
</mrow>
</mrow>
<mo>→</mo>
<mrow>
<mi>A</mi>
<mo>+</mo>
<msup>
<mi>B</mi>
<mo>+</mo>
</msup>
</mrow>
</mrow>
<annotation-xml encoding="MathML-Content">
<apply>
<ci>normal-→</ci>
<apply>
<plus></plus>
<apply>
<times></times>
<ci>A</ci>
<apply>
<csymbol cd="ambiguous">superscript</csymbol>
<ci>B</ci>
<plus></plus>
</apply>
</apply>
<apply>
<times></times>
<ci>h</ci>
<ci>ν</ci>
</apply>
</apply>
<apply>
<plus></plus>
<ci>A</ci>
<apply>
<csymbol cd="ambiguous">superscript</csymbol>
<ci>B</ci>
<plus></plus>
</apply>
</apply>
</apply>
</annotation-xml>
<annotation encoding="application/x-tex">
   AB^{+}+h\nu\to A+B^{+}
  </annotation>
</semantics>
</math>
</p>

<p>where 

<math display="inline" id="Tandem_mass_spectrometry:6">
<semantics>
<mrow>
<mi>h</mi>
<mi>ν</mi>
</mrow>
<annotation-xml encoding="MathML-Content">
<apply>
<times></times>
<ci>h</ci>
<ci>ν</ci>
</apply>
</annotation-xml>
<annotation encoding="application/x-tex">
   h\nu
  </annotation>
</semantics>
</math>

 represents the photon absorbed by the ion. Ultraviolet lasers can be used, but can lead to excessive fragmentation of biomolecules.<a class="footnoteRef" href="#fn30" id="fnref30"><sup>30</sup></a></p>
<h4 id="infrared-multiphoton-dissociation">Infrared multiphoton dissociation</h4>

<p><a class="uri" href="Infrared" title="wikilink">Infrared</a> photons will heat the ions and cause dissociation if enough of them are absorbed. This process is called <a href="infrared_multiphoton_dissociation" title="wikilink">infrared multiphoton dissociation</a> (IRMPD) and is often accomplished with a <a href="carbon_dioxide_laser" title="wikilink">carbon dioxide laser</a> and an ion trapping mass spectrometer such as a <a href="Fourier_transform_ion_cyclotron_resonance" title="wikilink">FTMS</a>.<a class="footnoteRef" href="#fn31" id="fnref31"><sup>31</sup></a></p>
<h4 id="blackbody-infrared-radiative-dissociation">Blackbody infrared radiative dissociation</h4>

<p><a href="Blackbody_radiation" title="wikilink">Blackbody radiation</a> can be used for photodissociation in a technique known as blackbody infrared radiative dissociation (BIRD).<a class="footnoteRef" href="#fn32" id="fnref32"><sup>32</sup></a> In the BIRD method, the entire mass spectrometer vacuum chamber is heated to create infrared radiation. BIRD uses the light from <a href="black_body_radiation" title="wikilink">black body radiation</a> to thermally (vibrationally) excite the ions until a bond breaks.<a class="footnoteRef" href="#fn33" id="fnref33"><sup>33</sup></a><a class="footnoteRef" href="#fn34" id="fnref34"><sup>34</sup></a> This is similar to <a href="infrared_multiphoton_dissociation" title="wikilink">infrared multiphoton dissociation</a> with the exception of the source of radiation.<a class="footnoteRef" href="#fn35" id="fnref35"><sup>35</sup></a> This technique is most often used with <a href="Fourier_transform_ion_cyclotron_resonance" title="wikilink">Fourier transform ion cyclotron resonance</a> mass spectrometers.</p>
<h3 id="surface-induced-dissociation">Surface induced dissociation</h3>

<p>With <a href="surface-induced_dissociation" title="wikilink">surface-induced dissociation</a> (SID), the fragmentation is a result of the collision of an ion with a surface under high vacuum.<a class="footnoteRef" href="#fn36" id="fnref36"><sup>36</sup></a><a class="footnoteRef" href="#fn37" id="fnref37"><sup>37</sup></a></p>
<h2 id="quantitative-proteomics">Quantitative proteomics</h2>

<p><a href="Quantitative_proteomics" title="wikilink">Quantitative proteomics</a> is used to determine the relative or absolution amount of <a href="protein" title="wikilink">proteins</a> in a sample.<a class="footnoteRef" href="#fn38" id="fnref38"><sup>38</sup></a><a class="footnoteRef" href="#fn39" id="fnref39"><sup>39</sup></a><a class="footnoteRef" href="#fn40" id="fnref40"><sup>40</sup></a> Several quantitative proteomics methods are based on tandem mass spectrometry. MS/MS has become a benchmark procedure for the structural elucidation of complex biomolecules.<a class="footnoteRef" href="#fn41" id="fnref41"><sup>41</sup></a></p>
<h3 id="isobaric-tag-for-relative-and-absolute-quantitation">Isobaric tag for relative and absolute quantitation</h3>

<p> An <a href="isobaric_tag_for_relative_and_absolute_quantitation" title="wikilink">isobaric tag for relative and absolute quantitation</a> (iTRAQ) is a reagent for tandem mass spectrometry that is used to determine the amount of proteins from different sources in a single experiment.<a class="footnoteRef" href="#fn42" id="fnref42"><sup>42</sup></a><a class="footnoteRef" href="#fn43" id="fnref43"><sup>43</sup></a><a class="footnoteRef" href="#fn44" id="fnref44"><sup>44</sup></a> It uses stable <a href="Isotopic_labeling" title="wikilink">isotope labeled</a> molecules that can be <a href="covalent_bond" title="wikilink">covalent bonded</a> to the <a class="uri" href="N-terminus" title="wikilink">N-terminus</a> and <a href="side_chain" title="wikilink">side chain</a> <a href="amine" title="wikilink">amines</a> of proteins. The iTRAQ reagents are used to label peptides from different samples that are pooled and analyzed by <a href="liquid_chromatography" title="wikilink">liquid chromatography</a> and tandem mass spectrometry. The fragmentation of the attached tag generates a low molecular mass reporter ion that can be used to relatively quantify the peptides and the proteins from which they originated.</p>
<h3 id="tandem-mass-tag">Tandem mass tag</h3>

<p>A <a href="tandem_mass_tag" title="wikilink">tandem mass tag</a> (TMT) is an isobaric mass tag chemical label used for protein quantification and identification.<a class="footnoteRef" href="#fn45" id="fnref45"><sup>45</sup></a> The tags contain four regions: mass reporter, cleavable linker, mass normalization, and protein reactive group.</p>
<h2 id="applications">Applications</h2>
<h3 id="peptides">Peptides</h3>

<p> Tandem mass spectrometry can be used for <a href="protein_sequencing" title="wikilink">protein sequencing</a>.<a class="footnoteRef" href="#fn46" id="fnref46"><sup>46</sup></a> When intact proteins are introduced to a mass analyzer, this is called "<a href="top-down_proteomics" title="wikilink">top-down proteomics</a>" and when proteins are digested into smaller <a class="uri" href="peptides" title="wikilink">peptides</a> and subsequently introduced into the mass spectrometer, this is called "<a href="bottom-up_proteomics" title="wikilink">bottom-up proteomics</a>". <a href="Shotgun_proteomics" title="wikilink">Shotgun proteomics</a> is a variant of bottom up proteomics in which proteins in a mixture are digested prior to separation and tandem mass spectrometry.</p>

<p>Tandem mass spectrometry can produce a <a href="peptide_sequence_tag" title="wikilink">peptide sequence tag</a> that can be used to identify a peptide in a protein database.<a class="footnoteRef" href="#fn47" id="fnref47"><sup>47</sup></a><a class="footnoteRef" href="#fn48" id="fnref48"><sup>48</sup></a><a class="footnoteRef" href="#fn49" id="fnref49"><sup>49</sup></a> A notation has been developed for indicating peptide fragments that arise from a tandem mass spectrum.<a class="footnoteRef" href="#fn50" id="fnref50"><sup>50</sup></a> Peptide fragment ions are indicated by a, b, or c if the charge is retained on the <a class="uri" href="N-terminus" title="wikilink">N-terminus</a> and by x, y or z if the charge is maintained on the <a class="uri" href="C-terminus" title="wikilink">C-terminus</a>. The subscript indicates the number of amino acid residues in the fragment. Superscripts are sometimes used to indicate neutral losses in addition to the backbone fragmentation, * for loss of ammonia and ° for loss of water. Although peptide backbone cleavage is the most useful for sequencing and peptide identification other fragment ions may be observed under high energy dissociation conditions. These include the side chain loss ions d, v, w and immonium ions<a class="footnoteRef" href="#fn51" id="fnref51"><sup>51</sup></a><a class="footnoteRef" href="#fn52" id="fnref52"><sup>52</sup></a> and additional sequence-specific fragment ions associated with particular amino acid residues.<a class="footnoteRef" href="#fn53" id="fnref53"><sup>53</sup></a><a class="footnoteRef" href="#fn54" id="fnref54"><sup>54</sup></a></p>
<h3 id="oligosaccharides">Oligosaccharides</h3>

<p><a href="Oligosaccharide" title="wikilink">Oligosaccharides</a> may be sequenced using tandem mass spectrometry in a similar manner to peptide sequencing.<a class="footnoteRef" href="#fn55" id="fnref55"><sup>55</sup></a> Fragmentation generally occurs on either side of the <a href="glycosidic_bond" title="wikilink">glycosidic bond</a> (b, c, y and z ions) but also under more energetic conditions through the sugar ring structure in a cross-ring cleavage (x ions). Again trailing subscripts are used to indicate position of the cleavage along the chain. For cross ring cleavage ions the nature of the cross ring cleavage is indicated by preceding superscripts.<a class="footnoteRef" href="#fn56" id="fnref56"><sup>56</sup></a><a class="footnoteRef" href="#fn57" id="fnref57"><sup>57</sup></a></p>
<h3 id="oligonucleotides">Oligonucleotides</h3>

<p>Tandem mass spectrometry has been applied to <a href="DNA_sequencing" title="wikilink">DNA</a> and <a href="RNA_sequencing" title="wikilink">RNA sequencing</a>.<a class="footnoteRef" href="#fn58" id="fnref58"><sup>58</sup></a><a class="footnoteRef" href="#fn59" id="fnref59"><sup>59</sup></a> A notation for gas-phase fragmentation of <a class="uri" href="oligonucleotide" title="wikilink">oligonucleotide</a> ions has been proposed.<a class="footnoteRef" href="#fn60" id="fnref60"><sup>60</sup></a></p>
<h3 id="newborn-screening">Newborn screening</h3>

<p>Newborn screening is the process of testing newborn babies for treatable <a href="genetic_disorder" title="wikilink">genetic</a>, <a href="endocrinology" title="wikilink">endocrinologic</a>, <a href="inborn_error_of_metabolism" title="wikilink">metabolic</a> and <a href="hematology" title="wikilink">hematologic</a> diseases.<a class="footnoteRef" href="#fn61" id="fnref61"><sup>61</sup></a><a class="footnoteRef" href="#fn62" id="fnref62"><sup>62</sup></a> The development of tandem mass spectrometry screening in the early 1990s led to a large expansion of potentially detectable <a href="congenital_metabolic_disease" title="wikilink">congenital metabolic diseases</a> that affect blood levels of organic acids.<a class="footnoteRef" href="#fn63" id="fnref63"><sup>63</sup></a></p>
<h2 id="see-also">See also</h2>
<ul>
<li><a href="Accelerator_mass_spectrometry" title="wikilink">Accelerator mass spectrometry</a></li>
<li><a href="Charge_remote_fragmentation" title="wikilink">Charge remote fragmentation</a></li>
<li><a href="Cross_section_(physics)" title="wikilink">Cross section (physics)</a></li>
<li><a href="Mass-analyzed_ion_kinetic_energy_spectrometry" title="wikilink">Mass-analyzed ion kinetic energy spectrometry</a></li>
<li><a href="Unimolecular_ion_decomposition" title="wikilink">Unimolecular ion decomposition</a></li>
</ul>
<h2 id="references">References</h2>
<h2 id="bibliography">Bibliography</h2>
<ul>
<li></li>
<li></li>
<li></li>
<li></li>
</ul>
<h2 id="external-links">External links</h2>
<ul>
<li><a href="http://www.astbury.leeds.ac.uk/facil/MStut/mstutorial.htm">An Introduction to Mass Spectrometry by Dr Alison E. Ashcroft</a></li>
</ul>

<p>"</p>

<p><a href="Category:Tandem_mass_spectrometry" title="wikilink">Category:Tandem mass spectrometry</a> <a href="Category:Mass_spectrometry" title="wikilink">Category:Mass spectrometry</a></p>
<section class="footnotes">
<hr/>
<ol>
<li id="fn1"><a href="#fnref1">↩</a></li>
<li id="fn2"><a href="#fnref2">↩</a></li>
<li id="fn3"><a href="#fnref3">↩</a></li>
<li id="fn4"><a href="#fnref4">↩</a></li>
<li id="fn5"><a href="#fnref5">↩</a></li>
<li id="fn6"><a href="#fnref6">↩</a></li>
<li id="fn7"><a href="#fnref7">↩</a></li>
<li id="fn8"><a href="#fnref8">↩</a></li>
<li id="fn9"><a href="#fnref9">↩</a></li>
<li id="fn10">IUPAC gold book definition of <em>metastable ion (in mass spectrometry)</em> <a href="http://dx.doi.org/10.1351/goldbook.M03874">1</a><a href="#fnref10">↩</a></li>
<li id="fn11"><a href="http://dx.doi.org/10.1351/goldbook.T06451">IUPAC gold book definition of <em>transient (chemical) species</em></a><a href="#fnref11">↩</a></li>
<li id="fn12"><a href="http://dx.doi.org/10.1016/1044-0305(95)00626-5">JAMS Vol. 7, Feb. 1996, pp 150-156</a><a href="#fnref12">↩</a></li>
<li id="fn13"><a href="#fnref13">↩</a></li>
<li id="fn14"><a href="#fnref14">↩</a></li>
<li id="fn15"><a href="#fnref15">↩</a></li>
<li id="fn16"><a href="#fnref16">↩</a></li>
<li id="fn17"><a href="#fnref17">↩</a></li>
<li id="fn18"><a href="#fnref18">↩</a></li>
<li id="fn19"><a href="#fnref19">↩</a></li>
<li id="fn20"><a href="#fnref20">↩</a></li>
<li id="fn21"><a href="#fnref21">↩</a></li>
<li id="fn22"><a href="#fnref22">↩</a></li>
<li id="fn23"><a href="#fnref23">↩</a></li>
<li id="fn24"><a href="#fnref24">↩</a></li>
<li id="fn25"><a href="#fnref25">↩</a></li>
<li id="fn26"><a href="#fnref26">↩</a></li>
<li id="fn27"><a href="#fnref27">↩</a></li>
<li id="fn28"><a href="#fnref28">↩</a></li>
<li id="fn29"><a href="#fnref29">↩</a></li>
<li id="fn30"><a href="#fnref30">↩</a></li>
<li id="fn31"><a href="#fnref31">↩</a></li>
<li id="fn32"><a href="#fnref32">↩</a></li>
<li id="fn33"><a href="#fnref33">↩</a></li>
<li id="fn34"><a href="#fnref34">↩</a></li>
<li id="fn35"><a href="#fnref35">↩</a></li>
<li id="fn36"><a href="#fnref36">↩</a></li>
<li id="fn37"><a href="#fnref37">↩</a></li>
<li id="fn38"><a href="#fnref38">↩</a></li>
<li id="fn39"><a href="#fnref39">↩</a></li>
<li id="fn40"><a href="#fnref40">↩</a></li>
<li id="fn41"><a href="#fnref41">↩</a></li>
<li id="fn42"><a href="#fnref42">↩</a></li>
<li id="fn43"><a href="#fnref43">↩</a></li>
<li id="fn44"><a href="#fnref44">↩</a></li>
<li id="fn45"><a href="#fnref45">↩</a></li>
<li id="fn46"><a href="#fnref46">↩</a></li>
<li id="fn47"><a href="#fnref47">↩</a></li>
<li id="fn48"><a href="#fnref48">↩</a></li>
<li id="fn49"><a href="#fnref49">↩</a></li>
<li id="fn50"><a href="#fnref50">↩</a></li>
<li id="fn51">Richard S. Johnson, Stephen A. Martin and Klaus Biemann, Collision-induced fragmentation of (M + H)+ ions of peptides. Side chain specific sequence ions, International Journal of Mass Spectrometry and Ion Processes, Volume 86, 29 December 1988, Pages 137-154. <a href="http://dx.doi.org/10.1016/0168-1176(88)80060-0">2</a><a href="#fnref51">↩</a></li>
<li id="fn52">A. M. Falick, W. M. Hines, K. F. Medzihradszky, M. A. Baldwin and B. W. Gibson, Low-mass ions produced from peptides by high-energy collision-induced dissociation in tandem mass spectrometry, Journal of the American Society for Mass Spectrometry, Volume 4, Issue 11, November 1993, Pages 882-893. <a href="http://dx.doi.org/10.1016/1044-0305(93)87006-X">3</a><a href="#fnref52">↩</a></li>
<li id="fn53">Kevin M. Downard and Klaus Biemann, Amino acid sequence prerequisites for the formation of cn ion, Journal of the American Society for Mass Spectrometry Volume 4, Issue 11, November 1993, Pages 874–881. <a href="http://dx.doi.org/10.1016/1044-0305(93)87005-W">4</a><a href="#fnref53">↩</a></li>
<li id="fn54">Kevin M. Downard and Klaus Biemann, Methionine specific sequence ions formed by the dissociation of protonated peptides at high collision energies, Journal of Mass Spectrometry, Volume 30, Issue 1, January 1995, pages 25–32. <a href="http://dx.doi.org/10.1002/jms.1190300106">5</a><a href="#fnref54">↩</a></li>
<li id="fn55"><a href="#fnref55">↩</a></li>
<li id="fn56"><a href="#fnref56">↩</a></li>
<li id="fn57"><a href="#fnref57">↩</a></li>
<li id="fn58"><a href="#fnref58">↩</a></li>
<li id="fn59"><a href="#fnref59">↩</a></li>
<li id="fn60"><a href="#fnref60">↩</a></li>
<li id="fn61"><a href="#fnref61">↩</a></li>
<li id="fn62"><a href="#fnref62">↩</a></li>
<li id="fn63"><a href="#fnref63">↩</a></li>
</ol>
</section>
</body>
</html>
