<html lang="en">
<head>
<meta charset="utf-8"/>
<title offset="774">Teiresias algorithm</title>
<script src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_SVG.js" type="text/javascript">
</script>
</head>
<body>
<h1>Teiresias algorithm</h1>
<hr/>

<p>The <strong>Teiresias algorithm</strong> is a combinatorial algorithm for the discovery of rigid patterns (motifs) in biological sequences. It is named after the Greek prophet <a class="uri" href="Teiresias" title="wikilink">Teiresias</a> and was created in 1997 by <a href="Isidore_Rigoutsos" title="wikilink">Isidore Rigoutsos</a> and <a href="Aris_Floratos" title="wikilink">Aris Floratos</a>.<a class="footnoteRef" href="#fn1" id="fnref1"><sup>1</sup></a></p>

<p>The problem of finding sequence similarities in the primary structure of related proteins or genes is one of the problems arising in the analysis of biological sequences. It can be shown that pattern discovery in its general form is <a class="uri" href="NP-hard" title="wikilink">NP-hard</a>.<a class="footnoteRef" href="#fn2" id="fnref2"><sup>2</sup></a> The Teiresias algorithm, is based on the observation that if a pattern spans many positions and appears exactly k times in the input then all fragments (sub patterns) of the pattern have to appear <u>at least</u> k times in the input. The algorithm is able to produce all patterns that have a user-defined number of copies in the given input, and manages to be very efficient by avoiding the enumeration of the entire space. Finally, the algorithm reports motifs that are maximal in both length and composition.</p>

<p>A new implementation of the Teiresias algorithm was recently made available by the . Teiresias is also accessible through an interactive web-based user interface by the same center. See external links for both.</p>
<h2 id="pattern-description">Pattern description</h2>

<p>The Teiresias algorithm uses <a href="regular_expressions" title="wikilink">regular expressions</a> to define the patterns. This allows the patterns reported to consist not only from the characters that appear in each position (literals) but from a specific group of characters (bracketed literals) or even from any character (wild card). The patterns created by the algorithm are <l> patterns that have at least k instances in the input, where L ≤ W and L, W, k positive integers. A pattern is called an <l> pattern if and only if any L consecutive literals or bracketed literals span at most W positions (i.e. there can be no more than W-L wild cards).</l></l></p>

<p>The algorithm reports only maximal patterns. Given a set of sequences S, a pattern P that appears k times in S is called maximal if and only if there exists no pattern P' which is more specific than P and also appears exactly k times in S. If there exists such a pattern P' then we say that P cannot be maximal and P is considered to be subsumed by P'. A pattern P' is said to be more specific than a pattern P if and only if P' can be obtained from P by a) dereferencing a wild card or b) instantiating a bracketed literal to a literal, or c) by appending a string of literals, bracketed literals or/and wild cards to the right of P,or d) by prepending a string of literals, bracketed literals or/and wild cards to the left of P.<a class="footnoteRef" href="#fn3" id="fnref3"><sup>3</sup></a></p>
<h2 id="algorithm-description">Algorithm description</h2>

<p>Teiresias consists of two phases. Scanning and Convolution. During the first phase the input is scanned for the patterns that satisfy the minimum requirements, the elementary patterns. The <u>elementary patterns</u> consist of exactly L literals and/or bracketed literals and includes at most W-L wild cards. During convolution, the elementary patterns are recursively combined and maximal patterns are created. The order in which the convolutions are performed is very important since it guarantees that all patterns will be generated and all maximal patterns are generated before all the patterns that are subsumed by them. The order is dictated by the following rules</p>
<ul>
<li>The priority of each pattern is defined by its contents from left to right.</li>
<li>A literal has higher priority than a bracketed literal and both have higher priority than wild cards (the more specific first).</li>
<li>Longer patterns have higher priority than shorter ones.</li>
<li>Ties are resolved alphabetically.</li>
</ul>

<p>Given the assurance that all maximal patterns will be created first, it is easy to check a newly created pattern against all maximal ones to ensure that it is subsumed in which case it is discarded. If the newly created pattern is not subsumed then it is added to the list of maximal patterns. When no more patterns can be combined to form new maximal patterns then the algorithm terminates. The length of any maximal pattern is bounded from above by the length of the longest input sequence.</p>
<h2 id="time-complexity">Time complexity</h2>

<p>The algorithm is "output-sensitive." The time complexity of the TEIRESIAS algorithm is <a class="footnoteRef" href="#fn4" id="fnref4"><sup>4</sup></a> 

<math display="inline" id="Teiresias_algorithm:0">
 <semantics>
  <mrow>
   <mi>O</mi>
   <mrow>
    <mo>(</mo>
    <mrow>
     <mrow>
      <msup>
       <mi>W</mi>
       <mi>L</mi>
      </msup>
      <mi>m</mi>
      <mrow>
       <mi>log</mi>
       <mi>m</mi>
      </mrow>
     </mrow>
     <mo>+</mo>
     <mrow>
      <mrow>
       <mo stretchy="false">(</mo>
       <mrow>
        <msup>
         <mi>C</mi>
         <mi>m</mi>
        </msup>
        <mo>+</mo>
        <msub>
         <mi>t</mi>
         <mi>H</mi>
        </msub>
       </mrow>
       <mo stretchy="false">)</mo>
      </mrow>
      <mrow>
       <msub>
        <mo largeop="true" symmetric="true">∑</mo>
        <msub>
         <mi>P</mi>
         <mrow>
          <mi>m</mi>
          <mi>a</mi>
          <mi>x</mi>
         </mrow>
        </msub>
       </msub>
       <mrow>
        <mi>r</mi>
        <mi>c</mi>
        <mrow>
         <mo stretchy="false">(</mo>
         <mi>P</mi>
         <mo stretchy="false">)</mo>
        </mrow>
       </mrow>
      </mrow>
     </mrow>
    </mrow>
    <mo>)</mo>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <times></times>
    <ci>O</ci>
    <apply>
     <plus></plus>
     <apply>
      <times></times>
      <apply>
       <csymbol cd="ambiguous">superscript</csymbol>
       <ci>W</ci>
       <ci>L</ci>
      </apply>
      <ci>m</ci>
      <apply>
       <log></log>
       <ci>m</ci>
      </apply>
     </apply>
     <apply>
      <times></times>
      <apply>
       <plus></plus>
       <apply>
        <csymbol cd="ambiguous">superscript</csymbol>
        <ci>C</ci>
        <ci>m</ci>
       </apply>
       <apply>
        <csymbol cd="ambiguous">subscript</csymbol>
        <ci>t</ci>
        <ci>H</ci>
       </apply>
      </apply>
      <apply>
       <apply>
        <csymbol cd="ambiguous">subscript</csymbol>
        <sum></sum>
        <apply>
         <csymbol cd="ambiguous">subscript</csymbol>
         <ci>P</ci>
         <apply>
          <times></times>
          <ci>m</ci>
          <ci>a</ci>
          <ci>x</ci>
         </apply>
        </apply>
       </apply>
       <apply>
        <times></times>
        <ci>r</ci>
        <ci>c</ci>
        <ci>P</ci>
       </apply>
      </apply>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   O\left(W^{L}m\log m+(C^{m}+t_{H})\sum_{P_{max}}{rc(P)}\right)
  </annotation>
 </semantics>
</math>

</p>

<p>where <em>L</em> and <em>W</em> are user-specified parameters that define the "minimum density" of a pattern (any <em>L</em> literals or brackets cannot span more than <em>W</em> positions), <em>m</em> is the number of characters the input includes, <em>C</em> ≤ 1 is the average number of patterns found in a hash entry, <em>t<sub>H</sub></em> is the time needed for locating the hash entry corresponding to any given hash value, and the summation Σ is the maximum number of patterns that will ever be placed in the stack that keeps the patterns for extension during convolution.</p>
<h2 id="external-links">External Links</h2>
<ul>
<li><a href="http://cm.jefferson.edu/teiresias-code/">A C++ based implementation of the algorithm can be found here</a>.</li>
<li><a href="https://cm.jefferson.edu/Teiresias/">The interactive web-based user interface of Teiresias can be found here</a>.</li>
</ul>
<h2 id="references">References</h2>

<p>"</p>

<p><a href="Category:Data_mining_algorithms" title="wikilink">Category:Data mining algorithms</a> <a href="Category:Pattern_matching" title="wikilink">Category:Pattern matching</a></p>
<section class="footnotes">
<hr/>
<ol>
<li id="fn1"><a href="http://bioinformatics.oxfordjournals.org/content/14/1/55.short">Rigoutsos, I, Floratos, A (1998) Combinatorial pattern discovery in biological sequences: The TEIRESIAS algorithm. Bioinformatics 14: 55-67</a><a href="#fnref1">↩</a></li>
<li id="fn2">Maier, D., "The Complexity of Some Problems on Subsequences and Supersequences", Journal of the ACM, 322-336, 1978<a href="#fnref2">↩</a></li>
<li id="fn3">Floratos A., and Rigoutsos, I., "On the time complexity of the Teiresias algorithm", IBM technical report RC 21161 (94582), IBM TJ Watson Research Center, 1998<a href="#fnref3">↩</a></li>
<li id="fn4"></li>
</ol>
</section>
</body>
</html>
