<html lang="en">
<head>
<meta charset="utf-8"/>
<title offset="960">Tree alignment</title>
<script src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_SVG.js" type="text/javascript">
</script>
</head>
<body>
<h1>Tree alignment</h1>
<hr/>

<p>In computational phylogenetics, <strong>tree alignment</strong> is the problem of producing a <a href="multiple_sequence_alignment" title="wikilink">multiple sequence alignment</a>,which can be used to analyse a set of sequences with evolutionary relationship using a fixed tree. Essentially,<strong>tree alignment</strong> is an algorithm for optimizing <a href="phylogenetic_tree" title="wikilink">phylogenetic tree</a> by calculating the <a href="edit_distance" title="wikilink">edit distance</a> to achieve the minimum value.To be specific, <a href="phylogenetic_tree" title="wikilink">phylogenetic tree</a> shows an evolutionary relationship between different species and taxa joined together are assumed to have the same ancestor.</p>

<p>Formally, <strong>tree alignment</strong> is the following optimization problem.</p>

<p><strong>Input</strong>: A set 

<math display="inline" id="Tree_alignment:0">
 <semantics>
  <mi>S</mi>
  <annotation-xml encoding="MathML-Content">
   <ci>S</ci>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   S
  </annotation>
 </semantics>
</math>

 of sequences, a <a href="phylogenetic_tree" title="wikilink">phylogenetic tree</a> 

<math display="inline" id="Tree_alignment:1">
 <semantics>
  <mi>T</mi>
  <annotation-xml encoding="MathML-Content">
   <ci>T</ci>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   T
  </annotation>
 </semantics>
</math>

 leaf-labeled by 

<math display="inline" id="Tree_alignment:2">
 <semantics>
  <mi>S</mi>
  <annotation-xml encoding="MathML-Content">
   <ci>S</ci>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   S
  </annotation>
 </semantics>
</math>

 and an <a href="edit_distance" title="wikilink">edit distance</a> function 

<math display="inline" id="Tree_alignment:3">
 <semantics>
  <mi>d</mi>
  <annotation-xml encoding="MathML-Content">
   <ci>d</ci>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   d
  </annotation>
 </semantics>
</math>

 between sequences,</p>

<p><strong>Output</strong>: A labeling of the internal vertices of 

<math display="inline" id="Tree_alignment:4">
 <semantics>
  <mi>T</mi>
  <annotation-xml encoding="MathML-Content">
   <ci>T</ci>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   T
  </annotation>
 </semantics>
</math>

 such that 

<math display="inline" id="Tree_alignment:5">
 <semantics>
  <mrow>
   <msub>
    <mi mathvariant="normal">Σ</mi>
    <mrow>
     <mi>e</mi>
     <mo>∈</mo>
     <mi>T</mi>
    </mrow>
   </msub>
   <mi>d</mi>
   <mrow>
    <mo stretchy="false">(</mo>
    <mi>e</mi>
    <mo stretchy="false">)</mo>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <times></times>
    <apply>
     <csymbol cd="ambiguous">subscript</csymbol>
     <ci>normal-Σ</ci>
     <apply>
      <in></in>
      <ci>e</ci>
      <ci>T</ci>
     </apply>
    </apply>
    <ci>d</ci>
    <ci>e</ci>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \Sigma_{e\in T}d(e)
  </annotation>
 </semantics>
</math>

 is minimized, where 

<math display="inline" id="Tree_alignment:6">
 <semantics>
  <mrow>
   <mi>d</mi>
   <mrow>
    <mo stretchy="false">(</mo>
    <mi>e</mi>
    <mo stretchy="false">)</mo>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <times></times>
    <ci>d</ci>
    <ci>e</ci>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   d(e)
  </annotation>
 </semantics>
</math>

 is the <a href="edit_distance" title="wikilink">edit distance</a> between the endpoints of 

<math display="inline" id="Tree_alignment:7">
 <semantics>
  <mi>e</mi>
  <annotation-xml encoding="MathML-Content">
   <ci>e</ci>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   e
  </annotation>
 </semantics>
</math>

.</p>

<p>The task is <a class="uri" href="NP-hard" title="wikilink">NP-hard</a><a class="footnoteRef" href="#fn1" id="fnref1"><sup>1</sup></a></p>
<h2 id="background">Background</h2>
<h3 id="sequence-alignment">Sequence alignment</h3>

<p><embed src="Sequence Alignment Otakuftf&amp;Kingftf.png;" title="fig:This is a simple Sequence Alignment of Insulin gene between rat, human and chicken. The labeled nucleotides are the different nucleotides with rat Ⅰ and --- means the missing nucleotides"></embed> In <a class="uri" href="bioinformatics" title="wikilink">bioinformatics</a>, the basic method of information processing is to contrast the sequence data. It has a very important significance when <a href="biologist" title="wikilink">biologists</a> use it to discover the function, structure and evolutionary information in biological sequences. From the <a href="sequence_assembly" title="wikilink">sequence assembly</a>, the <a class="uri" href="phylogenetic" title="wikilink">phylogenetic</a> analysis, the <a class="uri" href="haplotype" title="wikilink">haplotype</a> comparison, and the prediction of <a class="uri" href="RNA" title="wikilink">RNA</a> structure are all based on sequence alignment, so the efficiency of sequence alignment, especially multiple sequence alignment, will directly affect the effect of these problems’ solution. Therefore, to design a rational and efficient sequence alignment algorithm becomes a very important branch of research in the field of bioinformatics.</p>

<p>Generally, sequence alignment means constructing a string from two or more given strings with the greatest similarity by adding, deleting letters, or adding a space for each <a href="Connection_string" title="wikilink">string</a>. The multiple sequence alignment problem is generally based on pairwise sequence alignment and currently, for pairwise sequence alignment problem, biologists can use <a href="dynamic_programming" title="wikilink">dynamic programming</a> approach to obtain its optimal solution. However, the multiple sequence alignment problem is still one of the intractable problems in bioinformatics, because finding the optimal solution of multiple sequence alignment has been proved as a <a class="uri" href="NP-complete" title="wikilink">NP-complete</a> problem so that only approximate optimal solution can be obtained.<a class="footnoteRef" href="#fn2" id="fnref2"><sup>2</sup></a></p>
<h3 id="edit-distance">Edit distance</h3>

<p><a href="Edit_distance" title="wikilink">Edit distance</a> measures the minimum operation number of character <a href="insertion_(genetics)" title="wikilink">insertions</a>, <a href="deletion_(genetics)" title="wikilink">deletions</a> and <a href="Base-pair_substitution" title="wikilink">substitutions</a> that are required to transform one sequence u to the other sequence v when being operated on a pair of strings. The calculation of edit distance can be based on <a href="dynamic_programming" title="wikilink">dynamic programming</a>,and the equation is in O(|u|∗|v|) time, where |u| and |v| are the lengths of u and v<a class="footnoteRef" href="#fn3" id="fnref3"><sup>3</sup></a> <a href="Edit_distance" title="wikilink">Edit distance</a> are the basic principle in computational biology, thus an efficient estimation of edit distance is very essential .<a class="footnoteRef" href="#fn4" id="fnref4"><sup>4</sup></a> There are some functions to calculate edit distance, including “symmetrization” used for functions of hereditary properties. Because there are a series of functions being used to calculate edit distance, different functions may result in distinct results. Finding an optimal edit distance function seems essential for further explanation.</p>
<h2 id="the-problem-of-tree-alignment">The problem of Tree alignment</h2>

<p> Tree alignment problem is a <a class="uri" href="NP-hard" title="wikilink">NP-hard</a> problem when we restrict its scoring mode and alphabet size, and it can be found an algorithm, which uses to find the optimized solution. However, there is an exponential relationship between its efficiency and the number of sequence, it means when the number of sequence is very large, the runtime before getting results is an enormous figure and it is unacceptable. Using star alignment is faster than tree alignment to get the approximate optimized solution. However, whatever the degree of multiple-sequence similarity is, the time complexity of star alignment has a proportional relationship with the square of sequence number and the square of the sequence average length. In usual, the sequence in MSA is so long that it is also inefficient or even unacceptable. Therefore, how to reduce the time complexity to linear is one of the core issues in the Tree alignment.</p>
<h2 id="combinatorial-optimization-strategy">Combinatorial Optimization Strategy</h2>

<p><a href="Combinatorial_optimization" title="wikilink">Combinatorial optimization</a> is a good strategy to solve MSA problem. The idea of combinatorial optimization strategy is to transform the multiple sequence alignment into pair sequence alignment to solve this problem. Depending on its transformation strategy, the combinatorial optimization strategy can be divided into the tree alignment algorithm and the star alignment algorithm. For a given multi sequences set 

<math display="inline" id="Tree_alignment:8">
 <semantics>
  <mi>S</mi>
  <annotation-xml encoding="MathML-Content">
   <ci>S</ci>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   S
  </annotation>
 </semantics>
</math>

 ={

<math display="inline" id="Tree_alignment:9">
 <semantics>
  <msub>
   <mi>s</mi>
   <mn>1</mn>
  </msub>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <csymbol cd="ambiguous">subscript</csymbol>
    <ci>s</ci>
    <cn type="integer">1</cn>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   s_{1}
  </annotation>
 </semantics>
</math>

,…,

<math display="inline" id="Tree_alignment:10">
 <semantics>
  <msub>
   <mi>s</mi>
   <mi>n</mi>
  </msub>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <csymbol cd="ambiguous">subscript</csymbol>
    <ci>s</ci>
    <ci>n</ci>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   s_{n}
  </annotation>
 </semantics>
</math>

 }, finding an <a href="evolutionary_tree" title="wikilink">evolutionary tree</a> which has n leaf nodes and establishing one to one relationship between this evolutionary tree and the set S. By assigning the sequence to the internal nodes of the evolutionary tree, we calculate the total score of each edge and the sum of all edges’ score is the score of the evolutionary tree. The aim of tree alignment is to find an assigned sequence, which can obtain a maximum score, and get the final matching result by the evolutionary tree and its nodes’ assigned sequence. Star alignment can be seen as a special case of the tree alignment. When we use star alignment, the evolutionary tree has only one internal node and n leaf nodes. The sequence, which is assigned to the internal node, is called core sequence.<a class="footnoteRef" href="#fn5" id="fnref5"><sup>5</sup></a></p>
<h3 id="the-keyword-tree-theory-and-aho-corasick-search-algorithm">The Keyword Tree Theory and Aho-Corasick search algorithm</h3>

<p>When we use combinatorial optimization strategy to transform the multiple sequence alignment into pair sequence alignment, the main problem is changed from how to improve the efficiency of multiple sequence alignment to how to improve the efficiency of pairwise sequence alignment. The Keyword Tree Theory and Aho-Corasick search algorithm is an efficient approach to solve the pairwise sequence alignment problem. The aim of combining the keyword tree theory and Aho-Corasick search algorithm is to solve this kind of problem: for a given long string T and a short strings set 

<math display="inline" id="Tree_alignment:11">
 <semantics>
  <mi>P</mi>
  <annotation-xml encoding="MathML-Content">
   <ci>P</ci>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   P
  </annotation>
 </semantics>
</math>

={

<math display="inline" id="Tree_alignment:12">
 <semantics>
  <msub>
   <mi>p</mi>
   <mn>1</mn>
  </msub>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <csymbol cd="ambiguous">subscript</csymbol>
    <ci>p</ci>
    <cn type="integer">1</cn>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   p_{1}
  </annotation>
 </semantics>
</math>

,

<math display="inline" id="Tree_alignment:13">
 <semantics>
  <msub>
   <mi>p</mi>
   <mn>2</mn>
  </msub>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <csymbol cd="ambiguous">subscript</csymbol>
    <ci>p</ci>
    <cn type="integer">2</cn>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   p_{2}
  </annotation>
 </semantics>
</math>

,… ,

<math display="inline" id="Tree_alignment:14">
 <semantics>
  <msub>
   <mi>p</mi>
   <mi>z</mi>
  </msub>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <csymbol cd="ambiguous">subscript</csymbol>
    <ci>p</ci>
    <ci>z</ci>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   p_{z}
  </annotation>
 </semantics>
</math>

} (z∈N,z&gt;1), find the location of all 

<math display="inline" id="Tree_alignment:15">
 <semantics>
  <msub>
   <mi>P</mi>
   <mi>i</mi>
  </msub>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <csymbol cd="ambiguous">subscript</csymbol>
    <ci>P</ci>
    <ci>i</ci>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   P_{i}
  </annotation>
 </semantics>
</math>

 in the T. We use keyword tree produced by set 

<math display="inline" id="Tree_alignment:16">
 <semantics>
  <mi>P</mi>
  <annotation-xml encoding="MathML-Content">
   <ci>P</ci>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   P
  </annotation>
 </semantics>
</math>

, and then search in the T with this keyword tree by Aho-Corasick search algorithm.<a class="footnoteRef" href="#fn6" id="fnref6"><sup>6</sup></a> The total time complexity of using this method to find all 

<math display="inline" id="Tree_alignment:17">
 <semantics>
  <msub>
   <mi>P</mi>
   <mi>i</mi>
  </msub>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <csymbol cd="ambiguous">subscript</csymbol>
    <ci>P</ci>
    <ci>i</ci>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   P_{i}
  </annotation>
 </semantics>
</math>

’s location in the T is O(m+n+k), where m=|T| (the length of T), n=∑|

<math display="inline" id="Tree_alignment:18">
 <semantics>
  <msub>
   <mi>P</mi>
   <mi>i</mi>
  </msub>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <csymbol cd="ambiguous">subscript</csymbol>
    <ci>P</ci>
    <ci>i</ci>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   P_{i}
  </annotation>
 </semantics>
</math>

| (the sum of all 

<math display="inline" id="Tree_alignment:19">
 <semantics>
  <msub>
   <mi>P</mi>
   <mi>i</mi>
  </msub>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <csymbol cd="ambiguous">subscript</csymbol>
    <ci>P</ci>
    <ci>i</ci>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   P_{i}
  </annotation>
 </semantics>
</math>

’s length) and k means the sum of occurrence for all 

<math display="inline" id="Tree_alignment:20">
 <semantics>
  <msub>
   <mi>P</mi>
   <mi>i</mi>
  </msub>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <csymbol cd="ambiguous">subscript</csymbol>
    <ci>P</ci>
    <ci>i</ci>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   P_{i}
  </annotation>
 </semantics>
</math>

 in the T.</p>
<h4 id="keyword-tree-theory">Keyword Tree Theory</h4>

<p>The keyword tree of the set 

<math display="inline" id="Tree_alignment:21">
 <semantics>
  <mi>P</mi>
  <annotation-xml encoding="MathML-Content">
   <ci>P</ci>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   P
  </annotation>
 </semantics>
</math>

={

<math display="inline" id="Tree_alignment:22">
 <semantics>
  <msub>
   <mi>p</mi>
   <mn>1</mn>
  </msub>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <csymbol cd="ambiguous">subscript</csymbol>
    <ci>p</ci>
    <cn type="integer">1</cn>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   p_{1}
  </annotation>
 </semantics>
</math>

,

<math display="inline" id="Tree_alignment:23">
 <semantics>
  <msub>
   <mi>p</mi>
   <mn>2</mn>
  </msub>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <csymbol cd="ambiguous">subscript</csymbol>
    <ci>p</ci>
    <cn type="integer">2</cn>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   p_{2}
  </annotation>
 </semantics>
</math>

,… ,

<math display="inline" id="Tree_alignment:24">
 <semantics>
  <msub>
   <mi>p</mi>
   <mi>z</mi>
  </msub>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <csymbol cd="ambiguous">subscript</csymbol>
    <ci>p</ci>
    <ci>z</ci>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   p_{z}
  </annotation>
 </semantics>
</math>

 } (z∈N,z&gt;1) is a rooted tree, whose root denoted by K, and this keyword tree satisfies: (1): Each edge clearly demarcates one letter. (2): Any two edges separated from the same node are to correspond to different letters. (3) Each pattern 

<math display="inline" id="Tree_alignment:25">
 <semantics>
  <msub>
   <mi>P</mi>
   <mi>i</mi>
  </msub>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <csymbol cd="ambiguous">subscript</csymbol>
    <ci>P</ci>
    <ci>i</ci>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   P_{i}
  </annotation>
 </semantics>
</math>

 (i=1,2,…,z) corresponds to a node 

<math display="inline" id="Tree_alignment:26">
 <semantics>
  <mi>v</mi>
  <annotation-xml encoding="MathML-Content">
   <ci>v</ci>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   v
  </annotation>
 </semantics>
</math>

, and the path from the root K to the node 

<math display="inline" id="Tree_alignment:27">
 <semantics>
  <mi>v</mi>
  <annotation-xml encoding="MathML-Content">
   <ci>v</ci>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   v
  </annotation>
 </semantics>
</math>

 can exactly correctly spell the string 

<math display="inline" id="Tree_alignment:28">
 <semantics>
  <msub>
   <mi>P</mi>
   <mi>i</mi>
  </msub>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <csymbol cd="ambiguous">subscript</csymbol>
    <ci>P</ci>
    <ci>i</ci>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   P_{i}
  </annotation>
 </semantics>
</math>

. For each leaf node of this K tree, it corresponds to one of certain patterns of set 

<math display="inline" id="Tree_alignment:29">
 <semantics>
  <mi>P</mi>
  <annotation-xml encoding="MathML-Content">
   <ci>P</ci>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   P
  </annotation>
 </semantics>
</math>

.</p>

<p>And We use 

<math display="inline" id="Tree_alignment:30">
 <semantics>
  <mrow>
   <mi>L</mi>
   <mrow>
    <mo stretchy="false">(</mo>
    <mi>v</mi>
    <mo stretchy="false">)</mo>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <times></times>
    <ci>L</ci>
    <ci>v</ci>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   L(v)
  </annotation>
 </semantics>
</math>

 to represent the STRING which is connected from the root node to the node 

<math display="inline" id="Tree_alignment:31">
 <semantics>
  <mi>v</mi>
  <annotation-xml encoding="MathML-Content">
   <ci>v</ci>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   v
  </annotation>
 </semantics>
</math>

. We also use 

<math display="inline" id="Tree_alignment:32">
 <semantics>
  <mrow>
   <mi>L</mi>
   <mi>p</mi>
   <mrow>
    <mo stretchy="false">(</mo>
    <mi>v</mi>
    <mo stretchy="false">)</mo>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <times></times>
    <ci>L</ci>
    <ci>p</ci>
    <ci>v</ci>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   Lp(v)
  </annotation>
 </semantics>
</math>

 to represent the length of the longest suffix(also, this suffix is the prefix of one of patterns in the set 

<math display="inline" id="Tree_alignment:33">
 <semantics>
  <mi>P</mi>
  <annotation-xml encoding="MathML-Content">
   <ci>P</ci>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   P
  </annotation>
 </semantics>
</math>

). Searching this prefix from the root node in the keyword tree, and the last node denoted by 

<math display="inline" id="Tree_alignment:34">
 <semantics>
  <msub>
   <mi>n</mi>
   <mi>v</mi>
  </msub>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <csymbol cd="ambiguous">subscript</csymbol>
    <ci>n</ci>
    <ci>v</ci>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   n_{v}
  </annotation>
 </semantics>
</math>

 when the search is over. When 

<math display="inline" id="Tree_alignment:35">
 <semantics>
  <mrow>
   <mi>L</mi>
   <mi>p</mi>
   <mrow>
    <mo stretchy="false">(</mo>
    <mi>v</mi>
    <mo stretchy="false">)</mo>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <times></times>
    <ci>L</ci>
    <ci>p</ci>
    <ci>v</ci>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   Lp(v)
  </annotation>
 </semantics>
</math>

=0, 

<math display="inline" id="Tree_alignment:36">
 <semantics>
  <msub>
   <mi>n</mi>
   <mi>v</mi>
  </msub>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <csymbol cd="ambiguous">subscript</csymbol>
    <ci>n</ci>
    <ci>v</ci>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   n_{v}
  </annotation>
 </semantics>
</math>

 =K. The ordered pair (

<math display="inline" id="Tree_alignment:37">
 <semantics>
  <mi>v</mi>
  <annotation-xml encoding="MathML-Content">
   <ci>v</ci>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   v
  </annotation>
 </semantics>
</math>

, 

<math display="inline" id="Tree_alignment:38">
 <semantics>
  <msub>
   <mi>n</mi>
   <mi>v</mi>
  </msub>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <csymbol cd="ambiguous">subscript</csymbol>
    <ci>n</ci>
    <ci>v</ci>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   n_{v}
  </annotation>
 </semantics>
</math>

) called a failure link.<a class="footnoteRef" href="#fn7" id="fnref7"><sup>7</sup></a></p>

<p>For example, The set 

<math display="inline" id="Tree_alignment:39">
 <semantics>
  <mi>P</mi>
  <annotation-xml encoding="MathML-Content">
   <ci>P</ci>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   P
  </annotation>
 </semantics>
</math>

={potato, tattoo, theater, other}, and the keyword tree is shown on the right. Obviously, in that example if 

<math display="inline" id="Tree_alignment:40">
 <semantics>
  <mrow>
   <mi>L</mi>
   <mrow>
    <mo stretchy="false">(</mo>
    <mi>v</mi>
    <mo stretchy="false">)</mo>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <times></times>
    <ci>L</ci>
    <ci>v</ci>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   L(v)
  </annotation>
 </semantics>
</math>

=potat, then 

<math display="inline" id="Tree_alignment:41">
 <semantics>
  <mrow>
   <mi>L</mi>
   <mi>p</mi>
   <mrow>
    <mo stretchy="false">(</mo>
    <mi>v</mi>
    <mo stretchy="false">)</mo>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <times></times>
    <ci>L</ci>
    <ci>p</ci>
    <ci>v</ci>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   Lp(v)
  </annotation>
 </semantics>
</math>

=|tat|=3, and the failure link of the node 

<math display="inline" id="Tree_alignment:42">
 <semantics>
  <mi>v</mi>
  <annotation-xml encoding="MathML-Content">
   <ci>v</ci>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   v
  </annotation>
 </semantics>
</math>

 is shown in that figure.</p>

<p>To establish failure link is the key to improve the time complexity of Aho-Corasick algorithm. It can reduced the original polynomial time to the linear time for searching. Therefore, the core of keyword tree theory is to find all failure links(also means find all 

<math display="inline" id="Tree_alignment:43">
 <semantics>
  <msub>
   <mi>n</mi>
   <mi>v</mi>
  </msub>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <csymbol cd="ambiguous">subscript</csymbol>
    <ci>n</ci>
    <ci>v</ci>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   n_{v}
  </annotation>
 </semantics>
</math>

) of a keyword tree in the linear time. We assume that we find every 

<math display="inline" id="Tree_alignment:44">
 <semantics>
  <msub>
   <mi>n</mi>
   <mi>v</mi>
  </msub>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <csymbol cd="ambiguous">subscript</csymbol>
    <ci>n</ci>
    <ci>v</ci>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   n_{v}
  </annotation>
 </semantics>
</math>

 of all nodes 

<math display="inline" id="Tree_alignment:45">
 <semantics>
  <mi>v</mi>
  <annotation-xml encoding="MathML-Content">
   <ci>v</ci>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   v
  </annotation>
 </semantics>
</math>

 whose distance from the root node is less than or equal k, and now we are seeking the 

<math display="inline" id="Tree_alignment:46">
 <semantics>
  <msub>
   <mi>n</mi>
   <mi>v</mi>
  </msub>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <csymbol cd="ambiguous">subscript</csymbol>
    <ci>n</ci>
    <ci>v</ci>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   n_{v}
  </annotation>
 </semantics>
</math>

 of the node 

<math display="inline" id="Tree_alignment:47">
 <semantics>
  <mi>v</mi>
  <annotation-xml encoding="MathML-Content">
   <ci>v</ci>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   v
  </annotation>
 </semantics>
</math>

 whose distance from the root node is k +1. Its parent node is 

<math display="inline" id="Tree_alignment:48">
 <semantics>
  <msup>
   <mi>v</mi>
   <mo>′</mo>
  </msup>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <csymbol cd="ambiguous">superscript</csymbol>
    <ci>v</ci>
    <ci>normal-′</ci>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   v^{\prime}
  </annotation>
 </semantics>
</math>

, and the letter represented by the node 

<math display="inline" id="Tree_alignment:49">
 <semantics>
  <mi>v</mi>
  <annotation-xml encoding="MathML-Content">
   <ci>v</ci>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   v
  </annotation>
 </semantics>
</math>

 and 

<math display="inline" id="Tree_alignment:50">
 <semantics>
  <msup>
   <mi>v</mi>
   <mo>′</mo>
  </msup>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <csymbol cd="ambiguous">superscript</csymbol>
    <ci>v</ci>
    <ci>normal-′</ci>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   v^{\prime}
  </annotation>
 </semantics>
</math>

, is x. (1): If the next letter of the node 

<math display="inline" id="Tree_alignment:51">
 <semantics>
  <msubsup>
   <mi>n</mi>
   <mi>v</mi>
   <mo>′</mo>
  </msubsup>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <csymbol cd="ambiguous">superscript</csymbol>
    <apply>
     <csymbol cd="ambiguous">subscript</csymbol>
     <ci>n</ci>
     <ci>v</ci>
    </apply>
    <ci>normal-′</ci>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   n_{v}^{\prime}
  </annotation>
 </semantics>
</math>

 is x, we set the other node of this edge as 

<math display="inline" id="Tree_alignment:52">
 <semantics>
  <mi>w</mi>
  <annotation-xml encoding="MathML-Content">
   <ci>w</ci>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   w
  </annotation>
 </semantics>
</math>

, and 

<math display="inline" id="Tree_alignment:53">
 <semantics>
  <msub>
   <mi>n</mi>
   <mi>v</mi>
  </msub>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <csymbol cd="ambiguous">subscript</csymbol>
    <ci>n</ci>
    <ci>v</ci>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   n_{v}
  </annotation>
 </semantics>
</math>

=

<math display="inline" id="Tree_alignment:54">
 <semantics>
  <mi>w</mi>
  <annotation-xml encoding="MathML-Content">
   <ci>w</ci>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   w
  </annotation>
 </semantics>
</math>

. (2): If all letters is not x by searching all edges between 

<math display="inline" id="Tree_alignment:55">
 <semantics>
  <msubsup>
   <mi>n</mi>
   <mi>v</mi>
   <mo>′</mo>
  </msubsup>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <csymbol cd="ambiguous">superscript</csymbol>
    <apply>
     <csymbol cd="ambiguous">subscript</csymbol>
     <ci>n</ci>
     <ci>v</ci>
    </apply>
    <ci>normal-′</ci>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   n_{v}^{\prime}
  </annotation>
 </semantics>
</math>

 and its child nodes, 

<math display="inline" id="Tree_alignment:56">
 <semantics>
  <mrow>
   <mi>L</mi>
   <mrow>
    <mo stretchy="false">(</mo>
    <msub>
     <mi>n</mi>
     <mi>v</mi>
    </msub>
    <mo stretchy="false">)</mo>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <times></times>
    <ci>L</ci>
    <apply>
     <csymbol cd="ambiguous">subscript</csymbol>
     <ci>n</ci>
     <ci>v</ci>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   L(n_{v})
  </annotation>
 </semantics>
</math>

 is a suffix of 

<math display="inline" id="Tree_alignment:57">
 <semantics>
  <mrow>
   <mi>L</mi>
   <mrow>
    <mo stretchy="false">(</mo>
    <msubsup>
     <mi>n</mi>
     <mi>v</mi>
     <mo>′</mo>
    </msubsup>
    <mo stretchy="false">)</mo>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <times></times>
    <ci>L</ci>
    <apply>
     <csymbol cd="ambiguous">superscript</csymbol>
     <apply>
      <csymbol cd="ambiguous">subscript</csymbol>
      <ci>n</ci>
      <ci>v</ci>
     </apply>
     <ci>normal-′</ci>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   L(n_{v}^{\prime})
  </annotation>
 </semantics>
</math>

 plus x. Because this suffix matches the STRING begin with the root node (similar to prefix), we can detect if there is x after 

<math display="inline" id="Tree_alignment:58">
 <semantics>
  <msubsup>
   <mi>n</mi>
   <mi>v</mi>
   <mo>′</mo>
  </msubsup>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <csymbol cd="ambiguous">superscript</csymbol>
    <apply>
     <csymbol cd="ambiguous">subscript</csymbol>
     <ci>n</ci>
     <ci>v</ci>
    </apply>
    <ci>normal-′</ci>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   n_{v}^{\prime}
  </annotation>
 </semantics>
</math>

 or not. And if not, continue this process until find x or find the root node.</p>
<h4 id="aho-corasick-search-algorithm">Aho-Corasick search algorithm</h4>

<p>After establishing all failure links in the keyword tree, we use Aho-Corasick search algorithm to find the locations of all 

<math display="inline" id="Tree_alignment:59">
 <semantics>
  <msub>
   <mi>P</mi>
   <mi>i</mi>
  </msub>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <csymbol cd="ambiguous">subscript</csymbol>
    <ci>P</ci>
    <ci>i</ci>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   P_{i}
  </annotation>
 </semantics>
</math>

 (i=1,2,…,z) in the linear time. In this step, the time complexity is O(m+k).</p>
<h2 id="other-strategies">Other strategies</h2>

<p>In MSA ,DNA,RNA, and proteins sequences are usually generated and they are assumed to have evolutionary relationship .By comparing generated maps of RNA,DNA and sequences from evolutionary family ,people can assess conservation of protein ,find functional gene domains by comparing differences between evolutionary sequences. Generally ,heuristic algorithm and tree alignment graph are also adopted to solve multiple sequence alignment problems.</p>
<h3 id="heuristic-algorithm">Heuristic Algorithm</h3>

<p>Generally <a href="heuristic_algorithm" title="wikilink">heuristic algorithm</a> relies on the <a class="uri" href="iterative" title="wikilink">iterative</a> strategy, scilicet based on a comparison method, optimizing the results of multiple sequence alignment by the iterative process. Davie M proposed using <a href="particle_swarm_optimization" title="wikilink">particle swarm optimization</a> algorithm to solve the multiple sequence alignment problem; Ikeda T proposed a heuristic algorithm which is based on <a href="A*_search_algorithm" title="wikilink">A* search algorithm</a>; Bimey E first proposed using <a href="hidden_Markov_model" title="wikilink">hidden Markov model</a> to solve the multiple sequence alignment problem; and many other biologists use <a href="genetic_algorithm" title="wikilink">genetic algorithm</a> to solve it.<a class="footnoteRef" href="#fn8" id="fnref8"><sup>8</sup></a><a class="footnoteRef" href="#fn9" id="fnref9"><sup>9</sup></a> All these algorithms generally are robust and insensitive to the number of sequences, but they also have shortcoming, for example, the result got from particle swarm optimization algorithm is unstable and its merits depend on the selection of random numbers, the runtime of A * search algorithm is too long and the genetic algorithm is easy to fall into local excellent.</p>
<h3 id="tree-alignment-graph">Tree Alignment Graph</h3>

<p>Roughly ,tree alignment graph aims to align trees into a graph and finally synthesis them to develop statistics.For biologist,tree alignment graph (TAGs) are used to remove the evolutionary conflicts or overlapping taxa from sets of trees and can be queried to explore uncertainty and conflict.By integrating methods of aligning ,synthsizing and analyzing ,the TAG aims to solve the conflicting relationships and partial overlapping <a class="uri" href="taxon" title="wikilink">taxon</a> sets obtained from a wide range of sequence.Also ,tree alignment graph serves as a fundamental approach for <a class="uri" href="supertree" title="wikilink">supertree</a> and <a class="uri" href="grafting" title="wikilink">grafting</a> exercise,which have been successfully tested to construct supertrees by Berry et al.<a class="footnoteRef" href="#fn10" id="fnref10"><sup>10</sup></a> Because the transformation from trees to a graph contain similar <a href="node_(graph_theory)" title="wikilink">nodes</a> and <a href="Glossary_of_graph_theory#Basics" title="wikilink">edges</a> from their source trees ,TAGs also can provide extraction of original source trees for further analysis . TAG is a combination of a set of aligning trees,it can store conflicting hypotheses evolutionary relationship and synthesize the source trees to develop evolutionary hypotheses ,therefore ,it is a basic method to solve other alignment problems.<a class="footnoteRef" href="#fn11" id="fnref11"><sup>11</sup></a></p>
<h2 id="see-also">See also</h2>
<ul>
<li><a href="Generalized_tree_alignment" title="wikilink">Generalized tree alignment</a></li>
</ul>
<h2 id="references">References</h2>

<p>"</p>

<p><a href="Category:Computational_phylogenetics" title="wikilink">Category:Computational phylogenetics</a></p>
<section class="footnotes">
<hr/>
<ol>
<li id="fn1"><a href="#fnref1">↩</a></li>
<li id="fn2">L Wang,T Jiang. On the complexity of multiple sequence alignment[J]. Journal of Computational Biology, 194,1(4):337— 34.<a href="#fnref2">↩</a></li>
<li id="fn3">Yen Hung Chen, On the bottleneck tree alignment problems, INFORMATION SCIENCES; JUN 1, 2010; 180; 11; p2134-p2141<a href="#fnref3">↩</a></li>
<li id="fn4">Ostrovsky,Rafail ;Rabani,Yuval. Journal of the ACM. Oct 2007, Vol.54 Issue 5, p1-16. 16p. DOI: 10.1145/1284320.1284322.<a href="#fnref4">↩</a></li>
<li id="fn5">Serafim Batzoglou. The many faces of sequence alignment[J]. Briefings in Bioinformatics. <a href="http://bib.oxfordjournals.org/content/6/1/6.full.pdf+html">2005,6(1):6—22</a><a href="#fnref5">↩</a></li>
<li id="fn6">Aho A V, Corasick M J. Efficient string matching: an aid to bibliographic search[J]. Communications of ACM, <a href="ftp://163.13.200.222/assistant/bearhero/prog/%A8%E4%A5%A6/ac_bm.pdf">1975,18(6): 333—340</a>.<a href="#fnref6">↩</a></li>
<li id="fn7">D Gusfield. Algorithms on strings, trees and sequences: computer science and computational biology[M]. <a href="http://www.cambridge.org/us/academic/subjects/computer-science/algorithmics-complexity-computer-algebra-and-computational-g/algorithms-strings-trees-and-sequences-computer-science-and-computational-biology">Cambridge: Cambridge University Press.1997.</a><a href="#fnref7">↩</a></li>
<li id="fn8">RobertC Edgar,Serafim Batzoglou. Multiple sequence alignment[J]. Current opinion in structural biology. <a href="http://robotics.stanford.edu/~serafim/Publications/2006_MSA_COSB.pdf">2006,16(3):368— 373</a>.<a href="#fnref8">↩</a></li>
<li id="fn9">Notredame C, Higgins D.G. SAGA:sequence alignment by genetic algorithm [J]. Nucleic Acids Research. <a href="http://www.tcoffee.org/Publications/Ps_pdf/saga_paper.pdf">1996,24(8):1515-1524</a>.<a href="#fnref9">↩</a></li>
<li id="fn10">Wilkinson M,Pisani D ,Measuring support and finding unsupported relationships in supertrees ,Systematic Biology 54:823-831.<a href="#fnref10">↩</a></li>
<li id="fn11">Stephen A.Smith,Joseph W.Brown,analyzing and synthesizing phylogenies using tree alignment graphs,PLoS Computational Biology 9(9).<a href="#fnref11">↩</a></li>
</ol>
</section>
</body>
</html>
