<html lang="en">
<head>
<meta charset="utf-8"/>
<title offset="1424">Tryptophan 2,3-dioxygenase</title>
<script src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_SVG.js" type="text/javascript">
</script>
</head>
<body>
<h1>Tryptophan 2,3-dioxygenase</h1>
<hr/>

<p>In <a class="uri" href="enzymology" title="wikilink">enzymology</a>, a <strong>tryptophan 2,3-dioxygenase</strong> () is an <a class="uri" href="enzyme" title="wikilink">enzyme</a> that <a href="catalysis" title="wikilink">catalyzes</a> the <a href="chemical_reaction" title="wikilink">chemical reaction</a></p>
<dl>
<dd>L-tryptophan + O<sub>2</sub> 

<math display="inline" id="Tryptophan_2,3-dioxygenase:0">
 <semantics>
  <mo>⇌</mo>
  <annotation-xml encoding="MathML-Content">
   <ci>normal-⇌</ci>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \rightleftharpoons
  </annotation>
 </semantics>
</math>

 N-formyl-L-kynurenine
</dd>
</dl>

<p>Thus, the two <a href="substrate_(biochemistry)" title="wikilink">substrates</a> of this enzyme are <a class="uri" href="L-tryptophan" title="wikilink">L-tryptophan</a> and <a href="oxygen" title="wikilink">O<sub>2</sub></a>, whereas its <a href="product_(chemistry)" title="wikilink">product</a> is <a class="uri" href="N-formyl-L-kynurenine" title="wikilink">N-formyl-L-kynurenine</a>. This enzyme participates in <a href="tryptophan_metabolism" title="wikilink">tryptophan metabolism</a>.</p>

<p>In humans, tryptophan 2,3-dioxygenase is encoded by the <em>TDO2</em> <a class="uri" href="gene" title="wikilink">gene</a>.<a class="footnoteRef" href="#fn1" id="fnref1"><sup>1</sup></a></p>

<p>Tryptophan 2,3-dioxygenase plays a central role in the physiological regulation of tryptophan flux in the human body. It catalyses the first and rate limiting step of tryptophan degradation along the kynurenine pathway and thereby regulates systemic tryptophan levels.<a class="footnoteRef" href="#fn2" id="fnref2"><sup>2</sup></a></p>
<h2 id="function">Function</h2>

<p>This enzyme belongs to the family of <a href="oxidoreductase" title="wikilink">oxidoreductases</a>, specifically those acting on single donors with O<sub>2</sub> as oxidant and incorporation of two atoms of oxygen into the substrate (oxygenases).<a class="footnoteRef" href="#fn3" id="fnref3"><sup>3</sup></a> The oxygen incorporated need not be derived from O<sub>2</sub>.<a class="footnoteRef" href="#fn4" id="fnref4"><sup>4</sup></a> This family of enzymes includes <strong>tryptophan 2,3-dioxygenase</strong> (TDO, also known as <strong>tryptophan oxygenase</strong> and <strong>L-tryptophan pyrrolase</strong>) and <strong>indoleamine 2,3-dioxygenase</strong> (IDO, also known as <strong>tryptophan pyrrolase</strong>).<a class="footnoteRef" href="#fn5" id="fnref5"><sup>5</sup></a> These two enzymes are oxidoreductase enzymes that contain one noncovalently bound iron–<a href="protoporphyrin_IX" title="wikilink">protoporphyrin IX</a> per monomer. These enzymes catalyze the dioxygenation of L-tryptophan (L-Trp) to N-formyl-L-kynurenine in the first and rate-limiting step of the <a href="kynurenine_pathway" title="wikilink">kynurenine pathway</a>.<a class="footnoteRef" href="#fn6" id="fnref6"><sup>6</sup></a></p>

<p>The same family of enzymes also includes sIDO from <em><a href="Shewanella_oneidensis" title="wikilink">Shewanella oneidensis</a></em><a class="footnoteRef" href="#fn7" id="fnref7"><sup>7</sup></a> and PrnB, the second enzyme in the <a class="uri" href="pyrrolnitrin" title="wikilink">pyrrolnitrin</a> biosynthesis pathway from <em><a href="Pseudomonas_fluorescens" title="wikilink">Pseudomonas fluorescens</a></em>,<a class="footnoteRef" href="#fn8" id="fnref8"><sup>8</sup></a> although dioxygenase activity has not been demonstrated for either as yet. Recently, a new enzyme with the ability to catalyze L-tryptophan dioxygenation, INDOL1, was identified.<a class="footnoteRef" href="#fn9" id="fnref9"><sup>9</sup></a></p>

<p>Tryptophan 2,3-dioxygenase was initially discovered in the 1930s<a class="footnoteRef" href="#fn10" id="fnref10"><sup>10</sup></a> and is found in both <a class="uri" href="eukaryotes" title="wikilink">eukaryotes</a> (human,<a class="footnoteRef" href="#fn11" id="fnref11"><sup>11</sup></a> rat,<a class="footnoteRef" href="#fn12" id="fnref12"><sup>12</sup></a> and rabbit<a class="footnoteRef" href="#fn13" id="fnref13"><sup>13</sup></a>) and <a class="uri" href="prokaryotes" title="wikilink">prokaryotes</a> (<em><a href="Xanthomonas_campestris" title="wikilink">Xanthomonas campestris</a></em>,<a class="footnoteRef" href="#fn14" id="fnref14"><sup>14</sup></a> and <em>P. fluorescens</em><a class="footnoteRef" href="#fn15" id="fnref15"><sup>15</sup></a>) Expression of tryptophan 2,3-dioxygenase in mammals is normally restricted to the liver, but it has been identified in the brain and <a class="uri" href="epididymis" title="wikilink">epididymis</a> of some species, and, in some tissues, its production can be induced in response to stimuli.<a class="footnoteRef" href="#fn16" id="fnref16"><sup>16</sup></a></p>
<h2 id="structure">Structure</h2>

<p>Tryptophan 2,3-dioxygenase is a <a class="uri" href="heme" title="wikilink">heme</a>-containing <a class="uri" href="cytosolic" title="wikilink">cytosolic</a> enzyme encoded by <a class="uri" href="gene" title="wikilink">gene</a> TDO2.<a class="footnoteRef" href="#fn17" id="fnref17"><sup>17</sup></a> <a class="uri" href="Crystallographic" title="wikilink">Crystallographic</a> studies of xcTDO (<em>Xanthomonas campestris</em> TDO)<a class="footnoteRef" href="#fn18" id="fnref18"><sup>18</sup></a> and rmTDO (<em><a href="Ralstonia_metallidurans" title="wikilink">Ralstonia metallidurans</a></em> TDO)<a class="footnoteRef" href="#fn19" id="fnref19"><sup>19</sup></a> have revealed that the crystal structures of xcTDO and rmTDO are essentially identical and are intimately associated <a class="uri" href="homotetrameric" title="wikilink">homotetrameric</a> enzymes.<a class="footnoteRef" href="#fn20" id="fnref20"><sup>20</sup></a> They are best described as a dimer of dimers because the N terminal residues of each monomer form part of the substrate binding site in an adjacent monomer. The proteins are completely helical, and a flexible loop, involved in L-tryptophan binding, is observed just outside the active-site pocket. Interestingly, this loop appears to be substrate-binding induced, as it is observed only in crystals grown in the presence of L-tryptophan.<a class="footnoteRef" href="#fn21" id="fnref21"><sup>21</sup></a></p>

<p>The only structure available with <a href="substrate_(chemistry)" title="wikilink">substrate</a> bound at the active site in the catalytically active <a class="uri" href="ferrous" title="wikilink">ferrous</a> state is xcTDO. In the structure of xcTDO, the carboxy group of L-tryptophan interacts with Arg117, Tyr113 and Thr254. Amino acid residues equivalent to Arg117 and Tyr113 are found in nearly all TDO and IDO proteins. This carboxy-binding motif appears to be essential for substrate binding; <a class="uri" href="arginine" title="wikilink">arginine</a> reorients in the presence of substrate, co-ordinating the carboxy group of L-tryptophan. The substrate <a class="uri" href="ammonium" title="wikilink">ammonium</a> group is hydrogen-bonded to the side-chain <a class="uri" href="hydroxyl" title="wikilink">hydroxyl</a> group of Thr254, the 7-<a class="uri" href="propionate" title="wikilink">propionate</a> group of the heme, and a water molecule.<a class="footnoteRef" href="#fn22" id="fnref22"><sup>22</sup></a></p>
<h2 id="mechanism">Mechanism</h2>
<figure><b>(Figure)</b>
<figcaption>Mechanism of action of tryptophan 2,3-dioxygenase enzyme, which catalyzes the reaction L-tryptophan + dioxygen -&gt; N''-formylkynurenine</figcaption>
</figure>

<p>The initial formation of the ternary complex (<strong>1</strong>) occurs by substrate binding, followed by dioxygen binding to the ferrous protein. The ternary complex activates O<sub>2</sub> and allows the otherwise spin-forbidden reaction to proceeed.<a class="footnoteRef" href="#fn23" id="fnref23"><sup>23</sup></a> The formation of the <a class="uri" href="hydroperoxide" title="wikilink">hydroperoxide</a> intermediate (<strong>2</strong>) is catalyzed by the loss of the <a class="uri" href="indole" title="wikilink">indole</a> proton. Two mechanisms are possible: base-catalysed deprotonation or proton abstraction by bound dioxygen.<a class="footnoteRef" href="#fn24" id="fnref24"><sup>24</sup></a> However, catalysis by the ironbound dioxygen is generally proposed, as a result of experiments showing that catalytic activity is maintained upon substitution of alanine for His55 (the only basic residue in the active site of the enzyme).<a class="footnoteRef" href="#fn25" id="fnref25"><sup>25</sup></a></p>

<p>The rearrangement of the hydroperoxide intermediate to form the product could occur via the <a class="uri" href="dioxetane" title="wikilink">dioxetane</a> intermediate (see figure) or a <a class="uri" href="Criegee" title="wikilink">Criegee</a> intermediate. However, <a href="density_functional_theory" title="wikilink">density functional theory</a> calculations on the catalytic mechanism of tryptophan 2,3-dioxygenase have cast doubt on the relevance of the Criegee mechanism.<a class="footnoteRef" href="#fn26" id="fnref26"><sup>26</sup></a></p>
<h2 id="clinical-significance">Clinical significance</h2>

<p>It has been shown that tryptophan 2,3-dioxygenase is expressed in a significant proportion of human <a class="uri" href="tumors" title="wikilink">tumors</a>.<a class="footnoteRef" href="#fn27" id="fnref27"><sup>27</sup></a> In the same study, tryptophan 2,3-dioxygenase expression by tumors prevented their rejection by immunized mice. A tryptophan 2,3-dioxygenase inhibitor developed by the group restored the ability of these mice to reject tryptophan 2,3-dioxygenase-expressed tumors, demonstrating that tryptophan 2,3-dioxygenase inhibitors display potential in cancer therapy.</p>

<p>Another study showed that tryptophan 2,3-dioxygenase is potentially involved in the metabolic pathway responsible for <a class="uri" href="anxiety" title="wikilink">anxiety</a>-related behavior.<a class="footnoteRef" href="#fn28" id="fnref28"><sup>28</sup></a> Generating mice deficient for tryptophan 2,3-dioxygenase and comparing them to the <a href="wild_type" title="wikilink">wild type</a>, the group found that the tryptophan 2,3-dioxygenase-deficient mice showed increased <a href="Blood_plasma" title="wikilink">plasma</a> levels not only of tryptophan, but also of <a class="uri" href="serotonin" title="wikilink">serotonin</a> and <a class="uri" href="5-HIAA" title="wikilink">5-HIAA</a> in the <a class="uri" href="hippocampus" title="wikilink">hippocampus</a> and <a class="uri" href="midbrain" title="wikilink">midbrain</a>. A variety of tests, such as <a href="elevated_plus_maze" title="wikilink">elevated plus maze</a> and open-field tests showed anxiolytic modulation in these <a class="uri" href="knock-out" title="wikilink">knock-out</a> mice, the findings demonstrating a direct link between tryptophan 2,3-dioxygenase and tryptophan metabolism and anxiety-related behavior under physiological conditions.</p>
<h2 id="see-also">See also</h2>
<ul>
<li><a href="Indoleamine_2,3-dioxygenase" title="wikilink">Indoleamine 2,3-dioxygenase</a></li>
<li><a class="uri" href="Kynurenine" title="wikilink">Kynurenine</a></li>
<li><a class="uri" href="5-HTP" title="wikilink">5-HTP</a></li>
</ul>
<h2 id="references">References</h2>
<h2 id="further-reading">Further reading</h2>
<ul>
<li></li>
<li></li>
<li></li>
<li></li>
<li></li>
<li></li>
<li></li>
<li></li>
<li></li>
</ul>

<p>"</p>

<p><a href="Category:EC_1.13.11" title="wikilink">Category:EC 1.13.11</a> <a href="Category:Heme_enzymes" title="wikilink">Category:Heme enzymes</a> <a href="Category:Enzymes_of_known_structure" title="wikilink">Category:Enzymes of known structure</a></p>
<section class="footnotes">
<hr/>
<ol>
<li id="fn1"><a href="#fnref1">↩</a></li>
<li id="fn2"></li>
<li id="fn3"><a href="#fnref3">↩</a></li>
<li id="fn4"><a href="#fnref4">↩</a></li>
<li id="fn5"><a href="#fnref5">↩</a></li>
<li id="fn6"></li>
<li id="fn7"><a href="#fnref7">↩</a></li>
<li id="fn8"><a href="#fnref8">↩</a></li>
<li id="fn9"><a href="#fnref9">↩</a></li>
<li id="fn10"><a href="#fnref10">↩</a></li>
<li id="fn11"><a href="#fnref11">↩</a></li>
<li id="fn12"><a href="#fnref12">↩</a></li>
<li id="fn13"><a href="#fnref13">↩</a></li>
<li id="fn14"></li>
<li id="fn15"></li>
<li id="fn16"></li>
<li id="fn17"><a href="#fnref17">↩</a></li>
<li id="fn18"></li>
<li id="fn19"><a href="#fnref19">↩</a></li>
<li id="fn20"><a href="#fnref20">↩</a></li>
<li id="fn21"></li>
<li id="fn22"></li>
<li id="fn23"></li>
<li id="fn24"><a href="#fnref24">↩</a></li>
<li id="fn25"></li>
<li id="fn26"><a href="#fnref26">↩</a></li>
<li id="fn27"></li>
<li id="fn28"><a href="#fnref28">↩</a></li>
</ol>
</section>
</body>
</html>
