<html lang="en">
<head>
<meta charset="utf-8"/>
<title offset="796">Zygosity</title>
<script src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_SVG.js" type="text/javascript">
</script>
</head>
<body>
<h1>Zygosity</h1>
<hr/>

<p><strong>Zygosity</strong> is the degree of similarity of the <a href="allele" title="wikilink">alleles</a> for a trait in an organism.</p>

<p>Most <a href="eukaryote" title="wikilink">eukaryotes</a> have two matching sets of <a class="uri" href="chromosomes" title="wikilink">chromosomes</a>; that is, they are <a class="uri" href="diploid" title="wikilink">diploid</a>. Diploid organisms have the same loci on each of their two sets of homologous chromosomes, except that the sequences at these loci may differ between the two chromosomes in a matching pair and that a few chromosomes may be mismatched as part of a chromosomal <a href="Sex-determination_system#Chromosomal_determination" title="wikilink">sex-determination system</a>. If both alleles of a diploid organism are the same, the organism is <a href="Zygosity#Homozygous" title="wikilink"> homozygous</a> at that locus. If they are different, the organism is <a href="Zygosity#Heterozygous" title="wikilink"> heterozygous</a> at that locus. If one allele is missing, it is <a href="Zygosity#Hemizygous" title="wikilink"> hemizygous</a>, and, if both alleles are missing, it is <a href="Zygosity#Nullizygous" title="wikilink"> nullizygous</a>.</p>

<p>The DNA sequence of a gene often varies from one individual to another. Those variations are called <a class="uri" href="alleles" title="wikilink">alleles</a>. While some genes have only one allele because there is low variation, others have only one allele because only that allele can function properly. Any variation from the DNA sequence of that allele will be fatal in the embryo, and the organism will never survive to be born. But most genes have two or more alleles. The frequency of different alleles varies throughout the population. Some genes may have two alleles with equal distribution. For other genes, one allele may be common, and another allele may be rare. Sometimes, one allele is a <a href="Genetic_disorder" title="wikilink">disease</a>-causing variation while the other allele is healthy. Sometimes, the different variations in the alleles make no difference at all in the function of the organism.</p>

<p>In diploid organisms, one allele is inherited from the male parent and one from the female parent. Zygosity is a description of whether those two alleles have identical or different DNA sequences. In some cases the term "zygosity" is used in the context of a single chromosome.<a class="footnoteRef" href="#fn1" id="fnref1"><sup>1</sup></a></p>
<h2 id="types">Types</h2>

<p>The words homozygous, heterozygous, and hemizygous are used to describe the <a class="uri" href="genotype" title="wikilink">genotype</a> of a diploid organism at a single <a href="locus_(genetics)" title="wikilink">locus</a> on the DNA. <em>Homozygous</em> describes a genotype consisting of two identical alleles at a given locus, <em>heterozygous</em> describes a genotype consisting of two different alleles at a locus, <em>hemizygous</em> describes a genotype consisting of only a single copy of a particular gene in an otherwise diploid organism, and <em>nullizygous</em> refers to an otherwise-diploid organism in which both copies of the gene are missing.</p>
<h3 id="homozygous">Homozygous</h3>

<p>A cell is said to be homozygous for a particular gene when identical alleles of the gene are present on both <a href="homologous_chromosomes" title="wikilink">homologous chromosomes</a>.<a class="footnoteRef" href="#fn2" id="fnref2"><sup>2</sup></a> The cell or organism in question is called a <strong>homozygote</strong>. <a href="True_breeding_organism" title="wikilink">True breeding organisms</a> are always homozygous for the traits that are to be held constant.</p>

<p>An individual that is <strong>homozygous-dominant</strong> for a particular trait carries two copies of the allele that codes for the <a href="Dominance_(genetics)" title="wikilink">dominant</a> trait. This allele, often called the "dominant allele", is normally represented by a capital letter (such as "P" for the dominant allele producing purple flowers in pea plants). When an organism is homozygous-dominant for a particular trait, the genotype is represented by a doubling of the symbol for that trait, such as "PP".</p>

<p>An individual that is <strong>homozygous-recessive</strong> for a particular trait carries two copies of the allele that codes for the <a href="recessive_trait" title="wikilink">recessive trait</a>. This allele, often called the "recessive allele", is usually represented by the lowercase form of the letter used for the corresponding dominant trait (such as, with reference to the example above, "p" for the recessive allele producing white flowers in pea plants). The genotype of an organism that is homozygous-recessive for a particular trait is represented by a doubling of the appropriate letter, such as "pp".</p>
<h3 id="heterozygous">Heterozygous</h3>

<p>A <a href="Ploidy#Diploid" title="wikilink">diploid</a> organism is heterozygous at a gene locus when its cells contain two different <a href="allele" title="wikilink">alleles</a> of a gene.<a class="footnoteRef" href="#fn3" id="fnref3"><sup>3</sup></a> The cell or organism is called a <strong>heterozygote</strong> <em>specifically</em> for the allele in question, therefore, heterozygosity refers to a specific genotype. Heterozygous genotypes are represented by a capital letter (representing the dominant allele) and a lowercase letter (representing the recessive allele), such as "Rr" or "Ss". Alternatively, a heterozygote for gene "R" is assumed to be "Rr". The capital letter is usually written first.</p>

<p>If the trait in question is determined by simple (complete) dominance, a heterozygote will express only the trait coded by the dominant allele, and the trait coded by the recessive allele will not be present. In <a href="Dominant_allele#Types_of_dominances" title="wikilink">more complex dominance schemes</a> the results of heterozygosity can be more complex.</p>
<h3 id="hemizygous">Hemizygous</h3>

<p>A chromosome in a diploid organism is hemizygous when only one copy is present.<a class="footnoteRef" href="#fn4" id="fnref4"><sup>4</sup></a> The cell or organism is called a <em>hemizygote</em>. Hemizygosity is also observed when one copy of a gene is deleted, or in the <a href="sex_linkage" title="wikilink">heterogametic</a> sex when a gene is located on a sex chromosome. For organisms in which the male is heterogametic, such as humans, almost all X-linked genes are hemizygous in males with normal chromosomes because they have only one <a href="X_chromosome" title="wikilink">X chromosome</a> and few of the same genes are on the <a href="Y_chromosome" title="wikilink">Y chromosome</a>. Transgenic mice generated through exogenous DNA microinjection of an embryo's pronucleus are also considered to be hemizygous because the introduced allele is expected to be incorporated into only one copy of any locus. A transgenic can later be bred to homozygosity and maintained as an inbred line to reduce the need to confirm the genotypes of each litter.</p>

<p>In cultured mammalian cells, such as the <a href="Chinese_hamster_ovary_cell" title="wikilink">Chinese hamster ovary cell</a> line, a number of genetic loci are present in a functional hemizygous state, due to mutations or deletions in the other alleles. <a class="footnoteRef" href="#fn5" id="fnref5"><sup>5</sup></a></p>
<h3 id="nullizygous">Nullizygous</h3>

<p>A nullizygous organism carries two mutant alleles for the same gene. The mutant alleles are both complete loss-of-function or 'null' alleles, so homozygous null and nullizygous are synonymous.<a class="footnoteRef" href="#fn6" id="fnref6"><sup>6</sup></a> The mutant cell or organism is called a <em>nullizygote</em>.</p>
<h2 id="autozygous-and-allozygous">Autozygous and allozygous</h2>

<p>Zygosity may also refer to the origin(s) of the alleles in a genotype. When the two alleles at a locus originate from a common ancestor by way of nonrandom mating (<a class="uri" href="inbreeding" title="wikilink">inbreeding</a>), the genotype is said to be <em>autozygous</em>. This is also known as being "identical by descent", or IBD. When the two alleles come (at least to the extent that the descent can be traced) from completely different sources, as is the case in most normal, random mating, the genotype is called <em>allozygous</em>. This is known as being "identical by state", or IBS.</p>

<p>Because the alleles of autozygous genotypes come from the same source, they are always homozygous, but allozygous genotypes may be homozygous too. All heterozygous genotypes are, by definition, allozygous because they contain two completely different alleles. Hemizygous and nullizygous genotypes do not contain enough alleles to allow for comparison of sources, so this classification is irrelevant for them.</p>
<h2 id="monozygotic-and-dizygotic-twins">Monozygotic and dizygotic twins</h2>

<p>As discussed above, "zygosity" can be used in the context of a specific genetic locus (example<a class="footnoteRef" href="#fn7" id="fnref7"><sup>7</sup></a>). In addition, the word "zygosity" may also be used to describe the genetic similarity or dissimilarity of twins.<a class="footnoteRef" href="#fn8" id="fnref8"><sup>8</sup></a> Identical twins are <strong>monozygotic</strong>, meaning that they develop from one zygote that splits and forms two embryos. Fraternal twins are <strong>dizygotic</strong> because they develop from two separate eggs that are fertilized by two separate <a class="uri" href="sperm" title="wikilink">sperm</a>.</p>
<h2 id="heterozygosity-in-population-genetics">Heterozygosity in population genetics</h2>

<p>In <a href="population_genetics" title="wikilink">population genetics</a>, the concept of heterozygosity is commonly extended to refer to the population as a whole, i.e., the fraction of individuals in a population that are heterozygous for a particular locus. It can also refer to the fraction of loci within an individual that are heterozygous.</p>

<p>Typically, the observed (

<math display="inline" id="Zygosity:0">
 <semantics>
  <msub>
   <mi>H</mi>
   <mi>o</mi>
  </msub>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <csymbol cd="ambiguous">subscript</csymbol>
    <ci>H</ci>
    <ci>o</ci>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   H_{o}
  </annotation>
 </semantics>
</math>

) and expected (

<math display="inline" id="Zygosity:1">
 <semantics>
  <msub>
   <mi>H</mi>
   <mi>e</mi>
  </msub>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <csymbol cd="ambiguous">subscript</csymbol>
    <ci>H</ci>
    <ci>e</ci>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   H_{e}
  </annotation>
 </semantics>
</math>

) heterozygosities are compared, defined as follows for diploid individuals in a population:</p>
<dl>
<dt>Observed</dt>
<dd>

<math display="inline" id="Zygosity:2">
 <semantics>
  <mrow>
   <msub>
    <mi>H</mi>
    <mi>o</mi>
   </msub>
   <mo>=</mo>
   <mfrac>
    <mrow>
     <mstyle displaystyle="false">
      <munderover>
       <mo largeop="true" movablelimits="false" symmetric="true">∑</mo>
       <mrow>
        <mi>i</mi>
        <mo>=</mo>
        <mn>1</mn>
       </mrow>
       <mi>n</mi>
      </munderover>
     </mstyle>
     <mrow>
      <mo stretchy="false">(</mo>
      <mpadded width="+5pt">
       <mn>1</mn>
      </mpadded>
      <mpadded width="+5pt">
       <mtext>if</mtext>
      </mpadded>
      <msub>
       <mi>a</mi>
       <mrow>
        <mi>i</mi>
        <mn>1</mn>
       </mrow>
      </msub>
      <mo>≠</mo>
      <msub>
       <mi>a</mi>
       <mrow>
        <mi>i</mi>
        <mn>2</mn>
       </mrow>
      </msub>
      <mo stretchy="false">)</mo>
     </mrow>
    </mrow>
    <mi>n</mi>
   </mfrac>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <eq></eq>
    <apply>
     <csymbol cd="ambiguous">subscript</csymbol>
     <ci>H</ci>
     <ci>o</ci>
    </apply>
    <apply>
     <divide></divide>
     <cerror>
      <csymbol cd="ambiguous">fragments</csymbol>
      <apply>
       <csymbol cd="ambiguous">superscript</csymbol>
       <apply>
        <csymbol cd="ambiguous">subscript</csymbol>
        <sum></sum>
        <apply>
         <eq></eq>
         <ci>i</ci>
         <cn type="integer">1</cn>
        </apply>
       </apply>
       <ci>n</ci>
      </apply>
      <cerror>
       <csymbol cd="ambiguous">fragments</csymbol>
       <ci>normal-(</ci>
       <cn type="integer">1</cn>
       <mtext>if</mtext>
       <apply>
        <csymbol cd="ambiguous">subscript</csymbol>
        <ci>a</ci>
        <apply>
         <times></times>
         <ci>i</ci>
         <cn type="integer">1</cn>
        </apply>
       </apply>
       <neq></neq>
       <apply>
        <csymbol cd="ambiguous">subscript</csymbol>
        <ci>a</ci>
        <apply>
         <times></times>
         <ci>i</ci>
         <cn type="integer">2</cn>
        </apply>
       </apply>
       <ci>normal-)</ci>
      </cerror>
     </cerror>
     <ci>n</ci>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   H_{o}=\frac{\sum\limits_{i=1}^{n}{(1\ \textrm{if}\ a_{i1}\neq a_{i2})}}{n}
  </annotation>
 </semantics>
</math>


</dd>
</dl>

<p>where 

<math display="inline" id="Zygosity:3">
 <semantics>
  <mi>n</mi>
  <annotation-xml encoding="MathML-Content">
   <ci>n</ci>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   n
  </annotation>
 </semantics>
</math>

 is the number of individuals in the population, and 

<math display="inline" id="Zygosity:4">
 <semantics>
  <mrow>
   <msub>
    <mi>a</mi>
    <mrow>
     <mi>i</mi>
     <mn>1</mn>
    </mrow>
   </msub>
   <mo>,</mo>
   <msub>
    <mi>a</mi>
    <mrow>
     <mi>i</mi>
     <mn>2</mn>
    </mrow>
   </msub>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <list>
    <apply>
     <csymbol cd="ambiguous">subscript</csymbol>
     <ci>a</ci>
     <apply>
      <times></times>
      <ci>i</ci>
      <cn type="integer">1</cn>
     </apply>
    </apply>
    <apply>
     <csymbol cd="ambiguous">subscript</csymbol>
     <ci>a</ci>
     <apply>
      <times></times>
      <ci>i</ci>
      <cn type="integer">2</cn>
     </apply>
    </apply>
   </list>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   a_{i1},a_{i2}
  </annotation>
 </semantics>
</math>

 are the alleles of individual 

<math display="inline" id="Zygosity:5">
 <semantics>
  <mi>i</mi>
  <annotation-xml encoding="MathML-Content">
   <ci>i</ci>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   i
  </annotation>
 </semantics>
</math>

 at the target locus.</p>
<dl>
<dt>Expected</dt>
<dd><math>
</math></dd>
</dl>

<p>H_e = 1 - \sum\limits_{i=1}^{m}{(f_i)^2}  where 

<math display="inline" id="Zygosity:6">
 <semantics>
  <mi>m</mi>
  <annotation-xml encoding="MathML-Content">
   <ci>m</ci>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   m
  </annotation>
 </semantics>
</math>

 is the number of alleles at the target locus, and 

<math display="inline" id="Zygosity:7">
 <semantics>
  <msub>
   <mi>f</mi>
   <mi>i</mi>
  </msub>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <csymbol cd="ambiguous">subscript</csymbol>
    <ci>f</ci>
    <ci>i</ci>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   f_{i}
  </annotation>
 </semantics>
</math>

 is the <a href="allele_frequency" title="wikilink">allele frequency</a> of the 

<math display="inline" id="Zygosity:8">
 <semantics>
  <msup>
   <mi>i</mi>
   <mrow>
    <mi>t</mi>
    <mi>h</mi>
   </mrow>
  </msup>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <csymbol cd="ambiguous">superscript</csymbol>
    <ci>i</ci>
    <apply>
     <times></times>
     <ci>t</ci>
     <ci>h</ci>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   i^{th}
  </annotation>
 </semantics>
</math>

 allele at the target locus.</p>
<h2 id="see-also">See also</h2>
<ul>
<li><a class="uri" href="Heterosis" title="wikilink">Heterosis</a></li>
<li><a href="Heterozygote_advantage" title="wikilink">Heterozygote advantage</a></li>
<li><a href="Loss_of_heterozygosity" title="wikilink">Loss of heterozygosity</a></li>
<li><a href="Nucleotide_diversity" title="wikilink">Nucleotide diversity</a> measures polymorphisms on the level of nucleotides rather than on level of loci.</li>
</ul>
<h2 id="references">References</h2>

<p>"</p>

<p><a href="Category:Classical_genetics" title="wikilink">Category:Classical genetics</a></p>
<section class="footnotes">
<hr/>
<ol>
<li id="fn1"><a href="#fnref1">↩</a></li>
<li id="fn2">Lawrence, Eleanor: <em>Henderson's Dictionary of Biology, 14e</em>, 2008<a href="#fnref2">↩</a></li>
<li id="fn3"><a href="http://www.ncbi.nlm.nih.gov/bookshelf/br.fcgi?book=mcb">Molecular Cell Biology</a>, 4th edition by Harvey Lodish, et al [<a class="uri" href="http://www.ncbi.nlm.nih.gov/bookshelf/br.fcgi?book=mcb">http://www.ncbi.nlm.nih.gov/bookshelf/br.fcgi?book=mcb</a>∂=A1876#A1877 Chapter 8]. (2000)<a href="#fnref3">↩</a></li>
<li id="fn4"></li>
<li id="fn5">Gupta, R.S., Chan, D.H.Y. and Siminovitch, L. (1978). Evidence obtained by segregation analysis for functional hemizygosity at the Emtr locus in CHO cells. Cell 15: 1007-1013<a href="#fnref5">↩</a></li>
<li id="fn6"></li>
<li id="fn7"><a href="#fnref7">↩</a></li>
<li id="fn8"><a href="http://www.ncbi.nlm.nih.gov/bookshelf/br.fcgi?book=hmg">Human Molecular Genetics</a>, 2nd edition by Tom Strachan and Andrew P. Read [<a class="uri" href="http://www.ncbi.nlm.nih.gov/bookshelf/br.fcgi?book=hmg">http://www.ncbi.nlm.nih.gov/bookshelf/br.fcgi?book=hmg</a>∂=A2178#A2306 Chapter 17]. (1999).<a href="#fnref8">↩</a></li>
</ol>
</section>
</body>
</html>
