<html lang="en">
<head>
<meta charset="utf-8"/>
<title offset="676">Aminoacylase</title>
<script src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_SVG.js" type="text/javascript">
</script>
</head>
<body>
<h1>Aminoacylase</h1>
<hr/>

<p>In <a class="uri" href="enzymology" title="wikilink">enzymology</a>, an <strong>aminoacylase</strong> () is an <a class="uri" href="enzyme" title="wikilink">enzyme</a> that <a href="catalysis" title="wikilink">catalyzes</a> the <a href="chemical_reaction" title="wikilink">chemical reaction</a></p>

<p><a href="amino_acids" title="wikilink">N-acyl-L-amino acid</a> +  <a href="water" title="wikilink">H<sub>2</sub>O</a> 

<math display="inline" id="Aminoacylase:0">
 <semantics>
  <mo>⇌</mo>
  <annotation-xml encoding="MathML-Content">
   <ci>normal-⇌</ci>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \rightleftharpoons
  </annotation>
 </semantics>
</math>

 <a class="uri" href="carboxylate" title="wikilink">carboxylate</a>  + <a href="L-amino_acid" title="wikilink">L-amino acid</a></p>
<figure><b>(Figure)</b>
<figcaption>Mechanism Summary rightleft.png</figcaption>
</figure>

<p>Thus, the two <a href="substrate_(biochemistry)" title="wikilink">substrates</a> of this enzyme are <a href="amino_acids" title="wikilink">N-acyl-L-amino acid</a> and <a href="water" title="wikilink">H<sub>2</sub>O</a>, whereas its two <a href="product_(chemistry)" title="wikilink">products</a> are <a class="uri" href="carboxylate" title="wikilink">carboxylate</a> and <a href="L-amino_acid" title="wikilink">L-amino acid</a>.</p>

<p>This <a class="uri" href="enzyme" title="wikilink">enzyme</a> belongs to the family of <a href="hydrolase" title="wikilink">hydrolases</a>, those acting on <a href="carbon-nitrogen_bonds" title="wikilink">carbon-nitrogen bonds</a> other than <a href="peptide_bonds" title="wikilink">peptide bonds</a>, specifically in linear <a class="uri" href="amides" title="wikilink">amides</a>. The systematic name of this <a class="uri" href="enzyme" title="wikilink">enzyme</a> class is <em><a href="amino_acids" title="wikilink">N-acyl-L-amino acid</a> amidohydrolase</em>'. Other names in common use include <strong>dehydropeptidase II</strong>, <strong>histozyme</strong>, <strong>hippuricase</strong>, <strong>benzamidase</strong>, <strong>acylase I</strong>, <strong>hippurase</strong>, <strong>amido acid deacylase</strong>, <strong>L-aminoacylase</strong>, <strong>acylase</strong>, <strong>aminoacylase I</strong>, <strong>L-amino-acid acylase</strong>, <strong>alpha-N-acylaminoacid <a class="uri" href="hydrolase" title="wikilink">hydrolase</a></strong>, <strong>long acyl amidoacylase</strong>, and <strong>short acyl amidoacylase</strong>. This <a class="uri" href="enzyme" title="wikilink">enzyme</a> participates in <a href="urea_cycle_and_metabolism_of_amino_groups" title="wikilink">urea cycle and metabolism of amino groups</a>.</p>
<h2 id="enzyme-structure">Enzyme structure</h2>
<figure><b>(Figure)</b>
<figcaption>The quaternary structure of an Aminoacylase 1 (PDB 1Q7L)</figcaption>
</figure>

<p>As of late 2007, two <a href="tertiary_structure" title="wikilink">structures</a> have been solved for this class of <a class="uri" href="enzymes" title="wikilink">enzymes</a>, with <a href="Protein_Data_Bank" title="wikilink">PDB</a> accession codes  and . These <a href="Protein_structure_prediction" title="wikilink">structures</a> also correspond to two known <a href="Protein_primary_structure" title="wikilink">primary amino acid sequences</a> for <a class="uri" href="aminoacylases" title="wikilink">aminoacylases</a>. The associated papers identify two types of <a href="Protein_domain" title="wikilink">domains</a> comprising aminoacylases: <a class="uri" href="Zinc" title="wikilink">Zinc</a> <a href="binding_domain" title="wikilink">binding domains</a> - which bind Zn<sup>2+</sup> ions - and <a href="Protein_domain" title="wikilink">domains</a> that facilitate <a href="Protein_dimer" title="wikilink">dimerization</a> of <a class="uri" href="Zinc" title="wikilink">Zinc</a> <a href="binding_domains" title="wikilink">binding domains</a>.<a class="footnoteRef" href="#fn1" id="fnref1"><sup>1</sup></a><a class="footnoteRef" href="#fn2" id="fnref2"><sup>2</sup></a> It is this <a href="Protein_dimer" title="wikilink">dimerization</a> that allows <a class="uri" href="catalysis" title="wikilink">catalysis</a> to occur, since aminoacylase's <a href="active_site" title="wikilink">active site</a> lies between its two <a class="uri" href="Zinc" title="wikilink">Zinc</a> binding <a href="protein_domain" title="wikilink">domains</a>.<a class="footnoteRef" href="#fn3" id="fnref3"><sup>3</sup></a></p>

<p>Bound <a class="uri" href="Zinc" title="wikilink">Zinc</a> facilitates the <a href="enzyme_binding" title="wikilink">binding</a> of the <a href="amino_acids" title="wikilink">N-acyl-L-amino acid</a> <a href="enzyme_substrate" title="wikilink">substrate</a>, causing a <a href="conformational_shift" title="wikilink">conformational shift</a> that brings the <a href="protein" title="wikilink">protein's</a> <a href="Protein_subunit" title="wikilink">subunits</a> together around the <a href="Substrate_(biology)" title="wikilink">substrate</a> and allowing <a class="uri" href="catalysis" title="wikilink">catalysis</a> to occur.<a class="footnoteRef" href="#fn4" id="fnref4"><sup>4</sup></a> <a href="Aminoacylase_1" title="wikilink">Aminoacylase 1</a> exists in a <a class="uri" href="heterotetrameric" title="wikilink">heterotetrameric</a> structure, meaning 2 <a class="uri" href="Zinc" title="wikilink">Zinc</a> <a href="binding_domains" title="wikilink">binding domains</a> and 2 dimerization <a href="protein_domain" title="wikilink">domains</a> come together to make <a href="aminoacylase_1" title="wikilink">aminoacylase 1</a>'s <a href="quaternary_structure" title="wikilink">quaternary structure</a>.</p>
<h2 id="enzyme-mechanism">Enzyme Mechanism</h2>
<figure><b>(Figure)</b>
<figcaption>Aminoacylase Reaction Mechanism (click for larger image)</figcaption>
</figure>

<p>Aminoacylase is a <a href="Metalloprotein" title="wikilink">metallo-enzyme</a> that needs <a class="uri" href="Zinc" title="wikilink">Zinc</a> (<a href="Zinc" title="wikilink">Zn<sup>2+</sup></a>) as a <a href="Cofactor_(biochemistry)" title="wikilink">cofactor</a> to function.<a class="footnoteRef" href="#fn5" id="fnref5"><sup>5</sup></a><a class="footnoteRef" href="#fn6" id="fnref6"><sup>6</sup></a> The <a class="uri" href="Zinc" title="wikilink">Zinc</a> <a href="ion" title="wikilink">ions</a> inside of aminoacylase are each coordinated to <a class="uri" href="histidine" title="wikilink">histidine</a>, <a class="uri" href="glutamate" title="wikilink">glutamate</a>, <a class="uri" href="aspartate" title="wikilink">aspartate</a>, and <a class="uri" href="water" title="wikilink">water</a>.<a class="footnoteRef" href="#fn7" id="fnref7"><sup>7</sup></a><a class="footnoteRef" href="#fn8" id="fnref8"><sup>8</sup></a><a class="footnoteRef" href="#fn9" id="fnref9"><sup>9</sup></a> The <a class="uri" href="Zinc" title="wikilink">Zinc</a> <a class="uri" href="ion" title="wikilink">ion</a> <a href="chemical_polarity" title="wikilink">polarizes</a> the <a href="wat" title="wikilink">water</a>, facilitating its <a class="uri" href="deprotonation" title="wikilink">deprotonation</a> by a nearby <a href="Acid-base_chemistry" title="wikilink">basic</a> <a href="amino_acid" title="wikilink">residue</a>.<a class="footnoteRef" href="#fn10" id="fnref10"><sup>10</sup></a><a class="footnoteRef" href="#fn11" id="fnref11"><sup>11</sup></a> The negatively charged <a href="hydroxide_ion" title="wikilink">hydroxide ion</a> is <a href="nucleophile" title="wikilink">nucleophilic</a> and attacks the <a href="electrophile" title="wikilink">electrophilic</a> <a class="uri" href="carbonyl" title="wikilink">carbonyl</a> <a class="uri" href="carbon" title="wikilink">carbon</a> of the <a href="Enzyme_substrate_(biology)" title="wikilink">substrate's</a> <a href="acyl_group" title="wikilink">acyl group</a>.<a class="footnoteRef" href="#fn12" id="fnref12"><sup>12</sup></a> The exact <a href="Chemical_reaction_mechanism" title="wikilink">mechanism</a> after this point is unknown, with one possibility being that the carbonyl then reforms, breaks the <a href="amide_bond" title="wikilink">amide bond</a>, and forms the two <a href="chemical_reaction" title="wikilink">products</a>. At some point in the <a href="Chemical_reaction_mechanism" title="wikilink">mechanism</a>, another <a class="uri" href="water" title="wikilink">water</a> molecule enters and <a href="metal_coordination" title="wikilink">coordinates</a> with <a class="uri" href="Zinc" title="wikilink">Zinc</a>, returning the <a class="uri" href="enzyme" title="wikilink">enzyme</a> to its original state.<a class="footnoteRef" href="#fn13" id="fnref13"><sup>13</sup></a></p>
<figure><b>(Figure)</b>
<figcaption>Michaelis-Menten Kinetics of Aminoacylase Reaction</figcaption>
</figure>

<p>The nucleophilic attack by water is the rate-limiting step of aminoacylase's <a href="catalysis" title="wikilink">catalytic</a> <a href="chemical_reaction_mechanism" title="wikilink">mechanism</a>.<a class="footnoteRef" href="#fn14" id="fnref14"><sup>14</sup></a> This <a href="nucleophilic_attack" title="wikilink">nucleophilic attack</a> is reversible while the subsequent steps are fast and irreversible.<a class="footnoteRef" href="#fn15" id="fnref15"><sup>15</sup></a> This reaction sequence is an example of <a href="Michaelis–Menten_kinetics" title="wikilink">Michaelis–Menten kinetics</a>, allowing one to determine <a href="Michaelis–Menten_kinetics" title="wikilink">K<sub>M</sub></a>, <a href="Michaelis–Menten_kinetics" title="wikilink">K<sub>cat</sub></a>, <a href="Michaelis–Menten_kinetics" title="wikilink">V<sub>max</sub></a>, <a href="turnover_number" title="wikilink">turnover number</a>, and <a href="Specificity_constant" title="wikilink">substrate specificity</a> through classic <a href="Michaelis–Menten_kinetics" title="wikilink">Michaelis-Menten</a> <a class="uri" href="enzyme" title="wikilink">enzyme</a> experiments.<a class="footnoteRef" href="#fn16" id="fnref16"><sup>16</sup></a> The second and third forward steps cause the formation and release of the reaction's <a href="chemical_reaction" title="wikilink">products</a>.<a class="footnoteRef" href="#fn17" id="fnref17"><sup>17</sup></a> </p>
<h2 id="biological-function">Biological function</h2>

<p> Aminoacylases are expressed in the <a class="uri" href="kidney" title="wikilink">kidney</a>, where they recycle<a href="amino_a" title="wikilink">N-acyl-L-amino acids</a> as <a href="amino_acids" title="wikilink">L-amino acids</a> and aid in urea cycle regulation.</p>

<p><a href="amino_acid" title="wikilink">N-acyl-L-amino acids</a> are formed when <a href="amino_acid" title="wikilink">L-amino acids</a> have their <a class="uri" href="N-terminus" title="wikilink">N-terminus</a> <a href="covalent_bond" title="wikilink">covalently bonded</a> to an <a href="acyl_group" title="wikilink">acyl group</a>. The <a href="acyl_group" title="wikilink">acyl group</a> provides stability for the <a href="amino_acid" title="wikilink">amino acid</a>, making it more resistant to degradation. Additionally, <a href="amino_acid" title="wikilink">N-acyl-L-amino acids</a> cannot be used directly as building blocks for <a class="uri" href="proteins" title="wikilink">proteins</a> and must first be converted to <a href="amino_acid" title="wikilink">L-amino acids</a> by aminoacylase. Again, the <a href="amino_acid" title="wikilink">L-amino acid</a> products can be used for <a class="uri" href="biosynthesis" title="wikilink">biosynthesis</a> or catabolized <a href="energy_production_biology" title="wikilink">energy</a>.</p>

<p>Aminoacylase is involved in the <a href="biological_regulation" title="wikilink">regulation</a> of the <a href="urea_cycle" title="wikilink">urea cycle</a>. <a class="uri" href="N-acetyl-L-glutamate" title="wikilink">N-acetyl-L-glutamate</a> is an <a class="uri" href="allosteric" title="wikilink">allosteric</a> <a href="enzyme_regulation" title="wikilink">activator</a> of <a href="carbamoyl_phosphate_synthetase" title="wikilink">carbamoyl phosphate synthetase</a>, a crucial <a class="uri" href="enzyme" title="wikilink">enzyme</a> that commits NH<sub>4</sub><sup>+</sup> <a class="uri" href="molecules" title="wikilink">molecules</a> to the <a href="urea_cycle" title="wikilink">urea cycle</a>.<a class="footnoteRef" href="#fn18" id="fnref18"><sup>18</sup></a> The urea cycle gets rid of excess <a class="uri" href="ammonia" title="wikilink">ammonia</a> (<a href="ammonia" title="wikilink">NH<sub>4</sub><sup>+</sup></a>) in the body, a process that must be up-regulated during times of increased <a href="protein_catabolism" title="wikilink">protein catabolism</a>, as <a href="amino_acid" title="wikilink">amino acid</a> breakdown produces large amounts of <a href="ammonia" title="wikilink">NH<sub>4</sub><sup>+</sup></a>.<a class="footnoteRef" href="#fn19" id="fnref19"><sup>19</sup></a> When amino acid catabolism increases, <a href="N-Acetylglutamate_synthase" title="wikilink">N-Acetylglutamate synthase</a> is up-regulated, producing more N-acetyl-L-glutamate, which up-regulates <a href="carbamoyl_phosphate_synthetase" title="wikilink">carbamoyl phosphate synthetase</a> and allows it to dispose of the excess <a href="ammonia" title="wikilink">NH<sub>4</sub><sup>+</sup></a> from <a class="uri" href="catabolism" title="wikilink">catabolism</a>.<a class="footnoteRef" href="#fn20" id="fnref20"><sup>20</sup></a></p>

<p>Aminoacylase is up-regulated during times of nutrient deficit or <a class="uri" href="starvation" title="wikilink">starvation</a>, causing <a class="uri" href="N-acetyl-L-glutamate" title="wikilink">N-acetyl-L-glutamate</a> breakdown, which <a href="biological_regulation" title="wikilink">down-regulates</a> <a href="carbamoyl_phosphate_synthetase" title="wikilink">carbamoyl phosphate synthetase</a> and the rest of the <a href="urea_cycle" title="wikilink">urea cycle</a>. This response is evolutionarily advantageous, since a nutrient deficit means there isn't as much <a href="ammonia" title="wikilink">NH<sub>4</sub><sup>+</sup></a> that needs to be disposed of and since the body wants to salvage as many amino acids as it can.<a class="footnoteRef" href="#fn21" id="fnref21"><sup>21</sup></a></p>
<h2 id="disease-relevance">Disease relevance</h2>

<p><a href="Aminoacylase_1_deficiency" title="wikilink">Aminoacylase 1 deficiency</a> (<a href="Aminoacylase_1_deficiency" title="wikilink">A1D</a>) is a rare <a class="uri" href="disease" title="wikilink">disease</a> caused by an <a href="autosomal_recessive_mutation" title="wikilink">autosomal recessive mutation</a> in the <a href="aminoacylase_1" title="wikilink">aminoacylase 1</a> <a class="uri" href="gene" title="wikilink">gene</a> (<a class="uri" href="ACY1" title="wikilink">ACY1</a>) on <a href="chromosome_3p21" title="wikilink">chromosome 3p21</a>.<a class="footnoteRef" href="#fn22" id="fnref22"><sup>22</sup></a><a class="footnoteRef" href="#fn23" id="fnref23"><sup>23</sup></a><a class="footnoteRef" href="#fn24" id="fnref24"><sup>24</sup></a><a class="footnoteRef" href="#fn25" id="fnref25"><sup>25</sup></a><a class="footnoteRef" href="#fn26" id="fnref26"><sup>26</sup></a> The lack of functional <a href="aminoacylase_1" title="wikilink">aminoacylase 1</a> caused by <a href="Aminoacylase_1_deficiency" title="wikilink">A1D</a> results in a dysfunctional <a href="urea_cycle" title="wikilink">urea cycle</a>, causing an array of <a href="neurological_disorders" title="wikilink">neurological disorders</a> including <a class="uri" href="seizures" title="wikilink">seizures</a>, <a href="muscular_hypotonia" title="wikilink">muscular hypotonia</a>, <a href="mental_retardation" title="wikilink">mental retardation</a>, and <a href="impaired_psychomotor_development" title="wikilink">impaired psychomotor development</a>.<a class="footnoteRef" href="#fn27" id="fnref27"><sup>27</sup></a><a class="footnoteRef" href="#fn28" id="fnref28"><sup>28</sup></a><a class="footnoteRef" href="#fn29" id="fnref29"><sup>29</sup></a><a class="footnoteRef" href="#fn30" id="fnref30"><sup>30</sup></a><a class="footnoteRef" href="#fn31" id="fnref31"><sup>31</sup></a> <a href="Aminoacylase_1_deficiency" title="wikilink">A1D</a> has also been associated with <a class="uri" href="autism" title="wikilink">autism</a> .<a class="footnoteRef" href="#fn32" id="fnref32"><sup>32</sup></a> Patients with <a href="Aminoacylase_1_deficiency" title="wikilink">A1D</a> often start expressing symptoms shortly after <a class="uri" href="birth" title="wikilink">birth</a> but seem to recover fully in the next few years.<a class="footnoteRef" href="#fn33" id="fnref33"><sup>33</sup></a><a class="footnoteRef" href="#fn34" id="fnref34"><sup>34</sup></a><a class="footnoteRef" href="#fn35" id="fnref35"><sup>35</sup></a><a class="footnoteRef" href="#fn36" id="fnref36"><sup>36</sup></a></p>

<p><a href="Aminoacylase_1_deficiency" title="wikilink">Aminoacylase 2 Deficiency</a> - also known as <a href="Canavan_disease" title="wikilink">Canavan's Disease</a> - is another <a href="rare_disease" title="wikilink">rare disease</a> caused by a <a class="uri" href="mutation" title="wikilink">mutation</a> in the <a href="ASPA_gene" title="wikilink">ASPA gene</a> (on <a href="chromosome_17" title="wikilink">chromosome 17</a>) that leads to a deficiency in the <a class="uri" href="enzyme" title="wikilink">enzyme</a> <a href="aminoacylase_2" title="wikilink">aminoacylase 2</a>. <a href="Aminoacylase_2" title="wikilink">Aminoacylase 2</a> is known for the fact that it can <a href="protein_hydrolysis" title="wikilink">hydrolyze</a> <a class="uri" href="N-acetylaspartate" title="wikilink">N-acetylaspartate</a> while <a href="aminoacylase_1" title="wikilink">aminoacylase 1</a> cannot.<a class="footnoteRef" href="#fn37" id="fnref37"><sup>37</sup></a></p>
<h2 id="industrial-relevance">Industrial relevance</h2>

<p>Aminoacylases have been used for the production of L-<a href="amino_acids" title="wikilink">amino acids</a> in <a href="chemical_industry" title="wikilink">industrial settings</a> since the late 1950s.<a class="footnoteRef" href="#fn38" id="fnref38"><sup>38</sup></a> Since aminoacylases are <a href="enzyme_substrate" title="wikilink">substrate</a> specific for <a href="amino_acid" title="wikilink">N-acyl-L-amino acids</a> and not <a href="amino_acid" title="wikilink">N-acyl-D-amino acids</a>, aminoacylases can be used to reliably take a <a href="Chemical_Mixture" title="wikilink">mixture</a> of these two <a href="Chemical_reactant" title="wikilink">reactants</a> and only convert the L <a class="uri" href="enantiomers" title="wikilink">enantiomers</a> into <a href="chemical_product" title="wikilink">products</a> - which can then be isolated by <a href="amino_acid_solubility" title="wikilink">solubility</a> from the unreacted <a href="amino_acid" title="wikilink">N-acyl-D-amino acids</a>.<a class="footnoteRef" href="#fn39" id="fnref39"><sup>39</sup></a><a class="footnoteRef" href="#fn40" id="fnref40"><sup>40</sup></a> While this <a href="Chemical_process" title="wikilink">process</a> was done in a <a href="batch_reactor" title="wikilink">batch reactor</a> for many years, a faster and less wasteful <a href="chemical_process" title="wikilink">process</a> was developed in the late 1970s that placed aminoacylases in a <a href="column_chromatography" title="wikilink">column</a> that N-<a class="uri" href="acyl" title="wikilink">acyl</a>-<a href="amino_acids" title="wikilink">amino acids</a> were then continuously washed through.<a class="footnoteRef" href="#fn41" id="fnref41"><sup>41</sup></a><a class="footnoteRef" href="#fn42" id="fnref42"><sup>42</sup></a> This <a href="chemical_process" title="wikilink">process</a> is still used in <a href="Chemical_industry" title="wikilink">industrial settings</a> today to convert <a href="amino_acid" title="wikilink">N-acyl-amino acids</a> to <a href="amino_acids" title="wikilink">amino acids</a> in an <a href="enantiomer" title="wikilink">enantiomerically</a> specific way. </p>
<h2 id="evolution">Evolution</h2>

<p>Many <a href="scientific_studies" title="wikilink">scientific studies</a> throughout the past half <a class="uri" href="century" title="wikilink">century</a> have used porcine aminoacylase as their model aminoacylase <a class="uri" href="enzyme" title="wikilink">enzyme</a>.<a class="footnoteRef" href="#fn43" id="fnref43"><sup>43</sup></a> The <a href="amino_acid" title="wikilink">amino acid</a> sequence and <a href="primary_structure" title="wikilink">primary structure</a> of porcine aminoacylase have been determined.<a class="footnoteRef" href="#fn44" id="fnref44"><sup>44</sup></a> Porcine <a href="aminoacylase_1" title="wikilink">aminoacylase 1</a> is composed of two identical <a class="uri" href="heterodimeric" title="wikilink">heterodimeric</a> subunits each consisting of 406 amino acids, with <a class="uri" href="acetylalanine" title="wikilink">acetylalanine</a> at the N-terminus of each.<a class="footnoteRef" href="#fn45" id="fnref45"><sup>45</sup></a> <a href="pig" title="wikilink">Porcine</a> aminoacylase differs from <a class="uri" href="human" title="wikilink">human</a> aminoacylase in <a href="Protein_structure_prediction" title="wikilink">structure</a> but replicates its function.<a class="footnoteRef" href="#fn46" id="fnref46"><sup>46</sup></a><a class="footnoteRef" href="#fn47" id="fnref47"><sup>47</sup></a><a class="footnoteRef" href="#fn48" id="fnref48"><sup>48</sup></a><a class="footnoteRef" href="#fn49" id="fnref49"><sup>49</sup></a> It can be inferred from this data that these two <a class="uri" href="enzymes" title="wikilink">enzymes</a> evolved from a common ancestral <a class="uri" href="protein" title="wikilink">protein</a>, retaining function but diverging in <a href="protein_structure" title="wikilink">structure</a> over time.<a class="footnoteRef" href="#fn50" id="fnref50"><sup>50</sup></a><a class="footnoteRef" href="#fn51" id="fnref51"><sup>51</sup></a> </p>
<h2 id="references">References</h2>

<p>"</p>

<p><a href="Category:EC_3.5.1" title="wikilink">Category:EC 3.5.1</a> <a href="Category:Enzymes_of_known_structure" title="wikilink">Category:Enzymes of known structure</a></p>
<section class="footnotes">
<hr/>
<ol>
<li id="fn1"></li>
<li id="fn2"></li>
<li id="fn3"></li>
<li id="fn4"></li>
<li id="fn5"></li>
<li id="fn6"></li>
<li id="fn7"></li>
<li id="fn8"></li>
<li id="fn9"></li>
<li id="fn10"></li>
<li id="fn11"></li>
<li id="fn12"></li>
<li id="fn13"></li>
<li id="fn14"></li>
<li id="fn15"></li>
<li id="fn16"></li>
<li id="fn17"></li>
<li id="fn18"></li>
<li id="fn19"></li>
<li id="fn20"></li>
<li id="fn21"></li>
<li id="fn22"></li>
<li id="fn23"></li>
<li id="fn24"></li>
<li id="fn25"></li>
<li id="fn26"></li>
<li id="fn27"></li>
<li id="fn28"></li>
<li id="fn29"></li>
<li id="fn30"></li>
<li id="fn31"></li>
<li id="fn32"></li>
<li id="fn33"></li>
<li id="fn34"></li>
<li id="fn35"></li>
<li id="fn36"></li>
<li id="fn37"></li>
<li id="fn38"></li>
<li id="fn39"></li>
<li id="fn40"></li>
<li id="fn41"></li>
<li id="fn42"></li>
<li id="fn43"></li>
<li id="fn44"></li>
<li id="fn45"></li>
<li id="fn46"></li>
<li id="fn47"></li>
<li id="fn48"></li>
<li id="fn49"></li>
<li id="fn50"></li>
<li id="fn51"></li>
</ol>
</section>
</body>
</html>
