<html lang="en">
<head>
<meta charset="utf-8"/>
<title offset="1864">Aptamer</title>
<script src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_SVG.js" type="text/javascript">
</script>
</head>
<body>
<h1>Aptamer</h1>
<hr/>

<p> <strong>Aptamers</strong> (from the Latin <em>aptus</em> - fit, and Greek <em>meros</em> - part) are <a class="uri" href="oligonucleotide" title="wikilink">oligonucleotide</a> or <a class="uri" href="peptide" title="wikilink">peptide</a> molecules that bind to a specific target molecule. Aptamers are usually created by selecting them from a large random <a href="Sequencing" title="wikilink">sequence</a> pool, but natural aptamers also exist in <a href="riboswitch" title="wikilink">riboswitches</a>. Aptamers can be used for both basic research and clinical purposes as macromolecular drugs. Aptamers can be combined with <a href="ribozyme" title="wikilink">ribozymes</a> to self-cleave in the presence of their target molecule. These compound molecules have additional research, industrial and clinical applications.</p>

<p>More specifically, aptamers can be classified as:</p>
<ul>
<li><a href="Deoxyribonucleic_acid" title="wikilink">DNA</a> or <a href="ribonucleic_acid" title="wikilink">RNA</a> or <a href="Nucleic_acid_analogues" title="wikilink">XNA</a> aptamers. They consist of (usually short) strands of oligonucleotides.</li>
<li><a class="uri" href="Peptide" title="wikilink">Peptide</a> aptamers. They consist of a short variable peptide domain, attached at both ends to a protein scaffold.</li>
</ul>
<h2 id="nucleic-acid-aptamers">Nucleic Acid aptamers</h2>

<p>Nucleic acid aptamers are <a href="nucleic_acid" title="wikilink">nucleic acid</a> species that have been engineered through repeated rounds of <em>in vitro <a href="Selection_(biology)" title="wikilink">selection</a></em> or equivalently, SELEX (<a href="systematic_evolution_of_ligands_by_exponential_enrichment" title="wikilink">systematic evolution of ligands by exponential enrichment</a>) to bind to various molecular targets such as small molecules, proteins, nucleic acids, and even cells, tissues and organisms. Aptamers are useful in <a href="biotechnology" title="wikilink">biotechnological</a> and therapeutic applications as they offer molecular recognition properties that rival that of the commonly used biomolecule, <a href="antibody" title="wikilink">antibodies</a>. In addition to their discriminate recognition, aptamers offer advantages over antibodies as they can be engineered completely in a test tube, are readily produced by chemical synthesis, possess desirable storage properties, and elicit little or no <a class="uri" href="immunogenicity" title="wikilink">immunogenicity</a> in therapeutic applications.</p>

<p>In 1990, two labs independently developed the technique of selection: the Gold lab, using the term SELEX for their process of selecting RNA <a href="ligand_(biochemistry)" title="wikilink">ligands</a> against T4 <a href="DNA_polymerase" title="wikilink">DNA polymerase</a>; and the Szostak lab, coining the term <em>in vitro selection</em>, selecting RNA ligands against various organic dyes. The Szostak lab also coined the term aptamer (from the Latin, <em>apto</em>, meaning ‘to fit’) for these nucleic acid-based ligands. Two years later, the Szostak lab and <a href="Gilead_Sciences" title="wikilink">Gilead Sciences</a>, independent of one another, used <em>in vitro selection</em> schemes to evolve single stranded DNA ligands for organic dyes and human coagulant, thrombin (see <a href="Anti-thrombin_aptamers" title="wikilink">Anti-thrombin aptamers</a>), respectively. There does not appear to be any systematic differences between RNA and DNA aptamers, save the greater intrinsic chemical stability of DNA.</p>

<p>Interestingly enough, the notion of selection <em>in vitro</em> was actually preceded twenty-plus years prior when <a href="Sol_Spiegelman" title="wikilink">Sol Spiegelman</a> used a <a href="Bacteriophage_Qβ" title="wikilink">Qbeta</a> replication system as a way to evolve a self-replicating molecule.<a class="footnoteRef" href="#fn1" id="fnref1"><sup>1</sup></a> In addition, a year before the publishing of <em>in vitro selection</em> and SELEX, <a href="Gerald_Joyce" title="wikilink">Gerald Joyce</a> used a system that he termed ‘directed evolution’ to alter the cleavage activity of a ribozyme.</p>

<p>Since the discovery of aptamers, many researchers have used aptamer selection as a means for application and discovery. In 2001, the process of <em>in vitro selection</em> was automated<a class="footnoteRef" href="#fn2" id="fnref2"><sup>2</sup></a><a class="footnoteRef" href="#fn3" id="fnref3"><sup>3</sup></a><a class="footnoteRef" href="#fn4" id="fnref4"><sup>4</sup></a> by J. Colin Cox in the Ellington lab at the <a href="University_of_Texas_at_Austin" title="wikilink">University of Texas at Austin</a>, reducing the duration of a selection experiment from six weeks to three days.</p>

<p>While the process of artificial engineering of nucleic acid ligands is highly interesting to biology and biotechnology, the notion of aptamers in the natural world had yet to be uncovered until 2002 when two groups led by <a href="Ronald_Breaker" title="wikilink">Ronald Breaker</a> and Evgeny Nudler discovered a nucleic acid-based genetic regulatory element (which was named <a class="uri" href="riboswitch" title="wikilink">riboswitch</a>) that possesses similar molecular recognition properties to the artificially made aptamers. In addition to the discovery of a new mode of genetic regulation, this adds further credence to the notion of an ‘<a href="RNA_world_hypothesis" title="wikilink">RNA World</a>,’ a postulated stage in time in the origins of life on Earth.</p>

<p>Both DNA and RNA aptamers show robust binding affinities for various targets.<a class="footnoteRef" href="#fn5" id="fnref5"><sup>5</sup></a><a class="footnoteRef" href="#fn6" id="fnref6"><sup>6</sup></a><a class="footnoteRef" href="#fn7" id="fnref7"><sup>7</sup></a> DNA and RNA aptamers have been selected for the same target. These targets include <a class="uri" href="lysozyme" title="wikilink">lysozyme</a>,<a class="footnoteRef" href="#fn8" id="fnref8"><sup>8</sup></a> <a class="uri" href="thrombin" title="wikilink">thrombin</a>,<a class="footnoteRef" href="#fn9" id="fnref9"><sup>9</sup></a> human immunodeficiency virus trans-acting responsive element (HIV TAR),<a class="footnoteRef" href="#fn10" id="fnref10"><sup>10</sup></a> <a class="uri" href="hemin" title="wikilink">hemin</a>,<a class="footnoteRef" href="#fn11" id="fnref11"><sup>11</sup></a> <a href="interferon_γ" title="wikilink">interferon γ</a>,<a class="footnoteRef" href="#fn12" id="fnref12"><sup>12</sup></a> <a href="vascular_endothelial_growth_factor" title="wikilink">vascular endothelial growth factor</a> (VEGF),<a class="footnoteRef" href="#fn13" id="fnref13"><sup>13</sup></a> <a href="prostate_specific_antigen" title="wikilink">prostate specific antigen</a> (PSA),<a class="footnoteRef" href="#fn14" id="fnref14"><sup>14</sup></a><a class="footnoteRef" href="#fn15" id="fnref15"><sup>15</sup></a> <a class="uri" href="dopamine" title="wikilink">dopamine</a>,<a class="footnoteRef" href="#fn16" id="fnref16"><sup>16</sup></a> and the non-classical oncogene, heat shock factor 1 (HSF1).<a class="footnoteRef" href="#fn17" id="fnref17"><sup>17</sup></a> In the case of lysozyme, HIV TAR, VEGF and dopamine the DNA aptamer is the analog of the RNA aptamer, with thymine replacing uracil. The hemin, thrombin, and interferon γ, DNA and RNA aptamers were selected through independent selections and have unique sequences. Considering that not all DNA analogs of RNA aptamers show functionality the correlation between DNA and RNA sequence and their structure and function requires further investigation.</p>

<p>Lately, a concept of <a href="smart_aptamers" title="wikilink">smart aptamers</a>, and <a href="smart_ligands" title="wikilink">smart ligands</a> in general, has been introduced. It describes aptamers that are selected with pre-defined equilibrium (

<math display="inline" id="Aptamer:0">
 <semantics>
  <msub>
   <mi>K</mi>
   <mi>d</mi>
  </msub>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <csymbol cd="ambiguous">subscript</csymbol>
    <ci>K</ci>
    <ci>d</ci>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   K_{d}
  </annotation>
 </semantics>
</math>

), rate (

<math display="inline" id="Aptamer:1">
 <semantics>
  <msub>
   <mi>k</mi>
   <mrow>
    <mi>o</mi>
    <mi>f</mi>
    <mi>f</mi>
   </mrow>
  </msub>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <csymbol cd="ambiguous">subscript</csymbol>
    <ci>k</ci>
    <apply>
     <times></times>
     <ci>o</ci>
     <ci>f</ci>
     <ci>f</ci>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   k_{off}
  </annotation>
 </semantics>
</math>

, 

<math display="inline" id="Aptamer:2">
 <semantics>
  <msub>
   <mi>k</mi>
   <mrow>
    <mi>o</mi>
    <mi>n</mi>
   </mrow>
  </msub>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <csymbol cd="ambiguous">subscript</csymbol>
    <ci>k</ci>
    <apply>
     <times></times>
     <ci>o</ci>
     <ci>n</ci>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   k_{on}
  </annotation>
 </semantics>
</math>

) constants and thermodynamic (ΔH, ΔS) parameters of aptamer-target interaction. <a href="Kinetic_capillary_electrophoresis" title="wikilink">Kinetic capillary electrophoresis</a> is the technology used for the selection of smart aptamers. It obtains aptamers in a few rounds of selection.</p>

<p>Recent developments in aptamer-based therapeutics have been rewarded in the form of the first aptamer-based drug approved by the U.S. <a href="Food_and_Drug_Administration" title="wikilink">Food and Drug Administration</a> (FDA) in treatment for <a href="age-related_macular_degeneration" title="wikilink">age-related macular degeneration</a> (AMD), called <a class="uri" href="Macugen" title="wikilink">Macugen</a> offered by <a href="OSI_Pharmaceuticals" title="wikilink">OSI Pharmaceuticals</a>. In addition, the company NeoVentures Biotechnology Inc. (http://www.neoventures.ca) has successfully commercialized the first aptamer based diagnostic platform for analysis of mycotoxins in grain. Many contract companies develop aptamers and aptabodies to replace antibodies in research, diagnostic platforms, drug discovery, and therapeutics.</p>

<p>Non-modified aptamers are cleared rapidly from the bloodstream, with a half-life of minutes to hours, mainly due to <a class="uri" href="nuclease" title="wikilink">nuclease</a> degradation and clearance from the body by the kidneys, a result of the aptamer's inherently low molecular weight. Unmodified aptamer applications currently focus on treating transient conditions such as blood clotting, or treating organs such as the eye where local delivery is possible. This rapid clearance can be an advantage in applications such as <em>in vivo</em> diagnostic imaging. An example is a tenascin-binding aptamer under development by <a href="Schering_AG" title="wikilink">Schering AG</a> for cancer imaging. Several modifications, such as 2'-fluorine-substituted <a class="uri" href="pyrimidines" title="wikilink">pyrimidines</a>, <a href="polyethylene_glycol" title="wikilink">polyethylene glycol</a> (PEG) linkage, etc. (both of which are used in Macugen, an FDA-approved aptamer) are available to scientists with which to increase the serum <a class="uri" href="half-life" title="wikilink">half-life</a> of aptamers easily to the day or even week time scale.</p>

<p>Another approach to increase the nuclease resistance of aptamers is to develop <a href="Spiegelmer" title="wikilink">Spiegelmers</a>, which are composed entirely of an unnatural L-ribonucleic acid backbone. A spiegelmer of the same sequence has the same binding properties of the corresponding RNA aptamer, except it binds to the mirror image of its target molecule.</p>

<p>In addition to the development of aptamer-based therapeutics, many researchers such as the Ellington lab have been developing diagnostic techniques for aptamer based plasma protein profiling called <a href="aptamer_plasma_proteomics" title="wikilink">aptamer plasma proteomics</a>. This technology will enable future multi-biomarker protein measurements that can aid diagnostic distinction of disease versus healthy states.</p>

<p>Furthermore, the Hirao lab applied a genetic alphabet expansion using an unnatural base pair<a class="footnoteRef" href="#fn18" id="fnref18"><sup>18</sup></a><a class="footnoteRef" href="#fn19" id="fnref19"><sup>19</sup></a> to SELEX and achieved the generation of high affinity DNA aptamers.<a class="footnoteRef" href="#fn20" id="fnref20"><sup>20</sup></a> Only few hydrophobic unnatural base as a fifth base significantly augment the aptamer affinity to target proteins.</p>

<p>As a resource for all <em>in vitro selection</em> and SELEX experiments, the Ellington lab has developed the <a href="Aptamer_Database" title="wikilink">Aptamer Database</a> cataloging all published experiments. This is found at <a class="uri" href="http://aptamer.icmb.utexas.edu/">http://aptamer.icmb.utexas.edu/</a>.</p>
<h2 id="peptide-aptamers">Peptide aptamers</h2>

<p>Peptide aptamers are proteins that are designed to interfere with other <a href="protein-protein_interaction" title="wikilink">protein interactions</a> inside cells. They consist of a variable peptide loop attached at both ends to a protein scaffold. This double structural constraint greatly increases the binding affinity of the peptide aptamer to levels comparable to an antibody's (nanomolar range).</p>

<p>The variable loop length is typically composed of ten to twenty <a href="amino_acid" title="wikilink">amino acids</a>, and the scaffold may be any protein which has good <a class="uri" href="solubility" title="wikilink">solubility</a> and <a class="uri" href="compacity" title="wikilink">compacity</a> properties. Currently, the <a href="bacterial_protein" title="wikilink">bacterial protein</a> <a class="uri" href="Thioredoxin" title="wikilink">Thioredoxin</a>-A is the most used scaffold protein, the variable loop being inserted within the reducing active site, which is a -Cys-Gly-Pro-Cys- loop in the wild protein, the two <a href="Cysteine" title="wikilink">Cysteines</a> lateral chains being able to form a disulfide bridge.</p>

<p>Peptide aptamer selection can be made using different systems, but the most used is currently the <a class="uri" href="yeast" title="wikilink">yeast</a> <a href="two-hybrid_system" title="wikilink">two-hybrid system</a>.</p>

<p>Selection of Ligand Regulated Peptide Aptamers (LiRPAs) has been demonstrated. By displaying 7 amino acid peptides from a novel <a href="scaffold_protein" title="wikilink">scaffold protein</a> based on the <a href="protein_trimer" title="wikilink">trimeric</a> FKBP-rapamycin-FRB structure, interaction between the randomized peptide and target molecule can be controlled by the small molecule <a class="uri" href="Rapamycin" title="wikilink">Rapamycin</a> or non-immunosuppressive analogs.</p>

<p>Peptide aptamer can also be selected from combinatorial peptide libraries constructed by <a href="phage_display" title="wikilink">phage display</a> and other surface display technologies such as <a href="mRNA_display" title="wikilink">mRNA display</a>, <a href="ribosome_display" title="wikilink">ribosome display</a>, <a href="bacterial_display" title="wikilink">bacterial display</a> and <a href="yeast_display" title="wikilink">yeast display</a>. These experimental procedures are also known as <a href="biopanning" title="wikilink">biopannings</a>. Among peptides obtained from biopannings, <a href="mimotope" title="wikilink">mimotopes</a> can be considered as a kind of peptide aptamers. All the peptides panned from combinatorial peptide libraries have been stored in a special database with the name <a class="uri" href="MimoDB" title="wikilink">MimoDB</a>,<a class="footnoteRef" href="#fn21" id="fnref21"><sup>21</sup></a> which is freely available at <a class="uri" href="http://immunet.cn/mimodb">http://immunet.cn/mimodb</a>.</p>
<h2 id="affimers">Affimers</h2>

<p><a class="uri" href="Affimers" title="wikilink">Affimers</a>, an evolution of peptide aptamers, are small, highly stable <a class="uri" href="proteins" title="wikilink">proteins</a> engineered to display <a class="uri" href="peptide" title="wikilink">peptide</a> loops which provide a high affinity binding surface for a specific target protein. They are proteins of low molecular weight, 12–14 kDa,<a class="footnoteRef" href="#fn22" id="fnref22"><sup>22</sup></a> derived from the <a href="cysteine_protease" title="wikilink">cysteine protease</a> inhibitor family of <a class="uri" href="cystatins" title="wikilink">cystatins</a>.<a class="footnoteRef" href="#fn23" id="fnref23"><sup>23</sup></a><a class="footnoteRef" href="#fn24" id="fnref24"><sup>24</sup></a><a class="footnoteRef" href="#fn25" id="fnref25"><sup>25</sup></a><a class="footnoteRef" href="#fn26" id="fnref26"><sup>26</sup></a></p>

<p>Affimers are composed of a scaffold, which is a stable protein based on the cystatin protein fold. They display two peptide loops and an N-terminal sequence that can be randomised to bind different target proteins with high affinity and specificity similar to antibodies. Stabilisation of the peptide upon the protein scaffold constrains the possible conformations which the peptide may take, thus increasing the binding affinity and specificity compared to libraries of free peptides.</p>

<p>Affimers were developed initially at the <a href="MRC_Cancer_Cell_Unit" title="wikilink">MRC Cancer Cell Unit</a> in Cambridge then across two laboratories at the <a href="University_of_Leeds" title="wikilink">University of Leeds</a>.<a class="footnoteRef" href="#fn27" id="fnref27"><sup>27</sup></a><a class="footnoteRef" href="#fn28" id="fnref28"><sup>28</sup></a><a class="footnoteRef" href="#fn29" id="fnref29"><sup>29</sup></a><a class="footnoteRef" href="#fn30" id="fnref30"><sup>30</sup></a> Affimers have been commercialised and developed by Avacta Life Sciences, who are developing them as reagents for research and therapeutic applications.</p>
<h2 id="aptabid">AptaBiD</h2>

<p>AptaBiD or Aptamer-Facilitated Biomarker Discovery is a technology for <a class="uri" href="biomarker" title="wikilink">biomarker</a> discovery.<a class="footnoteRef" href="#fn31" id="fnref31"><sup>31</sup></a> AptaBiD is based on multi-round generation of an aptamer or a pool of aptamers for differential molecular targets on the cells which facilitates exponential detection of biomarkers. It involves three major stages: (i) differential multi-round selection of aptamers for biomarker of target cells; (ii) aptamer-based isolation of biomarkers from target cells; and (iii) <a href="mass_spectrometry" title="wikilink">mass spectrometry</a> identification of biomarkers. The important feature of the AptaBiD technology is that it produces synthetic affinity probes (aptamers) simultaneously with biomarker discovery. In AptaBiD, aptamers are developed for cell surface biomarkers in their native state and conformation. In addition to facilitating biomarker identification, such aptamers can be directly used for cell isolation, cell visualization, and tracking cells <em>in vivo</em>. They can also be used to modulate activities of cell receptors and deliver different agents (e.g., <a href="small_interfering_RNA" title="wikilink">siRNA</a> and drugs) into the cells.</p>
<h2 id="see-also">See also</h2>
<div style="-moz-column-count:3; column-count:3;">
<ul>
<li><a href="Anti-thrombin_aptamers" title="wikilink">Anti-thrombin aptamers</a></li>
<li><a href="Synthetic_antibody" title="wikilink">Synthetic antibody</a></li>
</ul>
</div>
<h2 id="references">References</h2>
<h2 id="further-reading">Further reading</h2>
<ul>
<li></li>
<li></li>
<li></li>
<li></li>
<li></li>
<li></li>
<li></li>
<li></li>
<li></li>
<li></li>
</ul>
<h2 id="external-links">External links</h2>
<ul>
<li><a href="http://www.avactalifesciences.com">Avacta Life Sciences</a></li>
<li><a href="http://aptamergroup.co.uk">Aptamer Group - The world's premier team in Aptamer Design, Therapeutics, Biomarker Discovery, and Diagnostics Formatting</a></li>
<li><a href="http://nanopore.weebly.com">Aptamer biosensor group</a></li>
<li><a href="http://aptamer.freebase.com">Aptamer - Freebase</a></li>
<li><a href="http://aptamerbase.semanticscience.org">Aptamer Base</a></li>
<li><a href="http://immunet.cn/mimodb">The MimoDB database</a></li>
</ul>

<p>"</p>

<p><a href="Category:Nucleic_acids" title="wikilink">Category:Nucleic acids</a> <a class="uri" href="Category:Peptides" title="wikilink">Category:Peptides</a> <a class="uri" href="Category:Biotechnology" title="wikilink">Category:Biotechnology</a> <a href="Category:Medical_research" title="wikilink">Category:Medical research</a></p>
<section class="footnotes">
<hr/>
<ol>
<li id="fn1"><a href="#fnref1">↩</a></li>
<li id="fn2"><a href="#fnref2">↩</a></li>
<li id="fn3"><a href="#fnref3">↩</a></li>
<li id="fn4"><a href="#fnref4">↩</a></li>
<li id="fn5"><a href="#fnref5">↩</a></li>
<li id="fn6"><a href="#fnref6">↩</a></li>
<li id="fn7"><a href="#fnref7">↩</a></li>
<li id="fn8"><a href="#fnref8">↩</a></li>
<li id="fn9"><a href="#fnref9">↩</a></li>
<li id="fn10"><a href="#fnref10">↩</a></li>
<li id="fn11"><a href="#fnref11">↩</a></li>
<li id="fn12"><a href="#fnref12">↩</a></li>
<li id="fn13"><a href="#fnref13">↩</a></li>
<li id="fn14"><a href="#fnref14">↩</a></li>
<li id="fn15"><a href="#fnref15">↩</a></li>
<li id="fn16"><a href="#fnref16">↩</a></li>
<li id="fn17"><a href="#fnref17">↩</a></li>
<li id="fn18"><a href="#fnref18">↩</a></li>
<li id="fn19"><a href="#fnref19">↩</a></li>
<li id="fn20"><a href="#fnref20">↩</a></li>
<li id="fn21"><a href="#fnref21">↩</a></li>
<li id="fn22"><a href="#fnref22">↩</a></li>
<li id="fn23"><a href="#fnref23">↩</a></li>
<li id="fn24"><a href="#fnref24">↩</a></li>
<li id="fn25"><a href="#fnref25">↩</a></li>
<li id="fn26"><a href="#fnref26">↩</a></li>
<li id="fn27"></li>
<li id="fn28"></li>
<li id="fn29"></li>
<li id="fn30"></li>
<li id="fn31"><a href="#fnref31">↩</a></li>
</ol>
</section>
</body>
</html>
