<html lang="en">
<head>
<meta charset="utf-8"/>
<title offset="1499">Biclustering</title>
<script src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_SVG.js" type="text/javascript">
</script>
</head>
<body>
<h1>Biclustering</h1>
<hr/>

<p><strong>Biclustering</strong>, <strong>block clustering</strong> ,<a class="footnoteRef" href="#fn1" id="fnref1"><sup>1</sup></a> <strong>co-clustering</strong>, or <strong>two-<a href="mode_(statistics)" title="wikilink">mode</a> clustering</strong> <a class="footnoteRef" href="#fn2" id="fnref2"><sup>2</sup></a> <ref></ref></p>

<p></p>

<p> is a <a href="data_mining" title="wikilink">data mining</a> technique which allows simultaneous <a href="cluster_analysis" title="wikilink">clustering</a> of the rows and columns of a <a href="matrix_(mathematics)" title="wikilink">matrix</a>. The term was first introduced by Mirkin,<a class="footnoteRef" href="#fn3" id="fnref3"><sup>3</sup></a> although the technique was originally introduced much earlier<a class="footnoteRef" href="#fn4" id="fnref4"><sup>4</sup></a> (i.e., by J.A. Hartigan<ref></ref></p>

<p></p>

<p>).</p>

<p>Given a set of 

<math display="inline" id="Biclustering:0">
 <semantics>
  <mi>m</mi>
  <annotation-xml encoding="MathML-Content">
   <ci>m</ci>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   m
  </annotation>
 </semantics>
</math>

 rows in 

<math display="inline" id="Biclustering:1">
 <semantics>
  <mi>n</mi>
  <annotation-xml encoding="MathML-Content">
   <ci>n</ci>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   n
  </annotation>
 </semantics>
</math>

 columns (i.e., an 

<math display="inline" id="Biclustering:2">
 <semantics>
  <mrow>
   <mi>m</mi>
   <mo>×</mo>
   <mi>n</mi>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <times></times>
    <ci>m</ci>
    <ci>n</ci>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   m\times n
  </annotation>
 </semantics>
</math>

 matrix), the biclustering algorithm generates biclusters - a subset of rows which exhibit similar behavior across a subset of columns, or vice versa.</p>
<h2 id="development">Development</h2>

<p>Biclustering was originally introduced by J.A.Hartigan in 1972.<a class="footnoteRef" href="#fn5" id="fnref5"><sup>5</sup></a> The term biclustering was later used by Mirkin. This algorithm was not generalized until 2000 when Y.Cheng and G.M.Church proposed a biclustering algorithm based on variance and applied it to biological gene expression data.<a class="footnoteRef" href="#fn6" id="fnref6"><sup>6</sup></a> Their paper is still the most important literature in the gene expression biclustering field.</p>

<p>In 2001 and 2003, I.S.Dhillon put forward two algorithms applying biclustering to files and words. One version was based on bipartite spectral graph partitioning.<a class="footnoteRef" href="#fn7" id="fnref7"><sup>7</sup></a> The other was based on information theory. Dhillon assumed the loss of mutual information during biclustering was equal to the KL(<a href="Kullback-Leibler_divergence" title="wikilink">Kullback-Leibler)-distance</a> between P and Q. P means the distribution of files and feature words before biclustering. Q is the distribution after biclustering. KL-distance is for measuring the difference between two random distributions. KL=0 when the two distributions are the same and KL increases as the difference increases.<a class="footnoteRef" href="#fn8" id="fnref8"><sup>8</sup></a> Thus, the aim of the algorithm was to find the minimum KL-distance between P and Q. In 2004, A.Banerjee used a weighted-Bregman distance instead of KL-distance to design a biclustering algorithm which was suitable for any kind of matrix, unlike the KL-distance algorithm.<a class="footnoteRef" href="#fn9" id="fnref9"><sup>9</sup></a></p>

<p>To cluster more than two types of objects, in 2005, Bekkerman expanded the mutual information in Dhillon’s theorem from a single pair into multiple pairs.</p>
<h2 id="complexity">Complexity</h2>

<p>The complexity of the biclustering problem depends on the exact problem formulation, and particularly on the merit function used to evaluate the quality of a given bicluster. However most interesting variants of this problem are <a class="uri" href="NP-complete" title="wikilink">NP-complete</a>. NP-complete have two conditions. In the simple case that there is only element a_(i,j) either 0 or 1 in the binary matrix A, a bicluster is equal to a biclique in the corresponding bipartite graph. The maximum size bicluster is equivalent to maximum edge biclique in bipartite graph. In the complex case, the element in matrix A is used to compute the quality of a given bicluster and solve the more restricted version of the problem.<a class="footnoteRef" href="#fn10" id="fnref10"><sup>10</sup></a> It requires either large <a href="computer" title="wikilink">computational</a> effort or the use of lossy <a class="uri" href="heuristics" title="wikilink">heuristics</a> to short-circuit the calculation.<a class="footnoteRef" href="#fn11" id="fnref11"><sup>11</sup></a></p>
<h2 id="type-of-bicluster">Type of Bicluster</h2>

<p>Different biclustering algorithms have different definitions of bicluster.<a class="footnoteRef" href="#fn12" id="fnref12"><sup>12</sup></a></p>

<p>They are:</p>
<ol>
<li>Bicluster with constant values (a),</li>
<li>Bicluster with constant values on rows (b) or columns (c),</li>
<li>Bicluster with coherent values (d, e).</li>
</ol>

<p><strong>1.Bicluster with constant values</strong></p>

<p>When a biclustering algorithm tries to find a constant bicluster, the normal way for it is to reorder the rows and columns of the matrix so it can group together similar rows/columns and find biclusters with similar values. This method is OK when the data is tidy. But as the data can be noisy most of the times, so it can’t satisfy us. More sophisticated methods should be used. A perfect constant bicluster is a matrix(I,J) where all values a(i,j) are equal to μ. In real data, a(i,j) can be seen as n(i,j) +μ where n(i,j) is the noise. According to Hartigan’s algorithm, by splitting the original data matrix into a set of biclusters. Variance is used to compute constant biclusters. So a perfect bicluster is a matrix with variance zero. Also, in order to prevent the partitioning of the data matrix into biclusters with only one row and one column. Hartigan assumes that there are K biclusters within the data matrix. When the data matrix is partitioned into K biclusters, the algorithm ends.</p>

<p><strong>2.Biclusters with constant values on rows or columns</strong></p>

<p>This kind of biclusters can’t be evaluated just by variance of its values. To finish the identification, the columns and the rows should be normalized at first. There are other algorithms, without normalization step, can find biclusters have rows and columns with different approaches.</p>

<p><strong>3.Biclusters with coherent values</strong></p>

<p>For biclusters with coherent values on rows and columns, an overall improvement over the algorithms for biclusters with constant values on rows or on columns should be considered. That means a sophisticated algorithm is needed. This algorithm may contain analysis of variance between groups, using co-variance between both rows and columns.In Cheng and Churchs’ theorem, a bicluster is defined as a subset of rows and columns with almost the same score.the similarity score is used to measure the coherence of rows and columns.</p>
<table>
<tbody>
<tr class="odd">
<td style="text-align: left;"><table>
<tbody>
<tr class="odd">
<td style="text-align: left;">
<p>a) Bicluster with constant values</p></td>
</tr>
<tr class="even">
<td style="text-align: left;">
<p>2.0</p></td>
</tr>
<tr class="odd">
<td style="text-align: left;">
<p>2.0</p></td>
</tr>
<tr class="even">
<td style="text-align: left;">
<p>2.0</p></td>
</tr>
<tr class="odd">
<td style="text-align: left;">
<p>2.0</p></td>
</tr>
<tr class="even">
<td style="text-align: left;">
<p>2.0</p></td>
</tr>
</tbody>
</table></td>
<td style="text-align: left;"><table>
<tbody>
<tr class="odd">
<td style="text-align: left;">
<p>b) Bicluster with constant values on rows</p></td>
</tr>
<tr class="even">
<td style="text-align: left;">
<p>1.0</p></td>
</tr>
<tr class="odd">
<td style="text-align: left;">
<p>2.0</p></td>
</tr>
<tr class="even">
<td style="text-align: left;">
<p>3.0</p></td>
</tr>
<tr class="odd">
<td style="text-align: left;">
<p>4.0</p></td>
</tr>
<tr class="even">
<td style="text-align: left;">
<p>4.0</p></td>
</tr>
</tbody>
</table></td>
<td style="text-align: left;"><table>
<tbody>
<tr class="odd">
<td style="text-align: left;">
<p>c) Bicluster with constant values on columns</p></td>
</tr>
<tr class="even">
<td style="text-align: left;">
<p>1.0</p></td>
</tr>
<tr class="odd">
<td style="text-align: left;">
<p>1.0</p></td>
</tr>
<tr class="even">
<td style="text-align: left;">
<p>1.0</p></td>
</tr>
<tr class="odd">
<td style="text-align: left;">
<p>1.0</p></td>
</tr>
<tr class="even">
<td style="text-align: left;">
<p>1.0</p></td>
</tr>
</tbody>
</table></td>
</tr>
</tbody>
</table>
<table>
<tbody>
<tr class="odd">
<td style="text-align: left;"><table>
<tbody>
<tr class="odd">
<td style="text-align: left;">
<p>d) Bicluster with coherent values (additive)</p></td>
</tr>
<tr class="even">
<td style="text-align: left;">
<p>1.0</p></td>
</tr>
<tr class="odd">
<td style="text-align: left;">
<p>4.0</p></td>
</tr>
<tr class="even">
<td style="text-align: left;">
<p>3.0</p></td>
</tr>
<tr class="odd">
<td style="text-align: left;">
<p>5.0</p></td>
</tr>
<tr class="even">
<td style="text-align: left;">
<p>2.0</p></td>
</tr>
</tbody>
</table></td>
<td style="text-align: left;"><table>
<tbody>
<tr class="odd">
<td style="text-align: left;">
<p>e) Bicluster with coherent values (multiplicative)</p></td>
</tr>
<tr class="even">
<td style="text-align: left;">
<p>1.0</p></td>
</tr>
<tr class="odd">
<td style="text-align: left;">
<p>2.0</p></td>
</tr>
<tr class="even">
<td style="text-align: left;">
<p>3.0</p></td>
</tr>
<tr class="odd">
<td style="text-align: left;">
<p>4.0</p></td>
</tr>
<tr class="even">
<td style="text-align: left;">
<p>5.0</p></td>
</tr>
</tbody>
</table></td>
</tr>
</tbody>
</table>

<p>The relationship between these cluster models and other types of clustering such as <a href="correlation_clustering" title="wikilink">correlation clustering</a> is discussed in.<a class="footnoteRef" href="#fn13" id="fnref13"><sup>13</sup></a></p>
<h2 id="algorithms">Algorithms</h2>

<p>There are many biclustering <a class="uri" href="algorithms" title="wikilink">algorithms</a> developed for <a class="uri" href="bioinformatics" title="wikilink">bioinformatics</a>, including: block clustering, CTWC (Coupled Two-Way Clustering), ITWC (Interrelated Two-Way Clustering), δ-bicluster, δ-pCluster, δ-pattern, FLOC, OPC, Plaid Model, OPSMs (Order-preserving submatrixes), Gibbs, SAMBA (Statistical-Algorithmic Method for Bicluster Analysis),<a class="footnoteRef" href="#fn14" id="fnref14"><sup>14</sup></a> Robust Biclustering Algorithm (RoBA), Crossing Minimization,<a class="footnoteRef" href="#fn15" id="fnref15"><sup>15</sup></a> cMonkey,<a class="footnoteRef" href="#fn16" id="fnref16"><sup>16</sup></a> PRMs, DCC, LEB (Localize and Extract Biclusters), QUBIC (QUalitative BIClustering), BCCA (Bi-Correlation Clustering Algorithm) BIMAX, ISA, SAMBA and FABIA (Factor Analysis for Bicluster Acquisition).<a class="footnoteRef" href="#fn17" id="fnref17"><sup>17</sup></a> Biclustering algorithms have also been proposed and used in other application fields under the names coclustering, bidimensional clustering, and subspace clustering.<a class="footnoteRef" href="#fn18" id="fnref18"><sup>18</sup></a></p>

<p>Given the known importance of discovering local patterns in <a href="time-series_data" title="wikilink">time-series data</a>, recent proposals have addressed the biclustering problem in the specific case of time series <a href="gene_expression" title="wikilink">gene expression</a> data. In this case, the interesting biclusters can be restricted to those with <a class="uri" href="contiguous" title="wikilink">contiguous</a> columns. This restriction leads to a <a href="tractable_problem" title="wikilink">tractable problem</a> and enables the development of efficient exhaustive <a class="uri" href="enumeration" title="wikilink">enumeration</a> algorithms such as CCC-Biclustering <a class="footnoteRef" href="#fn19" id="fnref19"><sup>19</sup></a> and <em>e</em>-CCC-Biclustering.<a class="footnoteRef" href="#fn20" id="fnref20"><sup>20</sup></a> The approximate patterns in CCC-Biclustering algorithms allow a given number of errors, per gene, relatively to an expression profile respresenting the expression pattern in the bicluster. The e-CCC-Biclustering algorithm uses approximate expressions to find and report all maximal CCC-Biclusters by a discretized matrix A and efficient string processing techniques.</p>

<p>These <a href="algorithm" title="wikilink">algorithms</a> ﬁnd and report all maximal biclusters with coherent and contiguous columns with perfect/approximate expression patterns, in time linear/<a class="uri" href="polynomial" title="wikilink">polynomial</a> which is obtained by manipulating a discretized version of original expression matrix in the size of the time series gene expression <a href="matrix_(mathematics)" title="wikilink">matrix</a> using eﬃcient <a href="string_processing" title="wikilink">string processing</a> techniques based on <a href="suffix_tree" title="wikilink">suffix trees</a>. These algorithms are also applied to solve problems and sketch the analysis of computational complexity.</p>

<p>Some recent algorithms have attempted to include additional support for biclustering rectangular matrices in the form of other <a href="datatype" title="wikilink">datatypes</a>, including cMonkey.</p>

<p>There is an ongoing debate about how to judge the results of these methods, as biclustering allows overlap between clusters and some <a class="uri" href="algorithms" title="wikilink">algorithms</a> allow the exclusion of hard-to-reconcile columns/conditions. Not all of the available algorithms are deterministic and the analyst must pay attention to the degree to which results represent stable <a class="uri" href="minima" title="wikilink">minima</a>. Because this is an <a href="unsupervised_classification" title="wikilink">unsupervised classification</a> problem, the lack of a <a href="gold_standard_(test)" title="wikilink">gold standard</a> makes it difficult to spot errors in the results. One approach is to utilize multiple biclustering algorithms, with majority or <a class="uri" href="super-majority" title="wikilink">super-majority</a> voting amongst them deciding the best result. Another way is to analyse the quality of shifting and scaling patterns in biclusters.<a class="footnoteRef" href="#fn21" id="fnref21"><sup>21</sup></a> Biclustering has been used in the domain of text mining (or classification) where it is popularly known as co-clustering .<a class="footnoteRef" href="#fn22" id="fnref22"><sup>22</sup></a> Text corpora are represented in a <a href="vector_(mathematics_and_physics)" title="wikilink">vectorial</a> form as a <a href="matrix_(mathematics)" title="wikilink">matrix</a> D whose rows denote the documents and whose columns denote the words in the dictionary. Matrix elements D<sub>ij</sub> denote occurrence of word j in document i. <a class="uri" href="Co-clustering" title="wikilink">Co-clustering</a> algorithms are then applied to discover blocks in D that correspond to a group of documents (rows) characterized by a group of words(columns).</p>

<p>Test clustering can solve the high-dimensional sparse problem, which means clustering text and words at the same time. When clustering text, we need to think about not only the words information, but also the information of words clusters that was composed by words. Then according to similarity of feature words in the text, will eventually cluster the feature words. This is called co-clustering. There are two advantages of co-clustering: one is clustering the test based on words clusters can extremely decrease the dimension of clustering, it can also appropriate to measure the distance between the tests. Second is mining more useful information and can get the corresponding information in test clusters and words clusters. This corresponding information can be used to describe the type of texts and words, at the same time, the result of words clustering can be also used to text mining and information retrival.</p>

<p>Several approaches have been proposed based on the information contents of the resulting blocks: matrix-based approaches such as <a href="singular_value_decomposition" title="wikilink">SVD</a> and BVD, and graph-based approaches. <a class="uri" href="Information-theoretic" title="wikilink">Information-theoretic</a> algorithms <a href="iterative" title="wikilink">iteratively</a> assign each row to a cluster of documents and each column to a cluster of words such that the mutual information is maximized. Matrix-based methods focus on the decomposition of matrices into blocks such that the error between the original matrix and the regenerated matrices from the decomposition is minimized. Graph-based methods tend to minimize the cuts between the clusters. Given two groups of documents d<sub>1</sub> and d<sub>2</sub>, the number of cuts can be measured as the number of words that occur in documents of groups d<sub>1</sub> and d<sub>2</sub>.</p>

<p>More recently (Bisson and Hussain)<a class="footnoteRef" href="#fn23" id="fnref23"><sup>23</sup></a> have proposed a new approach of using the similarity between words and the similarity between documents to <a href="co-clustering" title="wikilink">co-cluster</a> the matrix. Their method (known as <strong>χ-Sim</strong>, for cross similarity) is based on finding document-document similarity and word-word similarity, and then using classical clustering methods such as <a href="hierarchical_clustering" title="wikilink">hierarchical clustering</a>. Instead of explicitly clustering rows and columns alternately, they consider higher-order occurrences of words, inherently taking into account the documents in which they occur. Thus, the similarity between two words is calculated based on the documents in which they occur and also the documents in which "similar" words occur. The idea here is that two documents about the same topic do not necessarily use the same set of words to describe it but a subset of the words and other similar words that are characteristic of that topic. This approach of taking higher-order similarities takes the <a href="latent_semantic_analysis" title="wikilink">latent semantic</a> structure of the whole corpus into consideration with the result of generating a better clustering of the documents and words.</p>

<p>In text databases, for a document collection defined by a document by term D matrix (of size m by n, m: number of documents, n: number of terms) the cover-coefficient based clustering methodology<a class="footnoteRef" href="#fn24" id="fnref24"><sup>24</sup></a> yields the same number of clusters both for documents and terms (words) using a double-stage probability experiment. According to the cover coefficient concept number of clusters can also be roughly estimated by the following formula 

<math display="inline" id="Biclustering:3">
 <semantics>
  <mrow>
   <mrow>
    <mo stretchy="false">(</mo>
    <mrow>
     <mi>m</mi>
     <mo>×</mo>
     <mi>n</mi>
    </mrow>
    <mo stretchy="false">)</mo>
   </mrow>
   <mo>/</mo>
   <mi>t</mi>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <divide></divide>
    <apply>
     <times></times>
     <ci>m</ci>
     <ci>n</ci>
    </apply>
    <ci>t</ci>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   (m\times n)/t
  </annotation>
 </semantics>
</math>

 where t is the number of non-zero entries in D. Note that in D each row and each column must contain at least one non-zero element.</p>

<p>In contrast to other approaches, FABIA is a multiplicative model that assumes realistic <a href="non-Gaussianity" title="wikilink">non-Gaussian</a> signal distributions with <a href="heavy_tails" title="wikilink">heavy tails</a>. FABIA utilizes well understood model selection techniques like variational approaches and applies the <a href="Bayesian_probability" title="wikilink">Bayesian</a> framework. The generative framework allows FABIA to determine the <a href="information_content" title="wikilink">information content</a> of each bicluster to separate spurious biclusters from true biclusters.</p>
<h2 id="see-also">See also</h2>
<ul>
<li><a href="Formal_concept_analysis" title="wikilink">Formal concept analysis</a></li>
<li><a class="uri" href="Biclique" title="wikilink">Biclique</a></li>
<li><a href="Galois_connection" title="wikilink">Galois connection</a></li>
</ul>
<h2 id="references">References</h2>
<h3 id="others">Others</h3>
<ul>
<li>A. Tanay. R. Sharan, and R. Shamir, "Biclustering Algorithms: A Survey", In <em>Handbook of Computational Molecular Biology</em>, Edited by Srinivas Aluru, Chapman (2004)</li>
<li></li>
</ul>
<h2 id="external-links">External links</h2>
<ul>
<li><a href="http://www.bioinf.jku.at/software/fabia/fabia.html">FABIA: Factor Analysis for Bicluster Acquisition, an R package</a> —software</li>
</ul>

<p>"</p>

<p><a href="Category:Cluster_analysis" title="wikilink">Category:Cluster analysis</a> <a class="uri" href="Category:Bioinformatics" title="wikilink">Category:Bioinformatics</a></p>
<section class="footnotes">
<hr/>
<ol>
<li id="fn1"><a href="#fnref1">↩</a></li>
<li id="fn2"><a href="#fnref2">↩</a></li>
<li id="fn3"><a href="#fnref3">↩</a></li>
<li id="fn4"></li>
<li id="fn5"><a href="http://amstat.tandfonline.com/doi/abs/10.1080/01621459.1972.10481214#.VBLA6vmSwjA">Hartigan J A. Direct clustering of a data matrix[J</a>. Journal of the american statistical association, 1972, 67(337): 123-129.]<a href="#fnref5">↩</a></li>
<li id="fn6"><a class="uri" href="https://www.cs.princeton.edu/courses/archive/fall03/cs597F/Articles/biclustering_of_expression_data.pdf">https://www.cs.princeton.edu/courses/archive/fall03/cs597F/Articles/biclustering_of_expression_data.pdf</a> Cheng Y, Church G M. Biclustering of expression data[C]//Ismb. 2000, 8: 93-103.<a href="#fnref6">↩</a></li>
<li id="fn7"><a href="http://dl.acm.org/citation.cfm?id=502550">Dhillon I S. Co-clustering documents and words using bipartite spectral graph partitioning[C</a>//Proceedings of the seventh ACM SIGKDD international conference on Knowledge discovery and data mining. ACM, 2001: 269-274.]<a href="#fnref7">↩</a></li>
<li id="fn8"><a href="http://dl.acm.org/citation.cfm?id=956764">Dhillon I S, Mallela S, Modha D S. Information-theoretic co-clustering[C</a>//Proceedings of the ninth ACM SIGKDD international conference on KKluwer Academic Publishersnowledge discovery and data mining. ACM, 2003: 89-98.]<a href="#fnref8">↩</a></li>
<li id="fn9"><a href="http://dl.acm.org/citation.cfm?id=1014111">Banerjee A, Dhillon I, Ghosh J, et al. A generalized maximum entropy approach to bregman co-clustering and matrix approximation[C</a>//Proceedings of the tenth ACM SIGKDD international conference on Knowledge discovery and data mining. ACM, 2004: 509-514.]<a href="#fnref9">↩</a></li>
<li id="fn10"><a href="http://www.sciencedirect.com/science/article/pii/S0166218X03003330">Peeters R. The maximum edge biclique problem is NP-complete[J</a>. Discrete Applied Mathematics, 2003, 131(3): 651-654.]<a href="#fnref10">↩</a></li>
<li id="fn11"></li>
<li id="fn12"><a href="#fnref12">↩</a></li>
<li id="fn13"><a href="#fnref13">↩</a></li>
<li id="fn14"><a href="#fnref14">↩</a></li>
<li id="fn15"></li>
<li id="fn16"><a href="#fnref16">↩</a></li>
<li id="fn17"><a href="#fnref17">↩</a></li>
<li id="fn18"></li>
<li id="fn19"><a href="#fnref19">↩</a></li>
<li id="fn20"><a href="#fnref20">↩</a></li>
<li id="fn21"><a href="#fnref21">↩</a></li>
<li id="fn22"><a href="#fnref22">↩</a></li>
<li id="fn23"></li>
<li id="fn24"><a href="#fnref24">↩</a></li>
</ol>
</section>
</body>
</html>
