<html lang="en">
<head>
<meta charset="utf-8"/>
<title offset="685">Binning (Metagenomics)</title>
<script src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_SVG.js" type="text/javascript">
</script>
</head>
<body>
<h1>Binning (Metagenomics)</h1>
<hr/>

<p>In <a class="uri" href="metagenomics" title="wikilink">metagenomics</a>, <strong>binning</strong> is the process of grouping reads or <a href="contig" title="wikilink">contigs</a> and assigning them to <a href="operational_taxonomic_unit" title="wikilink">operational taxonomic units</a>. Binning methods can be based on either compositional features or alignment (similarity), or both.</p>
<h2 id="introduction">Introduction</h2>

<p>Metagenomic samples can contain reads from a huge number of organisms. For example, in a single gram of soil, there can be up to 18000 different types of organisms, each with its own genome.<a class="footnoteRef" href="#fn1" id="fnref1"><sup>1</sup></a> Metagenomic studies sample DNA from the whole community, and make it available as nucleotide sequences of <a href="DNA_sequencing" title="wikilink">certain length</a>. In most cases, the incomplete nature of the obtained sequences makes it hard to assemble individual genes,<a class="footnoteRef" href="#fn2" id="fnref2"><sup>2</sup></a> much less recovering the full <a href="genome" title="wikilink">genomes</a> of each organism. Thus, binning techniques represent a "best effort" to identify reads or <a href="contig" title="wikilink">contigs</a> with certain groups of organisms designated as <a href="operational_taxonomic_unit" title="wikilink">operational taxonomic units</a> (OTUs).<a class="footnoteRef" href="#fn3" id="fnref3"><sup>3</sup></a></p>

<p>The first studies that sampled DNA from multiple organisms used specific genes to assess diversity and origin of each sample.<a class="footnoteRef" href="#fn4" id="fnref4"><sup>4</sup></a><a class="footnoteRef" href="#fn5" id="fnref5"><sup>5</sup></a> These marker genes had been previously sequenced from clonal cultures from known organisms, so, whenever one of such genes appeared in a read or contig from the metagenomic sample that read could be assigned to a known species or to the OTU of that species. The problem with this method was that only a tiny fraction of the sequences carried a marker gene, leaving most of the data unassigned.</p>

<p>Modern binning techniques use both previously available information independent from the sample and intrinsic information present in the sample. Depending on the diversity and complexity of the sample, their degree of success vary: in some cases they can resolve the sequences up to individual species, while in some others the sequences are identified at best with very broad taxonomic groups.</p>
<h2 id="algorithms">Algorithms</h2>

<p>Binning algorithms can employ previous information, and thus act as <a href="Supervised_learning" title="wikilink">supervised classifiers</a>, or they can try to find new groups, those act as <a href="Unsupervised_learning" title="wikilink">unsupervised classifiers</a>. Many, of course, do both. The classifiers exploit the previously known sequences by performing <a href="Sequence_alignment" title="wikilink">alignments</a> against <a href="Biological_databases" title="wikilink">databases</a>, and try to separate sequence based in organism-specific characteristics of the DNA,<a class="footnoteRef" href="#fn6" id="fnref6"><sup>6</sup></a> like <a class="uri" href="GC-content" title="wikilink">GC-content</a>.</p>

<p>Mande et al., (2012) <a class="footnoteRef" href="#fn7" id="fnref7"><sup>7</sup></a> provides an excellent review of the premise, methodologies, advantages, limitations and challenges of various methods available for binning of metagenomic datasets obtained using the shotgun sequencing approach. Some of the prominent binning algorithms are described below.</p>
<h3 id="tetra">TETRA</h3>

<p>TETRA is a statistical classifier that uses tetranucleotide usage patterns in genomic fragments.<a class="footnoteRef" href="#fn8" id="fnref8"><sup>8</sup></a> There are four possible nucleotides in <a class="uri" href="DNA" title="wikilink">DNA</a>, therefore there can be 

<math display="inline" id="Binning_(Metagenomics):0">
 <semantics>
  <mrow>
   <msup>
    <mn>4</mn>
    <mn>4</mn>
   </msup>
   <mo>=</mo>
   <mn>256</mn>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <eq></eq>
    <apply>
     <csymbol cd="ambiguous">superscript</csymbol>
     <cn type="integer">4</cn>
     <cn type="integer">4</cn>
    </apply>
    <cn type="integer">256</cn>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   4^{4}=256
  </annotation>
 </semantics>
</math>

 different fragments of four consecutive nucleotides; these fragments are called tetramers. TETRA works by tabulating the frequencies of each tetramer for a given sequence. From these frequencies <a href="Standard_score" title="wikilink">z-scores</a> are then calculated, which indicate how over- or under-represented the tetramer is in contraposition with what would be expected by looking to individual nucleotide compositions. The z-scores for each tetramer are assembled in a vector, and the vectors corresponding to different sequences are compared pair-wise, to yield a measure of how similar different sequences from the sample are. It is expected that the most similar sequences belong to organisms in the same OTU.</p>
<h3 id="phylopythia">Phylopythia</h3>

<p>Phylopythia is one supervised classifier developed by researchers at IBM labs, and is basically a <a href="support_vector_machine" title="wikilink">support vector machine</a> trained with DNA-kmers from known sequences.<a class="footnoteRef" href="#fn9" id="fnref9"><sup>9</sup></a></p>
<h3 id="sort-items">SOrt-ITEMS</h3>

<p>SOrt-ITEMS (Monzoorul et al., 2009) <a class="footnoteRef" href="#fn10" id="fnref10"><sup>10</sup></a> is an alignment-based binning algorithm developed by Innovations Labs of Tata Consultancy Services (TCS) Ltd., India. Users need to perform a similarity search of the input metagenomic sequences (reads) against the nr protein database using BLASTx search. The generated blastx output is then taken as input by the SOrt-ITEMS program. The method uses a range of BLAST alignment parameter thresholds to first identify an appropriate taxonomic level (or rank) where the read can be assigned. An orthology-based approach is then adopted for the final assignment of the metagenomic read. Other alignment-based binning algorithms developed by the Innovation Labs of Tata Consultancy Services (TCS) include DiScRIBinATE,<a class="footnoteRef" href="#fn11" id="fnref11"><sup>11</sup></a> ProViDE <a class="footnoteRef" href="#fn12" id="fnref12"><sup>12</sup></a> and SPHINX.<a class="footnoteRef" href="#fn13" id="fnref13"><sup>13</sup></a> The methodologies of these algorithms are summarized below.</p>
<h3 id="discribinate">DiScRIBinATE</h3>

<p>DiScRIBinATE (Ghosh et al., 2010) <a class="footnoteRef" href="#fn14" id="fnref14"><sup>14</sup></a> is an alignment-based binning algorithms developed by the Innovations Labs of Tata Consultancy Services (TCS) Ltd., India. DiScRIBinATE replaces the orthology approach of SOrt-ITEMS with a quicker 'alignment-free' approach. Incorporating this alternate strategy was observed to reduce the binning time by half without any significant loss in the accuracy and specificity of assignments. Besides, a novel reclassification strategy incorporated in DiScRIBinATE was seem to reduce the overall misclassification rate.</p>
<h3 id="provide">ProViDE</h3>

<p>ProViDE (Ghosh et al., 2011) <a class="footnoteRef" href="#fn15" id="fnref15"><sup>15</sup></a> is an alignment-based binning approach developed by the Innovation Labs of Tata Consultancy Services (TCS) Ltd. for the estimation of viral diversity in metagenomic samples. ProViDE adopts the reverse orthlogy based approach similar to SOrt-ITEMS for the taxonomic classification of metagenomic sequences obtained from virome datasets. It a customized set of BLAST parameter thresholds, specifically suited for viral metagenomic sequences. These thresholds capture the pattern of sequence divergence and the non-uniform taxonomic hierarchy observed within/across various taxonomic groups of the viral kingdom.</p>
<h3 id="pcahier">PCAHIER</h3>

<p>PCAHIER (Zheng et al., 2010),<a class="footnoteRef" href="#fn16" id="fnref16"><sup>16</sup></a> another binning algorithm developed by the Georgia Institute of Technology., employs n-mer oligonucleotide frequencies as the features and adopts a hierarchical classifier (PCAHIER) for binning short metagenomic fragments. The principal component analysis was used to reduce the high dimensionality of the feature space. The effectiveness of the PCAHIER was demonstrated through comparisons against a non-hierarchical classifier, and two existing binning algorithms (TETRA and Phylopythia).</p>
<h3 id="sphinx">SPHINX</h3>

<p>SPHINX (Mohammed et al., 2011),<a class="footnoteRef" href="#fn17" id="fnref17"><sup>17</sup></a> another binning algorithm developed by the Innovation Labs of Tata Consultancy Services (TCS) Ltd., adopts a hybrid strategy that achieves high binning efficiency by utilizing the principles of both 'composition'- and 'alignment'-based binning algorithms. The approach was designed with the objective of analyzing metagenomic datasets as rapidly as composition-based approaches, but nevertheless with the accuracy and specificity of alignment-based algorithms. SPHINX was observed to classify metagenomic sequences as rapidly as composition-based algorithms. In addition, the binning efficiency (in terms of accuracy and specificity of assignments) of SPHINX was observed to be comparable with results obtained using alignment-based algorithms.</p>
<h3 id="indus-and-twarit">INDUS<a class="footnoteRef" href="#fn18" id="fnref18"><sup>18</sup></a> and TWARIT<a class="footnoteRef" href="#fn19" id="fnref19"><sup>19</sup></a></h3>

<p>Represent other composition-based binning algorithms developed by the Innovation Labs of Tata Consultancy Services (TCS) Ltd. These algorithms utilize a range of oligonucleotide compositional (as well as statistical) parameters to improve binning time while maintaining the accuracy and specificity of taxonomic assignments.</p>
<h3 id="other-algorithms">Other algorithms</h3>

<p>This list is not exhaustive:</p>
<ul>
<li>TACOA (Diaz et al., 2009)</li>
<li>Parallel-META (Su et al., 2011)</li>
<li>PhyloPythiaS (Patil et al., 2011)</li>
<li>RITA (MacDonald et al., 2012)<ref name="mac2012"></ref></li>
</ul>

<p></p>
<ul>
<li>MetaPhlAn (Segata et al., 2012) <ref name="segata2012"></ref></li>
</ul>

<p></p>
<ul>
<li>Quikr (Koslicki et al., 2013) <ref name="koslicki2013"></ref></li>
</ul>

<p> All these algorithms employ different schemes for binning sequences, such as hierarchical classification, and operate in either a supervised or unsupervised manner. These algorithms provide a global view of how diverse the samples are, and can potentially connect community composition and function in metagenomes.</p>
<h2 id="references">References</h2>
<ul>
<li></li>
</ul>

<p>"</p>

<p><a class="uri" href="Category:Bioinformatics" title="wikilink">Category:Bioinformatics</a></p>
<section class="footnotes">
<hr/>
<ol>
<li id="fn1"><a href="#fnref1">↩</a></li>
<li id="fn2"><a href="#fnref2">↩</a></li>
<li id="fn3"><a href="#fnref3">↩</a></li>
<li id="fn4"><a href="#fnref4">↩</a></li>
<li id="fn5"><a href="#fnref5">↩</a></li>
<li id="fn6"><a href="#fnref6">↩</a></li>
<li id="fn7"><a href="#fnref7">↩</a></li>
<li id="fn8"><a href="#fnref8">↩</a></li>
<li id="fn9"></li>
<li id="fn10"><a href="#fnref10">↩</a></li>
<li id="fn11"><a href="#fnref11">↩</a></li>
<li id="fn12"><a href="#fnref12">↩</a></li>
<li id="fn13"><a href="#fnref13">↩</a></li>
<li id="fn14"></li>
<li id="fn15"></li>
<li id="fn16"><a href="#fnref16">↩</a></li>
<li id="fn17"></li>
<li id="fn18"><a href="#fnref18">↩</a></li>
<li id="fn19"><a href="#fnref19">↩</a></li>
</ol>
</section>
</body>
</html>
