<html lang="en">
<head>
<meta charset="utf-8"/>
<title offset="1737">Biosynthesis</title>
<script src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_SVG.js" type="text/javascript">
</script>
</head>
<body>
<h1>Biosynthesis</h1>
<hr/>

<p><strong>Biosynthesis</strong> (also called <strong>biogenesis</strong> or <strong>anabolism</strong>) is a multi-step, <a href="catalyst" title="wikilink">enzyme-catalyzed</a> process where <a href="substrate_(chemistry)" title="wikilink">substrates</a> are converted into more complex <a href="Product_(chemistry)" title="wikilink">products</a> in living organisms. In biosynthesis, simple <a href="Chemical_compound" title="wikilink">compounds</a> are modified, converted into other compounds, or joined together to form <a class="uri" href="macromolecules" title="wikilink">macromolecules</a>. This process often consists of <a href="metabolic_pathway" title="wikilink">metabolic pathways</a>. Some of these biosynthetic pathways are located within a single cellular <a class="uri" href="organelle" title="wikilink">organelle</a>, while others involve enzymes that are located within multiple cellular organelles. Examples of these biosynthetic pathways include the production of <a href="lipid_membrane" title="wikilink">lipid membrane</a> components and <a href="nucleotide" title="wikilink">nucleotides</a>.</p>

<p>The prerequisite elements for biosynthesis include: <a href="Precursor_(chemistry)" title="wikilink">precursor</a> compounds, <a href="chemical_energy" title="wikilink">chemical energy</a> (e.g. <a href="adenosine_triphosphate" title="wikilink">ATP</a>), and catalytic <a class="uri" href="enzymes" title="wikilink">enzymes</a> which may require <a class="uri" href="coenzymes" title="wikilink">coenzymes</a> (e.g.<a class="uri" href="NADH" title="wikilink">NADH</a>, <a class="uri" href="NADPH" title="wikilink">NADPH</a>). These elements create <a class="uri" href="monomers" title="wikilink">monomers</a>, the building blocks for macromolecules. Some important biological macromolecules include: <a class="uri" href="proteins" title="wikilink">proteins</a>, which are composed of <a href="amino_acid" title="wikilink">amino acid</a> monomers joined via <a href="peptide_bonds" title="wikilink">peptide bonds</a>, and <a class="uri" href="DNA" title="wikilink">DNA</a> molecules, which are composed of nucleotides joined via <a href="phosphodiester_bonds" title="wikilink">phosphodiester bonds</a>. </p>
<h2 id="properties-of-chemical-reactions">Properties of chemical reactions</h2>

<p>Biosynthesis occurs due to a series of chemical reactions. For these reactions to take place, the following elements are necessary:<a class="footnoteRef" href="#fn1" id="fnref1"><sup>1</sup></a></p>
<ul>
<li><a href="Precursor_(chemistry)" title="wikilink">Precursor compounds</a>: these compounds are the starting molecules or <a href="Substrate_(biochemistry)" title="wikilink">substrates</a> in a reaction. These may also be viewed as the <a class="uri" href="reactants" title="wikilink">reactants</a> in a given chemical process.</li>
<li><a href="Chemical_energy" title="wikilink">Chemical energy</a>: chemical energy can be found in the form of high energy molecules. These molecules are required for energetically unfavorable reactions. Furthermore, the <a class="uri" href="hydrolysis" title="wikilink">hydrolysis</a> of these compounds drives a reaction forward. High energy molecules, such as <a href="adenosine_triphosphate" title="wikilink">ATP</a>, have three <a class="uri" href="phosphates" title="wikilink">phosphates</a>. Often, the terminal phosphate is split off during hydrolysis and transferred to another molecule.</li>
<li><a href="Catalysis" title="wikilink">Catalytic enzymes</a>: these molecules are special <a class="uri" href="proteins" title="wikilink">proteins</a> that catalyze a reaction by increasing the <a href="Reaction_rate" title="wikilink">rate of the reaction</a> and lowering the <a href="Energy_of_activation" title="wikilink">activation energy</a>.</li>
<li><a class="uri" href="Coenzymes" title="wikilink">Coenzymes</a> or <a href="Cofactor_(biochemistry)" title="wikilink">cofactors</a>: cofactors are molecules that assist in chemical reactions. These may be <a href="metal_ions" title="wikilink">metal ions</a>, vitamin derivatives such as <a class="uri" href="NADH" title="wikilink">NADH</a> and <a href="Acetyl-CoA" title="wikilink">acetyl CoA</a>, or non-vitamin derivatives such as ATP. In the case of NADH, the molecule transfers a hydrogen, whereas acetyl CoA transfers an <a href="acetyl_group" title="wikilink">acetyl group</a>, and ATP transfers a phosphate.</li>
</ul>

<p>In the simplest sense, the reactions that occur in biosynthesis have the following format:<a class="footnoteRef" href="#fn2" id="fnref2"><sup>2</sup></a></p>
<dl>
<dd><dl>
<dd>

<math display="inline" id="Biosynthesis:0">
 <semantics>
  <mrow>
   <mrow>
    <mi>R</mi>
    <mi>e</mi>
    <mi>a</mi>
    <mi>c</mi>
    <mi>t</mi>
    <mi>a</mi>
    <mi>n</mi>
    <mi>t</mi>
   </mrow>
   <munderover accent="true" accentunder="true">
    <mo>→</mo>
    <mrow>
     <mi>e</mi>
     <mi>n</mi>
     <mi>z</mi>
     <mi>y</mi>
     <mi>m</mi>
     <mi>e</mi>
    </mrow>
    <mi></mi>
   </munderover>
   <mrow>
    <mi>P</mi>
    <mi>r</mi>
    <mi>o</mi>
    <mi>d</mi>
    <mi>u</mi>
    <mi>c</mi>
    <mi>t</mi>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <apply>
     <apply>
      <times></times>
      <ci>e</ci>
      <ci>n</ci>
      <ci>z</ci>
      <ci>y</ci>
      <ci>m</ci>
      <ci>e</ci>
     </apply>
     <apply>
      <csymbol cd="latexml">absent</csymbol>
      <ci>normal-→</ci>
     </apply>
    </apply>
    <apply>
     <times></times>
     <ci>R</ci>
     <ci>e</ci>
     <ci>a</ci>
     <ci>c</ci>
     <ci>t</ci>
     <ci>a</ci>
     <ci>n</ci>
     <ci>t</ci>
    </apply>
    <apply>
     <times></times>
     <ci>P</ci>
     <ci>r</ci>
     <ci>o</ci>
     <ci>d</ci>
     <ci>u</ci>
     <ci>c</ci>
     <ci>t</ci>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   Reactant\xrightarrow[enzyme]{}Product
  </annotation>
 </semantics>
</math>


</dd>
</dl>
</dd>
</dl>

<p><br/>
Some variations of this basic equation which will be discussed later in more detail are:<a class="footnoteRef" href="#fn3" id="fnref3"><sup>3</sup></a></p>

<p>1. Simple compounds which are converted into other compounds, usually as part of a multiple step reaction pathway. Two examples of this type of reaction occur during the formation of <a href="nucleic_acid" title="wikilink">nucleic acids</a> and the <a href="Aminoacyl_tRNA_synthetase" title="wikilink">charging</a> of <a class="uri" href="tRNA" title="wikilink">tRNA</a> prior to <a href="Translation_(biology)" title="wikilink">translation</a>. For some of these steps, chemical energy is required:<br/>
:

<math display="block" id="Biosynthesis:1">
 <semantics>
  <mrow>
   <mrow>
    <mrow>
     <mi>P</mi>
     <mi>r</mi>
     <mi>e</mi>
     <mi>c</mi>
     <mi>u</mi>
     <mi>r</mi>
     <mi>s</mi>
     <mi>o</mi>
     <mpadded width="+3.3pt">
      <mi>r</mi>
     </mpadded>
     <mi>m</mi>
     <mi>o</mi>
     <mi>l</mi>
     <mi>e</mi>
     <mi>c</mi>
     <mi>u</mi>
     <mi>l</mi>
     <mi>e</mi>
    </mrow>
    <mo>+</mo>
    <mrow>
     <mi>A</mi>
     <mi>T</mi>
     <mi>P</mi>
    </mrow>
   </mrow>
   <mo>⇌</mo>
   <mrow>
    <mrow>
     <mi>p</mi>
     <mi>r</mi>
     <mi>o</mi>
     <mi>d</mi>
     <mi>u</mi>
     <mi>c</mi>
     <mpadded width="+3.3pt">
      <mi>t</mi>
     </mpadded>
     <mi>A</mi>
     <mi>M</mi>
     <mi>P</mi>
    </mrow>
    <mo>+</mo>
    <mrow>
     <mi>P</mi>
     <msub>
      <mi>P</mi>
      <mi>i</mi>
     </msub>
    </mrow>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <ci>normal-⇌</ci>
    <apply>
     <plus></plus>
     <apply>
      <times></times>
      <ci>P</ci>
      <ci>r</ci>
      <ci>e</ci>
      <ci>c</ci>
      <ci>u</ci>
      <ci>r</ci>
      <ci>s</ci>
      <ci>o</ci>
      <ci>r</ci>
      <ci>m</ci>
      <ci>o</ci>
      <ci>l</ci>
      <ci>e</ci>
      <ci>c</ci>
      <ci>u</ci>
      <ci>l</ci>
      <ci>e</ci>
     </apply>
     <apply>
      <times></times>
      <ci>A</ci>
      <ci>T</ci>
      <ci>P</ci>
     </apply>
    </apply>
    <apply>
     <plus></plus>
     <apply>
      <times></times>
      <ci>p</ci>
      <ci>r</ci>
      <ci>o</ci>
      <ci>d</ci>
      <ci>u</ci>
      <ci>c</ci>
      <ci>t</ci>
      <ci>A</ci>
      <ci>M</ci>
      <ci>P</ci>
     </apply>
     <apply>
      <times></times>
      <ci>P</ci>
      <apply>
       <csymbol cd="ambiguous">subscript</csymbol>
       <ci>P</ci>
       <ci>i</ci>
      </apply>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   Precursor~{}molecule+ATP\rightleftharpoons{}product~{}AMP+PP_{i}
  </annotation>
 </semantics>
</math>

<br/>
2. Simple compounds that are converted into other compounds with the assistance of cofactors. For example, the synthesis of <a href="phospholipid" title="wikilink">phospholipids</a> requires acetyl CoA, while the synthesis of another membrane component, <a href="Sphingolipid" title="wikilink">shingolipids</a>, requires NADH and FADH for the formation the <a class="uri" href="sphingosine" title="wikilink">sphingosine</a> backbone. The general equation for these examples is:<br/>
:

<math display="block" id="Biosynthesis:2">
 <semantics>
  <mrow>
   <mrow>
    <mrow>
     <mi>P</mi>
     <mi>r</mi>
     <mi>e</mi>
     <mi>c</mi>
     <mi>u</mi>
     <mi>r</mi>
     <mi>s</mi>
     <mi>o</mi>
     <mpadded width="+3.3pt">
      <mi>r</mi>
     </mpadded>
     <mi>m</mi>
     <mi>o</mi>
     <mi>l</mi>
     <mi>e</mi>
     <mi>c</mi>
     <mi>u</mi>
     <mi>l</mi>
     <mi>e</mi>
    </mrow>
    <mo>+</mo>
    <mrow>
     <mi>C</mi>
     <mi>o</mi>
     <mi>f</mi>
     <mi>a</mi>
     <mi>c</mi>
     <mi>t</mi>
     <mi>o</mi>
     <mi>r</mi>
    </mrow>
   </mrow>
   <munderover accent="true" accentunder="true">
    <mo>→</mo>
    <mrow>
     <mi>e</mi>
     <mi>n</mi>
     <mi>z</mi>
     <mi>y</mi>
     <mi>m</mi>
     <mi>e</mi>
    </mrow>
    <mi></mi>
   </munderover>
   <mrow>
    <mi>m</mi>
    <mi>a</mi>
    <mi>c</mi>
    <mi>r</mi>
    <mi>o</mi>
    <mi>m</mi>
    <mi>o</mi>
    <mi>l</mi>
    <mi>e</mi>
    <mi>c</mi>
    <mi>u</mi>
    <mi>l</mi>
    <mi>e</mi>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <apply>
     <apply>
      <times></times>
      <ci>e</ci>
      <ci>n</ci>
      <ci>z</ci>
      <ci>y</ci>
      <ci>m</ci>
      <ci>e</ci>
     </apply>
     <apply>
      <csymbol cd="latexml">absent</csymbol>
      <ci>normal-→</ci>
     </apply>
    </apply>
    <apply>
     <plus></plus>
     <apply>
      <times></times>
      <ci>P</ci>
      <ci>r</ci>
      <ci>e</ci>
      <ci>c</ci>
      <ci>u</ci>
      <ci>r</ci>
      <ci>s</ci>
      <ci>o</ci>
      <ci>r</ci>
      <ci>m</ci>
      <ci>o</ci>
      <ci>l</ci>
      <ci>e</ci>
      <ci>c</ci>
      <ci>u</ci>
      <ci>l</ci>
      <ci>e</ci>
     </apply>
     <apply>
      <times></times>
      <ci>C</ci>
      <ci>o</ci>
      <ci>f</ci>
      <ci>a</ci>
      <ci>c</ci>
      <ci>t</ci>
      <ci>o</ci>
      <ci>r</ci>
     </apply>
    </apply>
    <apply>
     <times></times>
     <ci>m</ci>
     <ci>a</ci>
     <ci>c</ci>
     <ci>r</ci>
     <ci>o</ci>
     <ci>m</ci>
     <ci>o</ci>
     <ci>l</ci>
     <ci>e</ci>
     <ci>c</ci>
     <ci>u</ci>
     <ci>l</ci>
     <ci>e</ci>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   Precursor~{}molecule+Cofactor\xrightarrow[enzyme]{}macromolecule
  </annotation>
 </semantics>
</math>

<br/>
3. Simple compounds that join together to create a macromolecule. For example, <a href="fatty_acid" title="wikilink">fatty acids</a> join together to form phopspholipids. In turn, phospholipids and <a class="uri" href="cholesterol" title="wikilink">cholesterol</a> interact <a href="Noncovalent_bonding" title="wikilink">noncovalently</a> in order to form the <a href="lipid_bilayer" title="wikilink">lipid bilayer</a>. This reaction may be depicted as follows:<br/>
:

<math display="block" id="Biosynthesis:3">
 <semantics>
  <mrow>
   <mrow>
    <mrow>
     <mi>M</mi>
     <mi>o</mi>
     <mi>l</mi>
     <mi>e</mi>
     <mi>c</mi>
     <mi>u</mi>
     <mi>l</mi>
     <mpadded width="+3.3pt">
      <mi>e</mi>
     </mpadded>
     <mn>1</mn>
    </mrow>
    <mo>+</mo>
    <mrow>
     <mi>M</mi>
     <mi>o</mi>
     <mi>l</mi>
     <mi>e</mi>
     <mi>c</mi>
     <mi>u</mi>
     <mi>l</mi>
     <mpadded width="+3.3pt">
      <mi>e</mi>
     </mpadded>
     <mn>2</mn>
    </mrow>
   </mrow>
   <mover accent="true">
    <mo>→</mo>
    <mi></mi>
   </mover>
   <mrow>
    <mi>m</mi>
    <mi>a</mi>
    <mi>c</mi>
    <mi>r</mi>
    <mi>o</mi>
    <mi>m</mi>
    <mi>o</mi>
    <mi>l</mi>
    <mi>e</mi>
    <mi>c</mi>
    <mi>u</mi>
    <mi>l</mi>
    <mi>e</mi>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <apply>
     <csymbol cd="latexml">absent</csymbol>
     <ci>normal-→</ci>
    </apply>
    <apply>
     <plus></plus>
     <apply>
      <times></times>
      <ci>M</ci>
      <ci>o</ci>
      <ci>l</ci>
      <ci>e</ci>
      <ci>c</ci>
      <ci>u</ci>
      <ci>l</ci>
      <ci>e</ci>
      <cn type="integer">1</cn>
     </apply>
     <apply>
      <times></times>
      <ci>M</ci>
      <ci>o</ci>
      <ci>l</ci>
      <ci>e</ci>
      <ci>c</ci>
      <ci>u</ci>
      <ci>l</ci>
      <ci>e</ci>
      <cn type="integer">2</cn>
     </apply>
    </apply>
    <apply>
     <times></times>
     <ci>m</ci>
     <ci>a</ci>
     <ci>c</ci>
     <ci>r</ci>
     <ci>o</ci>
     <ci>m</ci>
     <ci>o</ci>
     <ci>l</ci>
     <ci>e</ci>
     <ci>c</ci>
     <ci>u</ci>
     <ci>l</ci>
     <ci>e</ci>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   Molecule~{}1+Molecule~{}2\xrightarrow{}macromolecule
  </annotation>
 </semantics>
</math>

 </p>
<h2 id="lipids">Lipids</h2>

<p> Many intricate macromolecules are synthesized in a pattern of simple, repeated structures.<a class="footnoteRef" href="#fn4" id="fnref4"><sup>4</sup></a> For example, the simplest structures of lipids are <a href="fatty_acids" title="wikilink">fatty acids</a>. Fatty acids are <a class="uri" href="hydrocarbon" title="wikilink">hydrocarbon</a> derivatives; they contain a <a href="carboxyl_group" title="wikilink">carboxyl group</a> “head” and a hydrocarbon chain “tail.”<a class="footnoteRef" href="#fn5" id="fnref5"><sup>5</sup></a> These fatty acids create larger components, which in turn incorporate noncovalent interactions to form the lipid bilayer.<a class="footnoteRef" href="#fn6" id="fnref6"><sup>6</sup></a> Fatty acid chains are found in two major components of membrane lipids: <a class="uri" href="phospholipids" title="wikilink">phospholipids</a> and <a class="uri" href="sphingolipids" title="wikilink">sphingolipids</a>. A third major membrane component, <a class="uri" href="cholesterol" title="wikilink">cholesterol</a>, does not contain these fatty acid units.<a class="footnoteRef" href="#fn7" id="fnref7"><sup>7</sup></a></p>
<h3 id="phospholipids">Phospholipids</h3>

<p>The foundation of all biomembranes consists of a <a class="uri" href="bilayer" title="wikilink">bilayer</a> structure of phospholipids.<a class="footnoteRef" href="#fn8" id="fnref8"><sup>8</sup></a> The phospholipid molecule is <a class="uri" href="amphipathic" title="wikilink">amphipathic</a>; it contains a <a class="uri" href="hydrophilic" title="wikilink">hydrophilic</a> polar head and a <a class="uri" href="hydrophobic" title="wikilink">hydrophobic</a> nonpolar tail.<a class="footnoteRef" href="#fn9" id="fnref9"><sup>9</sup></a> The phospholipid heads interact with each other and aqueous media, while the hydrocarbon tails orient themselves in the center, away from water.<a class="footnoteRef" href="#fn10" id="fnref10"><sup>10</sup></a> These latter interactions drive the bilayer structure that acts as a barrier for ions and molecules.<a class="footnoteRef" href="#fn11" id="fnref11"><sup>11</sup></a></p>

<p>There are various types of phospholipids; consequently, their synthesis pathways differ. However, the first step in phospholipid synthesis involves the formation of <a class="uri" href="phosphatidate" title="wikilink">phosphatidate</a> or diacylglycerol 3-phosphate at the <a href="endoplasmic_reticulum" title="wikilink">endoplasmic reticulum</a> and <a href="outer_mitochondrial_membrane" title="wikilink">outer mitochondrial membrane</a>.<a class="footnoteRef" href="#fn12" id="fnref12"><sup>12</sup></a> The synthesis pathway is found below: </p>

<p>The pathway starts with glycerol 3-phosphate, which gets converted to lysophosphatidate via the addition of a fatty acid chain provided by <a href="Acyl_coenzyme_a" title="wikilink">acyl coenzyme A</a>.<a class="footnoteRef" href="#fn13" id="fnref13"><sup>13</sup></a> Then, lysophosphatidate is converted to phosphatidate via the addition of another fatty acid chain contributed by a second acyl CoA; all of these steps are catalyzed by the glycerol phosphate <a class="uri" href="acyltransferase" title="wikilink">acyltransferase</a> enzyme.<a class="footnoteRef" href="#fn14" id="fnref14"><sup>14</sup></a> Phospholipid synthesis continues in the endoplasmic reticulum, and the biosynthesis pathway diverges depending on the components of the particular phospholipid.<a class="footnoteRef" href="#fn15" id="fnref15"><sup>15</sup></a></p>
<h3 id="sphingolipids">Sphingolipids</h3>

<p>Like phospholipids, these fatty acid derivatives have a polar head and nonpolar tails.<a class="footnoteRef" href="#fn16" id="fnref16"><sup>16</sup></a> Unlike phospholipids, sphingolipids have a <a class="uri" href="sphingosine" title="wikilink">sphingosine</a> backbone.<a class="footnoteRef" href="#fn17" id="fnref17"><sup>17</sup></a> Sphingolipids exist in <a class="uri" href="eukaryotic" title="wikilink">eukaryotic</a> cells and are particularly abundant in the <a href="central_nervous_system" title="wikilink">central nervous system</a>.<a class="footnoteRef" href="#fn18" id="fnref18"><sup>18</sup></a> For example, sphingomyelin is part of the <a href="myelin_sheath" title="wikilink">myelin sheath</a> of nerve fibers.<a class="footnoteRef" href="#fn19" id="fnref19"><sup>19</sup></a></p>

<p>Sphingolipids are formed from <a href="ceramide" title="wikilink">ceramides</a> that consist of a fatty acid chain attached to the amino group of a sphingosine backbone. These ceramides are synthesized from the <a class="uri" href="acylation" title="wikilink">acylation</a> of sphingosine.<a class="footnoteRef" href="#fn20" id="fnref20"><sup>20</sup></a> The biosynthetic pathway for sphingosine is found below:  As the image denotes, during sphingosine synthesis, palmitoyl CoA and serine undergo a <a href="condensation_reaction" title="wikilink">condensation reaction</a> which results in the formation of dehydrosphingosine.<a class="footnoteRef" href="#fn21" id="fnref21"><sup>21</sup></a> This product is then reduced to form dihydrospingosine, which is converted to sphingosine via the <a href="oxidation_reaction" title="wikilink">oxidation reaction</a> by <a class="uri" href="FAD" title="wikilink">FAD</a>.<a class="footnoteRef" href="#fn22" id="fnref22"><sup>22</sup></a></p>
<h3 id="cholesterol">Cholesterol</h3>

<p>This <a class="uri" href="lipid" title="wikilink">lipid</a> belongs to a class of molecules called <a class="uri" href="sterols" title="wikilink">sterols</a>.<a class="footnoteRef" href="#fn23" id="fnref23"><sup>23</sup></a> Sterols have four fused rings and a <a href="hydroxyl_group" title="wikilink">hydroxyl group</a>.<a class="footnoteRef" href="#fn24" id="fnref24"><sup>24</sup></a> Cholesterol is a particularly important molecule. Not only does it serve as a component of lipid membranes, it is also a precursor to several <a class="uri" href="steroid" title="wikilink">steroid</a> hormones, including <a class="uri" href="cortisol" title="wikilink">cortisol</a>, <a class="uri" href="testosterone" title="wikilink">testosterone</a>, and <a class="uri" href="estrogen" title="wikilink">estrogen</a>.<a class="footnoteRef" href="#fn25" id="fnref25"><sup>25</sup></a></p>

<p>Cholesterol is synthesized from <a href="acetyl_CoA" title="wikilink">acetyl CoA</a>.<a class="footnoteRef" href="#fn26" id="fnref26"><sup>26</sup></a> The pathway is shown below:</p>
<figure><b>(Figure)</b>
<figcaption><code>Cholesterol</code> <code>synthesis</code> <code>pathway</code></figcaption>
</figure>

<p>More generally, this synthesis occurs in three stages, with the first stage taking place in the <a class="uri" href="cytoplasm" title="wikilink">cytoplasm</a> and the second and third stages occurring in the endoplasmic reticulum.<a class="footnoteRef" href="#fn27" id="fnref27"><sup>27</sup></a> The stages are as follows:<a class="footnoteRef" href="#fn28" id="fnref28"><sup>28</sup></a></p>
<dl>
<dd><dl>
<dd>1. The synthesis of <a href="isopentenyl_pyrophosphate" title="wikilink">isopentenyl pyrophosphate</a>, the “building block” of cholesterol
</dd>
</dl>
</dd>
</dl>
<dl>
<dd><dl>
<dd>2. The formation of <a class="uri" href="squalene" title="wikilink">squalene</a> via the condensation of six molecules of isopentenyl phosphate
</dd>
</dl>
</dd>
</dl>
<dl>
<dd><dl>
<dd>3. The conversion of squalene into cholesterol via several enzymatic reactions
</dd>
</dl>
</dd>
</dl>
<h2 id="nucleotides">Nucleotides</h2>

<p>The biosynthesis of <a class="uri" href="nucleotides" title="wikilink">nucleotides</a> involves enzyme-<a href="Catalyst" title="wikilink">catalyzed</a> reactions that convert substrates into more complex products.<a class="footnoteRef" href="#fn29" id="fnref29"><sup>29</sup></a> Nucleotides are the building blocks of <a class="uri" href="DNA" title="wikilink">DNA</a> and <a class="uri" href="RNA" title="wikilink">RNA</a>. Nucleotides are composed of a five-membered ring formed from <a class="uri" href="ribose" title="wikilink">ribose</a> sugar in RNA, and <a class="uri" href="deoxyribose" title="wikilink">deoxyribose</a> sugar in DNA; these sugars are linked to a <a class="uri" href="purine" title="wikilink">purine</a> or <a class="uri" href="pyrimidine" title="wikilink">pyrimidine</a> base with a <a href="glycosidic_bond" title="wikilink">glycosidic bond</a> and a <a class="uri" href="phosphate" title="wikilink">phosphate</a> group at the <a href="Ribonucleotide" title="wikilink">5’ location</a> of the sugar.<a class="footnoteRef" href="#fn30" id="fnref30"><sup>30</sup></a></p>
<h3 id="purine-nucleotides">Purine nucleotides</h3>

<p>[[<a class="uri" href="File:Nucleotides">File:Nucleotides</a> syn1.svg|thumb|380x380px|</p>
<div style="border-width: 0px; border-bottom: 1px solid black; text-align: left;">

<p><strong>The synthesis of IMP</strong>.</p>
</div>

<p>]] The DNA nucleosides <a class="uri" href="adenosine" title="wikilink">adenosine</a> and <a class="uri" href="guanosine" title="wikilink">guanosine</a> consist of a purine base attached to a ribose sugar with a glycosidic bond. In the case of RNA nucleotides <a class="uri" href="deoxyadenosine" title="wikilink">deoxyadenosine</a> and <a class="uri" href="deoxyguanosine" title="wikilink">deoxyguanosine</a>, the purine bases are attached to a deoxyribose sugar with a glycosidic bond. The purine bases on DNA and RNA nucleotides are synthesized in a twelve-step reaction mechanism present in most single-celled organisms. Higher <a class="uri" href="eukaryotes" title="wikilink">eukaryotes</a> employ a similar <a href="reaction_mechanism" title="wikilink">reaction mechanism</a> in ten reaction steps. Purine bases are synthesized by converting <a href="phosphoribosyl_pyrophosphate" title="wikilink">phosphoribosyl pyrophosphate</a> (PRPP) to <a href="inosine_monophosphate" title="wikilink">inosine monophosphate</a> (IMP), which is the first key intermediate in purine base biosynthesis.<a class="footnoteRef" href="#fn31" id="fnref31"><sup>31</sup></a> Further enzymatic modification of <a href="Inosine_monophosphate" title="wikilink">IMP</a> produces the adenosine and guanosine bases of nucleotides.</p>
<ol>
<li>The first step in purine biosynthesis is a <a href="condensation_reaction" title="wikilink">condensation reaction</a>, performed by <a href="amidophosphoribosyltransferase" title="wikilink">glutamine-PRPP amidotransferase</a>. This enzyme transfers the <a href="amino_group" title="wikilink">amino group</a> from <a class="uri" href="glutamine" title="wikilink">glutamine</a> to PRPP, forming <a class="uri" href="5-phosphoribosylamine" title="wikilink">5-phosphoribosylamine</a>. The following step requires the activation of <a class="uri" href="glycine" title="wikilink">glycine</a> by the addition of a <a class="uri" href="phosphate" title="wikilink">phosphate</a> group from <a href="adenosine_triphosphate" title="wikilink">ATP</a>.</li>
<li>GAR synthetase<a class="footnoteRef" href="#fn32" id="fnref32"><sup>32</sup></a> performs the condensation of activated glycine onto PRPP, forming <a href="glycineamide_ribonucleotide" title="wikilink">glycineamide ribonucleotide</a> (GAR).</li>
<li><a href="GAR_transformylase" title="wikilink">GAR transformylase</a> adds a <a href="formyl_group" title="wikilink">formyl group</a> onto the amino group of GAR, forming formylglycinamide ribonucleotide (FGAR).</li>
<li>FGAR amidotransferase<a class="footnoteRef" href="#fn33" id="fnref33"><sup>33</sup></a> catalyzes the addition of a nitrogen group to FGAR, forming formylglycinamidine ribonucleotide (FGAM).</li>
<li><a href="AIR_synthetase_(FGAM_cyclase)" title="wikilink">FGAM cyclase</a> catalyzes ring closure, which involves removal of a water molecule, forming the 5-membered <a class="uri" href="imidazole" title="wikilink">imidazole</a> ring <a href="5-Aminoimidazole_ribotide" title="wikilink">5-aminoimidazole ribonucleotide</a> (AIR).</li>
<li>N5-CAIR synthetase transfers a <a class="uri" href="carboxyl" title="wikilink">carboxyl</a> group, forming the intermediate N5-carboxyaminoimidazole ribonucleotide (N5-CAIR).<a class="footnoteRef" href="#fn34" id="fnref34"><sup>34</sup></a></li>
<li><a href="5-(carboxyamino)imidazole_ribonucleotide_mutase" title="wikilink">N5-CAIR mutase</a> rearranges the carboxyl functional group and transfers it onto the imidazole ring, forming <a href="5'-phosphoribosyl-4-carboxy-5-aminoimidazole" title="wikilink">carboxyamino- imidazole ribonucleotide</a> (CAIR). The two step mechanism of CAIR formation from AIR is mostly found in single celled organisms. Higher eukaryotes contain the enzyme AIR carboxylase,<a class="footnoteRef" href="#fn35" id="fnref35"><sup>35</sup></a> which transfers a carboxyl group directly to AIR imidazole ring, forming CAIR.</li>
<li><a href="Phosphoribosylaminoimidazolesuccinocarboxamide_synthase" title="wikilink">SAICAR synthetase</a> forms a <a href="peptide_bond" title="wikilink">peptide bond</a> between <a class="uri" href="aspartate" title="wikilink">aspartate</a> and the added carboxyl group of the imidazole ring, forming <a href="phosphoribosylaminoimidazolesuccinocarboxamide" title="wikilink">N-succinyl-5-aminoimidazole-4-carboxamide ribonucleotide</a> (SAICAR).</li>
<li><a href="Adenylosuccinate_lyase" title="wikilink">SAICAR lyase</a> removes the carbon skeleton of the added aspartate, leaving the amino group and forming <a href="AICA_ribonucleotide" title="wikilink">5-aminoimidazole-4-carboxamide ribonucleotide</a> (AICAR).</li>
<li><a href="AICAR_transformylase" title="wikilink">AICAR transformylase</a> transfers a carbonyl group to AICAR, forming <a href="5-Formamidoimidazole-4-carboxamide_ribotide" title="wikilink">N-formylaminoimidazole- 4-carboxamide ribonucleotide</a> (FAICAR).</li>
<li>The final step involves the enzyme <a href="inosine_monophosphate_synthase" title="wikilink">IMP synthase</a>, which performs the purine ring closure and forms the <a href="inosine_monophosphate" title="wikilink">inosine monophosphate</a> (IMP) intermediate.<a class="footnoteRef" href="#fn36" id="fnref36"><sup>36</sup></a></li>
</ol>
<h3 id="pyrimidine-nucleotides">Pyrimidine nucleotides</h3>

<p> Other DNA and RNA nucleotide bases that are linked to the ribose sugar via a glycosidic bond are <a class="uri" href="thymine" title="wikilink">thymine</a>, <a class="uri" href="cytosine" title="wikilink">cytosine</a> and <a class="uri" href="uracil" title="wikilink">uracil</a> (which is only found in RNA). <a href="Uridine_monophosphate" title="wikilink">Uridine monophosphate</a> biosynthesis involves an enzyme that is located in the <a href="Inner_mitochondrial_membrane" title="wikilink">mitochondrial inner membrane</a> and multifunctional enzymes that are located in the <a class="uri" href="cytosol" title="wikilink">cytosol</a>.<a class="footnoteRef" href="#fn37" id="fnref37"><sup>37</sup></a></p>
<ol>
<li>The first step involves the enzyme <a href="carbamoyl_phosphate_synthase" title="wikilink">carbamoyl phosphate synthase</a> combining <a class="uri" href="glutamine" title="wikilink">glutamine</a> with <a href="CO2" title="wikilink">CO<sub>2</sub></a> in an ATP dependent reaction to form <a href="carbamoyl_phosphate" title="wikilink">carbamoyl phosphate</a>.</li>
<li><a href="Aspartate_carbamoyltransferase" title="wikilink">Aspartate carbamoyltransferase</a> <a href="Condensation_reaction" title="wikilink">condenses</a> carbamoyl phosphate with aspartate to form uridosuccinate.</li>
<li><a class="uri" href="Dihydroorotase" title="wikilink">Dihydroorotase</a> performs <a href="Ring-closure_reaction" title="wikilink">ring closure</a>, a reaction that loses water, to form <a class="uri" href="dihydroorotate" title="wikilink">dihydroorotate</a>.</li>
<li><a href="Dihydroorotate_dehydrogenase" title="wikilink">Dihydroorotate dehydrogenase</a>, located within the mitochondrial inner membrane,<a class="footnoteRef" href="#fn38" id="fnref38"><sup>38</sup></a> oxidizes dihydroorotate to <a class="uri" href="orotate" title="wikilink">orotate</a>.</li>
<li>Orotate phosphoribosyl hydrolase (OMP pyrophosphorylase) condenses orotate with <a class="uri" href="PRPP" title="wikilink">PRPP</a> to form <a href="orotidine_5'-monophosphate" title="wikilink">orotidine-5’-phosphate</a>.</li>
<li><a href="OMP_decarboxylase" title="wikilink">OMP decarboxylase</a> catalyzes the conversion of orotidine-5’-phosphate to <a href="Uridine_monophosphate" title="wikilink">UMP</a>.<a class="footnoteRef" href="#fn39" id="fnref39"><sup>39</sup></a></li>
</ol>

<p>After the uridine nucleotide base is synthesized, the other bases, cytosine and thymine are synthesized. Cytosine biosynthesis is a two-step reaction which involves the conversion of UMP to <a href="Uridine_triphosphate" title="wikilink">UTP</a>. <a class="uri" href="Phosphate" title="wikilink">Phosphate</a> addition to UMP is catalyzed by a <a class="uri" href="kinase" title="wikilink">kinase</a> enzyme. The enzyme <a href="CTP_synthase" title="wikilink">CTP synthase</a> catalyzes the next reaction step: the conversion of UTP to <a href="cytidine_triphosphate" title="wikilink">CTP</a> by transferring an <a href="amino_group" title="wikilink">amino group</a> from glutamine to uridine; this forms the cytosine base of CTP.<a class="footnoteRef" href="#fn40" id="fnref40"><sup>40</sup></a> The mechanism, which depicts the reaction UTP + ATP + glutamine ⇔ CTP + ADP + glutamate, is below:</p>

<p> </p>

<p>Cytosine is a nucleotide that is present in both DNA and RNA. However, uracil is only found in RNA. Therefore, after UTP is synthesized, it is must be converted into a <a class="uri" href="deoxy" title="wikilink">deoxy</a> form to be incorporated into DNA. This conversion involves the enzyme <a href="Ribonucleoside-triphosphate_reductase" title="wikilink">ribonucleoside triphosphate reductase</a>. This reaction that removes the 2’-OH of the ribose sugar to generate deoxyribose is not affected by the bases attached to the sugar. This non-specificity allows ribonucleoside triphosphate reductase to convert all <a href="nucleotide_triphosphates" title="wikilink">nucleotide triphosphates</a> to <a class="uri" href="deoxyribonucleotide" title="wikilink">deoxyribonucleotide</a> by a similar mechanism.<a class="footnoteRef" href="#fn41" id="fnref41"><sup>41</sup></a></p>

<p>In contrast to uracil, thymine bases are found mostly in DNA, not RNA. Cells do not normally contain thymine bases that are linked to ribose sugars in RNA, thus indicating that cells only synthesize deoxyribose-linked thymine. The enzyme <a href="thymidylate_synthetase" title="wikilink">thymidylate synthetase</a> is responsible for synthesizing thymine residues from <a class="uri" href="dUMP" title="wikilink">dUMP</a> to <a class="uri" href="dTMP" title="wikilink">dTMP</a>. This reaction transfers a <a class="uri" href="methyl" title="wikilink">methyl</a> group onto the uracil base of dUMP to generate dTMP.<a class="footnoteRef" href="#fn42" id="fnref42"><sup>42</sup></a> The thymidylate synthase reaction, dUMP + 5,10-methylenetetrahydrofolate ⇔ dTMP + dihydrofolate, is shown to the right. </p>
<h2 id="dna">DNA</h2>

<p><a href="File:DNA_replication_en.svg" title="wikilink">240x250px|framed|right|As DNA polymerase moves in a 3’ to 5’ direction along the template strand, it synthesizes a new strand in the 5’ to 3’ direction</a> Although there are differences between <a class="uri" href="eukaryotic" title="wikilink">eukaryotic</a> and <a class="uri" href="prokaryotic" title="wikilink">prokaryotic</a> DNA synthesis, the following section denotes key characteristics of DNA replication shared by both organisms.</p>

<p><a class="uri" href="DNA" title="wikilink">DNA</a> is composed of <a href="nucleotide" title="wikilink">nucleotides</a> that are joined by <a href="phosphodiester_bonds" title="wikilink">phosphodiester bonds</a>.<a class="footnoteRef" href="#fn43" id="fnref43"><sup>43</sup></a> <a href="DNA_replication" title="wikilink">DNA synthesis</a>, which takes place in the <a href="Cell_nucleus" title="wikilink">nucleus</a>, is a <a href="Semiconservative_replication" title="wikilink">semiconservative</a> process, which means that the resulting DNA molecule contains an original strand from the parent structure and a new strand.<a class="footnoteRef" href="#fn44" id="fnref44"><sup>44</sup></a> DNA synthesis is catalyzed by a family of <a href="DNA_polymerases" title="wikilink">DNA polymerases</a> that require four deoxynucleoside triphosphates, a <a href="Template_strand#Elongation" title="wikilink">template strand</a>, and a <a href="Rna_primer" title="wikilink">primer</a> with a free 3’OH in which to incorporate nucleotides.<a class="footnoteRef" href="#fn45" id="fnref45"><sup>45</sup></a></p>

<p>In order for DNA replication to occur, a <a href="Replication_fork#Replication_fork" title="wikilink">replication fork</a> is created by enzymes called <a href="helicase" title="wikilink">helicases</a> which unwind the DNA helix.<a class="footnoteRef" href="#fn46" id="fnref46"><sup>46</sup></a> <a href="Topoisomerase" title="wikilink">Topoisomerases</a> at the replication fork remove <a href="DNA_supercoil" title="wikilink">supercoils</a> caused by DNA unwinding, and <a href="Single-strand_binding_protein" title="wikilink">single-stranded DNA binding proteins</a> maintain the two single-stranded DNA templates stabilized prior to replication.<a class="footnoteRef" href="#fn47" id="fnref47"><sup>47</sup></a></p>

<p>DNA synthesis is initiated by the <a href="RNA_polymerase" title="wikilink">RNA polymerase</a> <a class="uri" href="primase" title="wikilink">primase</a>, which makes an RNA primer with a free 3’OH.<a class="footnoteRef" href="#fn48" id="fnref48"><sup>48</sup></a> This primer is attached to the single-stranded DNA template, and DNA polymerase elongates the chain by incorporating nucleotides; DNA polymerase also proofreads the newly synthesized DNA strand.<a class="footnoteRef" href="#fn49" id="fnref49"><sup>49</sup></a></p>

<p>During the polymerization reaction catalyzed by DNA polymerase, a <a href="nucleophilic_attack" title="wikilink">nucleophilic attack</a> occurs by the 3’OH of the growing chain on the innermost phosphorus atom of a deoxynucleoside triphosphate; this yields the formation of a <a href="Phosphodiester_bridges" title="wikilink">phosphodiester bridge</a> that attaches a new nucleotide and releases <a class="uri" href="pyrophosphate" title="wikilink">pyrophosphate</a>.<a class="footnoteRef" href="#fn50" id="fnref50"><sup>50</sup></a></p>

<p>Two types of strands are created simultaneously during replication: the <a href="Leading_strand#Replication_fork" title="wikilink">leading strand</a>, which is synthesized continuously and grows towards the replication fork, and the <a href="Lagging_strand#Replication_fork" title="wikilink">lagging strand</a>, which is made discontinuously in <a href="Okazaki_fragments" title="wikilink">Okazaki fragments</a> and grows away from the replication fork.<a class="footnoteRef" href="#fn51" id="fnref51"><sup>51</sup></a> Okazaki fragments are <a class="uri" href="covalently" title="wikilink">covalently</a> joined by <a href="DNA_ligase" title="wikilink">DNA ligase</a> to form a continuous strand.<a class="footnoteRef" href="#fn52" id="fnref52"><sup>52</sup></a> Then, to complete DNA replication, RNA primers are removed, and the resulting gaps are replaced with DNA and joined via DNA ligase.<a class="footnoteRef" href="#fn53" id="fnref53"><sup>53</sup></a> </p>
<h2 id="amino-acids">Amino acids</h2>

<p>A protein is a polymer that is composed from <a href="amino_acids" title="wikilink">amino acids</a> that are linked by <a href="peptide_bonds" title="wikilink">peptide bonds</a>. There are more than <a href="Amino_acid#Non-standard_amino_acids" title="wikilink">300 amino acids</a> found in nature of which only twenty, known as the <a href="Amino_acid#Essential_amino_acids" title="wikilink">standard amino acids</a>, are the building blocks for protein.<a class="footnoteRef" href="#fn54" id="fnref54"><sup>54</sup></a> Only <a href="green_plants" title="wikilink">green plants</a> and most <a class="uri" href="microbes" title="wikilink">microbes</a> are able to <a href="Amino_acid_synthesis" title="wikilink">synthesize</a> all of the 20 standard amino acids that are needed by all living species. <a class="uri" href="Mammals" title="wikilink">Mammals</a> can only synthesize ten of the twenty standard amino acids. The other amino acids, <a class="uri" href="valine" title="wikilink">valine</a>, <a class="uri" href="methionine" title="wikilink">methionine</a>, <a class="uri" href="leucine" title="wikilink">leucine</a>, <a class="uri" href="isoleucine" title="wikilink">isoleucine</a>, <a class="uri" href="phenylalanine" title="wikilink">phenylalanine</a>, <a class="uri" href="lysine" title="wikilink">lysine</a>, <a class="uri" href="threonine" title="wikilink">threonine</a> and <a class="uri" href="tryptophan" title="wikilink">tryptophan</a> for adults and <a class="uri" href="histidine" title="wikilink">histidine</a>, and <a class="uri" href="arginine" title="wikilink">arginine</a> for babies are obtained through diet.<a class="footnoteRef" href="#fn55" id="fnref55"><sup>55</sup></a></p>
<h3 id="amino-acid-basic-structure">Amino acid basic structure</h3>

<p>The general structure of the standard amino acids includes a <a href="Amine" title="wikilink">primary amino group</a>, a <a href="carboxyl_group" title="wikilink">carboxyl group</a> and the <a href="functional_group" title="wikilink">functional group</a> attached to the <a href="Alpha-carbon" title="wikilink">α-carbon</a>. The different amino acids are identified by the functional group. As a result of the three different groups attached to the α-carbon, amino acids are <a href="Chirality_(chemistry)#Symmetry" title="wikilink">asymmetrical molecules</a>. For all standard amino acids, except <a class="uri" href="glycine" title="wikilink">glycine</a>, the α-carbon is a <a href="chiral_center" title="wikilink">chiral center</a>. In the case of glycine, the α-carbon has two hydrogen atoms, thus adding symmetry to this molecule. With the exception of <a class="uri" href="proline" title="wikilink">proline</a>, all of the amino acids found in life have the <a href="Chirality_(chemistry)#By_configuration:_D-_and_L-" title="wikilink">L-isoform</a> conformation. Proline has a functional group on the α-carbon that forms a ring with the amino group.<a class="footnoteRef" href="#fn56" id="fnref56"><sup>56</sup></a></p>
<figure><b>(Figure)</b>
<figcaption>Glutamine oxoglutarate aminotransferase and glutamine synthetase</figcaption>
</figure>
<h3 id="nitrogen-source">Nitrogen source</h3>

<p>One major step in amino acid biosynthesis involves incorporating a nitrogen group onto the α-carbon. In cells, there are two major pathways of incorporating nitrogen groups. One pathway involves the enzyme <a href="glutamine_oxoglutarate_aminotransferase" title="wikilink">glutamine oxoglutarate aminotransferase</a> (GOGAT) which removes the <a class="uri" href="amide" title="wikilink">amide</a> amino group of <a class="uri" href="glutamine" title="wikilink">glutamine</a> and transfers it onto <a class="uri" href="2-oxoglutarate" title="wikilink">2-oxoglutarate</a>, producing two <a class="uri" href="glutamate" title="wikilink">glutamate</a> molecules. In this catalysis reaction, glutamine serves as the nitrogen source. An image illustrating this reaction is found to the right.</p>

<p>The other pathway for incorporating nitrogen onto the α-carbon of amino acids involves the enzyme <a href="glutamate_dehydrogenase" title="wikilink">glutamate dehydrogenase</a> (GDH). GDH is able to transfer <a class="uri" href="ammonia" title="wikilink">ammonia</a> onto 2-oxoglutarate and form glutamate. Furthermore, the enzyme <a href="glutamine_synthetase" title="wikilink">glutamine synthetase</a> (GS) is able to transfer ammonia onto glutamate and synthesize glutamine, replenishing glutamine.<a class="footnoteRef" href="#fn57" id="fnref57"><sup>57</sup></a></p>
<h3 id="the-glutamate-family-of-amino-acids">The glutamate family of amino acids</h3>

<p>The <a class="uri" href="glutamate" title="wikilink">glutamate</a> family of amino acids includes the amino acids that derive from the amino acid glutamate. This family includes: glutamate, <a class="uri" href="glutamine" title="wikilink">glutamine</a>, <a class="uri" href="proline" title="wikilink">proline</a>, and <a class="uri" href="arginine" title="wikilink">arginine</a>. This family also includes the amino acid <a class="uri" href="lysine" title="wikilink">lysine</a>, which is derived from <a class="uri" href="α-ketoglutarate" title="wikilink">α-ketoglutarate</a>.<a class="footnoteRef" href="#fn58" id="fnref58"><sup>58</sup></a></p>

<p>The biosynthesis of glutamate and glutamine is a key step in the nitrogen assimilation discussed above. The enzymes <a class="uri" href="GOGAT" title="wikilink">GOGAT</a> and <a href="Glutamate_dehydrogenase" title="wikilink">GDH</a> catalyze the <a href="nitrogen_assimilation" title="wikilink">nitrogen assimilation</a> reactions.</p>

<p>In bacteria, the enzyme <a href="glutamate_5-kinase" title="wikilink">glutamate 5-kinase</a> initiates the biosynthesis of proline by transferring a phosphate group from ATP onto glutamate. The next reaction is catalyzed by the enzyme <a href="Glutamate-5-semialdehyde_dehydrogenase" title="wikilink">pyrroline-5-carboxylate synthase</a> (P5CS), which catalyzes the reduction of the <a href="Gamma-carboxylation" title="wikilink">ϒ-carboxyl</a> group of L-glutamate 5-phosphate. This results in the formation of glutamate semialdehyde, which spontaneously cyclizes to pyrroline-5-carboxylate. Pyrroline-5-carboxylate is further reduced by the enzyme pyrroline-5-carboxylate reductase (P5CR) to yield a proline amino acid.<a class="footnoteRef" href="#fn59" id="fnref59"><sup>59</sup></a></p>

<p>In the first step of arginine biosynthesis in bacteria, glutamate is <a class="uri" href="acetylated" title="wikilink">acetylated</a> by transferring the acetyl group from acetyl-CoA at the N-α position; this prevents spontaneous cyclization. The enzyme <a href="N-acetylglutamate_synthase" title="wikilink">N-acetylglutamate synthase</a> (glutamate N-acetyltransferase) is responsible for catalyzing the acetylation step. Subsequent steps are catalyzed by the enzymes <a href="acetylglutamate_kinase" title="wikilink">N-acetylglutamate kinase</a>, <a href="N-acetyl-gamma-glutamyl-phosphate_reductase" title="wikilink">N-acetyl-gamma-glutamyl-phosphate reductase</a>, and <a href="acetylornithine_transaminase" title="wikilink">acetylornithine/succinyldiamino pimelate aminotransferase</a> and yield the N-acetyl-L-ornithine. The acetyl group of acetylornithine is removed by the enzyme <a href="Acetylornithine_deacetylase" title="wikilink">acetylornithinase</a> (AO) or <a href="Glutamate_N-acetyltransferase" title="wikilink">ornithine acetyltransferase</a> (OAT), and this yields <a class="uri" href="ornithine" title="wikilink">ornithine</a>. Then, the enzymes <a class="uri" href="citrulline" title="wikilink">citrulline</a> and <a class="uri" href="argininosuccinate" title="wikilink">argininosuccinate</a> convert ornithine to arginine.<a class="footnoteRef" href="#fn60" id="fnref60"><sup>60</sup></a></p>
<figure><b>(Figure)</b>
<figcaption>The diaminopimelic acid pathway</figcaption>
</figure>

<p>There are two distinct lysine biosynthetic pathways: the diaminopimelic acid pathway and the <a href="Alpha-aminoadipate_pathway" title="wikilink">α-aminoadipate pathway</a>. The most common of the two synthetic pathways is the diaminopimelic acid pathway; it consists of several enzymatic reactions that add carbon groups to aspartate to yield lysine:<a class="footnoteRef" href="#fn61" id="fnref61"><sup>61</sup></a></p>
<ol>
<li><a href="Aspartate_kinase" title="wikilink">Aspartate kinase</a> initiates the diaminopimelic acid pathway by phosphorylating aspartate and producing aspartyl phosphate.</li>
<li><a href="Aspartate-semialdehyde_dehydrogenase" title="wikilink">Aspartate semialdehyde dehydrogenase</a> catalyzes the <a class="uri" href="NADPH" title="wikilink">NADPH</a>-dependent reduction of aspartyl phosphate to yield aspartate semialdehyde.</li>
<li><a href="4-hydroxy-tetrahydrodipicolinate_synthase" title="wikilink">4-hydroxy-tetrahydrodipicolinate synthase</a> adds a <a href="Pyruvic_acid" title="wikilink">pyruvate</a> group to the β-aspartyl-4-semialdehyde, and a water molecule is removed. This causes <a href="Cyclic_compound" title="wikilink">cyclization</a> and gives rise to (2S,4S)-4-hydroxy-2,3,4,5-tetrahydrodipicolinate.</li>
<li><a href="Dihydrodipicolinate_reductase" title="wikilink">4-hydroxy-tetrahydrodipicolinate reductase</a> catalyzes the reduction of (2S,4S)-4-hydroxy-2,3,4,5-tetrahydrodipicolinate by NADPH to yield Δ’-piperideine-2,6-dicarboxylate (2,3,4,5-tetrahydrodipicolinate) and H<sub>2</sub>O.</li>
<li><a href="Tetrahydrodipicolinate_N-acetyltransferase" title="wikilink">Tetrahydrodipicolinate acyltransferase</a> catalyzes the acetylation reaction that results in ring opening and yields N-acetyl α-amino-ε-ketopimelate.</li>
<li><a href="Succinyldiaminopimelate_transaminase" title="wikilink">N-succinyl-α-amino-ε-ketopimelate-glutamate aminotransaminase</a> catalyzes the transamination reaction that removes the keto group of N-acetyl α-amino-ε-ketopimelate and replaces it with an amino group to yield N-succinyl-L-diaminopimelate.<a class="footnoteRef" href="#fn62" id="fnref62"><sup>62</sup></a></li>
<li><a href="succinyl-diaminopimelate_desuccinylase" title="wikilink">N-acyldiaminopimelate deacylase</a> catalyzes the deacylation of N-succinyl-L-diaminopimelate to yield L,L-diaminopimelate.<a class="footnoteRef" href="#fn63" id="fnref63"><sup>63</sup></a></li>
<li><a href="diaminopimelate_epimerase" title="wikilink">DAP epimerase</a> catalyzes the conversion of L,L-diaminopimelate to the <a href="Meso_form" title="wikilink">meso</a> form of L,L-diaminopimelate.<a class="footnoteRef" href="#fn64" id="fnref64"><sup>64</sup></a></li>
<li><a href="Diaminopimelate_decarboxylase" title="wikilink">DAP decarboxylase</a> catalyzes the removal of the carboxyl group, yielding L-lysine.</li>
</ol>
<h3 id="the-serine-family-of-amino-acids">The serine family of amino acids</h3>

<p>The <a class="uri" href="serine" title="wikilink">serine</a> family of amino acid includes: serine, <a class="uri" href="cysteine" title="wikilink">cysteine</a>, and <a class="uri" href="glycine" title="wikilink">glycine</a>. Most microorganisms and plants obtain the sulfur for synthesizing <a class="uri" href="methionine" title="wikilink">methionine</a> from the amino acid cysteine. Furthermore, the conversion of serine to glycine provides the carbons needed for the biosynthesis of the methionine and <a class="uri" href="histidine" title="wikilink">histidine</a>.<a class="footnoteRef" href="#fn65" id="fnref65"><sup>65</sup></a></p>

<p>During serine biosynthesis,<a class="footnoteRef" href="#fn66" id="fnref66"><sup>66</sup></a> the enzyme <a href="phosphoglycerate_dehydrogenase" title="wikilink">phosphoglycerate dehydrogenase</a> catalyzes the initial reaction that <a class="uri" href="oxidizes" title="wikilink">oxidizes</a> <a class="uri" href="3-phospho-D-glycerate" title="wikilink">3-phospho-D-glycerate</a> to yield <a href="Phosphohydroxypyruvic_acid" title="wikilink">3-phosphonooxypyruvate</a>.<a class="footnoteRef" href="#fn67" id="fnref67"><sup>67</sup></a> The following reaction is catalyzed by the enzyme <a href="phosphoserine_aminotransferase" title="wikilink">phosphoserine aminotransferase</a>, which transfers an amino group from glutamate onto 3-phosphonooxypyruvate to yield <a href="Phosphoserine" title="wikilink">L-phosphoserine</a>.<a class="footnoteRef" href="#fn68" id="fnref68"><sup>68</sup></a> The final step is catalyzed by the enzyme <a href="phosphoserine_phosphatase" title="wikilink">phosphoserine phosphatase</a>, which <a href="Dephosphorylation" title="wikilink">dephosphorylates</a> L-phosphoserine to yield <a class="uri" href="L-serine" title="wikilink">L-serine</a>.<a class="footnoteRef" href="#fn69" id="fnref69"><sup>69</sup></a></p>

<p>There are two known pathways for the biosynthesis of glycine. Organisms that use <a class="uri" href="ethanol" title="wikilink">ethanol</a> and <a class="uri" href="acetate" title="wikilink">acetate</a> as the major carbon source utilize the <a href="Glyconeogenesis" title="wikilink">glyconeogenic</a> pathway to synthesize <a class="uri" href="glycine" title="wikilink">glycine</a>. The other pathway of glycine biosynthesis is known as the <a class="uri" href="glycolytic" title="wikilink">glycolytic</a> pathway. This pathway converts serine synthesized from the intermediates of <a class="uri" href="glycolysis" title="wikilink">glycolysis</a> to glycine. In the glycolytic pathway, the enzyme <a href="serine_hydroxymethyltransferase" title="wikilink">serine hydroxymethyltransferase</a> catalyzes the cleavage of serine to yield glycine and transfers the cleaved carbon group of serine onto <a class="uri" href="tetrahydrofolate" title="wikilink">tetrahydrofolate</a>, forming <a href="5,10-Methylenetetrahydrofolate" title="wikilink">5,10-methylene-tetrahydrofolate</a>.<a class="footnoteRef" href="#fn70" id="fnref70"><sup>70</sup></a></p>

<p>Cysteine biosynthesis is a two-step reaction that involves the incorporation of inorganic <a class="uri" href="sulfur" title="wikilink">sulfur</a>. In microorganisms and plants, the enzyme <a href="Serine_O-acetyltransferase" title="wikilink">serine acetyltransferase</a> catalyzes the transfer of acetyl group from <a class="uri" href="acetyl-CoA" title="wikilink">acetyl-CoA</a> onto L-serine to yield <a class="uri" href="O-acetyl-L-serine" title="wikilink">O-acetyl-L-serine</a>.<a class="footnoteRef" href="#fn71" id="fnref71"><sup>71</sup></a> The following reaction step, catalyzed by the enzyme <a href="Cysteine_synthase" title="wikilink">O-acetyl serine (thiol) lyase</a>, replaces the acetyl group of O-acetyl-L-serine with sulfide to yield cysteine.<a class="footnoteRef" href="#fn72" id="fnref72"><sup>72</sup></a></p>
<h3 id="the-aspartate-family-of-amino-acids">The aspartate family of amino acids</h3>

<p>The <a class="uri" href="aspartate" title="wikilink">aspartate</a> family of amino acids includes: <a class="uri" href="threonine" title="wikilink">threonine</a>, <a class="uri" href="lysine" title="wikilink">lysine</a>, <a class="uri" href="methionine" title="wikilink">methionine</a>, <a class="uri" href="isoleucine" title="wikilink">isoleucine</a>, and aspartate. Lysine and isoleucine are considered part of the aspartate family even though part of their carbon skeleton is derived from <a class="uri" href="pyruvate" title="wikilink">pyruvate</a>. In the case of methionine, the methyl carbon is derived from serine and the sulfur group, but in most organisms, it is derived from cysteine.<a class="footnoteRef" href="#fn73" id="fnref73"><sup>73</sup></a></p>

<p>The biosynthesis of aspartate is a one step reaction that is catalyzed by a single enzyme. The enzyme <a href="aspartate_aminotransferase" title="wikilink">aspartate aminotransferase</a> catalyzes the transfer of an amino group from aspartate onto <a class="uri" href="α-ketoglutarate" title="wikilink">α-ketoglutarate</a> to yield glutamate and <a class="uri" href="oxaloacetate" title="wikilink">oxaloacetate</a>.<a class="footnoteRef" href="#fn74" id="fnref74"><sup>74</sup></a> Asparagine is synthesized by an ATP-dependent addition of an amino group onto aspartate; <a href="asparagine_synthetase" title="wikilink">asparagine synthetase</a> catalyzes the addition of nitrogen from glutamine or soluble ammonia to aspartate to yield asparagine.<a class="footnoteRef" href="#fn75" id="fnref75"><sup>75</sup></a> <a href="File:Lysine_Biosynthesis.png" title="wikilink">thumb|framed|400x400px|The diaminopimelic acid lysine biosynthetic pathway</a></p>

<p>The diaminopimelic acid biosynthetic pathway of lysine belongs to the aspartate family of amino acids. This pathway involves nine enzyme-catalyzed reactions that convert aspartate to lysine.<a class="footnoteRef" href="#fn76" id="fnref76"><sup>76</sup></a></p>
<ol>
<li><a href="Aspartate_kinase" title="wikilink">Aspartate kinase</a> catalyzes the initial step in the diaminopimelic acid pathway by transferring a <a class="uri" href="phosphoryl" title="wikilink">phosphoryl</a> from ATP onto the carboxylate group of aspartate, which yields aspartyl-β-phosphate.<a class="footnoteRef" href="#fn77" id="fnref77"><sup>77</sup></a></li>
<li><a href="Aspartate-semialdehyde_dehydrogenase" title="wikilink">Aspartate-semialdehyde dehydrogenase</a> catalyzes the reduction reaction by <a class="uri" href="dephosphorylation" title="wikilink">dephosphorylation</a> of aspartyl-β-phosphate to yield aspartate-β-semialdehyde.<a class="footnoteRef" href="#fn78" id="fnref78"><sup>78</sup></a></li>
<li><a href="4-hydroxy-tetrahydrodipicolinate_synthase" title="wikilink">Dihydrodipicolinate synthase</a> catalyzes the <a href="Condensation_reaction" title="wikilink">condensation</a> reaction of aspartate-β-semialdehyde with pyruvate to yield dihydrodipicolinic acid.<a class="footnoteRef" href="#fn79" id="fnref79"><sup>79</sup></a></li>
<li><a href="Dihydrodipicolinate_reductase" title="wikilink">4-hydroxy-tetrahydrodipicolinate reductase</a> catalyzes the reduction of dihydrodipicolinic acid to yield tetrahydrodipicolinic acid.<a class="footnoteRef" href="#fn80" id="fnref80"><sup>80</sup></a></li>
<li><a href="2,3,4,5-tetrahydropyridine-2,6-dicarboxylate_N-succinyltransferase" title="wikilink">Tetrahydrodipicolinate N-succinyltransferase</a> catalyzes the transfer of a succinyl group from succinyl-CoA on to tetrahydrodipicolinic acid to yield N-succinyl-L-2,6-diaminoheptanedioate.<a class="footnoteRef" href="#fn81" id="fnref81"><sup>81</sup></a></li>
<li>N-succinyldiaminopimelate aminotransferase catalyzes the transfer of an amino group from glutamate onto N-succinyl-L-2,6-diaminoheptanedioate to yield N-succinyl-L,L-diaminopimelic acid.<a class="footnoteRef" href="#fn82" id="fnref82"><sup>82</sup></a></li>
<li><a href="Succinyldiaminopimelate_transaminase" title="wikilink">Succinyl-diaminopimelate desuccinylase</a> catalyzes the removal of acyl group from N-succinyl-L,L-diaminopimelic acid to yield L,L-diaminopimelic acid.<a class="footnoteRef" href="#fn83" id="fnref83"><sup>83</sup></a></li>
<li><a href="Diaminopimelate_epimerase" title="wikilink">Diaminopimelate epimerase</a> catalyzes the inversion of the α-carbon of L,L-diaminopimelic acid to yield <a href="meso-diaminopimelic_acid" title="wikilink">meso-diaminopimelic acid</a>.<a class="footnoteRef" href="#fn84" id="fnref84"><sup>84</sup></a></li>
<li>Siaminopimelate decarboxylase catalyzes the final step in lysine biosynthesis that removes the carbon dioxide group from meso-diaminopimelic acid to yield L-lysine.<a class="footnoteRef" href="#fn85" id="fnref85"><sup>85</sup></a></li>
</ol>
<h2 id="proteins">Proteins</h2>

<p><a href="File:Peptide_syn.png" title="wikilink">200x200px|framed|right|The tRNA anticodon interacts with the mRNA codon in order to bind an amino acid to growing polypeptide chain.</a><a href="File:Charge_tRNA.png" title="wikilink">200x200px|framed|right|The process of tRNA charging</a></p>

<p>Protein synthesis occurs via a process called <a href="Translation_(biology)" title="wikilink">translation</a>.<a class="footnoteRef" href="#fn86" id="fnref86"><sup>86</sup></a> During translation, genetic material called <a class="uri" href="mRNA" title="wikilink">mRNA</a> is read by <a class="uri" href="ribosomes" title="wikilink">ribosomes</a> to generate a protein <a class="uri" href="polypeptide" title="wikilink">polypeptide</a> chain.<a class="footnoteRef" href="#fn87" id="fnref87"><sup>87</sup></a> This process requires <a href="TRNA" title="wikilink">transfer RNA</a> (tRNA) which serves as an adaptor by binding <a href="amino_acids" title="wikilink">amino acids</a> on one end and interacting with mRNA at the other end; the latter pairing between the tRNA and mRNA ensures that the correct amino acid is added to the chain.<a class="footnoteRef" href="#fn88" id="fnref88"><sup>88</sup></a> Protein synthesis occurs in three phases: initiation, elongation, and termination.<a class="footnoteRef" href="#fn89" id="fnref89"><sup>89</sup></a> <a href="Prokaryotic_translation" title="wikilink">Prokaryotic translation</a> differs from <a href="eukaryotic_translation" title="wikilink">eukaryotic translation</a>; however, this section will mostly focus on the commonalities between the two organisms.</p>
<h3 id="additional-background">Additional background</h3>

<p>Before translation can begin, the process of binding a specific amino acid to its corresponding tRNA must occur. This reaction, called tRNA charging, is catalyzed by <a href="aminoacyl_tRNA_synthetase" title="wikilink">aminoacyl tRNA synthetase</a>.<a class="footnoteRef" href="#fn90" id="fnref90"><sup>90</sup></a> A specific tRNA synthetase is responsible for recognizing and charging a particular amino acid.<a class="footnoteRef" href="#fn91" id="fnref91"><sup>91</sup></a> Furthermore, this enzyme has special discriminator regions to ensure the correct binding between tRNA and its cognate amino acid.<a class="footnoteRef" href="#fn92" id="fnref92"><sup>92</sup></a> The first step for joining an amino acid to its corresponding tRNA is the formation of aminoacyl-AMP:<a class="footnoteRef" href="#fn93" id="fnref93"><sup>93</sup></a> 

<math display="inline" id="Biosynthesis:4">
 <semantics>
  <mrow>
   <mrow>
    <mrow>
     <mi>A</mi>
     <mi>m</mi>
     <mi>i</mi>
     <mi>n</mi>
     <mpadded width="+3.3pt">
      <mi>o</mi>
     </mpadded>
     <mi>a</mi>
     <mi>c</mi>
     <mi>i</mi>
     <mi>d</mi>
    </mrow>
    <mo>+</mo>
    <mrow>
     <mi>A</mi>
     <mi>T</mi>
     <mi>P</mi>
    </mrow>
   </mrow>
   <mo>⇌</mo>
   <mrow>
    <mrow>
     <mi>a</mi>
     <mi>m</mi>
     <mi>i</mi>
     <mi>n</mi>
     <mi>o</mi>
     <mi>a</mi>
     <mi>c</mi>
     <mi>y</mi>
     <mpadded width="+3.3pt">
      <mi>l</mi>
     </mpadded>
     <mi>A</mi>
     <mi>M</mi>
     <mi>P</mi>
    </mrow>
    <mo>+</mo>
    <mrow>
     <mi>P</mi>
     <msub>
      <mi>P</mi>
      <mi>i</mi>
     </msub>
    </mrow>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <ci>normal-⇌</ci>
    <apply>
     <plus></plus>
     <apply>
      <times></times>
      <ci>A</ci>
      <ci>m</ci>
      <ci>i</ci>
      <ci>n</ci>
      <ci>o</ci>
      <ci>a</ci>
      <ci>c</ci>
      <ci>i</ci>
      <ci>d</ci>
     </apply>
     <apply>
      <times></times>
      <ci>A</ci>
      <ci>T</ci>
      <ci>P</ci>
     </apply>
    </apply>
    <apply>
     <plus></plus>
     <apply>
      <times></times>
      <ci>a</ci>
      <ci>m</ci>
      <ci>i</ci>
      <ci>n</ci>
      <ci>o</ci>
      <ci>a</ci>
      <ci>c</ci>
      <ci>y</ci>
      <ci>l</ci>
      <ci>A</ci>
      <ci>M</ci>
      <ci>P</ci>
     </apply>
     <apply>
      <times></times>
      <ci>P</ci>
      <apply>
       <csymbol cd="ambiguous">subscript</csymbol>
       <ci>P</ci>
       <ci>i</ci>
      </apply>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   Amino~{}acid+ATP\rightleftharpoons{}aminoacyl~{}AMP+PP_{i}
  </annotation>
 </semantics>
</math>

</p>

<p>This is followed by the transfer of the aminoacyl group from aminoacyl-AMP to a tRNA molecule. The resulting molecule is <a class="uri" href="aminoacyl-tRNA" title="wikilink">aminoacyl-tRNA</a>:<a class="footnoteRef" href="#fn94" id="fnref94"><sup>94</sup></a> 

<math display="inline" id="Biosynthesis:5">
 <semantics>
  <mrow>
   <mrow>
    <mrow>
     <mi>A</mi>
     <mi>m</mi>
     <mi>i</mi>
     <mi>n</mi>
     <mi>o</mi>
     <mi>a</mi>
     <mi>c</mi>
     <mi>y</mi>
     <mpadded width="+3.3pt">
      <mi>l</mi>
     </mpadded>
     <mi>A</mi>
     <mi>M</mi>
     <mi>P</mi>
    </mrow>
    <mo>+</mo>
    <mrow>
     <mi>t</mi>
     <mi>R</mi>
     <mi>N</mi>
     <mi>A</mi>
    </mrow>
   </mrow>
   <mo>⇌</mo>
   <mrow>
    <mrow>
     <mi>a</mi>
     <mi>m</mi>
     <mi>i</mi>
     <mi>n</mi>
     <mi>o</mi>
     <mi>a</mi>
     <mi>c</mi>
     <mi>y</mi>
     <mpadded width="+3.3pt">
      <mi>l</mi>
     </mpadded>
     <mi>t</mi>
     <mi>R</mi>
     <mi>N</mi>
     <mi>A</mi>
    </mrow>
    <mo>+</mo>
    <mrow>
     <mi>A</mi>
     <mi>M</mi>
     <mi>P</mi>
    </mrow>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <ci>normal-⇌</ci>
    <apply>
     <plus></plus>
     <apply>
      <times></times>
      <ci>A</ci>
      <ci>m</ci>
      <ci>i</ci>
      <ci>n</ci>
      <ci>o</ci>
      <ci>a</ci>
      <ci>c</ci>
      <ci>y</ci>
      <ci>l</ci>
      <ci>A</ci>
      <ci>M</ci>
      <ci>P</ci>
     </apply>
     <apply>
      <times></times>
      <ci>t</ci>
      <ci>R</ci>
      <ci>N</ci>
      <ci>A</ci>
     </apply>
    </apply>
    <apply>
     <plus></plus>
     <apply>
      <times></times>
      <ci>a</ci>
      <ci>m</ci>
      <ci>i</ci>
      <ci>n</ci>
      <ci>o</ci>
      <ci>a</ci>
      <ci>c</ci>
      <ci>y</ci>
      <ci>l</ci>
      <ci>t</ci>
      <ci>R</ci>
      <ci>N</ci>
      <ci>A</ci>
     </apply>
     <apply>
      <times></times>
      <ci>A</ci>
      <ci>M</ci>
      <ci>P</ci>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   Aminoacyl~{}AMP+tRNA\rightleftharpoons{}aminoacyl~{}tRNA+AMP
  </annotation>
 </semantics>
</math>

</p>

<p>The combination of these two steps, both of which are catalyzed by aminoacyl tRNA synthetase, produces a charged tRNA that is ready to add amino acids to the growing polypeptide chain.</p>

<p>In addition to binding an amino acid, tRNA has a three nucleotide unit called an <a href="Anticodon#Anticodon" title="wikilink">anticodon</a> that <a href="Basepairing" title="wikilink">base pairs</a> with specific nucleotide triplets on the mRNA called <a class="uri" href="codons" title="wikilink">codons</a>; codons encode a specific amino acid.<a class="footnoteRef" href="#fn95" id="fnref95"><sup>95</sup></a> This interaction is possible thanks to the ribosome, which serves as the site for protein synthesis. The ribosome possesses three tRNA binding sites: the aminoacyl site (A site), the peptidyl site (P site), and the exit site (E site).<a class="footnoteRef" href="#fn96" id="fnref96"><sup>96</sup></a></p>

<p>There are numerous codons within an mRNA transcript, and it is very common for an amino acid to be specified by more than one codon; this phenomenon is called <a href="Codon_degeneracy" title="wikilink">degeneracy</a>.<a class="footnoteRef" href="#fn97" id="fnref97"><sup>97</sup></a> In all, there are 64 codons, 61 of each code for one of the 20 amino acids, while the remaining codons specify chain termination.<a class="footnoteRef" href="#fn98" id="fnref98"><sup>98</sup></a></p>
<h3 id="translation-in-steps">Translation in steps</h3>

<p>As previously mentioned, translation occurs in three phases: initiation, elongation, and termination. <a href="File:TRNA_ribosomes_diagram_en.svg" title="wikilink">311x440px|framed|left|Translation</a></p>
<h4 id="step-1-initiation">Step 1: Initiation</h4>

<p>The completion of the initiation phase is dependent on the following three events:<a class="footnoteRef" href="#fn99" id="fnref99"><sup>99</sup></a></p>

<p>1. The recruitment of the ribosome to mRNA</p>

<p>2. The binding of a charged initiator tRNA into the P site of the ribosome</p>

<p>3. The proper alignment of the ribosome with mRNA’s start codon</p>
<h4 id="step-2-elongation">Step 2: Elongation</h4>

<p>Following initiation, the polypeptide chain is extended via anticodon:codon interactions, with the ribosome adding amino acids to the polypeptide chain one at a time. The following steps must occur to ensure the correct addition of amino acids:<a class="footnoteRef" href="#fn100" id="fnref100"><sup>100</sup></a></p>

<p>1. The binding of the correct tRNA into the A site of the ribosome</p>

<p>2. The formation of a <a href="peptide_bond" title="wikilink">peptide bond</a> between the tRNA in the A site and the polypeptide chain attached to the tRNA in the P site</p>

<p>3. <a href="Protein_translocation#Protein_translocation" title="wikilink">Translocation</a> or advancement of the tRNA-mRNA complex by three nucleotides</p>

<p>Translocation “kicks off” the tRNA at the E site and shifts the tRNA from the A site into the P site, leaving the A site free for an incoming tRNA to add another amino acid.</p>
<h4 id="step-3-termination">Step 3: Termination</h4>

<p>The last stage of translation occurs when a <a href="stop_codon" title="wikilink">stop codon</a> enters the A site.<a class="footnoteRef" href="#fn101" id="fnref101"><sup>101</sup></a> Then, the following steps occur:</p>

<p>1. The recognition of codons by <a href="release_factor" title="wikilink">release factors</a>, which causes the <a class="uri" href="hydrolysis" title="wikilink">hydrolysis</a> of the polypeptide chain from the tRNA located in the P site<a class="footnoteRef" href="#fn102" id="fnref102"><sup>102</sup></a></p>

<p>2. The release of the polypeptide chain <a class="footnoteRef" href="#fn103" id="fnref103"><sup>103</sup></a></p>

<p>3. The dissociation and "recycling" of the ribosome for future translation processes <a class="footnoteRef" href="#fn104" id="fnref104"><sup>104</sup></a></p>

<p>A summary table of the key players in translation is found below:</p>
<table>
<tbody>
<tr class="odd">
<td style="text-align: left;"></td>
<td style="text-align: left;">
<p><strong>Key players in Translation</strong></p></td>
<td style="text-align: left;">
<p><strong>Translation Stage</strong></p></td>
<td style="text-align: left;">
<p><strong>Purpose</strong></p></td>
</tr>
<tr class="even">
<td style="text-align: left;">
<p>tRNA synthetase'''</p></td>
<td style="text-align: left;">
<p>before initiation</p></td>
<td style="text-align: left;">
<p>Responsible for tRNA charging</p></td>
</tr>
<tr class="odd">
<td style="text-align: left;">
<p>mRNA'''</p></td>
<td style="text-align: left;">
<p>initiation, elongation, termination</p></td>
<td style="text-align: left;">
<p>Template for protein synthesis; contains regions named codons which encode amino acids</p></td>
</tr>
<tr class="even">
<td style="text-align: left;">
<p>tRNA'''</p></td>
<td style="text-align: left;">
<p>initiation, elongation, termination</p></td>
<td style="text-align: left;">
<p>Binds ribosomes sites A, P, E; anticodon base pairs with mRNA codon to ensure that the correct amino acid is incorporated into the growing polypeptide chain</p></td>
</tr>
<tr class="odd">
<td style="text-align: left;">
<p>ribosome'''</p></td>
<td style="text-align: left;">
<p>initiation, elongation, termination</p></td>
<td style="text-align: left;">
<p>Directs protein synthesis and catalyzes the formation of the peptide bond</p></td>
</tr>
<tr class="even">
</tr>
</tbody>
</table>
<h2 id="diseases-associated-with-macromolecule-deficiency">Diseases associated with macromolecule deficiency</h2>

<p> Errors in biosynthetic pathways can have deleterious consequences including the malformation of macromolecules or the underproduction of functional molecules. Below are examples that illustrate the disruptions that occur due to these inefficiencies.</p>
<ul>
<li><a href="Familial_hypercholesterolemia" title="wikilink">Familial hypercholesterolemia</a>: this disorder is characterized by the absence of functional <a href="Receptor_(biochemistry)" title="wikilink">receptors</a> for <a class="uri" href="LDL" title="wikilink">LDL</a>.<a class="footnoteRef" href="#fn105" id="fnref105"><sup>105</sup></a> Deficiencies in the formation of LDL receptors may cause faulty receptors which disrupt the <a href="Endocytosis" title="wikilink">endocytic</a> pathway, inhibiting the entry of LDL into the liver and other cells.<a class="footnoteRef" href="#fn106" id="fnref106"><sup>106</sup></a> This causes a buildup of LDL in the blood plasma, which results in <a href="atherosclerotic_plaques" title="wikilink">atherosclerotic plaques</a> that narrow arteries and increase the risk of heart attacks.<a class="footnoteRef" href="#fn107" id="fnref107"><sup>107</sup></a></li>
</ul>
<ul>
<li><a href="Lesch-Nyhan_syndrome" title="wikilink">Lesch-Nyhan syndrome</a>: this genetic disease is characterized by <a href="Self_mutilation" title="wikilink">self- mutilation</a>, mental deficiency, and <a class="uri" href="gout" title="wikilink">gout</a>.<a class="footnoteRef" href="#fn108" id="fnref108"><sup>108</sup></a> It is caused by the absence of <a href="hypoxanthine-guanine_phosphoribosyltransferase" title="wikilink">hypoxanthine-guanine phosphoribosyltransferase</a>, which is a necessary enzyme for purine nucleotide formation.<a class="footnoteRef" href="#fn109" id="fnref109"><sup>109</sup></a> The lack of enzyme reduces the level of necessary nucleotides and causes the accumulation of biosynthesis <a href="Reaction_intermediate" title="wikilink">intermediates</a>, which results in the aforementioned unusual behavior.<a class="footnoteRef" href="#fn110" id="fnref110"><sup>110</sup></a></li>
</ul>
<ul>
<li><a href="Severe_combined_immunodeficiency" title="wikilink">Severe combined immunodeficiency (SCID)</a>: SCID is characterized by a loss of <a href="T_cells" title="wikilink">T cells</a>.<a class="footnoteRef" href="#fn111" id="fnref111"><sup>111</sup></a> Shortage of these immune system components increases the susceptibility to infectious agents because the affected individuals cannot develop <a href="Immunological_memory" title="wikilink">immunological</a> memory.<a class="footnoteRef" href="#fn112" id="fnref112"><sup>112</sup></a> This immunological disorder results from a deficiency in <a href="Adenosine_deaminase" title="wikilink">adenosine deanimase</a> activity, which causes a buildup of <a class="uri" href="dATP" title="wikilink">dATP</a>. These dATP molecules then inhibit ribonucleotide reductase, which prevents of DNA synthesis.<a class="footnoteRef" href="#fn113" id="fnref113"><sup>113</sup></a></li>
</ul>
<ul>
<li><a href="Huntingtons_disease" title="wikilink">Huntington’s disease</a>: this <a class="uri" href="neurological" title="wikilink">neurological</a> disease is caused from errors that occur during DNA synthesis.<a class="footnoteRef" href="#fn114" id="fnref114"><sup>114</sup></a> These errors or mutations lead to the expression of a mutant <a class="uri" href="huntingtin" title="wikilink">huntingtin</a> protein, which contains repetitive <a class="uri" href="glutamine" title="wikilink">glutamine</a> residues that are encoded by expanding <a href="Trinucleotide_repeat#CAG_Repeats" title="wikilink">CAG trinucleotide repeats</a> in the gene.<a class="footnoteRef" href="#fn115" id="fnref115"><sup>115</sup></a> Huntington’s disease is characterized by neuronal loss and <a class="uri" href="gliosis" title="wikilink">gliosis</a>. Symptoms of the disease include: movement disorder, <a class="uri" href="cognitive" title="wikilink">cognitive</a> decline, and behavioral disorder.<a class="footnoteRef" href="#fn116" id="fnref116"><sup>116</sup></a></li>
</ul>
<h2 id="see-also">See also</h2>
<ul>
<li><a class="uri" href="Lipids" title="wikilink">Lipids</a></li>
<li><a href="Phosopholipid_bilayer" title="wikilink">Phospholipid bilayer</a></li>
<li><a class="uri" href="Nucleotides" title="wikilink">Nucleotides</a></li>
<li><a class="uri" href="DNA" title="wikilink">DNA</a></li>
<li><a href="DNA_replication" title="wikilink">DNA replication</a></li>
<li><a href="Amino_acids" title="wikilink">Amino acids</a></li>
<li><a href="Proteinogenic_amino_acid" title="wikilink">Proteinogenic amino acid</a></li>
<li><a href="Codon_table#RNA_codon_table" title="wikilink">Codon table</a></li>
<li><a class="uri" href="Proteins" title="wikilink">Proteins</a></li>
<li><a href="Translation_(biology)" title="wikilink">Translation</a></li>
</ul>
<h2 id="references">References</h2>

<p>"</p>

<p><a class="uri" href="Category:Metabolism" title="wikilink">Category:Metabolism</a> <a href="Category:Biosynthesis" title="wikilink"> </a></p>
<section class="footnotes">
<hr/>
<ol>
<li id="fn1"></li>
<li id="fn2"><a href="#fnref2">↩</a></li>
<li id="fn3"><a href="#fnref3">↩</a></li>
<li id="fn4"><a href="#fnref4">↩</a></li>
<li id="fn5"></li>
<li id="fn6"></li>
<li id="fn7"><a href="#fnref7">↩</a></li>
<li id="fn8"><a href="#fnref8">↩</a></li>
<li id="fn9"></li>
<li id="fn10"></li>
<li id="fn11"><a href="#fnref11">↩</a></li>
<li id="fn12"></li>
<li id="fn13"><a href="#fnref13">↩</a></li>
<li id="fn14"></li>
<li id="fn15"></li>
<li id="fn16"></li>
<li id="fn17"><a href="#fnref17">↩</a></li>
<li id="fn18"></li>
<li id="fn19"></li>
<li id="fn20"><a href="#fnref20">↩</a></li>
<li id="fn21"></li>
<li id="fn22"><a href="#fnref22">↩</a></li>
<li id="fn23"></li>
<li id="fn24"></li>
<li id="fn25"><a href="#fnref25">↩</a></li>
<li id="fn26"></li>
<li id="fn27"></li>
<li id="fn28"></li>
<li id="fn29"><a href="#fnref29">↩</a></li>
<li id="fn30"></li>
<li id="fn31"></li>
<li id="fn32"><a href="#fnref32">↩</a></li>
<li id="fn33"><a href="#fnref33">↩</a></li>
<li id="fn34"><a href="#fnref34">↩</a></li>
<li id="fn35"><a href="#fnref35">↩</a></li>
<li id="fn36"></li>
<li id="fn37"></li>
<li id="fn38"></li>
<li id="fn39"></li>
<li id="fn40"></li>
<li id="fn41"></li>
<li id="fn42"></li>
<li id="fn43"></li>
<li id="fn44"><a href="#fnref44">↩</a></li>
<li id="fn45"><a href="#fnref45">↩</a></li>
<li id="fn46"></li>
<li id="fn47"><a href="#fnref47">↩</a></li>
<li id="fn48"></li>
<li id="fn49"></li>
<li id="fn50"></li>
<li id="fn51"></li>
<li id="fn52"></li>
<li id="fn53"></li>
<li id="fn54"><a href="#fnref54">↩</a></li>
<li id="fn55"><a href="#fnref55">↩</a></li>
<li id="fn56"></li>
<li id="fn57"><a href="#fnref57">↩</a></li>
<li id="fn58"><a href="#fnref58">↩</a></li>
<li id="fn59"><a href="#fnref59">↩</a></li>
<li id="fn60"><a href="#fnref60">↩</a></li>
<li id="fn61"><a href="#fnref61">↩</a></li>
<li id="fn62"><a href="#fnref62">↩</a></li>
<li id="fn63"><a href="#fnref63">↩</a></li>
<li id="fn64"><a href="#fnref64">↩</a></li>
<li id="fn65"></li>
<li id="fn66"><a href="#fnref66">↩</a></li>
<li id="fn67"><a href="#fnref67">↩</a></li>
<li id="fn68"><a href="#fnref68">↩</a></li>
<li id="fn69"><a href="#fnref69">↩</a></li>
<li id="fn70"><a href="#fnref70">↩</a></li>
<li id="fn71"><a href="#fnref71">↩</a></li>
<li id="fn72"><a href="#fnref72">↩</a></li>
<li id="fn73"></li>
<li id="fn74"><a href="#fnref74">↩</a></li>
<li id="fn75"><a href="#fnref75">↩</a></li>
<li id="fn76"><a href="#fnref76">↩</a></li>
<li id="fn77"><a href="#fnref77">↩</a></li>
<li id="fn78"><a href="#fnref78">↩</a></li>
<li id="fn79"><a href="#fnref79">↩</a></li>
<li id="fn80"><a href="#fnref80">↩</a></li>
<li id="fn81"><a href="#fnref81">↩</a></li>
<li id="fn82"><a href="#fnref82">↩</a></li>
<li id="fn83"><a href="#fnref83">↩</a></li>
<li id="fn84"><a href="#fnref84">↩</a></li>
<li id="fn85"><a href="#fnref85">↩</a></li>
<li id="fn86"><a href="#fnref86">↩</a></li>
<li id="fn87"></li>
<li id="fn88"></li>
<li id="fn89"></li>
<li id="fn90"><a href="#fnref90">↩</a></li>
<li id="fn91"></li>
<li id="fn92"></li>
<li id="fn93"></li>
<li id="fn94"></li>
<li id="fn95"><a href="#fnref95">↩</a></li>
<li id="fn96"><a href="#fnref96">↩</a></li>
<li id="fn97"><a href="#fnref97">↩</a></li>
<li id="fn98"></li>
<li id="fn99"></li>
<li id="fn100"><a href="#fnref100">↩</a></li>
<li id="fn101"></li>
<li id="fn102"></li>
<li id="fn103"></li>
<li id="fn104"></li>
<li id="fn105"><a href="#fnref105">↩</a></li>
<li id="fn106"></li>
<li id="fn107"></li>
<li id="fn108"><a href="#fnref108">↩</a></li>
<li id="fn109"></li>
<li id="fn110"></li>
<li id="fn111"><a href="#fnref111">↩</a></li>
<li id="fn112"></li>
<li id="fn113"></li>
<li id="fn114"><a href="#fnref114">↩</a></li>
<li id="fn115"></li>
<li id="fn116"><a href="#fnref116">↩</a></li>
</ol>
</section>
</body>
</html>
