<html lang="en">
<head>
<meta charset="utf-8"/>
<title offset="1191">Cellular noise</title>
<script src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_SVG.js" type="text/javascript">
</script>
</head>
<body>
<h1>Cellular noise</h1>
<hr/>

<p><strong>Cellular noise</strong> is random variability in quantities arising in <a href="cellular_biology" title="wikilink">cellular biology</a>. For example, cells which are genetically identical, even within the same tissue, are often observed to have different expression levels of proteins, different sizes and structures.<a class="footnoteRef" href="#fn1" id="fnref1"><sup>1</sup></a><a class="footnoteRef" href="#fn2" id="fnref2"><sup>2</sup></a> These apparently random differences can have important biological and medical consequences.<a class="footnoteRef" href="#fn3" id="fnref3"><sup>3</sup></a></p>

<p>Cellular noise was originally, and is still often, examined in the context of gene expression levels ‚Äì either the concentration or copy number of the products of genes within and between cells. As <a href="gene_expression" title="wikilink">gene expression</a> levels are responsible for many fundamental properties in cellular biology, including cells' physical appearance, behaviour in response to stimuli, and ability to process information and control internal processes, the presence of noise in gene expression has profound implications for many processes in cellular biology.</p>
<h2 id="definitions">Definitions</h2>

<p>The most frequent quantitative definition of noise is the <a href="coefficient_of_variation" title="wikilink">coefficient of variation</a>:</p>

<p>

<math display="block" id="Cellular_noise:0">
 <semantics>
  <mrow>
   <mrow>
    <msub>
     <mi>Œ∑</mi>
     <mi>X</mi>
    </msub>
    <mo>=</mo>
    <mfrac>
     <msub>
      <mi>œÉ</mi>
      <mi>X</mi>
     </msub>
     <msub>
      <mi>Œº</mi>
      <mi>X</mi>
     </msub>
    </mfrac>
   </mrow>
   <mo>,</mo>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <eq></eq>
    <apply>
     <csymbol cd="ambiguous">subscript</csymbol>
     <ci>Œ∑</ci>
     <ci>X</ci>
    </apply>
    <apply>
     <divide></divide>
     <apply>
      <csymbol cd="ambiguous">subscript</csymbol>
      <ci>œÉ</ci>
      <ci>X</ci>
     </apply>
     <apply>
      <csymbol cd="ambiguous">subscript</csymbol>
      <ci>Œº</ci>
      <ci>X</ci>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \eta_{X}=\frac{\sigma_{X}}{\mu_{X}},
  </annotation>
 </semantics>
</math>

</p>

<p>where 

<math display="inline" id="Cellular_noise:1">
 <semantics>
  <msub>
   <mi>Œ∑</mi>
   <mi>X</mi>
  </msub>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <csymbol cd="ambiguous">subscript</csymbol>
    <ci>Œ∑</ci>
    <ci>X</ci>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \eta_{X}
  </annotation>
 </semantics>
</math>

 is the noise in a quantity 

<math display="inline" id="Cellular_noise:2">
 <semantics>
  <mi>X</mi>
  <annotation-xml encoding="MathML-Content">
   <ci>X</ci>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   X
  </annotation>
 </semantics>
</math>

, 

<math display="inline" id="Cellular_noise:3">
 <semantics>
  <msub>
   <mi>Œº</mi>
   <mi>X</mi>
  </msub>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <csymbol cd="ambiguous">subscript</csymbol>
    <ci>Œº</ci>
    <ci>X</ci>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \mu_{X}
  </annotation>
 </semantics>
</math>

 is the <a class="uri" href="mean" title="wikilink">mean</a> value of 

<math display="inline" id="Cellular_noise:4">
 <semantics>
  <mi>X</mi>
  <annotation-xml encoding="MathML-Content">
   <ci>X</ci>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   X
  </annotation>
 </semantics>
</math>

 and 

<math display="inline" id="Cellular_noise:5">
 <semantics>
  <msub>
   <mi>œÉ</mi>
   <mi>X</mi>
  </msub>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <csymbol cd="ambiguous">subscript</csymbol>
    <ci>œÉ</ci>
    <ci>X</ci>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \sigma_{X}
  </annotation>
 </semantics>
</math>

 is the <a href="standard_deviation" title="wikilink">standard deviation</a> of 

<math display="inline" id="Cellular_noise:6">
 <semantics>
  <mi>X</mi>
  <annotation-xml encoding="MathML-Content">
   <ci>X</ci>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   X
  </annotation>
 </semantics>
</math>

. This measure is <a class="uri" href="dimensionless" title="wikilink">dimensionless</a>, allowing a relative comparison of the importance of noise, without necessitating knowledge of the absolute mean.</p>

<p>Another quantity often used for mathematical convenience is the <a href="Fano_factor" title="wikilink">Fano factor</a>:</p>

<p>

<math display="block" id="Cellular_noise:7">
 <semantics>
  <mrow>
   <mrow>
    <msub>
     <mi>F</mi>
     <mi>X</mi>
    </msub>
    <mo>=</mo>
    <mfrac>
     <msubsup>
      <mi>œÉ</mi>
      <mi>X</mi>
      <mn>2</mn>
     </msubsup>
     <msub>
      <mi>Œº</mi>
      <mi>X</mi>
     </msub>
    </mfrac>
   </mrow>
   <mo>.</mo>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <eq></eq>
    <apply>
     <csymbol cd="ambiguous">subscript</csymbol>
     <ci>F</ci>
     <ci>X</ci>
    </apply>
    <apply>
     <divide></divide>
     <apply>
      <csymbol cd="ambiguous">superscript</csymbol>
      <apply>
       <csymbol cd="ambiguous">subscript</csymbol>
       <ci>œÉ</ci>
       <ci>X</ci>
      </apply>
      <cn type="integer">2</cn>
     </apply>
     <apply>
      <csymbol cd="ambiguous">subscript</csymbol>
      <ci>Œº</ci>
      <ci>X</ci>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   F_{X}=\frac{\sigma_{X}^{2}}{\mu_{X}}.
  </annotation>
 </semantics>
</math>

</p>
<h2 id="intrinsic-and-extrinsic-noise">Intrinsic and extrinsic noise</h2>
<figure><b>(Figure)</b>
<figcaption>A schematic illustration of a dual reporter study. Each data point corresponds to a measurement of the expression level of two identically-regulated genes in a single cell: the scatter reflects measurements of a population of cells. Extrinsic noise is characterised by expression levels of both genes covarying between cells, intrinsic by internal differences.</figcaption>
</figure>

<p>Cellular noise is often investigated in the framework of <em>intrinsic</em> and <em>extrinsic</em> noise. Intrinsic noise refers to variation in identically-regulated quantities within a single cell: for example, the intra-cell variation in expression levels of two identically-controlled genes. Extrinsic noise refers to variation in identically-regulated quantities between different cells: for example, the cell-to-cell variation in expression of a given gene.</p>

<p>Intrinsic and extrinsic noise levels are often compared in <strong>dual reporter studies</strong>, in which the expression levels of two identically-regulated genes (often fluorescent reporters like <a href="Green_fluorescent_protein" title="wikilink">GFP</a> and <a href="Yellow_fluorescent_protein" title="wikilink">YFP</a>) are plotted for each cell in a population.<a class="footnoteRef" href="#fn4" id="fnref4"><sup>4</sup></a></p>
<h2 id="sources-of-cellular-noise">Sources of cellular noise</h2>

<p><em>Note</em>: These lists are illustrative, not exhaustive, and identification of noise sources is an active and expanding area of research.</p>
<dl>
<dt>Intrinsic noise</dt>
</dl>
<ul>
<li><em>Low copy-number effects (including discrete birth and death events)</em>: the random nature of production and degradation of cellular components means that noise is high for components at low copy number (as the magnitude of these random fluctuations is not negligible with respect to the copy number);</li>
<li><em>Diffusive cellular dynamics</em>:<a class="footnoteRef" href="#fn5" id="fnref5"><sup>5</sup></a> many important cellular processes rely on collisions between reactants (for example, RNA polymerase and DNA) and other physical criteria which, given the <a href="Diffusion" title="wikilink">diffusive</a> dynamic nature of the cell, occur stochastically.</li>
<li><em>Noise propagation</em>: Low copy-number effects and diffusive dynamics result in each of the biochemical reactions in a cell occurring randomly. Stochasticity of reactions can be either attenuated or amplified. Contribution each reaction makes to the intrinsic variability in copy numbers can be quantified via <a href="system_size_expansion" title="wikilink">Van Kampen's system size expansion</a>.<a class="footnoteRef" href="#fn6" id="fnref6"><sup>6</sup></a><a class="footnoteRef" href="#fn7" id="fnref7"><sup>7</sup></a></li>
</ul>
<dl>
<dt>Extrinsic noise</dt>
</dl>
<ul>
<li><em>Cellular age / cell cycle stage</em>: cells in a dividing population that is not synchronised will, at a given snapshot in time, be at different <a href="cell_cycle" title="wikilink">cell cycle</a> stages, with corresponding biochemical and physical differences;<a class="footnoteRef" href="#fn8" id="fnref8"><sup>8</sup></a></li>
<li><em>Physical environment (temperature, pressure, ...)</em>: physical quantities and chemical concentrations (particularly in the case of cell-to-cell signalling) may vary spatially across a population of cells, provoking extrinsic differences as a function of position;<a class="footnoteRef" href="#fn9" id="fnref9"><sup>9</sup></a></li>
<li><em>Organelle distributions</em>: random factors in the quantity and quality of organelles (for example, the number and functionality of <a class="uri" href="mitochondria" title="wikilink">mitochondria</a><a class="footnoteRef" href="#fn10" id="fnref10"><sup>10</sup></a>) lead to significant cell-to-cell differences in a range of processes<a class="footnoteRef" href="#fn11" id="fnref11"><sup>11</sup></a> (as, for example, mitochondria play a central role in the energy budget of eukaryotic cells);</li>
<li><em>Inheritance noise</em>: uneven partitioning of cellular components between daughter cells at <a class="uri" href="mitosis" title="wikilink">mitosis</a> can result in large extrinsic differences in a dividing population.<a class="footnoteRef" href="#fn12" id="fnref12"><sup>12</sup></a></li>
</ul>

<p>Note that extrinsic noise can affect levels and types of intrinsic noise:<a class="footnoteRef" href="#fn13" id="fnref13"><sup>13</sup></a> for example, extrinsic differences in the mitochondrial content of cells lead, through differences in <a href="Adenosine_triphosphate" title="wikilink">ATP</a> levels, to some cells transcribing faster than others, affecting the rates of gene expression and the magnitude of intrinsic noise across the population.<a class="footnoteRef" href="#fn14" id="fnref14"><sup>14</sup></a></p>
<h2 id="effects-of-cellular-noise">Effects of cellular noise</h2>

<p><em>Note</em>: These lists are illustrative, not exhaustive, and identification of noise effects is an active and expanding area of research.</p>
<ul>
<li><em>Gene expression levels</em>: noise in gene expression causes differences in the fundamental properties of cells, limits their ability to biochemically control cellular dynamics,<a class="footnoteRef" href="#fn15" id="fnref15"><sup>15</sup></a> and directly or indirectly induce many of the specific effects below;</li>
<li><em>Energy levels and transcription rate</em>: <a href="Transcriptional_noise" title="wikilink">noise in transcription rate</a>, arising from sources including <a href="transcriptional_bursting" title="wikilink">transcriptional bursting</a>, is a significant source of noise in expression levels of genes. Extrinsic noise in <a href="mitochondria" title="wikilink">mitochondrial</a> content has been suggested to propagate to differences in the ATP concentrations and transcription rates (with functional relationships implied between these three quantities) in cells, affecting cells' energetic competence and ability to express genes;<a class="footnoteRef" href="#fn16" id="fnref16"><sup>16</sup></a></li>
<li><em>Phenotype selection</em>: bacterial populations exploit extrinsic noise to choose a population subset to enter a quiescent state.<a class="footnoteRef" href="#fn17" id="fnref17"><sup>17</sup></a> In a bacterial infection, for example, this subset will not propagate quickly but will be more robust when the population is threatened by antibiotic treatment: the rapidly replicating, infectious bacteria will be killed more quickly than the quiescent subset, which may be capable of restarting the infection. This phenomenon is why courses of antibiotics should be finished even when symptoms seem to have disappeared;</li>
<li><em>Development and stem cell differentiation</em>: <a href="developmental_noise" title="wikilink">developmental noise</a> in biochemical processes which need to be tightly controlled (for example, patterning of gene expression levels that develop into different body parts) during organismal development can have dramatic consequences, necessitating the evolution of robust cellular machinery. <a href="Stem_cells" title="wikilink">Stem cells</a> differentiate into different cell types depending on the expression levels of various characteristic genes:<a class="footnoteRef" href="#fn18" id="fnref18"><sup>18</sup></a> noise in gene expression can clearly perturb and influence this process, and noise in transcription rate can affect the structure of the dynamic landscape that differentiation occurs on;<a class="footnoteRef" href="#fn19" id="fnref19"><sup>19</sup></a></li>
<li><em>Cancer treatments</em>: recent work has found extrinsic differences, linked to gene expression levels, in the response of cancer cells to anti-cancer treatments, potentially linking the phenomenon of fractional killing (whereby each treatment kills some but not all of a tumour) to noise in gene expression.<a class="footnoteRef" href="#fn20" id="fnref20"><sup>20</sup></a> Because individual cells could repeatedly and stochastically perform transitions between states associated with differences in responsiveness to a therapeutic modality (chemotherapy, targeted agent, radiation, etc.), therapy might need to be administered frequently (to ensure cells are treated soon after entering a therapy-responsive state, before they can rejoin the therapy-resistant subpopulation and proliferate) and over long times (to treat even those cells emerging late from the final residue of the therapy-resistant subpopulation).<a class="footnoteRef" href="#fn21" id="fnref21"><sup>21</sup></a></li>
<li><em>Information processing</em>: as cellular regulation is performed with components that are themselves subject to noise, the ability of cells to process information and perform control is fundamentally limited by intrinsic noise<a class="footnoteRef" href="#fn22" id="fnref22"><sup>22</sup></a><a class="footnoteRef" href="#fn23" id="fnref23"><sup>23</sup></a></li>
</ul>
<h2 id="analysis">Analysis</h2>
<figure><b>(Figure)</b>
<figcaption>A canonical model for stochastic gene expression. DNA flips between "inactive" and "active" states (involving, for example, <a href="chromatin" title="wikilink">chromatin remodelling</a> and <a href="transcription_factor" title="wikilink">transcription factor</a> binding). Active DNA is transcribed to produce mRNA which is translated to produce protein, both of which are degraded. All processes are <a href="Poisson_process" title="wikilink">Poissonian</a> with given rates.</figcaption>
</figure>

<p>As many quantities of cell biological interest are present in discrete copy number within the cell (single DNAs, dozens of mRNAs, hundreds of proteins), tools from discrete <a href="Stochastic_process" title="wikilink">stochastic mathematics</a> are often used to analyse and model cellular noise.<a class="footnoteRef" href="#fn24" id="fnref24"><sup>24</sup></a><a class="footnoteRef" href="#fn25" id="fnref25"><sup>25</sup></a> In particular, <a href="master_equation" title="wikilink">master equation</a> treatments ‚Äì where the probabilities 

<math display="inline" id="Cellular_noise:8">
 <semantics>
  <mrow>
   <mi>P</mi>
   <mrow>
    <mo stretchy="false">(</mo>
    <mi>ùê±</mi>
    <mo>,</mo>
    <mi>t</mi>
    <mo stretchy="false">)</mo>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <times></times>
    <ci>P</ci>
    <interval closure="open">
     <ci>ùê±</ci>
     <ci>t</ci>
    </interval>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   P(\mathbf{x},t)
  </annotation>
 </semantics>
</math>

 of observing a system in a state 

<math display="inline" id="Cellular_noise:9">
 <semantics>
  <mi>ùê±</mi>
  <annotation-xml encoding="MathML-Content">
   <ci>ùê±</ci>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \mathbf{x}
  </annotation>
 </semantics>
</math>

 at time 

<math display="inline" id="Cellular_noise:10">
 <semantics>
  <mi>t</mi>
  <annotation-xml encoding="MathML-Content">
   <ci>t</ci>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   t
  </annotation>
 </semantics>
</math>

 are linked through <a href="Ordinary_differential_equation" title="wikilink">ODEs</a> ‚Äì have proved particularly fruitful. A canonical model for noise gene expression, where the processes of <a href="Transcription_factor" title="wikilink">DNA activation</a>, <a href="Transcription_(genetics)" title="wikilink">transcription</a> and <a href="Translation_(biology)" title="wikilink">translation</a> are all represented as <a href="Poisson_processes" title="wikilink">Poisson processes</a> with given rates, gives a master equation which may be solved exactly (with <a href="generating_functions" title="wikilink">generating functions</a>) under various assumptions or approximated with stochastic tools like <a href="system_size_expansion" title="wikilink">Van Kampen's system size expansion</a>.</p>

<p>Numerically, the <a href="Gillespie_algorithm" title="wikilink">Gillespie algorithm</a> or stochastic simulation algorithm is often used to create realisations of stochastic cellular processes, from which statistics can be calculated.</p>

<p>The problem of inferring the values of parameters in stochastic models (<a href="Parametric_statistics" title="wikilink">parametric inference</a>) for biological processes, which are typically characterised by sparse and noisy experimental data, is an active field of research, with methods including <a href="Markov_chain_Monte_Carlo" title="wikilink">Bayesian MCMC</a> and <a href="approximate_Bayesian_computation" title="wikilink">approximate Bayesian computation</a> proving adaptable and robust. </p>
<h2 id="references">References</h2>

<p> <a class="footnoteRef" href="#fn26" id="fnref26"><sup>26</sup></a> <a class="footnoteRef" href="#fn27" id="fnref27"><sup>27</sup></a> <a class="footnoteRef" href="#fn28" id="fnref28"><sup>28</sup></a> <a class="footnoteRef" href="#fn29" id="fnref29"><sup>29</sup></a> <a class="footnoteRef" href="#fn30" id="fnref30"><sup>30</sup></a> <a class="footnoteRef" href="#fn31" id="fnref31"><sup>31</sup></a> <a class="footnoteRef" href="#fn32" id="fnref32"><sup>32</sup></a> <a class="footnoteRef" href="#fn33" id="fnref33"><sup>33</sup></a> <a class="footnoteRef" href="#fn34" id="fnref34"><sup>34</sup></a> <a class="footnoteRef" href="#fn35" id="fnref35"><sup>35</sup></a> <a class="footnoteRef" href="#fn36" id="fnref36"><sup>36</sup></a> <a class="footnoteRef" href="#fn37" id="fnref37"><sup>37</sup></a> <a class="footnoteRef" href="#fn38" id="fnref38"><sup>38</sup></a> }}</p>

<p>"</p>

<p><a href="Category:Cell_biology" title="wikilink">Category:Cell biology</a> <a class="uri" href="Category:Biophysics" title="wikilink">Category:Biophysics</a> <a href="Category:Molecular_biology" title="wikilink">Category:Molecular biology</a> <a class="uri" href="Category:Biostatistics" title="wikilink">Category:Biostatistics</a> <a class="uri" href="Category:Noise" title="wikilink">Category:Noise</a></p>
<section class="footnotes">
<hr/>
<ol>
<li id="fn1"></li>
<li id="fn2"></li>
<li id="fn3"></li>
<li id="fn4"></li>
<li id="fn5"></li>
<li id="fn6"></li>
<li id="fn7"></li>
<li id="fn8"></li>
<li id="fn9"></li>
<li id="fn10"></li>
<li id="fn11"></li>
<li id="fn12"></li>
<li id="fn13"></li>
<li id="fn14"></li>
<li id="fn15"></li>
<li id="fn16"></li>
<li id="fn17"></li>
<li id="fn18"></li>
<li id="fn19"></li>
<li id="fn20"></li>
<li id="fn21"></li>
<li id="fn22"></li>
<li id="fn23"></li>
<li id="fn24"></li>
<li id="fn25"></li>
<li id="fn26"><a href="#fnref26">‚Ü©</a></li>
<li id="fn27"><a href="#fnref27">‚Ü©</a></li>
<li id="fn28"><a href="#fnref28">‚Ü©</a></li>
<li id="fn29"><a href="#fnref29">‚Ü©</a></li>
<li id="fn30"><a href="#fnref30">‚Ü©</a></li>
<li id="fn31"><a href="#fnref31">‚Ü©</a></li>
<li id="fn32"><a href="#fnref32">‚Ü©</a></li>
<li id="fn33"><a href="#fnref33">‚Ü©</a></li>
<li id="fn34"><a href="#fnref34">‚Ü©</a></li>
<li id="fn35"><a href="#fnref35">‚Ü©</a></li>
<li id="fn36"><a href="#fnref36">‚Ü©</a></li>
<li id="fn37"><a href="#fnref37">‚Ü©</a></li>
<li id="fn38"><a href="#fnref38">‚Ü©</a></li>
</ol>
</section>
</body>
</html>
