<html lang="en">
<head>
<meta charset="utf-8"/>
<title offset="1954">Chorismate lyase</title>
<script src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_SVG.js" type="text/javascript">
</script>
</head>
<body>
<h1>Chorismate lyase</h1>
<hr/>

<p>In <a class="uri" href="enzymology" title="wikilink">enzymology</a>, a <strong>chorismate lyase</strong> () is an <a class="uri" href="enzyme" title="wikilink">enzyme</a> that <a href="catalysis" title="wikilink">catalyzes</a> the <a href="chemical_reaction" title="wikilink">chemical reaction</a></p>
<dl>
<dd>chorismate 

<math display="inline" id="Chorismate_lyase:0">
 <semantics>
  <mo>⇌</mo>
  <annotation-xml encoding="MathML-Content">
   <ci>normal-⇌</ci>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \rightleftharpoons
  </annotation>
 </semantics>
</math>

 4-hydroxybenzoate + pyruvate
</dd>
<dd><embed src="72 2015 05 08 CPL.tif" title="fig:The chorismate pyruvate lyase (CPL) catalyzed reaction.|thumb|none"></embed>
</dd>
<dd>
</dd>
<dd>
</dd>
<dd>
</dd>
<dd>
</dd>
<dd>
</dd>
</dl>

<p>Hence, this enzyme has one <a href="substrate_(biochemistry)" title="wikilink">substrate</a>, <a class="uri" href="chorismate" title="wikilink">chorismate</a>, and two <a href="product_(chemistry)" title="wikilink">products</a>, <a class="uri" href="4-hydroxybenzoate" title="wikilink">4-hydroxybenzoate</a> and <a class="uri" href="pyruvate" title="wikilink">pyruvate</a>.</p>

<p>This enzyme belongs to the family of <a href="lyase" title="wikilink">lyases</a>, specifically the oxo-acid-lyases, which cleave carbon-carbon bonds. The systematic name of this enzyme class is <strong>chorismate pyruvate-lyase (4-hydroxybenzoate-forming)</strong>. Other names in common use include <strong>CL</strong>, <strong>CPL</strong>, and <strong>UbiC</strong>.</p>

<p>This enzyme <a href="catalysis" title="wikilink">catalyses</a> the first step in <a href="ubiquinone_biosynthesis" title="wikilink">ubiquinone biosynthesis</a>, the removal of pyruvate from chorismate, to yield 4-hydroxybenzoate in <em><a href="Escherichia_coli" title="wikilink">Escherichia coli</a></em> and other <a class="uri" href="Gram-negative" title="wikilink">Gram-negative</a> <a class="uri" href="bacteria" title="wikilink">bacteria</a>.<a class="footnoteRef" href="#fn1" id="fnref1"><sup>1</sup></a> Its activity does not require metal <a href="Cofactor_(biochemistry)" title="wikilink">cofactors</a>.<a class="footnoteRef" href="#fn2" id="fnref2"><sup>2</sup></a></p>
<h2 id="activity">Activity<a class="footnoteRef" href="#fn3" id="fnref3"><sup>3</sup></a></h2>
<h3 id="catayltic-activity">Catayltic activity</h3>
<ul>
<li>Chorismate = 4HB + pyruvate<a class="footnoteRef" href="#fn4" id="fnref4"><sup>4</sup></a></li>
<li>This enzyme has an optimum pH at 7.5</li>
</ul>
<h3 id="enzymatic-activity">Enzymatic activity</h3>

<p>Inhibited by:</p>
<ul>
<li>Vanillate</li>
<li>4-hydroxybenzaldehyde</li>
<li>3-carboxylmethylaminmethyl-4-hydroxybenzoic acid</li>
<li>4HB - ubiC is inhibited by the product of the reaction, which scientists believe serves as a control mechanism for the pathway</li>
</ul>
<h2 id="pathway">Pathway</h2>

<p>The pathway used is called the ubiquinone biosynthesis pathway, it catalyzes the first step in the biosynthesis of ubiquinone in <em>E. coli.</em> Ubiquinone is a lipid-soluble electron-transporting coenzyme. They are essential electron carriers in prokaryotes and are essential in aerobic organisms to achieve ATP.<a href="http://www.genome.jp/kegg-bin/show_pathway?map00130+4.1.3.40">1</a></p>
<h2 id="nomenclature">Nomenclature<a class="footnoteRef" href="#fn5" id="fnref5"><sup>5</sup></a></h2>

<p>There are several different names for chorismate lyase. it is also called chorismate pyruvate lyase (4-hydroxybenzoate-forming) and it is also abbreviated several different ways: CPL, CL, and ubiC. It is sometimes referred to as ubiC, because that is the gene name. This enzyme belongs to the class Lyases; more specifically the ox-acid-lyase or the carbon-carbon-lyases.</p>
<ul>
<li><ul>
<li>taxonomic lineage: bacteria → proteobacteria → gammaproteobacteria → enterobacteriales → enterobacteriaceae → escherichia → escherichia coli</li>
</ul></li>
</ul>
<h2 id="structure">Structure<a class="footnoteRef" href="#fn6" id="fnref6"><sup>6</sup></a></h2>

<p>This enzyme is a monomer. Its secondary structure contains helixes, turns, and beta strands.</p>
<ul>
<li>It has a mass of 18,777 daltons</li>
<li>Its sequence is 165 amino acids long</li>
</ul>
<h3 id="binding-sites">Binding Sites</h3>
<ul>
<li>position: 35(M)</li>
<li>position: 77(R)</li>
<li>position: 115(L)</li>
</ul>
<h2 id="mutagenesis">Mutagenesis<a class="footnoteRef" href="#fn7" id="fnref7"><sup>7</sup></a></h2>
<ul>
<li>position: 91- G → A; increases product inhibition by 40%. No effect on substrate affinity.</li>
<li>position: 156 - E → K; loss of activity</li>
</ul>
<h2 id="references">References</h2>
<h2 id="further-reading">Further reading</h2>
<ul>
<li></li>
<li></li>
<li></li>
</ul>

<p>"</p>

<p><a href="Category:Protein_families" title="wikilink">Category:Protein families</a> <a href="Category:EC_4.1.3" title="wikilink">Category:EC 4.1.3</a> <a href="Category:Enzymes_of_unknown_structure" title="wikilink">Category:Enzymes of unknown structure</a></p>
<section class="footnotes">
<hr/>
<ol>
<li id="fn1"><a href="#fnref1">↩</a></li>
<li id="fn2"><a href="#fnref2">↩</a></li>
<li id="fn3"><a href="#fnref3">↩</a></li>
<li id="fn4"><a href="#fnref4">↩</a></li>
<li id="fn5"></li>
<li id="fn6"></li>
<li id="fn7"></li>
</ol>
</section>
</body>
</html>
