<html lang="en">
<head>
<meta charset="utf-8"/>
<title offset="714">Combined bisulfite restriction analysis</title>
<script src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_SVG.js" type="text/javascript">
</script>
</head>
<body>
<h1>Combined bisulfite restriction analysis</h1>
<hr/>
<figure><b>(Figure)</b>
<figcaption>The first few steps of COBRA, and the molecular changes caused by each step to methylated and unmethylated CpG sites.</figcaption>
</figure>

<p><strong>Combined Bisulfite Restriction Analysis</strong> (or <strong>COBRA</strong>) is a molecular biology technique that allows for the sensitive quantification of <a href="DNA_methylation" title="wikilink">DNA methylation</a> levels at a specific genomic locus on a <a href="DNA_sequence" title="wikilink">DNA sequence</a> in a small sample of genomic DNA.<a class="footnoteRef" href="#fn1" id="fnref1"><sup>1</sup></a> The technique is a variation of <a href="bisulfite_sequencing" title="wikilink">bisulfite sequencing</a>, and combines <a href="Sodium_bisulfite#Bisulfite_DNA_sequencing" title="wikilink">bisulfite conversion</a> based <a href="polymerase_chain_reaction" title="wikilink">polymerase chain reaction</a> with <a href="restriction_digest" title="wikilink">restriction digestion</a>. Originally developed to reliably handle minute amounts of genomic DNA from microdissected <a href="Paraffin_wax" title="wikilink">paraffin</a>-embedded tissue samples, the technique has since seen widespread usage in <a href="cancer_research" title="wikilink">cancer research</a> and <a class="uri" href="epigenetics" title="wikilink">epigenetics</a> studies.<a class="footnoteRef" href="#fn2" id="fnref2"><sup>2</sup></a></p>
<h2 id="procedure">Procedure</h2>
<h3 id="bisulfite-treatment">Bisulfite Treatment</h3>

<p>Genomic DNA of interest is treated with <a href="sodium_bisulfite" title="wikilink">sodium bisulfite</a>, which introduces methylation-dependent sequence differences. During sodium bisulfite treatment, unmethylated <a class="uri" href="cytosine" title="wikilink">cytosine</a> residues are converted to <a class="uri" href="uracil" title="wikilink">uracil</a>, while <a href="5-Methylcytosine" title="wikilink">methylated cytosine</a> residues are unaffected.</p>
<figure><b>(Figure)</b>
<figcaption><a href="Sodium_bisulfite#Bisulfite_DNA_sequencing" title="wikilink">Bisulfite conversion</a>, changing unmethylated <a class="uri" href="cytosine" title="wikilink">cytosine</a> to <a class="uri" href="uracil" title="wikilink">uracil</a>, while <a class="uri" href="5-methylcytosine" title="wikilink">5-methylcytosine</a> remains unaffected by the treatment.</figcaption>
</figure>
<h3 id="pcr-amplification">PCR Amplification</h3>

<p>, taking place after <a href="Wikipedia:restriction_digest" title="wikilink">restriction digestion</a>. The digested <a href="Wikipedia:PCR" title="wikilink">PCR</a> fragments are separated and visualized by <a href="Wikipedia:gel_electrophoresis" title="wikilink">gel electrophoresis</a>, and the <a href="Wikipedia:DNA_methylation" title="wikilink">DNA methylation</a> level of the input DNA can be determined by quantifying the comparing the amount of DNA in the digested and undigested bands.|The final quantification step of COBRA, where the DNA methylation levels of the original input sample can be determined by comparing and quantifying the number of digested and undigested fragments.]]</p>

<p>Bisulfite treated DNA is then PCR amplified, resulting in cytosine residues at originally methylated positions, and thymine residues at originally unmethylated position (that were converted to uracil). Primers used during this step do not contain <a href="CpG_sites" title="wikilink">CpG sites</a> (the common target of cytosine methylation), so the amplificiation process does not discriminate between templates based on methylation status. PCR products are <a href="Polymerase_chain_reaction#PCR_optimization" title="wikilink">purified</a> to ensure complete digestion in the following step.</p>
<h3 id="restriction-digest">Restriction Digest</h3>

<p>The above steps lead to the methylation dependent retention or loss of CpG-containing <a href="restriction_sites" title="wikilink">restriction enzyme sites</a>, such as those for <a class="uri" href="TaqI" title="wikilink">TaqI</a> (TCGA) and <a class="uri" href="BstUI" title="wikilink">BstUI</a> (CGCG), depending on whether the cytosine residue was originally methylated or not, respectively. Due to the methylation-independent amplification in the above step, the resulting PCR products will be a mixed population of fragments that have lost or retained CpG-containing restriction enzyme sites, whose respective percentages will be directly correlated to the original level of DNA methylation in the sample DNA.</p>

<p>PCR products are then treated with a restriction enzyme (<em>e.g.</em> BstUI), which will only cleave sites that were originally methylated (CGCG), while leaving sites that were originally unmethylated (TGTG). To ensure that all CpG sites are retained due to originally being methylated, and not a remnant of incomplete bisulfite conversion, a control digestion is performed, with enzymes such as <a class="uri" href="Hsp92II" title="wikilink">Hsp92II</a> which recognizes the sequence CATG, none of which should be remaining after bisulfite conversion (with the rare exception of non-CpG methylation) and thus no cleavage should occur if bisulfite conversion was complete.</p>
<h3 id="quantification">Quantification</h3>

<p>The digested fragments are then separated by <a href="polyacrylamide_gel_electrophoresis" title="wikilink">polyacrylamide gel electrophoresis</a> with the expected appearance of bands corresponding to a single large undigested fragment, and multiple smaller bands corresponding to digested fragments. Quantitative amount of DNA in these bands can be determined with a device such as a <a href="photostimulated_luminescence" title="wikilink">phosphoimager</a>, after which the methylation percentage of the original sample can be calculated by:</p>
<dl>
<dd><dl>
<dd>

<math display="inline" id="Combined_bisulfite_restriction_analysis:0">
 <semantics>
  <mrow>
   <mo lspace="0pt" rspace="3.5pt">%</mo>
   <mtext>Methylation</mtext>
   <mo>=</mo>
   <mn>100</mn>
   <mo>×</mo>
   <mfrac>
    <mtext>Digested Fragments</mtext>
    <mtext>Undigested Fragments + Digested Fragments</mtext>
   </mfrac>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <cerror>
    <csymbol cd="ambiguous">fragments</csymbol>
    <csymbol cd="latexml">percent</csymbol>
    <mtext>Methylation</mtext>
    <eq></eq>
    <cn type="integer">100</cn>
    <times></times>
    <apply>
     <divide></divide>
     <mtext>Digested Fragments</mtext>
     <mtext>Undigested Fragments + Digested Fragments</mtext>
    </apply>
   </cerror>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \%\;\text{Methylation}=100\times\frac{\text{Digested Fragments}}{\text{%
Undigested Fragments + Digested Fragments}}
  </annotation>
 </semantics>
</math>


</dd>
</dl>
</dd>
</dl>
<h2 id="usage-and-applications">Usage and Applications</h2>

<p>COBRA has been used extensively in many research-based applications such as screening for DNA methylation changes at <a href="promoter_(biology)" title="wikilink">gene promoters</a> in cancer studies,<a class="footnoteRef" href="#fn3" id="fnref3"><sup>3</sup></a><a class="footnoteRef" href="#fn4" id="fnref4"><sup>4</sup></a> detecting altered methylation patterns at <a href="Genomic_imprinting" title="wikilink">imprinted genes</a>,<a class="footnoteRef" href="#fn5" id="fnref5"><sup>5</sup></a> and characterizing methylation patterns in the genome during development in mammals.<a class="footnoteRef" href="#fn6" id="fnref6"><sup>6</sup></a><a class="footnoteRef" href="#fn7" id="fnref7"><sup>7</sup></a></p>

<p>In medicine, COBRA has been used as a tool to help diagnose human disease involving aberrant DNA methylation. Researchers utilized COBRA in conjunction with <a href="SNP_genotyping#Denaturing_high_performance_liquid_chromatography" title="wikilink">denaturing high performance liquid chromatography</a> in the diagnosis of the <a href="genetic_imprinting" title="wikilink">genetic imprinting</a> disorder <a href="Russell-Silver_syndrome" title="wikilink">Russell-Silver syndrome</a> where <a href="Wiktionary:hypomethylation" title="wikilink">hypomethylation</a> of the imprinted gene <a href="H19_(gene)" title="wikilink">H19</a> is responsible for the disorder in up to 50% of patients.<a class="footnoteRef" href="#fn8" id="fnref8"><sup>8</sup></a></p>
<h2 id="strengths">Strengths</h2>
<ul>
<li><strong>Simple, fast and inexpensive</strong>: In COBRA, DNA methylation levels are easily and quickly measured without the need for laborious sub-cloning and <a href="DNA_sequencing" title="wikilink">sequencing</a>, as with bisulfite sequencing. The assay is straightforward and can be done with standard inexpensive molecular biology reagents.</li>
</ul>
<ul>
<li><strong>High compatibility</strong>: Due to the PCR and purification steps, the method not only works with very small amounts of genomic DNA, but also samples that have been treated with paraffin, both of which can be problems in other DNA methylation quantification protocols such as <a href="Southern_blot" title="wikilink">Southern blotting</a> and methylation-sensitive restriction enzyme digestion followed by PCR.</li>
</ul>
<ul>
<li><strong>Quantitative</strong>: This is in contrast to <a href="methylation-specific_PCR" title="wikilink">methylation-specific PCR</a>, which is qualitative. With COBRA, DNA methylation levels can be directly quantified at a given locus, yielding more information per assay.</li>
</ul>
<ul>
<li><strong>Scalability for high-throughput sample processing</strong>: With COBRA, many regions of interest can be processed in parallel in separate samples digested with the same restriction enzyme. This is in contrast to bisulfite sequencing analysis, where each region needs to be examined rigorously by sequencing many clones per locus, costing more time.</li>
</ul>
<ul>
<li><strong>Multiple queries per assay</strong>: Methylation status can be interrogated at multiple CpG-containing restriction sites in a single digestion assay.</li>
</ul>
<h2 id="weaknesses">Weaknesses</h2>
<ul>
<li>The assay is <strong>limited to using existing restriction sites</strong> in the region of interest, and methylation that does not occur in the context of a specific restriction site will not be assayed.</li>
</ul>
<ul>
<li><strong>Incomplete digestion</strong> by restriction enzymes after PCR can confound the analysis: incomplete digestion would suggest lack of DNA methylation (if cutting with a methylation-sensitive enzyme such as HpaII). It is also known that BstUI can cut at unconverted sites, leading to overestimation of methylation levels and so the use of HpaII is often needed.<a class="footnoteRef" href="#fn9" id="fnref9"><sup>9</sup></a></li>
</ul>
<ul>
<li>In complex samples, <strong>cell-type heterogeneity can confound the analysis</strong> since the DNA is not being sequenced, heterogeneity in sequences from different cells in the sample (<em>i.e.</em> different cell populations within a tumor) that have acquired mutations in the interrogated region, such as changing the CG dinucleotide to CA or CT, would result in loss of the restriction site giving rise to an apparently methylated region due to lack of digestion. This would skew the quantification of DNA methylation levels in a given sample.</li>
</ul>
<h2 id="alternatives">Alternatives</h2>

<p>In general, COBRA is often combined with other DNA methylation analyses and is frequently used in the initial screening of a loci of interest. If COBRA suggests altered methylation patterns, then more rigorous, labor-intensive techniques can be applied, such as bisulfite sequencing or <a class="uri" href="MeDIP" title="wikilink">MeDIP</a>.</p>
<h2 id="references">References</h2>
<references>
</references>

<p>"</p>

<p><a href="Category:Molecular_biology_techniques" title="wikilink">Category:Molecular biology techniques</a></p>
<section class="footnotes">
<hr/>
<ol>
<li id="fn1"><a href="#fnref1">↩</a></li>
<li id="fn2"> This paper had 576 references as of February 2–11, according to <a class="uri" href="Scopus" title="wikilink">Scopus</a><a href="#fnref2">↩</a></li>
<li id="fn3"><a href="#fnref3">↩</a></li>
<li id="fn4"><a href="#fnref4">↩</a></li>
<li id="fn5"><a href="#fnref5">↩</a></li>
<li id="fn6"><a href="#fnref6">↩</a></li>
<li id="fn7"><a href="#fnref7">↩</a></li>
<li id="fn8"><a href="#fnref8">↩</a></li>
<li id="fn9"><a href="#fnref9">↩</a></li>
</ol>
</section>
</body>
</html>
