<html lang="en">
<head>
<meta charset="utf-8"/>
<title offset="1397">Contact order</title>
<script src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_SVG.js" type="text/javascript">
</script>
</head>
<body>
<h1>Contact order</h1>
<hr>The '''contact order''' of a [[protein]] is a measure of the locality of the inter-[[amino acid]] contacts in the protein's [[native state]] [[tertiary structure]]. It is calculated as the average sequence distance between residues that form [[native contact]]s in the folded protein divided by the total length of the protein. Higher contact orders indicate longer [[protein folding|folding times]],<ref name="plaxco">{{cite journal|last=Plaxco|first=Kevin W|author2=Simons, Kim T |author3=Baker, David |t
<p>itle=Contact order, transition state placement and the refolding rates of single domain proteins|journal=Journal of Molecular Biology|date=April 1998|volume=277|issue=4|pages=985–994|doi=10.1006/jmbi.1998.1645}} <a class="footnoteRef" href="#fn1" id="fnref1"><sup>1</sup></a> and low contact order has been suggested as a predictor of potential <a href="downhill_folding" title="wikilink">downhill folding</a>, or protein folding that occurs without a <a href="Thermodynamic_free_energy" title="wikilink">free energy</a> barrier.<a class="footnoteRef" href="#fn2" id="fnref2"><sup>2</sup></a> This effect is thought to be due to the lower loss of <a href="conformational_entropy" title="wikilink">conformational entropy</a> associated with the formation of local as opposed to nonlocal contacts.<a class="footnoteRef" href="#fn3" id="fnref3"><sup>3</sup></a></p>

<p>Contact order (CO) is formally defined as:</p>

<p>

<math display="block" id="Contact_order:0">
 <semantics>
  <mrow>
   <mrow>
    <mi>C</mi>
    <mi>O</mi>
   </mrow>
   <mo>=</mo>
   <mrow>
    <mfrac>
     <mn>1</mn>
     <mrow>
      <mi>L</mi>
      <mo>⋅</mo>
      <mi>N</mi>
     </mrow>
    </mfrac>
    <mrow>
     <mover>
      <mo largeop="true" movablelimits="false" symmetric="true">∑</mo>
      <mi>N</mi>
     </mover>
     <mrow>
      <mi mathvariant="normal">Δ</mi>
      <msub>
       <mi>S</mi>
       <mrow>
        <mi>i</mi>
        <mo>,</mo>
        <mi>j</mi>
       </mrow>
      </msub>
     </mrow>
    </mrow>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <eq></eq>
    <apply>
     <times></times>
     <ci>C</ci>
     <ci>O</ci>
    </apply>
    <apply>
     <times></times>
     <apply>
      <divide></divide>
      <cn type="integer">1</cn>
      <apply>
       <ci>normal-⋅</ci>
       <ci>L</ci>
       <ci>N</ci>
      </apply>
     </apply>
     <apply>
      <apply>
       <csymbol cd="ambiguous">superscript</csymbol>
       <sum></sum>
       <ci>N</ci>
      </apply>
      <apply>
       <times></times>
       <ci>normal-Δ</ci>
       <apply>
        <csymbol cd="ambiguous">subscript</csymbol>
        <ci>S</ci>
        <list>
         <ci>i</ci>
         <ci>j</ci>
        </list>
       </apply>
      </apply>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   CO={1\over{L\cdot N}}\sum^{N}\Delta S_{i,j}
  </annotation>
 </semantics>
</math>

 where <em>N</em> is the total number of contacts, Δ<em>Si,j</em> is the sequence separation, in residues, between contacting residues <em>i</em> and <em>j</em>, and <em>L</em> is the total number of residues in the protein.<a class="footnoteRef" href="#fn4" id="fnref4"><sup>4</sup></a> The value of contact order typically ranges from 5% to 25% for single-domain proteins, with lower contact order belonging to mainly helical proteins, and higher contact order belonging to proteins with a high beta-sheet content.</p>

<p><a href="Protein_structure_prediction" title="wikilink">Protein structure prediction</a> methods are more accurate in predicting the structures of proteins with low contact orders. This may be partly because low contact order proteins tend to be small, but is likely to be explained by the smaller number of possible long-range residue-residue interactions to be considered during <a href="global_optimization" title="wikilink">global optimization</a> procedures that minimize an <a href="objective_function" title="wikilink">energy function</a>.<a class="footnoteRef" href="#fn5" id="fnref5"><sup>5</sup></a> Even successful structure prediction methods such as the <a href="Rosetta@Home" title="wikilink">Rosetta</a> method overproduce low-contact-order structure predictions compared to the distributions observed in experimentally determined protein structures.<a class="footnoteRef" href="#fn6" id="fnref6"><sup>6</sup></a></p>

<p>The percentage of the natively folded contact order can also be used as a measure of the "nativeness" of folding <a href="transition_state" title="wikilink">transition states</a>. <a href="Phi_value_analysis" title="wikilink">Phi value analysis</a> in concert with <a href="molecular_dynamics" title="wikilink">molecular dynamics</a> has produced transition-state models whose contact order is close to that of the folded state in proteins that are small and fast-folding.<a class="footnoteRef" href="#fn7" id="fnref7"><sup>7</sup></a> Further, contact orders in transition states as well as those in native states are highly correlated with overall folding time.<a class="footnoteRef" href="#fn8" id="fnref8"><sup>8</sup></a></p>

<p>In addition to their role in structure prediction, contact orders can themselves be predicted based on a <a href="sequence_alignment" title="wikilink">sequence alignment</a>, which can be useful in classifying the fold of a novel sequence with some degree of <a href="homology_(biology)" title="wikilink">homology</a> to known sequences.<a class="footnoteRef" href="#fn9" id="fnref9"><sup>9</sup></a></p>
<h2 id="references">References</h2>
<references>
</references>

<p>"</p>

<p><a class="uri" href="Category:Bioinformatics" title="wikilink">Category:Bioinformatics</a> <a href="Category:Protein_structure" title="wikilink">Category:Protein structure</a></p>
<section class="footnotes">
<hr/>
<ol>
<li id="fn1"><a href="#fnref1">↩</a></li>
<li id="fn2">Zuo G, Wang J, Wang W. (2006). Folding with downhill behavior and low cooperativity of proteins. <em>Proteins</em> 63(1):165-73.<a href="#fnref2">↩</a></li>
<li id="fn3"></li>
<li id="fn4"></li>
<li id="fn5">Mount DM. (2004). <em>Bioinformatics: Sequence and Genome Analysis</em> 2nd ed. Cold Spring Harbor Laboratory Press: Cold Spring Harbor, NY.<a href="#fnref5">↩</a></li>
<li id="fn6"></li>
<li id="fn7">Pandit AD, Jha A, Freed KF, Sosnick TR. (2006). Small proteins fold through transition states with native-like topologies. <em>J Mol Biol</em> 361(4):755-70.<a href="#fnref7">↩</a></li>
<li id="fn8">Paci E, Lindorff-Larsen K, Dobson CM, Karplus M, Vendruscolo M. (2005). Transition state contact orders correlate with protein folding rates. <em>J Mol Biol</em> 352(3):495-500.<a href="#fnref8">↩</a></li>
<li id="fn9">Yi Shi, Jianjun Zhou, David Arndt, David S. Wishart and Guohui Lin (2008). Protein contact order prediction from primary sequences. <em>BMC Bioinformatics.</em> 9:255.<a href="#fnref9">↩</a></li>
</ol>
</section>
</ref></hr></body>
</html>
