<html lang="en">
<head>
<meta charset="utf-8"/>
<title offset="1615">Crossover (genetic algorithm)</title>
<script src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_SVG.js" type="text/javascript">
</script>
</head>
<body>
<h1>Crossover (genetic algorithm)</h1>
<hr/>

<p>In <a href="genetic_algorithm" title="wikilink">genetic algorithms</a>, <strong>crossover</strong> is a <a href="genetic_operator" title="wikilink">genetic operator</a> used to vary the programming of a chromosome or <a href="chromosome_(genetic_algorithm)" title="wikilink">chromosomes</a> from one generation to the next. It is analogous to <a class="uri" href="reproduction" title="wikilink">reproduction</a> and <a href="chromosomal_crossover" title="wikilink">biological crossover</a>, upon which genetic algorithms are based. Cross over is a process of taking more than one parent solutions and producing a child solution from them. There are methods for selection of the chromosomes. Those are also given below.</p>
<h2 id="methods-of-selection-of-chromosomes-for-crossover">Methods of selection of chromosomes for crossover</h2>
<ul>
<li><strong><a href="Fitness_proportionate_selection" title="wikilink">Fitness proportionate selection</a></strong> (SCX)<a class="footnoteRef" href="#fn1" id="fnref1"><sup>1</sup></a> It is also known as fitness proportionate selection. The individual is selected on the basis of fitness. The probability of an individual to be selected increases with the fitness of the individual greater or less than its competitor's fitness.</li>
<li><strong>Boltzmann selection</strong></li>
<li><strong><a href="Tournament_selection" title="wikilink">Tournament selection</a></strong></li>
<li><strong>Rank selection</strong></li>
<li><strong>Steady state selection</strong></li>
<li><strong><a href="Truncation_selection" title="wikilink">Truncation selection</a></strong></li>
<li><strong>Local selection</strong></li>
</ul>
<h2 id="crossover-techniques">Crossover techniques</h2>

<p>Many crossover techniques exist for organisms which use different data structures to store themselves.</p>
<h3 id="one-point-crossover">One-point crossover</h3>

<p>A single crossover point on both parents' organism strings is selected. All data beyond that point in either organism string is swapped between the two parent organisms. The resulting organisms are the children:</p>
<figure><b>(Figure)</b>
<figcaption>OnePointCrossover.svg</figcaption>
</figure>
<h3 id="two-point-crossover">Two-point crossover</h3>

<p>Two-point crossover calls for two points to be selected on the parent organism strings. Everything between the two points is swapped between the parent organisms, rendering two child organisms:</p>
<figure><b>(Figure)</b>
<figcaption>TwoPointCrossover.svg</figcaption>
</figure>
<h3 id="cut-and-splice">"Cut and splice"</h3>

<p>Another crossover variant, the "cut and splice" approach, results in a change in length of the children strings. The reason for this difference is that each parent string has a separate choice of crossover point.</p>
<figure><b>(Figure)</b>
<figcaption>CutSpliceCrossover.png</figcaption>
</figure>
<h3 id="uniform-crossover-and-half-uniform-crossover">Uniform Crossover and Half Uniform Crossover</h3>

<p>The Uniform Crossover uses a fixed mixing ratio between two parents. Unlike one- and two-point crossover, the Uniform Crossover enables the parent chromosomes to contribute the gene level rather than the segment level.</p>

<p>If the mixing ratio is 0.5, the offspring has approximately half of the genes from first parent and the other half from second parent, although cross over points can be randomly chosen as seen below:</p>
<figure><b>(Figure)</b>
<figcaption>UniformCrossover.png</figcaption>
</figure>

<p>The Uniform Crossover evaluates each bit in the parent strings for exchange with a probability of 0.5. Even though the uniform crossover is a poor method , empirical evidence suggest that it is a more exploratory approach to crossover than the traditional exploitative approach that maintains longer schemata. This results in a more complete search of the design space with maintaining the exchange of good information. Unfortunately, no satisfactory theory exists to explain the discrepancies between the Uniform Crossover and the traditional approaches. <a class="footnoteRef" href="#fn2" id="fnref2"><sup>2</sup></a></p>

<p>In the uniform crossover scheme (UX) individual bits in the string are compared between two parents. The bits are swapped with a fixed probability, typically 0.5.</p>

<p>In the half uniform crossover scheme (HUX), exactly half of the nonmatching bits are swapped. Thus first the <a href="Hamming_distance" title="wikilink">Hamming distance</a> (the number of differing bits) is calculated. This number is divided by two. The resulting number is how many of the bits that do not match between the two parents will be swapped.</p>
<h3 id="three-parent-crossover">Three parent crossover</h3>

<p>In this technique, the child is derived from three parents. They are randomly chosen. Each bit of first parent is checked with bit of second parent whether they are same. If same then the bit is taken for the offspring otherwise the bit from the third parent is taken for the offspring. For example, the following three parents:</p>

<p>parent1 1 1 0 1 0 0 0 1 0</p>

<p>parent2 0 1 1 0 0 1 0 0 1</p>

<p>parent3 1 1 0 1 1 0 1 0 1<br/>
produces the following offspring:</p>

<p>offspring 1 1 0 1 0 0 0 0 1<a class="footnoteRef" href="#fn3" id="fnref3"><sup>3</sup></a></p>
<h3 id="crossover-for-ordered-chromosomes">Crossover for Ordered Chromosomes</h3>

<p>Depending on how the chromosome represents the solution, a direct swap may not be possible. One such case is when the chromosome is an ordered list, such as an ordered list of the cities to be travelled for the <a href="traveling_salesman_problem" title="wikilink">traveling salesman problem</a>. There are many crossover methods for ordered chromosomes. The already mentioned N-point crossover can be applied for ordered chromosomes also, but this always need a corresponding repair process, actually, some ordered crossover methods are derived from the idea. However, sometimes a crossover of chromosomes produces recombinations which violate the constraint of ordering and thus need to be repaired. Several examples for crossover operators (also mutation operator) preserving a given order are given in:<a class="footnoteRef" href="#fn4" id="fnref4"><sup>4</sup></a></p>
<ol>
<li>partially matched crossover (PMX): In this method, two crossover points are selected at random and PMX proceeds by position wise exchanges. The two crossover points give matching selection. It affects cross by position-by-position exchange operations. In this method parents are mapped to each other, hence we can also call it partially mapped crossover.<a class="footnoteRef" href="#fn5" id="fnref5"><sup>5</sup></a></li>
<li>cycle crossover (CX): Beginning at any gene 

<math display="inline" id="Crossover_(genetic_algorithm):0">
 <semantics>
  <mi>i</mi>
  <annotation-xml encoding="MathML-Content">
   <ci>i</ci>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   i
  </annotation>
 </semantics>
</math>

 in parent 1, the 

<math display="inline" id="Crossover_(genetic_algorithm):1">
 <semantics>
  <mi>i</mi>
  <annotation-xml encoding="MathML-Content">
   <ci>i</ci>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   i
  </annotation>
 </semantics>
</math>

-th gene in parent 2 becomes replaced by it. The same is repeated for the displaced gene until the gene which is equal to the first inserted gene becomes replaced (cycle).</li>
<li>order crossover operator (OX1): A portion of one parent is mapped to a portion of the other parent. From the replaced portion on, the rest is filled up by the remaining genes, where already present genes are omitted and the order is preserved.</li>
<li>order-based crossover operator (OX2)</li>
<li>position-based crossover operator (POS)</li>
<li>voting recombination crossover operator (VR)</li>
<li>alternating-position crossover operator (AP)</li>
<li>sequential constructive crossover operator (SCX)<a class="footnoteRef" href="#fn6" id="fnref6"><sup>6</sup></a></li>
</ol>

<p>Other possible methods include the <a href="edge_recombination_operator" title="wikilink">edge recombination operator</a>.</p>
<h2 id="crossover-biases">Crossover biases</h2>

<p>For crossover operators which exchange contiguous sections of the chromosomes (e.g. k-point) the ordering of the variables may become important. This is particularly true when good solutions contain <a href="Genetic_algorithm#The_building_block_hypothesis" title="wikilink">building blocks</a> which might be disrupted by a non-respectful crossover operator.</p>
<h2 id="see-also">See also</h2>
<ul>
<li><a href="Mutation_(genetic_algorithm)" title="wikilink">Mutation (genetic algorithm)</a></li>
</ul>
<ul>
<li><a href="Chromosome_(genetic_algorithm)" title="wikilink">Chromosome (genetic algorithm)</a></li>
</ul>
<ul>
<li><a href="Fitness_function" title="wikilink">Fitness function</a></li>
</ul>
<ul>
<li><a href="Fitness_approximation" title="wikilink">Fitness approximation</a></li>
</ul>
<h2 id="references">References</h2>
<ul>
<li>John Holland, <em>Adaptation in Natural and Artificial Systems</em>, University of Michigan Press, Ann Arbor, Michigan. 1975. ISBN 0-262-58111-6.</li>
<li>Larry J. Eshelman, <em>The CHC Adaptive Search Algorithm: How to Have Safe Search When Engaging in Nontraditional Genetic Recombination</em>, in Gregory J. E. Rawlins editor, Proceedings of the First Workshop on Foundations of Genetic Algorithms. pages 265-283. Morgan Kaufmann, 1991. ISBN 1-55860-170-8.</li>
<li>Tomasz D. Gwiazda, <em>Genetic Algorithms Reference Vol.1 Crossover for single-objective numerical optimization problems</em>, Tomasz Gwiazda, Lomianki, 2006. ISBN 83-923958-3-2.</li>
</ul>
<h2 id="external-links">External links</h2>
<ul>
<li><a href="http://www.faqs.org/faqs/ai-faq/genetic/part2/">Newsgroup: comp.ai.genetic FAQ</a> - see section on crossover (also known as recombination).</li>
</ul>

<p>"</p>

<p><a href="Category:Genetic_algorithms" title="wikilink">Category:Genetic algorithms</a></p>
<section class="footnotes">
<hr/>
<ol>
<li id="fn1"><http: en.wikipedia.org="" fitness_proportionate_selection="" wiki=""></http:><a href="#fnref1">↩</a></li>
<li id="fn2"><a href="#fnref2">↩</a></li>
<li id="fn3">Introduction to genetic algorithms By S. N. Sivanandam, S. N. Deepa<a href="#fnref3">↩</a></li>
<li id="fn4">Pedro Larrañaga et al., "Learning Bayesian Network Structures by searching for the best ordering with genetic algorithms", IEEE Transactions on systems, man and cybernetics, Vol 26, No. 4, 1996<a href="#fnref4">↩</a></li>
<li id="fn5">Introduction to genetic algorithms By S. N. Sivanandam, S. N. Deepa<a href="#fnref5">↩</a></li>
<li id="fn6">Ahmed, Zakir H. "Genetic Algorithm for the Traveling Salesman Problem Using Sequential Constructive Crossover Operator." International Journal of Biometric and Bioinformatics 3.6 (2010). Computer Science Journals. Web. <http: csc="" ijbb="" ijbb-41.pdf="" issue6="" journals="" manuscript="" volume3="" www.cscjournals.org="">.</http:><a href="#fnref6">↩</a></li>
</ol>
</section>
</body>
</html>
