<html lang="en">
<head>
<meta charset="utf-8"/>
<title offset="950">Denoising Algorithm based on Relevance network Topology</title>
<script src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_SVG.js" type="text/javascript">
</script>
</head>
<body>
<h1>Denoising Algorithm based on Relevance network Topology</h1>
<hr/>

<p><strong>Denoising Algorithm based on Relevance network Topology</strong> (DART) is an unsupervised algorithm that estimates an activity score for a pathway in a gene expression matrix, following a denoising step.<a class="footnoteRef" href="#fn1" id="fnref1"><sup>1</sup></a> In DART, a weighted average is used where the weights reflect the degree of the nodes in the pruned network.<a class="footnoteRef" href="#fn2" id="fnref2"><sup>2</sup></a> The denoising step removes prior information that is inconsistent with a data set. This strategy substantially improves unsupervised predictions of pathway activity that are based on a prior model, which was learned from a different biological system or context.<a class="footnoteRef" href="#fn3" id="fnref3"><sup>3</sup></a></p>

<p>Pre-existing methods such as gene set enrichment analysis method attempt to infer.<a class="footnoteRef" href="#fn4" id="fnref4"><sup>4</sup></a> However, it did not construct a structured list of genes. SPIA (Signaling Pathway Impact analysis)<a class="footnoteRef" href="#fn5" id="fnref5"><sup>5</sup></a> is a method that uses the phenotype information to evaluate the pathway activity between two phenotypes. However, it does not identify the pathway gene subset that could be used to differentiate individual samples.<a class="footnoteRef" href="#fn6" id="fnref6"><sup>6</sup></a> <a class="uri" href="CORG" title="wikilink">CORG</a> is used to identify a relevant gene subset. It is a supervised method, which does not perform as well as DART in analyzing independent data set<a class="footnoteRef" href="#fn7" id="fnref7"><sup>7</sup></a></p>

<p>Understanding molecular pathway activity is crucial for risk assessment, clinical diagnosis and treatment. <a class="uri" href="Meta-analysis" title="wikilink">Meta-analysis</a> of complex genomic data is often associated with difficulties such as extracting useful information from big data, eliminating confounding factors and providing more sensible interpretation. Different approaches have been taken to highlight the identification of relevant pathway in order to provide better <a href="gene_expression" title="wikilink">gene expression</a> prediction.</p>
<h2 id="method">Method</h2>

<p><a class="uri" href="File:DART">File:DART</a> method.jpg|Caption1</p>
<h2 id="strategy">Strategy</h2>
<ol>
<li>Build a network of all genes that are involved in the pathway</li>
<li>Evaluate the consistency of the prior regulatory information</li>
<li>Remove inconsistent prior information-the denoising step</li>
<li>Estimate pathway activity</li>
</ol>

<p>Pearson correlations were first computed between regulatory genes at the level of transcription and a gene expression data set. The correlation coefficient then underwent a <a href="Fisher's_transform" title="wikilink">Fisher's transform</a>:</p>

<p>

<math display="block" id="Denoising_Algorithm_based_on_Relevance_network_Topology:0">
 <semantics>
  <mrow>
   <msub>
    <mi>γ</mi>
    <mrow>
     <mi>i</mi>
     <mi>j</mi>
    </mrow>
   </msub>
   <mo>=</mo>
   <mrow>
    <mfrac>
     <mn>1</mn>
     <mn>2</mn>
    </mfrac>
    <mrow>
     <mi>log</mi>
     <mfrac>
      <mrow>
       <mn>1</mn>
       <mo>+</mo>
       <msub>
        <mi>c</mi>
        <mrow>
         <mi>i</mi>
         <mi>j</mi>
        </mrow>
       </msub>
      </mrow>
      <mrow>
       <mn>1</mn>
       <mo>-</mo>
       <msub>
        <mi>c</mi>
        <mrow>
         <mi>i</mi>
         <mi>j</mi>
        </mrow>
       </msub>
      </mrow>
     </mfrac>
    </mrow>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <eq></eq>
    <apply>
     <csymbol cd="ambiguous">subscript</csymbol>
     <ci>γ</ci>
     <apply>
      <times></times>
      <ci>i</ci>
      <ci>j</ci>
     </apply>
    </apply>
    <apply>
     <times></times>
     <apply>
      <divide></divide>
      <cn type="integer">1</cn>
      <cn type="integer">2</cn>
     </apply>
     <apply>
      <log></log>
      <apply>
       <divide></divide>
       <apply>
        <plus></plus>
        <cn type="integer">1</cn>
        <apply>
         <csymbol cd="ambiguous">subscript</csymbol>
         <ci>c</ci>
         <apply>
          <times></times>
          <ci>i</ci>
          <ci>j</ci>
         </apply>
        </apply>
       </apply>
       <apply>
        <minus></minus>
        <cn type="integer">1</cn>
        <apply>
         <csymbol cd="ambiguous">subscript</csymbol>
         <ci>c</ci>
         <apply>
          <times></times>
          <ci>i</ci>
          <ci>j</ci>
         </apply>
        </apply>
       </apply>
      </apply>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \gamma_{ij}=\frac{1}{2}\log{\frac{1+c_{ij}}{1-c_{ij}}}
  </annotation>
 </semantics>
</math>

</p>

<p>Where c<sub>ij</sub> is the correlation coefficient between gene i and j, and where γ<sub>ij</sub> is the variable that under the null hypothesis, its mean is zero and standard deviation 1/√(n_s-3), where ns is the number of tumor samples. The threshold of p-value was set at 0.0001. Gene pairs with significant correlation will be considered relevant in the network. To predict the activity score in which genes that are nearby are also taken into consideration:</p>

<p><a class="uri" href="File:Activity">File:Activity</a> score.jpg|</p>

<p>Where ki is the number of neighbors of gene i, zi is the normalized z-score and σi is a binary variable ( i.e 1 means upregulated upon activation and -1 means downregulated). This step is to estimate the activation level, in which sw AV is the activity score. A linear regression model was then applied to estimate the pathway activation levels. Thus, tij and pij denote the t-statistics and p-value associated with, whereas p <a class="uri" href="File:Consistency.jpg">File:Consistency.jpg</a>|</p>

<p>Where S is defined by</p>

<p><a class="uri" href="File:Sdenotes_value.jpg">File:Sdenotes_value.jpg</a>|</p>

<p>S is the threshold function of a given pair of pathways. And where</p>

<p><a class="uri" href="File:Significane_of_correlation.jpg">File:Significane_of_correlation.jpg</a>|</p>

<p>σ<sub>ij</sub> is the score that tells the directionality of a correlation, in which an opposite prediction will be panelized by given a value of -1. t<sub>ij</sub> is the t-statistics of interpathway correlation. The performance measure Vij accounts for the significance of correlation between pathways, the direction of correlation, and the weights in the magnitude of the correlation. A two-tailed paired Wilcoxon test is performed to compare the distribution under hypothesis. Advantages and limitation: DART gives an improved performance and higher accuracy in inferring pathway activity from prior information of pathway databases. Pre-existed information and large database are needed in order for DART to run.In other words, DART requires well-established prior gene expression data to start with, and then it can proceed evaluation of consistency and denoise any irrelevant information.</p>
<h2 id="application">Application</h2>

<p>DART is an algorithm that is applicable and used successfully in Cancer Genomics. The DART <a class="uri" href="algorithm" title="wikilink">algorithm</a> has been shown to be a strong method for estimating the pathway activity and perturbation signature activity in breast and lung cancer gene expression data sets.<a class="footnoteRef" href="#fn8" id="fnref8"><sup>8</sup></a> Imaging traits such as <a class="uri" href="mammography" title="wikilink">mammography</a> (Mammography is the process of using low-energy X-rays to examine the human breast tissue) plays an important role in cancer tumor diagnosis. Studies have shown that women with increased mammographic density have a higher risk of developing Breast cancer.<a class="footnoteRef" href="#fn9" id="fnref9"><sup>9</sup></a> Estrogen receptor alpha gene 1 encodes Estrogen Receptor-alpha, which is activated by estrogen. Polymorphisms in <a class="uri" href="ESR1" title="wikilink">ESR1</a> are associated with breast cancer risk through differences in different level of breast density. DART successfully predicted an inverse correlation between ESR1 signaling and MMD. It can be used in simulated and real multidimensional cancer genomic data. It gives more reliable prediction about pathway activation, which would be helpful in association studies.</p>
<h2 id="references">References</h2>

<p>"</p>

<p><a href="Category:Computational_biology" title="wikilink">Category:Computational biology</a></p>
<section class="footnotes">
<hr/>
<ol>
<li id="fn1"></li>
<li id="fn2"><a href="#fnref2">↩</a></li>
<li id="fn3"></li>
<li id="fn4"><a href="#fnref4">↩</a></li>
<li id="fn5"><a href="#fnref5">↩</a></li>
<li id="fn6"></li>
<li id="fn7"></li>
<li id="fn8"></li>
<li id="fn9"><a href="#fnref9">↩</a></li>
</ol>
</section>
</body>
</html>
