<html lang="en">
<head>
<meta charset="utf-8"/>
<title offset="1636">Diaminopimelate epimerase</title>
<script src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_SVG.js" type="text/javascript">
</script>
</head>
<body>
<h1>Diaminopimelate epimerase</h1>
<hr/>

<p>In <a class="uri" href="enzymology" title="wikilink">enzymology</a>, a <strong>diaminopimelate epimerase</strong> () is an <a class="uri" href="enzyme" title="wikilink">enzyme</a> that <a href="catalysis" title="wikilink">catalyzes</a> the <a href="chemical_reaction" title="wikilink">chemical reaction</a></p>
<dl>
<dd>LL-2,6-diaminoheptanedioate 

<math display="inline" id="Diaminopimelate_epimerase:0">
 <semantics>
  <mo>⇌</mo>
  <annotation-xml encoding="MathML-Content">
   <ci>normal-⇌</ci>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \rightleftharpoons
  </annotation>
 </semantics>
</math>

 meso-diaminoheptanedioate
</dd>
</dl>

<p>Hence, this enzyme has one <a href="substrate_(biochemistry)" title="wikilink">substrate</a>, <a class="uri" href="LL-2,6-diaminoheptanedioate" title="wikilink">LL-2,6-diaminoheptanedioate</a>, and one <a href="product_(chemistry)" title="wikilink">product</a>, <a class="uri" href="meso-diaminoheptanedioate" title="wikilink">meso-diaminoheptanedioate</a>.</p>

<p>This enzyme belongs to the family of <a href="isomerase" title="wikilink">isomerases</a>, specifically those <a href="racemase" title="wikilink">racemases</a> and <a href="epimerase" title="wikilink">epimerases</a> acting on <a href="amino_acid" title="wikilink">amino acids</a> and derivatives. The systematic name of this enzyme class is <strong>LL-2,6-diaminoheptanedioate 2-epimerase</strong>. This enzyme participates in <a href="lysine_biosynthesis" title="wikilink">lysine biosynthesis</a>.</p>
<h2 id="background">Background</h2>

<p>Bacteria, plants and fungi metabolise aspartic acid to produce four amino acids - lysine, threonine, methionine and isoleucine - in a series of reactions known as the aspartate pathway. Additionally, several important metabolic intermediates are produced by these reactions, such as diaminopimelic acid, an essential component of bacterial cell wall biosynthesis, and dipicolinic acid, which is involved in sporulation in Gram-positive bacteria. Members of the animal kingdom do not possess this pathway and must therefore acquire these essential amino acids through their diet. Research into improving the metabolic flux through this pathway has the potential to increase the yield of the essential amino acids in important crops, thus improving their nutritional value. Additionally, since the enzymes are not present in animals, inhibitors of them are promising targets for the development of novel antibiotics and herbicides. For more information see.<a class="footnoteRef" href="#fn1" id="fnref1"><sup>1</sup></a></p>

<p>The lysine/diaminopimelic acid branch of the aspartate pathway produces the essential amino acid lysine via the intermediate meso-diaminopimelic acid (meso-DAP), which is also a vital cell wall component in Gram-negative bacteria.<a class="footnoteRef" href="#fn2" id="fnref2"><sup>2</sup></a> The production of dihydropicolinate from aspartate-semialdehyde controls flux into the lysine/diaminopimelic acid pathway. Three variants of this pathway exist, differing in how tetrahydropicolinate (formed by reduction of dihydropicolinate) is metabolised to meso-DAP. One variant, the most commonly found one in archaea and bacteria, uses primarily succinyl intermediates, while a second variant, found only in Bacillus, utilises primarily acetyl intermediates. In the third variant, found in some Gram-positive bacteria, a dehydrogenase converts tetrahydropicolinate directly to meso-DAP. In all variants meso-DAP is subsequently converted to lysine by a decarboxylase, or, in Gram-negative bacteria, assimilated into the cell wall. Evidence exists that a fourth, currently unknown, variant of this pathway may function in plants.<a class="footnoteRef" href="#fn3" id="fnref3"><sup>3</sup></a></p>

<p>Diaminopimelate epimerase (), which catalyses the isomerisation of L,L-dimaminopimelate to meso-DAP in the biosynthetic pathway leading from aspartate to lysine. It is a member of the broader family of PLP-independent amino acid racemases. This enzyme is a monomeric protein of about 30 kDa consisting of two domains which are homologus in structure though they share little sequence similarity.<a class="footnoteRef" href="#fn4" id="fnref4"><sup>4</sup></a> Each domain consists of mixed beta-sheets which fold into a barrel around the central helix. The active site cleft is formed from both domains and contains two conserved cysteines thought to function as the acid and base in the catalytic reaction.<a class="footnoteRef" href="#fn5" id="fnref5"><sup>5</sup></a> Other PLP-independent racemases such as glutamate racemase have been shown to share a similar structure and mechanism of catalysis.</p>
<h2 id="structural-studies">Structural studies</h2>

<p>As of late 2007, 4 <a href="tertiary_structure" title="wikilink">structures</a> have been solved for this class of enzymes, with <a href="Protein_Data_Bank" title="wikilink">PDB</a> accession codes , , , and .</p>
<h2 id="references">References</h2>
<h2 id="further-reading">Further reading</h2>
<ul>
<li></li>
</ul>

<p>"</p>

<p><a href="Category:EC_5.1.1" title="wikilink">Category:EC 5.1.1</a> <a href="Category:Enzymes_of_known_structure" title="wikilink">Category:Enzymes of known structure</a></p>
<section class="footnotes">
<hr/>
<ol>
<li id="fn1"><a href="#fnref1">↩</a></li>
<li id="fn2"><a href="#fnref2">↩</a></li>
<li id="fn3"><a href="#fnref3">↩</a></li>
<li id="fn4"><a href="#fnref4">↩</a></li>
<li id="fn5"><a href="#fnref5">↩</a></li>
</ol>
</section>
</body>
</html>
