<html lang="en">
<head>
<meta charset="utf-8"/>
<title offset="1738">Drug design</title>
<script src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_SVG.js" type="text/javascript">
</script>
</head>
<body>
<h1>Drug design</h1>
<hr/>

<p><strong>Drug design</strong>, sometimes referred to as <strong>rational drug design</strong> or simply <a href="rational_design" title="wikilink">rational design</a>, is the <a href="invention" title="wikilink">inventive</a> process of finding new <a class="uri" href="medications" title="wikilink">medications</a> based on the knowledge of a <a href="biological_target" title="wikilink">biological target</a>.<a class="footnoteRef" href="#fn1" id="fnref1"><sup>1</sup></a> The drug is most commonly an <a href="organic_compound" title="wikilink">organic</a> <a href="small_molecule" title="wikilink">small molecule</a> that activates or inhibits the function of a <a class="uri" href="biomolecule" title="wikilink">biomolecule</a> such as a <a class="uri" href="protein" title="wikilink">protein</a>, which in turn results in a <a href="therapeutic_effect" title="wikilink">therapeutic</a> benefit to the <a class="uri" href="patient" title="wikilink">patient</a>. In the most basic sense, drug design involves the design of molecules that are complementary in <a class="uri" href="shape" title="wikilink">shape</a> and <a href="electric_charge" title="wikilink">charge</a> to the biomolecular target with which they interact and therefore will bind to it. Drug design frequently but not necessarily relies on <a href="molecular_modelling" title="wikilink">computer modeling</a> techniques.<a class="footnoteRef" href="#fn2" id="fnref2"><sup>2</sup></a> This type of modeling is often referred to as <strong>computer-aided drug design</strong>. Finally, drug design that relies on the knowledge of the three-dimensional structure of the biomolecular target is known as <strong>structure-based drug design</strong>.<a class="footnoteRef" href="#fn3" id="fnref3"><sup>3</sup></a> In addition to small molecules, <a href="biopharmaceutical" title="wikilink">biopharmaceuticals</a> and especially <a href="therapeutic_antibodies" title="wikilink">therapeutic antibodies</a> are an increasingly important class of drugs and computational methods for improving the affinity, selectivity, and stability of these protein-based therapeutics have also been developed.<a class="footnoteRef" href="#fn4" id="fnref4"><sup>4</sup></a></p>

<p>The phrase "drug design" is to some extent a <a class="uri" href="misnomer" title="wikilink">misnomer</a>. A more accurate term is <a href="ligand_(biochemistry)" title="wikilink">ligand</a> design (i.e., design of a molecule that will bind tightly to its target).<a class="footnoteRef" href="#fn5" id="fnref5"><sup>5</sup></a> Although modeling techniques for prediction of binding affinity are reasonably successful, there are many other properties, such as <a class="uri" href="bioavailability" title="wikilink">bioavailability</a>, <a href="biological_half-life" title="wikilink">metabolic half-life</a>, <a href="adverse_drug_reaction" title="wikilink">side effects</a>, etc., that first must be optimized before a ligand can become a safe and efficacious drug. These other characteristics are often difficult to predict using computer-aided drug design techniques. Nevertheless due to high attrition rates, especially during <a href="clinical_trial" title="wikilink">clinical phases</a> of <a href="drug_development" title="wikilink">drug development</a>, more attention is being focused early in the drug design process on selecting candidate drugs whose <a href="physical_chemistry" title="wikilink">physicochemical</a> properties are predicted to result in fewer complications during development and hence more likely to lead to an approved, marketed drug.<a class="footnoteRef" href="#fn6" id="fnref6"><sup>6</sup></a> Furthermore <a href="in_vitro" title="wikilink">in vitro</a> experiments complemented with computation methods are increasingly used in early <a href="drug_discovery" title="wikilink">drug discovery</a> to select compounds with more favorable <a class="uri" href="ADME" title="wikilink">ADME</a> (absorption, distribution, metabolism, and excretion) and <a class="uri" href="toxicological" title="wikilink">toxicological</a> profiles.<a class="footnoteRef" href="#fn7" id="fnref7"><sup>7</sup></a></p>
<h2 id="drug-targets">Drug targets</h2>

<p>A <a href="biomolecular_target" title="wikilink">biomolecular target</a> (most commonly a <a class="uri" href="protein" title="wikilink">protein</a> or <a href="nucleic_acid" title="wikilink">nucleic acid</a>) is a key molecule involved in a particular <a href="metabolic_pathway" title="wikilink">metabolic</a> or <a href="signal_transduction" title="wikilink">signaling</a> pathway that is associated with a specific disease condition or <a class="uri" href="pathology" title="wikilink">pathology</a> or to the <a class="uri" href="infectivity" title="wikilink">infectivity</a> or survival of a <a href="microorganism" title="wikilink">microbial</a> <a class="uri" href="pathogen" title="wikilink">pathogen</a>. Potential drug targets are not necessarily disease causing but must by definition be disease modifying.<a class="footnoteRef" href="#fn8" id="fnref8"><sup>8</sup></a> In some cases, <a href="small_molecule" title="wikilink">small molecules</a> will be designed to enhance or inhibit the target function in the specific disease modifying pathway. Small molecules (for example receptor <a href="agonist" title="wikilink">agonists</a>, <a href="receptor_antagonist" title="wikilink">antagonists</a>, <a href="inverse_agonist" title="wikilink">inverse agonists</a>, or <a href="selective_receptor_modulator" title="wikilink">modulators</a>; enzyme <a href="enzyme_activator" title="wikilink">activators</a> or <a href="enzyme_inhibitor" title="wikilink">inhibitors</a>; or ion channel <a href="channel_opener" title="wikilink">openers</a> or <a href="channel_blocker" title="wikilink">blockers</a>)<a class="footnoteRef" href="#fn9" id="fnref9"><sup>9</sup></a> will be designed that are complementary to the <a href="binding_site" title="wikilink">binding site</a> site of target.<a class="footnoteRef" href="#fn10" id="fnref10"><sup>10</sup></a> Small molecules (drugs) can be designed so as not to affect any other important "off-target" molecules (often referred to as <a href="antitarget" title="wikilink">antitargets</a>) since drug interactions with off-target molecules may lead to undesirable <a href="adverse_effect" title="wikilink">side effects</a>.<a class="footnoteRef" href="#fn11" id="fnref11"><sup>11</sup></a> Due to similarities in binding sites, closely related targets identified through <a href="sequence_homology" title="wikilink">sequence homology</a> have the highest chance of cross reactivity and hence highest side effect potential.</p>

<p>Most commonly, drugs are <a href="organic_compound" title="wikilink">organic</a> <a href="small_molecule" title="wikilink">small molecules</a> produced through chemical synthesis, but biopolymer-based drugs (also known as <a href="biopharmaceutical" title="wikilink">biopharmaceuticals</a>) produced through biological processes are becoming increasingly more common.<a class="footnoteRef" href="#fn12" id="fnref12"><sup>12</sup></a> In addition, <a class="uri" href="mRNA" title="wikilink">mRNA</a>-based <a href="gene_silencing" title="wikilink">gene silencing</a> technologies may have therapeutic applications.<a class="footnoteRef" href="#fn13" id="fnref13"><sup>13</sup></a></p>
<h2 id="rational-drug-discovery">Rational drug discovery</h2>

<p>In contrast to traditional methods of <a href="drug_discovery" title="wikilink">drug discovery</a> (known as <a href="classical_pharmacology" title="wikilink">forward pharmacology</a>), which rely on <a class="uri" href="trial-and-error" title="wikilink">trial-and-error</a> testing of chemical substances on <a href="cell_culture" title="wikilink">cultured cells</a> or <a href="animal" title="wikilink">animals</a>, and matching the apparent effects to treatments, rational drug design (also called <a href="reverse_pharmacology" title="wikilink">reverse pharmacology</a>) begins with a hypothesis that modulation of a specific biological target may have therapeutic value. In order for a biomolecule to be selected as a drug target, two essential pieces of information are required. The first is evidence that modulation of the target will be disease modifying. This knowledge may come from, for example, disease linkage studies that show an association between mutations in the biological target and certain disease states.<a class="footnoteRef" href="#fn14" id="fnref14"><sup>14</sup></a> The second is that the target is "<a href="druggability" title="wikilink">druggable</a>". This means that it is capable of binding to a small molecule and that its activity can be modulated by the small molecule.<a class="footnoteRef" href="#fn15" id="fnref15"><sup>15</sup></a></p>

<p>Once a suitable target has been identified, the target is normally <a href="molecular_cloning" title="wikilink">cloned</a> and <a href="protein_expression" title="wikilink">expressed</a>. The expressed target is then used to establish a <a href="Drug_discovery#Screening_and_design" title="wikilink">screening assay</a>. In addition, the three-dimensional structure of the target may be determined.</p>

<p>The search for small molecules that bind to the target is begun by screening libraries of potential drug compounds. This may be done by using the screening assay (a "wet screen"). In addition, if the structure of the target is available, a <a href="virtual_screening" title="wikilink">virtual screen</a> may be performed of candidate drugs. Ideally the candidate drug compounds should be "<a href="druglikeness" title="wikilink">drug-like</a>", that is they should possess properties that are predicted to lead to <a href="oral_bioavailability" title="wikilink">oral bioavailability</a>, adequate chemical and metabolic stability, and minimal toxic effects.<a class="footnoteRef" href="#fn16" id="fnref16"><sup>16</sup></a> Several methods are available to estimate druglikeness such as <a href="Lipinski's_Rule_of_Five" title="wikilink">Lipinski's Rule of Five</a> and a range of scoring methods such as <a href="lipophilic_efficiency" title="wikilink">lipophilic efficiency</a>.<a class="footnoteRef" href="#fn17" id="fnref17"><sup>17</sup></a> Several methods for predicting drug metabolism have also been proposed in the scientific literature.<a class="footnoteRef" href="#fn18" id="fnref18"><sup>18</sup></a></p>

<p>Due to the large number of drug properties that must be simultaneously optimized during the design process, <a href="multi-objective_optimization" title="wikilink">multi-objective optimization</a> techniques are sometimes employed.<a class="footnoteRef" href="#fn19" id="fnref19"><sup>19</sup></a> Finally because of the limitations in the current methods for prediction of activity, drug design is still very much reliant on <a href="Serendipity#Pharmacology" title="wikilink">serendipity</a><a class="footnoteRef" href="#fn20" id="fnref20"><sup>20</sup></a> and <a href="bounded_rationality" title="wikilink">bounded rationality</a>.<a class="footnoteRef" href="#fn21" id="fnref21"><sup>21</sup></a></p>
<h2 id="computer-aided-drug-design">Computer-aided drug design </h2>

<p>The most fundamental goal in drug design is to predict whether a given molecule will bind to a target and if so how strongly. <a href="Molecular_mechanics" title="wikilink">Molecular mechanics</a> or <a href="molecular_dynamics" title="wikilink">molecular dynamics</a> are most often used to predict the conformation of the <a href="small_molecule" title="wikilink">small molecule</a> and to model conformational changes in the biological target that may occur when the small molecule binds to it. <a href="Semi-empirical_quantum_chemistry_method" title="wikilink">Semi-empirical</a>, <a href="ab_initio_quantum_chemistry_methods" title="wikilink">ab initio quantum chemistry methods</a>, or <a href="density_functional_theory" title="wikilink">density functional theory</a> are often used to provide optimized parameters for the molecular mechanics calculations and also provide an estimate of the electronic properties (electrostatic potential, <a class="uri" href="polarizability" title="wikilink">polarizability</a>, etc.) of the drug candidate that will influence binding affinity.<a class="footnoteRef" href="#fn22" id="fnref22"><sup>22</sup></a></p>

<p>Molecular mechanics methods may also be used to provide semi-quantitative prediction of the binding affinity. Also, knowledge-based <a href="scoring_functions_for_docking" title="wikilink">scoring function</a> may be used to provide binding affinity estimates. These methods use <a href="linear_regression" title="wikilink">linear regression</a>, <a href="machine_learning" title="wikilink">machine learning</a>, <a href="neural_net" title="wikilink">neural nets</a> or other statistical techniques to derive predictive binding affinity equations by fitting experimental affinities to computationally derived interaction energies between the small molecule and the target.<a class="footnoteRef" href="#fn23" id="fnref23"><sup>23</sup></a><a class="footnoteRef" href="#fn24" id="fnref24"><sup>24</sup></a></p>

<p>Ideally, the computational method will be able to predict affinity before a compound is synthesized and hence in theory only one compound needs to be synthesized, saving enormous time and cost. The reality is that present computational methods are imperfect and provide, at best, only qualitatively accurate estimates of affinity. In practice it still takes several iterations of design, synthesis, and testing before an optimal drug is discovered. Computational methods have accelerated discovery by reducing the number of iterations required and have often provided novel structures.<a class="footnoteRef" href="#fn25" id="fnref25"><sup>25</sup></a><a class="footnoteRef" href="#fn26" id="fnref26"><sup>26</sup></a></p>

<p>Drug design with the help of computers may be used at any of the following stages of drug discovery:</p>
<ol>
<li>hit identification using <a href="virtual_screening" title="wikilink">virtual screening</a> (structure- or ligand-based design)</li>
<li><a href="drug_discovery_hit_to_lead" title="wikilink">hit-to-lead</a> optimization of affinity and selectivity (structure-based design, <a href="Quantitative_structure-activity_relationship" title="wikilink">QSAR</a>, etc.)</li>
<li><a href="drug_development" title="wikilink">lead optimization</a> optimization of other pharmaceutical properties while maintaining affinity</li>
</ol>

<p> In order to overcome the insufficient prediction of binding affinity calculated by recent scoring functions, the protein-ligand interaction and compound 3D structure information are used for analysis. For structure-based drug design, several post-screening analyses focusing on protein-ligand interaction have been developed for improving enrichment and effectively mining potential candidates:</p>
<ul>
<li>Consensus scoring<a class="footnoteRef" href="#fn27" id="fnref27"><sup>27</sup></a><a class="footnoteRef" href="#fn28" id="fnref28"><sup>28</sup></a>
<ul>
<li>Selecting candidates by voting of multiple scoring functions</li>
<li>May lose the relationship between protein-ligand structural information and scoring criterion</li>
</ul></li>
<li>Cluster analysis<a class="footnoteRef" href="#fn29" id="fnref29"><sup>29</sup></a><a class="footnoteRef" href="#fn30" id="fnref30"><sup>30</sup></a>
<ul>
<li>Represent and cluster candidates according to protein-ligand 3D information</li>
<li>Needs meaningful representation of protein-ligand interactions.</li>
</ul></li>
</ul>
<h2 id="types">Types</h2>
<figure><b>(Figure)</b>
<figcaption>Drug discovery cycle highlighting both ligand-based (indirect) and structure-based (direct) drug design strategies.</figcaption>
</figure>

<p>There are two major types of drug design. The first is referred to as <strong>ligand-based drug design</strong> and the second, <strong>structure-based drug design</strong>.<a class="footnoteRef" href="#fn31" id="fnref31"><sup>31</sup></a></p>
<h3 id="ligand-based">Ligand-based</h3>

<p>Ligand-based drug design (or <strong>indirect drug design</strong>) relies on knowledge of other molecules that bind to the biological target of interest. These other molecules may be used to derive a <a class="uri" href="pharmacophore" title="wikilink">pharmacophore</a> model that defines the minimum necessary structural characteristics a molecule must possess in order to bind to the target.<a class="footnoteRef" href="#fn32" id="fnref32"><sup>32</sup></a> In other words, a model of the biological target may be built based on the knowledge of what binds to it, and this model in turn may be used to design new molecular entities that interact with the target. Alternatively, a <a href="quantitative_structure-activity_relationship" title="wikilink">quantitative structure-activity relationship</a> (QSAR), in which a correlation between calculated properties of molecules and their experimentally determined biological activity, may be derived. These QSAR relationships in turn may be used to predict the activity of new analogs.<a class="footnoteRef" href="#fn33" id="fnref33"><sup>33</sup></a></p>
<h3 id="structure-based">Structure-based</h3>

<p>Structure-based drug design (or <strong>direct drug design</strong>) relies on knowledge of the <a href="tertiary_structure" title="wikilink">three dimensional structure</a> of the biological target obtained through methods such as <a href="X-ray_crystallography#Protein_crystallography" title="wikilink">x-ray crystallography</a> or <a href="Protein_nuclear_magnetic_resonance_spectroscopy" title="wikilink">NMR spectroscopy</a>.<a class="footnoteRef" href="#fn34" id="fnref34"><sup>34</sup></a> If an experimental structure of a target is not available, it may be possible to create a <a href="homology_modeling" title="wikilink">homology model</a> of the target based on the experimental structure of a related protein. Using the structure of the biological target, candidate drugs that are predicted to bind with high <a href="dissociation_constant" title="wikilink">affinity</a> and <a href="Ligand_(biochemistry)#Selective_and_non-selective" title="wikilink">selectivity</a> to the target may be designed using interactive graphics and the intuition of a <a href="medicinal_chemistry" title="wikilink">medicinal chemist</a>. Alternatively various automated computational procedures may be used to suggest new drug candidates.<a class="footnoteRef" href="#fn35" id="fnref35"><sup>35</sup></a></p>

<p>Current methods for structure-based drug design can be divided roughly into three main categories.<a class="footnoteRef" href="#fn36" id="fnref36"><sup>36</sup></a> The first method is identification of new ligands for a given receptor by searching large databases of 3D structures of small molecules to find those fitting the binding pocket of the receptor using fast approximate <a href="docking_(molecular)" title="wikilink">docking</a> programs. This method is known as <a href="virtual_screening" title="wikilink">virtual screening</a>. A second category is de novo design of new ligands. In this method, ligand molecules are built up within the constraints of the binding pocket by assembling small pieces in a stepwise manner. These pieces can be either individual atoms or molecular fragments. The key advantage of such a method is that novel structures, not contained in any database, can be suggested.<a class="footnoteRef" href="#fn37" id="fnref37"><sup>37</sup></a><a class="footnoteRef" href="#fn38" id="fnref38"><sup>38</sup></a><a class="footnoteRef" href="#fn39" id="fnref39"><sup>39</sup></a> A third method is the optimization of known ligands by evaluating proposed analogs within the binding cavity.<a class="footnoteRef" href="#fn40" id="fnref40"><sup>40</sup></a></p>
<h4 id="binding-site-identification">Binding site identification</h4>

<p><a href="Binding_site" title="wikilink">Binding site</a> identification is the first step in structure based design.<a class="footnoteRef" href="#fn41" id="fnref41"><sup>41</sup></a><a class="footnoteRef" href="#fn42" id="fnref42"><sup>42</sup></a> If the structure of the target or a sufficiently similar <a href="protein_homology" title="wikilink">homolog</a> is determined in the presence of a bound ligand, then the ligand should be observable in the structure in which case location of the binding site is trivial. However there may be unoccupied <a href="allosteric_modulator" title="wikilink">allosteric binding sites</a> that may be of interest. Furthermore it may be that only <a href="wiktionary:apoprotein" title="wikilink">apoprotein</a> (protein without ligand) structures are available and the reliable identification of unoccupied sites that have the potential to bind ligands with high affinity is non-trivial. In brief, binding site identification usually relies on identification of <a href="wiktionary:concave" title="wikilink">concave</a> surfaces on the protein that can accommodate drug sized molecules that also possess appropriate "hot spots" (<a class="uri" href="hydrophobic" title="wikilink">hydrophobic</a> surfaces, <a href="hydrogen_bonding" title="wikilink">hydrogen bonding</a> sites, etc.) that drive ligand binding.<a class="footnoteRef" href="#fn43" id="fnref43"><sup>43</sup></a><a class="footnoteRef" href="#fn44" id="fnref44"><sup>44</sup></a></p>
<h4 id="scoring-functions">Scoring functions</h4>

<p>Structure-based drug design attempts to use the structure of proteins as a basis for designing new ligands by applying the principles of <a href="molecular_recognition" title="wikilink">molecular recognition</a>. <a href="Binding_selectivity" title="wikilink">Selective</a> high <a href="affinity_(pharmacology)" title="wikilink">affinity</a> binding to the target is generally desirable since it leads to more <a href="efficacy" title="wikilink">efficacious</a> drugs with fewer side effects. Thus, one of the most important principles for designing or obtaining potential new ligands is to predict the binding affinity of a certain ligand to its target (and known <a href="antitarget" title="wikilink">antitargets</a>) and use the predicted affinity as a criterion for selection.<a class="footnoteRef" href="#fn45" id="fnref45"><sup>45</sup></a></p>

<p>One early general-purposed empirical scoring function to describe the binding energy of ligands to receptors was developed by B√∂hm.<a class="footnoteRef" href="#fn46" id="fnref46"><sup>46</sup></a><a class="footnoteRef" href="#fn47" id="fnref47"><sup>47</sup></a> This empirical scoring function took the form:</p>

<p>

<math display="inline" id="Drug_design:0">
 <semantics>
  <mrow>
   <mrow>
    <mi mathvariant="normal">Œî</mi>
    <msub>
     <mi>G</mi>
     <mtext>bind</mtext>
    </msub>
   </mrow>
   <mo>=</mo>
   <mrow>
    <mrow>
     <mi mathvariant="normal">Œî</mi>
     <msub>
      <mi>G</mi>
      <mtext>0</mtext>
     </msub>
    </mrow>
    <mo>+</mo>
    <mrow>
     <mi mathvariant="normal">Œî</mi>
     <msub>
      <mi>G</mi>
      <mtext>hb</mtext>
     </msub>
     <msub>
      <mi mathvariant="normal">Œ£</mi>
      <mrow>
       <mi>h</mi>
       <mo>-</mo>
       <mrow>
        <mi>b</mi>
        <mi>o</mi>
        <mi>n</mi>
        <mi>d</mi>
        <mi>s</mi>
       </mrow>
      </mrow>
     </msub>
    </mrow>
    <mo>+</mo>
    <mrow>
     <mi mathvariant="normal">Œî</mi>
     <msub>
      <mi>G</mi>
      <mtext>ionic</mtext>
     </msub>
     <msub>
      <mi mathvariant="normal">Œ£</mi>
      <mrow>
       <mrow>
        <mi>i</mi>
        <mi>o</mi>
        <mi>n</mi>
        <mi>i</mi>
        <mi>c</mi>
       </mrow>
       <mo>-</mo>
       <mrow>
        <mi>i</mi>
        <mi>n</mi>
        <mi>t</mi>
       </mrow>
      </mrow>
     </msub>
    </mrow>
    <mo>+</mo>
    <mrow>
     <mi mathvariant="normal">Œî</mi>
     <msub>
      <mi>G</mi>
      <mtext>lipophilic</mtext>
     </msub>
     <mrow>
      <mo>|</mo>
      <mi>A</mi>
      <mo>|</mo>
     </mrow>
    </mrow>
    <mo>+</mo>
    <mrow>
     <mi mathvariant="normal">Œî</mi>
     <msub>
      <mi>G</mi>
      <mtext>rot</mtext>
     </msub>
     <mi>ùëÅùëÖùëÇùëá</mi>
    </mrow>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <eq></eq>
    <apply>
     <times></times>
     <ci>normal-Œî</ci>
     <apply>
      <csymbol cd="ambiguous">subscript</csymbol>
      <ci>G</ci>
      <mtext>bind</mtext>
     </apply>
    </apply>
    <apply>
     <plus></plus>
     <apply>
      <times></times>
      <ci>normal-Œî</ci>
      <apply>
       <csymbol cd="ambiguous">subscript</csymbol>
       <ci>G</ci>
       <mtext>0</mtext>
      </apply>
     </apply>
     <apply>
      <times></times>
      <ci>normal-Œî</ci>
      <apply>
       <csymbol cd="ambiguous">subscript</csymbol>
       <ci>G</ci>
       <mtext>hb</mtext>
      </apply>
      <apply>
       <csymbol cd="ambiguous">subscript</csymbol>
       <ci>normal-Œ£</ci>
       <apply>
        <minus></minus>
        <ci>h</ci>
        <apply>
         <times></times>
         <ci>b</ci>
         <ci>o</ci>
         <ci>n</ci>
         <ci>d</ci>
         <ci>s</ci>
        </apply>
       </apply>
      </apply>
     </apply>
     <apply>
      <times></times>
      <ci>normal-Œî</ci>
      <apply>
       <csymbol cd="ambiguous">subscript</csymbol>
       <ci>G</ci>
       <mtext>ionic</mtext>
      </apply>
      <apply>
       <csymbol cd="ambiguous">subscript</csymbol>
       <ci>normal-Œ£</ci>
       <apply>
        <minus></minus>
        <apply>
         <times></times>
         <ci>i</ci>
         <ci>o</ci>
         <ci>n</ci>
         <ci>i</ci>
         <ci>c</ci>
        </apply>
        <apply>
         <times></times>
         <ci>i</ci>
         <ci>n</ci>
         <ci>t</ci>
        </apply>
       </apply>
      </apply>
     </apply>
     <apply>
      <times></times>
      <ci>normal-Œî</ci>
      <apply>
       <csymbol cd="ambiguous">subscript</csymbol>
       <ci>G</ci>
       <mtext>lipophilic</mtext>
      </apply>
      <apply>
       <abs></abs>
       <ci>A</ci>
      </apply>
     </apply>
     <apply>
      <times></times>
      <ci>normal-Œî</ci>
      <apply>
       <csymbol cd="ambiguous">subscript</csymbol>
       <ci>G</ci>
       <mtext>rot</mtext>
      </apply>
      <ci>ùëÅùëÖùëÇùëá</ci>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \Delta G_{\text{bind}}=\Delta G_{\text{0}}+\Delta G_{\text{hb}}\Sigma_{h-bonds%
}+\Delta G_{\text{ionic}}\Sigma_{ionic-int}+\Delta G_{\text{lipophilic}}\left|%
A\right|+\Delta G_{\text{rot}}\mathit{NROT}
  </annotation>
 </semantics>
</math>

</p>

<p>where:</p>
<ul>
<li>ŒîG<sub>0</sub> ‚Äì empirically derived offset that in part corresponds to the overall loss of translational and rotational entropy of the ligand upon binding.</li>
<li>ŒîG<sub>hb</sub> ‚Äì contribution from hydrogen bonding</li>
<li>ŒîG<sub>ionic</sub> ‚Äì contribution from ionic interactions</li>
<li>ŒîG<sub>lip</sub> ‚Äì contribution from lipophilic interactions where |A<sub>lipo</sub>| is surface area of lipophilic contact between the ligand and receptor</li>
<li>ŒîG<sub>rot</sub> ‚Äì entropy penalty due to freezing a rotatable in the ligand bond upon binding</li>
</ul>

<p>A more general thermodynamic "master" equation is as follows:<a class="footnoteRef" href="#fn48" id="fnref48"><sup>48</sup></a> 

<math display="inline" id="Drug_design:1">
 <semantics>
  <mtable>
   <mtr>
    <mtd columnalign="left">
     <mrow>
      <mrow>
       <mi mathvariant="normal">Œî</mi>
       <msub>
        <mi>G</mi>
        <mtext>bind</mtext>
       </msub>
      </mrow>
      <mo>=</mo>
      <mrow>
       <mo>-</mo>
       <mrow>
        <mi>R</mi>
        <mi>T</mi>
        <mrow>
         <mi>ln</mi>
         <msub>
          <mi>K</mi>
          <mtext>d</mtext>
         </msub>
        </mrow>
       </mrow>
      </mrow>
     </mrow>
    </mtd>
    <mtd></mtd>
    <mtd></mtd>
   </mtr>
   <mtr>
    <mtd columnalign="left">
     <mrow>
      <msub>
       <mi>K</mi>
       <mtext>d</mtext>
      </msub>
      <mo>=</mo>
      <mstyle displaystyle="true">
       <mfrac>
        <mrow>
         <mrow>
          <mo stretchy="false">[</mo>
          <mtext>Ligand</mtext>
          <mo stretchy="false">]</mo>
         </mrow>
         <mrow>
          <mo stretchy="false">[</mo>
          <mtext>Receptor</mtext>
          <mo stretchy="false">]</mo>
         </mrow>
        </mrow>
        <mrow>
         <mo stretchy="false">[</mo>
         <mtext>Complex</mtext>
         <mo stretchy="false">]</mo>
        </mrow>
       </mfrac>
      </mstyle>
     </mrow>
    </mtd>
    <mtd></mtd>
    <mtd></mtd>
   </mtr>
   <mtr>
    <mtd columnalign="left">
     <mrow>
      <mrow>
       <mi mathvariant="normal">Œî</mi>
       <msub>
        <mi>G</mi>
        <mtext>bind</mtext>
       </msub>
      </mrow>
      <mo>=</mo>
      <mrow>
       <mrow>
        <mi mathvariant="normal">Œî</mi>
        <msub>
         <mi>G</mi>
         <mtext>desolvation</mtext>
        </msub>
       </mrow>
       <mo>+</mo>
       <mrow>
        <mi mathvariant="normal">Œî</mi>
        <msub>
         <mi>G</mi>
         <mtext>motion</mtext>
        </msub>
       </mrow>
       <mo>+</mo>
       <mrow>
        <mi mathvariant="normal">Œî</mi>
        <msub>
         <mi>G</mi>
         <mtext>configuration</mtext>
        </msub>
       </mrow>
       <mo>+</mo>
       <mrow>
        <mi mathvariant="normal">Œî</mi>
        <msub>
         <mi>G</mi>
         <mtext>interaction</mtext>
        </msub>
       </mrow>
      </mrow>
     </mrow>
    </mtd>
    <mtd></mtd>
    <mtd></mtd>
   </mtr>
  </mtable>
  <annotation-xml encoding="MathML-Content">
   <matrix>
    <matrixrow>
     <apply>
      <eq></eq>
      <apply>
       <times></times>
       <ci>normal-Œî</ci>
       <apply>
        <csymbol cd="ambiguous">subscript</csymbol>
        <ci>G</ci>
        <mtext>bind</mtext>
       </apply>
      </apply>
      <apply>
       <minus></minus>
       <apply>
        <times></times>
        <ci>R</ci>
        <ci>T</ci>
        <apply>
         <ln></ln>
         <apply>
          <csymbol cd="ambiguous">subscript</csymbol>
          <ci>K</ci>
          <mtext>d</mtext>
         </apply>
        </apply>
       </apply>
      </apply>
     </apply>
     <cerror>
      <csymbol cd="ambiguous">missing-subexpression</csymbol>
     </cerror>
     <cerror>
      <csymbol cd="ambiguous">missing-subexpression</csymbol>
     </cerror>
    </matrixrow>
    <matrixrow>
     <apply>
      <eq></eq>
      <apply>
       <csymbol cd="ambiguous">subscript</csymbol>
       <ci>K</ci>
       <mtext>d</mtext>
      </apply>
      <apply>
       <divide></divide>
       <apply>
        <times></times>
        <apply>
         <csymbol cd="latexml">delimited-[]</csymbol>
         <mtext>Ligand</mtext>
        </apply>
        <apply>
         <csymbol cd="latexml">delimited-[]</csymbol>
         <mtext>Receptor</mtext>
        </apply>
       </apply>
       <apply>
        <csymbol cd="latexml">delimited-[]</csymbol>
        <mtext>Complex</mtext>
       </apply>
      </apply>
     </apply>
     <cerror>
      <csymbol cd="ambiguous">missing-subexpression</csymbol>
     </cerror>
     <cerror>
      <csymbol cd="ambiguous">missing-subexpression</csymbol>
     </cerror>
    </matrixrow>
    <matrixrow>
     <apply>
      <eq></eq>
      <apply>
       <times></times>
       <ci>normal-Œî</ci>
       <apply>
        <csymbol cd="ambiguous">subscript</csymbol>
        <ci>G</ci>
        <mtext>bind</mtext>
       </apply>
      </apply>
      <apply>
       <plus></plus>
       <apply>
        <times></times>
        <ci>normal-Œî</ci>
        <apply>
         <csymbol cd="ambiguous">subscript</csymbol>
         <ci>G</ci>
         <mtext>desolvation</mtext>
        </apply>
       </apply>
       <apply>
        <times></times>
        <ci>normal-Œî</ci>
        <apply>
         <csymbol cd="ambiguous">subscript</csymbol>
         <ci>G</ci>
         <mtext>motion</mtext>
        </apply>
       </apply>
       <apply>
        <times></times>
        <ci>normal-Œî</ci>
        <apply>
         <csymbol cd="ambiguous">subscript</csymbol>
         <ci>G</ci>
         <mtext>configuration</mtext>
        </apply>
       </apply>
       <apply>
        <times></times>
        <ci>normal-Œî</ci>
        <apply>
         <csymbol cd="ambiguous">subscript</csymbol>
         <ci>G</ci>
         <mtext>interaction</mtext>
        </apply>
       </apply>
      </apply>
     </apply>
     <cerror>
      <csymbol cd="ambiguous">missing-subexpression</csymbol>
     </cerror>
     <cerror>
      <csymbol cd="ambiguous">missing-subexpression</csymbol>
     </cerror>
    </matrixrow>
   </matrix>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \begin{array}[]{lll}\Delta G_{\text{bind}}=-RT\ln K_{\text{d}}\\
\par
 K_{\text{d}}=\dfrac{[\text{Ligand}][\text{Receptor}]}{[\text{Complex}]}%
\\
\par
\Delta G_{\text{bind}}=\Delta G_{\text{desolvation}}+\Delta G_{\text{%
motion}}+\Delta G_{\text{configuration}}+\Delta G_{\text{interaction}}\end{array}
  </annotation>
 </semantics>
</math>

</p>

<p>where:</p>
<ul>
<li>desolvation ‚Äì <a href="enthalpy" title="wikilink">enthalpic</a> penalty for removing the ligand from solvent</li>
<li>motion ‚Äì <a href="entropy" title="wikilink">entropic</a> penalty for reducing the degrees of freedom when a ligand binds to its receptor</li>
<li>configuration ‚Äì conformational strain energy required to put the ligand in its "active" conformation</li>
<li>interaction ‚Äì enthalpic gain for "resolvating" the ligand with its receptor</li>
</ul>

<p>The basic idea is that the overall binding free energy can be decomposed into independent components that are known to be important for the binding process. Each component reflects a certain kind of free energy alteration during the binding process between a ligand and its target receptor. The Master Equation is the linear combination of these components. According to Gibbs free energy equation, the relation between dissociation equilibrium constant, K<sub>d</sub>, and the components of free energy was built.</p>

<p>Various computational methods are used to estimate each of the components of the master equation. For example, the change in polar surface area upon ligand binding can be used to estimate the desolvation energy. The number of rotatable bonds frozen upon ligand binding is proportional to the motion term. The configurational or strain energy can be estimated using <a href="molecular_mechanics" title="wikilink">molecular mechanics</a> calculations. Finally the interaction energy can be estimated using methods such as the change in non polar surface, statistically derived <a href="potential_of_mean_force" title="wikilink">potentials of mean force</a>, the number of hydrogen bonds formed, etc. In practice, the components of the master equation are fit to experimental data using multiple linear regression. This can be done with a diverse training set including many types of ligands and receptors to produce a less accurate but more general "global" model or a more restricted set of ligands and receptors to produce a more accurate but less general "local" model.<a class="footnoteRef" href="#fn49" id="fnref49"><sup>49</sup></a></p>
<h2 id="examples">Examples</h2>

<p>A particular example of rational drug design involves the use of three-dimensional information about biomolecules obtained from such techniques as X-ray crystallography and NMR spectroscopy. Computer-aided drug design in particular becomes much more tractable when there is a high-resolution structure of a target protein bound to a potent ligand. This approach to drug discovery is sometimes referred to as structure-based drug design. The first unequivocal example of the application of <a href="QSAR" title="wikilink">structure-based drug design</a> leading to an approved drug is the carbonic anhydrase inhibitor <a class="uri" href="dorzolamide" title="wikilink">dorzolamide</a>, which was approved in 1995.<a class="footnoteRef" href="#fn50" id="fnref50"><sup>50</sup></a><a class="footnoteRef" href="#fn51" id="fnref51"><sup>51</sup></a></p>

<p>Another important case study in rational drug design is <a class="uri" href="imatinib" title="wikilink">imatinib</a>, a <a href="tyrosine_kinase" title="wikilink">tyrosine kinase</a> inhibitor designed specifically for the <em>bcr-abl</em> fusion protein that is characteristic for <a href="Philadelphia_chromosome" title="wikilink">Philadelphia chromosome</a>-positive <a href="leukemia" title="wikilink">leukemias</a> (<a href="chronic_myelogenous_leukemia" title="wikilink">chronic myelogenous leukemia</a> and occasionally <a href="acute_lymphocytic_leukemia" title="wikilink">acute lymphocytic leukemia</a>). Imatinib is substantially different from previous drugs for <a class="uri" href="cancer" title="wikilink">cancer</a>, as most agents of <a class="uri" href="chemotherapy" title="wikilink">chemotherapy</a> simply target rapidly dividing cells, not differentiating between cancer cells and other tissues.<a class="footnoteRef" href="#fn52" id="fnref52"><sup>52</sup></a></p>

<p>Additional examples include: </p>
<ul>
<li>Many of the <a href="atypical_antipsychotic" title="wikilink">atypical antipsychotics</a></li>
<li><a class="uri" href="Cimetidine" title="wikilink">Cimetidine</a>, the prototypical <a href="H2-receptor_antagonist" title="wikilink">H<sub>2</sub>-receptor antagonist</a> from which the later members of the class were developed</li>
<li>Selective <a href="Cyclooxygenase" title="wikilink">COX-2</a> inhibitor <a href="NSAID" title="wikilink">NSAIDs</a></li>
<li><a class="uri" href="Enfuvirtide" title="wikilink">Enfuvirtide</a>, a peptide HIV entry inhibitor</li>
<li><a class="uri" href="Nonbenzodiazepines" title="wikilink">Nonbenzodiazepines</a> like <a class="uri" href="zolpidem" title="wikilink">zolpidem</a> and <a class="uri" href="zopiclone" title="wikilink">zopiclone</a></li>
<li><a class="uri" href="Raltegravir" title="wikilink">Raltegravir</a>, an <a href="HIV_integrase" title="wikilink">HIV integrase</a> inhibitor<a class="footnoteRef" href="#fn53" id="fnref53"><sup>53</sup></a></li>
<li><a href="Selective_serotonin_reuptake_inhibitor" title="wikilink">SSRIs</a> (selective serotonin reuptake inhibitors), a class of <a href="antidepressant" title="wikilink">antidepressants</a></li>
<li><a class="uri" href="Zanamivir" title="wikilink">Zanamivir</a>, an <a href="antiviral_drug" title="wikilink">antiviral drug</a></li>
</ul>
<h2 id="case-studies">Case Studies</h2>
<ul>
<li><a href="5-HT3_antagonist#5-HT3_antagonists_drug_design" title="wikilink">5-HT3 antagonists</a></li>
<li><a href="Nicotinic_agonist" title="wikilink">Acetylcholine receptor agonists</a></li>
<li><a href="Discovery_and_development_of_angiotensin_receptor_blockers" title="wikilink">Angiotensin receptor antagonists</a></li>
<li><a href="Bcr-Abl_tyrosine-kinase_inhibitor" title="wikilink">Bcr-Abl tyrosine-kinase inhibitors</a></li>
<li><a href="Cannabinoid_receptor_antagonist#Drug_design" title="wikilink">Cannabinoid receptor antagonists</a></li>
<li><a href="CCR5_receptor_antagonist" title="wikilink">CCR5 receptor antagonists</a></li>
<li><a href="Discovery_and_development_of_cyclooxygenase_2_inhibitors" title="wikilink">Cyclooxygenase 2 inhibitors</a></li>
<li><a href="Development_of_dipeptidyl_peptidase-4_inhibitors" title="wikilink">Dipeptidyl peptidase-4 inhibitors</a></li>
<li><a href="Discovery_and_development_of_HIV_protease_inhibitors" title="wikilink">HIV protease inhibitors</a></li>
<li><a href="NK1_receptor_antagonist#Drug_discovery_and_development" title="wikilink">NK1 receptor antagonists</a></li>
<li><a href="Discovery_and_development_of_non-nucleoside_reverse_transcriptase_inhibitors" title="wikilink">Non-nucleoside reverse transcriptase inhibitors</a></li>
<li><a href="Discovery_and_development_of_phosphodiesterase_5_inhibitors" title="wikilink">PDE5 inhibitors</a></li>
<li><a href="Discovery_and_development_of_proton_pump_inhibitors" title="wikilink">Proton pump inibitors</a></li>
<li><a href="Discovery_and_Development_of_Renin_Inhibitors" title="wikilink">Renin inhibitors</a></li>
<li><a href="Discovery_and_development_of_triptans" title="wikilink">Triptans</a></li>
<li><a href="Discovery_and_development_of_TRPV1_antagonists" title="wikilink">TRPV1 antagonists</a></li>
<li><a href="c-Met_inhibitors" title="wikilink">c-Met inhibitors</a></li>
</ul>
<h2 id="criticism">Criticism</h2>

<p>It has been argued that the highly rigid and focused nature of rational drug design suppresses serendipity in drug discovery.<a class="footnoteRef" href="#fn54" id="fnref54"><sup>54</sup></a> Because many of the most significant medical discoveries have been inadvertent, the recent focus on rational drug design may limit the progress of drug discovery. Furthermore, the rational design of a drug may be limited by a crude or incomplete understanding of the underlying molecular processes of the disease it is intended to treat.</p>
<h2 id="see-also">See also</h2>
<ul>
<li><a class="uri" href="Bioisostere" title="wikilink">Bioisostere</a></li>
<li><a class="uri" href="Bioinformatics" title="wikilink">Bioinformatics</a></li>
<li><a class="uri" href="Cheminformatics" title="wikilink">Cheminformatics</a></li>
<li><a href="Drug_development" title="wikilink">Drug development</a></li>
<li><a href="Drug_discovery" title="wikilink">Drug discovery</a></li>
<li><a href="List_of_pharmaceutical_companies" title="wikilink">List of pharmaceutical companies</a></li>
<li><a href="Medicinal_chemistry" title="wikilink">Medicinal chemistry</a></li>
<li><a href="Molecular_design_software" title="wikilink">Molecular design software</a></li>
<li><a href="Retrometabolic_drug_design" title="wikilink">Retrometabolic drug design</a></li>
</ul>
<h2 id="references">References</h2>
<h2 id="external-links">External links</h2>
<ul>
<li></li>
</ul>

<p>"</p>

<p><a class="uri" href="Category:Pharmacology" title="wikilink">Category:Pharmacology</a> <a href="Category:Design_of_experiments" title="wikilink">Category:Design of experiments</a> <a href="Category:Clinical_research" title="wikilink">Category:Clinical research</a> <a href="Category:Medicinal_chemistry" title="wikilink">Category:Medicinal chemistry</a> <a href="Category:Drug_discovery" title="wikilink">Category:Drug discovery</a></p>
<section class="footnotes">
<hr/>
<ol>
<li id="fn1"><a href="#fnref1">‚Ü©</a></li>
<li id="fn2"><a href="#fnref2">‚Ü©</a></li>
<li id="fn3"></li>
<li id="fn4"><a href="#fnref4">‚Ü©</a></li>
<li id="fn5"><a href="#fnref5">‚Ü©</a></li>
<li id="fn6"><a href="#fnref6">‚Ü©</a></li>
<li id="fn7"><a href="#fnref7">‚Ü©</a></li>
<li id="fn8"><a href="#fnref8">‚Ü©</a></li>
<li id="fn9"><a href="#fnref9">‚Ü©</a></li>
<li id="fn10"><a href="#fnref10">‚Ü©</a></li>
<li id="fn11"><a href="#fnref11">‚Ü©</a></li>
<li id="fn12"><a href="#fnref12">‚Ü©</a></li>
<li id="fn13"><a href="#fnref13">‚Ü©</a></li>
<li id="fn14"><a href="#fnref14">‚Ü©</a></li>
<li id="fn15"><a href="#fnref15">‚Ü©</a></li>
<li id="fn16"><a href="#fnref16">‚Ü©</a></li>
<li id="fn17"><a href="#fnref17">‚Ü©</a></li>
<li id="fn18"><a href="#fnref18">‚Ü©</a></li>
<li id="fn19"><a href="#fnref19">‚Ü©</a></li>
<li id="fn20"><a href="#fnref20">‚Ü©</a></li>
<li id="fn21"><a href="#fnref21">‚Ü©</a></li>
<li id="fn22"><a href="#fnref22">‚Ü©</a></li>
<li id="fn23"><a href="#fnref23">‚Ü©</a></li>
<li id="fn24"><a href="#fnref24">‚Ü©</a></li>
<li id="fn25"><a href="#fnref25">‚Ü©</a></li>
<li id="fn26"><a href="#fnref26">‚Ü©</a></li>
<li id="fn27"><a href="#fnref27">‚Ü©</a></li>
<li id="fn28"><a href="#fnref28">‚Ü©</a></li>
<li id="fn29"><a href="#fnref29">‚Ü©</a></li>
<li id="fn30"><a href="#fnref30">‚Ü©</a></li>
<li id="fn31"></li>
<li id="fn32"><a href="#fnref32">‚Ü©</a></li>
<li id="fn33"><a href="#fnref33">‚Ü©</a></li>
<li id="fn34"><a href="#fnref34">‚Ü©</a></li>
<li id="fn35"><a href="#fnref35">‚Ü©</a></li>
<li id="fn36"><a href="#fnref36">‚Ü©</a></li>
<li id="fn37"><a href="#fnref37">‚Ü©</a></li>
<li id="fn38"><a href="#fnref38">‚Ü©</a></li>
<li id="fn39"><a href="#fnref39">‚Ü©</a></li>
<li id="fn40"></li>
<li id="fn41"></li>
<li id="fn42"><a href="#fnref42">‚Ü©</a></li>
<li id="fn43"></li>
<li id="fn44"></li>
<li id="fn45"><a href="#fnref45">‚Ü©</a></li>
<li id="fn46"><a href="#fnref46">‚Ü©</a></li>
<li id="fn47"><a href="#fnref47">‚Ü©</a></li>
<li id="fn48"><a href="#fnref48">‚Ü©</a></li>
<li id="fn49"><a href="#fnref49">‚Ü©</a></li>
<li id="fn50"><a href="#fnref50">‚Ü©</a></li>
<li id="fn51"><a href="#fnref51">‚Ü©</a></li>
<li id="fn52"><a href="#fnref52">‚Ü©</a></li>
<li id="fn53"><a href="#fnref53">‚Ü©</a></li>
<li id="fn54"><a href="#fnref54">‚Ü©</a></li>
</ol>
</section>
</body>
</html>
