<html lang="en">
<head>
<meta charset="utf-8"/>
<title offset="1642">DTDP-4-dehydrorhamnose 3,5-epimerase</title>
<script src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_SVG.js" type="text/javascript">
</script>
</head>
<body>
<h1>DTDP-4-dehydrorhamnose 3,5-epimerase</h1>
<hr/>

<p>In <a class="uri" href="enzymology" title="wikilink">enzymology</a>, a <strong>dTDP-4-dehydrorhamnose 3,5-epimerase</strong> () is an <a class="uri" href="enzyme" title="wikilink">enzyme</a> that <a href="catalysis" title="wikilink">catalyzes</a> the <a href="chemical_reaction" title="wikilink">chemical reaction</a></p>
<dl>
<dd>dTDP-4-dehydro-6-deoxy-D-glucose 

<math display="inline" id="DTDP-4-dehydrorhamnose_3,5-epimerase:0">
 <semantics>
  <mo>⇌</mo>
  <annotation-xml encoding="MathML-Content">
   <ci>normal-⇌</ci>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \rightleftharpoons
  </annotation>
 </semantics>
</math>

 dTDP-4-dehydro-6-deoxy-L-mannose
</dd>
</dl>

<p>Hence, this enzyme has one <a href="substrate_(biochemistry)" title="wikilink">substrate</a>, <a class="uri" href="dTDP-4-dehydro-6-deoxy-D-glucose" title="wikilink">dTDP-4-dehydro-6-deoxy-D-glucose</a>, and one <a href="product_(chemistry)" title="wikilink">product</a>, <a class="uri" href="dTDP-4-dehydro-6-deoxy-L-mannose" title="wikilink">dTDP-4-dehydro-6-deoxy-L-mannose</a>.</p>

<p>This enzyme belongs to the family of <a href="isomerase" title="wikilink">isomerases</a>, specifically those <a href="racemase" title="wikilink">racemases</a> and <a href="epimerase" title="wikilink">epimerases</a> acting on <a href="carbohydrate" title="wikilink">carbohydrates</a> and derivatives. The systematic name of this enzyme class is <strong>dTDP-4-dehydro-6-deoxy-D-glucose 3,5-epimerase</strong>. Other names in common use include <strong>dTDP-L-rhamnose synthetase</strong>, <strong>dTDP-L-rhamnose synthetase</strong>, <strong>thymidine diphospho-4-ketorhamnose 3,5-epimerase</strong>, <strong>TDP-4-ketorhamnose 3,5-epimerase</strong>, <strong>dTDP-4-dehydro-6-deoxy-D-glucose 3,5-epimerase</strong>, and <strong>TDP-4-keto-L-rhamnose-3,5-epimerase</strong>. This enzyme participates in 3 <a href="metabolism" title="wikilink">metabolic pathways</a>: <a href="nucleotide_sugars_metabolism" title="wikilink">nucleotide sugars metabolism</a>, <a href="streptomycin_biosynthesis" title="wikilink">streptomycin biosynthesis</a>, and <a href="polyketide_sugar_unit_biosynthesis" title="wikilink">polyketide sugar unit biosynthesis</a>.</p>
<h2 id="structural-studies">Structural studies</h2>

<p>The <a href="crystal_structure" title="wikilink">crystal structure</a> of RmlC from <em>Methanobacterium thermoautotrophicum</em> was determined in the presence and absence of a <a href="Enzyme_substrate" title="wikilink">substrate</a> <a href="Substrate_analog" title="wikilink">analogue</a>. RmlC is a <a class="uri" href="homodimer" title="wikilink">homodimer</a> comprising a central jelly roll motif, which extends in two directions into longer <a href="beta-sheet" title="wikilink">beta-sheets</a>. <a href="Binding_(molecular)" title="wikilink">Binding</a> of <a class="uri" href="dTDP" title="wikilink">dTDP</a> is stabilised by ionic interactions to the <a class="uri" href="phosphate" title="wikilink">phosphate</a> group and by a combination of ionic and <a class="uri" href="hydrophobic" title="wikilink">hydrophobic</a> interactions with the base. The <a href="active_site" title="wikilink">active site</a>, which is located in the centre of the jelly roll, is formed by residues that are <a href="conserved_sequence" title="wikilink">conserved</a> in all known RmlC <a href="sequence_(biology)" title="wikilink">sequence</a> <a href="Homology_(biology)" title="wikilink">homologues</a>. The <a href="active_site" title="wikilink">active site</a> is lined with a number of charged residues and a number of residues with <a class="uri" href="hydrogen-bonding" title="wikilink">hydrogen-bonding</a> potentials, which together comprise a potential network for substrate binding and <a class="uri" href="catalysis" title="wikilink">catalysis</a>. The active site is also lined with <a href="Aromatic_amino-acid" title="wikilink">aromatic residues</a> which provide favourable environments for the base moiety of dTDP and potentially for the <a class="uri" href="sugar" title="wikilink">sugar</a> moiety of the <a href="enzyme_substrate" title="wikilink">substrate</a>.<a class="footnoteRef" href="#fn1" id="fnref1"><sup>1</sup></a></p>

<p>As of late 2007, 14 <a href="tertiary_structure" title="wikilink">structures</a> have been solved for this class of enzymes, with <a href="Protein_Data_Bank" title="wikilink">PDB</a> accession codes , , , , , , , , , , , , , and .</p>
<h2 id="references">References</h2>
<h2 id="further-reading">Further reading</h2>
<ul>
<li></li>
<li></li>
</ul>

<p>"</p>

<p><a href="Category:EC_5.1.3" title="wikilink">Category:EC 5.1.3</a> <a href="Category:Enzymes_of_known_structure" title="wikilink">Category:Enzymes of known structure</a></p>
<section class="footnotes">
<hr/>
<ol>
<li id="fn1"><a href="#fnref1">↩</a></li>
</ol>
</section>
</body>
</html>
