<html lang="en">
<head>
<meta charset="utf-8"/>
<title offset="310">Enzyme</title>
<script src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_SVG.js" type="text/javascript">
</script>
</head>
<body>
<h1>Enzyme</h1>
<hr/>
<figure><b>(Figure)</b>
<figcaption>The enzyme <a class="uri" href="glucosidase" title="wikilink">glucosidase</a> converts sugar <a class="uri" href="maltose" title="wikilink">maltose</a> to two <a class="uri" href="glucose" title="wikilink">glucose</a> sugars. Active site residues in red, maltose substrate in black, and NAD cofactor in yellow. ()|alt=Ribbon diagram of glycosidase with an arrow showing the cleavage of the maltose sugar substrate into two glucose products.</figcaption>
</figure>

<p><strong>Enzymes</strong>  are <a class="uri" href="macromolecular" title="wikilink">macromolecular</a> <a class="uri" href="biological" title="wikilink">biological</a> <a href="catalyst" title="wikilink">catalysts</a>. Enzymes accelerate, or <a class="uri" href="catalyze" title="wikilink">catalyze</a>, <a href="chemical_reactions" title="wikilink">chemical reactions</a>. The molecules at the beginning of the process are called <a href="substrate_(chemistry)" title="wikilink">substrates</a> and the enzyme converts these into different molecules, called <a href="product_(chemistry)" title="wikilink">products</a>. Almost all <a href="metabolism" title="wikilink">metabolic processes</a> in the <a href="cell_(biology)" title="wikilink">cell</a> need enzymes in order to occur at rates fast enough to sustain life.<a class="footnoteRef" href="#fn1" id="fnref1"><sup>1</sup></a> The set of enzymes made in a cell determines which <a href="metabolic_pathway" title="wikilink">metabolic pathways</a> occur in that cell. The study of enzymes is called <em>enzymology</em>.</p>

<p>Enzymes are known to catalyze more than 5,000 biochemical reaction types.<a class="footnoteRef" href="#fn2" id="fnref2"><sup>2</sup></a> Most enzymes are <a href="protein" title="wikilink">proteins</a>, although a few are <a href="Ribozyme" title="wikilink">catalytic RNA molecules</a>. Enzymes' specificity comes from their unique <a href="tertiary_structure" title="wikilink">three-dimensional structures</a>.</p>

<p>Like all catalysts, enzymes increase the <a href="reaction_rate" title="wikilink">rate of a reaction</a> by lowering its <a href="activation_energy" title="wikilink">activation energy</a>. Some enzymes can make their conversion of substrate to product occur many millions of times faster. An extreme example is <a href="orotidine_5'-phosphate_decarboxylase" title="wikilink">orotidine 5'-phosphate decarboxylase</a>, which allows a reaction that would otherwise take millions of years to occur in milliseconds.<a class="footnoteRef" href="#fn3" id="fnref3"><sup>3</sup></a><a class="footnoteRef" href="#fn4" id="fnref4"><sup>4</sup></a> Chemically, enzymes are like any catalyst and are not consumed in chemical reactions, nor do they alter the <a href="Chemical_equilibrium" title="wikilink">equilibrium</a> of a reaction. Enzymes differ from most other catalysts by being much more specific. Enzyme activity can be affected by other molecules: <a href="Enzyme_inhibitor" title="wikilink">inhibitors</a> are molecules that decrease enzyme activity, and <a href="enzyme_activator" title="wikilink">activators</a> are molecules that increase activity. Many <a href="drug" title="wikilink">drugs</a> and <a href="poison" title="wikilink">poisons</a> are enzyme inhibitors. An enzyme's activity decreases markedly outside its optimal <a class="uri" href="temperature" title="wikilink">temperature</a> and <a class="uri" href="pH" title="wikilink">pH</a>.</p>

<p>Some enzymes are used commercially, for example, in the synthesis of <a class="uri" href="antibiotics" title="wikilink">antibiotics</a>. Some household products use enzymes to speed up chemical reactions: enzymes in biological <a href="washing_powder" title="wikilink">washing powders</a> break down protein, starch or <a class="uri" href="fat" title="wikilink">fat</a> stains on clothes, and enzymes in <a href="papain" title="wikilink">meat tenderizer</a> break down proteins into smaller molecules, making the meat easier to chew.</p>
<h2 id="etymology-and-history">Etymology and history</h2>

<p> By the late 17th and early 18th centuries, the digestion of <a class="uri" href="meat" title="wikilink">meat</a> by stomach secretions<a class="footnoteRef" href="#fn5" id="fnref5"><sup>5</sup></a> and the conversion of <a class="uri" href="starch" title="wikilink">starch</a> to <a href="sugar" title="wikilink">sugars</a> by plant extracts and <a class="uri" href="saliva" title="wikilink">saliva</a> were known but the mechanisms by which these occurred had not been identified.<a class="footnoteRef" href="#fn6" id="fnref6"><sup>6</sup></a></p>

<p>French chemist <a href="Anselme_Payen" title="wikilink">Anselme Payen</a> was the first to discover an enzyme, <a class="uri" href="diastase" title="wikilink">diastase</a>, in 1833.<a class="footnoteRef" href="#fn7" id="fnref7"><sup>7</sup></a> A few decades later, when studying the <a href="fermentation_(food)" title="wikilink">fermentation</a> of sugar to <a class="uri" href="alcohol" title="wikilink">alcohol</a> by <a class="uri" href="yeast" title="wikilink">yeast</a>, <a href="Louis_Pasteur" title="wikilink">Louis Pasteur</a> concluded that this fermentation was caused by a <a href="Vitalism" title="wikilink">vital force</a> contained within the yeast cells called "ferments", which were thought to function only within living organisms. He wrote that "alcoholic fermentation is an act correlated with the life and organization of the yeast cells, not with the death or putrefaction of the cells."<a class="footnoteRef" href="#fn8" id="fnref8"><sup>8</sup></a></p>

<p>In 1877, German physiologist <a href="Wilhelm_Kühne" title="wikilink">Wilhelm Kühne</a> (1837–1900) first used the term <em><a href="wiktionary:enzyme" title="wikilink">enzyme</a></em>, which comes from <a href="Ancient_Greek" title="wikilink">Greek</a> ἔνζυμον, "leavened", to describe this process.<a class="footnoteRef" href="#fn9" id="fnref9"><sup>9</sup></a> The word <em>enzyme</em> was used later to refer to nonliving substances such as <a class="uri" href="pepsin" title="wikilink">pepsin</a>, and the word <em>ferment</em> was used to refer to chemical activity produced by living organisms.<a class="footnoteRef" href="#fn10" id="fnref10"><sup>10</sup></a></p>

<p><a href="Eduard_Buchner" title="wikilink">Eduard Buchner</a> submitted his first paper on the study of yeast extracts in 1897. In a series of experiments at the <a href="Humboldt_University_of_Berlin" title="wikilink">University of Berlin</a>, he found that sugar was fermented by yeast extracts even when there were no living yeast cells in the mixture.<a class="footnoteRef" href="#fn11" id="fnref11"><sup>11</sup></a> He named the enzyme that brought about the fermentation of sucrose "<a class="uri" href="zymase" title="wikilink">zymase</a>".<a class="footnoteRef" href="#fn12" id="fnref12"><sup>12</sup></a> In 1907, he received the <a href="Nobel_Prize_in_Chemistry" title="wikilink">Nobel Prize in Chemistry</a> for "his discovery of cell-free fermentation". Following Buchner's example, enzymes are usually named according to the reaction they carry out: the suffix <em><a class="uri" href="-ase" title="wikilink">-ase</a></em> is combined with the name of the <a href="substrate_(biochemistry)" title="wikilink">substrate</a> (e.g., <a class="uri" href="lactase" title="wikilink">lactase</a> is the enzyme that cleaves <a class="uri" href="lactose" title="wikilink">lactose</a>) or to the type of reaction (e.g., <a href="DNA_polymerase" title="wikilink">DNA polymerase</a> forms DNA polymers).<a class="footnoteRef" href="#fn13" id="fnref13"><sup>13</sup></a></p>

<p>The biochemical identity of enzymes was still unknown in the early 1900s. Many scientists observed that enzymatic activity was associated with proteins, but others (such as Nobel laureate <a href="Richard_Willstätter" title="wikilink">Richard Willstätter</a>) argued that proteins were merely carriers for the true enzymes and that proteins <em>per se</em> were incapable of catalysis.<a class="footnoteRef" href="#fn14" id="fnref14"><sup>14</sup></a> In 1926, <a href="James_B._Sumner" title="wikilink">James B. Sumner</a> showed that the enzyme <a class="uri" href="urease" title="wikilink">urease</a> was a pure protein and crystallized it; he did likewise for the enzyme <a class="uri" href="catalase" title="wikilink">catalase</a> in 1937. The conclusion that pure proteins can be enzymes was definitively demonstrated by <a href="John_Howard_Northrop" title="wikilink">John Howard Northrop</a> and <a href="Wendell_Meredith_Stanley" title="wikilink">Wendell Meredith Stanley</a>, who worked on the digestive enzymes <a class="uri" href="pepsin" title="wikilink">pepsin</a> (1930), <a class="uri" href="trypsin" title="wikilink">trypsin</a> and <a class="uri" href="chymotrypsin" title="wikilink">chymotrypsin</a>. These three scientists were awarded the 1946 Nobel Prize in Chemistry.<a class="footnoteRef" href="#fn15" id="fnref15"><sup>15</sup></a></p>

<p>The discovery that enzymes could be crystallized eventually allowed their structures to be solved by <a href="x-ray_crystallography" title="wikilink">x-ray crystallography</a>. This was first done for <a class="uri" href="lysozyme" title="wikilink">lysozyme</a>, an enzyme found in tears, saliva and <a href="egg_white" title="wikilink">egg whites</a> that digests the coating of some bacteria; the structure was solved by a group led by <a href="David_Chilton_Phillips" title="wikilink">David Chilton Phillips</a> and published in 1965.<a class="footnoteRef" href="#fn16" id="fnref16"><sup>16</sup></a> This high-resolution structure of lysozyme marked the beginning of the field of <a href="structural_biology" title="wikilink">structural biology</a> and the effort to understand how enzymes work at an atomic level of detail.<a class="footnoteRef" href="#fn17" id="fnref17"><sup>17</sup></a></p>
<h2 id="structure">Structure</h2>
<figure><b>(Figure)</b>
<figcaption>Organisation of <a href="protein_structure" title="wikilink">enzyme structure</a> and <a class="uri" href="lysozyme" title="wikilink">lysozyme</a> example. Binding sites in blue, catalytic site in red and <a class="uri" href="peptidoglycan" title="wikilink">peptidoglycan</a> substrate in black. ()</figcaption>
</figure>

<p>Enzymes are generally <a href="globular_protein" title="wikilink">globular proteins</a>, acting alone or in larger <a href="protein_complex" title="wikilink">complexes</a>. Like all proteins, enzymes are linear chains of <a href="amino_acids" title="wikilink">amino acids</a> that <a href="protein_folding" title="wikilink">fold</a> to produce a <a href="tertiary_structure" title="wikilink">three-dimensional structure</a>. The sequence of the amino acids specifies the structure which in turn determines the catalytic activity of the enzyme.<a class="footnoteRef" href="#fn18" id="fnref18"><sup>18</sup></a> Although structure determines function, a novel enzyme's activity cannot yet be predicted from its structure alone.<a class="footnoteRef" href="#fn19" id="fnref19"><sup>19</sup></a> Enzyme structures unfold (<a href="denaturation_(biochemistry)" title="wikilink">denature</a>) when heated or exposed to chemical denaturants and this disruption to the structure typically causes a loss of activity.<a class="footnoteRef" href="#fn20" id="fnref20"><sup>20</sup></a></p>

<p>Enzymes are usually much larger than their substrates. Sizes range from just 62 amino acid residues, for the <a class="uri" href="monomer" title="wikilink">monomer</a> of <a href="4-Oxalocrotonate_tautomerase" title="wikilink">4-oxalocrotonate tautomerase</a>,<a class="footnoteRef" href="#fn21" id="fnref21"><sup>21</sup></a> to over 2,500 residues in the animal <a href="fatty_acid_synthase" title="wikilink">fatty acid synthase</a>.<a class="footnoteRef" href="#fn22" id="fnref22"><sup>22</sup></a> Only a small portion of their structure (around 2–4 amino acids) is directly involved in catalysis: the catalytic site.<a class="footnoteRef" href="#fn23" id="fnref23"><sup>23</sup></a> This catalytic site is located next to one or more <a href="binding_site" title="wikilink">binding sites</a> where residues orient the substrates. The catalytic site and binding site together comprise the enzyme's <a href="active_site" title="wikilink">active site</a>. The remaining majority of the enzyme structure serves to maintain the precise orientation and dynamics of the active site.<a class="footnoteRef" href="#fn24" id="fnref24"><sup>24</sup></a></p>

<p>In some enzymes, no amino acids are directly involved in catalysis; instead, the enzyme contains sites to bind and orient catalytic <a href="cofactor_(biochemistry)" title="wikilink">cofactors</a>.<a class="footnoteRef" href="#fn25" id="fnref25"><sup>25</sup></a> Enzyme structures may also contain <a href="allosteric_site" title="wikilink">allosteric sites</a> where the binding of a small molecule causes a <a href="conformational_change" title="wikilink">conformational change</a> that increases or decreases activity.<a class="footnoteRef" href="#fn26" id="fnref26"><sup>26</sup></a></p>

<p>A small number of <a href="Ribonucleic_acid" title="wikilink">RNA</a>-based biological catalysts called <a href="ribozyme" title="wikilink">ribozymes</a> exist, which again can act alone or in complex with proteins. The most common of these is the <a class="uri" href="ribosome" title="wikilink">ribosome</a> which is a complex of protein and catalytic RNA components.<a class="footnoteRef" href="#fn27" id="fnref27"><sup>27</sup></a></p>
<h2 id="mechanism">Mechanism</h2>
<h3 id="substrate-binding">Substrate binding</h3>

<p>Enzymes must bind their substrates before they can catalyse any chemical reaction. Enzymes are usually very specific as to what <a href="substrate_(biochemistry)" title="wikilink">substrates</a> they bind and then the chemical reaction catalysed. Specificity is achieved by binding pockets with complementary shape, charge and <a class="uri" href="hydrophilic" title="wikilink">hydrophilic</a>/<a class="uri" href="hydrophobic" title="wikilink">hydrophobic</a> characteristics to the substrates. Enzymes can therefore distinguish between very similar substrate molecules to be <a href="chemoselectivity" title="wikilink">chemoselective</a>, <a href="regioselectivity" title="wikilink">regioselective</a> and <a href="stereospecificity" title="wikilink">stereospecific</a>.<a class="footnoteRef" href="#fn28" id="fnref28"><sup>28</sup></a></p>

<p>Some of the enzymes showing the highest specificity and accuracy are involved in the copying and <a href="Gene_expression" title="wikilink">expression</a> of the <a class="uri" href="genome" title="wikilink">genome</a>. Some of these enzymes have "<a href="Proofreading_(biology)" title="wikilink">proof-reading</a>" mechanisms. Here, an enzyme such as <a href="DNA_polymerase" title="wikilink">DNA polymerase</a> catalyzes a reaction in a first step and then checks that the product is correct in a second step.<a class="footnoteRef" href="#fn29" id="fnref29"><sup>29</sup></a> This two-step process results in average error rates of less than 1 error in 100 million reactions in high-fidelity mammalian polymerases.<a class="footnoteRef" href="#fn30" id="fnref30"><sup>30</sup></a> Similar proofreading mechanisms are also found in <a href="RNA_polymerase" title="wikilink">RNA polymerase</a>,<a class="footnoteRef" href="#fn31" id="fnref31"><sup>31</sup></a> <a href="aminoacyl_tRNA_synthetase" title="wikilink">aminoacyl tRNA synthetases</a><a class="footnoteRef" href="#fn32" id="fnref32"><sup>32</sup></a> and <a href="ribosome" title="wikilink">ribosomes</a>.<a class="footnoteRef" href="#fn33" id="fnref33"><sup>33</sup></a></p>

<p>Conversely, some enzymes display <a href="enzyme_promiscuity" title="wikilink">enzyme promiscuity</a>, having broad specificity and acting on a range of different physiologically relevant substrates. Many enzymes possess small side activities which arose fortuitously (i.e. <a href="Neutral_evolution" title="wikilink">neutrally</a>), which may be the starting point for the evolutionary selection of a new function.<a class="footnoteRef" href="#fn34" id="fnref34"><sup>34</sup></a><a class="footnoteRef" href="#fn35" id="fnref35"><sup>35</sup></a></p>
<figure><b>(Figure)</b>
<figcaption>Enzyme changes shape by induced fit upon substrate binding to form enzyme-substrate complex. <a class="uri" href="Hexokinase" title="wikilink">Hexokinase</a> has a large induced fit motion that closes over the substrates <a href="adenosine_triphosphate" title="wikilink">adenosine triphosphate</a> and <a class="uri" href="xylose" title="wikilink">xylose</a>. Binding sites in blue, substrates in black and <a href="magnesium" title="wikilink">Mg<sup>2+</sup></a> cofactor in yellow. (, )</figcaption>
</figure>
<h4 id="lock-and-key-model">"Lock and key" model</h4>

<p>To explain the observed specificity of enzymes, in 1894 <a href="Hermann_Emil_Fischer" title="wikilink">Emil Fischer</a> proposed that both the enzyme and the substrate possess specific complementary geometric shapes that fit exactly into one another.<a class="footnoteRef" href="#fn36" id="fnref36"><sup>36</sup></a> This is often referred to as "the lock and key" model.<a class="footnoteRef" href="#fn37" id="fnref37"><sup>37</sup></a> This early model explains enzyme specificity, but fails to explain the stabilization of the transition state that enzymes achieve.<a class="footnoteRef" href="#fn38" id="fnref38"><sup>38</sup></a></p>
<h4 id="induced-fit-model">Induced fit model</h4>

<p>In 1958, <a href="Daniel_E._Koshland,_Jr." title="wikilink">Daniel Koshland</a> suggested a modification to the lock and key model: since enzymes are rather flexible structures, the active site is continuously reshaped by interactions with the substrate as the substrate interacts with the enzyme.<a class="footnoteRef" href="#fn39" id="fnref39"><sup>39</sup></a> As a result, the substrate does not simply bind to a rigid active site; the amino acid <a href="Side_chain" title="wikilink">side-chains</a> that make up the active site are molded into the precise positions that enable the enzyme to perform its catalytic function. In some cases, such as glycosidases, the substrate <a class="uri" href="molecule" title="wikilink">molecule</a> also changes shape slightly as it enters the active site.<a class="footnoteRef" href="#fn40" id="fnref40"><sup>40</sup></a> The active site continues to change until the substrate is completely bound, at which point the final shape and charge distribution is determined.<a class="footnoteRef" href="#fn41" id="fnref41"><sup>41</sup></a> Induced fit may enhance the fidelity of molecular recognition in the presence of competition and noise via the <a href="conformational_proofreading" title="wikilink">conformational proofreading</a> mechanism.<a class="footnoteRef" href="#fn42" id="fnref42"><sup>42</sup></a></p>
<h3 id="catalysis">Catalysis</h3>

<p>Enzymes can accelerate reactions in several ways, all of which lower the <a href="activation_energy" title="wikilink">activation energy</a> (ΔG<sup>‡</sup>, <a href="Gibbs_free_energy" title="wikilink">Gibbs free energy</a>)<a class="footnoteRef" href="#fn43" id="fnref43"><sup>43</sup></a></p>
<ol>
<li>By stabilizing the transition state:
<ul>
<li>Creating an environment with a charge distribution complementary to that of the transition state to lower its energy.<a class="footnoteRef" href="#fn44" id="fnref44"><sup>44</sup></a></li>
</ul></li>
<li>By providing an alternative reaction pathway:
<ul>
<li>Temporarily reacting with the substrate, forming a covalent intermediate to provide a lower energy transition state.<a class="footnoteRef" href="#fn45" id="fnref45"><sup>45</sup></a></li>
</ul></li>
<li>By destabilising the substrate ground state:
<ul>
<li>Distorting bound substrate(s) into their transition state form to reduce the energy required to reach the transition state.<a class="footnoteRef" href="#fn46" id="fnref46"><sup>46</sup></a></li>
<li>By orienting the substrates into a productive arrangement to reduce the reaction <a class="uri" href="entropy" title="wikilink">entropy</a> change.<a class="footnoteRef" href="#fn47" id="fnref47"><sup>47</sup></a> The contribution of this mechanism to catalysis is relatively small.<a class="footnoteRef" href="#fn48" id="fnref48"><sup>48</sup></a></li>
</ul></li>
</ol>

<p>Enzymes may use several of these mechanisms simultaneously. For example, <a href="protease" title="wikilink">proteases</a> such as <a class="uri" href="trypsin" title="wikilink">trypsin</a> perform covalent catalysis using a <a href="catalytic_triad" title="wikilink">catalytic triad</a>, stabilise charge build-up on the transition states using an <a href="oxyanion_hole" title="wikilink">oxyanion hole</a>, complete <a class="uri" href="hydrolysis" title="wikilink">hydrolysis</a> using an oriented water substrate.</p>
<h3 id="dynamics">Dynamics</h3>

<p>Enzymes are not rigid, static structures; instead they have complex internal dynamic motions – that is, movements of parts of the enzyme's structure such as individual amino acid residues, groups of residues forming a <a href="turn_(biochemistry)" title="wikilink">protein loop</a> or unit of <a href="protein_secondary_structure" title="wikilink">secondary structure</a>, or even an entire <a href="protein_domain" title="wikilink">protein domain</a>. These motions give rise to a <a href="conformational_ensemble" title="wikilink">conformational ensemble</a> of slightly different structures that interconvert with one another at <a href="thermodynamic_equilibrium" title="wikilink">equilibrium</a>. Different states within this ensemble may be associated with different aspects of an enzyme's function. For example, different conformations of the enzyme <a href="dihydrofolate_reductase" title="wikilink">dihydrofolate reductase</a> are associated with the substrate binding, catalysis, cofactor release, and product release steps of the catalytic cycle.<a class="footnoteRef" href="#fn49" id="fnref49"><sup>49</sup></a></p>
<h3 id="allosteric-modulation">Allosteric modulation</h3>

<p>Allosteric sites are pockets on the enzyme, distinct from the active site, that bind to molecules in the cellular environment. These molecules then cause a change in the conformation or dynamics of the enzyme that is transduced to the active site and thus affects the reaction rate of the enzyme.<a class="footnoteRef" href="#fn50" id="fnref50"><sup>50</sup></a> In this way, allosteric interactions can either inhibit or activate enzymes. Allosteric interactions with metabolites upstream or downstream in an enzyme's metabolic pathway cause <a class="uri" href="feedback" title="wikilink">feedback</a> regulation, altering the activity of the enzyme according to the <a href="Flux_(metabolism)" title="wikilink">flux</a> through the rest of the pathway.<a class="footnoteRef" href="#fn51" id="fnref51"><sup>51</sup></a></p>
<h2 id="cofactors">Cofactors</h2>
<figure><b>(Figure)</b>
<figcaption>Chemical structure for <a href="thiamine_pyrophosphate" title="wikilink">thiamine pyrophosphate</a> and protein structure of <a class="uri" href="transketolase" title="wikilink">transketolase</a>. Thiamine pyrophosphate cofactor in yellow and <a href="xylulose_5-phosphate" title="wikilink">xylulose 5-phosphate</a> substrate in black. ()</figcaption>
</figure>

<p>Some enzymes do not need additional components to show full activity. Others require non-protein molecules called cofactors to be bound for activity.<a class="footnoteRef" href="#fn52" id="fnref52"><sup>52</sup></a> Cofactors can be either <a class="uri" href="inorganic" title="wikilink">inorganic</a> (e.g., <a href="ion" title="wikilink">metal ions</a> and <a href="iron-sulfur_cluster" title="wikilink">iron-sulfur clusters</a>) or <a href="organic_molecules" title="wikilink">organic compounds</a> (e.g., <a href="flavin_group" title="wikilink">flavin</a> and <a class="uri" href="heme" title="wikilink">heme</a>). Organic cofactors can be either <a href="coenzyme" title="wikilink">coenzymes</a>, which are released from the enzyme's active site during the reaction, or <a href="prosthetic_groups" title="wikilink">prosthetic groups</a>, which are tightly bound to an enzyme. Organic prosthetic groups can be covalently bound (e.g., <a class="uri" href="biotin" title="wikilink">biotin</a> in enzymes such as <a href="pyruvate_carboxylase" title="wikilink">pyruvate carboxylase</a>).<a class="footnoteRef" href="#fn53" id="fnref53"><sup>53</sup></a></p>

<p>An example of an enzyme that contains a cofactor is <a href="carbonic_anhydrase" title="wikilink">carbonic anhydrase</a>, which is shown in the <a href="ribbon_diagram" title="wikilink">ribbon diagram</a> above with a zinc cofactor bound as part of its active site.<a class="footnoteRef" href="#fn54" id="fnref54"><sup>54</sup></a> These tightly bound ions or molecules are usually found in the active site and are involved in catalysis.<a class="footnoteRef" href="#fn55" id="fnref55"><sup>55</sup></a> For example, flavin and heme cofactors are often involved in <a class="uri" href="redox" title="wikilink">redox</a> reactions.<a class="footnoteRef" href="#fn56" id="fnref56"><sup>56</sup></a></p>

<p>Enzymes that require a cofactor but do not have one bound are called <em>apoenzymes</em> or <em>apoproteins</em>. An enzyme together with the cofactor(s) required for activity is called a <em>holoenzyme</em> (or haloenzyme). The term <em>holoenzyme</em> can also be applied to enzymes that contain multiple protein subunits, such as the <a href="DNA_polymerase" title="wikilink">DNA polymerases</a>; here the holoenzyme is the complete complex containing all the subunits needed for activity.<a class="footnoteRef" href="#fn57" id="fnref57"><sup>57</sup></a></p>
<h3 id="coenzymes">Coenzymes</h3>

<p>Coenzymes are small organic molecules that can be loosely or tightly bound to an enzyme. Coenzymes transport chemical groups from one enzyme to another.<a class="footnoteRef" href="#fn58" id="fnref58"><sup>58</sup></a> Examples include <a href="Nicotinamide_adenine_dinucleotide" title="wikilink">NADH</a>, <a href="Nicotinamide_adenine_dinucleotide_phosphate" title="wikilink">NADPH</a> and <a href="adenosine_triphosphate" title="wikilink">adenosine triphosphate</a> (ATP). Some coenzymes, such as <a class="uri" href="riboflavin" title="wikilink">riboflavin</a>, <a class="uri" href="thiamine" title="wikilink">thiamine</a> and <a href="folic_acid" title="wikilink">folic acid</a>, are <a class="uri" href="vitamins" title="wikilink">vitamins</a>, or compounds that cannot be synthesized by the body and must be acquired from the diet. The chemical groups carried include the <a class="uri" href="hydride" title="wikilink">hydride</a> ion (H<sup>−</sup>) carried by <a href="nicotinamide_adenine_dinucleotide" title="wikilink">NAD or NADP<sup>+</sup></a>, the phosphate group carried by <a href="adenosine_triphosphate" title="wikilink">adenosine triphosphate</a>, the acetyl group carried by <a href="coenzyme_A" title="wikilink">coenzyme A</a>, formyl, methenyl or methyl groups carried by <a href="folic_acid" title="wikilink">folic acid</a> and the methyl group carried by <a class="uri" href="S-adenosylmethionine" title="wikilink">S-adenosylmethionine</a>.<a class="footnoteRef" href="#fn59" id="fnref59"><sup>59</sup></a></p>

<p>Since coenzymes are chemically changed as a consequence of enzyme action, it is useful to consider coenzymes to be a special class of substrates, or second substrates, which are common to many different enzymes. For example, about 1000 enzymes are known to use the coenzyme NADH.<a class="footnoteRef" href="#fn60" id="fnref60"><sup>60</sup></a></p>

<p>Coenzymes are usually continuously regenerated and their concentrations maintained at a steady level inside the cell. For example, NADPH is regenerated through the <a href="pentose_phosphate_pathway" title="wikilink">pentose phosphate pathway</a> and <em>S</em>-adenosylmethionine by <a href="methionine_adenosyltransferase" title="wikilink">methionine adenosyltransferase</a>. This continuous regeneration means that small amounts of coenzymes can be used very intensively. For example, the human body turns over its own weight in ATP each day.<a class="footnoteRef" href="#fn61" id="fnref61"><sup>61</sup></a></p>
<h2 id="thermodynamics">Thermodynamics</h2>
<figure><b>(Figure)</b>
<figcaption>The energies of the stages of a <a href="chemical_reaction" title="wikilink">chemical reaction</a>. Uncatalysed (dashed line), substrates need a lot of <a href="activation_energy" title="wikilink">activation energy</a> to reach a <a href="transition_state" title="wikilink">transition state</a>, which then decays into lower-energy products. When enzyme catalysed (solid line), the enzyme binds the substrates (ES), then stabilizes the transition state (ES<sup>‡</sup>) to reduce the activation energy required to produce products (EP) which are finally released.</figcaption>
</figure>

<p>As with all catalysts, enzymes do not alter the position of the chemical equilibrium of the reaction. In the presence of an enzyme, the reaction runs in the same direction as it would without the enzyme, just more quickly.<a class="footnoteRef" href="#fn62" id="fnref62"><sup>62</sup></a> For example, <a href="carbonic_anhydrase" title="wikilink">carbonic anhydrase</a> catalyzes its reaction in either direction depending on the concentration of its reactants:<a class="footnoteRef" href="#fn63" id="fnref63"><sup>63</sup></a></p>

<p>

<math display="block" id="Enzyme:0">
 <semantics>
  <mrow>
   <mrow>
    <msub>
     <mi>CO</mi>
     <mn>2</mn>
    </msub>
    <mo>+</mo>
    <mrow>
     <msub>
      <mi mathvariant="normal">H</mi>
      <mn>2</mn>
     </msub>
     <mi mathvariant="normal">O</mi>
    </mrow>
   </mrow>
   <mover accent="true">
    <mo>→</mo>
    <mrow>
     <mpadded width="+5pt">
      <mi>Carbonic</mi>
     </mpadded>
     <mi>anhydrase</mi>
    </mrow>
   </mover>
   <mrow>
    <msub>
     <mi mathvariant="normal">H</mi>
     <mn>2</mn>
    </msub>
    <msub>
     <mi>CO</mi>
     <mn>3</mn>
    </msub>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <apply>
     <apply>
      <times></times>
      <ci>Carbonic</ci>
      <ci>anhydrase</ci>
     </apply>
     <ci>normal-→</ci>
    </apply>
    <apply>
     <plus></plus>
     <apply>
      <csymbol cd="ambiguous">subscript</csymbol>
      <ci>CO</ci>
      <cn type="integer">2</cn>
     </apply>
     <apply>
      <times></times>
      <apply>
       <csymbol cd="ambiguous">subscript</csymbol>
       <ci>normal-H</ci>
       <cn type="integer">2</cn>
      </apply>
      <ci>normal-O</ci>
     </apply>
    </apply>
    <apply>
     <times></times>
     <apply>
      <csymbol cd="ambiguous">subscript</csymbol>
      <ci>normal-H</ci>
      <cn type="integer">2</cn>
     </apply>
     <apply>
      <csymbol cd="ambiguous">subscript</csymbol>
      <ci>CO</ci>
      <cn type="integer">3</cn>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \mathrm{CO_{2}+H_{2}O\xrightarrow{Carbonic\ anhydrase}H_{2}CO_{3}}
  </annotation>
 </semantics>
</math>

 (in <a href="Biological_tissue" title="wikilink">tissues</a>; high CO<sub>2</sub> concentration)</p>

<p>

<math display="block" id="Enzyme:1">
 <semantics>
  <mrow>
   <mrow>
    <msub>
     <mi mathvariant="normal">H</mi>
     <mn>2</mn>
    </msub>
    <msub>
     <mi>CO</mi>
     <mn>3</mn>
    </msub>
   </mrow>
   <mover accent="true">
    <mo>→</mo>
    <mrow>
     <mpadded width="+5pt">
      <mi>Carbonic</mi>
     </mpadded>
     <mi>anhydrase</mi>
    </mrow>
   </mover>
   <mrow>
    <msub>
     <mi>CO</mi>
     <mn>2</mn>
    </msub>
    <mo>+</mo>
    <mrow>
     <msub>
      <mi mathvariant="normal">H</mi>
      <mn>2</mn>
     </msub>
     <mi mathvariant="normal">O</mi>
    </mrow>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <apply>
     <apply>
      <times></times>
      <ci>Carbonic</ci>
      <ci>anhydrase</ci>
     </apply>
     <ci>normal-→</ci>
    </apply>
    <apply>
     <times></times>
     <apply>
      <csymbol cd="ambiguous">subscript</csymbol>
      <ci>normal-H</ci>
      <cn type="integer">2</cn>
     </apply>
     <apply>
      <csymbol cd="ambiguous">subscript</csymbol>
      <ci>CO</ci>
      <cn type="integer">3</cn>
     </apply>
    </apply>
    <apply>
     <plus></plus>
     <apply>
      <csymbol cd="ambiguous">subscript</csymbol>
      <ci>CO</ci>
      <cn type="integer">2</cn>
     </apply>
     <apply>
      <times></times>
      <apply>
       <csymbol cd="ambiguous">subscript</csymbol>
       <ci>normal-H</ci>
       <cn type="integer">2</cn>
      </apply>
      <ci>normal-O</ci>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \mathrm{H_{2}CO_{3}\xrightarrow{Carbonic\ anhydrase}CO_{2}+H_{2}O}
  </annotation>
 </semantics>
</math>

 (in <a href="lung" title="wikilink">lungs</a>; low CO<sub>2</sub> concentration)</p>

<p>The rate of a reaction is dependent on the <a href="activation_energy" title="wikilink">activation energy</a> needed to form the <a href="transition_state" title="wikilink">transition state</a> which then decays into products. Enzymes increase reaction rates by lowering the energy of the transition state. First, binding forms a low energy enzyme-substrate complex (ES). Secondly the enzyme stabilises the transition state such that it requires less energy to achieve compared to the uncatalyzed reaction (ES<sup>‡</sup>). Finally the enzyme-product complex (EP) dissociates to release the products.<a class="footnoteRef" href="#fn64" id="fnref64"><sup>64</sup></a></p>

<p>Enzymes can couple two or more reactions, so that a thermodynamically favorable reaction can be used to "drive" a thermodynamically unfavourable one so that the combined energy of the products is lower than the substrates. For example, the hydrolysis of <a href="Adenosine_triphosphate" title="wikilink">ATP</a> is often used to drive other chemical reactions.<a class="footnoteRef" href="#fn65" id="fnref65"><sup>65</sup></a></p>
<h2 id="kinetics">Kinetics</h2>

<p></p>

<p>Enzyme kinetics is the investigation of how enzymes bind substrates and turn them into products. The rate data used in kinetic analyses are commonly obtained from <a href="enzyme_assay" title="wikilink">enzyme assays</a>. In 1913 <a href="Leonor_Michaelis" title="wikilink">Leonor Michaelis</a> and <a href="Maud_Leonora_Menten" title="wikilink">Maud Leonora Menten</a> proposed a quantitative theory of enzyme kinetics, which is referred to as <a href="Michaelis–Menten_kinetics" title="wikilink">Michaelis–Menten kinetics</a>.<a class="footnoteRef" href="#fn66" id="fnref66"><sup>66</sup></a> The major contribution of Michaelis and Menten was to think of enzyme reactions in two stages. In the first, the substrate binds reversibly to the enzyme, forming the enzyme-substrate complex. This is sometimes called the Michaelis-Menten complex in their honor. The enzyme then catalyzes the chemical step in the reaction and releases the product. This work was further developed by <a href="George_Edward_Briggs" title="wikilink">G. E. Briggs</a> and <a href="J._B._S._Haldane" title="wikilink">J. B. S. Haldane</a>, who derived kinetic equations that are still widely used today.<a class="footnoteRef" href="#fn67" id="fnref67"><sup>67</sup></a></p>

<p>Enzyme rates depend on <a class="uri" href="solution" title="wikilink">solution</a> conditions and substrate <a class="uri" href="concentration" title="wikilink">concentration</a>. To find the maximum speed of an enzymatic reaction, the substrate concentration is increased until a constant rate of product formation is seen. This is shown in the saturation curve on the right. Saturation happens because, as substrate concentration increases, more and more of the free enzyme is converted into the substrate-bound ES complex. At the maximum reaction rate (<em>V</em><sub>max</sub>) of the enzyme, all the enzyme active sites are bound to substrate, and the amount of ES complex is the same as the total amount of enzyme.<a class="footnoteRef" href="#fn68" id="fnref68"><sup>68</sup></a></p>

<p><em>V</em><sub>max</sub> is only one of several important kinetic parameters. The amount of substrate needed to achieve a given rate of reaction is also important. This is given by the <a href="Michaelis-Menten_constant" title="wikilink">Michaelis-Menten constant</a> (<em>K</em><sub>m</sub>), which is the substrate concentration required for an enzyme to reach one-half its maximum reaction rate; generally, each enzyme has a characteristic <em>K</em><sub>m</sub> for a given substrate. Another useful constant is <em>k</em><sub>cat</sub>, also called the <em>turnover number</em>, which is the number of substrate molecules handled by one active site per second.<a class="footnoteRef" href="#fn69" id="fnref69"><sup>69</sup></a></p>

<p>The efficiency of an enzyme can be expressed in terms of <em>k</em><sub>cat</sub>/<em>K</em><sub>m</sub>. This is also called the specificity constant and incorporates the <a href="rate_constant" title="wikilink">rate constants</a> for all steps in the reaction up to and including the first irreversible step. Because the specificity constant reflects both affinity and catalytic ability, it is useful for comparing different enzymes against each other, or the same enzyme with different substrates. The theoretical maximum for the specificity constant is called the diffusion limit and is about 10<sup>8</sup> to 10<sup>9</sup> (M<sup>−1</sup> s<sup>−1</sup>). At this point every collision of the enzyme with its substrate will result in catalysis, and the rate of product formation is not limited by the reaction rate but by the diffusion rate. Enzymes with this property are called <em><a href="catalytically_perfect_enzyme" title="wikilink">catalytically perfect</a></em> or <em>kinetically perfect</em>. Example of such enzymes are <a href="triosephosphateisomerase" title="wikilink">triose-phosphate isomerase</a>, <a href="carbonic_anhydrase" title="wikilink">carbonic anhydrase</a>, <a class="uri" href="acetylcholinesterase" title="wikilink">acetylcholinesterase</a>, <a class="uri" href="catalase" title="wikilink">catalase</a>, <a class="uri" href="fumarase" title="wikilink">fumarase</a>, <a class="uri" href="β-lactamase" title="wikilink">β-lactamase</a>, and <a href="superoxide_dismutase" title="wikilink">superoxide dismutase</a>.<a class="footnoteRef" href="#fn70" id="fnref70"><sup>70</sup></a> The turnover of such enzymes can reach several million reactions per second.<a class="footnoteRef" href="#fn71" id="fnref71"><sup>71</sup></a></p>

<p>Michaelis–Menten kinetics relies on the <a href="law_of_mass_action" title="wikilink">law of mass action</a>, which is derived from the assumptions of free <a class="uri" href="diffusion" title="wikilink">diffusion</a> and thermodynamically driven random collision. Many biochemical or cellular processes deviate significantly from these conditions, because of <a href="macromolecular_crowding" title="wikilink">macromolecular crowding</a> and constrained molecular movement.<a class="footnoteRef" href="#fn72" id="fnref72"><sup>72</sup></a> More recent, complex extensions of the model attempt to correct for these effects.<a class="footnoteRef" href="#fn73" id="fnref73"><sup>73</sup></a></p>
<h2 id="inhibition">Inhibition</h2>

<p></p>

<p>Enzyme reaction rates can be decreased by various types of <a href="enzyme_inhibitor" title="wikilink">enzyme inhibitors</a>.<a class="footnoteRef" href="#fn74" id="fnref74"><sup>74</sup></a></p>
<h3 id="types-of-inhibition">Types of inhibition</h3>
<dl>
<dt>Competitive: A <a href="competitive_inhibitor" title="wikilink">competitive inhibitor</a> and substrate cannot bind to the enzyme at the same time.<a class="footnoteRef" href="#fn75" id="fnref75"><sup>75</sup></a> Often competitive inhibitors strongly resemble the real substrate of the enzyme. For example, the drug <a class="uri" href="methotrexate" title="wikilink">methotrexate</a> is a competitive inhibitor of the enzyme <a href="dihydrofolate_reductase" title="wikilink">dihydrofolate reductase</a>, which catalyzes the reduction of <a href="folic_acid" title="wikilink">dihydrofolate</a> to tetrahydrofolate. The similarity between the structures of dihydrofolate and this drug are shown in the accompanying figure. This type of inhibition can be overcome with high substrate concentration. In some cases, the inhibitor can bind to a site other than the binding-site of the usual substrate and exert an <a href="#Allosteric_modulation" title="wikilink">allosteric effect</a> to change the shape of the usual binding-site.<br/>
Non-competitive: A <a href="non-competitive_inhibition" title="wikilink">non-competitive inhibitor</a> binds to a site other than where the substrate binds. The substrate still binds with its usual affinity and hence K<sub>m</sub> remains the same. However the inhibitor reduces the catalytic efficiency of the enzyme so that V<sub>max</sub> is reduced. In contrast to competitive inhibition, non-competitive inhibition cannot be overcome with high substrate concentration.<a class="footnoteRef" href="#fn76" id="fnref76"><sup>76</sup></a><br/>
Uncompetitive: An <a href="uncompetitive_inhibitor" title="wikilink">uncompetitive inhibitor</a> cannot bind to the free enzyme, only to the enzyme-substrate complex; hence, these types of inhibitors are most effective at high substrate concentration. In the presence of the inhibitor, the enzyme-substrate complex is inactive.<a class="footnoteRef" href="#fn77" id="fnref77"><sup>77</sup></a> This type of inhibition is rare.<a class="footnoteRef" href="#fn78" id="fnref78"><sup>78</sup></a><br/>
Mixed: A <a href="mixed_inhibition" title="wikilink">mixed inhibitor</a> binds to an allosteric site and the binding of the substrate and the inhibitor affect each other. The enzyme's function is reduced but not eliminated when bound to the inhibitor. This type of inhibitor does not follow the Michaelis-Menten equation.<a class="footnoteRef" href="#fn79" id="fnref79"><sup>79</sup></a><br/>
Irreversible: An <a href="irreversible_inhibitor" title="wikilink">irreversible inhibitor</a> permanently inactivates the enzyme, usually by forming a <a href="covalent_bond" title="wikilink">covalent bond</a> to the protein. <a class="uri" href="Penicillin" title="wikilink">Penicillin</a><a class="footnoteRef" href="#fn80" id="fnref80"><sup>80</sup></a> and <a class="uri" href="aspirin" title="wikilink">aspirin</a><a class="footnoteRef" href="#fn81" id="fnref81"><sup>81</sup></a> are common drugs that act in this manner.</dt>
</dl>
<h3 id="functions-of-inhibitors">Functions of inhibitors</h3>

<p>In many organisms, inhibitors may act as part of a <a class="uri" href="feedback" title="wikilink">feedback</a> mechanism. If an enzyme produces too much of one substance in the organism, that substance may act as an inhibitor for the enzyme at the beginning of the pathway that produces it, causing production of the substance to slow down or stop when there is sufficient amount. This is a form of <a href="negative_feedback" title="wikilink">negative feedback</a>. Major metabolic pathways such as the <a href="citric_acid_cycle" title="wikilink">citric acid cycle</a> make use of this mechanism.<a class="footnoteRef" href="#fn82" id="fnref82"><sup>82</sup></a></p>

<p>Since inhibitors modulate the function of enzymes they are often used as drugs. Many such drugs are reversible competitive inhibitors that resemble the enzyme's native substrate, similar to <a class="uri" href="methotrexate" title="wikilink">methotrexate</a> above; other well-known examples include <a href="statin" title="wikilink">statins</a> used to treat high <a class="uri" href="cholesterol" title="wikilink">cholesterol</a>,<a class="footnoteRef" href="#fn83" id="fnref83"><sup>83</sup></a> and <a href="protease_inhibitor_(pharmacology)" title="wikilink">protease inhibitors</a> used to treat <a href="retrovirus" title="wikilink">retroviral</a> infections such as <a class="uri" href="HIV" title="wikilink">HIV</a>.<a class="footnoteRef" href="#fn84" id="fnref84"><sup>84</sup></a> A common example of an irreversible inhibitor that is used as a drug is <a class="uri" href="aspirin" title="wikilink">aspirin</a>, which inhibits the <a href="Cyclooxygenase" title="wikilink">COX-1</a> and <a href="Cyclooxygenase" title="wikilink">COX-2</a> enzymes that produce the <a class="uri" href="inflammation" title="wikilink">inflammation</a> messenger <a class="uri" href="prostaglandin" title="wikilink">prostaglandin</a>.<a class="footnoteRef" href="#fn85" id="fnref85"><sup>85</sup></a> Other enzyme inhibitors are poisons. For example, the poison <a class="uri" href="cyanide" title="wikilink">cyanide</a> is an irreversible enzyme inhibitor that combines with the copper and iron in the active site of the enzyme <a href="cytochrome_c_oxidase" title="wikilink">cytochrome c oxidase</a> and blocks <a href="cellular_respiration" title="wikilink">cellular respiration</a>.<a class="footnoteRef" href="#fn86" id="fnref86"><sup>86</sup></a></p>
<h2 id="biological-function">Biological function</h2>

<p>Enzymes serve a wide variety of <a href="function_(biology)" title="wikilink">functions</a> inside living organisms. They are indispensable for <a href="signal_transduction" title="wikilink">signal transduction</a> and cell regulation, often via <a href="kinase" title="wikilink">kinases</a> and <a href="phosphatase" title="wikilink">phosphatases</a>.<a class="footnoteRef" href="#fn87" id="fnref87"><sup>87</sup></a> They also generate movement, with <a class="uri" href="myosin" title="wikilink">myosin</a> hydrolyzing ATP to generate <a href="muscle_contraction" title="wikilink">muscle contraction</a> and also moving cargo around the cell as part of the <a class="uri" href="cytoskeleton" title="wikilink">cytoskeleton</a>.<a class="footnoteRef" href="#fn88" id="fnref88"><sup>88</sup></a> Other ATPases in the cell membrane are <a href="ion_pump_(biology)" title="wikilink">ion pumps</a> involved in <a href="active_transport" title="wikilink">active transport</a>. Enzymes are also involved in more exotic functions, such as <a class="uri" href="luciferase" title="wikilink">luciferase</a> generating light in <a href="firefly" title="wikilink">fireflies</a>.<a class="footnoteRef" href="#fn89" id="fnref89"><sup>89</sup></a> <a href="Virus" title="wikilink">Viruses</a> can also contain enzymes for infecting cells, such as the <a href="HIV_integrase" title="wikilink">HIV integrase</a> and <a href="reverse_transcriptase" title="wikilink">reverse transcriptase</a>, or for viral release from cells, like the <a class="uri" href="influenza" title="wikilink">influenza</a> virus <a class="uri" href="neuraminidase" title="wikilink">neuraminidase</a>.<a class="footnoteRef" href="#fn90" id="fnref90"><sup>90</sup></a></p>

<p>An important function of enzymes is in the <a href="digestive_systems" title="wikilink">digestive systems</a> of animals. Enzymes such as <a class="uri" href="amylases" title="wikilink">amylases</a> and <a class="uri" href="proteases" title="wikilink">proteases</a> break down large molecules (<a class="uri" href="starch" title="wikilink">starch</a> or <a href="protein" title="wikilink">proteins</a>, respectively) into smaller ones, so they can be absorbed by the intestines. Starch molecules, for example, are too large to be absorbed from the intestine, but enzymes hydrolyze the starch chains into smaller molecules such as <a class="uri" href="maltose" title="wikilink">maltose</a> and eventually <a class="uri" href="glucose" title="wikilink">glucose</a>, which can then be absorbed. Different enzymes digest different food substances. In <a class="uri" href="ruminants" title="wikilink">ruminants</a>, which have <a class="uri" href="herbivorous" title="wikilink">herbivorous</a> diets, microorganisms in the gut produce another enzyme, <a class="uri" href="cellulase" title="wikilink">cellulase</a>, to break down the cellulose cell walls of plant fiber.<a class="footnoteRef" href="#fn91" id="fnref91"><sup>91</sup></a></p>
<h3 id="metabolism">Metabolism</h3>
<figure><b>(Figure)</b>
<figcaption>The <a href="metabolic_pathway" title="wikilink">metabolic pathway</a> of <a class="uri" href="glycolysis" title="wikilink">glycolysis</a> releases energy by converting <a class="uri" href="glucose" title="wikilink">glucose</a> to <a class="uri" href="pyruvate" title="wikilink">pyruvate</a> by via a series of intermediate metabolites. Each chemical modification (red box) is performed by a different enzyme.</figcaption>
</figure>

<p>Several enzymes can work together in a specific order, creating <a href="metabolic_pathway" title="wikilink">metabolic pathways</a>.<a class="footnoteRef" href="#fn92" id="fnref92"><sup>92</sup></a> In a metabolic pathway, one enzyme takes the product of another enzyme as a substrate. After the catalytic reaction, the product is then passed on to another enzyme. Sometimes more than one enzyme can catalyze the same reaction in parallel; this can allow more complex regulation: with, for example, a low constant activity provided by one enzyme but an inducible high activity from a second enzyme.<a class="footnoteRef" href="#fn93" id="fnref93"><sup>93</sup></a></p>

<p>Enzymes determine what steps occur in these pathways. Without enzymes, metabolism would neither progress through the same steps and could not be regulated to serve the needs of the cell. Most central metabolic pathways are regulated at a few key steps, typically through enzymes whose activity involves the hydrolysis of <a href="Adenosine_triphosphate" title="wikilink">ATP</a>. Because this reaction releases so much energy, other reactions that are <a href="endothermic" title="wikilink">thermodynamically unfavorable</a> can be coupled to ATP hydrolysis, driving the overall series of linked metabolic reactions.<a class="footnoteRef" href="#fn94" id="fnref94"><sup>94</sup></a></p>
<h3 id="control-of-activity">Control of activity</h3>

<p>There are five main ways that enzyme activity is controlled in the cell.<a class="footnoteRef" href="#fn95" id="fnref95"><sup>95</sup></a></p>
<dl>
<dt>Regulation: Enzymes can be either <a href="enzyme_activator" title="wikilink">activated</a> or <a href="enzyme_inhibitor" title="wikilink">inhibited</a> by other molecules. For example, the end product(s) of a metabolic pathway are often inhibitors for one of the first enzymes of the pathway (usually the first irreversible step, called committed step), thus regulating the amount of end product made by the pathways. Such a regulatory mechanism is called a <a href="negative_feedback" title="wikilink">negative feedback mechanism</a>, because the amount of the end product produced is regulated by its own concentration.<a class="footnoteRef" href="#fn96" id="fnref96"><sup>96</sup></a> Negative feedback mechanism can effectively adjust the rate of synthesis of intermediate metabolites according to the demands of the cells. This helps with effective allocations of materials and energy economy, and it prevents the excess manufacture of end products. Like other <a href="homeostasis" title="wikilink">homeostatic devices</a>, the control of enzymatic action helps to maintain a stable internal environment in living organisms.<a class="footnoteRef" href="#fn97" id="fnref97"><sup>97</sup></a></dt>
</dl>
<dl>
<dt>Post-translational modification: Examples of <a href="post-translational_modification" title="wikilink">post-translational modification</a> include <a class="uri" href="phosphorylation" title="wikilink">phosphorylation</a>, <a class="uri" href="myristoylation" title="wikilink">myristoylation</a> and <a class="uri" href="glycosylation" title="wikilink">glycosylation</a>.<a class="footnoteRef" href="#fn98" id="fnref98"><sup>98</sup></a> For example, in the response to <a class="uri" href="insulin" title="wikilink">insulin</a>, the <a class="uri" href="phosphorylation" title="wikilink">phosphorylation</a> of multiple enzymes, including <a href="glycogen_synthase" title="wikilink">glycogen synthase</a>, helps control the synthesis or degradation of <a class="uri" href="glycogen" title="wikilink">glycogen</a> and allows the cell to respond to changes in <a href="blood_sugar" title="wikilink">blood sugar</a>.<a class="footnoteRef" href="#fn99" id="fnref99"><sup>99</sup></a> Another example of post-translational modification is the cleavage of the polypeptide chain. <a class="uri" href="Chymotrypsin" title="wikilink">Chymotrypsin</a>, a digestive <a class="uri" href="protease" title="wikilink">protease</a>, is produced in inactive form as <a class="uri" href="chymotrypsinogen" title="wikilink">chymotrypsinogen</a> in the <a class="uri" href="pancreas" title="wikilink">pancreas</a> and transported in this form to the <a class="uri" href="stomach" title="wikilink">stomach</a> where it is activated. This stops the enzyme from digesting the pancreas or other tissues before it enters the gut. This type of inactive precursor to an enzyme is known as a <a class="uri" href="zymogen" title="wikilink">zymogen</a>.<a class="footnoteRef" href="#fn100" id="fnref100"><sup>100</sup></a></dt>
</dl>
<dl>
<dt>Quantity: Enzyme production (<a href="Transcription_(genetics)" title="wikilink">transcription</a> and <a href="Translation_(genetics)" title="wikilink">translation</a> of enzyme genes) can be enhanced or diminished by a cell in response to changes in the cell's environment. This form of <a href="regulation_of_gene_expression" title="wikilink">gene regulation</a> is called <a href="enzyme_induction" title="wikilink">enzyme induction</a>. For example, bacteria may become <a href="antibiotic_resistance" title="wikilink">resistant to antibiotics</a> such as <a class="uri" href="penicillin" title="wikilink">penicillin</a> because enzymes called <a href="beta-lactamase" title="wikilink">beta-lactamases</a> are induced that hydrolyse the crucial <a href="Beta-lactam" title="wikilink">beta-lactam ring</a> within the penicillin molecule.<a class="footnoteRef" href="#fn101" id="fnref101"><sup>101</sup></a> Another example comes from enzymes in the <a class="uri" href="liver" title="wikilink">liver</a> called <a href="cytochrome_P450_oxidase" title="wikilink">cytochrome P450 oxidases</a>, which are important in <a href="drug_metabolism" title="wikilink">drug metabolism</a>. Induction or inhibition of these enzymes can cause <a href="drug_interaction" title="wikilink">drug interactions</a>.<a class="footnoteRef" href="#fn102" id="fnref102"><sup>102</sup></a> Enzyme levels can also be regulated by changing the rate of enzyme <a href="catabolism" title="wikilink">degradation</a>.<a class="footnoteRef" href="#fn103" id="fnref103"><sup>103</sup></a></dt>
</dl>
<dl>
<dt>Subcellular distribution: Enzymes can be compartmentalized, with different metabolic pathways occurring in different <a href="cellular_compartment" title="wikilink">cellular compartments</a>. For example, <a href="fatty_acids" title="wikilink">fatty acids</a> are synthesized by one set of enzymes in the <a class="uri" href="cytosol" title="wikilink">cytosol</a>, <a href="endoplasmic_reticulum" title="wikilink">endoplasmic reticulum</a> and <a href="golgi_apparatus" title="wikilink">golgi</a> and used by a different set of enzymes as a source of energy in the <a class="uri" href="mitochondrion" title="wikilink">mitochondrion</a>, through <a class="uri" href="β-oxidation" title="wikilink">β-oxidation</a>.<a class="footnoteRef" href="#fn104" id="fnref104"><sup>104</sup></a> In addition, <a href="protein_targeting" title="wikilink">trafficking</a> of the enzyme to different compartments may change the degree of <a class="uri" href="protonation" title="wikilink">protonation</a> (<a class="uri" href="cytoplasm" title="wikilink">cytoplasm</a> neutral and <a class="uri" href="lysosome" title="wikilink">lysosome</a> acidic) or oxidative state [e.g., oxidized (<a class="uri" href="periplasm" title="wikilink">periplasm</a>) or reduced (<a class="uri" href="cytoplasm" title="wikilink">cytoplasm</a>)] which in turn affects enzyme activity.<a class="footnoteRef" href="#fn105" id="fnref105"><sup>105</sup></a></dt>
</dl>
<dl>
<dt>Organ specialization: In <a class="uri" href="multicellular" title="wikilink">multicellular</a> <a href="eukaryote" title="wikilink">eukaryotes</a>, cells in different <a href="organ_(anatomy)" title="wikilink">organs</a> and <a href="tissue_(biology)" title="wikilink">tissues</a> have different patterns of <a href="gene_expression" title="wikilink">gene expression</a> and therefore have different sets of enzymes (known as <a href="isozyme" title="wikilink">isozymes</a>) available for metabolic reactions. This provides a mechanism for regulating the overall metabolism of the organism. For example, <a class="uri" href="hexokinase" title="wikilink">hexokinase</a>, the first enzyme in the <a class="uri" href="glycolysis" title="wikilink">glycolysis</a> pathway, has a specialized form called <a class="uri" href="glucokinase" title="wikilink">glucokinase</a> expressed in the <a class="uri" href="liver" title="wikilink">liver</a> and <a class="uri" href="pancreas" title="wikilink">pancreas</a> that has a lower <a href="affinity_(pharmacology)" title="wikilink">affinity</a> for glucose yet is more sensitive to glucose concentration.<a class="footnoteRef" href="#fn106" id="fnref106"><sup>106</sup></a> This enzyme is involved in sensing <a href="blood_sugar" title="wikilink">blood sugar</a> and regulating <a class="uri" href="insulin" title="wikilink">insulin</a> production.<a class="footnoteRef" href="#fn107" id="fnref107"><sup>107</sup></a></dt>
</dl>
<h3 id="involvement-in-disease">Involvement in disease</h3>
<figure><b>(Figure)</b>
<figcaption>In <a href="phenylalanine_hydroxylase" title="wikilink">phenylalanine hydroxylase</a> over 300 different mutations throughout the structure cause <a class="uri" href="phenylketonuria" title="wikilink">phenylketonuria</a>. <a class="uri" href="Phenylalanine" title="wikilink">Phenylalanine</a> substrate and <a class="uri" href="tetrahydrobiopterin" title="wikilink">tetrahydrobiopterin</a> coenzyme in black, and <a href="Iron" title="wikilink">Fe<sup>2+</sup></a> cofactor in yellow. ()</figcaption>
</figure>

<p>Since the tight control of enzyme activity is essential for <a class="uri" href="homeostasis" title="wikilink">homeostasis</a>, any malfunction (mutation, overproduction, underproduction or deletion) of a single critical enzyme can lead to a <a href="genetic_disease" title="wikilink">genetic disease</a>. The malfunction of just one type of enzyme out of the thousands of types present in the human body can be fatal. An example of a fatal <a href="genetic_disease" title="wikilink">genetic disease</a> due to enzyme insufficiency is <a class="uri" href="Tay-Sachs" title="wikilink">Tay-Sachs</a> disease, in which patients lack the enzyme <a class="uri" href="hexosaminidase" title="wikilink">hexosaminidase</a>.<a class="footnoteRef" href="#fn108" id="fnref108"><sup>108</sup></a><a class="footnoteRef" href="#fn109" id="fnref109"><sup>109</sup></a></p>

<p>One example of enzyme deficiency is the most common type of <a class="uri" href="phenylketonuria" title="wikilink">phenylketonuria</a>. Many different single amino acid mutations in the enzyme <a href="phenylalanine_hydroxylase" title="wikilink">phenylalanine hydroxylase</a>, which catalyzes the first step in the degradation of <a class="uri" href="phenylalanine" title="wikilink">phenylalanine</a>, result in build-up of phenylalanine and related products. Some mutations are in the active site, directly disrupting binding and catalysis, but many are far from the active site and reduce activity by destabilising the protein structure, or affecting correct oligomerisation.<a class="footnoteRef" href="#fn110" id="fnref110"><sup>110</sup></a><a class="footnoteRef" href="#fn111" id="fnref111"><sup>111</sup></a> This can lead to <a href="intellectual_disability" title="wikilink">intellectual disability</a> if the disease is untreated.<a class="footnoteRef" href="#fn112" id="fnref112"><sup>112</sup></a> Another example is <a href="pseudocholinesterase_deficiency" title="wikilink">pseudocholinesterase deficiency</a>, in which the body's ability to break down choline ester drugs is impaired.<a class="footnoteRef" href="#fn113" id="fnref113"><sup>113</sup></a> Oral administration of enzymes can be used to treat some functional enzyme deficiencies, such as <a href="pancreatic_insufficiency" title="wikilink">pancreatic insufficiency</a><a class="footnoteRef" href="#fn114" id="fnref114"><sup>114</sup></a> and <a href="lactose_intolerance" title="wikilink">lactose intolerance</a>.<a class="footnoteRef" href="#fn115" id="fnref115"><sup>115</sup></a></p>

<p>Another way enzyme malfunctions can cause disease comes from <a href="germline_mutation" title="wikilink">germline mutations</a> in genes coding for <a href="DNA_repair" title="wikilink">DNA repair</a> enzymes. Defects in these enzymes cause cancer because cells are less able to repair mutations in their <a href="genome" title="wikilink">genomes</a>. This causes a slow accumulation of mutations and results in the <a href="carcinogenesis" title="wikilink">development of cancers</a>. An example of such a hereditary <a href="cancer_syndrome" title="wikilink">cancer syndrome</a> is <a href="xeroderma_pigmentosum" title="wikilink">xeroderma pigmentosum</a>, which causes the development of <a href="skin_cancer" title="wikilink">skin cancers</a> in response to even minimal exposure to <a href="ultraviolet_light" title="wikilink">ultraviolet light</a>.<a class="footnoteRef" href="#fn116" id="fnref116"><sup>116</sup></a><a class="footnoteRef" href="#fn117" id="fnref117"><sup>117</sup></a></p>
<h2 id="naming-conventions">Naming conventions</h2>

<p>An enzyme's name is often derived from its substrate or the chemical reaction it catalyzes, with the word ending in <em>-ase</em>.<a class="footnoteRef" href="#fn118" id="fnref118"><sup>118</sup></a> Examples are <a class="uri" href="lactase" title="wikilink">lactase</a>, <a href="alcohol_dehydrogenase" title="wikilink">alcohol dehydrogenase</a> and <a href="DNA_polymerase" title="wikilink">DNA polymerase</a>. Different enzymes that catalyze the same chemical reaction are called <a class="uri" href="isozymes" title="wikilink">isozymes</a>.<a class="footnoteRef" href="#fn119" id="fnref119"><sup>119</sup></a></p>

<p>The <a href="International_Union_of_Biochemistry_and_Molecular_Biology" title="wikilink">International Union of Biochemistry and Molecular Biology</a> have developed a <a class="uri" href="nomenclature" title="wikilink">nomenclature</a> for enzymes, the <a href="Enzyme_Commission_number" title="wikilink">EC numbers</a>; each enzyme is described by a sequence of four numbers preceded by "EC". The first number broadly classifies the enzyme based on its mechanism.<a class="footnoteRef" href="#fn120" id="fnref120"><sup>120</sup></a></p>

<p>The top-level classification is:</p>
<ul>
<li>EC 1, <a href="Oxidoreductase" title="wikilink">Oxidoreductases</a>: catalyze <a class="uri" href="oxidation" title="wikilink">oxidation</a>/reduction reactions</li>
<li>EC 2, <a href="Transferase" title="wikilink">Transferases</a>: transfer a <a href="functional_group" title="wikilink">functional group</a> (<em>e.g.</em> a methyl or phosphate group)</li>
<li>EC 3, <a href="Hydrolase" title="wikilink">Hydrolases</a>: catalyze the <a class="uri" href="hydrolysis" title="wikilink">hydrolysis</a> of various bonds</li>
<li>EC 4, <a href="Lyase" title="wikilink">Lyases</a>: cleave various bonds by means other than hydrolysis and oxidation</li>
<li>EC 5, <a href="Isomerase" title="wikilink">Isomerases</a>: catalyze <a href="isomer" title="wikilink">isomerization</a> changes within a single molecule</li>
<li>EC 6, <a href="Ligase" title="wikilink">Ligases</a>: join two molecules with <a href="covalent_bond" title="wikilink">covalent bonds</a>.</li>
</ul>

<p>These sections are subdivided by other features such as the substrate, products, and <a href="chemical_mechanism" title="wikilink">chemical mechanism</a>. An enzyme is fully specified by four numerical designations. For example, <a class="uri" href="hexokinase" title="wikilink">hexokinase</a> (EC 2.7.1.1) is a transferase (EC 2) that adds a phosphate group (EC 2.7) to a hexose sugar, a molecule containing an alcohol group (EC 2.7.1).<a class="footnoteRef" href="#fn121" id="fnref121"><sup>121</sup></a></p>
<h2 id="industrial-applications">Industrial applications</h2>

<p>Enzymes are used in the <a href="chemical_industry" title="wikilink">chemical industry</a> and other industrial applications when extremely specific catalysts are required. Enzymes in general are limited in the number of reactions they have evolved to catalyze and also by their lack of stability in <a href="organic_solvent" title="wikilink">organic solvents</a> and at high temperatures. As a consequence, <a href="protein_engineering" title="wikilink">protein engineering</a> is an active area of research and involves attempts to create new enzymes with novel properties, either through rational design or <em>in vitro</em> evolution.<a class="footnoteRef" href="#fn122" id="fnref122"><sup>122</sup></a><a class="footnoteRef" href="#fn123" id="fnref123"><sup>123</sup></a> These efforts have begun to be successful, and a few enzymes have now been designed "from scratch" to catalyze reactions that do not occur in nature.<a class="footnoteRef" href="#fn124" id="fnref124"><sup>124</sup></a></p>
<table>
<thead>
<tr class="header">
<th style="text-align: left;">
<p>Application</p></th>
<th style="text-align: left;">
<p>Enzymes used</p></th>
<th style="text-align: left;">
<p>Uses</p></th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;">
<p>style="border-top:solid 3px #aaa;" <strong><a href="Biofuel" title="wikilink">Biofuel industry</a></strong></p></td>
<td style="text-align: left;">
<p><a href="Cellulase" title="wikilink">Cellulases</a></p></td>
<td style="text-align: left;">
<p>Break down cellulose into sugars that can be fermented to produce <a href="cellulosic_ethanol" title="wikilink">cellulosic ethanol</a>.<a class="footnoteRef" href="#fn125" id="fnref125"><sup>125</sup></a></p></td>
</tr>
<tr class="even">
<td style="text-align: left;">
<p><a href="Ligninase" title="wikilink">Ligninases</a></p></td>
<td style="text-align: left;">
<p>Pretreatment of <a class="uri" href="biomass" title="wikilink">biomass</a> for biofuel production.<a class="footnoteRef" href="#fn126" id="fnref126"><sup>126</sup></a></p></td>
</tr>
<tr class="odd">
<td style="text-align: left;">
<p>style="border-top:solid 3px #aaa;" <strong><a href="Biological_detergent" title="wikilink">Biological detergent</a></strong></p></td>
<td style="text-align: left;">
<p><a href="Protease" title="wikilink">Proteases</a>, <a href="amylase" title="wikilink">amylases</a>, <a href="lipase" title="wikilink">lipases</a></p></td>
<td style="text-align: left;">
<p>Remove protein, starch, and fat or oil stains from laundry and dishware.<a class="footnoteRef" href="#fn127" id="fnref127"><sup>127</sup></a></p></td>
</tr>
<tr class="even">
<td style="text-align: left;">
<p><a href="Mannanase" title="wikilink">Mannanases</a></p></td>
<td style="text-align: left;">
<p>Remove food stains from the common food additive <a href="guar_gum" title="wikilink">guar gum</a>.<a class="footnoteRef" href="#fn128" id="fnref128"><sup>128</sup></a></p></td>
</tr>
<tr class="odd">
<td style="text-align: left;">
<p>style="border-top:solid 3px #aaa;" <strong><a href="Brewing" title="wikilink">Brewing industry</a></strong></p></td>
<td style="text-align: left;">
<p><a class="uri" href="Amylase" title="wikilink">Amylase</a>, <a href="glucanase" title="wikilink">glucanases</a>, <a href="protease" title="wikilink">proteases</a></p></td>
<td style="text-align: left;">
<p>Split polysaccharides and proteins in the <a class="uri" href="malt" title="wikilink">malt</a>.<a class="footnoteRef" href="#fn129" id="fnref129"><sup>129</sup></a></p></td>
</tr>
<tr class="even">
<td style="text-align: left;">
<p><a href="Betaglucanase" title="wikilink">Betaglucanases</a></p></td>
<td style="text-align: left;">
<p>Improve the <a class="uri" href="wort" title="wikilink">wort</a> and beer filtration characteristics.<a class="footnoteRef" href="#fn130" id="fnref130"><sup>130</sup></a></p></td>
</tr>
<tr class="odd">
<td style="text-align: left;">
<p><a class="uri" href="Amyloglucosidase" title="wikilink">Amyloglucosidase</a> and <a href="pullulanase" title="wikilink">pullulanases</a></p></td>
<td style="text-align: left;">
<p>Make low-calorie <a class="uri" href="beer" title="wikilink">beer</a> and adjust fermentability.<a class="footnoteRef" href="#fn131" id="fnref131"><sup>131</sup></a></p></td>
</tr>
<tr class="even">
<td style="text-align: left;">
<p><a href="Acetolactate_decarboxylase" title="wikilink">Acetolactate decarboxylase</a> (ALDC)</p></td>
<td style="text-align: left;">
<p>Increase fermentation efficiency by reducing <a class="uri" href="diacetyl" title="wikilink">diacetyl</a> formation.<a class="footnoteRef" href="#fn132" id="fnref132"><sup>132</sup></a></p></td>
</tr>
<tr class="odd">
<td style="text-align: left;">
<p><strong><a href="Cooking" title="wikilink">Culinary uses</a></strong></p></td>
<td style="text-align: left;">
<p><a class="uri" href="Papain" title="wikilink">Papain</a></p></td>
<td style="text-align: left;">
<p><a href="Tenderizer" title="wikilink">Tenderize</a> meat for cooking.<a class="footnoteRef" href="#fn133" id="fnref133"><sup>133</sup></a></p></td>
</tr>
<tr class="even">
<td style="text-align: left;">
<p>style="border-top:solid 3px #aaa;" <strong><a href="Dairy" title="wikilink">Dairy industry</a></strong></p></td>
<td style="text-align: left;">
<p><a href="Chymosin" title="wikilink">Rennin</a></p></td>
<td style="text-align: left;">
<p><a class="uri" href="Hydrolyze" title="wikilink">Hydrolyze</a> protein in the manufacture of <a class="uri" href="cheese" title="wikilink">cheese</a>.<a class="footnoteRef" href="#fn134" id="fnref134"><sup>134</sup></a></p></td>
</tr>
<tr class="odd">
<td style="text-align: left;">
<p><a href="Lipase" title="wikilink">Lipases</a></p></td>
<td style="text-align: left;">
<p>Produce <a href="Camembert_cheese" title="wikilink">Camembert cheese</a> and <a href="blue_cheese" title="wikilink">blue cheeses</a> such as <a class="uri" href="Roquefort" title="wikilink">Roquefort</a>.<a class="footnoteRef" href="#fn135" id="fnref135"><sup>135</sup></a></p></td>
</tr>
<tr class="even">
<td style="text-align: left;">
<p>style="border-top:solid 3px #aaa;" <strong><a href="Food_processing" title="wikilink">Food processing</a></strong></p></td>
<td style="text-align: left;">
<p><a href="Amylase" title="wikilink">Amylases</a></p></td>
<td style="text-align: left;">
<p>Produce sugars from <a class="uri" href="starch" title="wikilink">starch</a>, such as in making <a href="high-fructose_corn_syrup" title="wikilink">high-fructose corn syrup</a>.<a class="footnoteRef" href="#fn136" id="fnref136"><sup>136</sup></a></p></td>
</tr>
<tr class="odd">
<td style="text-align: left;">
<p><a href="Protease" title="wikilink">Proteases</a></p></td>
<td style="text-align: left;">
<p>Lower the protein level of <a class="uri" href="flour" title="wikilink">flour</a>, as in <a class="uri" href="biscuit" title="wikilink">biscuit</a>-making.<a class="footnoteRef" href="#fn137" id="fnref137"><sup>137</sup></a></p></td>
</tr>
<tr class="even">
<td style="text-align: left;">
<p>|<a class="uri" href="Trypsin" title="wikilink">Trypsin</a></p></td>
<td style="text-align: left;">
<p>Manufacture <a class="uri" href="hypoallergenic" title="wikilink">hypoallergenic</a> baby foods.<a class="footnoteRef" href="#fn138" id="fnref138"><sup>138</sup></a></p></td>
</tr>
<tr class="odd">
<td style="text-align: left;">
<p><a href="Cellulase" title="wikilink">Cellulases</a>, <a href="pectinase" title="wikilink">pectinases</a></p></td>
<td style="text-align: left;">
<p>Clarify <a href="fruit_juice" title="wikilink">fruit juices</a>.<a class="footnoteRef" href="#fn139" id="fnref139"><sup>139</sup></a></p></td>
</tr>
<tr class="even">
<td style="text-align: left;">
<p><strong><a href="Molecular_biology" title="wikilink">Molecular biology</a></strong></p></td>
<td style="text-align: left;">
<p><a href="Nuclease" title="wikilink">Nucleases</a>, <a href="DNA_ligase" title="wikilink">DNA ligase</a> and <a class="uri" href="polymerases" title="wikilink">polymerases</a></p></td>
<td style="text-align: left;">
<p>Use <a href="restriction_enzyme" title="wikilink">restriction digestion</a> and the <a href="polymerase_chain_reaction" title="wikilink">polymerase chain reaction</a> to create <a href="recombinant_DNA" title="wikilink">recombinant DNA</a>.<a class="footnoteRef" href="#fn140" id="fnref140"><sup>140</sup></a></p></td>
</tr>
<tr class="odd">
<td style="text-align: left;">
<p><strong><a href="Paper" title="wikilink">Paper industry</a></strong></p></td>
<td style="text-align: left;">
<p><a href="Xylanase" title="wikilink">Xylanases</a>, <a href="hemicellulase" title="wikilink">hemicellulases</a> and <a href="lignin_peroxidase" title="wikilink">lignin peroxidases</a></p></td>
<td style="text-align: left;">
<p>Remove <a class="uri" href="lignin" title="wikilink">lignin</a> from <a href="kraft_pulp" title="wikilink">kraft pulp</a>.<a class="footnoteRef" href="#fn141" id="fnref141"><sup>141</sup></a></p></td>
</tr>
<tr class="even">
<td style="text-align: left;">
<p><strong><a href="Personal_care" title="wikilink">Personal care</a></strong></p></td>
<td style="text-align: left;">
<p><a class="uri" href="Proteases" title="wikilink">Proteases</a></p></td>
<td style="text-align: left;">
<p>Remove proteins on <a href="contact_lens" title="wikilink">contact lenses</a> to prevent infections.<a class="footnoteRef" href="#fn142" id="fnref142"><sup>142</sup></a></p></td>
</tr>
<tr class="odd">
<td style="text-align: left;">
<p>style="border-top:solid 3px #aaa;" <strong><a href="Starch" title="wikilink">Starch industry</a></strong></p></td>
<td style="text-align: left;">
<p><a href="Amylase" title="wikilink">Amylases</a></p></td>
<td style="text-align: left;">
<p>Convert <a class="uri" href="starch" title="wikilink">starch</a> into <a class="uri" href="glucose" title="wikilink">glucose</a> and various <a href="Inverted_sugar_syrup" title="wikilink">syrups</a>.<a class="footnoteRef" href="#fn143" id="fnref143"><sup>143</sup></a></p></td>
</tr>
</tbody>
</table>
<h2 id="see-also">See also</h2>
<ul>
<li><a href="List_of_enzymes" title="wikilink">List of enzymes</a></li>
<li>Enzyme databases
<ul>
<li><a class="uri" href="BRENDA" title="wikilink">BRENDA</a></li>
<li><a class="uri" href="ExPASy" title="wikilink">ExPASy</a></li>
<li><a class="uri" href="IntEnz" title="wikilink">IntEnz</a></li>
<li><a class="uri" href="KEGG" title="wikilink">KEGG</a></li>
<li><a class="uri" href="MetaCyc" title="wikilink">MetaCyc</a></li>
</ul></li>
</ul>
<h2 id="references">References</h2>
<h2 id="further-reading">Further reading</h2>

<p><strong>General</strong></p>
<ul>
<li>

<p>, A biochemistry textbook available free online through NCBI Bookshelf.</p></li>
</ul>

<p><strong>Etymology and history</strong></p>
<ul>
<li>

<p>, A history of early enzymology.</p></li>
</ul>

<p><strong>Enzyme structure and mechanism</strong></p>
<ul>
<li></li>
</ul>

<p><strong>Kinetics and inhibition</strong></p>
<ul>
<li></li>
</ul>

<p>"</p>

<p><a href="Category:Enzymes" title="wikilink">*</a> <a class="uri" href="Category:Biomolecules" title="wikilink">Category:Biomolecules</a> <a class="uri" href="Category:Catalysis" title="wikilink">Category:Catalysis</a> <a class="uri" href="Category:Metabolism" title="wikilink">Category:Metabolism</a> <a href="Category:Process_chemicals" title="wikilink">Category:Process chemicals</a></p>
<section class="footnotes">
<hr/>
<ol>
<li id="fn1"><a href="#fnref1">↩</a></li>
<li id="fn2"><a href="#fnref2">↩</a></li>
<li id="fn3"><a href="#fnref3">↩</a></li>
<li id="fn4"><a href="#fnref4">↩</a></li>
<li id="fn5"><a href="#fnref5">↩</a></li>
<li id="fn6"><a href="#fnref6">↩</a></li>
<li id="fn7"><a href="#fnref7">↩</a></li>
<li id="fn8"><a href="#fnref8">↩</a></li>
<li id="fn9">Kühne coined the word "enzyme" in:  The relevant passage occurs on page 190: ''"Um Missverständnissen vorzubeugen und lästige Umschreibungen zu vermeiden schlägt Vortragender vor, die ungeformten oder nicht organisirten Fermente, deren Wirkung ohne Anwesenheit von Organismen und ausserhalb derselben erfolgen kann, als <em>Enzyme</em> zu bezeichnen."'' (Translation: In order to obviate misunderstandings and avoid cumbersome periphrases, [the author, a university lecturer] suggests designating as "enzymes" the unformed or not organized ferments, whose action can occur without the presence of organisms and outside of the same.)<a href="#fnref9">↩</a></li>
<li id="fn10"><a href="#fnref10">↩</a></li>
<li id="fn11"><a href="#fnref11">↩</a></li>
<li id="fn12"><a href="#fnref12">↩</a></li>
<li id="fn13">The naming of enzymes by adding the suffix "-ase" to the substrate on which the enzyme acts, has been traced to French scientist <a href="Émile_Duclaux" title="wikilink">Émile Duclaux</a> (1840–1904), who intended to honor the discoverers of <a class="uri" href="diastase" title="wikilink">diastase</a> – the first enzyme to be isolated – by introducing this practice in his book  See Chapter 1, especially page 9.<a href="#fnref13">↩</a></li>
<li id="fn14"> quoted in <a href="#fnref14">↩</a></li>
<li id="fn15"><a href="#fnref15">↩</a></li>
<li id="fn16"><a href="#fnref16">↩</a></li>
<li id="fn17"><a href="#fnref17">↩</a></li>
<li id="fn18"><a href="#fnref18">↩</a></li>
<li id="fn19"><a href="#fnref19">↩</a></li>
<li id="fn20"><a href="#fnref20">↩</a></li>
<li id="fn21"><a href="#fnref21">↩</a></li>
<li id="fn22"><a href="#fnref22">↩</a></li>
<li id="fn23"><a href="#fnref23">↩</a></li>
<li id="fn24"><a href="#fnref24">↩</a></li>
<li id="fn25"></li>
<li id="fn26"><a href="#fnref26">↩</a></li>
<li id="fn27"></li>
<li id="fn28"><a href="#fnref28">↩</a></li>
<li id="fn29"><a href="#fnref29">↩</a></li>
<li id="fn30"></li>
<li id="fn31"><a href="#fnref31">↩</a></li>
<li id="fn32"><a href="#fnref32">↩</a></li>
<li id="fn33"><a href="#fnref33">↩</a></li>
<li id="fn34"><a href="#fnref34">↩</a></li>
<li id="fn35"><a href="#fnref35">↩</a></li>
<li id="fn36"> From page 2992: <em>"Um ein Bild zu gebrauchen, will ich sagen, dass Enzym und Glucosid wie Schloss und Schlüssel zu einander passen müssen, um eine chemische Wirkung auf einander ausüben zu können."</em> (To use an image, I will say that an enzyme and a glucoside [i.e., glucose derivative] must fit like a lock and key, in order to be able to exert a chemical effect on each other.)<a href="#fnref36">↩</a></li>
<li id="fn37"></li>
<li id="fn38"><a href="#fnref38">↩</a></li>
<li id="fn39"><a href="#fnref39">↩</a></li>
<li id="fn40"><a href="#fnref40">↩</a></li>
<li id="fn41"><a href="#fnref41">↩</a></li>
<li id="fn42"><a href="#fnref42">↩</a></li>
<li id="fn43"><a href="#fnref43">↩</a></li>
<li id="fn44"><a href="#fnref44">↩</a></li>
<li id="fn45"><a href="#fnref45">↩</a></li>
<li id="fn46"><a href="#fnref46">↩</a></li>
<li id="fn47"><a href="#fnref47">↩</a></li>
<li id="fn48"><a href="#fnref48">↩</a></li>
<li id="fn49"><a href="#fnref49">↩</a></li>
<li id="fn50"><a href="#fnref50">↩</a></li>
<li id="fn51"><a href="#fnref51">↩</a></li>
<li id="fn52"><a href="#fnref52">↩</a></li>
<li id="fn53"><a href="#fnref53">↩</a></li>
<li id="fn54"><a href="#fnref54">↩</a></li>
<li id="fn55"></li>
<li id="fn56"></li>
<li id="fn57"></li>
<li id="fn58"><a href="#fnref58">↩</a></li>
<li id="fn59"></li>
<li id="fn60"><a href="#fnref60">↩</a></li>
<li id="fn61"><a href="#fnref61">↩</a></li>
<li id="fn62"></li>
<li id="fn63"><a href="#fnref63">↩</a></li>
<li id="fn64"></li>
<li id="fn65"><a href="#fnref65">↩</a></li>
<li id="fn66">; <a href="#fnref66">↩</a></li>
<li id="fn67"><a href="#fnref67">↩</a></li>
<li id="fn68"></li>
<li id="fn69"></li>
<li id="fn70"></li>
<li id="fn71"></li>
<li id="fn72"><a href="#fnref72">↩</a></li>
<li id="fn73"><a href="#fnref73">↩</a></li>
<li id="fn74"><a href="#fnref74">↩</a></li>
<li id="fn75"><a href="#fnref75">↩</a></li>
<li id="fn76"></li>
<li id="fn77"></li>
<li id="fn78"><a href="#fnref78">↩</a></li>
<li id="fn79"></li>
<li id="fn80"><a href="#fnref80">↩</a></li>
<li id="fn81"><a href="#fnref81">↩</a></li>
<li id="fn82"></li>
<li id="fn83"><a href="#fnref83">↩</a></li>
<li id="fn84"><a href="#fnref84">↩</a></li>
<li id="fn85"></li>
<li id="fn86"><a href="#fnref86">↩</a></li>
<li id="fn87"><a href="#fnref87">↩</a></li>
<li id="fn88"><a href="#fnref88">↩</a></li>
<li id="fn89"><a href="#fnref89">↩</a></li>
<li id="fn90"><a href="#fnref90">↩</a></li>
<li id="fn91"><a href="#fnref91">↩</a></li>
<li id="fn92"></li>
<li id="fn93"><a href="#fnref93">↩</a></li>
<li id="fn94"></li>
<li id="fn95"></li>
<li id="fn96"></li>
<li id="fn97"></li>
<li id="fn98"><a href="#fnref98">↩</a></li>
<li id="fn99"><a href="#fnref99">↩</a></li>
<li id="fn100"></li>
<li id="fn101"><a href="#fnref101">↩</a></li>
<li id="fn102"><a href="#fnref102">↩</a></li>
<li id="fn103"></li>
<li id="fn104"><a href="#fnref104">↩</a></li>
<li id="fn105"><a href="#fnref105">↩</a></li>
<li id="fn106"><a href="#fnref106">↩</a></li>
<li id="fn107"><a href="#fnref107">↩</a></li>
<li id="fn108"><a href="#fnref108">↩</a></li>
<li id="fn109"><a href="#fnref109">↩</a></li>
<li id="fn110"><a href="#fnref110">↩</a></li>
<li id="fn111"><a href="#fnref111">↩</a></li>
<li id="fn112"><a href="#fnref112">↩</a></li>
<li id="fn113"><a href="#fnref113">↩</a></li>
<li id="fn114"><a href="#fnref114">↩</a></li>
<li id="fn115"><a href="#fnref115">↩</a></li>
<li id="fn116"><a href="#fnref116">↩</a></li>
<li id="fn117"><a href="#fnref117">↩</a></li>
<li id="fn118"></li>
<li id="fn119"></li>
<li id="fn120"><a href="#fnref120">↩</a></li>
<li id="fn121"><a href="#fnref121">↩</a></li>
<li id="fn122"><a href="#fnref122">↩</a></li>
<li id="fn123"><a href="#fnref123">↩</a></li>
<li id="fn124"><a href="#fnref124">↩</a></li>
<li id="fn125"><a href="#fnref125">↩</a></li>
<li id="fn126"></li>
<li id="fn127"><a href="#fnref127">↩</a></li>
<li id="fn128"></li>
<li id="fn129"><a href="#fnref129">↩</a></li>
<li id="fn130"></li>
<li id="fn131"></li>
<li id="fn132"><a href="#fnref132">↩</a></li>
<li id="fn133"><a href="#fnref133">↩</a></li>
<li id="fn134"><a href="#fnref134">↩</a></li>
<li id="fn135"><a href="#fnref135">↩</a></li>
<li id="fn136"><a href="#fnref136">↩</a></li>
<li id="fn137"></li>
<li id="fn138"><a href="#fnref138">↩</a></li>
<li id="fn139"><a href="#fnref139">↩</a></li>
<li id="fn140"></li>
<li id="fn141"><a href="#fnref141">↩</a></li>
<li id="fn142"><a href="#fnref142">↩</a></li>
<li id="fn143"><a href="#fnref143">↩</a></li>
</ol>
</section>
</body>
</html>
