<html lang="en">
<head>
<meta charset="utf-8"/>
<title offset="1724">Enzyme kinetics</title>
<script src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_SVG.js" type="text/javascript">
</script>
</head>
<body>
<h1>Enzyme kinetics</h1>
<hr/>
<figure><b>(Figure)</b>
<figcaption><a href="Dihydrofolate_reductase" title="wikilink">Dihydrofolate reductase</a> from <em><a href="Escherichia_coli" title="wikilink">E. coli</a></em> with its two substrates <a href="Dihydrofolic_acid" title="wikilink">dihydrofolate</a> (right) and <a href="Nicotinamide_adenine_dinucleotide_phosphate" title="wikilink">NADPH</a> (left), bound in the active site. The protein is shown as a <a href="ribbon_diagram" title="wikilink">ribbon diagram</a>, with alpha helices in red, beta sheets in yellow and loops in blue. Generated from <a href="http://www.rcsb.org/pdb/explore.do?structureId=7DFR">7DFR</a>.</figcaption>
</figure>

<p><strong>Enzyme kinetics</strong> is the study of the <a href="chemical_reaction" title="wikilink">chemical reactions</a> that are <a href="catalyst" title="wikilink">catalysed</a> by <a class="uri" href="enzymes" title="wikilink">enzymes</a>. In enzyme kinetics, the <a href="reaction_rate" title="wikilink">reaction rate</a> is measured and the effects of varying the conditions of the reaction are investigated. Studying an enzyme's <a href="chemical_kinetics" title="wikilink">kinetics</a> in this way can reveal the catalytic mechanism of this enzyme, its role in <a class="uri" href="metabolism" title="wikilink">metabolism</a>, how its activity is controlled, and how a <a class="uri" href="drug" title="wikilink">drug</a> or an agonist might <a href="enzyme_inhibitor" title="wikilink">inhibit</a> the enzyme.</p>

<p>Enzymes are usually <a class="uri" href="protein" title="wikilink">protein</a> <a href="molecule" title="wikilink">molecules</a> that manipulate other molecules — the enzymes' <a href="Substrate_(biochemistry)" title="wikilink">substrates</a>. These target molecules bind to an enzyme's <a href="active_site" title="wikilink">active site</a> and are transformed into <a href="product_(biology)" title="wikilink">products</a> through a series of steps known as the <a href="enzyme_catalysis" title="wikilink">enzymatic mechanism</a></p>

<p>E + S  ES  ES* EP  E + P</p>

<p>. These mechanisms can be divided into single-substrate and multiple-substrate mechanisms. Kinetic studies on enzymes that only bind one substrate, such as <a href="Triosephosphateisomerase" title="wikilink">triosephosphate isomerase</a>, aim to measure the <a href="dissociation_constant" title="wikilink">affinity</a> with which the enzyme binds this substrate and the turnover rate. Some other examples of enzymes are phosphofructokinase and hexokinase, both of which are important for cellular respiration (glycolysis).</p>

<p>When enzymes bind multiple substrates, such as <a href="dihydrofolate_reductase" title="wikilink">dihydrofolate reductase</a> (shown right), enzyme kinetics can also show the sequence in which these substrates bind and the sequence in which products are released. An example of enzymes that bind a single substrate and release multiple products are <a href="protease" title="wikilink">proteases</a>, which cleave one protein substrate into two polypeptide products. Others join two substrates together, such as <a href="DNA_polymerase" title="wikilink">DNA polymerase</a> linking a <a class="uri" href="nucleotide" title="wikilink">nucleotide</a> to <a class="uri" href="DNA" title="wikilink">DNA</a>. Although these mechanisms are often a complex series of steps, there is typically one <em>rate-determining step</em> that determines the overall kinetics. This <a href="rate-determining_step" title="wikilink">rate-determining step</a> may be a chemical reaction or a <a href="conformational_isomerism" title="wikilink">conformational</a> change of the enzyme or substrates, such as those involved in the release of product(s) from the enzyme.</p>

<p>Knowledge of the <a href="protein_structure" title="wikilink">enzyme's structure</a> is helpful in interpreting kinetic data. For example, the structure can suggest how substrates and products bind during catalysis; what changes occur during the reaction; and even the role of particular <a href="amino_acid" title="wikilink">amino acid</a> residues in the mechanism. Some enzymes change shape significantly during the mechanism; in such cases, it is helpful to determine the enzyme structure with and without bound substrate analogues that do not undergo the enzymatic reaction.</p>

<p>Not all biological catalysts are protein enzymes; <a class="uri" href="RNA" title="wikilink">RNA</a>-based catalysts such as <a class="uri" href="ribozymes" title="wikilink">ribozymes</a> and <a class="uri" href="ribosomes" title="wikilink">ribosomes</a> are essential to many cellular functions, such as <a href="splicing_(genetics)" title="wikilink">RNA splicing</a> and <a href="translation_(biology)" title="wikilink">translation</a>. The main difference between ribozymes and enzymes is that RNA catalysts are composed of nucleotides, whereas enzymes are composed of amino acids. Ribozymes also perform a more limited set of reactions, although their <a href="reaction_mechanism" title="wikilink">reaction mechanisms</a> and kinetics can be analysed and classified by the same methods.</p>
<h2 id="general-principles">General principles</h2>
<figure><b>(Figure)</b>
<figcaption>As larger amounts of <a href="Substrate_(biochemistry)" title="wikilink">substrate</a> are added to a reaction, the available enzyme <a href="binding_site" title="wikilink">binding sites</a> become filled to the limit of <a href="Michaelis-Menten_kinetics" title="wikilink">

<math display="inline" id="Enzyme_kinetics:0">
 <semantics>
  <msub>
   <mi>V</mi>
   <mi>max</mi>
  </msub>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <csymbol cd="ambiguous">subscript</csymbol>
    <ci>V</ci>
    <max></max>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   V_{\max}
  </annotation>
 </semantics>
</math>

</a>. Beyond this limit the enzyme is saturated with substrate and the reaction rate ceases to increase.</figcaption>
</figure>

<p>The reaction catalysed by an enzyme uses exactly the same reactants and produces exactly the same products as the uncatalysed reaction. Like other <a class="uri" href="catalysts" title="wikilink">catalysts</a>, enzymes do not alter the position of <a href="chemical_equilibrium" title="wikilink">equilibrium</a> between substrates and products.<a class="footnoteRef" href="#fn1" id="fnref1"><sup>1</sup></a> However, unlike uncatalysed chemical reactions, enzyme-catalysed reactions display saturation kinetics. For a given enzyme concentration and for relatively low substrate concentrations, the reaction rate increases linearly with substrate concentration; the enzyme molecules are largely free to catalyse the reaction, and increasing substrate concentration means an increasing rate at which the enzyme and substrate molecules encounter one another. However, at relatively high substrate concentrations, the reaction rate <a href="asymptote" title="wikilink">asymptotically</a> approaches the theoretical maximum; the enzyme active sites are almost all occupied and the reaction rate is determined by the intrinsic turnover rate of the enzyme. The substrate concentration midway between these two limiting cases is denoted by <em>K</em><sub>M</sub>.</p>

<p>The two most important kinetic properties of an enzyme are how quickly the enzyme becomes saturated with a particular substrate, and the maximum rate it can achieve. Knowing these properties suggests what an enzyme might do in the cell and can show how the enzyme will respond to changes in these conditions.</p>
<h2 id="enzyme-assays">Enzyme assays</h2>

<p> <a href="Enzyme_assay" title="wikilink">Enzyme assays</a> are laboratory procedures that measure the rate of enzyme reactions. Because enzymes are not consumed by the reactions they catalyse, enzyme assays usually follow changes in the concentration of either substrates or products to measure the rate of reaction. There are many methods of measurement. <a href="Ultraviolet-visible_spectroscopy" title="wikilink">Spectrophotometric</a> assays observe change in the <a class="uri" href="absorbance" title="wikilink">absorbance</a> of light between products and reactants; radiometric assays involve the incorporation or release of <a class="uri" href="radioactivity" title="wikilink">radioactivity</a> to measure the amount of product made over time. Spectrophotometric assays are most convenient since they allow the rate of the reaction to be measured continuously. Although radiometric assays require the removal and counting of samples (i.e., they are discontinuous assays) they are usually extremely sensitive and can measure very low levels of enzyme activity.<a class="footnoteRef" href="#fn2" id="fnref2"><sup>2</sup></a> An analogous approach is to use <a href="mass_spectrometry" title="wikilink">mass spectrometry</a> to monitor the incorporation or release of <a href="stable_isotope" title="wikilink">stable isotopes</a> as substrate is converted into product.</p>

<p>The most sensitive enzyme assays use <a href="laser" title="wikilink">lasers</a> focused through a <a class="uri" href="microscope" title="wikilink">microscope</a> to observe changes in single enzyme molecules as they catalyse their reactions. These measurements either use changes in the <a class="uri" href="fluorescence" title="wikilink">fluorescence</a> of <a href="Cofactor_(biochemistry)" title="wikilink">cofactors</a> during an enzyme's reaction mechanism, or of <a href="fluorescent_dyes" title="wikilink">fluorescent dyes</a> added onto specific sites of the <a class="uri" href="protein" title="wikilink">protein</a> to report movements that occur during catalysis.<a class="footnoteRef" href="#fn3" id="fnref3"><sup>3</sup></a> These studies are providing a new view of the kinetics and dynamics of single enzymes, as opposed to traditional enzyme kinetics, which observes the average behaviour of populations of millions of enzyme molecules.<a class="footnoteRef" href="#fn4" id="fnref4"><sup>4</sup></a><a class="footnoteRef" href="#fn5" id="fnref5"><sup>5</sup></a></p>

<p>An example progress curve for an enzyme assay is shown above. The enzyme produces product at an initial rate that is approximately linear for a short period after the start of the reaction. As the reaction proceeds and substrate is consumed, the rate continuously slows (so long as substrate is not still at saturating levels). To measure the initial (and maximal) rate, enzyme assays are typically carried out while the reaction has progressed only a few percent towards total completion. The length of the initial rate period depends on the assay conditions and can range from milliseconds to hours. However, equipment for rapidly mixing liquids allows fast kinetic measurements on initial rates of less than one second.<a class="footnoteRef" href="#fn6" id="fnref6"><sup>6</sup></a> These very rapid assays are essential for measuring pre-steady-state kinetics, which are discussed below.</p>

<p>Most enzyme kinetics studies concentrate on this initial, approximately linear part of enzyme reactions. However, it is also possible to measure the complete reaction curve and fit this data to a non-linear <a href="rate_equation" title="wikilink">rate equation</a>. This way of measuring enzyme reactions is called progress-curve analysis.<a class="footnoteRef" href="#fn7" id="fnref7"><sup>7</sup></a> This approach is useful as an alternative to <a href="rapid_kinetics" title="wikilink">rapid kinetics</a> when the initial rate is too fast to measure accurately.</p>
<h2 id="single-substrate-reactions">Single-substrate reactions</h2>

<p>Enzymes with single-substrate mechanisms include <a href="isomerase" title="wikilink">isomerases</a> such as <a class="uri" href="triosephosphateisomerase" title="wikilink">triosephosphateisomerase</a> or <a href="bisphosphoglycerate_mutase" title="wikilink">bisphosphoglycerate mutase</a>, intramolecular <a href="lyase" title="wikilink">lyases</a> such as <a href="adenylate_cyclase" title="wikilink">adenylate cyclase</a> and the <a href="ribozyme" title="wikilink">hammerhead ribozyme</a>, an RNA lyase.<a class="footnoteRef" href="#fn8" id="fnref8"><sup>8</sup></a> However, some enzymes that only have a single substrate do not fall into this category of mechanisms. <a class="uri" href="Catalase" title="wikilink">Catalase</a> is an example of this, as the enzyme reacts with a first molecule of <a href="hydrogen_peroxide" title="wikilink">hydrogen peroxide</a> substrate, becomes oxidised and is then reduced by a second molecule of substrate. Although a single substrate is involved, the existence of a modified enzyme intermediate means that the mechanism of catalase is actually a ping–pong mechanism, a type of mechanism that is discussed in the <em>Multi-substrate reactions</em> section below.</p>
<h3 id="michaelismenten-kinetics">Michaelis–Menten kinetics</h3>

<p></p>

<p>As enzyme-catalysed reactions are saturable, their rate of catalysis does not show a linear response to increasing substrate. If the initial rate of the reaction is measured over a range of substrate concentrations (denoted as [S]), the reaction rate (<em>v</em>) increases as [S] increases, as shown on the right. However, as [S] gets higher, the enzyme becomes saturated with substrate and the rate reaches <em>V</em><sub>max</sub>, the enzyme's maximum rate.</p>

<p>The <a href="Michaelis–Menten_kinetics" title="wikilink">Michaelis–Menten kinetic model of a single-substrate reaction</a> is shown on the right. There is an initial <a href="chemical_kinetics" title="wikilink">bimolecular reaction</a> between the enzyme E and substrate S to form the enzyme–substrate complex ES but the rate of enzymatic reaction increase with the increase of the substrate concentration up to a certain level but then more increase in substrate concentration does not cause any increase in reaction rate as there no more E remain available for reacting with S and the rate of reaction become dependent on ES and the reaction become <a href="chemical_kinetics" title="wikilink">unimolecular reaction</a>. Although the enzymatic mechanism for the <a href="chemical_kinetics" title="wikilink">unimolecular reaction</a> 

<math display="inline" id="Enzyme_kinetics:1">
 <semantics>
  <mrow>
   <mrow>
    <mi>E</mi>
    <mi>S</mi>
    <mover accent="true">
     <mo>⟶</mo>
     <msub>
      <mi>k</mi>
      <mrow>
       <mi>c</mi>
       <mi>a</mi>
       <mi>t</mi>
      </mrow>
     </msub>
    </mover>
    <mi>E</mi>
   </mrow>
   <mo>+</mo>
   <mi>P</mi>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <plus></plus>
    <apply>
     <times></times>
     <ci>E</ci>
     <ci>S</ci>
     <apply>
      <apply>
       <csymbol cd="ambiguous">subscript</csymbol>
       <ci>k</ci>
       <apply>
        <times></times>
        <ci>c</ci>
        <ci>a</ci>
        <ci>t</ci>
       </apply>
      </apply>
      <ci>normal-⟶</ci>
     </apply>
     <ci>E</ci>
    </apply>
    <ci>P</ci>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   ES\overset{k_{cat}}{\longrightarrow}E+P
  </annotation>
 </semantics>
</math>

 can be quite complex, there is typically one rate-determining enzymatic step that allows this reaction to be modelled as a single catalytic step with an apparent unimolecular rate constant <em>k</em><sub>cat</sub>. If the reaction path proceeds over one or several intermediates, <em>k</em><sub>cat</sub> will be a function of several elementary rate constants, whereas in the simplest case of a single elementary reaction (e.g. no intermediates) it will be identical to the elementary unimolecular rate constant <em>k</em><sub>2</sub>. The apparent unimolecular rate constant <em>k</em><sub>cat</sub> is also called <em>turnover number</em> and denotes the maximum number of enzymatic reactions catalysed per second.</p>

<p>The <a href="Michaelis–Menten_kinetics" title="wikilink">Michaelis–Menten equation</a><a class="footnoteRef" href="#fn9" id="fnref9"><sup>9</sup></a> describes how the (initial) reaction rate <em>v</em><sub>0</sub> depends on the position of the substrate-binding <a href="chemical_equilibrium" title="wikilink">equilibrium</a> and the rate constant <em>k</em><sub>2</sub>.</p>

<p>

<math display="block" id="Enzyme_kinetics:2">
 <semantics>
  <mrow>
   <msub>
    <mi>v</mi>
    <mn>0</mn>
   </msub>
   <mo>=</mo>
   <mfrac>
    <mrow>
     <msub>
      <mi>V</mi>
      <mi>max</mi>
     </msub>
     <mrow>
      <mo stretchy="false">[</mo>
      <mtext>S</mtext>
      <mo stretchy="false">]</mo>
     </mrow>
    </mrow>
    <mrow>
     <msub>
      <mi>K</mi>
      <mi>M</mi>
     </msub>
     <mo>+</mo>
     <mrow>
      <mo stretchy="false">[</mo>
      <mtext>S</mtext>
      <mo stretchy="false">]</mo>
     </mrow>
    </mrow>
   </mfrac>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <eq></eq>
    <apply>
     <csymbol cd="ambiguous">subscript</csymbol>
     <ci>v</ci>
     <cn type="integer">0</cn>
    </apply>
    <apply>
     <divide></divide>
     <apply>
      <times></times>
      <apply>
       <csymbol cd="ambiguous">subscript</csymbol>
       <ci>V</ci>
       <max></max>
      </apply>
      <apply>
       <csymbol cd="latexml">delimited-[]</csymbol>
       <mtext>S</mtext>
      </apply>
     </apply>
     <apply>
      <plus></plus>
      <apply>
       <csymbol cd="ambiguous">subscript</csymbol>
       <ci>K</ci>
       <ci>M</ci>
      </apply>
      <apply>
       <csymbol cd="latexml">delimited-[]</csymbol>
       <mtext>S</mtext>
      </apply>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   v_{0}=\frac{V_{\max}[\mbox{S}]}{K_{M}+[\mbox{S}]}
  </annotation>
 </semantics>
</math>

    <em>(Michaelis–Menten equation)</em> with the constants</p>

<p>

<math display="inline" id="Enzyme_kinetics:3">
 <semantics>
  <msub>
   <mi>K</mi>
   <mi>M</mi>
  </msub>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <csymbol cd="ambiguous">subscript</csymbol>
    <ci>K</ci>
    <ci>M</ci>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \displaystyle K_{M}
  </annotation>
 </semantics>
</math>


</p>

<p>This Michaelis–Menten equation is the basis for most single-substrate enzyme kinetics. Two crucial assumptions underlie this equation (apart from the general assumption about the mechanism only involving no intermediate or product inhibition, and there is no <a href="Allosteric_regulation" title="wikilink">allostericity</a> or <a href="Cooperative_binding" title="wikilink">cooperativity</a>). The first assumption is the so-called <a href="steady_state_(chemistry)" title="wikilink">quasi-steady-state assumption</a> (or pseudo-steady-state hypothesis), namely that the concentration of the substrate-bound enzyme (and hence also the unbound enzyme) changes much more slowly than those of the product and substrate and thus the change over time of the complex can be set to zero 

<math display="inline" id="Enzyme_kinetics:4">
 <semantics>
  <mrow>
   <mrow>
    <mrow>
     <mi>d</mi>
     <mrow>
      <mo stretchy="false">[</mo>
      <mrow>
       <mi>E</mi>
       <mi>S</mi>
      </mrow>
      <mo stretchy="false">]</mo>
     </mrow>
    </mrow>
    <mo>/</mo>
    <mi>d</mi>
   </mrow>
   <mpadded width="+2.8pt">
    <mi>t</mi>
   </mpadded>
   <mover accent="true">
    <mo>=</mo>
    <mo>!</mo>
   </mover>
   <mn>0</mn>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <times></times>
    <apply>
     <divide></divide>
     <apply>
      <times></times>
      <ci>d</ci>
      <apply>
       <csymbol cd="latexml">delimited-[]</csymbol>
       <apply>
        <times></times>
        <ci>E</ci>
        <ci>S</ci>
       </apply>
      </apply>
     </apply>
     <ci>d</ci>
    </apply>
    <ci>t</ci>
    <apply>
     <factorial></factorial>
     <eq></eq>
    </apply>
    <cn type="float">0</cn>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   d{[}ES{]}/{dt}\;\overset{!}{=}\;0
  </annotation>
 </semantics>
</math>

. The second assumption is that the total enzyme concentration does not change over time, thus 

<math display="inline" id="Enzyme_kinetics:5">
 <semantics>
  <mrow>
   <msub>
    <mrow>
     <mo stretchy="false">[</mo>
     <mi>E</mi>
     <mo stretchy="false">]</mo>
    </mrow>
    <mtext>tot</mtext>
   </msub>
   <mo>=</mo>
   <mrow>
    <mrow>
     <mo stretchy="false">[</mo>
     <mi>E</mi>
     <mo stretchy="false">]</mo>
    </mrow>
    <mo>+</mo>
    <mrow>
     <mrow>
      <mo stretchy="false">[</mo>
      <mrow>
       <mi>E</mi>
       <mi>S</mi>
      </mrow>
      <mo rspace="5.3pt" stretchy="false">]</mo>
     </mrow>
     <mpadded width="+2.8pt">
      <mover accent="true">
       <mo>=</mo>
       <mo>!</mo>
      </mover>
     </mpadded>
     <mtext>const</mtext>
    </mrow>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <eq></eq>
    <apply>
     <csymbol cd="ambiguous">subscript</csymbol>
     <apply>
      <csymbol cd="latexml">delimited-[]</csymbol>
      <ci>E</ci>
     </apply>
     <mtext>tot</mtext>
    </apply>
    <apply>
     <plus></plus>
     <apply>
      <csymbol cd="latexml">delimited-[]</csymbol>
      <ci>E</ci>
     </apply>
     <apply>
      <times></times>
      <apply>
       <csymbol cd="latexml">delimited-[]</csymbol>
       <apply>
        <times></times>
        <ci>E</ci>
        <ci>S</ci>
       </apply>
      </apply>
      <apply>
       <factorial></factorial>
       <eq></eq>
      </apply>
      <mtext>const</mtext>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   {[}E{]}_{\text{tot}}={[}E{]}+{[}ES{]}\;\overset{!}{=}\;\text{const}
  </annotation>
 </semantics>
</math>

. A complete derivation can be found <a href="Michaelis–Menten_kinetics#Equation" title="wikilink">here</a>.</p>

<p>The Michaelis constant <em>K</em><sub>M</sub> is experimentally defined as the concentration at which the rate of the enzyme reaction is half <em>V</em><sub>max</sub>, which can be verified by substituting [S] = <em>K</em><sub>m</sub> into the Michaelis–Menten equation and can also be seen graphically. If the rate-determining enzymatic step is slow compared to substrate dissociation (

<math display="inline" id="Enzyme_kinetics:6">
 <semantics>
  <mrow>
   <msub>
    <mi>k</mi>
    <mn>2</mn>
   </msub>
   <mo>≪</mo>
   <msub>
    <mi>k</mi>
    <mrow>
     <mo>-</mo>
     <mn>1</mn>
    </mrow>
   </msub>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <csymbol cd="latexml">much-less-than</csymbol>
    <apply>
     <csymbol cd="ambiguous">subscript</csymbol>
     <ci>k</ci>
     <cn type="integer">2</cn>
    </apply>
    <apply>
     <csymbol cd="ambiguous">subscript</csymbol>
     <ci>k</ci>
     <apply>
      <minus></minus>
      <cn type="integer">1</cn>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   k_{2}\ll k_{-1}
  </annotation>
 </semantics>
</math>


), the Michaelis constant <em>K</em><sub>M</sub> is roughly the <a href="dissociation_constant" title="wikilink">dissociation constant</a> <em>K</em><sub>D</sub> of the ES complex.</p>

<p>If 

<math display="inline" id="Enzyme_kinetics:7">
 <semantics>
  <mrow>
   <mo stretchy="false">[</mo>
   <mi>S</mi>
   <mo stretchy="false">]</mo>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <csymbol cd="latexml">delimited-[]</csymbol>
    <ci>S</ci>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   [S]
  </annotation>
 </semantics>
</math>

 is small compared to 

<math display="inline" id="Enzyme_kinetics:8">
 <semantics>
  <msub>
   <mi>K</mi>
   <mi>M</mi>
  </msub>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <csymbol cd="ambiguous">subscript</csymbol>
    <ci>K</ci>
    <ci>M</ci>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   K_{M}
  </annotation>
 </semantics>
</math>

 then the term 

<math display="inline" id="Enzyme_kinetics:9">
 <semantics>
  <mrow>
   <mrow>
    <mrow>
     <mo stretchy="false">[</mo>
     <mi>S</mi>
     <mo stretchy="false">]</mo>
    </mrow>
    <mo>/</mo>
    <mrow>
     <mo stretchy="false">(</mo>
     <mrow>
      <msub>
       <mi>K</mi>
       <mi>M</mi>
      </msub>
      <mo>+</mo>
      <mrow>
       <mo stretchy="false">[</mo>
       <mi>S</mi>
       <mo stretchy="false">]</mo>
      </mrow>
     </mrow>
     <mo stretchy="false">)</mo>
    </mrow>
   </mrow>
   <mo>≈</mo>
   <mrow>
    <mrow>
     <mo stretchy="false">[</mo>
     <mi>S</mi>
     <mo stretchy="false">]</mo>
    </mrow>
    <mo>/</mo>
    <msub>
     <mi>K</mi>
     <mi>M</mi>
    </msub>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <approx></approx>
    <apply>
     <divide></divide>
     <apply>
      <csymbol cd="latexml">delimited-[]</csymbol>
      <ci>S</ci>
     </apply>
     <apply>
      <plus></plus>
      <apply>
       <csymbol cd="ambiguous">subscript</csymbol>
       <ci>K</ci>
       <ci>M</ci>
      </apply>
      <apply>
       <csymbol cd="latexml">delimited-[]</csymbol>
       <ci>S</ci>
      </apply>
     </apply>
    </apply>
    <apply>
     <divide></divide>
     <apply>
      <csymbol cd="latexml">delimited-[]</csymbol>
      <ci>S</ci>
     </apply>
     <apply>
      <csymbol cd="ambiguous">subscript</csymbol>
      <ci>K</ci>
      <ci>M</ci>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   [S]/(K_{M}+[S])\approx[S]/K_{M}
  </annotation>
 </semantics>
</math>

 and also very little ES complex is formed, thus 

<math display="inline" id="Enzyme_kinetics:10">
 <semantics>
  <mrow>
   <msub>
    <mrow>
     <mo stretchy="false">[</mo>
     <mi>E</mi>
     <mo stretchy="false">]</mo>
    </mrow>
    <mn>0</mn>
   </msub>
   <mo>≈</mo>
   <mrow>
    <mo stretchy="false">[</mo>
    <mi>E</mi>
    <mo stretchy="false">]</mo>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <approx></approx>
    <apply>
     <csymbol cd="ambiguous">subscript</csymbol>
     <apply>
      <csymbol cd="latexml">delimited-[]</csymbol>
      <ci>E</ci>
     </apply>
     <cn type="integer">0</cn>
    </apply>
    <apply>
     <csymbol cd="latexml">delimited-[]</csymbol>
     <ci>E</ci>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   [E]_{0}\approx[E]
  </annotation>
 </semantics>
</math>

. Therefore, the rate of product formation is</p>

<p>

<math display="block" id="Enzyme_kinetics:11">
 <semantics>
  <mrow>
   <mrow>
    <msub>
     <mi>v</mi>
     <mn>0</mn>
    </msub>
    <mo>≈</mo>
    <mrow>
     <mfrac>
      <msub>
       <mi>k</mi>
       <mrow>
        <mi>c</mi>
        <mi>a</mi>
        <mi>t</mi>
       </mrow>
      </msub>
      <msub>
       <mi>K</mi>
       <mi>M</mi>
      </msub>
     </mfrac>
     <mrow>
      <mo stretchy="false">[</mo>
      <mi>E</mi>
      <mo stretchy="false">]</mo>
     </mrow>
     <mrow>
      <mo stretchy="false">[</mo>
      <mi>S</mi>
      <mo stretchy="false">]</mo>
     </mrow>
    </mrow>
   </mrow>
   <mrow>
    <mrow>
     <mtext>if</mtext>
     <mrow>
      <mo stretchy="false">[</mo>
      <mi>S</mi>
      <mo stretchy="false">]</mo>
     </mrow>
    </mrow>
    <mo>≪</mo>
    <msub>
     <mi>K</mi>
     <mi>M</mi>
    </msub>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <csymbol cd="ambiguous">formulae-sequence</csymbol>
    <apply>
     <approx></approx>
     <apply>
      <csymbol cd="ambiguous">subscript</csymbol>
      <ci>v</ci>
      <cn type="integer">0</cn>
     </apply>
     <apply>
      <times></times>
      <apply>
       <divide></divide>
       <apply>
        <csymbol cd="ambiguous">subscript</csymbol>
        <ci>k</ci>
        <apply>
         <times></times>
         <ci>c</ci>
         <ci>a</ci>
         <ci>t</ci>
        </apply>
       </apply>
       <apply>
        <csymbol cd="ambiguous">subscript</csymbol>
        <ci>K</ci>
        <ci>M</ci>
       </apply>
      </apply>
      <apply>
       <csymbol cd="latexml">delimited-[]</csymbol>
       <ci>E</ci>
      </apply>
      <apply>
       <csymbol cd="latexml">delimited-[]</csymbol>
       <ci>S</ci>
      </apply>
     </apply>
    </apply>
    <apply>
     <csymbol cd="latexml">much-less-than</csymbol>
     <apply>
      <times></times>
      <mtext>if</mtext>
      <apply>
       <csymbol cd="latexml">delimited-[]</csymbol>
       <ci>S</ci>
      </apply>
     </apply>
     <apply>
      <csymbol cd="ambiguous">subscript</csymbol>
      <ci>K</ci>
      <ci>M</ci>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   v_{0}\approx\frac{k_{cat}}{K_{M}}[E][S]\qquad\qquad\text{if }[S]\ll K_{M}
  </annotation>
 </semantics>
</math>

 Thus the product formation rate depends on the enzyme concentration as well as on the substrate concentration, the equation resembles a bimolecular reaction with a corresponding pseudo-second order rate constant 

<math display="inline" id="Enzyme_kinetics:12">
 <semantics>
  <mrow>
   <msub>
    <mi>k</mi>
    <mn>2</mn>
   </msub>
   <mo>/</mo>
   <msub>
    <mi>K</mi>
    <mi>M</mi>
   </msub>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <divide></divide>
    <apply>
     <csymbol cd="ambiguous">subscript</csymbol>
     <ci>k</ci>
     <cn type="integer">2</cn>
    </apply>
    <apply>
     <csymbol cd="ambiguous">subscript</csymbol>
     <ci>K</ci>
     <ci>M</ci>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   k_{2}/K_{M}
  </annotation>
 </semantics>
</math>

. This constant is a measure of <a href="catalytic_efficiency" title="wikilink">catalytic efficiency</a>. The most efficient enzymes reach a 

<math display="inline" id="Enzyme_kinetics:13">
 <semantics>
  <mrow>
   <msub>
    <mi>k</mi>
    <mn>2</mn>
   </msub>
   <mo>/</mo>
   <msub>
    <mi>K</mi>
    <mi>M</mi>
   </msub>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <divide></divide>
    <apply>
     <csymbol cd="ambiguous">subscript</csymbol>
     <ci>k</ci>
     <cn type="integer">2</cn>
    </apply>
    <apply>
     <csymbol cd="ambiguous">subscript</csymbol>
     <ci>K</ci>
     <ci>M</ci>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   k_{2}/K_{M}
  </annotation>
 </semantics>
</math>

 in the range of 10<sup>8</sup> – 10<sup>10</sup> <a href="Concentration#Molality" title="wikilink">M</a><sup>−1</sup> <a href="second" title="wikilink">s</a><sup>−1</sup>. These enzymes are so efficient they effectively catalyse a reaction each time they encounter a substrate molecule and have thus reached an upper theoretical limit for efficiency (<a href="Diffusion_limited_enzyme" title="wikilink">diffusion limit</a>); and are sometimes referred to as <a href="Diffusion_limited_enzyme" title="wikilink">kinetically perfect enzymes</a>.<a class="footnoteRef" href="#fn10" id="fnref10"><sup>10</sup></a></p>
<h3 id="direct-use-of-the-michaelismenten-equation-for-time-course-kinetic-analysis">Direct use of the Michaelis–Menten equation for time course kinetic analysis</h3>

<p>The observed velocities predicted by the Michaelis–Menten equation can be used to directly model the <a href="Reaction_progress_kinetic_analysis" title="wikilink">time course disappearance of substrate</a> and the production of product through incorporation of the Michaelis–Menten equation into the equation for first order chemical kinetics. This can only be achieved however if one recognises the problem associated with the use of <a href="Euler's_number" title="wikilink">Euler's number</a> in the description of first order chemical kinetics. i.e. <em>e</em><sup>−<em>k</em></sup> is a split constant that introduces a systematic error into calculations and can be rewritten as a single constant which represents the remaining substrate after each time period.<a class="footnoteRef" href="#fn11" id="fnref11"><sup>11</sup></a></p>

<p>

<math display="block" id="Enzyme_kinetics:14">
 <semantics>
  <mrow>
   <mrow>
    <mo stretchy="false">[</mo>
    <mi>S</mi>
    <mo stretchy="false">]</mo>
   </mrow>
   <mo>=</mo>
   <mrow>
    <msub>
     <mrow>
      <mo stretchy="false">[</mo>
      <mi>S</mi>
      <mo stretchy="false">]</mo>
     </mrow>
     <mn>0</mn>
    </msub>
    <mpadded width="+1.7pt">
     <msup>
      <mrow>
       <mo stretchy="false">(</mo>
       <mrow>
        <mn>1</mn>
        <mo>-</mo>
        <mi>k</mi>
       </mrow>
       <mo stretchy="false">)</mo>
      </mrow>
      <mi>t</mi>
     </msup>
    </mpadded>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <eq></eq>
    <apply>
     <csymbol cd="latexml">delimited-[]</csymbol>
     <ci>S</ci>
    </apply>
    <apply>
     <times></times>
     <apply>
      <csymbol cd="ambiguous">subscript</csymbol>
      <apply>
       <csymbol cd="latexml">delimited-[]</csymbol>
       <ci>S</ci>
      </apply>
      <cn type="integer">0</cn>
     </apply>
     <apply>
      <csymbol cd="ambiguous">superscript</csymbol>
      <apply>
       <minus></minus>
       <cn type="integer">1</cn>
       <ci>k</ci>
      </apply>
      <ci>t</ci>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   [S]=[S]_{0}(1-k)^{t}\,
  </annotation>
 </semantics>
</math>

</p>

<p>

<math display="block" id="Enzyme_kinetics:15">
 <semantics>
  <mrow>
   <mrow>
    <mo stretchy="false">[</mo>
    <mi>S</mi>
    <mo stretchy="false">]</mo>
   </mrow>
   <mo>=</mo>
   <mrow>
    <msub>
     <mrow>
      <mo stretchy="false">[</mo>
      <mi>S</mi>
      <mo stretchy="false">]</mo>
     </mrow>
     <mn>0</mn>
    </msub>
    <mpadded width="+1.7pt">
     <msup>
      <mrow>
       <mo stretchy="false">(</mo>
       <mrow>
        <mn>1</mn>
        <mo>-</mo>
        <mrow>
         <mi>v</mi>
         <mo>/</mo>
         <msub>
          <mrow>
           <mo stretchy="false">[</mo>
           <mi>S</mi>
           <mo stretchy="false">]</mo>
          </mrow>
          <mn>0</mn>
         </msub>
        </mrow>
       </mrow>
       <mo stretchy="false">)</mo>
      </mrow>
      <mi>t</mi>
     </msup>
    </mpadded>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <eq></eq>
    <apply>
     <csymbol cd="latexml">delimited-[]</csymbol>
     <ci>S</ci>
    </apply>
    <apply>
     <times></times>
     <apply>
      <csymbol cd="ambiguous">subscript</csymbol>
      <apply>
       <csymbol cd="latexml">delimited-[]</csymbol>
       <ci>S</ci>
      </apply>
      <cn type="integer">0</cn>
     </apply>
     <apply>
      <csymbol cd="ambiguous">superscript</csymbol>
      <apply>
       <minus></minus>
       <cn type="integer">1</cn>
       <apply>
        <divide></divide>
        <ci>v</ci>
        <apply>
         <csymbol cd="ambiguous">subscript</csymbol>
         <apply>
          <csymbol cd="latexml">delimited-[]</csymbol>
          <ci>S</ci>
         </apply>
         <cn type="integer">0</cn>
        </apply>
       </apply>
      </apply>
      <ci>t</ci>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   [S]=[S]_{0}(1-v/[S]_{0})^{t}\,
  </annotation>
 </semantics>
</math>

</p>

<p>

<math display="block" id="Enzyme_kinetics:16">
 <semantics>
  <mrow>
   <mrow>
    <mo stretchy="false">[</mo>
    <mi>S</mi>
    <mo stretchy="false">]</mo>
   </mrow>
   <mo>=</mo>
   <mrow>
    <msub>
     <mrow>
      <mo stretchy="false">[</mo>
      <mi>S</mi>
      <mo stretchy="false">]</mo>
     </mrow>
     <mn>0</mn>
    </msub>
    <mpadded width="+1.7pt">
     <msup>
      <mrow>
       <mo stretchy="false">(</mo>
       <mrow>
        <mn>1</mn>
        <mo>-</mo>
        <mrow>
         <mo stretchy="false">(</mo>
         <mrow>
          <mrow>
           <msub>
            <mi>V</mi>
            <mi>max</mi>
           </msub>
           <msub>
            <mrow>
             <mo stretchy="false">[</mo>
             <mi>S</mi>
             <mo stretchy="false">]</mo>
            </mrow>
            <mn>0</mn>
           </msub>
          </mrow>
          <mo>/</mo>
          <mrow>
           <mo stretchy="false">(</mo>
           <mrow>
            <msub>
             <mi>K</mi>
             <mi>M</mi>
            </msub>
            <mo>+</mo>
            <msub>
             <mrow>
              <mo stretchy="false">[</mo>
              <mi>S</mi>
              <mo stretchy="false">]</mo>
             </mrow>
             <mn>0</mn>
            </msub>
           </mrow>
           <mo stretchy="false">)</mo>
          </mrow>
          <mo>/</mo>
          <msub>
           <mrow>
            <mo stretchy="false">[</mo>
            <mi>S</mi>
            <mo stretchy="false">]</mo>
           </mrow>
           <mn>0</mn>
          </msub>
         </mrow>
         <mo stretchy="false">)</mo>
        </mrow>
       </mrow>
       <mo stretchy="false">)</mo>
      </mrow>
      <mi>t</mi>
     </msup>
    </mpadded>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <eq></eq>
    <apply>
     <csymbol cd="latexml">delimited-[]</csymbol>
     <ci>S</ci>
    </apply>
    <apply>
     <times></times>
     <apply>
      <csymbol cd="ambiguous">subscript</csymbol>
      <apply>
       <csymbol cd="latexml">delimited-[]</csymbol>
       <ci>S</ci>
      </apply>
      <cn type="integer">0</cn>
     </apply>
     <apply>
      <csymbol cd="ambiguous">superscript</csymbol>
      <apply>
       <minus></minus>
       <cn type="integer">1</cn>
       <apply>
        <divide></divide>
        <apply>
         <times></times>
         <apply>
          <csymbol cd="ambiguous">subscript</csymbol>
          <ci>V</ci>
          <max></max>
         </apply>
         <apply>
          <csymbol cd="ambiguous">subscript</csymbol>
          <apply>
           <csymbol cd="latexml">delimited-[]</csymbol>
           <ci>S</ci>
          </apply>
          <cn type="integer">0</cn>
         </apply>
        </apply>
        <apply>
         <plus></plus>
         <apply>
          <csymbol cd="ambiguous">subscript</csymbol>
          <ci>K</ci>
          <ci>M</ci>
         </apply>
         <apply>
          <csymbol cd="ambiguous">subscript</csymbol>
          <apply>
           <csymbol cd="latexml">delimited-[]</csymbol>
           <ci>S</ci>
          </apply>
          <cn type="integer">0</cn>
         </apply>
        </apply>
        <apply>
         <csymbol cd="ambiguous">subscript</csymbol>
         <apply>
          <csymbol cd="latexml">delimited-[]</csymbol>
          <ci>S</ci>
         </apply>
         <cn type="integer">0</cn>
        </apply>
       </apply>
      </apply>
      <ci>t</ci>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   [S]=[S]_{0}(1-(V_{\max}[S]_{0}/(K_{M}+[S]_{0})/[S]_{0}))^{t}\,
  </annotation>
 </semantics>
</math>

</p>

<p>In 1983 Stuart Beal (and also independently <a href="Santiago_Schnell" title="wikilink">Santiago Schnell</a> and Claudio Mendoza in 1997) derived a closed form solution for the time course kinetics analysis of the Michaelis-Menten mechanism.<a class="footnoteRef" href="#fn12" id="fnref12"><sup>12</sup></a><a class="footnoteRef" href="#fn13" id="fnref13"><sup>13</sup></a> The solution, known as the Schnell-Mendoza equation, has the form:</p>

<p>

<math display="block" id="Enzyme_kinetics:17">
 <semantics>
  <mrow>
   <mfrac>
    <mrow>
     <mo stretchy="false">[</mo>
     <mi>S</mi>
     <mo stretchy="false">]</mo>
    </mrow>
    <msub>
     <mi>K</mi>
     <mi>M</mi>
    </msub>
   </mfrac>
   <mo>=</mo>
   <mrow>
    <mi>W</mi>
    <mrow>
     <mo>[</mo>
     <mrow>
      <mi>F</mi>
      <mrow>
       <mo stretchy="false">(</mo>
       <mi>t</mi>
       <mo stretchy="false">)</mo>
      </mrow>
     </mrow>
     <mo rspace="4.2pt">]</mo>
    </mrow>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <eq></eq>
    <apply>
     <divide></divide>
     <apply>
      <csymbol cd="latexml">delimited-[]</csymbol>
      <ci>S</ci>
     </apply>
     <apply>
      <csymbol cd="ambiguous">subscript</csymbol>
      <ci>K</ci>
      <ci>M</ci>
     </apply>
    </apply>
    <apply>
     <times></times>
     <ci>W</ci>
     <apply>
      <csymbol cd="latexml">delimited-[]</csymbol>
      <apply>
       <times></times>
       <ci>F</ci>
       <ci>t</ci>
      </apply>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \frac{[S]}{K_{M}}=W\left[F(t)\right]\,
  </annotation>
 </semantics>
</math>

</p>

<p>where W[] is the <a href="Lambert_W_function" title="wikilink">Lambert-W function</a>.<a class="footnoteRef" href="#fn14" id="fnref14"><sup>14</sup></a><a class="footnoteRef" href="#fn15" id="fnref15"><sup>15</sup></a> and where F(t) is</p>

<p>

<math display="block" id="Enzyme_kinetics:18">
 <semantics>
  <mrow>
   <mrow>
    <mi>F</mi>
    <mrow>
     <mo stretchy="false">(</mo>
     <mi>t</mi>
     <mo stretchy="false">)</mo>
    </mrow>
   </mrow>
   <mo>=</mo>
   <mrow>
    <mfrac>
     <msub>
      <mrow>
       <mo stretchy="false">[</mo>
       <mi>S</mi>
       <mo stretchy="false">]</mo>
      </mrow>
      <mn>0</mn>
     </msub>
     <msub>
      <mi>K</mi>
      <mi>M</mi>
     </msub>
    </mfrac>
    <mrow>
     <mpadded width="-1.7pt">
      <mi>exp</mi>
     </mpadded>
     <mrow>
      <mo>(</mo>
      <mrow>
       <mfrac>
        <msub>
         <mrow>
          <mo stretchy="false">[</mo>
          <mi>S</mi>
          <mo stretchy="false">]</mo>
         </mrow>
         <mn>0</mn>
        </msub>
        <msub>
         <mi>K</mi>
         <mi>M</mi>
        </msub>
       </mfrac>
       <mo>-</mo>
       <mrow>
        <mpadded width="+1.7pt">
         <mfrac>
          <msub>
           <mi>V</mi>
           <mi>max</mi>
          </msub>
          <msub>
           <mi>K</mi>
           <mi>M</mi>
          </msub>
         </mfrac>
        </mpadded>
        <mi>t</mi>
       </mrow>
      </mrow>
      <mo rspace="4.2pt">)</mo>
     </mrow>
    </mrow>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <eq></eq>
    <apply>
     <times></times>
     <ci>F</ci>
     <ci>t</ci>
    </apply>
    <apply>
     <times></times>
     <apply>
      <divide></divide>
      <apply>
       <csymbol cd="ambiguous">subscript</csymbol>
       <apply>
        <csymbol cd="latexml">delimited-[]</csymbol>
        <ci>S</ci>
       </apply>
       <cn type="integer">0</cn>
      </apply>
      <apply>
       <csymbol cd="ambiguous">subscript</csymbol>
       <ci>K</ci>
       <ci>M</ci>
      </apply>
     </apply>
     <apply>
      <exp></exp>
      <apply>
       <minus></minus>
       <apply>
        <divide></divide>
        <apply>
         <csymbol cd="ambiguous">subscript</csymbol>
         <apply>
          <csymbol cd="latexml">delimited-[]</csymbol>
          <ci>S</ci>
         </apply>
         <cn type="integer">0</cn>
        </apply>
        <apply>
         <csymbol cd="ambiguous">subscript</csymbol>
         <ci>K</ci>
         <ci>M</ci>
        </apply>
       </apply>
       <apply>
        <times></times>
        <apply>
         <divide></divide>
         <apply>
          <csymbol cd="ambiguous">subscript</csymbol>
          <ci>V</ci>
          <max></max>
         </apply>
         <apply>
          <csymbol cd="ambiguous">subscript</csymbol>
          <ci>K</ci>
          <ci>M</ci>
         </apply>
        </apply>
        <ci>t</ci>
       </apply>
      </apply>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   F(t)=\frac{[S]_{0}}{K_{M}}\exp\!\left(\frac{[S]_{0}}{K_{M}}-\frac{V_{\max}}{K_%
{M}}\,t\right)\,
  </annotation>
 </semantics>
</math>

</p>

<p>This equation is encompassed by the equation below, obtained by Berberan-Santos (MATCH Commun. Math. Comput. Chem. 63 (2010) 283), which is also valid when the initial substrate concentration is close to that of enzyme,</p>

<p>

<math display="block" id="Enzyme_kinetics:19">
 <semantics>
  <mrow>
   <mfrac>
    <mrow>
     <mo stretchy="false">[</mo>
     <mi>S</mi>
     <mo stretchy="false">]</mo>
    </mrow>
    <msub>
     <mi>K</mi>
     <mi>M</mi>
    </msub>
   </mfrac>
   <mo>=</mo>
   <mrow>
    <mrow>
     <mi>W</mi>
     <mrow>
      <mo>[</mo>
      <mrow>
       <mi>F</mi>
       <mrow>
        <mo stretchy="false">(</mo>
        <mi>t</mi>
        <mo stretchy="false">)</mo>
       </mrow>
      </mrow>
      <mo>]</mo>
     </mrow>
    </mrow>
    <mo>-</mo>
    <mrow>
     <mpadded width="+5pt">
      <mfrac>
       <msub>
        <mi>V</mi>
        <mi>max</mi>
       </msub>
       <mrow>
        <msub>
         <mi>k</mi>
         <mrow>
          <mi>c</mi>
          <mi>a</mi>
          <mi>t</mi>
         </mrow>
        </msub>
        <msub>
         <mi>K</mi>
         <mi>M</mi>
        </msub>
       </mrow>
      </mfrac>
     </mpadded>
     <mpadded width="+1.7pt">
      <mfrac>
       <mrow>
        <mi>W</mi>
        <mrow>
         <mo>[</mo>
         <mrow>
          <mi>F</mi>
          <mrow>
           <mo stretchy="false">(</mo>
           <mi>t</mi>
           <mo stretchy="false">)</mo>
          </mrow>
         </mrow>
         <mo>]</mo>
        </mrow>
       </mrow>
       <mrow>
        <mn>1</mn>
        <mo>+</mo>
        <mrow>
         <mi>W</mi>
         <mrow>
          <mo>[</mo>
          <mrow>
           <mi>F</mi>
           <mrow>
            <mo stretchy="false">(</mo>
            <mi>t</mi>
            <mo stretchy="false">)</mo>
           </mrow>
          </mrow>
          <mo>]</mo>
         </mrow>
        </mrow>
       </mrow>
      </mfrac>
     </mpadded>
    </mrow>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <eq></eq>
    <apply>
     <divide></divide>
     <apply>
      <csymbol cd="latexml">delimited-[]</csymbol>
      <ci>S</ci>
     </apply>
     <apply>
      <csymbol cd="ambiguous">subscript</csymbol>
      <ci>K</ci>
      <ci>M</ci>
     </apply>
    </apply>
    <apply>
     <minus></minus>
     <apply>
      <times></times>
      <ci>W</ci>
      <apply>
       <csymbol cd="latexml">delimited-[]</csymbol>
       <apply>
        <times></times>
        <ci>F</ci>
        <ci>t</ci>
       </apply>
      </apply>
     </apply>
     <apply>
      <times></times>
      <apply>
       <divide></divide>
       <apply>
        <csymbol cd="ambiguous">subscript</csymbol>
        <ci>V</ci>
        <max></max>
       </apply>
       <apply>
        <times></times>
        <apply>
         <csymbol cd="ambiguous">subscript</csymbol>
         <ci>k</ci>
         <apply>
          <times></times>
          <ci>c</ci>
          <ci>a</ci>
          <ci>t</ci>
         </apply>
        </apply>
        <apply>
         <csymbol cd="ambiguous">subscript</csymbol>
         <ci>K</ci>
         <ci>M</ci>
        </apply>
       </apply>
      </apply>
      <apply>
       <divide></divide>
       <apply>
        <times></times>
        <ci>W</ci>
        <apply>
         <csymbol cd="latexml">delimited-[]</csymbol>
         <apply>
          <times></times>
          <ci>F</ci>
          <ci>t</ci>
         </apply>
        </apply>
       </apply>
       <apply>
        <plus></plus>
        <cn type="integer">1</cn>
        <apply>
         <times></times>
         <ci>W</ci>
         <apply>
          <csymbol cd="latexml">delimited-[]</csymbol>
          <apply>
           <times></times>
           <ci>F</ci>
           <ci>t</ci>
          </apply>
         </apply>
        </apply>
       </apply>
      </apply>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \frac{[S]}{K_{M}}=W\left[F(t)\right]-\frac{V_{\max}}{k_{cat}K_{M}}\ \frac{W%
\left[F(t)\right]}{1+W\left[F(t)\right]}\,
  </annotation>
 </semantics>
</math>

</p>

<p>where W[] is again the <a href="Lambert_W_function" title="wikilink">Lambert-W function</a>.</p>
<h3 id="linear-plots-of-the-michaelismenten-equation">Linear plots of the Michaelis–Menten equation</h3>
<figure><b>(Figure)</b>
<figcaption>Lineweaver–Burk or double-reciprocal plot of kinetic data, showing the significance of the axis intercepts and gradient.</figcaption>
</figure>

<p>The plot of <em>v</em> versus [S] above is not linear; although initially linear at low [S], it bends over to saturate at high [S]. Before the modern era of <a href="nonlinear_regression" title="wikilink">nonlinear curve-fitting</a> on computers, this nonlinearity could make it difficult to estimate <em>K</em><sub>M</sub> and <em>V</em><sub>max</sub> accurately. Therefore, several researchers developed linearisations of the Michaelis–Menten equation, such as the <a href="Lineweaver–Burk_plot" title="wikilink">Lineweaver–Burk plot</a>, the <a href="Eadie–Hofstee_diagram" title="wikilink">Eadie–Hofstee diagram</a> and the <a href="Hanes–Woolf_plot" title="wikilink">Hanes–Woolf plot</a>. All of these linear representations can be useful for visualising data, but none should be used to determine kinetic parameters, as computer software is readily available that allows for more accurate determination by <a href="nonlinear_regression" title="wikilink">nonlinear regression</a> methods.<a class="footnoteRef" href="#fn16" id="fnref16"><sup>16</sup></a></p>

<p>The <a href="Lineweaver–Burk_plot" title="wikilink">Lineweaver–Burk plot</a> or double reciprocal plot is a common way of illustrating kinetic data. This is produced by taking the <a href="Multiplicative_inverse" title="wikilink">reciprocal</a> of both sides of the Michaelis–Menten equation. As shown on the right, this is a linear form of the Michaelis–Menten equation and produces a straight line with the equation <em>y</em> = m<em>x</em> + c with a <em>y</em>-intercept equivalent to 1/<em>V</em><sub>max</sub> and an <em>x</em>-intercept of the graph representing −1/<em>K</em><sub>M</sub>.</p>

<p>

<math display="block" id="Enzyme_kinetics:20">
 <semantics>
  <mrow>
   <mfrac>
    <mn>1</mn>
    <mi>v</mi>
   </mfrac>
   <mo>=</mo>
   <mrow>
    <mfrac>
     <msub>
      <mi>K</mi>
      <mi>M</mi>
     </msub>
     <mrow>
      <msub>
       <mi>V</mi>
       <mi>max</mi>
      </msub>
      <mrow>
       <mo stretchy="false">[</mo>
       <mtext>S</mtext>
       <mo stretchy="false">]</mo>
      </mrow>
     </mrow>
    </mfrac>
    <mo>+</mo>
    <mfrac>
     <mn>1</mn>
     <msub>
      <mi>V</mi>
      <mi>max</mi>
     </msub>
    </mfrac>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <eq></eq>
    <apply>
     <divide></divide>
     <cn type="integer">1</cn>
     <ci>v</ci>
    </apply>
    <apply>
     <plus></plus>
     <apply>
      <divide></divide>
      <apply>
       <csymbol cd="ambiguous">subscript</csymbol>
       <ci>K</ci>
       <ci>M</ci>
      </apply>
      <apply>
       <times></times>
       <apply>
        <csymbol cd="ambiguous">subscript</csymbol>
        <ci>V</ci>
        <max></max>
       </apply>
       <apply>
        <csymbol cd="latexml">delimited-[]</csymbol>
        <mtext>S</mtext>
       </apply>
      </apply>
     </apply>
     <apply>
      <divide></divide>
      <cn type="integer">1</cn>
      <apply>
       <csymbol cd="ambiguous">subscript</csymbol>
       <ci>V</ci>
       <max></max>
      </apply>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \frac{1}{v}=\frac{K_{M}}{V_{\max}[\mbox{S}]}+\frac{1}{V_{\max}}
  </annotation>
 </semantics>
</math>

</p>

<p>Naturally, no experimental values can be taken at negative 1/[S]; the lower limiting value 1/[S] = 0 (the <em>y</em>-intercept) corresponds to an infinite substrate concentration, where <em>1/v=1/V<sub>max</sub></em> as shown at the right; thus, the <em>x</em>-intercept is an <a class="uri" href="extrapolation" title="wikilink">extrapolation</a> of the experimental data taken at positive concentrations. More generally, the Lineweaver–Burk plot skews the importance of measurements taken at low substrate concentrations and, thus, can yield inaccurate estimates of <em>V</em><sub>max</sub> and <em>K</em><sub>M</sub>.<a class="footnoteRef" href="#fn17" id="fnref17"><sup>17</sup></a> A more accurate linear plotting method is the <a href="Eadie-Hofstee_diagram" title="wikilink">Eadie-Hofstee plot</a>. In this case, <em>v</em> is plotted against <em>v</em>/[S]. In the third common linear representation, the <a href="Hanes-Woolf_plot" title="wikilink">Hanes-Woolf plot</a>, [S]/<em>v</em> is plotted against [S]. In general, data normalisation can help diminish the amount of experimental work and can increase the reliability of the output, and is suitable for both graphical and numerical analysis.<a class="footnoteRef" href="#fn18" id="fnref18"><sup>18</sup></a></p>
<h3 id="practical-significance-of-kinetic-constants">Practical significance of kinetic constants</h3>

<p>The study of enzyme kinetics is important for two basic reasons. Firstly, it helps explain how enzymes work, and secondly, it helps predict how enzymes behave in living organisms. The kinetic constants defined above, <em>K</em><sub>M</sub> and <em>V</em><sub>max</sub>, are critical to attempts to understand how enzymes work together to control <a class="uri" href="metabolism" title="wikilink">metabolism</a>.</p>

<p>Making these predictions is not trivial, even for simple systems. For example, <a class="uri" href="oxaloacetate" title="wikilink">oxaloacetate</a> is formed by <a href="malate_dehydrogenase" title="wikilink">malate dehydrogenase</a> within the <a class="uri" href="mitochondrion" title="wikilink">mitochondrion</a>. Oxaloacetate can then be consumed by <a href="citrate_synthase" title="wikilink">citrate synthase</a>, <a href="phosphoenolpyruvate_carboxykinase" title="wikilink">phosphoenolpyruvate carboxykinase</a> or <a href="aspartate_aminotransferase" title="wikilink">aspartate aminotransferase</a>, feeding into the <a href="citric_acid_cycle" title="wikilink">citric acid cycle</a>, <a class="uri" href="gluconeogenesis" title="wikilink">gluconeogenesis</a> or <a href="aspartic_acid" title="wikilink">aspartic acid</a> biosynthesis, respectively. Being able to predict how much oxaloacetate goes into which pathway requires knowledge of the concentration of oxaloacetate as well as the concentration and kinetics of each of these enzymes. This aim of predicting the behaviour of metabolic pathways reaches its most complex expression in the synthesis of huge amounts of kinetic and <a href="gene_expression" title="wikilink">gene expression</a> data into mathematical models of entire organisms. Alternatively, one useful simplification of the metabolic modelling problem is to ignore the underlying enzyme kinetics and only rely on information about the reaction network's stoichiometry, a technique called <a href="flux_balance_analysis" title="wikilink">flux balance analysis</a>.<a class="footnoteRef" href="#fn19" id="fnref19"><sup>19</sup></a><a class="footnoteRef" href="#fn20" id="fnref20"><sup>20</sup></a></p>
<h3 id="michaelismenten-kinetics-with-intermediate">Michaelis–Menten kinetics with intermediate</h3>

<p>One could also consider the less simple case</p>

<p>

<math display="inline" id="Enzyme_kinetics:21">
 <semantics>
  <mrow>
   <mi>E</mi>
   <mo>+</mo>
   <mrow>
    <mi>S</mi>
    <munder accentunder="true">
     <mover accent="true">
      <mtable>
       <mtr>
        <mtd columnalign="center">
         <mo mathsize="70%" stretchy="false">⟶</mo>
        </mtd>
       </mtr>
       <mtr>
        <mtd columnalign="center">
         <mo mathsize="70%" stretchy="false">⟵</mo>
        </mtd>
       </mtr>
      </mtable>
      <msub>
       <mi>k</mi>
       <mn>1</mn>
      </msub>
     </mover>
     <msub>
      <mi>k</mi>
      <mrow>
       <mo>-</mo>
       <mn>1</mn>
      </mrow>
     </msub>
    </munder>
    <mi>E</mi>
    <mi>S</mi>
    <mover accent="true">
     <mo>⟶</mo>
     <msub>
      <mi>k</mi>
      <mn>2</mn>
     </msub>
    </mover>
    <mi>E</mi>
    <mi>I</mi>
    <mover accent="true">
     <mo>⟶</mo>
     <msub>
      <mi>k</mi>
      <mn>3</mn>
     </msub>
    </mover>
    <mi>E</mi>
   </mrow>
   <mo>+</mo>
   <mi>P</mi>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <plus></plus>
    <ci>E</ci>
    <apply>
     <times></times>
     <ci>S</ci>
     <apply>
      <apply>
       <csymbol cd="ambiguous">subscript</csymbol>
       <ci>k</ci>
       <apply>
        <minus></minus>
        <cn type="integer">1</cn>
       </apply>
      </apply>
      <apply>
       <apply>
        <csymbol cd="ambiguous">subscript</csymbol>
        <ci>k</ci>
        <cn type="integer">1</cn>
       </apply>
       <matrix>
        <matrixrow>
         <ci>normal-⟶</ci>
        </matrixrow>
        <matrixrow>
         <ci>normal-⟵</ci>
        </matrixrow>
       </matrix>
      </apply>
     </apply>
     <ci>E</ci>
     <ci>S</ci>
     <apply>
      <apply>
       <csymbol cd="ambiguous">subscript</csymbol>
       <ci>k</ci>
       <cn type="integer">2</cn>
      </apply>
      <ci>normal-⟶</ci>
     </apply>
     <ci>E</ci>
     <ci>I</ci>
     <apply>
      <apply>
       <csymbol cd="ambiguous">subscript</csymbol>
       <ci>k</ci>
       <cn type="integer">3</cn>
      </apply>
      <ci>normal-⟶</ci>
     </apply>
     <ci>E</ci>
    </apply>
    <ci>P</ci>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \displaystyle E+S\underset{k_{-1}}{\overset{k_{1}}{\begin{smallmatrix}%
\displaystyle\longrightarrow\\
\displaystyle\longleftarrow\end{smallmatrix}}}ES\overset{k_{2}}{%
\longrightarrow}EI\overset{k_{3}}{\longrightarrow}E+P
  </annotation>
 </semantics>
</math>


 where a complex with the enzyme and an intermediate exists and the intermediate is converted into product in a second step. In this case we have a very similar equation<a class="footnoteRef" href="#fn21" id="fnref21"><sup>21</sup></a></p>

<p>

<math display="inline" id="Enzyme_kinetics:22">
 <semantics>
  <msub>
   <mi>v</mi>
   <mn>0</mn>
  </msub>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <csymbol cd="ambiguous">subscript</csymbol>
    <ci>v</ci>
    <cn type="integer">0</cn>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \displaystyle v_{0}
  </annotation>
 </semantics>
</math>


 but the constants are different</p>

<p>

<math display="inline" id="Enzyme_kinetics:23">
 <semantics>
  <msubsup>
   <mi>K</mi>
   <mi>M</mi>
   <mo>′</mo>
  </msubsup>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <csymbol cd="ambiguous">superscript</csymbol>
    <apply>
     <csymbol cd="ambiguous">subscript</csymbol>
     <ci>K</ci>
     <ci>M</ci>
    </apply>
    <ci>normal-′</ci>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \displaystyle K_{M}^{\prime}
  </annotation>
 </semantics>
</math>


 We see that for the limiting case 

<math display="inline" id="Enzyme_kinetics:24">
 <semantics>
  <mrow>
   <msub>
    <mi>k</mi>
    <mn>3</mn>
   </msub>
   <mo>≫</mo>
   <msub>
    <mi>k</mi>
    <mn>2</mn>
   </msub>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <csymbol cd="latexml">much-greater-than</csymbol>
    <apply>
     <csymbol cd="ambiguous">subscript</csymbol>
     <ci>k</ci>
     <cn type="integer">3</cn>
    </apply>
    <apply>
     <csymbol cd="ambiguous">subscript</csymbol>
     <ci>k</ci>
     <cn type="integer">2</cn>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   k_{3}\gg k_{2}
  </annotation>
 </semantics>
</math>

, thus when the last step from <em>EI</em> to <em>E</em> + <em>P</em> is much faster than the previous step, we get again the original equation. Mathematically we have then 

<math display="inline" id="Enzyme_kinetics:25">
 <semantics>
  <mrow>
   <msubsup>
    <mi>K</mi>
    <mi>M</mi>
    <mo>′</mo>
   </msubsup>
   <mo>≈</mo>
   <msub>
    <mi>K</mi>
    <mi>M</mi>
   </msub>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <approx></approx>
    <apply>
     <csymbol cd="ambiguous">superscript</csymbol>
     <apply>
      <csymbol cd="ambiguous">subscript</csymbol>
      <ci>K</ci>
      <ci>M</ci>
     </apply>
     <ci>normal-′</ci>
    </apply>
    <apply>
     <csymbol cd="ambiguous">subscript</csymbol>
     <ci>K</ci>
     <ci>M</ci>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   K_{M}^{\prime}\approx K_{M}
  </annotation>
 </semantics>
</math>

 and 

<math display="inline" id="Enzyme_kinetics:26">
 <semantics>
  <mrow>
   <msub>
    <mi>k</mi>
    <mrow>
     <mi>c</mi>
     <mi>a</mi>
     <mi>t</mi>
    </mrow>
   </msub>
   <mo>≈</mo>
   <msub>
    <mi>k</mi>
    <mn>2</mn>
   </msub>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <approx></approx>
    <apply>
     <csymbol cd="ambiguous">subscript</csymbol>
     <ci>k</ci>
     <apply>
      <times></times>
      <ci>c</ci>
      <ci>a</ci>
      <ci>t</ci>
     </apply>
    </apply>
    <apply>
     <csymbol cd="ambiguous">subscript</csymbol>
     <ci>k</ci>
     <cn type="integer">2</cn>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   k_{cat}\approx k_{2}
  </annotation>
 </semantics>
</math>

.</p>
<h2 id="multi-substrate-reactions">Multi-substrate reactions</h2>

<p>Multi-substrate reactions follow complex rate equations that describe how the substrates bind and in what sequence. The analysis of these reactions is much simpler if the concentration of substrate A is kept constant and substrate B varied. Under these conditions, the enzyme behaves just like a single-substrate enzyme and a plot of <em>v</em> by [S] gives apparent <em>K</em><sub>M</sub> and <em>V</em><sub>max</sub> constants for substrate B. If a set of these measurements is performed at different fixed concentrations of A, these data can be used to work out what the mechanism of the reaction is. For an enzyme that takes two substrates A and B and turns them into two products P and Q, there are two types of mechanism: ternary complex and ping–pong.</p>
<h3 id="ternary-complex-mechanisms">Ternary-complex mechanisms</h3>
<figure><b>(Figure)</b>
<figcaption>Random-order ternary-complex mechanism for an enzyme reaction. The reaction path is shown as a line and enzyme intermediates containing substrates A and B or products P and Q are written below the line.</figcaption>
</figure>

<p>In these enzymes, both substrates bind to the enzyme at the same time to produce an EAB ternary complex. The order of binding can either be random (in a random mechanism) or substrates have to bind in a particular sequence (in an ordered mechanism). When a set of <em>v</em> by [S] curves (fixed A, varying B) from an enzyme with a ternary-complex mechanism are plotted in a <a href="Lineweaver–Burk_plot" title="wikilink">Lineweaver–Burk plot</a>, the set of lines produced will intersect.</p>

<p>Enzymes with ternary-complex mechanisms include <a href="Glutathione_S-transferase" title="wikilink">glutathione <em>S</em>-transferase</a>,<a class="footnoteRef" href="#fn22" id="fnref22"><sup>22</sup></a> <a href="dihydrofolate_reductase" title="wikilink">dihydrofolate reductase</a><a class="footnoteRef" href="#fn23" id="fnref23"><sup>23</sup></a> and <a href="DNA_polymerase" title="wikilink">DNA polymerase</a>.<a class="footnoteRef" href="#fn24" id="fnref24"><sup>24</sup></a> The following links show short animations of the ternary-complex mechanisms of the enzymes dihydrofolate reductase and DNA polymerase.</p>
<h3 id="pingpong-mechanisms">Ping–pong mechanisms</h3>

<p> As shown on the right, enzymes with a ping-pong mechanism can exist in two states, E and a chemically modified form of the enzyme E*; this modified enzyme is known as an <a href="Reactive_intermediate" title="wikilink">intermediate</a>. In such mechanisms, substrate A binds, changes the enzyme to E* by, for example, transferring a chemical group to the active site, and is then released. Only after the first substrate is released can substrate B bind and react with the modified enzyme, regenerating the unmodified E form. When a set of <em>v</em> by [S] curves (fixed A, varying B) from an enzyme with a ping–pong mechanism are plotted in a Lineweaver–Burk plot, a set of parallel lines will be produced. This is called a <a href="Secondary_plot_(kinetics)" title="wikilink">secondary plot</a>.</p>

<p>Enzymes with ping–pong mechanisms include some <a class="uri" href="oxidoreductases" title="wikilink">oxidoreductases</a> such as <a href="peroxidase" title="wikilink">thioredoxin peroxidase</a>,<a class="footnoteRef" href="#fn25" id="fnref25"><sup>25</sup></a> <a class="uri" href="transferases" title="wikilink">transferases</a> such as acylneuraminate cytidylyltransferase<a class="footnoteRef" href="#fn26" id="fnref26"><sup>26</sup></a> and <a href="serine_protease" title="wikilink">serine proteases</a> such as <a class="uri" href="trypsin" title="wikilink">trypsin</a> and <a class="uri" href="chymotrypsin" title="wikilink">chymotrypsin</a>.<a class="footnoteRef" href="#fn27" id="fnref27"><sup>27</sup></a> Serine proteases are a very common and diverse family of enzymes, including <a href="digestion" title="wikilink">digestive</a> enzymes (trypsin, chymotrypsin, and elastase), several enzymes of the <a href="Coagulation" title="wikilink">blood clotting cascade</a> and many others. In these serine proteases, the E* intermediate is an acyl-enzyme species formed by the attack of an active site <a class="uri" href="serine" title="wikilink">serine</a> residue on a <a href="peptide_bond" title="wikilink">peptide bond</a> in a protein substrate. A short animation showing the mechanism of chymotrypsin is linked here.</p>
<h2 id="non-michaelismenten-kinetics">Non-Michaelis–Menten kinetics</h2>

<p> Some enzymes produce a <a href="sigmoid_function" title="wikilink">sigmoid</a> <em>v</em> by [S] plot, which often indicates <a href="cooperative_binding" title="wikilink">cooperative binding</a> of substrate to the active site. This means that the binding of one substrate molecule affects the binding of subsequent substrate molecules. This behavior is most common in <a href="protein_structure" title="wikilink">multimeric</a> enzymes with several interacting active sites.<a class="footnoteRef" href="#fn28" id="fnref28"><sup>28</sup></a> Here, the mechanism of cooperation is similar to that of <a class="uri" href="hemoglobin" title="wikilink">hemoglobin</a>, with binding of substrate to one active site altering the affinity of the other active sites for substrate molecules. Positive cooperativity occurs when binding of the first substrate molecule <em>increases</em> the affinity of the other active sites for substrate. Negative cooperativity occurs when binding of the first substrate <em>decreases</em> the affinity of the enzyme for other substrate molecules.</p>

<p>Allosteric enzymes include mammalian tyrosyl tRNA-synthetase, which shows negative cooperativity,<a class="footnoteRef" href="#fn29" id="fnref29"><sup>29</sup></a> and bacterial <a href="aspartate_transcarbamoylase" title="wikilink">aspartate transcarbamoylase</a><a class="footnoteRef" href="#fn30" id="fnref30"><sup>30</sup></a> and <a class="uri" href="phosphofructokinase" title="wikilink">phosphofructokinase</a>,<a class="footnoteRef" href="#fn31" id="fnref31"><sup>31</sup></a> which show positive cooperativity.</p>

<p>Cooperativity is surprisingly common and can help regulate the responses of enzymes to changes in the concentrations of their substrates. Positive cooperativity makes enzymes much more sensitive to [S] and their activities can show large changes over a narrow range of substrate concentration. Conversely, negative cooperativity makes enzymes insensitive to small changes in [S].</p>

<p>The <a href="Hill_equation_(biochemistry)" title="wikilink">Hill equation (biochemistry)</a><a class="footnoteRef" href="#fn32" id="fnref32"><sup>32</sup></a> is often used to describe the degree of cooperativity quantitatively in non-Michaelis–Menten kinetics. The derived Hill coefficient <em>n</em> measures how much the binding of substrate to one active site affects the binding of substrate to the other active sites. A Hill coefficient of 1 indicates positive <a class="uri" href="cooperativity" title="wikilink">cooperativity</a>.</p>
<h2 id="pre-steady-state-kinetics">Pre-steady-state kinetics</h2>

<p> In the first moment after an enzyme is mixed with substrate, no product has been formed and no <a href="reactive_intermediate" title="wikilink">intermediates</a> exist. The study of the next few milliseconds of the reaction is called Pre-steady-state kinetics also referred to as <a href="Burst_kinetics" title="wikilink">Burst kinetics</a>. Pre-steady-state kinetics is therefore concerned with the formation and consumption of enzyme–substrate intermediates (such as ES or E*) until their <a href="steady_state_(chemistry)" title="wikilink">steady-state concentrations</a> are reached.</p>

<p>This approach was first applied to the hydrolysis reaction catalysed by <a class="uri" href="chymotrypsin" title="wikilink">chymotrypsin</a>.<a class="footnoteRef" href="#fn33" id="fnref33"><sup>33</sup></a> Often, the detection of an intermediate is a vital piece of evidence in investigations of what mechanism an enzyme follows. For example, in the ping–pong mechanisms that are shown above, rapid kinetic measurements can follow the release of product P and measure the formation of the modified enzyme intermediate E*.<a class="footnoteRef" href="#fn34" id="fnref34"><sup>34</sup></a> In the case of chymotrypsin, this intermediate is formed by an attack on the substrate by the <a href="nucleophile" title="wikilink">nucleophilic</a> serine in the active site and the formation of the acyl-enzyme intermediate.</p>

<p>In the figure to the right, the enzyme produces E* rapidly in the first few seconds of the reaction. The rate then slows as steady state is reached. This rapid burst phase of the reaction measures a single turnover of the enzyme. Consequently, the amount of product released in this burst, shown as the intercept on the <em>y</em>-axis of the graph, also gives the amount of functional enzyme which is present in the assay.<a class="footnoteRef" href="#fn35" id="fnref35"><sup>35</sup></a></p>
<h2 id="chemical-mechanism">Chemical mechanism</h2>

<p>An important goal of measuring enzyme kinetics is to determine the chemical mechanism of an enzyme reaction, i.e., the sequence of chemical steps that transform substrate into product. The kinetic approaches discussed above will show at what rates <a href="Reactive_intermediate" title="wikilink">intermediates</a> are formed and inter-converted, but they cannot identify exactly what these intermediates are.</p>

<p>Kinetic measurements taken under various solution conditions or on slightly modified enzymes or substrates often shed light on this chemical mechanism, as they reveal the rate-determining step or intermediates in the reaction. For example, the breaking of a <a href="covalent_bond" title="wikilink">covalent bond</a> to a <a class="uri" href="hydrogen" title="wikilink">hydrogen</a> <a class="uri" href="atom" title="wikilink">atom</a> is a common rate-determining step. Which of the possible hydrogen transfers is rate determining can be shown by measuring the kinetic effects of substituting each hydrogen by <a class="uri" href="deuterium" title="wikilink">deuterium</a>, its stable <a class="uri" href="isotope" title="wikilink">isotope</a>. The rate will change when the critical hydrogen is replaced, due to a primary <a href="kinetic_isotope_effect" title="wikilink">kinetic isotope effect</a>, which occurs because bonds to deuterium are harder to break than bonds to hydrogen.<a class="footnoteRef" href="#fn36" id="fnref36"><sup>36</sup></a> It is also possible to measure similar effects with other isotope substitutions, such as <sup>13</sup>C/<sup>12</sup>C and <sup>18</sup>O/<sup>16</sup>O, but these effects are more subtle.<a class="footnoteRef" href="#fn37" id="fnref37"><sup>37</sup></a></p>

<p>Isotopes can also be used to reveal the fate of various parts of the substrate molecules in the final products. For example, it is sometimes difficult to discern the origin of an <a class="uri" href="oxygen" title="wikilink">oxygen</a> atom in the final product; since it may have come from water or from part of the substrate. This may be determined by systematically substituting oxygen's stable isotope <sup>18</sup>O into the various molecules that participate in the reaction and checking for the isotope in the product.<a class="footnoteRef" href="#fn38" id="fnref38"><sup>38</sup></a> The chemical mechanism can also be elucidated by examining the kinetics and isotope effects under different pH conditions,<a class="footnoteRef" href="#fn39" id="fnref39"><sup>39</sup></a> by altering the metal ions or other bound <a href="Cofactor_(biochemistry)" title="wikilink">cofactors</a>,<a class="footnoteRef" href="#fn40" id="fnref40"><sup>40</sup></a> by <a href="site-directed_mutagenesis" title="wikilink">site-directed mutagenesis</a> of conserved amino acid residues, or by studying the behaviour of the enzyme in the presence of analogues of the substrate(s).<a class="footnoteRef" href="#fn41" id="fnref41"><sup>41</sup></a></p>
<h2 id="enzyme-inhibition-and-activation">Enzyme inhibition and activation</h2>
<figure><b>(Figure)</b>
<figcaption>Kinetic scheme for reversible enzyme inhibitors.</figcaption>
</figure>

<p>Enzyme inhibitors are molecules that reduce or abolish enzyme activity, while enzyme activators are molecules that increase the catalytic rate of enzymes. These interactions can be either <em>reversible</em> (i.e., removal of the inhibitor restores enzyme activity) or <em>irreversible</em> (i.e., the inhibitor permanently inactivates the enzyme).</p>
<h3 id="reversible-inhibitors">Reversible inhibitors</h3>

<p>Traditionally reversible enzyme inhibitors have been classified as competitive, uncompetitive, or non-competitive, according to their effects on <em>K</em><sub>m</sub> and <em>V</em><sub>max</sub>. These different effects result from the inhibitor binding to the enzyme E, to the enzyme–substrate complex ES, or to both, respectively. The division of these classes arises from a problem in their derivation and results in the need to use two different binding constants for one binding event. The binding of an inhibitor and its effect on the enzymatic activity are two distinctly different things, another problem the traditional equations fail to acknowledge. In noncompetitive inhibition the binding of the inhibitor results in 100% inhibition of the enzyme only, and fails to consider the possibility of anything in between.<a class="footnoteRef" href="#fn42" id="fnref42"><sup>42</sup></a> The common form of the inhibitory term also obscures the relationship between the inhibitor binding to the enzyme and its relationship to any other binding term be it the Michaelis–Menten equation or a dose response curve associated with ligand receptor binding. To demonstrate the relationship the following rearrangement can be made:</p>

<p>

<math display="block" id="Enzyme_kinetics:27">
 <semantics>
  <mfrac>
   <msub>
    <mi>V</mi>
    <mi>max</mi>
   </msub>
   <mrow>
    <mn>1</mn>
    <mo>+</mo>
    <mstyle displaystyle="true">
     <mfrac>
      <mrow>
       <mo stretchy="false">[</mo>
       <mi>I</mi>
       <mo stretchy="false">]</mo>
      </mrow>
      <msub>
       <mi>K</mi>
       <mi>i</mi>
      </msub>
     </mfrac>
    </mstyle>
   </mrow>
  </mfrac>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <csymbol cd="latexml">continued-fraction</csymbol>
    <apply>
     <csymbol cd="ambiguous">subscript</csymbol>
     <ci>V</ci>
     <max></max>
    </apply>
    <apply>
     <plus></plus>
     <cn type="integer">1</cn>
     <apply>
      <csymbol cd="latexml">continued-fraction</csymbol>
      <apply>
       <csymbol cd="latexml">delimited-[]</csymbol>
       <ci>I</ci>
      </apply>
      <apply>
       <csymbol cd="ambiguous">subscript</csymbol>
       <ci>K</ci>
       <ci>i</ci>
      </apply>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \cfrac{V_{\max}}{1+\cfrac{[I]}{K_{i}}}
  </annotation>
 </semantics>
</math>

</p>

<p>

<math display="block" id="Enzyme_kinetics:28">
 <semantics>
  <mfrac>
   <msub>
    <mi>V</mi>
    <mi>max</mi>
   </msub>
   <mstyle displaystyle="true">
    <mfrac>
     <mrow>
      <mrow>
       <mo stretchy="false">[</mo>
       <mi>I</mi>
       <mo stretchy="false">]</mo>
      </mrow>
      <mo>+</mo>
      <msub>
       <mi>K</mi>
       <mi>i</mi>
      </msub>
     </mrow>
     <msub>
      <mi>K</mi>
      <mi>i</mi>
     </msub>
    </mfrac>
   </mstyle>
  </mfrac>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <csymbol cd="latexml">continued-fraction</csymbol>
    <apply>
     <csymbol cd="ambiguous">subscript</csymbol>
     <ci>V</ci>
     <max></max>
    </apply>
    <apply>
     <csymbol cd="latexml">continued-fraction</csymbol>
     <apply>
      <plus></plus>
      <apply>
       <csymbol cd="latexml">delimited-[]</csymbol>
       <ci>I</ci>
      </apply>
      <apply>
       <csymbol cd="ambiguous">subscript</csymbol>
       <ci>K</ci>
       <ci>i</ci>
      </apply>
     </apply>
     <apply>
      <csymbol cd="ambiguous">subscript</csymbol>
      <ci>K</ci>
      <ci>i</ci>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \cfrac{V_{\max}}{\cfrac{[I]+K_{i}}{K_{i}}}
  </annotation>
 </semantics>
</math>

</p>

<p>Adding zero to the bottom ([I]-[I])</p>

<p>

<math display="block" id="Enzyme_kinetics:29">
 <semantics>
  <mfrac>
   <msub>
    <mi>V</mi>
    <mi>max</mi>
   </msub>
   <mstyle displaystyle="true">
    <mfrac>
     <mrow>
      <mrow>
       <mo stretchy="false">[</mo>
       <mi>I</mi>
       <mo stretchy="false">]</mo>
      </mrow>
      <mo>+</mo>
      <msub>
       <mi>K</mi>
       <mi>i</mi>
      </msub>
     </mrow>
     <mrow>
      <mrow>
       <mrow>
        <mo stretchy="false">[</mo>
        <mi>I</mi>
        <mo stretchy="false">]</mo>
       </mrow>
       <mo>+</mo>
       <msub>
        <mi>K</mi>
        <mi>i</mi>
       </msub>
      </mrow>
      <mo>-</mo>
      <mrow>
       <mo stretchy="false">[</mo>
       <mi>I</mi>
       <mo stretchy="false">]</mo>
      </mrow>
     </mrow>
    </mfrac>
   </mstyle>
  </mfrac>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <csymbol cd="latexml">continued-fraction</csymbol>
    <apply>
     <csymbol cd="ambiguous">subscript</csymbol>
     <ci>V</ci>
     <max></max>
    </apply>
    <apply>
     <csymbol cd="latexml">continued-fraction</csymbol>
     <apply>
      <plus></plus>
      <apply>
       <csymbol cd="latexml">delimited-[]</csymbol>
       <ci>I</ci>
      </apply>
      <apply>
       <csymbol cd="ambiguous">subscript</csymbol>
       <ci>K</ci>
       <ci>i</ci>
      </apply>
     </apply>
     <apply>
      <minus></minus>
      <apply>
       <plus></plus>
       <apply>
        <csymbol cd="latexml">delimited-[]</csymbol>
        <ci>I</ci>
       </apply>
       <apply>
        <csymbol cd="ambiguous">subscript</csymbol>
        <ci>K</ci>
        <ci>i</ci>
       </apply>
      </apply>
      <apply>
       <csymbol cd="latexml">delimited-[]</csymbol>
       <ci>I</ci>
      </apply>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \cfrac{V_{\max}}{\cfrac{[I]+K_{i}}{[I]+K_{i}-[I]}}
  </annotation>
 </semantics>
</math>

</p>

<p>Dividing by [I]+K<sub>i</sub></p>

<p>

<math display="block" id="Enzyme_kinetics:30">
 <semantics>
  <mfrac>
   <msub>
    <mi>V</mi>
    <mi>max</mi>
   </msub>
   <mstyle displaystyle="true">
    <mfrac>
     <mn>1</mn>
     <mrow>
      <mn>1</mn>
      <mo>-</mo>
      <mstyle displaystyle="true">
       <mfrac>
        <mrow>
         <mo stretchy="false">[</mo>
         <mi>I</mi>
         <mo stretchy="false">]</mo>
        </mrow>
        <mrow>
         <mrow>
          <mo stretchy="false">[</mo>
          <mi>I</mi>
          <mo stretchy="false">]</mo>
         </mrow>
         <mo>+</mo>
         <msub>
          <mi>K</mi>
          <mi>i</mi>
         </msub>
        </mrow>
       </mfrac>
      </mstyle>
     </mrow>
    </mfrac>
   </mstyle>
  </mfrac>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <csymbol cd="latexml">continued-fraction</csymbol>
    <apply>
     <csymbol cd="ambiguous">subscript</csymbol>
     <ci>V</ci>
     <max></max>
    </apply>
    <apply>
     <csymbol cd="latexml">continued-fraction</csymbol>
     <cn type="integer">1</cn>
     <apply>
      <minus></minus>
      <cn type="integer">1</cn>
      <apply>
       <csymbol cd="latexml">continued-fraction</csymbol>
       <apply>
        <csymbol cd="latexml">delimited-[]</csymbol>
        <ci>I</ci>
       </apply>
       <apply>
        <plus></plus>
        <apply>
         <csymbol cd="latexml">delimited-[]</csymbol>
         <ci>I</ci>
        </apply>
        <apply>
         <csymbol cd="ambiguous">subscript</csymbol>
         <ci>K</ci>
         <ci>i</ci>
        </apply>
       </apply>
      </apply>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \cfrac{V_{\max}}{\cfrac{1}{1-\cfrac{[I]}{[I]+K_{i}}}}
  </annotation>
 </semantics>
</math>

</p>

<p>

<math display="block" id="Enzyme_kinetics:31">
 <semantics>
  <mrow>
   <msub>
    <mi>V</mi>
    <mi>max</mi>
   </msub>
   <mo>-</mo>
   <mrow>
    <msub>
     <mi>V</mi>
     <mi>max</mi>
    </msub>
    <mfrac>
     <mrow>
      <mo stretchy="false">[</mo>
      <mi>I</mi>
      <mo stretchy="false">]</mo>
     </mrow>
     <mrow>
      <mrow>
       <mo stretchy="false">[</mo>
       <mi>I</mi>
       <mo stretchy="false">]</mo>
      </mrow>
      <mo>+</mo>
      <msub>
       <mi>K</mi>
       <mi>i</mi>
      </msub>
     </mrow>
    </mfrac>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <minus></minus>
    <apply>
     <csymbol cd="ambiguous">subscript</csymbol>
     <ci>V</ci>
     <max></max>
    </apply>
    <apply>
     <times></times>
     <apply>
      <csymbol cd="ambiguous">subscript</csymbol>
      <ci>V</ci>
      <max></max>
     </apply>
     <apply>
      <csymbol cd="latexml">continued-fraction</csymbol>
      <apply>
       <csymbol cd="latexml">delimited-[]</csymbol>
       <ci>I</ci>
      </apply>
      <apply>
       <plus></plus>
       <apply>
        <csymbol cd="latexml">delimited-[]</csymbol>
        <ci>I</ci>
       </apply>
       <apply>
        <csymbol cd="ambiguous">subscript</csymbol>
        <ci>K</ci>
        <ci>i</ci>
       </apply>
      </apply>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   V_{\max}-V_{\max}\cfrac{[I]}{[I]+K_{i}}
  </annotation>
 </semantics>
</math>

</p>

<p>This notation demonstrates that similar to the Michaelis–Menten equation, where the rate of reaction depends on the percent of the enzyme population interacting with substrate</p>

<p>fraction of the enzyme population bound by substrate</p>

<p>

<math display="block" id="Enzyme_kinetics:32">
 <semantics>
  <mfrac>
   <mrow>
    <mo stretchy="false">[</mo>
    <mi>S</mi>
    <mo stretchy="false">]</mo>
   </mrow>
   <mrow>
    <mrow>
     <mo stretchy="false">[</mo>
     <mi>S</mi>
     <mo stretchy="false">]</mo>
    </mrow>
    <mo>+</mo>
    <msub>
     <mi>K</mi>
     <mi>m</mi>
    </msub>
   </mrow>
  </mfrac>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <csymbol cd="latexml">continued-fraction</csymbol>
    <apply>
     <csymbol cd="latexml">delimited-[]</csymbol>
     <ci>S</ci>
    </apply>
    <apply>
     <plus></plus>
     <apply>
      <csymbol cd="latexml">delimited-[]</csymbol>
      <ci>S</ci>
     </apply>
     <apply>
      <csymbol cd="ambiguous">subscript</csymbol>
      <ci>K</ci>
      <ci>m</ci>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \cfrac{[S]}{[S]+K_{m}}
  </annotation>
 </semantics>
</math>

</p>

<p>fraction of the enzyme population bound by inhibitor</p>

<p>

<math display="block" id="Enzyme_kinetics:33">
 <semantics>
  <mfrac>
   <mrow>
    <mo stretchy="false">[</mo>
    <mi>I</mi>
    <mo stretchy="false">]</mo>
   </mrow>
   <mrow>
    <mrow>
     <mo stretchy="false">[</mo>
     <mi>I</mi>
     <mo stretchy="false">]</mo>
    </mrow>
    <mo>+</mo>
    <msub>
     <mi>K</mi>
     <mi>i</mi>
    </msub>
   </mrow>
  </mfrac>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <csymbol cd="latexml">continued-fraction</csymbol>
    <apply>
     <csymbol cd="latexml">delimited-[]</csymbol>
     <ci>I</ci>
    </apply>
    <apply>
     <plus></plus>
     <apply>
      <csymbol cd="latexml">delimited-[]</csymbol>
      <ci>I</ci>
     </apply>
     <apply>
      <csymbol cd="ambiguous">subscript</csymbol>
      <ci>K</ci>
      <ci>i</ci>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \cfrac{[I]}{[I]+K_{i}}
  </annotation>
 </semantics>
</math>

</p>

<p>the effect of the inhibitor is a result of the percent of the enzyme population interacting with inhibitor. The only problem with this equation in its present form is that it assumes absolute inhibition of the enzyme with inhibitor binding, when in fact there can be a wide range of effects anywhere from 100% inhibition of substrate turn over to just &gt;0%. To account for this the equation can be easily modified to allow for different degrees of inhibition by including a delta <em>V</em><sub>max</sub> term.</p>

<p>

<math display="block" id="Enzyme_kinetics:34">
 <semantics>
  <mrow>
   <msub>
    <mi>V</mi>
    <mi>max</mi>
   </msub>
   <mo>-</mo>
   <mrow>
    <mi mathvariant="normal">Δ</mi>
    <msub>
     <mi>V</mi>
     <mi>max</mi>
    </msub>
    <mfrac>
     <mrow>
      <mo stretchy="false">[</mo>
      <mi>I</mi>
      <mo stretchy="false">]</mo>
     </mrow>
     <mrow>
      <mrow>
       <mo stretchy="false">[</mo>
       <mi>I</mi>
       <mo stretchy="false">]</mo>
      </mrow>
      <mo>+</mo>
      <msub>
       <mi>K</mi>
       <mi>i</mi>
      </msub>
     </mrow>
    </mfrac>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <minus></minus>
    <apply>
     <csymbol cd="ambiguous">subscript</csymbol>
     <ci>V</ci>
     <max></max>
    </apply>
    <apply>
     <times></times>
     <ci>normal-Δ</ci>
     <apply>
      <csymbol cd="ambiguous">subscript</csymbol>
      <ci>V</ci>
      <max></max>
     </apply>
     <apply>
      <csymbol cd="latexml">continued-fraction</csymbol>
      <apply>
       <csymbol cd="latexml">delimited-[]</csymbol>
       <ci>I</ci>
      </apply>
      <apply>
       <plus></plus>
       <apply>
        <csymbol cd="latexml">delimited-[]</csymbol>
        <ci>I</ci>
       </apply>
       <apply>
        <csymbol cd="ambiguous">subscript</csymbol>
        <ci>K</ci>
        <ci>i</ci>
       </apply>
      </apply>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   V_{\max}-\Delta V_{\max}\cfrac{[I]}{[I]+K_{i}}
  </annotation>
 </semantics>
</math>

</p>

<p>or</p>

<p>

<math display="block" id="Enzyme_kinetics:35">
 <semantics>
  <mrow>
   <mrow>
    <msub>
     <mi>V</mi>
     <mi>max</mi>
    </msub>
    <mn>1</mn>
   </mrow>
   <mo>-</mo>
   <mrow>
    <mrow>
     <mo stretchy="false">(</mo>
     <mrow>
      <mrow>
       <msub>
        <mi>V</mi>
        <mi>max</mi>
       </msub>
       <mn>1</mn>
      </mrow>
      <mo>-</mo>
      <mrow>
       <msub>
        <mi>V</mi>
        <mi>max</mi>
       </msub>
       <mn>2</mn>
      </mrow>
     </mrow>
     <mo stretchy="false">)</mo>
    </mrow>
    <mfrac>
     <mrow>
      <mo stretchy="false">[</mo>
      <mi>I</mi>
      <mo stretchy="false">]</mo>
     </mrow>
     <mrow>
      <mrow>
       <mo stretchy="false">[</mo>
       <mi>I</mi>
       <mo stretchy="false">]</mo>
      </mrow>
      <mo>+</mo>
      <msub>
       <mi>K</mi>
       <mi>i</mi>
      </msub>
     </mrow>
    </mfrac>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <minus></minus>
    <apply>
     <times></times>
     <apply>
      <csymbol cd="ambiguous">subscript</csymbol>
      <ci>V</ci>
      <max></max>
     </apply>
     <cn type="integer">1</cn>
    </apply>
    <apply>
     <times></times>
     <apply>
      <minus></minus>
      <apply>
       <times></times>
       <apply>
        <csymbol cd="ambiguous">subscript</csymbol>
        <ci>V</ci>
        <max></max>
       </apply>
       <cn type="integer">1</cn>
      </apply>
      <apply>
       <times></times>
       <apply>
        <csymbol cd="ambiguous">subscript</csymbol>
        <ci>V</ci>
        <max></max>
       </apply>
       <cn type="integer">2</cn>
      </apply>
     </apply>
     <apply>
      <csymbol cd="latexml">continued-fraction</csymbol>
      <apply>
       <csymbol cd="latexml">delimited-[]</csymbol>
       <ci>I</ci>
      </apply>
      <apply>
       <plus></plus>
       <apply>
        <csymbol cd="latexml">delimited-[]</csymbol>
        <ci>I</ci>
       </apply>
       <apply>
        <csymbol cd="ambiguous">subscript</csymbol>
        <ci>K</ci>
        <ci>i</ci>
       </apply>
      </apply>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   V_{\max}1-(V_{\max}1-V_{\max}2)\cfrac{[I]}{[I]+K_{i}}
  </annotation>
 </semantics>
</math>

</p>

<p>This term can then define the residual enzymatic activity present when the inhibitor is interacting with individual enzymes in the population. However the inclusion of this term has the added value of allowing for the possibility of activation if the secondary <em>V</em><sub>max</sub> term turns out to be higher than the initial term. To account for the possibly of activation as well the notation can then be rewritten replacing the inhibitor "I" with a modifier term denoted here as "X".</p>

<p>

<math display="block" id="Enzyme_kinetics:36">
 <semantics>
  <mrow>
   <mrow>
    <msub>
     <mi>V</mi>
     <mi>max</mi>
    </msub>
    <mn>1</mn>
   </mrow>
   <mo>-</mo>
   <mrow>
    <mrow>
     <mo stretchy="false">(</mo>
     <mrow>
      <mrow>
       <msub>
        <mi>V</mi>
        <mi>max</mi>
       </msub>
       <mn>1</mn>
      </mrow>
      <mo>-</mo>
      <mrow>
       <msub>
        <mi>V</mi>
        <mi>max</mi>
       </msub>
       <mn>2</mn>
      </mrow>
     </mrow>
     <mo stretchy="false">)</mo>
    </mrow>
    <mfrac>
     <mrow>
      <mo stretchy="false">[</mo>
      <mi>X</mi>
      <mo stretchy="false">]</mo>
     </mrow>
     <mrow>
      <mrow>
       <mo stretchy="false">[</mo>
       <mi>X</mi>
       <mo stretchy="false">]</mo>
      </mrow>
      <mo>+</mo>
      <msub>
       <mi>K</mi>
       <mi>x</mi>
      </msub>
     </mrow>
    </mfrac>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <minus></minus>
    <apply>
     <times></times>
     <apply>
      <csymbol cd="ambiguous">subscript</csymbol>
      <ci>V</ci>
      <max></max>
     </apply>
     <cn type="integer">1</cn>
    </apply>
    <apply>
     <times></times>
     <apply>
      <minus></minus>
      <apply>
       <times></times>
       <apply>
        <csymbol cd="ambiguous">subscript</csymbol>
        <ci>V</ci>
        <max></max>
       </apply>
       <cn type="integer">1</cn>
      </apply>
      <apply>
       <times></times>
       <apply>
        <csymbol cd="ambiguous">subscript</csymbol>
        <ci>V</ci>
        <max></max>
       </apply>
       <cn type="integer">2</cn>
      </apply>
     </apply>
     <apply>
      <csymbol cd="latexml">continued-fraction</csymbol>
      <apply>
       <csymbol cd="latexml">delimited-[]</csymbol>
       <ci>X</ci>
      </apply>
      <apply>
       <plus></plus>
       <apply>
        <csymbol cd="latexml">delimited-[]</csymbol>
        <ci>X</ci>
       </apply>
       <apply>
        <csymbol cd="ambiguous">subscript</csymbol>
        <ci>K</ci>
        <ci>x</ci>
       </apply>
      </apply>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   V_{\max}1-(V_{\max}1-V_{\max}2)\cfrac{[X]}{[X]+K_{x}}
  </annotation>
 </semantics>
</math>

</p>

<p>While this terminology results in a simplified way of dealing with kinetic effects relating to the maximum velocity of the Michaelis–Menten equation, it highlights potential problems with the term used to describe effects relating to the <em>K</em><sub>m</sub>. The <em>K</em><sub>m</sub> relating to the affinity of the enzyme for the substrate should in most cases relate to potential changes in the binding site of the enzyme which would directly result from enzyme inhibitor interactions. As such a term similar to the one proposed above to modulate <em>V</em><sub>max</sub> should be appropriate in most situations:<a class="footnoteRef" href="#fn43" id="fnref43"><sup>43</sup></a><a class="footnoteRef" href="#fn44" id="fnref44"><sup>44</sup></a></p>

<p>

<math display="block" id="Enzyme_kinetics:37">
 <semantics>
  <mrow>
   <mrow>
    <msub>
     <mi>K</mi>
     <mi>m</mi>
    </msub>
    <mn>1</mn>
   </mrow>
   <mo>-</mo>
   <mrow>
    <mrow>
     <mo stretchy="false">(</mo>
     <mrow>
      <mrow>
       <msub>
        <mi>K</mi>
        <mi>m</mi>
       </msub>
       <mn>1</mn>
      </mrow>
      <mo>-</mo>
      <mrow>
       <msub>
        <mi>K</mi>
        <mi>m</mi>
       </msub>
       <mn>2</mn>
      </mrow>
     </mrow>
     <mo stretchy="false">)</mo>
    </mrow>
    <mfrac>
     <mrow>
      <mo stretchy="false">[</mo>
      <mi>X</mi>
      <mo stretchy="false">]</mo>
     </mrow>
     <mrow>
      <mrow>
       <mo stretchy="false">[</mo>
       <mi>X</mi>
       <mo stretchy="false">]</mo>
      </mrow>
      <mo>+</mo>
      <msub>
       <mi>K</mi>
       <mi>x</mi>
      </msub>
     </mrow>
    </mfrac>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <minus></minus>
    <apply>
     <times></times>
     <apply>
      <csymbol cd="ambiguous">subscript</csymbol>
      <ci>K</ci>
      <ci>m</ci>
     </apply>
     <cn type="integer">1</cn>
    </apply>
    <apply>
     <times></times>
     <apply>
      <minus></minus>
      <apply>
       <times></times>
       <apply>
        <csymbol cd="ambiguous">subscript</csymbol>
        <ci>K</ci>
        <ci>m</ci>
       </apply>
       <cn type="integer">1</cn>
      </apply>
      <apply>
       <times></times>
       <apply>
        <csymbol cd="ambiguous">subscript</csymbol>
        <ci>K</ci>
        <ci>m</ci>
       </apply>
       <cn type="integer">2</cn>
      </apply>
     </apply>
     <apply>
      <csymbol cd="latexml">continued-fraction</csymbol>
      <apply>
       <csymbol cd="latexml">delimited-[]</csymbol>
       <ci>X</ci>
      </apply>
      <apply>
       <plus></plus>
       <apply>
        <csymbol cd="latexml">delimited-[]</csymbol>
        <ci>X</ci>
       </apply>
       <apply>
        <csymbol cd="ambiguous">subscript</csymbol>
        <ci>K</ci>
        <ci>x</ci>
       </apply>
      </apply>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   K_{m}1-(K_{m}1-K_{m}2)\cfrac{[X]}{[X]+K_{x}}
  </annotation>
 </semantics>
</math>

</p>
<h3 id="irreversible-inhibitors">Irreversible inhibitors</h3>

<p>Enzyme inhibitors can also irreversibly inactivate enzymes, usually by covalently modifying active site residues. These reactions, which may be called suicide substrates, follow <a href="exponential_decay" title="wikilink">exponential decay</a> functions and are usually saturable. Below saturation, they follow <a href="reaction_rate" title="wikilink">first order</a> kinetics with respect to inhibitor.</p>
<h2 id="mechanisms-of-catalysis">Mechanisms of catalysis</h2>
<figure><b>(Figure)</b>
<figcaption>The energy variation as a function of <a href="reaction_coordinate" title="wikilink">reaction coordinate</a> shows the stabilisation of the transition state by an enzyme.</figcaption>
</figure>

<p>The favoured model for the enzyme–substrate interaction is the induced fit model.<a class="footnoteRef" href="#fn45" id="fnref45"><sup>45</sup></a> This model proposes that the initial interaction between enzyme and substrate is relatively weak, but that these weak interactions rapidly induce <a href="conformational_change" title="wikilink">conformational changes</a> in the enzyme that strengthen binding. These <a href="Tertiary_structure" title="wikilink">conformational</a> changes also bring catalytic residues in the active site close to the chemical bonds in the substrate that will be altered in the reaction.<a class="footnoteRef" href="#fn46" id="fnref46"><sup>46</sup></a> Conformational changes can be measured using <a href="circular_dichroism" title="wikilink">circular dichroism</a> or <a href="dual_polarisation_interferometry" title="wikilink">dual polarisation interferometry</a>. After binding takes place, one or more mechanisms of catalysis lower the energy of the reaction's <a href="transition_state" title="wikilink">transition state</a> by providing an alternative chemical pathway for the reaction. Mechanisms of catalysis include catalysis by bond strain; by proximity and orientation; by active-site proton donors or acceptors; covalent catalysis and quantum tunnelling.<a class="footnoteRef" href="#fn47" id="fnref47"><sup>47</sup></a><a class="footnoteRef" href="#fn48" id="fnref48"><sup>48</sup></a></p>

<p>Enzyme kinetics cannot prove which modes of catalysis are used by an enzyme. However, some kinetic data can suggest possibilities to be examined by other techniques. For example, a ping–pong mechanism with burst-phase pre-steady-state kinetics would suggest covalent catalysis might be important in this enzyme's mechanism. Alternatively, the observation of a strong pH effect on <em>V</em><sub>max</sub> but not <em>K</em><sub>m</sub> might indicate that a residue in the active site needs to be in a particular <a href="Ionization" title="wikilink">ionisation</a> state for catalysis to occur.</p>
<h2 id="history">History</h2>

<p>In 1902 <a href="Victor_Henri" title="wikilink">Victor Henri</a> proposed a quantitative theory of enzyme kinetics,<a class="footnoteRef" href="#fn49" id="fnref49"><sup>49</sup></a> but at the time the experimental significance of the <a href="pH" title="wikilink">hydrogen ion concentration</a> was not yet recognized. After <a href="S._P._L._Sørensen" title="wikilink">Peter Lauritz Sørensen</a> had defined the logarithmic pH-scale and introduced the concept of <a href="buffer_solution" title="wikilink">buffering</a> in 1909<a class="footnoteRef" href="#fn50" id="fnref50"><sup>50</sup></a> the German chemist <a href="Leonor_Michaelis" title="wikilink">Leonor Michaelis</a> and his Canadian postdoc <a href="Maud_Leonora_Menten" title="wikilink">Maud Leonora Menten</a> repeated Henri's experiments and confirmed his equation, which is now generally referred to as <a href="Michaelis-Menten_kinetics" title="wikilink">Michaelis-Menten kinetics</a> (sometimes also <em>Henri-Michaelis-Menten kinetics</em>).<a class="footnoteRef" href="#fn51" id="fnref51"><sup>51</sup></a> Their work was further developed by <a href="George_Edward_Briggs" title="wikilink">G. E. Briggs</a> and <a href="J._B._S._Haldane" title="wikilink">J. B. S. Haldane</a>, who derived kinetic equations that are still widely considered today a starting point in modeling enzymatic activity.<a class="footnoteRef" href="#fn52" id="fnref52"><sup>52</sup></a></p>

<p>The major contribution of the Henri-Michaelis-Menten approach was to think of enzyme reactions in two stages. In the first, the substrate binds reversibly to the enzyme, forming the enzyme-substrate complex. This is sometimes called the Michaelis complex. The enzyme then catalyzes the chemical step in the reaction and releases the product. The kinetics of many enzymes is adequately described by the simple Michaelis-Menten model, but all enzymes have <a href="protein_dynamics" title="wikilink">internal motions</a> that are not accounted for in the model and can have significant contributions to the overall reaction kinetics. This can be modeled by introducing several Michaelis-Menten pathways that are connected with fluctuating rates,<a class="footnoteRef" href="#fn53" id="fnref53"><sup>53</sup></a><a class="footnoteRef" href="#fn54" id="fnref54"><sup>54</sup></a><a class="footnoteRef" href="#fn55" id="fnref55"><sup>55</sup></a> which is a mathematical extension of the basic Michaelis Menten mechanism.<a class="footnoteRef" href="#fn56" id="fnref56"><sup>56</sup></a></p>
<h2 id="software">Software</h2>
<h3 id="enzo">ENZO</h3>

<p><strong>ENZO</strong> (Enzyme Kinetics) is a graphical interface tool for building kinetic models of enzyme catalyzed reactions. ENZO automatically generates the corresponding differential equations from a stipulated enzyme reaction scheme. These differential equations are processed by a numerical solver and a regression algorithm which fits the coefficients of differential equations to experimentally observed time course curves. ENZO allows rapid evaluation of rival reaction schemes and can be used for routine tests in enzyme kinetics.<a class="footnoteRef" href="#fn57" id="fnref57"><sup>57</sup></a></p>
<h2 id="see-also">See also</h2>
<ul>
<li><a href="Protein_dynamics" title="wikilink">Protein dynamics</a></li>
<li><a href="Diffusion_limited_enzyme" title="wikilink">Diffusion limited enzyme</a></li>
</ul>
<h2 id="footnotes">Footnotes</h2>

<p><strong>α.</strong> <a href="http://cti.itc.virginia.edu/~cmg/Demo/scriptFrame.html">Link: Interactive Michaelis–Menten kinetics tutorial (Java required)</a></p>

<p><strong>β.</strong> <a href="http://chem-faculty.ucsd.edu/kraut/dhfr.html">Link: dihydrofolate reductase mechanism (Gif)</a></p>

<p><strong>γ.</strong> <a href="http://chem-faculty.ucsd.edu/kraut/dNTP.html">Link: DNA polymerase mechanism (Gif)</a></p>

<p><strong>δ.</strong> <a href="http://web.archive.org/web/20070319235224/http://courses.cm.utexas.edu/jrobertus/ch339k/overheads-2/06_21_chymotrypsin.html">Link: Chymotrypsin mechanism (Flash required)</a></p>
<h2 id="references">References</h2>
<h2 id="further-reading">Further reading</h2>

<p><strong>Introductory</strong></p>
<ul>
<li></li>
<li></li>
<li></li>
</ul>

<p><strong>Advanced</strong></p>
<ul>
<li></li>
<li></li>
<li><mtpl></mtpl></li>
<li></li>
<li></li>
</ul>
<h2 id="external-links">External links</h2>
<ul>
<li><a href="http://www.kscience.co.uk/animations/model.swf">Animation of an enzyme assay</a> — Shows effects of manipulating assay conditions</li>
<li><a href="http://www.ebi.ac.uk/thornton-srv/databases/MACiE/">MACiE</a> — A database of enzyme reaction mechanisms</li>
<li><a href="http://us.expasy.org/enzyme/">ENZYME</a> — Expasy enzyme nomenclature database</li>
<li><a href="http://enzo.cmm.ki.si">ENZO</a> — Web application for easy construction and quick testing of kinetic models of enzyme catalyzed reactions.</li>
<li><a href="http://mbs.cbrc.jp/EzCatDB/">ExCatDB</a> — A database of enzyme catalytic mechanisms</li>
<li><a href="http://www.brenda-enzymes.info/">BRENDA</a> — Comprehensive enzyme database, giving substrates, inhibitors and reaction diagrams</li>
<li><a href="http://sabio.h-its.org">SABIO-RK</a> — A database of reaction kinetics</li>
<li><a href="http://chem-faculty.ucsd.edu/kraut/dhfr.html">Joseph Kraut's Research Group, University of California San Diego</a> — Animations of several enzyme reaction mechanisms</li>
<li><a href="http://www.chem.qmul.ac.uk/iubmb/kinetics/">Symbolism and Terminology in Enzyme Kinetics</a> — A comprehensive explanation of concepts and terminology in enzyme kinetics</li>
<li><a href="http://web.archive.org/web/20040612065857/http://orion1.paisley.ac.uk/kinetics/contents.html">An introduction to enzyme kinetics</a> — An accessible set of on-line tutorials on enzyme kinetics</li>
<li><a href="http://www.wiley.com/college/pratt/0471393878/student/animations/enzyme_kinetics/index.html">Enzyme kinetics animated tutorial</a> — An animated tutorial with audio</li>
</ul>

<p>"</p>

<p><a href="Category:Enzyme_kinetics" title="wikilink"> </a> <a class="uri" href="Category:Catalysis" title="wikilink">Category:Catalysis</a></p>
<section class="footnotes">
<hr/>
<ol>
<li id="fn1"><a href="#fnref1">↩</a></li>
<li id="fn2"><a href="#fnref2">↩</a></li>
<li id="fn3"><a href="#fnref3">↩</a></li>
<li id="fn4"><a href="#fnref4">↩</a></li>
<li id="fn5"><a href="#fnref5">↩</a></li>
<li id="fn6"><a href="#fnref6">↩</a></li>
<li id="fn7"><a href="#fnref7">↩</a></li>
<li id="fn8"><a href="#fnref8">↩</a></li>
<li id="fn9">Michaelis L. and Menten M.L. <em>Kinetik der Invertinwirkung</em> Biochem. Z. 1913; 49:333–369 <a href="http://web.lemoyne.edu/~giunta/menten.html">English translation</a> Accessed 6 April 2007<a href="#fnref9">↩</a></li>
<li id="fn10"><a href="#fnref10">↩</a></li>
<li id="fn11">Walsh R, Martin E, Darvesh S. A method to describe enzyme-catalyzed reactions by combining steady state and time course enzyme kinetic parameters. Biochim Biophys Acta. 2010 Jan;1800:1–5<a href="#fnref11">↩</a></li>
<li id="fn12"><a href="#fnref12">↩</a></li>
<li id="fn13"><a href="#fnref13">↩</a></li>
<li id="fn14"><a href="#fnref14">↩</a></li>
<li id="fn15"><a href="#fnref15">↩</a></li>
<li id="fn16"><a href="#fnref16">↩</a></li>
<li id="fn17"><a href="#fnref17">↩</a></li>
<li id="fn18"><a href="#fnref18">↩</a></li>
<li id="fn19"><a href="#fnref19">↩</a></li>
<li id="fn20"><a href="#fnref20">↩</a></li>
<li id="fn21">for a complete derivation, see <a href="commons:File:Enzyme_Kinetics.pdf" title="wikilink">here</a><a href="#fnref21">↩</a></li>
<li id="fn22"><a href="#fnref22">↩</a></li>
<li id="fn23"><a href="#fnref23">↩</a></li>
<li id="fn24"><a href="#fnref24">↩</a></li>
<li id="fn25"><a href="#fnref25">↩</a></li>
<li id="fn26"><a href="#fnref26">↩</a></li>
<li id="fn27"><a href="#fnref27">↩</a></li>
<li id="fn28"><a href="#fnref28">↩</a></li>
<li id="fn29"><a href="#fnref29">↩</a></li>
<li id="fn30"><a href="#fnref30">↩</a></li>
<li id="fn31"><a href="#fnref31">↩</a></li>
<li id="fn32">Hill, A. V. The possible effects of the aggregation of the molecules of haemoglobin on its dissociation curves. <em>J. Physiol. (Lond.)</em>, 1910 40, iv–vii.<a href="#fnref32">↩</a></li>
<li id="fn33"><a href="#fnref33">↩</a></li>
<li id="fn34"><a href="#fnref34">↩</a></li>
<li id="fn35"><a href="#fnref35">↩</a></li>
<li id="fn36"><a href="#fnref36">↩</a></li>
<li id="fn37"><a href="#fnref37">↩</a></li>
<li id="fn38"><a href="#fnref38">↩</a></li>
<li id="fn39"><a href="#fnref39">↩</a></li>
<li id="fn40"><a href="#fnref40">↩</a></li>
<li id="fn41"><a href="#fnref41">↩</a></li>
<li id="fn42"><a href="#fnref42">↩</a></li>
<li id="fn43"><a href="#fnref43">↩</a></li>
<li id="fn44"><a href="#fnref44">↩</a></li>
<li id="fn45"><a href="#fnref45">↩</a></li>
<li id="fn46"><a href="#fnref46">↩</a></li>
<li id="fn47"></li>
<li id="fn48"><a href="#fnref48">↩</a></li>
<li id="fn49"><a href="#fnref49">↩</a></li>
<li id="fn50"><a href="#fnref50">↩</a></li>
<li id="fn51">; <a href="#fnref51">↩</a></li>
<li id="fn52"><a href="#fnref52">↩</a></li>
<li id="fn53"><a href="#fnref53">↩</a></li>
<li id="fn54"><a href="#fnref54">↩</a></li>
<li id="fn55"><a href="#fnref55">↩</a></li>
<li id="fn56"><a href="#fnref56">↩</a></li>
<li id="fn57"> <a href="http://enzo.cmm.ki.si/">ENZO server</a><a href="#fnref57">↩</a></li>
</ol>
</section>
</body>
</html>
