<html lang="en">
<head>
<meta charset="utf-8"/>
<title offset="342">FASTQ format</title>
<script src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_SVG.js" type="text/javascript">
</script>
</head>
<body>
<h1>FASTQ format</h1>
<hr/>

<p><strong>FASTQ format</strong> is a text-based <a href="File_format" title="wikilink">format</a> for storing both a biological sequence (usually <a href="nucleotide_sequence" title="wikilink">nucleotide sequence</a>) and its corresponding quality scores. Both the sequence letter and quality score are each encoded with a single <a class="uri" href="ASCII" title="wikilink">ASCII</a> character for brevity. It was originally developed at the <a href="Wellcome_Trust_Sanger_Institute" title="wikilink">Wellcome Trust Sanger Institute</a> to bundle a <a href="FASTA_format" title="wikilink">FASTA</a> sequence and its quality data, but has recently become the <em>de facto</em> standard for storing the output of high-throughput sequencing instruments such as the <a href="Illumina_(company)" title="wikilink">Illumina</a> Genome Analyzer.<a class="footnoteRef" href="#fn1" id="fnref1"><sup>1</sup></a></p>
<h2 id="format">Format</h2>

<p>A FASTQ file normally uses four lines per sequence.</p>
<ul>
<li>Line 1 begins with a '@' character and is followed by a sequence identifier and an <em>optional</em> description (like a <a href="FASTA_format" title="wikilink">FASTA</a> title line).</li>
<li>Line 2 is the raw sequence letters.</li>
<li>Line 3 begins with a '+' character and is <em>optionally</em> followed by the same sequence identifier (and any description) again.</li>
<li>Line 4 encodes the quality values for the sequence in Line 2, and must contain the same number of symbols as letters in the sequence.</li>
</ul>

<p>A FASTQ file containing a single sequence might look like this:</p>
<pre><code>@SEQ_ID
GATTTGGGGTTCAAAGCAGTATCGATCAAATAGTAAATCCATTTGTTCAACTCACAGTTT
+
!''*((((***+))%%%++)(%%%%).1***-+*''))**55CCF&gt;&gt;&gt;&gt;&gt;&gt;CCCCCCC65</code></pre>

<p>The character '!' represents the lowest quality while '~' is the highest. Here are the quality value characters in left-to-right increasing order of quality (<a class="uri" href="ASCII" title="wikilink">ASCII</a>):</p>
<pre><code> !"#$%&amp;'()*+,-./0123456789:;&lt;=&gt;?@ABCDEFGHIJKLMNOPQRSTUVWXYZ[\]^_`abcdefghijklmnopqrstuvwxyz{|}~</code></pre>

<p>The original Sanger FASTQ files also allowed the sequence and quality strings to be wrapped (split over multiple lines), but this is generally discouraged as it can make parsing complicated due to the unfortunate choice of "@" and "+" as markers (these characters can also occur in the quality string).<a class="footnoteRef" href="#fn2" id="fnref2"><sup>2</sup></a> An example of a tools that break the 4 line convention is vcfutils.pl from samtools.<a class="footnoteRef" href="#fn3" id="fnref3"><sup>3</sup></a></p>
<h3 id="illumina-sequence-identifiers">Illumina sequence identifiers</h3>

<p>Sequences from the <a href="Solexa" title="wikilink">Illumina</a> software use a systematic identifier:</p>
<pre><code>@HWUSI-EAS100R:6:73:941:1973#0/1</code></pre>
<table>
<thead>
<tr class="header">
<th style="text-align: left;">
<p>HWUSI-EAS100R</p></th>
<th style="text-align: left;">
<p>the unique instrument name</p></th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;">
<p>6</p></td>
<td style="text-align: left;">
<p>flowcell lane</p></td>
</tr>
<tr class="even">
<td style="text-align: left;">
<p>73</p></td>
<td style="text-align: left;">
<p>tile number within the flowcell lane</p></td>
</tr>
<tr class="odd">
<td style="text-align: left;">
<p>941</p></td>
<td style="text-align: left;">
<p>'x'-coordinate of the cluster within the tile</p></td>
</tr>
<tr class="even">
<td style="text-align: left;">
<p>1973</p></td>
<td style="text-align: left;">
<p>'y'-coordinate of the cluster within the tile</p></td>
</tr>
<tr class="odd">
<td style="text-align: left;"><ol>
<li>0</li>
</ol></td>
<td style="text-align: left;">
<p>index number for a multiplexed sample (0 for no indexing)</p></td>
</tr>
<tr class="even">
<td style="text-align: left;">
<p>/1</p></td>
<td style="text-align: left;">
<p>the member of a pair, /1 or /2 <em>(paired-end or mate-pair reads only)</em></p></td>
</tr>
</tbody>
</table>

<p>Versions of the Illumina pipeline since 1.4 appear to use <strong>#NNNNNN</strong> instead of <strong>#0</strong> for the multiplex ID, where <strong>NNNNNN</strong> is the sequence of the multiplex tag.</p>

<p>With Casava 1.8 the format of the '@' line has changed:</p>
<pre><code>@EAS139:136:FC706VJ:2:2104:15343:197393 1:Y:18:ATCACG</code></pre>
<table>
<thead>
<tr class="header">
<th style="text-align: left;">
<p>EAS139</p></th>
<th style="text-align: left;">
<p>the unique instrument name</p></th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;">
<p>136</p></td>
<td style="text-align: left;">
<p>the run id</p></td>
</tr>
<tr class="even">
<td style="text-align: left;">
<p>FC706VJ</p></td>
<td style="text-align: left;">
<p>the flowcell id</p></td>
</tr>
<tr class="odd">
<td style="text-align: left;">
<p>2</p></td>
<td style="text-align: left;">
<p>flowcell lane</p></td>
</tr>
<tr class="even">
<td style="text-align: left;">
<p>2104</p></td>
<td style="text-align: left;">
<p>tile number within the flowcell lane</p></td>
</tr>
<tr class="odd">
<td style="text-align: left;">
<p>15343</p></td>
<td style="text-align: left;">
<p>'x'-coordinate of the cluster within the tile</p></td>
</tr>
<tr class="even">
<td style="text-align: left;">
<p>197393</p></td>
<td style="text-align: left;">
<p>'y'-coordinate of the cluster within the tile</p></td>
</tr>
<tr class="odd">
<td style="text-align: left;">
<p>1</p></td>
<td style="text-align: left;">
<p>the member of a pair, 1 or 2 <em>(paired-end or mate-pair reads only)</em></p></td>
</tr>
<tr class="even">
<td style="text-align: left;">
<p>Y</p></td>
<td style="text-align: left;">
<p>Y if the read is filtered, N otherwise</p></td>
</tr>
<tr class="odd">
<td style="text-align: left;">
<p>18</p></td>
<td style="text-align: left;">
<p>0 when none of the control bits are on, otherwise it is an even number</p></td>
</tr>
<tr class="even">
<td style="text-align: left;">
<p>ATCACG</p></td>
<td style="text-align: left;">
<p>index sequence</p></td>
</tr>
</tbody>
</table>
<h3 id="ncbi-sequence-read-archive">NCBI Sequence Read Archive</h3>

<p>FASTQ files from the <a href="National_Center_for_Biotechnology_Information" title="wikilink">NCBI</a>/<a href="European_Bioinformatics_Institute" title="wikilink">EBI</a> <a href="Sequence_Read_Archive" title="wikilink">Sequence Read Archive</a> often include a description, e.g.</p>
<pre><code>@SRR001666.1 071112_SLXA-EAS1_s_7:5:1:817:345 length=36
GGGTGATGGCCGCTGCCGATGGCGTCAAATCCCACC
+SRR001666.1 071112_SLXA-EAS1_s_7:5:1:817:345 length=36
IIIIIIIIIIIIIIIIIIIIIIIIIIIIII9IG9IC</code></pre>

<p>In this example there is an NCBI-assigned identifier, and the description holds the original identifier from <a href="Solexa" title="wikilink">Solexa/Illumina</a> (as described above) plus the read length.</p>

<p>Also note that the NCBI have converted this FASTQ data from the original Solexa/Illumina encoding to the Sanger standard (see encodings below).</p>
<h2 id="variations">Variations</h2>
<h3 id="quality">Quality</h3>

<p>A quality value <em>Q</em> is an integer mapping of <em>p</em> (i.e., the probability that the corresponding base call is incorrect). Two different equations have been in use. The first is the standard Sanger variant to assess reliability of a base call, otherwise known as <a href="Phred_quality_score" title="wikilink">Phred quality score</a>:</p>

<p>

<math display="inline" id="FASTQ_format:0">
 <semantics>
  <mrow>
   <msub>
    <mi>Q</mi>
    <mtext>sanger</mtext>
   </msub>
   <mo>=</mo>
   <mrow>
    <mo>-</mo>
    <mrow>
     <mpadded width="+1.7pt">
      <mn>10</mn>
     </mpadded>
     <mrow>
      <msub>
       <mi>log</mi>
       <mn>10</mn>
      </msub>
      <mi>p</mi>
     </mrow>
    </mrow>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <eq></eq>
    <apply>
     <csymbol cd="ambiguous">subscript</csymbol>
     <ci>Q</ci>
     <mtext>sanger</mtext>
    </apply>
    <apply>
     <minus></minus>
     <apply>
      <times></times>
      <cn type="integer">10</cn>
      <apply>
       <apply>
        <csymbol cd="ambiguous">subscript</csymbol>
        <log></log>
        <cn type="integer">10</cn>
       </apply>
       <ci>p</ci>
      </apply>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   Q_{\text{sanger}}=-10\,\log_{10}p
  </annotation>
 </semantics>
</math>

</p>

<p>The Solexa pipeline (i.e., the software delivered with the Illumina Genome Analyzer) earlier used a different mapping, encoding the <a class="uri" href="odds" title="wikilink">odds</a> <em>p</em>/(1-<em>p</em>) instead of the probability <em>p</em>:</p>

<p>

<math display="inline" id="FASTQ_format:1">
 <semantics>
  <mrow>
   <msub>
    <mi>Q</mi>
    <mtext>solexa-prior to v.1.3</mtext>
   </msub>
   <mo>=</mo>
   <mrow>
    <mo>-</mo>
    <mrow>
     <mpadded width="+1.7pt">
      <mn>10</mn>
     </mpadded>
     <mrow>
      <msub>
       <mi>log</mi>
       <mn>10</mn>
      </msub>
      <mfrac>
       <mi>p</mi>
       <mrow>
        <mn>1</mn>
        <mo>-</mo>
        <mi>p</mi>
       </mrow>
      </mfrac>
     </mrow>
    </mrow>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <eq></eq>
    <apply>
     <csymbol cd="ambiguous">subscript</csymbol>
     <ci>Q</ci>
     <mtext>solexa-prior to v.1.3</mtext>
    </apply>
    <apply>
     <minus></minus>
     <apply>
      <times></times>
      <cn type="integer">10</cn>
      <apply>
       <apply>
        <csymbol cd="ambiguous">subscript</csymbol>
        <log></log>
        <cn type="integer">10</cn>
       </apply>
       <apply>
        <divide></divide>
        <ci>p</ci>
        <apply>
         <minus></minus>
         <cn type="integer">1</cn>
         <ci>p</ci>
        </apply>
       </apply>
      </apply>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   Q_{\text{solexa-prior to v.1.3}}=-10\,\log_{10}\frac{p}{1-p}
  </annotation>
 </semantics>
</math>

</p>

<p>Although both mappings are asymptotically identical at higher quality values, they differ at lower quality levels (i.e., approximately <em>p</em> &gt; 0.05, or equivalently, <em>Q</em> Sequencing Analysis Software User Guide: For Pipeline Version 1.4 and CASAVA Version 1.0, dated April 2009 <a href="http://genomecenter.ucdavis.edu/dna_technologies/documents/pipeline_1_4.pdf">PDF</a> In retrospect, this entry in the manual appears to have been an error. The user guide (What's New, page 5) for version 1.5 of the Illumina pipeline lists this description instead: "Important Changes in Pipeline v1.3 . The quality scoring scheme has changed to the Phred [i.e., Sanger] scoring scheme, encoded as an ASCII character by adding 64 to the Phred value. A Phred score of a base is

<math display="block" id="FASTQ_format:2">
 <semantics>
  <mrow>
   <msub>
    <mi>Q</mi>
    <mtext>phred</mtext>
   </msub>
   <mo>=</mo>
   <mrow>
    <mo>-</mo>
    <mrow>
     <mn>10</mn>
     <mrow>
      <msub>
       <mi>log</mi>
       <mtext>10</mtext>
      </msub>
      <mi>e</mi>
     </mrow>
    </mrow>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <eq></eq>
    <apply>
     <csymbol cd="ambiguous">subscript</csymbol>
     <ci>Q</ci>
     <mtext>phred</mtext>
    </apply>
    <apply>
     <minus></minus>
     <apply>
      <times></times>
      <cn type="integer">10</cn>
      <apply>
       <apply>
        <csymbol cd="ambiguous">subscript</csymbol>
        <log></log>
        <mtext>10</mtext>
       </apply>
       <ci>e</ci>
      </apply>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   Q_{\text{phred}}=-10\log_{\text{10}}e
  </annotation>
 </semantics>
</math>

, where <em>e</em> is the estimated probability of a base being wrong.<a class="footnoteRef" href="#fn4" id="fnref4"><sup>4</sup></a></p>
<h3 id="encoding">Encoding</h3>
<ul>
<li>Sanger format can encode a <a href="Phred_quality_score" title="wikilink">Phred quality score</a> from 0 to 93 using ASCII 33 to 126 (although in raw read data the Phred quality score rarely exceeds 60, higher scores are possible in assemblies or read maps). Also used in SAM format.<a class="footnoteRef" href="#fn5" id="fnref5"><sup>5</sup></a>Sequence/Alignment Map format Version 1.0, dated August 2009 <a href="http://samtools.sourceforge.net/SAM1.pdf">PDF</a> Coming to the end of February 2011, Illumina's newest version (1.8) of their pipeline CASAVA will directly produce fastq in Sanger format, according to the announcement on seqanswers.com forum.<a class="footnoteRef" href="#fn6" id="fnref6"><sup>6</sup></a></li>
<li>Solexa/Illumina 1.0 format can encode a Solexa/Illumina quality score from -5 to 62 using <a class="uri" href="ASCII" title="wikilink">ASCII</a> 59 to 126 (although in raw read data Solexa scores from -5 to 40 only are expected)</li>
<li>Starting with Illumina 1.3 and before Illumina 1.8, the format encoded a <a href="Phred_quality_score" title="wikilink">Phred quality score</a> from 0 to 62 using <a class="uri" href="ASCII" title="wikilink">ASCII</a> 64 to 126 (although in raw read data Phred scores from 0 to 40 only are expected).</li>
<li>Starting in Illumina 1.5 and before Illumina 1.8, the Phred scores 0 to 2 have a slightly different meaning. The values 0 and 1 are no longer used and the value 2, encoded by ASCII 66 "B", is used also at the end of reads as a <em>Read Segment Quality Control Indicator</em>.<a class="footnoteRef" href="#fn7" id="fnref7"><sup>7</sup></a> The Illumina manual<ref>[Using Genome Analyzer</ref></li>
</ul>

<p>Sequencing Control Software, Version 2.6, Catalog # SY-960-2601, Part # 15009921 Rev. A, November 2009]<a href="http://watson.nci.nih.gov/solexa/Using_SCSv2.6_15009921_A.pdf">2</a> (page 30) states the following: <em>If a read ends with a segment of mostly low quality (Q15 or below), then all of the quality values in the segment are replaced with a value of 2 (encoded as the letter B in Illumina's text-based encoding of quality scores)... This Q2 indicator does not predict a specific error rate, but rather indicates that a specific final portion of the read should not be used in further analyses.</em> Also, the quality score encoded as "B" letter may occur internally within reads at least as late as pipeline version 1.6, as shown in the following example:</p>
<pre><code>@HWI-EAS209_0006_FC706VJ:5:58:5894:21141#ATCACG/1
TTAATTGGTAAATAAATCTCCTAATAGCTTAGATNTTACCTTNNNNNNNNNNTAGTTTCTTGAGATTTGTTGGGGGAGACATTTTTGTGATTGCCTTGAT
+HWI-EAS209_0006_FC706VJ:5:58:5894:21141#ATCACG/1
efcfffffcfeefffcffffffddf`feed]`]_Ba_^__[YBBBBBBBBBBRTT\]][]dddd`ddd^dddadd^BBBBBBBBBBBBBBBBBBBBBBBB</code></pre>

<p>An alternative interpretation of this ASCII encoding has been proposed.<a class="footnoteRef" href="#fn8" id="fnref8"><sup>8</sup></a> Also, in Illumina runs using PhiX controls, the character 'B' was observed to represent an "unknown quality score". The error rate of 'B' reads was roughly 3 phred scores lower the mean observed score of a given run.</p>
<ul>
<li>Starting in Illumina 1.8, the quality scores have basically returned to the use of the Sanger format (Phred+33).</li>
</ul>

<p>For raw reads, the range of scores will depend on the technology and the base caller used, but will typically be up to 41 for recent Illumina chemistry. Since the maximum observed quality score was previously only 40, various scripts and tools break when they encounter data with quality values larger than 40. For processed reads, scores may be even higher. For example, quality values of 45 are observed in reads from Illumina's Long Read Sequencing Service (previously Moleculo).</p>

<p><code>  </code><span style="color: purple"><code>SSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSS</code></span><code>.....................................................</code><br/>
<code>  ..........................</code><span style="color: green"><code>XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX</code></span><code>......................</code><br/>
<code>  ...............................</code><span style="color: blue"><code>IIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIII</code></span><code>......................</code><br/>
<code>  .................................</code><span style="color: orange"><strong><code>J</code></strong><code>JJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJ</code></span><code>......................</code><br/>
<code>  </code><span style="color: brown"><code>LLLLLLLLLLLLLLLLLLLLLLLLLLLLLLLLLLLLLLLLLL</code></span><code>....................................................</code><br/>
<code>  !"#$%&amp;'()*+,-./0123456789:;</code><code>?@ABCDEFGHIJKLMNOPQRSTUVWXYZ[\]^_`abcdefghijklmnopqrstuvwxyz{|}~</code><br/>
<code>  |                         |    |        |                              |                     |</code><br/>
<code> 33                        59   64       73                            104                   126</code><br/>
<span style="color: purple"><code>  0........................26...31.......40                                </code></span></p>

<p><span style="color: green"> -5....0........9.............................40 </span> <span style="color: blue"> 0........9.............................40 </span> <span style="color: orange"> 3.....9.............................40 </span> <span style="color: brown"> 0.2......................26...31........41 </span> <span style="color: purple">S - Sanger Phred+33, raw reads typically (0, 40)</span> <span style="color: green">X - Solexa Solexa+64, raw reads typically (-5, 40)</span> <span style="color: blue">I - Illumina 1.3+ Phred+64, raw reads typically (0, 40)</span> <span style="color: orange">J - Illumina 1.5+ Phred+64, raw reads typically (3, 40)</span></p>

<p><code>     with 0=unused, 1=unused, 2=Read Segment Quality Control Indicator (bold) </code><br/>
<code>     (Note: See discussion above).</code></p>

<p><span style="color: brown">L - Illumina 1.8+ Phred+33, raw reads typically (0, 41)</span></p>
<h3 id="color-space">Color space</h3>

<p>For SOLiD data, the sequence is in color space, except the first position. The quality values are those of the Sanger format. Alignment tools differ in their preferred version of the quality values: some include a quality score (set to 0, i.e. '!') for the leading nucleotide, others do not. The sequence read archive includes this quality score.</p>
<h3 id="compression">Compression</h3>

<p>Quality values account for about half of the required disk space in the FASTQ format (before compression), and therefore the compression of the quality values can significantly reduce storage requirements and speed up analysis and transmission of sequencing data. Both lossless and lossy compression are recently being considered in the literature. For example, the algorithm QualComp <a class="footnoteRef" href="#fn9" id="fnref9"><sup>9</sup></a> performs lossy compression with a rate (number of bits per quality value) specified by the user. Based on rate-distortion theory results, it allocates the number of bits so as to minimize the MSE (mean squared error) between the original (uncompressed) and the reconstructed (after compression) quality values. Other algorithms for compression of quality values include SCALCE <a class="footnoteRef" href="#fn10" id="fnref10"><sup>10</sup></a> and Fastqz.<a class="footnoteRef" href="#fn11" id="fnref11"><sup>11</sup></a> Both are lossless compression algorithms that provide an optional controlled lossy transformation approach. For example, SCALCE reduces the alphabet size based on the observation that “neighboring” quality values are similar in general.</p>
<h2 id="file-extension">File extension</h2>

<p>There is no standard <a href="file_extension" title="wikilink">file extension</a> for a FASTQ file, but .fq and .fastq, are commonly used.</p>
<h2 id="format-converters">Format converters</h2>
<ul>
<li><a href="BioPython" title="wikilink">Biopython</a> version 1.51 onwards (interconverts Sanger, Solexa and Illumina 1.3+)</li>
<li><a class="uri" href="EMBOSS" title="wikilink">EMBOSS</a> version 6.1.0 patch 1 onwards (interconverts Sanger, Solexa and Illumina 1.3+)</li>
<li><a class="uri" href="BioPerl" title="wikilink">BioPerl</a> version 1.6.1 onwards (interconverts Sanger, Solexa and Illumina 1.3+)</li>
<li><a class="uri" href="BioRuby" title="wikilink">BioRuby</a> version 1.4.0 onwards (interconverts Sanger, Solexa and Illumina 1.3+)</li>
<li><a class="uri" href="BioJava" title="wikilink">BioJava</a> version 1.7.1 onwards (interconverts Sanger, Solexa and Illumina 1.3+)</li>
<li><a href="http://maq.sourceforge.net/">MAQ</a> can convert from Solexa to Sanger (use this [<a class="uri" href="http://sourceforge.net/tracker/index.php?func=detail&amp;aid">http://sourceforge.net/tracker/index.php?func=detail&amp;aid;</a>;=2824334&amp;group;_id=191815&amp;atid;=938893 patch] to support Illumina 1.3+ files).</li>
<li><a href="http://hannonlab.cshl.edu/fastx_toolkit/">fastx_toolkit</a> The included fastq_quality_converter program can convert Illumina to Sanger</li>
</ul>
<h3 id="command-line-conversions">Command line conversions</h3>

<p><strong>FASTQ to FASTA format:</strong></p>
<pre><code>zcat input_file.fastq.gz | awk 'NR%4==1{printf "&gt;%s\n", substr($0,2)}NR%4==2{print}' &gt; output_file.fa</code></pre>

<p><strong>Illumina FASTQ 1.8 to 1.3</strong></p>
<pre><code>sed -e '4~4y/!"#$%&amp;'\''()*+,-.\/0123456789:;&lt;=&gt;?@ABCDEFGHIJ/@ABCDEFGHIJKLMNOPQRSTUVWXYZ[\\]^_`abcdefghi/' myfile.fastq   # add -i to save the result to the same input file</code></pre>

<p><strong>Illumina FASTQ 1.3 to 1.8</strong></p>
<pre><code>sed -e '4~4y/@ABCDEFGHIJKLMNOPQRSTUVWXYZ[\\]^_`abcdefghi/!"#$%&amp;'\''()*+,-.\/0123456789:;&lt;=&gt;?@ABCDEFGHIJ/' myfile.fastq   # add -i to save the result to the same input file</code></pre>
<h2 id="see-also">See also</h2>
<ul>
<li><a href="FASTA_format" title="wikilink">FASTA format</a></li>
<li><a href="Phred_quality_score" title="wikilink">Phred quality score</a></li>
<li><a href="List_of_file_formats#Biology" title="wikilink">List of file formats for molecular biology</a></li>
</ul>
<h2 id="references">References</h2>
<h2 id="external-links">External links</h2>
<ul>
<li><a href="http://maq.sourceforge.net/fastq.shtml">MAQ</a> webpage discussing FASTQ variants</li>
<li><a href="http://hannonlab.cshl.edu/fastx_toolkit/">Fastx toolkit</a> collection of command line tools for Short-Reads FASTA/FASTQ files preprocessing</li>
<li><a href="http://www.bioinformatics.bbsrc.ac.uk/projects/fastqc/">Fastqc</a> quality control tool for high throughput sequence data</li>
<li><a href="http://prinseq.sourceforge.net/">PRINSEQ</a> can be used for QC and to filter, reformat, or trim sequence data (web-based and command line versions)</li>
</ul>

<p>"</p>

<p><a class="uri" href="Category:Bioinformatics" title="wikilink">Category:Bioinformatics</a> <a href="Category:Biological_sequence_format" title="wikilink">Category:Biological sequence format</a></p>
<section class="footnotes">
<hr/>
<ol>
<li id="fn1"><a href="#fnref1">↩</a></li>
<li id="fn2"></li>
<li id="fn3"><a href="http://cnr.lwlss.net/SeqConsensus/">accessed 14 March 2014</a><a href="#fnref3">↩</a></li>
<li id="fn4">Sequencing Analysis Software User Guide: For Pipeline Version 1.5 and CASAVA Version 1.0, dated August 2009 <a href="http://illumina.ucr.edu/illumina_docs/Pipeline1.5/Pipeline1.5_CASAVA1.0_User_Guide_15006500_A.pdf">PDF</a><a href="#fnref4">↩</a></li>
<li id="fn5"></li>
<li id="fn6">Seqanswer's topic of skruglyak, dated January 2011 [<a class="uri" href="http://seqanswers.com/forums/showthread.php?s=ba8c7dfba863815f637c0bf45882f14b&amp;t">http://seqanswers.com/forums/showthread.php?s=ba8c7dfba863815f637c0bf45882f14b&amp;t;</a>;=8895 website]<a href="#fnref6">↩</a></li>
<li id="fn7">Illumina Quality Scores, Tobias Mann, Bioinformatics, San Diego, Illumina <a href="http://seqanswers.com/forums/showthread.php?t=4721">1</a><a href="#fnref7">↩</a></li>
<li id="fn8"><a href="http://solexaqa.sourceforge.net/questions.htm#illumina">SolexaQA project website</a><a href="#fnref8">↩</a></li>
<li id="fn9">Ochoa, Idoia, et al. "QualComp: a new lossy compressor for quality scores based on rate distortion theory." BMC bioinformatics 14.1 (2013): 187. <a class="uri" href="http://www.biomedcentral.com/1471-2105/14/187/">http://www.biomedcentral.com/1471-2105/14/187/</a><a href="#fnref9">↩</a></li>
<li id="fn10">Hach F, Numanagi ́c I, Alkan C, Sahinalp SC:SCALCE: boosting sequencecompression algorithms using locally consistent encoding.Bioinformatics2012,28(23):3051–3057.<a href="#fnref10">↩</a></li>
<li id="fn11">fastqz.http://mattmahoney.net/dc/fastqz/<a href="#fnref11">↩</a></li>
</ol>
</section>
</body>
</html>
