<html lang="en">
<head>
<meta charset="utf-8"/>
<title offset="1282">Formaldehyde dehydrogenase</title>
<script src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_SVG.js" type="text/javascript">
</script>
</head>
<body>
<h1>Formaldehyde dehydrogenase</h1>
<hr/>

<p>In <a class="uri" href="enzymology" title="wikilink">enzymology</a>, a <strong>formaldehyde dehydrogenase</strong> () is an <a class="uri" href="enzyme" title="wikilink">enzyme</a> that <a href="catalysis" title="wikilink">catalyzes</a> the <a href="chemical_reaction" title="wikilink">chemical reaction</a></p>
<dl>
<dd>formaldehyde + NAD<sup>+</sup> + H<sub>2</sub>O 

<math display="inline" id="Formaldehyde_dehydrogenase:0">
 <semantics>
  <mo>⇌</mo>
  <annotation-xml encoding="MathML-Content">
   <ci>normal-⇌</ci>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \rightleftharpoons
  </annotation>
 </semantics>
</math>

 formate + NADH + H<sup>+</sup>
</dd>
</dl>

<p>The 3 <a href="substrate_(biochemistry)" title="wikilink">substrates</a> of this enzyme are <a class="uri" href="formaldehyde" title="wikilink">formaldehyde</a>, <a href="nicotinamide_adenine_dinucleotide" title="wikilink">NAD<sup>+</sup></a>, and <a href="water" title="wikilink">H<sub>2</sub>O</a>, whereas its 3 <a href="product_(chemistry)" title="wikilink">products</a> are <a class="uri" href="formate" title="wikilink">formate</a>, <a href="nicotinamide_adenine_dinucleotide" title="wikilink">NADH</a>, and <a href="hydrogen_ion" title="wikilink">H<sup>+</sup></a>.</p>

<p>This enzyme belongs to the family of <a href="oxidoreductase" title="wikilink">oxidoreductases</a>, specifically those acting on the aldehyde or oxo group of donor with NAD+ or NADP+ as acceptor. The systematic name of this enzyme class is <strong>formaldehyde:NAD+ oxidoreductase</strong>. Other names in common use include <strong>NAD+-linked formaldehyde dehydrogenase</strong>, <strong>s-nitrosoglutathione reductase</strong> (GSNO reductase) and <strong>NAD+-dependent formaldehyde dehydrogenase</strong>. This enzyme participates in <a href="methane_metabolism" title="wikilink">methane metabolism</a>.</p>
<h2 id="ubiquitous-function">Ubiquitous function</h2>

<p>S-nitrosoglutathione reductase (GSNOR) is a class III alcohol dehydrogenase (ADH) encoded by the ADH5 gene in humans. It is a primordial ADH that is ubiquitously expressed in plant and animals alike. GSNOR reduces <a class="uri" href="S-nitrosoglutathione" title="wikilink">S-nitrosoglutathione</a> (GSNO) to the unstable intermediate, S-hydroxylaminoglutathione, which then rearranges to form glutathione sulfinamide, or in the presence of GSH, forms oxidized glutathione (GSSG) and hydroxyl amine.<a class="footnoteRef" href="#fn1" id="fnref1"><sup>1</sup></a><a class="footnoteRef" href="#fn2" id="fnref2"><sup>2</sup></a><a class="footnoteRef" href="#fn3" id="fnref3"><sup>3</sup></a> Through this catabolic process, GSNOR regulates the cellular concentrations of GSNO and plays a central role in regulating the levels of endogenous S-nitrosothiols and controlling protein S-nitrosylation-based signaling. As an example of S-nitrosylation-based signaling, Barglow et al. showed that GSNO selectively S-nitrosylates reduced thioredoxin at cysteine 62.<a class="footnoteRef" href="#fn4" id="fnref4"><sup>4</sup></a> Nitrosylated thioredoxin, via directed protein-protein interaction, trans-nitrosylates the active site cysteine of caspase-3 thus inactivating caspase-3 and preventing induction of apoptosis.<a class="footnoteRef" href="#fn5" id="fnref5"><sup>5</sup></a></p>

<p>As might be expected of an enzyme involved in regulating NO levels and signaling, pleiotropic effects are observed in GSNOR knockout models. Deleting the GSNOR gene from both yeast and mice increased the cellular levels of GSNO and nitrosylated proteins, and the yeast cells showed increased susceptibility to nitrosative stress.<a class="footnoteRef" href="#fn6" id="fnref6"><sup>6</sup></a> Null mice show increased levels of S-nitrosated proteins, increased beta adrenergic receptor numbers in lung and heart,<a class="footnoteRef" href="#fn7" id="fnref7"><sup>7</sup></a> diminished tachyphylaxis to β2-adrenergic receptor agonists, hyporesponsiveness to methacholine and allergen challenge and reduced infarct size after occlusion of the coronary artery.<a class="footnoteRef" href="#fn8" id="fnref8"><sup>8</sup></a><a class="footnoteRef" href="#fn9" id="fnref9"><sup>9</sup></a> In addition, null mice show increased tissue damage and mortality following challenge with bacteria or endotoxin and are hypotensive under anesthesia yet normotensive in the conscious state.<a class="footnoteRef" href="#fn10" id="fnref10"><sup>10</sup></a> More related to its alcohol dehydrogenase activity, GSNOR null mice show a 30% reduction in the LD50 for formaldehyde and a decreased capacity to metabolize retinol, although it is clear from these studies that other pathways exist for the metabolism of these compounds.<a class="footnoteRef" href="#fn11" id="fnref11"><sup>11</sup></a><a class="footnoteRef" href="#fn12" id="fnref12"><sup>12</sup></a></p>
<h2 id="role-in-disease">Role in disease</h2>

<p>It has been shown that GSNOR may have an important role in respiratory diseases such as asthma. GSNOR expression has been inversely correlated with <a class="uri" href="s-nitrosothiol" title="wikilink">s-nitrosothiol</a> (SNO) levels in the alveolar lining fluid in the lung and with responsiveness to methacholine challenge in patients with mild asthma compared to normal subjects.<a class="footnoteRef" href="#fn13" id="fnref13"><sup>13</sup></a> Furthermore, there are lowered SNOs in tracheal irrigations in asthmatic children with respiratory failure in comparison to normal children undergoing elective surgery and NO species are elevated in asthma patients when exposed to antigen.<a class="footnoteRef" href="#fn14" id="fnref14"><sup>14</sup></a></p>

<p>Assessing the gene expression of the ADHs in nonalcoholic steatohepatitis (NASH) patients has shown elevated levels of all ADHs, but primarily ADH1 and ADH4 (up to 40-fold increased). ADH5 showed an ~4-fold increase in gene expression.<a class="footnoteRef" href="#fn15" id="fnref15"><sup>15</sup></a></p>
<h2 id="structural-studies">Structural studies</h2>

<p>As of late 2007, only one <a href="tertiary_structure" title="wikilink">structure</a> has been solved for this class of enzymes, with the <a href="Protein_Data_Bank" title="wikilink">PDB</a> accession code .</p>
<h2 id="references">References</h2>
<h2 id="further-reading">Further reading</h2>
<ul>
<li></li>
</ul>

<p>"</p>

<p><a href="Category:EC_1.2.1" title="wikilink">Category:EC 1.2.1</a> <a href="Category:NADH-dependent_enzymes" title="wikilink">Category:NADH-dependent enzymes</a> <a href="Category:Enzymes_of_known_structure" title="wikilink">Category:Enzymes of known structure</a></p>
<section class="footnotes">
<hr/>
<ol>
<li id="fn1"><a href="#fnref1">↩</a></li>
<li id="fn2"><a href="#fnref2">↩</a></li>
<li id="fn3"><a href="#fnref3">↩</a></li>
<li id="fn4"><a href="#fnref4">↩</a></li>
<li id="fn5"><a href="#fnref5">↩</a></li>
<li id="fn6"><a href="#fnref6">↩</a></li>
<li id="fn7"><a href="#fnref7">↩</a></li>
<li id="fn8"><a href="#fnref8">↩</a></li>
<li id="fn9"><a href="#fnref9">↩</a></li>
<li id="fn10"><a href="#fnref10">↩</a></li>
<li id="fn11"><a href="#fnref11">↩</a></li>
<li id="fn12"><a href="#fnref12">↩</a></li>
<li id="fn13"><a href="#fnref13">↩</a></li>
<li id="fn14"><a href="#fnref14">↩</a></li>
<li id="fn15"><a href="#fnref15">↩</a></li>
</ol>
</section>
</body>
</html>
