<html lang="en">
<head>
<meta charset="utf-8"/>
<title offset="1401">Gaussian network model</title>
<script src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_SVG.js" type="text/javascript">
</script>
</head>
<body>
<h1>Gaussian network model</h1>
<hr/>

<p>The <strong>Gaussian network model (GNM)</strong> is a representation of a biological <a class="uri" href="macromolecule" title="wikilink">macromolecule</a> as an elastic mass-and-<a href="spring_(device)" title="wikilink">spring</a> network to study, understand, and characterize mechanical aspects of its long-scale <a href="dynamics_(mechanics)" title="wikilink">dynamics</a>. The model has a wide range of applications from small proteins such as enzymes composed of a single <a href="protein_domain" title="wikilink">domain</a>, to large <a href="Macromolecular_Assembly" title="wikilink">macromolecular assemblies</a> such as a <a class="uri" href="ribosome" title="wikilink">ribosome</a> or a viral <a class="uri" href="capsid" title="wikilink">capsid</a>.</p>

<p>The Gaussian network model is a minimalist, coarse-grained approach to study biological molecules. In the model, proteins are represented by nodes corresponding to alpha carbons of the amino acid residues. Similarly, DNA and RNA structures are represented with one to three nodes for each <a class="uri" href="nucleotide" title="wikilink">nucleotide</a>. The model uses the harmonic approximation to model interactions, i.e. the spatial interactions between nodes (amino acids or nucleotides) are modeled with a uniform harmonic spring. This coarse-grained representation makes the calculations computationally inexpensive.</p>

<p>At molecular level, many biological phenomena, such as catalytic activity of an <a class="uri" href="enzyme" title="wikilink">enzyme</a>, occur within the range of nano- to millisecond timescales. All atom simulation techniques, such as <a href="molecular_dynamics" title="wikilink">molecular dynamics</a>, rarely reach microsecond trajectory length, depending on the size of the system and accessible computational resources. Normal mode analysis in the context of GNM or elastic network (EN) models, in general, provides insights on the longer-scale functional behaviors of macromolecules. Here, the model captures native state functional motions of a biomolecule in the cost of atomic detail. The inference obtained from this model is complementary to atomic detail simulation techniques.</p>

<p>Another model for protein dynamics based on elastic mass-and-spring networks is the <a href="Anisotropic_Network_Model" title="wikilink">Anisotropic Network Model</a>.</p>
<h2 id="gaussian-network-model-theory">Gaussian network model theory</h2>
<figure><b>(Figure)</b>
<figcaption><strong>Figure 2:</strong> Schematic representation of nodes in elastic network of GNM. Every node is connected to its spatial neighbors by uniform springs. Distance vector between two nodes, <strong>i</strong> and <strong>j</strong>, is shown by an arrow and labeled <strong>R</strong><sub>ij</sub>. Equilibrium positions of the <strong>i</strong>th and <strong>j</strong>th nodes, <strong>R</strong><sup>0</sup><sub>i</sub> and <strong>R</strong><sup>0</sup><sub>j</sub>, are shown in xyz coordinate system. <strong>R</strong><sup>0</sup><sub>ij</sub> is the equilibrium distance between nodes <strong>i</strong> and <strong>j</strong>. Instantaneous fluctuation vectors, <strong>ΔR</strong><sub>i</sub> and <strong>ΔR</strong><sub>j</sub>, and instantaneous distance vector, <strong>R</strong><sub>ij</sub>, are shown by the dashed arrows.|400px</figcaption>
</figure>

<p>The Gaussian network model was first proposed in 1996 by Tirion at the atomic level<a class="footnoteRef" href="#fn1" id="fnref1"><sup>1</sup></a> and then one year later reconsidered at the amino-acid level by Bahar, Atilgan, Haliloglu and Erman.<a class="footnoteRef" href="#fn2" id="fnref2"><sup>2</sup></a><a class="footnoteRef" href="#fn3" id="fnref3"><sup>3</sup></a> The model was influenced by work of PJ Flory on polymer networks <a class="footnoteRef" href="#fn4" id="fnref4"><sup>4</sup></a> and other works that utilized normal mode analysis and simplified harmonic potentials to study dynamics of proteins.<a class="footnoteRef" href="#fn5" id="fnref5"><sup>5</sup></a></p>
<h3 id="the-elastic-network">The elastic network</h3>

<p>Figure 2 shows a schematic view of elastic network studied in GNM. Metal beads represent the nodes in this Gaussian network (residues of a protein) and springs represent the connections between the nodes of this network (covalent and non-covalent interactions between residues). For nodes <strong>i</strong> and <strong>j</strong>, equilibrium position vectors, <strong>R</strong><sup>0</sup><sub>i</sub> and <strong>R</strong><sup>0</sup><sub>j</sub>, equilibrium distance vector, <strong>R</strong><sup>0</sup><sub>ij</sub>, instantaneous fluctuation vectors, <strong>ΔR</strong><sub>i</sub> and <strong>ΔR</strong><sub>j</sub>, and instantaneous distance vector, <strong>R</strong><sub>ij</sub>, are shown in Figure 2. Instantaneous position vectors of these nodes are defined by <strong>R</strong><sub>i</sub> and <strong>R</strong><sub>j</sub>. The difference between equilibrium position vector and instantaneous position vector of residue <strong>i</strong> gives the instantaneous fluctuation vector, <strong>ΔR</strong><sub>i</sub> = <strong>R</strong><sub>i</sub> - <strong>R</strong><sup>0</sup><sub>i</sub>. Hence, the instantaneous fluctuation vector between nodes <strong>i</strong> and <strong>j</strong> is expressed as <strong>ΔR</strong><sub>ij</sub> = <strong>ΔR</strong><sub>j</sub> - <strong>ΔR</strong><sub>i</sub> = <strong>R</strong><sub>ij</sub> - <strong>R</strong><sup>0</sup><sub>ij</sub>.</p>
<h3 id="potential-of-the-gaussian-network">Potential of the Gaussian network</h3>

<p>Using the harmonic potential approximation, potential energy of the network in terms of <strong>ΔR</strong><sub>i</sub> is</p>

<p>

<math display="block" id="Gaussian_network_model:0">
 <semantics>
  <mrow>
   <msub>
    <mi>V</mi>
    <mrow>
     <mi>G</mi>
     <mi>N</mi>
     <mi>M</mi>
    </mrow>
   </msub>
   <mo>=</mo>
   <mrow>
    <mfrac>
     <mi>γ</mi>
     <mn>2</mn>
    </mfrac>
    <mrow>
     <mo>[</mo>
     <mrow>
      <munderover>
       <mo largeop="true" movablelimits="false" symmetric="true">∑</mo>
       <mrow>
        <mi>i</mi>
        <mo>,</mo>
        <mi>j</mi>
       </mrow>
       <mi>N</mi>
      </munderover>
      <msup>
       <mrow>
        <mo stretchy="false">(</mo>
        <mrow>
         <mrow>
          <mi mathvariant="normal">Δ</mi>
          <msub>
           <mi>R</mi>
           <mi>j</mi>
          </msub>
         </mrow>
         <mo>-</mo>
         <mrow>
          <mi mathvariant="normal">Δ</mi>
          <msub>
           <mi>R</mi>
           <mi>i</mi>
          </msub>
         </mrow>
        </mrow>
        <mo stretchy="false">)</mo>
       </mrow>
       <mn>2</mn>
      </msup>
     </mrow>
     <mo>]</mo>
    </mrow>
   </mrow>
   <mo>=</mo>
   <mrow>
    <mfrac>
     <mi>γ</mi>
     <mn>2</mn>
    </mfrac>
    <mrow>
     <mo>[</mo>
     <mrow>
      <munderover>
       <mo largeop="true" movablelimits="false" symmetric="true">∑</mo>
       <mrow>
        <mi>i</mi>
        <mo>,</mo>
        <mi>j</mi>
       </mrow>
       <mi>N</mi>
      </munderover>
      <mrow>
       <mi mathvariant="normal">Δ</mi>
       <msub>
        <mi>R</mi>
        <mi>i</mi>
       </msub>
       <msub>
        <mi mathvariant="normal">Γ</mi>
        <mrow>
         <mi>i</mi>
         <mi>j</mi>
        </mrow>
       </msub>
       <mi mathvariant="normal">Δ</mi>
       <msub>
        <mi>R</mi>
        <mi>j</mi>
       </msub>
      </mrow>
     </mrow>
     <mo>]</mo>
    </mrow>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <and></and>
    <apply>
     <eq></eq>
     <apply>
      <csymbol cd="ambiguous">subscript</csymbol>
      <ci>V</ci>
      <apply>
       <times></times>
       <ci>G</ci>
       <ci>N</ci>
       <ci>M</ci>
      </apply>
     </apply>
     <apply>
      <times></times>
      <apply>
       <divide></divide>
       <ci>γ</ci>
       <cn type="integer">2</cn>
      </apply>
      <apply>
       <csymbol cd="latexml">delimited-[]</csymbol>
       <apply>
        <apply>
         <csymbol cd="ambiguous">superscript</csymbol>
         <apply>
          <csymbol cd="ambiguous">subscript</csymbol>
          <sum></sum>
          <list>
           <ci>i</ci>
           <ci>j</ci>
          </list>
         </apply>
         <ci>N</ci>
        </apply>
        <apply>
         <csymbol cd="ambiguous">superscript</csymbol>
         <apply>
          <minus></minus>
          <apply>
           <times></times>
           <ci>normal-Δ</ci>
           <apply>
            <csymbol cd="ambiguous">subscript</csymbol>
            <ci>R</ci>
            <ci>j</ci>
           </apply>
          </apply>
          <apply>
           <times></times>
           <ci>normal-Δ</ci>
           <apply>
            <csymbol cd="ambiguous">subscript</csymbol>
            <ci>R</ci>
            <ci>i</ci>
           </apply>
          </apply>
         </apply>
         <cn type="integer">2</cn>
        </apply>
       </apply>
      </apply>
     </apply>
    </apply>
    <apply>
     <eq></eq>
     <share href="#.cmml">
     </share>
     <apply>
      <times></times>
      <apply>
       <divide></divide>
       <ci>γ</ci>
       <cn type="integer">2</cn>
      </apply>
      <apply>
       <csymbol cd="latexml">delimited-[]</csymbol>
       <apply>
        <apply>
         <csymbol cd="ambiguous">superscript</csymbol>
         <apply>
          <csymbol cd="ambiguous">subscript</csymbol>
          <sum></sum>
          <list>
           <ci>i</ci>
           <ci>j</ci>
          </list>
         </apply>
         <ci>N</ci>
        </apply>
        <apply>
         <times></times>
         <ci>normal-Δ</ci>
         <apply>
          <csymbol cd="ambiguous">subscript</csymbol>
          <ci>R</ci>
          <ci>i</ci>
         </apply>
         <apply>
          <csymbol cd="ambiguous">subscript</csymbol>
          <ci>normal-Γ</ci>
          <apply>
           <times></times>
           <ci>i</ci>
           <ci>j</ci>
          </apply>
         </apply>
         <ci>normal-Δ</ci>
         <apply>
          <csymbol cd="ambiguous">subscript</csymbol>
          <ci>R</ci>
          <ci>j</ci>
         </apply>
        </apply>
       </apply>
      </apply>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   V_{GNM}=\frac{\gamma}{2}\left[\sum_{i,j}^{N}(\Delta R_{j}-\Delta R_{i})^{2}%
\right]=\frac{\gamma}{2}\left[\sum_{i,j}^{N}\Delta R_{i}\Gamma_{ij}\Delta R_{j%
}\right]
  </annotation>
 </semantics>
</math>

</p>

<p>where <strong>γ</strong> is a force constant uniform for all springs and <strong>Γ</strong><sub>ij</sub> is the <strong>ij</strong>th element of the <a href="Kirchhoff_matrix" title="wikilink">Kirchhoff</a> (or connectivity) matrix of inter-residue contacts, <strong>Γ</strong>, defined by</p>

<p>

<math display="block" id="Gaussian_network_model:1">
 <semantics>
  <mrow>
   <msub>
    <mi mathvariant="normal">Γ</mi>
    <mrow>
     <mi>i</mi>
     <mi>j</mi>
    </mrow>
   </msub>
   <mo>=</mo>
   <mrow>
    <mo>{</mo>
    <mtable displaystyle="true">
     <mtr>
      <mtd columnalign="center">
       <mrow>
        <mrow>
         <mo>-</mo>
         <mn>1</mn>
        </mrow>
        <mo>,</mo>
       </mrow>
      </mtd>
      <mtd columnalign="center">
       <mrow>
        <mrow>
         <mtext>if</mtext>
         <mi>i</mi>
        </mrow>
        <mo>≠</mo>
        <mi>j</mi>
       </mrow>
      </mtd>
      <mtd columnalign="center">
       <mrow>
        <mrow>
         <mtext>and</mtext>
         <msub>
          <mi>R</mi>
          <mrow>
           <mi>i</mi>
           <mi>j</mi>
          </mrow>
         </msub>
        </mrow>
        <mo>≤</mo>
        <msub>
         <mi>r</mi>
         <mi>c</mi>
        </msub>
       </mrow>
      </mtd>
     </mtr>
     <mtr>
      <mtd columnalign="center">
       <mrow>
        <mn>0</mn>
        <mo>,</mo>
       </mrow>
      </mtd>
      <mtd columnalign="center">
       <mrow>
        <mrow>
         <mtext>if</mtext>
         <mi>i</mi>
        </mrow>
        <mo>≠</mo>
        <mi>j</mi>
       </mrow>
      </mtd>
      <mtd columnalign="center">
       <mrow>
        <mrow>
         <mtext>and</mtext>
         <msub>
          <mi>R</mi>
          <mrow>
           <mi>i</mi>
           <mi>j</mi>
          </mrow>
         </msub>
        </mrow>
        <mo>></mo>
        <msub>
         <mi>r</mi>
         <mi>c</mi>
        </msub>
       </mrow>
      </mtd>
     </mtr>
     <mtr>
      <mtd columnalign="center">
       <mrow>
        <mrow>
         <mo>-</mo>
         <mrow>
          <mstyle displaystyle="false">
           <msubsup>
            <mo largeop="true" symmetric="true">∑</mo>
            <mrow>
             <mrow>
              <mi>j</mi>
              <mo>,</mo>
              <mi>j</mi>
             </mrow>
             <mo>≠</mo>
             <mi>i</mi>
            </mrow>
            <mi>N</mi>
           </msubsup>
          </mstyle>
          <msub>
           <mi mathvariant="normal">Γ</mi>
           <mrow>
            <mi>i</mi>
            <mi>j</mi>
           </mrow>
          </msub>
         </mrow>
        </mrow>
        <mo>,</mo>
       </mrow>
      </mtd>
      <mtd columnalign="center">
       <mrow>
        <mrow>
         <mtext>if</mtext>
         <mi>i</mi>
        </mrow>
        <mo>=</mo>
        <mi>j</mi>
       </mrow>
      </mtd>
     </mtr>
    </mtable>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <cerror>
    <csymbol cd="ambiguous">fragments</csymbol>
    <apply>
     <csymbol cd="ambiguous">subscript</csymbol>
     <ci>normal-Γ</ci>
     <apply>
      <times></times>
      <ci>i</ci>
      <ci>j</ci>
     </apply>
    </apply>
    <eq></eq>
    <cerror>
     <csymbol cd="ambiguous">fragments</csymbol>
     <ci>normal-{</ci>
     <matrix>
      <matrixrow>
       <apply>
        <minus></minus>
        <cn type="integer">1</cn>
       </apply>
       <apply>
        <neq></neq>
        <apply>
         <times></times>
         <mtext>if</mtext>
         <ci>i</ci>
        </apply>
        <ci>j</ci>
       </apply>
       <apply>
        <leq></leq>
        <apply>
         <times></times>
         <mtext>and</mtext>
         <apply>
          <csymbol cd="ambiguous">subscript</csymbol>
          <ci>R</ci>
          <apply>
           <times></times>
           <ci>i</ci>
           <ci>j</ci>
          </apply>
         </apply>
        </apply>
        <apply>
         <csymbol cd="ambiguous">subscript</csymbol>
         <ci>r</ci>
         <ci>c</ci>
        </apply>
       </apply>
      </matrixrow>
      <matrixrow>
       <cn type="integer">0</cn>
       <apply>
        <neq></neq>
        <apply>
         <times></times>
         <mtext>if</mtext>
         <ci>i</ci>
        </apply>
        <ci>j</ci>
       </apply>
       <apply>
        <gt></gt>
        <apply>
         <times></times>
         <mtext>and</mtext>
         <apply>
          <csymbol cd="ambiguous">subscript</csymbol>
          <ci>R</ci>
          <apply>
           <times></times>
           <ci>i</ci>
           <ci>j</ci>
          </apply>
         </apply>
        </apply>
        <apply>
         <csymbol cd="ambiguous">subscript</csymbol>
         <ci>r</ci>
         <ci>c</ci>
        </apply>
       </apply>
      </matrixrow>
      <matrixrow>
       <apply>
        <minus></minus>
        <apply>
         <apply>
          <csymbol cd="ambiguous">superscript</csymbol>
          <apply>
           <csymbol cd="ambiguous">subscript</csymbol>
           <sum></sum>
           <apply>
            <neq></neq>
            <list>
             <ci>j</ci>
             <ci>j</ci>
            </list>
            <ci>i</ci>
           </apply>
          </apply>
          <ci>N</ci>
         </apply>
         <apply>
          <csymbol cd="ambiguous">subscript</csymbol>
          <ci>normal-Γ</ci>
          <apply>
           <times></times>
           <ci>i</ci>
           <ci>j</ci>
          </apply>
         </apply>
        </apply>
       </apply>
       <apply>
        <eq></eq>
        <apply>
         <times></times>
         <mtext>if</mtext>
         <ci>i</ci>
        </apply>
        <ci>j</ci>
       </apply>
      </matrixrow>
     </matrix>
    </cerror>
   </cerror>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \Gamma_{ij}=\left\{\begin{matrix}-1,&\mbox{if }i\neq j&\mbox{and }R_{ij}\leq r%
_{c}\\
0,&\mbox{if }i\neq j&\mbox{and }R_{ij}>r_{c}\\
-\sum_{j,j\neq i}^{N}\Gamma_{ij},&\mbox{if }i=j\end{matrix}\right.
  </annotation>
 </semantics>
</math>

</p>

<p><em>r</em><sub>c</sub> is a cutoff distance for spatial interactions and taken to be 7 Å for proteins.</p>

<p>Expressing the X, Y and Z components of the fluctuation vectors <strong>ΔR</strong><sub>i</sub> as <strong>ΔX</strong><sup>T</sup> = [ΔX<sub>1</sub> ΔX<sub>2</sub> ..... ΔX<sub>N</sub>], <strong>ΔY</strong><sup>T</sup> = [ΔY<sub>1</sub> ΔY<sub>2</sub> ..... ΔY<sub>N</sub>], and <strong>ΔZ</strong><sup>T</sup> = [ΔZ<sub>1</sub> ΔZ<sub>2</sub> ..... ΔZ<sub>N</sub>], above equation simplifies to</p>

<p>

<math display="block" id="Gaussian_network_model:2">
 <semantics>
  <mrow>
   <msub>
    <mi>V</mi>
    <mrow>
     <mi>G</mi>
     <mi>N</mi>
     <mi>M</mi>
    </mrow>
   </msub>
   <mo>=</mo>
   <mrow>
    <mfrac>
     <mi>γ</mi>
     <mn>2</mn>
    </mfrac>
    <mrow>
     <mo stretchy="false">[</mo>
     <mrow>
      <mrow>
       <mi mathvariant="normal">Δ</mi>
       <msup>
        <mi>X</mi>
        <mi>T</mi>
       </msup>
       <mi mathvariant="normal">Γ</mi>
       <mi mathvariant="normal">Δ</mi>
       <mi>X</mi>
      </mrow>
      <mo>+</mo>
      <mrow>
       <mi mathvariant="normal">Δ</mi>
       <msup>
        <mi>Y</mi>
        <mi>T</mi>
       </msup>
       <mi mathvariant="normal">Γ</mi>
       <mi mathvariant="normal">Δ</mi>
       <mi>Y</mi>
      </mrow>
      <mo>+</mo>
      <mrow>
       <mi mathvariant="normal">Δ</mi>
       <msup>
        <mi>Z</mi>
        <mi>T</mi>
       </msup>
       <mi mathvariant="normal">Γ</mi>
       <mi mathvariant="normal">Δ</mi>
       <mi>Z</mi>
      </mrow>
     </mrow>
     <mo stretchy="false">]</mo>
    </mrow>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <eq></eq>
    <apply>
     <csymbol cd="ambiguous">subscript</csymbol>
     <ci>V</ci>
     <apply>
      <times></times>
      <ci>G</ci>
      <ci>N</ci>
      <ci>M</ci>
     </apply>
    </apply>
    <apply>
     <times></times>
     <apply>
      <divide></divide>
      <ci>γ</ci>
      <cn type="integer">2</cn>
     </apply>
     <apply>
      <csymbol cd="latexml">delimited-[]</csymbol>
      <apply>
       <plus></plus>
       <apply>
        <times></times>
        <ci>normal-Δ</ci>
        <apply>
         <csymbol cd="ambiguous">superscript</csymbol>
         <ci>X</ci>
         <ci>T</ci>
        </apply>
        <ci>normal-Γ</ci>
        <ci>normal-Δ</ci>
        <ci>X</ci>
       </apply>
       <apply>
        <times></times>
        <ci>normal-Δ</ci>
        <apply>
         <csymbol cd="ambiguous">superscript</csymbol>
         <ci>Y</ci>
         <ci>T</ci>
        </apply>
        <ci>normal-Γ</ci>
        <ci>normal-Δ</ci>
        <ci>Y</ci>
       </apply>
       <apply>
        <times></times>
        <ci>normal-Δ</ci>
        <apply>
         <csymbol cd="ambiguous">superscript</csymbol>
         <ci>Z</ci>
         <ci>T</ci>
        </apply>
        <ci>normal-Γ</ci>
        <ci>normal-Δ</ci>
        <ci>Z</ci>
       </apply>
      </apply>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   V_{GNM}=\frac{\gamma}{2}[\Delta X^{T}\Gamma\Delta X+\Delta Y^{T}\Gamma\Delta Y%
+\Delta Z^{T}\Gamma\Delta Z]
  </annotation>
 </semantics>
</math>

</p>
<h3 id="statistical-mechanics-foundations">Statistical mechanics foundations</h3>

<p>In the GNM, the probability distribution of all fluctuations, <em>P</em>(<strong>ΔR</strong>) is <em>isotropic</em></p>

<p>

<math display="block" id="Gaussian_network_model:3">
 <semantics>
  <mrow>
   <mrow>
    <mi>P</mi>
    <mrow>
     <mo stretchy="false">(</mo>
     <mrow>
      <mi mathvariant="normal">Δ</mi>
      <mi>R</mi>
     </mrow>
     <mo stretchy="false">)</mo>
    </mrow>
   </mrow>
   <mo>=</mo>
   <mrow>
    <mi>P</mi>
    <mrow>
     <mo stretchy="false">(</mo>
     <mrow>
      <mi mathvariant="normal">Δ</mi>
      <mi>X</mi>
     </mrow>
     <mo>,</mo>
     <mrow>
      <mi mathvariant="normal">Δ</mi>
      <mi>Y</mi>
     </mrow>
     <mo>,</mo>
     <mrow>
      <mi mathvariant="normal">Δ</mi>
      <mi>Z</mi>
     </mrow>
     <mo stretchy="false">)</mo>
    </mrow>
   </mrow>
   <mo>=</mo>
   <mrow>
    <mi>p</mi>
    <mrow>
     <mo stretchy="false">(</mo>
     <mrow>
      <mi mathvariant="normal">Δ</mi>
      <mi>X</mi>
     </mrow>
     <mo stretchy="false">)</mo>
    </mrow>
    <mi>p</mi>
    <mrow>
     <mo stretchy="false">(</mo>
     <mrow>
      <mi mathvariant="normal">Δ</mi>
      <mi>Y</mi>
     </mrow>
     <mo stretchy="false">)</mo>
    </mrow>
    <mi>p</mi>
    <mrow>
     <mo stretchy="false">(</mo>
     <mrow>
      <mi mathvariant="normal">Δ</mi>
      <mi>Z</mi>
     </mrow>
     <mo stretchy="false">)</mo>
    </mrow>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <and></and>
    <apply>
     <eq></eq>
     <apply>
      <times></times>
      <ci>P</ci>
      <apply>
       <times></times>
       <ci>normal-Δ</ci>
       <ci>R</ci>
      </apply>
     </apply>
     <apply>
      <times></times>
      <ci>P</ci>
      <vector>
       <apply>
        <times></times>
        <ci>normal-Δ</ci>
        <ci>X</ci>
       </apply>
       <apply>
        <times></times>
        <ci>normal-Δ</ci>
        <ci>Y</ci>
       </apply>
       <apply>
        <times></times>
        <ci>normal-Δ</ci>
        <ci>Z</ci>
       </apply>
      </vector>
     </apply>
    </apply>
    <apply>
     <eq></eq>
     <share href="#.cmml">
     </share>
     <apply>
      <times></times>
      <ci>p</ci>
      <apply>
       <times></times>
       <ci>normal-Δ</ci>
       <ci>X</ci>
      </apply>
      <ci>p</ci>
      <apply>
       <times></times>
       <ci>normal-Δ</ci>
       <ci>Y</ci>
      </apply>
      <ci>p</ci>
      <apply>
       <times></times>
       <ci>normal-Δ</ci>
       <ci>Z</ci>
      </apply>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   P(\Delta R)=P(\Delta X,\Delta Y,\Delta Z)=p(\Delta X)p(\Delta Y)p(\Delta Z)
  </annotation>
 </semantics>
</math>

</p>

<p>and <em>Gaussian</em></p>

<p>

<math display="block" id="Gaussian_network_model:4">
 <semantics>
  <mrow>
   <mrow>
    <mi>p</mi>
    <mrow>
     <mo stretchy="false">(</mo>
     <mrow>
      <mi mathvariant="normal">Δ</mi>
      <mi>X</mi>
     </mrow>
     <mo stretchy="false">)</mo>
    </mrow>
   </mrow>
   <mo>∝</mo>
   <mrow>
    <mi>exp</mi>
    <mrow>
     <mo>{</mo>
     <mrow>
      <mo>-</mo>
      <mrow>
       <mfrac>
        <mi>γ</mi>
        <mrow>
         <mn>2</mn>
         <msub>
          <mi>k</mi>
          <mi>B</mi>
         </msub>
         <mi>T</mi>
        </mrow>
       </mfrac>
       <mi mathvariant="normal">Δ</mi>
       <msup>
        <mi>X</mi>
        <mi>T</mi>
       </msup>
       <mi mathvariant="normal">Γ</mi>
       <mi mathvariant="normal">Δ</mi>
       <mi>X</mi>
      </mrow>
     </mrow>
     <mo>}</mo>
    </mrow>
   </mrow>
   <mo>=</mo>
   <mrow>
    <mi>exp</mi>
    <mrow>
     <mo>{</mo>
     <mrow>
      <mo>-</mo>
      <mrow>
       <mfrac>
        <mn>1</mn>
        <mn>2</mn>
       </mfrac>
       <mrow>
        <mo>(</mo>
        <mrow>
         <mi mathvariant="normal">Δ</mi>
         <msup>
          <mi>X</mi>
          <mi>T</mi>
         </msup>
         <msup>
          <mrow>
           <mo>(</mo>
           <mrow>
            <mfrac>
             <mrow>
              <msub>
               <mi>k</mi>
               <mi>B</mi>
              </msub>
              <mi>T</mi>
             </mrow>
             <mi>γ</mi>
            </mfrac>
            <msup>
             <mi mathvariant="normal">Γ</mi>
             <mrow>
              <mo>-</mo>
              <mn>1</mn>
             </mrow>
            </msup>
           </mrow>
           <mo>)</mo>
          </mrow>
          <mrow>
           <mo>-</mo>
           <mn>1</mn>
          </mrow>
         </msup>
         <mi mathvariant="normal">Δ</mi>
         <mi>X</mi>
        </mrow>
        <mo>)</mo>
       </mrow>
      </mrow>
     </mrow>
     <mo>}</mo>
    </mrow>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <and></and>
    <apply>
     <csymbol cd="latexml">proportional-to</csymbol>
     <apply>
      <times></times>
      <ci>p</ci>
      <apply>
       <times></times>
       <ci>normal-Δ</ci>
       <ci>X</ci>
      </apply>
     </apply>
     <apply>
      <exp></exp>
      <apply>
       <minus></minus>
       <apply>
        <times></times>
        <apply>
         <divide></divide>
         <ci>γ</ci>
         <apply>
          <times></times>
          <cn type="integer">2</cn>
          <apply>
           <csymbol cd="ambiguous">subscript</csymbol>
           <ci>k</ci>
           <ci>B</ci>
          </apply>
          <ci>T</ci>
         </apply>
        </apply>
        <ci>normal-Δ</ci>
        <apply>
         <csymbol cd="ambiguous">superscript</csymbol>
         <ci>X</ci>
         <ci>T</ci>
        </apply>
        <ci>normal-Γ</ci>
        <ci>normal-Δ</ci>
        <ci>X</ci>
       </apply>
      </apply>
     </apply>
    </apply>
    <apply>
     <eq></eq>
     <share href="#.cmml">
     </share>
     <apply>
      <exp></exp>
      <apply>
       <minus></minus>
       <apply>
        <times></times>
        <apply>
         <divide></divide>
         <cn type="integer">1</cn>
         <cn type="integer">2</cn>
        </apply>
        <apply>
         <times></times>
         <ci>normal-Δ</ci>
         <apply>
          <csymbol cd="ambiguous">superscript</csymbol>
          <ci>X</ci>
          <ci>T</ci>
         </apply>
         <apply>
          <csymbol cd="ambiguous">superscript</csymbol>
          <apply>
           <times></times>
           <apply>
            <divide></divide>
            <apply>
             <times></times>
             <apply>
              <csymbol cd="ambiguous">subscript</csymbol>
              <ci>k</ci>
              <ci>B</ci>
             </apply>
             <ci>T</ci>
            </apply>
            <ci>γ</ci>
           </apply>
           <apply>
            <csymbol cd="ambiguous">superscript</csymbol>
            <ci>normal-Γ</ci>
            <apply>
             <minus></minus>
             <cn type="integer">1</cn>
            </apply>
           </apply>
          </apply>
          <apply>
           <minus></minus>
           <cn type="integer">1</cn>
          </apply>
         </apply>
         <ci>normal-Δ</ci>
         <ci>X</ci>
        </apply>
       </apply>
      </apply>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   p(\Delta X)\propto\exp\left\{-\frac{\gamma}{2k_{B}T}\Delta X^{T}\Gamma\Delta X%
\right\}=\exp\left\{-\frac{1}{2}\left(\Delta X^{T}\left(\frac{k_{B}T}{\gamma}%
\Gamma^{-1}\right)^{-1}\Delta X\right)\right\}
  </annotation>
 </semantics>
</math>

</p>

<p>where <em>k</em><sub><em>B</em></sub> is the Boltzmann constant and <em>T</em> is the absolute temperature. <em>p</em>(<strong>ΔY</strong>) and <em>p</em>(<strong>ΔZ</strong>) are expressed similarly. N-dimensional Gaussian probability density function with random variable vector <strong>x</strong>, mean vector <strong>μ</strong> and covariance matrix <strong>Σ</strong> is</p>

<p>

<math display="block" id="Gaussian_network_model:5">
 <semantics>
  <mrow>
   <mrow>
    <mi>W</mi>
    <mrow>
     <mo stretchy="false">(</mo>
     <mi>x</mi>
     <mo>,</mo>
     <mi>μ</mi>
     <mo>,</mo>
     <mi mathvariant="normal">Σ</mi>
     <mo stretchy="false">)</mo>
    </mrow>
   </mrow>
   <mo>=</mo>
   <mrow>
    <mfrac>
     <mn>1</mn>
     <msqrt>
      <mrow>
       <msup>
        <mrow>
         <mo stretchy="false">(</mo>
         <mrow>
          <mn>2</mn>
          <mi>π</mi>
         </mrow>
         <mo stretchy="false">)</mo>
        </mrow>
        <mi>N</mi>
       </msup>
       <mrow>
        <mo stretchy="false">|</mo>
        <mi mathvariant="normal">Σ</mi>
        <mo stretchy="false">|</mo>
       </mrow>
      </mrow>
     </msqrt>
    </mfrac>
    <mrow>
     <mi>exp</mi>
     <mrow>
      <mo>{</mo>
      <mrow>
       <mo>-</mo>
       <mrow>
        <mfrac>
         <mn>1</mn>
         <mn>2</mn>
        </mfrac>
        <msup>
         <mrow>
          <mo stretchy="false">(</mo>
          <mrow>
           <mi>x</mi>
           <mo>-</mo>
           <mi>μ</mi>
          </mrow>
          <mo stretchy="false">)</mo>
         </mrow>
         <mi>T</mi>
        </msup>
        <msup>
         <mi mathvariant="normal">Σ</mi>
         <mrow>
          <mo>-</mo>
          <mn>1</mn>
         </mrow>
        </msup>
        <mrow>
         <mo stretchy="false">(</mo>
         <mrow>
          <mi>x</mi>
          <mo>-</mo>
          <mi>μ</mi>
         </mrow>
         <mo stretchy="false">)</mo>
        </mrow>
       </mrow>
      </mrow>
      <mo>}</mo>
     </mrow>
    </mrow>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <eq></eq>
    <apply>
     <times></times>
     <ci>W</ci>
     <vector>
      <ci>x</ci>
      <ci>μ</ci>
      <ci>normal-Σ</ci>
     </vector>
    </apply>
    <apply>
     <times></times>
     <apply>
      <divide></divide>
      <cn type="integer">1</cn>
      <apply>
       <root></root>
       <apply>
        <times></times>
        <apply>
         <csymbol cd="ambiguous">superscript</csymbol>
         <apply>
          <times></times>
          <cn type="integer">2</cn>
          <ci>π</ci>
         </apply>
         <ci>N</ci>
        </apply>
        <apply>
         <abs></abs>
         <ci>normal-Σ</ci>
        </apply>
       </apply>
      </apply>
     </apply>
     <apply>
      <exp></exp>
      <apply>
       <minus></minus>
       <apply>
        <times></times>
        <apply>
         <divide></divide>
         <cn type="integer">1</cn>
         <cn type="integer">2</cn>
        </apply>
        <apply>
         <csymbol cd="ambiguous">superscript</csymbol>
         <apply>
          <minus></minus>
          <ci>x</ci>
          <ci>μ</ci>
         </apply>
         <ci>T</ci>
        </apply>
        <apply>
         <csymbol cd="ambiguous">superscript</csymbol>
         <ci>normal-Σ</ci>
         <apply>
          <minus></minus>
          <cn type="integer">1</cn>
         </apply>
        </apply>
        <apply>
         <minus></minus>
         <ci>x</ci>
         <ci>μ</ci>
        </apply>
       </apply>
      </apply>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   W(x,\mu,\Sigma)=\frac{1}{\sqrt{(2\pi)^{N}|\Sigma|}}\exp\left\{-\frac{1}{2}(x-%
\mu)^{T}\Sigma^{-1}(x-\mu)\right\}
  </annotation>
 </semantics>
</math>

 

<math display="inline" id="Gaussian_network_model:6">
 <semantics>
  <msqrt>
   <mrow>
    <msup>
     <mrow>
      <mo stretchy="false">(</mo>
      <mrow>
       <mn>2</mn>
       <mi>π</mi>
      </mrow>
      <mo stretchy="false">)</mo>
     </mrow>
     <mi>N</mi>
    </msup>
    <mrow>
     <mo stretchy="false">|</mo>
     <mi mathvariant="normal">Σ</mi>
     <mo stretchy="false">|</mo>
    </mrow>
   </mrow>
  </msqrt>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <root></root>
    <apply>
     <times></times>
     <apply>
      <csymbol cd="ambiguous">superscript</csymbol>
      <apply>
       <times></times>
       <cn type="integer">2</cn>
       <ci>π</ci>
      </apply>
      <ci>N</ci>
     </apply>
     <apply>
      <abs></abs>
      <ci>normal-Σ</ci>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \sqrt{(2\pi)^{N}|\Sigma|}
  </annotation>
 </semantics>
</math>

 normalizes the distribution and <strong>|Σ|</strong> is the determinant of the covariance matrix.</p>

<p>Similar to Gaussian distribution, normalized distribution for <strong>ΔX</strong><sup>T</sup> = [ΔX<sub>1</sub> ΔX<sub>2</sub> ..... ΔX<sub>N</sub>] around the equilibrium positions can be expressed as</p>

<p>

<math display="block" id="Gaussian_network_model:7">
 <semantics>
  <mrow>
   <mrow>
    <mi>p</mi>
    <mrow>
     <mo stretchy="false">(</mo>
     <mrow>
      <mi mathvariant="normal">Δ</mi>
      <mi>X</mi>
     </mrow>
     <mo stretchy="false">)</mo>
    </mrow>
   </mrow>
   <mo>=</mo>
   <mrow>
    <mfrac>
     <mn>1</mn>
     <msqrt>
      <mrow>
       <msup>
        <mrow>
         <mo stretchy="false">(</mo>
         <mrow>
          <mn>2</mn>
          <mi>π</mi>
         </mrow>
         <mo stretchy="false">)</mo>
        </mrow>
        <mi>N</mi>
       </msup>
       <mfrac>
        <mrow>
         <msub>
          <mi>k</mi>
          <mi>B</mi>
         </msub>
         <mi>T</mi>
        </mrow>
        <mi>γ</mi>
       </mfrac>
       <mrow>
        <mo stretchy="false">|</mo>
        <msup>
         <mi mathvariant="normal">Γ</mi>
         <mrow>
          <mo>-</mo>
          <mn>1</mn>
         </mrow>
        </msup>
        <mo stretchy="false">|</mo>
       </mrow>
      </mrow>
     </msqrt>
    </mfrac>
    <mrow>
     <mi>exp</mi>
     <mrow>
      <mo>{</mo>
      <mrow>
       <mo>-</mo>
       <mrow>
        <mfrac>
         <mn>1</mn>
         <mn>2</mn>
        </mfrac>
        <mrow>
         <mo>(</mo>
         <mrow>
          <mi mathvariant="normal">Δ</mi>
          <msup>
           <mi>X</mi>
           <mi>T</mi>
          </msup>
          <msup>
           <mrow>
            <mo>(</mo>
            <mrow>
             <mfrac>
              <mrow>
               <msub>
                <mi>k</mi>
                <mi>B</mi>
               </msub>
               <mi>T</mi>
              </mrow>
              <mi>γ</mi>
             </mfrac>
             <msup>
              <mi mathvariant="normal">Γ</mi>
              <mrow>
               <mo>-</mo>
               <mn>1</mn>
              </mrow>
             </msup>
            </mrow>
            <mo>)</mo>
           </mrow>
           <mrow>
            <mo>-</mo>
            <mn>1</mn>
           </mrow>
          </msup>
          <mi mathvariant="normal">Δ</mi>
          <mi>X</mi>
         </mrow>
         <mo>)</mo>
        </mrow>
       </mrow>
      </mrow>
      <mo>}</mo>
     </mrow>
    </mrow>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <eq></eq>
    <apply>
     <times></times>
     <ci>p</ci>
     <apply>
      <times></times>
      <ci>normal-Δ</ci>
      <ci>X</ci>
     </apply>
    </apply>
    <apply>
     <times></times>
     <apply>
      <divide></divide>
      <cn type="integer">1</cn>
      <apply>
       <root></root>
       <apply>
        <times></times>
        <apply>
         <csymbol cd="ambiguous">superscript</csymbol>
         <apply>
          <times></times>
          <cn type="integer">2</cn>
          <ci>π</ci>
         </apply>
         <ci>N</ci>
        </apply>
        <apply>
         <divide></divide>
         <apply>
          <times></times>
          <apply>
           <csymbol cd="ambiguous">subscript</csymbol>
           <ci>k</ci>
           <ci>B</ci>
          </apply>
          <ci>T</ci>
         </apply>
         <ci>γ</ci>
        </apply>
        <apply>
         <abs></abs>
         <apply>
          <csymbol cd="ambiguous">superscript</csymbol>
          <ci>normal-Γ</ci>
          <apply>
           <minus></minus>
           <cn type="integer">1</cn>
          </apply>
         </apply>
        </apply>
       </apply>
      </apply>
     </apply>
     <apply>
      <exp></exp>
      <apply>
       <minus></minus>
       <apply>
        <times></times>
        <apply>
         <divide></divide>
         <cn type="integer">1</cn>
         <cn type="integer">2</cn>
        </apply>
        <apply>
         <times></times>
         <ci>normal-Δ</ci>
         <apply>
          <csymbol cd="ambiguous">superscript</csymbol>
          <ci>X</ci>
          <ci>T</ci>
         </apply>
         <apply>
          <csymbol cd="ambiguous">superscript</csymbol>
          <apply>
           <times></times>
           <apply>
            <divide></divide>
            <apply>
             <times></times>
             <apply>
              <csymbol cd="ambiguous">subscript</csymbol>
              <ci>k</ci>
              <ci>B</ci>
             </apply>
             <ci>T</ci>
            </apply>
            <ci>γ</ci>
           </apply>
           <apply>
            <csymbol cd="ambiguous">superscript</csymbol>
            <ci>normal-Γ</ci>
            <apply>
             <minus></minus>
             <cn type="integer">1</cn>
            </apply>
           </apply>
          </apply>
          <apply>
           <minus></minus>
           <cn type="integer">1</cn>
          </apply>
         </apply>
         <ci>normal-Δ</ci>
         <ci>X</ci>
        </apply>
       </apply>
      </apply>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   p(\Delta X)=\frac{1}{\sqrt{(2\pi)^{N}\frac{k_{B}T}{\gamma}|\Gamma^{-1}|}}\exp%
\left\{-\frac{1}{2}\left(\Delta X^{T}\left(\frac{k_{B}T}{\gamma}\Gamma^{-1}%
\right)^{-1}\Delta X\right)\right\}
  </annotation>
 </semantics>
</math>

</p>

<p>The normalization constant, also the partition function <em>Z</em><sub>X</sub>, is given by</p>

<p>

<math display="block" id="Gaussian_network_model:8">
 <semantics>
  <mrow>
   <msub>
    <mi>Z</mi>
    <mi>X</mi>
   </msub>
   <mo>=</mo>
   <mrow>
    <msubsup>
     <mo largeop="true" symmetric="true">∫</mo>
     <mn>0</mn>
     <mi mathvariant="normal">∞</mi>
    </msubsup>
    <mrow>
     <mrow>
      <mi>exp</mi>
      <mrow>
       <mo>{</mo>
       <mrow>
        <mo>-</mo>
        <mrow>
         <mfrac>
          <mn>1</mn>
          <mn>2</mn>
         </mfrac>
         <mrow>
          <mo>(</mo>
          <mrow>
           <mi mathvariant="normal">Δ</mi>
           <msup>
            <mi>X</mi>
            <mi>T</mi>
           </msup>
           <msup>
            <mrow>
             <mo>(</mo>
             <mrow>
              <mfrac>
               <mrow>
                <msub>
                 <mi>k</mi>
                 <mi>B</mi>
                </msub>
                <mi>T</mi>
               </mrow>
               <mi>γ</mi>
              </mfrac>
              <msup>
               <mi mathvariant="normal">Γ</mi>
               <mrow>
                <mo>-</mo>
                <mn>1</mn>
               </mrow>
              </msup>
             </mrow>
             <mo>)</mo>
            </mrow>
            <mrow>
             <mo>-</mo>
             <mn>1</mn>
            </mrow>
           </msup>
           <mi mathvariant="normal">Δ</mi>
           <mi>X</mi>
          </mrow>
          <mo>)</mo>
         </mrow>
        </mrow>
       </mrow>
       <mo>}</mo>
      </mrow>
     </mrow>
     <mi>d</mi>
     <mi mathvariant="normal">Δ</mi>
     <mi>X</mi>
    </mrow>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <eq></eq>
    <apply>
     <csymbol cd="ambiguous">subscript</csymbol>
     <ci>Z</ci>
     <ci>X</ci>
    </apply>
    <apply>
     <apply>
      <csymbol cd="ambiguous">superscript</csymbol>
      <apply>
       <csymbol cd="ambiguous">subscript</csymbol>
       <int></int>
       <cn type="integer">0</cn>
      </apply>
      <infinity></infinity>
     </apply>
     <apply>
      <times></times>
      <apply>
       <exp></exp>
       <apply>
        <minus></minus>
        <apply>
         <times></times>
         <apply>
          <divide></divide>
          <cn type="integer">1</cn>
          <cn type="integer">2</cn>
         </apply>
         <apply>
          <times></times>
          <ci>normal-Δ</ci>
          <apply>
           <csymbol cd="ambiguous">superscript</csymbol>
           <ci>X</ci>
           <ci>T</ci>
          </apply>
          <apply>
           <csymbol cd="ambiguous">superscript</csymbol>
           <apply>
            <times></times>
            <apply>
             <divide></divide>
             <apply>
              <times></times>
              <apply>
               <csymbol cd="ambiguous">subscript</csymbol>
               <ci>k</ci>
               <ci>B</ci>
              </apply>
              <ci>T</ci>
             </apply>
             <ci>γ</ci>
            </apply>
            <apply>
             <csymbol cd="ambiguous">superscript</csymbol>
             <ci>normal-Γ</ci>
             <apply>
              <minus></minus>
              <cn type="integer">1</cn>
             </apply>
            </apply>
           </apply>
           <apply>
            <minus></minus>
            <cn type="integer">1</cn>
           </apply>
          </apply>
          <ci>normal-Δ</ci>
          <ci>X</ci>
         </apply>
        </apply>
       </apply>
      </apply>
      <ci>d</ci>
      <ci>normal-Δ</ci>
      <ci>X</ci>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   Z_{X}=\int_{0}^{\infty}\exp\left\{-\frac{1}{2}\left(\Delta X^{T}\left(\frac{k_%
{B}T}{\gamma}\Gamma^{-1}\right)^{-1}\Delta X\right)\right\}d\Delta X
  </annotation>
 </semantics>
</math>

</p>

<p>where 

<math display="inline" id="Gaussian_network_model:9">
 <semantics>
  <mrow>
   <mfrac>
    <mrow>
     <msub>
      <mi>k</mi>
      <mi>B</mi>
     </msub>
     <mi>T</mi>
    </mrow>
    <mi>γ</mi>
   </mfrac>
   <msup>
    <mi mathvariant="normal">Γ</mi>
    <mrow>
     <mo>-</mo>
     <mn>1</mn>
    </mrow>
   </msup>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <times></times>
    <apply>
     <divide></divide>
     <apply>
      <times></times>
      <apply>
       <csymbol cd="ambiguous">subscript</csymbol>
       <ci>k</ci>
       <ci>B</ci>
      </apply>
      <ci>T</ci>
     </apply>
     <ci>γ</ci>
    </apply>
    <apply>
     <csymbol cd="ambiguous">superscript</csymbol>
     <ci>normal-Γ</ci>
     <apply>
      <minus></minus>
      <cn type="integer">1</cn>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \frac{k_{B}T}{\gamma}\Gamma^{-1}
  </annotation>
 </semantics>
</math>

 is the covariance matrix in this case. <em>Z</em><sub>Y</sub> and <em>Z</em><sub>Z</sub> are expressed similarly. This formulation requires inversion of the Kirchhoff matrix. In the GNM, the determinant of the Kirchhoff matrix is zero, hence calculation of its inverse requires <a href="Spectral_theorem" title="wikilink">eigenvalue decomposition</a>. <strong>Γ</strong><sup>−1</sup> is constructed using the N-1 non-zero eigenvalues and associated eigenvectors. Expressions for <em>p</em>(<strong>ΔY</strong>) and <em>p</em>(<strong>ΔZ</strong>) are similar to that of <em>p</em>(<strong>ΔX</strong>). The probability distribution of all fluctuations in GNM becomes</p>

<p>

<math display="block" id="Gaussian_network_model:10">
 <semantics>
  <mrow>
   <mrow>
    <mi>P</mi>
    <mrow>
     <mo stretchy="false">(</mo>
     <mrow>
      <mi mathvariant="normal">Δ</mi>
      <mi>R</mi>
     </mrow>
     <mo stretchy="false">)</mo>
    </mrow>
   </mrow>
   <mo>=</mo>
   <mrow>
    <mi>p</mi>
    <mrow>
     <mo stretchy="false">(</mo>
     <mrow>
      <mi mathvariant="normal">Δ</mi>
      <mi>X</mi>
     </mrow>
     <mo stretchy="false">)</mo>
    </mrow>
    <mi>p</mi>
    <mrow>
     <mo stretchy="false">(</mo>
     <mrow>
      <mi mathvariant="normal">Δ</mi>
      <mi>Y</mi>
     </mrow>
     <mo stretchy="false">)</mo>
    </mrow>
    <mi>p</mi>
    <mrow>
     <mo stretchy="false">(</mo>
     <mrow>
      <mi mathvariant="normal">Δ</mi>
      <mi>Z</mi>
     </mrow>
     <mo stretchy="false">)</mo>
    </mrow>
   </mrow>
   <mo>=</mo>
   <mrow>
    <mfrac>
     <mn>1</mn>
     <msqrt>
      <mrow>
       <msup>
        <mrow>
         <mo stretchy="false">(</mo>
         <mrow>
          <mn>2</mn>
          <mi>π</mi>
         </mrow>
         <mo stretchy="false">)</mo>
        </mrow>
        <mrow>
         <mn>3</mn>
         <mi>N</mi>
        </mrow>
       </msup>
       <msup>
        <mrow>
         <mo stretchy="false">|</mo>
         <mrow>
          <mfrac>
           <mrow>
            <msub>
             <mi>k</mi>
             <mi>B</mi>
            </msub>
            <mi>T</mi>
           </mrow>
           <mi>γ</mi>
          </mfrac>
          <msup>
           <mi mathvariant="normal">Γ</mi>
           <mrow>
            <mo>-</mo>
            <mn>1</mn>
           </mrow>
          </msup>
         </mrow>
         <mo stretchy="false">|</mo>
        </mrow>
        <mn>3</mn>
       </msup>
      </mrow>
     </msqrt>
    </mfrac>
    <mrow>
     <mi>exp</mi>
     <mrow>
      <mo>{</mo>
      <mrow>
       <mo>-</mo>
       <mrow>
        <mfrac>
         <mn>3</mn>
         <mn>2</mn>
        </mfrac>
        <mrow>
         <mo>(</mo>
         <mrow>
          <mi mathvariant="normal">Δ</mi>
          <msup>
           <mi>X</mi>
           <mi>T</mi>
          </msup>
          <msup>
           <mrow>
            <mo>(</mo>
            <mrow>
             <mfrac>
              <mrow>
               <msub>
                <mi>k</mi>
                <mi>B</mi>
               </msub>
               <mi>T</mi>
              </mrow>
              <mi>γ</mi>
             </mfrac>
             <msup>
              <mi mathvariant="normal">Γ</mi>
              <mrow>
               <mo>-</mo>
               <mn>1</mn>
              </mrow>
             </msup>
            </mrow>
            <mo>)</mo>
           </mrow>
           <mrow>
            <mo>-</mo>
            <mn>1</mn>
           </mrow>
          </msup>
          <mi mathvariant="normal">Δ</mi>
          <mi>X</mi>
         </mrow>
         <mo>)</mo>
        </mrow>
       </mrow>
      </mrow>
      <mo>}</mo>
     </mrow>
    </mrow>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <and></and>
    <apply>
     <eq></eq>
     <apply>
      <times></times>
      <ci>P</ci>
      <apply>
       <times></times>
       <ci>normal-Δ</ci>
       <ci>R</ci>
      </apply>
     </apply>
     <apply>
      <times></times>
      <ci>p</ci>
      <apply>
       <times></times>
       <ci>normal-Δ</ci>
       <ci>X</ci>
      </apply>
      <ci>p</ci>
      <apply>
       <times></times>
       <ci>normal-Δ</ci>
       <ci>Y</ci>
      </apply>
      <ci>p</ci>
      <apply>
       <times></times>
       <ci>normal-Δ</ci>
       <ci>Z</ci>
      </apply>
     </apply>
    </apply>
    <apply>
     <eq></eq>
     <share href="#.cmml">
     </share>
     <apply>
      <times></times>
      <apply>
       <divide></divide>
       <cn type="integer">1</cn>
       <apply>
        <root></root>
        <apply>
         <times></times>
         <apply>
          <csymbol cd="ambiguous">superscript</csymbol>
          <apply>
           <times></times>
           <cn type="integer">2</cn>
           <ci>π</ci>
          </apply>
          <apply>
           <times></times>
           <cn type="integer">3</cn>
           <ci>N</ci>
          </apply>
         </apply>
         <apply>
          <csymbol cd="ambiguous">superscript</csymbol>
          <apply>
           <abs></abs>
           <apply>
            <times></times>
            <apply>
             <divide></divide>
             <apply>
              <times></times>
              <apply>
               <csymbol cd="ambiguous">subscript</csymbol>
               <ci>k</ci>
               <ci>B</ci>
              </apply>
              <ci>T</ci>
             </apply>
             <ci>γ</ci>
            </apply>
            <apply>
             <csymbol cd="ambiguous">superscript</csymbol>
             <ci>normal-Γ</ci>
             <apply>
              <minus></minus>
              <cn type="integer">1</cn>
             </apply>
            </apply>
           </apply>
          </apply>
          <cn type="integer">3</cn>
         </apply>
        </apply>
       </apply>
      </apply>
      <apply>
       <exp></exp>
       <apply>
        <minus></minus>
        <apply>
         <times></times>
         <apply>
          <divide></divide>
          <cn type="integer">3</cn>
          <cn type="integer">2</cn>
         </apply>
         <apply>
          <times></times>
          <ci>normal-Δ</ci>
          <apply>
           <csymbol cd="ambiguous">superscript</csymbol>
           <ci>X</ci>
           <ci>T</ci>
          </apply>
          <apply>
           <csymbol cd="ambiguous">superscript</csymbol>
           <apply>
            <times></times>
            <apply>
             <divide></divide>
             <apply>
              <times></times>
              <apply>
               <csymbol cd="ambiguous">subscript</csymbol>
               <ci>k</ci>
               <ci>B</ci>
              </apply>
              <ci>T</ci>
             </apply>
             <ci>γ</ci>
            </apply>
            <apply>
             <csymbol cd="ambiguous">superscript</csymbol>
             <ci>normal-Γ</ci>
             <apply>
              <minus></minus>
              <cn type="integer">1</cn>
             </apply>
            </apply>
           </apply>
           <apply>
            <minus></minus>
            <cn type="integer">1</cn>
           </apply>
          </apply>
          <ci>normal-Δ</ci>
          <ci>X</ci>
         </apply>
        </apply>
       </apply>
      </apply>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   P(\Delta R)=p(\Delta X)p(\Delta Y)p(\Delta Z)=\frac{1}{\sqrt{(2\pi)^{3N}|\frac%
{k_{B}T}{\gamma}\Gamma^{-1}|^{3}}}\exp\left\{-\frac{3}{2}\left(\Delta X^{T}%
\left(\frac{k_{B}T}{\gamma}\Gamma^{-1}\right)^{-1}\Delta X\right)\right\}
  </annotation>
 </semantics>
</math>

</p>

<p>For this mass and spring system, the normalization constant in the preceding expression is the overall GNM partition function, <em>Z</em><sub>GNM</sub>,</p>

<p>

<math display="block" id="Gaussian_network_model:11">
 <semantics>
  <mrow>
   <msub>
    <mi>Z</mi>
    <mrow>
     <mi>G</mi>
     <mi>N</mi>
     <mi>M</mi>
    </mrow>
   </msub>
   <mo>=</mo>
   <mrow>
    <msub>
     <mi>Z</mi>
     <mi>X</mi>
    </msub>
    <msub>
     <mi>Z</mi>
     <mi>Y</mi>
    </msub>
    <msub>
     <mi>Z</mi>
     <mi>Z</mi>
    </msub>
   </mrow>
   <mo>=</mo>
   <mfrac>
    <mn>1</mn>
    <msqrt>
     <mrow>
      <msup>
       <mrow>
        <mo stretchy="false">(</mo>
        <mrow>
         <mn>2</mn>
         <mi>π</mi>
        </mrow>
        <mo stretchy="false">)</mo>
       </mrow>
       <mrow>
        <mn>3</mn>
        <mi>N</mi>
       </mrow>
      </msup>
      <msup>
       <mrow>
        <mo stretchy="false">|</mo>
        <mrow>
         <mfrac>
          <mrow>
           <msub>
            <mi>k</mi>
            <mi>B</mi>
           </msub>
           <mi>T</mi>
          </mrow>
          <mi>γ</mi>
         </mfrac>
         <msup>
          <mi mathvariant="normal">Γ</mi>
          <mrow>
           <mo>-</mo>
           <mn>1</mn>
          </mrow>
         </msup>
        </mrow>
        <mo stretchy="false">|</mo>
       </mrow>
       <mn>3</mn>
      </msup>
     </mrow>
    </msqrt>
   </mfrac>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <and></and>
    <apply>
     <eq></eq>
     <apply>
      <csymbol cd="ambiguous">subscript</csymbol>
      <ci>Z</ci>
      <apply>
       <times></times>
       <ci>G</ci>
       <ci>N</ci>
       <ci>M</ci>
      </apply>
     </apply>
     <apply>
      <times></times>
      <apply>
       <csymbol cd="ambiguous">subscript</csymbol>
       <ci>Z</ci>
       <ci>X</ci>
      </apply>
      <apply>
       <csymbol cd="ambiguous">subscript</csymbol>
       <ci>Z</ci>
       <ci>Y</ci>
      </apply>
      <apply>
       <csymbol cd="ambiguous">subscript</csymbol>
       <ci>Z</ci>
       <ci>Z</ci>
      </apply>
     </apply>
    </apply>
    <apply>
     <eq></eq>
     <share href="#.cmml">
     </share>
     <apply>
      <divide></divide>
      <cn type="integer">1</cn>
      <apply>
       <root></root>
       <apply>
        <times></times>
        <apply>
         <csymbol cd="ambiguous">superscript</csymbol>
         <apply>
          <times></times>
          <cn type="integer">2</cn>
          <ci>π</ci>
         </apply>
         <apply>
          <times></times>
          <cn type="integer">3</cn>
          <ci>N</ci>
         </apply>
        </apply>
        <apply>
         <csymbol cd="ambiguous">superscript</csymbol>
         <apply>
          <abs></abs>
          <apply>
           <times></times>
           <apply>
            <divide></divide>
            <apply>
             <times></times>
             <apply>
              <csymbol cd="ambiguous">subscript</csymbol>
              <ci>k</ci>
              <ci>B</ci>
             </apply>
             <ci>T</ci>
            </apply>
            <ci>γ</ci>
           </apply>
           <apply>
            <csymbol cd="ambiguous">superscript</csymbol>
            <ci>normal-Γ</ci>
            <apply>
             <minus></minus>
             <cn type="integer">1</cn>
            </apply>
           </apply>
          </apply>
         </apply>
         <cn type="integer">3</cn>
        </apply>
       </apply>
      </apply>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   Z_{GNM}=Z_{X}Z_{Y}Z_{Z}=\frac{1}{\sqrt{(2\pi)^{3N}|\frac{k_{B}T}{\gamma}\Gamma%
^{-1}|^{3}}}
  </annotation>
 </semantics>
</math>

</p>
<h3 id="expectation-values-of-fluctuations-and-correlations">Expectation values of fluctuations and correlations</h3>

<p>Based on the statistical mechanics foundations of GNM, expectation values of residue fluctuations, i<sup>2</sup>&gt;, and correlations, i · <strong>ΔR</strong><sub>j</sub>&gt;, can be calculated. Covariance matrix for <strong>ΔX</strong> is given by</p>

<p>:<math> = \int \Delta X \cdot \Delta X^T p(\Delta X)d\Delta X=\frac{k_B T}{\gamma}\Gamma^{-1} </math></p>

<p>Since,</p>

<p>:<math> =  =  =\frac{1}{3} </math></p>

<p>i<sup>2</sup>&gt; and i · <strong>ΔR</strong><sub>j</sub>&gt; follows</p>

<p>:<math> = \frac{3 k_B T}{\gamma}(\Gamma^{-1})_{ii}</math></p>

<p>:<math> = \frac{3 k_B T}{\gamma}(\Gamma^{-1})_{ij}</math></p>
<h3 id="mode-decomposition">Mode decomposition</h3>

<p>The GNM normal modes are found by diagonalization of the Kirchhoff matrix, <strong>Γ</strong> = <strong>UΛU</strong><sup><em>T</em></sup>. Here, <strong>U</strong> is a unitary matrix, <strong>U</strong><sup><em>T</em></sup> = <strong>U</strong><sup>−1</sup>, of the eigenvectors <strong>u</strong><sub>i</sub> of <strong>Γ</strong> and <strong>Λ</strong> is the diagonal matrix of eigenvalues <strong>λ</strong><sub>i</sub>. The frequency and shape of a mode is represented by its eigenvalue and eigenvector, respectively. Since the Kirchhoff matrix is positive semi-definite, the first eigenvalue, <strong>λ</strong><sub>1</sub>, is zero and the corresponding eigenvector have all its elements equal to 1/√N. This shows that the network model is translation invariant.</p>

<p>Cross-correlations between residue fluctuations can be written as a sum over the N-1 nonzero modes as</p>

<p>:<math> = \frac{3 k_B T}{\gamma}[U\Lambda^{-1}U^T]_{ij}=\frac{3 k_B T}{\gamma}\sum_{k=1}^{N-1}\lambda_k^{-1} [u_k u_k^T]_{ij}</math></p>

<p>It follows that, [<strong>ΔR</strong><sub>i</sub> · <strong>ΔR</strong><sub>j</sub>], the contribution of an individual mode is expressed as</p>

<p>

<math display="block" id="Gaussian_network_model:12">
 <semantics>
  <mrow>
   <msub>
    <mrow>
     <mo stretchy="false">[</mo>
     <mrow>
      <mrow>
       <mrow>
        <mi mathvariant="normal">Δ</mi>
        <msub>
         <mi>R</mi>
         <mi>i</mi>
        </msub>
       </mrow>
       <mo>⋅</mo>
       <mi mathvariant="normal">Δ</mi>
      </mrow>
      <msub>
       <mi>R</mi>
       <mi>j</mi>
      </msub>
     </mrow>
     <mo stretchy="false">]</mo>
    </mrow>
    <mi>k</mi>
   </msub>
   <mo>=</mo>
   <mrow>
    <mfrac>
     <mrow>
      <mn>3</mn>
      <msub>
       <mi>k</mi>
       <mi>B</mi>
      </msub>
      <mi>T</mi>
     </mrow>
     <mi>γ</mi>
    </mfrac>
    <msubsup>
     <mi>λ</mi>
     <mi>k</mi>
     <mrow>
      <mo>-</mo>
      <mn>1</mn>
     </mrow>
    </msubsup>
    <msub>
     <mrow>
      <mo stretchy="false">[</mo>
      <msub>
       <mi>u</mi>
       <mi>k</mi>
      </msub>
      <mo stretchy="false">]</mo>
     </mrow>
     <mi>i</mi>
    </msub>
    <msub>
     <mrow>
      <mo stretchy="false">[</mo>
      <msub>
       <mi>u</mi>
       <mi>k</mi>
      </msub>
      <mo stretchy="false">]</mo>
     </mrow>
     <mi>j</mi>
    </msub>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <eq></eq>
    <apply>
     <csymbol cd="ambiguous">subscript</csymbol>
     <apply>
      <csymbol cd="latexml">delimited-[]</csymbol>
      <apply>
       <times></times>
       <apply>
        <ci>normal-⋅</ci>
        <apply>
         <times></times>
         <ci>normal-Δ</ci>
         <apply>
          <csymbol cd="ambiguous">subscript</csymbol>
          <ci>R</ci>
          <ci>i</ci>
         </apply>
        </apply>
        <ci>normal-Δ</ci>
       </apply>
       <apply>
        <csymbol cd="ambiguous">subscript</csymbol>
        <ci>R</ci>
        <ci>j</ci>
       </apply>
      </apply>
     </apply>
     <ci>k</ci>
    </apply>
    <apply>
     <times></times>
     <apply>
      <divide></divide>
      <apply>
       <times></times>
       <cn type="integer">3</cn>
       <apply>
        <csymbol cd="ambiguous">subscript</csymbol>
        <ci>k</ci>
        <ci>B</ci>
       </apply>
       <ci>T</ci>
      </apply>
      <ci>γ</ci>
     </apply>
     <apply>
      <csymbol cd="ambiguous">superscript</csymbol>
      <apply>
       <csymbol cd="ambiguous">subscript</csymbol>
       <ci>λ</ci>
       <ci>k</ci>
      </apply>
      <apply>
       <minus></minus>
       <cn type="integer">1</cn>
      </apply>
     </apply>
     <apply>
      <csymbol cd="ambiguous">subscript</csymbol>
      <apply>
       <csymbol cd="latexml">delimited-[]</csymbol>
       <apply>
        <csymbol cd="ambiguous">subscript</csymbol>
        <ci>u</ci>
        <ci>k</ci>
       </apply>
      </apply>
      <ci>i</ci>
     </apply>
     <apply>
      <csymbol cd="ambiguous">subscript</csymbol>
      <apply>
       <csymbol cd="latexml">delimited-[]</csymbol>
       <apply>
        <csymbol cd="ambiguous">subscript</csymbol>
        <ci>u</ci>
        <ci>k</ci>
       </apply>
      </apply>
      <ci>j</ci>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   [\Delta R_{i}\cdot\Delta R_{j}]_{k}=\frac{3k_{B}T}{\gamma}\lambda_{k}^{-1}[u_{%
k}]_{i}[u_{k}]_{j}
  </annotation>
 </semantics>
</math>

</p>

<p>where [<strong>u</strong><sub>k</sub>]<sub>i</sub> is the <strong>i</strong>th element of <strong>u</strong><sub>k</sub>.</p>
<h3 id="influence-of-local-packing-density">Influence of local packing density</h3>

<p>By definition, a diagonal element of the Kirchhoff matrix, <strong>Γ</strong><sub>ii</sub>, is equal to the degree of a node in GNM that represents the corresponding residue’s coordination number. This number is a measure of the local packing density around a given residue. The influence of local packing density can be assessed by series expansion of <strong>Γ</strong><sup>−1</sup> matrix. <strong>Γ</strong> can be written as a sum of two matrices, <strong>Γ</strong> = <strong>D</strong> + <strong>O</strong>, containing diagonal elements and off-diagonal elements of <strong>Γ</strong>.</p>
<dl>
<dd><strong>Γ</strong><sup>−1</sup> = (<strong>D</strong> + <strong>O</strong>)<sup>−1</sup> = [ <strong>D</strong> (<strong>I</strong> + <strong>D</strong><sup>−1</sup><strong>O</strong>) ]<sup>−1</sup> = (<strong>I</strong> + <strong>D</strong><sup>−1</sup><strong>O</strong>)<sup>−1</sup><strong>D</strong><sup>−1</sup> = (<strong>I</strong> - <strong>D</strong><sup>−1</sup><strong>O</strong> + ...)<sup>−1</sup><strong>D</strong><sup>−1</sup> = <strong>D</strong><sup>−1</sup> - <strong>D</strong><sup>−1</sup><strong>O</strong> <strong>D</strong><sup>−1</sup> + ...
</dd>
</dl>

<p>This expression shows that local packing density makes a significant contribution to expected fluctuations of residues.<a class="footnoteRef" href="#fn6" id="fnref6"><sup>6</sup></a> The terms that follow inverse of the diagonal matrix, are contributions of positional correlations to expected fluctuations.</p>
<h2 id="gnm-applications">GNM applications</h2>
<figure><b>(Figure)</b>
<figcaption><strong>Figure 3:</strong> Example of theoretical prediction of expected residue fluctuations for the catalytic domain of the protein Cdc25B, a cell division cycle dual-specificity phosphatase. <strong>A.</strong> Comparison of β-factors from X-ray structure (yellow) and theoretical calculations (red). <strong>B.</strong> Structure of catalytic domain of Cdc25B colored according to theoretical motility of regions. Light blue regions, e.g. topmost alpha-helix next to the catalytic site of this protein, are expected to be more mobile than the rest of the domain. <strong>C.</strong> Cross-correlation map i.e. normalized i·<strong>ΔR</strong><sub>j</sub>&gt; values. Red-colored regions correspond to collective residue motions and blue-colored regions correspond to uncorrelated motions. The results are retrieved iGNM server. PDB ID of Cdc25B is 1QB0.|400px</figcaption>
</figure>
<h3 id="equilibrium-fluctuations">Equilibrium fluctuations</h3>

<p>Equilibrium fluctuations of biological molecules can be experimentally measured. In <a href="X-ray_crystallography" title="wikilink">X-ray crystallography</a> β-factor (or temperature factor) of each atom is a measure of mean-squared fluctuation of the native structure. In NMR experiments, this measure can be obtained by calculating root-mean-squared differences between different models. In many applications and publications, including the original articles, it has been shown that expected residue fluctuations obtained from GNM is in good agreement with the experimentally measured native state fluctuations.<a class="footnoteRef" href="#fn7" id="fnref7"><sup>7</sup></a><a class="footnoteRef" href="#fn8" id="fnref8"><sup>8</sup></a> The relation between b-factors, for example, and expected residue fluctuations obtained from GNM is as follows</p>

<p>

<math display="block" id="Gaussian_network_model:13">
 <semantics>
  <mrow>
   <msub>
    <mi>B</mi>
    <mi>i</mi>
   </msub>
   <mo>=</mo>
   <mfrac>
    <mrow>
     <mn>8</mn>
     <msup>
      <mi>π</mi>
      <mn>2</mn>
     </msup>
    </mrow>
    <mn>3</mn>
   </mfrac>
   <mo><</mo>
   <mrow>
    <mrow>
     <mrow>
      <mi mathvariant="normal">Δ</mi>
      <msub>
       <mi>R</mi>
       <mi>i</mi>
      </msub>
     </mrow>
     <mo>⋅</mo>
     <mi mathvariant="normal">Δ</mi>
    </mrow>
    <msub>
     <mi>R</mi>
     <mi>i</mi>
    </msub>
   </mrow>
   <mo>≥</mo>
   <mrow>
    <mfrac>
     <mrow>
      <mn>8</mn>
      <msup>
       <mi>π</mi>
       <mn>2</mn>
      </msup>
      <msub>
       <mi>k</mi>
       <mi>B</mi>
      </msub>
      <mi>T</mi>
     </mrow>
     <mi>γ</mi>
    </mfrac>
    <msub>
     <mrow>
      <mo stretchy="false">(</mo>
      <msup>
       <mi mathvariant="normal">Γ</mi>
       <mrow>
        <mo>-</mo>
        <mn>1</mn>
       </mrow>
      </msup>
      <mo stretchy="false">)</mo>
     </mrow>
     <mrow>
      <mi>i</mi>
      <mi>i</mi>
     </mrow>
    </msub>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <and></and>
    <apply>
     <eq></eq>
     <apply>
      <csymbol cd="ambiguous">subscript</csymbol>
      <ci>B</ci>
      <ci>i</ci>
     </apply>
     <apply>
      <divide></divide>
      <apply>
       <times></times>
       <cn type="integer">8</cn>
       <apply>
        <csymbol cd="ambiguous">superscript</csymbol>
        <ci>π</ci>
        <cn type="integer">2</cn>
       </apply>
      </apply>
      <cn type="integer">3</cn>
     </apply>
    </apply>
    <apply>
     <lt></lt>
     <share href="#.cmml">
     </share>
     <apply>
      <times></times>
      <apply>
       <ci>normal-⋅</ci>
       <apply>
        <times></times>
        <ci>normal-Δ</ci>
        <apply>
         <csymbol cd="ambiguous">subscript</csymbol>
         <ci>R</ci>
         <ci>i</ci>
        </apply>
       </apply>
       <ci>normal-Δ</ci>
      </apply>
      <apply>
       <csymbol cd="ambiguous">subscript</csymbol>
       <ci>R</ci>
       <ci>i</ci>
      </apply>
     </apply>
    </apply>
    <apply>
     <geq></geq>
     <share href="#.cmml">
     </share>
     <apply>
      <times></times>
      <apply>
       <divide></divide>
       <apply>
        <times></times>
        <cn type="integer">8</cn>
        <apply>
         <csymbol cd="ambiguous">superscript</csymbol>
         <ci>π</ci>
         <cn type="integer">2</cn>
        </apply>
        <apply>
         <csymbol cd="ambiguous">subscript</csymbol>
         <ci>k</ci>
         <ci>B</ci>
        </apply>
        <ci>T</ci>
       </apply>
       <ci>γ</ci>
      </apply>
      <apply>
       <csymbol cd="ambiguous">subscript</csymbol>
       <apply>
        <csymbol cd="ambiguous">superscript</csymbol>
        <ci>normal-Γ</ci>
        <apply>
         <minus></minus>
         <cn type="integer">1</cn>
        </apply>
       </apply>
       <apply>
        <times></times>
        <ci>i</ci>
        <ci>i</ci>
       </apply>
      </apply>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   B_{i}=\frac{8\pi^{2}}{3}<\Delta R_{i}\cdot\Delta R_{i}>=\frac{8\pi^{2}k_{B}T}{%
\gamma}(\Gamma^{-1})_{ii}
  </annotation>
 </semantics>
</math>

</p>

<p>Figure 3 shows an example of GNM calculation for the catalytic domain of the protein Cdc25B, a <a href="cell_division_cycle" title="wikilink">cell division cycle</a> dual-specificity phosphatase. </p>
<h3 id="physical-meanings-of-slow-and-fast-modes">Physical meanings of slow and fast modes</h3>

<p>Diagonalization of the Kirchhoff matrix decomposes the normal modes of collective motions of the Gaussian network model of a biomolecule. The expected values of fluctuations and cross-correlations are obtained from linear combinations of fluctuations along these normal modes. The contribution of each mode is scaled with the inverse of that modes frequency. Hence, slow (low frequency) modes contribute most to the expected fluctuations. Along the few slowest modes, motions are shown to be collective and global and potentially relevant to functionality of the biomolecules [9,13,15-18]. Fast (high frequency) modes, on the other hand, describe uncorrelated motions not inducing notable changes in the structure.</p>
<h3 id="other-specific-applications">Other specific applications</h3>

<p>There are several major areas in which the Gaussian network model and other elastic network models are applied and found to be useful.<a class="footnoteRef" href="#fn9" id="fnref9"><sup>9</sup></a> These include:</p>
<ul>
<li>Decomposition of flexible/rigid regions and domains of proteins <a class="footnoteRef" href="#fn10" id="fnref10"><sup>10</sup></a><a class="footnoteRef" href="#fn11" id="fnref11"><sup>11</sup></a><a class="footnoteRef" href="#fn12" id="fnref12"><sup>12</sup></a></li>
<li>Characterization of functional motions and functionally important sites/residues of proteins, enzymes and large macromolecular assemblies <a class="footnoteRef" href="#fn13" id="fnref13"><sup>13</sup></a><a class="footnoteRef" href="#fn14" id="fnref14"><sup>14</sup></a><a class="footnoteRef" href="#fn15" id="fnref15"><sup>15</sup></a><a class="footnoteRef" href="#fn16" id="fnref16"><sup>16</sup></a><a class="footnoteRef" href="#fn17" id="fnref17"><sup>17</sup></a><a class="footnoteRef" href="#fn18" id="fnref18"><sup>18</sup></a><a class="footnoteRef" href="#fn19" id="fnref19"><sup>19</sup></a><a class="footnoteRef" href="#fn20" id="fnref20"><sup>20</sup></a><a class="footnoteRef" href="#fn21" id="fnref21"><sup>21</sup></a><a class="footnoteRef" href="#fn22" id="fnref22"><sup>22</sup></a></li>
<li>Refinement and dynamics of low-resolution structural data, e.g. <a href="Cryo-electron_microscopy" title="wikilink">Cryo-electron microscopy</a> <a class="footnoteRef" href="#fn23" id="fnref23"><sup>23</sup></a><a class="footnoteRef" href="#fn24" id="fnref24"><sup>24</sup></a><a class="footnoteRef" href="#fn25" id="fnref25"><sup>25</sup></a><a class="footnoteRef" href="#fn26" id="fnref26"><sup>26</sup></a></li>
<li><a href="Molecular_replacement" title="wikilink">Molecular replacement</a> for solving <a href="X-ray_structure" title="wikilink">X-ray structures</a>, when a <a href="conformational_change" title="wikilink">conformational change</a> occurred, with respect to a known structure<ref>Suhre, K. and Sanejouand, Y.H. On the potential of normal mode analysis for solving difficult molecular replacement problems, Acta Cryst. D 60, 796, 2004.</ref></li>
</ul>

<p></p>
<ul>
<li>Integration with atomistic models and simulations <a class="footnoteRef" href="#fn27" id="fnref27"><sup>27</sup></a><a class="footnoteRef" href="#fn28" id="fnref28"><sup>28</sup></a></li>
<li>Investigation of folding/unfolding pathways and kinetics.<a class="footnoteRef" href="#fn29" id="fnref29"><sup>29</sup></a><a class="footnoteRef" href="#fn30" id="fnref30"><sup>30</sup></a></li>
<li>Annotation of functional implication in molecular evolution <a class="footnoteRef" href="#fn31" id="fnref31"><sup>31</sup></a><a class="footnoteRef" href="#fn32" id="fnref32"><sup>32</sup></a></li>
</ul>
<h2 id="web-servers">Web servers</h2>

<p>In practice, two kinds of calculations can be performed. The first kind (the GNM per se) makes use of the <a href="Kirchhoff_matrix" title="wikilink">Kirchhoff matrix</a>.<a class="footnoteRef" href="#fn33" id="fnref33"><sup>33</sup></a><a class="footnoteRef" href="#fn34" id="fnref34"><sup>34</sup></a> The second kind (more specifically called either the Elastic Network Model or the Anisotropic Network Model) makes use of the <a href="Hessian_matrix" title="wikilink">Hessian matrix</a> associated to the corresponding set of harmonic springs.<a class="footnoteRef" href="#fn35" id="fnref35"><sup>35</sup></a> Both kinds of models can be used online, using the following servers.</p>
<h3 id="gnm-servers">GNM servers</h3>
<ul>
<li>iGNM: A database of protein functional motions based on GNM <a class="uri" href="http://ignm.ccbb.pitt.edu/Index.htm">http://ignm.ccbb.pitt.edu/Index.htm</a></li>
<li>oGNM: Online calculation of structural dynamics using GNM <a class="uri" href="http://ignm.ccbb.pitt.edu/GNM_Online_Calculation.htm">http://ignm.ccbb.pitt.edu/GNM_Online_Calculation.htm</a></li>
<li>GNM server <a class="uri" href="http://gor.bb.iastate.edu/gnm/gnm.htm">http://gor.bb.iastate.edu/gnm/gnm.htm</a></li>
</ul>
<h3 id="enmanm-servers">ENM/ANM servers</h3>
<ul>
<li><a href="Anisotropic_Network_Model" title="wikilink">Anisotropic Network Model</a> web server <a class="uri" href="http://www.ccbb.pitt.edu/anm">http://www.ccbb.pitt.edu/anm</a> <a class="footnoteRef" href="#fn36" id="fnref36"><sup>36</sup></a></li>
<li><a href="Anisotropic_Network_Model" title="wikilink">ANM</a> server <a class="uri" href="http://gor.bb.iastate.edu/anm/anm.htm">http://gor.bb.iastate.edu/anm/anm.htm</a></li>
<li>elNemo: Web-interface to The Elastic Network Model <a class="uri" href="http://www.igs.cnrs-mrs.fr/elnemo/">http://www.igs.cnrs-mrs.fr/elnemo/</a></li>
<li>AD-ENM: Analysis of Dynamics of an Elastic Network Model <a class="uri" href="http://enm.lobos.nih.gov/">http://enm.lobos.nih.gov/</a></li>
<li>WEBnm@: Web-server for Normal Mode Analysis of proteins <a class="uri" href="http://apps.cbu.uib.no/webnma/home">http://apps.cbu.uib.no/webnma/home</a></li>
</ul>
<h3 id="other-relevant-servers">Other relevant servers</h3>
<ul>
<li>ProMode: Database of normal mode analysis of proteins <a class="uri" href="http://cube.socs.waseda.ac.jp/pages/jsp/index.jsp">http://cube.socs.waseda.ac.jp/pages/jsp/index.jsp</a></li>
<li>HingeProt: An algorithm for protein hinge prediction using elastic network models <a class="uri" href="http://www.prc.boun.edu.tr/appserv/prc/hingeprot/">http://www.prc.boun.edu.tr/appserv/prc/hingeprot/</a>, or <a class="uri" href="http://bioinfo3d.cs.tau.ac.il/HingeProt/hingeprot.html">http://bioinfo3d.cs.tau.ac.il/HingeProt/hingeprot.html</a></li>
<li>DNABindProt: A Server for Determination of Potential DNA Binding Sites of Proteins <a class="uri" href="http://www.prc.boun.edu.tr/appserv/prc/dnabindprot/">http://www.prc.boun.edu.tr/appserv/prc/dnabindprot/</a></li>
<li>MolMovDB: A database of macromolecular motions: <a class="uri" href="http://www.molmovdb.org/">http://www.molmovdb.org/</a></li>
<li>The <a href="Protein_Data_Bank" title="wikilink">Protein Data Bank</a> (PDB) <a class="uri" href="http://www.pdb.org/">http://www.pdb.org/</a></li>
<li>A comprephensive elastic network model server: <a class="uri" href="http://omega.psi.iastate.edu">http://omega.psi.iastate.edu</a></li>
</ul>
<h2 id="see-also">See also</h2>
<ul>
<li><a href="Normal_distribution" title="wikilink">Gaussian distribution</a></li>
<li><a href="Harmonic_oscillator" title="wikilink">Harmonic oscillator</a></li>
<li><a href="Hooke's_law" title="wikilink">Hooke's law</a></li>
<li><a href="Molecular_dynamics" title="wikilink">Molecular dynamics</a></li>
<li><a href="Normal_mode" title="wikilink">Normal mode</a></li>
<li><a href="Principal_component_analysis" title="wikilink">Principal component analysis</a></li>
<li><a href="Protein_dynamics" title="wikilink">Protein dynamics</a></li>
<li><a href="Rubber_elasticity" title="wikilink">Rubber elasticity</a></li>
<li><a href="Statistical_mechanics" title="wikilink">Statistical mechanics</a></li>
</ul>
<h2 id="references">References</h2>
<h3 id="primary-sources">Primary sources</h3>
<ul>
<li>Direct evaluation of thermal fluctuations in protein using a single parameter harmonic potential, I. Bahar, A. R. Atilgan, and B. Erman Folding &amp; Design 2, 173-181, 1997.</li>
<li>Gaussian dynamics of folded proteins, Haliloglu, T. Bahar, I. &amp; Erman, B. Phys. Rev. Lett. 79, 3090-3093, 1997.</li>
<li>Cui Q, Bahar I, (2006). Normal Mode Analysis: Theory and applications to biological and chemical systems, Chapman &amp; Hall/CRC, London, UK</li>
</ul>
<h3 id="specific-citations">Specific citations</h3>

<p>"</p>

<p><a href="Category:Molecular_modelling" title="wikilink">Category:Molecular modelling</a></p>
<section class="footnotes">
<hr/>
<ol>
<li id="fn1">Tirion, M.M. Large amplitude elastic motions in proteins from a single-parameter, atomic analysis, Phys. Rev. Lett., 77, 1905, 1996.<a href="#fnref1">↩</a></li>
<li id="fn2">Direct evaluation of thermal fluctuations in protein using a single parameter harmonic potential, I. Bahar, A. R. Atilgan, and B. Erman Folding &amp; Design 2, 173-181, 1997.<a href="#fnref2">↩</a></li>
<li id="fn3">Gaussian dynamics of folded proteins, Haliloglu, T. Bahar, I. &amp; Erman, B. Phys. Rev. Lett. 79, 3090-3093, 1997.<a href="#fnref3">↩</a></li>
<li id="fn4">Flory, P.J., Statistical thermodynamics of random networks, Proc. Roy. Soc. Lond. A, 351, 351, 1976.<a href="#fnref4">↩</a></li>
<li id="fn5">Go, N., Noguti, T. and Nishikawa, T. Dynamics of a small globular protein in terms of low-frequency vibrational modes, Proc. Natl. Acad. Sci. USA, 80, 3696, 1983.<a href="#fnref5">↩</a></li>
<li id="fn6">Halle, B. Flexibility and packing in proteins, Proc. Natl. Acad. Sci. USA, 99, 1274, 2002.<a href="#fnref6">↩</a></li>
<li id="fn7">Correlation between native state hydrogen exchange and cooperative residue fluctuations from a simple model, I. Bahar, A. Wallqvist, D. G. Covell, &amp; R.L. Jernigan Biochemistry 37, 1067-1075, 1998.<a href="#fnref7">↩</a></li>
<li id="fn8">Vibrational dynamics of proteins: Significance of slow and fast modes in relation to function and stability, I. Bahar, A. R. Atilgan, M. C. Demirel, &amp; B. Erman, Phys. Rev. Lett. 80, 2733-2736, 1998.<a href="#fnref8">↩</a></li>
<li id="fn9">Chennubhotla C, Rader AJ, Yang LW, Bahar I (2005). Elastic network models for understanding biomolecular machinery: from enzymes to supramolecular assemblies. Phys. Biol. 2:S173-S180 PMID 16280623<a href="#fnref9">↩</a></li>
<li id="fn10">Analysis of domain motions by approximate normal mode calculations, Hinsen, K, Proteins 33(3), 417-429, 1999<a href="#fnref10">↩</a></li>
<li id="fn11">Identification of core amino acids stabilizing rhodopsin, Rader, AJ., G. Anderson, B. Isin, H. G. Khorana, I. Bahar, &amp; J. Klein-Seetharaman. Proc. Natl. Acad Sci USA 101: 7246-7251, 2004.<a href="#fnref11">↩</a></li>
<li id="fn12">Automatic domain decomposition of proteins by a Gaussian Network Model, Kundu, S., Sorensen, D.C., Phillips, G.N. Jr., Proteins 57(4), 725-733, 2004.<a href="#fnref12">↩</a></li>
<li id="fn13">Keskin, O. et al. Relating molecular flexibility to function: a case study of tubulin, Biophys. J., 83, 663, 2002.<a href="#fnref13">↩</a></li>
<li id="fn14">Inhibitor binding alters the directions of domain motions in HIV-1 reverse transcriptase, Temiz NA &amp; Bahar I, Proteins: Structure, Function and Genetics 49, 61-70, 2002.<a href="#fnref14">↩</a></li>
<li id="fn15">Xu, C., Tobi, D. and Bahar, I. Allosteric changes in protein structure computed by a simple mechanical model: hemoglobin T R2 transition, J. Mol. Biol., 333, 153, 2003.<a href="#fnref15">↩</a></li>
<li id="fn16">Structural Changes Involved in Protein Binding Correlate with intrinsic Motions of Proteins in the Unbound State, Dror Tobi &amp; Ivet Bahar. Proc Natl Acad Sci (USA) 102, 18908-18913, 2005.<a href="#fnref16">↩</a></li>
<li id="fn17">Common Mechanism of Pore Opening Shared by Five Different Potassium Channels, Indira H. Shrivastava &amp; Ivet Bahar. Biophys J 90, 3929-3940, 2006.<a href="#fnref17">↩</a></li>
<li id="fn18">Yang LW, Bahar I (2005). Coupling between Catalytic Site and Collective Dynamics: A requirement for Mechanochemical Activity of Enzymes. Structure 13:893-904  PMID 15939021<a href="#fnref18">↩</a></li>
<li id="fn19">Markov Methods for Hierarchical Coarse-Graining of Large Protein Dynamics, Chakra Chennubhotla &amp; Ivet Bahar. Lecture Notes in Computer Science 3909, 379-393, 2006.<a href="#fnref19">↩</a></li>
<li id="fn20">Global Ribosome Motions Revealed with Elastic Network Model, Wang, Y. Rader, AJ, Bahar, I. &amp; Jernigan, RL. , J. Struct Biol 147: 302-314, 2004.<a href="#fnref20">↩</a></li>
<li id="fn21">Maturation Dynamics of Bacteriophage HK97 Capsid, AJ Rader, Daniel Vlad &amp; Ivet Bahar. Structure (Camb) 13:413-21, 2005.<a href="#fnref21">↩</a></li>
<li id="fn22">Hamacher, K., Trylska, J., McCammon, J.A. Dependency Map of Proteins in the Small Ribosomal Subunit. PLoS Comput. Biol. 2(2): e10, 2006<a href="#fnref22">↩</a></li>
<li id="fn23">Ming, D. et al. How to describe protein motion without amino acid sequence and atomic coordinates, Proc. Natl. Acad. Sci. USA, 99, 8620, 2002.<a href="#fnref23">↩</a></li>
<li id="fn24">Tama, F., Wriggers, W. and Brooks III, C.L. Exploring global distortions of biological macromolecules and assemblies from low-resolution structural information and elastic network theory, J. Mol. Biol., 321, 297, 2002.<a href="#fnref24">↩</a></li>
<li id="fn25">Delarue, M. and Dumas, P. On the use of low-frequency normal modes to enforce collective movements in refining macromolecular structural models, Proc. Natl. Acad. Sci. USA, 101, 6957, 2004.<a href="#fnref25">↩</a></li>
<li id="fn26">Micheletti, C., Carloni, P. and Maritan, A. Accurate and efficient description of protein vibrational dynamics: comparing molecular dynamics and gaussian models, Proteins, 55, 635, 2004.<a href="#fnref26">↩</a></li>
<li id="fn27">Zhang, Z.Y., Shi, Y.Y. and Liu, H.Y. Molecular dynamics simulations of peptides and proteins with amplified collective motions, Bipohys. J., 84, 3583, 2003.<a href="#fnref27">↩</a></li>
<li id="fn28">Micheletti, C., Lattanzi, G. and Maritan, A. Elastic properties of proteins: insight on the folding process and evolutionary selection of native structures, J. Mol. Biol., 321, 909, 2002.<a href="#fnref28">↩</a></li>
<li id="fn29">Micheletti, C. et al. Crucial stages of protein folding through a solvable model: predicting target sites for enzyme-inhibiting drugs, Prot. Sci., 11, 1878, 2002.<a href="#fnref29">↩</a></li>
<li id="fn30">Portman, J.J., Takada, S. and Wolynes, P.G. Microscopic theory of protein folding rates. I. fine structure of the free energy profile and folding routes from a variational approach, J. Chem. Phys., 114, 5069, 2001.<a href="#fnref30">↩</a></li>
<li id="fn31">Hamacher, K. Relating Sequence Evolution of HIV1-Protease to Its Underlying Molecular Mechanics, Gene, 422, 30-36, 2008.<a href="#fnref31">↩</a></li>
<li id="fn32">Hamacher, K., McCammon J.A. Computing the amino acid specificity of fluctuations in biomolecular systems, J.Chem.Theo.Comp. 2:873, 2006.<a href="#fnref32">↩</a></li>
<li id="fn33"></li>
<li id="fn34"></li>
<li id="fn35"></li>
<li id="fn36">Anisotropy of fluctuation dynamics of proteins with an elastic network model, Atilgan, AR, Durrell, SR, Jernigan, RL, Demirel, MC, Keskin, O. &amp; Bahar, I. Biophys. J. 80, 505-515, 2001.<a href="#fnref36">↩</a></li>
</ol>
</section>
</body>
</html>
