<html lang="en">
<head>
<meta charset="utf-8"/>
<title offset="1526">GC-content</title>
<script src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_SVG.js" type="text/javascript">
</script>
</head>
<body>
<h1>GC-content</h1>
<hr/>

<p> In <a href="molecular_biology" title="wikilink">molecular biology</a> and <a class="uri" href="genetics" title="wikilink">genetics</a>, <strong>GC-content</strong> (or <strong>guanine-cytosine content</strong>) is the percentage of <a href="nitrogenous_bases" title="wikilink">nitrogenous bases</a> on a <a class="uri" href="DNA" title="wikilink">DNA</a> molecule that are either <a class="uri" href="guanine" title="wikilink">guanine</a> or <a class="uri" href="cytosine" title="wikilink">cytosine</a> (from a possibility of four different ones, also including <a class="uri" href="adenine" title="wikilink">adenine</a> and <a class="uri" href="thymine" title="wikilink">thymine</a>).<a class="footnoteRef" href="#fn1" id="fnref1"><sup>1</sup></a> This may refer to a specific fragment of DNA or <a class="uri" href="RNA" title="wikilink">RNA</a>, or that of the <a href="Genome" title="wikilink">whole genome</a>. When it refers to a fragment of the genetic material, it may denote the GC-content of part of a gene (domain), single gene, group of genes (or gene clusters), or even a non-coding region. G (guanine) and C (cytosine) undergo a specific <a href="hydrogen_bonding" title="wikilink">hydrogen bonding</a>, whereas A (adenine) bonds specifically with T (thymine).</p>

<p>The GC pair is bound by three <a href="hydrogen_bond" title="wikilink">hydrogen bonds</a>, while AT pairs are bound by two hydrogen bonds. DNA with high GC-content is more stable than DNA with low GC-content; however, the hydrogen bonds do not stabilize the DNA significantly, and stabilization is due mainly to stacking interactions.<a class="footnoteRef" href="#fn2" id="fnref2"><sup>2</sup></a> In spite of the higher <a class="uri" href="thermostability" title="wikilink">thermostability</a> conferred to the genetic material, it is envisaged that cells with DNA of high GC-content undergo <a href="Autolysis_(biology)" title="wikilink">autolysis</a>, thereby reducing the longevity of the cell <em>per se</em>.<a class="footnoteRef" href="#fn3" id="fnref3"><sup>3</sup></a> Due to the robustness endowed to the genetic materials in high GC organisms, it was commonly believed that the GC content played a vital part in adaptation temperatures, a hypothesis that has recently been refuted.<a class="footnoteRef" href="#fn4" id="fnref4"><sup>4</sup></a> However, the same study showed a strong correlation between higher temperatures and the GC content of structured <a href="RNA" title="wikilink">RNAs</a> (such as <a href="ribosomal_RNA" title="wikilink">ribosomal RNA</a>, <a href="transfer_RNA" title="wikilink">transfer RNA</a>, and many other <a href="non-coding_RNA" title="wikilink">non-coding RNAs</a>); GC <a href="base_pair" title="wikilink">base pairs</a> are more stable than AU base pairs, due to the fact that GC bonds have 3 hydrogen bonds and AU only has 2 hydrogen bonds, which makes high-GC-content RNA structures more tolerant of high temperatures. More recently, the first large-scale systematic gene-centric association analysis demonstrated the correlation between GC content and temperature for certain genomic regions while not for others.<a class="footnoteRef" href="#fn5" id="fnref5"><sup>5</sup></a></p>

<p>In <a class="uri" href="PCR" title="wikilink">PCR</a> experiments, the GC-content of <a href="primer_(molecular_biology)" title="wikilink">primers</a> are used to predict their <a href="Polymerase_chain_reaction" title="wikilink">annealing temperature</a> to the template DNA. A higher GC-content level indicates a higher melting temperature.</p>
<h2 id="determination-of-gc-content">Determination of GC content</h2>

<p>GC content is usually expressed as a percentage value, but sometimes as a ratio (called <strong>G+C ratio</strong> or <strong>GC-ratio</strong>). GC-content percentage is calculated as<a class="footnoteRef" href="#fn6" id="fnref6"><sup>6</sup></a></p>

<p>

<math display="block" id="GC-content:0">
 <semantics>
  <mrow>
   <mfrac>
    <mrow>
     <mi>G</mi>
     <mo>+</mo>
     <mi>C</mi>
    </mrow>
    <mrow>
     <mi>A</mi>
     <mo>+</mo>
     <mi>T</mi>
     <mo>+</mo>
     <mi>G</mi>
     <mo>+</mo>
     <mi>C</mi>
    </mrow>
   </mfrac>
   <mo>×</mo>
   <mn>100</mn>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <times></times>
    <apply>
     <csymbol cd="latexml">continued-fraction</csymbol>
     <apply>
      <plus></plus>
      <ci>G</ci>
      <ci>C</ci>
     </apply>
     <apply>
      <plus></plus>
      <ci>A</ci>
      <ci>T</ci>
      <ci>G</ci>
      <ci>C</ci>
     </apply>
    </apply>
    <cn type="float">100</cn>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \cfrac{G+C}{A+T+G+C}\times\ 100
  </annotation>
 </semantics>
</math>

</p>

<p>whereas the AT/GC ratio is calculated as<a class="footnoteRef" href="#fn7" id="fnref7"><sup>7</sup></a></p>

<p>

<math display="block" id="GC-content:1">
 <semantics>
  <mfrac>
   <mrow>
    <mi>A</mi>
    <mo>+</mo>
    <mi>T</mi>
   </mrow>
   <mrow>
    <mi>G</mi>
    <mo>+</mo>
    <mi>C</mi>
   </mrow>
  </mfrac>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <csymbol cd="latexml">continued-fraction</csymbol>
    <apply>
     <plus></plus>
     <ci>A</ci>
     <ci>T</ci>
    </apply>
    <apply>
     <plus></plus>
     <ci>G</ci>
     <ci>C</ci>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \cfrac{A+T}{G+C}
  </annotation>
 </semantics>
</math>

 .</p>

<p>The GC-content percentages as well as GC-ratio can be measured by several means, but one of the simplest methods is to measure what is called the <a href="DNA_melting" title="wikilink">melting temperature</a> of the <a class="uri" href="DNA" title="wikilink">DNA</a> <a href="double_helix" title="wikilink">double helix</a> using <a class="uri" href="spectrophotometry" title="wikilink">spectrophotometry</a>. The <a class="uri" href="absorbance" title="wikilink">absorbance</a> of <a class="uri" href="DNA" title="wikilink">DNA</a> at a <a class="uri" href="wavelength" title="wikilink">wavelength</a> of 260 <a href="Nanometer" title="wikilink">nm</a> increases fairly sharply when the double-stranded <a class="uri" href="DNA" title="wikilink">DNA</a> separates into two single strands when sufficiently heated.<a class="footnoteRef" href="#fn8" id="fnref8"><sup>8</sup></a> The most commonly used protocol for determining GC ratios uses <a href="flow_cytometry" title="wikilink">flow cytometry</a> for large number of samples.<a class="footnoteRef" href="#fn9" id="fnref9"><sup>9</sup></a></p>

<p>In alternative manner, if the DNA or RNA molecule under investigation has been <a class="uri" href="sequenced" title="wikilink">sequenced</a> then the GC-content can be accurately calculated by simple arithmetic or by using the <a href="http://www.basic.northwestern.edu/biotools/oligocalc.html">free online GC calculator</a>.</p>
<h2 id="gc-ratio-of-genomes">GC ratio of genomes</h2>

<p>GC ratios within a genome is found to be markedly variable. These variations in GC ratio within the genomes of more complex organisms result in a mosaic-like formation with islet regions called <a href="Isochore_(genetics)" title="wikilink">isochores</a>.<a class="footnoteRef" href="#fn10" id="fnref10"><sup>10</sup></a> This results in the variations in staining intensity in the <a class="uri" href="chromosomes" title="wikilink">chromosomes</a>.<a class="footnoteRef" href="#fn11" id="fnref11"><sup>11</sup></a> GC-rich isochores include in them many protein coding genes, and thus determination of ratio of these specific regions contributes in mapping gene-rich regions of the genome.<a class="footnoteRef" href="#fn12" id="fnref12"><sup>12</sup></a><a class="footnoteRef" href="#fn13" id="fnref13"><sup>13</sup></a></p>
<h2 id="gc-ratios-and-coding-sequence">GC ratios and coding sequence</h2>

<p>Within a long region of genomic sequence, genes are often characterised by having a higher GC-content in contrast to the background GC-content for the entire genome. Evidence of GC ratio with that of length of the <a href="coding_region" title="wikilink">coding region</a> of a <a class="uri" href="gene" title="wikilink">gene</a> has shown that the length of the coding sequence is directly proportional to higher G+C content.<a class="footnoteRef" href="#fn14" id="fnref14"><sup>14</sup></a> This has been pointed to the fact that the <a href="stop_codon" title="wikilink">stop codon</a> has a bias towards A and T nucleotides, and, thus, the shorter the sequence the higher the AT bias.<a class="footnoteRef" href="#fn15" id="fnref15"><sup>15</sup></a></p>
<h2 id="application-in-systematics">Application in systematics</h2>

<p>GC content is found to be variable with different organisms, the process of which is envisaged to be contributed to by variation in <a href="Gene-centered_view_of_evolution" title="wikilink">selection</a>, mutational bias, and biased recombination-associated <a href="DNA_repair" title="wikilink">DNA repair</a>.<a class="footnoteRef" href="#fn16" id="fnref16"><sup>16</sup></a> The <a href="species_problem" title="wikilink">species problem</a> in prokaryotic taxonomy has led to various suggestions in classifying bacteria, and the <em>ad hoc committee on reconciliation of approaches to bacterial systematics</em> has recommended use of GC ratios in higher level hierarchical classification.<a class="footnoteRef" href="#fn17" id="fnref17"><sup>17</sup></a> For example, the <a class="uri" href="Actinobacteria" title="wikilink">Actinobacteria</a> are characterised as "high GC-content <a href="bacterium" title="wikilink">bacteria</a>".<a class="footnoteRef" href="#fn18" id="fnref18"><sup>18</sup></a> In <em><a href="Streptomyces_coelicolor" title="wikilink">Streptomyces coelicolor</a></em> A3(2), GC content is 72%.<a class="footnoteRef" href="#fn19" id="fnref19"><sup>19</sup></a> The GC-content of <a class="uri" href="Yeast" title="wikilink">Yeast</a> (<em><a href="Saccharomyces_cerevisiae" title="wikilink">Saccharomyces cerevisiae</a></em>) is 38%,<a class="footnoteRef" href="#fn20" id="fnref20"><sup>20</sup></a> and that of another common <a href="model_organism" title="wikilink">model organism</a>, Thale Cress (<em><a href="Arabidopsis_thaliana" title="wikilink">Arabidopsis thaliana</a></em>), is 36%.<a class="footnoteRef" href="#fn21" id="fnref21"><sup>21</sup></a> Because of the nature of the <a href="genetic_code" title="wikilink">genetic code</a>, it is virtually impossible for an organism to have a genome with a GC-content approaching either 0% or 100%. A species with an extremely low GC-content is <em><a href="Plasmodium_falciparum" title="wikilink">Plasmodium falciparum</a></em> (GC% = ~20%),<a class="footnoteRef" href="#fn22" id="fnref22"><sup>22</sup></a> and it is usually common to refer to such examples as being AT-rich instead of GC-poor.<a class="footnoteRef" href="#fn23" id="fnref23"><sup>23</sup></a></p>
<h2 id="references">References</h2>
<h2 id="external-links">External links</h2>
<ol>
<li><a href="http://insilico.ehu.es/oligoweb/index2.php?m=all">Table with GC-content of all sequenced prokaryotes</a></li>
<li>[<a class="uri" href="http://www.ncbi.nlm.nih.gov/Taxonomy/Browser/wwwtax.cgi?mode=Tree&amp;id">http://www.ncbi.nlm.nih.gov/Taxonomy/Browser/wwwtax.cgi?mode=Tree&amp;id;</a>;=2&amp;lvl;=3&amp;srchmode;=1&amp;keep;=1&amp;unlock; Taxonomic browser of bacteria based on GC ratio on NCBI website].</li>
<li><a href="http://esper.lab.nig.ac.jp/study/genome/?page=genome_composition_database_species_list">GC ratio in diverse species</a>.</li>
</ol>

<p>"</p>

<p><a class="uri" href="Category:DNA" title="wikilink">Category:DNA</a> <a href="Category:Molecular_biology" title="wikilink">Category:Molecular biology</a> <a href="Category:Biological_classification" title="wikilink">Category:Biological classification</a> <a class="uri" href="Category:Genetics" title="wikilink">Category:Genetics</a></p>
<section class="footnotes">
<hr/>
<ol>
<li id="fn1"><a href="http://cancerweb.ncl.ac.uk/cgi-bin/omd?GC+content">Definition of GC – content on CancerWeb of </a><a href="Newcastle_University" title="wikilink">Newcastle University</a>,UK<a href="#fnref1">↩</a></li>
<li id="fn2"><a href="#fnref2">↩</a></li>
<li id="fn3"><a href="#fnref3">↩</a></li>
<li id="fn4"><a href="#fnref4">↩</a></li>
<li id="fn5"><a href="#fnref5">↩</a></li>
<li id="fn6"><a href="#fnref6">↩</a></li>
<li id="fn7"><a href="http://www.biochem.northwestern.edu/holmgren/Glossary/Definitions/Def-A/A+T_G+C_ratio.html">Definition of GC-ratio on Northwestern University, IL, USA</a><a href="#fnref7">↩</a></li>
<li id="fn8"><a href="#fnref8">↩</a></li>
<li id="fn9"><a href="#fnref9">↩</a></li>
<li id="fn10"><a href="#fnref10">↩</a></li>
<li id="fn11"><a href="#fnref11">↩</a></li>
<li id="fn12"><a href="#fnref12">↩</a></li>
<li id="fn13"><a href="#fnref13">↩</a></li>
<li id="fn14"><a href="#fnref14">↩</a></li>
<li id="fn15"><a href="#fnref15">↩</a></li>
<li id="fn16"><a href="#fnref16">↩</a></li>
<li id="fn17"><a href="#fnref17">↩</a></li>
<li id="fn18">[<a class="uri" href="http://www.ncbi.nlm.nih.gov/Taxonomy/Browser/wwwtax.cgi?mode=Tree&amp;id">http://www.ncbi.nlm.nih.gov/Taxonomy/Browser/wwwtax.cgi?mode=Tree&amp;id;</a>;=1760&amp;lvl;=3&amp;lin;=f&amp;keep;=1&amp;srchmode;=1&amp;unlock; Taxonomy browser on NCBI]<a href="#fnref18">↩</a></li>
<li id="fn19">[<a class="uri" href="http://www.ncbi.nlm.nih.gov/sites/entrez?db=genomeprj&amp;cmd">http://www.ncbi.nlm.nih.gov/sites/entrez?db=genomeprj&amp;cmd;</a>;=Retrieve&amp;dopt;=Overview&amp;list;_uids=242 Whole genome data of "Streptomyces coelicolor" A3(2) on NCBI]<a href="#fnref19">↩</a></li>
<li id="fn20">[<a class="uri" href="http://www.ncbi.nlm.nih.gov/sites/entrez?db=genomeprj&amp;cmd">http://www.ncbi.nlm.nih.gov/sites/entrez?db=genomeprj&amp;cmd;</a>;=Retrieve&amp;dopt;=Overview&amp;list;_uids=128 Whole genome data of <em>Saccharomyces cerevisiae</em> on NCBI]<a href="#fnref20">↩</a></li>
<li id="fn21">[<a class="uri" href="http://www.ncbi.nlm.nih.gov/sites/entrez?db=genomeprj&amp;cmd">http://www.ncbi.nlm.nih.gov/sites/entrez?db=genomeprj&amp;cmd;</a>;=Retrieve&amp;dopt;=Overview&amp;list;_uids=116 Whole genome data of '' Arabidopsis thaliana'' on NCBI]<a href="#fnref21">↩</a></li>
<li id="fn22">[<a class="uri" href="http://www.ncbi.nlm.nih.gov/sites/entrez?db=genomeprj&amp;cmd">http://www.ncbi.nlm.nih.gov/sites/entrez?db=genomeprj&amp;cmd;</a>;=Retrieve&amp;dopt;=Overview&amp;list;_uids=148 Whole genome data of <em>Plasmodium falciparum</em> on NCBI]<a href="#fnref22">↩</a></li>
<li id="fn23"><a href="#fnref23">↩</a></li>
</ol>
</section>
</body>
</html>
