<html lang="en">
<head>
<meta charset="utf-8"/>
<title offset="1205">Genetic load</title>
<script src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_SVG.js" type="text/javascript">
</script>
</head>
<body>
<h1>Genetic load</h1>
<hr>'''Genetic load''' is the difference between the [[fitness (biology)|fitness]] of the theoretically optimal [[genotype]] and the fitness of the observed average genotype in a [[population]]. This means that populations with a low genetic load will tend to be healthier, have fitness that is less dispersed, and have offspring that are more likely to survive to reproduction.<ref>Whit
<p>lock M., Bourguet D. 2000. Factors affecting the genetic load in Drosophila synergistic epistasis and correlations among fitness components, Evolution 54:5. On the other hand, populations with a high genetic load will have a greater variance in fitness levels; furthermore, a population with a high genetic load is not likely to have as many organisms at or near the optimal fitness.<a class="footnoteRef" href="#fn1" id="fnref1"><sup>1</sup></a> Genetic load can be viewed as the probability that an organism will fail to reach its reproductive age.<a class="footnoteRef" href="#fn2" id="fnref2"><sup>2</sup></a> This is important to scientists because if genetic load gets too high the population will be in danger of going extinct because the organisms are not able to survive and reproduce.</p>

<p>Genetic load is the reduction in selective value for a population compared to what the population would have if all individuals had the most favored genotype.<a class="footnoteRef" href="#fn3" id="fnref3"><sup>3</sup></a> It is normally stated in terms of fitness as the reduction in the mean fitness for a population compared to the maximum fitness.</p>
<h2 id="causes">Causes</h2>
<h3 id="mutation">Mutation</h3>

<p>This is the portion of the genetic load due to mutation, which is almost always negative since more mutations on a genome are deleterious than beneficial. Mutation loads are higher in populations which experience a higher mutation rate. Mutation loads are also higher in populations with positive selection. This is because positive selection means that the genome is more likely to keep changes that are made through mutation than genomes that are not undergoing positive selection. This means that the population is more likely to keep mutations that are deleterious. It is important to remember that not all mutations are negative ones, and that mutations which are beneficial will also be kept, but they will occur much less often than the ones that are deleterious. If a population is lucky enough to receive a high amount of beneficial mutations, then that mutation load might actually help the population, though this is unlikely to happen. Mutation load also tends to be higher in larger populations as well as populations that have been around for longer.<a class="footnoteRef" href="#fn4" id="fnref4"><sup>4</sup></a> Mutation load is larger in big populations because there are more individuals which can generate and pass on the mutation. Mutation load is bigger in older populations because they have had a longer amount of time to accrue. Mutation load is the main contributing factor to genetic load overall.<a class="footnoteRef" href="#fn5" id="fnref5"><sup>5</sup></a></p>
<h3 id="inbreeding">Inbreeding</h3>

<p>Inbreeding increases genetic load in the short run, but actually decreases genetic load in the long run. Overall, inbreeding increases the rates of homozygosity. In the short run, this means that offspring are more likely to get two recessive and deleterious alleles, so they will have lower fitnesses and experience an <a href="inbreeding_depression" title="wikilink">inbreeding depression</a>. The result is that you could begin to weed out deleterious alleles, some scientists have gone so far as to claim that if a population begins to decline because of higher mutation rates then getting the remaining organisms to breed with their relatives will help push out some of the more harmful mutations.<a class="footnoteRef" href="#fn6" id="fnref6"><sup>6</sup></a> However, once the rate of the allele gets high enough the pairing frequency will get so high that it will start to weed out those alleles at a high rate since the organisms which get them will die before reproducing. This is referred to as a purging of the population as a result of inbreeding.<a class="footnoteRef" href="#fn7" id="fnref7"><sup>7</sup></a> This kind of load is dependent on sexual reproduction, so isn‚Äôt present in asexually reproducing species.</p>
<h3 id="sexual-vs.-asexual-reproduction">Sexual vs. asexual reproduction</h3>

<p>Sexual and asexual reproduction both have ways that they directly increase or decrease genetic load. Sexual reproduction can increase genetic load by pairing together two different deleterious alleles who‚Äôs net effect is higher than just the sum of their two deleterious effects, this is referred to as a concealed portion of the genetic load.<a class="footnoteRef" href="#fn8" id="fnref8"><sup>8</sup></a> However, sexual reproduction can also prevent deleterious alleles from spreading and building up in a population, because less fit individuals won‚Äôt have as much reproductive success. This can also apply to those organisms which reproduce asexually because all organisms will create copies of themselves, thus passing on all of their deleterious mutations, causing a buildup in the population.<a class="footnoteRef" href="#fn9" id="fnref9"><sup>9</sup></a> This is called <a href="Muller‚Äôs_ratchet" title="wikilink">Muller‚Äôs ratchet</a>. If the mutations that are spread to all offspring are only slightly deleterious to the point where the organisms can still live and reproduce then the genetic load will increase at a steady rate.<a class="footnoteRef" href="#fn10" id="fnref10"><sup>10</sup></a> If the mutations are quite harmful then eventually this may lead to a rapid fall in the population size. With both of these in mind, sexually reproducing species are still expected to have lower genetic loads.<a class="footnoteRef" href="#fn11" id="fnref11"><sup>11</sup></a></p>
<h3 id="recombinationsegregation-load">Recombination/segregation load</h3>

<p>This is the fitness cost of breaking down genetic correlations and linkage disequilibria that were established by natural selection.<a class="footnoteRef" href="#fn12" id="fnref12"><sup>12</sup></a> Especially useful linkages or correlations happen when two beneficial alleles co-occur, organisms that do inherit them are well off, and those who do not experience a much depleted fitness. By breaking down this relationship you open up the possibility of inheriting one of the ‚Äúgood‚Äù genes and one of the ‚Äúbad‚Äù ones, which will increase genetic load as opposed to receiving both of the good ones. It is known as recombination load when this acts on two different loci and segregation load when it happens at a single locus.<a class="footnoteRef" href="#fn13" id="fnref13"><sup>13</sup></a> Because this is the result of the joining together of two sex cells, this kind of load is only present in sexual reproduction.</p>
<h3 id="migration-load">Migration load</h3>

<p>Migration load is the result of nonnative organisms that aren‚Äôt adapted to a particular environment coming into that environment and breeding with individuals who are adapted to that environment. This will mean that their offspring will not be as fit as they would have been if both of their parents had been adapted to that particular environment.<a class="footnoteRef" href="#fn14" id="fnref14"><sup>14</sup></a> The offspring will have lower viability rates as well as a lower overall fitness.<a class="footnoteRef" href="#fn15" id="fnref15"><sup>15</sup></a> This can also be referred to as an <a href="outbreeding_depression" title="wikilink">outbreeding depression</a>. This will create a higher load the more different the two environments are that the organisms are from, as well as if there is either a small population that is native, or a large population of immigrating organisms.<a class="footnoteRef" href="#fn16" id="fnref16"><sup>16</sup></a> This can also happen in asexually reproducing organisms if they come into the area. However, since the adapted and unadapted genes don‚Äôt mix, the effect won‚Äôt be as drastic. In either case, this means that there is often a steady selection against immigrant phenotypes and genotypes.<a class="footnoteRef" href="#fn17" id="fnref17"><sup>17</sup></a></p>
<h2 id="mathematics">Mathematics</h2>

<p>Consider a single <a href="gene_locus" title="wikilink">gene locus</a> with the <a href="allele" title="wikilink">alleles</a> 

<math display="inline" id="Genetic_load:0">
 <semantics>
  <mrow>
   <msub>
    <mi>ùêÄ</mi>
    <mn>1</mn>
   </msub>
   <mi mathvariant="normal">‚Ä¶</mi>
   <msub>
    <mi>ùêÄ</mi>
    <mi>n</mi>
   </msub>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <times></times>
    <apply>
     <csymbol cd="ambiguous">subscript</csymbol>
     <ci>ùêÄ</ci>
     <cn type="integer">1</cn>
    </apply>
    <ci>normal-‚Ä¶</ci>
    <apply>
     <csymbol cd="ambiguous">subscript</csymbol>
     <ci>ùêÄ</ci>
     <ci>n</ci>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \mathbf{A}_{1}\dots\mathbf{A}_{n}
  </annotation>
 </semantics>
</math>

, which have the <a href="fitness_(biology)" title="wikilink">fitnesses</a> 

<math display="inline" id="Genetic_load:1">
 <semantics>
  <mrow>
   <msub>
    <mi>w</mi>
    <mn>1</mn>
   </msub>
   <mi mathvariant="normal">‚Ä¶</mi>
   <msub>
    <mi>w</mi>
    <mi>n</mi>
   </msub>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <times></times>
    <apply>
     <csymbol cd="ambiguous">subscript</csymbol>
     <ci>w</ci>
     <cn type="integer">1</cn>
    </apply>
    <ci>normal-‚Ä¶</ci>
    <apply>
     <csymbol cd="ambiguous">subscript</csymbol>
     <ci>w</ci>
     <ci>n</ci>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   w_{1}\dots w_{n}
  </annotation>
 </semantics>
</math>

 and the <a href="allele_frequency" title="wikilink">allele frequencies</a> 

<math display="inline" id="Genetic_load:2">
 <semantics>
  <mrow>
   <msub>
    <mi>p</mi>
    <mn>1</mn>
   </msub>
   <mi mathvariant="normal">‚Ä¶</mi>
   <msub>
    <mi>p</mi>
    <mi>n</mi>
   </msub>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <times></times>
    <apply>
     <csymbol cd="ambiguous">subscript</csymbol>
     <ci>p</ci>
     <cn type="integer">1</cn>
    </apply>
    <ci>normal-‚Ä¶</ci>
    <apply>
     <csymbol cd="ambiguous">subscript</csymbol>
     <ci>p</ci>
     <ci>n</ci>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   p_{1}\dots p_{n}
  </annotation>
 </semantics>
</math>

 respectively. Ignoring <a href="frequency-dependent_selection" title="wikilink">frequency-dependent selection</a>, then genetic load (

<math display="inline" id="Genetic_load:3">
 <semantics>
  <mi>L</mi>
  <annotation-xml encoding="MathML-Content">
   <ci>L</ci>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   L
  </annotation>
 </semantics>
</math>


) may be calculated as:</p>

<p>

<math display="block" id="Genetic_load:4">
 <semantics>
  <mrow>
   <mi>L</mi>
   <mo>=</mo>
   <mrow>
    <mfrac>
     <mrow>
      <msub>
       <mi>w</mi>
       <mi>max</mi>
      </msub>
      <mo>-</mo>
      <mover accent="true">
       <mi>w</mi>
       <mo stretchy="false">¬Ø</mo>
      </mover>
     </mrow>
     <msub>
      <mi>w</mi>
      <mi>max</mi>
     </msub>
    </mfrac>
    <mrow>
     <mo stretchy="false">(</mo>
     <mn>1</mn>
     <mo stretchy="false">)</mo>
    </mrow>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <eq></eq>
    <ci>L</ci>
    <list>
     <apply>
      <divide></divide>
      <apply>
       <minus></minus>
       <apply>
        <csymbol cd="ambiguous">subscript</csymbol>
        <ci>w</ci>
        <max></max>
       </apply>
       <apply>
        <ci>normal-¬Ø</ci>
        <ci>w</ci>
       </apply>
      </apply>
      <apply>
       <csymbol cd="ambiguous">subscript</csymbol>
       <ci>w</ci>
       <max></max>
      </apply>
     </apply>
     <cn type="integer">1</cn>
    </list>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   L={{w_{\max}-\bar{w}}\over w_{\max}}~{}~{}~{}~{}~{}~{}~{}~{}~{}~{}(1)
  </annotation>
 </semantics>
</math>

</p>

<p>where 

<math display="inline" id="Genetic_load:5">
 <semantics>
  <msub>
   <mi>w</mi>
   <mi>max</mi>
  </msub>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <csymbol cd="ambiguous">subscript</csymbol>
    <ci>w</ci>
    <max></max>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   w_{\max}
  </annotation>
 </semantics>
</math>

 is the <a class="uri" href="maximum" title="wikilink">maximum</a> value of the fitnesses 

<math display="inline" id="Genetic_load:6">
 <semantics>
  <mrow>
   <msub>
    <mi>w</mi>
    <mn>1</mn>
   </msub>
   <mi mathvariant="normal">‚Ä¶</mi>
   <msub>
    <mi>w</mi>
    <mi>n</mi>
   </msub>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <times></times>
    <apply>
     <csymbol cd="ambiguous">subscript</csymbol>
     <ci>w</ci>
     <cn type="integer">1</cn>
    </apply>
    <ci>normal-‚Ä¶</ci>
    <apply>
     <csymbol cd="ambiguous">subscript</csymbol>
     <ci>w</ci>
     <ci>n</ci>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   w_{1}\dots w_{n}
  </annotation>
 </semantics>
</math>

 and 

<math display="inline" id="Genetic_load:7">
 <semantics>
  <mover accent="true">
   <mi>w</mi>
   <mo stretchy="false">¬Ø</mo>
  </mover>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <ci>normal-¬Ø</ci>
    <ci>w</ci>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \bar{w}
  </annotation>
 </semantics>
</math>

 is <a class="uri" href="mean" title="wikilink">mean</a> <a href="fitness_(biology)" title="wikilink">fitness</a> which is calculated as the mean of all the fitnesses weighted by their corresponding allele frequency:</p>

<p>

<math display="block" id="Genetic_load:8">
 <semantics>
  <mrow>
   <mover accent="true">
    <mi>w</mi>
    <mo stretchy="false">¬Ø</mo>
   </mover>
   <mo>=</mo>
   <mrow>
    <mrow>
     <munderover>
      <mo largeop="true" movablelimits="false" symmetric="true">‚àë</mo>
      <mrow>
       <mi>i</mi>
       <mo>=</mo>
       <mn>1</mn>
      </mrow>
      <mi>n</mi>
     </munderover>
     <mrow>
      <msub>
       <mi>p</mi>
       <mi>i</mi>
      </msub>
      <msub>
       <mi>w</mi>
       <mi>i</mi>
      </msub>
     </mrow>
    </mrow>
    <mrow>
     <mo stretchy="false">(</mo>
     <mn>2</mn>
     <mo stretchy="false">)</mo>
    </mrow>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <eq></eq>
    <apply>
     <ci>normal-¬Ø</ci>
     <ci>w</ci>
    </apply>
    <list>
     <apply>
      <apply>
       <csymbol cd="ambiguous">superscript</csymbol>
       <apply>
        <csymbol cd="ambiguous">subscript</csymbol>
        <sum></sum>
        <apply>
         <eq></eq>
         <ci>i</ci>
         <cn type="integer">1</cn>
        </apply>
       </apply>
       <ci>n</ci>
      </apply>
      <apply>
       <times></times>
       <apply>
        <csymbol cd="ambiguous">subscript</csymbol>
        <ci>p</ci>
        <ci>i</ci>
       </apply>
       <apply>
        <csymbol cd="ambiguous">subscript</csymbol>
        <ci>w</ci>
        <ci>i</ci>
       </apply>
      </apply>
     </apply>
     <cn type="integer">2</cn>
    </list>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \bar{w}={\sum_{i=1}^{n}{p_{i}w_{i}}}~{}~{}~{}~{}~{}~{}~{}~{}~{}~{}(2)
  </annotation>
 </semantics>
</math>

</p>

<p>where the 

<math display="inline" id="Genetic_load:9">
 <semantics>
  <msup>
   <mi>i</mi>
   <mi>th</mi>
  </msup>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <csymbol cd="ambiguous">superscript</csymbol>
    <ci>i</ci>
    <ci>th</ci>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   i^{\mathrm{th}}
  </annotation>
 </semantics>
</math>

 allele is 

<math display="inline" id="Genetic_load:10">
 <semantics>
  <msub>
   <mi>ùêÄ</mi>
   <mi>i</mi>
  </msub>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <csymbol cd="ambiguous">subscript</csymbol>
    <ci>ùêÄ</ci>
    <ci>i</ci>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \mathbf{A}_{i}
  </annotation>
 </semantics>
</math>

 and has the fitness and frequency 

<math display="inline" id="Genetic_load:11">
 <semantics>
  <msub>
   <mi>w</mi>
   <mi>i</mi>
  </msub>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <csymbol cd="ambiguous">subscript</csymbol>
    <ci>w</ci>
    <ci>i</ci>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   w_{i}
  </annotation>
 </semantics>
</math>

 and 

<math display="inline" id="Genetic_load:12">
 <semantics>
  <msub>
   <mi>p</mi>
   <mi>i</mi>
  </msub>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <csymbol cd="ambiguous">subscript</csymbol>
    <ci>p</ci>
    <ci>i</ci>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   p_{i}
  </annotation>
 </semantics>
</math>

 respectively.</p>

<p>When the 

<math display="inline" id="Genetic_load:13">
 <semantics>
  <mrow>
   <msub>
    <mi>w</mi>
    <mi>max</mi>
   </msub>
   <mo>=</mo>
   <mn>1</mn>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <eq></eq>
    <apply>
     <csymbol cd="ambiguous">subscript</csymbol>
     <ci>w</ci>
     <max></max>
    </apply>
    <cn type="integer">1</cn>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   w_{\max}=1
  </annotation>
 </semantics>
</math>


, then (1) simplifies to</p>

<p>

<math display="block" id="Genetic_load:14">
 <semantics>
  <mrow>
   <mi>L</mi>
   <mo>=</mo>
   <mn>1</mn>
   <mo>-</mo>
   <mover accent="true">
    <mi>w</mi>
    <mo stretchy="false">¬Ø</mo>
   </mover>
   <mo>.</mo>
   <mrow>
    <mo stretchy="false">(</mo>
    <mn>3</mn>
    <mo stretchy="false">)</mo>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <cerror>
    <csymbol cd="ambiguous">fragments</csymbol>
    <csymbol cd="unknown">L</csymbol>
    <eq></eq>
    <cn type="integer">1</cn>
    <minus></minus>
    <apply>
     <ci>normal-¬Ø</ci>
     <ci>w</ci>
    </apply>
    <ci>normal-.</ci>
    <ci>italic-</ci>
    <cerror>
     <csymbol cd="ambiguous">fragments</csymbol>
     <ci>normal-(</ci>
     <cn type="integer">3</cn>
     <ci>normal-)</ci>
    </cerror>
   </cerror>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   L=1-\bar{w}.~{}~{}~{}~{}~{}~{}~{}~{}~{}~{}(3)
  </annotation>
 </semantics>
</math>

</p>
<h2 id="problems">Problems</h2>

<p>One problem with calculating genetic load is that in order to do so you have to a have a ‚Äúperfect‚Äù or ‚Äúoptimal‚Äù genotype with which to compare the population to; this kind of genotype simply does not exist. This is problem because it means that it is harder for scientists to gauge with accuracy how much load a population has, and how much load it can bear without being in danger. This means that all perceptions of genetic load should be taken with a grain of salt.<a class="footnoteRef" href="#fn18" id="fnref18"><sup>18</sup></a></p>
<h2 id="references">References</h2>

<p>"</p>

<p><a href="Category:Population_genetics" title="wikilink">Category:Population genetics</a></p>
<section class="footnotes">
<hr/>
<ol>
<li id="fn1">Saccheri I., Lloyd H., Helyar S., Brakefield P. 2005. Inbreeding uncovers fundamental differences in the genetic load affecting male and female fertility in a butterfly, Proceedings of the Royal Society 272:1.<a href="#fnref1">‚Ü©</a></li>
<li id="fn2">Crist K., Farrar D. 1982. Genetic load and long- distance dispersal in Asplenium platyneruon, Can. J. Botany 61:1.<a href="#fnref2">‚Ü©</a></li>
<li id="fn3"><a href="#fnref3">‚Ü©</a></li>
<li id="fn4">Crist K., Farrar D. 1982. Genetic load and long- distance dispersal in Asplenium platyneruon, Can. J. Botany 61:1.<a href="#fnref4">‚Ü©</a></li>
<li id="fn5">Klekowski E. 1988. Genetic load and its causes in long- lived plants, Trees 2:4.<a href="#fnref5">‚Ü©</a></li>
<li id="fn6">Byers D., Waller D. 1999. Do Plant Populations Purge Their Genetic Load? Effects of Population Size and Mating, Annual Review of Ecology and Systematics 30:1.<a href="#fnref6">‚Ü©</a></li>
<li id="fn7">Barrett S., Charlesworth D. 1991. Effects of a change in the level of inbreeding on the genetic load, Letters to Nature 352:1.<a href="#fnref7">‚Ü©</a></li>
<li id="fn8">King J. 1965. The gene interaction component of the genetic load, Donner Laboratory and Lawrence Radiation Laboratory, University of California, Berkeley<a href="#fnref8">‚Ü©</a></li>
<li id="fn9">Zimmer C., Emlen D. 2013. Evolution: Making Sense of Life- Sex: Causes and Consequences.<a href="#fnref9">‚Ü©</a></li>
<li id="fn10">Takhtajan A. 2009. Flowering Plants: Asexual vs. Sexual Reproduction: Advantages and Disadvantages. Springer Verlag: New York.<a href="#fnref10">‚Ü©</a></li>
<li id="fn11">Marriage T. 2009. Mutation, asexual reproduction and genetic load: A study in three parts, University of Kansas.<a href="#fnref11">‚Ü©</a></li>
<li id="fn12">Haag C., Roze D. 2007. Genetic Load in Sexual and Asexual Diploids: Segregation, Dominance and Genetic Drift, Genetics 176:3.<a href="#fnref12">‚Ü©</a></li>
<li id="fn13">Haag C., Roze D. 2007. Genetic Load in Sexual and Asexual Diploids: Segregation, Dominance and Genetic Drift, Genetics 176:3.<a href="#fnref13">‚Ü©</a></li>
<li id="fn14">Bolnick D., Nosil P. 2007. Natural selection in populations subject to a migration load, Evolution 61:9.<a href="#fnref14">‚Ü©</a></li>
<li id="fn15">Hu X., Li B. 2003. On migration load of seeds and pollen grains in a local population, Heredity 90:1.<a href="#fnref15">‚Ü©</a></li>
<li id="fn16">Hanski,I., Gaggiotti O. 2004. Ecology, Genetics, and Evolution of Metapopulations. Elsevier Academic Press: London, UK.<a href="#fnref16">‚Ü©</a></li>
<li id="fn17">Bolnick D., Nosil P. 2007. Natural selection in populations subject to a migration load, Evolution 61:9.<a href="#fnref17">‚Ü©</a></li>
<li id="fn18">Agrawal A., Whitlock M. 2012. Mutation Load: The Fitness of Individuals in Populations Where Deleterious Alleles Are Abundant, Annual Review of Ecology, Evolution, and Systematics 43:1.<a href="#fnref18">‚Ü©</a></li>
</ol>
</section>
</ref></hr></body>
</html>
