<html lang="en">
<head>
<meta charset="utf-8"/>
<title offset="304">Gene regulatory network</title>
<script src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_SVG.js" type="text/javascript">
</script>
</head>
<body>
<h1>Gene regulatory network</h1>
<hr/>
<figure><b>(Figure)</b>
<figcaption>Structure of a gene regulatory network</figcaption>
</figure>
<figure><b>(Figure)</b>
<figcaption>Control process of a gene regulatory network</figcaption>
</figure>

<p>A <strong><a class="uri" href="gene" title="wikilink">gene</a> regulatory network</strong> or <strong>genetic regulatory network (GRN)</strong> is a collection of regulators that interact with each other and with other substances in the cell to govern the gene expression levels of <a class="uri" href="mRNA" title="wikilink">mRNA</a> and proteins. The regulator can be <a class="uri" href="DNA" title="wikilink">DNA</a>, <a class="uri" href="RNA" title="wikilink">RNA</a>, <a class="uri" href="protein" title="wikilink">protein</a> and their complex. The interaction can be direct or indirect (through their transcribed RNA or translated protein). In general, each mRNA molecule goes on to make a specific protein (or set of proteins). In some cases this protein will be <a href="Protein#Structural_proteins" title="wikilink">structural</a>, and will accumulate at the cell membrane or within the cell to give it particular structural properties. In other cases the protein will be an <a class="uri" href="enzyme" title="wikilink">enzyme</a>, i.e., a micro-machine that catalyses a certain reaction, such as the breakdown of a food source or toxin. Some proteins though serve only to activate other genes, and these are the <a href="transcription_factors" title="wikilink">transcription factors</a> that are the main players in regulatory networks or cascades. By binding to the <a href="promoter_(biology)" title="wikilink">promoter</a> region at the start of other genes they turn them on, initiating the production of another protein, and so on. Some transcription factors are inhibitory.</p>

<p>In single-celled organisms, regulatory networks respond to the external environment, optimising the cell at a given time for survival in this environment. Thus a yeast cell, finding itself in a sugar solution, will turn on genes to make enzymes that process the sugar to alcohol.<a class="footnoteRef" href="#fn1" id="fnref1"><sup>1</sup></a> This process, which we associate with wine-making, is how the yeast cell makes its living, gaining energy to multiply, which under normal circumstances would enhance its survival prospects.</p>

<p>In multicellular animals the same principle has been put in the service of gene cascades that control body-shape.<a class="footnoteRef" href="#fn2" id="fnref2"><sup>2</sup></a> Each time a cell divides, two cells result which, although they contain the same genome in full, can differ in which genes are turned on and making proteins. Sometimes a 'self-sustaining feedback loop' ensures that a cell maintains its identity and passes it on. Less understood is the mechanism of <a class="uri" href="epigenetics" title="wikilink">epigenetics</a> by which <a class="uri" href="chromatin" title="wikilink">chromatin</a> modification may provide cellular memory by blocking or allowing transcription. A major feature of multicellular animals is the use of <a class="uri" href="morphogen" title="wikilink">morphogen</a> gradients, which in effect provide a positioning system that tells a cell where in the body it is, and hence what sort of cell to become. A gene that is turned on in one cell may make a product that leaves the cell and <a href="diffuse" title="wikilink">diffuses</a> through adjacent cells, entering them and turning on genes only when it is present above a certain threshold level. These cells are thus induced into a new fate, and may even generate other <a class="uri" href="morphogens" title="wikilink">morphogens</a> that signal back to the original cell. Over longer distances morphogens may use the active process of <a href="signal_transduction" title="wikilink">signal transduction</a>. Such signalling controls <a class="uri" href="embryogenesis" title="wikilink">embryogenesis</a>, the building of a <a href="body_plan" title="wikilink">body plan</a> from scratch through a series of sequential steps. They also control and maintain adult bodies through <a class="uri" href="feedback" title="wikilink">feedback</a> processes, and the loss of such feedback because of a mutation can be responsible for the cell proliferation that is seen in <a class="uri" href="cancer" title="wikilink">cancer</a>. In parallel with this process of building structure, the gene cascade turns on genes that make <a href="Protein#Structural_proteins" title="wikilink">structural proteins</a> that give each cell the physical properties it needs.</p>

<p>It has been suggested that, because biological molecular interactions are intrinsically stochastic, gene networks are the result of cellular processes and not their cause (i.e. cellular Darwinism). However, recent experimental evidence has favored the attractor view of cell fates.</p>
<h2 id="overview">Overview</h2>

<p>At one level, biological cells can be thought of as "partially mixed bags" of biological chemicals – in the discussion of gene regulatory networks, these chemicals are mostly the mRNAs and proteins that arise from gene expression. These mRNA and proteins interact with each other with various degrees of specificity. Some diffuse around the cell. Others are bound to cell membranes, interacting with molecules in the environment. Still others pass through cell membranes and mediate long range signals to other cells in a multi-cellular organism. These molecules and their interactions comprise a <em>gene regulatory network</em>. A typical gene regulatory network looks something like this:</p>
<figure><b>(Figure)</b>
<figcaption>Example of a regulatory network built in Pathway Studio</figcaption>
</figure>

<p>The nodes of this network are proteins, their corresponding mRNAs, and protein/protein complexes. Nodes that are depicted as lying along vertical lines are associated with the cell/environment interfaces, while the others are free-floating and diffusible. Implied are genes, the DNA sequences which are transcribed into the mRNAs that translate into proteins. Edges between nodes represent individual molecular reactions, the protein/protein and protein/mRNA interactions through which the products of one gene affect those of another, though the lack of experimentally obtained information often implies that some reactions are not modeled at such a fine level of detail. These interactions can be inductive (the arrowheads), with an increase in the concentration of one leading to an increase in the other, or inhibitory (the filled circles), with an increase in one leading to a decrease in the other. A series of edges indicates a chain of such dependences, with cycles corresponding to feedback loops. The network structure is an abstraction of the system's chemical dynamics, describing the manifold ways in which one substance affects all the others to which it is connected. In practice, such GRNs are inferred from the biological literature on a given system and represent a distillation of the collective knowledge about a set of related biochemical reactions. To speed up the manual curation of GRNs, some recent efforts try to use text mining and information extraction technologies for this purpose. <a class="footnoteRef" href="#fn3" id="fnref3"><sup>3</sup></a></p>

<p>Genes can be viewed as nodes in the network, with input being proteins such as <a href="transcription_factor" title="wikilink">transcription factors</a>, and outputs being the level of <a href="gene_expression" title="wikilink">gene expression</a>. The node itself can also be viewed as a function which can be obtained by combining basic functions upon the inputs (in the Boolean network described below these are <a href="Boolean_functions" title="wikilink">Boolean functions</a>, typically AND, OR, and NOT). These functions have been interpreted as performing a kind of <a href="information_processing" title="wikilink">information processing</a> within the cell, which determines cellular behavior. The basic drivers within cells are concentrations of some proteins, which determine both spatial (location within the cell or tissue) and temporal (cell cycle or developmental stage) coordinates of the cell, as a kind of "cellular memory". The gene networks are only beginning to be understood, and it is a next step for biology to attempt to deduce the functions for each gene "node", to help understand the behavior of the system in increasing levels of complexity, from gene to signaling pathway, cell or tissue level (see <a href="systems_biology" title="wikilink">systems biology</a>).</p>

<p><a href="Mathematical_model" title="wikilink">Mathematical models</a> of GRNs have been developed to capture the behavior of the system being modeled, and in some cases generate predictions corresponding with experimental observations. In some other cases, models have proven to make accurate novel predictions, which can be tested experimentally, thus suggesting new approaches to explore in an experiment that sometimes wouldn't be considered in the design of the protocol of an experimental laboratory. The most common modeling technique involves the use of coupled ordinary <a href="differential_equation" title="wikilink">differential equations</a> (ODEs). Several other promising modeling techniques have been used, including <a href="Boolean_network" title="wikilink">Boolean networks</a>, <a href="Petri_net" title="wikilink">Petri nets</a>, <a href="Bayesian_network" title="wikilink">Bayesian networks</a>, <a href="graphical_Gaussian_model" title="wikilink">graphical Gaussian models</a>, <a class="uri" href="Stochastic" title="wikilink">Stochastic</a>, and <a href="Process_calculus" title="wikilink">Process Calculi</a>. Conversely, techniques have been proposed for generating models of GRNs that best explain a set of <a href="time_series" title="wikilink">time series</a> observations.Recently it has been shown that ChIP-seq signal of Histone modification are more correlated with transcription factor motifs at promoters in comparison to RNA level.<a class="footnoteRef" href="#fn4" id="fnref4"><sup>4</sup></a> Hence it is proposed that time-series histone modification ChIP-seq could provide more reliable inference of gene-regulatory networks in comparison to methods based on expression levels.</p>
<h2 id="modelling">Modelling</h2>
<h3 id="coupled-odes">Coupled ODEs</h3>

<p>It is common to model such a network with a set of coupled <a href="ordinary_differential_equation" title="wikilink">ordinary differential equations</a> (ODEs) or <a href="Stochastic_differential_equation" title="wikilink">stochastic ODEs</a>, describing the reaction kinetics of the constituent parts. Suppose that our regulatory network has 

<math display="inline" id="Gene_regulatory_network:0">
 <semantics>
  <mi>N</mi>
  <annotation-xml encoding="MathML-Content">
   <ci>N</ci>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   N
  </annotation>
 </semantics>
</math>

 nodes, and let 

<math display="inline" id="Gene_regulatory_network:1">
 <semantics>
  <mrow>
   <mrow>
    <msub>
     <mi>S</mi>
     <mn>1</mn>
    </msub>
    <mrow>
     <mo stretchy="false">(</mo>
     <mi>t</mi>
     <mo stretchy="false">)</mo>
    </mrow>
   </mrow>
   <mo>,</mo>
   <mrow>
    <msub>
     <mi>S</mi>
     <mn>2</mn>
    </msub>
    <mrow>
     <mo stretchy="false">(</mo>
     <mi>t</mi>
     <mo stretchy="false">)</mo>
    </mrow>
   </mrow>
   <mo>,</mo>
   <mi mathvariant="normal">…</mi>
   <mo>,</mo>
   <mrow>
    <msub>
     <mi>S</mi>
     <mi>N</mi>
    </msub>
    <mrow>
     <mo stretchy="false">(</mo>
     <mi>t</mi>
     <mo stretchy="false">)</mo>
    </mrow>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <list>
    <apply>
     <times></times>
     <apply>
      <csymbol cd="ambiguous">subscript</csymbol>
      <ci>S</ci>
      <cn type="integer">1</cn>
     </apply>
     <ci>t</ci>
    </apply>
    <apply>
     <times></times>
     <apply>
      <csymbol cd="ambiguous">subscript</csymbol>
      <ci>S</ci>
      <cn type="integer">2</cn>
     </apply>
     <ci>t</ci>
    </apply>
    <ci>normal-…</ci>
    <apply>
     <times></times>
     <apply>
      <csymbol cd="ambiguous">subscript</csymbol>
      <ci>S</ci>
      <ci>N</ci>
     </apply>
     <ci>t</ci>
    </apply>
   </list>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   S_{1}(t),S_{2}(t),\ldots,S_{N}(t)
  </annotation>
 </semantics>
</math>

 represent the concentrations of the 

<math display="inline" id="Gene_regulatory_network:2">
 <semantics>
  <mi>N</mi>
  <annotation-xml encoding="MathML-Content">
   <ci>N</ci>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   N
  </annotation>
 </semantics>
</math>

 corresponding substances at time 

<math display="inline" id="Gene_regulatory_network:3">
 <semantics>
  <mi>t</mi>
  <annotation-xml encoding="MathML-Content">
   <ci>t</ci>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   t
  </annotation>
 </semantics>
</math>

. Then the temporal evolution of the system can be described approximately by</p>

<p>

<math display="block" id="Gene_regulatory_network:4">
 <semantics>
  <mrow>
   <mfrac>
    <mrow>
     <mi>d</mi>
     <msub>
      <mi>S</mi>
      <mi>j</mi>
     </msub>
    </mrow>
    <mrow>
     <mi>d</mi>
     <mi>t</mi>
    </mrow>
   </mfrac>
   <mo>=</mo>
   <mrow>
    <msub>
     <mi>f</mi>
     <mi>j</mi>
    </msub>
    <mrow>
     <mo>(</mo>
     <msub>
      <mi>S</mi>
      <mn>1</mn>
     </msub>
     <mo>,</mo>
     <msub>
      <mi>S</mi>
      <mn>2</mn>
     </msub>
     <mo>,</mo>
     <mi mathvariant="normal">…</mi>
     <mo>,</mo>
     <msub>
      <mi>S</mi>
      <mi>N</mi>
     </msub>
     <mo>)</mo>
    </mrow>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <eq></eq>
    <apply>
     <divide></divide>
     <apply>
      <times></times>
      <ci>d</ci>
      <apply>
       <csymbol cd="ambiguous">subscript</csymbol>
       <ci>S</ci>
       <ci>j</ci>
      </apply>
     </apply>
     <apply>
      <times></times>
      <ci>d</ci>
      <ci>t</ci>
     </apply>
    </apply>
    <apply>
     <times></times>
     <apply>
      <csymbol cd="ambiguous">subscript</csymbol>
      <ci>f</ci>
      <ci>j</ci>
     </apply>
     <vector>
      <apply>
       <csymbol cd="ambiguous">subscript</csymbol>
       <ci>S</ci>
       <cn type="integer">1</cn>
      </apply>
      <apply>
       <csymbol cd="ambiguous">subscript</csymbol>
       <ci>S</ci>
       <cn type="integer">2</cn>
      </apply>
      <ci>normal-…</ci>
      <apply>
       <csymbol cd="ambiguous">subscript</csymbol>
       <ci>S</ci>
       <ci>N</ci>
      </apply>
     </vector>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \frac{dS_{j}}{dt}=f_{j}\left(S_{1},S_{2},\ldots,S_{N}\right)
  </annotation>
 </semantics>
</math>

</p>

<p>where the functions 

<math display="inline" id="Gene_regulatory_network:5">
 <semantics>
  <msub>
   <mi>f</mi>
   <mi>j</mi>
  </msub>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <csymbol cd="ambiguous">subscript</csymbol>
    <ci>f</ci>
    <ci>j</ci>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   f_{j}
  </annotation>
 </semantics>
</math>

 express the dependence of 

<math display="inline" id="Gene_regulatory_network:6">
 <semantics>
  <msub>
   <mi>S</mi>
   <mi>j</mi>
  </msub>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <csymbol cd="ambiguous">subscript</csymbol>
    <ci>S</ci>
    <ci>j</ci>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   S_{j}
  </annotation>
 </semantics>
</math>

 on the concentrations of other substances present in the cell. The functions 

<math display="inline" id="Gene_regulatory_network:7">
 <semantics>
  <msub>
   <mi>f</mi>
   <mi>j</mi>
  </msub>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <csymbol cd="ambiguous">subscript</csymbol>
    <ci>f</ci>
    <ci>j</ci>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   f_{j}
  </annotation>
 </semantics>
</math>

 are ultimately derived from basic <a href="rate_equation" title="wikilink">principles of chemical kinetics</a> or simple expressions derived from these e.g. <a class="uri" href="Michaelis-Menten" title="wikilink">Michaelis-Menten</a> enzymatic kinetics. Hence, the functional forms of the 

<math display="inline" id="Gene_regulatory_network:8">
 <semantics>
  <msub>
   <mi>f</mi>
   <mi>j</mi>
  </msub>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <csymbol cd="ambiguous">subscript</csymbol>
    <ci>f</ci>
    <ci>j</ci>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   f_{j}
  </annotation>
 </semantics>
</math>

 are usually chosen as low-order <a class="uri" href="polynomials" title="wikilink">polynomials</a> or <a href="Hill_equation_(biochemistry)" title="wikilink">Hill functions</a> that serve as an <a class="uri" href="ansatz" title="wikilink">ansatz</a> for the real molecular dynamics. Such models are then studied using the mathematics of <a href="dynamical_system" title="wikilink">nonlinear dynamics</a>. System-specific information, like <a href="reaction_rate" title="wikilink">reaction rate</a> constants and sensitivities, are encoded as constant parameters.<a class="footnoteRef" href="#fn5" id="fnref5"><sup>5</sup></a></p>

<p>By solving for the <a href="Fixed_point_(mathematics)" title="wikilink">fixed point</a> of the system:</p>

<p>

<math display="block" id="Gene_regulatory_network:9">
 <semantics>
  <mrow>
   <mfrac>
    <mrow>
     <mi>d</mi>
     <msub>
      <mi>S</mi>
      <mi>j</mi>
     </msub>
    </mrow>
    <mrow>
     <mi>d</mi>
     <mi>t</mi>
    </mrow>
   </mfrac>
   <mo>=</mo>
   <mn>0</mn>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <eq></eq>
    <apply>
     <divide></divide>
     <apply>
      <times></times>
      <ci>d</ci>
      <apply>
       <csymbol cd="ambiguous">subscript</csymbol>
       <ci>S</ci>
       <ci>j</ci>
      </apply>
     </apply>
     <apply>
      <times></times>
      <ci>d</ci>
      <ci>t</ci>
     </apply>
    </apply>
    <cn type="integer">0</cn>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \frac{dS_{j}}{dt}=0
  </annotation>
 </semantics>
</math>

</p>

<p>for all 

<math display="inline" id="Gene_regulatory_network:10">
 <semantics>
  <mi>j</mi>
  <annotation-xml encoding="MathML-Content">
   <ci>j</ci>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   j
  </annotation>
 </semantics>
</math>

, one obtains (possibly several) concentration profiles of proteins and mRNAs that are theoretically sustainable (though not necessarily <a href="stability_(mathematics)" title="wikilink">stable</a>). <a href="Steady_state" title="wikilink">Steady states</a> of kinetic equations thus correspond to potential cell types, and <a href="Oscillation" title="wikilink">oscillatory</a> solutions to the above equation to naturally cyclic cell types. Mathematical stability of these <a href="attractor" title="wikilink">attractors</a> can usually be characterized by the sign of higher derivatives at critical points, and then correspond to <a href="Steady_state_(biochemistry)" title="wikilink">biochemical stability</a> of the concentration profile. <a href="Critical_point_(mathematics)" title="wikilink">Critical points</a> and <a href="Bifurcation_theory" title="wikilink">bifurcations</a> in the equations correspond to critical cell states in which small state or parameter perturbations could switch the system between one of several stable differentiation fates. Trajectories correspond to the unfolding of biological pathways and transients of the equations to short-term biological events. For a more mathematical discussion, see the articles on <a class="uri" href="nonlinearity" title="wikilink">nonlinearity</a>, <a href="dynamical_systems" title="wikilink">dynamical systems</a>, <a href="bifurcation_theory" title="wikilink">bifurcation theory</a>, and <a href="chaos_theory" title="wikilink">chaos theory</a>.</p>
<h3 id="boolean-network">Boolean network</h3>

<p>The following example illustrates how a <a href="Boolean_network" title="wikilink">Boolean network</a> can model a GRN together with its gene products (the outputs) and the substances from the environment that affect it (the inputs). <a href="Stuart_Kauffman" title="wikilink">Stuart Kauffman</a> was amongst the first biologists to use the metaphor of Boolean networks to model genetic regulatory networks.<a class="footnoteRef" href="#fn6" id="fnref6"><sup>6</sup></a><a class="footnoteRef" href="#fn7" id="fnref7"><sup>7</sup></a></p>
<ol>
<li>Each gene, each input, and each output is represented by a node in a <a href="directed_graph" title="wikilink">directed graph</a> in which there is an arrow from one node to another if and only if there is a causal link between the two nodes.</li>
<li>Each node in the graph can be in one of two states: on or off.</li>
<li>For a gene, "on" corresponds to the gene being expressed; for inputs and outputs, "off" corresponds to the substance being present.</li>
<li>Time is viewed as proceeding in discrete steps. At each step, the new state of a node is a <a href="Boolean_function" title="wikilink">Boolean function</a> of the prior states of the nodes with arrows pointing towards it.</li>
</ol>

<p>The validity of the model can be tested by comparing simulation results with time series observations. A partial validation of a Boolean network model can also come from testing the predicted existence of a yet unknown regulatory connection between two particular transcription factors that each are nodes of the model. <a class="footnoteRef" href="#fn8" id="fnref8"><sup>8</sup></a></p>
<h3 id="continuous-networks">Continuous networks</h3>

<p>Continuous network models of GRNs are an extension of the boolean networks described above. Nodes still represent genes and connections between them regulatory influences on gene expression. Genes in biological systems display a continuous range of activity levels and it has been argued that using a continuous representation captures several properties of gene regulatory networks not present in the Boolean model.<a class="footnoteRef" href="#fn9" id="fnref9"><sup>9</sup></a> Formally most of these approaches are similar to an <a href="artificial_neural_network" title="wikilink">artificial neural network</a>, as inputs to a node are summed up and the result serves as input to a sigmoid function, e.g.,<a class="footnoteRef" href="#fn10" id="fnref10"><sup>10</sup></a> but proteins do often control gene expression in a synergistic, i.e. non-linear, way.<a class="footnoteRef" href="#fn11" id="fnref11"><sup>11</sup></a> However there is now a continuous network model<a class="footnoteRef" href="#fn12" id="fnref12"><sup>12</sup></a> that allows grouping of inputs to a node thus realizing another level of regulation. This model is formally closer to a higher order <a href="recurrent_neural_network" title="wikilink">recurrent neural network</a>. The same model has also been used to mimic the evolution of <a href="cellular_differentiation" title="wikilink">cellular differentiation</a><a class="footnoteRef" href="#fn13" id="fnref13"><sup>13</sup></a> and even multicellular <a class="uri" href="morphogenesis" title="wikilink">morphogenesis</a>.<a class="footnoteRef" href="#fn14" id="fnref14"><sup>14</sup></a></p>
<h3 id="stochastic-gene-networks">Stochastic gene networks</h3>

<p>Recent experimental results<a class="footnoteRef" href="#fn15" id="fnref15"><sup>15</sup></a> <a class="footnoteRef" href="#fn16" id="fnref16"><sup>16</sup></a> have demonstrated that gene expression is a stochastic process. Thus, many authors are now using the stochastic formalism, after the work by.<a class="footnoteRef" href="#fn17" id="fnref17"><sup>17</sup></a> Works on single gene expression<a class="footnoteRef" href="#fn18" id="fnref18"><sup>18</sup></a> and small synthetic genetic networks,<a class="footnoteRef" href="#fn19" id="fnref19"><sup>19</sup></a><a class="footnoteRef" href="#fn20" id="fnref20"><sup>20</sup></a> such as the genetic toggle switch of Tim Gardner and <a href="James_Collins_(bioengineer)" title="wikilink">Jim Collins</a>, provided additional experimental data on the phenotypic variability and the stochastic nature of gene expression. The first versions of stochastic models of gene expression involved only instantaneous reactions and were driven by the <a href="Gillespie_algorithm" title="wikilink">Gillespie algorithm</a>.<a class="footnoteRef" href="#fn21" id="fnref21"><sup>21</sup></a></p>

<p>Since some processes, such as gene transcription, involve many reactions and could not be correctly modeled as an instantaneous reaction in a single step, it was proposed to model these reactions as single step multiple delayed reactions in order to account for the time it takes for the entire process to be complete.<a class="footnoteRef" href="#fn22" id="fnref22"><sup>22</sup></a></p>

<p>From here, a set of reactions were proposed<a class="footnoteRef" href="#fn23" id="fnref23"><sup>23</sup></a> that allow generating GRNs. These are then simulated using a modified version of the Gillespie algorithm, that can simulate multiple time delayed reactions (chemical reactions where each of the products is provided a time delay that determines when will it be released in the system as a "finished product").</p>

<p>For example, basic transcription of a gene can be represented by the following single-step reaction (RNAP is the RNA polymerase, RBS is the RNA ribosome binding site, and Pro<sub> <em>i</em></sub> is the promoter region of gene <em>i</em>):</p>

<p>

<math display="block" id="Gene_regulatory_network:11">
 <semantics>
  <mrow>
   <mrow>
    <mtext>RNAP</mtext>
    <mo>+</mo>
    <mrow>
     <msub>
      <mtext>Pro</mtext>
      <mi>i</mi>
     </msub>
     <mover accent="true">
      <mi></mi>
      <msub>
       <mi>k</mi>
       <mrow>
        <mi>i</mi>
        <mo>,</mo>
        <mrow>
         <mi>b</mi>
         <mi>a</mi>
         <mi>s</mi>
        </mrow>
       </mrow>
      </msub>
     </mover>
    </mrow>
   </mrow>
   <mo>⟶</mo>
   <mrow>
    <mrow>
     <msub>
      <mtext>Pro</mtext>
      <mi>i</mi>
     </msub>
     <mrow>
      <mo stretchy="false">(</mo>
      <msubsup>
       <mi>τ</mi>
       <mi>i</mi>
       <mn>1</mn>
      </msubsup>
      <mo stretchy="false">)</mo>
     </mrow>
    </mrow>
    <mo>+</mo>
    <mrow>
     <msub>
      <mtext>RBS</mtext>
      <mi>i</mi>
     </msub>
     <mrow>
      <mo stretchy="false">(</mo>
      <msubsup>
       <mi>τ</mi>
       <mi>i</mi>
       <mn>1</mn>
      </msubsup>
      <mo stretchy="false">)</mo>
     </mrow>
    </mrow>
    <mo>+</mo>
    <mrow>
     <mtext>RNAP</mtext>
     <mrow>
      <mo stretchy="false">(</mo>
      <msubsup>
       <mi>τ</mi>
       <mi>i</mi>
       <mn>2</mn>
      </msubsup>
      <mo stretchy="false">)</mo>
     </mrow>
    </mrow>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <ci>normal-⟶</ci>
    <apply>
     <plus></plus>
     <mtext>RNAP</mtext>
     <apply>
      <times></times>
      <apply>
       <csymbol cd="ambiguous">subscript</csymbol>
       <mtext>Pro</mtext>
       <ci>i</ci>
      </apply>
      <apply>
       <apply>
        <csymbol cd="ambiguous">subscript</csymbol>
        <ci>k</ci>
        <list>
         <ci>i</ci>
         <apply>
          <times></times>
          <ci>b</ci>
          <ci>a</ci>
          <ci>s</ci>
         </apply>
        </list>
       </apply>
       <csymbol cd="latexml">absent</csymbol>
      </apply>
     </apply>
    </apply>
    <apply>
     <plus></plus>
     <apply>
      <times></times>
      <apply>
       <csymbol cd="ambiguous">subscript</csymbol>
       <mtext>Pro</mtext>
       <ci>i</ci>
      </apply>
      <apply>
       <csymbol cd="ambiguous">superscript</csymbol>
       <apply>
        <csymbol cd="ambiguous">subscript</csymbol>
        <ci>τ</ci>
        <ci>i</ci>
       </apply>
       <cn type="integer">1</cn>
      </apply>
     </apply>
     <apply>
      <times></times>
      <apply>
       <csymbol cd="ambiguous">subscript</csymbol>
       <mtext>RBS</mtext>
       <ci>i</ci>
      </apply>
      <apply>
       <csymbol cd="ambiguous">superscript</csymbol>
       <apply>
        <csymbol cd="ambiguous">subscript</csymbol>
        <ci>τ</ci>
        <ci>i</ci>
       </apply>
       <cn type="integer">1</cn>
      </apply>
     </apply>
     <apply>
      <times></times>
      <mtext>RNAP</mtext>
      <apply>
       <csymbol cd="ambiguous">superscript</csymbol>
       <apply>
        <csymbol cd="ambiguous">subscript</csymbol>
        <ci>τ</ci>
        <ci>i</ci>
       </apply>
       <cn type="integer">2</cn>
      </apply>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \text{RNAP}+\text{Pro}_{i}\overset{k_{i,bas}}{\par
}\longrightarrow\text{Pro}_%
{i}(\tau_{i}^{1})+\text{RBS}_{i}(\tau_{i}^{1})+\text{RNAP}(\tau_{i}^{2})
  </annotation>
 </semantics>
</math>

</p>

<p>Furthermore, there seems to be a trade-off between the noise in gene expression, the speed with which genes can switch, and the metabolic cost associated their functioning. More specifically, for any given level of metabolic cost, there is an optimal trade-off between noise and processing speed and increasing the metabolic cost leads to better speed-noise trade-offs.<a class="footnoteRef" href="#fn24" id="fnref24"><sup>24</sup></a><a class="footnoteRef" href="#fn25" id="fnref25"><sup>25</sup></a><a class="footnoteRef" href="#fn26" id="fnref26"><sup>26</sup></a></p>

<p>A recent work proposed a simulator (SGNSim, <em>Stochastic Gene Networks Simulator</em>),<a class="footnoteRef" href="#fn27" id="fnref27"><sup>27</sup></a> that can model GRNs where transcription and translation are modeled as multiple time delayed events and its dynamics is driven by a stochastic simulation algorithm (SSA) able to deal with multiple time delayed events. The time delays can be drawn from several distributions and the reaction rates from complex functions or from physical parameters. SGNSim can generate ensembles of GRNs within a set of user-defined parameters, such as topology. It can also be used to model specific GRNs and systems of chemical reactions. Genetic perturbations such as gene deletions, gene over-expression, insertions, frame shift mutations can also be modeled as well.</p>

<p>The GRN is created from a graph with the desired topology, imposing in-degree and out-degree distributions. Gene promoter activities are affected by other genes expression products that act as inputs, in the form of monomers or combined into multimers and set as direct or indirect. Next, each direct input is assigned to an operator site and different transcription factors can be allowed, or not, to compete for the same operator site, while indirect inputs are given a target. Finally, a function is assigned to each gene, defining the gene's response to a combination of transcription factors (promoter state). The transfer functions (that is, how genes respond to a combination of inputs) can be assigned to each combination of promoter states as desired.</p>

<p>In other recent work, multiscale models of gene regulatory networks have been developed that focus on synthetic biology applications. Simulations have been used that model all biomolecular interactions in transcription, translation, regulation, and induction of gene regulatory networks, guiding the design of synthetic systems.<a class="footnoteRef" href="#fn28" id="fnref28"><sup>28</sup></a></p>
<h2 id="prediction">Prediction</h2>

<p>Other work has focused on predicting the gene expression levels in a gene regulatory network. The approaches used to model gene regulatory networks have been constrained to be interpretable and, as a result, are generally simplified versions of the network. For example, Boolean networks have been used due to their simplicity and ability to handle noisy data but lose data information by having a binary representation of the genes. Also, artificial neural networks omit using a hidden layer so that they can be interpreted, losing the ability to model higher order correlations in the data. Using a model that is not constrained to be interpretable, a more accurate model can be produced. Being able to predict gene expressions more accurately provides a way to explore how drugs affect a system of genes as well as for finding which genes are interrelated in a process. This has been encouraged by the DREAM competition<a class="footnoteRef" href="#fn29" id="fnref29"><sup>29</sup></a> which promotes a competition for the best prediction algorithms.<a class="footnoteRef" href="#fn30" id="fnref30"><sup>30</sup></a> Some other recent work has used artificial neural networks with a hidden layer.<a class="footnoteRef" href="#fn31" id="fnref31"><sup>31</sup></a></p>
<h2 id="structure-and-evolution">Structure and evolution</h2>

<p>Gene regulatory networks are generally thought to be made up of a few highly connected <a href="node_(graph_theory)" title="wikilink">nodes</a> (<a href="network_topology" title="wikilink">hubs</a>) and many poorly connected nodes nested within a hierarchical regulatory regime. Thus gene regulatory networks approximate a <a href="Network_topology" title="wikilink">hierarchical</a> <a href="Scale-free_network" title="wikilink">scale free network</a> topology.<a class="footnoteRef" href="#fn32" id="fnref32"><sup>32</sup></a> This is consistent with the view that most genes have limited <a class="uri" href="pleiotropy" title="wikilink">pleiotropy</a> and operate within regulatory <a href="Modularity_(biology)" title="wikilink">modules</a>.<a class="footnoteRef" href="#fn33" id="fnref33"><sup>33</sup></a> This structure is thought to evolve due to the <a href="preferential_attachment" title="wikilink">preferential attachment</a> of <a href="Gene_duplication" title="wikilink">duplicated genes</a> to more highly connected genes.<a class="footnoteRef" href="#fn34" id="fnref34"><sup>34</sup></a> Recent work has also shown that natural selection tends to favor networks with sparse connectivity.<a class="footnoteRef" href="#fn35" id="fnref35"><sup>35</sup></a></p>

<p>There are primarily two ways that networks can evolve, both of which can occur simultaneously. The first is that network topology can be changed by the addition or subtraction of nodes (genes) or parts of the network (modules) may be expressed in different contexts. The'' <a class="uri" href="Drosophila" title="wikilink">Drosophila</a>'' <a href="Hippo_signaling_pathway" title="wikilink">Hippo signaling pathway</a> provides a good example. The Hippo signaling pathway controls both mitotic growth and post-mitotic cellular differentiation.<a class="footnoteRef" href="#fn36" id="fnref36"><sup>36</sup></a> Recently it was found that the network the Hippo signaling pathway operates in differs between these two functions which in turn changes the behavior of the Hippo signaling pathway. This suggests that the Hippo signaling pathway operates as a conserved regulatory module that can be used for multiple functions depending on context.<a class="footnoteRef" href="#fn37" id="fnref37"><sup>37</sup></a> Thus, changing network topology can allow a conserved module to serve multiple functions and alter the final output of the network. The second way networks can evolve is by changing the strength of interactions between nodes, such as how strongly a transcription factor may bind to a cis-regulatory element. Such variation in strength of network edges has been shown to underlie between species variation in vulva cell fate patterning of <em><a class="uri" href="Caenorhabditis" title="wikilink">Caenorhabditis</a></em> worms.<a class="footnoteRef" href="#fn38" id="fnref38"><sup>38</sup></a></p>
<h2 id="bacterial-regulatory-networks">Bacterial regulatory networks</h2>

<p>Regulatory networks allow <a class="uri" href="bacteria" title="wikilink">bacteria</a> to adapt to almost every environmental niche on earth.<a class="footnoteRef" href="#fn39" id="fnref39"><sup>39</sup></a><a class="footnoteRef" href="#fn40" id="fnref40"><sup>40</sup></a> A network of interactions among diverse types of molecules including DNA, RNA, proteins and metabolites, is utilised by the bacteria to achieve regulation of gene expression. In bacteria, the principal function of regulatory networks is to control the response to environmental changes, for example nutritional status and environmental stress.<a class="footnoteRef" href="#fn41" id="fnref41"><sup>41</sup></a> A complex organization of networks permits the microorganism to coordinate and integrate multiple environmental signals.<a class="footnoteRef" href="#fn42" id="fnref42"><sup>42</sup></a></p>
<h2 id="see-also">See also</h2>
<ul>
<li><a href="Body_plan" title="wikilink">Body plan</a></li>
<li><a href="Cis-regulatory_module" title="wikilink">Cis-regulatory module</a></li>
<li><a class="uri" href="Genenetwork" title="wikilink">Genenetwork</a> (database)</li>
<li><a class="uri" href="Morphogen" title="wikilink">Morphogen</a></li>
<li><a class="uri" href="Operon" title="wikilink">Operon</a></li>
<li><a class="uri" href="Synexpression" title="wikilink">Synexpression</a></li>
<li><a href="Systems_biology" title="wikilink">Systems biology</a></li>
<li><a href="Weighted_correlation_network_analysis" title="wikilink">Weighted gene co-expression network analysis</a></li>
</ul>
<h2 id="notes">Notes</h2>
<h2 id="references">References</h2>
<ul>
<li></li>
</ul>
<ul>
<li></li>
</ul>
<h2 id="external-links">External links</h2>
<ul>
<li><a href="http://www.genenetwork.org/">Open source web service for GRN analysis</a></li>
<li><a href="http://sergi5.com/bio">BIB: Yeast Biological Interaction Browser</a></li>
<li><a href="http://strimmerlab.org/notes/ggm.html">Graphical Gaussian models for genome data</a> — Inference of gene association networks with GGMs</li>
<li><a href="http://www.molgen.mpg.de/~markowet/docs/network-bib.pdf">A bibliography on learning causal networks of gene interactions</a> - regularly updated, contains hundreds of links to papers from bioinformatics, statistics, machine learning.</li>
<li><a class="uri" href="http://mips.gsf.de/proj/biorel/">http://mips.gsf.de/proj/biorel/</a> BIOREL is a web-based resource for quantitative estimation of the gene network bias in relation to available database information about gene activity/function/properties/associations/interactio.</li>
<li><a href="http://panmental.de/GRNclocks">Evolving Biological Clocks using Genetic Regulatory Networks</a> - Information page with model source code and Java applet.</li>
<li><a href="http://www.bu.edu/abl">Engineered Gene Networks</a></li>
<li><a href="http://panmental.de/ICSBtut/">Tutorial: Genetic Algorithms and their Application to the Artificial Evolution of Genetic Regulatory Networks</a></li>
<li><a href="http://crn.vistainformatics.com/">BEN: a web-based resource for exploring the connections between genes, diseases, and other biomedical entities</a></li>
<li><a href="http://zope.bioinfo.cnio.es/cellcyle_addmaterial">Global protein-protein interaction and gene regulation network of Arabidopsis thaliana</a></li>
</ul>

<p>"</p>

<p><a href="Category:Gene_expression" title="wikilink">Category:Gene expression</a> <a class="uri" href="Category:Networks" title="wikilink">Category:Networks</a> <a href="Category:Systems_biology" title="wikilink">Category:Systems biology</a></p>
<section class="footnotes">
<hr/>
<ol>
<li id="fn1"><a href="#fnref1">↩</a></li>
<li id="fn2"><a href="#fnref2">↩</a></li>
<li id="fn3">Florian Leitner, Martin Krallinger, Sushil Tripathi, Martin Kuiper, Astrid Lægreid and Alfonso Valencia, Mining cis-Regulatory Transcription Networks from Literature, Proceedings of BioLINK Special Interest Group, 5-12, ISBM/ECCB, 2013<a href="#fnref3">↩</a></li>
<li id="fn4">Vibhor Kumar, Masafumi Muratani, Nirmala Arul Rayan, Petra Kraus, Thomas Lufkin, Huck Hui Ng and Shyam Prabhakar, Uniform, optimal signal processing of mapped deep-sequencing data, Nature biotechnology, 2013<a href="#fnref4">↩</a></li>
<li id="fn5"><a href="#fnref5">↩</a></li>
<li id="fn6"><a href="#fnref6">↩</a></li>
<li id="fn7"><a href="#fnref7">↩</a></li>
<li id="fn8"><a href="#fnref8">↩</a></li>
<li id="fn9"><a href="#fnref9">↩</a></li>
<li id="fn10"><a href="#fnref10">↩</a></li>
<li id="fn11"><a href="#fnref11">↩</a></li>
<li id="fn12"><a href="#fnref12">↩</a></li>
<li id="fn13"><a href="#fnref13">↩</a></li>
<li id="fn14"><a href="#fnref14">↩</a></li>
<li id="fn15"><a href="#fnref15">↩</a></li>
<li id="fn16"><a href="#fnref16">↩</a></li>
<li id="fn17"><a href="#fnref17">↩</a></li>
<li id="fn18"><a href="#fnref18">↩</a></li>
<li id="fn19"><a href="#fnref19">↩</a></li>
<li id="fn20"><a href="#fnref20">↩</a></li>
<li id="fn21"><a href="#fnref21">↩</a></li>
<li id="fn22"><a href="#fnref22">↩</a></li>
<li id="fn23"><a href="#fnref23">↩</a></li>
<li id="fn24"><a href="#fnref24">↩</a></li>
<li id="fn25"><a href="#fnref25">↩</a></li>
<li id="fn26"><a href="#fnref26">↩</a></li>
<li id="fn27"><a href="#fnref27">↩</a></li>
<li id="fn28"><a href="#fnref28">↩</a></li>
<li id="fn29"><a href="#fnref29">↩</a></li>
<li id="fn30"><a href="#fnref30">↩</a></li>
<li id="fn31"><a href="#fnref31">↩</a></li>
<li id="fn32"><a href="#fnref32">↩</a></li>
<li id="fn33">Wagner, G. P. and J. Zhang. 2011. <a href="The_pleiotropic_structure_of_the_genotype-phenotype_map:_the_evolvability_of_complex_organisms" title="wikilink">The pleiotropic structure of the genotype-phenotype map: the evolvability of complex organisms</a>. Nature Review Genetics 12: 204-213<a href="#fnref33">↩</a></li>
<li id="fn34"></li>
<li id="fn35"><a href="#fnref35">↩</a></li>
<li id="fn36"><a href="#fnref36">↩</a></li>
<li id="fn37"></li>
<li id="fn38"><a href="#fnref38">↩</a></li>
<li id="fn39"><a href="#fnref39">↩</a></li>
<li id="fn40"><a href="#fnref40">↩</a></li>
<li id="fn41"><a href="#fnref41">↩</a></li>
<li id="fn42"></li>
</ol>
</section>
</body>
</html>
