<html lang="en">
<head>
<meta charset="utf-8"/>
<title offset="18">Genetic drift</title>
<script src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_SVG.js" type="text/javascript">
</script>
</head>
<body>
<h1>Genetic drift</h1>
<hr/>

<p><strong>Genetic drift</strong> (or <strong>allelic drift</strong>) is the change in the frequency of a gene variant (<a class="uri" href="allele" title="wikilink">allele</a>) in a population due to random sampling of organisms.<a class="footnoteRef" href="#fn1" id="fnref1"><sup>1</sup></a> The alleles in the offspring are a sample of those in the parents, and chance has a role in determining whether a given individual survives and reproduces. A population's <a href="allele_frequency" title="wikilink">allele frequency</a> is the fraction of the copies of one gene that share a particular form.<a class="footnoteRef" href="#fn2" id="fnref2"><sup>2</sup></a> Genetic drift may cause gene variants to disappear completely and thereby reduce <a href="genetic_variation" title="wikilink">genetic variation</a>.</p>

<p>When there are few copies of an allele, the effect of genetic drift is larger, and when there are many copies the effect is smaller. In the early twentieth century vigorous debates occurred over the relative importance of <a href="natural_selection" title="wikilink">natural selection</a> versus neutral processes, including genetic drift. <a href="Ronald_Fisher" title="wikilink">Ronald Fisher</a> held the view that genetic drift plays at the most a minor role in evolution, and this remained the dominant view for several decades. In 1968, <a href="Motoo_Kimura" title="wikilink">Motoo Kimura</a> rekindled the debate with his <a href="neutral_theory_of_molecular_evolution" title="wikilink">neutral theory of molecular evolution</a>, which claims that most instances where a genetic change <a href="Fixation_(population_genetics)" title="wikilink">spreads across a population</a> (although not necessarily changes in <a class="uri" href="phenotypes" title="wikilink">phenotypes</a>) are caused by genetic drift.<a class="footnoteRef" href="#fn3" id="fnref3"><sup>3</sup></a></p>
<h2 id="analogy-with-marbles-in-a-jar">Analogy with marbles in a jar</h2>

<p>The process of genetic drift can be illustrated using 20 marbles in a jar to represent 20 organisms in a population.<a class="footnoteRef" href="#fn4" id="fnref4"><sup>4</sup></a> Consider this jar of marbles as the starting population. Half of the marbles in the jar are red and half blue, and both colors correspond to two different <a href="allele" title="wikilink">alleles</a> of one <a class="uri" href="gene" title="wikilink">gene</a> in the population. In each new generation the organisms reproduce at random. To represent this reproduction, randomly select a marble from the original jar and deposit a new marble with the same color as its "parent" into a new jar. (The selected marble remains in the original jar.) Repeat this process until there are 20 new marbles in the second jar. The second jar then contains a second generation of "offspring," consisting of 20 marbles of various colors. Unless the second jar contains exactly 10 red marbles and 10 blue marbles, a random shift occurred in the allele frequencies.</p>

<p>Repeat this process a number of times, randomly reproducing each generation of marbles to form the next. The numbers of red and blue marbles picked each generation fluctuates: sometimes more red, sometimes more blue. This fluctuation is analogous to genetic drift – a change in the population's allele frequency resulting from a random variation in the distribution of alleles from one generation to the next.</p>

<p>It is even possible that in any one generation no marbles of a particular color are chosen, meaning they have no offspring. In this example, if no red marbles are selected the jar representing the new generation contains only blue offspring. If this happens, the red allele has been lost permanently in the population, while the remaining blue allele has become <a href="fixation_(population_genetics)" title="wikilink">fixed</a>: all future generations are entirely blue. In small populations, fixation can occur in just a few generations. </p>
<h2 id="probability-and-allele-frequency">Probability and allele frequency</h2>

<p>The mechanisms of genetic drift can be illustrated with a simplified example. Consider a very large colony of bacteria isolated in a drop of solution. The bacteria are genetically identical except for a single gene with two alleles labeled <strong>A</strong> and <strong>B</strong>. <strong>A</strong> and <strong>B</strong> are neutral alleles meaning that they do not affect the bacteria's ability to survive and reproduce; all bacteria in this colony are equally likely to survive and reproduce. Suppose that half the bacteria have allele <strong>A</strong> and the other half have allele <strong>B</strong>. Thus <strong>A</strong> and <strong>B</strong> each have allele frequency 1/2.</p>

<p>The drop of solution then shrinks until it has only enough food to sustain four bacteria. All other bacteria die without reproducing. Among the four who survive, there are sixteen possible <a href="combination" title="wikilink">combinations</a> for the <strong>A</strong> and <strong>B</strong> alleles:</p>

<p><strong>(A-A-A-A), (B-A-A-A), (A-B-A-A), (B-B-A-A),<br/>
 (A-A-B-A), (B-A-B-A), (A-B-B-A), (B-B-B-A),<br/>
 (A-A-A-B), (B-A-A-B), (A-B-A-B), (B-B-A-B),<br/>
 (A-A-B-B), (B-A-B-B), (A-B-B-B), (B-B-B-B).</strong></p>

<p>Since all bacteria in the original solution are equally likely to survive when the solution shrinks, the four survivors are a random sample from the original colony. The probability that each of the four survivors has a given allele is 1/2, and so the probability that any particular allele combination occurs when the solution shrinks is</p>

<p>

<math display="block" id="Genetic_drift:0">
 <semantics>
  <mrow>
   <mrow>
    <mrow>
     <mfrac>
      <mn>1</mn>
      <mn>2</mn>
     </mfrac>
     <mo>⋅</mo>
     <mfrac>
      <mn>1</mn>
      <mn>2</mn>
     </mfrac>
     <mo>⋅</mo>
     <mfrac>
      <mn>1</mn>
      <mn>2</mn>
     </mfrac>
     <mo>⋅</mo>
     <mfrac>
      <mn>1</mn>
      <mn>2</mn>
     </mfrac>
    </mrow>
    <mo>=</mo>
    <mfrac>
     <mn>1</mn>
     <mn>16</mn>
    </mfrac>
   </mrow>
   <mo>.</mo>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <eq></eq>
    <apply>
     <ci>normal-⋅</ci>
     <apply>
      <divide></divide>
      <cn type="integer">1</cn>
      <cn type="integer">2</cn>
     </apply>
     <apply>
      <divide></divide>
      <cn type="integer">1</cn>
      <cn type="integer">2</cn>
     </apply>
     <apply>
      <divide></divide>
      <cn type="integer">1</cn>
      <cn type="integer">2</cn>
     </apply>
     <apply>
      <divide></divide>
      <cn type="integer">1</cn>
      <cn type="integer">2</cn>
     </apply>
    </apply>
    <apply>
     <divide></divide>
     <cn type="integer">1</cn>
     <cn type="integer">16</cn>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \frac{1}{2}\cdot\frac{1}{2}\cdot\frac{1}{2}\cdot\frac{1}{2}=\frac{1}{16}.
  </annotation>
 </semantics>
</math>

</p>

<p>(The original population size is so large that the sampling effectively happens without replacement). In other words, each of the sixteen possible allele combinations is equally likely to occur, with probability 1/16.</p>

<p>Counting the combinations with the same number of <strong>A</strong> and <strong>B</strong>, we get the following table.</p>
<table>
<tbody>
<tr class="odd">
<td style="text-align: left;">
<p><strong>A</strong></p></td>
<td style="text-align: left;">
<p><strong>B</strong></p></td>
<td style="text-align: left;">
<p>Combinations</p></td>
<td style="text-align: left;">
<p>Probability</p></td>
</tr>
<tr class="even">
<td style="text-align: left;">
<p>4</p></td>
<td style="text-align: left;">
<p>0</p></td>
<td style="text-align: left;">
<p>1</p></td>
<td style="text-align: left;">
<p>1/16</p></td>
</tr>
<tr class="odd">
<td style="text-align: left;">
<p>3</p></td>
<td style="text-align: left;">
<p>1</p></td>
<td style="text-align: left;">
<p>4</p></td>
<td style="text-align: left;">
<p>4/16</p></td>
</tr>
<tr class="even">
<td style="text-align: left;">
<p>2</p></td>
<td style="text-align: left;">
<p>2</p></td>
<td style="text-align: left;">
<p>6</p></td>
<td style="text-align: left;">
<p>6/16</p></td>
</tr>
<tr class="odd">
<td style="text-align: left;">
<p>1</p></td>
<td style="text-align: left;">
<p>3</p></td>
<td style="text-align: left;">
<p>4</p></td>
<td style="text-align: left;">
<p>4/16</p></td>
</tr>
<tr class="even">
<td style="text-align: left;">
<p>0</p></td>
<td style="text-align: left;">
<p>4</p></td>
<td style="text-align: left;">
<p>1</p></td>
<td style="text-align: left;">
<p>1/16</p></td>
</tr>
</tbody>
</table>

<p>As shown in the table, the total number of possible combinations to have an equal (conserved) number of <strong>A</strong> and <strong>B</strong> alleles is six, and its probability is 6/16. The total number of possible alternative combinations is ten, and the probability of unequal number of <strong>A</strong> and <strong>B</strong> alleles is 10/16. Thus, although the original colony began with an equal number of <strong>A</strong> and <strong>B</strong> alleles, chances are that the number of alleles in the remaining population of four members will not be equal. In the latter case, genetic drift has occurred because the population's allele frequencies have changed due to random sampling. In this example the population contracted to just four random survivors, a phenomenon known as <a href="population_bottleneck" title="wikilink">population bottleneck</a>.</p>

<p>The probabilities for the number of copies of allele <strong>A</strong> (or <strong>B</strong>) that survive (given in the last column of the above table) can be calculated directly from the <a href="Binomial_distribution" title="wikilink">Binomial distribution</a> where the "success" probability (probability of a given allele being present) is 1/2 i.e. the probability that there are <em>k</em> copies of <strong>A</strong> (or <strong>B</strong>) alleles in the combination is given by</p>

<p>

<math display="block" id="Genetic_drift:1">
 <semantics>
  <mrow>
   <mrow>
    <mrow>
     <mo>(</mo>
     <mtable columnspacing="0.4em" rowspacing="0.2ex">
      <mtr>
       <mtd>
        <mi>n</mi>
       </mtd>
      </mtr>
      <mtr>
       <mtd>
        <mi>k</mi>
       </mtd>
      </mtr>
     </mtable>
     <mo>)</mo>
    </mrow>
    <msup>
     <mrow>
      <mo>(</mo>
      <mfrac>
       <mn>1</mn>
       <mn>2</mn>
      </mfrac>
      <mo>)</mo>
     </mrow>
     <mi>k</mi>
    </msup>
    <msup>
     <mrow>
      <mo>(</mo>
      <mrow>
       <mn>1</mn>
       <mo>-</mo>
       <mfrac>
        <mn>1</mn>
        <mn>2</mn>
       </mfrac>
      </mrow>
      <mo>)</mo>
     </mrow>
     <mrow>
      <mi>n</mi>
      <mo>-</mo>
      <mi>k</mi>
     </mrow>
    </msup>
   </mrow>
   <mo>=</mo>
   <mrow>
    <mrow>
     <mo>(</mo>
     <mtable columnspacing="0.4em" rowspacing="0.2ex">
      <mtr>
       <mtd>
        <mi>n</mi>
       </mtd>
      </mtr>
      <mtr>
       <mtd>
        <mi>k</mi>
       </mtd>
      </mtr>
     </mtable>
     <mo>)</mo>
    </mrow>
    <mpadded width="-1.7pt">
     <msup>
      <mrow>
       <mo>(</mo>
       <mfrac>
        <mn>1</mn>
        <mn>2</mn>
       </mfrac>
       <mo>)</mo>
      </mrow>
      <mi>n</mi>
     </msup>
    </mpadded>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <eq></eq>
    <apply>
     <times></times>
     <apply>
      <csymbol cd="latexml">binomial</csymbol>
      <ci>n</ci>
      <ci>k</ci>
     </apply>
     <apply>
      <csymbol cd="ambiguous">superscript</csymbol>
      <apply>
       <divide></divide>
       <cn type="integer">1</cn>
       <cn type="integer">2</cn>
      </apply>
      <ci>k</ci>
     </apply>
     <apply>
      <csymbol cd="ambiguous">superscript</csymbol>
      <apply>
       <minus></minus>
       <cn type="integer">1</cn>
       <apply>
        <divide></divide>
        <cn type="integer">1</cn>
        <cn type="integer">2</cn>
       </apply>
      </apply>
      <apply>
       <minus></minus>
       <ci>n</ci>
       <ci>k</ci>
      </apply>
     </apply>
    </apply>
    <apply>
     <times></times>
     <apply>
      <csymbol cd="latexml">binomial</csymbol>
      <ci>n</ci>
      <ci>k</ci>
     </apply>
     <apply>
      <csymbol cd="ambiguous">superscript</csymbol>
      <apply>
       <divide></divide>
       <cn type="integer">1</cn>
       <cn type="integer">2</cn>
      </apply>
      <ci>n</ci>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   {n\choose k}\left(\frac{1}{2}\right)^{k}\left(1-\frac{1}{2}\right)^{n-k}={n%
\choose k}\left(\frac{1}{2}\right)^{n}\!
  </annotation>
 </semantics>
</math>

</p>

<p>where <em>n=4</em> is the number of surviving bacteria.</p>
<h2 id="mathematical-models-of-genetic-drift">Mathematical models of genetic drift</h2>

<p>Mathematical models of genetic drift can be designed using either <a href="branching_process" title="wikilink">branching processes</a> or a <a href="diffusion_equation" title="wikilink">diffusion equation</a> describing changes in allele frequency in an <a href="idealised_population" title="wikilink">idealised population</a>.<a class="footnoteRef" href="#fn5" id="fnref5"><sup>5</sup></a></p>
<h3 id="wrightfisher-model">Wright–Fisher model</h3>

<p>Think of a gene with two alleles, <strong>A</strong> or <strong>B</strong>. In <a class="uri" href="diploid" title="wikilink">diploid</a> populations consisting of <em>N</em> individuals there are 2<em>N</em> copies of each gene. An individual can have two copies of the same allele or two different alleles. We can call the frequency of one allele <em>p</em> and the frequency of the other <em>q</em>. The Wright–Fisher model (named after <a href="Sewall_Wright" title="wikilink">Sewall Wright</a> and <a href="Ronald_Fisher" title="wikilink">Ronald Fisher</a>) assumes that generations do not overlap (for example, <a href="annual_plants" title="wikilink">annual plants</a> have exactly one generation per year) and that each copy of the gene found in the new generation is drawn independently at random from all copies of the gene in the old generation. The formula to calculate the probability of obtaining <em>k</em> copies of an allele that had frequency <em>p</em> in the last generation is then<a class="footnoteRef" href="#fn6" id="fnref6"><sup>6</sup></a><a class="footnoteRef" href="#fn7" id="fnref7"><sup>7</sup></a></p>

<p>

<math display="block" id="Genetic_drift:2">
 <semantics>
  <mrow>
   <mfrac>
    <mrow>
     <mrow>
      <mo stretchy="false">(</mo>
      <mrow>
       <mn>2</mn>
       <mi>N</mi>
      </mrow>
      <mo stretchy="false">)</mo>
     </mrow>
     <mo lspace="0pt" rspace="3.5pt">!</mo>
    </mrow>
    <mrow>
     <mrow>
      <mi>k</mi>
      <mo lspace="0pt" rspace="3.5pt">!</mo>
     </mrow>
     <mrow>
      <mrow>
       <mo stretchy="false">(</mo>
       <mrow>
        <mrow>
         <mn>2</mn>
         <mi>N</mi>
        </mrow>
        <mo>-</mo>
        <mi>k</mi>
       </mrow>
       <mo stretchy="false">)</mo>
      </mrow>
      <mo lspace="0pt" rspace="3.5pt">!</mo>
     </mrow>
    </mrow>
   </mfrac>
   <msup>
    <mi>p</mi>
    <mi>k</mi>
   </msup>
   <msup>
    <mi>q</mi>
    <mrow>
     <mrow>
      <mn>2</mn>
      <mi>N</mi>
     </mrow>
     <mo>-</mo>
     <mi>k</mi>
    </mrow>
   </msup>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <times></times>
    <apply>
     <divide></divide>
     <apply>
      <factorial></factorial>
      <apply>
       <times></times>
       <cn type="integer">2</cn>
       <ci>N</ci>
      </apply>
     </apply>
     <apply>
      <times></times>
      <apply>
       <factorial></factorial>
       <ci>k</ci>
      </apply>
      <apply>
       <factorial></factorial>
       <apply>
        <minus></minus>
        <apply>
         <times></times>
         <cn type="integer">2</cn>
         <ci>N</ci>
        </apply>
        <ci>k</ci>
       </apply>
      </apply>
     </apply>
    </apply>
    <apply>
     <csymbol cd="ambiguous">superscript</csymbol>
     <ci>p</ci>
     <ci>k</ci>
    </apply>
    <apply>
     <csymbol cd="ambiguous">superscript</csymbol>
     <ci>q</ci>
     <apply>
      <minus></minus>
      <apply>
       <times></times>
       <cn type="integer">2</cn>
       <ci>N</ci>
      </apply>
      <ci>k</ci>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \frac{(2N)!}{k!(2N-k)!}p^{k}q^{2N-k}
  </annotation>
 </semantics>
</math>

</p>

<p>where the symbol "<strong>!</strong>" signifies the <a class="uri" href="factorial" title="wikilink">factorial</a> function. This expression can also be formulated using the binomial coefficient,</p>

<p>

<math display="block" id="Genetic_drift:3">
 <semantics>
  <mrow>
   <mrow>
    <mo>(</mo>
    <mtable columnspacing="0.4em" rowspacing="0.2ex">
     <mtr>
      <mtd>
       <mrow>
        <mn>2</mn>
        <mi>N</mi>
       </mrow>
      </mtd>
     </mtr>
     <mtr>
      <mtd>
       <mi>k</mi>
      </mtd>
     </mtr>
    </mtable>
    <mo>)</mo>
   </mrow>
   <msup>
    <mi>p</mi>
    <mi>k</mi>
   </msup>
   <msup>
    <mi>q</mi>
    <mrow>
     <mrow>
      <mn>2</mn>
      <mi>N</mi>
     </mrow>
     <mo>-</mo>
     <mi>k</mi>
    </mrow>
   </msup>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <times></times>
    <apply>
     <csymbol cd="latexml">binomial</csymbol>
     <apply>
      <times></times>
      <cn type="integer">2</cn>
      <ci>N</ci>
     </apply>
     <ci>k</ci>
    </apply>
    <apply>
     <csymbol cd="ambiguous">superscript</csymbol>
     <ci>p</ci>
     <ci>k</ci>
    </apply>
    <apply>
     <csymbol cd="ambiguous">superscript</csymbol>
     <ci>q</ci>
     <apply>
      <minus></minus>
      <apply>
       <times></times>
       <cn type="integer">2</cn>
       <ci>N</ci>
      </apply>
      <ci>k</ci>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   {2N\choose k}p^{k}q^{2N-k}
  </annotation>
 </semantics>
</math>

</p>
<h3 id="moran-model">Moran model</h3>

<p>The <a href="Moran_process" title="wikilink">Moran model</a> assumes overlapping generations. At each time step, one individual is chosen to reproduce and one individual is chosen to die. So in each timestep, the number of copies of a given allele can go up by one, go down by one, or can stay the same. This means that the <a href="transition_matrix" title="wikilink">transition matrix</a> is <a href="tridiagonal_matrix" title="wikilink">tridiagonal</a>, which means that mathematical solutions are easier for the Moran model than for the Wright–Fisher model. On the other hand, <a href="computer_simulations" title="wikilink">computer simulations</a> are usually easier to perform using the Wright–Fisher model, because fewer time steps need to be calculated. In the Moran model, it takes <em>N</em> timesteps to get through one generation, where <em>N</em> is the <a href="effective_population_size" title="wikilink">effective population size</a>. In the Wright–Fisher model, it takes just one.</p>

<p>In practice, the Moran model and Wright–Fisher model give qualitatively similar results, but genetic drift runs twice as fast in the Moran model.</p>
<h3 id="other-models-of-drift">Other models of drift</h3>

<p>If the variance in the number of offspring is much greater than that given by the <a href="binomial_distribution" title="wikilink">binomial distribution</a> assumed by the Wright–Fisher model, then given the same overall speed of genetic drift (the <a href="Effective_population_size#Variance_effective_size" title="wikilink">variance effective population size</a>), genetic drift is a less powerful force compared to selection.<a class="footnoteRef" href="#fn8" id="fnref8"><sup>8</sup></a> Even for the same variance, if higher <a href="Moment_(mathematics)" title="wikilink">moments</a> of the offspring number distribution exceed those of the binomial distribution then again the force of genetic drift is substantially weakened.<a class="footnoteRef" href="#fn9" id="fnref9"><sup>9</sup></a></p>
<h3 id="random-effects-other-than-sampling-error">Random effects other than sampling error</h3>

<p>Random changes in allele frequencies can also be caused by effects other than sampling error, for example random changes in selection pressure.<a class="footnoteRef" href="#fn10" id="fnref10"><sup>10</sup></a></p>

<p>One important alternative source of stochasticity, perhaps more important than genetic drift, is <a href="genetic_hitchhiking" title="wikilink">genetic draft</a>.<a class="footnoteRef" href="#fn11" id="fnref11"><sup>11</sup></a> Genetic draft is the effect on a locus by selection on <a href="genetic_linkage" title="wikilink">linked</a> loci. The mathematical properties of genetic draft are different from those of genetic drift.<a class="footnoteRef" href="#fn12" id="fnref12"><sup>12</sup></a> The direction of the random change in allele frequency is <a href="autocorrelation" title="wikilink">autocorrelated</a> across generations.<a class="footnoteRef" href="#fn13" id="fnref13"><sup>13</sup></a></p>
<h2 id="drift-and-fixation">Drift and fixation</h2>

<p>The <a href="Hardy–Weinberg_principle" title="wikilink">Hardy–Weinberg principle</a> states that within sufficiently large populations, the allele frequencies remain constant from one generation to the next unless the equilibrium is disturbed by <a href="gene_flow" title="wikilink">migration</a>, <a href="genetic_mutation" title="wikilink">genetic mutations</a>, or <a href="Selection_(biology)" title="wikilink">selection</a>.<a class="footnoteRef" href="#fn14" id="fnref14"><sup>14</sup></a></p>

<p>However, in finite populations, no new alleles are gained from the random sampling of alleles passed to the next generation, but the sampling can cause an existing allele to disappear. Because <a href="random_sample" title="wikilink">random sampling</a> can remove, but not replace, an allele, and because random declines or increases in allele frequency influence expected allele distributions for the next generation, genetic drift drives a population towards genetic uniformity over time. When an allele reaches a frequency of 1 (100%) it is said to be "fixed" in the population and when an allele reaches a frequency of 0 (0%) it is lost. Smaller populations achieve fixation faster, whereas in the limit of an infinite population, fixation is not achieved. Once an allele becomes <a href="Fixation_(population_genetics)" title="wikilink">fixed</a>, genetic drift comes to a halt, and the allele frequency cannot change unless a new allele is introduced in the population via mutation or <a href="gene_flow" title="wikilink">gene flow</a>. Thus even while genetic drift is a random, directionless process, it acts to eliminate <a href="genetic_diversity" title="wikilink">genetic variation</a> over time.<a class="footnoteRef" href="#fn15" id="fnref15"><sup>15</sup></a></p>
<h3 id="rate-of-allele-frequency-change-due-to-drift">Rate of allele frequency change due to drift</h3>

<p> Assuming genetic drift is the only evolutionary force acting on an allele, after t generations in many replicated populations, starting with allele frequencies of p and q, the variance in allele frequency across those populations is</p>

<p>

<math display="block" id="Genetic_drift:4">
 <semantics>
  <mrow>
   <msub>
    <mi>V</mi>
    <mi>t</mi>
   </msub>
   <mo>≈</mo>
   <mrow>
    <mi>p</mi>
    <mi>q</mi>
    <mrow>
     <mo>(</mo>
     <mrow>
      <mn>1</mn>
      <mo>-</mo>
      <mrow>
       <mi>exp</mi>
       <mrow>
        <mo>(</mo>
        <mrow>
         <mo>-</mo>
         <mfrac>
          <mi>t</mi>
          <mrow>
           <mn>2</mn>
           <msub>
            <mi>N</mi>
            <mi>e</mi>
           </msub>
          </mrow>
         </mfrac>
        </mrow>
        <mo>)</mo>
       </mrow>
      </mrow>
     </mrow>
     <mo>)</mo>
    </mrow>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <approx></approx>
    <apply>
     <csymbol cd="ambiguous">subscript</csymbol>
     <ci>V</ci>
     <ci>t</ci>
    </apply>
    <apply>
     <times></times>
     <ci>p</ci>
     <ci>q</ci>
     <apply>
      <minus></minus>
      <cn type="integer">1</cn>
      <apply>
       <exp></exp>
       <apply>
        <minus></minus>
        <apply>
         <divide></divide>
         <ci>t</ci>
         <apply>
          <times></times>
          <cn type="integer">2</cn>
          <apply>
           <csymbol cd="ambiguous">subscript</csymbol>
           <ci>N</ci>
           <ci>e</ci>
          </apply>
         </apply>
        </apply>
       </apply>
      </apply>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   V_{t}\approx pq\left(1-\exp\left(-\frac{t}{2N_{e}}\right)\right)
  </annotation>
 </semantics>
</math>

<a class="footnoteRef" href="#fn16" id="fnref16"><sup>16</sup></a></p>
<h3 id="time-to-fixation-or-loss">Time to fixation or loss</h3>

<p>Assuming genetic drift is the only evolutionary force acting on an allele, at any given time the probability that an allele will eventually become fixed in the population is simply its frequency in the population at that time.<a class="footnoteRef" href="#fn17" id="fnref17"><sup>17</sup></a> For example, if the frequency <em>p</em> for allele <strong>A</strong> is 75% and the frequency <em>q</em> for allele <strong>B</strong> is 25%, then given unlimited time the probability <strong>A</strong> will ultimately become fixed in the population is 75% and the probability that <strong>B</strong> will become fixed is 25%.</p>

<p>The expected number of generations for fixation to occur is <a href="Proportionality_(mathematics)" title="wikilink">proportional</a> to the population size, such that fixation is predicted to occur much more rapidly in smaller populations.<a class="footnoteRef" href="#fn18" id="fnref18"><sup>18</sup></a> Normally the <a href="effective_population_size" title="wikilink">effective population size</a>, which is smaller than the total population, is used to determine these probabilities. The effective population (<em>N</em><sub><em>e</em></sub>) takes into account factors such as the level of inbreeding, the stage of the lifecycle in which the population is the smallest, and the fact that some neutral genes are genetically linked to others that are under selection.<a class="footnoteRef" href="#fn19" id="fnref19"><sup>19</sup></a> The effective population size may not be the same for every gene in the same population.<a class="footnoteRef" href="#fn20" id="fnref20"><sup>20</sup></a></p>

<p>One forward-looking formula used for approximating the expected time before a neutral allele becomes fixed through genetic drift, according to the Wright–Fisher model, is</p>

<p>

<math display="block" id="Genetic_drift:5">
 <semantics>
  <mrow>
   <msub>
    <mover accent="true">
     <mi>T</mi>
     <mo stretchy="false">¯</mo>
    </mover>
    <mtext>fixed</mtext>
   </msub>
   <mo>=</mo>
   <mfrac>
    <mrow>
     <mo>-</mo>
     <mrow>
      <mn>4</mn>
      <msub>
       <mi>N</mi>
       <mi>e</mi>
      </msub>
      <mrow>
       <mo stretchy="false">(</mo>
       <mrow>
        <mn>1</mn>
        <mo>-</mo>
        <mi>p</mi>
       </mrow>
       <mo stretchy="false">)</mo>
      </mrow>
      <mrow>
       <mi>ln</mi>
       <mrow>
        <mo stretchy="false">(</mo>
        <mrow>
         <mn>1</mn>
         <mo>-</mo>
         <mi>p</mi>
        </mrow>
        <mo stretchy="false">)</mo>
       </mrow>
      </mrow>
     </mrow>
    </mrow>
    <mi>p</mi>
   </mfrac>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <eq></eq>
    <apply>
     <csymbol cd="ambiguous">subscript</csymbol>
     <apply>
      <ci>normal-¯</ci>
      <ci>T</ci>
     </apply>
     <mtext>fixed</mtext>
    </apply>
    <apply>
     <divide></divide>
     <apply>
      <minus></minus>
      <apply>
       <times></times>
       <cn type="integer">4</cn>
       <apply>
        <csymbol cd="ambiguous">subscript</csymbol>
        <ci>N</ci>
        <ci>e</ci>
       </apply>
       <apply>
        <minus></minus>
        <cn type="integer">1</cn>
        <ci>p</ci>
       </apply>
       <apply>
        <ln></ln>
        <apply>
         <minus></minus>
         <cn type="integer">1</cn>
         <ci>p</ci>
        </apply>
       </apply>
      </apply>
     </apply>
     <ci>p</ci>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \bar{T}_{\text{fixed}}=\frac{-4N_{e}(1-p)\ln(1-p)}{p}
  </annotation>
 </semantics>
</math>

</p>

<p>where <em>T</em> is the number of generations, <em>N</em><sub><em>e</em></sub> is the effective population size, and <em>p</em> is the initial frequency for the given allele. The result is the number of generations <a href="Expected_value" title="wikilink">expected</a> to pass before fixation occurs for a given allele in a population with given size (<em>N</em><sub><em>e</em></sub>) and allele frequency (<em>p</em>).<a class="footnoteRef" href="#fn21" id="fnref21"><sup>21</sup></a></p>

<p>The expected time for the neutral allele to be lost through genetic drift can be calculated as<a class="footnoteRef" href="#fn22" id="fnref22"><sup>22</sup></a></p>

<p>

<math display="block" id="Genetic_drift:6">
 <semantics>
  <mrow>
   <mrow>
    <msub>
     <mover accent="true">
      <mi>T</mi>
      <mo stretchy="false">¯</mo>
     </mover>
     <mtext>lost</mtext>
    </msub>
    <mo>=</mo>
    <mrow>
     <mfrac>
      <mrow>
       <mo>-</mo>
       <mrow>
        <mn>4</mn>
        <msub>
         <mi>N</mi>
         <mi>e</mi>
        </msub>
        <mi>p</mi>
       </mrow>
      </mrow>
      <mrow>
       <mn>1</mn>
       <mo>-</mo>
       <mi>p</mi>
      </mrow>
     </mfrac>
     <mrow>
      <mi>ln</mi>
      <mi>p</mi>
     </mrow>
    </mrow>
   </mrow>
   <mo>.</mo>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <eq></eq>
    <apply>
     <csymbol cd="ambiguous">subscript</csymbol>
     <apply>
      <ci>normal-¯</ci>
      <ci>T</ci>
     </apply>
     <mtext>lost</mtext>
    </apply>
    <apply>
     <times></times>
     <apply>
      <divide></divide>
      <apply>
       <minus></minus>
       <apply>
        <times></times>
        <cn type="integer">4</cn>
        <apply>
         <csymbol cd="ambiguous">subscript</csymbol>
         <ci>N</ci>
         <ci>e</ci>
        </apply>
        <ci>p</ci>
       </apply>
      </apply>
      <apply>
       <minus></minus>
       <cn type="integer">1</cn>
       <ci>p</ci>
      </apply>
     </apply>
     <apply>
      <ln></ln>
      <ci>p</ci>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \bar{T}_{\text{lost}}=\frac{-4N_{e}p}{1-p}\ln p.
  </annotation>
 </semantics>
</math>

</p>

<p>When a mutation appears only once in a population large enough for the initial frequency to be negligible, the formulas can be simplified to<a class="footnoteRef" href="#fn23" id="fnref23"><sup>23</sup></a></p>

<p>

<math display="block" id="Genetic_drift:7">
 <semantics>
  <mrow>
   <msub>
    <mover accent="true">
     <mi>T</mi>
     <mo stretchy="false">¯</mo>
    </mover>
    <mtext>fixed</mtext>
   </msub>
   <mo>=</mo>
   <mrow>
    <mn>4</mn>
    <msub>
     <mi>N</mi>
     <mi>e</mi>
    </msub>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <eq></eq>
    <apply>
     <csymbol cd="ambiguous">subscript</csymbol>
     <apply>
      <ci>normal-¯</ci>
      <ci>T</ci>
     </apply>
     <mtext>fixed</mtext>
    </apply>
    <apply>
     <times></times>
     <cn type="integer">4</cn>
     <apply>
      <csymbol cd="ambiguous">subscript</csymbol>
      <ci>N</ci>
      <ci>e</ci>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \bar{T}_{\text{fixed}}=4N_{e}
  </annotation>
 </semantics>
</math>

</p>

<p>for average number of generations expected before fixation of a neutral mutation, and</p>

<p>

<math display="block" id="Genetic_drift:8">
 <semantics>
  <mrow>
   <msub>
    <mover accent="true">
     <mi>T</mi>
     <mo stretchy="false">¯</mo>
    </mover>
    <mtext>lost</mtext>
   </msub>
   <mo>=</mo>
   <mrow>
    <mn>2</mn>
    <mrow>
     <mo>(</mo>
     <mfrac>
      <msub>
       <mi>N</mi>
       <mi>e</mi>
      </msub>
      <mi>N</mi>
     </mfrac>
     <mo>)</mo>
    </mrow>
    <mrow>
     <mi>ln</mi>
     <mrow>
      <mo stretchy="false">(</mo>
      <mrow>
       <mn>2</mn>
       <mi>N</mi>
      </mrow>
      <mo stretchy="false">)</mo>
     </mrow>
    </mrow>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <eq></eq>
    <apply>
     <csymbol cd="ambiguous">subscript</csymbol>
     <apply>
      <ci>normal-¯</ci>
      <ci>T</ci>
     </apply>
     <mtext>lost</mtext>
    </apply>
    <apply>
     <times></times>
     <cn type="integer">2</cn>
     <apply>
      <divide></divide>
      <apply>
       <csymbol cd="ambiguous">subscript</csymbol>
       <ci>N</ci>
       <ci>e</ci>
      </apply>
      <ci>N</ci>
     </apply>
     <apply>
      <ln></ln>
      <apply>
       <times></times>
       <cn type="integer">2</cn>
       <ci>N</ci>
      </apply>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \bar{T}_{\text{lost}}=2\left(\frac{N_{e}}{N}\right)\ln(2N)
  </annotation>
 </semantics>
</math>

</p>

<p>for the average number of generations expected before the loss of a neutral mutation.<a class="footnoteRef" href="#fn24" id="fnref24"><sup>24</sup></a></p>
<h3 id="time-to-loss-with-both-drift-and-mutation">Time to loss with both drift and mutation</h3>

<p>The formulae above apply to an allele that is already present in a population, and which is subject to neither <a class="uri" href="mutation" title="wikilink">mutation</a> nor <a href="natural_selection" title="wikilink">selection</a>. If an allele is lost by mutation much more often than it is gained by mutation, then mutation, as well as drift, may influence the time to loss. If the allele prone to mutational loss begins as fixed in the population, and is lost by mutation at rate m per replication, then the expected time in generations until its loss in a haploid population is given by</p>

<p>

<math display="block" id="Genetic_drift:9">
 <semantics>
  <mrow>
   <msub>
    <mover accent="true">
     <mi>T</mi>
     <mo stretchy="false">¯</mo>
    </mover>
    <mtext>lost</mtext>
   </msub>
   <mo>≈</mo>
   <mrow>
    <mo>{</mo>
    <mtable displaystyle="true">
     <mtr>
      <mtd columnalign="left">
       <mrow>
        <mrow>
         <mstyle displaystyle="false">
          <mfrac>
           <mn>1</mn>
           <mi>m</mi>
          </mfrac>
         </mstyle>
         <mo>,</mo>
         <mrow>
          <mtext>if</mtext>
          <mi>m</mi>
          <msub>
           <mi>N</mi>
           <mi>e</mi>
          </msub>
         </mrow>
        </mrow>
        <mo>≪</mo>
        <mn>1</mn>
       </mrow>
      </mtd>
      <mtd></mtd>
     </mtr>
     <mtr>
      <mtd columnalign="left">
       <mrow>
        <mrow>
         <mstyle displaystyle="false">
          <mfrac>
           <mrow>
            <mrow>
             <mi>ln</mi>
             <mrow>
              <mo stretchy="false">(</mo>
              <mrow>
               <mi>m</mi>
               <msub>
                <mi>N</mi>
                <mi>e</mi>
               </msub>
              </mrow>
              <mo stretchy="false">)</mo>
             </mrow>
            </mrow>
            <mo>+</mo>
            <mi>γ</mi>
           </mrow>
           <mi>m</mi>
          </mfrac>
         </mstyle>
         <mtext>if</mtext>
         <mi>m</mi>
         <msub>
          <mi>N</mi>
          <mi>e</mi>
         </msub>
        </mrow>
        <mo>≫</mo>
        <mn>1</mn>
       </mrow>
      </mtd>
      <mtd></mtd>
     </mtr>
    </mtable>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <approx></approx>
    <apply>
     <csymbol cd="ambiguous">subscript</csymbol>
     <apply>
      <ci>normal-¯</ci>
      <ci>T</ci>
     </apply>
     <mtext>lost</mtext>
    </apply>
    <apply>
     <csymbol cd="latexml">cases</csymbol>
     <apply>
      <csymbol cd="latexml">much-less-than</csymbol>
      <list>
       <apply>
        <divide></divide>
        <cn type="integer">1</cn>
        <ci>m</ci>
       </apply>
       <apply>
        <times></times>
        <mtext>if</mtext>
        <ci>m</ci>
        <apply>
         <csymbol cd="ambiguous">subscript</csymbol>
         <ci>N</ci>
         <ci>e</ci>
        </apply>
       </apply>
      </list>
      <cn type="integer">1</cn>
     </apply>
     <mtext>otherwise</mtext>
     <apply>
      <csymbol cd="latexml">much-greater-than</csymbol>
      <apply>
       <times></times>
       <apply>
        <divide></divide>
        <apply>
         <plus></plus>
         <apply>
          <ln></ln>
          <apply>
           <times></times>
           <ci>m</ci>
           <apply>
            <csymbol cd="ambiguous">subscript</csymbol>
            <ci>N</ci>
            <ci>e</ci>
           </apply>
          </apply>
         </apply>
         <ci>γ</ci>
        </apply>
        <ci>m</ci>
       </apply>
       <mtext>if</mtext>
       <ci>m</ci>
       <apply>
        <csymbol cd="ambiguous">subscript</csymbol>
        <ci>N</ci>
        <ci>e</ci>
       </apply>
      </apply>
      <cn type="integer">1</cn>
     </apply>
     <mtext>otherwise</mtext>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \bar{T}_{\text{lost}}\approx\begin{cases}\frac{1}{m},\text{ if }mN_{e}\ll 1\\
\frac{\ln{(mN_{e})}+\gamma}{m}\text{ if }mN_{e}\gg 1\end{cases}
  </annotation>
 </semantics>
</math>

</p>

<p>where 

<math display="inline" id="Genetic_drift:10">
 <semantics>
  <mi>γ</mi>
  <annotation-xml encoding="MathML-Content">
   <ci>γ</ci>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \gamma
  </annotation>
 </semantics>
</math>

 is equal to <a href="Euler's_constant" title="wikilink">Euler's constant</a>.<a class="footnoteRef" href="#fn25" id="fnref25"><sup>25</sup></a> The first approximation represents the waiting time until the first mutant destined for loss, with loss then occurring relatively rapidly by genetic drift, taking time N<sub>e</sub></p></body></html>

<p>Although both processes affect evolution, genetic drift operates randomly while natural selection functions non-randomly. While natural selection has a direction, guiding evolution towards heritable <a href="adaptation" title="wikilink">adaptations</a> to the current environment, genetic drift has no direction and is guided only by the <a href="probability" title="wikilink">mathematics of chance</a>.<a class="footnoteRef" href="#fn26" id="fnref26"><sup>26</sup></a> As a result, drift acts upon the <a href="genotype_frequency" title="wikilink">genotypic frequencies</a> within a population without regard to their phenotypic effects. In contrast, selection favors the spread of alleles whose phenotypic effects increase survival and/or reproduction of their carriers, lowers the frequencies of alleles that cause unfavorable traits, and ignores those that are neutral.<a class="footnoteRef" href="#fn27" id="fnref27"><sup>27</sup></a></p>

<p>In natural populations, genetic drift and <a href="natural_selection" title="wikilink">natural selection</a> do not act in isolation; both forces are always at play, together with mutation and migration. However, the magnitude of drift on allele frequencies per generation is larger when the absolute number of copies of the allele is small, e.g. in small populations. The magnitude of drift is large enough to overwhelm selection when the <a href="selection_coefficient" title="wikilink">selection coefficient</a> is less than 1 divided by the <a href="effective_population_size" title="wikilink">effective population size</a>.</p>

<p>The mathematics of genetic drift depend on the <a href="effective_population_size" title="wikilink">effective population size</a>, but it is not clear how this is related to the actual number of individuals in a population.<a class="footnoteRef" href="#fn28" id="fnref28"><sup>28</sup></a> <a href="Genetic_linkage" title="wikilink">Genetic linkage</a> to other genes that are under selection can reduce the effective population size experienced by a neutral allele. With a higher <a href="homologous_recombination" title="wikilink">recombination</a> rate, linkage decreases and with it this local effect on effective population size.<a class="footnoteRef" href="#fn29" id="fnref29"><sup>29</sup></a><a class="footnoteRef" href="#fn30" id="fnref30"><sup>30</sup></a> This effect is visible in molecular data as a correlation between local recombination rate and genetic diversity,<a class="footnoteRef" href="#fn31" id="fnref31"><sup>31</sup></a> and negative correlation between gene density and diversity at noncoding sites.<a class="footnoteRef" href="#fn32" id="fnref32"><sup>32</sup></a> Stochasticity associated with linkage to other genes that are under selection is not the same as sampling error, and is sometimes known as <a href="genetic_hitchhiking" title="wikilink">genetic draft</a> in order to distinguish it from genetic drift.<a class="footnoteRef" href="#fn33" id="fnref33"><sup>33</sup></a></p>

<p>When the allele frequency is very small, drift can also overpower selection even in large populations. For example, while disadvantageous mutations are usually eliminated quickly in large populations, new advantageous mutations are almost as vulnerable to loss through genetic drift as are neutral mutations. Not until the allele frequency for the advantageous mutation reaches a certain threshold will genetic drift have no effect.<a class="footnoteRef" href="#fn34" id="fnref34"><sup>34</sup></a></p>

<p>In general, "global" solutions to many adaptive challenges at once can evolve at a smaller effective population size than "local" solutions that must evolve separately to each adaptive challenge.<a class="footnoteRef" href="#fn35" id="fnref35"><sup>35</sup></a></p>
<h2 id="population-bottleneck">Population bottleneck</h2>

<p> A <a href="population_bottleneck" title="wikilink">population bottleneck</a> is when a population contracts to a significantly smaller size over a short period of time due to some random environmental event.<a class="footnoteRef" href="#fn36" id="fnref36"><sup>36</sup></a> In a true population bottleneck, the odds for survival of any member of the population are purely random, and are not improved by any particular inherent genetic advantage. The bottleneck can result in radical changes in allele frequencies, completely independent of selection.</p>

<p>The impact of a population bottleneck can be sustained, even when the bottleneck is caused by a one-time event such as a natural catastrophe. An interesting example of a bottleneck causing unusual genetic distribution is the relatively high proportion of individuals with total rod <a class="uri" href="color-blindness" title="wikilink">color-blindness</a> (achromatopsia) on <a href="Pingelap" title="wikilink">Pingelap atoll</a> in Micronesia. After a bottleneck, <a class="uri" href="inbreeding" title="wikilink">inbreeding</a> increases. This increases the damage done by recessive deleterious mutations, in a process known as <a href="inbreeding_depression" title="wikilink">inbreeding depression</a>. The worst of these mutations are selected against, leading to the loss of other alleles that are <a href="linkage_disequilibrium" title="wikilink">genetically linked</a> to them, in a process of <a href="background_selection" title="wikilink">background selection</a>.<a class="footnoteRef" href="#fn37" id="fnref37"><sup>37</sup></a> For recessive harmful mutations, this selection can be enhanced as a consequence of the bottleneck, due to <a href="genetic_purging" title="wikilink">genetic purging</a>.This leads to a further loss of genetic diversity. In addition, a sustained reduction in population size increases the likelihood of further allele fluctuations from drift in generations to come.</p>

<p>A population's genetic variation can be greatly reduced by a bottleneck, and even beneficial adaptations may be permanently eliminated.<a class="footnoteRef" href="#fn38" id="fnref38"><sup>38</sup></a> The loss of variation leaves the surviving population vulnerable to any new selection pressures such as disease, climate change or shift in the available food source, because adapting in response to environmental changes requires sufficient genetic variation in the population for natural selection to take place.<a class="footnoteRef" href="#fn39" id="fnref39"><sup>39</sup></a><a class="footnoteRef" href="#fn40" id="fnref40"><sup>40</sup></a></p>

<p>There have been many known cases of population bottleneck in the recent past. Prior to the arrival of Europeans, North American prairies were habitat for millions of <a href="greater_prairie_chicken" title="wikilink">greater prairie chickens</a>. In <a class="uri" href="Illinois" title="wikilink">Illinois</a> alone, their numbers plummeted from about 100 million birds in 1900 to about 50 birds in the 1990s. The declines in population resulted from hunting and habitat destruction, but the random consequence has been a loss of most of the species' <a href="genetic_diversity" title="wikilink">genetic diversity</a>. DNA analysis comparing birds from the mid century to birds in the 1990s documents a steep decline in the genetic variation in just in the latter few decades. Currently the greater prairie chicken is experiencing low <a href="reproductive_success" title="wikilink">reproductive success</a>.<a class="footnoteRef" href="#fn41" id="fnref41"><sup>41</sup></a></p>

<p>Over-hunting also caused a severe population bottleneck in the <a href="northern_elephant_seal" title="wikilink">northern elephant seal</a> in the 19th century. Their resulting decline in genetic variation can be deduced by comparing it to that of the <a href="southern_elephant_seal" title="wikilink">southern elephant seal</a>, which were not so aggressively hunted.<a class="footnoteRef" href="#fn42" id="fnref42"><sup>42</sup></a></p>
<h3 id="founder-effect">Founder effect</h3>

<p> The founder effect is a special case of a population bottleneck, occurring when a small group in a population splinters off from the original population and forms a new one. The random sample of alleles in the just formed new colony is expected to grossly misrepresent the original population in at least some respects.<a class="footnoteRef" href="#fn43" id="fnref43"><sup>43</sup></a> It is even possible that the number of alleles for some genes in the original population is larger than the number of gene copies in the founders, making complete representation impossible. When a newly formed colony is small, its founders can strongly affect the population's genetic make-up far into the future.</p>

<p>A well documented example is found in the <a class="uri" href="Amish" title="wikilink">Amish</a> migration to <a class="uri" href="Pennsylvania" title="wikilink">Pennsylvania</a> in 1744. Two members of the new colony shared the recessive allele for <a href="Ellis–van_Creveld_syndrome" title="wikilink">Ellis–van Creveld syndrome</a>. Members of the colony and their descendants tend to be religious isolates and remain relatively insular. As a result of many generations of inbreeding, Ellis-van Creveld syndrome is now much more prevalent among the Amish than in the general population.<a class="footnoteRef" href="#fn44" id="fnref44"><sup>44</sup></a><a class="footnoteRef" href="#fn45" id="fnref45"><sup>45</sup></a></p>

<p>The difference in gene frequencies between the original population and colony may also trigger the two groups to <a href="genetic_divergence" title="wikilink">diverge</a> significantly over the course of many generations. As the difference, or <a href="genetic_distance" title="wikilink">genetic distance</a>, increases, the two separated populations may become distinct, both <a href="genotype" title="wikilink">genetically</a> and <a href="phenotype" title="wikilink">phenetically</a>, although not only genetic drift but also natural selection, gene flow, and mutation contribute to this divergence. This potential for relatively rapid changes in the colony's gene frequency led most scientists to consider the founder effect (and by extension, genetic drift) a significant driving force in the evolution of <a href="speciation" title="wikilink">new species</a>. <a href="Sewall_Wright" title="wikilink">Sewall Wright</a> was the first to attach this significance to random drift and small, newly isolated populations with his <a href="shifting_balance_theory" title="wikilink">shifting balance theory</a> of speciation.<a class="footnoteRef" href="#fn46" id="fnref46"><sup>46</sup></a> Following after Wright, <a href="Ernst_Mayr" title="wikilink">Ernst Mayr</a> created many persuasive models to show that the decline in genetic variation and small population size following the founder effect were critically important for new species to develop.<a class="footnoteRef" href="#fn47" id="fnref47"><sup>47</sup></a> However, there is much less support for this view today since the hypothesis has been tested repeatedly through experimental research and the results have been equivocal at best.<a class="footnoteRef" href="#fn48" id="fnref48"><sup>48</sup></a> Founder effect was first well investigated in USSR by <a href="Science_and_technology_in_the_Soviet_Union" title="wikilink">Soviet scientists</a> Lisovskiy V.V., Kuznetsov M.A. and <a href="Nikolay_Dubinin" title="wikilink">Nikolay Dubinin</a>.</p>
<h2 id="history-of-the-concept">History of the concept</h2>

<p>The concept of genetic drift was first introduced by one of the founders in the field of <a href="population_genetics" title="wikilink">population genetics</a>, <a href="Sewall_Wright" title="wikilink">Sewall Wright</a>. His first use of the term "drift" was in 1929,<a class="footnoteRef" href="#fn49" id="fnref49"><sup>49</sup></a> though at the time he was using it in the sense of a directed process of change, or natural selection. Random drift by means of sampling error came to be known as the "Sewall–Wright effect," though he was never entirely comfortable to see his name given to it. Wright referred to all changes in allele frequency as either "steady drift" (e.g. selection) or "random drift" (e.g. sampling error).<a class="footnoteRef" href="#fn50" id="fnref50"><sup>50</sup></a> "Drift" came to be adopted as a technical term in the <a class="uri" href="stochastic" title="wikilink">stochastic</a> sense exclusively.<a class="footnoteRef" href="#fn51" id="fnref51"><sup>51</sup></a> Today it is usually defined still more narrowly, in terms of sampling error,<a class="footnoteRef" href="#fn52" id="fnref52"><sup>52</sup></a> although this narrow definition is not universal.<a class="footnoteRef" href="#fn53" id="fnref53"><sup>53</sup></a><a class="footnoteRef" href="#fn54" id="fnref54"><sup>54</sup></a> Wright wrote that the "restriction of "random drift" or even "drift" to only one component, the effects of accidents of sampling, tends to lead to confusion."<a class="footnoteRef" href="#fn55" id="fnref55"><sup>55</sup></a> Sewall Wright considered the process of random genetic drift by means of sampling error equivalent to that by means of <a class="uri" href="inbreeding" title="wikilink">inbreeding</a>, but later work has shown them to be distinct.<a class="footnoteRef" href="#fn56" id="fnref56"><sup>56</sup></a></p>

<p>In the early days of the <a href="modern_evolutionary_synthesis" title="wikilink">modern evolutionary synthesis</a>, scientists were just beginning to blend the new science of population genetics with <a href="Charles_Darwin" title="wikilink">Charles Darwin</a>'s theory of natural selection. Working within this new framework, Wright focused on the effects of inbreeding on small relatively isolated populations. He introduced the concept of an <a href="adaptive_landscape" title="wikilink">adaptive landscape</a> in which phenomena such as cross breeding and genetic drift in small populations could push them away from adaptive peaks, which in turn allow natural selection to push them towards new adaptive peaks.<a class="footnoteRef" href="#fn57" id="fnref57"><sup>57</sup></a> Wright thought smaller populations were more suited for natural selection because "inbreeding was sufficiently intense to create new interaction systems through random drift but not intense enough to cause random nonadaptive fixation of genes."<a class="footnoteRef" href="#fn58" id="fnref58"><sup>58</sup></a></p>

<p>Wright's views on the role of genetic drift in the evolutionary scheme were controversial almost from the very beginning. One of the most vociferous and influential critics was colleague <a href="Ronald_Fisher" title="wikilink">Ronald Fisher</a>. Fisher conceded genetic drift played some role in evolution, but an insignificant one. Fisher has been accused of misunderstanding Wright's views because in his criticisms Fisher seemed to argue Wright had rejected selection almost entirely. To Fisher, viewing the process of evolution as a long, steady, adaptive progression was the only way to explain the ever-increasing complexity from simpler forms. But the debates have continued between the "gradualists" and those who lean more toward the Wright model of evolution where selection and drift together play an important role.<a class="footnoteRef" href="#fn59" id="fnref59"><sup>59</sup></a></p>

<p>In 1968,<a class="footnoteRef" href="#fn60" id="fnref60"><sup>60</sup></a> population geneticist <a href="Motoo_Kimura" title="wikilink">Motoo Kimura</a> rekindled the debate with his <a href="neutral_theory_of_molecular_evolution" title="wikilink">neutral theory of molecular evolution</a>, which claims that most of the genetic changes are caused by genetic drift acting on neutral mutations.<a class="footnoteRef" href="#fn61" id="fnref61"><sup>61</sup></a></p>

<p>The role of genetic drift by means of sampling error in evolution has been criticized by <a href="John_H_Gillespie" title="wikilink">John H Gillespie</a><a class="footnoteRef" href="#fn62" id="fnref62"><sup>62</sup></a> and <a href="Will_Provine" title="wikilink">Will Provine</a>, who argue that selection on linked sites is a more important stochastic force.</p>
<h2 id="see-also">See also</h2>
<ul>
<li><a href="Allopatric_speciation" title="wikilink">Allopatric speciation</a></li>
<li><a href="Antigenic_drift" title="wikilink">Antigenic drift</a></li>
<li><a href="Gene_pool" title="wikilink">Gene pool</a></li>
<li><a href="Small_population_size" title="wikilink">Small population size</a></li>
<li><a href="Neutral_theory_of_molecular_evolution" title="wikilink">Neutral theory of molecular evolution</a></li>
<li><a href="Coalescent_theory" title="wikilink">Coalescent theory</a></li>
<li><a href="Fixation_(population_genetics)" title="wikilink">Fixation</a></li>
</ul>
<h2 id="notes-and-references">Notes and references</h2>
<h2 id="external-links">External links</h2>
<ul>
<li><a href="http://www.talkorigins.org/faqs/genetic-drift.html">The TalkOrigins Archive</a></li>
<li><a href="http://www.evolution-textbook.org/content/free/figures/ch15.html">Genetic drift illustrations in Barton et al.</a></li>
<li>[<a class="uri" href="http://authors.library.caltech.edu/5456/1/hrst.mit.edu/groups/evolution/bboard/q-and-a-fetch-msg.tcl_msg_id=000190&amp;topic;_id=54&amp;topic">http://authors.library.caltech.edu/5456/1/hrst.mit.edu/groups/evolution/bboard/q-and-a-fetch-msg.tcl_msg_id=000190&amp;topic;_id=54&amp;topic;</a>;=Genetic_Draft.html Drift vs. draft]</li>
<li><a href="http://www.radford.edu/~rsheehy/Gen_flash/popgen/">Online simulation of genetic drift</a></li>
</ul>

<p>"</p>

<p><a href="Category:Population_genetics" title="wikilink">Category:Population genetics</a> <a href="Category:Evolutionary_biology" title="wikilink">Category:Evolutionary biology</a> <a href="Category:Genetic_genealogy" title="wikilink">Category:Genetic genealogy</a></p>
<section class="footnotes">
<hr/>
<ol>
<li id="fn1"><a href="#fnref1">↩</a></li>
<li id="fn2"><a href="#fnref2">↩</a></li>
<li id="fn3"><a href="#fnref3">↩</a></li>
<li id="fn4"><a href="#fnref4">↩</a></li>
<li id="fn5"><a href="#fnref5">↩</a></li>
<li id="fn6"><a href="#fnref6">↩</a></li>
<li id="fn7"><a href="#fnref7">↩</a></li>
<li id="fn8"></li>
<li id="fn9"><a href="#fnref9">↩</a></li>
<li id="fn10"><a href="#fnref10">↩</a></li>
<li id="fn11"><a href="#fnref11">↩</a></li>
<li id="fn12"><a href="#fnref12">↩</a></li>
<li id="fn13"></li>
<li id="fn14"><a href="#fnref14">↩</a></li>
<li id="fn15"><a href="#fnref15">↩</a></li>
<li id="fn16"><a href="#fnref16">↩</a></li>
<li id="fn17"><a href="#fnref17">↩</a></li>
<li id="fn18"><a href="#fnref18">↩</a></li>
<li id="fn19"><a href="#fnref19">↩</a></li>
<li id="fn20"><a href="#fnref20">↩</a></li>
<li id="fn21"><a href="#fnref21">↩</a></li>
<li id="fn22"></li>
<li id="fn23"><a href="#fnref23">↩</a></li>
<li id="fn24"><a href="#fnref24">↩</a></li>
<li id="fn25"><a href="#fnref25">↩</a></li>
<li id="fn26"><a href="#fnref26">↩</a></li>
<li id="fn27"><a href="#fnref27">↩</a></li>
<li id="fn28"></li>
<li id="fn29"><a href="#fnref29">↩</a></li>
<li id="fn30"><a href="#fnref30">↩</a></li>
<li id="fn31"><a href="#fnref31">↩</a></li>
<li id="fn32"><a href="#fnref32">↩</a></li>
<li id="fn33"></li>
<li id="fn34"></li>
<li id="fn35"><a href="#fnref35">↩</a></li>
<li id="fn36"><a href="http://www.bookrags.com/research/population-bottleneck-gen-03/">Population Bottleneck | Macmillan Genetics</a><a href="#fnref36">↩</a></li>
<li id="fn37"></li>
<li id="fn38"><a href="#fnref38">↩</a></li>
<li id="fn39"><a href="#fnref39">↩</a></li>
<li id="fn40"><a href="#fnref40">↩</a></li>
<li id="fn41"><a href="#fnref41">↩</a></li>
<li id="fn42"><a href="#fnref42">↩</a></li>
<li id="fn43"><a href="#fnref43">↩</a></li>
<li id="fn44"></li>
<li id="fn45"><a href="#fnref45">↩</a></li>
<li id="fn46"><a href="#fnref46">↩</a></li>
<li id="fn47"><a href="#fnref47">↩</a></li>
<li id="fn48"><a href="#fnref48">↩</a></li>
<li id="fn49"><a href="#fnref49">↩</a></li>
<li id="fn50"><a href="#fnref50">↩</a></li>
<li id="fn51"><a href="#fnref51">↩</a></li>
<li id="fn52">Freeman and Herron. <em>Evolutionary Analysis.</em> 2007. Pearson Education, NJ. p.801<a href="#fnref52">↩</a></li>
<li id="fn53"><a href="#fnref53">↩</a></li>
<li id="fn54"><a href="#fnref54">↩</a></li>
<li id="fn55"></li>
<li id="fn56"><a href="#fnref56">↩</a></li>
<li id="fn57"><a href="#fnref57">↩</a></li>
<li id="fn58">; Quote attributed to William B. Provine in <em>The Origins of Theoretical Population Genetics</em>', Chicago: University of Chicago Press (1971)<a href="#fnref58">↩</a></li>
<li id="fn59"><a href="#fnref59">↩</a></li>
<li id="fn60"><a href="#fnref60">↩</a></li>
<li id="fn61"></li>
<li id="fn62"><a href="#fnref62">↩</a></li>
</ol>
</section>


