<html lang="en">
<head>
<meta charset="utf-8"/>
<title offset="1662">Gene expression programming</title>
<script src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_SVG.js" type="text/javascript">
</script>
</head>
<body>
<h1>Gene expression programming</h1>
<hr/>

<p>In <a href="computer_programming" title="wikilink">computer programming</a>, <strong>gene expression programming (GEP)</strong> is an <a href="evolutionary_algorithm" title="wikilink">evolutionary algorithm</a> that creates <a href="computer_program" title="wikilink">computer programs</a> or models. These computer programs are complex <a href="tree_structure" title="wikilink">tree structures</a> that learn and adapt by changing their sizes, shapes, and composition, much like a living organism. And like living organisms, the computer programs of GEP are also encoded in simple linear <a href="chromosome" title="wikilink">chromosomes</a> of fixed length. Thus, GEP is a <a href="Genotype-phenotype_distinction" title="wikilink">genotype-phenotype system</a>, benefiting from a simple <a href="genotype" title="wikilink">genome</a> to keep and transmit the genetic information and a complex <a class="uri" href="phenotype" title="wikilink">phenotype</a> to explore the environment and adapt to it.</p>
<h2 id="background">Background</h2>

<p><a href="Evolutionary_algorithms" title="wikilink">Evolutionary algorithms</a> use populations of individuals, select individuals according to fitness, and introduce genetic variation using one or more <a href="genetic_operators" title="wikilink">genetic operators</a>. Their use in artificial computational systems dates back to the 1950s where they were used to solve optimization problems (e.g. Box 1957<a class="footnoteRef" href="#fn1" id="fnref1"><sup>1</sup></a> and Friedman 1959<a class="footnoteRef" href="#fn2" id="fnref2"><sup>2</sup></a>). But it was with the introduction of <a href="evolution_strategies" title="wikilink">evolution strategies</a> by Rechenberg in 1965<a class="footnoteRef" href="#fn3" id="fnref3"><sup>3</sup></a> that evolutionary algorithms gained popularity. A good overview text on evolutionary algorithms is the book “An Introduction to Genetic Algorithms” by Mitchell (1996).<a class="footnoteRef" href="#fn4" id="fnref4"><sup>4</sup></a></p>

<p>Gene expression programming<a class="footnoteRef" href="#fn5" id="fnref5"><sup>5</sup></a> belongs to the family of <a href="evolutionary_algorithm" title="wikilink">evolutionary algorithms</a> and is closely related to <a href="genetic_algorithms" title="wikilink">genetic algorithms</a> and <a href="genetic_programming" title="wikilink">genetic programming</a>. From genetic algorithms it inherited the linear chromosomes of fixed length; and from genetic programming it inherited the expressive <a href="parse_tree" title="wikilink">parse trees</a> of varied sizes and shapes.</p>

<p>In gene expression programming the linear chromosomes work as the genotype and the parse trees as the phenotype, creating a <a href="Genotype-phenotype_distinction" title="wikilink">genotype/phenotype system</a>. This genotype/phenotype system is <a href="Gene" title="wikilink">multigenic</a>, thus encoding multiple parse trees in each chromosome. This means that the computer programs created by GEP are composed of multiple parse trees. Because these parse trees are the result of gene expression, in GEP they are called <a href="expression_tree" title="wikilink">expression trees</a>.</p>
<h2 id="encoding-the-genotype">Encoding: the genotype</h2>

<p>The genome of gene expression programming consists of a linear, symbolic string or chromosome of fixed length composed of one or more genes of equal size. These genes, despite their fixed length, code for expression trees of different sizes and shapes. An example of a chromosome with two genes, each of size 9, is the string (position zero indicates the start of each gene):</p>
<dl>
<dd><code><nowiki>012345678012345678</nowiki></code>
</dd>
<dd>
</dd>
<dd><code><nowiki>L+a-baccd**cLabacd</nowiki></code>
</dd>
</dl>

<p>where “L” represents the natural logarithm function and “a”, “b”, “c”, and “d” represent the variables and constants used in a problem.</p>
<h2 id="expression-trees-the-phenotype">Expression trees: the phenotype</h2>

<p>As shown <a href="gene_expression_programming#Encoding:_the_genotype" title="wikilink">above</a>, the genes of gene expression programming have all the same size. However, these fixed length strings code for <a href="expression_tree" title="wikilink">expression trees</a> of different sizes. This means that the size of the coding regions varies from gene to gene, allowing for adaptation and evolution to occur smoothly.</p>

<p>For example, the mathematical expression:</p>

<p>

<math display="block" id="Gene_expression_programming:0">
 <semantics>
  <mpadded width="+1.7pt">
   <msqrt>
    <mrow>
     <mrow>
      <mo stretchy="false">(</mo>
      <mrow>
       <mi>a</mi>
       <mo>-</mo>
       <mi>b</mi>
      </mrow>
      <mo stretchy="false">)</mo>
     </mrow>
     <mrow>
      <mo stretchy="false">(</mo>
      <mrow>
       <mi>c</mi>
       <mo>+</mo>
       <mi>d</mi>
      </mrow>
      <mo stretchy="false">)</mo>
     </mrow>
    </mrow>
   </msqrt>
  </mpadded>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <root></root>
    <apply>
     <times></times>
     <apply>
      <minus></minus>
      <ci>a</ci>
      <ci>b</ci>
     </apply>
     <apply>
      <plus></plus>
      <ci>c</ci>
      <ci>d</ci>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \sqrt{(a-b)(c+d)}\,
  </annotation>
 </semantics>
</math>

 can also be represented as an expression tree:</p>
<table>
<tbody>
<tr class="odd">
<td style="text-align: left;"><figure><b>(Figure)</b>
<figcaption>GEP expression tree, k-expression Q*-+abcd.png</figcaption>
</figure></td>
</tr>
</tbody>
</table>

<p>where “Q” represents the square root function.</p>

<p>This kind of expression tree consists of the phenotypic expression of GEP genes, whereas the genes are linear strings encoding these complex structures. For this particular example, the linear string corresponds to:</p>
<dl>
<dd><code><nowiki>01234567</nowiki></code>
</dd>
<dd>
</dd>
<dd><code><nowiki>Q*-+abcd</nowiki></code>
</dd>
</dl>

<p>which is the straightforward reading of the expression tree from top to bottom and from left to right. These linear strings are called k-expressions (from <a href="Karva_notation" title="wikilink">Karva notation</a>).</p>

<p>Going from k-expressions to expression trees is also very simple. For example, the following k-expression:</p>
<dl>
<dd><code><nowiki>01234567890</nowiki></code>
</dd>
<dd>
</dd>
<dd><code><nowiki>Q*b**+baQba</nowiki></code>
</dd>
</dl>

<p>is composed of two different terminals (the variables “a” and “b”), two different functions of two arguments (“*” and “+”), and a function of one argument (“Q”). Its expression gives:</p>
<table>
<tbody>
<tr class="odd">
<td style="text-align: left;"><figure><b>(Figure)</b>
<figcaption>GEP expression tree, k-expression Q*b**+baQba.png</figcaption>
</figure></td>
</tr>
</tbody>
</table>
<h2 id="k-expressions-and-genes">K-expressions and genes</h2>

<p>The k-expressions of gene expression programming correspond to the region of genes that gets expressed. This means that there might be sequences in the genes that are not expressed, which is indeed true for most genes. The reason for these noncoding regions is to provide a buffer of terminals so that all k-expressions encoded in GEP genes correspond always to valid programs or expressions.</p>

<p>The genes of gene expression programming are therefore composed of two different domains – a head and a tail – each with different properties and functions. The head is used mainly to encode the functions and variables chosen to solve the problem at hand, whereas the tail, while also used to encode the variables, provides essentially a reservoir of terminals to ensure that all programs are error-free.</p>

<p>For GEP genes the length of the tail is given by the formula:</p>

<p>

<math display="block" id="Gene_expression_programming:1">
 <semantics>
  <mrow>
   <mi>t</mi>
   <mo>=</mo>
   <mrow>
    <mrow>
     <mi>h</mi>
     <mrow>
      <mo stretchy="false">(</mo>
      <mrow>
       <msub>
        <mi>n</mi>
        <mi>max</mi>
       </msub>
       <mo>-</mo>
       <mn>1</mn>
      </mrow>
      <mo stretchy="false">)</mo>
     </mrow>
    </mrow>
    <mo>+</mo>
    <mn>1</mn>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <eq></eq>
    <ci>t</ci>
    <apply>
     <plus></plus>
     <apply>
      <times></times>
      <ci>h</ci>
      <apply>
       <minus></minus>
       <apply>
        <csymbol cd="ambiguous">subscript</csymbol>
        <ci>n</ci>
        <max></max>
       </apply>
       <cn type="integer">1</cn>
      </apply>
     </apply>
     <cn type="integer">1</cn>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   t=h(n_{\max}-1)+1
  </annotation>
 </semantics>
</math>

</p>

<p>where <em>h</em> is the head’s length and <em>n</em><sub>max</sub> is maximum arity. For example, for a gene created using the set of functions F = {Q, +, −, *, /} and the set of terminals T = {a, b}, <em>n</em><sub>max</sub> = 2. And if we choose a head length of 15, then <em>t</em> = 15 (2 − 1) + 1 = 16, which gives a gene length <em>g</em> of 15 + 16 = 31. The randomly generated string below is an example of one such gene:</p>
<dl>
<dd><code><nowiki>0123456789012345678901234567890</nowiki></code>
</dd>
<dd>
</dd>
<dd><code><nowiki>*b+a-aQab+//+b+babbabbbababbaaa</nowiki></code>
</dd>
</dl>

<p>It encodes the expression tree:</p>
<table>
<tbody>
<tr class="odd">
<td style="text-align: left;"><figure><b>(Figure)</b>
<figcaption>GEP expression tree, k-expression *b+a-aQa.png</figcaption>
</figure></td>
</tr>
</tbody>
</table>

<p>which, in this case, only uses 8 of the 31 elements that constitute the gene.</p>

<p>It’s not hard to see that, despite their fixed length, each gene has the potential to code for expression trees of different sizes and shapes, with the simplest composed of only one node (when the first element of a gene is a terminal) and the largest composed of as many nodes as there are elements in the gene (when all the elements in the head are functions with maximum arity).</p>

<p>It’s also not hard to see that it is trivial to implement all kinds of <a href="genetic_operator" title="wikilink">genetic modification</a> (<a href="mutation_(genetic_algorithm)" title="wikilink">mutation</a>, inversion, insertion, <a href="crossover_(genetic_algorithm)" title="wikilink">recombination</a>, and so on) with the guarantee that all resulting offspring encode correct, error-free programs.</p>
<h2 id="multigenic-chromosomes">Multigenic chromosomes</h2>

<p>The chromosomes of gene expression programming are usually composed of more than one gene of equal length. Each gene codes for a sub-expression tree (sub-ET) or sub-program. Then the sub-ETs can interact with one another in different ways, forming a more complex program. The figure shows an example of a program composed of three sub-ETs.</p>
<figure><b>(Figure)</b>
<figcaption>Expression of GEP genes as sub-ETs. a) A three-genic chromosome with the tails shown in bold. b) The sub-ETs encoded by each gene.</figcaption>
</figure>

<p>In the final program the sub-ETs could be linked by addition or some other function, as there are no restrictions to the kind of linking function one might choose. Some examples of more complex linkers include taking the average, the median, the midrange, thresholding their sum to make a binomial classification, applying the sigmoid function to compute a probability, and so on. These linking functions are usually chosen a priori for each problem, but they can also be evolved elegantly and efficiently by the <a href="gene_expression_programming#Cells_and_code_reuse" title="wikilink">cellular system</a><a class="footnoteRef" href="#fn6" id="fnref6"><sup>6</sup></a><a class="footnoteRef" href="#fn7" id="fnref7"><sup>7</sup></a> of gene expression programming.</p>
<h2 id="cells-and-code-reuse">Cells and code reuse</h2>

<p>In gene expression programming, <a href="gene_expression_programming#Homeotic_genes_and_the_cellular_system" title="wikilink">homeotic genes</a> control the interactions of the different sub-ETs or modules of the main program. The expression of such genes results in different main programs or cells, that is, they determine which genes are expressed in each cell and how the sub-ETs of each cell interact with one another. In other words, homeotic genes determine which sub-ETs are called upon and how often in which main program or cell and what kind of connections they establish with one another.</p>
<h3 id="homeotic-genes-and-the-cellular-system">Homeotic genes and the cellular system</h3>

<p>Homeotic genes have exactly the same kind of structural organization as normal genes and they are built using an identical process. They also contain a head domain and a tail domain, with the difference that the heads contain now linking functions and a special kind of terminals – genic terminals – that represent the normal genes. The expression of the normal genes results as usual in different sub-ETs, which in the cellular system are called ADFs (automatically defined functions). As for the tails, they contain only genic terminals, that is, derived features generated on the fly by the algorithm.</p>

<p>For example, the chromosome in the figure has three normal genes and one homeotic gene and encodes a main program that invokes three different functions a total of four times, linking them in a particular way.</p>
<figure><b>(Figure)</b>
<figcaption>Expression of a unicellular system with three ADFs. a) The chromosome composed of three conventional genes and one homeotic gene (shown in bold). b) The ADFs encoded by each conventional gene. c) The main program or cell.</figcaption>
</figure>

<p>From this example it is clear that the cellular system not only allows the unconstrained evolution of linking functions but also code reuse. And it shouldn't be hard to implement <a href="Recursion_(computer_science)" title="wikilink">recursion</a> in this system.</p>
<h3 id="multiple-main-programs-and-multicellular-systems">Multiple main programs and multicellular systems</h3>

<p>Multicellular systems are composed of more than one <a href="gene_expression_programming#Homeotic_genes_and_the_cellular_system" title="wikilink">homeotic gene</a>. Each homeotic gene in this system puts together a different combination of sub-expression trees or ADFs, creating multiple cells or main programs.</p>

<p>For example, the program shown in the figure was created using a cellular system with two cells and three normal genes.</p>
<figure><b>(Figure)</b>
<figcaption>Expression of a multicellular system with three ADFs and two main programs. a) The chromosome composed of three conventional genes and two homeotic genes (shown in bold). b) The ADFs encoded by each conventional gene. c) Two different main programs expressed in two different cells.</figcaption>
</figure>

<p>The applications of these multicellular systems are multiple and varied and, like the <a href="gene_expression_programming#Multigenic_chromosomes" title="wikilink">multigenic systems</a>, they can be used both in problems with just one output and in problems with multiple outputs.</p>
<h2 id="other-levels-of-complexity">Other levels of complexity</h2>

<p>The head/tail domain of GEP genes (both normal and homeotic) is the basic building block of all GEP algorithms. However, gene expression programming also explores other chromosomal organizations that are more complex than the head/tail structure. Essentially these complex structures consist of functional units or genes with a basic head/tail domain plus one or more extra domains. These extra domains usually encode random numerical constants that the algorithm relentlessly fine-tunes in order to find a good solution. For instance, these numerical constants may be the weights or factors in a function approximation problem (see the <a href="gene_expression_programming#The_GEP-RNC_algorithm" title="wikilink">GEP-RNC algorithm</a> below); they may be the weights and thresholds of a neural network (see the <a href="gene_expression_programming#Neural_networks" title="wikilink">GEP-NN algorithm</a> below); the numerical constants needed for the design of decision trees (see the <a href="gene_expression_programming#Decision_trees" title="wikilink">GEP-DT algorithm</a> below); the weights needed for polynomial induction; or the random numerical constants used to discover the parameter values in a parameter optimization task.</p>
<h2 id="the-basic-gene-expression-algorithm">The basic gene expression algorithm</h2>

<p>The fundamental steps of the basic gene expression algorithm are listed below in pseudocode:</p>
<dl>
<dd>1. Select function set;
</dd>
<dd>2. Select terminal set;
</dd>
<dd>3. Load dataset for fitness evaluation;
</dd>
<dd>4. Create chromosomes of initial population randomly;
</dd>
<dd>5. For each program in population:
<dl>
<dd>a) Express chromosome;
</dd>
<dd>b) Execute program;
</dd>
<dd>c) Evaluate fitness;
</dd>
</dl>
</dd>
<dd>6. Verify stop condition;
</dd>
<dd>7. Select programs;
</dd>
<dd>8. Replicate selected programs to form the next population;
</dd>
<dd>9. Modify chromosomes using genetic operators;
</dd>
<dd>10. Go to step 5.
</dd>
</dl>

<p>The first four steps prepare all the ingredients that are needed for the iterative loop of the algorithm (steps 5 through 10). Of these preparative steps, the crucial one is the creation of the initial population, which is created randomly using the elements of the function and terminal sets.</p>
<h3 id="populations-of-programs">Populations of programs</h3>

<p>Like all evolutionary algorithms, gene expression programming works with populations of individuals, which in this case are computer programs. Therefore some kind of initial population must be created to get things started. Subsequent populations are descendants, via <a href="gene_expression_programming#Selection_and_elitism" title="wikilink">selection</a> and <a href="gene_expression_programming#Reproduction_with_modification" title="wikilink">genetic modification</a>, of the initial population.</p>

<p>In the genotype/phenotype system of gene expression programming, it is only necessary to create the simple linear chromosomes of the individuals without worrying about the structural soundness of the programs they code for, as their expression always results in syntactically correct programs.</p>
<h3 id="fitness-functions-and-the-selection-environment">Fitness functions and the selection environment</h3>

<p>Fitness functions and selection environments (called training datasets in <a href="machine_learning" title="wikilink">machine learning</a>) are the two facets of fitness and are therefore intricately connected. Indeed, the fitness of a program depends not only on the <a href="Loss_function" title="wikilink">cost function</a> used to measure its performance but also on the training data chosen to evaluate fitness</p>
<h4 id="the-selection-environment-or-training-data">The selection environment or training data</h4>

<p>The selection environment consists of the set of training records, which are also called fitness cases. These fitness cases could be a set of observations or measurements concerning some problem, and they form what is called the training dataset.</p>

<p>The quality of the training data is essential for the evolution of good solutions. A good training set should be representative of the problem at hand and also well-balanced, otherwise the algorithm might get stuck at some local optimum. In addition, it is also important to avoid using unnecessarily large datasets for training as this will slow things down unnecessarily. A good rule of thumb is to choose enough records for training to enable a good generalization in the validation data and leave the remaining records for validation and testing.</p>
<h4 id="fitness-functions">Fitness functions</h4>

<p>Broadly speaking, there are essentially three different kinds of problems based on the kind of prediction being made:</p>
<dl>
<dd>1. Problems involving numeric (continuous) predictions;
</dd>
<dd>2. Problems involving categorical or nominal predictions, both binomial and multinomial;
</dd>
<dd>3. Problems involving binary or Boolean predictions.
</dd>
</dl>

<p>The first type of problem goes by the name of <a href="Regression_analysis" title="wikilink">regression</a>; the second is known as <a href="Statistical_classification" title="wikilink">classification</a>, with <a href="logistic_regression" title="wikilink">logistic regression</a> as a special case where, besides the crisp classifications like “Yes” or “No”, a probability is also attached to each outcome; and the last one is related to <a href="Boolean_algebra_(logic)" title="wikilink">Boolean algebra</a> and <a href="logic_synthesis" title="wikilink">logic synthesis</a>.</p>
<h5 id="fitness-functions-for-regression">Fitness functions for regression</h5>

<p>In <a href="Regression_analysis" title="wikilink">regression</a>, the response or dependent variable is numeric (usually continuous) and therefore the output of a regression model is also continuous. So it’s quite straightforward to evaluate the fitness of the evolving models by comparing the output of the model to the value of the response in the training data.</p>

<p>There are several basic <a href="fitness_function" title="wikilink">fitness functions</a> for evaluating model performance, with the most common being based on the error or residual between the model output and the actual value. Such functions include the <a href="mean_squared_error" title="wikilink">mean squared error</a>, <a href="root_mean_squared_error" title="wikilink">root mean squared error</a>, <a href="mean_absolute_error" title="wikilink">mean absolute error</a>, relative squared error, root relative squared error, relative absolute error, and others.</p>

<p>All these standard measures offer a fine granularity or smoothness to the solution space and therefore work very well for most applications. But some problems might require a coarser evolution, such as determining if a prediction is within a certain interval, for instance less than 10% of the actual value. However, even if one is only interested in counting the hits (that is, a prediction that is within the chosen interval), making populations of models evolve based on just the number of hits each program scores is usually not very efficient due to the coarse granularity of the <a href="fitness_landscape" title="wikilink">fitness landscape</a>. Thus the solution usually involves combining these coarse measures with some kind of smooth function such as the standard error measures listed above.</p>

<p>Fitness functions based on the <a href="Pearson_product-moment_correlation_coefficient" title="wikilink">correlation coefficient</a> and <a class="uri" href="R-square" title="wikilink">R-square</a> are also very smooth. For regression problems, these functions work best by combining them with other measures because, by themselves, they only tend to measure <a href="Correlation_and_dependence" title="wikilink">correlation</a>, not caring for the range of values of the model output. So by combining them with functions that work at approximating the range of the target values, they form very efficient fitness functions for finding models with good correlation and good fit between predicted and actual values.</p>
<h5 id="fitness-functions-for-classification-and-logistic-regression">Fitness functions for classification and logistic regression</h5>

<p>The design of fitness functions for <a href="Statistical_classification" title="wikilink">classification</a> and <a href="logistic_regression" title="wikilink">logistic regression</a> takes advantage of three different characteristics of classification models. The most obvious is just counting the hits, that is, if a record is classified correctly it is counted as a hit. This fitness function is very simple and works well for simple problems, but for more complex problems or datasets highly unbalanced it gives poor results.</p>

<p>One way to improve this type of hits-based fitness function consists of expanding the notion of correct and incorrect classifications. In a binary classification task, correct classifications can be 00 or 11. The “00” representation means that a negative case (represented by “0”) was correctly classified, whereas the “11” means that a positive case (represented by “1”) was correctly classified. Classifications of the type “00” are called true negatives (TN) and “11” true positives (TP).</p>

<p>There are also two types of incorrect classifications and they are represented by 01 and 10. They are called false positives (FP) when the actual value is 0 and the model predicts a 1; and false negatives (FN) when the target is 1 and the model predicts a 0. The counts of TP, TN, FP, and FN are usually kept on a table known as the <a href="confusion_matrix" title="wikilink">confusion matrix</a>.</p>
<figure><b>(Figure)</b>
<figcaption><a href="Confusion_matrix" title="wikilink">Confusion matrix</a> for a binomial classification task.</figcaption>
</figure>

<p>So by counting the TP, TN, FP, and FN and further assigning different weights to these four types of classifications, it is possible to create smoother and therefore more efficient fitness functions. Some popular fitness functions based on the confusion matrix include <a href="Sensitivity_and_specificity" title="wikilink">sensitivity/specificity</a>, <a href="Precision_and_recall" title="wikilink">recall/precision</a>, <a class="uri" href="F-measure" title="wikilink">F-measure</a>, <a href="Jaccard_similarity" title="wikilink">Jaccard similarity</a>, <a href="Matthews_correlation_coefficient" title="wikilink">Matthews correlation coefficient</a>, and cost/gain matrix which combines the costs and gains assigned to the 4 different types of classifications.</p>

<p>These functions based on the confusion matrix are quite sophisticated and are adequate to solve most problems efficiently. But there is another dimension to classification models which is key to exploring more efficiently the solution space and therefore results in the discovery of better classifiers. This new dimension involves exploring the structure of the model itself, which includes not only the domain and range, but also the distribution of the model output and the classifier margin.</p>

<p>By exploring this other dimension of classification models and then combining the information about the model with the confusion matrix, it is possible to design very sophisticated fitness functions that allow the smooth exploration of the solution space. For instance, one can combine some measure based on the confusion matrix with the <a href="mean_squared_error" title="wikilink">mean squared error</a> evaluated between the raw model outputs and the actual values. Or combine the <a class="uri" href="F-measure" title="wikilink">F-measure</a> with the <a class="uri" href="R-square" title="wikilink">R-square</a> evaluated for the raw model output and the target; or the cost/gain matrix with the <a href="Pearson_product-moment_correlation_coefficient" title="wikilink">correlation coefficient</a>, and so on. More exotic fitness functions that explore model granularity include the area under the <a href="Receiver_operating_characteristic" title="wikilink">ROC curve</a> and rank measure.</p>

<p>Also related to this new dimension of classification models, is the idea of assigning probabilities to the model output, which is what is done in <a href="logistic_regression" title="wikilink">logistic regression</a>. Then it is also possible to use these probabilities and evaluate the <a href="mean_squared_error" title="wikilink">mean squared error</a> (or some other similar measure) between the probabilities and the actual values, then combine this with the confusion matrix to create very efficient fitness functions for logistic regression. Popular examples of fitness functions based on the probabilities include <a href="maximum_likelihood_estimation" title="wikilink">maximum likelihood estimation</a> and <a href="hinge_loss" title="wikilink">hinge loss</a>.</p>
<h5 id="fitness-functions-for-boolean-problems">Fitness functions for Boolean problems</h5>

<p>In logic there is no model structure (as defined <a href="gene_expression_programming#Fitness_functions_for_classification_and_logistic_regression" title="wikilink">above</a> for classification and logistic regression) to explore: the domain and range of logical functions comprises only 0’s and 1’s or false and true. So, the fitness functions available for <a href="Boolean_algebra_(logic)" title="wikilink">Boolean algebra</a> can only be based on the hits or on the confusion matrix as explained in the section <a href="gene_expression_programming#Fitness_functions_for_classification_and_logistic_regression" title="wikilink">above</a>.</p>
<h3 id="selection-and-elitism">Selection and elitism</h3>

<p><a href="Roulette-wheel_selection" title="wikilink">Roulette-wheel selection</a> is perhaps the most popular selection scheme used in evolutionary computation. It involves mapping the fitness of each program to a slice of the roulette wheel proportional to its fitness. Then the roulette is spun as many times as there are programs in the population in order to keep the population size constant. So, with roulette-wheel selection programs are selected both according to fitness and the luck of the draw, which means that some times the best traits might be lost. However, by combining roulette-wheel selection with the cloning of the best program of each generation, one guarantees that at least the very best traits are not lost. This technique of cloning the best-of-generation program is known as simple elitism and is used by most stochastic selection schemes.</p>
<h3 id="reproduction-with-modification">Reproduction with modification</h3>

<p>The reproduction of programs involves first the selection and then the reproduction of their genomes. Genome modification is not required for reproduction, but without it adaptation and evolution won’t take place.</p>
<h4 id="replication-and-selection">Replication and selection</h4>

<p>The selection operator selects the programs for the replication operator to copy. Depending on the selection scheme, the number of copies one program originates may vary, with some programs getting copied more than once while others are copied just once or not at all. In addition, selection is usually set up so that the population size remains constant from one generation to another.</p>

<p>The replication of genomes in nature is very complex and it took scientists a long time to discover the <a href="DNA_double_helix" title="wikilink">DNA double helix</a> and propose a mechanism for its replication. But the replication of strings is trivial in artificial evolutionary systems, where only an instruction to copy strings is required to pass all the information in the genome from generation to generation.</p>

<p>The replication of the selected programs is a fundamental piece of all artificial evolutionary systems, but for evolution to occur it needs to be implemented not with the usual precision of a copy instruction, but rather with a few errors thrown in. Indeed, genetic diversity is created with <a href="genetic_operator" title="wikilink">genetic operators</a> such as <a href="Mutation_(genetic_algorithm)" title="wikilink">mutation</a>, <a href="Crossover_(genetic_algorithm)" title="wikilink">recombination</a>, <a href="Transposition_(genetics)" title="wikilink">transposition</a>, inversion, and many others.</p>
<h4 id="mutation">Mutation</h4>

<p>In gene expression programming mutation is the most important genetic operator.<a class="footnoteRef" href="#fn8" id="fnref8"><sup>8</sup></a> It changes genomes by changing an element by another. The accumulation of many small changes over time can create great diversity.</p>

<p>In gene expression programming mutation is totally unconstrained, which means that in each gene domain any domain symbol can be replaced by another. For example, in the heads of genes any function can be replaced by a terminal or another function, regardless of the number of arguments in this new function; and a terminal can be replaced by a function or another terminal.</p>
<h4 id="recombination">Recombination</h4>

<p><a href="Crossover_(genetic_algorithm)" title="wikilink">Recombination</a> usually involves two parent chromosomes to create two new chromosomes by combining different parts from the parent chromosomes. And as long as the parent chromosomes are aligned and the exchanged fragments are homologous (that is, occupy the same position in the chromosome), the new chromosomes created by recombination will always encode syntactically correct programs.</p>

<p>Different kinds of crossover are easily implemented either by changing the number of parents involved (there’s no reason for choosing only two); the number of split points; or the way one chooses to exchange the fragments, for example, either randomly or in some orderly fashion. For example, gene recombination, which is a special case of recombination, can be done by exchanging homologous genes (genes that occupy the same position in the chromosome) or by exchanging genes chosen at random from any position in the chromosome.</p>
<h4 id="transposition">Transposition</h4>

<p><a href="Transposition_(genetics)" title="wikilink">Transposition</a> involves the introduction of an insertion sequence somewhere in a chromosome. In gene expression programming insertion sequences might appear anywhere in the chromosome, but they are only inserted in the heads of genes. This method guarantees that even insertion sequences from the tails result in error-free programs.</p>

<p>For transposition to work properly, it must preserve chromosome length and gene structure. So, in gene expression programming transposition can be implemented using two different methods: the first creates a shift at the insertion site, followed by a deletion at the end of the head; the second overwrites the local sequence at the target site and therefore is easier to implement. Both methods can be implemented to operate between chromosomes or within a chromosome or even within a single gene.</p>
<h4 id="inversion">Inversion</h4>

<p>Inversion is an interesting operator, especially powerful for combinatorial optimization.<a class="footnoteRef" href="#fn9" id="fnref9"><sup>9</sup></a> It consists of inverting a small sequence within a chromosome.</p>

<p>In gene expression programming it can be easily implemented in all gene domains and, in all cases, the offspring produced is always syntactically correct. For any gene domain, a sequence (ranging from at least two elements to as big as the domain itself) is chosen at random within that domain and then inverted.</p>
<h4 id="other-genetic-operators">Other genetic operators</h4>

<p>Several other genetic operators exist and in gene expression programming, with its different genes and gene domains, the possibilities are endless. For example, genetic operators such as one-point recombination, two-point recombination, gene recombination, uniform recombination, gene transposition, root transposition, domain-specific mutation, domain-specific inversion, domain-specific transposition, and so on, are easily implemented and widely used.</p>
<h2 id="the-gep-rnc-algorithm">The GEP-RNC algorithm</h2>

<p>Numerical constants are essential elements of mathematical and statistical models and therefore it is important to allow their integration in the models designed by evolutionary algorithms.</p>

<p>Gene expression programming solves this problem very elegantly through the use of an extra gene domain – the Dc – for handling random numerical constants (RNC). By combining this domain with a special terminal placeholder for the RNCs, a richly expressive system can be created.</p>

<p>Structurally, the Dc comes after the tail, has a length equal to the size of the tail <em>t</em>, and is composed of the symbols used to represent the RNCs.</p>

<p>For example, below is shown a simple chromosome composed of only one gene a head size of 7 (the Dc stretches over positions 15–22):</p>
<dl>
<dd><code><nowiki>01234567890123456789012</nowiki></code>
</dd>
<dd>
</dd>
<dd><code><nowiki>+?*+?**aaa??aaa68083295</nowiki></code>
</dd>
</dl>

<p>where the terminal “?” represents the placeholder for the RNCs. This kind of chromosome is expressed exactly as shown <a href="gene_expression_programming#Expression_trees:_the_phenotype" title="wikilink">above</a>, giving:</p>
<table>
<tbody>
<tr class="odd">
<td style="text-align: left;"><figure><b>(Figure)</b>
<figcaption>GEP expression tree with placeholder for RNCs.png</figcaption>
</figure></td>
</tr>
</tbody>
</table>

<p>Then the ?’s in the expression tree are replaced from left to right and from top to bottom by the symbols (for simplicity represented by numerals) in the Dc, giving:</p>
<table>
<tbody>
<tr class="odd">
<td style="text-align: left;"><figure><b>(Figure)</b>
<figcaption>GEP expression tree with symbols (numerals) for RNCs.png</figcaption>
</figure></td>
</tr>
</tbody>
</table>

<p>The values corresponding to these symbols are kept in an array. (For simplicity, the number represented by the numeral indicates the order in the array.) For instance, for the following 10 element array of RNCs:</p>
<dl>
<dd>C = {0.611, 1.184, 2.449, 2.98, 0.496, 2.286, 0.93, 2.305, 2.737, 0.755}
</dd>
</dl>

<p>the expression tree above gives:</p>
<table>
<tbody>
<tr class="odd">
<td style="text-align: left;"><figure><b>(Figure)</b>
<figcaption>GEP expression tree with RNCs.png</figcaption>
</figure></td>
</tr>
</tbody>
</table>

<p>This elegant structure for handling random numerical constants is at the heart of different GEP systems, such as <a href="gene_expression_programming#Neural_networks" title="wikilink">GEP neural networks</a> and <a href="gene_expression_programming#Decision_trees" title="wikilink">GEP decision trees</a>.</p>

<p>Like the <a href="gene_expression_programming#The_basic_gene_expression_algorithm" title="wikilink">basic gene expression algorithm</a>, the GEP-RNC algorithm is also multigenic and its chromosomes are decoded as usual by expressing one gene after another and then linking them all together by the same kind of linking process.</p>

<p>The genetic operators used in the GEP-RNC system are an extension to the genetic operators of the basic GEP algorithm (see <a href="gene_expression_programming#Reproduction_with_modification" title="wikilink">above</a>), and they all can be straightforwardly implemented in these new chromosomes. On the other hand, the basic operators of mutation, inversion, transposition, and recombination are also used in the GEP-RNC algorithm. Furthermore, special Dc-specific operators such as mutation, inversion, and transposition, are also used to aid in a more efficient circulation of the RNCs among individual programs. In addition, there is also a special mutation operator that allows the permanent introduction of variation in the set of RNCs. The initial set of RNCs is randomly created at the beginning of a run, which means that, for each gene in the initial population, a specified number of numerical constants, chosen from a certain range, are randomly generated. Then their circulation and mutation is enabled by the genetic operators.</p>
<h2 id="neural-networks">Neural networks</h2>

<p>An <a href="artificial_neural_network" title="wikilink">artificial neural network</a> (ANN or NN) is a computational device that consists of many simple connected units or neurons. The connections between the units are usually weighted by real-valued weights. These weights are the primary means of learning in neural networks and a learning algorithm is usually used to adjust them.</p>

<p>Structurally, a neural network has three different classes of units: input units, hidden units, and output units. An activation pattern is presented at the input units and then spreads in a forward direction from the input units through one or more layers of hidden units to the output units. The activation coming into one unit from other unit is multiplied by the weights on the links over which it spreads. All incoming activation is then added together and the unit becomes activated only if the incoming result is above the unit’s threshold.</p>

<p>In summary, the basic components of a neural network are the units, the connections between the units, the weights, and the thresholds. So, in order to fully simulate an artificial neural network one must somehow encode these components in a linear chromosome and then be able to express them in a meaningful way.</p>

<p>In GEP neural networks (GEP-NN or GEP nets), the network architecture is encoded in the usual structure of a head/tail domain.<a class="footnoteRef" href="#fn10" id="fnref10"><sup>10</sup></a> The head contains special functions/neurons that activate the hidden and output units (in the GEP context, all these units are more appropriately called functional units) and terminals that represent the input units. The tail, as usual, contains only terminals/input units.</p>

<p>Besides the head and the tail, these neural network genes contain two additional domains, Dw and Dt, for encoding the weights and thresholds of the neural network. Structurally, the Dw comes after the tail and its length <em>d<sub>w</sub></em> depends on the head size <em>h</em> and maximum arity <em>n</em><sub>max</sub> and is evaluated by the formula:</p>

<p>

<math display="block" id="Gene_expression_programming:2">
 <semantics>
  <mrow>
   <msub>
    <mi>d</mi>
    <mi>w</mi>
   </msub>
   <mo>=</mo>
   <mrow>
    <mi>h</mi>
    <msub>
     <mi>n</mi>
     <mi>max</mi>
    </msub>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <eq></eq>
    <apply>
     <csymbol cd="ambiguous">subscript</csymbol>
     <ci>d</ci>
     <ci>w</ci>
    </apply>
    <apply>
     <times></times>
     <ci>h</ci>
     <apply>
      <csymbol cd="ambiguous">subscript</csymbol>
      <ci>n</ci>
      <max></max>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   d_{w}=hn_{\max}
  </annotation>
 </semantics>
</math>

</p>

<p>The Dt comes after Dw and has a length <em>d<sub>t</sub></em> equal to <em>t</em>. Both domains are composed of symbols representing the weights and thresholds of the neural network.</p>

<p>For each NN-gene, the weights and thresholds are created at the beginning of each run, but their circulation and adaptation are guaranteed by the usual genetic operators of <a href="gene_expression_programming#Mutation" title="wikilink">mutation</a>, <a href="gene_expression_programming#Transposition" title="wikilink">transposition</a>, <a href="gene_expression_programming#Inversion" title="wikilink">inversion</a>, and <a href="gene_expression_programming#Recombination" title="wikilink">recombination</a>. In addition, special operators are also used to allow a constant flow of genetic variation in the set of weights and thresholds.</p>

<p>For example, below is shown a neural network with two input units (<em>i</em><sub>1</sub> and <em>i</em><sub>2</sub>), two hidden units (<em>h</em><sub>1</sub> and <em>h</em><sub>2</sub>), and one output unit (<em>o</em><sub>1</sub>). It has a total of six connections with six corresponding weights represented by the numerals 1–6 (for simplicity, the thresholds are all equal to 1 and are omitted):</p>
<table>
<tbody>
<tr class="odd">
<td style="text-align: left;"><figure><b>(Figure)</b>
<figcaption>Neural network with 5 units.png</figcaption>
</figure></td>
</tr>
</tbody>
</table>

<p>This representation is the canonical neural network representation, but neural networks can also be represented by a tree, which, in this case, corresponds to:</p>
<table>
<tbody>
<tr class="odd">
<td style="text-align: left;"><figure><b>(Figure)</b>
<figcaption>GEP neural network with 7 nodes.png</figcaption>
</figure></td>
</tr>
</tbody>
</table>

<p>where “a” and “b” represent the two inputs <em>i</em><sub>1</sub> and <em>i</em><sub>2</sub> and “D” represents a function with connectivity two. This function adds all its weighted arguments and then thresholds this activation in order to determine the forwarded output. This output (zero or one in this simple case) depends on the threshold of each unit, that is, if the total incoming activation is equal to or greater than the threshold, then the output is one, zero otherwise.</p>

<p>The above NN-tree can be linearized as follows:</p>
<dl>
<dd><code><nowiki>0123456789012</nowiki></code>
</dd>
<dd>
</dd>
<dd><code><nowiki>DDDabab654321</nowiki></code>
</dd>
</dl>

<p>where the structure in positions 7–12 (Dw) encodes the weights. The values of each weight are kept in an array and retrieved as necessary for expression.</p>

<p>As a more concrete example, below is shown a neural net gene for the <a href="exclusive_or" title="wikilink">exclusive-or</a> problem. It has a head size of 3 and Dw size of 6:</p>
<dl>
<dd><code><nowiki>0123456789012</nowiki></code>
</dd>
<dd>
</dd>
<dd><code><nowiki>DDDabab393257</nowiki></code>
</dd>
</dl>

<p>Its expression results in the following neural network:</p>
<table>
<tbody>
<tr class="odd">
<td style="text-align: left;"><figure><b>(Figure)</b>
<figcaption>Expression of a GEP neural network for the exclusive-or.png</figcaption>
</figure></td>
</tr>
</tbody>
</table>

<p>which, for the set of weights:</p>
<dl>
<dd><em>W</em> = {−1.978, 0.514, −0.465, 1.22, −1.686, −1.797, 0.197, 1.606, 0, 1.753}
</dd>
</dl>

<p>it gives:</p>
<table>
<tbody>
<tr class="odd">
<td style="text-align: left;"><figure><b>(Figure)</b>
<figcaption>GEP neural network solution for the exclusive-or.png</figcaption>
</figure></td>
</tr>
</tbody>
</table>

<p>which is a perfect solution to the exclusive-or function.</p>

<p>Besides simple Boolean functions with binary inputs and binary outputs, the GEP-nets algorithm can handle all kinds of functions or neurons (linear neuron, tanh neuron, atan neuron, logistic neuron, limit neuron, radial basis and triangular basis neurons, all kinds of step neurons, and so on). Also interesting is that the GEP-nets algorithm can use all these neurons together and let evolution decide which ones work best to solve the problem at hand. So, GEP-nets can be used not only in Boolean problems but also in <a href="logistic_regression" title="wikilink">logistic regression</a>, <a href="Statistical_classification" title="wikilink">classification</a>, and <a href="Regression_analysis" title="wikilink">regression</a>. In all cases, GEP-nets can be implemented not only with <a href="gene_expression_programming#Multigenic_chromosomes" title="wikilink">multigenic systems</a> but also <a href="gene_expression_programming#Cells_and_code_reuse" title="wikilink">cellular systems</a>, both unicellular and multicellular. Furthermore, multinomial classification problems can also be tackled in one go by GEP-nets both with multigenic systems and multicellular systems.</p>
<h2 id="decision-trees">Decision trees</h2>

<p><a href="Decision_trees" title="wikilink">Decision trees</a> (DT) are classification models where a series of questions and answers are mapped using nodes and directed edges.</p>

<p>Decision trees have three types of nodes: a root node, internal nodes, and leaf or terminal nodes. The root node and all internal nodes represent test conditions for different attributes or variables in a dataset. Leaf nodes specify the class label for all different paths in the tree.</p>

<p>Most decision tree induction algorithms involve selecting an attribute for the root node and then make the same kind of informed decision about all the nodes in a tree.</p>

<p>Decision trees can also be created by gene expression programming,<a class="footnoteRef" href="#fn11" id="fnref11"><sup>11</sup></a> with the advantage that all the decisions concerning the growth of the tree are made by the algorithm itself without any kind of human input.</p>

<p>There are basically two different types of DT algorithms: one for inducing decision trees with only nominal attributes and another for inducing decision trees with both numeric and nominal attributes. This aspect of decision tree induction also carries to gene expression programming and there are two GEP algorithms for decision tree induction: the evolvable decision trees (EDT) algorithm for dealing exclusively with nominal attributes and the EDT-RNC (EDT with random numerical constants) for handling both nominal and numeric attributes.</p>

<p>In the decision trees induced by gene expression programming, the attributes behave as function nodes in the <a href="gene_expression_programming#The_basic_gene_expression_algorithm" title="wikilink">basic gene expression algorithm</a>, whereas the class labels behave as terminals. This means that attribute nodes have also associated with them a specific arity or number of branches that will determine their growth and, ultimately, the growth of the tree. Class labels behave like terminals, which means that for a <em>k</em>-class classification task, a terminal set with <em>k</em> terminals is used, representing the <em>k</em> different classes.</p>

<p>The rules for encoding a decision tree in a linear genome are very similar to the rules used to encode mathematical expressions (see <a href="gene_expression_programming#K-expressions_and_genes" title="wikilink">above</a>). So, for decision tree induction the genes also have a head and a tail, with the head containing attributes and terminals and the tail containing only terminals. This again ensures that all decision trees designed by GEP are always valid programs. Furthermore, the size of the tail <em>t</em> is also dictated by the head size <em>h</em> and the number of branches of the attribute with more branches <em>n</em><sub>max</sub> and is evaluated by the equation:</p>

<p>

<math display="block" id="Gene_expression_programming:3">
 <semantics>
  <mrow>
   <mi>t</mi>
   <mo>=</mo>
   <mrow>
    <mrow>
     <mi>h</mi>
     <mrow>
      <mo stretchy="false">(</mo>
      <mrow>
       <msub>
        <mi>n</mi>
        <mi>max</mi>
       </msub>
       <mo>-</mo>
       <mn>1</mn>
      </mrow>
      <mo stretchy="false">)</mo>
     </mrow>
    </mrow>
    <mo>+</mo>
    <mpadded width="+1.7pt">
     <mn>1</mn>
    </mpadded>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <eq></eq>
    <ci>t</ci>
    <apply>
     <plus></plus>
     <apply>
      <times></times>
      <ci>h</ci>
      <apply>
       <minus></minus>
       <apply>
        <csymbol cd="ambiguous">subscript</csymbol>
        <ci>n</ci>
        <max></max>
       </apply>
       <cn type="integer">1</cn>
      </apply>
     </apply>
     <cn type="integer">1</cn>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   t=h(n_{\max}-1)+1\,
  </annotation>
 </semantics>
</math>

</p>

<p>For example, consider the decision tree below to decide whether to play outside:</p>
<table>
<tbody>
<tr class="odd">
<td style="text-align: left;"><figure><b>(Figure)</b>
<figcaption>Decision tree for playing outside.png</figcaption>
</figure></td>
</tr>
</tbody>
</table>

<p>It can be linearly encoded as:</p>
<dl>
<dd><code><nowiki>01234567</nowiki></code>
</dd>
<dd>
</dd>
<dd><code><nowiki>HOWbaaba</nowiki></code>
</dd>
</dl>

<p>where “H” represents the attribute Humidity, “O” the attribute Outlook, “W” represents Windy, and “a” and “b” the class labels "Yes" and "No" respectively. Note that the edges connecting the nodes are properties of the data, specifying the type and number of branches of each attribute, and therefore don’t have to be encoded.</p>

<p>The process of decision tree induction with gene expression programming starts, as usual, with an initial population of randomly created chromosomes. Then the chromosomes are expressed as decision trees and their fitness evaluated against a training dataset. According to fitness they are then selected to reproduce with modification. The genetic operators are exactly the same that are used in a conventional unigenic system, for example, <a href="gene_expression_programming#Mutation" title="wikilink">mutation</a>, <a href="gene_expression_programming#Inversion" title="wikilink">inversion</a>, <a href="gene_expression_programming#Transposition" title="wikilink">transposition</a>, and <a href="gene_expression_programming#Recombination" title="wikilink">recombination</a>.</p>

<p>Decision trees with both nominal and numeric attributes are also easily induced with gene expression programming using the framework described <a href="gene_expression_programming#The_GEP-RNC_algorithm" title="wikilink">above</a> for dealing with random numerical constants. The chromosomal architecture includes an extra domain for encoding random numerical constants, which are used as thresholds for splitting the data at each branching node. For example, the gene below with a head size of 5 (the Dc starts at position 16):</p>
<dl>
<dd><code><nowiki>012345678901234567890</nowiki></code>
</dd>
<dd>
</dd>
<dd><code><nowiki>WOTHabababbbabba46336</nowiki></code>
</dd>
</dl>

<p>encodes the decision tree shown below:</p>
<table>
<tbody>
<tr class="odd">
<td style="text-align: left;"><figure><b>(Figure)</b>
<figcaption>GEP decision tree, k-expression WOTHababab.png</figcaption>
</figure></td>
</tr>
</tbody>
</table>

<p>In this system, every node in the head, irrespective of its type (numeric attribute, nominal attribute, or terminal), has associated with it a random numerical constant, which for simplicity in the example above is represented by a numeral 0–9. These random numerical constants are encoded in the Dc domain and their expression follows a very simple scheme: from top to bottom and from left to right, the elements in Dc are assigned one-by-one to the elements in the decision tree. So, for the following array of RNCs:</p>
<dl>
<dd><em>C</em> = {62, 51, 68, 83, 86, 41, 43, 44, 9, 67}
</dd>
</dl>

<p>the decision tree above results in:</p>
<table>
<tbody>
<tr class="odd">
<td style="text-align: left;"><figure><b>(Figure)</b>
<figcaption>GEP decision tree with numeric and nominal attributes, k-expression WOTHababab.png</figcaption>
</figure></td>
</tr>
</tbody>
</table>

<p>which can also be represented more colorfully as a conventional decision tree:</p>
<table>
<tbody>
<tr class="odd">
<td style="text-align: left;"><figure><b>(Figure)</b>
<figcaption>GEP decision tree with numeric and nominal attributes.png</figcaption>
</figure></td>
</tr>
</tbody>
</table>
<h2 id="criticism">Criticism</h2>

<p>GEP has been criticized for not being a major improvement over other <a href="genetic_programming" title="wikilink">genetic programming</a> techniques. In many experiments, it did not perform better than existing methods.<a class="footnoteRef" href="#fn12" id="fnref12"><sup>12</sup></a></p>
<h2 id="software">Software</h2>
<h3 id="commercial-applications">Commercial applications</h3>
<dl>
<dt><a class="uri" href="GeneXproTools" title="wikilink">GeneXproTools</a>: GeneXproTools is a <a href="predictive_analytics" title="wikilink">predictive analytics</a> suite developed by <a class="uri" href="Gepsoft" title="wikilink">Gepsoft</a>. GeneXproTools modeling frameworks include <a href="logistic_regression" title="wikilink">logistic regression</a>, <a href="Statistical_classification" title="wikilink">classification</a>, <a href="Regression_analysis" title="wikilink">regression</a>, <a href="time_series_prediction" title="wikilink">time series prediction</a>, and <a href="logic_synthesis" title="wikilink">logic synthesis</a>. GeneXproTools implements the basic <a href="gene_expression_programming#The_basic_gene_expression_algorithm" title="wikilink">gene expression algorithm</a> and the <a href="gene_expression_programming#The_GEP-RNC_algorithm" title="wikilink">GEP-RNC algorithm</a>, both used in all the modeling frameworks of GeneXproTools.</dt>
</dl>
<h3 id="open-source-libraries">Open source libraries</h3>
<dl>
<dt><a href="http://code.google.com/p/gep4j/">GEP4J – GEP for Java Project</a>: Created by Jason Thomas, GEP4J is an open-source implementation of gene expression programming in <a href="Java_(programming_language)" title="wikilink">Java</a>. It implements different GEP algorithms, including evolving <a href="gene_expression_programming#Decision_trees" title="wikilink">decision trees</a> (with nominal, numeric, or mixed attributes) and <a href="gene_expression_programming#Cells_and_code_reuse" title="wikilink">automatically defined functions</a>. GEP4J is hosted at <a href="Google_Code" title="wikilink">Google Code</a>.</dt>
</dl>
<dl>
<dt><a href="http://code.google.com/p/pygep/">PyGEP – Gene Expression Programming for Python</a>: Created by Ryan O'Neil with the goal to create a simple library suitable for the academic study of gene expression programming in <a href="Python_(programming_language)" title="wikilink">Python</a>, aiming for ease of use and rapid implementation. It implements standard <a href="gene_expression_programming#Multigenic_chromosomes" title="wikilink">multigenic chromosomes</a> and the genetic operators mutation, crossover, and transposition. PyGEP is hosted at <a href="Google_Code" title="wikilink">Google Code</a>.</dt>
</dl>
<dl>
<dt><a href="http://sourceforge.net/projects/jgep/">jGEP – Java GEP toolkit</a>: Created by Matthew Sottile to rapidly build <a href="Java_(programming_language)" title="wikilink">Java</a> prototype codes that use GEP, which can then be written in a language such as <a href="C_(programming_language)" title="wikilink">C</a> or <a class="uri" href="Fortran" title="wikilink">Fortran</a> for real speed. jGEP is hosted at <a class="uri" href="SourceForge" title="wikilink">SourceForge</a>.</dt>
</dl>
<h2 id="further-reading">Further reading</h2>
<ul>
<li></li>
<li></li>
</ul>
<h2 id="see-also">See also</h2>
<ul>
<li><a href="Artificial_intelligence" title="wikilink">Artificial intelligence</a></li>
<li><a href="Decision_trees" title="wikilink">Decision trees</a></li>
<li><a href="Evolutionary_algorithms" title="wikilink">Evolutionary algorithms</a></li>
<li><a href="Genetic_algorithms" title="wikilink">Genetic algorithms</a></li>
<li><a href="Genetic_programming" title="wikilink">Genetic programming</a></li>
<li><a class="uri" href="GeneXproTools" title="wikilink">GeneXproTools</a></li>
<li><a href="Machine_learning" title="wikilink">Machine learning</a></li>
<li><a href="Artificial_neural_network" title="wikilink">Neural networks</a></li>
</ul>
<h2 id="references">References</h2>
<h2 id="external-links">External links</h2>
<ul>
<li><a href="http://www.gene-expression-programming.com/">GEP home page</a>, maintained by the inventor of gene expression programming.</li>
<li><a href="http://www.gepsoft.com/">GeneXproTools</a>, commercial GEP software.</li>
</ul>

<p>"</p>

<p><a href="Category:Gene_expression_programming" title="wikilink"> </a> <a href="Category:Evolutionary_algorithms" title="wikilink">Category:Evolutionary algorithms</a> <a href="Category:Evolutionary_computation" title="wikilink">Category:Evolutionary computation</a> <a href="Category:Genetic_algorithms" title="wikilink">Category:Genetic algorithms</a> <a href="Category:Genetic_programming" title="wikilink">Category:Genetic programming</a></p>
<section class="footnotes">
<hr/>
<ol>
<li id="fn1">Box, G. E. P., 1957. Evolutionary operation: A method for increasing industrial productivity. Applied Statistics, 6, 81–101.<a href="#fnref1">↩</a></li>
<li id="fn2">Friedman, G. J., 1959. Digital simulation of an evolutionary process. General Systems Yearbook, 4, 171–184.<a href="#fnref2">↩</a></li>
<li id="fn3"><a href="#fnref3">↩</a></li>
<li id="fn4"><a href="#fnref4">↩</a></li>
<li id="fn5"><a href="#fnref5">↩</a></li>
<li id="fn6"><a href="#fnref6">↩</a></li>
<li id="fn7"><a href="#fnref7">↩</a></li>
<li id="fn8"><a href="#fnref8">↩</a></li>
<li id="fn9"><a href="#fnref9">↩</a></li>
<li id="fn10"><a href="#fnref10">↩</a></li>
<li id="fn11"><a href="#fnref11">↩</a></li>
<li id="fn12"><a href="#fnref12">↩</a></li>
</ol>
</section>
</body>
</html>
