<html lang="en">
<head>
  <meta charset="UTF-8" />
  <title offset="472">Heritability</title>
   <script type="text/javascript"
      src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_SVG.js">
    </script>
</head>
<body>
<h1>Heritability</h1>
<hr/>
<figure><b>(Figure)</b>
<figcaption>Heritability asks how much genetics are playing a role in <em>differences</em> in height between people. This is not the same as asking how much genetics influences height in any one person.</figcaption>
</figure>
<p><strong>Heritability</strong> is a concept in biology that describes how much of the <a class="uri" href="variation" title="wikilink">variation</a> of a <a href="phenotypic_trait" title="wikilink">trait</a> in a population is due to <a href="genetics" title="wikilink">genetic</a> differences in that population.<a class="footnoteRef" href="#fn1" id="fnref1"><sup>1</sup></a> Other causes of variation in a trait are <a href="environment_(biophysical)" title="wikilink">environment</a> factors and <a href="genetic_drift" title="wikilink">genetic drift</a>.</p>
<p>For instance, some humans in a population are taller than others; heritability attempts to identify how much genetics play a role in part of the population being taller.</p>
<p>Heritability is measured by estimating the relative contributions of genetic and non-genetic differences to the total phenotypic variation in a population. Heritability is an important concept in <a href="quantitative_genetics" title="wikilink">quantitative genetics</a>, particularly in <a href="selective_breeding" title="wikilink">selective breeding</a> and behavior genetics (for instance, <a href="twin_study" title="wikilink">twin studies</a>), but is less widely used in <a href="population_genetics" title="wikilink">population genetics</a>.</p>
<p><a href="Geoffrey_Miller_(psychologist)" title="wikilink">Geoffrey Miller</a>, an <a href="Evolutionary_psychology" title="wikilink">evolutionary psychologist</a>, has said "The concept of heritability applies only to traits that differ between individuals. If a trait exists in precisely the same form across all individuals, it may be inherited, but it cannot be heritable."<a class="footnoteRef" href="#fn2" id="fnref2"><sup>2</sup></a></p>
<h2 id="overview">Overview</h2>
<p> </p>
<p>Heritability measures the fraction of phenotype variability that can be attributed to <a href="genetic_variation" title="wikilink">genetic variation</a>. This is not the same as saying that this fraction of an individual phenotype is caused by genetics. In addition, heritability can change without any genetic change occurring, such as when the environment starts contributing to more variation. A case in point, consider that both <a href="gene" title="wikilink">genes</a> and environment have the potential to influence intelligence. Heritability could increase if genetic variation increases, causing individuals to show more phenotypic variation, like showing different levels of intelligence. On the other hand, heritability might also increase if the environmental variation decreases, causing individuals to show less phenotypic variation, like showing more similar levels of intelligence. Heritability increases when genetics are contributing more variation or because non-genetic factors are contributing less variation; what matters is the relative contribution. Heritability is specific to a particular population in a particular environment.</p>
<p>The extent of dependence of phenotype on environment can also be a function of the genes involved. Matters of heritability are complicated because genes may <a href="canalization_(genetics)" title="wikilink">canalize</a> a phenotype, making its expression almost inevitable in all occurring environments. Individuals with the same genotype can also exhibit different phenotypes through a mechanism called <a href="phenotypic_plasticity" title="wikilink">phenotypic plasticity</a>, which makes heritability difficult to measure in some cases. Recent insights in molecular biology have identified changes in <a href="transcription_(genetics)" title="wikilink">transcriptional</a> activity of individual genes associated with environmental changes. However, there are a large number of genes whose transcription is not affected by the environment.<ref></ref></p>
<ul>
<li><em>review of</em>: </li>
</ul>
<p>Estimates of heritability use <a href="statistical_analyses" title="wikilink">statistical analyses</a> to help to identify the causes of differences between individuals. Since heritability is concerned with variance, it is necessarily an account of the differences between individuals in a population. Heritability can be <a class="uri" href="univariate" title="wikilink">univariate</a> – examining a single trait – or multivariate – examining the genetic and environmental associations between multiple traits at once. This allows a test of the genetic overlap between different phenotypes: for instance <a href="hair_color" title="wikilink">hair color</a> and <a href="eye_color" title="wikilink">eye color</a>. Environment and genetics may also interact, and heritability analyses can test for and examine these interactions (GxE models).</p>
<p>A prerequisite for heritability analyses is that there is some population variation to account for. In practice, all traits vary and almost all traits show some heritability.<a class="footnoteRef" href="#fn3" id="fnref3"><sup>3</sup></a> For example, in a population with no diversity in hair color, "heritability" of hair color would be undefined. In populations with varying values of a trait (e.g. see the image on the left), variance could be due to environment (hair dye for instance) or genetic differences, and heritability could vary from 0-100%.</p>
<p>This last point highlights the fact that heritability cannot take into account the effect of factors which are invariant in the population. Factors may be invariant if they are absent and do not exist in the population, such as no one having access to a particular <a class="uri" href="antibiotic" title="wikilink">antibiotic</a>, or because they are omni-present, like if everyone is drinking <a class="uri" href="coffee" title="wikilink">coffee</a>.</p>
<h2 id="definition">Definition</h2>
<p>Any particular phenotype can be <a href="statistical_model" title="wikilink">modeled</a> as the sum of genetic and environmental effects:<a class="footnoteRef" href="#fn4" id="fnref4"><sup>4</sup></a></p>
<dl>
<dd>Phenotype (<em>P</em>) = Genotype (<em>G</em>) + Environment (<em>E</em>).
</dd>
</dl>
<p>Likewise the variance in the trait – Var (P) – is the sum of effects as follows:</p>
<dl>
<dd>Var(<em>P</em>) = Var(<em>G</em>) + Var(<em>E</em>) + 2 Cov(<em>G</em>,<em>E</em>).
</dd>
</dl>
<p>In a planned experiment Cov(<em>G</em>,<em>E</em>) can be controlled and held at 0. In this case, heritability is defined as:</p>
<p><span class="LaTeX">$$H^2 = \frac{\mathrm{Var}(G)}{\mathrm{Var}(P)}$$</span> .</p>
<p><em>H</em><sup>2</sup> is the broad-sense heritability. This reflects all the genetic contributions to a population's phenotypic variance including additive, <a href="dominance_relationship" title="wikilink">dominant</a>, and <a href="epistasis" title="wikilink">epistatic</a> (multi-genic interactions), as well as <a href="maternal_effect" title="wikilink">maternal and paternal effects</a>, where individuals are directly affected by their parents' phenotype, such as with <a class="uri" href="milk" title="wikilink">milk</a> production in mammals.</p>
<p>A particularly important component of the genetic variance is the additive variance, Var(A), which is the variance due to the average effects (additive effects) of the <a href="allele" title="wikilink">alleles</a>. Since each parent passes a single allele per <a href="locus_(genetics)" title="wikilink">locus</a> to each offspring, parent-offspring resemblance depends upon the average effect of single alleles. Additive variance represents, therefore, the genetic component of variance responsible for parent-offspring resemblance. The additive genetic portion of the phenotypic variance is known as Narrow-sense heritability and is defined as</p>
<p><span class="LaTeX">$$h^2 = \frac{\mathrm{Var}(A)}{\mathrm{Var}(P)}$$</span></p>
<p>An upper case <em>H</em><sup>2</sup> is used to denote broad sense, and lower case <em>h</em><sup>2</sup> for narrow sense.</p>
<p>Additive variance is important for <a href="natural_selection" title="wikilink">selection</a>. If a selective pressure such as improving livestock is exerted, the response of the trait is directly related to narrow-sense heritability. The mean of the trait will increase in the next generation as a function of how much the mean of the selected parents differs from the mean of the population from which the selected parents were chosen. The observed <em>response to selection</em> leads to an estimate of the narrow-sense heritability (called <strong>realized heritability</strong>). This is the principle underlying <a href="artificial_selection" title="wikilink">artificial selection</a> or breeding.</p>
<h3 id="example">Example</h3>
<figure><b>(Figure)</b>
<figcaption>Figure 1. Relationship of phenotypic values to additive and dominance effects using a completely dominant locus.</figcaption>
</figure>
<p>The simplest genetic model involves a single locus with two alleles (b and B) affecting one quantitative phenotype.</p>
<p>The number of <strong>B</strong> alleles can vary from 0, 1, or 2. For any genotype, B<sub>i</sub>B<sub>j</sub>, the expected phenotype can then be written as the sum of the overall mean, a linear effect, and a dominance deviation:</p>
<p><span class="LaTeX">$$P_{ij} = \mu + \alpha_i + \alpha_j + d_{ij}$$</span> = <a href="Mean" title="wikilink">Population mean</a> + Additive Effect (<span class="LaTeX">$a_{ij}=\alpha_i+\alpha_j$</span>) + Dominance Deviation (<span class="LaTeX">$d_{ij}$</span>).</p>
<p>The additive genetic variance at this locus is the <a href="Weighted_mean" title="wikilink">weighted average</a> of the squares of the additive effects:</p>
<p><span class="LaTeX">$$\mathrm{Var}(A) = f(bb)a^2_{bb}+f(Bb)a^2_{Bb}+f(BB)a^2_{BB},$$</span></p>
<p>where <span class="LaTeX">$f(bb)a_{bb}+f(Bb)a_{Bb}+f(BB)a_{BB} = 0.$</span></p>
<p>There is a similar relationship for variance of dominance deviations:</p>
<p><span class="LaTeX">$$\mathrm{Var}(D) = f(bb)d^2_{bb}+f(Bb)d^2_{Bb}+f(BB)d^2_{BB},$$</span></p>
<p>where <span class="LaTeX">$f(bb)d_{bb}+f(Bb)d_{Bb}+f(BB)d_{BB} = 0.$</span></p>
<p>The <a href="linear_regression" title="wikilink">linear regression</a> of phenotype on genotype is shown in Figure 1.</p>
<h2 id="estimating-heritability">Estimating heritability</h2>
<p>Since only <em>P</em> can be observed or measured directly, heritability must be estimated from the similarities observed in subjects varying in their level of genetic or environmental similarity. The <a href="Statistics" title="wikilink">statistical</a> analyses required to estimate the <a href="genetics" title="wikilink">genetic</a> and <a href="Environment_(biophysical)" title="wikilink">environmental</a> components of variance depend on the sample characteristics. Briefly, better estimates are obtained using data from individuals with widely varying levels of genetic relationship - such as <a class="uri" href="twins" title="wikilink">twins</a>, siblings, parents and offspring, rather than from more distantly related (and therefore less similar) subjects. The <a href="standard_error_(statistics)" title="wikilink">standard error</a> for heritability estimates is improved with large sample sizes.</p>
<p>In non-human populations it is often possible to collect information in a controlled way. For example, among farm animals it is easy to arrange for a bull to produce offspring from a large number of cows and to control environments. Such <a href="experimental_control" title="wikilink">experimental control</a> is impossible when gathering human data, relying on naturally occurring relationships and environments.</p>
<p>Studies of human heritability often utilize adoption study designs, often with <a href="Twin" title="wikilink">identical twins</a> who have been separated early in life and raised in different environments (see for example Fig. 2). Such individuals have identical genotypes and can be used to separate the effects of genotype and environment. A limit of this design is the common prenatal environment and the relatively low numbers of twins reared apart. A second and more common design is the <a href="twin_study" title="wikilink">twin study</a> in which the similarity of identical and fraternal twins is used to estimate heritability. These studies can be limited by the fact that identical twins are <a href="Twin#Genetic_and_epigenetic_similarity" title="wikilink">not completely genetically identical</a>, potentially resulting in an underestimation of heritability. Studies of twins also examine differences between twins and non-twin siblings, for instance to examine phenomena such as intrauterine competition (for example, <a href="twin-to-twin_transfusion_syndrome" title="wikilink">twin-to-twin transfusion syndrome</a>).</p>
<p>Heritability estimates are always relative to the genetic and environmental factors in the population, and are not absolute measurements of the contribution of genetic and environmental factors to a phenotype. Heritability estimates reflect the amount of variation in genotypic effects compared to variation in environmental effects.</p>
<p>Heritability can be made larger by diversifying the genetic background, e.g., by using only very out bred individuals (which increases Var<sub>G</sub>) and/or by minimizing environmental effects (decreasing Var<sub>E</sub>). The converse also holds. Due to such effects, different populations of a species might have different heritabilities for the same trait.</p>
<p>In <a href="Observational_study" title="wikilink">observational studies</a>, or because of evokative effects (where a genome evokes environments by its effect on them), G and E may covary: <a href="gene_environment_correlation" title="wikilink">gene environment correlation</a>. Depending on the methods used to estimate heritability, correlations between genetic factors and shared or non-shared environments may or may not be confounded with heritability.<a class="footnoteRef" href="#fn5" id="fnref5"><sup>5</sup></a></p>
<h3 id="common-misunderstandings-of-heritability-estimates">Common misunderstandings of heritability estimates</h3>
<p>A common estimate of heritability is called the Heritability Index (HI), which ranges from 0 - 1. A HI index of 0 means that none of the variability between people in the study sample on the trait under investigation is due to genetic factors; an HI of 1 indicates the opposite.</p>
<p>Heritability estimates are often misinterpreted if it is not understood that they refer to the <em>proportion of variation between individuals</em> on a trait that is due to genetic factors. It does not indicate the degree of genetic influence on the development of a trait of an individual. For example, it is incorrect to say that since the heritability of personality traits is about .6, that means that 60% of your personality is inherited from your parents and 40% comes from the environment.</p>
<p>Even a highly heritable trait (such as <a href="eye_color" title="wikilink">eye color</a>) assumes environmental inputs which are required for development: for instance temperatures and an atmosphere supporting life, etc. A more useful distinction than "nature vs. nurture" is "obligate vs. facultative"—under typical environmental ranges, what traits are more "obligate" (e.g., the nose—everyone has a nose) or more "facultative" (sensitive to environmental variations, such as specific language learned during infancy). Another useful distinction is between traits that are likely to be adaptations (such as the umbilical cord) vs. those that are byproducts of adaptations (such as the belly button), or are due to random variation (non-adaptive variation in belly button shape, e.g., convex or concave).</p>
<h2 id="estimation-methods">Estimation methods</h2>
<p>There are essentially two schools of thought regarding estimation of heritability.</p>
<p>One <a href="School_(discipline)" title="wikilink">school of thought</a> was developed by <a href="Sewall_Wright" title="wikilink">Sewall Wright</a> at <a href="University_of_Chicago" title="wikilink">The University of Chicago</a>, and further popularized by <a href="C._C._Li" title="wikilink">C. C. Li</a> (<a href="University_of_Chicago" title="wikilink">University of Chicago</a>) and <a href="Jay_Laurence_Lush" title="wikilink">J. L. Lush</a> (<a href="Iowa_State_University" title="wikilink">Iowa State University</a>). It is based on the analysis of correlations and, by extension, regression. <a href="path_analysis_(statistics)" title="wikilink">Path Analysis</a> was developed by <a href="Sewall_Wright" title="wikilink">Sewall Wright</a> as a way of estimating heritability.</p>
<p>The second was originally developed by <a href="Ronald_Fisher" title="wikilink">R. A. Fisher</a> and expanded at <a href="University_of_Edinburgh" title="wikilink">The University of Edinburgh</a>, <a href="Iowa_State_University" title="wikilink">Iowa State University</a>, and <a href="North_Carolina_State_University" title="wikilink">North Carolina State University</a>, as well as other schools. It is based on the <a href="analysis_of_variance" title="wikilink">analysis of variance</a> of breeding studies, using the intraclass correlation of relatives. Various methods of estimating components of variance (and, hence, heritability) from <a href="Analysis_of_variance" title="wikilink">ANOVA</a> are used in these analyses.</p>
<h2 id="regressioncorrelation-methods-of-estimation">Regression/correlation methods of estimation</h2>
<p>The first school of estimation uses regression and correlation to estimate heritability.</p>
<h3 id="selection-experiments">Selection experiments</h3>
<p> Calculating the strength of selection, <em>S</em> (the difference in mean trait between the population as a whole and the selected parents of the next generation, also called the <em>selection differential</em><a class="footnoteRef" href="#fn6" id="fnref6"><sup>6</sup></a>) and response to selection <em>R</em> (the difference in offspring and whole parental generation mean trait) in an artificial selection experiment will allow calculation of realized heritability as the response to selection relative to the strength of selection, <em>h</em><sup>2</sup>=R/S as in Fig. 3.</p>
<h3 id="comparison-of-close-relatives">Comparison of close relatives</h3>
<p>In the comparison of relatives, we find that in general,</p>
<p><span class="LaTeX">$h^2 = \frac{b}{r} = \frac{t^2}{r}$</span> where <em>r</em> can be thought of as the <a href="Coefficient_of_relationship" title="wikilink">coefficient of relatedness</a>, <em>b</em> is the coefficient of regression and <em>t</em> the coefficient of correlation.</p>
<h4 id="parent-offspring-regression">Parent-offspring regression</h4>
<figure><b>(Figure)</b>
<figcaption>Figure 4. Sir <a href="Francis_Galton" title="wikilink">Francis Galton</a>'s (1889) data showing the relationship between offspring height (928 individuals) as a function of mean parent height (205 sets of parents).</figcaption>
</figure>
<p>Heritability may be estimated by comparing parent and offspring traits (as in Fig. 4). The slope of the line (0.57) approximates the heritability of the trait when offspring values are regressed against the average trait in the parents. If only one parent's value is used then heritability is twice the slope. (Note that this is the source of the term "<a href="regression_analysis" title="wikilink">regression</a>," since the offspring values always tend to <a href="regression_toward_the_mean" title="wikilink">regress to the mean</a> value for the population, <em>i.e.</em>, the slope is always less than one). This regression effect also underlies the <a href="John_DeFries" title="wikilink">DeFries</a> <a href="David_Fulker" title="wikilink">Fulker</a> method for analyzing twins selected for one member being affected.<a class="footnoteRef" href="#fn7" id="fnref7"><sup>7</sup></a></p>
<h4 id="sibling-comparison">Sibling comparison</h4>
<p>A basic approach to heritability can be taken using full-Sib designs: comparing similarity between siblings who share both a biological mother and a father.<a class="footnoteRef" href="#fn8" id="fnref8"><sup>8</sup></a> When there is only additive gene action, this sibling phenotypic correlation is an index of <em>familiarity</em> – the sum of half the additive genetic variance plus full effect of the common environment. It thus places an upper-limit on additive heritability of twice the full-Sib phenotypic correlation. Half-Sib designs compare phenotypic traits of siblings that share one parent with other sibling groups.</p>
<h4 id="twin-studies">Twin studies</h4>
<p> Heritability for traits in humans is most frequently estimated by comparing resemblances between twins (Fig. 2 & 5). "The advantage of twin studies, is that the total variance can be split up into genetic, shared or common environmental, and unique environmental components, enabling an accurate estimation of heritability".<a class="footnoteRef" href="#fn9" id="fnref9"><sup>9</sup></a> Fraternal or dizygotic (DZ) twins on average share half their genes (assuming there is no <a href="assortative_mating" title="wikilink">assortative mating</a> for the trait), and so identical or monozygotic (MZ) twins on average are twice as genetically similar as DZ twins. A crude estimate of heritability, then, is approximately twice the difference in <a class="uri" href="correlation" title="wikilink">correlation</a> between MZ and DZ twins, i.e. <a href="Falconer's_formula" title="wikilink">Falconer's formula</a> <em>H</em><sup>2</sup>=2(r(MZ)-r(DZ)).</p>
<p>The effect of shared environment, <em>c</em><sup>2</sup>, contributes to similarity between siblings due to the commonality of the environment they are raised in. Shared environment is approximated by the DZ correlation minus half heritability, which is the degree to which DZ twins share the same genes, <em>c</em><sup>2</sup>=DZ-1/2<em>h</em><sup>2</sup>. Unique environmental variance, <em>e</em><sup>2</sup>, reflects the degree to which identical twins raised together are dissimilar, <em>e</em><sup>2</sup>=1-r(MZ).</p>
<p>The methodology of the classical <a href="twin_study" title="wikilink">twin study</a> has been <a href="Twin_study#Criticism" title="wikilink">criticized</a>, but some of these criticisms do not take into account the methodological innovations and refinements described above.</p>
<h3 id="extended-pedigree-design">Extended pedigree design</h3>
<p>While often heritability is analyzed in single generations: comparing MZ twins raised apart, or comparing the similarity of MZ and DZ twins, considerable power can be gained using more complex relationships. By studying a trait in multi-generational families, the multiple recombination of genetic and environmental effects can be decomposed using software such as <a class="uri" href="ASReml" title="wikilink">ASReml</a> and heritability estimated.<a class="footnoteRef" href="#fn10" id="fnref10"><sup>10</sup></a> This design is helpful for untangling confounds such as <a href="reverse_causality" title="wikilink">reverse causality</a>, <a href="maternal_effects" title="wikilink">maternal effects</a> such as the <a href="prenatal_environment" title="wikilink">prenatal environment</a>, and confounding of <a href="genetic_dominance" title="wikilink">genetic dominance</a>, shared environment, and maternal gene effects<a class="footnoteRef" href="#fn11" id="fnref11"><sup>11</sup></a><a class="footnoteRef" href="#fn12" id="fnref12"><sup>12</sup></a> but is generally less powerful than the twin design for obtaining heritability estimates.</p>
<h2 id="analysis-of-variance-methods-of-estimation">Analysis of variance methods of estimation</h2>
<p>The second set of methods of estimation of heritability involves ANOVA and estimation of variance components.</p>
<h3 id="basic-model">Basic model</h3>
<p>We use the basic discussion of Kempthorne (1957 [1969]). Considering only the most basic of genetic models, we can look at the quantitative contribution of a single locus with genotype <strong>G<sub>i</sub></strong> as</p>
<p><span class="LaTeX">$y_i = \mu + g_i + e$</span></p>
<p>where</p>
<p><span class="LaTeX">$g_i$</span> is the effect of genotype <strong>G<sub>i</sub></strong></p>
<p>and <span class="LaTeX">$e$</span> is the environmental effect.</p>
<p>Consider an experiment with a group of sires and their progeny from random dams. Since the progeny get half of their genes from the father and half from their (random) mother, the progeny equation is</p>
<p><span class="LaTeX">$z_i = \mu + \frac{1}{2}g_i + e$</span></p>
<h4 id="intraclass-correlations">Intraclass correlations</h4>
<p>Consider the experiment above. We have two groups of progeny we can compare. The first is comparing the various progeny for an individual sire (called <em>within sire group</em>). The variance will include terms for genetic variance (since they did not all get the same genotype) and environmental variance. This is thought of as an <em>error</em> term.</p>
<p>The second group of progeny are comparisons of means of half sibs with each other (called <em>among sire group</em>). In addition to the <a href="Errors_and_residuals_in_statistics" title="wikilink">error term</a> as in the within sire groups, we have an addition term due to the differences among different means of half sibs. The intraclass correlation is</p>
<p><span class="LaTeX">$$\mathrm{corr}(z,z') = \mathrm{corr}(\mu + \frac{1}{2}g + e, \mu + \frac{1}{2}g + e') = \frac{1}{4}V_g$$</span> , since environmental effects are independent of each other.</p>
<h4 id="the-anova">The ANOVA</h4>
<p>In an experiment with <span class="LaTeX">$n$</span> sires and <span class="LaTeX">$r$</span> progeny per sire, we can calculate the following ANOVA, using <span class="LaTeX">$V_g$</span> as the genetic variance and <span class="LaTeX">$V_e$</span> as the environmental variance:</p>
<table>
<tbody>
<tr class="odd">
<td style="text-align: left;"><p>Table 1: ANOVA for Sire experiment</p></td>
<td style="text-align: left;"><p>Source</p></td>
<td style="text-align: left;"><p>d.f.</p></td>
<td style="text-align: left;"><p>Mean Square</p></td>
<td style="text-align: left;"><p>Expected Mean Square</p></td>
</tr>
<tr class="even">
<td style="text-align: left;"><p>Among sire groups</p></td>
<td style="text-align: left;"><p><span class="LaTeX">$n-1$</span></p></td>
<td style="text-align: left;"><p><span class="LaTeX">$S$</span></p></td>
<td style="text-align: left;"><p><span class="LaTeX">$\frac{3}{4}V_g + V_e + r({\frac{1}{4}V_g})$</span></p></td>
</tr>
<tr class="odd">
<td style="text-align: left;"><p>Within sire groups</p></td>
<td style="text-align: left;"><p><span class="LaTeX">$n(r-1)$</span></p></td>
<td style="text-align: left;"><p><span class="LaTeX">$W$</span></p></td>
<td style="text-align: left;"><p><span class="LaTeX">$\frac{3}{4}V_g + V_e$</span></p></td>
</tr>
</tbody>
</table>
<p>The <span class="LaTeX">$\frac{1}{4}V_g$</span> term is the <a href="intraclass_correlation" title="wikilink">intraclass correlation</a> among half sibs. We can easily calculate <span class="LaTeX">$H^2 = \frac{V_g}{V_g+V_e} = \frac{4(S-W)}{S+(r-1)W}$</span>. The Expected Mean Square is calculated from the relationship of the individuals (progeny within a sire are all half-sibs, for example), and an understanding of intraclass correlations.</p>
<h3 id="model-with-additive-and-dominance-terms">Model with additive and dominance terms</h3>
<p>For a model with additive and dominance terms, but not others, the equation for a single locus is</p>
<p><span class="LaTeX">$$y_{ij} = \mu + \alpha_i + \alpha_j + d_{ij} + e,$$</span></p>
<p>where</p>
<p><span class="LaTeX">$\alpha_i$</span> is the additive effect of the i<sup>th</sup> allele, <span class="LaTeX">$\alpha_j$</span> is the additive effect of the j<sup>th</sup> allele, <span class="LaTeX">$d_{ij}$</span> is the dominance deviation for the ij<sup>th</sup> genotype, and <span class="LaTeX">$e$</span> is the environment.</p>
<p>Experiments can be run with a similar setup to the one given in Table 1. Using different relationship groups, we can evaluate different intraclass correlations. Using <span class="LaTeX">$V_a$</span> as the additive genetic variance and <span class="LaTeX">$V_d$</span> as the dominance deviation variance, intraclass correlations become <a href="linear_function" title="wikilink">linear functions</a> of these parameters. In general,</p>
<dl>
<dd>Intraclass correlation<span class="LaTeX">$= r V_a + \theta V_d,$</span>
</dd>
</dl>
<p>where <span class="LaTeX">$r$</span> and <span class="LaTeX">$\theta$</span> are found as</p>
<p><span class="LaTeX">$r =$</span>P[ <a class="uri" href="alleles" title="wikilink">alleles</a> drawn at random from the relationship pair are <a href="identity_by_descent" title="wikilink">identical by descent</a>], and</p>
<p><span class="LaTeX">$\theta =$</span>P[ <a class="uri" href="genotypes" title="wikilink">genotypes</a> drawn at random from the relationship pair are <a href="identity_by_descent" title="wikilink">identical by descent</a>].</p>
<p>Some common relationships and their coefficients are given in Table 2.</p>
<table>
<tbody>
<tr class="odd">
<td style="text-align: left;"><p>Table 2: Coefficients for calculating variance components</p></td>
<td style="text-align: left;"><p>Relationship</p></td>
<td style="text-align: left;"><p><span class="LaTeX">$r$</span></p></td>
<td style="text-align: left;"><p><span class="LaTeX">$\theta$</span></p></td>
</tr>
<tr class="even">
<td style="text-align: left;"><p>Identical Twins</p></td>
<td style="text-align: left;"><p><span class="LaTeX">$1$</span></p></td>
<td style="text-align: left;"><p><span class="LaTeX">$1$</span></p></td>
</tr>
<tr class="odd">
<td style="text-align: left;"><p>Parent-Offspring</p></td>
<td style="text-align: left;"><p><span class="LaTeX">$\frac{1}{2}$</span></p></td>
<td style="text-align: left;"><p><span class="LaTeX">$0$</span></p></td>
</tr>
<tr class="even">
<td style="text-align: left;"><p>Half Siblings</p></td>
<td style="text-align: left;"><p><span class="LaTeX">$\frac{1}{4}$</span></p></td>
<td style="text-align: left;"><p><span class="LaTeX">$0$</span></p></td>
</tr>
<tr class="odd">
<td style="text-align: left;"><p>Full Siblings</p></td>
<td style="text-align: left;"><p><span class="LaTeX">$\frac{1}{2}$</span></p></td>
<td style="text-align: left;"><p><span class="LaTeX">$\frac{1}{4}$</span></p></td>
</tr>
<tr class="even">
<td style="text-align: left;"><p>First Cousins</p></td>
<td style="text-align: left;"><p><span class="LaTeX">$\frac{1}{8}$</span></p></td>
<td style="text-align: left;"><p><span class="LaTeX">$0$</span></p></td>
</tr>
<tr class="odd">
<td style="text-align: left;"><p>Double First Cousins</p></td>
<td style="text-align: left;"><p><span class="LaTeX">$\frac{1}{4}$</span></p></td>
<td style="text-align: left;"><p><span class="LaTeX">$\frac{1}{16}$</span></p></td>
</tr>
</tbody>
</table>
<h3 id="larger-models">Larger models</h3>
<p>When a large, complex pedigree is available for estimating heritability, the most efficient use of the data is in a <a href="restricted_maximum_likelihood" title="wikilink">restricted maximum likelihood</a> (REML) model. The <a href="raw_data" title="wikilink">raw data</a> will usually have three or more data points for each individual: a code for the sire, a code for the dam and one or several trait values. Different trait values may be for different traits or for different time points of measurement.</p>
<p>The currently popular methodology relies on high degrees of certainty over the identities of the sire and dam; it is not common to treat the sire identity probabilistically. This is not usually a problem, since the methodology is rarely applied to wild populations (although it has been used for several wild ungulate and bird populations), and sires are invariably known with a very high degree of certainty in breeding programmes. There are also algorithms that account for uncertain paternity.</p>
<p>The pedigrees can be viewed using programs such as Pedigree Viewer <a href="http://www-personal.une.edu.au/~bkinghor/pedigree.htm">1</a>, and analyzed with programs such as <a class="uri" href="ASReml" title="wikilink">ASReml</a>, VCE <a href="http://vce.tzv.fal.de/index.pl">2</a>, WOMBAT <a href="http://agbu.une.edu.au/~kmeyer/wombat.html">3</a> or BLUPF90 family's programs <a href="http://nce.ads.uga.edu/~ignacy/programs.html">4</a></p>
<h2 id="response-to-selection">Response to selection</h2>
<p>In <a href="selective_breeding" title="wikilink">selective breeding</a> of plants and animals, the expected response to selection of a trait with known narrow-sense heritability h can be estimated using the <em>breeder's equation</em>:<a class="footnoteRef" href="#fn13" id="fnref13"><sup>13</sup></a> <span class="LaTeX">$R = h^2 S$</span></p>
<p>In this equation, the Response to Selection (R) is defined as the realized average difference between the parent generation and the next generation, and the Selection Differential (S) is defined as the average difference between the parent generation and the selected parents.</p>
<p>For example, imagine that a plant breeder is involved in a selective breeding project with the aim of increasing the number of kernels per ear of corn. For the sake of argument, let us assume that the average ear of corn in the parent generation has 100 kernels. Let us also assume that the selected parents produce corn with an average of 120 kernels per ear. If h<sup>2</sup> equals 0.5, then the next generation will produce corn with an average of 0.5(120-100) = 10 additional kernels per ear. Therefore, the total number of kernels per ear of corn will equal, on average, 110.</p>
<p>Note that heritability in the above equation is equal to the ratio <span class="LaTeX">$\mathrm{Var}(A)/\mathrm{Var}(P)$</span> only if the genotype and the environmental noise follow <a href="Gaussian_distributions" title="wikilink">Gaussian distributions</a> .</p>
<h2 id="controversies">Controversies</h2>
<p>Some authors like <a href="Steven_Rose" title="wikilink">Steven Rose</a><a class="footnoteRef" href="#fn14" id="fnref14"><sup>14</sup></a> and <a href="Jay_Joseph" title="wikilink">Jay Joseph</a><a class="footnoteRef" href="#fn15" id="fnref15"><sup>15</sup></a> have dismissed heritability estimates as useless. Criticisms have also been raised with regards to the lack of progress in identifying causal genetic variants for many traits (<a href="Missing_heritability_problem" title="wikilink">Missing heritability problem</a>).</p>
<p>However, a 2008 paper in <em><a href="Nature_Reviews_Genetics" title="wikilink">Nature Reviews Genetics</a></em> stated however: "Despite continuous misunderstandings and controversies over its use and application, heritability remains key to the response to selection in evolutionary biology and agriculture, and to the prediction of disease risk in medicine. Recent reports of substantial heritability for gene expression and new estimation methods using marker data highlight the relevance of heritability in the genomics era."<a class="footnoteRef" href="#fn16" id="fnref16"><sup>16</sup></a> According to Eric Turkheimer, new molecular genetic methods of heritability estimation have demonstrated that the underlying assumptions of the classic twin design of human behavioral genetics are sound, "putting an end to an entire misguided school of thought among traditional opponents of classical quantitative (and by association behavioral) genetics."<a class="footnoteRef" href="#fn17" id="fnref17"><sup>17</sup></a></p>
<h2 id="see-also">See also</h2>
<ul>
<li><a class="uri" href="Heredity" title="wikilink">Heredity</a></li>
<li><a href="Heritability_of_IQ" title="wikilink">Heritability of IQ</a></li>
</ul>
<h2 id="references">References</h2>
<h3 id="notes">Notes</h3>
<h3 id="books">Books</h3>
<ul>
<li>Falconer, D. S. and T. Mackay. 1996. Introduction to Quantitative Genetics, Ed. 4. Longman, Essex, England.</li>
<li>Kempthorne, O (1957 [1969]) <em>An Introduction to Genetic Statistics</em>. John Wiley. Reprinted, 1969 by Iowa State University Press.</li>
</ul>
<h2 id="further-reading">Further reading</h2>
<ul>
<li>Lynch, M. & Walsh, B. 1996. <em>Genetics and Analysis of Quantitative Traits</em>. Sinauer Associates. ISBN 0-87893-481-2.</li>
<li></li>
</ul>
<h2 id="external-links">External links</h2>
<ul>
<li><a href="http://plato.stanford.edu/entries/heredity/">Stanford Encyclopedia of Philosophy entry on Heredity and Heritability</a></li>
<li><a href="http://nitro.biosci.arizona.edu/zbook/book.html">Quantitative Genetics Resources website, including the two volume book by Lynch and Walsh. Free access</a></li>
</ul>
<p>"</p>
<p><a class="uri" href="Category:Genetics" title="wikilink">Category:Genetics</a></p>
<section class="footnotes">
<hr/>
<ol>
<li id="fn1"><a href="#fnref1">↩</a></li>
<li id="fn2"><a href="Geoffrey_Miller_(psychologist)" title="wikilink">Miller, Geoffrey</a> (2000). <em>The mating mind: how sexual choice shaped the evolution of human nature</em>, London, Heineman, ISBN 0-434-00741-2 (also Doubleday, ISBN 0-385-49516-1) p.115<a href="#fnref2">↩</a></li>
<li id="fn3"><a href="#fnref3">↩</a></li>
<li id="fn4"><a href="#fnref4">↩</a></li>
<li id="fn5"><a href="#fnref5">↩</a></li>
<li id="fn6">; or , for example.<a href="#fnref6">↩</a></li>
<li id="fn7"><a href="#fnref7">↩</a></li>
<li id="fn8"><a href="#fnref8">↩</a></li>
<li id="fn9">Gielen, M., Lindsey, P.J., Derom, C., Smeets, H.J.M., Souren, N.Y., Paulussen, A.D.C., Derom, R., & Nijhuis, J.G. (2008) "Modeling Genetic and Environmental Factors to IncreaseHeritability and Ease the Identification of Candidate Genes for Birth Weight: A Twin Study". Behavioral Genetics. 38(44-54):45. DOI 10.1007/s10519-007-9170-3<a href="#fnref9">↩</a></li>
<li id="fn10"><a href="#fnref10">↩</a></li>
<li id="fn11"><a href="#fnref11">↩</a></li>
<li id="fn12"><a href="#fnref12">↩</a></li>
<li id="fn13">Plomin, R., DeFries, J. C., & McClearn, G. E. (1990). <a href="Behavioural_genetics" title="wikilink">Behavioral genetics</a>. New York: Freeman.<a href="#fnref13">↩</a></li>
<li id="fn14"><a href="#fnref14">↩</a></li>
<li id="fn15">Joseph, J. (2004), <em><a href="The_Gene_Illusion" title="wikilink">The Gene Illusion</a></em>, <a class="uri" href="Algora" title="wikilink">Algora</a>. (2003 <a href="United_Kingdom" title="wikilink">United Kingdom</a> Edition by <a href="PCCS_Books" title="wikilink">PCCS Books</a>) (Chapter 5 contains a critique of the heritability concept)<a href="#fnref15">↩</a></li>
<li id="fn16"></li>
<li id="fn17"><a href="#fnref17">↩</a></li>
</ol>
</section>
</body>
</html>
