<html lang="en">
<head>
<meta charset="utf-8"/>
<title offset="352">Kinetic proofreading</title>
<script src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_SVG.js" type="text/javascript">
</script>
</head>
<body>
<h1>Kinetic proofreading</h1>
<hr>'''Kinetic proofreading''' (or '''kinetic amplification''') is a mechanism for error correction in [[enzyme catalysis|biochemical reactions]]
<p>, proposed independently by <a href="John_Hopfield" title="wikilink">John Hopfield</a> (1974) and Jacques Ninio (1975). Kinetic proofreading allows <a href="enzyme" title="wikilink">enzymes</a> to discriminate between two possible <a href="reaction_mechanism" title="wikilink">reaction pathways</a> leading to correct or incorrect products with an accuracy higher than what one would predict based on the difference in the <a href="activation_energy" title="wikilink">activation energy</a> between these two pathways.<a class="footnoteRef" href="#fn1" id="fnref1"><sup>1</sup></a><a class="footnoteRef" href="#fn2" id="fnref2"><sup>2</sup></a></p>

<p>Increased specificity is obtained by introducing an irreversible step exiting the pathway, with <a href="reaction_intermediate" title="wikilink">reaction intermediates</a> leading to incorrect products more likely to prematurely exit the pathway than reaction intermediates leading to the correct product. If the exit step is fast relative to the next step in the pathway, the specificity can be increased by a factor of up to the ratio between the two exit rate constants. (If the next step is fast relative to the exit step, specificity will not be increased because there will not be enough time for exit to occur.) This can be repeated more than once to increase specificity further.</p>
<h2 id="specificity-paradox">Specificity paradox</h2>

<p>In <a href="protein_synthesis" title="wikilink">protein synthesis</a>, the error rate is on the order of 1 in 10,000. This means that when a <a class="uri" href="ribosome" title="wikilink">ribosome</a> is matching <a href="anticodon" title="wikilink">anticodons</a> of <a class="uri" href="tRNA" title="wikilink">tRNA</a> to the <a href="codon" title="wikilink">codons</a> of <a class="uri" href="mRNA" title="wikilink">mRNA</a>, it matches complementary sequences correctly nearly all the time. Hopfield noted that because of how similar the substrates are (the difference between a wrong codon and a right codon can be as small as a difference in a single base), an error rate that small is unachievable with a one-step mechanism. Both wrong and right tRNA can bind to the ribosome, and if the ribosome can only discriminate between them by complementary matching of the anticodon, it must rely on the small <a href="Thermodynamic_free_energy" title="wikilink">free energy</a> difference between binding three matched complementary bases or only two.</p>

<p>A one-shot machine which tests whether the codons match or not by examining whether the codon and anticodon are bound will not be able to tell the difference between wrong and right codon with an error rate less than 

<math display="inline" id="Kinetic_proofreading:0">
 <semantics>
  <msup>
   <mi>e</mi>
   <mrow>
    <mo>-</mo>
    <mn>10</mn>
   </mrow>
  </msup>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <csymbol cd="ambiguous">superscript</csymbol>
    <ci>e</ci>
    <apply>
     <minus></minus>
     <cn type="integer">10</cn>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   e^{-10}
  </annotation>
 </semantics>
</math>

 unless the free energy difference is at least 10<a href="kT_(energy)" title="wikilink">kT</a>, which is much larger than the free energy difference for single codon binding. This is a thermodynamic bound, so it cannot be evaded by building a different machine. However, this can be overcome by kinetic proofreading, which introduces an irreversible step through the input of energy.<a class="footnoteRef" href="#fn3" id="fnref3"><sup>3</sup></a></p>

<p>Another molecular recognition mechanism, which does <em>not</em> require expenditure of free energy is that of <a href="conformational_proofreading" title="wikilink">conformational proofreading</a>. The incorrect product may also be formed but hydrolyzed at a greater rate than the correct product, giving the possibility of theoretically infinite specificity the longer you let this reaction run, but at the cost of large amounts of the correct product as well. (Thus there is a tradeoff between product production and its efficiency.) The hydrolytic activity may be on the same enzyme, as in DNA polymerases with editing functions, or on different enzymes.</p>
<h2 id="multistep-ratchet">Multistep ratchet</h2>

<p>Hopfield suggested a simple way to achieve smaller error rates using a molecular ratchet which takes many irreversible steps, each testing to see if the sequences match. At each step, energy is expended and specificity (the ratio of correct substrate to incorrect substrate at that point in the pathway) increases.</p>

<p>The requirement for energy in each step of the ratchet is due to the need for the steps to be irreversible; for specificity to increase, entry of substrate and analogue must occur largely through the entry pathway, and exit largely through the exit pathway. If entry were an equilibrium, the earlier steps would form a pre-equilibrium and the specificity benefits of entry into the pathway (less likely for the substrate analogue) would be lost; if the exit step were an equilibrium, then the substrate analogue would be able to re-enter the pathway through the exit step, bypassing the specificity of earlier steps altogether.</p>

<p>Although one test will only be able to discriminate between mismatched and matched sequences a fraction 

<math display="inline" id="Kinetic_proofreading:1">
 <semantics>
  <mrow>
   <mi>p</mi>
   <mo>=</mo>
   <msup>
    <mi>e</mi>
    <mrow>
     <mo>-</mo>
     <mrow>
      <mrow>
       <mrow>
        <mi mathvariant="normal">Δ</mi>
        <mi>F</mi>
       </mrow>
       <mo>/</mo>
       <mi>k</mi>
      </mrow>
      <mi>T</mi>
     </mrow>
    </mrow>
   </msup>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <eq></eq>
    <ci>p</ci>
    <apply>
     <csymbol cd="ambiguous">superscript</csymbol>
     <ci>e</ci>
     <apply>
      <minus></minus>
      <apply>
       <times></times>
       <apply>
        <divide></divide>
        <apply>
         <times></times>
         <ci>normal-Δ</ci>
         <ci>F</ci>
        </apply>
        <ci>k</ci>
       </apply>
       <ci>T</ci>
      </apply>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   p=e^{-\Delta F/kT}
  </annotation>
 </semantics>
</math>

 of the time, two tests will both fail only 

<math display="inline" id="Kinetic_proofreading:2">
 <semantics>
  <msup>
   <mi>p</mi>
   <mn>2</mn>
  </msup>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <csymbol cd="ambiguous">superscript</csymbol>
    <ci>p</ci>
    <cn type="integer">2</cn>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   p^{2}
  </annotation>
 </semantics>
</math>

 of the time, and N tests will fail 

<math display="inline" id="Kinetic_proofreading:3">
 <semantics>
  <msup>
   <mi>p</mi>
   <mi>N</mi>
  </msup>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <csymbol cd="ambiguous">superscript</csymbol>
    <ci>p</ci>
    <ci>N</ci>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   p^{N}
  </annotation>
 </semantics>
</math>

 of the time. In terms of free energy, the discrimination power of N successive tests for two states with a free energy 

<math display="inline" id="Kinetic_proofreading:4">
 <semantics>
  <mrow>
   <mi mathvariant="normal">Δ</mi>
   <mi>F</mi>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <times></times>
    <ci>normal-Δ</ci>
    <ci>F</ci>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \Delta F
  </annotation>
 </semantics>
</math>

 is the same as one test between two states with a free energy 

<math display="inline" id="Kinetic_proofreading:5">
 <semantics>
  <mrow>
   <mi>N</mi>
   <mi mathvariant="normal">Δ</mi>
   <mi>F</mi>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <times></times>
    <ci>N</ci>
    <ci>normal-Δ</ci>
    <ci>F</ci>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   N\Delta F
  </annotation>
 </semantics>
</math>

.</p>

<p>To achieve an error rate of 

<math display="inline" id="Kinetic_proofreading:6">
 <semantics>
  <msup>
   <mi>e</mi>
   <mrow>
    <mo>-</mo>
    <mn>10</mn>
   </mrow>
  </msup>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <csymbol cd="ambiguous">superscript</csymbol>
    <ci>e</ci>
    <apply>
     <minus></minus>
     <cn type="integer">10</cn>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   e^{-10}
  </annotation>
 </semantics>
</math>

 requires several comparison steps. Hopfield predicted on the basis of this theory that there is a multistage ratchet in the ribosome which tests the match several times before incorporating the next amino acid into the protein.</p>
<h2 id="experimental-examples">Experimental examples</h2>
<ul>
<li><strong>Charging tRNAs with their respective amino-acids</strong> – the enzyme that charges the <a class="uri" href="tRNA" title="wikilink">tRNA</a> is called <a href="aminoacyl_tRNA_synthetase" title="wikilink">aminoacyl tRNA synthetase</a>. This enzyme utilizes a high energy intermediate state to increase the fidelity of binding the right pair of tRNA and amino-acid.<a class="footnoteRef" href="#fn4" id="fnref4"><sup>4</sup></a> In this case, energy is used to make the high-energy intermediate (making the entry pathway irreversible), and the exit pathway is irreversible by virtue of the high energy difference in dissociation.</li>
</ul>
<ul>
<li><strong>Homologous recombination</strong> – <a href="Homologous_recombination" title="wikilink">Homologous recombination</a> facilitates the exchange between homologous or almost homologous DNA strands. During this process, the RecA protein polymerizes along a DNA and this DNA-protein filament searches for a homologous DNA sequence. Both processes of RecA polymerization<a class="footnoteRef" href="#fn5" id="fnref5"><sup>5</sup></a><a class="footnoteRef" href="#fn6" id="fnref6"><sup>6</sup></a> and homology search<a class="footnoteRef" href="#fn7" id="fnref7"><sup>7</sup></a> utilize the kinetic proofreading mechanism.</li>
</ul>
<ul>
<li><strong>DNA damage recognition and repair</strong> – a certain <a href="DNA_repair" title="wikilink">DNA repair</a> mechanism utilizes kinetic proofreading to discriminate damaged DNA.<a class="footnoteRef" href="#fn8" id="fnref8"><sup>8</sup></a> Some DNA polymerases can also detect when they have added an incorrect base and are able to hydrolyze it immediately; in this case, the irreversible (energy-requiring) step is addition of the base.</li>
</ul>
<h2 id="theoretical-considerations">Theoretical considerations</h2>
<h3 id="universal-first-passage-time">Universal first passage time</h3>

<p>Biochemical processes that use kinetic proofreading to improve specificity implement the delay-inducing multistep ratchet by a variety of distinct biochemical networks. Nonetheless, many such networks result in the times to completion of the molecular assembly and the proofreading steps (also known as the <a href="first_passage_time" title="wikilink">first passage time</a>) that approach a near-universal, exponential shape for high proofreading rates and large network sizes.<a class="footnoteRef" href="#fn9" id="fnref9"><sup>9</sup></a> Since exponential completion times are characteristic of a two-state <a href="Markov_process" title="wikilink">Markov process</a>, this observation makes kinetic proofreading one of only a few examples of biochemical processes where structural complexity results in a much simpler large-scale, phenomenological dynamics.</p>
<h3 id="topology-of-kinetic-proofreading">Topology of kinetic proofreading</h3>

<p>The increase in specificity, or the overall amplification factor of a kinetic proofreading network that may include multiple pathways and especially loops is intimately related to the topology of the network: the specificity grows exponentially with the number of loops in the network.<a class="footnoteRef" href="#fn10" id="fnref10"><sup>10</sup></a><a class="footnoteRef" href="#fn11" id="fnref11"><sup>11</sup></a> An example is homologous recombination in which the number of loops scales like the square of DNA length.<a class="footnoteRef" href="#fn12" id="fnref12"><sup>12</sup></a><a class="footnoteRef" href="#fn13" id="fnref13"><sup>13</sup></a> <a href="#Universal_first_passage_time" title="wikilink">The universal completion time</a> emerges precisely in this regime of large number of loops and high amplification.<a class="footnoteRef" href="#fn14" id="fnref14"><sup>14</sup></a></p>
<h2 id="references">References</h2>
<h2 id="further-reading">Further reading</h2>
<ul>
<li></li>
<li></li>
</ul>

<p>"</p>

<p><a href="Category:Biological_processes" title="wikilink">Category:Biological processes</a> <a href="Category:DNA_replication" title="wikilink">Category:DNA replication</a> <a href="Category:Mathematical_and_theoretical_biology" title="wikilink"> </a></p>
<section class="footnotes">
<hr/>
<ol>
<li id="fn1"><a href="#fnref1">↩</a></li>
<li id="fn2"><a href="#fnref2">↩</a></li>
<li id="fn3"><a href="#fnref3">↩</a></li>
<li id="fn4"><a href="#fnref4">↩</a></li>
<li id="fn5"><a href="#fnref5">↩</a></li>
<li id="fn6"><a href="#fnref6">↩</a></li>
<li id="fn7"><a href="#fnref7">↩</a></li>
<li id="fn8"><a href="#fnref8">↩</a></li>
<li id="fn9"><a href="#fnref9">↩</a></li>
<li id="fn10"><a href="#fnref10">↩</a></li>
<li id="fn11"><a href="#fnref11">↩</a></li>
<li id="fn12"></li>
<li id="fn13"></li>
<li id="fn14"></li>
</ol>
</section>
</hr></body>
</html>
