<html lang="en">
<head>
<meta charset="utf-8"/>
<title offset="578">Malate dehydrogenase (oxaloacetate-decarboxylating) (NADP+)</title>
<script src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_SVG.js" type="text/javascript">
</script>
</head>
<body>
<h1>Malate dehydrogenase (oxaloacetate-decarboxylating) (NADP+)</h1>
<hr/>

<p><mtpl></mtpl> <strong>Malate dehydrogenase (oxaloacetate-decarboxylating) (NADP<sup>+</sup>)</strong> () or <strong>NADP-malic enzyme</strong> (NADP-ME) is an <a class="uri" href="enzyme" title="wikilink">enzyme</a> that <a href="catalysis" title="wikilink">catalyzes</a> the <a href="chemical_reaction" title="wikilink">chemical reaction</a> in the presence of a bivalent metal ion:<a class="footnoteRef" href="#fn1" id="fnref1"><sup>1</sup></a></p>
<dl>
<dd>(S)-malate + NADP<sup>+</sup> 

<math display="inline" id="Malate_dehydrogenase_(oxaloacetate-decarboxylating)_(NADP+):0">
 <semantics>
  <mo>⇌</mo>
  <annotation-xml encoding="MathML-Content">
   <ci>normal-⇌</ci>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \rightleftharpoons
  </annotation>
 </semantics>
</math>

 pyruvate + CO<sub>2</sub> + NADPH
</dd>
</dl>

<p>Thus, the two <a href="substrate_(biochemistry)" title="wikilink">substrates</a> of this enzyme are <a href="Malic_acid" title="wikilink">(S)-malate</a> and <a href="nicotinamide_adenine_dinucleotide_phosphate" title="wikilink">NADP<sup>+</sup></a>, whereas its 3 <a href="product_(chemistry)" title="wikilink">products</a> are <a class="uri" href="pyruvate" title="wikilink">pyruvate</a>, <a href="carbon_dioxide" title="wikilink">CO<sub>2</sub></a>, and <a href="nicotinamide_adenine_dinucleotide_phosphate" title="wikilink">NADPH</a>. Malate is <a href="redox" title="wikilink">oxidized</a> to pyruvate and CO<sub>2</sub>, and NADP<sup>+</sup> is reduced to NADPH.</p>

<p>This enzyme belongs to the family of <a href="oxidoreductase" title="wikilink">oxidoreductases</a>, to be specific those acting on the CH-OH group of donor with NAD<sup>+</sup> or NADP<sup>+</sup> as acceptor. The systematic name of this enzyme class is <strong>(S)-malate:NADP<sup>+</sup> oxidoreductase (oxaloacetate-decarboxylating)</strong>. This enzyme participates in <a href="pyruvate_metabolism" title="wikilink">pyruvate metabolism</a> and <a href="carbon_fixation" title="wikilink">carbon fixation</a>. NADP-malic enzyme is one of three decarboxylation enzymes used in the inorganic carbon concentrating mechanisms of <a href="C4_carbon_fixation" title="wikilink">C4</a> and <a href="Crassulacean_acid_metabolism" title="wikilink">CAM</a> plants. The others are <a href="NAD-malic_enzyme" title="wikilink">NAD-malic enzyme</a> and <a href="PEP_carboxykinase" title="wikilink">PEP carboxykinase</a>.<a class="footnoteRef" href="#fn2" id="fnref2"><sup>2</sup></a><a class="footnoteRef" href="#fn3" id="fnref3"><sup>3</sup></a> Although often one of the three photosynthetic decarboxylases predominate, the simultaneous operation of all three is also shown to exist.<a class="footnoteRef" href="#fn4" id="fnref4"><sup>4</sup></a><a class="footnoteRef" href="#fn5" id="fnref5"><sup>5</sup></a></p>
<figure><b>(Figure)</b>
<figcaption>ChemDraw generated NADP-ME chemical reaction detailing the chemical structures involved.</figcaption>
</figure>
<h2 id="enzyme-structure">Enzyme Structure</h2>

<p> Based on <a class="uri" href="crystallography" title="wikilink">crystallography</a> data of <a href="Homology_(biology)" title="wikilink">homologous</a> NADP-dependent malic enzymes of mammalian origin, a 3D model for C<sub>4</sub> pathway NADP-ME in plants has been developed, identifying the key residues involved in substrate-binding or catalysis. <a href="Nicotinamide_adenine_dinucleotide_phosphate" title="wikilink">Dinucleotide</a> binding involves two <a class="uri" href="glycine" title="wikilink">glycine</a> rich GXGXXG motifs, a <a class="uri" href="hydrophobic" title="wikilink">hydrophobic</a> groove involving at least six amino acid residues, and a negatively charged residue at the end of the βB-strand.<a class="footnoteRef" href="#fn6" id="fnref6"><sup>6</sup></a><a class="footnoteRef" href="#fn7" id="fnref7"><sup>7</sup></a> The <a href="primary_sequence" title="wikilink">primary sequence</a> of the first motif, <sup>240</sup>GLGDLG<sup>245</sup>, is a consensus marker for phosphate binding, evidencing involvement with NADP binding, while the other glycine rich motif adopts a classical <a href="Rossman_fold" title="wikilink">Rossman fold</a>—also a typical marker for <a href="Nicotinamide_adenine_dinucleotide_phosphate" title="wikilink">NADP</a> <a href="Cofactor_(biochemistry)" title="wikilink">cofactor</a> binding.<a class="footnoteRef" href="#fn8" id="fnref8"><sup>8</sup></a> <a class="uri" href="Mutagenesis" title="wikilink">Mutagenesis</a> experiments in maize NADP-ME have supported the current model.<a class="footnoteRef" href="#fn9" id="fnref9"><sup>9</sup></a> <a class="uri" href="Valine" title="wikilink">Valine</a> substitution for glycine in either motif region rendered the enzyme completely inactive while spectral analysis indicated no major changes from wild-type form. The data is suggestive of direct impairment at a key residue involved in binding or catalysis rather than an inter-domain residue influencing conformational stability. Additionally, a key <a class="uri" href="arginine" title="wikilink">arginine</a> residue at site 237 has been shown to interact both with <a class="uri" href="malate" title="wikilink">malate</a> and NADP<sup>+</sup> substrates, forming key favorable <a href="Ionic_bond" title="wikilink">electrostatic</a> interactions to the negatively charged carboxylic-acid and phosphate group respectively. Elucidation of whether the residue plays a role in substrate binding or substrate positioning for catalysis has yet to be determined.<a class="footnoteRef" href="#fn10" id="fnref10"><sup>10</sup></a> <a class="uri" href="Lysine" title="wikilink">Lysine</a> residue 255 has been implicated as a catalytic <a href="Base_(chemistry)" title="wikilink">base</a> for the enzymes reactivity; however, further studies are still required to conclusively establish its biochemical role.<a class="footnoteRef" href="#fn11" id="fnref11"><sup>11</sup></a></p>
<h3 id="structural-studies">Structural studies</h3>

<p>, 3 <a href="tertiary_structure" title="wikilink">structures</a> have been solved for this class of enzymes, with <a href="Protein_Data_Bank" title="wikilink">PDB</a> accession codes , , and .</p>
<h2 id="biological-function">Biological Function</h2>

<p>In a broader context, malic enzymes are found within a wide range of <a class="uri" href="eukaryotic" title="wikilink">eukaryotic</a> organisms, from fungi to mammals, and beyond that, are shown to localize in range of subcellular locations, including the <a class="uri" href="cytosol" title="wikilink">cytosol</a>, <a class="uri" href="mitochondria" title="wikilink">mitochondria</a>, and <a class="uri" href="chloroplast" title="wikilink">chloroplast</a>. C<sub>4</sub> NADP-ME, specifically, is in plants localized in <a href="bundle_sheath" title="wikilink">bundle sheath</a> chloroplasts.<a class="footnoteRef" href="#fn12" id="fnref12"><sup>12</sup></a></p>

<p>During C<sub>4</sub> <a class="uri" href="photosynthesis" title="wikilink">photosynthesis</a>, an evolved pathway to increase localized CO<sub>2</sub> concentrations under the threat of enhanced <a class="uri" href="photorespiration" title="wikilink">photorespiration</a>, <a href="Carbon_dioxide" title="wikilink">CO<sub>2</sub></a> is captured within <a href="mesophyll_tissue" title="wikilink">mesophyll</a> cells, fixed as <a class="uri" href="oxaloacetate" title="wikilink">oxaloacetate</a>, converted into malate and released internally within bundle sheath cells to directly feed <a class="uri" href="RuBisCO" title="wikilink">RuBisCO</a> activity.<a class="footnoteRef" href="#fn13" id="fnref13"><sup>13</sup></a> This release of fixed CO<sub>2</sub>, triggered by the favorable decarboxylation of malate into <a class="uri" href="pyruvate" title="wikilink">pyruvate</a>, is mediated by NADP-dependent malic enzyme. In fact, the significance of NADP-ME activity in CO<sub>2</sub> conservation is evidenced by a study performed with transgenic plants exhibiting a NADP-ME loss of function mutation. Plants with the <a class="uri" href="mutation" title="wikilink">mutation</a> experienced 40% the activity of wild-type NADP-ME and achieved significantly reduced CO<sub>2</sub> uptake even at high intercellular levels of CO<sub>2</sub>, evidencing the biological importance of NADP-ME at regulating carbon <a href="Flux_(metabolism)" title="wikilink">flux</a> towards the <a href="Light-independent_reactions#Calvin_Cycle" title="wikilink">Calvin cycle</a>.<a class="footnoteRef" href="#fn14" id="fnref14"><sup>14</sup></a><a class="footnoteRef" href="#fn15" id="fnref15"><sup>15</sup></a></p>
<h2 id="enzyme-regulation">Enzyme Regulation</h2>

<p>NADP-ME <a href="Protein_expression" title="wikilink">expression</a> has been shown to be regulated by <a class="uri" href="abiotic" title="wikilink">abiotic</a> stress factors. For <a href="Cam_Plants" title="wikilink">CAM plants</a>, <a class="uri" href="drought" title="wikilink">drought</a> conditions cause <a class="uri" href="stoma" title="wikilink">stoma</a> to largely remain shut to avoid water loss by <a class="uri" href="evapotranspiration" title="wikilink">evapotranspiration</a>, which unfortunately leads to CO<sub>2</sub> starvation. In compensation, closed <a class="uri" href="stoma" title="wikilink">stoma</a> activates the translation of NADP-ME to reinforce high efficiency of CO<sub>2</sub> assimilation during the brief intervals of CO<sub>2</sub> intake, allowing for <a href="carbon_fixation" title="wikilink">carbon fixation</a> to continue.<a class="footnoteRef" href="#fn16" id="fnref16"><sup>16</sup></a> In addition to regulation at the longer time scale by means of expression control, regulation at the short-time scale can occur through <a href="Allosteric_regulation" title="wikilink">allosteric</a> mechanisms. C<sub>4</sub> NADP-ME has been shown to be partially inhibited by its <a href="Substrate_(biochemistry)" title="wikilink">substrate</a>, malate, suggesting two independent binding sites: one at the active site and one at an allosteric site. However, the inhibitory effect exhibits <a class="uri" href="pH" title="wikilink">pH</a>-dependence – existent at a pH of 7 but not a pH of 8. The control of <a class="uri" href="enzyme" title="wikilink">enzyme</a> activity due to pH changes align with the hypothesis that NADP-ME is most active while <a class="uri" href="photosynthesis" title="wikilink">photosynthesis</a> is in progress: Active light reactions leads to a rise in basicity within the <a class="uri" href="chloroplast" title="wikilink">chloroplast</a> <a href="stroma_(fluid)" title="wikilink">stroma</a>, the location of NADP-ME, leading to a diminished <a href="Negative_allosteric_modulator" title="wikilink">inhibitory</a> effect of malate on NADP-ME and thereby promoting a more active state. Conversely, slowed light reactions leads to a rise in <a class="uri" href="acidity" title="wikilink">acidity</a> within the stroma, promoting the inhibition of NADP-ME by malate. Because the high energy products of the <a href="light_reactions" title="wikilink">light reactions</a>, <a class="uri" href="NADPH" title="wikilink">NADPH</a> and <a href="Adenosine_triphosphate" title="wikilink">ATP</a>, are required for the <a href="Calvin_Cycle" title="wikilink">Calvin Cycle</a> to proceed, a buildup of CO<sub>2</sub> without them is not useful, explaining the need for the regulatory mechanism.<a class="footnoteRef" href="#fn17" id="fnref17"><sup>17</sup></a></p>

<p>This protein may use the <a class="uri" href="morpheein" title="wikilink">morpheein</a> model of <a href="allosteric_regulation" title="wikilink">allosteric regulation</a>.<a class="footnoteRef" href="#fn18" id="fnref18"><sup>18</sup></a></p>
<h2 id="evolution">Evolution</h2>

<p>NADP-malic enzyme, as all other C<sub>4</sub> decarboxylases, did not evolve de novo for CO<sub>2</sub> pooling to aid <a class="uri" href="RuBisCO" title="wikilink">RuBisCO</a>.<a class="footnoteRef" href="#fn19" id="fnref19"><sup>19</sup></a> Rather, NADP-ME was directly transformed from a C<sub>3</sub> species in <a class="uri" href="photosynthesis" title="wikilink">photosynthesis</a>, and even earlier origins from an ancient cystolic <a class="uri" href="ancestor" title="wikilink">ancestor</a>. In the <a class="uri" href="cytosol" title="wikilink">cytosol</a>, the enzyme existed as a series of housekeeping <a href="isoform" title="wikilink">isoforms</a> purposed towards a variety of functions including malate level maintenance during <a href="hypoxia_(environmental)" title="wikilink">hypoxia</a>, <a class="uri" href="microspore" title="wikilink">microspore</a> separation, and <a class="uri" href="pathogen" title="wikilink">pathogen</a> defense. In regards to the mechanism of evolution, the C<sub>4</sub> functionality is thought to have stemmed from gene duplication error both within <a href="Promoter_(genetics)" title="wikilink">promoter</a> regions, triggering <a class="uri" href="overexpression" title="wikilink">overexpression</a> in bundle-sheath cells, and within the coding region, generating <a class="uri" href="neofunctionalization" title="wikilink">neofunctionalization</a>.<a class="footnoteRef" href="#fn20" id="fnref20"><sup>20</sup></a> Selection for CO<sub>2</sub> preservation function as well as enhanced water and nitrogen utilization under stressed conditions was then shaped by natural pressures.<a class="footnoteRef" href="#fn21" id="fnref21"><sup>21</sup></a></p>
<h2 id="see-also">See also</h2>
<ul>
<li><em><a href="ME1_(gene)" title="wikilink">ME1 (gene)</a></em> (human gene)</li>
</ul>
<h2 id="references">References</h2>
<h2 id="further-reading">Further reading</h2>
<ul>
<li></li>
<li></li>
<li></li>
<li></li>
<li></li>
<li></li>
</ul>

<p>"</p>

<p><a href="Category:EC_1.1.1" title="wikilink">Category:EC 1.1.1</a> <a href="Category:NADPH-dependent_enzymes" title="wikilink">Category:NADPH-dependent enzymes</a> <a href="Category:Enzymes_of_known_structure" title="wikilink">Category:Enzymes of known structure</a></p>
<section class="footnotes">
<hr/>
<ol>
<li id="fn1"></li>
<li id="fn2"><mtpl></mtpl><a href="#fnref2">↩</a></li>
<li id="fn3"><a href="#fnref3">↩</a></li>
<li id="fn4"><a href="#fnref4">↩</a></li>
<li id="fn5"><a href="#fnref5">↩</a></li>
<li id="fn6"><a href="#fnref6">↩</a></li>
<li id="fn7"><a href="#fnref7">↩</a></li>
<li id="fn8"><a href="#fnref8">↩</a></li>
<li id="fn9"></li>
<li id="fn10"><a href="#fnref10">↩</a></li>
<li id="fn11"></li>
<li id="fn12"><a href="#fnref12">↩</a></li>
<li id="fn13"><a href="#fnref13">↩</a></li>
<li id="fn14"><a href="#fnref14">↩</a></li>
<li id="fn15"><a href="#fnref15">↩</a></li>
<li id="fn16"><a href="#fnref16">↩</a></li>
<li id="fn17"><a href="#fnref17">↩</a></li>
<li id="fn18"><a href="#fnref18">↩</a></li>
<li id="fn19"><a href="#fnref19">↩</a></li>
<li id="fn20"><a href="#fnref20">↩</a></li>
<li id="fn21"><a href="#fnref21">↩</a></li>
</ol>
</section>
</body>
</html>
