<html lang="en">
<head>
<meta charset="utf-8"/>
<title offset="225">Methylglyoxal synthase</title>
<script src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_SVG.js" type="text/javascript">
</script>
</head>
<body>
<h1>Methylglyoxal synthase</h1>
<hr/>

<p>In <a class="uri" href="enzymology" title="wikilink">enzymology</a>, a <strong>methylglyoxal synthase</strong> () is a <a class="uri" href="enzyme" title="wikilink">enzyme</a> that <a href="catalysis" title="wikilink">catalyzes</a> the <a href="chemical_reaction" title="wikilink">chemical reaction</a></p>
<dl>
<dd>dihydroxyacetone phosphate 

<math display="inline" id="Methylglyoxal_synthase:0">
 <semantics>
  <mo>⇌</mo>
  <annotation-xml encoding="MathML-Content">
   <ci>normal-⇌</ci>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \rightleftharpoons
  </annotation>
 </semantics>
</math>

 methylglyoxal + phosphate
</dd>
</dl>

<p>Hence, this enzyme has one <a href="substrate_(biochemistry)" title="wikilink">substrate</a>, <a href="dihydroxyacetone_phosphate" title="wikilink">DHAP</a>, and two <a href="product_(chemistry)" title="wikilink">products</a>, <a class="uri" href="methylglyoxal" title="wikilink">methylglyoxal</a> and <a class="uri" href="phosphate" title="wikilink">phosphate</a>. Attempts to observe <a href="Reversible_reaction" title="wikilink">reversibility</a> of this reaction have been unsuccessful.<a class="footnoteRef" href="#fn1" id="fnref1"><sup>1</sup></a></p>

<p>This enzyme belongs to the family of <a href="lyase" title="wikilink">lyases</a>, specifically those carbon-oxygen lyases acting on phosphates. The systematic name of this enzyme class is <strong>glycerone-phosphate phosphate-lyase (methylglyoxal-forming)</strong>. Other names in common use include <strong>methylglyoxal synthetase</strong>, and <strong>glycerone-phosphate phospho-lyase</strong>. This enzyme participates in <a href="pyruvate_metabolism" title="wikilink">pyruvate metabolism</a> and is constitutively expressed.<a class="footnoteRef" href="#fn2" id="fnref2"><sup>2</sup></a></p>
<h2 id="structural-studies">Structural Studies</h2>
<figure><b>(Figure)</b>
<figcaption>Crystal structure (PDB ID: 1EGH) of methylglyoxal synthase homohexamer with residues of one active site highlighted. Three active sites exists within the three major grooves. Image generated with <a class="uri" href="PyMOL" title="wikilink">PyMOL</a>.</figcaption>
</figure>
<figure><b>(Figure)</b>
<figcaption>Active site residues of methylglyoxal synthase (Lys23, Thr45, Thr48, Gly66, His19, His98, Asp71). Image generated from crystal structure (PDB ID: 1EGH) with PyMOL.</figcaption>
</figure>

<p>As of late 2007, 7 <a href="tertiary_structure" title="wikilink">structures</a> have been solved for this class of enzymes, with <a href="Protein_Data_Bank" title="wikilink">PDB</a> accession codes , , , , , , and .</p>

<p>Methylglyoxal synthase (MGS) is a 152-amino acid homohexamer that has a <a href="molecular_mass" title="wikilink">molecular weight</a> of approximately 67,000 kD.<a class="footnoteRef" href="#fn3" id="fnref3"><sup>3</sup></a><a class="footnoteRef" href="#fn4" id="fnref4"><sup>4</sup></a><a class="footnoteRef" href="#fn5" id="fnref5"><sup>5</sup></a> The total <a class="uri" href="solvent" title="wikilink">solvent</a>-accessible surface area of the MGS homohexamer is 18,510 square Angstroms, roughly 40% of the total possible surface area if the <a href="Protein_subunit" title="wikilink">subunits</a> were separated.<a class="footnoteRef" href="#fn6" id="fnref6"><sup>6</sup></a> Each <a class="uri" href="monomer" title="wikilink">monomer</a> consists of five <a href="Alpha_helix" title="wikilink">alpha helices</a> surrounding five <a href="beta_sheets" title="wikilink">beta sheets</a>. Of these, two <a href="Antiparallel_(biochemistry)" title="wikilink">antiparallel</a> beta sheets and one alpha helix are located in a subdomain where the N-terminus and C-terminus are in close juxtaposition.<a class="footnoteRef" href="#fn7" id="fnref7"><sup>7</sup></a> The homohexamer exhibits a threefold axis perpendicular to a twofold axis. Within the wide V-groove, there are twelve <a href="hydrogen_bonds" title="wikilink">hydrogen bonds</a> and six <a href="Salt_bridge_(protein_and_supramolecular)" title="wikilink">salt bridges</a> between the monomers in the presence of phosphate binding. In the absence of phosphate binding, ten hydrogen bonds and two salt bridges hold the monomers together. At the peak interfaces, ten hydrogen bonds and no salt bridges connect the monomers regardless of phosphate binding.<a class="footnoteRef" href="#fn8" id="fnref8"><sup>8</sup></a></p>

<p>The MGS homohexamer is slightly asymmetrical. All three monomers within the asymmetrical region contain a <a class="uri" href="formate" title="wikilink">formate</a> molecule within their respective actives sites. Only one of the monomers within the asymmetrical region is additionally bound to a phosphate.<a class="footnoteRef" href="#fn9" id="fnref9"><sup>9</sup></a></p>

<p>The active site contains many <a href="conserved_sequence" title="wikilink">conserved</a> residues for function (Asp, His, Thr) and structure (Gly, Pro). Inorganic phosphate interacts with Lys23, Thr45, Thr47, Thr48, and Gly66. Formate interacts with His19, His98, and Asp71. The active site is exposed to the solvent via a perpendicular channel that consists of Arg150, Tyr146, Asp20, Pro67, His98, and His19.<a class="footnoteRef" href="#fn10" id="fnref10"><sup>10</sup></a></p>

<p>Although mechanistically similar to <a href="triosephosphate_isomerase" title="wikilink">triosephosphate isomerase</a> (TIM), MGS contains widely dissimilar protein folding that prevents structural alignment with TIM which suggests <a href="convergent_evolution" title="wikilink">convergent evolution</a> of their chemical reactions. However, Asp71 in MGS may act similarly to the Glu165, the catalytic base in TIM. Additionally, His19 and His98 may perform the role of the electrophilic catalyst similar to His95 in TIM. <a href="Protein-glutamate_methylesterase" title="wikilink">CheB methylesterase</a> has the highest <a href="structural_similarity" title="wikilink">structural similarity</a> with MGS.<a class="footnoteRef" href="#fn11" id="fnref11"><sup>11</sup></a></p>
<h2 id="mechanism">Mechanism</h2>

<p>Methylglyoxal synthase is highly specific for DHAP with <a href="Michaelis–Menten_kinetics" title="wikilink">Km</a> 0.47mM at its optimal <a class="uri" href="pH" title="wikilink">pH</a> of 7.5.<a class="footnoteRef" href="#fn12" id="fnref12"><sup>12</sup></a><a class="footnoteRef" href="#fn13" id="fnref13"><sup>13</sup></a> Contrary to early reports, the purified enzyme does not react with other glycolytic metabolites such as <a class="uri" href="glyceraldehyde-3-phosphate" title="wikilink">glyceraldehyde-3-phosphate</a> or <a href="fructose_1,6-diphosphate" title="wikilink">fructose 1,6-diphosphate</a>.<a class="footnoteRef" href="#fn14" id="fnref14"><sup>14</sup></a><a class="footnoteRef" href="#fn15" id="fnref15"><sup>15</sup></a> The mechanism of MGS is similar to that of TIM; both enzymes react with dihydroxyacetone phosphate to form an <a href="enol" title="wikilink">ene-diol</a> phosphate intermediate as the first step of their reaction pathways.<a class="footnoteRef" href="#fn16" id="fnref16"><sup>16</sup></a> However, the second step involves the <a href="Elimination_reaction" title="wikilink">elimination</a> of phosphate to form methylglyoxal instead of reprotonation to form glyceraldehyde-3-phosphate.<a class="footnoteRef" href="#fn17" id="fnref17"><sup>17</sup></a> The overall reaction is characterized as an intramolecular oxidation-reduction followed by a dephosphorylation. The C-3 of DHAP is oxidized to an <a class="uri" href="aldehyde" title="wikilink">aldehyde</a>, while C-1, which bears the <a href="phosphate_ester" title="wikilink">phosphate ester</a>, is dephosphorylated and reduced to a <a href="methyl_group" title="wikilink">methyl group</a>.<a class="footnoteRef" href="#fn18" id="fnref18"><sup>18</sup></a> MGS does not require the use of metal ions or a <a href="Schiff_base" title="wikilink">Schiff base</a> as part of catalysis.<a class="footnoteRef" href="#fn19" id="fnref19"><sup>19</sup></a></p>

<p><a href="File:Methylglyoxal_synthase_mechanism.png" title="wikilink">thumb|center|upright=2.8|Methylglyoxal synthase arrow-pushing mechanism. Recreated on </a><a class="uri" href="ChemDraw" title="wikilink">ChemDraw</a> and adapted from Saadat, D; Harrison, D. H. (1999). "The crystal structure of methylglyoxal synthase from Escherichia coli". Structure (London, England : 1993) 7 (3): 309–17. PMID 10368300 </p>

<p>The enzyme first uses Asp71 to specifically abstract the pro-S hydrogen from the C-3 of DHAP to form an ene-diol(ate)-enzyme intermediate, unlike the abstraction of C-3 pro-R hydrogen in TIM by Glu165.<a class="footnoteRef" href="#fn20" id="fnref20"><sup>20</sup></a><a class="footnoteRef" href="#fn21" id="fnref21"><sup>21</sup></a><a class="footnoteRef" href="#fn22" id="fnref22"><sup>22</sup></a> A second base deprotonates the <a href="hydroxyl_group" title="wikilink">hydroxyl group</a>, leading to the collapse of the en-diol(ate) to form the 2-hydroxy 2-propenal enol intermediate along with dissocation of inorganic phosophate (–OPO

<math display="inline" id="Methylglyoxal_synthase:1">
 <semantics>
  <msub>
   <mi></mi>
   <mn>3</mn>
  </msub>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <cn type="integer">3</cn>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   {}_{3}
  </annotation>
 </semantics>
</math>

) through the cleavage of a C-O bond rather than an O-P bond.<a class="footnoteRef" href="#fn23" id="fnref23"><sup>23</sup></a><a class="footnoteRef" href="#fn24" id="fnref24"><sup>24</sup></a><a class="footnoteRef" href="#fn25" id="fnref25"><sup>25</sup></a> This deprotonation is catalyzed by either Asp71 or Asp101.<a class="footnoteRef" href="#fn26" id="fnref26"><sup>26</sup></a><a class="footnoteRef" href="#fn27" id="fnref27"><sup>27</sup></a> Protonatation of the <a href="methylene_group" title="wikilink">methylene group</a> of the enolate is non-<a href="stereospecificity" title="wikilink">stereospecific</a>.<a class="footnoteRef" href="#fn28" id="fnref28"><sup>28</sup></a><a class="footnoteRef" href="#fn29" id="fnref29"><sup>29</sup></a> The reaction products are released sequentially with methylglyoxal leaving before the inorganic phosphate.<a class="footnoteRef" href="#fn30" id="fnref30"><sup>30</sup></a> MGS is responsible for the <a href="racemic_mixture" title="wikilink">racemic mixture</a> of <a href="lactic_acid" title="wikilink">lactate</a> in cells; the production of methylglyoxal and its further metabolism yields L-(+)-lactate and D-(-)-lactate, while deletion of the MGS gene leads to observation of optically pure D-(-)-lactate.<a class="footnoteRef" href="#fn31" id="fnref31"><sup>31</sup></a></p>
<h2 id="regulation">Regulation</h2>

<p>Binding of phosphate to the enzyme increases its <a class="uri" href="cooperativity" title="wikilink">cooperativity</a> via structural changes that open three DHAP-binding sites.<a class="footnoteRef" href="#fn32" id="fnref32"><sup>32</sup></a> At higher concentrations, however, phosphate acts as an <a href="Competitive_inhibition" title="wikilink">competitive</a> <a href="allostery" title="wikilink">allosteric</a> inhibitor to turn off enzymatic activity, suggesting that diversion to methylglyoxal production occurs under conditions of phosphate starvation.<a class="footnoteRef" href="#fn33" id="fnref33"><sup>33</sup></a><a class="footnoteRef" href="#fn34" id="fnref34"><sup>34</sup></a><a class="footnoteRef" href="#fn35" id="fnref35"><sup>35</sup></a> This inhibition is believed to be caused by bound phosphate and formate mimicking the reaction intermediates (enolate and inorganic phosphate).<a class="footnoteRef" href="#fn36" id="fnref36"><sup>36</sup></a> Additionally, phosphate binding causes rotation of threonine residues that close the active site.<a class="footnoteRef" href="#fn37" id="fnref37"><sup>37</sup></a></p>

<p>Ser55 in the active site of MGS is responsible for discriminating the binding of an inorganic phosphate from the phosphate group of the substrate (DHAP) by hydrogen bonding and undergoing a <a href="conformational_change" title="wikilink">conformational change</a> of location.<a class="footnoteRef" href="#fn38" id="fnref38"><sup>38</sup></a> Transmittance of the allosteric signal is determined to pass through Arg97 and Val101 because none of these are located in the active site, yet mutations at these residues negates any inhibitory effect of phosphate binding. Pro82 is necessary to transmit the signal from one subunit to the Ar97 and Val101 of another subunit.<a class="footnoteRef" href="#fn39" id="fnref39"><sup>39</sup></a> The induction of salt-bridge formation between Asp10 and Arg140 is an additional inter-subunit signal transmission pathway for organisms that retain the last 10 amino acids of the monomer peptide.<a class="footnoteRef" href="#fn40" id="fnref40"><sup>40</sup></a> The final acceptor of this allosteric signal is the catalytic Gly56 within the active site.<a class="footnoteRef" href="#fn41" id="fnref41"><sup>41</sup></a></p>

<p>Inorganic pyrophosphate has 95% the ability of phosphate in inhibiting MGS. <a class="uri" href="3-phosphoglycerate" title="wikilink">3-phosphoglycerate</a> and <a class="uri" href="phosphoenolpyruvate" title="wikilink">phosphoenolpyruvate</a> also have 50% and 70% inhibition, respectively.<a class="footnoteRef" href="#fn42" id="fnref42"><sup>42</sup></a> <a class="uri" href="2-phosphoglycolate" title="wikilink">2-phosphoglycolate</a> also acts as a competitive inhibitor by mimicking the ene-diolate intermediate.<a class="footnoteRef" href="#fn43" id="fnref43"><sup>43</sup></a> <a href="Adenosine_triphosphate" title="wikilink">ATP</a> has been shown to have weak inhibition in some bacterial strains.<a class="footnoteRef" href="#fn44" id="fnref44"><sup>44</sup></a> The reaction product, methylglyoxal, does not exhibit any <a href="feedback_inhibition" title="wikilink">feedback inhibition</a> on MGS.<a class="footnoteRef" href="#fn45" id="fnref45"><sup>45</sup></a><a class="footnoteRef" href="#fn46" id="fnref46"><sup>46</sup></a></p>
<h2 id="biological-function">Biological Function</h2>

<p>Methylglyoxal synthase provides an alternative <a href="catabolism" title="wikilink">catabolic</a> pathway for triose phosphates created in <a class="uri" href="glycolysis" title="wikilink">glycolysis</a>.<a class="footnoteRef" href="#fn47" id="fnref47"><sup>47</sup></a> It has activity levels similar to that of <a href="glyceraldehyde-3-phosphate_dehydrogenase" title="wikilink">glyceraldehyde-3-phosphate dehydrogenase</a> from glycolysis, suggesting an interplay between the two enzymes in the breakdown of triose phosphates. Indeed, MGS is strongly inhibited by phosphate concentrations that are close to the Km of phosphate serving as substrate for glyceraldehyde-3-phosphate dehydrogenase and is, therefore, inactive at normal intracellular conditions.<a class="footnoteRef" href="#fn48" id="fnref48"><sup>48</sup></a><a class="footnoteRef" href="#fn49" id="fnref49"><sup>49</sup></a> Triose phosphate catabolism switches over to MGS when phosphate concentrations are too low for glyceraldehyde-3-phosphate dehydrogenase activity.</p>

<p>In situations when glycolysis is restricted by phosphate starvation, the switch to MGS serves to release phosphate from glycolytic metabolites for glyceraldehyde-3-phosphate dehydrogenase and to produce methylglyoxal, which is converted to pyruvate via lactate with the uncoupling of <a href="ATP_synthesis" title="wikilink">ATP synthesis</a>.<a class="footnoteRef" href="#fn50" id="fnref50"><sup>50</sup></a><a class="footnoteRef" href="#fn51" id="fnref51"><sup>51</sup></a> This interplay between the two enzymes allows the cell to shift triose catabolism between the formation of <a href="1,3-Bisphosphoglyceric_acid" title="wikilink">1,3-biphosphoglycerate</a> and methylglyoxal based on available phosphates.</p>
<h2 id="other-applications">Other Applications</h2>

<p>For fuel <a href="Ethanol#Production" title="wikilink">ethanol production</a>, complete metabolism of complex combinations of sugars in <a href="E._coli" title="wikilink">E. coli</a> by synthetic biocatalysts is necessary. Deletion of the methylglyoxal synthase gene in E. coli increases <a class="uri" href="fermentation" title="wikilink">fermentation</a> rate of ethanogenic E. coli by promoting the co-metabolism of sugar mixtures containing the five principal sugars found in biomass (<a class="uri" href="glucose" title="wikilink">glucose</a>, <a class="uri" href="xylose" title="wikilink">xylose</a>, <a class="uri" href="arabinose" title="wikilink">arabinose</a>, <a class="uri" href="galactose" title="wikilink">galactose</a>, and <a class="uri" href="mannose" title="wikilink">mannose</a>).<a class="footnoteRef" href="#fn52" id="fnref52"><sup>52</sup></a> This suggests that MGS production of methylglyoxal plays a role in controlling expression of sugar-specific transporters and catabolic genes in native E.coli.</p>

<p>MGS also has industrial importance in the production of lactate, <a class="uri" href="hydroxyacetone" title="wikilink">hydroxyacetone</a> (acetol), and <a href="Propylene_glycol" title="wikilink">1,2-propandiol</a>.<a class="footnoteRef" href="#fn53" id="fnref53"><sup>53</sup></a><a class="footnoteRef" href="#fn54" id="fnref54"><sup>54</sup></a><a class="footnoteRef" href="#fn55" id="fnref55"><sup>55</sup></a> Introduction of the MGS gene in bacteria that natively lack MGS increased useful production of 1,2-propandiol by 141%.<a class="footnoteRef" href="#fn56" id="fnref56"><sup>56</sup></a></p>

<p>For biotech and synthetic applications, phosphate binding helps stabilize and protect the enzyme against cold- and heat-induced <a href="denaturation_(biochemistry)" title="wikilink">denaturation</a>.<a class="footnoteRef" href="#fn57" id="fnref57"><sup>57</sup></a> His-His interaction via the insertion of one histidine residue between Arg22 and His23 is also known to confer greater <a class="uri" href="thermostability" title="wikilink">thermostability</a> by increasing its <a href="Biological_half-life" title="wikilink">half-life</a> 4.6-fold.<a class="footnoteRef" href="#fn58" id="fnref58"><sup>58</sup></a></p>
<h2 id="references">References</h2>
<ul>
<li></li>
<li></li>
<li></li>
</ul>

<p>"</p>

<p><a href="Category:EC_4.2.3" title="wikilink">Category:EC 4.2.3</a> <a href="Category:Enzymes_of_known_structure" title="wikilink">Category:Enzymes of known structure</a></p>
<section class="footnotes">
<hr/>
<ol>
<li id="fn1"><a href="#fnref1">↩</a></li>
<li id="fn2"></li>
<li id="fn3"><a href="#fnref3">↩</a></li>
<li id="fn4"><a href="#fnref4">↩</a></li>
<li id="fn5"><a href="#fnref5">↩</a></li>
<li id="fn6"></li>
<li id="fn7"></li>
<li id="fn8"></li>
<li id="fn9"></li>
<li id="fn10"></li>
<li id="fn11"></li>
<li id="fn12"></li>
<li id="fn13"><a href="#fnref13">↩</a></li>
<li id="fn14"></li>
<li id="fn15"><a href="#fnref15">↩</a></li>
<li id="fn16"></li>
<li id="fn17"></li>
<li id="fn18"><a href="#fnref18">↩</a></li>
<li id="fn19"><a href="#fnref19">↩</a></li>
<li id="fn20"></li>
<li id="fn21"></li>
<li id="fn22"><a href="#fnref22">↩</a></li>
<li id="fn23"></li>
<li id="fn24"></li>
<li id="fn25"></li>
<li id="fn26"></li>
<li id="fn27"></li>
<li id="fn28"></li>
<li id="fn29"></li>
<li id="fn30"></li>
<li id="fn31"><a href="#fnref31">↩</a></li>
<li id="fn32"></li>
<li id="fn33"></li>
<li id="fn34"></li>
<li id="fn35"><a href="#fnref35">↩</a></li>
<li id="fn36"></li>
<li id="fn37"></li>
<li id="fn38"></li>
<li id="fn39"></li>
<li id="fn40"><a href="#fnref40">↩</a></li>
<li id="fn41"></li>
<li id="fn42"></li>
<li id="fn43"></li>
<li id="fn44"></li>
<li id="fn45"></li>
<li id="fn46"></li>
<li id="fn47"></li>
<li id="fn48"></li>
<li id="fn49"></li>
<li id="fn50"></li>
<li id="fn51"></li>
<li id="fn52"><a href="#fnref52">↩</a></li>
<li id="fn53"></li>
<li id="fn54"></li>
<li id="fn55"></li>
<li id="fn56"><a href="#fnref56">↩</a></li>
<li id="fn57"></li>
<li id="fn58"><a href="#fnref58">↩</a></li>
</ol>
</section>
</body>
</html>
