<html lang="en">
<head>
<meta charset="utf-8"/>
<title offset="603">Multiple EM for Motif Elicitation</title>
<script src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_SVG.js" type="text/javascript">
</script>
</head>
<body>
<h1>Multiple EM for Motif Elicitation</h1>
<hr/>

<p><strong>Multiple EM for Motif Elicitation</strong> or <strong>MEME</strong> is a tool for discovering motifs in a group of related <a class="uri" href="DNA" title="wikilink">DNA</a> or <a class="uri" href="protein" title="wikilink">protein</a> sequences.<a class="footnoteRef" href="#fn1" id="fnref1"><sup>1</sup></a></p>

<p>A <a href="Sequence_motif" title="wikilink">motif</a> is a sequence pattern that occurs repeatedly in a group of related protein or DNA sequences. MEME represents motifs as <a href="position_weight_matrix" title="wikilink">position-dependent letter-probability matrices</a> which describe the probability of each possible letter at each position in the pattern. Individual MEME motifs do not contain gaps. Patterns with variable-length gaps are split by MEME into two or more separate motifs.</p>

<p>MEME takes as input a group of DNA or protein sequences (the training set) and outputs as many motifs as requested. It uses statistical modeling techniques to automatically choose the best width, number of occurrences, and description for each motif.</p>

<p>MEME is the first of a collection of tools for analyzing motifs called the <a href="MEME_suite" title="wikilink">MEME suite</a>.</p>
<h2 id="definition">Definition</h2>

<p>What the MEME algorithms actually does can be understood from two different perspectives. From a biological point of view, MEME identifies and characterizes shared motifs in a set of unaligned sequences. From the computer science aspect, MEME finds a set of non-overlapping, approximately matching substrings given a starting set of strings.</p>
<h2 id="use">Use</h2>

<p>With MEME one can find similar biological functions and structures in different sequences. One has to take into account that the sequences variation can be significant and that the motifs are sometimes very small. It is also useful to take into account that the binding sites for proteins are very specific. This makes it easier to reduce wet-lab experiments (reduces costs and time). Indeed to better discover the motifs relevant from a biological point of view one has to carefully choose:</p>
<ul>
<li>The best width of motifs.</li>
<li>The number of occurrences in each sequence.</li>
<li>The composition of each motif.</li>
</ul>
<h2 id="algorithm-components">Algorithm Components</h2>

<p>The algorithm uses several types of well known functions:</p>
<ul>
<li><a href="Expectation_maximization" title="wikilink">Expectation maximization</a> (EM).</li>
<li>EM based heuristic for choosing the EM starting point.</li>
<li><a href="Maximum_likelihood" title="wikilink">Maximum likelihood</a> ratio based (LRT-based). Heuristic for determining the best number of model-free parameters.</li>
<li>Multi-start for searching over possible motif widths.</li>
<li><a href="Greedy_search" title="wikilink">Greedy search</a> for finding multiple motifs.</li>
</ul>

<p>However, one often doesn't know where the starting position is. Several possibilities exist:</p>
<ul>
<li>Exactly one motif per sequence.</li>
<li>One or zero motif per sequence.</li>
<li>Any number of motifs per sequence.</li>
</ul>
<h2 id="example">Example</h2>

<p>In the following example, one has a weight matrix of 3 different sequences, without gaps.</p>
<table>
<tbody>
<tr class="odd">
<td style="text-align: left;">
<p>1:</p></td>
<td style="text-align: left;">
<p>C G G G T A A G T</p></td>
</tr>
<tr class="even">
<td style="text-align: left;">
<p>2:</p></td>
<td style="text-align: left;">
<p>A A G G T A T G C</p></td>
</tr>
<tr class="odd">
<td style="text-align: left;">
<p>3:</p></td>
<td style="text-align: left;">
<p>C A G G T G A G G</p></td>
</tr>
</tbody>
</table>

<p>Now one counts the number of nucleotides contained in all sequences:</p>
<table>
<tbody>
<tr class="odd">
<td style="text-align: left;">
<p>A:</p></td>
<td style="text-align: left;">
<p>1 2 0 0 0 2 2 0 0</p></td>
<td style="text-align: left;">
<p>7</p></td>
</tr>
<tr class="even">
<td style="text-align: left;">
<p>C:</p></td>
<td style="text-align: left;">
<p>2 0 0 0 0 0 0 0 1</p></td>
<td style="text-align: left;">
<p>3</p></td>
</tr>
<tr class="odd">
<td style="text-align: left;">
<p>G:</p></td>
<td style="text-align: left;">
<p>0 1 3 3 0 1 0 3 1</p></td>
<td style="text-align: left;">
<p>12</p></td>
</tr>
<tr class="even">
<td style="text-align: left;">
<p>T:</p></td>
<td style="text-align: left;">
<p>0 0 0 0 3 0 1 0 1</p></td>
<td style="text-align: left;">
<p>5</p></td>
</tr>
</tbody>
</table>

<p>Now one needs to sum up the total: 7+3+12+5 = 27; this gives us a "dividing factor" for each base, or the equivalent probability of each nucleotides.</p>
<table>
<tbody>
<tr class="odd">
<td style="text-align: left;">
<p>A:</p></td>
<td style="text-align: left;">
<p>7/27 ≈ 0.26</p></td>
</tr>
<tr class="even">
<td style="text-align: left;">
<p>C:</p></td>
<td style="text-align: left;">
<p>3/27 ≈ 0.11</p></td>
</tr>
<tr class="odd">
<td style="text-align: left;">
<p>G:</p></td>
<td style="text-align: left;">
<p>12/27 ≈ 0.44</p></td>
</tr>
<tr class="even">
<td style="text-align: left;">
<p>T:</p></td>
<td style="text-align: left;">
<p>5/27 ≈ 0.19</p></td>
</tr>
</tbody>
</table>

<p>Now one can "redo" the weight matrix (WM) by dividing it by the total number of sequences (in our case 3):<br/>
A: 0.33 0.66 0.00 0.00 0.00 0.66 0.66 0.00 0.00<br/>
C: 0.66 0.00 0.00 0.00 0.00 0.00 0.00 0.00 0.33<br/>
G: 0.00 0.33 1.00 1.00 0.00 0.33 0.00 1.00 0.33<br/>
T: 0.00 0.00 0.00 0.00 1.00 0.00 0.33 0.00 0.33</p>

<p>Next, one divides the entries of the WM at position 

<math display="inline" id="Multiple_EM_for_Motif_Elicitation:0">
 <semantics>
  <msub>
   <mi>x</mi>
   <mi>i</mi>
  </msub>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <csymbol cd="ambiguous">subscript</csymbol>
    <ci>x</ci>
    <ci>i</ci>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   x_{i}
  </annotation>
 </semantics>
</math>

 with the probability of the base 

<math display="inline" id="Multiple_EM_for_Motif_Elicitation:1">
 <semantics>
  <mi>x</mi>
  <annotation-xml encoding="MathML-Content">
   <ci>x</ci>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   x
  </annotation>
 </semantics>
</math>

.<br/>
A: 1.29 2.57 0.00 0.00 0.00 2.57 2.57 0.00 0.00<br/>
C: 6.00 0.00 0.00 0.00 0.00 0.00 0.00 0.00 3.00<br/>
G: 0.00 0.75 2.25 2.25 0.00 0.75 0.00 2.25 0.75<br/>
T: 0.00 0.00 0.00 0.00 5.40 0.00 1.80 0.00 1.80</p>

<p>In general one would now multiply the probabilities. In our case one would have zero for every one. Due to this we define 

<math display="inline" id="Multiple_EM_for_Motif_Elicitation:2">
 <semantics>
  <mrow>
   <mrow>
    <msub>
     <mi>log</mi>
     <mn>10</mn>
    </msub>
    <mn>0</mn>
   </mrow>
   <mo>:=</mo>
   <mrow>
    <mo>-</mo>
    <mn>10</mn>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <csymbol cd="latexml">assign</csymbol>
    <apply>
     <apply>
      <csymbol cd="ambiguous">subscript</csymbol>
      <log></log>
      <cn type="integer">10</cn>
     </apply>
     <cn type="integer">0</cn>
    </apply>
    <apply>
     <minus></minus>
     <cn type="integer">10</cn>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \log_{10}0:=-10
  </annotation>
 </semantics>
</math>

 and take the (base 10) logarithm:</p>
<table>
<thead>
<tr class="header">
<th style="text-align: left;">
<p>A :</p></th>
<th style="text-align: left;">
<p>0.11</p></th>
<th style="text-align: left;">
<p>0.41</p></th>
<th style="text-align: left;">
<p>-10</p></th>
<th style="text-align: left;">
<p>-10</p></th>
<th style="text-align: left;">
<p>-10</p></th>
<th style="text-align: left;">
<p>0.41</p></th>
<th style="text-align: left;">
<p>0.41</p></th>
<th style="text-align: left;">
<p>-10</p></th>
<th style="text-align: left;">
<p>-10</p></th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;">
<p>C :</p></td>
<td style="text-align: left;">
<p>0.78</p></td>
<td style="text-align: left;">
<p>-10</p></td>
<td style="text-align: left;">
<p>-10</p></td>
<td style="text-align: left;">
<p>-10</p></td>
<td style="text-align: left;">
<p>-10</p></td>
<td style="text-align: left;">
<p>-10</p></td>
<td style="text-align: left;">
<p>-10</p></td>
<td style="text-align: left;">
<p>-10</p></td>
<td style="text-align: left;">
<p>0.48</p></td>
</tr>
<tr class="even">
<td style="text-align: left;">
<p>G :</p></td>
<td style="text-align: left;">
<p>-10</p></td>
<td style="text-align: left;">
<p>-0.12</p></td>
<td style="text-align: left;">
<p>0.35</p></td>
<td style="text-align: left;">
<p>0.35</p></td>
<td style="text-align: left;">
<p>-10</p></td>
<td style="text-align: left;">
<p>-0.12</p></td>
<td style="text-align: left;">
<p>-10</p></td>
<td style="text-align: left;">
<p>0.35</p></td>
<td style="text-align: left;">
<p>-0.12</p></td>
</tr>
<tr class="odd">
<td style="text-align: left;">
<p>T :</p></td>
<td style="text-align: left;">
<p>-10</p></td>
<td style="text-align: left;">
<p>-10</p></td>
<td style="text-align: left;">
<p>-10</p></td>
<td style="text-align: left;">
<p>-10</p></td>
<td style="text-align: left;">
<p>0.73</p></td>
<td style="text-align: left;">
<p>-10</p></td>
<td style="text-align: left;">
<p>0.26</p></td>
<td style="text-align: left;">
<p>-10</p></td>
<td style="text-align: left;">
<p>0.26</p></td>
</tr>
</tbody>
</table>

<p>This is our new weight matrix (WM). One is ready to use an example of a promoter sequence to determine its score. To do this, one has to add the numbers found at the position 

<math display="inline" id="Multiple_EM_for_Motif_Elicitation:3">
 <semantics>
  <msub>
   <mi>x</mi>
   <mi>i</mi>
  </msub>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <csymbol cd="ambiguous">subscript</csymbol>
    <ci>x</ci>
    <ci>i</ci>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   x_{i}
  </annotation>
 </semantics>
</math>

 of the logarithmic WM. For instance, if one takes the AGGCTGATC promoter:<br/>
0.11 - 0.12 + 0.35 - 10 + 0.73 - 0.12 + 0.41 - 10 + 0.48 = -18.17<br/>
This is then divided by the number of entries (in our case 9) yielding a score of -2.02.</p>
<h2 id="shortcomings">Shortcomings</h2>

<p>The MEME algorithms has several drawbacks including:</p>
<ul>
<li>Allowance for gaps/substitutions/insertions not included.</li>
<li>Ability to test significance often not included.</li>
<li>Erased input data each time a new motif is discovered (the algorithm assumes the new motif is correct).</li>
<li>Limitation to two component case.</li>
<li>Time complexity is high, scaling O(n^2). A faster implementation of MEME, known as EXTREME, uses the online EM algorithm to significantly speed up motif discovery.<a class="footnoteRef" href="#fn2" id="fnref2"><sup>2</sup></a></li>
</ul>
<ul>
<li>Very pessimistic about alignment (which might lead to missed signals).</li>
</ul>
<h2 id="see-also">See also</h2>
<ul>
<li><a href="Sequence_motif" title="wikilink">Sequence motif</a></li>
<li><a href="Sequence_alignment" title="wikilink">Sequence alignment</a></li>
</ul>
<h2 id="references">References</h2>
<h2 id="external-links">External links</h2>
<ul>
<li><a href="http://meme.sdsc.edu/meme/intro.html">The MEME Suite</a> — Motif-based sequence analysis tools</li>
<li><a href="http://www.nvidia.com/object/meme_on_tesla.html">GPU Accelerated version of MEME</a></li>
<li><a href="https://github.com/uci-cbcl/EXTREME">EXTREME</a> — An online EM implementation of the MEME model for fast motif discovery in large ChIP-Seq and DNase-Seq Footprinting data</li>
</ul>

<p>"</p>

<p><a class="uri" href="Category:Bioinformatics" title="wikilink">Category:Bioinformatics</a></p>
<section class="footnotes">
<hr/>
<ol>
<li id="fn1"><a href="#fnref1">↩</a></li>
<li id="fn2"><a href="#fnref2">↩</a></li>
</ol>
</section>
</body>
</html>
