<html lang="en">
<head>
<meta charset="utf-8"/>
<title offset="500">Nucleic acid thermodynamics</title>
<script src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_SVG.js" type="text/javascript">
</script>
</head>
<body>
<h1>Nucleic acid thermodynamics</h1>
<hr/>

<p><strong>Nucleic acid thermodynamics</strong> is the study of how <a class="uri" href="temperature" title="wikilink">temperature</a> affects the <a href="nucleic_acid_structure" title="wikilink">nucleic acid structure</a> of double-stranded <a class="uri" href="DNA" title="wikilink">DNA</a> (dsDNA). The melting temperature (<em>T<sub>m</sub></em>) is defined as the temperature at which half of the DNA strands are in the <a href="random_coil" title="wikilink">random coil</a> or single-stranded (ssDNA) state. <em>T<sub>m</sub></em> depends on the length of the DNA molecule and its specific <a class="uri" href="nucleotide" title="wikilink">nucleotide</a> sequence. DNA, when in a state where its two strands are dissociated (i.e., the dsDNA molecule exists as two independent strands), is referred to as having been <a href="denaturation_(biochemistry)" title="wikilink">denatured</a> by the high temperature.</p>
<h2 id="concepts">Concepts</h2>
<h3 id="hybridization">Hybridization</h3>

<p>Hybridization is the process of establishing a <a class="uri" href="non-covalent" title="wikilink">non-covalent</a>, sequence-specific interaction between two or more <a href="Complementarity_(molecular_biology)" title="wikilink">complementary</a> strands of <a href="nucleic_acid" title="wikilink">nucleic acids</a> into a single complex, which in the case of two strands is referred to as a <a href="Nucleic_acid_double_helix" title="wikilink">duplex</a>. <a href="Oligonucleotide" title="wikilink">Oligonucleotides</a>, <a class="uri" href="DNA" title="wikilink">DNA</a>, or <a class="uri" href="RNA" title="wikilink">RNA</a> will bind to their complement under normal conditions, so two perfectly complementary strands will bind to each other readily. In order to reduce the diversity and obtain the most energetically preferred complexes, a technique called <a href="Annealing_(biology)#Annealing" title="wikilink">annealing</a> is used in laboratory practice. However, due to the different molecular geometries of the nucleotides, a single inconsistency between the two strands will make binding between them less energetically favorable. Measuring the effects of base incompatibility by quantifying the temperature at which two strands anneal can provide information as to the similarity in base sequence between the two strands being annealed. The complexes may be dissociated by thermal <a href="denaturation_(biochemistry)" title="wikilink">denaturation</a>, also referred to as melting. In the absence of external negative factors, the processes of hybridization and melting may be repeated in succession indefinitely, which lays the ground for <a href="polymerase_chain_reaction" title="wikilink">polymerase chain reaction</a>. Most commonly, the pairs of nucleic bases A=T and G≡C are formed, of which the latter is more stable.</p>
<h3 id="denaturation">Denaturation</h3>

<p>DNA denaturation, also called DNA melting, is the process by which double-stranded <a href="deoxyribonucleic_acid" title="wikilink">deoxyribonucleic acid</a> unwinds and separates into single-stranded strands through the breaking of <a class="uri" href="hydrophobic" title="wikilink">hydrophobic</a> stacking attractions between the bases. See <a href="Hydrophobic_effect" title="wikilink">Hydrophobic effect</a>. Both terms are used to refer to the process as it occurs when a mixture is heated, although "denaturation" can also refer to the separation of DNA strands induced by chemicals like <a class="uri" href="formamide" title="wikilink">formamide</a> or <a class="uri" href="urea" title="wikilink">urea</a>.</p>

<p>The process of DNA denaturation can be used to analyze some aspects of DNA. Because cytosine / guanine base-pairing is generally stronger than adenosine / thymine base-pairing, the amount of cytosine and guanine in a genome (called the "<a href="GC_content" title="wikilink">GC content</a>") can be estimated by measuring the temperature at which the genomic DNA melts.<a class="footnoteRef" href="#fn1" id="fnref1"><sup>1</sup></a> Higher temperatures are associated with high GC content.</p>

<p>DNA denaturation can also be used to detect sequence differences between two different DNA sequences. DNA is heated and denatured into single-stranded state, and the mixture is cooled to allow strands to rehybridize. Hybrid molecules are formed between similar sequences and any differences between those sequences will result in a disruption of the base-pairing. On a genomic scale, the method has been used by researchers to estimate the <a href="genetic_distance" title="wikilink">genetic distance</a> between two species, a process known as <a href="DNA-DNA_hybridization" title="wikilink">DNA-DNA hybridization</a>.<a class="footnoteRef" href="#fn2" id="fnref2"><sup>2</sup></a> In the context of a single isolated region of DNA, denaturing gradient gels and temperature gradient gels can be used to detect the presence of small mismatches between two sequences, a process known as <a href="temperature_gradient_gel_electrophoresis" title="wikilink">temperature gradient gel electrophoresis</a>.<a class="footnoteRef" href="#fn3" id="fnref3"><sup>3</sup></a><a class="footnoteRef" href="#fn4" id="fnref4"><sup>4</sup></a></p>

<p>Methods of DNA analysis based on melting temperature have the disadvantage of being proxies for studying the underlying sequence; <a href="DNA_sequencing" title="wikilink">DNA sequencing</a> is generally considered a more accurate method.</p>

<p>The process of DNA melting is also used in molecular biology techniques, notably in the <a href="polymerase_chain_reaction" title="wikilink">polymerase chain reaction</a>. Although the temperature of DNA melting is not diagnostic in the technique, methods for estimating <em>T<sub>m</sub></em> are important for determining the appropriate temperatures to use in a protocol. DNA melting temperatures can also be used as a proxy for equalizing the hybridization strengths of a set of molecules, e.g. the oligonucleotide probes of <a href="DNA_microarray" title="wikilink">DNA microarrays</a>.</p>
<h3 id="annealing">Annealing</h3>

<p>Annealing, in <a class="uri" href="genetics" title="wikilink">genetics</a>, means for <a href="complementarity_(molecular_biology)" title="wikilink">complementary sequences</a> of single-stranded <a class="uri" href="DNA" title="wikilink">DNA</a> or <a class="uri" href="RNA" title="wikilink">RNA</a> to pair by <a href="hydrogen_bond" title="wikilink">hydrogen bonds</a> to form a double-stranded <a href="nucleotide" title="wikilink">polynucleotide</a>. The term is often used to describe the binding of a <a href="DNA_probe" title="wikilink">DNA probe</a>, or the binding of a <a href="primer_(molecular_biology)" title="wikilink">primer</a> to a DNA strand during a <a href="polymerase_chain_reaction" title="wikilink">polymerase chain reaction</a>. The term is also often used to describe the reformation (<a href="Denaturation_(biochemistry)#Nucleic_acid_denaturation" title="wikilink">renaturation</a>) of reverse-complementary strands that were separated by heat (thermally denatured). Proteins such as <a class="uri" href="RAD52" title="wikilink">RAD52</a> can help DNA anneal.</p>
<h2 id="thermodynamics-of-the-two-state-model">Thermodynamics of the two-state model</h2>

<p>Several formulas are used to calculate <em>T<sub>m</sub></em> values.<a class="footnoteRef" href="#fn5" id="fnref5"><sup>5</sup></a><a class="footnoteRef" href="#fn6" id="fnref6"><sup>6</sup></a> Some formulas are more accurate in predicting melting temperatures of DNA duplexes.<a class="footnoteRef" href="#fn7" id="fnref7"><sup>7</sup></a> For DNA oligonucleotides, i.e. short sequences of DNA, the thermodynamics of hybridization can be accurately described as a two-state process. In this approximation one neglects the possibility of intermediate partial binding states in the formation of a double strand state from two single stranded oligonucleotides. Under this assumption one can elegantly describe the thermodynamic parameters for forming double-stranded nucleic acid AB from single-stranded nucleic acids A and B.</p>
<dl>
<dd>AB ↔ A + B
</dd>
</dl>

<p>The equilibrium constant for this reaction is 

<math display="inline" id="Nucleic_acid_thermodynamics:0">
 <semantics>
  <mrow>
   <mi>K</mi>
   <mo>=</mo>
   <mfrac>
    <mrow>
     <mrow>
      <mo stretchy="false">[</mo>
      <mi>A</mi>
      <mo stretchy="false">]</mo>
     </mrow>
     <mrow>
      <mo stretchy="false">[</mo>
      <mi>B</mi>
      <mo stretchy="false">]</mo>
     </mrow>
    </mrow>
    <mrow>
     <mo stretchy="false">[</mo>
     <mrow>
      <mi>A</mi>
      <mi>B</mi>
     </mrow>
     <mo stretchy="false">]</mo>
    </mrow>
   </mfrac>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <eq></eq>
    <ci>K</ci>
    <apply>
     <divide></divide>
     <apply>
      <times></times>
      <apply>
       <csymbol cd="latexml">delimited-[]</csymbol>
       <ci>A</ci>
      </apply>
      <apply>
       <csymbol cd="latexml">delimited-[]</csymbol>
       <ci>B</ci>
      </apply>
     </apply>
     <apply>
      <csymbol cd="latexml">delimited-[]</csymbol>
      <apply>
       <times></times>
       <ci>A</ci>
       <ci>B</ci>
      </apply>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   K=\frac{[A][B]}{[AB]}
  </annotation>
 </semantics>
</math>

. According to the Van´t Hoff equation, the relation between free energy, Δ<em>G</em>, and <em>K</em> is Δ<em>G°</em> = -<em>RT</em>ln <em>K</em>, where <em>R</em> is the ideal gas law constant, and <em>T</em> is the kelvin temperature of the reaction. This gives, for the nucleic acid system,</p>

<p>

<math display="inline" id="Nucleic_acid_thermodynamics:1">
 <semantics>
  <mrow>
   <mrow>
    <mi mathvariant="normal">Δ</mi>
    <msup>
     <mi>G</mi>
     <mo>∘</mo>
    </msup>
   </mrow>
   <mo>=</mo>
   <mrow>
    <mo>-</mo>
    <mrow>
     <mi>R</mi>
     <mi>T</mi>
     <mrow>
      <mi>ln</mi>
      <mfrac>
       <mrow>
        <mrow>
         <mo stretchy="false">[</mo>
         <mi>A</mi>
         <mo stretchy="false">]</mo>
        </mrow>
        <mrow>
         <mo stretchy="false">[</mo>
         <mi>B</mi>
         <mo stretchy="false">]</mo>
        </mrow>
       </mrow>
       <mrow>
        <mo stretchy="false">[</mo>
        <mrow>
         <mi>A</mi>
         <mi>B</mi>
        </mrow>
        <mo stretchy="false">]</mo>
       </mrow>
      </mfrac>
     </mrow>
    </mrow>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <eq></eq>
    <apply>
     <times></times>
     <ci>normal-Δ</ci>
     <apply>
      <csymbol cd="ambiguous">superscript</csymbol>
      <ci>G</ci>
      <compose></compose>
     </apply>
    </apply>
    <apply>
     <minus></minus>
     <apply>
      <times></times>
      <ci>R</ci>
      <ci>T</ci>
      <apply>
       <ln></ln>
       <apply>
        <divide></divide>
        <apply>
         <times></times>
         <apply>
          <csymbol cd="latexml">delimited-[]</csymbol>
          <ci>A</ci>
         </apply>
         <apply>
          <csymbol cd="latexml">delimited-[]</csymbol>
          <ci>B</ci>
         </apply>
        </apply>
        <apply>
         <csymbol cd="latexml">delimited-[]</csymbol>
         <apply>
          <times></times>
          <ci>A</ci>
          <ci>B</ci>
         </apply>
        </apply>
       </apply>
      </apply>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \Delta G^{\circ}=-RT\ln\frac{[A][B]}{[AB]}
  </annotation>
 </semantics>
</math>

.</p>

<p>The melting temperature, <em>T</em><sub>m</sub>, occurs when half of the double-stranded nucleic acid has dissociated. If no additional nucleic acids are present, then [A], [B], and [AB] will be equal, and equal to half the initial concentration of double-stranded nucleic acid, [AB]<sub>initial</sub>. This gives an expression for the melting point of a nucleic acid duplex of</p>

<p>

<math display="inline" id="Nucleic_acid_thermodynamics:2">
 <semantics>
  <mrow>
   <msub>
    <mi>T</mi>
    <mi>m</mi>
   </msub>
   <mo>=</mo>
   <mrow>
    <mo>-</mo>
    <mfrac>
     <mrow>
      <mi mathvariant="normal">Δ</mi>
      <msup>
       <mi>G</mi>
       <mo>∘</mo>
      </msup>
     </mrow>
     <mrow>
      <mi>R</mi>
      <mrow>
       <mi>ln</mi>
       <mfrac>
        <msub>
         <mrow>
          <mo stretchy="false">[</mo>
          <mrow>
           <mi>A</mi>
           <mi>B</mi>
          </mrow>
          <mo stretchy="false">]</mo>
         </mrow>
         <mrow>
          <mi>i</mi>
          <mi>n</mi>
          <mi>i</mi>
          <mi>t</mi>
          <mi>i</mi>
          <mi>a</mi>
          <mi>l</mi>
         </mrow>
        </msub>
        <mn>2</mn>
       </mfrac>
      </mrow>
     </mrow>
    </mfrac>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <eq></eq>
    <apply>
     <csymbol cd="ambiguous">subscript</csymbol>
     <ci>T</ci>
     <ci>m</ci>
    </apply>
    <apply>
     <minus></minus>
     <apply>
      <divide></divide>
      <apply>
       <times></times>
       <ci>normal-Δ</ci>
       <apply>
        <csymbol cd="ambiguous">superscript</csymbol>
        <ci>G</ci>
        <compose></compose>
       </apply>
      </apply>
      <apply>
       <times></times>
       <ci>R</ci>
       <apply>
        <ln></ln>
        <apply>
         <divide></divide>
         <apply>
          <csymbol cd="ambiguous">subscript</csymbol>
          <apply>
           <csymbol cd="latexml">delimited-[]</csymbol>
           <apply>
            <times></times>
            <ci>A</ci>
            <ci>B</ci>
           </apply>
          </apply>
          <apply>
           <times></times>
           <ci>i</ci>
           <ci>n</ci>
           <ci>i</ci>
           <ci>t</ci>
           <ci>i</ci>
           <ci>a</ci>
           <ci>l</ci>
          </apply>
         </apply>
         <cn type="integer">2</cn>
        </apply>
       </apply>
      </apply>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   T_{m}=-\frac{\Delta G^{\circ}}{R\ln\frac{[AB]_{initial}}{2}}
  </annotation>
 </semantics>
</math>

.</p>

<p>Because Δ<em>G</em>° = Δ<em>H</em>° -<em>T</em>Δ<em>S</em>°, <em>T</em><sub>m</sub> is also given by</p>

<p>

<math display="inline" id="Nucleic_acid_thermodynamics:3">
 <semantics>
  <mrow>
   <msub>
    <mi>T</mi>
    <mi>m</mi>
   </msub>
   <mo>=</mo>
   <mfrac>
    <mrow>
     <mi mathvariant="normal">Δ</mi>
     <msup>
      <mi>H</mi>
      <mo>∘</mo>
     </msup>
    </mrow>
    <mrow>
     <mrow>
      <mi mathvariant="normal">Δ</mi>
      <msup>
       <mi>S</mi>
       <mo>∘</mo>
      </msup>
     </mrow>
     <mo>-</mo>
     <mrow>
      <mi>R</mi>
      <mrow>
       <mi>ln</mi>
       <mfrac>
        <msub>
         <mrow>
          <mo stretchy="false">[</mo>
          <mrow>
           <mi>A</mi>
           <mi>B</mi>
          </mrow>
          <mo stretchy="false">]</mo>
         </mrow>
         <mrow>
          <mi>i</mi>
          <mi>n</mi>
          <mi>i</mi>
          <mi>t</mi>
          <mi>i</mi>
          <mi>a</mi>
          <mi>l</mi>
         </mrow>
        </msub>
        <mn>2</mn>
       </mfrac>
      </mrow>
     </mrow>
    </mrow>
   </mfrac>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <eq></eq>
    <apply>
     <csymbol cd="ambiguous">subscript</csymbol>
     <ci>T</ci>
     <ci>m</ci>
    </apply>
    <apply>
     <divide></divide>
     <apply>
      <times></times>
      <ci>normal-Δ</ci>
      <apply>
       <csymbol cd="ambiguous">superscript</csymbol>
       <ci>H</ci>
       <compose></compose>
      </apply>
     </apply>
     <apply>
      <minus></minus>
      <apply>
       <times></times>
       <ci>normal-Δ</ci>
       <apply>
        <csymbol cd="ambiguous">superscript</csymbol>
        <ci>S</ci>
        <compose></compose>
       </apply>
      </apply>
      <apply>
       <times></times>
       <ci>R</ci>
       <apply>
        <ln></ln>
        <apply>
         <divide></divide>
         <apply>
          <csymbol cd="ambiguous">subscript</csymbol>
          <apply>
           <csymbol cd="latexml">delimited-[]</csymbol>
           <apply>
            <times></times>
            <ci>A</ci>
            <ci>B</ci>
           </apply>
          </apply>
          <apply>
           <times></times>
           <ci>i</ci>
           <ci>n</ci>
           <ci>i</ci>
           <ci>t</ci>
           <ci>i</ci>
           <ci>a</ci>
           <ci>l</ci>
          </apply>
         </apply>
         <cn type="integer">2</cn>
        </apply>
       </apply>
      </apply>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   T_{m}=\frac{\Delta H^{\circ}}{\Delta S^{\circ}-R\ln\frac{[AB]_{initial}}{2}}
  </annotation>
 </semantics>
</math>

.</p>

<p>The terms Δ<em>H</em>° and Δ<em>S</em>° are usually given for the association and not the dissociation reaction (see the nearest-neighbor method for example). This formula then turns into:<a class="footnoteRef" href="#fn8" id="fnref8"><sup>8</sup></a> 

<math display="inline" id="Nucleic_acid_thermodynamics:4">
 <semantics>
  <mrow>
   <msub>
    <mi>T</mi>
    <mi>m</mi>
   </msub>
   <mo>=</mo>
   <mfrac>
    <mrow>
     <mi mathvariant="normal">Δ</mi>
     <msup>
      <mi>H</mi>
      <mo>∘</mo>
     </msup>
    </mrow>
    <mrow>
     <mrow>
      <mi mathvariant="normal">Δ</mi>
      <msup>
       <mi>S</mi>
       <mo>∘</mo>
      </msup>
     </mrow>
     <mo>+</mo>
     <mrow>
      <mi>R</mi>
      <mrow>
       <mi>ln</mi>
       <mrow>
        <mo stretchy="false">(</mo>
        <mrow>
         <msub>
          <mrow>
           <mo stretchy="false">[</mo>
           <mi>A</mi>
           <mo stretchy="false">]</mo>
          </mrow>
          <mrow>
           <mi>t</mi>
           <mi>o</mi>
           <mi>t</mi>
           <mi>a</mi>
           <mi>l</mi>
          </mrow>
         </msub>
         <mo>-</mo>
         <mrow>
          <msub>
           <mrow>
            <mo stretchy="false">[</mo>
            <mi>B</mi>
            <mo stretchy="false">]</mo>
           </mrow>
           <mrow>
            <mi>t</mi>
            <mi>o</mi>
            <mi>t</mi>
            <mi>a</mi>
            <mi>l</mi>
           </mrow>
          </msub>
          <mo>/</mo>
          <mn>2</mn>
         </mrow>
        </mrow>
        <mo stretchy="false">)</mo>
       </mrow>
      </mrow>
     </mrow>
    </mrow>
   </mfrac>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <eq></eq>
    <apply>
     <csymbol cd="ambiguous">subscript</csymbol>
     <ci>T</ci>
     <ci>m</ci>
    </apply>
    <apply>
     <divide></divide>
     <apply>
      <times></times>
      <ci>normal-Δ</ci>
      <apply>
       <csymbol cd="ambiguous">superscript</csymbol>
       <ci>H</ci>
       <compose></compose>
      </apply>
     </apply>
     <apply>
      <plus></plus>
      <apply>
       <times></times>
       <ci>normal-Δ</ci>
       <apply>
        <csymbol cd="ambiguous">superscript</csymbol>
        <ci>S</ci>
        <compose></compose>
       </apply>
      </apply>
      <apply>
       <times></times>
       <ci>R</ci>
       <apply>
        <ln></ln>
        <apply>
         <minus></minus>
         <apply>
          <csymbol cd="ambiguous">subscript</csymbol>
          <apply>
           <csymbol cd="latexml">delimited-[]</csymbol>
           <ci>A</ci>
          </apply>
          <apply>
           <times></times>
           <ci>t</ci>
           <ci>o</ci>
           <ci>t</ci>
           <ci>a</ci>
           <ci>l</ci>
          </apply>
         </apply>
         <apply>
          <divide></divide>
          <apply>
           <csymbol cd="ambiguous">subscript</csymbol>
           <apply>
            <csymbol cd="latexml">delimited-[]</csymbol>
            <ci>B</ci>
           </apply>
           <apply>
            <times></times>
            <ci>t</ci>
            <ci>o</ci>
            <ci>t</ci>
            <ci>a</ci>
            <ci>l</ci>
           </apply>
          </apply>
          <cn type="integer">2</cn>
         </apply>
        </apply>
       </apply>
      </apply>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   T_{m}=\frac{\Delta H^{\circ}}{\Delta S^{\circ}+R\ln([A]_{total}-[B]_{total}/2)}
  </annotation>
 </semantics>
</math>

, where [B]<sub>total</sub> total.</p>

<p>As mentioned, this equation is based on the assumption that only two states are involved in melting: the double stranded state and the random-coil state. However, nucleic acids may melt via several intermediate states. To account for such complicated behavior, the methods of <a href="statistical_mechanics" title="wikilink">statistical mechanics</a> must be used, which is especially relevant for long sequences.</p>
<h2 id="estimating-thermodynamic-properties-from-nucleic-acid-sequence">Estimating thermodynamic properties from nucleic acid sequence</h2>

<p>The previous paragraph shows how melting temperature and thermodynamic parameters (Δ<em>G</em>° or Δ<em>H</em>° &amp; Δ<em>S</em>°) are related to each other. From the observation of melting temperatures one can experimentally determine the thermodynamic parameters. Vice versa, and important for applications, when the thermodynamic parameters of a given nucleic acid sequence are known, the melting temperature can be predicted. It turns out that for oligonucleotides, these parameters can be well approximated by the nearest-neighbor model.</p>
<h3 id="nearest-neighbor-method">Nearest-neighbor method</h3>

<p>The interaction between bases on different strands depends somewhat on the neighboring bases. Instead of treating a DNA helix as a string of interactions between base pairs, the nearest-neighbor model treats a DNA helix as a string of interactions between 'neighboring' base pairs.<a class="footnoteRef" href="#fn9" id="fnref9"><sup>9</sup></a> So, for example, the DNA shown below has nearest-neighbor interactions indicated by the arrows.</p>
<dl>
<dd><code>    ↓↓↓↓↓</code>
</dd>
<dd><code>5' C-G-T-T-G-A 3'</code>
</dd>
<dd><code>3' G-C-A-A-C-T 5'</code>
</dd>
</dl>

<p>The free energy of forming this DNA from the individual strands, Δ<em>G</em>°, is represented (at 37 °C) as</p>

<p>Δ<em>G</em>°<sub>37</sub>(predicted) = Δ<em>G</em>°<sub>37</sub>(CG initiation) + Δ<em>G</em>°<sub>37</sub>(CG/GC) + Δ<em>G</em>°<sub>37</sub>(GT/CA) + Δ<em>G</em>°<sub>37</sub>(TT/AA) + Δ<em>G</em>°<sub>37</sub>(TG/AC) + Δ<em>G</em>°<sub>37</sub>(GA/CT) + Δ<em>G</em>°<sub>37</sub>(AT initiation)</p>

<p>The first term represents the free energy of the first base pair, CG, in the absence of a nearest neighbor. The second term includes both the free energy of formation of the second base pair, GC, and stacking interaction between this base pair and the previous base pair. The remaining terms are similarly defined. In general, the free energy of forming a nucleic acid duplex is</p>

<p>

<math display="inline" id="Nucleic_acid_thermodynamics:5">
 <semantics>
  <mrow>
   <mrow>
    <mi mathvariant="normal">Δ</mi>
    <msubsup>
     <mi>G</mi>
     <mn>37</mn>
     <mo>∘</mo>
    </msubsup>
    <mrow>
     <mo stretchy="false">(</mo>
     <mi>total</mi>
     <mo stretchy="false">)</mo>
    </mrow>
   </mrow>
   <mo>=</mo>
   <mrow>
    <mrow>
     <mi mathvariant="normal">Δ</mi>
     <msubsup>
      <mi>G</mi>
      <mn>37</mn>
      <mo>∘</mo>
     </msubsup>
     <mrow>
      <mo stretchy="false">(</mo>
      <mi>initiations</mi>
      <mo stretchy="false">)</mo>
     </mrow>
    </mrow>
    <mo>+</mo>
    <mrow>
     <msubsup>
      <mo largeop="true" symmetric="true">∑</mo>
      <mrow>
       <mi>i</mi>
       <mo>=</mo>
       <mn>1</mn>
      </mrow>
      <mn>10</mn>
     </msubsup>
     <mrow>
      <msub>
       <mi>n</mi>
       <mi>i</mi>
      </msub>
      <mi mathvariant="normal">Δ</mi>
      <msubsup>
       <mi>G</mi>
       <mn>37</mn>
       <mo>∘</mo>
      </msubsup>
      <mrow>
       <mo stretchy="false">(</mo>
       <mi>i</mi>
       <mo stretchy="false">)</mo>
      </mrow>
     </mrow>
    </mrow>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <eq></eq>
    <apply>
     <times></times>
     <ci>normal-Δ</ci>
     <apply>
      <csymbol cd="ambiguous">superscript</csymbol>
      <apply>
       <csymbol cd="ambiguous">subscript</csymbol>
       <ci>G</ci>
       <cn type="integer">37</cn>
      </apply>
      <compose></compose>
     </apply>
     <ci>total</ci>
    </apply>
    <apply>
     <plus></plus>
     <apply>
      <times></times>
      <ci>normal-Δ</ci>
      <apply>
       <csymbol cd="ambiguous">superscript</csymbol>
       <apply>
        <csymbol cd="ambiguous">subscript</csymbol>
        <ci>G</ci>
        <cn type="integer">37</cn>
       </apply>
       <compose></compose>
      </apply>
      <ci>initiations</ci>
     </apply>
     <apply>
      <apply>
       <csymbol cd="ambiguous">superscript</csymbol>
       <apply>
        <csymbol cd="ambiguous">subscript</csymbol>
        <sum></sum>
        <apply>
         <eq></eq>
         <ci>i</ci>
         <cn type="integer">1</cn>
        </apply>
       </apply>
       <cn type="integer">10</cn>
      </apply>
      <apply>
       <times></times>
       <apply>
        <csymbol cd="ambiguous">subscript</csymbol>
        <ci>n</ci>
        <ci>i</ci>
       </apply>
       <ci>normal-Δ</ci>
       <apply>
        <csymbol cd="ambiguous">superscript</csymbol>
        <apply>
         <csymbol cd="ambiguous">subscript</csymbol>
         <ci>G</ci>
         <cn type="integer">37</cn>
        </apply>
        <compose></compose>
       </apply>
       <ci>i</ci>
      </apply>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \Delta G_{37}^{\circ}(\mathrm{total})=\Delta G_{37}^{\circ}(\mathrm{%
initiations})+\sum_{i=1}^{10}n_{i}\Delta G_{37}^{\circ}(i)
  </annotation>
 </semantics>
</math>

.</p>

<p>Each Δ<em>G</em>° term has enthalpic, Δ<em>H</em>°, and entropic, Δ<em>S</em>°, parameters, so the change in free energy is also given by</p>

<p>

<math display="inline" id="Nucleic_acid_thermodynamics:6">
 <semantics>
  <mrow>
   <mrow>
    <mi mathvariant="normal">Δ</mi>
    <msup>
     <mi>G</mi>
     <mo>∘</mo>
    </msup>
    <mrow>
     <mo stretchy="false">(</mo>
     <mi>total</mi>
     <mo stretchy="false">)</mo>
    </mrow>
   </mrow>
   <mo>=</mo>
   <mrow>
    <mrow>
     <mi mathvariant="normal">Δ</mi>
     <msubsup>
      <mi>H</mi>
      <mi>total</mi>
      <mo>∘</mo>
     </msubsup>
    </mrow>
    <mo>-</mo>
    <mrow>
     <mi>T</mi>
     <mi mathvariant="normal">Δ</mi>
     <msubsup>
      <mi>S</mi>
      <mi>total</mi>
      <mo>∘</mo>
     </msubsup>
    </mrow>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <eq></eq>
    <apply>
     <times></times>
     <ci>normal-Δ</ci>
     <apply>
      <csymbol cd="ambiguous">superscript</csymbol>
      <ci>G</ci>
      <compose></compose>
     </apply>
     <ci>total</ci>
    </apply>
    <apply>
     <minus></minus>
     <apply>
      <times></times>
      <ci>normal-Δ</ci>
      <apply>
       <csymbol cd="ambiguous">superscript</csymbol>
       <apply>
        <csymbol cd="ambiguous">subscript</csymbol>
        <ci>H</ci>
        <ci>total</ci>
       </apply>
       <compose></compose>
      </apply>
     </apply>
     <apply>
      <times></times>
      <ci>T</ci>
      <ci>normal-Δ</ci>
      <apply>
       <csymbol cd="ambiguous">superscript</csymbol>
       <apply>
        <csymbol cd="ambiguous">subscript</csymbol>
        <ci>S</ci>
        <ci>total</ci>
       </apply>
       <compose></compose>
      </apply>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \Delta G^{\circ}(\mathrm{total})=\Delta H_{\mathrm{total}}^{\circ}-T\Delta S_{%
\mathrm{total}}^{\circ}
  </annotation>
 </semantics>
</math>

.</p>

<p>Values of Δ<em>H</em>° and Δ<em>S</em>° have been determined for the ten possible pairs of interactions. These are given in Table 1, along with the value of Δ<em>G</em>° calculated at 37 °C. Using these values, the value of Δ<em>G</em><sub>37</sub>° for the DNA helix shown above is calculated to be −22.4 kJ/mol. The experimental value is −21.8 kJ/mol.</p>
<table>
<tbody>
<tr class="odd">
<td style="text-align: left;">
<p>Table 1. Nearest-neighbor parameters for DNA/DNA duplexes in 1 M NaCl.<a class="footnoteRef" href="#fn10" id="fnref10"><sup>10</sup></a></p></td>
</tr>
<tr class="even">
<td style="text-align: left;">
<p>Nearest-neighbor sequence<br/>
 (5'-3'/3'-5')</p></td>
</tr>
<tr class="odd">
<td style="text-align: left;">
<p>AA/TT</p></td>
</tr>
<tr class="even">
<td style="text-align: left;">
<p>AT/TA</p></td>
</tr>
<tr class="odd">
<td style="text-align: left;">
<p>TA/AT</p></td>
</tr>
<tr class="even">
<td style="text-align: left;">
<p>CA/GT</p></td>
</tr>
<tr class="odd">
<td style="text-align: left;">
<p>GT/CA</p></td>
</tr>
<tr class="even">
<td style="text-align: left;">
<p>CT/GA</p></td>
</tr>
<tr class="odd">
<td style="text-align: left;">
<p>GA/CT</p></td>
</tr>
<tr class="even">
<td style="text-align: left;">
<p>CG/GC</p></td>
</tr>
<tr class="odd">
<td style="text-align: left;">
<p>GC/CG</p></td>
</tr>
<tr class="even">
<td style="text-align: left;">
<p>GG/CC</p></td>
</tr>
<tr class="odd">
<td style="text-align: left;">
<p>Terminal A-T base pair</p></td>
</tr>
<tr class="even">
<td style="text-align: left;">
<p>Terminal G-C base pair</p></td>
</tr>
</tbody>
</table>

<p>The parameters associated with the ten groups of neighbors shown in table 1 are determined from melting points of short oligonucleotide duplexes. Curiously, it works out that only eight of the ten groups are independent.</p>

<p>The nearest-neighbor model can be extended beyond the Watson-Crick pairs to include parameters for interactions between mismatches and neighboring base pairs.<a class="footnoteRef" href="#fn11" id="fnref11"><sup>11</sup></a> This allows the estimation of the thermodynamic parameters of sequences containing isolated mismatches, like e.g. (arrows indicating mismatch)</p>
<dl>
<dd><code>           ↓↓↓</code>
</dd>
<dd><code>5' G-G-A-C-T-G-A-C-G 3'</code>
</dd>
<dd><code>3' C-C-T-G-G-C-T-G-C 5'</code>
</dd>
</dl>

<p>These parameters have been fitted from melting experiments and an extension of Table 1 which includes mismatches can be found in literature.</p>

<p>A more realistic way of modeling the behavior of nucleic acids would seem to be to have parameters that depend on the neighboring groups on both sides of a nucleotide, giving a table with entries like "TCG/AGC". However, this would involve around 32 groups for Watson-Crick pairing and even more for sequences containing mismatches; the number of DNA melting experiments needed to get reliable data for so many groups would be inconveniently high. However, other means exist to access thermodynamic parameters of nucleic acids: microarray technology allows hybridization monitoring of tens of thousands sequences in parallel. This data, in combination with molecular adsorption theory allows the determination of many thermodynamic parameters in a single experiment<a class="footnoteRef" href="#fn12" id="fnref12"><sup>12</sup></a> and to go beyond the nearest neighbor model.<a class="footnoteRef" href="#fn13" id="fnref13"><sup>13</sup></a> In general the predictions from the nearest neighbor method agree reasonably well with experimental results, but some unexpected outlying sequences, calling for further insights, do exist.<a class="footnoteRef" href="#fn14" id="fnref14"><sup>14</sup></a></p>
<h2 id="see-also">See also</h2>
<ul>
<li><a href="Melting_point" title="wikilink">Melting point</a></li>
<li><a href="Primer_(molecular_biology)" title="wikilink">Primer (molecular biology)</a> for calculations of <em>T<sub>m</sub></em></li>
<li><a href="Base_pair" title="wikilink">Base pair</a></li>
<li><a href="Complementary_DNA" title="wikilink">Complementary DNA</a></li>
<li><a href="Western_blot" title="wikilink">Western blot</a></li>
</ul>
<h2 id="references">References</h2>
<h2 id="external-links">External links</h2>
<ul>
<li><a href="http://www.idtdna.com/analyzer/Applications/OligoAnalyzer/">T<sub>m</sub> calculations in OligoAnalyzer</a> – <a href="Integrated_DNA_Technologies" title="wikilink">Integrated DNA Technologies</a></li>
<li><a href="http://biophysics.idtdna.com">DNA thermodynamics calculations – T<sub>m</sub>, melting profile, mismatches, free energy calculations</a></li>
<li><a href="http://www.biophp.org/minitools/melting_temperature/demo.php">T<sub>m</sub> calculation</a> – by bioPHP.org.</li>
<li><a class="uri" href="http://www.promega.com/biomath/calc11.htm#disc">http://www.promega.com/biomath/calc11.htm#disc</a></li>
<li><a href="http://www.invitrogen.com/content/sfs/appendix/PCR_RTPCR/Primer%20Tm%20Calculations.pdf">Invitrogen T<sub>m</sub> calculation</a></li>
<li><a href="http://www.bioinformatics.org/annhyb">AnnHyb Open Source software for T<sub>m</sub> calculation using the Nearest-neighbour method</a></li>
<li><a href="http://www.sigmaaldrich.com/Brands/Sigma_Genosys/Custom_DNA/Key_Resources/Oligos_Melting_Temp.html">Sigma-aldrich technical notes</a></li>
<li><a href="http://fokker.wi.mit.edu/primer3/input-help.htm#PRIMER_TM">Primer3 calculation</a></li>
<li><a href="http://www.jbc.org/cgi/content/full/281/12/7693">"Discovery of the Hybrid Helix and the First DNA-RNA Hybridization"</a> by <a href="Alexander_Rich" title="wikilink">Alexander Rich</a></li>
<li><a href="https://www.dna.utah.edu/umelt/umelt.html">uMelt: Melting Curve Prediction</a></li>
<li><a href="https://www.dna.utah.edu/utensils/RCC.php">Sequence Tm Utility v1.5</a></li>
<li><a href="http://rna.urmc.rochester.edu/NNDB">Nearest Neighbor Database: Provides a description of RNA-RNA interaction nearest neighbor parameters and examples of their use.</a></li>
</ul>

<p>"</p>

<p><a class="uri" href="Category:DNA" title="wikilink">Category:DNA</a> <a href="Category:Nucleic_acids" title="wikilink">Category:Nucleic acids</a> <a href="Category:Molecular_biology" title="wikilink">Category:Molecular biology</a> <a class="uri" href="Category:Biotechnology" title="wikilink">Category:Biotechnology</a> <a href="Category:Biochemical_engineering" title="wikilink">Category:Biochemical engineering</a></p>
<section class="footnotes">
<hr/>
<ol>
<li id="fn1"><a href="#fnref1">↩</a></li>
<li id="fn2"><a href="#fnref2">↩</a></li>
<li id="fn3"><a href="#fnref3">↩</a></li>
<li id="fn4"><a href="#fnref4">↩</a></li>
<li id="fn5"> [<a class="uri" href="http://www.pubmedcentral.gov/picrender.fcgi?artid=323600&amp;blobtype">http://www.pubmedcentral.gov/picrender.fcgi?artid=323600&amp;blobtype;</a>;=pdf (pdf)]<a href="#fnref5">↩</a></li>
<li id="fn6"><a href="#fnref6">↩</a></li>
<li id="fn7"> <a href="http://www3.interscience.wiley.com/cgi-bin/abstract/40314/ABSTRACT">(pdf)</a><a href="#fnref7">↩</a></li>
<li id="fn8"><a href="#fnref8">↩</a></li>
<li id="fn9"></li>
<li id="fn10"></li>
<li id="fn11"><a href="#fnref11">↩</a></li>
<li id="fn12"><a href="#fnref12">↩</a></li>
<li id="fn13"><a href="#fnref13">↩</a></li>
<li id="fn14"></li>
</ol>
</section>
</body>
</html>
