<html lang="en">
<head>
<meta charset="utf-8"/>
<title offset="1149">Phosphoribosylglycinamide formyltransferase</title>
<script src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_SVG.js" type="text/javascript">
</script>
</head>
<body>
<h1>Phosphoribosylglycinamide formyltransferase</h1>
<hr/>

<p><strong>Phosphoribosylglycinamide formyltransferase</strong> (, <em>2-amino-N-ribosylacetamide 5'-phosphate transformylase</em>, <em>GAR formyltransferase</em>, <em>GAR transformylase</em>, <em>glycinamide ribonucleotide transformylase</em>, <em>GAR TFase</em>, <em>5,10-methenyltetrahydrofolate:2-amino-N-ribosylacetamide ribonucleotide transformylase</em>) is an <a class="uri" href="enzyme" title="wikilink">enzyme</a> with system name <em>10-formyltetrahydrofolate:5'-phosphoribosylglycinamide N-formyltransferase</em>.<a class="footnoteRef" href="#fn1" id="fnref1"><sup>1</sup></a><a class="footnoteRef" href="#fn2" id="fnref2"><sup>2</sup></a><a class="footnoteRef" href="#fn3" id="fnref3"><sup>3</sup></a> This enzyme <a href="catalysis" title="wikilink">catalyses</a> the following <a href="chemical_reaction" title="wikilink">chemical reaction</a></p>
<dl>
<dd>10-<a class="uri" href="formyltetrahydrofolate" title="wikilink">formyltetrahydrofolate</a> + N<sup>1</sup>-(5-phospho-D-ribosyl)glycinamide 

<math display="inline" id="Phosphoribosylglycinamide_formyltransferase:0">
 <semantics>
  <mo>⇌</mo>
  <annotation-xml encoding="MathML-Content">
   <ci>normal-⇌</ci>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \rightleftharpoons
  </annotation>
 </semantics>
</math>

 <a class="uri" href="tetrahydrofolate" title="wikilink">tetrahydrofolate</a> + N<sup>2</sup>-formyl-N<sup>1</sup>-(5-phospho-D-ribosyl)glycinamide
</dd>
</dl>

<p><a href="File:Overall_Reaction_of_GAR_transformylase.jpg" title="wikilink">thumb|center|upright=1.2|Overall reaction of GAR transformylase</a> This THF dependent enzyme catalyzes a nucleophilic acyl substitution of the formyl group from 10-<a class="uri" href="formyltetrahydrofolate" title="wikilink">formyltetrahydrofolate</a> (fTHF) to N<sup>1</sup>-(5-phospho-D-ribosyl)glycinamide (GAR) to form N<sup>2</sup>-formyl-N<sup>1</sup>-(5-phospho-D-ribosyl)glycinamide (fGAR) as shown above.<a class="footnoteRef" href="#fn4" id="fnref4"><sup>4</sup></a> This reaction plays an important role in the formation of purine through the <em>de novo</em> purine biosynthesis pathway. This pathway creates inosine monophosphate (IMP), a precursor to adenosine monophosphate (AMP) and guanosine monophosphate (GMP). AMP is a building block for important energy carriers such as ATP, NAD<sup>+</sup> and FAD, and signaling molecules such as cAMP. GARTfase's role in <em>de novo</em> purine biosynthesis makes it a target for anti-cancer drugs<a class="footnoteRef" href="#fn5" id="fnref5"><sup>5</sup></a> and its overexpression during postnatal development has been connected to Down syndrome.<a class="footnoteRef" href="#fn6" id="fnref6"><sup>6</sup></a> There are two known types of genes encoding GAR transformylase in E.coli: purN and purT, while only purN is found in humans.<a class="footnoteRef" href="#fn7" id="fnref7"><sup>7</sup></a> Many residues in the active site are conserved across bacterial, yeast, avian and human enzymes.<a class="footnoteRef" href="#fn8" id="fnref8"><sup>8</sup></a></p>
<h2 id="enzyme-structure">Enzyme Structure</h2>
<figure><b>(Figure)</b>
<figcaption>Structure of GAR transformylase in rainbow from N(blue)-&gt;C(red). Folate binding loop is highlighted in black. GAR substrate (red) and THF analogue (blue) are shown in binding pockets.<a class="footnoteRef" href="#fn9" id="fnref9"><sup>9</sup></a></figcaption>
</figure>

<p>In humans, GARTfase is part of trifunctional enzyme which also includes glycinamide ribnucleotide synthase (<a href="Phosphoribosylamine—glycine_ligase" title="wikilink">GARS</a>) and aminoimidazole ribonucleotide synthetase (<a href="AIR_synthetase_(FGAM_cyclase)" title="wikilink">AIRS</a>). This protein (110kDa) catalyzes steps 2, 3 and 5 of de novo purine biosynthesis. The proximity of these enzyme units and flexibility of the protein serves to increase pathway throughput. GARTfase is located on the C-terminal end of the protein.<a class="footnoteRef" href="#fn10" id="fnref10"><sup>10</sup></a></p>

<p>Human GARTfase has been crystallized by vapor-diffusion sitting drop method and imaged at the <a href="Stanford_Synchrotron_Radiation_Laboratory" title="wikilink">Stanford Synchrotron Radiation Laboratory</a> (SSRL) by at least two groups.<a class="footnoteRef" href="#fn11" id="fnref11"><sup>11</sup></a><a class="footnoteRef" href="#fn12" id="fnref12"><sup>12</sup></a></p>

<p>The structure can be described by two subdomains which are connected by a seven-stranded beta sheet. The N- terminal domain consists of a Rossman type mononucleotide fold, with a four strand part of the beta sheet surrounded on each side by two alpha helices. The beta sheet continues into the C terminal domain, where on one side it is covered by a long alpha helix and on the other it is partially exposed to solvent. It is the cleft between the two subdomains where the active site lies.<a class="footnoteRef" href="#fn13" id="fnref13"><sup>13</sup></a></p>

<p>The cleft consists of the GAR binding site and the folate binding pocket. The folate binding pocket is delineated by pteridine-binding cleft, the formyl transfer region and the benzoylglutamate region which bind thepteridine head and a benzoylglutamate tail connected by a formyl bound nitrogen of fTHF. This folate binding region has been the subject of much research because its inhibition by small molecules has led to the discovery of antineoplastic drugs. The folate binding loop has been shown to change conformation depending on the pH of solution and as such Human GAR transformylase shows highest activity around pH 7.5-8. Lower pH (~4.2) conditions change the conformation of the substrate (GAR) binding loops as well.<a class="footnoteRef" href="#fn14" id="fnref14"><sup>14</sup></a></p>
<h2 id="mechanism">Mechanism</h2>
<h3 id="mechanism-of-purn-gartfase">Mechanism of purN GARTfase</h3>
<figure><b>(Figure)</b>
<figcaption>GAR transformylase water assisted mechanism suggested by Qiao et al</figcaption>
</figure>

<p>Klein et al first suggested a water molecule assisted mechanism. A single water molecule possibly held in place by hydrogen bonding with the carboxylate group of the persistent Asp144 residue transfers protons from the GAR-N to the THF-N. The nucleophilic nitrogen on the terminal amino group of GAR attacks the carbonyl carbon of the formyl group on THF pushing negative charge onto the oxygen. Klein suggests that His108 stabilizes the transition state by hydrogen bonding with the negatively charged oxygen and that the reformation of the carbonyl double bond results in breaking the THF-N - formyl bond. Calculations by Qiao et al suggest that the water assisted stepwise proton transfer from Gar-N to THF-N is 80-100 kj/mol more favorable than the concerted transfer suggested by Klein. The mechanism shown is suggested by Qiao et al, whom admittedly did not consider surrounding residues in their calculations.<a class="footnoteRef" href="#fn15" id="fnref15"><sup>15</sup></a><a class="footnoteRef" href="#fn16" id="fnref16"><sup>16</sup></a></p>
<h3 id="mechanism-of-purt-gartfase">Mechanism of purT GARTfase</h3>

<p>Studies of the purT variant of GAR transformylase in E.coli found that the reaction proceeds through a formyl phosphate intermediate. While the in vitro reaction can proceed without THF, overall the in vivo reaction is the same.<a class="footnoteRef" href="#fn17" id="fnref17"><sup>17</sup></a></p>
<h2 id="involvement-in-de-novo-purine-biosynthesis">Involvement in <em>de novo</em> Purine Biosynthesis</h2>

<p>GART catalyzes the third step in the ten step <em>de novo</em> purine biosynthesis, the formation of N<sup>2</sup>-formyl-N<sup>1</sup>-(5-phospho-D-ribosyl)glycinamide (fGAR) by formyl addition to N<sup>1</sup>-(5-phospho-D-ribosyl)glycinamide (GAR).<a class="footnoteRef" href="#fn18" id="fnref18"><sup>18</sup></a> In E. coli, the purN enzyme is a 23 kDa protein<a class="footnoteRef" href="#fn19" id="fnref19"><sup>19</sup></a> but in humans it is part of a trifunctional protein of 110 kDa which includes AIRS and GARS functionalities.<a class="footnoteRef" href="#fn20" id="fnref20"><sup>20</sup></a> This protein catalyzes three different steps of the <em>de novo</em> purine pathway.</p>
<h2 id="disease-relevance">Disease Relevance</h2>
<h3 id="cancer-target">Cancer Target</h3>

<p>Due to their increased growth rate and metabolic requirements, cancer cells rely on <em>de novo</em> nucleotide biosynthesis to achieve levels of AMP and GMP necessary.<a class="footnoteRef" href="#fn21" id="fnref21"><sup>21</sup></a> Being able to block any of the steps of the <em>de novo</em> purine pathway would present significant reduction in tumor growth. Studies have been done both on the substrate binding<a class="footnoteRef" href="#fn22" id="fnref22"><sup>22</sup></a> and folate binding site<a class="footnoteRef" href="#fn23" id="fnref23"><sup>23</sup></a> to find inhibitors.</p>
<h3 id="down-syndrome">Down Syndrome</h3>

<p>GARTfase is suspected to be connected with Down syndrome. The gene encoding the trifunctional protein human GARS-AIRS-GART is located on chromosome 21q22.1, in the Down syndrome critical region. The protein is overexpressed in the cerebellum during the postnatal development of individuals with Down syndrome. Typically, this protein is undetectable in cerebellum shortly after birth, but found in high levels in prenatal development.<a class="footnoteRef" href="#fn24" id="fnref24"><sup>24</sup></a><a class="footnoteRef" href="#fn25" id="fnref25"><sup>25</sup></a></p>
<h2 id="see-also">See also</h2>
<ul>
<li><a href="Trifunctional_purine_biosynthetic_protein_adenosine-3" title="wikilink">Trifunctional purine biosynthetic protein adenosine-3</a></li>
</ul>
<h2 id="references">References</h2>
<h2 id="external-links">External links</h2>
<ul>
<li></li>
</ul>

<p>"</p>

<p><a href="Category:EC_2.1.2" title="wikilink">Category:EC 2.1.2</a></p>
<section class="footnotes">
<hr/>
<ol>
<li id="fn1"><a href="#fnref1">↩</a></li>
<li id="fn2"><a href="#fnref2">↩</a></li>
<li id="fn3"><a href="#fnref3">↩</a></li>
<li id="fn4">McMurry, J. and Tadhg, B. The Organic Chemistry of Biological Pathways<a href="#fnref4">↩</a></li>
<li id="fn5"><a href="#fnref5">↩</a></li>
<li id="fn6"><a href="#fnref6">↩</a></li>
<li id="fn7"><a href="#fnref7">↩</a></li>
<li id="fn8"></li>
<li id="fn9">Zhang, Y., Desharnais, J., Boger, D.L., Wilson, I.A. (2005) "Human GAR Tfase complex structure with 10-(trifluoroacetyl)-5,10-dideazaacyclic-5,6,7,8-tetrahydrofolic acid and substrate beta-GAR." Unplubished. PDB: <a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=1RBY">1RBY</a>.<a href="#fnref9">↩</a></li>
<li id="fn10"><a href="#fnref10">↩</a></li>
<li id="fn11"></li>
<li id="fn12"><a href="#fnref12">↩</a></li>
<li id="fn13"><a href="#fnref13">↩</a></li>
<li id="fn14"></li>
<li id="fn15"><a href="#fnref15">↩</a></li>
<li id="fn16"><a href="#fnref16">↩</a></li>
<li id="fn17"><a href="#fnref17">↩</a></li>
<li id="fn18"></li>
<li id="fn19"><a href="#fnref19">↩</a></li>
<li id="fn20"></li>
<li id="fn21"><a href="#fnref21">↩</a></li>
<li id="fn22"><a href="#fnref22">↩</a></li>
<li id="fn23"><a href="#fnref23">↩</a></li>
<li id="fn24"></li>
<li id="fn25"><a href="#fnref25">↩</a></li>
</ol>
</section>
</body>
</html>
