<html lang="en">
<head>
<meta charset="utf-8"/>
<title offset="121">Pi helix</title>
<script src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_SVG.js" type="text/javascript">
</script>
</head>
<body>
<h1>Pi helix</h1>
<hr/>
<figure><b>(Figure)</b>
<figcaption>Side view of a standard π-helix of L-<a class="uri" href="alanine" title="wikilink">alanine</a> residues in <a href="atom" title="wikilink">atomic</a> detail. Two <a href="hydrogen_bond" title="wikilink">hydrogen bonds</a> to the same <a href="peptide_bond" title="wikilink">peptide group</a> are highlighted in magenta; the oxygen-hydrogen distance is 1.65 Å (165 pm). The <a class="uri" href="protein" title="wikilink">protein</a> chain runs upwards, i.e., its N-terminus is at the bottom and its C-terminus at the top of the figure. Note that the sidechains point slightly <em>downwards</em>, i.e., towards the N-terminus.</figcaption>
</figure>

<p>A <strong>pi helix</strong> (or <strong>π-helix</strong>) is a type of <a href="secondary_structure" title="wikilink">secondary structure</a> found in <a href="protein" title="wikilink">proteins</a>.<a class="footnoteRef" href="#fn1" id="fnref1"><sup>1</sup></a> Although once thought to be rare, short π-helices are found in 15% of known protein structures and are believed to be an evolutionary adaptation derived by the insertion of a single amino acid into an <a class="uri" href="α-helix" title="wikilink">α-helix</a>.<a class="footnoteRef" href="#fn2" id="fnref2"><sup>2</sup></a> Because such insertions are highly destabilizing,<a class="footnoteRef" href="#fn3" id="fnref3"><sup>3</sup></a> the formation of π-helices would tend to be selected against unless it provided some functional advantage to the protein. π-helices therefore are typically found near functional sites of proteins.<a class="footnoteRef" href="#fn4" id="fnref4"><sup>4</sup></a><a class="footnoteRef" href="#fn5" id="fnref5"><sup>5</sup></a><a class="footnoteRef" href="#fn6" id="fnref6"><sup>6</sup></a></p>
<h2 id="standard-structure">Standard structure</h2>

<p>The <a href="amino_acid" title="wikilink">amino acids</a> in a standard π-helix are arranged in a right-handed <a href="helix" title="wikilink">helical</a> structure. Each amino acid corresponds to an 87° turn in the helix (i.e., the helix has 4.1 residues per turn), and a translation of 1.15 <a href="Ångström" title="wikilink">Å</a> (=0.115 <a href="Nanometre" title="wikilink">nm</a>) along the helical axis. Most importantly, the <a href="amine" title="wikilink">N-H</a> group of an amino acid forms a <a href="hydrogen_bond" title="wikilink">hydrogen bond</a> with the <a href="carbonyl" title="wikilink">C=O</a> group of the amino acid <em>five</em> residues earlier; this repeated <em>i</em>+5→<em>i</em> hydrogen bonding <strong>defines</strong> a π-helix. Similar structures include the <a href="310_helix" title="wikilink">3<sub>10</sub> helix</a> (<em>i</em>+3→<em>i</em> hydrogen bonding) and the <a href="Alpha_helix" title="wikilink">α-helix</a> (<em>i</em>+4→<em>i</em> hydrogen bonding).</p>
<figure><b>(Figure)</b>
<figcaption>Top view of the same helix shown above. Four <a class="uri" href="carbonyl" title="wikilink">carbonyl</a> groups are pointing upwards towards the viewer, spaced roughly 87° apart on the circle, corresponding to 4.1 <a href="amino_acid" title="wikilink">amino-acid</a> residues per turn of the helix.</figcaption>
</figure>

<p>The majority of π-helices are only 7 residues in length and do not adopt regularly repeating (φ, ψ) <a href="dihedral_angle" title="wikilink">dihedral angles</a> throughout the entire structure like that of α-helices or β-sheets. Because of this, textbooks that provide single dihedral values for all residues in the π-helix are misleading. Some generalizations can be made, however. When the first and last residue pairs are excluded, dihedral angles exist such that the ψ <a href="dihedral_angle" title="wikilink">dihedral angle</a> of one residue and the φ dihedral angle of the <em>next</em> residue sum to roughly -125°. The first and last residue pairs sum to -95° and -105°, respectively. For comparison, the sum of the dihedral angles for a 3<sub>10</sub> helix is roughly -75°, whereas that for the α-helix is roughly -105°. Proline is often seen immediately following the end of π-helices. The general formula for the rotation angle Ω per residue of any polypeptide helix with <em>trans</em> isomers is given by the equation</p>

<p>

<math display="block" id="Pi_helix:0">
 <semantics>
  <mrow>
   <mrow>
    <mn>3</mn>
    <mrow>
     <mi>cos</mi>
     <mi mathvariant="normal">Ω</mi>
    </mrow>
   </mrow>
   <mo>=</mo>
   <mrow>
    <mn>1</mn>
    <mo>-</mo>
    <mrow>
     <mn>4</mn>
     <mrow>
      <msup>
       <mi>cos</mi>
       <mn>2</mn>
      </msup>
      <mrow>
       <mo>[</mo>
       <mrow>
        <mrow>
         <mo>(</mo>
         <mrow>
          <mi>ϕ</mi>
          <mo>+</mo>
          <mi>ψ</mi>
         </mrow>
         <mo>)</mo>
        </mrow>
        <mo>/</mo>
        <mn>2</mn>
       </mrow>
       <mo>]</mo>
      </mrow>
     </mrow>
    </mrow>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <eq></eq>
    <apply>
     <times></times>
     <cn type="integer">3</cn>
     <apply>
      <cos></cos>
      <ci>normal-Ω</ci>
     </apply>
    </apply>
    <apply>
     <minus></minus>
     <cn type="integer">1</cn>
     <apply>
      <times></times>
      <cn type="integer">4</cn>
      <apply>
       <apply>
        <csymbol cd="ambiguous">superscript</csymbol>
        <cos></cos>
        <cn type="integer">2</cn>
       </apply>
       <apply>
        <divide></divide>
        <apply>
         <plus></plus>
         <ci>ϕ</ci>
         <ci>ψ</ci>
        </apply>
        <cn type="integer">2</cn>
       </apply>
      </apply>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   3\cos\Omega=1-4\cos^{2}\left[\left(\phi+\psi\right)/2\right]
  </annotation>
 </semantics>
</math>

</p>
<h2 id="left-handed-structure">Left-handed structure</h2>

<p>In principle, a left-handed version of the π-helix is possible by reversing the sign of the (φ, ψ) <a href="dihedral_angle" title="wikilink">dihedral angles</a> to (55°, 70°). This pseudo-"mirror-image" helix has roughly the same number of residues per turn (4.1) and helical pitch (1.5 angstroms or 150 <a href="picometer" title="wikilink">picometers</a>). It is not a true mirror image, because the <a href="amino_acid" title="wikilink">amino-acid</a> residues still have a left-handed <a href="chirality_(chemistry)" title="wikilink">chirality</a>. A long left-handed π-helix is unlikely to be observed in proteins because, among the naturally occurring amino acids, only <a class="uri" href="glycine" title="wikilink">glycine</a> is likely to adopt positive φ dihedral angles such as 55°.</p>
<h2 id="π-helices-in-nature">π-helices in nature</h2>
<figure><b>(Figure)</b>
<figcaption>A short, 7 residue π-helix (orange) is embedded within a longer, α-helix (green). The "bulge" of the π-helix can be clearly seen, and was created as the result of a single amino acid that has been inserted into an α-helix. PBD code 3QHB.</figcaption>
</figure>

<p>Commonly used automated secondary structure assignment programs, such as <a href="DSSP_(protein)" title="wikilink">DSSP</a>, suggest </p>

<p>Natural π-helices can easily be identified in a structure as a "bulge" within a longer α-helix. Such helical bulges have previously been referred to as α-aneurisms, α-bulges, π-bulges, wide-turns,looping outs and π-turns, but in fact are π-helices as determined by their repeating <em>i</em>+5→<em>i</em> hydrogen bonds.<a class="footnoteRef" href="#fn7" id="fnref7"><sup>7</sup></a> Evidence suggests that these bulges, or π-helices, are created by the insertion of a single additional amino acid into a pre-existing α-helix. Thus, α-helices and π-helices can be inter-converted by the insertion and deletion of a single amino acid.<a class="footnoteRef" href="#fn8" id="fnref8"><sup>8</sup></a><a class="footnoteRef" href="#fn9" id="fnref9"><sup>9</sup></a> Given both the relatively high rate of occurrence of π-helices and their noted association with functional sites (i.e. active sites) of proteins, this ability to inter-convert between α-helices and π-helices has been an important mechanism of altering and diversifying protein functionality over the course of evolution.<a class="footnoteRef" href="#fn10" id="fnref10"><sup>10</sup></a></p>

<p>One of the most notable group of proteins whose functional diversification appears to have been heavily influenced by such an evolutionary mechanism is the ferritin-like superfamily, which includes <a href="ferritin" title="wikilink">ferritins</a>, <a href="bacterioferritin" title="wikilink">bacterioferritins</a>, <a href="rubrerythrin" title="wikilink">rubrerythrins</a>, class I <a href="ribonucleotide_reductase" title="wikilink">ribonucleotide reductases</a> and <a href="Methane_monooxygenase#Soluble_methane_Monooxygenase_.28MMO.29_Systems" title="wikilink">soluble methane monooxygenases</a>. Soluble methane monooxygenase is the current record holder for the most number of π-helices in a single enzyme with 13 (PDB code 1MTY). However, the bacterial homologue of a Na+/Cl− dependent neurotransmitter transporter (PDB code 2A65) holds the record for the most π-helices in a single peptide chain with 8.<a class="footnoteRef" href="#fn11" id="fnref11"><sup>11</sup></a></p>
<h2 id="see-also">See also</h2>
<ul>
<li><a href="alpha_helix" title="wikilink">alpha helix</a></li>
<li><a href="3_10_helix" title="wikilink">3 10 helix</a></li>
<li><a href="secondary_structure" title="wikilink">secondary structure</a></li>
</ul>
<h2 id="references">References</h2>

<p>"</p>

<p><a href="Category:Protein_structural_motifs" title="wikilink">Category:Protein structural motifs</a> <a class="uri" href="Category:Helices" title="wikilink">Category:Helices</a></p>
<section class="footnotes">
<hr/>
<ol>
<li id="fn1"><a href="#fnref1">↩</a></li>
<li id="fn2"><a href="#fnref2">↩</a></li>
<li id="fn3"><a href="#fnref3">↩</a></li>
<li id="fn4"></li>
<li id="fn5"><a href="#fnref5">↩</a></li>
<li id="fn6"><a href="#fnref6">↩</a></li>
<li id="fn7"></li>
<li id="fn8"></li>
<li id="fn9"></li>
<li id="fn10"></li>
<li id="fn11"></li>
</ol>
</section>
</body>
</html>
