<html lang="en">
<head>
<meta charset="utf-8"/>
<title offset="701">Population stratification</title>
<script src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_SVG.js" type="text/javascript">
</script>
</head>
<body>
<h1>Population stratification</h1>
<hr/>

<p><strong>Population stratification</strong> is the presence of a systematic difference in <a class="uri" href="allele" title="wikilink">allele</a> <a href="Allele_frequency" title="wikilink">frequencies</a> between subpopulations in a <a class="uri" href="population" title="wikilink">population</a> possibly due to different <a href="Ancestor" title="wikilink">ancestry</a>, especially in the context of association studies. Population stratification is also referred to as population structure, in this context.</p>
<h2 id="causes-of-population-stratification">Causes of population stratification</h2>

<p>The basic cause of population stratification is nonrandom mating between groups, often due to their physical separation (e.g., for populations of African and European descent) followed by genetic drift of allele frequencies in each group. In some contemporary populations there has been recent admixture between individuals from different populations, leading to populations in which ancestry is variable (as in African Americans). Over tens of generations, random mating can eliminate this type of stratification. In some parts of the globe (e.g., in Europe), population structure is best modeled by isolation-by-distance, in which allele frequencies tend to vary smoothly with location.</p>
<h2 id="population-stratification-and-association-studies">Population stratification and association studies</h2>

<p>Population stratification can be a problem for association studies, such as case-control studies, where the association could be found due to the underlying structure of the population and not a disease associated locus. By analogy, one might imagine a scenario in which certain small beads are made out of a certain type of unique foam, and that children tend to choke on these beads; one might wrongly conclude that the foam material causes choking when in fact it is the small size of the beads. Also the real disease causing locus might not be found in the study if the locus is less prevalent in the population where the case subjects are chosen. For this reason, it was common in the 1990s to use family-based data where the effect of population stratification can easily be controlled for using methods such as the <a href="Transmission_disequilibrium_test" title="wikilink">TDT</a>. But if the structure is known or a putative structure is found, there are a number of possible ways to implement this structure in the association studies and thus compensate for any population bias. Most contemporary genome-wide association studies take the view that the problem of population stratification is manageable,<a class="footnoteRef" href="#fn1" id="fnref1"><sup>1</sup></a> and that the logistic advantages of using unrelated cases and controls make these studies preferable to family-based association studies.</p>

<p>The two most widely used approaches to this problem include genomic control, which is a relatively nonparametric method for controlling the inflation of test statistics,<a class="footnoteRef" href="#fn2" id="fnref2"><sup>2</sup></a> and structured association methods,<a class="footnoteRef" href="#fn3" id="fnref3"><sup>3</sup></a> which use genetic information to estimate and control for population structure. Currently, the most widely used structured association method is Eigenstrat, developed by Alkes Price and colleagues.<a class="footnoteRef" href="#fn4" id="fnref4"><sup>4</sup></a></p>
<h2 id="genomic-control">Genomic Control</h2>

<p>The assumption of population homogeneity in association studies, especially case-control studies, can easily be violated and can lead to both <a href="type_I_and_type_II_errors" title="wikilink">type I and type II errors</a>. It is therefore important for the models used in the study to compensate for the population structure. The problem in case control studies is that if there is a genetic involvement in the disease, the case population is more likely to be related than the individuals in the control population. This means that the assumption of independence of observations is violated. Often this will lead to an overestimation of the significance of an association but it depends on the way the sample was chosen. As long as there is a higher allele frequency in a subpopulation you will find association with any trait more prevalent in the case population.<a class="footnoteRef" href="#fn5" id="fnref5"><sup>5</sup></a> This kind of spurious association increases as the sample population grows so the problem should be of special concern in large scale association studies when loci only cause relatively small effects on the trait. A method that in some cases can compensate for the above described problems has been developed by Devlin and Roeder (1999).<a class="footnoteRef" href="#fn6" id="fnref6"><sup>6</sup></a> It uses both a frequentist and a Bayesian approach (the latter being appropriate when dealing with a large number of candidate genes). Here is a short description of how the frequentist way of correcting for population stratification works. It works by using markers that are not linked with the trait in question to correct for any inflation of the statistic caused by population stratification. The method was first developed for binary traits but has since been generalized for quantitative ones.<a class="footnoteRef" href="#fn7" id="fnref7"><sup>7</sup></a> For the binary one, which applies to finding genetic differences between the case and control populations, Devlin and Roeder (1999) use <a href="Cochran–Armitage_test_for_trend" title="wikilink">Armitage's trend test</a></p>

<p>

<math display="inline" id="Population_stratification:0">
 <semantics>
  <mrow>
   <msup>
    <mi>Y</mi>
    <mn>2</mn>
   </msup>
   <mo>=</mo>
   <mfrac>
    <mrow>
     <mi>N</mi>
     <msup>
      <mrow>
       <mo stretchy="false">(</mo>
       <mrow>
        <mrow>
         <mi>N</mi>
         <mrow>
          <mo stretchy="false">(</mo>
          <mrow>
           <msub>
            <mi>r</mi>
            <mn>1</mn>
           </msub>
           <mo>+</mo>
           <mrow>
            <mn>2</mn>
            <msub>
             <mi>r</mi>
             <mn>2</mn>
            </msub>
           </mrow>
          </mrow>
          <mo stretchy="false">)</mo>
         </mrow>
        </mrow>
        <mo>-</mo>
        <mrow>
         <mi>R</mi>
         <mrow>
          <mo stretchy="false">(</mo>
          <mrow>
           <msub>
            <mi>n</mi>
            <mn>1</mn>
           </msub>
           <mo>+</mo>
           <mrow>
            <mn>2</mn>
            <msub>
             <mi>n</mi>
             <mn>2</mn>
            </msub>
           </mrow>
          </mrow>
          <mo stretchy="false">)</mo>
         </mrow>
        </mrow>
       </mrow>
       <mo stretchy="false">)</mo>
      </mrow>
      <mn>2</mn>
     </msup>
    </mrow>
    <mrow>
     <mi>R</mi>
     <mrow>
      <mo stretchy="false">(</mo>
      <mrow>
       <mi>N</mi>
       <mo>-</mo>
       <mi>R</mi>
      </mrow>
      <mo stretchy="false">)</mo>
     </mrow>
     <mrow>
      <mo stretchy="false">(</mo>
      <mrow>
       <mrow>
        <mi>N</mi>
        <mrow>
         <mo stretchy="false">(</mo>
         <mrow>
          <msub>
           <mi>n</mi>
           <mn>1</mn>
          </msub>
          <mo>+</mo>
          <mrow>
           <mn>4</mn>
           <msub>
            <mi>n</mi>
            <mn>2</mn>
           </msub>
          </mrow>
         </mrow>
         <mo stretchy="false">)</mo>
        </mrow>
       </mrow>
       <mo>-</mo>
       <msup>
        <mrow>
         <mo stretchy="false">(</mo>
         <mrow>
          <msub>
           <mi>n</mi>
           <mn>1</mn>
          </msub>
          <mo>+</mo>
          <mrow>
           <mn>2</mn>
           <msub>
            <mi>n</mi>
            <mn>2</mn>
           </msub>
          </mrow>
         </mrow>
         <mo stretchy="false">)</mo>
        </mrow>
        <mn>2</mn>
       </msup>
      </mrow>
      <mo stretchy="false">)</mo>
     </mrow>
    </mrow>
   </mfrac>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <eq></eq>
    <apply>
     <csymbol cd="ambiguous">superscript</csymbol>
     <ci>Y</ci>
     <cn type="integer">2</cn>
    </apply>
    <apply>
     <divide></divide>
     <apply>
      <times></times>
      <ci>N</ci>
      <apply>
       <csymbol cd="ambiguous">superscript</csymbol>
       <apply>
        <minus></minus>
        <apply>
         <times></times>
         <ci>N</ci>
         <apply>
          <plus></plus>
          <apply>
           <csymbol cd="ambiguous">subscript</csymbol>
           <ci>r</ci>
           <cn type="integer">1</cn>
          </apply>
          <apply>
           <times></times>
           <cn type="integer">2</cn>
           <apply>
            <csymbol cd="ambiguous">subscript</csymbol>
            <ci>r</ci>
            <cn type="integer">2</cn>
           </apply>
          </apply>
         </apply>
        </apply>
        <apply>
         <times></times>
         <ci>R</ci>
         <apply>
          <plus></plus>
          <apply>
           <csymbol cd="ambiguous">subscript</csymbol>
           <ci>n</ci>
           <cn type="integer">1</cn>
          </apply>
          <apply>
           <times></times>
           <cn type="integer">2</cn>
           <apply>
            <csymbol cd="ambiguous">subscript</csymbol>
            <ci>n</ci>
            <cn type="integer">2</cn>
           </apply>
          </apply>
         </apply>
        </apply>
       </apply>
       <cn type="integer">2</cn>
      </apply>
     </apply>
     <apply>
      <times></times>
      <ci>R</ci>
      <apply>
       <minus></minus>
       <ci>N</ci>
       <ci>R</ci>
      </apply>
      <apply>
       <minus></minus>
       <apply>
        <times></times>
        <ci>N</ci>
        <apply>
         <plus></plus>
         <apply>
          <csymbol cd="ambiguous">subscript</csymbol>
          <ci>n</ci>
          <cn type="integer">1</cn>
         </apply>
         <apply>
          <times></times>
          <cn type="integer">4</cn>
          <apply>
           <csymbol cd="ambiguous">subscript</csymbol>
           <ci>n</ci>
           <cn type="integer">2</cn>
          </apply>
         </apply>
        </apply>
       </apply>
       <apply>
        <csymbol cd="ambiguous">superscript</csymbol>
        <apply>
         <plus></plus>
         <apply>
          <csymbol cd="ambiguous">subscript</csymbol>
          <ci>n</ci>
          <cn type="integer">1</cn>
         </apply>
         <apply>
          <times></times>
          <cn type="integer">2</cn>
          <apply>
           <csymbol cd="ambiguous">subscript</csymbol>
           <ci>n</ci>
           <cn type="integer">2</cn>
          </apply>
         </apply>
        </apply>
        <cn type="integer">2</cn>
       </apply>
      </apply>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   Y^{2}=\frac{N(N(r_{1}+2r_{2})-R(n_{1}+2n_{2}))^{2}}{R(N-R)(N(n_{1}+4n_{2})-(n_%
{1}+2n_{2})^{2})}
  </annotation>
 </semantics>
</math>

</p>

<p>and the 

<math display="inline" id="Population_stratification:1">
 <semantics>
  <msup>
   <mi>χ</mi>
   <mn>2</mn>
  </msup>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <csymbol cd="ambiguous">superscript</csymbol>
    <ci>χ</ci>
    <cn type="integer">2</cn>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \chi^{2}
  </annotation>
 </semantics>
</math>

 test for allelic frequencies</p>

<p>

<math display="inline" id="Population_stratification:2">
 <semantics>
  <mrow>
   <msup>
    <mi>χ</mi>
    <mn>2</mn>
   </msup>
   <mo>∼</mo>
   <msubsup>
    <mi>X</mi>
    <mi>A</mi>
    <mn>2</mn>
   </msubsup>
   <mo>=</mo>
   <mfrac>
    <mrow>
     <mn>2</mn>
     <mi>N</mi>
     <msup>
      <mrow>
       <mo stretchy="false">(</mo>
       <mrow>
        <mrow>
         <mn>2</mn>
         <mi>N</mi>
         <mrow>
          <mo stretchy="false">(</mo>
          <mrow>
           <msub>
            <mi>r</mi>
            <mn>1</mn>
           </msub>
           <mo>+</mo>
           <mrow>
            <mn>2</mn>
            <msub>
             <mi>r</mi>
             <mn>2</mn>
            </msub>
           </mrow>
          </mrow>
          <mo stretchy="false">)</mo>
         </mrow>
        </mrow>
        <mo>-</mo>
        <mrow>
         <mi>R</mi>
         <mrow>
          <mo stretchy="false">(</mo>
          <mrow>
           <msub>
            <mi>n</mi>
            <mn>1</mn>
           </msub>
           <mo>+</mo>
           <mrow>
            <mn>2</mn>
            <msub>
             <mi>n</mi>
             <mn>2</mn>
            </msub>
           </mrow>
          </mrow>
          <mo stretchy="false">)</mo>
         </mrow>
        </mrow>
       </mrow>
       <mo stretchy="false">)</mo>
      </mrow>
      <mn>2</mn>
     </msup>
    </mrow>
    <mrow>
     <mn>4</mn>
     <mi>R</mi>
     <mrow>
      <mo stretchy="false">(</mo>
      <mrow>
       <mi>N</mi>
       <mo>-</mo>
       <mi>R</mi>
      </mrow>
      <mo stretchy="false">)</mo>
     </mrow>
     <mrow>
      <mo stretchy="false">(</mo>
      <mrow>
       <mrow>
        <mn>2</mn>
        <mi>N</mi>
        <mrow>
         <mo stretchy="false">(</mo>
         <mrow>
          <msub>
           <mi>n</mi>
           <mn>1</mn>
          </msub>
          <mo>+</mo>
          <mrow>
           <mn>2</mn>
           <msub>
            <mi>n</mi>
            <mn>2</mn>
           </msub>
          </mrow>
         </mrow>
         <mo stretchy="false">)</mo>
        </mrow>
       </mrow>
       <mo>-</mo>
       <msup>
        <mrow>
         <mo stretchy="false">(</mo>
         <mrow>
          <msub>
           <mi>n</mi>
           <mn>1</mn>
          </msub>
          <mo>+</mo>
          <mrow>
           <mn>2</mn>
           <msub>
            <mi>n</mi>
            <mn>2</mn>
           </msub>
          </mrow>
         </mrow>
         <mo stretchy="false">)</mo>
        </mrow>
        <mn>2</mn>
       </msup>
      </mrow>
      <mo stretchy="false">)</mo>
     </mrow>
    </mrow>
   </mfrac>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <and></and>
    <apply>
     <csymbol cd="latexml">similar-to</csymbol>
     <apply>
      <csymbol cd="ambiguous">superscript</csymbol>
      <ci>χ</ci>
      <cn type="integer">2</cn>
     </apply>
     <apply>
      <csymbol cd="ambiguous">superscript</csymbol>
      <apply>
       <csymbol cd="ambiguous">subscript</csymbol>
       <ci>X</ci>
       <ci>A</ci>
      </apply>
      <cn type="integer">2</cn>
     </apply>
    </apply>
    <apply>
     <eq></eq>
     <share href="#.cmml">
     </share>
     <apply>
      <divide></divide>
      <apply>
       <times></times>
       <cn type="integer">2</cn>
       <ci>N</ci>
       <apply>
        <csymbol cd="ambiguous">superscript</csymbol>
        <apply>
         <minus></minus>
         <apply>
          <times></times>
          <cn type="integer">2</cn>
          <ci>N</ci>
          <apply>
           <plus></plus>
           <apply>
            <csymbol cd="ambiguous">subscript</csymbol>
            <ci>r</ci>
            <cn type="integer">1</cn>
           </apply>
           <apply>
            <times></times>
            <cn type="integer">2</cn>
            <apply>
             <csymbol cd="ambiguous">subscript</csymbol>
             <ci>r</ci>
             <cn type="integer">2</cn>
            </apply>
           </apply>
          </apply>
         </apply>
         <apply>
          <times></times>
          <ci>R</ci>
          <apply>
           <plus></plus>
           <apply>
            <csymbol cd="ambiguous">subscript</csymbol>
            <ci>n</ci>
            <cn type="integer">1</cn>
           </apply>
           <apply>
            <times></times>
            <cn type="integer">2</cn>
            <apply>
             <csymbol cd="ambiguous">subscript</csymbol>
             <ci>n</ci>
             <cn type="integer">2</cn>
            </apply>
           </apply>
          </apply>
         </apply>
        </apply>
        <cn type="integer">2</cn>
       </apply>
      </apply>
      <apply>
       <times></times>
       <cn type="integer">4</cn>
       <ci>R</ci>
       <apply>
        <minus></minus>
        <ci>N</ci>
        <ci>R</ci>
       </apply>
       <apply>
        <minus></minus>
        <apply>
         <times></times>
         <cn type="integer">2</cn>
         <ci>N</ci>
         <apply>
          <plus></plus>
          <apply>
           <csymbol cd="ambiguous">subscript</csymbol>
           <ci>n</ci>
           <cn type="integer">1</cn>
          </apply>
          <apply>
           <times></times>
           <cn type="integer">2</cn>
           <apply>
            <csymbol cd="ambiguous">subscript</csymbol>
            <ci>n</ci>
            <cn type="integer">2</cn>
           </apply>
          </apply>
         </apply>
        </apply>
        <apply>
         <csymbol cd="ambiguous">superscript</csymbol>
         <apply>
          <plus></plus>
          <apply>
           <csymbol cd="ambiguous">subscript</csymbol>
           <ci>n</ci>
           <cn type="integer">1</cn>
          </apply>
          <apply>
           <times></times>
           <cn type="integer">2</cn>
           <apply>
            <csymbol cd="ambiguous">subscript</csymbol>
            <ci>n</ci>
            <cn type="integer">2</cn>
           </apply>
          </apply>
         </apply>
         <cn type="integer">2</cn>
        </apply>
       </apply>
      </apply>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \chi^{2}\sim X_{A}^{2}=\frac{2N(2N(r_{1}+2r_{2})-R(n_{1}+2n_{2}))^{2}}{4R(N-R)%
(2N(n_{1}+2n_{2})-(n_{1}+2n_{2})^{2})}
  </annotation>
 </semantics>
</math>

</p>
<table>
<thead>
<tr class="header">
<th style="text-align: left;">
<p>Alleles</p></th>
<th style="text-align: left;">
<p>aa</p></th>
<th style="text-align: left;">
<p>Aa</p></th>
<th style="text-align: left;">
<p>AA</p></th>
<th style="text-align: left;">
<p>total</p></th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;">
<p>Case</p></td>
<td style="text-align: left;">
<p>r0</p></td>
<td style="text-align: left;">
<p>r1</p></td>
<td style="text-align: left;">
<p>r2</p></td>
<td style="text-align: left;">
<p>R</p></td>
</tr>
<tr class="even">
<td style="text-align: left;">
<p>Control</p></td>
<td style="text-align: left;">
<p>s0</p></td>
<td style="text-align: left;">
<p>s1</p></td>
<td style="text-align: left;">
<p>s2</p></td>
<td style="text-align: left;">
<p>S</p></td>
</tr>
<tr class="odd">
<td style="text-align: left;">
<p>total</p></td>
<td style="text-align: left;">
<p>n0</p></td>
<td style="text-align: left;">
<p>n1</p></td>
<td style="text-align: left;">
<p>n2</p></td>
<td style="text-align: left;">
<p>N</p></td>
</tr>
</tbody>
</table>

<p>If the population is in Hardy-Weinberg equilibrium the two statistics are approximately equal. Under the null hypothesis of no population stratification the trend test is asymptotic 

<math display="inline" id="Population_stratification:3">
 <semantics>
  <msup>
   <mi>χ</mi>
   <mn>2</mn>
  </msup>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <csymbol cd="ambiguous">superscript</csymbol>
    <ci>χ</ci>
    <cn type="integer">2</cn>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \chi^{2}
  </annotation>
 </semantics>
</math>

 distribution with one degree of freedom. The idea is that the statistic is inflated by a factor 

<math display="inline" id="Population_stratification:4">
 <semantics>
  <mi>λ</mi>
  <annotation-xml encoding="MathML-Content">
   <ci>λ</ci>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \lambda
  </annotation>
 </semantics>
</math>

 so that 

<math display="inline" id="Population_stratification:5">
 <semantics>
  <mrow>
   <msup>
    <mi>Y</mi>
    <mn>2</mn>
   </msup>
   <mo>∼</mo>
   <mrow>
    <mi>λ</mi>
    <msubsup>
     <mi>χ</mi>
     <mn>1</mn>
     <mn>2</mn>
    </msubsup>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <csymbol cd="latexml">similar-to</csymbol>
    <apply>
     <csymbol cd="ambiguous">superscript</csymbol>
     <ci>Y</ci>
     <cn type="integer">2</cn>
    </apply>
    <apply>
     <times></times>
     <ci>λ</ci>
     <apply>
      <csymbol cd="ambiguous">superscript</csymbol>
      <apply>
       <csymbol cd="ambiguous">subscript</csymbol>
       <ci>χ</ci>
       <cn type="integer">1</cn>
      </apply>
      <cn type="integer">2</cn>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   Y^{2}\sim\lambda\chi_{1}^{2}
  </annotation>
 </semantics>
</math>

 where 

<math display="inline" id="Population_stratification:6">
 <semantics>
  <mi>λ</mi>
  <annotation-xml encoding="MathML-Content">
   <ci>λ</ci>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \lambda
  </annotation>
 </semantics>
</math>

 depends on the effect of stratification. The above method rests upon the assumption that the inflation factor 

<math display="inline" id="Population_stratification:7">
 <semantics>
  <mi>λ</mi>
  <annotation-xml encoding="MathML-Content">
   <ci>λ</ci>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \lambda
  </annotation>
 </semantics>
</math>

 is constant, which means that the loci should have roughly equal mutation rates, should not be under different selection in the two populations, and the amount of Hardy-Weinberg disequilibrium measured in Wright’s coefficient of inbreeding F should not differ between the different loci. The latter being of greatest concern. If the effect of the stratification is similar across the different loci 

<math display="inline" id="Population_stratification:8">
 <semantics>
  <mi>λ</mi>
  <annotation-xml encoding="MathML-Content">
   <ci>λ</ci>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \lambda
  </annotation>
 </semantics>
</math>

 can be estimated from the unlinked markers 

<math display="inline" id="Population_stratification:9">
 <semantics>
  <mrow>
   <mover accent="true">
    <mi>λ</mi>
    <mo stretchy="false">^</mo>
   </mover>
   <mo>=</mo>
   <mrow>
    <mrow>
     <mi>m</mi>
     <mi>e</mi>
     <mi>d</mi>
     <mi>i</mi>
     <mi>a</mi>
     <mi>n</mi>
     <mrow>
      <mo stretchy="false">(</mo>
      <msubsup>
       <mi>Y</mi>
       <mn>1</mn>
       <mn>2</mn>
      </msubsup>
      <mo>,</mo>
      <msubsup>
       <mi>Y</mi>
       <mn>2</mn>
       <mn>2</mn>
      </msubsup>
      <mo>,</mo>
      <mrow>
       <mi mathvariant="normal">…</mi>
       <msubsup>
        <mi>Y</mi>
        <mi>L</mi>
        <mn>2</mn>
       </msubsup>
      </mrow>
      <mo stretchy="false">)</mo>
     </mrow>
    </mrow>
    <mo>/</mo>
    <mn>0.456</mn>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <eq></eq>
    <apply>
     <ci>normal-^</ci>
     <ci>λ</ci>
    </apply>
    <apply>
     <divide></divide>
     <apply>
      <times></times>
      <ci>m</ci>
      <ci>e</ci>
      <ci>d</ci>
      <ci>i</ci>
      <ci>a</ci>
      <ci>n</ci>
      <vector>
       <apply>
        <csymbol cd="ambiguous">superscript</csymbol>
        <apply>
         <csymbol cd="ambiguous">subscript</csymbol>
         <ci>Y</ci>
         <cn type="integer">1</cn>
        </apply>
        <cn type="integer">2</cn>
       </apply>
       <apply>
        <csymbol cd="ambiguous">superscript</csymbol>
        <apply>
         <csymbol cd="ambiguous">subscript</csymbol>
         <ci>Y</ci>
         <cn type="integer">2</cn>
        </apply>
        <cn type="integer">2</cn>
       </apply>
       <apply>
        <times></times>
        <ci>normal-…</ci>
        <apply>
         <csymbol cd="ambiguous">superscript</csymbol>
         <apply>
          <csymbol cd="ambiguous">subscript</csymbol>
          <ci>Y</ci>
          <ci>L</ci>
         </apply>
         <cn type="integer">2</cn>
        </apply>
       </apply>
      </vector>
     </apply>
     <cn type="float">0.456</cn>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \hat{\lambda}=median(Y_{1}^{2},Y_{2}^{2},\ldots Y_{L}^{2})/0.456
  </annotation>
 </semantics>
</math>

 where L is the number of unlinked markers. The denominator is derived from the gamma distribution as a robust estimator of 

<math display="inline" id="Population_stratification:10">
 <semantics>
  <mi>λ</mi>
  <annotation-xml encoding="MathML-Content">
   <ci>λ</ci>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \lambda
  </annotation>
 </semantics>
</math>

. Other estimators have been suggested, for example, Reich and Goldstein<a class="footnoteRef" href="#fn8" id="fnref8"><sup>8</sup></a> suggested using the mean of the statistics instead. This is not the only way to estimate 

<math display="inline" id="Population_stratification:11">
 <semantics>
  <mi>λ</mi>
  <annotation-xml encoding="MathML-Content">
   <ci>λ</ci>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \lambda
  </annotation>
 </semantics>
</math>

 but according to Bacanu et al.<a class="footnoteRef" href="#fn9" id="fnref9"><sup>9</sup></a> it is an appropriate estimate even if some of the unlinked markers are actually in disequilibrium with a disease causing locus or are themselves associated with the disease. Under the null hypothesis and when correcting for stratification using L unlinked genes, 

<math display="inline" id="Population_stratification:12">
 <semantics>
  <msup>
   <mi>Y</mi>
   <mn>2</mn>
  </msup>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <csymbol cd="ambiguous">superscript</csymbol>
    <ci>Y</ci>
    <cn type="integer">2</cn>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   Y^{2}
  </annotation>
 </semantics>
</math>

 is approximately 

<math display="inline" id="Population_stratification:13">
 <semantics>
  <msubsup>
   <mi>χ</mi>
   <mn>1</mn>
   <mn>2</mn>
  </msubsup>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <csymbol cd="ambiguous">subscript</csymbol>
    <apply>
     <csymbol cd="ambiguous">superscript</csymbol>
     <ci>χ</ci>
     <cn type="integer">2</cn>
    </apply>
    <cn type="integer">1</cn>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \chi^{2}_{1}
  </annotation>
 </semantics>
</math>

 distributed. With this correction the overall type I error rate should be approximately equal to 

<math display="inline" id="Population_stratification:14">
 <semantics>
  <mi>α</mi>
  <annotation-xml encoding="MathML-Content">
   <ci>α</ci>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \alpha
  </annotation>
 </semantics>
</math>

 even when the population is stratified. Devlin and Roeder (1999)<a class="footnoteRef" href="#fn10" id="fnref10"><sup>10</sup></a> mostly considered the situation where 

<math display="inline" id="Population_stratification:15">
 <semantics>
  <mrow>
   <mi>α</mi>
   <mo>=</mo>
   <mn>0.05</mn>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <eq></eq>
    <ci>α</ci>
    <cn type="float">0.05</cn>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \alpha=0.05
  </annotation>
 </semantics>
</math>

 gives a 95% confidence level and not smaller p-values. Marchini et al. (2004)<a class="footnoteRef" href="#fn11" id="fnref11"><sup>11</sup></a> demonstrates by simulation that genomic control can lead to an anti-conservative p-value if this value is very small and the two populations (case and control) are extremely distinct. This was especially a problem if the number of unlinked markers were in the order 50 − 100. This can result in false positives (at that significance level).</p>
<h2 id="notes-and-references">Notes and references</h2>

<p>"</p>

<p><a href="Category:Population_genetics" title="wikilink">Category:Population genetics</a> <a href="Category:Medical_genetics" title="wikilink">Category:Medical genetics</a></p>
<section class="footnotes">
<hr/>
<ol>
<li id="fn1"><a href="Jonathan_K._Pritchard" title="wikilink">Pritchard, J.K.</a> and Rosenberg, N.A. (1999). Use of unlinked genetic markers to detect population stratification in association studies. American Journal of Human Genetics 65:220-228.<a href="#fnref1">↩</a></li>
<li id="fn2">Devlin, B. and Roeder, K. (1999). Genomic control for association studies, Biometrics 55(4): 997–1004.<a href="#fnref2">↩</a></li>
<li id="fn3"><a href="Jonathan_K._Pritchard" title="wikilink">Pritchard, JK</a>, Stephens, M, Rosenberg NA, and Donnelly P. (2000) Association mapping in structured populations. American Journal of Human Genetics 67: 170-181<a href="#fnref3">↩</a></li>
<li id="fn4">Price AL, Patterson NJ, Plenge RM, Weinblatt ME, Shadick NA, Reich D. (2006) Principal components analysis corrects for stratification in genome-wide association. Nature Genetics 38:904-909<a href="#fnref4">↩</a></li>
<li id="fn5">Lander, E. S. and Schork, N. J. (1994). Genetic dissection of complex traits, Science 265(5181): 2037–2048.<a href="#fnref5">↩</a></li>
<li id="fn6">Devlin, B. and Roeder, K. (1999). Genomic control for association studies, Biometrics 55(4): 997–1004.<a href="#fnref6">↩</a></li>
<li id="fn7">Bacanu, S.-A., Devlin, B. and Roeder, K. (2002). Association studies for quantitative traits in structured populations, Genet Epidemiol 22(1): 78–93.<a href="#fnref7">↩</a></li>
<li id="fn8">Reich, D. E. and Goldstein, D. B. (2001). Detecting association in a case-control study while correcting for population stratification, Genet Epidemiol 20(1): 4–16.<a href="#fnref8">↩</a></li>
<li id="fn9">Bacanu, S. A., Devlin, B. and Roeder, K. (2000). The power of genomic control, Am J Hum Genet 66(6): 1933–1944.<a href="#fnref9">↩</a></li>
<li id="fn10">Devlin, B. and Roeder, K. (1999). Genomic control for association studies, Biometrics 55(4): 997–1004.<a href="#fnref10">↩</a></li>
<li id="fn11">Marchini, J., Cardon, L. R., Phillips, M. S. and Donnelly, P. (2004). The effects of human population structure on large genetic association studies, Nat Genet 36(5): 512–517.<a href="#fnref11">↩</a></li>
</ol>
</section>
</body>
</html>
