<html lang="en">
<head>
<meta charset="utf-8"/>
<title offset="818">Power graph analysis</title>
<script src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_SVG.js" type="text/javascript">
</script>
</head>
<body>
<h1>Power graph analysis</h1>
<hr/>

<p>In <a href="computational_biology" title="wikilink">computational biology</a>, <strong>power graph analysis</strong> is a method for the analysis and representation of <a href="complex_network" title="wikilink">complex networks</a>. Power graph analysis is the computation, analysis and visual representation of a power graph from a <a href="Graph_(mathematics)" title="wikilink">graph</a> (<a href="network_(mathematics)" title="wikilink">networks</a>).</p>

<p>Power graph analysis can be thought of as a <a href="Lossless_data_compression" title="wikilink">lossless compression algorithm</a> for graphs.<a class="footnoteRef" href="#fn1" id="fnref1"><sup>1</sup></a> It extends graph syntax with representations of <a href="Clique_(graph_theory)" title="wikilink">cliques</a>, <a href="Complete_bipartite_graph" title="wikilink">bicliques</a> and <a href="star_graph" title="wikilink">stars</a>. Compression levels of up to 95% have been obtained for complex <a href="biological_network" title="wikilink">biological networks</a>.</p>

<p><a href="Hypergraph" title="wikilink">Hypergraphs</a> are a generalization of graphs in which <a href="edge_(graph_theory)" title="wikilink">edges</a> are not just couples of <a href="node_(graph_theory)" title="wikilink">nodes</a> but arbitrary <a href="n-tuple" title="wikilink">n-tuples</a>. Power graphs are not another generalization of graphs, but instead a novel representation of graphs that proposes a shift from the "node and edge" language to one using cliques, bicliques and stars as primitives.</p>
<h2 id="power-graphs">Power graphs</h2>
<figure><b>(Figure)</b>
<figcaption>The primitive motifs used for power graph analysis and their corresponding diagrammatic representation: biclique, clique and star.</figcaption>
</figure>
<h3 id="graphical-representation">Graphical representation</h3>

<p>Graphs are drawn with circles or points that represent <strong>nodes</strong> and lines connecting pairs of nodes that represent <strong>edges</strong>. Power graphs extend the syntax of graphs with <strong>power nodes</strong>, which are drawn as a circle enclosing nodes or <em>other power nodes</em>, and <strong>power edges</strong>, which are lines between power nodes.</p>

<p><a href="Complete_bipartite_graph" title="wikilink">Bicliques</a> are two sets of nodes with an edge between every member of one set and every member of the other set. In a power graph, a biclique is represented as an edge between two power nodes.</p>

<p><a href="Clique_(graph_theory)" title="wikilink">Cliques</a> are a set of nodes with an edge between every pair of nodes. In a power graph, a clique is represented by a power node with a <a href="Loop_(graph_theory)" title="wikilink">loop</a>.</p>

<p><a href="Star_(graph_theory)" title="wikilink">Stars</a> are a set of nodes with an edge between every member of that set and a single node outside the set. In a power graph, a star is represented by a power edge between a regular node and a power node.</p>
<h3 id="formal-definition">Formal definition</h3>

<p>Given a graph 

<math display="inline" id="Power_graph_analysis:0">
 <semantics>
  <mrow>
   <mi>G</mi>
   <mo>=</mo>
   <mrow>
    <mo maxsize="120%" minsize="120%">(</mo>
    <mi>V</mi>
    <mo>,</mo>
    <mi>E</mi>
    <mo maxsize="120%" minsize="120%">)</mo>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <eq></eq>
    <ci>G</ci>
    <interval closure="open">
     <ci>V</ci>
     <ci>E</ci>
    </interval>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   G=\bigl({V,E}\bigr)
  </annotation>
 </semantics>
</math>


 where 

<math display="inline" id="Power_graph_analysis:1">
 <semantics>
  <mrow>
   <mi>V</mi>
   <mo>=</mo>
   <mrow>
    <mo maxsize="120%" minsize="120%">{</mo>
    <msub>
     <mi>v</mi>
     <mn>0</mn>
    </msub>
    <mo>,</mo>
    <mi mathvariant="normal">‚Ä¶</mi>
    <mo>,</mo>
    <msub>
     <mi>v</mi>
     <mi>n</mi>
    </msub>
    <mo maxsize="120%" minsize="120%">}</mo>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <eq></eq>
    <ci>V</ci>
    <set>
     <apply>
      <csymbol cd="ambiguous">subscript</csymbol>
      <ci>v</ci>
      <cn type="integer">0</cn>
     </apply>
     <ci>normal-‚Ä¶</ci>
     <apply>
      <csymbol cd="ambiguous">subscript</csymbol>
      <ci>v</ci>
      <ci>n</ci>
     </apply>
    </set>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   V=\bigl\{v_{0},\dots,v_{n}\bigr\}
  </annotation>
 </semantics>
</math>

 is the set of nodes and 

<math display="inline" id="Power_graph_analysis:2">
 <semantics>
  <mrow>
   <mi>E</mi>
   <mo>‚äÜ</mo>
   <mrow>
    <mi>V</mi>
    <mo>√ó</mo>
    <mi>V</mi>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <subset></subset>
    <ci>E</ci>
    <apply>
     <times></times>
     <ci>V</ci>
     <ci>V</ci>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   E\subseteq V\times V
  </annotation>
 </semantics>
</math>

 is the set of edges, a <strong>power graph</strong> 

<math display="inline" id="Power_graph_analysis:3">
 <semantics>
  <mrow>
   <msup>
    <mi>G</mi>
    <mo>‚Ä≤</mo>
   </msup>
   <mo>=</mo>
   <mrow>
    <mo maxsize="120%" minsize="120%">(</mo>
    <msup>
     <mi>V</mi>
     <mo>‚Ä≤</mo>
    </msup>
    <mo>,</mo>
    <msup>
     <mi>E</mi>
     <mo>‚Ä≤</mo>
    </msup>
    <mo maxsize="120%" minsize="120%">)</mo>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <eq></eq>
    <apply>
     <csymbol cd="ambiguous">superscript</csymbol>
     <ci>G</ci>
     <ci>normal-‚Ä≤</ci>
    </apply>
    <interval closure="open">
     <apply>
      <csymbol cd="ambiguous">superscript</csymbol>
      <ci>V</ci>
      <ci>normal-‚Ä≤</ci>
     </apply>
     <apply>
      <csymbol cd="ambiguous">superscript</csymbol>
      <ci>E</ci>
      <ci>normal-‚Ä≤</ci>
     </apply>
    </interval>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   G^{\prime}=\bigl({V^{\prime},E^{\prime}}\bigr)
  </annotation>
 </semantics>
</math>

 is a graph defined on the power set 

<math display="inline" id="Power_graph_analysis:4">
 <semantics>
  <mrow>
   <msup>
    <mi>V</mi>
    <mo>‚Ä≤</mo>
   </msup>
   <mo>‚äÜ</mo>
   <mrow>
    <mi class="ltx_font_mathcaligraphic">ùí´</mi>
    <mrow>
     <mo maxsize="120%" minsize="120%">(</mo>
     <mi>V</mi>
     <mo maxsize="120%" minsize="120%">)</mo>
    </mrow>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <subset></subset>
    <apply>
     <csymbol cd="ambiguous">superscript</csymbol>
     <ci>V</ci>
     <ci>normal-‚Ä≤</ci>
    </apply>
    <apply>
     <times></times>
     <ci>ùí´</ci>
     <ci>V</ci>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   V^{\prime}\subseteq\mathcal{P}\bigl(V\bigr)
  </annotation>
 </semantics>
</math>

 of <strong>power nodes</strong> connected to each other by <strong>power edges</strong>

<math display="block" id="Power_graph_analysis:5">
 <semantics>
  <mrow>
   <msup>
    <mi>E</mi>
    <mo>‚Ä≤</mo>
   </msup>
   <mo>‚äÜ</mo>
   <mrow>
    <msup>
     <mi>V</mi>
     <mo>‚Ä≤</mo>
    </msup>
    <mo>√ó</mo>
    <msup>
     <mi>V</mi>
     <mo>‚Ä≤</mo>
    </msup>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <subset></subset>
    <apply>
     <csymbol cd="ambiguous">superscript</csymbol>
     <ci>E</ci>
     <ci>normal-‚Ä≤</ci>
    </apply>
    <apply>
     <times></times>
     <apply>
      <csymbol cd="ambiguous">superscript</csymbol>
      <ci>V</ci>
      <ci>normal-‚Ä≤</ci>
     </apply>
     <apply>
      <csymbol cd="ambiguous">superscript</csymbol>
      <ci>V</ci>
      <ci>normal-‚Ä≤</ci>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   E^{\prime}\subseteq V^{\prime}\times V^{\prime}
  </annotation>
 </semantics>
</math>

. Hence power graphs are defined on the <a href="power_set" title="wikilink">power set</a> of nodes as well as on the <a href="power_set" title="wikilink">power set</a> of edges of the graph 

<math display="inline" id="Power_graph_analysis:6">
 <semantics>
  <mi>G</mi>
  <annotation-xml encoding="MathML-Content">
   <ci>G</ci>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   G
  </annotation>
 </semantics>
</math>

.</p>

<p>The semantics of power graphs are as follows: if two power nodes are connected by a power edge, this means that all nodes of the first power node are connected to all nodes of the second power node. Similarly, if a power node is connected to itself by a power edge, this signifies that all nodes in the power node are connected to each other by edges.</p>

<p>The following two conditions are required:</p>
<ul>
<li>Power node hierarchy condition: Any two power nodes are either disjoint, or one is included in the other.</li>
<li>Power edge disjointness condition: There is an <a href="Surjective_function" title="wikilink">onto mapping</a> from edges of the original graph to power edges.</li>
</ul>
<h2 id="analogy-to-fourier-analysis">Analogy to Fourier analysis</h2>

<p>The <a href="Fourier_analysis" title="wikilink">Fourier analysis</a> of a function can be seen as a rewriting of the function in terms of harmonic functions instead of 

<math display="inline" id="Power_graph_analysis:7">
 <semantics>
  <mrow>
   <mi>t</mi>
   <mo>‚Ü¶</mo>
   <mi>x</mi>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <csymbol cd="latexml">maps-to</csymbol>
    <ci>t</ci>
    <ci>x</ci>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   t\mapsto x
  </annotation>
 </semantics>
</math>

 pairs. This transformation changes the point of view from <strong>time domain</strong> to <strong>frequency domain</strong> and enables many interesting applications in <a href="signal_analysis" title="wikilink">signal analysis</a>, <a href="data_compression" title="wikilink">data compression</a>, and filtering. Similarly, Power Graph Analysis is a rewriting or decomposition of a network using bicliques, cliques and stars as primitive elements (just as harmonic functions for Fourier analysis). It can be used to analyze, compress and filter networks. There are, however, several key differences. First, in Fourier analysis the two spaces (time and frequency domains) are the same function space - but stricto sensu, power graphs are not graphs. Second, there is not a unique power graph representing a given graph. Yet a very interesting class of power graphs are <strong>minimal power graphs</strong> which have the least number of power edges and power nodes necessary to represent a given graph.</p>
<h2 id="minimal-power-graphs">Minimal power graphs</h2>

<p> In general, there is no unique minimal power graph for a given graph. In this example (right) a graph of four nodes and five edges admits two minimal power graphs of two power edges each. The main difference between these two minimal power graphs is the higher nesting level of the second power graph as well as a loss of symmetry with respect to the underlying graph. Loss of symmetry is only a problem in small toy examples since complex networks rarely exhibit such symmetries in the first place. Additionally, one can minimize the nesting level but even then, there is in general not a unique minimal power graph of minimal nesting level.</p>
<h2 id="power-graph-greedy-algorithm">Power graph greedy algorithm</h2>

<p>The power graph greedy algorithm relies on two simple steps to perform the decomposition:</p>

<p>The <strong>first step</strong> identifies candidate power nodes through a <a href="hierarchical_clustering" title="wikilink">hierarchical clustering</a> of the nodes in the network based on the similarity of their neighboring nodes. The similarity of two sets of neighbors is taken as the <a href="Jaccard_index" title="wikilink">Jaccard index</a> of the two sets.</p>

<p>The <strong>second step</strong> performs a greedy search for possible power edges between candidate power nodes. Power edges abstracting the most edges in the original network are added first to the power graph. Thus bicliques, cliques and stars are incrementally replaced with power edges, until all remaining single edges are also added. Candidate power nodes that are not the end point of any power edge are ignored.</p>
<h2 id="modular-decomposition">Modular decomposition</h2>

<p><a href="Modular_decomposition" title="wikilink">Modular decomposition</a> can be used to compute a power graph by using the strong modules of the modular decomposition. Modules in modular decomposition are groups of nodes in a graph that have identical neighbors. A Strong Module is a module that does not overlap with another module. However, in <a href="complex_networks" title="wikilink">complex networks</a> strong modules are more the exception than the rule. Therefore the power graphs obtained through modular decomposition are far from minimality. The main difference between modular decomposition and power graph analysis is the emphasis of power graph analysis in decomposing graphs not only using modules of nodes but also modules of edges (cliques, bicliques). Indeed, power graph analysis can be seen as a loss-less simultaneous clustering of both nodes and edges.</p>
<h2 id="applications">Applications</h2>
<h3 id="biological-networks">Biological networks</h3>

<p>Power Graph Analysis has been shown to be useful for the analysis of several types of biological networks such as <a href="Protein-protein_interaction" title="wikilink">Protein-protein interaction</a> networks,<a class="footnoteRef" href="#fn2" id="fnref2"><sup>2</sup></a> domain-peptide binding motifs, <a href="Gene_regulatory_networks" title="wikilink">Gene regulatory networks</a><a class="footnoteRef" href="#fn3" id="fnref3"><sup>3</sup></a> and Homology/Paralogy networks. Also a network of significant disease-trait pairs<a class="footnoteRef" href="#fn4" id="fnref4"><sup>4</sup></a> have been recently visualized and analyzed with Power Graphs.</p>

<p>Network compression, a new measure derived from Power Graphs, has been proposed as a quality measure for protein interaction networks.<a class="footnoteRef" href="#fn5" id="fnref5"><sup>5</sup></a></p>
<h3 id="drug-repositioning">Drug repositioning</h3>

<p>Power Graphs have been also applied to the analysis of drug-target-disease networks<a class="footnoteRef" href="#fn6" id="fnref6"><sup>6</sup></a> for <a href="Drug_repositioning" title="wikilink">Drug repositioning</a>.</p>
<h3 id="social-networks">Social networks</h3>

<p>Power Graphs have been applied to large-scale data in social networks, for community mining<a class="footnoteRef" href="#fn7" id="fnref7"><sup>7</sup></a> or for modeling author types.<a class="footnoteRef" href="#fn8" id="fnref8"><sup>8</sup></a></p>
<h2 id="see-also">See also</h2>
<ul>
<li><a href="Computational_biology" title="wikilink">Computational biology</a></li>
<li><a href="network_(mathematics)" title="wikilink">Networks</a>/<a href="Graph_(mathematics)" title="wikilink">Graph</a></li>
<li><a href="Complex_networks" title="wikilink">Complex networks</a></li>
<li><a href="Modular_decomposition" title="wikilink">Modular decomposition</a></li>
</ul>
<h2 id="references">References</h2>
<h2 id="external-links">External links</h2>
<ul>
<li><a href="http://www.biotec.tu-dresden.de/research/schroeder/powergraphs/">1</a> Power Graph Analysis tools (CyOog v2.8.2) and example applications</li>
<li><a href="http://wiki.cytoscape.org/CyOogPowerGraphAnalysisRecipe">2</a> Power Graph Analysis with CyOog v2.6</li>
</ul>

<p>"</p>

<p><a href="Category:Computational_science" title="wikilink">Category:Computational science</a> <a class="uri" href="Category:Bioinformatics" title="wikilink">Category:Bioinformatics</a> <a href="Category:Application-specific_graphs" title="wikilink">Category:Application-specific graphs</a></p>
<section class="footnotes">
<hr/>
<ol>
<li id="fn1"></li>
<li id="fn2"></li>
<li id="fn3"></li>
<li id="fn4"></li>
<li id="fn5"></li>
<li id="fn6"></li>
<li id="fn7"></li>
<li id="fn8"></li>
</ol>
</section>
</body>
</html>
