<html lang="en">
<head>
<meta charset="utf-8"/>
<title offset="613">Protein contact map</title>
<script src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_SVG.js" type="text/javascript">
</script>
</head>
<body>
<h1>Protein contact map</h1>
<hr/>

<p> A <strong>protein contact map</strong> represents the distance between all possible <a href="Amino_acid" title="wikilink">amino acid residue</a> pairs of a three-dimensional <a href="protein_structure" title="wikilink">protein structure</a> using a binary two-dimensional <a href="Matrix_(mathematics)" title="wikilink">matrix</a>. For two residues 

<math display="inline" id="Protein_contact_map:0">
 <semantics>
  <mi>i</mi>
  <annotation-xml encoding="MathML-Content">
   <ci>i</ci>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   i
  </annotation>
 </semantics>
</math>

 and 

<math display="inline" id="Protein_contact_map:1">
 <semantics>
  <mi>j</mi>
  <annotation-xml encoding="MathML-Content">
   <ci>j</ci>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   j
  </annotation>
 </semantics>
</math>

, the 

<math display="inline" id="Protein_contact_map:2">
 <semantics>
  <mrow>
   <mi>i</mi>
   <mi>j</mi>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <times></times>
    <ci>i</ci>
    <ci>j</ci>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   ij
  </annotation>
 </semantics>
</math>

 element of the matrix is 1 if the two residues are closer than a predetermined threshold, and 0 otherwise. Various contact definitions have been proposed: The distance between the C<sub>α</sub>-C<sub>α</sub> atom with threshold 6-12 <a href="Ångström" title="wikilink">Å</a>; distance between C<sub>β</sub>-C<sub>β</sub> atoms with threshold 6-12 Å (C<sub>α</sub> is used for <a class="uri" href="Glycine" title="wikilink">Glycine</a>); and distance between the side-chain <a href="Center_of_mass" title="wikilink">centers of mass</a>.</p>
<h2 id="overview">Overview</h2>

<p>Contact maps provide a more reduced representation of a protein structure than its full 3D atomic coordinates. The advantage is that contact maps are invariant to rotations and translations. They are more easily predicted by <a href="machine_learning" title="wikilink">machine learning</a> methods. It has also been shown that under certain circumstances it is possible to reconstruct the 3D coordinates of a protein using its contact map.<a class="footnoteRef" href="#fn1" id="fnref1"><sup>1</sup></a></p>

<p>Contact maps are also used for protein <a class="uri" href="superimposition" title="wikilink">superimposition</a> and to describe similarity between protein structures.<a class="footnoteRef" href="#fn2" id="fnref2"><sup>2</sup></a> They are either predicted from <a href="protein_sequence" title="wikilink">protein sequence</a> or calculated from a given structure.</p>
<h2 id="contact-map-prediction">Contact map prediction</h2>

<p>With the availability of high numbers of genomic sequences it becomes feasible to analyze such sequences for <em><a href="coevolution" title="wikilink">coevolving</a> residues</em>. The effectiveness of this approach results from the fact that a mutation in position <em>i</em> of a protein is more likely to be associated with a mutation in position <em>j</em> than with a back-mutation in <em>i</em> if both positions are functionally coupled (e.g. by taking part in an enzymatic domain, or by being adjacent in a folded protein, or even by being adjacent in an oligomer of that protein).<a class="footnoteRef" href="#fn3" id="fnref3"><sup>3</sup></a></p>

<p>Several statistical methods exist to extract from a <a href="multiple_sequence_alignment" title="wikilink">multiple sequence alignment</a> such coupled residue pairs: observed versus expected frequencies of residue pairs (OMES);<a class="footnoteRef" href="#fn4" id="fnref4"><sup>4</sup></a> the McLachlan Based Substitution correlation (McBASC);<a class="footnoteRef" href="#fn5" id="fnref5"><sup>5</sup></a> <a href="statistical_coupling_analysis" title="wikilink">statistical coupling analysis</a>; <a href="Mutual_Information" title="wikilink">Mutual Information</a> (MI) based methods;<a class="footnoteRef" href="#fn6" id="fnref6"><sup>6</sup></a> and, most recently, <a href="direct_coupling_analysis" title="wikilink">direct coupling analysis</a> (DCA)<a class="footnoteRef" href="#fn7" id="fnref7"><sup>7</sup></a><a class="footnoteRef" href="#fn8" id="fnref8"><sup>8</sup></a></p>

<p>Predicted contact maps have been used in the prediction of <a href="membrane_proteins" title="wikilink">membrane proteins</a> where helix-helix interactions are targeted.<a class="footnoteRef" href="#fn9" id="fnref9"><sup>9</sup></a></p>
<h2 id="hb-plot">HB Plot</h2>

<p>Knowledge of the relationship between a <a class="uri" href="protein" title="wikilink">protein</a>'s structure and its dynamic behavior is essential for understanding protein function. The description of a protein three dimensional structure as a network of <a href="hydrogen_bonding" title="wikilink">hydrogen bonding</a> interactions (<strong>HB plot</strong>)<a class="footnoteRef" href="#fn10" id="fnref10"><sup>10</sup></a> was introduced as a tool for exploring protein structure and function. By analyzing the network of tertiary interactions the possible spread of information within a protein can be investigated.</p>

<p>HB plot offers a simple way of analyzing protein <a href="secondary_structure" title="wikilink">secondary structure</a> and <a href="tertiary_structure" title="wikilink">tertiary structure</a>. <a href="Hydrogen_bonds" title="wikilink">Hydrogen bonds</a> stabilizing secondary structural elements (<strong><a href="secondary_hydrogen_bond" title="wikilink">secondary hydrogen bonds</a></strong>) and those formed between distant <a href="amino_acid" title="wikilink">amino acid</a> residues - defined as <strong><a href="tertiary_hydrogen_bond" title="wikilink">tertiary hydrogen bonds</a></strong> - can be easily distinguished in HB plot, thus, amino acid residues involved in stabilizing <a href="protein_structure" title="wikilink">protein structure</a> and function can be identified.</p>
<h3 id="features">Features</h3>

<p>The plot distinguishes between main chain-main chain, main chain-<a href="side_chain" title="wikilink">side chain</a> and side chain-side chain <a href="hydrogen_bonding" title="wikilink">hydrogen bonding</a> interactions. Bifurcated hydrogen bonds and multiple hydrogen bonds between <a href="amino_acid" title="wikilink">amino acid</a> residues; and intra- and interchain <a href="hydrogen_bonds" title="wikilink">hydrogen bonds</a> are also indicated on the plots. Three classes of hydrogen bondings are distinguished by color-coding; short (distance smaller than 2.5 <a href="Ångström" title="wikilink">Å</a> between donor and acceptor), intermediate (between 2.5 Å and 3.2 Å) and long hydrogen bonds (greater than 3.2 Å).</p>
<h3 id="secondary-structure-elements-in-hb-plot">Secondary structure elements in HB plot</h3>
<figure><b>(Figure)</b>
<figcaption>Secondary structure elements in HB plot, there is swaped parallel and anti-parallel sheets|300px</figcaption>
</figure>

<p>In representations of the HB plot, characteristic patterns of <a href="secondary_structure" title="wikilink">secondary structure</a> elements can be recognised easily, as follows:</p>
<ol>
<li><a class="uri" href="Helices" title="wikilink">Helices</a> can be identified as strips directly adjacent to the diagonal.</li>
<li>Antiparallel <a href="beta_sheet" title="wikilink">beta sheets</a> appear in HB plot as cross-diagonal.</li>
<li>Parallel <a href="beta_sheet" title="wikilink">beta sheets</a> appears in the HB plot as parallel to the diagonal.</li>
<li><a href="Loop_(graph_theory)" title="wikilink">Loops</a> appear as breaks in the diagonal between the cross-diagonal <a class="uri" href="beta-sheet" title="wikilink">beta-sheet</a> motifs.</li>
</ol>
<h3 id="examples-of-usage">Examples of usage</h3>
<h4 id="cytochrome-p450s">Cytochrome P450s</h4>

<p><strong>The <a href="cytochrome_P450" title="wikilink">cytochrome P450s</a> (P450s)</strong> are <a class="uri" href="xenobiotic" title="wikilink">xenobiotic</a>-metabolizing <a href="Membrane_(selective_barrier)" title="wikilink">membrane</a>-bound <a class="uri" href="heme" title="wikilink">heme</a>-containing enzymes that use molecular <a class="uri" href="oxygen" title="wikilink">oxygen</a> and electrons from <a href="NADPH_cytochrome_P450_reductase" title="wikilink">NADPH cytochrome P450 reductase</a> to oxidize their <a href="Substrate_(biochemistry)" title="wikilink">substrates</a>. <a class="uri" href="CYP2B4" title="wikilink">CYP2B4</a>, a member of the cytochrome P450 family is the only protein within this family, whose <a href="X-ray_crystallography" title="wikilink">X-ray structure</a> in both open 11 and closed form 12 is published. The comparison of the open and closed structures of CYP2B4 structures reveals large-scale <a href="Conformational_change" title="wikilink">conformational</a> rearrangement between the two states, with the greatest conformational change around the residues 215-225, which is widely open in ligand-free state and shut after ligand binding; and the region around loop C near the heme.</p>
<figure><b>(Figure)</b>
<figcaption>HB Plot and structure of Cytochrome P450 2B4 in closed form</figcaption>
</figure>

<p>Examining the HB plot of the closed and open state of CYP2B4 revealed that the rearrangement of tertiary hydrogen bonds was in excellent agreement with the current knowledge of the cytochrome P450 <a href="catalytic_cycle" title="wikilink">catalytic cycle</a>.</p>

<p>The first step in <a class="uri" href="P450" title="wikilink">P450</a> catalytic cycle is identified as substrate binding. Preliminary binding of a ligand near to the entrance breaks hydrogen bonds S212-E474, S207-H172 in the open form of CYP2B4 and hydrogen bonds E218-A102, Q215-L51 are formed that fix the entrance in the closed form as the HB plot reveals.</p>

<p>The second step is the transfer of the first electron from <a class="uri" href="NADPH" title="wikilink">NADPH</a> via an electron transfer chain. For the electron transfer a conformational change occurs that triggers interaction of the P450 with the NADPH cytochrome P450 reductase. Breaking of hydrogen bonds between S128-N287, S128-T291, L124-N287 and forming S96-R434, A116-R434, R125-I435, D82-R400 at the NADPH cytochrome P450 reductase <a href="binding_site" title="wikilink">binding site</a>—as seen in HB plot—transform CYP2B4 to a conformation state, where binding of NADPH cytochrome P450 reductase occurs.</p>

<p>In the third step, oxygen enters CYP2B4 in the closed state - the state where newly formed hydrogen bonds S176-T300, H172-S304, N167-R308 open a tunnel which is exactly the size and shape of an <a class="uri" href="oxygen" title="wikilink">oxygen</a> molecule.</p>
<h3 id="lipocalin-family">Lipocalin family</h3>

<p> The <strong><a class="uri" href="lipocalin" title="wikilink">lipocalin</a> family</strong> is a large and diverse family of proteins with functions as small <a class="uri" href="hydrophobic" title="wikilink">hydrophobic</a> molecule transporters. <a class="uri" href="Beta-lactoglobulin" title="wikilink">Beta-lactoglobulin</a> is a typical member of the lipocalin family. Beta-lactoglobulin was found to have a role in the transport of hydrophobic ligands such as <a class="uri" href="retinol" title="wikilink">retinol</a> or <a href="fatty_acid" title="wikilink">fatty acids</a>.<a class="footnoteRef" href="#fn11" id="fnref11"><sup>11</sup></a> Its <a href="crystal_structure" title="wikilink">crystal structure</a> were determined [e.g. Qin, 1998] with different ligands and in ligand-free form as well. The crystal structures determined so far reveal that the typical lipocalin contains eight-stranded <a href="Antiparallel_(biochemistry)" title="wikilink">antiparallel</a>-barrel arranged to form a conical central cavity in which the hydrophobic ligand is bound. The structure of beta-lactoglobulin reveals that the barrel-form structure with the central cavity of the protein has an "entrance" surrounded by five <a href="beta-loop" title="wikilink">beta-loops</a> with centers around 26, 35, 63, 87, and 111, which undergo a conformational change during the ligand binding and close the cavity.</p>

<p>The overall shape of beta-lactoglobulin is characteristic of the lipocalin family. In the absence of <a class="uri" href="alpha-helices" title="wikilink">alpha-helices</a>, the main diagonal almost disappears and the cross-diagonals representing the <a href="beta-sheet" title="wikilink">beta-sheets</a> dominate the plot. Relatively low number of tertiary hydrogen bonds can be found in the plot, with three high-density regions, one of which is connected to a loop at the residues around 63, a second is connected to the loop around 87, and a third region which is connected to the regions 26 and 35. The fifth loop around 111 is represented only one tertiary hydrogen bond in the HB plot.</p>

<p>In the three-dimensional structure, tertiary hydrogen bonds are formed (1) near to the entrance, directly involved in conformational rearrangement during ligand binding; and (2) at the bottom of the "barrel". HB plots of the open and closed forms of beta-lactoglobulin are very similar, all unique motifs can be recognized in both forms. Difference in HB plots of open and ligand-bound form show few important individual changes in tertiary hydrogen bonding pattern. Especially, the formation of hydrogen bonds between Y20-E157 and S21-H161 in closed form might be crucial in conformational rearrangement. These hydrogen bonds lie at the bottom of the cavity, which suggests that the closure of the entrance of a lipocalin starts when a ligand reached the bottom of the cavity and broke hydrogen bonds R123-Y99, R123-T18, and V41-Q120. Lipocalins are known to have very low sequence similarity with high structural similarity. The only conserved regions are exactly the region around 20 and 160 with an unknown role.</p>
<figure><b>(Figure)</b>
<figcaption>HB Plots of beta-lactoglobulin in open (2BLG) and ligand-bound (2AKQ) form|700px</figcaption>
</figure>
<h2 id="see-also">See also</h2>
<ul>
<li><a href="Ramachandran_plot" title="wikilink">Ramachandran plot</a></li>
<li><a href="Structural_classification_of_proteins" title="wikilink">Structural classification of proteins</a></li>
<li><a class="uri" href="CATH" title="wikilink">CATH</a></li>
<li><a href="HB_plot" title="wikilink">HB plot</a></li>
<li><a href="Dot_plot_(bioinformatics)" title="wikilink">Dot plot (bioinformatics)</a></li>
</ul>
<h2 id="references">References</h2>
<h2 id="external-links">External links</h2>
<ul>
<li><a href="http://distill.ucd.ie/distill/">DISTILL</a> — prediction of protein structural features (including protein residue contact maps)</li>
<li>[<a class="uri" href="http://www.igb.uci.edu/?page=tools&amp;subPage">http://www.igb.uci.edu/?page=tools&amp;subPage;</a>;=psss Structural Proteomics Tools] — includes amino acid contact maps</li>
<li><a href="http://rostlab.org/services/profcon/index.php">ProfCon</a> — prediction of inter-residue contacts</li>
<li><a href="http://webclu.bio.wzw.tum.de/tmhcon/">TMHcon</a> — prediction of helix-helix contacts specifically within the transmembrane parts of membrane proteins</li>
<li><a href="http://bio-cluster.iis.sinica.edu.tw/TMhit/">TMhit</a> — A new transmembrane helix-helix interaction prediction method based on residue contacts</li>
<li><a href="http://www.ics.uci.edu/~baldig/index.html">CMAPpro</a> — A protein contact map prediction server</li>
<li><a href="http://emptyewer.github.io/CMPyMOL/">CMPyMOL</a> —A Tool for Protein Contact-Map Visualization in PyMOL</li>
</ul>

<p>"</p>

<p><a class="uri" href="Category:Proteomics" title="wikilink">Category:Proteomics</a> <a href="Category:Protein_structure" title="wikilink">Category:Protein structure</a></p>
<section class="footnotes">
<hr/>
<ol>
<li id="fn1"><a href="#fnref1">↩</a></li>
<li id="fn2"><a href="#fnref2">↩</a></li>
<li id="fn3">Fitch, W. M. and Markowitz, E. (1970): <em>An improved method for determining codon variability in a gene and its application to the rate of fixation of mutations in evolution.</em> In: <em>Biochem. Genet.</em>, 4, 579–593.<a href="#fnref3">↩</a></li>
<li id="fn4">Kass, I. and Horovitz, A. (2002): <em>Mapping pathways of allosteric communication in GroEL by analysis of correlated mutations.</em> In: <em>Proteins</em>, 48, 611–617.<a href="#fnref4">↩</a></li>
<li id="fn5">Gobel, U. et al. (1994): <em>Correlated mutations and residue contacts in proteins.</em> In: <em>Proteins</em>, 18, 309–317.<a href="#fnref5">↩</a></li>
<li id="fn6">Wollenberg, K. R. and Atchley, W. R. (2000): <em>Separation of phylogenetic and functional associations in biological sequences by using the parametric bootstrap.</em> In: <em>Proc. Natl Acad. Sci. USA</em>, 97, 3288–3291.<a href="#fnref6">↩</a></li>
<li id="fn7">Weigt M, White RA, Szurmant H, Hoch JA, Hwa T (2009): <em>Identification of direct residue contacts in protein–protein interaction by message passing.</em> In: <em>Proc Natl Acad Sci USA</em> 106:67–72.<a href="#fnref7">↩</a></li>
<li id="fn8">Morcos F, et al. (2011): <em>Direct-coupling analysis of residue coevolution captures native contacts across many protein families.</em> In: <em>Proc Natl Acad Sci USA</em> 108:E1293–E1301.<a href="#fnref8">↩</a></li>
<li id="fn9"><a href="#fnref9">↩</a></li>
<li id="fn10"><a href="#fnref10">↩</a></li>
<li id="fn11"><a href="#fnref11">↩</a></li>
</ol>
</section>
</body>
</html>
