<html lang="en">
<head>
<meta charset="utf-8"/>
<title offset="1810">Quantitative genetics</title>
<script src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_SVG.js" type="text/javascript">
</script>
</head>
<body>
<h1>Quantitative genetics</h1>
<hr/>

<p><strong>Quantitative genetics</strong> is that branch of <a href="population_genetics" title="wikilink">population genetics</a> which deals with <a href="phenotype" title="wikilink">phenotypes</a> which vary continuously (such as height or mass), rather than with phenotypes and gene-products which are discretely identifiable (such as eye-colour, or the presence of a particular biochemical). Both employ the frequencies of different <a href="allele" title="wikilink">alleles</a> of a <a class="uri" href="gene" title="wikilink">gene</a> in breeding populations (gamodemes), and combine them with concepts arising from simple <a href="Mendelian_inheritance" title="wikilink">Mendelian inheritance</a> in order to analyze inheritance patterns across generations and descendant lines. While <a href="population_genetics" title="wikilink">population genetics</a> can focus on particular genes and their subsequent metabolic products, quantitative genetics focuses more on the outward phenotypes, and makes summaries only of the underlying genetics. This, however, can be viewed as its strength, because it facilitates an interface with the biological macrocosm, including micro-evolution and artificial selection in plant and animal breeding. Both branches share some common history; and some mathematics: for example, they use expansion of the quadratic equation to represent the fertilization of gametes to form the zygote. However, because of the continuous distribution of phenotypic values, quantitative genetics needs also to employ many other statistical methods (such as the <em>effect</em>, the <em>mean</em> and the <em>variance</em>) in order to link the phenotype to underlying genetics principles. Some phenotypes (attributes) may be analyzed either as discrete categories or as continuous phenotypes, depending on the definition of cut-off points, or on the <em>metric</em> used to quantify them.<a class="footnoteRef" href="#fn1" id="fnref1"><sup>1</sup></a> Mendel himself had to discuss this matter in his famous paper,<a class="footnoteRef" href="#fn2" id="fnref2"><sup>2</sup></a> especially with respect to his peas attribute <em>tall/dwarf</em>, which actually was "length of stem".<a class="footnoteRef" href="#fn3" id="fnref3"><sup>3</sup></a><a class="footnoteRef" href="#fn4" id="fnref4"><sup>4</sup></a> Analysis of <a href="Quantitative_trait_locus" title="wikilink">quantitative trait loci</a>, or QTL,<a class="footnoteRef" href="#fn5" id="fnref5"><sup>5</sup></a> is a more recent addition to quantitative genetics, linking it more directly to <a href="molecular_genetics" title="wikilink">molecular genetics</a>.</p>
<h2 id="basic-principles">Basic principles</h2>
<h3 id="gene-effects">Gene effects</h3>

<p>In <a class="uri" href="diploid" title="wikilink">diploid</a> organisms, the average <a class="uri" href="genotypic" title="wikilink">genotypic</a> "value" (locus value) may be defined by the allele "effect" together with a <a href="dominance_(genetics)" title="wikilink">dominance</a> effect, and also by how genes interact with genes at other loci (<a class="uri" href="epistasis" title="wikilink">epistasis</a>). The founder of quantitative genetics - Sir Ronald Fisher - perceived much of this when he proposed the first mathematics of this branch of genetics.<a class="footnoteRef" href="#fn6" id="fnref6"><sup>6</sup></a></p>

<p><a href="File:Gene_effects.jpg" title="wikilink"><code>thumb</code> <code>|</code> <code>right</code> <code>|</code> <code>300px</code> <code>|</code> <code>Gene</code> <code>effects</code> <code>and</code> <code>Phenotype</code> <code>values.</code></a><code> </code></p>

<p>Being a statistician, he defined the gene effects as deviations from a central value: thereby enabling the use of statistical concepts such as the mean and variance, which utilize this idea.<a class="footnoteRef" href="#fn7" id="fnref7"><sup>7</sup></a> The central value he chose for the gene was the midpoint between the two opposing homozygotes at the one locus. The deviation from there to the "greater" homozygous genotype can be named "<em>+a</em>" ; and therefore it is "<em>-a</em>" from that same midpoint to the "lesser" homozygote genotype. This is the "allele" effect mentioned above. The heterozygote deviation from the same midpoint can be named "<em>d</em>", this being the "dominance" effect referred to above.<a class="footnoteRef" href="#fn8" id="fnref8"><sup>8</sup></a> The diagram depicts the idea. However, in reality we measure phenotypes, and the figure also shows how observed phenotypes relate to the gene effects. Formal definitions of these effects recognize this phenotypic focus.<a class="footnoteRef" href="#fn9" id="fnref9"><sup>9</sup></a><a class="footnoteRef" href="#fn10" id="fnref10"><sup>10</sup></a> Epistasis has been approached statistically as interaction (<em>ie</em> "inconsistencies"),<a class="footnoteRef" href="#fn11" id="fnref11"><sup>11</sup></a> but <strong>epigenetics</strong> suggests a new approach may be needed.</p>

<p>If <strong>0</strong><strong>a</strong> was known as "over-dominance".<a class="footnoteRef" href="#fn12" id="fnref12"><sup>12</sup></a></p>

<p>Mendel's pea attribute "length of stem" provides us with a good example.<a class="footnoteRef" href="#fn13" id="fnref13"><sup>13</sup></a> Mendel stated that the tall true-breeding parents ranged from 6–7 feet in stem length (183 – 213 cm), giving a median of 198 cm (= P1). The short parents ranged from 0.75 - 1.25 feet in stem length (23 – 46 cm), with a rounded median of 34 cm (= P2). Their hybrid ranged from 6–7.5 feet in length (183–229 cm), with a median of 206 cm (= F1). The mean of P1 and P2 is 116 cm, this being the phenotypic value of the homozygotes midpoint (mp). The allele affect (<em>a</em>) is [P1-mp] = 82 cm = -[P2-mp]. The dominance effect (<em>d</em>) is [F1-mp] = 90 cm.<a class="footnoteRef" href="#fn14" id="fnref14"><sup>14</sup></a> This historical example illustrates clearly how phenotype values and gene effects are linked.</p>
<h3 id="allele-and-genotype-frequencies">Allele and Genotype frequencies</h3>

<p>To obtain means, variances and other statistics, both <em>quantities</em> and their <em>occurrences</em> are required. The gene effects (above) provide the framework for <em>quantities</em>: and the <em>frequencies</em> of the contrasting alleles in the fertilization gamete-pool provide the information on <em>occurrences</em>. <a href="File:Sexual-Repro-simpl.jpg" title="wikilink">thumb | left | 400px | Analysis of Sexual reproduction.</a> Commonly, the frequency of the allele causing "more" in the phenotype (including dominance) is given the symbol <strong><em>p</em></strong>, while the frequency of the contrasting allele is <strong><em>q</em></strong>. An initial assumption made when establishing the algebra was that the parental population was infinite and random mating, which was made simply to facilitate the derivation. The subsequent mathematical development also implied that the frequency distribution within the effective gamete-pool was uniform: there were no local perturbations where <strong><em>p</em></strong> and <strong><em>q</em></strong> varied. Looking at the diagrammatic analysis of sexual reproduction, this is the same as declaring that <strong><em>p<sub>P</sub></em></strong> = <strong><em>p<sub>g</sub></em></strong> = <strong><em>p</em></strong>; and similarly for <strong><em>q</em></strong>.<a class="footnoteRef" href="#fn15" id="fnref15"><sup>15</sup></a> This mating system, dependent upon these assumptions, became known as "panmixia".</p>

<p>Panmixia rarely actually occurs in nature,<a class="footnoteRef" href="#fn16" id="fnref16"><sup>16</sup></a><a class="footnoteRef" href="#fn17" id="fnref17"><sup>17</sup></a><a class="footnoteRef" href="#fn18" id="fnref18"><sup>18</sup></a> as gamete distribution may be limited, for example by dispersal restrictions or by behaviour, or by chance sampling (those local perturbations mentioned above). It is well-known that there is a huge wastage of gametes in Nature, which is why the diagram depicts a <em>potential</em> gamete-pool separately to the <em>actual</em> gamete-pool. Only the latter sets the definitive frequencies for the zygotes: this is the true "gamodeme" ("gamo" refers to the gametes, and "deme" derives from Greek for "population"). But, under Fisher's assumptions, the <em>gamodeme</em> can be effectively extended back to the <em>potential</em> gamete-pool, and even back to the parental base-population (the "source" population). The random sampling arising when small "actual" gamete-pools are sampled from a large "potential" gamete-pool is known as <em>genetic drift</em>, and will be considered subsequently.</p>

<p>While panmixia may not be widely extant, the <em>potential</em> for it does occur, although it may be only ephemeral because of those local perturbations. It has been shown, for example, that the F2 derived from <em>random fertilization of F1 individuals</em> (an <em>allogamous</em> F2), following hybridization, is an <strong>origin</strong> of a new <em>potentially</em> panmictic population.<a class="footnoteRef" href="#fn19" id="fnref19"><sup>19</sup></a><a class="footnoteRef" href="#fn20" id="fnref20"><sup>20</sup></a> It has also been shown that <em>if</em> panmictic random fertilization <em>did</em> occur continually, it would maintain the same allele and genotype frequencies across each successive panmictic sexual generation - this being the <strong>Hardy Weinberg</strong> equilibrium.<a class="footnoteRef" href="#fn21" id="fnref21"><sup>21</sup></a><a class="footnoteRef" href="#fn22" id="fnref22"><sup>22</sup></a><a class="footnoteRef" href="#fn23" id="fnref23"><sup>23</sup></a><a class="footnoteRef" href="#fn24" id="fnref24"><sup>24</sup></a><a class="footnoteRef" href="#fn25" id="fnref25"><sup>25</sup></a> However, as soon as genetic drift was initiated by local random sampling of gametes, the equilibrium would cease.</p>
<h4 id="random-mating">Random Mating</h4>

<p>Male and female gametes within the actual fertilizing pool are considered usually to have the same frequencies for their corresponding alleles. (Exceptions have been considered.) This means that when <strong><em>p</em></strong> male gametes carrying the <strong><em>A</em></strong> allele randomly fertilize <strong><em>p</em></strong> female gametes carrying that same allele, the resulting zygote will have genotype <strong><em>AA</em></strong>, and, under random fertilization, the combination will occur with a frequency of <strong><em>p</em></strong> x <strong><em>p</em></strong> (= <strong><em>p<sup>2</sup></em></strong>). Similarly, the zygote <strong><em>aa</em></strong> will occur with a frequency of <strong><em>q<sup>2</sup><strong><em>. Heterozygotes (</em></strong>Aa</em></strong>) can arise in two ways: when <strong><em>p<strong>'' male (</strong></em>A</strong>'' allele) randomly fertilize <strong><em>q<strong>'' female (</strong></em>a</strong>'' allele) gametes, and <em>vice versa</em>. The resulting frequency for the heterozygous zygotes is thus <strong><em>2pq</em></strong>.<a class="footnoteRef" href="#fn26" id="fnref26"><sup>26</sup></a> Notice that such a population is never more than half heterozygous, this maximum occurring when <strong>p</strong>=<strong>q</strong>= 0.5.</p>

<p>In summary then, under random fertilization, the zygote (genotype) frequencies are the quadratic expansion of the gametic (allelic) frequencies: ( <strong><em>p</em></strong> + <strong><em>q</em></strong> )<sup><strong>2</strong></sup> = ( <strong><em>p</em></strong><sup><em>2</em></sup> + <strong>2</strong> <strong><em>p</em></strong> <strong><em>q</em></strong> + <strong><em>q</em></strong><sup><strong>2</strong></sup>) = <strong>1</strong>. (The "=1" states that the frequencies are in fraction form, not percentages; and that there are no omissions within the framework proposed.) Also recall that "random fertilization" and "panmixia" are <em>not</em> synonyms, as discussed in the previous section.</p>
<h4 id="mendels-research-cross---a-contrast">Mendel's Research Cross - a Contrast</h4>

<p>Mendel's pea experiments were constructed by establishing true-breeding parents with "opposite" phenotypes for each attribute.<a class="footnoteRef" href="#fn27" id="fnref27"><sup>27</sup></a> This meant that each opposite parent was homozygous for its respective allele only. In our example, "tall <em>vs</em> dwarf", the tall parent would be genotype <strong><em>TT</em></strong> with <strong><em>p</em></strong> = <strong>1</strong> (and <strong><em>q</em></strong> = <strong>0</strong>); while the dwarf parent would be genotype <strong><em>tt</em></strong> with <strong><em>q</em></strong> = <strong>1</strong> (and <strong><em>p</em></strong> = <strong>0</strong>). After controlled crossing, their hybrid will be <strong><em>Tt</em></strong>, with <strong><em>p</em></strong> = <strong><em>q</em></strong> = <strong>½</strong>. However, the frequency of this heterozygote = <strong>1</strong>, because this is the F1 of an artificial cross: it has not arisen through random fertilization.<a class="footnoteRef" href="#fn28" id="fnref28"><sup>28</sup></a> The F2 generation was produced by natural self-pollination of the F1 (with monitoring against insect contamination), resulting in <strong><em>p</em></strong> = <strong><em>q</em></strong> = <strong>½</strong> being maintained. Such an F2 is said to be "autogamous". However, the genotype frequencies (0.25 <strong><em>TT</em></strong>, 0.5 <strong><em>Tt</em></strong>, 0.25 <strong><em>tt</em></strong>) have arisen through a mating system very different from random fertilization, and therefore the use of the quadratic expansion has been avoided. The numerical values obtained were the same as those for random fertilization only because this is the special case of having originally crossed homozygous opposite parents.<a class="footnoteRef" href="#fn29" id="fnref29"><sup>29</sup></a> We can notice that, because of the dominance of <strong><em>T-</em></strong> [frequency (0.25 + 0.5)] over <strong><em>tt</em></strong> [frequency 0.25], the 3:1 ratio is still obtained.</p>

<p>A cross such as Mendel's, where true-breeding (largely homozygous) opposite parents are crossed in a controlled way to produce an F1, is a special case of hybrid structure. The F1 is often regarded as being "entirely heterozygous" for the gene under consideration. However, this is an over-simplification which does not apply generally: for example when individual parents are not homozygous, or when <em>populations</em> inter-hybridise to form "hybrid swarms".<a class="footnoteRef" href="#fn30" id="fnref30"><sup>30</sup></a> The general properties of intra-species hybrids (F1) and F2 (both "autogamous" and "allogamous") will be considered in a later section.</p>
<h4 id="self-fertilization---an-alternative">Self Fertilization - an Alternative</h4>

<p>Having noticed that the pea is naturally self-pollinated, we cannot continue to use it as an example for illustrating random fertilization properties. Self-fertilization ("selfing") is a major alternative to random fertilization, especially within Plants. Most of the Earth's cereals are naturally self-pollinated (rice, wheat, barley, for example), as well as the pulses. Considering the millions of individuals of each of these on Earth at any time, it's obvious that self-fertilization is at least as significant as random fertilization! Self-fertilization is the most intensive form of <em>inbreeding</em>, which arises whenever there is restricted independence in the genetical origins of gametes. Such reduction in independence arises if parents are already related, and/or from genetic drift or other spatial restrictions on gamete dispersal. Path analysis demonstrates that these are tantamount to the same thing.<a class="footnoteRef" href="#fn31" id="fnref31"><sup>31</sup></a><a class="footnoteRef" href="#fn32" id="fnref32"><sup>32</sup></a> Arising from this background, the <em>inbreeding coefficient</em> (often symbolized as <strong>F</strong> or <strong><em>f</em></strong>) quantifies the effect of inbreeding from whatever cause. There are several formal definitions of <strong><em>f</em></strong>, and some of these will be considered in later sections. For the present, note that for a long-term self-fertilized species <strong><em>f</em></strong> = <strong>1</strong>. Natural self-fertilized populations are not single " <em>pure lines</em> ", however, but mixtures of such lines. This becomes particularly obvious when considering more than one gene at a time. Therefore, allele frequencies (<strong><em>p</em></strong> and <strong><em>q</em></strong>) other than <strong>1</strong> or <strong>0</strong> are still relevant in these cases (refer back to the Mendel Cross section). The genotype frequencies take a different form, however.</p>

<p>In general, the genotype frequencies become <strong><em>[p<sup>2</sup>(1-f) + pf]</em></strong> for <strong>AA</strong> and <strong><em>[2pq(1-f)]</em></strong> for <strong>Aa</strong> and <strong><em>[q<sup>2</sup>(1-f) + qf]</em></strong> for <strong>aa</strong>.<a class="footnoteRef" href="#fn33" id="fnref33"><sup>33</sup></a>  Notice that the frequency of the heterozygote declines in proportion to <strong><em>f</em></strong>. When <strong><em>f</em> = 1</strong>, these three frequencies become respectively <strong>p</strong>, <strong>0</strong> and <strong>q</strong> !! Conversely, when <strong>f = 0</strong>, they reduce to the random-fertilization quadratic expansion shown previously.</p>
<h3 id="population-mean">Population Mean</h3>

<p>The population mean shifts the central reference point from the homozygote midpoint (<strong>mp</strong>) to the mean of a sexually reproduced population. This is important not only to relocate the focus into the natural world, but also to use a measure of <em>central tendency</em> utilized by Statistics/Biometrics. In particular, the square of this mean is the Correction Factor, which is used to obtain the Genotypic Variances later.<a class="footnoteRef" href="#fn34" id="fnref34"><sup>34</sup></a> </p>

<p>For each genotype in turn, its allele effect is multiplied by its genotype frequency; and the products are accumulated across all genotypes in the model. Some algebraic simplification usually follows to reach a succinct result.</p>
<h4 id="the-mean-after-random-fertilization">The Mean after random fertilization</h4>

<p>The contribution of <strong>AA</strong> is <strong>p<sup>2</sup> (+)a</strong>, that of <strong>Aa</strong> is <strong>2pq d</strong>. and that of <strong>aa</strong> is <strong>q<sup>2</sup> (-)a</strong>. Gathering together the two <strong>a</strong> terms and accumulating over all, the result is: <strong>a (p<sup>2</sup> - q<sup>2</sup>) + 2pq d</strong>. Simplification is achieved by noting that <strong>(p<sup>2</sup>-q<sup>2</sup>)</strong> = <strong>(p-q)(p+q)</strong>, and by recalling that <strong>(p+q) = 1</strong>, thereby reducing the left-hand term to <strong>(p-q)</strong>. The succinct result is therefore <strong>G</strong> = <strong>a(p-q) + 2pqd</strong>.<a class="footnoteRef" href="#fn35" id="fnref35"><sup>35</sup></a>  This defines the population mean as an "offset" from the homozygote midpoint (recall <strong>a</strong> and <strong>d</strong> are defined as <em>deviations</em> from that midpoint). The Figure depicts <strong>G</strong> across all values of <strong>p</strong> for several values of <strong>d</strong>, including one case of slight over-dominance. Notice that <strong>G</strong> is often negative, thereby emphasizing that it is itself a <em>deviation</em> (from <strong>mp</strong>). Finally, to obtain the <em>actual</em> Population Mean in "phenotypic space", the midpoint value is added to this offset: <strong>P = G + mp</strong>.</p>

<p>An example arises from data on ear length in maize.<a class="footnoteRef" href="#fn36" id="fnref36"><sup>36</sup></a> Assuming for now that one gene only is represented, <strong>a</strong> = 5.45 cm, <strong>d</strong> = 0.12 cm [virtually "0", really], <strong>mp</strong> = 12.05 cm. Further assuming that <strong>p</strong> = 0.6 and <strong>q</strong> = 0.4 in this example population, then:-</p>

<p><strong>G</strong> = 5.45 (0.6 - 0.4) + (0.48)0.12 = <strong>1.15 cm</strong> (rounded); and</p>

<p><strong>P</strong> = 1.15 + 12.05 = <strong>13.20 cm</strong> (rounded).</p>
<h4 id="the-mean-after-self-fertilization">The Mean after self fertilization</h4>

<p>The contribution of <strong>AA</strong> is <strong>p (+)a</strong>, while that of <strong>aa</strong> is <strong>q (-)a</strong>. (See above for the frequencies.) Gathering these two <strong>a</strong> terms together leads to an immediately very simple final result:- <strong>G<sub>S</sub></strong> = <strong>a(p-q)</strong>. <strong>P</strong> is obtained as above.</p>

<p>Mendel's peas can provide us with the allele effects and midpoint (see previously); and a mixed self-pollinated population with <strong>p</strong> = 0.6 and <strong>q</strong> = 0.4 provides example frequencies. Thus:-</p>

<p><strong>G<sub>S</sub></strong> = 82 (0.6 - .04) = <strong>59.6 cm</strong> (rounded); and</p>

<p><strong>P<sub>S</sub></strong> = 59.6 + 116 = <strong>175.6 cm</strong> (rounded).</p>
<h4 id="the-mean---generalized-fertilization">The Mean - Generalized fertilization</h4>

<p>A general formula incorporates the inbreeding coefficient <strong><em>f</em></strong>, and can then accommodate any situation. The procedure is exactly the same as before, using the weighted genotype frequencies given earlier. After translation into our symbols, and further rearrangement:<a class="footnoteRef" href="#fn37" id="fnref37"><sup>37</sup></a> </p>

<p><strong>G<sub><em>f</em></sub></strong> = <strong>a(q-p) + [2pqd - <em>f</em> 2pqd]</strong> = <strong>a(q-p) + (1-<em>f</em>) 2pqd</strong> = <strong>G - <em>f</em> 2pqd</strong> !</p>

<p>Supposing that the maize example (earlier) had been constrained on a holme (a narrow riparian meadow), and had partial inbreeding to the extent of <em>'</em>f <em>'</em>= <strong>0.25</strong>, then, using the third version (above) of <strong>G<sub><em>f</em></sub></strong>:-</p>

<p><strong>G<sub><em>0.25</em></sub></strong> = 1.15 - 0.25 (0.48) 0.12 = <strong>1.136 cm</strong> (rounded), with <strong>P</strong> = <strong>13.194 cm</strong> (rounded).</p>

<p>There is hardly any effect from inbreeding in this example, which arises because there was virtually no dominance in this attribute (<strong>d</strong> → 0). Examination of all three versions of <strong>G<sub><em>f</em></sub></strong> reveals that this would lead to trivial change in the Population mean. Where dominance was notable, however, there would be considerable change.</p>
<h3 id="genetic-drift">Genetic drift</h3>

<p>Genetic drift was introduced when discussing the likelihood of panmixia being widely extant as a natural fertilization pattern. [See section on Allele and Genotype frequencies.] Here the sampling of gametes from the <em>potential</em> gamodeme is discussed in more detail. The sampling involves random fertilization between pairs of random gametes, each of which may contain either an <strong>A</strong> or an <strong>a</strong> allele. The sampling is therefore Binomial sampling.<a class="footnoteRef" href="#fn38" id="fnref38"><sup>38</sup></a> <a class="footnoteRef" href="#fn39" id="fnref39"><sup>39</sup></a> <a class="footnoteRef" href="#fn40" id="fnref40"><sup>40</sup></a> <a class="footnoteRef" href="#fn41" id="fnref41"><sup>41</sup></a> Each sampling "packet" involves <strong>2N</strong> alleles, and produces <strong>N</strong> zygotes (a "progeny" or a "line") as a result. During the course of the reproductive period, this sampling is repeated over and over, so that the final result is a mixture of sample progenies. These events, and the overall end-result, are examined here with an illustrative example.</p>

<p>The "base" allele frequencies of the example are those of the <em>potential gamodeme</em>: the frequency of <strong>A</strong> is <strong>p<sub>g</sub> = 0.75</strong>, while the frequency of <strong>a</strong> is <strong>q<sub>g</sub> = 0.25</strong>. [<em>White label</em> "<strong>1</strong>" in the Diagram.] Five example <strong>actual gamodemes</strong> are Binomially sampled out of this base (<strong>s = the number of samples = 5</strong>), and each sample is designated with an "index" <strong>k</strong>: with <strong>k = 1 .... s</strong> sequentially. (These are the sampling "packets" referred to in the previous paragraph.) The number of gametes involved in fertilization varies from sample to sample, and is given as <strong>2N<sub>k</sub></strong> [at <em>white label</em> "<strong>2</strong>" in the Diagram]. The total (Σ) number of gametes sampled overall is 52 [<em>white label</em> "<strong>3</strong>" in the Diagram]. Because each sample has its own size, <em>weights</em> are needed to obtain averages (and other statistics) when obtaining the overall results. These are <strong>ω<sub>k</sub> = 2N<sub>k</sub> / ( Σ<sub>k</sub> 2N<sub>k</sub> )</strong>, and are given at <em>white label</em> "<strong>4</strong>" in the Diagram. </p>
<h4 id="the-sample-gamodemes---genetic-drift">The sample gamodemes - Genetic drift</h4>

<p>Following completion of these five Binomial sampling events, the resultant <em>actual</em> gamodemes each contained <em><strong>different allele frequencies</strong></em> - (<strong>p<sub>k</sub></strong> and <strong>q<sub>k</sub></strong>). [These are given at <em>white label</em> "<strong>5</strong>" in the Diagram.] This outcome is actually the <strong>Genetic Drift</strong> itself. Notice that two samples (k = 1 and 5) happen to have the same frequencies as the <em>base</em> (<em>potential</em>) gamodeme. Another (k = 3) happens to have the <em>p</em> and <em>q</em> "reversed". Sample (k = 2) happens to be an "extreme" case, with <strong>p<sub>k</sub> = 0.9</strong> and <strong>q<sub>k</sub> = 0.1</strong> ; while the remaining sample (k = 4) is "middle of the range" in its allele frequencies. All of these results have arisen only by "chance", through Binomial sampling. Having occurred, however, they set in place all the downstream properties of the progenies.</p>

<p>Because sampling involves "chance", the <em><strong>Probabilities</strong></em> ( <mtpl></mtpl><sub>k</sub> ) of obtaining each of these samples become of interest. These Binomial Probabilities depend on the starting frequencies (<strong>p<sub>g</sub></strong> and <strong>q<sub>g</sub></strong>) and the sample size (<strong>2N<sub>k</sub></strong>). They are tedious to obtain,<a class="footnoteRef" href="#fn42" id="fnref42"><sup>42</sup></a> <a class="footnoteRef" href="#fn43" id="fnref43"><sup>43</sup></a> but are of considerable interest. [See <em>white label</em> "<strong>6</strong>" in the Diagram.] The two samples (k = 1, 5), with the allele frequencies the same as in the <em>potential gamodeme</em>, had higher "chances" of occurring than the other samples. Their Binomial Probabilities did differ, however, because of their different sample sizes (2N<sub>k</sub>). The "reversal" sample (k = 3) had a very low Probability of occurring, confirming perhaps what might be expected. The "extreme" allele frequency gamodeme (k = 2) was not "rare", however; and the "middle of the range" sample (k=4) <em>was</em> rare. These same Probabilities will apply also to the progenies arising from these fertilizations.</p>

<p>It is here that some <em>summarizing</em> can begin. The <strong><em>overall allele frequencies</em></strong> in the progenies bulk are supplied by weighted averages of the appropriate frequencies of the individual samples. That is:- <strong>p<sub>•</sub> = [Σ<sub>k</sub> (ω<sub>k</sub> p<sub>k</sub>)]</strong> and <strong>q<sub>•</sub> = [Σ<sub>k</sub> (ω<sub>k</sub> q<sub>k</sub>)]</strong>. (Notice that <strong>k</strong> is replaced by <strong>•</strong> for the overall result - a common practice.) <a class="footnoteRef" href="#fn44" id="fnref44"><sup>44</sup></a> The results for the example are <strong>p<sub>•</sub></strong> = <strong>0.631</strong> and <strong>q<sub>•</sub></strong> = <strong>0.369</strong> [<em>black label</em> "<strong>5</strong>" in the Diagram]. These values are quite different to the starting ones (<strong>p<sub>g</sub></strong> and <strong>q<sub>g</sub></strong>) [<em>white label</em> "<strong>1</strong>"]. The sample allele frequencies also have variance as well as an average. This has been obtained using the "Sum of Squares method" (<strong>SS</strong>),<a class="footnoteRef" href="#fn45" id="fnref45"><sup>45</sup></a> [see to the right of <em>black label</em> "<strong>5</strong>" in the Diagram]. [Further discussion on this variance occurs in the section below on Extensive genetic drift.]</p>
<h4 id="the-progeny-lines---dispersion">The progeny lines - Dispersion</h4>

<p>The <strong>genotype frequencies</strong> of the five sample progenies are obtained from the usual quadratic expansion of their respective allele frequencies (<em>random fertilization</em>). The results are given at the Diagram's <em>white label</em> "<strong>7</strong>" for the homozygotes, and at <em>white label</em> "<strong>8</strong>" for the heterozygotes. Re-arrangement in this manner prepares the way for monitoring <strong>inbreeding</strong> levels. This can be done either by examining the level of <em>total</em> homozygosis [(<strong>p<sup>2</sup><sub>k</sub> + q<sup>2</sup><sub>k</sub></strong>) = (<strong>1 - 2p<sub>k</sub>q<sub>k</sub></strong>)] , or by examining the level of heterozygosis (<strong>2p<sub>k</sub>q<sub>k</sub></strong>), as they are complementary.<a class="footnoteRef" href="#fn46" id="fnref46"><sup>46</sup></a> Notice that samples <em>k= 1, 3, 5</em> all had the same level of heterozygosis, despite one being the "mirror image" of the others with respect to allele frequencies. The "extreme" allele-frequency case (k= <em>2</em>) had the most homozygosis (least heterozygosis) of any sample. The "middle of the range" case (k= <em>4</em>) had the least homozygosity (most heterozygosity): they were each equal at 0.50, in fact.</p>

<p>The <em>overall summary</em> can continue by obtaining the <em>weighted average</em> of the respective genotype frequencies for the progeny bulk. Thus, for <strong>AA</strong>, it is <strong>p<sup>2</sup><sub>•</sub></strong> = <strong>[Σ<sub>k</sub> (ω<sub>k</sub> p<sup>2</sup><sub>k</sub>)]</strong> , for <strong>Aa</strong> , it is <strong>2p<sub>•</sub>q<sub>•</sub></strong> = <strong>[Σ<sub>k</sub> (ω<sub>k</sub> 2p<sub>k</sub>q<sub>k</sub>)]</strong> and for <strong>aa</strong>, it is <strong>q<sup>2</sup><sub>•</sub></strong> = <strong>[Σ<sub>k</sub> (ω<sub>k</sub> q<sup>2</sup><sub>k</sub>)]</strong>. The example results are given at <em>black label</em> "<strong>7</strong>" for the homozygotes, and at <em>black label</em> "<strong>8</strong>" for the heterozygote. Note that the heterozygosity mean is <strong>0.3588</strong>, which will be used in the next section to examine the <strong>inbreeding</strong> resulting from this <strong>Genetic Drift</strong>.</p>

<p>The next focus of interest is the <strong>Dispersion</strong> itself, which refers to the "spreading apart" of the progenies' <strong>population means</strong>. These are obtained as <strong>G<sub>k</sub></strong> = <strong>[a(p<sub>k</sub>-q<sub>k</sub>) + 2p<sub>k</sub>q<sub>k</sub> d]</strong> [see section on the Population mean], for each sample progeny in turn, using the example gene effects given at <em>white label</em> "<strong>9</strong>" in the Diagram. Then, each <strong>P<sub>k</sub></strong> = <strong>G<sub>k</sub> + mp</strong> is obtained also. The latter are given at <em>white label</em> "<strong>10</strong>" in the Diagram. Notice that the "best" line (k = 2) had the <em>highest</em> allele frequency for the "more" allele (<strong>A</strong>) (it also had the highest level of homozygosity). The <em>worst</em> progeny (k = 3) had the highest frequency for the "less" allele (<strong>a</strong>), which accounted for its poor performance. This "poor" line was less homozygous than the "best" line; and it shared the same level of homozygosity, in fact, as the two <em>second-best</em> lines (k = 1, 5). The progeny line with both the "more" and the "less" alleles present in equal frequency (k = 4) had a mean below the <em>overall average</em> (see next paragraph), and had the lowest level of homozygosity. These results reveal the fact that it is what alleles are most prevalent in the "gene-pool" (also called the "germplasm") which determines performance, not the level of homozygosity <em>per se</em>. Recall that it is Binomial sampling alone which has effected this Dispersion.</p>

<p>The <em>overall summary</em> can now be concluded by obtaining <strong>G<sub>•</sub> = [Σ<sub>k</sub> (ω<sub>k</sub>G<sub>k</sub>)]</strong> and <strong>P<sub>•</sub> = [Σ<sub>k</sub> (ω<sub>k</sub>P<sub>k</sub>)]</strong>. The example result for <strong>P<sub>•</sub></strong> is <strong>36.94</strong> (<em>black label</em> "<strong>10</strong>" in the Diagram). This will be used later to quantify <strong>inbreeding depression</strong> arising overall from the gamete sampling. However, recall that some very "non-depressed" progeny means have been identified already (k = 1, 2, 5)! This is an enigma of inbreeding - while there may be "depression" overall, there will usually be superior lines amongst the gamodeme samplings.</p>
<h4 id="the-equivalent-post-dispersion-panmictic---inbreeding">The equivalent post-dispersion panmictic - Inbreeding</h4>

<p>Included in the <em>overall summary</em> were the averaqe allele frequencies in the mixture of progeny lines (<strong>p<sub>•</sub></strong> and <strong>q<sub>•</sub></strong>). These can now be used to construct a hypothetical panmictic equivalent.<a class="footnoteRef" href="#fn47" id="fnref47"><sup>47</sup></a> <a class="footnoteRef" href="#fn48" id="fnref48"><sup>48</sup></a> <a class="footnoteRef" href="#fn49" id="fnref49"><sup>49</sup></a> This can be regarded as a "reference" to assess the changes wrought by the gamete sampling. The example appends such a panmictic to the right of the Diagram. The frequency of <strong>AA</strong> is therefore <strong>(p<sub>•</sub>)<sup>2</sup> = 0.3979</strong>. This is less than that found in the dispersed bulk (0.4513 at <em>black label</em> "<strong>7</strong>"). Similarly, for <strong>aa</strong>, <strong>(q<sub>•</sub>)<sup>2</sup> = 0.1303</strong> - again less than the equivalent in the progenies bulk (0.1898). Clearly, <em>genetic drift</em> has increased the overall level of homozygosis by the amount (0.6411 - 0.5342) = 0.1069. In a complementary approach, the heterozygosity could be used instead. The panmictic equivalent for <strong>Aa</strong> is <strong>2 p<sub>•</sub> q<sub>•</sub> = 0.4658</strong>, which is <em>higher</em> than that in the Binomially sampled bulk (0.3588) [<em>black label</em> "<strong>8</strong>"]. The sampling has caused the heterozygosity to decrease by 0.1070, which differs trivially from the earlier estimate because of rounding errors.</p>

<p>The <strong>inbreeding coefficient</strong> (<strong><em>f</em></strong>) was introduced in the early section on Self Fertilization. Here, a formal definition of it is considered:- <strong><em>f</em></strong> is the probability that two "same" alleles (that is <strong>A</strong> and <strong>A</strong>, or <strong>a</strong> and <strong>a</strong>) which fertilize together are of common ancestral origin - or (more formally) <strong><em>f</em></strong> is the probability that two homologous alleles are autozygous.<a class="footnoteRef" href="#fn50" id="fnref50"><sup>50</sup></a><a class="footnoteRef" href="#fn51" id="fnref51"><sup>51</sup></a> Consider any random gamete in the <em>potential</em> gamodeme which has its syngamy partner restricted by Binomial sampling. The probability that that second gamete is homologous autozygous to the first is <strong>1/(2N)</strong>, the reciprocal of the gamodeme size. For the five example progenies, these quantities are 0.1, 0.0833, 0.1, 0.0833 and 0.125 respectively, and their weighted average is <strong>0.0961</strong>. This is the <strong>inbreeding coefficient</strong> of the example progenies bulk, provided it is <em>unbiased</em> with respect to the full Binomial Distribution. An example based upon <em>s = 5</em> is likely to be biased, however, when compared to an appropriate entire Binomial Distribution based upon <em>s → ∞</em>. Another derived definition of <strong><em>f</em></strong> for the full Distribution is that <strong><em>f</em></strong> also equals the rise in homozygosity which equals the fall in heterozygosity.<a class="footnoteRef" href="#fn52" id="fnref52"><sup>52</sup></a> For the example, these frequency changes are <em>0.1069</em> and <em>0.1070</em>, respectively: which result is different to the above, indicating that bias with respect to the full underlying Distribution is present in the example. For the example <em>itself</em>, these latter values are the better ones to use, namely <strong><em>f<sub>•</sub></em></strong> = <strong>0.10695</strong>.</p>

<p>The <strong>population mean</strong> of the equivalent panmictic is found as <strong>[a (p<sub>•</sub>-q<sub>•</sub>) + 2 p<sub>•</sub>q<sub>•</sub> d] + mp</strong>. Using the example <em>gene effects</em> (<em>white label</em> "<strong>9</strong>" in the Diagram), this mean is <strong>37.87</strong>. The equivalent mean in the dispersed bulk is 36.94 (<em>black label</em> "<strong>10</strong>") which is depressed by the amount <em>0.93</em>. This is the <strong>inbreeding depression</strong> arising from this Genetic Drift. However, as noted previously, three progenies were <em>not</em> depressed (k = 1, 2, 5), and had means even greater than that of the panmictic equivalent. <strong><em>These</em></strong> are the lines the Plant Breeder would be looking for in his "Line Selection" programme.<a class="footnoteRef" href="#fn53" id="fnref53"><sup>53</sup></a></p>
<h4 id="extensive-binomial-sampling---is-panmixia-restored">Extensive Binomial sampling - is Panmixia restored?</h4>

<p>If the number of Binomial samplings is very large (<strong>s → ∞</strong> ), then <strong>p<sub>•</sub> → p<sub>g</sub></strong> and <strong>q<sub>•</sub> → q<sub>g</sub></strong>. It might be queried whether panmixia would effectively re-appear under these circumstances. However, the sampling of allele frequencies has still occurred, with the result that <strong>σ<sup>2</sup><sub>p, q</sub></strong> ≠ <strong>0</strong>. In fact, as <strong>s → ∞</strong>, the <strong>σ<sup>2</sup><sub>p, q</sub></strong> → <strong>[(p<sub>g</sub>q<sub>g</sub>)/2N]</strong>, which is the variance of the whole Binomial Distribution.<a class="footnoteRef" href="#fn54" id="fnref54"><sup>54</sup></a> <a class="footnoteRef" href="#fn55" id="fnref55"><sup>55</sup></a> Furthermore, the "Wahlund equations" show that the progeny-bulk <em>homozygote</em> frequencies can be obtained as the sums of their respective average values (<strong>p<sup>2</sup><sub>•</sub></strong> or <strong>q<sup>2</sup><sub>•</sub></strong>) and <strong>σ<sup>2</sup><sub>p, q</sub></strong>.<a class="footnoteRef" href="#fn56" id="fnref56"><sup>56</sup></a> Likewise, the bulk <em>heterozygote</em> frequency is <strong>(2 p<sub>•</sub> q<sub>•</sub>) <em>minus</em></strong> <em>twice</em> the <strong>σ<sup>2</sup><sub>p, q</sub></strong>. The variance arising from the Binomial sampling is conspicuously present indeed ! Thus, even when <strong>s → ∞</strong>, the progeny-bulk <em>genotype</em> frequencies will still reveal <em>increased homozygosis</em>, and <em>decreased heterozygosis</em>, there will still be <em>dispersion of progeny means</em>, and there will still be <em>inbreeding</em> and <em>inbreeding depression</em>. That is, panmixia is <em>not</em> re-attained once it is lost. But a new <em>potential</em> panmixia can be initiated <em>via</em> an allogamous F2 following hybridization,<a class="footnoteRef" href="#fn57" id="fnref57"><sup>57</sup></a> as noted earlier.</p>
<h4 id="continued-genetic-drift---increased-dispersion-and-inbreeding">Continued genetic drift - increased Dispersion and Inbreeding</h4>

<p>In the previous discussions on <strong>genetic drift</strong>, just one cycle (generation) of the process was examined. When the sampling continues over successive generations, conspicuous changes occur in <strong>σ<sup>2</sup><sub>p</sub></strong>, <strong>q</strong> and <strong><em>f</em></strong>. Another "index" is needed to keep track of "time": <strong>t</strong> = <strong>1 .... y</strong> where <strong>y</strong> = the number of "years" (generations) considered. The approach often is to add the "current Binomial increment" (<strong>Δ</strong> = "<em>de novo</em>") to what has occurred previously.<a class="footnoteRef" href="#fn58" id="fnref58"><sup>58</sup></a> The entire Binomial Distribution is examined here: there is no further benefit from an abbreviated example.</p>

<p>Consider firstly <strong>Dispersion</strong> <em>via</em> the '''σ<sup>2</sup><sub>p,q</sub> '''. Earlier this variance was seen to be '''(p<sub>g</sub> q<sub>g</sub>)/(2N) ''' = '''p<sub>g</sub> q<sub>g</sub> [1 /(2N)] ''' = <strong>p<sub>g</sub> q<sub>g</sub> <em>f</em></strong>. [Notice particularly this last version, where <strong>f</strong> = '''[1 /(2N)] ''' = <strong>Δf</strong> as the recurrences are followed.] With the extension over time, this is the result of the <em>first</em> cycle, so this can be designated as ''' σ<sup>2</sup> <sub>1</sub> <em>' (for brevity). At cycle 2, this variance is generated again, becoming this time the "</em>de novo''" variance, and accumulates to that which was present already. The <em>de novo</em> is given the <em>weight</em> of "<strong><em>1</em></strong>", while the "carry-over" is given the <em>weight</em> of <strong><em>1-[1/(2N)]</em></strong>, which is the same as ''' 1-Δf ''' . Thus, ''' σ<sup>2</sup><sub>2</sub> ''' = <strong><em>1</em></strong> ''' Δσ<sup>2</sup> ''' + ''' <em>(1-Δf)</em> σ<sup>2</sup><sub>1</sub> ''' . After gathering terms, simplifying and recalling previous symbols, this becomes ''' σ<sup>2</sup><sub>2</sub> ''' = ''' σ<sup>2</sup><sub>1</sub> {1 + [1-Δf]} ''' . The extension to generalize to any time <em>t</em> , after considerable simplification, becomes:- ''' σ<sup>2</sup><sub>t</sub> ''' = ''' p<sub>g</sub> q<sub>g</sub> {1 - [1 - Δf]<sup>t</sup> } <em>'.<a class="footnoteRef" href="#fn59" id="fnref59"><sup>59</sup></a> Recall that it was the variation in allele frequencies which caused the "spreading apart" of the progenies' means (</em>Dispersion''). Therefore, this ''' σ<sup>2</sup><sub>t</sub> ''' can be used to indicate the extent of <strong>Dispersion</strong> over the generations. </p>

<p>The method for examining <strong>Inbreeding</strong> is very similar. The same <em>weights</em> as before are used respectively for <em>de novo</em> <strong><em>f</em></strong> ( <strong>Δ<em>f</em></strong> ) [recall this is <strong>1/(2N)</strong> ] and <em>carry-over</em> <strong><em>f</em></strong>. Therefore, '''f<sub>2</sub> ''' = ''' <em>1</em> Δf + <em>(1 - Δf)</em> f<sub>1</sub> '''. In general, '''f<sub>t</sub> ''' = ''' Δf + (1- Δf) f<sub>t-1</sub> ''' = '''Δf (1- f<sub>t-1</sub> ) + f<sub>t-1</sub> ''' after rearrangement.<a class="footnoteRef" href="#fn60" id="fnref60"><sup>60</sup></a> The graphs to the left show levels of inbreeding over twenty generations arising from genetic drift (Binomial sampling of gamodemes) for various <em>actual gamodeme</em> sizes (2N).</p>

<p>Still further rearrangement of this general equation reveals some interesting relationships. After some simplification, <strong>(f<sub>t</sub> - f<sub>t-1</sub>)</strong> = <strong>Δf (1-f<sub>t-1</sub>)</strong>. The left-hand side is the difference between the current and previous levels of inbreeding: the <em>change in inbreeding</em> (<strong>δf<sub>t</sub></strong>). Notice, it is not <strong>Δf</strong>, unless the f<sub>t-1</sub> is <strong>0</strong>. Another item of note is the <strong>(1-f<sub>t-1</sub>)</strong> - an index of <em>non-inbreeding</em> - the <strong>panmictic index</strong> (<strong>π<sub>t-1</sub></strong>). Further rearrangements reveal that <strong>Δf = δf<sub>t</sub> / π<sub>t-1</sub></strong> = '''1 - [π<sub>t</sub> / π<sub>t-1</sub>] ''', all useful relationships. </p>

<p>One further rearrangement gives <em>' f<sub>t</sub> = 1 - (1-Δf)<sup>t</sup> (1-f<sub>0</sub>)</em>'; which, assuming that <strong>f<sub>0</sub></strong> = <strong>0</strong>, forms the section <em>within the braces</em> of the last equation above for the ''' σ<sup>2</sup><sub>t</sub> '''. That is, ''' σ<sup>2</sup><sub>t</sub> ''' = ''' p<sub>g</sub> q<sub>g</sub> f<sub>t</sub> ''' <strong>!</strong> Rearranged, this provides also the revelation that ''' f<sub>t</sub> ''' = ''' σ<sup>2</sup><sub>t</sub> / (p<sub>g</sub> q<sub>g</sub>) '''. The two principal threads of Binomial gamete sampling are thus cemented together, and are directly inter-changeable.</p>
<h5 id="selfing-within-random-fertilization">Selfing within Random fertilization</h5>

<p>It is easy to overlook that <strong>self fertilization</strong> is intrinsically included as a part of <em>random fertilization</em>. Sewall Wright showed that a proportion <strong>1/N</strong> of <em>random fertilizations</em> is actually <em>self fertilization</em>, with the remainder <strong>(N-1)/N</strong> being <em>cross fertilization</em>. Following path analysis and simplification, the divided <em>random fertilization</em> inbreeding was found to be:- <strong>f<sub>t</sub></strong> = <strong>selfing<sub>t</sub></strong> + <strong>crossing<sub>t</sub></strong> = <strong>Δf (1+ f<sub>t-1</sub> )</strong> + '''[(N-1)/N] f<sub>t-1</sub> '''.<a class="footnoteRef" href="#fn61" id="fnref61"><sup>61</sup></a><a class="footnoteRef" href="#fn62" id="fnref62"><sup>62</sup></a> Upon further rearrangement, the earlier results from the Binomial sampling were confirmed, along with some new arrangements. Two of these were potentially very useful, namely:- <strong>f<sub>t</sub></strong> = <strong>Δf [ 1 + f<sub>t-1</sub> (2N-1)]</strong>; and further <strong>f<sub>t</sub></strong> = <strong>Δf(1-f<sub>t-1</sub>) + f<sub>t-1</sub>)</strong>.</p>

<p>The insight provided by this leads to some issues about the use of the <strong>inbreeding coefficient</strong> for Binomial sampling <em>random fertilization</em>. Clearly, then, it is inappropriate for any species incapable of <em>self fertilization</em>, which includes plants with self-incompatibility mechanisms, dioecious plants, and <a href="bisexual_animals" title="wikilink">bisexual animals</a> (including, of course, mammals). The method developed by Wright was modified to develop a random fertilization inbreeding equation which involved only <em>cross fertilization</em> without any <em>self fertilization</em>. The proportion <strong>1/N</strong> formerly due to <em>selfing</em> now defined the <em>carry-over</em> inbreeding arising from the cycle previous to the current cycle. The "inbreeding for cross fertilization <em>only</em>" final result was:- <strong>f<sub>t</sub></strong> = <strong>f<sub>t-1</sub> + Δf (1 + f<sub>t-2</sub> - 2 f<sub>t-1</sub>)</strong>.<a class="footnoteRef" href="#fn63" id="fnref63"><sup>63</sup></a> The graphs to the right depict the differences between standard <em>random fertilization</em> (based on Binomial sampling, which includes <em>selfing</em>) <strong>RF</strong>, and Binomial sampling adjusted for "cross fertilization alone" <strong>CF</strong>. As can be seen, the issue is non-trivial for small gamodeme sample sizes.</p>

<p>It now is necessary to note that not only is "panmixia" <em>not</em> a synonym for "random fertilization", but also that "random fertilization" is <em>not</em> a synonym for "cross fertilization".</p>
<h4 id="homozygosity-and-heterozygosity">Homozygosity and Heterozygosity</h4>

<p>In the section on the "Sample gamodemes - Genetic drift", a series of gamete samplings was followed. An important result of this sampling was that homozygosity rose at the expense of heterozygosity. This is one view of the genotype frequencies following the inbreeding attendant upon the sampling. Another view is linked to the definition of the inbreeding coefficient, and examines homozygotes according to whether they arose as allozygotes or autozygotes. Recall that autozygous alleles have the same allelic origin, the likelihood (frequency) of which <strong><em>is</em></strong> the <strong>inbreeding coefficient (<em>f</em>)</strong> by definition. The proportion arising <em>allozygously</em> is therefore <strong>(1-f)</strong>. For the <strong>A</strong>-bearing gametes, which are present with a general frequency of <strong>p</strong>, the overall frequency of those which are autozygous is therefore (<strong>f</strong> <em>p</em>). Similarly, for <strong>a</strong>-bearing gametes, the autozygous frequency is (<strong>f</strong> <em>q</em>). [Remember that the issue of auto/allo -zygosity can arise only for <em>homologous</em> alleles (that is <em>A</em> and <em>A</em>, or <em>a</em> and <em>a</em>), and not for <em>non-homologous</em> alleles (<em>A</em> and <em>a</em>), which cannot possibly have the <em>same allelic origin</em>.] These two viewpoints regarding genotype frequencies need to be connected in order to establish consistency.</p>

<p>Following firstly the <strong>auto/allo -zygosity</strong> viewpoint, consider the <em>allozygous</em> component. This occurs with the frequency of <strong>(1-f)</strong>, and the alleles unite according to the <em>random fertilization</em> quadratic expansion. Thus:- <strong>(1-f)[(p<sub>0</sub> + q<sub>0</sub>)<sup>2</sup>]</strong> = '''(1-f)[ p<sup>2</sup><sub>0</sub> + q<sup>2</sup><sub>0</sub> ] + (1-f)[ 2 p<sub>0</sub> q<sub>0</sub> ] '''. Consider next the <em>autozygous</em> component. As these alleles <strong>are</strong> <em>autozygous</em> (same allelic origin), they are effectively <strong>selfings</strong>, and produce either <strong>AA</strong> or <strong>aa</strong> genotypes, but no heterozygotes. They therefore produce (<strong><em>f</em> p<sub>0</sub></strong> + <strong><em>f</em> q<sub>0</sub></strong>) genotypes, for <em>AA</em> and <em>aa</em> respectively. Adding the two components together results in:- <em>' [(1-f) p<sup>2</sup><sub>0</sub> + f p<sub>0</sub>]</em>' for the <strong>AA</strong> homozygote; <strong>[(1-f) q<sup>2</sup><sub>0</sub> + f q<sub>0</sub>]</strong> for the <strong>aa</strong> homozygote; and <strong>(1-f)[ 2 p<sub>0</sub> q<sub>0</sub> ]</strong> for the <strong>Aa</strong> heterozygote.<a class="footnoteRef" href="#fn64" id="fnref64"><sup>64</sup></a><a class="footnoteRef" href="#fn65" id="fnref65"><sup>65</sup></a> This is the same equation presented earlier in the section on "Self fertilization - an alternative". The reason for the decline in heterozygosity is made clear here. Heterozygotes can arise <strong><em>only</em></strong> from the allozygous component, and its frequency in the mix is just <strong>(1-f)</strong> - the same as the frequency used for the heterozygotes.</p>

<p>Secondly, the <strong>sampling rise/fall</strong> viewpoint is followed. The <strong><em>decline in heterozygotes</em></strong>, <strong>f (2 p<sub>0</sub> q<sub>0</sub>)</strong> , is distributed equally towards each homozygote and added to their basic <em>random fertilization</em> expectations. Therefore, the frequencies are:- <strong>(p<sup>2</sup><sub>0</sub> + <em>f</em> p<sub>0</sub> q<sub>0</sub> )</strong> for the <strong>AA</strong> homozygote; <strong>(q<sup>2</sup><sub>0</sub> + <em>f</em> p<sub>0</sub> q<sub>0</sub> )</strong> for the <strong>aa</strong> homozygote; and <strong>2 p<sub>0</sub> q<sub>0</sub> - <em>f</em> (2 p<sub>0</sub> q<sub>0</sub>)</strong> for the heterozygote.</p>

<p>Thirdly, the <strong>consistency</strong> proof begins with considering the <strong>AA</strong> homozygote's <strong><em>final</em></strong> equation in the <em>auto/allo -zygosity</em> paragraph above. Taking that equation "[ (1-f) p<sup>2</sup><sub>0</sub> + f p<sub>0</sub> ]", expand the parentheses, and follow by re-gathering [within the resultant] the two new terms with the common-factor <em>f</em> in them [resulting in:- '' p<sup>2</sup><sub>0</sub> - f ( '' <strong>p<sup>2</sup><sub>0</sub></strong> ''- p<sub>0</sub> ) '']. Next, in the " <strong>p<sup>2</sup><sub>0</sub></strong> " in that previous parenthesis part, one of the <strong>p</strong> is substituted by <strong>(1-q)</strong>, becoming " '''(1-q<sub>0</sub> ) p<sub>0</sub> ''' " instead of " <strong>p<sup>2</sup><sub>0</sub></strong> ". Following that substitution, it is a straightforward matter of multiplying-out, simplifying and watching signs. The end result is " <em>(p<sup>2</sup><sub>0</sub> + f p<sub>0</sub> q<sub>0</sub> )</em> " , which is exactly the result for <strong>AA</strong> in the <em>sampling rise/fall</em> paragraph. The two viewpoints are therefore <strong><em>consistent</em></strong> for the <strong>AA</strong> homozygote. In a like manner, the consistency of the <strong>aa</strong> viewpoints can also be proven. For the heterozygote, the two viewpoints are in agreement from the beginning.</p>
<h3 id="allele-shuffling---allele-substitution">Allele shuffling - allele Substitution</h3>

<p>The <strong>gene-model</strong> examines the heredity pathway from the point of view of "inputs" (alleles/gametes) and "outputs" (genotypes/zygotes), with fertilization being the "process" converting one to the other. An alternative viewpoint concentrates on the "process" itself, and considers the zygote genotypes as arising from allele shuffling. In particular, it regards the results as if one allele had "substituted" for the other during the shuffle, together with a residual which deviates from this view. This formed an integral part of Fisher's method,<a class="footnoteRef" href="#fn66" id="fnref66"><sup>66</sup></a> in addition to his use of frequencies and effects to generate his genetical statistics.<a class="footnoteRef" href="#fn67" id="fnref67"><sup>67</sup></a>  A discursive derivation of the <strong>Allele Substitution</strong> alternative follows.<a class="footnoteRef" href="#fn68" id="fnref68"><sup>68</sup></a> Suppose that the usual random fertilization of gametes in a "base" gamodeme - consisting of <em><strong>p</strong></em> gametes (<strong>A</strong>) and <em><strong>q</strong></em> gametes (<strong>a</strong>) - is replaced by fertilization with a "flood" of gametes all containing a single allele (<strong>A</strong> or <strong>a</strong>, but not both). The zygotic results can be interpreted in terms of the "flood" allele having "substituted for" the alternative allele in the underlying "base" gamodeme. The diagram assists in following this viewpoint: the upper part pictures an <strong>A</strong> substitution, while the lower part shows an <strong>a</strong> substitution. (The diagram's "RF allele" is the allele in the "base" gamodeme.)</p>

<p>Consider the upper part firstly. Because <em>base</em> <strong>A</strong> is present with a frequency of <strong><em>p</em></strong>, the <em>substitute</em> <strong>A</strong> will fertilize it with a frequency of <strong><em>p</em></strong> resulting in a zygote <strong>AA</strong> with an allele effect of <strong><em>a</em></strong>. Its contribution to the outcome will therefore be the product <strong><em>p a</em></strong>. Similarly, when the <em>substitute</em> fertilizes <em>base</em> <strong>a</strong> (resulting in <strong>Aa</strong> with a frequency of <strong><em>q</em></strong> and heterozygote effect of <strong><em>d</em></strong>), the contribution will be <strong><em>q d</em></strong>. The overall result of substitution by <strong>A</strong> will therefore be <strong>pa + qd</strong>. This is now oriented towards the population mean (see earlier section) by expressing it as a deviate from that mean : <strong>(pa + qd) - G</strong>. After some algebraic simplification, this becomes <strong>α<sub>A</sub> = q [a + (q-p)d]</strong> - the <strong><em>substitution effect</em></strong> of <strong>A</strong>.</p>

<p>A parallel reasoning can be applied to the lower part of the diagram, taking care with the differences in frequencies and gene effects. The result is the <strong><em>substitution effect</em></strong> of <strong>a</strong>, which is <strong>α<sub>a</sub> = -p [a + (q-p)d]</strong>.</p>

<p>The common factor inside the brackets is known as the <strong><em>average allele substitution effect</em></strong>, and is usually given as <strong>α = a + (q-p)d</strong>.<a class="footnoteRef" href="#fn69" id="fnref69"><sup>69</sup></a> It can be derived also in a more direct way, but the result is the same.</p>

<p>In subsequent sections, these <strong>substitution effects</strong> will be used to define the gene-model genotypes as consisting of a partition predicted by these new effects (substitution <em>expectations</em>), and a deviation between these expectations and the previous gene-model effect. The <em>expectations</em> are also known as the <strong>breeding value</strong>, and the deviations are also known as the <em>dominance deviations</em>. Ultimately, the variance arising from the <strong><em>substitution expectations</em></strong> will become the <strong>Additive Genetic variance (σ<sup>2</sup><sub>A</sub>)</strong> <a class="footnoteRef" href="#fn70" id="fnref70"><sup>70</sup></a> (also the <strong><em>Genic</em> variance</strong> <ref name="M&amp;J 1971">); while that arising from the deviations becomes the <strong>Dominance variance (σ<sup>2</sup><sub>D</sub>)</strong>.</ref></p>
<h2 id="extended-principles">Extended principles</h2>
<h3 id="gene-effects-redefined">Gene effects redefined</h3>

<p>The gene-model effects (<strong>a</strong>, <strong>d</strong> and <strong>-a</strong>) are important soon in the derivation of the <em>deviations from substitution expectations</em> (<strong>δ</strong>), which were first discussed in the previous <em>Allele Substitution</em> section. However, they need to be redefined themselves before they become useful in that exercise. They firstly need to be re-centralized around the population mean (<strong>G</strong>), and secondly they need to be re-arranged as functions of <strong>α</strong>, the <em>average allele substitution effect</em>.</p>

<p>The re-centralized effect for <strong>AA</strong>, therefore, is <strong>a′ = a - G</strong> which, after simplification, becomes <strong>a′ = 2<em>q</em>(a-<em>p</em>d)</strong>. The similar effect for <strong>Aa</strong> is <strong>d′ = d - G = a(<em>q</em>-<em>p</em>) + d(1-2<em>pq</em>)</strong>, after simplification. Finally, the re-centralized effect for <strong>aa</strong> is <strong>(-a)′ = -2<em>p</em>(a+<em>q</em>d)</strong>.<a class="footnoteRef" href="#fn71" id="fnref71"><sup>71</sup></a></p>

<p>These re-centralized effects eventually will have the <em>genotype substitution expectations</em> (see next section) subtracted from them in order to define subsequently the <em>genotype substitution deviations</em>. These <em>expectations</em> each are a function of the <em>average allele substitution effect</em>, and the present re-centralized effects have to be re-arranged still further to accommodate this last subtraction. Recalling that α = [a +(q-p)d], rearrangement gives <strong>a = [α -(q-p)d]</strong>. After substituting this for <strong>a</strong> in <strong>a′</strong> and simplifying, the final version becomes <strong>a′′ = 2q(α-qd)</strong>. Similarly, <strong>d′</strong> becomes <strong>d′′ = α(q-p) + 2pqd</strong>; and <strong>(-a)′</strong> becomes <strong>(-a)′′ = -2p(α+pd)</strong>.<a class="footnoteRef" href="#fn72" id="fnref72"><sup>72</sup></a></p>
<h3 id="genotype-substitution---expectations-and-deviations">Genotype substitution - Expectations and Deviations</h3>

<p>The zygote genotypes are the target of all this preparation. The homozygous genotype <strong>AA</strong> is a union of <em>two</em> <strong>substitution effects of A</strong>, one coming from each sex. Its <strong>substitution expectation</strong> is therefore <strong>α<sub>AA</sub> = 2α<sub>A</sub> = 2<em>q</em>α</strong> (see previous sections). Similarly, the <em>substitution expectation</em> of <strong>Aa</strong> is <strong>α<sub>Aa</sub> = α<sub>A</sub> + α<sub>a</sub> = (<em>q</em>-<em>p</em>)α</strong> ; and for <strong>aa</strong>, <strong>α<sub>aa</sub> = 2α<sub>a</sub> = -2<em>p</em>α</strong>. These <strong>substitution expectations</strong> of the genotypes are known also as <strong>"Breeding values"</strong>.<a class="footnoteRef" href="#fn73" id="fnref73"><sup>73</sup></a></p>

<p>The <strong>substitution deviations</strong> are the differences between these <em>expectations</em> and the <em>gene effects</em> after their two-stage redefinition in the previous section. Therefore, <strong>δ<sub>AA</sub> = a′′ - α<sub>AA</sub> = -2<em>q</em><sup>2</sup>d</strong> after simplification. Similarly, <strong>δ<sub>Aa</sub> = d′′ - α<sub>Aa</sub> = 2<em>pq</em>d</strong> after simplification. Finally, <strong>δ<sub>aa</sub> = (-a)′′ - α<sub>aa</sub> = -2<em>p</em><sup>2</sup>d</strong> after simplification.<a class="footnoteRef" href="#fn74" id="fnref74"><sup>74</sup></a></p>

<p>The <strong>genotype substitution <em>expectations</em></strong> will give rise ultimately to the <strong>σ<sup>2</sup><sub>A</sub></strong>, and the <strong>genotype substitution <em>deviations</em></strong> will give rise to the <strong>σ<sup>2</sup><sub>D</sub></strong>.</p>
<h3 id="genotypic-variance">Genotypic variance</h3>

<p>There are two major approaches to defining and partitioning the <strong>Genotypic variance</strong> : one is based on the <strong>gene-model effects</strong>,<ref name="M&amp;J 1971"> while the other is based on the <strong>genotype substitution effects</strong><a class="footnoteRef" href="#fn75" id="fnref75"><sup>75</sup></a> They are algebraically inter-convertible with each other.<a class="footnoteRef" href="#fn76" id="fnref76"><sup>76</sup></a> In this section, the basic <em>random fertilization</em> derivation is considered, with the effects of inbreeding and dispersion set aside. This will be dealt with later in order to arrive at a more general solution. Until this <em>mono-genic</em> treatment is replaced by a <em>multi-genic</em> one, and until <em>epistasis</em> is resolved in the light of the findings of <strong>epigenetics</strong>, the Genotypic variance will have only the components considered here.</ref></p>
<h4 id="gene-model-approach---mather-jinks-hayman">Gene-model approach - Mather Jinks Hayman</h4>

<p> It is convenient to follow the Biometrical approach : which is based on correcting the <em>unadjusted sum of squares</em> (<strong>USS</strong>) by subtracting the <em>correction factor</em> (<strong>CF</strong>). Because all of our effects have been examined through <em>frequencies</em>, the <strong>USS</strong> can be obtained as the sum of the products of each genotype's <strong>frequency</strong> and the <em>square</em> of its <strong>gene-effect</strong>. The <strong>CF</strong> in this particular case is the <strong>mean</strong> <em>squared</em>. The result is the <em>sum of squares</em> (<strong>SS</strong>), which, again because of the use of <em>frequencies</em>, is also immediately the <strong>variance</strong>.<a class="footnoteRef" href="#fn77" id="fnref77"><sup>77</sup></a></p>

<p>The <strong>USS</strong> = <strong><em>p</em><sup>2</sup>a<sup>2</sup> + 2<em>pq</em>d<sup>2</sup> + <em>q</em><sup>2</sup>(-a)<sup>2</sup></strong> , and the <strong>CF</strong> = <strong>G<sup>2</sup></strong> . The <strong>SS</strong> = <strong>USS - CF</strong> = <strong>σ<sup>2</sup><sub>G</sub></strong> .</p>

<p>After partial simplification,</p>

<p><em>' σ<sup>2</sup><sub>G</sub> = 2</em>pq<em>a<sup>2</sup> + (</em>q<em>-</em>p<em>)4</em>pq<em>ad + 2</em>pq<em>d<sup>2</sup> + (2</em>pq<em>)<sup>2</sup> d<sup>2</sup></em>' = '''σ<sup>2</sup><sub>a</sub> + (weighted_covariance)<sub>ad</sub> + σ<sup>2</sup><sub>d</sub> + σ<sup>2</sup><sub>D</sub> ''' = <strong>½D + ½F´ + ½H<sub>1</sub> + ¼H<sub>2</sub></strong> in Mather's terminology.<ref name="M&amp;J 1971"> <a class="footnoteRef" href="#fn78" id="fnref78"><sup>78</sup></a></ref></p>

<p>Here, <strong>σ<sup>2</sup><sub>a</sub></strong> represents the <em>homozygote</em> or <strong>allelic</strong> variance, and <strong>σ<sup>2</sup><sub>d</sub></strong> represents the <em>heterozygote</em> or <strong><em>gene-model</em> dominance</strong> variance. The <strong><em>random-fertilization</em> dominance</strong> variance (<strong>σ<sup>2</sup><sub>D</sub></strong>) is present also. These components are plotted across all values of <strong>p</strong> in the Figure accompanying. Notice that the <strong>(weighted_covariance)<sub>ad</sub></strong><a class="footnoteRef" href="#fn79" id="fnref79"><sup>79</sup></a> (hereafter abbreviated to <strong>cov<sub>ad</sub></strong>) is <em>negative</em> for <strong>0.5<p< strong="">. '''½d="" +="" further="" gathering="" leads="" of="" terms="" to="" ½f´="" ½h<sub>3</sub></p<></strong></p>

<p>+ ¼H<sub>2</sub>''', where <strong>½H<sub>3</sub> = (<em>q</em>-<em>p</em>)<sup>2</sup> ½H<sub>1</sub></strong> = <em>' (</em>q<em>-</em>p<em>)<sup>2</sup>2</em>pq<em>d<sup>2</sup></em>'. It will be useful later in Diallel analysis, which is an experimental design for estimating these genetical statistics.<a class="footnoteRef" href="#fn80" id="fnref80"><sup>80</sup></a></p>

<p>If, following the last-given rearrangements, the first three terms are amalgamated together, rearranged further and simplified, the result is the variance of the Fisherian <em>substitution expectation</em>. That is: <strong>σ<sup>2</sup><sub>A</sub></strong> = <strong>σ<sup>2</sup><sub>a</sub></strong> + <strong>cov<sub>ad</sub></strong> + <strong>σ<sup>2</sup><sub>d</sub></strong>, a revealing insight indeed. Notice particularly that <strong>σ<sup>2</sup><sub>A</sub></strong> is not <strong>σ<sup>2</sup><sub>a</sub></strong>.<a class="footnoteRef" href="#fn81" id="fnref81"><sup>81</sup></a> Notice also that <strong>σ<sup>2</sup><sub>D</sub></strong> = <strong>2pq σ<sup>2</sup><sub>d</sub></strong>. From the Figure, this can be visualized as accumulating <strong>σ<sup>2</sup><sub>a</sub></strong>, <strong>σ<sup>2</sup><sub>d</sub></strong> and <strong>cov</strong> to obtain <strong>σ<sup>2</sup><sub>A</sub></strong>, while leaving the <strong>σ<sup>2</sup><sub>D</sub></strong> still separated. It is clear also in the Figure that <strong>σ<sup>2</sup><sub>D</sub></strong> 2<sub>d</sub>''', as expected from the equations. </p>

<p>The overall result is <strong>σ<sup>2</sup><sub>G</sub></strong> = <strong>2<em>pq</em> [a+(<em>q</em>-<em>p</em>)d]<sup>2</sup> + (2<em>pq</em>)<sup>2</sup> d<sup>2</sup></strong> = '''σ<sup>2</sup><sub>A</sub> + σ<sup>2</sup><sub>D</sub> '''.</p>

<p>However, its derivation via the <em>substitution effects</em> themselves will be given also, in the next section.</p>
<h4 id="allele-substitution-approach---fisher">Allele-substitution approach - Fisher</h4>

<p>Reference to the several earlier sections on <strong>allele substitution</strong> reveals that the two ultimate effects are <strong>genotype substitution <em>expectations</em></strong> and '''genotype substitution <em>deviations<strong>''. Notice that these are each defined already as deviations from the <em>random fertilization</em> population mean (</strong>G</em>'). For each genotype in turn, the product of the frequency and the square of the relevant effect is obtained, and these are accumulated to obtain directly a <strong>SS</strong> and <strong>σ<sup>2</sup></strong>. Details follow.</p>

<p><strong>σ<sup>2</sup><sub>A</sub></strong> = <strong><em>p</em><sup>2</sup> α<sup>2</sup><sub>AA</sub></strong> + <strong>2<em>pq</em> α<sup>2</sup><sub>Aa</sub></strong> + <strong><em>q</em><sup>2</sup> α<sup>2</sup><sub>aa</sub></strong> which simplifies to <strong>σ<sup>2</sup><sub>A</sub> = 2<em>pq</em>α<sup>2</sup></strong>.</p>

<p><strong>σ<sup>2</sup><sub>D</sub></strong> = <strong><em>p</em><sup>2</sup> δ<sup>2</sup><sub>AA</sub></strong> + <strong>2<em>pq</em> δ<sup>2</sup><sub>Aa</sub></strong> + <strong><em>q</em><sup>2</sup> δ<sup>2</sup><sub>aa</sub></strong> which simplifies to <strong>σ<sup>2</sup><sub>D</sub> = (2<em>pq</em>)<sup>2</sup> d<sup>2</sup></strong>.</p>

<p>Once again, <strong>σ<sup>2</sup><sub>G</sub></strong> = '''σ<sup>2</sup><sub>A</sub> + σ<sup>2</sup><sub>D</sub> '''.</p>

<p>Note that this allele-substitution approach defined the components separately, and then totaled them to obtain the final Genotypic variance. Conversely, the gene-model approach derived the whole situation (components and total) as one exercise. Bonuses arising from this were (a) the revelations about the real structure of <strong>σ<sup>2</sup><sub>A</sub></strong>, and (b) the relative sizes of <strong>σ<sup>2</sup><sub>d</sub></strong> and <strong>σ<sup>2</sup><sub>D</sub></strong> (see previous sub-section). It is also apparent that a "Mather" analysis is more informative, and that a "Fisher" analysis can always be constructed from it. The opposite conversion is not possible, however, because information about <strong>cov<sub>ad</sub></strong> would be missing.</p>
<h3 id="other-fertilization-patterns">Other fertilization patterns</h3>
<figure><b>(Figure)</b>
<figcaption>Spatial fertilization patterns</figcaption>
</figure>

<p>In previous sections, dispersive random fertilization (<em>genetic drift</em>) has been considered comprehensively, and self-fertilization and hybridizing have been examined to varying degrees. The diagram to the left depicts the first two of these, along with another "spatially based" pattern: <strong>Islands</strong>. This is a pattern of <em>random fertilization</em> featuring <em>dispersed gamodemes</em>, with the addition of "overlaps" in which <em>non-dispersive</em> random fertilization occurs. With the <strong>Islands</strong> pattern, individual gamodeme sizes (<strong>2N</strong>) are observable, and overlaps (<strong>m</strong>) are minimal. This is one of Sewall Wright's array of possibilities.<a class="footnoteRef" href="#fn82" id="fnref82"><sup>82</sup></a> In addition to "spatially" based patterns of fertilization, there are others based on either "phenotypic" or "relationship" criteria. The <em>phenotypic</em> bases include <strong>assortative</strong> fertilization (between similar phenotypes) and <strong>disassortative</strong> fertilization (between opposite phenotypes). The <em>relationship</em> patterns include <strong>sib crossing</strong>, <strong>cousin crossing</strong> and <strong>backcrossing</strong>, and are considered in a separate section. <strong>Self fertilization</strong> may be considered both from a <em>spatial</em> and from a <em>relationship</em> point of view. <a href="File:RF_Inbreeding_C_c.jpg" title="wikilink">thumb|300px|rightt|"Islands" random fertilization.</a></p>
<h4 id="islands-random-fertilization">"Islands" random fertilization</h4>

<p>The breeding population consists of <strong>s</strong> small <em>dispersed random fertilization</em> gamodemes of sample size <strong>2N<sub>k</sub></strong> ( <strong>k</strong> = 1 ... <em>s</em> ) with " <strong>overlaps</strong> " of proportion <strong>m<sub>k</sub></strong> in which ''non-dispersive random fertilization '' occurs. The ''' dispersive proportion ''' is thus ''' (1 - m<sub>k</sub>) '''. The bulk population consists of <em>weighted averages</em> of sample sizes, allele and genotype frequencies and progeny means, as was done for <strong>genetic drift</strong> in an earlier section. However, each <em>gamete sample size</em> is reduced to allow for the <em>overlaps</em>, thus finding a ''' 2N<sub>k</sub> <em>effective for</em> (1 - m<sub>k</sub>) '''.</p>

<p>For brevity, the argument will be followed further with the subscripts omitted. Recall that <em>1 / (2N)</em> is <em>Δf</em>. Therefore, ''' <sub>Islands</sub> Δf = (1 - m)<sup>2</sup> / [2N - (2N-1)m<sup>2</sup> ] '''. Notice that when <em>m = 0</em> this reduces to the previous <em>Δf</em>. This is substituted into the <em>inbreeding coefficient</em> to obtain ''' <sub>Islands</sub>f<sub>t</sub> = <sub>Islands</sub>Δf<sub>t</sub> + ( 1 - <sub>Islands</sub>Δf<sub>t</sub> ) <sub>Islands</sub>f<sub>(t-1)</sub> ''', where <em>t</em> is the index over generations, as before. The effective <em>overlap proportion</em> can be obtained also, as '''m<sub>t</sub> = 1 - [ ( 2N <sub>Islands</sub>Δf<sub>t</sub> ) / ((2N - 1) <sub>Islands</sub>Δf<sub>t</sub> + 1 )] <sup>(1/2)</sup> '''. Here, the <em>2N</em> refers to the <strong><em>un-reduced</em></strong> sample size, not the <em>Islands</em> adjustment.</p>

<p>The graphs to the right show the <strong>inbreeding</strong> for a gamodeme size of <strong>2N = 50</strong> for ''ordinary dispersed random fertilization '' <strong>(RF) (m=0)</strong>, and for <em>four overlap levels</em> <strong>( m = 0.0625, 0.125, 0.25, 0.5 )</strong> of <strong>Islands</strong> <em>random fertilization</em>. There has indeed been reduction in the inbreeding resulting from the <em>non-dispersed random fertilization</em> in the overlaps: it is particularly notable as <strong>m <em>appraoches</em> 0.50</strong>. Sewall Wright suggested that this value should be the limit for the use of this approach.</p>
<h3 id="dispersion-and-the-genotypic-variance">Dispersion and the Genotypic Variance</h3>

<p>In the section on <strong>Genetic drift</strong> , and in other sections where <strong>Inbreeding</strong> has been discussed, a major outcome resulting from the allele frequency sampling has been the <strong>dispersion</strong> of progeny means. This collection of means has its own average, and also will have a variance - the <strong>amongst-line variance</strong>. (This is a variance of the attribute itself, not of <em>allele frequencies</em>.) As dispersion develops further over succeeding generations, this amongst-line variance would be expected to increase. Conversely, as homozygosity rises, the variance <strong>within-lines</strong> would be expected to decrease. The question arises therefore as to whether the <strong>total variance</strong> is changing, and, if so, in what direction. These issues are examined for both the <strong>genic (σ<sup>2</sup><sub>A</sub> )</strong> and the <strong>dominance ( σ<sup>2</sup><sub>D</sub> )</strong> variances. </p>

<p>The crucial <em>overview equation</em> comes from Sewall Wright,<a class="footnoteRef" href="#fn83" id="fnref83"><sup>83</sup></a> <a class="footnoteRef" href="#fn84" id="fnref84"><sup>84</sup></a> and is the outline of the <em>inbred <strong>genotypic</strong> variance</em> based on a weighted average of its extremes, the weights being quadratic with respect to the <strong>inbreeding coefficient ( f )</strong>. This equation is:-</p>

<p>''' σ<sup>2</sup><sub>G(f)</sub> = (1-f) σ<sup>2</sup><sub>G(0)</sub> + f σ<sup>2</sup><sub>G(1)</sub> + f (1-f) [ G<sub>0</sub> - G<sub>1</sub> ]<sup>2</sup> ''',</p>

<p>where <em>f</em> is the inbreeding coefficient, '' σ<sup>2</sup><sub>G(0)</sub> '' is the genotypic variance at f=0, '' σ<sup>2</sup><sub>G(1)</sub> '' is the genotypic variance at f=1, '' G<sub>0</sub> '' is the population mean at f=0, and '' G<sub>1</sub> '' is the population mean at f=1. The <strong>(1-f)</strong> component concerns the reduction of variance <strong>within progeny lines</strong> ; the <strong>f</strong> component addresses the increase in variance <strong>amongst progeny lines</strong> ; while the <strong>f (1-f)</strong> component will be seen (in the next line) to address a part of the <strong>dominance</strong> variance.<a class="footnoteRef" href="#fn85" id="fnref85"><sup>85</sup></a>  These components can be expanded to reveal further insight.</p>

<p>Thus:- ''' σ<sup>2</sup><sub>G(f)</sub> = (1-f) [ σ<sup>2</sup><sub>A(0)</sub> + σ<sup>2</sup><sub>D(0)</sub> ] + f (4pq a<sup>2</sup> ) + f (1-f) (2pq d)<sup>2</sup> '''.</p>

<p>In the first component, <em>σ<sup>2</sup><sub>G(0)</sub></em> has been expanded to show its two variance components as previously defined . The <em>σ<sup>2</sup><sub>G(1)</sub></em> in the second component is ''4pqa<sup>2</sup> '' [which, recall, equals <em>2 (σ<sup>2</sup><sub>a</sub></em>) ] and will be derived shortly. The third component's substitution is the result of the subtraction between the two "inbreeding extremes" of the population means (see section on the "Population Mean").<a class="footnoteRef" href="#fn86" id="fnref86"><sup>86</sup></a></p>

<p><strong><em>Summarising</em></strong> therefore gives:- the <strong>within-line</strong> components are <em>(1-f) σ<sup>2</sup><sub>A(0)</sub></em> and <em>(1-f) σ<sup>2</sup><sub>D(0)</sub></em> ; and the <strong>amongst-line</strong> components are '' 2f σ<sup>2</sup><sub>a(0)</sub> '' and '' (f - f <sup>2</sup>) σ<sup>2</sup><sub>D(0)</sub>''.<a class="footnoteRef" href="#fn87" id="fnref87"><sup>87</sup></a></p>

<p>The '''total Genic variance <em>' (</em>Additive Genetic variance'') is thus ''' [ (1-f) σ<sup>2</sup><sub>A(0)</sub> + 2f σ<sup>2</sup><sub>a(0)</sub> ] = (1+f) σ<sup>2</sup><sub>A(f)</sub> ''', where the '' σ<sup>2</sup><sub>A(f)</sub> '' will be discussed shortly in a sub-section. Similarly, the <strong>total Dominance variance</strong> is thus ''' [ (1-f) σ<sup>2</sup><sub>D(0)</sub> + (f - f <sup>2</sup>) σ<sup>2</sup><sub>D(0)</sub> ] = (1-f<sup>2</sup>) σ<sup>2</sup><sub>D(0)</sub> '''. Graphs to the left show these three <em>Genic variances</em>, together with the three <em>Dominance variances</em>, across all values of <strong>f</strong>, for <strong>p = 0.5</strong> (at which the dominance variance is at a maximum). Graphs to the right show the <strong>Genotypic</strong> variance partitions (being the sums of the respective <em>genic</em> and <em>dominance</em> partitions) changing over ten generations with an example <em>f = 0.10</em>. </p>

<p>Answering, firstly, the questions posed at the beginning about the <strong>total variances</strong> [the <strong>Σ</strong> in the graphs] : the <strong>Genic variance</strong> rises linearly with the <strong>inbreeding coefficient</strong>, maximizing at twice its starting level ; the <strong>Dominance variance</strong> declines at the rate of '''(1 - f<sup>2</sup> ) ''' [and therefore declines only slowly at low levels of inbreeding] until it finishes at <em>zero</em>. Now, notice the other trends. It is probably intuitive that the <strong>within line</strong> variances decline to zero with continued inbreeding, and this is seen to be the case [both at the same linear rate <em>(1-f)</em> ]. The <strong>amongst line</strong> variances both increase with inbreeding up to <strong><em>f = 0.5</em></strong>, the <em>Genic variance</em> at the rate of <strong>2f</strong>, and the <em>Dominance variance</em> at the rate of '''(f - f<sup>2</sup>) '''. At '''0.5 2) ''' also declines with '''0.5 </p>
<h4 id="derivation-of-σ2g1">Derivation of <em>σ<sup>2</sup><sub>G(1)</sub></em></h4>

<p>Recall that when <em>f=1</em>, heterozygosity is zero, within-line variance is zero, and <strong>all</strong> genotypic variance is thus <em>amongst-line</em> variance and deplete of dominance variance. In other words, <strong>σ<sup>2</sup><sub>G(1)</sub></strong> is the variance amongst fully inbred line means. Recall further (from "The mean after self-fertilization" section) that such means (G<sub>1</sub>'s, in fact) are <strong>G = a(p-q)</strong>. Substituting <em>(1-q)</em> for the <em>p</em>, gives <strong>G<sub>1</sub> = a (1 -2q)</strong> = <strong>a - 2aq</strong>.<a class="footnoteRef" href="#fn88" id="fnref88"><sup>88</sup></a> Therefore, the <strong>σ<sup>2</sup><sub>G(1)</sub></strong> is the <strong>σ<sup>2</sup><sub>(a-2aq)</sub></strong> actually. Now, in general, the <strong><em>variance of a difference (x-y)</em></strong> is <em>' [ σ<sup>2</sup><sub>x</sub> + σ<sup>2</sup><sub>y</sub> - 2 cov<sub>xy</sub> ]</em>'.<a class="footnoteRef" href="#fn89" id="fnref89"><sup>89</sup></a><a class="footnoteRef" href="#fn90" id="fnref90"><sup>90</sup></a>  Therefore, '''σ<sup>2</sup><sub>G(1)</sub> = [ σ<sup>2</sup><sub>a</sub> + σ<sup>2</sup><sub>2aq</sub> - 2 cov<sub>(a, 2aq)</sub> ] '''. But <strong>a</strong> (an allele <em>effect</em>) and <strong>q</strong> (an allele <em>frequency</em>) are <strong><em>independent</em></strong> - so this <strong>covariance</strong> is zero. Furthermore, <strong>a</strong> is a constant from one line to the next, so <strong>σ<sup>2</sup><sub>a</sub></strong> is zero also. And still further, <strong>2a</strong> is another constant (k), so the <strong>σ<sup>2</sup><sub>2aq</sub></strong> is of the type <em>σ<sup>2</sup><sub>k X</sub></em>. In general, the variance <em>σ<sup>2</sup><sub>k X</sub></em> is equal to '''k<sup>2</sup> σ<sup>2</sup><sub>X</sub> '''.<a class="footnoteRef" href="#fn91" id="fnref91"><sup>91</sup></a> Putting all this together reveals that ''' σ<sup>2</sup><sub>(a-2aq)</sub> = (2a)<sup>2</sup> σ<sup>2</sup><sub>q</sub> '''. Recall (from the section on "Continued genetic drift") that ''σ<sup>2</sup><sub>q</sub> = pq f ''. With <em>f=1</em> here within this present derivation, this becomes <em>pq 1</em> (that is <strong>pq</strong>), and it is substituted into the previous.</p>

<p>The final result is:- '''σ<sup>2</sup><sub>G(1)</sub> = σ<sup>2</sup><sub>(a-2aq)</sub> = 4a<sup>2</sup> pq = 2(2pq a<sup>2</sup>) = 2 σ<sup>2</sup><sub>a</sub> '''.</p>

<p>It follows immediately that ''<em>'f</em> σ<sup>2</sup><sub>G(1)</sub> = <em>f</em> 2 σ<sup>2</sup><sub>a</sub> '''. [This last <em>f</em> comes from the <em>initial Sewall Wright equation</em> : it is <strong><em>not</em></strong> the ''f '' just set to "1" in the derivation concluded two lines above.]</p>
<h4 id="total-dispersed-genic-variance---σ2af-and-αf">Total dispersed Genic variance - σ<sup>2</sup><sub>A(f)</sub> and α<sub>f</sub></h4>

<p>From the previous sections, it was found that the <strong>within line</strong> <em>genic variance</em> was based upon the <strong><em>Allele Substitution</em></strong> "Additive genetic variance" <strong>( σ<sup>2</sup><sub>A</sub> )</strong> ; but the <strong>amongst line</strong> <em>genic variance</em> was based upon the <strong><em>Gene Model</em></strong> "Allele variance" <strong>( σ<sup>2</sup><sub>a</sub> )</strong>. These two cannot simply be added together to obtain the <strong>total Genic variance</strong>. [This was not a difficulty for the <em>Dominance variances</em> because all components referred to the same base - <strong>σ<sup>2</sup><sub>D</sub></strong>.] One approach to avoiding this problem was to re-visit the derivation of the " <em>average allele substitution effect</em> ", and to construct a version <strong>( α<sub><em>f</em></sub> )</strong> which would incorporate the effects of the dispersion. Crow and Kimura achieved this <a class="footnoteRef" href="#fn92" id="fnref92"><sup>92</sup></a>  using the re-centered allele effects ('''a’, d’, (-a)’ ''') discussed previously ["Gene effects re-defined"]. However, this was found subsequently to under-estimate slightly the <em>total Genic variance</em>, and a new variance-based derivation led to a refined version.<a class="footnoteRef" href="#fn93" id="fnref93"><sup>93</sup></a></p>

<p>The <em>refined</em> version is:- <em>' α<sub><em>f</em></sub> = { a<sup>2</sup> + [(1-</em>f'' ) / (1 + <em>f</em> )] 2(q - p ) ad + [(1-<em>f</em> ) / (1 + <em>f</em> )] (q - p )<sup>2</sup> d<sup>2</sup> } <sup>(1/2)</sup></p>

<p>Consequently, '''σ<sup>2</sup><sub>A(f)</sub> = (1 + <em>f</em> ) 2pq α<sub>f</sub> <sup>2</sup> ''' does now agree with <strong>[ (1-f) σ<sup>2</sup><sub>A(0)</sub> + 2f σ<sup>2</sup><sub>a(0)</sub> ]</strong> exactly.</p>
<h4 id="total-and-partitioned-dispersed-dominance-variances">Total and partitioned dispersed Dominance variances</h4>

<p>The <em>total genic variance</em> is of intrinsic interest in its own right. But it had had another important use as well: it was subtracted from the <strong><em>inbred Genotypic variance</em></strong> equation of Sewall Wright <a class="footnoteRef" href="#fn94" id="fnref94"><sup>94</sup></a> in order to provide an estimator for the <em>(total) Dominance variance</em> . An anomaly appeared, however, because the <em>total dominance variance</em> appeared to <strong><em>rise</em></strong> early in inbreeding despite the decline in heterozygosity.<a class="footnoteRef" href="#fn95" id="fnref95"><sup>95</sup></a>   Consequently, the <em>de novo</em> derivation [referred to above] refined the equation for <em>α<sub>f</sub></em>.<a class="footnoteRef" href="#fn96" id="fnref96"><sup>96</sup></a> At the same time, a direct solution for the <em>'total Dominance variance</em> was obtained, thus avoiding the need for the "subtraction" method of previous times. Furthermore, by incorporating the expanded Sewall Wright equation, direct solutions for the dispersion partitions of the <em>Dominance variance</em> were obtained also.</p>
<h3 id="environmental-variance">Environmental variance</h3>

<p>The environmental variance is phenotypic variability which cannot be ascribed to genetics. This sounds simple, but the experimental design needed to separate the two needs very careful planning. Even the "external" environment can be divided into spatial and temporal components, as well as partitions such as "litter" or "family" and "culture" or "history". Where does epigenetic variance get placed? Is it embedded within epistasis: or is it "internal environment"? These components are very dependent upon the actual experimental model used to do the research. Such issues are very important when doing the research itself, but in this article on quantitative genetics this overview may suffice.</p>

<p>It is an appropriate place, however, for a summary:</p>

<p>Phenotypic variance = genotypic variances + environmental variances + <a href="genotype-environment_interaction" title="wikilink">genotype-environment interaction</a> + experimental "error" variance</p>

<p><em>ie</em> σ²<sub>P</sub> = σ²<sub>G</sub> + σ²<sub>E</sub> + σ²<sub>GE</sub> + σ²</p>

<p>'' or'' σ²<sub>P</sub> = σ²<sub>A</sub> + σ²<sub>D</sub> + σ²<sub>I</sub> + σ²<sub>E</sub> + σ²<sub>GE</sub> + σ²</p>

<p>after partitioning the genotypic variance (G) into the components of "additive" (A), "dominance" (D), and "epistasic" (I) variance mentioned above.<a class="footnoteRef" href="#fn97" id="fnref97"><sup>97</sup></a></p>
<h3 id="heritability-and-repeatability">Heritability and repeatability</h3>

<p>The <a class="uri" href="heritability" title="wikilink">heritability</a> of a trait is the proportion of the total (phenotypic) variance (σ²<sub>P</sub>) that is explained by the total genotypic variance (σ²<sub>G</sub>). This is known as the "broad sense" heritability (H<sup>2</sup>).<a class="footnoteRef" href="#fn98" id="fnref98"><sup>98</sup></a> If only additive genetic variance (σ²<sub>A</sub>) is used in the numerator, the heritability is called "narrow sense" (h<sup>2</sup>).</p>

<p>The broad sense heritability indicates the proportion of the phenotypic variance due to the whole genotypical variance. In colloquial terms, it indicates the extent of "nature" while (1-H<sup>2</sup>) indicates the extent of "nurture". Narrow sense heritability indicates the proportion of the phenotypic variance attributable to the "additive" genetic variance, discussed above. It was pointed out there that this variance arises through substitution (<em>i.e.</em> phenotypic change) following fertilization. Fisher proposed that this narrow-sense heritability might be appropriate in considering the results of natural selection, focusing as it does on change-ability, and hence adaptation.<a class="footnoteRef" href="#fn99" id="fnref99"><sup>99</sup></a> It has been used also for predicting generally the results of artificial selection. In the latter case, however, the broad sense heritability may be more appropriate, as the whole attribute is being altered: not just adaptive capacity. Generally, advance from selection is more rapid with higher heritability. In animals, heritability of reproductive traits is typically low, while heritability of disease resistance and production are moderately low to moderate, and heritability of body conformation is high.</p>

<p>Repeatability (r<sup>2</sup>) is the proportion of phenotypic variance attributable to differences in repeated measures of the same subject, arising from later records. It is used particularly for long-lived species. This value can only be determined for traits that manifest multiple times in the organism's lifetime, such as adult body mass, metabolic rate or litter size. Individual birth mass, for example, would not have a repeatability value: but it would have a heritability value. Generally, but not always, repeatability indicates the upper level of the heritability.<a class="footnoteRef" href="#fn100" id="fnref100"><sup>100</sup></a></p>

<p>r<sup>2</sup> = (σ²<sub>G</sub> + σ²<sub>PE</sub>)/σ²<sub>P</sub></p>

<p>where σ²<sub>PE</sub> = phenotype-environment interaction ≡ repeatability.</p>

<p>The above concept of repeatability is, however, problematic for traits that necessarily change greatly between measurements. For example, body mass increases greatly in many organisms between birth and adult-hood. Nonetheless, within a given age range (or life-cycle stage), repeated measures could be done, and repeatability would be meaningful within that stage.</p>
<h2 id="relationship">Relationship</h2>
<figure><b>(Figure)</b>
<figcaption>Connection between the inbreeding and co-ancestry coefficients.</figcaption>
</figure>

<p>From the heredity perspective, relations are individuals which have inherited genes from one or more common ancestors. Therefore, their "relationship" can be <em>quantified</em> on the basis of the probability that they each have inherited a copy of an allele which had its origin in the common ancestor. In earlier sections, the <strong>Inbreding coefficient</strong> has been defined as "the probability that two <em>same</em> alleles ( <strong>A</strong> and <strong>A</strong>, or <strong>a</strong> and <strong>a</strong> ) have a common origin"; or, more formally, " <strong>the probability that two homologous alleles are autozygous</strong> ". Previously, the emphasis was on an individual's likelihood of having two such alleles, and the coefficient was framed accordingly. It is obvious, however, that this probability of autozygosity for an individual must also be the probability that each of its <em>two parents</em> had this autozygous allele. In this re-focused form, the probability is called the <strong>co-ancestry coefficient</strong> for the two individuals <em>i</em> and <em>j</em> ( <strong><em>f</em> <sub>ij</sub></strong> ). In this form, it can be used to quantify relationship between two individuals, and may also be known as the " <strong>coefficient of kinship</strong> " or the " <strong>consanguinity coefficient</strong> ".<a class="footnoteRef" href="#fn101" id="fnref101"><sup>101</sup></a> <a class="footnoteRef" href="#fn102" id="fnref102"><sup>102</sup></a></p>
<h3 id="pedigree-analysis">Pedigree analysis</h3>

<p> <strong>Pedigrees</strong> are diagrams which show the familial connections between individuals and their ancestors, and possibly between other members of the group sharing genetical inheritance with them. They are "relationship maps". A pedigree can be analyzed, therefore, to reveal coefficients of inbreeding and co-ancestry. Such pedigrees actually are informal depictions of <strong>path diagrams</strong> as used in <strong>path analysis</strong>, which was invented by Sewall Wright when he formulated his studies on inbreeding.<a class="footnoteRef" href="#fn103" id="fnref103"><sup>103</sup></a> Using the diagram to the left, the probability that individuals "B" and "C" have received autozygous alleles from ancestor "A" is <strong>1/2</strong> (one out of the two diploid alleles). This is the "de novo" inbreeding (<strong>Δf<sub>Ped</sub></strong>) at this step. However, the "other" allele may have had "carry-over" autozygosity from previous generations, so the probability of this occurring is ('' de novo complement '' <strong>multiplied by the</strong> '' inbreeding of ancestor A'' ), that is ''' (1 - Δf<sub>Ped</sub> ) f<sub>A</sub> ''' = ''' (1/2) f<sub>A</sub> '''. Therefore, the total probability of autozygosity in B and C, following the bi-furcation of the pedigree, is the sum of these two components, namely ''' (1/2) + (1/2)f<sub>A</sub> ''' = ''' (1/2) (1+f <sub>A</sub> ) '''. This can be viewed as the <em>'</em>probability that two random gametes from ancestor A bear autozygous alleles <em>'</em>, and in that context is called the " <strong>coefficient of parentage</strong> " = ''' f<sub>AA</sub> '''.<a class="footnoteRef" href="#fn104" id="fnref104"><sup>104</sup></a> <a class="footnoteRef" href="#fn105" id="fnref105"><sup>105</sup></a> It will re-appear often in the following paragraphs.</p>

<p>Following the "B" path, the probability that any autozygous allele is "passed on" to each successive parent is again <strong>(1/2)</strong> at each step (including the last one to the "target" <strong>X</strong> ). The overall probability of transfer down the "B path" is therefore ''' (1/2)<sup>3</sup> '''. The power that (1/2) is raised to can be viewed as "the number of intermediates in the path between <strong>A</strong> and <strong>X</strong> ", '''n<sub>B</sub> = 3 '''. Similarly, for the "C path", ''' n<sub>C</sub> = 2 ''', and the "transfer probability" is ''' (1/2)<sup>2</sup> '''. The combined probability of autozygous transfer from <strong>A</strong> to <strong>X</strong> is therefore '''[ f<sub>AA</sub> (1/2)<sup>(n<sub>B</sub>)</sup> (1/2)<sup>(n<sub>C</sub>)</sup> ] '''. Recalling that '' f<sub>AA</sub> = (1/2) (1+f <sub>A</sub> ) '', ''' f<sub>X</sub> = f<sub>PQ</sub> ''' = ''' (1/2)<sup>(n<sub>B</sub> + n<sub>C</sub> + 1)</sup> (1 + f<sub>A</sub> ) '''. In this example, assuming that f<sub>A</sub> = 0, ''' f<sub>X</sub> = 0.0156 ''' (rounded) = '''f<sub>PQ</sub> ''', one measure of the "relatedness" between <strong>P</strong> and <strong>Q</strong>.</p>
<figure><b>(Figure)</b>
<figcaption>Cross-multiplication rules.</figcaption>
</figure>
<h3 id="cross-multiplication-rules">Cross-multiplication rules</h3>

<p>In the following sections on sib-crossing, and similar, a number of "averaging rules" will be found useful. These are derived from Path Analysis.<a class="footnoteRef" href="#fn106" id="fnref106"><sup>106</sup></a> The rules show that any <strong>co-ancestry coefficient</strong> can be obtained as the <em>average</em> of " <em>cross-over co-ancestries</em> " between appropriate grand-parental and parental combinations. Thus, referring to the diagram to the right, <strong>Cross-Multiplier 1</strong> is that <strong>f<sub>PQ</sub></strong> = average of ( <strong>f<sub>AC</sub></strong> , <strong>f<sub>AD</sub></strong> , <strong>f<sub>BC</sub></strong> , <strong>f<sub>BD</sub></strong> ) = ''' (1/4) [f<sub>AC</sub> + f<sub>AD</sub> + f<sub>BC</sub> + f<sub>BD</sub> ] ''' = '''f<sub>Y</sub> ''' also. In a similar fashion, '''Cross-Multiplier 2 ''' states that <strong>f<sub>PC</sub> = (1/2) [ f<sub>AC</sub> + f<sub>BC</sub> ]</strong> ; while '''Cross-Multiplier 3 ''' states that <strong>f<sub>PD</sub> = (1/2) [ f<sub>AD</sub> + f<sub>BD</sub> ]</strong> . Returning to the first Multiplier, it can now be seen also to be <strong>f<sub>PQ</sub> = (1/2) [ f<sub>PC</sub> + f<sub>PD</sub> ]</strong>, which, after substituting <strong>Multipliers 2 and 3</strong>, resumes its original form.</p>

<p>In much of the following, the grand-parental generation will be referred to as <strong>(t-2)</strong> , the parent generation as <strong>(t-1)</strong> , and the "target" generation will be <strong>t</strong>.</p>
<h3 id="full-sib-crossing-fs">Full-Sib crossing (FS)</h3>
<figure><b>(Figure)</b>
<figcaption>Inbreeding in sibling relationships</figcaption>
</figure>

<p>The diagram to the right shows that ''Full Sib crossing '' is a direct application of '''Cross-Multiplier 1 ''', with the slight modification that <em>parents A and B</em> are repeated (in lieu of <em>C and D</em>) in order to indicate that individuals <em>P1</em> and <em>P2</em> have both of <em>their</em> parents in common - that is they are '''full siblings '''. Individual '''Y ''' is the result of the crossing of two full siblings. Therefore, '''f<sub>Y</sub> = f<sub>P1,P2</sub> = (1/4) [ f<sub>AA</sub> + 2 f<sub>AB</sub> + f<sub>BB</sub> ] '''. Recall that <strong>f<sub>AA</sub></strong> and <strong>f<sub>BB</sub></strong> were defined earlier (in Pedigree analysis) as '''coefficients of parentage ''', being equal to '' (1/2)[1+f<sub>A</sub> ] '' and '' (1/2)[1+f<sub>B</sub> ] '' respectively, in the present context. Recognize that, in this guise, the grandparents <em>A</em> and <em>B</em> represent '''generation (t-2) '''. Thus, assuming that in any one generation all levels of inbreeding are the same, these two ''coefficients of parentage '' each represent '''(1/2) [1 + f<sub>(t-2)</sub> ] '''.  Now, examine '''f<sub>AB</sub> '''. Recall that this also is ''f<sub>P1</sub> '' or ''f<sub>P2</sub> '', and so represents ''their '' generation - '''f<sub>(t-1)</sub> '''. Putting it all together, '''f<sub>t</sub> = (1/4) [ 2 f<sub>AA</sub> + 2 f<sub>AB</sub> ] ''' = ''' (1/4) [ 1 + f<sub>(t-2)</sub> + 2 f<sub>(t-1)</sub> ] '''. That is the '''inbreeding coefficient ''' for <em>Full-Sib crossing</em> .<a class="footnoteRef" href="#fn107" id="fnref107"><sup>107</sup></a> <a class="footnoteRef" href="#fn108" id="fnref108"><sup>108</sup></a> The graph to the left shows the rate of this inbreeding over twenty repetitive generations. The "repetition" means that the progeny after cycle <strong>t</strong> become the crossing parents that generate cycle (<strong>t+1</strong> ), and so on successively. The graphs also show the inbreeding for <em>random fertilization 2N=20</em> for comparison. Recall that this inbreeding coefficient for progeny <em>Y</em> is also the <strong><em>co-ancestry coefficient</em></strong> for its parents, and so is a measure of the <em>relatedness of the two Fill siblings</em>.</p>
<h3 id="half-sib-crossing-hs">Half-Sib crossing (HS)</h3>

<p>Derivation of the <em>Half Sib crossing</em> takes a slightly different path to that for Full sibs. In the diagram to the right, the two half-sibs at generation (t-1) have only one parent in common - parent "A" at generation (t-2). The <strong>cross-multiplier 1</strong> is utilized again, giving '''f<sub>Y</sub> = f<sub>(P1,P2)</sub> = (1/4) [ f<sub>AA</sub> + f<sub>AC</sub> + f<sub>BA</sub> + f<sub>BC</sub> ] '''. There is just one <em>coefficient of parentage</em> this time, but three <em>co-ancestry coefficients</em> at the (t-2) level (one of them - f<sub>BC</sub> - being a "dummy" and not representing an actual individual in the (t-1) generation). As before, the <em>coefficient of parentage</em> is '''(1/2)[1+f<sub>A</sub> ] ''', and the three <em>co-ancestries</em> each represent '''f<sub>(t-1)</sub> '''. Recalling that '' f<sub>A</sub> '' represents '' f<sub>(t-2)</sub> '', the final gathering and simplifying of terms gives ''' f<sub>Y</sub> = f<sub>t</sub> = (1/8) [ 1 + f<sub>(t-2)</sub> + 6 f<sub>(t-1)</sub> ] '''.<a class="footnoteRef" href="#fn109" id="fnref109"><sup>109</sup></a> <a class="footnoteRef" href="#fn110" id="fnref110"><sup>110</sup></a> The graphs at left include this '''half-sib (HS) inbreeding ''' over twenty successive generations.  As before, this also quantifies the <em>relatedness</em> of the two half-sibs at generation (t-1) in its alternative form of '''f<sub>(P1, P2)</sub> '''.</p>
<h3 id="self-fertilization-sf">Self fertilization (SF)</h3>

<p>A pedigree diagram for selfing is on the right. It is so straightforward it doesn't require any cross-multiplication rules. It employs just the basic juxtaposition of the <em>inbreeding coefficient</em> and its alternative the <em>co-ancestry coefficient</em>; followed by recognizing that, in this case, the latter is also a <em>coefficient of parentage</em>. Thus, ''' f<sub>Y</sub> = f<sub>(P1, P1)</sub> = f<sub>t</sub> = (1/2) [ 1 + f<sub>(t-1)</sub> ] ''' !! <a class="footnoteRef" href="#fn111" id="fnref111"><sup>111</sup></a> <a class="footnoteRef" href="#fn112" id="fnref112"><sup>112</sup></a> This is the fastest rate of inbreeding of all types, as can be seen in the graphs above. The selfing curve is, in fact, a graph of the '<strong>'coefficient of parentage</strong> .</p>
<h3 id="cousins-crossings">Cousins crossings</h3>
<figure><b>(Figure)</b>
<figcaption>Pedigree analysis First cousins</figcaption>
</figure>

<p>These are derived with methods similar to those for siblings.<a class="footnoteRef" href="#fn113" id="fnref113"><sup>113</sup></a> <a class="footnoteRef" href="#fn114" id="fnref114"><sup>114</sup></a> As before, the <strong>co-ancestry</strong> viewpoint of the <em>inbreeding coefficient</em> provides a measure of "relatedness" between the parents <strong>P1</strong> and <strong>P2</strong> in these cousin expressions.</p>

<p>The pedigree for <strong>First Cousins (FC)</strong> is given to the right. The prime equation is <em>' f<sub>Y</sub> = f<sub>t</sub> = f<sub>P1,P2</sub> = (1/4) [ f<sub>1D</sub> + f<sub>12</sub> + f<sub>CD</sub> + f<sub>C2</sub> ]</em>'. After substitution with corresponding inbreeding coefficients, gathering of terms and simplifying, this becomes ''' f<sub>t</sub> = (1/4) [ 3 f<sub>(t-1)</sub> + (1/4) [2 f<sub>(t-2)</sub> + f<sub>(t-3)</sub> + 1 ]] ''', which is a version for iteration - useful for observing the general pattern, and for computer programming. A "final" version is ''' f<sub>t</sub> = (1/16) [ 12 f<sub>(t-1)</sub> + 2 f<sub>(t-2)</sub> + f<sub>(t-3)</sub> + 1 ] '''. </p>

<p>The <strong>Second Cousins (SC)</strong> pedigree is on the left. Parents in the pedigree which are <strong><em>not related</em></strong> to the <em>Common Ancestor</em> are indicated by numerals instead of letters. Here, the prime equation is <em>' f<sub>Y</sub> = f<sub>t</sub> = f<sub>P1,P2</sub> = (1/4) [ f<sub>3F</sub> + f<sub>34</sub> + f<sub>EF</sub> + f<sub>E4</sub> ]</em>'. After working through the appropriate algebra, this becomes ''' f<sub>t</sub> = (1/4) [ 3 f<sub>(t-1)</sub> + (1/4) [3 f<sub>(t-2)</sub> + (1/4) [2 f<sub>(t-3)</sub> + f<sub>(t-4)</sub> + 1 ]]] ''', which is the iteration version. A "final" version is ''' f<sub>t</sub> = (1/64) [ 48 f<sub>(t-1)</sub> + 12 f<sub>(t-2)</sub> + 2 f<sub>(t-3)</sub> + f<sub>(t-4)</sub> + 1 ] '''. </p>

<p>In order to visualize the <strong>pattern in Full Cousin</strong> equations, start the series with the <strong>full sib</strong> equation re-written in iteration form:- <em>' f<sub>t</sub> = (1/4)[2 f<sub>(t-1)</sub> + f<sub>(t-2)</sub> + 1 ]</em>'. Notice that this is the "essential plan" of the last term in each of the cousin iterative forms: with the small difference that the generation indices increment by "1" at each cousin "level". Now, define the <em>cousin level</em> as <strong>k = 1</strong> (for First cousins), <strong>= 2</strong> (for Second cousins), <strong>= 3</strong> (for Third cousins), etc., etc.; and <strong>= 0</strong> (for Full Sibs, which are "zero level cousins"). The <strong><em>last term</em></strong> can be written now as:- ''' (1/4) [ 2 f<sub>(t-(1+k))</sub> + f<sub>(t-(2+k))</sub> + 1] '''. Stacked in front of this <em>last term</em> are one or more <strong>iteration increments</strong> of the form '''(1/4) [ 3 f<sub>(t-j)</sub> + ... ''', where <strong>j</strong> is the <strong>iteration index</strong> and takes values from <strong>1 ... k</strong> over the successive iterations as needed. Putting all this together provides a general formula for all levels of <strong>Full cousin</strong> possible, including <em>Full Sibs</em>. For <strong>k</strong>th <strong>level</strong> Full cousins, '''f{k}<sub>t</sub> = <em>Ιter</em><sub>j = 1</sub><sup>k</sup> { (1/4) [ 3 f<sub>(t-j)</sub> + }<sub>j</sub> + (1/4) [ 2 f<sub>(t-(1+k))</sub> + f<sub>(t-(2+k))</sub> + 1] '''. At the commencement of iteration, all f<sub>(t-<em>x</em>)</sub> are set at "0", and each has its value substituted as it is calculated through the generations. The graphs to the right show the successive inbreeding for several levels of Full Cousins. </p>

<p>For <strong>First Half-cousins (FHC)</strong>, the pedigree is to the left. Notice that there is just <em>one</em> Common Ancestor (individual <strong>A</strong>). Also, as for <em>Second cousins</em>, parents not related to the Common Ancestor are indicated by numerals. Here, the prime equation is <em>' f<sub>Y</sub> = f<sub>t</sub> = f<sub>P1,P2</sub> = (1/4) [ f<sub>3D</sub> + f<sub>34</sub> + f<sub>CD</sub> + f<sub>C4</sub> ]</em>'. After working through the appropriate algebra, this becomes ''' f<sub>t</sub> = (1/4) [ 3 f<sub>(t-1)</sub> + (1/8) [6 f<sub>(t-2)</sub> + f<sub>(t-3)</sub> + 1 ]] ''', which is the iteration version. A "final" version is ''' f<sub>t</sub> = (1/32) [ 24 f<sub>(t-1)</sub> + 6 f<sub>(t-2)</sub> + f<sub>(t-3)</sub> + 1 ] '''. The <strong>iteration algorithm</strong> is similar to that of <em>Full Cousins</em>, except that the <strong>last term</strong> is ''' (1/8) [ 6 f<sub>(t-(1+k))</sub> + f<sub>(t-(2+k))</sub> + 1 ] '''. Notice that this <em>last term</em> is basically similar to the Half Sib equation, in parallel to the pattern found for Full Cousins and Full Sibs. In other words, Half Sibs are "zero level" Half Cousins.</p>

<p>There is a tendency to regard cousin crossing with a human-oriented point of view, possibly because of a wide interest in Genealogy. The use of pedigrees to derive the inbreeding perhaps reinforces this "Family History" view. However, such kinds of inter-crossing occur also in natural populations, especially those which are sedentary, or have a "breeding area" which is re-visited from season to season. The progeny-group of a harem with a dominant male, for example, may contain elements of sib-crossing, cousin crossing, and backcrossing, as well as genetic drift, especially of the "island" type. In addition to all of that, the occasional "outcross" will add an element of hybridization to the mix. It certainly is <em>not</em> panmixia !</p>
<h3 id="backcrossing-bc">Backcrossing (BC)</h3>

<p> </p>

<p>Following the hybridizing between <strong>A</strong> and <strong>R</strong>, the <strong>F1</strong> (individual <strong>B</strong>) is crossed back (<strong>''BC1</strong>'') to an original parent (<strong>R</strong>) to produce the <strong>BC1</strong> generation (individual <strong>C</strong>). [It is usual to use the same label for the act of <em>making</em> the back-cross <em>and</em> for the generation produced by it. The act of back-crossing is here in <em>italics</em>. ] Parent <strong>R</strong> is the <strong><em>Recurrent</em> parent</strong>. Two successive backcrosses are depicted, with individual <strong>D</strong> being the <strong>BC2</strong> generation. These generations have been given <em>t</em> indices also, as indicated. As before, '''f<sub>D</sub> = f<sub>t</sub> = f<sub>CR</sub> = (1/2) [ f<sub>RB</sub> + f<sub>RR</sub> ] ''', using <strong>cross-multiplier 2</strong> previously given. The <em>f<sub>RB</sub></em> just defined is the one that involves generation <em>(t-1)</em> with <em>(t-2)</em>. However, there is another such <em>f<sub>RB</sub></em> contained wholly <em>within</em> generation <em>(t-2)</em> as well, and it is <em>this</em> one that is used now:- as the <em>co-ancestry</em> of the <em>parents</em> of individual <strong>C</strong> in generation <em>(t-1)</em>. As such, it is also the <em>inbreeding coefficient</em> of <strong>C</strong>, and hence is <strong>f<sub>(t-1)</sub></strong>. The remaining <strong>f<sub>RR</sub></strong> is the <em>coefficient of parentage</em> of the <em>recurrent parent</em>, and so is ''(1/2) [1 + f<sub>R</sub> ] ''. Putting all this together :- '''f<sub>t</sub> = (1/2) [ (1/2) [ 1 + f<sub>R</sub> ] + f<sub>(t-1)</sub> ] ''' = ''' (1/4) [ 1 + f<sub>R</sub> + 2 f<sub>(t-1)</sub> ] '''. The graphs at right illustrate Backcross inbreeding over twenty backcrosses for three different levels of (fixed) inbreeding in the Recurrent parent.</p>

<p>This routine is commonly used in Animal and Plant Breeding programmes. Often after making the hybrid (especially if individuals are short-lived), the recurrent parent needs separate "line breeding" for its maintenance as a future recurrent parent in the backcrossing. This maintenance may be through selfing, or through full-sib or half-sib crossing, or through restricted randomly-fertilized populations, depending on the species' reproductive possibilities. Of course, this incremental rise in <strong>f<sub>R</sub></strong> carries-over into the <strong>f<sub>t</sub></strong> of the backcrossing. The result is a more gradual curve rising to the asymptotes than shown in the present graphs, because the <em>f<sub>R</sub></em> is not at a fixed level from the outset.</p>
<h3 id="relatedness-between-relatives">Relatedness between relatives</h3>

<p>Central in estimating the variances for the various components is the principle of relatedness. A child has a father and a mother. Consequently, the child and father share 50% of their alleles, as do the child and the mother. However, the mother and father normally do not share alleles as a result of shared ancestors. Similarly, two full siblings share also on average 50% of the alleles with each other, while half siblings share only 25% of their alleles. This variation in relatedness can be used to estimate which proportion of the total phenotypic variance (σ²<sub>P</sub>) is explained by the above-mentioned components.</p>

<p>The principle of relationship (R) is central to understanding the resemblances within families and can be useful when calculating inbreeding. Relationship has two definitions that can be applied: -The probable portion of genes that are the same for two individuals due to common ancestry exceeding that of the base population -Additive/numerator relationship: the relationship coefficient (Rxy¬) = twice the probability of two genes at loci in different individuals being identical by descent. Rxy values can range from 0 to 1. Relationship can be calculated in several ways; from the known relationships of the individual, from bracket pedigrees, and from pedigree path diagrams.</p>
<h4 id="calculating-relationship-from-known-relationships">Calculating relationship from known relationships</h4>
<table>
<thead>
<tr class="header">
<th style="text-align: left;">
<p>Relationship</p></th>
<th style="text-align: left;">
<p>Relationship Coefficient</p></th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;">
<p>Individual and itself</p></td>
<td style="text-align: left;">
<p>1.00</p></td>
</tr>
<tr class="even">
<td style="text-align: left;">
<p>Individual and a monozygotic twin</p></td>
<td style="text-align: left;">
<p>1.00</p></td>
</tr>
<tr class="odd">
<td style="text-align: left;">
<p>Individual and parent</p></td>
<td style="text-align: left;">
<p>0.50</p></td>
</tr>
<tr class="even">
<td style="text-align: left;">
<p>Full siblings</p></td>
<td style="text-align: left;">
<p>0.50</p></td>
</tr>
<tr class="odd">
<td style="text-align: left;">
<p>Half siblings</p></td>
<td style="text-align: left;">
<p>0.25</p></td>
</tr>
<tr class="even">
<td style="text-align: left;">
<p>Individual and grandparent</p></td>
<td style="text-align: left;">
<p>0.25</p></td>
</tr>
<tr class="odd">
<td style="text-align: left;">
<p>Son of sire and daughter of sire</p></td>
<td style="text-align: left;">
<p>0.125</p></td>
</tr>
<tr class="even">
<td style="text-align: left;">
<p>Grandson and granddaughter of sire</p></td>
<td style="text-align: left;">
<p>0.0625</p></td>
</tr>
</tbody>
</table>
<ul>
<li>Note: if the common ancestor is inbred, multiply the relationship by (1+inbreeding coefficient)</li>
</ul>
<h4 id="calculating-relationship-from-pathway-diagrams">Calculating relationship from pathway diagrams</h4>

<p>RXY = Σ(.5)n(1+FCA)</p>

<p>n = number of segregations between X and Y through their common ancestor FCA = the inbreeding coefficient of the common ancestor</p>

<p>Example: calculating <strong>R</strong>AE and <strong>R</strong>BE Note: valid pathways only go through ancestors (only go against the direction of the arrow). For example, to calculate the relationship of A and B, the pathway A-D-B would be acceptable, whereas the pathway A-X-B would be not. The reason behind this is that having progeny together does not make two individuals related.</p>

<p>RAB: there are two possible pathways from A to E. A-D-F-E = (1/2)3 = .125 A-D-E = (1/2)2 = .25 Total: .375</p>

<p>RBE: there are four possible pathways from B to E. B-D-E = (1/2)2 = .25 B-D-F-E = (1/2)3 = .125 B-C-D-E = (1/2)3 = .125 B-C-D-F-E = (1/2)4 = .0625 Total: .5625</p>

<p>The square root of h^2 equals the correlation between additive genotype and expressed phenotype, as shown through the general procedures of Path Analysis.<a class="footnoteRef" href="#fn115" id="fnref115"><sup>115</sup></a></p>
<h2 id="resemblances-amongst-relatives">Resemblances amongst relatives</h2>

<p>These, in like manner to the Genotypic variances, can be derived through either the gene-model ("Mather") approach or the allele-substitution ("Fisher") approach. Here, each method will be demonstrated for alternate cases.</p>
<h3 id="parent-offspring-covariance">Parent-offspring covariance</h3>

<p>These can be viewed either as the covariance between any offspring and <em>any one</em> of its parents (<strong>PO</strong>), or as the covariance between any offspring and the '' "mid-parent" '' value of both its parents (<strong>MPO</strong>).</p>
<h4 id="one-parent-and-offspring-po">One-parent and offspring (PO)</h4>

<p>This can be derived as the <em>sum of cross-products</em> between parent gene-effects and <em>one-half</em> of the progeny expectations using the allele-substitution approach. The <em>one-half</em> of the progeny expectation accounts for the fact that <em>only one of the two parents</em> is being considered. The appropriate parental gene-effects are therefore the second-stage redefined gene effects used to define the Genotypic variances earlier, that is:- <strong>a′′ = 2q(α - qd)</strong> and <strong>d′′ = (q-p)α + 2pqd</strong> and also <strong>(-a)′′ = -2p(α + pd)</strong> [see section "Gene effects redefined"]. Similarly, the appropriate progeny effects, <strong><em>for allele-substitution expectations</em></strong> are one-half of the earlier <em>breeding values</em>, the latter being:- <strong>α<sub>AA</sub> = 2qα</strong>, and <strong>α<sub>Aa</sub> = (q-p)α</strong> and also <strong>α<sub>aa</sub> = -2pα</strong> [see section on "Genotype substitution - Expectations and Deviations"].</p>

<p>Because all of these effects are defined already as deviates from the <strong>Genotypic mean</strong>, the cross-product sum using {<strong>genotype-frequency * parental gene-effect * half-breeding-value</strong>} will provide immediately the <strong>allele-substitution-expectation covariance</strong> between any one parent and its offspring. After careful gathering of terms and simplification, this becomes '''cov(PO)<sub>A</sub> = pqα<sup>2</sup> = ½ σ<sup>2</sup><sub>A</sub> '''.<a class="footnoteRef" href="#fn116" id="fnref116"><sup>116</sup></a> <a class="footnoteRef" href="#fn117" id="fnref117"><sup>117</sup></a> </p>

<p>Unfortunately, the <em>allele-substitution-deviations</em> are usually overlooked, but they have not "ceased to exist" nonetheless! Recall that these deviations are:- <strong>δ<sub>AA</sub> = -2q<sup>2</sup> d</strong>, and <strong>δ<sub>Aa</sub> = 2pq d</strong> and also <strong>δ<sub>aa</sub> = -2p<sup>2</sup> d</strong> [see section on "Genotype substitution - Expectations and Deviations"]. Consequently, the cross-product sum using {<strong>genotype-frequency * parental gene-effect * half-substitution-deviations</strong>} also will provide immediately the <strong>allele-substitution-deviations covariance</strong> between any one parent and its offspring. Once more, after careful gathering of terms and simplification, this becomes '''cov(PO)<sub>D</sub> = 2p<sup>2</sup>q<sup>2</sup>d<sup>2</sup> = ½ σ<sup>2</sup><sub>D</sub> '''.</p>

<p>It follows therefore that:- '''cov(PO) = cov(PO)<sub>A</sub> + cov(PO)<sub>D</sub> = ½ σ<sup>2</sup><sub>A</sub> + ½ σ<sup>2</sup><sub>D</sub> ''', when dominance is <em>not</em> overlooked !</p>
<h4 id="mid-parent-and-offspring-mpo">Mid-parent and offspring (MPO)</h4>

<p>Because there are many combinations of parental genotypes, there are many different mid-parents and offspring means to consider, together with the varying frequencies of obtaining each parental pairing. The gene-model approach is the most expedient in this case. Therefore, an <strong><em>unadjusted sum of cross-products</em> (USCP)</strong> - using all products {''' parent-pair-frequency * mid-parent-gene-effect * offspring-genotype-mean '''} - is adjusted by subtracting the '''{overall genotypic mean}<sup>2</sup> ''' as <strong><em>correction factor</em> (CF)</strong>. After multiplying out all the various combinations, carefully gathering terms, simplifying, factoring and cancelling-out where applicable, this becomes:-</p>

<p>'''cov(MPO) = pq [a + (q-p)d ]<sup>2</sup> = pq α<sup>2</sup> = ½ σ<sup>2</sup><sub>A</sub> ''', with no dominance having been overlooked in this case, as it had been "used-up" in defining the <strong>α</strong>.<a class="footnoteRef" href="#fn118" id="fnref118"><sup>118</sup></a> <a class="footnoteRef" href="#fn119" id="fnref119"><sup>119</sup></a> </p>
<h4 id="applications-parent-offspring">Applications (Parent-Offspring)</h4>

<p>The most obvious application is an <strong><em>experiment containing all parents and their offspring</em></strong>, with or without reciprocal crosses, preferably replicated without bias, enabling estimation of all appropriate means, variances and covariances, together with their standard errors. These estimated statistics can then be used to estimate the genetical variances. Twice <em>the difference between the estimates of the two forms of (corrected) parent-offspring covariance</em> provides an estimate of <strong>σ<sup>2</sup><sub>D</sub></strong>; and twice the <em>cov(MPO)</em> estimates <strong>σ<sup>2</sup><sub>A</sub></strong>. With appropriate experimental design and analysis,<a class="footnoteRef" href="#fn120" id="fnref120"><sup>120</sup></a><a class="footnoteRef" href="#fn121" id="fnref121"><sup>121</sup></a><a class="footnoteRef" href="#fn122" id="fnref122"><sup>122</sup></a> standard errors can be obtained for these genetical statistics as well. This is the basic core of an experiment known as <strong>Diallel analysis</strong>, the Mather, Jinks and Hayman version of which will be discussed in another section.</p>

<p>A second application involves the use of <strong><em>regression analysis</em></strong>, which estimates from statistics the ordinate (Y-estimate), derivative (regression coefficient) and constant (Y-intercept) of calculus.<a class="footnoteRef" href="#fn123" id="fnref123"><sup>123</sup></a><a class="footnoteRef" href="#fn124" id="fnref124"><sup>124</sup></a><a class="footnoteRef" href="#fn125" id="fnref125"><sup>125</sup></a><a class="footnoteRef" href="#fn126" id="fnref126"><sup>126</sup></a> The <em>regression coefficient</em> estimates the <em>rate of change</em> of the function predicting <strong>Y</strong> from <strong>X</strong>, based on minimizing the residuals between the fitted curve and the observed data (MINRES). No alternative method of estimating such a function satisfies this basic requirement of MINRES. In general, the regression coefficient is estimated as the <strong><em>ratio of the covariance(XY) to the variance of the determinator (X)</em></strong>. In practice, the sample size is usually the same for both X and Y, so that this can be written as <strong>SCP(XY) / SS(X)</strong>, where all terms have been defined previously.<a class="footnoteRef" href="#fn127" id="fnref127"><sup>127</sup></a><a class="footnoteRef" href="#fn128" id="fnref128"><sup>128</sup></a><a class="footnoteRef" href="#fn129" id="fnref129"><sup>129</sup></a> In the present context, the parents are viewed as the "determinative variable" (X), and the offspring as the "determined variable" (Y), and the regression coefficient as the "functional relationship" (β<sub>PO</sub>) between the two. Taking '''cov(MPO) = ½ σ<sup>2</sup><sub>A</sub> ''' as <strong>cov(XY)</strong>, and ''' σ<sup>2</sup><sub>P</sub> / 2 ''' (the variance of the mean of two parents - the mid-parent) as <strong>σ<sup>2</sup><sub>X</sub></strong>, it can be seen that '''β<sub>MPO</sub> = [½ σ<sup>2</sup><sub>A</sub>] / [½ σ<sup>2</sup><sub>P</sub>] = h<sup>2</sup> '''.<a class="footnoteRef" href="#fn130" id="fnref130"><sup>130</sup></a> Next, utilizing <strong>cov(PO) = [ ½ σ<sup>2</sup><sub>A</sub> + ½ σ<sup>2</sup><sub>D</sub> ]</strong> as <strong>cov(XY)</strong>, and <em>' σ<sup>2</sup><sub>P</sub></em>' as <strong>σ<sup>2</sup><sub>X</sub></strong>, it is seen that ''' 2 β<sub>PO</sub> = [ 2 (½ σ<sup>2</sup><sub>A</sub> + ½ σ<sup>2</sup><sub>D</sub> )] / σ<sup>2</sup><sub>P</sub> = H<sup>2</sup> '''.</p>

<p>Analysis of <strong><em>epistasis</em></strong> has previously been attempted via an <em>interaction variance</em> approach of the type '' σ<sup>2</sup><sub>AA</sub> '', and '' σ<sup>2</sup><sub>AD</sub>'' and also '' σ<sup>2</sup><sub>DD</sub>''. This has been integrated with these present covariances in an effort to provide estimators for the epistasis variances. However, the findings of epigenetics suggest that this may not be an appropriate way to define epistasis.</p>
<h3 id="siblings-covariances">Siblings covariances</h3>

<p>Covariance between half-sibs (<strong>HS</strong>) is defined easily using allele-substitution methods; but, once again, the dominance contribution has historically been omitted. However, as with the mid-parent/offspring covariance, the covariance between full-sibs (<strong>FS</strong>) requires a "parent-combination" approach, thereby necessitating the use of the gene-model corrected-cross-product method; and the dominance contribution has not historically been overlooked. The superiority of the gene-model derivations is as evident here as it was for the Genotypic variances.</p>
<h4 id="half-sibs-of-the-same-common-parent-hs">Half-sibs of the same common-parent (HS)</h4>

<p>The sum of the cross-products <strong>{ common-parent frequency * half-breeding-value of one half-sib * half-breeding-value of any other half-sib in that same common-parent-group }</strong> immediately provides one of the required covariances, because the effects used [<em>breeding values</em> - representing the allele-substitution expectations] are already defined as deviates from the genotypic mean [see section on "Allele substitution - Expectations and deviations"]. After simplification. this becomes:- ''' cov(HS)<sub>A</sub> = ½ pq α<sup>2</sup> = ¼ σ<sup>2</sup><sub>A</sub> '''.<a class="footnoteRef" href="#fn131" id="fnref131"><sup>131</sup></a> <a class="footnoteRef" href="#fn132" id="fnref132"><sup>132</sup></a>  However, the <em>substitution deviations</em> also exist, defining the sum of the cross-products <strong>{ common-parent frequency * half-substitution-deviation of one half-sib * half-substitution-deviation of any other half-sib in that same common-parent-group }</strong>, which ultimately leads to:- ''' cov(HS)<sub>D</sub> = p<sup>2</sup> q<sup>2</sup> d<sup>2</sup> = ¼ σ<sup>2</sup><sub>D</sub> '''. Adding the two components gives:-</p>

<p>'''cov(HS) = cov(HS)<sub>A</sub> + cov(HS)<sub>D</sub> = ¼ σ<sup>2</sup><sub>A</sub> + ¼ σ<sup>2</sup><sub>D</sub> '''.</p>
<h4 id="full-sibs-fs">Full-sibs (FS)</h4>

<p>As explained in the introduction, a method similar to that used for mid-parent/progeny covariance is used. Therefore, an <strong><em>unadjusted sum of cross-products</em> (USCP)</strong> - using all products {''' parent-pair-frequency * the square of the offspring-genotype-mean '''} - is adjusted by subtracting the '''{overall genotypic mean}<sup>2</sup> ''' as <strong><em>correction factor</em> (CF)</strong>. In this case, the multiplying out of all the various combinations, carefully gathering terms, simplifying, factoring and cancelling-out is very protracted. It eventually becomes:-</p>

<p>'''cov(FS) = pq α<sup>2</sup> + p<sup>2</sup> q<sup>2</sup> d<sup>2</sup> = ½ σ<sup>2</sup><sub>A</sub> + ¼ σ<sup>2</sup><sub>D</sub> ''', with no dominance having been overlooked.<a class="footnoteRef" href="#fn133" id="fnref133"><sup>133</sup></a> <a class="footnoteRef" href="#fn134" id="fnref134"><sup>134</sup></a> </p>
<h4 id="applications-siblings">Applications (Siblings)</h4>

<p>The most useful application here for genetical statistics is the <em>correlation between half-sibs</em>. Recall that the correlation coefficient (<em>r</em>) is the ratio of the covariance to the variance [see section on "Associated attributes" for example]. Therefore, ''' r<sub>HS</sub> = cov(HS) / σ<sup>2</sup><sub>all HS together</sub> ''' = ''' [¼ σ<sup>2</sup><sub>A</sub> + ¼ σ<sup>2</sup><sub>D</sub> ] / σ<sup>2</sup><sub>P</sub> = ¼ H<sup>2</sup> '''.<a class="footnoteRef" href="#fn135" id="fnref135"><sup>135</sup></a> The correlation between full-sibs is of little utility, being ''' r<sub>FS</sub> = cov(FS) / σ<sup>2</sup><sub>all FS together</sub> ''' = ''' [½ σ<sup>2</sup><sub>A</sub> + ¼ σ<sup>2</sup><sub>D</sub> ] / σ<sup>2</sup><sub>P</sub> <em>'. The suggestion that it "approximates" (</em>½ h<sup>2</sup>'') is poor advice.</p>

<p>Of course, the correlations between siblings are of intrinsic interest in their own right, quite apart from any utility they may have for estimating heritabilities or genotypic variances.</p>

<p>It may be worth noting that '''[ cov(FS) - cov(HS)] = ¼ σ<sup>2</sup><sub>A</sub> '''. Experiments consisting of FS and HS families could utilize this by using intra-class correlation to equate experiment variance components to these covariances [see section on "Coefficient of relationship as an intra-class correlation" for the rationale behind this].</p>

<p>The earlier comments regarding epistasis apply again here [see section on "Applications (Parent-offspring"].</p>
<h2 id="selection">Selection</h2>
<h3 id="basic-principles-1">Basic principles</h3>
<figure><b>(Figure)</b>
<figcaption>Genetic advance and Selection pressure repeated.</figcaption>
</figure>

<p>Selection operates on the attribute (phenotype), such that individuals which equal or exceed a <strong><em>selection threshold</em> (z<sub>P</sub>)</strong> become effective parents to create the next generation. The <strong><em>proportion</em></strong> they represent of the base population is the <strong><em>selection pressure</em> (Prob.)</strong>. The <em>smaller</em> the proportion, the <em>stronger</em> is the pressure! The <strong><em>mean of the selected group</em> (P<sub>s</sub>)</strong> is superior to the <strong><em>base-population mean</em> (P<sub>0</sub>)</strong> by the difference called the <strong><em>selection differential</em> (S)</strong>. All of these quantities are phenotypic. In order to "link" to the underlying genes, a <strong><em>heritability</em> (h<sup>2</sup>)</strong> is used, fulfilling the role of a <em>coefficient of determination</em> in the Biometrical sense. The <em>expected genetical change</em>, still expressed in <em>phenotypic units of measurement</em>, is called the <strong><em>genetic advance</em> (ΔG)</strong>, and is obtained by the product of the <em>selection differential (S)</em> and its <em>coefficient of determination (h<sup>2</sup>)</em>. The <em>expected</em> <strong><em>mean of the progeny</em> (P<sub>1</sub>)</strong> can be found by adding the <em>genetic advance (ΔG)</em> to the <em>base mean (P<sub>0</sub>)</em>. The graphs to the right show how the (initial) <strong>genetic advance</strong> is greater with stronger selection pressure (smaller <strong>Probability</strong>). They also show how progress from successive cycles of selection (even at the same selection pressure) steadily declines, because the Phenotypic variance and the Heritability are being diminished by the selection itself. This is discussed further shortly.</p>

<p>Thus <strong>ΔG = S h<sup>2</sup></strong>.<a class="footnoteRef" href="#fn136" id="fnref136"><sup>136</sup></a> </p>

<p>and '''P<sub>1</sub> = P<sub>0</sub> + ΔG .<a class="footnoteRef" href="#fn137" id="fnref137"><sup>137</sup></a> </p>

<p>The <strong>narrow-sense heritability (h<sup>2</sup>)</strong> is usually used, thereby linking to the '''additive genetic variance (σ<sup>2</sup><sub>A</sub>) '''. However, if appropriate, use of the <strong>broad-sense heritability (H<sup>2</sup>)</strong> would connect to the <strong>genotypic variance (σ<sup>2</sup><sub>G</sub>)</strong> ; and even possibly an <strong>allelic heritability [ ħ<sup>2</sup> = (σ<sup>2</sup><sub>a</sub>) / (σ<sup>2</sup><sub>P</sub>) ]</strong> might be contemplated, connecting to ( <strong>σ<sup>2</sup><sub>a</sub></strong> ).</p>

<p>In order to utilize these concepts <em>before</em> selection actually takes place, and so predict the outcome of alternatives (such as choice of <em>selection threshold</em>, for example), these phenotypic statistics are re-considered against the properties of the Normal Distribution, especially those concerning truncation of the <em>superior tail</em> of the Distribution. In such consideration, the '<strong><em>standardized</em> selection differential (i)</strong> and the <strong><em>standardized</em> selection threshold (z)</strong> are used instead of the previous "phenotypic" versions. The <strong><em>phenotypic standard deviate</em> (σ<sub>P(0)</sub>)</strong> also is needed. All of this will be elaborated in a subsequent section.</p>

<p>Therefore, <strong>ΔG = (i σ<sub>P</sub>) h<sup>2</sup></strong>, .....where <em>(i σ<sub>P(0)</sub>)</em> = <em>S</em> previously.<a class="footnoteRef" href="#fn138" id="fnref138"><sup>138</sup></a> </p>
<figure><b>(Figure)</b>
<figcaption>Changes arising from repeated selection</figcaption>
</figure>

<p>It was noted above that successive <strong>ΔG</strong> will decline because the "input" [the <strong>phenotypic variance ( σ<sup>2</sup><sub>P</sub> )</strong>] will be reduced by the previous selection.<a class="footnoteRef" href="#fn139" id="fnref139"><sup>139</sup></a> The heritability also will be reduced. The graphs to the left show these declines over ten cycles of repeated selection during which the same selection pressure is asserted. The accumulated genetic advance (<strong>ΣΔG</strong>) has virtually reached its asymptote by generation 6 in this example. This reduction depends partly upon truncation properties of the Normal Distribution, and partly upon the heritability together with <strong>meiosis determination ( b<sup>2</sup> )</strong>. The last two items quantify the extent to which the <em>truncation</em> is "offset" by new variation arising from segregation and assortment during meiosis.<a class="footnoteRef" href="#fn140" id="fnref140"><sup>140</sup></a> <a class="footnoteRef" href="#fn141" id="fnref141"><sup>141</sup></a> This will be discussed soon, but here note the simplified result for <strong><em>undispersed random fertilization</em> (f = 0)</strong>.</p>

<p>Thus :- <strong>σ<sup>2</sup><sub>P(1)</sub> = σ<sup>2</sup><sub>P(0)</sub> [1 - i ( i-z) ½ h<sup>2</sup>]</strong> , where <strong>i ( i-z) = K = truncation coefficient</strong>; and <strong>½ h<sup>2</sup> = R = reproduction coefficient</strong> <a class="footnoteRef" href="#fn142" id="fnref142"><sup>142</sup></a> <a class="footnoteRef" href="#fn143" id="fnref143"><sup>143</sup></a> This can be written also as <strong>σ<sup>2</sup><sub>P(1)</sub> = σ<sup>2</sup><sub>P(0)</sub> [1 - K R ]</strong>, which facilitates more detailed analysis of selection problems.</p>

<p>Here, <strong>i</strong> and <strong>z</strong> have already been defined, <strong>½</strong> is the <em>meiosis determination (<strong>b<sup>2</sup></strong>) for <strong>f=0</strong></em>, and the remaining symbol is the heritability. These all will be discussed further in following sections. Also notice that, more generally, <strong>R = b<sup>2</sup> h<sup>2</sup></strong>. If the general <strong>meiosis determination ( b<sup>2</sup> )</strong> is used, the results of prior inbreeding can be incorporated into the selection. The phenotypic variance equation then becomes:-</p>

<p><strong>σ<sup>2</sup><sub>P(1)</sub> = σ<sup>2</sup><sub>P(0)</sub> [1 - i ( i-z) b<sup>2</sup> h<sup>2</sup>]</strong>.</p>

<p>The <em>Phenotypic variance</em> truncated by the <em><strong>selected group</strong></em> (<em>' σ<sup>2</sup><sub>P(S)</sub></em>' ) is simply <strong>σ<sup>2</sup><sub>P(0)</sub> [1 - K]</strong>, and its contained <em>genic variance</em> is <strong>(h<sup>2</sup><sub>0</sub> σ<sup>2</sup><sub>P(S)</sub></strong> ). Assuming that selection has not altered the <em>environmental</em> variance, the <em>genic variance</em> for the progeny can be approximated by ''' σ<sup>2</sup><sub>A(1)</sub> = ( σ<sup>2</sup><sub>P(1)</sub> - σ<sup>2</sup><sub>E</sub>) ''' . From this, <strong>h<sup>2</sup><sub>1</sub> = ( σ<sup>2</sup><sub>A(1)</sub> / σ<sup>2</sup><sub>P(1)</sub> )</strong>. Similar estimates could be made for <strong>σ<sup>2</sup><sub>G(1)</sub></strong> and <strong>H<sup>2</sup><sub>1</sub></strong> , or for <strong>σ<sup>2</sup><sub>a(1)</sub></strong> and <strong>ħ<sup>2</sup><sub>1</sub></strong> if required.</p>
<h4 id="alternative-δg">Alternative ΔG</h4>

<p>A rearrangement follows which is useful for considering selection on multiple attributes (characters). It starts by expanding the heritability into its variance components. '''ΔG = i σ<sub>P</sub> ( σ<sup>2</sup><sub>A</sub> / σ<sup>2</sup><sub>P</sub> ) '''. The <em>σ<sub>P</sub></em> and <em>σ<sup>2</sup><sub>P</sub></em> partially cancel, leaving a solo <em>σ<sub>P</sub></em>. Next, the <em>σ<sup>2</sup><sub>A</sub></em> inside the heritability can be expanded as (<em>σ<sub>A</sub> * σ<sub>A</sub></em>), which leads to :  <strong>ΔG = i σ<sub>A</sub> ( σ<sub>A</sub> / σ<sub>P</sub> )</strong> = '''i σ<sub>A</sub> h '''. Corresponding re-arrangements could be made using the alternative heritabilities, giving <strong>ΔG = i σ<sub>G</sub> H</strong> or <strong>ΔG = i σ<sub>a</sub> ħ</strong>.</p>
<h3 id="background">Background</h3>
<h4 id="standardized-selection---the-normal-distribution">Standardized selection - the Normal distribution</h4>

<p>The entire <em>base population</em> is outlined by the Normal curve <a class="footnoteRef" href="#fn144" id="fnref144"><sup>144</sup></a> to the right. Along the <strong>Z axis</strong> is every value of the attribute from least to greatest, and the height from this axis to the curve itself is the frequency of the value at the axis below. The equation for finding these frequencies for the "Normal" curve (the curve of "common experience") is given in the ellipse: notice it includes the mean (<strong>μ</strong>) and the variance (<strong>σ<sup>2</sup></strong>). Moving infinitesimally along the z-axis, the frequencies of neighbouring values can be "stacked" beside the previous, thereby accumulating an "area representing the <strong>probability</strong> " of obtaining all the values within the "stack". [That's <strong>integration</strong> from calculus!] Selection is focused on such a Probability area, being the shaded-in one from the <strong>selection threshold (z)</strong> to the end of the superior tail of the curve. This " <strong>Prob.</strong> " is the <strong>selection pressure</strong>. The selected group (the effective parents of the next generation) include all phenotype values from <strong>z</strong> to the "end" of the "tail".<a class="footnoteRef" href="#fn145" id="fnref145"><sup>145</sup></a> The mean of the <strong><em>selected group</em></strong> is <strong>μ<sub>s</sub></strong>, and the difference between it and the base mean (<strong>μ</strong>) represents the <strong>selection differential (S)</strong>. By taking partial integrations over curve-sections of interest, and some rearranging of the algebra, it can be shown that the "selection differential" is '''S = [ y (σ / Prob.)] ''', where <strong>y</strong> is the <em>frequency</em> of the value at the "selection threshold" <strong>z</strong> (the <em>ordinate</em> of <em>z</em>).<a class="footnoteRef" href="#fn146" id="fnref146"><sup>146</sup></a> Rearranging this relationship gives <strong>S / σ = y / Prob.</strong>, the left-hand side of which is, in fact, the <em>selection differential <strong>divided by the standard deviation</strong></em> - that is the <strong>standardized selection differential (i)</strong>. The right-side of the relationship provides an "estimator" for <strong>i</strong> - the ordinate of the <strong><em>selection threshold</em></strong> divided by the <strong><em>selection pressure</em></strong>. Tables of the Normal Distribution <a class="footnoteRef" href="#fn147" id="fnref147"><sup>147</sup></a>  can be used, but tabulations of <strong>i</strong> itself are available also.<a class="footnoteRef" href="#fn148" id="fnref148"><sup>148</sup></a> The latter reference also gives values of <strong>i</strong> adjusted for small populations (400 and less),<a class="footnoteRef" href="#fn149" id="fnref149"><sup>149</sup></a> where "quasi-infinity" cannot be assumed (but <em>was</em> presumed in the "Normal Distribution" outline above). The <em>standardized selection differential (<strong>i</strong>)</em> is known also as the <strong><em>intensity of selection</em></strong>.<a class="footnoteRef" href="#fn150" id="fnref150"><sup>150</sup></a></p>

<p>Finally, a cross-link with the differing terminology in the previous sub-section may be useful:- <strong>μ</strong> (here) = "P<sub>0</sub>" (there), <strong>μ<sub>S</sub></strong> = "P<sub>S</sub>" and <strong>σ<sup>2</sup></strong> = "σ<sup>2</sup><sub>P</sub>".</p>
<h4 id="meiosis-determination---reproductive-path-analysis">Meiosis determination - Reproductive path analysis</h4>

<p> </p>

<p>The <strong>meiosis determination (b<sup>2</sup>)</strong> is the <em>coefficient of determination</em> of meiosis to the process whereby parents generate gametes. Following the principles of <em>standardized partial regression</em>, of which <strong>path analysis</strong> is a pictorially-oriented version, Sewall Wright analyzed the paths of gene-flow during sexual reproduction, and established the "strengths of contribution" (<strong><em>coefficients of determination</em></strong>) of various components to the overall result.<a class="footnoteRef" href="#fn151" id="fnref151"><sup>151</sup></a><a class="footnoteRef" href="#fn152" id="fnref152"><sup>152</sup></a> Path analysis includes <em>partial correlations</em> as well as <em>partial regression coefficients</em> (the latter are the <strong><em>path coefficients</em></strong>). Lines with a single arrow-head are directional "determinative paths", and lines with double arrow-heads are "correlation connections". By tracing various routes according to <em>path analysis rules</em>, the algebra of standardized partial regression is emulated.<a class="footnoteRef" href="#fn153" id="fnref153"><sup>153</sup></a></p>

<p>The path diagram to the left is a representation of this analysis of sexual reproduction. It has many items of interest within it, but the one of importance in the selection context is that of meiosis: for it is there that segregation and assortment occur - the processes which partially ameliorate the truncation of the phenotypic variance arising from selection. The path coefficients <strong>b</strong> are the meiosis paths; those labeled <strong>a</strong> are the fertilization paths. The correlation between gametes from the same parent (<strong>g</strong>) is the "meiotic correlation"; that between parents within the same generation is <strong>r<sub>A</sub></strong>; and that between gametes from different parents (<strong>f</strong>) became known subsequently as the "inbreeding coefficient".<a class="footnoteRef" href="#fn154" id="fnref154"><sup>154</sup></a> The primes ( ' ) indicate generation <strong>(t-1)</strong>, and the <em>un</em>primed indicate generation <strong>t</strong>. Here, some important results of the present analysis are given. Sewall Wright interpreted many in terms of inbreeding coefficients.<a class="footnoteRef" href="#fn155" id="fnref155"><sup>155</sup></a><a class="footnoteRef" href="#fn156" id="fnref156"><sup>156</sup></a></p>

<p>The meiosis determination (<strong>b<sup>2</sup></strong>) is <em>½ (1+g)</em> and equals '''½ (1 + f<sub>(t-1)</sub>) ''', implying that <strong>g = f<sub>(t-1)</sub></strong>. [Notice that this <strong>b<sup>2</sup></strong> is the <em>coefficient of parentage (<strong>f<sub>AA</sub></strong>)</em> of <em>Pedigree analysis</em> re-written with a "generation level" instead of an "A" inside the parentheses.] With non-dispersed random fertilization, f<sub>(t-1)</sub>) = 0, giving <strong>b<sup>2</sup> = ½</strong>, as used in the selection section above. However, being aware of its background, other fertilization patterns can be utilized as required. Another determination also involves inbreeding - the fertilization determination (<strong>a<sup>2</sup></strong>) equals <strong>1 / [ 2 ( 1 + f<sub>t</sub> ) ]</strong> . Also another correlation is an inbreeding indicator - <strong>r<sub>A</sub></strong> = <strong>2 f<sub>t</sub> / ( 1 + f<sub>(t-1)</sub> )</strong>, also known as the <strong><em>coefficient of relationship</em></strong> [not to be confused with the <em>coefficient of kinship</em> which is an alternative name for the <em>co-ancestry coefficient</em> - see introduction to "Relationship" section]. This <strong>r<sub>A</sub></strong> will re-occur in the sub-section on "Dispersion and Selection".</p>

<p>These links with inbreeding reveal interesting facets about sexual reproduction which are not immediately apparent. The graphs to the right plot the <strong>meiosis</strong> and <strong>syngamy (fertilization)</strong> coefficients of determination against the inbreeding coefficient. There it is revealed that as inbreeding increases, meiosis becomes more important (the coefficient increases), while syngamy becomes less important. The overall role of reproduction [the product of the previous two coefficients - <strong>r<sup>2</sup></strong>] remains the same.<a class="footnoteRef" href="#fn157" id="fnref157"><sup>157</sup></a> This <em>increase in <strong>m<sup>2</sup></strong></em> is particularly relevant for selection because it means that the <em>selection truncation of the Phenotypic variance</em> is offset to a lesser extent during a sequence of selections when accompanied by inbreeding (which is frequently the case).</p>
<h3 id="dispersion-and-selection">Dispersion and Selection</h3>
<figure><b>(Figure)</b>
<figcaption>px250|right|Genic intra-class correlation compared with Coefficient of Relationship</figcaption>
</figure>

<p>Selection basics concern the selection for future parenthood of individuals belonging to a population, on the basis of their phenotypic values. Gamete sampling results in that population becoming dispersed into progeny lines with variable allele and genotype frequencies, and with variable means. This necessitated the partitioning of the genotypic variance into within-line and amongst-line components. It remains to superimpose this dispersion structure onto the basics of selection. Sewall Wright's analysis of reproduction (see previous sub-section) provides a convenient key to enable this. This key is the <strong>coefficient of relationship (r<sub>A</sub>)</strong>, which is '''the genic correlation amongst individuals ''within the (t-1) generation <em>'</em> - that is, it is an <strong>intra-class correlation</strong>. This is a special class of correlations which can be defined in terms of <strong><em>variance components</em></strong> - in particular, they are generally the ratio of the <strong>inter</strong>-class variance to the <strong>total</strong> of both inter- and intra-class variances.<a class="footnoteRef" href="#fn158" id="fnref158"><sup>158</sup></a> <a class="footnoteRef" href="#fn159" id="fnref159"><sup>159</sup></a> This connection is demonstrated in the following sub-section. It is important to affirm this idea because the method of connecting selection and dispersion directly utilizes intra-class correlations.</p>
<h4 id="coefficient-of-relationship-as-an-intra-class-correlation">Coefficient of relationship as an Intra-class correlation</h4>

<p>It is sometimes "assumed" that the approximations ''' σ<sup>2</sup><sub>A(AL)</sub> = 2 <em>f</em> σ<sup>2</sup><sub>A(0)</sub>''' and ''' σ<sup>2</sup><sub>A(WL)</sub> = (1- <em>f</em>) σ<sup>2</sup><sub>A(0)</sub>''' and ''' σ<sup>2</sup><sub>A(Σ)</sub> = (1 + <em>f</em>) σ<sup>2</sup><sub>A(0)</sub>''' will "suffice" [see section on Dispersion and the Genotypic variance], and it is temporally useful here (it will be corrected subsequently). Furthermore, recall that the <em>inbreeding coefficient in the t-th generation (f<sub>t</sub>)</em> is the same as the <em>co-ancestry coefficient (f<sub>j j’</sub>) in the (t-1) generation</em> [see section on Pedigree analysis, <em>et seq</em>]. Putting all of this together,</p>

<p>'''r<sub>A(jj’)</sub> ''' = ''' σ<sup>2</sup><sub>A(AL)</sub> ''' / ''' σ<sup>2</sup><sub>A(Σ)</sub> ''' = <em>' [2</em>f<sub>jj’</sub>'' σ<sup>2</sup><sub>A(0)</sub>] ''' / ''' [(1 + <em>f<sub>(t-1)</sub></em>) σ<sup>2</sup><sub>A(0)</sub>] ''' = <em>' 2</em>f<sub>jj’</sub>'' ''' / ''' (1 + <em>f<sub>(t-1)</sub></em>) ''' (after cancelling the variances) = <em>' 2</em>f<sub>t</sub>'' ''' / ''' (1 + <em>f<sub>(t-1)</sub></em>) ''' = Sewall Wright's result.</p>

<p>Thus, the <strong>coefficient of relationship</strong> is indeed an <strong>intra-class correlation</strong> in principle.<a class="footnoteRef" href="#fn160" id="fnref160"><sup>160</sup></a>  The simplifying assumptions now need addressing.</p>

<p>Referring to the section on "Dispersion and the Genotypic variance", note that actually ''' σ<sup>2</sup><sub>A(AL)</sub> = 2 <em>f</em> σ<sup>2</sup><sub>a</sub>''' and ''' σ<sup>2</sup><sub>A(Σ)</sub> = (1 + <em>f</em>) σ<sup>2</sup><sub>A(f)</sub>'''. After substituting these into the <em>' r<sub>A</sub></em>' equation, and simplifying, the corrected <strong>genic intra-class correlation (c<sub>A</sub>)</strong> is:-</p>

<p>'''c<sub>A</sub> = r<sub>A</sub> ( a<sup>2</sup> / α<sup>2</sup><sub><em>f</em> (t-1)</sub> ) '''.</p>

<p>The difference between the two is important if dominance is non trivial. See the graphs to the right.</p>
<h4 id="phenotypic-intra-class-correlation">Phenotypic intra-class correlation</h4>

<p>This is the other prominent intra-class correlation relating selection to dispersion. It is based on partitioning the total phenotypic variance into amongst-line and within-line components, which have an underlying genetical cause. Its derivation serves to affirm that a correlation can be constructed with <strong>variance components</strong>. The following biometrical model forms the skeleton:-</p>

<p>''' X<sub>ij</sub> = μ + λ<sub>i</sub> + ω<sub>ij</sub> ''' ...where... '' X<sub>ij</sub> '' is the phenotype of the <em>j-th</em> individual within the <em>i-th</em> line, <em>i</em> equals <em>1 .... g</em> where <em>g = the number of lines</em>, <em>j</em> equals <em>1 .... n</em> where <em>n = number of individuals within the line</em>, <em>μ</em> is the grand mean, <em>λ<sub>i</sub></em> is the <em>line effect</em> of the <em>i-th</em> line [the <strong>expectation</strong> - the deviation between the grand mean and the mean of the individuals within the <em>i-th</em> line], and <em>ω<sub>ij</sub></em> is the deviation of the <em>j-th</em> individual within line <em>i</em> and the mean of all the individuals within <em>line i</em>.<a class="footnoteRef" href="#fn161" id="fnref161"><sup>161</sup></a><a class="footnoteRef" href="#fn162" id="fnref162"><sup>162</sup></a></p>

<p>The variance components associated with this model are:- ''' σ<sup>2</sup><sub>X( i j )</sub> ''' = ''' σ<sup>2</sup><sub>λ</sub> ''' + ''' σ<sup>2</sup><sub>ω</sub> ''' = ''' σ<sup>2</sup><sub>AL</sub> ''' + ''' σ<sup>2</sup><sub>WL</sub> ''' = ''' σ<sup>2</sup><sub>P(Σ)</sub> ''' .</p>

<p>Recalling that the correlation (succinctly stated) is the ratio of the covariance to the variance (or to the geometric mean of two variances if necessary) [see section on "correlated traits"], the covariance [ X( i j ), X( i j’ ) ] needs defining, where ''j ≠ j’ '' being separate individuals within the same line. This covariance can be found as the Expectation of the cross-product of the model-components defining each individual,<a class="footnoteRef" href="#fn163" id="fnref163"><sup>163</sup></a> as follows:-</p>

<p>''' cov [ X( i j ), X( i j’ ) ] = E{(λ<sub>i</sub> + ω<sub>ij</sub>) ( λ<sub>i</sub> + ω<sub>ij’</sub>)} ''', where <strong>E</strong> is the "Expectation", that is the <em>mean under infinite sampling of all background</em>.<a class="footnoteRef" href="#fn164" id="fnref164"><sup>164</sup></a><a class="footnoteRef" href="#fn165" id="fnref165"><sup>165</sup></a> Continuing :-</p>

<p>''' cov [ X( i j ), X( i j’ ) ] = E{( λ<sub>i</sub>) <sup>2</sup> } + E{ λ<sub>i</sub> ω<sub>ij’</sub>} + E{ λ<sub>i</sub> ω<sub>ij</sub>} + E{ω<sub>ij</sub> ω<sub>ij’</sub>} ''' = ''' σ<sup>2</sup><sub>AL</sub> + 0 + 0 + 0 ''', the first by definition of the variance, and the rest by the <em>fundamental assumption of the <strong>independence of effects</strong> in the model</em>.<a class="footnoteRef" href="#fn166" id="fnref166"><sup>166</sup></a><a class="footnoteRef" href="#fn167" id="fnref167"><sup>167</sup></a></p>

<p>Thus, <strong>r<sub>P</sub></strong> = c<sub>P</sub> = ''' cov [ X( i j ), X( i j’ ) ] / σ<sup>2</sup><sub>X( i j )</sub> ''' = ''' σ<sup>2</sup><sub>AL</sub> / [ σ<sup>2</sup><sub>AL</sub> ''' + ''' σ<sup>2</sup><sub>WL</sub> ] '''.</p>

<p>This satisfies the biometrical definition of any intra-class correlation, but it ignores the genetical origin of the matter. The very gamete sampling which has given rise to dispersion also has made likely that the uniting gametes were not independent, with the result that the ''' E{ω<sub>ij</sub> ω<sub>ij’</sub>} ≠ 0 ''' after all ! Sewall Wright's reproductive paths identify two correlations which would give rise to this lack of independence - <strong>g</strong> ( the meiosis correlation - <em>f<sub>(t-1)</sub></em> ) and <strong>f</strong> ( the gamete correlation of separate parents - <em>f<sub>t</sub></em> ) [refer back to that section]. The combined effect of these can be defined as ''' r<sub>g</sub> = f<sub>t</sub> + f<sub>(t-1)</sub> ''', which has a striking resemblance to <em>de novo inbreeding</em> plus <em>carry-over inbreeding</em> of previous sections !! Now, define ''' g<sup>2</sup> ''' as the coefficient of determination which quantifies the <strong><em>genotypic</em></strong> component of ''' σ<sup>2</sup><sub>WL</sub> ''', giving consequently ''' σ<sup>2</sup><sub>WL(g)</sub> = g<sup>2</sup> σ<sup>2</sup><sub>WL</sub> '''. Finally, recalling that <em>covariance equals correlation multiplied by variance</em>, the ''<em>'genotypic covariance amongst individuals within the same dispersion line</em> cov<sub>WL(g)</sub> = r<sub>g</sub> σ<sup>2</sup><sub>WL(g)</sub> = (f<sub>t</sub> + f<sub>(t-1)</sub>) [g<sup>2</sup> σ<sup>2</sup><sub>WL</sub>] '''. That is to say:-</p>

<p>''' E{ω<sub>ij</sub> ω<sub>ij’</sub>} = r<sub>g</sub> σ<sup>2</sup><sub>WL(g)</sub> ''' ....[instead of <strong>0</strong>], giving:-</p>

<p>''' k<sub>P</sub> = [ σ<sup>2</sup><sub>AL</sub> + r<sub>g</sub> σ<sup>2</sup><sub>WL(g)</sub> ] / σ<sup>2</sup><sub>P(Σ)</sub> ''' = ''' r<sub>P</sub> + [ r<sub>g</sub> σ<sup>2</sup><sub>WL(g)</sub> ] / σ<sup>2</sup><sub>P(Σ)</sub> ''', after including the effects of inbreeding. All symbols are defined within the derivation. The implications of this are that if ''' σ<sup>2</sup><sub>AL</sub> ''' and ''' σ<sup>2</sup><sub>WL</sub> ''' were estimated from the Mean-squares of an actual analysis-of-variance of a "nursery", the ''' "apparent" σ<sup>2</sup><sub>AL</sub> ''' would be biased upward by the amount ''' [ r<sub>g</sub> σ<sup>2</sup><sub>WL(g)</sub> ] / σ<sup>2</sup><sub>P(Σ)</sub> ''', for which adjustment would be needed. Unbiased '' r<sub>P</sub> (c<sub>P</sub>)'' could then be calculated.</p>
<h4 id="relating-intra-class-correlations-to-dispersion-and-to-heritabilities">Relating intra-class correlations to Dispersion and to Heritabilities</h4>
<h5 id="fundamentals">Fundamentals</h5>

<p>Both the phenotypic and genic intra-class correlations have been shown to be the ratios of their respective amongst-line variances to their respective total variances, as defined within the simple Dispersion model. Therefore:- <a class="footnoteRef" href="#fn168" id="fnref168"><sup>168</sup></a> </p>

<p>''' σ<sup>2</sup><sub>P(AL)</sub> = c<sub>P</sub> σ<sup>2</sup><sub>P(Σ)</sub> ''', ...and... ''' σ<sup>2</sup><sub>P(WL)</sub> = (1 - c<sub>P</sub>) σ<sup>2</sup><sub>P(Σ)</sub> '''.</p>

<p>Similarly, ''' σ<sup>2</sup><sub>A(AL)</sub> = c<sub>A</sub> σ<sup>2</sup><sub>A(Σ)</sub> ''', ...and... ''' σ<sup>2</sup><sub>A(WL)</sub> = (1 - c<sub>A</sub>) σ<sup>2</sup><sub>A(Σ)</sub> '''.</p>

<p>These are without error provided unbiased intra-class correlations (<strong>c<sub>P</sub></strong> and <strong>c<sub>A</sub></strong>) are used [see the sections above]. The amongst-line phenotypic variance is the variance of dispersed progeny means arising from <strong>genetic drift</strong> and/or <strong>gamete relationship</strong>. It is therefore a "genotypic variance" (of dispersed progenies). The within-line phenotypic variance is the variance of individuals within progeny lines, being their "genotypic variance" confounded probably with their "environmental variance". The "genotypic" component of this was ['''g<sup>2</sup> σ<sup>2</sup><sub>P(WL)</sub> '''] in the amendment giving rise to <strong>k<sub>P</sub></strong> in the section above. The genic variances are the previously-discussed dispersion genic variances.</p>

<p>It is possible also to define "Genotypic" (c<sub>G</sub>) and "Dominance" (c<sub>D</sub>) intra-class correlations which can be used in parallel ways to relate to their respective "amongst-line" and "within-line" variance components.</p>
<h5 id="corollaries">Corollaries</h5>

<p>Recalling the comment above that ''' σ<sup>2</sup><sub>P(AL)</sub> = σ<sup>2</sup><sub>G(AL)</sub> ''', it is often therefore equated to ''' σ<sup>2</sup><sub>A(AL)</sub> ''' = ''' c<sub>A</sub> σ<sup>2</sup><sub>A(Σ)</sub> '''. Now, recalling that ''' r<sub>P</sub> = σ<sup>2</sup><sub>P(AL)</sub> / σ<sup>2</sup><sub>P(Σ)</sub> ''', this new substitution leads to <em>' r<sub>P</sub> = c<sub>A</sub> σ<sup>2</sup><sub>A(Σ)</sub> / σ<sup>2</sup><sub>P(Σ)</sub></em>'. Further, recall that '''h<sup>2</sup><sub>Individuals</sub> = σ<sup>2</sup><sub>A(Σ)</sub> / σ<sup>2</sup><sub>P(Σ)</sub> '''. It can thus be seen that '''r<sub>P</sub> = c<sub>A</sub> h<sup>2</sup> <sub>Indiv.</sub> '''. That is, it is an "overview heritability" of <strong>line dispersion</strong> ! [Notice that this equality applies to '''r<sub>P</sub> ''' : not to <em>k<sub>P</sub></em>.] A simple "within-line" heritability { '''[(1 - c<sub>A</sub>) / (1 - c<sub>P</sub>)] h<sup>2</sup> <sub>Indiv.</sub> ''' }, and "amongst-line" heritability { '''[ c<sub>A</sub> / c<sub>P</sub>] h<sup>2</sup> <sub>Indiv.</sub> '''} can also be constructed : but more utilitarian versions will follow.<a class="footnoteRef" href="#fn169" id="fnref169"><sup>169</sup></a>  Notice that this simple "within-line" heritability is not the same as <em>g<sup>2</sup></em> of the previous section on the "phenotypic intra-class correlation", being a "narrow-sense" heritability instead of a "broad-sense" one. The <strong>g<sup>2</sup></strong> would be '''[(1 - c<sub>G</sub>) / (1 - c<sub>P</sub>)] H<sup>2</sup> <sub>Indiv.</sub> ''' instead.</p>
<h5 id="amongst-line-selection">Amongst-line Selection</h5>

<p>Within a natural dispersed bulk of progeny lines, it is probably impossible to identify the lines and their respective members: but, within the "nursery" of a controlled selection programme, parents and offspring are managed before-hand, and both progeny-lines and their members are certainly identifiable. It is thus possible to analyse the nursery as an "experiment", and to conduct a simple "single hypothesis" (one-way) analysis-of-variance (ANOVA) on it, extracting estimates of means and variance-components together with their standard errors.<a class="footnoteRef" href="#fn170" id="fnref170"><sup>170</sup></a><a class="footnoteRef" href="#fn171" id="fnref171"><sup>171</sup></a><a class="footnoteRef" href="#fn172" id="fnref172"><sup>172</sup></a> If this was <em>possible</em> for the natural bulk, this ANOVA approach <em>could</em> also be utilized.</p>

<p>As a result, <strong>amongst-line selection</strong> can actually be effected: "observe" the line means and apply (to these) amongst-line selection as derived from the selection basics. Because observed means are usable, the <em>variance of an estimated mean</em> has to be added to the <em>σ<sup>2</sup><sub>(AL)</sub></em>, whether it be phenotypic or genic. In general, the <em>variance of an estimated mean</em> '''σ<sup>2</sup><sub>ỹ</sub> = σ<sup>2</sup><sub>y</sub> / n<sub>ỹ</sub> '''.<a class="footnoteRef" href="#fn173" id="fnref173"><sup>173</sup></a><a class="footnoteRef" href="#fn174" id="fnref174"><sup>174</sup></a><a class="footnoteRef" href="#fn175" id="fnref175"><sup>175</sup></a> Similarly, the <em>amongst-line "narrow-sense" heritability</em> has to be re-defined in this new light. Finally, all these modifications have to be amalgamated into one <em>compound-coefficient (<strong>Κ<sub>AL′</sub></strong>)</em> so as to convert at once the <strong>ΔG<sub>indiv</sub></strong> into <strong>ΔG<sub>AL′</sub></strong> . The one remaining modification is to realize that ''selection pressure (<strong>Prob</strong>) '' now applies to the proportion of <em><strong>means</strong></em> which are selected, and is not focused upon the proportion of <em>individuals</em>. Consequently, the selection threshold (<strong>z</strong>), its ordinate (<strong>y</strong>), and the intensity of selection (<strong>i</strong>) are all focused in this same way.</p>

<p>Therefore,<a class="footnoteRef" href="#fn176" id="fnref176"><sup>176</sup></a> ''' σ<sup>2</sup><sub>P(AL′ )</sub> = σ<sup>2</sup><sub>P(AL)</sub> + σ<sup>2</sup><sub>P( ỹ )</sub> = σ<sup>2</sup><sub>P(AL)</sub> + ( σ<sup>2</sup><sub>P(WL)</sub> / n<sub>WL</sub> ) '''. That is:- ''' σ<sup>2</sup><sub>P(AL′ )</sub> = c<sub>P</sub> σ<sup>2</sup><sub>P(Σ)</sub> + [ (1-c<sub>P</sub>) σ<sup>2</sup><sub>P(Σ)</sub> ] / n<sub>WL</sub> '''.</p>

<p>After gathering of terms, and simplifying, this becomes:- ''' σ<sup>2</sup><sub>P(AL′ )</sub> = σ<sup>2</sup><sub>P(Σ)</sub> {[(c<sub>P</sub> (n<sub>WL</sub>-1) + 1) / n<sub>WL</sub> ]} '''.</p>

<p>In a like manner, ''' σ<sup>2</sup><sub>A(AL′ )</sub> = σ<sup>2</sup><sub>A(Σ)</sub> {[(c<sub>A</sub> (n<sub>WL</sub>-1) + 1) / n<sub>WL</sub> ]} '''.</p>

<p>The ratio of the latter to the former furnishes the appropriate "narrow-sense" heritability, which, after simplifying, becomes:- ''' h<sup>2</sup><sub>AL′</sub> = h<sup>2</sup><sub>Indiv</sub> { [c<sub>A</sub> (n<sub>WL</sub>-1) + 1] / [c<sub>P</sub> (n<sub>WL</sub>-1) + 1] } '''.</p>

<p>Lastly, <em>' ΔG<sub>AL′</sub> = i σ<sub>P(AL′ )</sub> h<sup>2</sup><sub>AL′</sub></em>' .... <strong>=</strong> .... <em>' [ i σ<sub>P(Σ)</sub> h<sup>2</sup><sub>Indiv</sub> ] {[(c<sub>A</sub> (n<sub>WL</sub>-1) + 1)] / [ n<sub>WL</sub>(c<sub>P</sub> (n<sub>WL</sub>-1) + 1)]<sup>(1/2)</sup> }</em>' .... <strong>=</strong> .... <em>' ΔG<sub>Indiv</sub> Κ<sub>AL′</sub></em>', ....where <em>' Κ<sub>AL′</sub> = {[(c<sub>A</sub> (n<sub>WL</sub>-1) + 1)] / [ n<sub>WL</sub>(c<sub>P</sub> (n<sub>WL</sub>-1) + 1)]<sup>(1/2)</sup> }</em>'.<a class="footnoteRef" href="#fn177" id="fnref177"><sup>177</sup></a></p>
<h5 id="within-line-selection-and-overall-individual-selection">Within-line Selection (and Overall Individual selection)</h5>

<p>An alternative to <em>amongst-line selection</em> is to select <strong>individuals</strong> from <em>within their progeny line context</em>. This means essentially to select them with respect to their immediate peers, and it is called <strong>within-line selection</strong>. Notice its distinction with respect to "straight" <strong>individual selection</strong>: the latter selects individuals without any reference to their immediate peers within their progeny-line. That is, with "individual selection", the best individuals in the whole nursery are selected with the dispersion subdivisions being "ignored" altogether. In that case, the basic <em>ΔG</em> is used, together with the <strong>total</strong> phenotypic and genic variances, the straightforward <strong>h<sup>2</sup><sub>Indiv</sub></strong>, and the "obvious" <em>selection pressure</em> for overall "non-dispersed" individuals.</p>

<p>However, in <strong>within-line selection</strong>, the within-line partitions of variances (both phenotypic and genic) are used, and furthermore, the <strong>variance of the line mean</strong> has to be <em>removed</em> because it is un-usable when selecting individuals from amongst their immediate peers. Once again, the <em>within-line "narrow-sense" heritability</em> has to be re-defined in this new light. Finally, as before, all these modifications have to be amalgamated into one <em>compound-coefficient (<strong>Κ<sub>WL′</sub></strong>)</em> so as to convert at once the <strong>ΔG<sub>indiv</sub></strong> algebra into <strong>ΔG<sub>WL′</sub></strong> . Also as before, recall that ''selection pressure (<strong>Prob</strong>) '' now applies to the proportion of <em><strong>individuals within a progeny line</strong></em> which are selected, and is not focused upon the proportion of <em>individuals overall in the nursery</em>. Consequently, the selection threshold (<strong>z</strong>), its ordinate (<strong>y</strong>), and the intensity of selection (<strong>i</strong>) are all focused in this alternative way.</p>

<p>Therefore,<a class="footnoteRef" href="#fn178" id="fnref178"><sup>178</sup></a> ''' σ<sup>2</sup><sub>P(WL′ )</sub> = σ<sup>2</sup><sub>P(WL)</sub> - σ<sup>2</sup><sub>P( ỹ )</sub> = σ<sup>2</sup><sub>P(WL)</sub> - ( σ<sup>2</sup><sub>P(WL)</sub> / n<sub>WL</sub> ) '''. That is:- ''' σ<sup>2</sup><sub>P(WL′ )</sub> = (1-c<sub>P</sub>) σ<sup>2</sup><sub>P(Σ)</sub> - [ (1-c<sub>P</sub>) σ<sup>2</sup><sub>P(Σ)</sub> ] / n<sub>WL</sub> '''.</p>

<p>After gathering of terms, and simplifying, this becomes:- ''' σ<sup>2</sup><sub>P(WL′ )</sub> = σ<sup>2</sup><sub>P(Σ)</sub> {[(1-c<sub>P</sub>)(n<sub>WL</sub>-1)] / n<sub>WL</sub> } '''.</p>

<p>In a like manner,''' σ<sup>2</sup><sub>A(WL′ )</sub> = σ<sup>2</sup><sub>A(Σ)</sub> {[(1-c<sub>A</sub>)(n<sub>WL</sub>-1)] / n<sub>WL</sub> } ''' .</p>

<p>The ratio of the latter to the former again furnishes the appropriate "narrow-sense" heritability, which, after simplifying, becomes:- ''' h<sup>2</sup><sub>WL′</sub> = h<sup>2</sup><sub>Indiv</sub> [(1-c<sub>A</sub>) / (1-c<sub>P</sub>)] '''.</p>

<p>Finally, <em>' ΔG<sub>WL′</sub> = i σ<sub>P(WL′ )</sub> h<sup>2</sup><sub>WL′</sub></em>' .... <strong>=</strong> .... <em>' [ i σ<sub>P(Σ)</sub> h<sup>2</sup><sub>Indiv</sub> ] {(1-c<sub>A</sub>) [(n<sub>WL</sub>-1) / ( n<sub>WL</sub>(1-c<sub>P</sub>))]<sup>(1/2)</sup> }</em>' .... <strong>=</strong> .... <em>' ΔG<sub>Indiv</sub> Κ<sub>WL′</sub></em>', ....where <em>' Κ<sub>WL′</sub> = {(1-c<sub>A</sub>) [(n<sub>WL</sub>-1) / ( n<sub>WL</sub>(1-c<sub>P</sub>))]<sup>(1/2)</sup> }</em>'.<a class="footnoteRef" href="#fn179" id="fnref179"><sup>179</sup></a> </p>
<h5 id="selecting-best-individuals-from-best-lines---combined-selection">Selecting best Individuals from best Lines - Combined selection</h5>

<p>Combining <strong>amongst-line selection</strong> and <strong>within-line selection</strong> necessitates two stages, each of which may have different selection pressures applied. The <em>final overall</em> selection pressure is the product of those of the separate stages:- '''Prob<sub>Cbtn</sub> = Prob<sub>AL</sub> Prob<sub>WL</sub> '''. For example, for a <em>Prob<sub>Cbtn</sub></em> of <strong>0.10</strong>, this can be achieved by several combinations of <em>Prob<sub>AL</sub></em> and <em>Prob<sub>WL</sub></em>, such as (0.316 * 0.316), or (0.5 * 0.2), or (0.2 * .05), and many other combinations. In the first of these examples, each stage is given equal "weight", as each is selected with the same selection pressure of 0.316 [this being the approximate square-root of <strong>Prob<sub>Cbtn</sub></strong>]. Recalling that <strong>i = y/Prob</strong> [see section on "standardized selection and the Normal curve"], it is clear that the result for the combination will be the same irrespective of the weights given to each stage, provided that the "final combination selection pressure" achieved is actually that which was planned. Using this '''Prob<sub>Cbtn</sub> ''' will lead to the appropriate <em>i</em>, <em>z</em> and <em>y</em> for the combined selection. </p>

<p>The amalgamation of the two <em>ΔG coefficients</em> from each stage will complete the algebra for <em>Combined selection</em>. Thus ''' Κ<sub>Cbtn</sub> = Κ<sub>AL</sub> + Κ<sub>WL</sub> ''', which, after simplification, becomes:-</p>

<p>''' Κ<sub>Cbtn</sub> = { 1 + [ (c<sub>A</sub> - c<sub>P</sub> )<sup>2</sup> / (1 - c<sub>P</sub> )] [(n<sub>WL</sub> - 1) / (1 + (n<sub>WL</sub> - 1) c<sub>P</sub>)]}<sup>(1/2)</sup> ''', where all symbols have been defined previously.</p>

<p>Finally, <strong>ΔG<sub>Cbnd</sub> = [ i σ<sub>P(Σ)</sub> h<sup>2</sup><sub>Indiv</sub> ] Κ<sub>Cbtn</sub> = ΔG<sub>Indiv</sub> Κ<sub>Cbtn</sub></strong>, remembering to use the appropriate <strong>i</strong> as discussed above.</p>
<h4 id="relative-efficiencies-of-selection-strategies">Relative efficiencies of Selection strategies</h4>

<p>Comparison of the values of the various <em>ΔG conversion coefficients (<strong>Κ<sub>x</sub></strong>)</em> will provide an immediate measure of the relative merits of the four strategies over time (generations) under various inbreeding regimes (dispersion strengths). For this purpose, the <strong>Κ<sub>Indiv</sub></strong> can be set at <strong>1</strong>. Using the <em>Κ<sub>x</sub></em> criteria, <em>combined selection</em> is found to be the best under every inbreeding regime. Therefore, the simplest method to visualize the relative efficiencies is to obtain the ratios of each <strong>Κ<sub>x (t)</sub></strong> to <strong>Κ<sub>Cbnd (t)</sub></strong>. Graphs to the right show these Κ-ratios over ten cycles of selection for <em>successive half-sib</em> inbreeding [see section on half-sib crossing]. It is immediately apparent that, for this inbreeding regime, within-line selection has no value for making genetic advance (ΔG). Its only purpose might be for purification of breeding-stocks.</p>

<p><strong>[An application note:-]</strong> However, even though <em>combined selection</em> is most efficient for ΔG, it is a two-stage selection activity, and therefore is more costly in time and money. It may not be so highly desirable when these new criteria are considered. For that reason, the graphs include a "0.9 efficiency judgment" line as well as the <em>Κ-ratios</em> themselves. This could be used in the following way. If a plant breeder was executing a "line selection" programme under such pollen management,<a class="footnoteRef" href="#fn180" id="fnref180"><sup>180</sup></a> he might decide to utilize cheaper "one-pass" selection strategies that fell well above the <strong><em>0.9 cut-off line</em></strong> in lieu of combined selection. On this basis, using these graphs, he might choose to use overall individual selection (that is "ignore" his dispersion structure in the nursery) for the first two cycles, then use combined selection for four further cycles, and finish with four cycles of amongst-line selection. Of course, this is only one possibility. He would not "abandon" his dispersion <em>maintenance</em>, however, and would continue with the rigorous pollen management which the half-sib crossing demands.</p>
<h3 id="genetic-drift-and-selection">Genetic drift and Selection</h3>

<p>In the foregoing sections, <strong>dispersion</strong> has been considered as an "assistant" to <strong>selection</strong>, and it became apparent that the two can work well together. In quantitative genetics, selection is usually examined in this "biometrical" fashion, but the changes in the means (as monitored by ΔG) reflect the changes in allele and genotype frequencies which lie beneath this surface. Referral to the section on "Genetic drift" brings to mind that it also effects changes in allele and genotype frequencies, and associated means; and that this is the companion aspect to the Dispersion considered here ("the other side of the same coin"). However, these two forces of frequency change will seldom be "in concert", and may often act contrary to each other. One (selection) is "directional" being driven by selection pressure acting on the phenotype: the other (genetic drift) is driven by "chance" at fertilization (Binomial probabilities of gamete samples). If the two are tending towards the same allele frequency, their "coincidence" will be the Probability of obtaining that <strong>frequencies sample</strong> in the genetic drift: the likelihood of their being "in conflict", however, is the <strong>sum of Probabilities of all the alternative frequency samples</strong> ! In extreme cases, a single syngamy sampling can undo what selection has achieved, and the probabilities of it happening are available. It is important to keep this in mind. However, genetic drift resulting in sample frequencies <strong><em>similar</em> to</strong> those of the "selection target" will not lead to so drastic an outcome, leading rather to what may be seen as "slowness in reaching selection goals".</p>
<h2 id="correlated-attributes">Correlated attributes</h2>
<figure><b>(Figure)</b>
<figcaption>Sources of attribute correlation.</figcaption>
</figure>

<p>Although some genes have only an effect on a single trait, many genes have an effect on various traits, which is termed <a class="uri" href="pleiotropy" title="wikilink">pleiotropy</a>. Because of this, a change in a single gene will have an effect on all those traits. This is calculated using <a href="covariance" title="wikilink">covariances</a>, and the phenotypic covariance (<strong>cov<sub>P</sub></strong>) between two traits can be partitioned in the same way as the variances described above [<em>e.g.</em> <strong>genic (cov<sub>A</sub> )</strong>, <strong>dominance (cov<sub>D</sub>)</strong>, <strong>environment (cov<sub>E</sub> )</strong>] . In general, the correlation coefficient is the ratio of the <strong>covariance</strong> to the <strong>geometric mean of the two variances</strong> of the traits.<a class="footnoteRef" href="#fn181" id="fnref181"><sup>181</sup></a>  Various correlation coefficients can be obtained, using the appropriate partitions of variances and covariances. Of course, the <em>Phenotypic correlation</em> is the "usual" correlation of Statistics/Biometrics.</p>

<p>

<math display="block" id="Quantitative_genetics:0">
 <semantics>
  <mrow>
   <mtext>Phenotypic correlation</mtext>
   <mo>=</mo>
   <mfrac>
    <mrow>
     <mi>cov</mi>
     <mrow>
      <mo stretchy="false">(</mo>
      <msub>
       <mi>P</mi>
       <mn>1</mn>
      </msub>
      <mo>,</mo>
      <msub>
       <mi>P</mi>
       <mn>2</mn>
      </msub>
      <mo stretchy="false">)</mo>
     </mrow>
    </mrow>
    <msqrt>
     <mrow>
      <msub>
       <mi>V</mi>
       <msub>
        <mi>P</mi>
        <mn>1</mn>
       </msub>
      </msub>
      <mo>*</mo>
      <msub>
       <mi>V</mi>
       <msub>
        <mi>P</mi>
        <mn>2</mn>
       </msub>
      </msub>
     </mrow>
    </msqrt>
   </mfrac>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <eq></eq>
    <mtext>Phenotypic correlation</mtext>
    <apply>
     <divide></divide>
     <apply>
      <times></times>
      <ci>cov</ci>
      <interval closure="open">
       <apply>
        <csymbol cd="ambiguous">subscript</csymbol>
        <ci>P</ci>
        <cn type="integer">1</cn>
       </apply>
       <apply>
        <csymbol cd="ambiguous">subscript</csymbol>
        <ci>P</ci>
        <cn type="integer">2</cn>
       </apply>
      </interval>
     </apply>
     <apply>
      <root></root>
      <apply>
       <times></times>
       <apply>
        <csymbol cd="ambiguous">subscript</csymbol>
        <ci>V</ci>
        <apply>
         <csymbol cd="ambiguous">subscript</csymbol>
         <ci>P</ci>
         <cn type="integer">1</cn>
        </apply>
       </apply>
       <apply>
        <csymbol cd="ambiguous">subscript</csymbol>
        <ci>V</ci>
        <apply>
         <csymbol cd="ambiguous">subscript</csymbol>
         <ci>P</ci>
         <cn type="integer">2</cn>
        </apply>
       </apply>
      </apply>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \mbox{Phenotypic correlation}=\frac{\mathrm{cov}(P_{1},P_{2})}{\sqrt{{V_{P_{1}%
}*V_{P_{2}}}}}
  </annotation>
 </semantics>
</math>

 <em>...and...</em> 

<math display="inline" id="Quantitative_genetics:1">
 <semantics>
  <mrow>
   <mtext>Genotypic correlation</mtext>
   <mo>=</mo>
   <mfrac>
    <mrow>
     <mi>cov</mi>
     <mrow>
      <mo stretchy="false">(</mo>
      <msub>
       <mi>G</mi>
       <mn>1</mn>
      </msub>
      <mo>,</mo>
      <msub>
       <mi>G</mi>
       <mn>2</mn>
      </msub>
      <mo stretchy="false">)</mo>
     </mrow>
    </mrow>
    <msqrt>
     <mrow>
      <msub>
       <mi>V</mi>
       <msub>
        <mi>G</mi>
        <mn>1</mn>
       </msub>
      </msub>
      <mo>*</mo>
      <msub>
       <mi>V</mi>
       <msub>
        <mi>G</mi>
        <mn>2</mn>
       </msub>
      </msub>
     </mrow>
    </msqrt>
   </mfrac>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <eq></eq>
    <mtext>Genotypic correlation</mtext>
    <apply>
     <divide></divide>
     <apply>
      <times></times>
      <ci>cov</ci>
      <interval closure="open">
       <apply>
        <csymbol cd="ambiguous">subscript</csymbol>
        <ci>G</ci>
        <cn type="integer">1</cn>
       </apply>
       <apply>
        <csymbol cd="ambiguous">subscript</csymbol>
        <ci>G</ci>
        <cn type="integer">2</cn>
       </apply>
      </interval>
     </apply>
     <apply>
      <root></root>
      <apply>
       <times></times>
       <apply>
        <csymbol cd="ambiguous">subscript</csymbol>
        <ci>V</ci>
        <apply>
         <csymbol cd="ambiguous">subscript</csymbol>
         <ci>G</ci>
         <cn type="integer">1</cn>
        </apply>
       </apply>
       <apply>
        <csymbol cd="ambiguous">subscript</csymbol>
        <ci>V</ci>
        <apply>
         <csymbol cd="ambiguous">subscript</csymbol>
         <ci>G</ci>
         <cn type="integer">2</cn>
        </apply>
       </apply>
      </apply>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \mbox{Genotypic correlation}=\frac{\mathrm{cov}(G_{1},G_{2})}{\sqrt{{V_{G_{1}}%
*V_{G_{2}}}}}
  </annotation>
 </semantics>
</math>

 <em>...and also...</em> 

<math display="inline" id="Quantitative_genetics:2">
 <semantics>
  <mrow>
   <mtext>Environmental correlation</mtext>
   <mo>=</mo>
   <mfrac>
    <mrow>
     <mi>cov</mi>
     <mrow>
      <mo stretchy="false">(</mo>
      <msub>
       <mi>E</mi>
       <mn>1</mn>
      </msub>
      <mo>,</mo>
      <msub>
       <mi>E</mi>
       <mn>2</mn>
      </msub>
      <mo stretchy="false">)</mo>
     </mrow>
    </mrow>
    <msqrt>
     <mrow>
      <msub>
       <mi>V</mi>
       <msub>
        <mi>E</mi>
        <mn>1</mn>
       </msub>
      </msub>
      <mo>*</mo>
      <msub>
       <mi>V</mi>
       <msub>
        <mi>E</mi>
        <mn>2</mn>
       </msub>
      </msub>
     </mrow>
    </msqrt>
   </mfrac>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <eq></eq>
    <mtext>Environmental correlation</mtext>
    <apply>
     <divide></divide>
     <apply>
      <times></times>
      <ci>cov</ci>
      <interval closure="open">
       <apply>
        <csymbol cd="ambiguous">subscript</csymbol>
        <ci>E</ci>
        <cn type="integer">1</cn>
       </apply>
       <apply>
        <csymbol cd="ambiguous">subscript</csymbol>
        <ci>E</ci>
        <cn type="integer">2</cn>
       </apply>
      </interval>
     </apply>
     <apply>
      <root></root>
      <apply>
       <times></times>
       <apply>
        <csymbol cd="ambiguous">subscript</csymbol>
        <ci>V</ci>
        <apply>
         <csymbol cd="ambiguous">subscript</csymbol>
         <ci>E</ci>
         <cn type="integer">1</cn>
        </apply>
       </apply>
       <apply>
        <csymbol cd="ambiguous">subscript</csymbol>
        <ci>V</ci>
        <apply>
         <csymbol cd="ambiguous">subscript</csymbol>
         <ci>E</ci>
         <cn type="integer">2</cn>
        </apply>
       </apply>
      </apply>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \mbox{Environmental correlation}=\frac{\mathrm{cov}(E_{1},E_{2})}{\sqrt{{V_{E_%
{1}}*V_{E_{2}}}}}
  </annotation>
 </semantics>
</math>

 .</p>

<p>The <strong>genic correlation</strong> {<a href="genetic_correlation" title="wikilink">genetic correlation</a>} is of particular interest, especially in quantifying the correlated effects of selection. It is as follows:-</p>

<p>

<math display="block" id="Quantitative_genetics:3">
 <semantics>
  <mrow>
   <mtext>Genic correlation</mtext>
   <mo>=</mo>
   <mfrac>
    <mrow>
     <mi>cov</mi>
     <mrow>
      <mo stretchy="false">(</mo>
      <msub>
       <mi>A</mi>
       <mn>1</mn>
      </msub>
      <mo>,</mo>
      <msub>
       <mi>A</mi>
       <mn>2</mn>
      </msub>
      <mo stretchy="false">)</mo>
     </mrow>
    </mrow>
    <msqrt>
     <mrow>
      <msub>
       <mi>V</mi>
       <msub>
        <mi>A</mi>
        <mn>1</mn>
       </msub>
      </msub>
      <mo>*</mo>
      <msub>
       <mi>V</mi>
       <msub>
        <mi>A</mi>
        <mn>2</mn>
       </msub>
      </msub>
     </mrow>
    </msqrt>
   </mfrac>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <eq></eq>
    <mtext>Genic correlation</mtext>
    <apply>
     <divide></divide>
     <apply>
      <times></times>
      <ci>cov</ci>
      <interval closure="open">
       <apply>
        <csymbol cd="ambiguous">subscript</csymbol>
        <ci>A</ci>
        <cn type="integer">1</cn>
       </apply>
       <apply>
        <csymbol cd="ambiguous">subscript</csymbol>
        <ci>A</ci>
        <cn type="integer">2</cn>
       </apply>
      </interval>
     </apply>
     <apply>
      <root></root>
      <apply>
       <times></times>
       <apply>
        <csymbol cd="ambiguous">subscript</csymbol>
        <ci>V</ci>
        <apply>
         <csymbol cd="ambiguous">subscript</csymbol>
         <ci>A</ci>
         <cn type="integer">1</cn>
        </apply>
       </apply>
       <apply>
        <csymbol cd="ambiguous">subscript</csymbol>
        <ci>V</ci>
        <apply>
         <csymbol cd="ambiguous">subscript</csymbol>
         <ci>A</ci>
         <cn type="integer">2</cn>
        </apply>
       </apply>
      </apply>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \mbox{Genic correlation}=\frac{\mathrm{cov}(A_{1},A_{2})}{\sqrt{{V_{A_{1}}*V_{%
A_{2}}}}}
  </annotation>
 </semantics>
</math>

</p>
<h2 id="see-also">See also</h2>
<ul>
<li><a href="Artificial_selection" title="wikilink">Artificial selection</a></li>
<li><a href="Diallel_cross" title="wikilink">Diallel cross</a></li>
<li><a href="Douglas_Scott_Falconer" title="wikilink">Douglas Scott Falconer</a></li>
<li><a href="Ewens's_sampling_formula" title="wikilink">Ewens's sampling formula</a></li>
<li><a href="Experimental_evolution" title="wikilink">Experimental evolution</a></li>
<li><a href="Genetic_architecture" title="wikilink">Genetic architecture</a></li>
<li><a href="Genetic_distance" title="wikilink">Genetic distance</a></li>
<li><a class="uri" href="Heritability" title="wikilink">Heritability</a></li>
<li><a href="Ronald_Fisher" title="wikilink">Ronald Fisher</a></li>
</ul>
<h2 id="footnotes-and-references">Footnotes and references</h2>
<h2 id="further-reading">Further reading</h2>
<ul>
<li>Lynch M &amp; Walsh B (1998). Genetics and Analysis of Quantitative Traits. Sinauer, Sunderland, MA.</li>
<li>Roff DA (1997). Evolutionary Quantitative Genetics. Chapman &amp; Hall, New York.</li>
<li>Seykora, Tony. Animal Science 3221 Animal Breeding. Tech. Minneapolis: University of Minnesota, 2011. Print.</li>
</ul>
<h2 id="external-links">External links</h2>
<ul>
<li><a href="http://www.nature.com/scitable/knowledge/library/the-breeder-s-equation-24204828">The Breeder's Equation</a></li>
<li><a href="http://nitro.biosci.arizona.edu/zbook/book.html">Quantitative Genetics Resources</a> by <a href="Michael_Lynch_(geneticist)" title="wikilink">Michael Lynch</a> and <a href="Bruce_Walsh_(geneticist)" title="wikilink">Bruce Walsh</a>, including the two volumes of their textbook, '' Genetics and Analysis of Quantitative Traits'' and <em>Evolution and Selection of Quantitative Traits</em>.</li>
<li><a href="http://www.evolution-textbook.org/content/free/figures/ch14.html">Resources by</a> <a href="Nick_Barton" title="wikilink">Nick Barton</a> <em>et al</em>. from the textbook, <em>Evolution</em>.</li>
</ul>

<p>"</p>

<p><a class="uri" href="Category:Genetics" title="wikilink">Category:Genetics</a></p>
<section class="footnotes">
<hr/>
<ol>
<li id="fn1"><a href="#fnref1">↩</a></li>
<li id="fn2"><a href="#fnref2">↩</a></li>
<li id="fn3"><a href="#fnref3">↩</a></li>
<li id="fn4">The Mendel G.; Bateson W. (1891) paper, with additional comments by Bateson, is reprinted in:- Sinnott E.W.; Dunn L.C.; Dobzhansky T. (1958). "Principles of genetics"; New York, McGraw-Hill: 419-443. Footnote 3, page 422 identifies Bateson as the original translator, and provides the reference for that translation.<a href="#fnref4">↩</a></li>
<li id="fn5">A <em>QTL</em> is a region in the DNA genome that affects, or is associated with, quantitative phenotypic traits.<a href="#fnref5">↩</a></li>
<li id="fn6"><a href="#fnref6">↩</a></li>
<li id="fn7"><a href="#fnref7">↩</a></li>
<li id="fn8">Other symbols are sometimes used, but these are common.<a href="#fnref8">↩</a></li>
<li id="fn9">The allele effect is the average phenotypic deviation of the homozygote from the mid-point of the two contrasting homozygote phenotypes at one locus, when observed over the infinity of all background genotypes and environments. In practice, estimates from large unbiased samples substitute for the parameter.<a href="#fnref9">↩</a></li>
<li id="fn10">The dominance effect is the average phenotypic deviation of the heterozygote from the mid-point of the two homozygotes at one locus, when observed over the infinity of all background genotypes and environments. In practice, estimates from large unbiased samples substitute for the parameter.<a href="#fnref10">↩</a></li>
<li id="fn11"><a href="#fnref11">↩</a></li>
<li id="fn12"></li>
<li id="fn13"></li>
<li id="fn14">Mendel commented on this particular tendency for F1 &gt; P1, <em>ie</em> evidence of hybrid vigour in stem length. However, the difference may not be sufficient to be judged significant. (The relationship between the range and the standard deviation is known [Steel and Torrie (1980): 576], permitting an approximate significance test to be made for this present difference.)<a href="#fnref14">↩</a></li>
<li id="fn15"></li>
<li id="fn16"><a href="#fnref16">↩</a></li>
<li id="fn17"><a href="#fnref17">↩</a></li>
<li id="fn18"><a href="#fnref18">↩</a></li>
<li id="fn19"><a href="#fnref19">↩</a></li>
<li id="fn20">An F2 derived by <em>self fertilizing F1 individuals</em> (an <em>autogamous</em> F2), however, is not an origin of a randomly fertilized population structure. See Gordon (2001).<a href="#fnref20">↩</a></li>
<li id="fn21"></li>
<li id="fn22"><a href="#fnref22">↩</a></li>
<li id="fn23"><a href="#fnref23">↩</a></li>
<li id="fn24"><a href="#fnref24">↩</a></li>
<li id="fn25">Usually in science ethics, a discovery is named after the earliest person to propose it. Castle, however, seems to have been overlooked: and later when re-found, the title "Hardy Weinberg" was so ubiquitous it seemed too late to update it. Perhaps the "Castle Hardy Weinberg" equlilbrium would be a good compromise?<a href="#fnref25">↩</a></li>
<li id="fn26"></li>
<li id="fn27"></li>
<li id="fn28"><a href="#fnref28">↩</a></li>
<li id="fn29"><a href="#fnref29">↩</a></li>
<li id="fn30"></li>
<li id="fn31"><a href="#fnref31">↩</a></li>
<li id="fn32"><a href="#fnref32">↩</a></li>
<li id="fn33"></li>
<li id="fn34"></li>
<li id="fn35"></li>
<li id="fn36"><a href="#fnref36">↩</a></li>
<li id="fn37"></li>
<li id="fn38"></li>
<li id="fn39"></li>
<li id="fn40"></li>
<li id="fn41"><a href="#fnref41">↩</a></li>
<li id="fn42"></li>
<li id="fn43"></li>
<li id="fn44"></li>
<li id="fn45">This is outlined subsequently in the Genotypic Variances section.<a href="#fnref45">↩</a></li>
<li id="fn46">Both are used commonly.<a href="#fnref46">↩</a></li>
<li id="fn47"></li>
<li id="fn48"></li>
<li id="fn49"></li>
<li id="fn50"></li>
<li id="fn51"></li>
<li id="fn52">See the earlier citations.<a href="#fnref52">↩</a></li>
<li id="fn53"><a href="#fnref53">↩</a></li>
<li id="fn54"></li>
<li id="fn55"></li>
<li id="fn56"></li>
<li id="fn57"></li>
<li id="fn58"></li>
<li id="fn59"></li>
<li id="fn60"></li>
<li id="fn61"></li>
<li id="fn62"><a href="#fnref62">↩</a></li>
<li id="fn63"></li>
<li id="fn64"></li>
<li id="fn65"></li>
<li id="fn66"></li>
<li id="fn67"></li>
<li id="fn68"></li>
<li id="fn69"></li>
<li id="fn70"></li>
<li id="fn71"></li>
<li id="fn72"></li>
<li id="fn73"></li>
<li id="fn74"></li>
<li id="fn75"><a href="#fnref75">↩</a></li>
<li id="fn76"><a href="#fnref76">↩</a></li>
<li id="fn77"></li>
<li id="fn78">These have been translated from Mather's symbols into Fisherian ones to facilitate the comparison.<a href="#fnref78">↩</a></li>
<li id="fn79">Covariance is the co-variability between two sets of data - in this case the <strong>a</strong> and the <strong>d</strong>. Similarly to the variance, it is based on a <em>sum of cross-products</em> (<strong>SCP</strong>) instead of a <strong>SS</strong>. From this, it is clear therefore that the variance is but a special form of the covariance!<a href="#fnref79">↩</a></li>
<li id="fn80"><a href="#fnref80">↩</a></li>
<li id="fn81">It has been observed that when <strong>p</strong> = <strong>q</strong>, or when <strong>d</strong> = <strong>0</strong>, <strong>α</strong> [= a+(q-p)d] "reduces" to <strong>a</strong>. In such circumstances, <strong>σ<sup>2</sup><sub>A</sub></strong> = <strong>σ<sup>2</sup><sub>a</sub></strong> - but only <strong><em>numerically</em></strong>. They still have not <em>become</em> the one and the same identity.<a href="#fnref81">↩</a></li>
<li id="fn82"></li>
<li id="fn83"></li>
<li id="fn84"></li>
<li id="fn85"></li>
<li id="fn86"></li>
<li id="fn87"></li>
<li id="fn88"></li>
<li id="fn89"><a href="#fnref89">↩</a></li>
<li id="fn90"><a href="#fnref90">↩</a></li>
<li id="fn91"></li>
<li id="fn92"></li>
<li id="fn93"></li>
<li id="fn94"></li>
<li id="fn95"></li>
<li id="fn96"></li>
<li id="fn97">It is common practice <em>not</em> to have a subscript on the experimental "error" variance.<a href="#fnref97">↩</a></li>
<li id="fn98">This type of variance-ratio is an example of a <a href="coefficient_of_determination" title="wikilink">coefficient of determination</a>. It is used particularly in <a href="regression_analysis" title="wikilink">regression analysis</a>. A standardized version of regression analysis is <a href="path_analysis_(statistics)" title="wikilink">path analysis</a>. Standardizing here means that the data were first divided by their own experimental standard errors, in order to unify the scales for all attributes.<a href="#fnref98">↩</a></li>
<li id="fn99"><a href="#fnref99">↩</a></li>
<li id="fn100"><a href="#fnref100">↩</a></li>
<li id="fn101"></li>
<li id="fn102"></li>
<li id="fn103"></li>
<li id="fn104"></li>
<li id="fn105"></li>
<li id="fn106"></li>
<li id="fn107"></li>
<li id="fn108"></li>
<li id="fn109"></li>
<li id="fn110"></li>
<li id="fn111"></li>
<li id="fn112"></li>
<li id="fn113"></li>
<li id="fn114"></li>
<li id="fn115"><a href="#fnref115">↩</a></li>
<li id="fn116"></li>
<li id="fn117"></li>
<li id="fn118"></li>
<li id="fn119"></li>
<li id="fn120"></li>
<li id="fn121"></li>
<li id="fn122"></li>
<li id="fn123"></li>
<li id="fn124"></li>
<li id="fn125"><a href="#fnref125">↩</a></li>
<li id="fn126"></li>
<li id="fn127"></li>
<li id="fn128"></li>
<li id="fn129"></li>
<li id="fn130">In the past, both forms of parent-offspring covariance have been applied to this task of estimating <strong>h<sup>2</sup></strong>, but, as noted in the sub-section above, only one of them (<strong>cov(MPO)</strong>) is actually appropriate. The <strong>cov(PO)</strong> is useful, however, for estimating <strong>H<sup>2</sup></strong> as seen in the main text following.<a href="#fnref130">↩</a></li>
<li id="fn131"></li>
<li id="fn132"></li>
<li id="fn133"></li>
<li id="fn134"></li>
<li id="fn135">Note that texts which ignore the dominance component of cov(HS) erroneously suggest that r<sub>HS</sub> "approximates" ( ¼ h<sup>2</sup> ).<a href="#fnref135">↩</a></li>
<li id="fn136"></li>
<li id="fn137"></li>
<li id="fn138"></li>
<li id="fn139"></li>
<li id="fn140"></li>
<li id="fn141"></li>
<li id="fn142"></li>
<li id="fn143"></li>
<li id="fn144"></li>
<li id="fn145">Theoretically, the tail is <strong><em>infinite</em></strong>, but in practice there is a <em>quasi-end</em>.<a href="#fnref145">↩</a></li>
<li id="fn146"></li>
<li id="fn147"></li>
<li id="fn148"><a href="#fnref148">↩</a></li>
<li id="fn149"></li>
<li id="fn150"></li>
<li id="fn151"></li>
<li id="fn152"></li>
<li id="fn153"></li>
<li id="fn154"></li>
<li id="fn155"></li>
<li id="fn156"></li>
<li id="fn157">There is a small "wobble" arising from the fact that <em>b<sup>2</sup></em> alters one generation behind <em>a<sup>2</sup></em> - examine their inbreeding equations.<a href="#fnref157">↩</a></li>
<li id="fn158"></li>
<li id="fn159"></li>
<li id="fn160"></li>
<li id="fn161"></li>
<li id="fn162"></li>
<li id="fn163"></li>
<li id="fn164"></li>
<li id="fn165"></li>
<li id="fn166"></li>
<li id="fn167"></li>
<li id="fn168"></li>
<li id="fn169"></li>
<li id="fn170"></li>
<li id="fn171"></li>
<li id="fn172"></li>
<li id="fn173"></li>
<li id="fn174"></li>
<li id="fn175"></li>
<li id="fn176"></li>
<li id="fn177"></li>
<li id="fn178"></li>
<li id="fn179"></li>
<li id="fn180"></li>
<li id="fn181"><a href="#fnref181">↩</a></li>
</ol>
</section>
</body>
</html>
