<html lang="en">
<head>
<meta charset="utf-8"/>
<title offset="1963">RNA helicase</title>
<script src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_SVG.js" type="text/javascript">
</script>
</head>
<body>
<h1>RNA helicase</h1>
<hr/>

<p>A helicase in general is defined as “any of various enzymes that catalyze the unwinding and separation of double-stranded DNA or RNA during its replication”.<a class="footnoteRef" href="#fn1" id="fnref1"><sup>1</sup></a> A helicase is more specifically considered a motor enzyme which receives its energy from NTP hydrolysis to unwind double stranded nucleic acids.<a class="footnoteRef" href="#fn2" id="fnref2"><sup>2</sup></a> In our case we are considering specifically RNA Helicase which is involved holistically in the metabolism of RNA which is found in all facets of life on earth.<a class="footnoteRef" href="#fn3" id="fnref3"><sup>3</sup></a> <strong>RNA helicase</strong> (, <em>CSFV NS3 helicase</em>, <em>DBP2</em>, <em>DbpA</em>, <em><a class="uri" href="DDX17" title="wikilink">DDX17</a></em>, <em>DDX25</em>, <em>DDX3</em>, <em><a class="uri" href="DDX3X" title="wikilink">DDX3X</a></em>, <em>DDX3Y</em>, <em>DDX4</em>, <em><a class="uri" href="DDX5" title="wikilink">DDX5</a></em>, <em>DEAD-box protein DED1</em>, <em>DEAD-box RNA helicase</em>, <em>DEAD-box protein 2</em>, <em>DEAH-box RNA helicase</em>, <em>DED1</em>, <em>Dex(H/D) RNA helicase</em>, <em>EhDEAD1</em>, <em>EhDEAD1 RNA helicase</em>, <em>eIF4A helicase</em>, <em>KOKV helicase</em>, <em>Mtr4p</em>, <em>nonstructural protein 3 helicase</em>, <em>NPH-II</em>, <em>RHA</em>, <em><a href="RNA_helicase_A" title="wikilink">RNA helicase A</a></em>, <em>RNA helicase DDX3</em>, <em>RNA helicase Hera</em>, <em>RNA-dependent ATPase</em>, <em>TGBp1 NTPase/helicase domain</em>, <em>VRH1</em>, <em>GRTH/DDX25</em>) is an <a class="uri" href="enzyme" title="wikilink">enzyme</a> with system name <em>ATP phosphohydrolase (RNA helix unwinding)</em>.<a class="footnoteRef" href="#fn4" id="fnref4"><sup>4</sup></a><a class="footnoteRef" href="#fn5" id="fnref5"><sup>5</sup></a><a class="footnoteRef" href="#fn6" id="fnref6"><sup>6</sup></a><a class="footnoteRef" href="#fn7" id="fnref7"><sup>7</sup></a><a class="footnoteRef" href="#fn8" id="fnref8"><sup>8</sup></a><a class="footnoteRef" href="#fn9" id="fnref9"><sup>9</sup></a><a class="footnoteRef" href="#fn10" id="fnref10"><sup>10</sup></a><a class="footnoteRef" href="#fn11" id="fnref11"><sup>11</sup></a> This enzyme <a href="catalysis" title="wikilink">catalyses</a> the following <a href="chemical_reaction" title="wikilink">chemical reaction</a></p>
<dl>
<dd>ATP + H<sub>2</sub>O 

<math display="inline" id="RNA_helicase:0">
 <semantics>
  <mo>⇌</mo>
  <annotation-xml encoding="MathML-Content">
   <ci>normal-⇌</ci>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \rightleftharpoons
  </annotation>
 </semantics>
</math>

 ADP + <a class="uri" href="phosphate" title="wikilink">phosphate</a>
</dd>
</dl>

<p>RNA helicases utilize the energy from ATP <a class="uri" href="hydrolysis" title="wikilink">hydrolysis</a> to unwind RNA.</p>
<h2 id="structure-and-superfamilies">Structure and Superfamilies</h2>

<p>RNA helicases are split into two main categories based on their ability to form oligomeric structures. Of the six superfamilies (SFs) that exist, SFs 1 and 2 do not form rings, whereas SFs 3, 4, 5, and 6 do.</p>

<p>The first two superfamilies, which are usually found in <a class="uri" href="eukaryotes" title="wikilink">eukaryotes</a>, are composed of a structurally conserved core that is usually surrounded by large N- and C- terminal domains that function as <a class="uri" href="RNA" title="wikilink">RNA</a> and <a class="uri" href="DNA" title="wikilink">DNA</a> <a href="binding_domains" title="wikilink">binding domains</a>, protein binding domains, and other molecular specific functions. The function of these domains is extremely significant in cellular interaction by increasing specificity of recruitment of proteins by using structurally specific complexes and sequences within families. These two helicases are also made up of at least 12 structural motifs are positioned in specific sequences that vary between families but are usually highly conserved within the same family.<a class="footnoteRef" href="#fn12" id="fnref12"><sup>12</sup></a></p>

<p>SFs 3 through 6 form hexameric rings, and are usually found in <a class="uri" href="bacteria" title="wikilink">bacteria</a> and <a class="uri" href="viruses" title="wikilink">viruses</a>.<a class="footnoteRef" href="#fn13" id="fnref13"><sup>13</sup></a> <a class="uri" href="Proteins" title="wikilink">Proteins</a> of the superfamilies 3 and 4 are most similar to each other. However, SF4 contains a packaging motor named P4 that plays a role of packaging the <a class="uri" href="RNA" title="wikilink">RNA</a> into a phage by first unwinding the structure and translocating the information into <a class="uri" href="capsids" title="wikilink">capsids</a>. SF5 contains a Bacterial <a href="Rho_factor" title="wikilink">Rho factor</a> that works to regulate <a href="transcription_(genetics)" title="wikilink">transcription</a> termination as well as removing <a href="RNA_polymerase" title="wikilink">RNA polymerase</a>. SF6 contains a structure that works similarly to the Bacterial <a href="Rho_factor" title="wikilink">Rho factor</a> but is different structurally.<a class="footnoteRef" href="#fn14" id="fnref14"><sup>14</sup></a></p>
<h2 id="mechanisms">Mechanisms</h2>

<p>There are two known mechanisms for RNA helicase unwinding: canonical duplex unwinding and unwinding by local strand separation.</p>

<p>During canonical duplex unwinding, the <a class="uri" href="helicase" title="wikilink">helicase</a> first binds to the single stranded region, then uses <a href="ATP_hydrolysis" title="wikilink">ATP hydrolysis</a> as a power stroke in order to <a href="Chromosomal_translocation" title="wikilink">translocate</a> the helicase across the strand. As the helicase slides down one strand, it is dissociating the two strands and removing the complementary. Finally, the helicase is removed. </p>

<p>The second unwinding mechanism is dependent on specific sequences called <a href="DEAD_box" title="wikilink">DEAD box</a> helicases. As opposed to the canonical unwinding, local strand separation loads the helicase directly on the duplex region. <a href="DEAD_box" title="wikilink">DEAD box</a> proteins, which are part of SF2, catalyze ATP-driven structural changes in RNA by unwinding substrates with the help of promoter sequences and accessory domains.<a class="footnoteRef" href="#fn15" id="fnref15"><sup>15</sup></a> Although both mechanisms require <a href="Adenosine_triphosphate" title="wikilink">ATP</a>, local strand separation does not need to hydrolyze <a href="Adenosine_triphosphate" title="wikilink">ATP</a> as long as the <a href="Adenosine_triphosphate" title="wikilink">ATP</a> binds to the duplex strands.<a class="footnoteRef" href="#fn16" id="fnref16"><sup>16</sup></a> </p>
<h2 id="diseases-related-to-rna-helicase">Diseases Related to RNA Helicase</h2>

<p>Most viruses are RNA viruses, usually containing an RNA helicase of their own (except in the case of retroviruses). These viruses depend on RNA helicase in order to replicate in the host, and helicase aids in <a href="transcription_(genetics)" title="wikilink">transcription</a>, <a class="uri" href="translation" title="wikilink">translation</a>, <a href="RNA_splicing" title="wikilink">splicing</a>, assembly, etc. in <a href="infectious_diseases" title="wikilink">infectious diseases</a> such as <a href="Hepatitis_C" title="wikilink">Hepatitis C</a>. RNA processing has also played in a role in <a href="neurological_disorders" title="wikilink">neurological disorders</a> such as <a href="Amyotrophic_lateral_sclerosis" title="wikilink">Amyotrophic lateral sclerosis</a> (<a class="uri" href="ALS" title="wikilink">ALS</a>, <a href="Lou_Gehrig’s_disease" title="wikilink">Lou Gehrig’s disease</a>) and <a href="Alzheimer_disease" title="wikilink">Alzheimer disease</a> (AD). <a href="Alternative_splicing" title="wikilink">Alternative splicing</a> that is carried out by the RNA helicase plays a central role in the <a href="nervous_system" title="wikilink">nervous system</a>, so defects in spliced <a class="uri" href="mRNA" title="wikilink">mRNA</a> may lead to serious <a href="degenerative_disease" title="wikilink">degenerative</a> effects in the <a href="spinal_cord" title="wikilink">spinal cord</a> and <a class="uri" href="brain" title="wikilink">brain</a>. Especially at a young age, it is crucial for the <a href="transcription_(genetics)" title="wikilink">transcription</a> of <a class="uri" href="mRNA" title="wikilink">mRNA</a> to be responsive to binding factors for RNA helicase because these signals are essential for proper growth. It has also been shown that there are several RNA helicases that have altered <a href="gene_expression" title="wikilink">gene expression</a> in <a class="uri" href="cancer" title="wikilink">cancer</a> cells. A specific helicase <a class="uri" href="DDX1" title="wikilink">DDX1</a> is involved in <a class="uri" href="mRNA" title="wikilink">mRNA</a> processing and translation that affects cellular proliferation and <a class="uri" href="tumor" title="wikilink">tumor</a> development. During the progression of cancer, tumor cells are growing at an increased rate by maintaining higher mRNA processing functions because of defects in <a class="uri" href="DDX1" title="wikilink">DDX1</a>. It is known that RNA helicases are extremely important for RNA metabolism, so problems in their function can be detrimental to human life. <a href="Antiviral_drug" title="wikilink">Antiviral</a> and anti-cancer therapies have been targeting defects in RNA helicase and the results have been promising thus far.<a class="footnoteRef" href="#fn17" id="fnref17"><sup>17</sup></a></p>
<h2 id="see-also">See also</h2>
<ul>
<li><a class="uri" href="Helicase" title="wikilink">Helicase</a></li>
<li><a href="DNA_helicase" title="wikilink">DNA helicase</a></li>
<li><a href="Dead_box" title="wikilink">Dead box</a></li>
</ul>
<h2 id="references">References</h2>
<h2 id="external-links">External links</h2>
<ul>
<li></li>
</ul>

<p>"</p>

<p><a href="Category:EC_3.6.4" title="wikilink">Category:EC 3.6.4</a></p>
<section class="footnotes">
<hr/>
<ol>
<li id="fn1">Merriam-Webster. Merriam-Webster, n.d. Web. 15 Oct. 2014.<a href="#fnref1">↩</a></li>
<li id="fn2">Kwong, Ann D., Govinda B. Rao, and Kuan-Teh Jeang. "Viral and Cellular RNA Helicases as Antiviral Targets." Nature.com. Nature Publishing Group, Oct. 2005. Web. 15 Oct. 2014.<a href="#fnref2">↩</a></li>
<li id="fn3">Jankowsky, Eckhard. RNA Helicases. Cambridge: Royal Society of Chemistry, 2010. Print.<a href="#fnref3">↩</a></li>
<li id="fn4"><a href="#fnref4">↩</a></li>
<li id="fn5"><a href="#fnref5">↩</a></li>
<li id="fn6"><a href="#fnref6">↩</a></li>
<li id="fn7"><a href="#fnref7">↩</a></li>
<li id="fn8"><a href="#fnref8">↩</a></li>
<li id="fn9"><a href="#fnref9">↩</a></li>
<li id="fn10"><a href="#fnref10">↩</a></li>
<li id="fn11"><a href="#fnref11">↩</a></li>
<li id="fn12">Jankowsky, Eckhard. "RNA Helicases at Work: Binding and Rearranging." Trends in Biochemical Sciences 36.1 (2011): 19-29. Web.<a href="#fnref12">↩</a></li>
<li id="fn13"></li>
<li id="fn14"></li>
<li id="fn15">Yang, Quansheng, Mark Del Campo, Alan M. Lambowitz, and Eckhard Jankowsky. "DEAD-Box Proteins Unwind Duplexes by Local Strand Separation." Molecular Cell 28.2 (2007): 253-63. Web.<a href="#fnref15">↩</a></li>
<li id="fn16"></li>
<li id="fn17">Steimer, Lenz, and Dagmar Klostermeier. "RNA Helicases in Infection and Disease."RNA Biology 9.6 (2012): 751-71. Web.<a href="#fnref17">↩</a></li>
</ol>
</section>
</body>
</html>
