<html lang="en">
<head>
<meta charset="utf-8"/>
<title offset="1502">Sequence logo</title>
<script src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_SVG.js" type="text/javascript">
</script>
</head>
<body>
<h1>Sequence logo</h1>
<hr/>
<figure><b>(Figure)</b>
<figcaption> A sequence logo showing the most conserved bases around the <a href="initiation_codon" title="wikilink">initiation codon</a> from all human <a href="mRNA" title="wikilink">mRNAs</a> (<a href="Kozak_consensus_sequence" title="wikilink">Kozak consensus sequence</a>). Note that the initiation codon is not drawn to scale, otherwise the letters AUG would each have a height of 2 bits.</figcaption>
</figure>

<p>In <a class="uri" href="bioinformatics" title="wikilink">bioinformatics</a>, a <strong>sequence logo</strong> is a graphical representation of the <a href="sequence_conservation" title="wikilink">sequence conservation</a> of <a href="nucleotide" title="wikilink">nucleotides</a> (in a strand of <a class="uri" href="DNA" title="wikilink">DNA</a>/<a class="uri" href="RNA" title="wikilink">RNA</a>) or <a href="amino_acid" title="wikilink">amino acids</a> (in <a href="protein_sequence" title="wikilink">protein sequences</a>).<a class="footnoteRef" href="#fn1" id="fnref1"><sup>1</sup></a> A sequence logo is created from a collection of aligned sequences and depicts the consensus sequence and diversity of the sequences. Sequence logos are frequently used to depict sequence characteristics such as protein-binding sites in DNA or functional units in proteins.</p>
<h2 id="overview">Overview</h2>

<p>A sequence logo consists of a stack of letters at each position. The relative sizes of the letters indicates their frequency in the sequences. The total height of the letters depicts the information content of the position, in bits.</p>
<h2 id="logo-creation">Logo creation</h2>

<p>To create sequence logos, related DNA, RNA or protein sequences, or DNA sequences that have common conserved binding sites, are aligned so that the most conserved parts create good alignments. A sequence logo can then be created from the conserved multiple sequence alignment. The sequence logo will show how well residues are conserved at each position: the higher the number of residues, the higher the letters will be, because the better the conservation is at that position. Different residues at the same position are scaled according to their frequency. The height of the entire stack of residues is the <a class="uri" href="information" title="wikilink">information</a> measured in <a href="bit" title="wikilink">bits</a>. Sequence logos can be used to represent conserved <a href="DNA_binding_site" title="wikilink">DNA binding sites</a>, where <a href="transcription_factor" title="wikilink">transcription factors</a> bind.</p>

<p>The information content (y-axis) of position 

<math display="inline" id="Sequence_logo:0">
 <semantics>
  <mi>i</mi>
  <annotation-xml encoding="MathML-Content">
   <ci>i</ci>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   i
  </annotation>
 </semantics>
</math>

 is given by:</p>
<dl>
<dd>for amino acids, 

<math display="inline" id="Sequence_logo:1">
 <semantics>
  <mrow>
   <msub>
    <mi>R</mi>
    <mi>i</mi>
   </msub>
   <mo>=</mo>
   <mrow>
    <mrow>
     <msub>
      <mi>log</mi>
      <mn>2</mn>
     </msub>
     <mrow>
      <mo stretchy="false">(</mo>
      <mn>20</mn>
      <mo stretchy="false">)</mo>
     </mrow>
    </mrow>
    <mo>-</mo>
    <mrow>
     <mo stretchy="false">(</mo>
     <mrow>
      <msub>
       <mi>H</mi>
       <mi>i</mi>
      </msub>
      <mo>+</mo>
      <msub>
       <mi>e</mi>
       <mi>n</mi>
      </msub>
     </mrow>
     <mo stretchy="false">)</mo>
    </mrow>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <eq></eq>
    <apply>
     <csymbol cd="ambiguous">subscript</csymbol>
     <ci>R</ci>
     <ci>i</ci>
    </apply>
    <apply>
     <minus></minus>
     <apply>
      <apply>
       <csymbol cd="ambiguous">subscript</csymbol>
       <log></log>
       <cn type="integer">2</cn>
      </apply>
      <cn type="integer">20</cn>
     </apply>
     <apply>
      <plus></plus>
      <apply>
       <csymbol cd="ambiguous">subscript</csymbol>
       <ci>H</ci>
       <ci>i</ci>
      </apply>
      <apply>
       <csymbol cd="ambiguous">subscript</csymbol>
       <ci>e</ci>
       <ci>n</ci>
      </apply>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   R_{i}=\log_{2}(20)-(H_{i}+e_{n})
  </annotation>
 </semantics>
</math>


</dd>
<dd>for nucleic acids, 

<math display="inline" id="Sequence_logo:2">
 <semantics>
  <mrow>
   <msub>
    <mi>R</mi>
    <mi>i</mi>
   </msub>
   <mo>=</mo>
   <mrow>
    <mn>2</mn>
    <mo>-</mo>
    <mrow>
     <mo stretchy="false">(</mo>
     <mrow>
      <msub>
       <mi>H</mi>
       <mi>i</mi>
      </msub>
      <mo>+</mo>
      <msub>
       <mi>e</mi>
       <mi>n</mi>
      </msub>
     </mrow>
     <mo stretchy="false">)</mo>
    </mrow>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <eq></eq>
    <apply>
     <csymbol cd="ambiguous">subscript</csymbol>
     <ci>R</ci>
     <ci>i</ci>
    </apply>
    <apply>
     <minus></minus>
     <cn type="integer">2</cn>
     <apply>
      <plus></plus>
      <apply>
       <csymbol cd="ambiguous">subscript</csymbol>
       <ci>H</ci>
       <ci>i</ci>
      </apply>
      <apply>
       <csymbol cd="ambiguous">subscript</csymbol>
       <ci>e</ci>
       <ci>n</ci>
      </apply>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   R_{i}=2-(H_{i}+e_{n})
  </annotation>
 </semantics>
</math>


</dd>
</dl>

<p>where 

<math display="inline" id="Sequence_logo:3">
 <semantics>
  <msub>
   <mi>H</mi>
   <mi>i</mi>
  </msub>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <csymbol cd="ambiguous">subscript</csymbol>
    <ci>H</ci>
    <ci>i</ci>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   H_{i}
  </annotation>
 </semantics>
</math>

 is the uncertainty (sometimes called the Shannon <a href="Entropy_(Information_theory)" title="wikilink">entropy</a>) of position 

<math display="inline" id="Sequence_logo:4">
 <semantics>
  <mi>i</mi>
  <annotation-xml encoding="MathML-Content">
   <ci>i</ci>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   i
  </annotation>
 </semantics>
</math>

</p>

<p>

<math display="block" id="Sequence_logo:5">
 <semantics>
  <mrow>
   <msub>
    <mi>H</mi>
    <mi>i</mi>
   </msub>
   <mo>=</mo>
   <mrow>
    <mo>-</mo>
    <mrow>
     <mo largeop="true" movablelimits="false" symmetric="true">∑</mo>
     <mrow>
      <msub>
       <mi>f</mi>
       <mrow>
        <mi>a</mi>
        <mo>,</mo>
        <mi>i</mi>
       </mrow>
      </msub>
      <mo>×</mo>
      <mrow>
       <msub>
        <mi>log</mi>
        <mn>2</mn>
       </msub>
       <msub>
        <mi>f</mi>
        <mrow>
         <mi>a</mi>
         <mo>,</mo>
         <mi>i</mi>
        </mrow>
       </msub>
      </mrow>
     </mrow>
    </mrow>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <eq></eq>
    <apply>
     <csymbol cd="ambiguous">subscript</csymbol>
     <ci>H</ci>
     <ci>i</ci>
    </apply>
    <apply>
     <minus></minus>
     <apply>
      <sum></sum>
      <apply>
       <times></times>
       <apply>
        <csymbol cd="ambiguous">subscript</csymbol>
        <ci>f</ci>
        <list>
         <ci>a</ci>
         <ci>i</ci>
        </list>
       </apply>
       <apply>
        <apply>
         <csymbol cd="ambiguous">subscript</csymbol>
         <log></log>
         <cn type="integer">2</cn>
        </apply>
        <apply>
         <csymbol cd="ambiguous">subscript</csymbol>
         <ci>f</ci>
         <list>
          <ci>a</ci>
          <ci>i</ci>
         </list>
        </apply>
       </apply>
      </apply>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   H_{i}=-\sum f_{a,i}\times\log_{2}f_{a,i}
  </annotation>
 </semantics>
</math>

</p>

<p>Here, 

<math display="inline" id="Sequence_logo:6">
 <semantics>
  <msub>
   <mi>f</mi>
   <mrow>
    <mi>a</mi>
    <mo>,</mo>
    <mi>i</mi>
   </mrow>
  </msub>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <csymbol cd="ambiguous">subscript</csymbol>
    <ci>f</ci>
    <list>
     <ci>a</ci>
     <ci>i</ci>
    </list>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   f_{a,i}
  </annotation>
 </semantics>
</math>

 is the <a href="Frequency_(statistics)" title="wikilink">relative frequency</a> of base or amino acid 

<math display="inline" id="Sequence_logo:7">
 <semantics>
  <mi>a</mi>
  <annotation-xml encoding="MathML-Content">
   <ci>a</ci>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   a
  </annotation>
 </semantics>
</math>

 at position 

<math display="inline" id="Sequence_logo:8">
 <semantics>
  <mi>i</mi>
  <annotation-xml encoding="MathML-Content">
   <ci>i</ci>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   i
  </annotation>
 </semantics>
</math>

, and 

<math display="inline" id="Sequence_logo:9">
 <semantics>
  <msub>
   <mi>e</mi>
   <mi>n</mi>
  </msub>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <csymbol cd="ambiguous">subscript</csymbol>
    <ci>e</ci>
    <ci>n</ci>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   e_{n}
  </annotation>
 </semantics>
</math>

 is the small-sample correction for an alignment of 

<math display="inline" id="Sequence_logo:10">
 <semantics>
  <mi>n</mi>
  <annotation-xml encoding="MathML-Content">
   <ci>n</ci>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   n
  </annotation>
 </semantics>
</math>

 letters. The height of letter 

<math display="inline" id="Sequence_logo:11">
 <semantics>
  <mi>a</mi>
  <annotation-xml encoding="MathML-Content">
   <ci>a</ci>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   a
  </annotation>
 </semantics>
</math>

 in column 

<math display="inline" id="Sequence_logo:12">
 <semantics>
  <mi>i</mi>
  <annotation-xml encoding="MathML-Content">
   <ci>i</ci>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   i
  </annotation>
 </semantics>
</math>

 is given by</p>

<p>

<math display="block" id="Sequence_logo:13">
 <semantics>
  <mrow>
   <mtext>height</mtext>
   <mo>=</mo>
   <mrow>
    <msub>
     <mi>f</mi>
     <mrow>
      <mi>a</mi>
      <mo>,</mo>
      <mi>i</mi>
     </mrow>
    </msub>
    <mo>×</mo>
    <msub>
     <mi>R</mi>
     <mi>i</mi>
    </msub>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <eq></eq>
    <mtext>height</mtext>
    <apply>
     <times></times>
     <apply>
      <csymbol cd="ambiguous">subscript</csymbol>
      <ci>f</ci>
      <list>
       <ci>a</ci>
       <ci>i</ci>
      </list>
     </apply>
     <apply>
      <csymbol cd="ambiguous">subscript</csymbol>
      <ci>R</ci>
      <ci>i</ci>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \text{height}=f_{a,i}\times R_{i}
  </annotation>
 </semantics>
</math>

</p>

<p>The approximation for the small-sample correction, 

<math display="inline" id="Sequence_logo:14">
 <semantics>
  <msub>
   <mi>e</mi>
   <mi>n</mi>
  </msub>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <csymbol cd="ambiguous">subscript</csymbol>
    <ci>e</ci>
    <ci>n</ci>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   e_{n}
  </annotation>
 </semantics>
</math>

, is given by:</p>

<p>

<math display="block" id="Sequence_logo:15">
 <semantics>
  <mrow>
   <msub>
    <mi>e</mi>
    <mi>n</mi>
   </msub>
   <mo>=</mo>
   <mrow>
    <mfrac>
     <mn>1</mn>
     <mrow>
      <mi>ln</mi>
      <mn>2</mn>
     </mrow>
    </mfrac>
    <mo>×</mo>
    <mfrac>
     <mrow>
      <mi>s</mi>
      <mo>-</mo>
      <mn>1</mn>
     </mrow>
     <mrow>
      <mn>2</mn>
      <mi>n</mi>
     </mrow>
    </mfrac>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <eq></eq>
    <apply>
     <csymbol cd="ambiguous">subscript</csymbol>
     <ci>e</ci>
     <ci>n</ci>
    </apply>
    <apply>
     <times></times>
     <apply>
      <divide></divide>
      <cn type="integer">1</cn>
      <apply>
       <ln></ln>
       <cn type="integer">2</cn>
      </apply>
     </apply>
     <apply>
      <divide></divide>
      <apply>
       <minus></minus>
       <ci>s</ci>
       <cn type="integer">1</cn>
      </apply>
      <apply>
       <times></times>
       <cn type="integer">2</cn>
       <ci>n</ci>
      </apply>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   e_{n}=\frac{1}{\ln{2}}\times\frac{s-1}{2n}
  </annotation>
 </semantics>
</math>

</p>

<p>where 

<math display="inline" id="Sequence_logo:16">
 <semantics>
  <mi>s</mi>
  <annotation-xml encoding="MathML-Content">
   <ci>s</ci>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   s
  </annotation>
 </semantics>
</math>

 is 4 for nucleotides, 20 for amino acids, and 

<math display="inline" id="Sequence_logo:17">
 <semantics>
  <mi>n</mi>
  <annotation-xml encoding="MathML-Content">
   <ci>n</ci>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   n
  </annotation>
 </semantics>
</math>

 is the number of sequences in the alignment.</p>
<h2 id="consensus-logo">Consensus logo</h2>

<p>A <strong>consensus logo</strong> is a simplified variation of a sequence logo that can be embedded in text format. Like a sequence logo, a consensus logo is created from a collection of aligned protein or DNA/RNA sequences and conveys information about the conservation of each position of a <a href="sequence_motif" title="wikilink">sequence motif</a> or <a href="sequence_alignment" title="wikilink">sequence alignment</a><a class="footnoteRef" href="#fn2" id="fnref2"><sup>2</sup></a><a class="footnoteRef" href="#fn3" id="fnref3"><sup>3</sup></a> . However, a consensus logo displays only conservation information, and not explicitly the frequency information of each <a class="uri" href="nucleotide" title="wikilink">nucleotide</a> or <a href="amino_acid" title="wikilink">amino acid</a> at each position. Instead of a stack made of several characters, denoting the relative frequency of each character, the consensus logo depicts the degree of conservation of each position using the height of the consensus character at that position.</p>

<p> </p>
<h3 id="advantages-and-drawbacks">Advantages and drawbacks</h3>

<p>The main, and obvious, advantage of consensus logos over sequence logos is their ability to be embedded as text in any <a href="Rich_Text_Format" title="wikilink">Rich Text Format</a> supporting editor/viewer and, therefore, in scientific manuscripts. As described above, the consensus logo is a cross between <a href="sequence_logo" title="wikilink">sequence logos</a> and <a href="consensus_sequence" title="wikilink">consensus sequences</a>. As a result, compared to a sequence logo, the consensus logo omits information (the relative contribution of each character to the conservation of that position in the motif/alignment). Hence, a sequence logo should be used preferentially whenever possible. That being said, the need to include graphic figures in order to display sequence logos has perpetuated the use of consensus sequences in scientific manuscripts, even though they fail to convey information on both conservation and frequency.<a class="footnoteRef" href="#fn4" id="fnref4"><sup>4</sup></a> Consensus logos represent therefore an improvement over consensus sequences whenever motif/alignment information has to be constrained to text.</p>
<h2 id="see-also">See also</h2>
<ul>
<li><a href="Sequence_motif" title="wikilink">Sequence motif</a></li>
<li><a href="Position-specific_scoring_matrix" title="wikilink">Position-specific scoring matrix</a></li>
<li><a href="DNA_binding_site" title="wikilink">DNA binding site</a></li>
</ul>
<h2 id="references">References</h2>
<h2 id="external-links">External links</h2>
<ul>
<li><a href="http://alum.mit.edu/www/toms/how.to.read.sequence.logos/">How to read sequence logos</a>.</li>
<li><a href="http://alum.mit.edu/www/toms/logorecommendations.html">Recommendations for Making Sequence Logos</a>.</li>
<li>Erill, I., "A gentle introduction to information content in transcription factor binding sites", <a href="http://compbio.umbc.edu/Documents/Introduction_Information_Theory.pdf">Eprint</a></li>
<li><a href="http://compbio.umbc.edu/Documents/Sequence_logo.pdf">What is (in) a sequence logo?</a></li>
</ul>
<h3 id="tools-for-creating-sequence-logos">Tools for creating sequence logos</h3>
<ul>
<li><a href="http://cran.r-project.org/web/packages/RWebLogo/index.html">RWebLogo</a> R Code, wrapper for python code (BSD licence)</li>
<li><a href="http://code.google.com/p/weblogo/">WebLogo Python Code</a> Python Code (BSD license, somewhat difficult to use)</li>
<li><a href="http://weblogo.threeplusone.com">WebLogo 3.0</a> (Online)</li>
<li><a href="http://www.cbs.dtu.dk/biotools/Seq2Logo/">Seq2Logo</a> (Online app. for peptide alignments feat. pseudo count, sequence weighting and two-sided representation)</li>
<li><a href="http://moraine.cebitec.uni-bielefeld.de">MoRAine</a> (Online application with integrated binding site re-annotation)</li>
<li><a href="http://biogenio.com/logo/">GENIO</a> (Online)</li>
<li><a href="http://demo.tinyray.com/weblogo">PWM-based logo</a> (Online application for motif PWM-based models)</li>
<li><a href="http://www.bionut.ki.se/groups/tbu/logobar/">LogoBar</a> (Java application)</li>
<li><a href="http://alum.mit.edu/www/toms/papers/correlogo/">CorreLogo</a> An online server for 3D sequence logos of RNA and DNA alignments</li>
<li><a href="http://www.malcolmmclean.site11.com/www/SeqLogo/seqlogo.html">seqlogo</a> C function to generate DNA sequence logos</li>
<li><a href="http://compbio.umbc.edu/software/bioword/">1</a> MS-Word AddOn Ribbon that allows generation of <a href="consensus_logo" title="wikilink">consensus logos</a></li>
<li><a href="http://rth.dk/resources/rilogo/">RILogo</a> program and web server for creating logos for two interacting RNAs</li>
<li><a href="http://skylign.org/">Skylign</a> Online tool for creating logos representing both sequence alignments and profile hidden Markov models</li>
</ul>

<p>"</p>

<p><a class="uri" href="Category:Bioinformatics" title="wikilink">Category:Bioinformatics</a> <a href="Category:Statistical_charts_and_diagrams" title="wikilink">Category:Statistical charts and diagrams</a></p>
<section class="footnotes">
<hr/>
<ol>
<li id="fn1"><a href="#fnref1">↩</a></li>
<li id="fn2"><a href="#fnref2">↩</a></li>
<li id="fn3"><a href="#fnref3">↩</a></li>
<li id="fn4"><a href="#fnref4">↩</a></li>
</ol>
</section>
</body>
</html>
