<html lang="en">
<head>
<meta charset="utf-8"/>
<title offset="80">Significance analysis of microarrays</title>
<script src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_SVG.js" type="text/javascript">
</script>
</head>
<body>
<h1>Significance analysis of microarrays</h1>
<hr/>

<p> <strong>Significance analysis of microarrays (SAM)</strong> is a <a href="statistics" title="wikilink">statistical technique</a>, established in 2001 by Virginia Tusher, <a href="Robert_Tibshirani" title="wikilink">Robert Tibshirani</a> and <a href="Gilbert_Chu" title="wikilink">Gilbert Chu</a>, for determining whether changes in <a href="gene_expression" title="wikilink">gene expression</a> are statistically significant. With the advent of <a href="DNA_microarray" title="wikilink">DNA microarrays</a>, it is now possible to measure the expression of thousands of genes in a single hybridization experiment. The data generated is considerable, and a method for sorting out what is significant and what isn't is essential. SAM is distributed by <a href="Stanford_University" title="wikilink">Stanford University</a> in an <a href="R_(programming_language)" title="wikilink">R-package</a>.</p>

<p>SAM identifies statistically significant genes by carrying out gene specific <a href="Student's_t-test" title="wikilink">t-tests</a> and computes a statistic <em>d<sub>j</sub></em> for each gene <em>j</em>, which measures the strength of the relationship between gene expression and a response variable.<a class="footnoteRef" href="#fn1" id="fnref1"><sup>1</sup></a><a class="footnoteRef" href="#fn2" id="fnref2"><sup>2</sup></a><a class="footnoteRef" href="#fn3" id="fnref3"><sup>3</sup></a> This analysis uses <a href="non-parametric_statistics" title="wikilink">non-parametric statistics</a>, since the data may not follow a <a href="normal_distribution" title="wikilink">normal distribution</a>. The response variable describes and groups the data based on experimental conditions. In this method, repeated <a class="uri" href="permutations" title="wikilink">permutations</a> of the data are used to determine if the expression of any gene is significant related to the response. The use of permutation-based analysis accounts for correlations in genes and avoids <a href="wikt:Special:Search/parametric" title="wikilink">parametric</a> assumptions about the distribution of individual genes. This is an advantage over other techniques (e.g., <a class="uri" href="ANOVA" title="wikilink">ANOVA</a> and <a class="uri" href="Bonferroni" title="wikilink">Bonferroni</a>), which assume equal variance and/or independence of genes.<a class="footnoteRef" href="#fn4" id="fnref4"><sup>4</sup></a></p>
<h2 id="basic-protocol">Basic protocol</h2>
<ul>
<li>Perform <a class="uri" href="microarray" title="wikilink">microarray</a> experiments — DNA microarray with oligo and cDNA primers, SNP arrays, protein arrays, etc.</li>
<li>Input Expression Analysis in Microsoft Excel — see below</li>
<li>Run SAM as a Microsoft Excel Add-Ins</li>
<li>Adjust the Delta tuning parameter to get a significant # of genes along with an acceptable false discovery rate (FDR)) and Assess Sample Size by calculating the mean difference in expression in the SAM Plot Controller</li>
<li>List Differentially Expressed Genes (Positively and Negatively Expressed Genes)</li>
</ul>
<h2 id="running-sam">Running SAM</h2>
<ul>
<li>SAM is available for download online at <a class="uri" href="http://www-stat.stanford.edu/~tibs/SAM/">http://www-stat.stanford.edu/~tibs/SAM/</a> for academic and non-academic users after completion of a registration step.</li>
<li>SAM is run as an Excel Add-In, and the SAM Plot Controller allows Customization of the False Discovery Rate and Delta, while the SAM Plot and SAM Output functionality generate a List of Significant Genes, Delta Table, and Assessment of Sample Sizes</li>
</ul>
<ul>
<li><a class="uri" href="Permutations" title="wikilink">Permutations</a> are calculated based on the number of samples</li>
</ul>
<ul>
<li>Block Permutations
<ul>
<li>Blocks are batches of microarrays; for example for eight samples split into two groups (control and affected) there are 4!=24 permutations for each block and the total number of permutations is (24)(24)= 576. A minimum of 1000 permutations are recommended;<a class="footnoteRef" href="#fn5" id="fnref5"><sup>5</sup></a><a class="footnoteRef" href="#fn6" id="fnref6"><sup>6</sup></a><a class="footnoteRef" href="#fn7" id="fnref7"><sup>7</sup></a></li>
</ul></li>
</ul>

<p>the number of permutations is set by the user when imputing correct values for the data set to run SAM</p>
<h3 id="response-formats">Response formats<a class="footnoteRef" href="#fn8" id="fnref8"><sup>8</sup></a></h3>

<p><strong>Types</strong></p>

<p>:*<strong>Quantitative</strong> — real-valued (such as heart rate)</p>

<p>:*<strong>One class</strong> — tests whether the mean gene expression differs from zero</p>

<p>:*<strong>Two class</strong> — two sets of measurements</p>

<p>::*<strong>Unpaired</strong> — measurement units are different in the two groups; e.g. control and treatment groups with samples from different patients</p>

<p>::*<strong>Paired</strong> — same experimental units are measured in the two groups; e.g. samples before and after treatment from the same patients</p>

<p>:*<strong>Multiclass</strong> — more than two groups with each containing different experimental units; generalization of two class unpaired type</p>

<p>:*<strong>Survival</strong> — data of a time until an event (for example death or relapse)</p>

<p>:*<strong>Time course</strong> — each experimental units is measured at more than one time point; experimental units fall into a one or two class design</p>

<p>:*<strong>Pattern discovery</strong> — no explicit response parameter is specified; the user specifies eigengene (principal component) of the expression data and treats it as a quantitative response</p>
<h2 id="algorithm">Algorithm</h2>

<p>SAM calculates a test statistic for relative difference in gene expression based on permutation analysis of expression data and calculates a false discovery rate. The principal calculations of the program are illustrated below.<a class="footnoteRef" href="#fn9" id="fnref9"><sup>9</sup></a><a class="footnoteRef" href="#fn10" id="fnref10"><sup>10</sup></a><a class="footnoteRef" href="#fn11" id="fnref11"><sup>11</sup></a></p>

<p> </p>

<p>The <em>s</em><sub>o</sub> constant is chosen to minimize the coefficient of variation of <em>d<sub>i</sub></em>. r<sub><em>i</em></sub> is equal to the expression levels (x) for gene <em>i</em> under y experimental conditions.</p>

<p>

<math display="inline" id="Significance_analysis_of_microarrays:0">
 <semantics>
  <mrow>
   <mrow>
    <mpadded width="+5pt">
     <mi>False</mi>
    </mpadded>
    <mpadded width="+5pt">
     <mi>discovery</mi>
    </mpadded>
    <mpadded width="+5pt">
     <mi>rate</mi>
    </mpadded>
    <mrow>
     <mo stretchy="false">(</mo>
     <mi>FDR</mi>
     <mo stretchy="false">)</mo>
    </mrow>
   </mrow>
   <mo>=</mo>
   <mfrac>
    <mrow>
     <mpadded width="+5pt">
      <mi>Median</mi>
     </mpadded>
     <mrow>
      <mo stretchy="false">(</mo>
      <mrow>
       <mi>or</mi>
       <mpadded width="+5pt">
        <msup>
         <mn>90</mn>
         <mi>th</mi>
        </msup>
       </mpadded>
       <mi>percentile</mi>
      </mrow>
      <mo rspace="7.5pt" stretchy="false">)</mo>
     </mrow>
     <mpadded width="+5pt">
      <mi>of</mi>
     </mpadded>
     <mpadded width="+5pt">
      <mi mathvariant="normal">#</mi>
     </mpadded>
     <mpadded width="+5pt">
      <mi>of</mi>
     </mpadded>
     <mpadded width="+5pt">
      <mi>falsely</mi>
     </mpadded>
     <mpadded width="+5pt">
      <mi>called</mi>
     </mpadded>
     <mi>genes</mi>
    </mrow>
    <mrow>
     <mpadded width="+5pt">
      <mi>Number</mi>
     </mpadded>
     <mpadded width="+5pt">
      <mi>of</mi>
     </mpadded>
     <mpadded width="+5pt">
      <mi>genes</mi>
     </mpadded>
     <mpadded width="+5pt">
      <mi>called</mi>
     </mpadded>
     <mi>significant</mi>
    </mrow>
   </mfrac>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <eq></eq>
    <apply>
     <times></times>
     <ci>False</ci>
     <ci>discovery</ci>
     <ci>rate</ci>
     <ci>FDR</ci>
    </apply>
    <apply>
     <divide></divide>
     <apply>
      <times></times>
      <ci>Median</ci>
      <apply>
       <times></times>
       <ci>or</ci>
       <apply>
        <csymbol cd="ambiguous">superscript</csymbol>
        <cn type="float">90</cn>
        <ci>th</ci>
       </apply>
       <ci>percentile</ci>
      </apply>
      <ci>of</ci>
      <ci>normal-#</ci>
      <ci>of</ci>
      <ci>falsely</ci>
      <ci>called</ci>
      <ci>genes</ci>
     </apply>
     <apply>
      <times></times>
      <ci>Number</ci>
      <ci>of</ci>
      <ci>genes</ci>
      <ci>called</ci>
      <ci>significant</ci>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \mathrm{False\ discovery\ rate\ (FDR)=\frac{Median\ (or\ 90^{th}\ percentile)%
\ of\ \#\ of\ falsely\ called\ genes}{Number\ of\ genes\ called\ significant}}
  </annotation>
 </semantics>
</math>

</p>

<p><strong>Fold changes</strong> (t) are specified to guarantee genes called significant change at least a pre-specified amount. This means that the absolute value of the average expression levels of a gene under each of two conditions must be greater than the fold change (t) to be called positive and less than the inverse of the fold change (t) to be called negative.</p>

<p>The SAM algorithm can be stated as:</p>
<ol>
<li>Order test statistics according to magnitude <a class="footnoteRef" href="#fn12" id="fnref12"><sup>12</sup></a><a class="footnoteRef" href="#fn13" id="fnref13"><sup>13</sup></a></li>
<li>For each permutation compute the ordered null (unaffected) scores <a class="footnoteRef" href="#fn14" id="fnref14"><sup>14</sup></a><a class="footnoteRef" href="#fn15" id="fnref15"><sup>15</sup></a></li>
<li>Plot the ordered test statistic against the expected null scores <a class="footnoteRef" href="#fn16" id="fnref16"><sup>16</sup></a><a class="footnoteRef" href="#fn17" id="fnref17"><sup>17</sup></a></li>
<li>Call each gene significant if the absolute value of the test statistic for that gene minus the mean test statistic for that gene is greater than a stated threshold <a class="footnoteRef" href="#fn18" id="fnref18"><sup>18</sup></a></li>
<li>Estimate the false discovery rate based on expected versus observed values <a class="footnoteRef" href="#fn19" id="fnref19"><sup>19</sup></a><a class="footnoteRef" href="#fn20" id="fnref20"><sup>20</sup></a></li>
</ol>
<h3 id="output">Output</h3>
<ul>
<li>Significant gene sets
<ul>
<li>Positive gene set — higher expression of most genes in the gene set correlates with higher values of the phenotype y</li>
<li>Negative gene set — lower expression of most genes in the gene set correlates with higher values of the phenotype y</li>
</ul></li>
</ul>
<h2 id="sam-features">SAM features</h2>
<ul>
<li>Data from Oligo or cDNA arrays, SNP array, protein arrays,etc. can be utilized in SAM<a class="footnoteRef" href="#fn21" id="fnref21"><sup>21</sup></a><a class="footnoteRef" href="#fn22" id="fnref22"><sup>22</sup></a></li>
<li>Correlates expression data to clinical parameters<a class="footnoteRef" href="#fn23" id="fnref23"><sup>23</sup></a></li>
<li>Correlates expression data with time<a class="footnoteRef" href="#fn24" id="fnref24"><sup>24</sup></a></li>
<li>Uses data permutation to estimates False Discovery Rate for multiple testing<a class="footnoteRef" href="#fn25" id="fnref25"><sup>25</sup></a><a class="footnoteRef" href="#fn26" id="fnref26"><sup>26</sup></a><a class="footnoteRef" href="#fn27" id="fnref27"><sup>27</sup></a><a class="footnoteRef" href="#fn28" id="fnref28"><sup>28</sup></a></li>
<li>Reports local false discovery rate (the FDR for genes having a similar d<sub>i</sub> as that gene)<a class="footnoteRef" href="#fn29" id="fnref29"><sup>29</sup></a> and miss rates <a class="footnoteRef" href="#fn30" id="fnref30"><sup>30</sup></a><a class="footnoteRef" href="#fn31" id="fnref31"><sup>31</sup></a></li>
<li>Can work with blocked design for when treatments are applied within different batches of arrays<a class="footnoteRef" href="#fn32" id="fnref32"><sup>32</sup></a></li>
<li>Can adjust threshold determining number of gene called significant<a class="footnoteRef" href="#fn33" id="fnref33"><sup>33</sup></a></li>
</ul>
<h2 id="references">References</h2>
<ul>
<li>Kooperberg, C., S. Sipione, et al. (2002). "Evaluating test statistics to select interesting genes in microarray experiments." Hum. Mol. Genet. 11(19): 2223–2232.</li>
</ul>
<h2 id="external-links">External links</h2>
<ul>
<li><a href="http://www-stat-class.stanford.edu/~tibs/clickwrap/sam.html">SAM download instructions</a></li>
</ul>

<p>"</p>

<p><a href="Category:Statistical_genetics" title="wikilink">Category:Statistical genetics</a> <a href="Category:Gene_expression" title="wikilink">Category:Gene expression</a> <a class="uri" href="Category:Microarrays" title="wikilink">Category:Microarrays</a></p>
<section class="footnotes">
<hr/>
<ol>
<li id="fn1"></li>
<li id="fn2"></li>
<li id="fn3"></li>
<li id="fn4"></li>
<li id="fn5"></li>
<li id="fn6"></li>
<li id="fn7"></li>
<li id="fn8"></li>
<li id="fn9"></li>
<li id="fn10"></li>
<li id="fn11"></li>
<li id="fn12"></li>
<li id="fn13"></li>
<li id="fn14"></li>
<li id="fn15"></li>
<li id="fn16"></li>
<li id="fn17"></li>
<li id="fn18"></li>
<li id="fn19"></li>
<li id="fn20"></li>
<li id="fn21"></li>
<li id="fn22"></li>
<li id="fn23"></li>
<li id="fn24"></li>
<li id="fn25"></li>
<li id="fn26"></li>
<li id="fn27"></li>
<li id="fn28"></li>
<li id="fn29"></li>
<li id="fn30"></li>
<li id="fn31"></li>
<li id="fn32"></li>
<li id="fn33"></li>
</ol>
</section>
</body>
</html>
