<html lang="en">
<head>
<meta charset="utf-8"/>
<title offset="1345">Similarity matrix</title>
<script src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_SVG.js" type="text/javascript">
</script>
</head>
<body>
<h1>Similarity matrix</h1>
<hr/>

<p>A <strong>similarity matrix</strong> is a <a href="matrix_(mathematics)" title="wikilink">matrix</a> of scores that represent the similarity between a number of data points. Each <em>element</em> of the similarity matrix contains a <a href="Similarity_measure" title="wikilink">measure of similarity</a> between two of the data points. Similarity matrices are strongly related to their counterparts, <a href="distance_matrix" title="wikilink">distance matrices</a> and <a href="substitution_matrix" title="wikilink">substitution matrices</a>.</p>
<h2 id="uses">Uses</h2>

<p>Similarity matrices have a wide range of uses:</p>
<ol>
<li>To find <a href="Cluster_analysis" title="wikilink">clusters</a> of data points.</li>
<li>To <a href="Sequence_alignment" title="wikilink">align sequences</a> of DNA.</li>
</ol>
<h2 id="use-in-clustering">Use in clustering</h2>

<p>In <a href="spectral_clustering" title="wikilink">spectral clustering</a>, a similarity, or affinity, matrix is used to transform data to overcome difficulties related to lack of convexity in the shape of the data distribution.<a class="footnoteRef" href="#fn1" id="fnref1"><sup>1</sup></a> The value of point 

<math display="inline" id="Similarity_matrix:0">
 <semantics>
  <mrow>
   <mo stretchy="false">(</mo>
   <mi>i</mi>
   <mo>,</mo>
   <mi>j</mi>
   <mo stretchy="false">)</mo>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <interval closure="open">
    <ci>i</ci>
    <ci>j</ci>
   </interval>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   (i,j)
  </annotation>
 </semantics>
</math>

 in the matrix can be simply the (negative of the) <a href="Euclidean_distance" title="wikilink">Euclidean distance</a> between 

<math display="inline" id="Similarity_matrix:1">
 <semantics>
  <mi>i</mi>
  <annotation-xml encoding="MathML-Content">
   <ci>i</ci>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   i
  </annotation>
 </semantics>
</math>

 and 

<math display="inline" id="Similarity_matrix:2">
 <semantics>
  <mi>j</mi>
  <annotation-xml encoding="MathML-Content">
   <ci>j</ci>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   j
  </annotation>
 </semantics>
</math>

, or it can be a more complex measure of distance such as the Gaussian 

<math display="inline" id="Similarity_matrix:3">
 <semantics>
  <msup>
   <mi>e</mi>
   <mrow>
    <mo>-</mo>
    <mrow>
     <mrow>
      <msup>
       <mrow>
        <mo fence="true">||</mo>
        <mrow>
         <msub>
          <mi>s</mi>
          <mn>1</mn>
         </msub>
         <mo>-</mo>
         <msub>
          <mi>s</mi>
          <mn>2</mn>
         </msub>
        </mrow>
        <mo fence="true">||</mo>
       </mrow>
       <mn>2</mn>
      </msup>
      <mo>/</mo>
      <mn>2</mn>
     </mrow>
     <msup>
      <mi>σ</mi>
      <mn>2</mn>
     </msup>
    </mrow>
   </mrow>
  </msup>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <csymbol cd="ambiguous">superscript</csymbol>
    <ci>e</ci>
    <apply>
     <minus></minus>
     <apply>
      <times></times>
      <apply>
       <divide></divide>
       <apply>
        <csymbol cd="ambiguous">superscript</csymbol>
        <apply>
         <csymbol cd="latexml">norm</csymbol>
         <apply>
          <minus></minus>
          <apply>
           <csymbol cd="ambiguous">subscript</csymbol>
           <ci>s</ci>
           <cn type="integer">1</cn>
          </apply>
          <apply>
           <csymbol cd="ambiguous">subscript</csymbol>
           <ci>s</ci>
           <cn type="integer">2</cn>
          </apply>
         </apply>
        </apply>
        <cn type="integer">2</cn>
       </apply>
       <cn type="integer">2</cn>
      </apply>
      <apply>
       <csymbol cd="ambiguous">superscript</csymbol>
       <ci>σ</ci>
       <cn type="integer">2</cn>
      </apply>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   e^{-||s_{1}-s_{2}||^{2}/2\sigma^{2}}
  </annotation>
 </semantics>
</math>

.<a class="footnoteRef" href="#fn2" id="fnref2"><sup>2</sup></a> Further modifying this result with network analysis techniques is also common.<a class="footnoteRef" href="#fn3" id="fnref3"><sup>3</sup></a></p>
<h2 id="use-in-sequence-alignment">Use in sequence alignment</h2>

<p>Similarity matrices are used in <a href="sequence_alignment" title="wikilink">sequence alignment</a>. Higher scores are given to more-similar characters, and lower or negative scores for dissimilar characters.</p>

<p><a class="uri" href="Nucleotide" title="wikilink">Nucleotide</a> similarity matrices are used to align <a href="nucleic_acid" title="wikilink">nucleic acid</a> sequences. Because there are only four nucleotides commonly found in <a class="uri" href="DNA" title="wikilink">DNA</a> (<a class="uri" href="Adenine" title="wikilink">Adenine</a> (A), <a class="uri" href="Cytosine" title="wikilink">Cytosine</a> (C), <a class="uri" href="Guanine" title="wikilink">Guanine</a> (G) and <a class="uri" href="Thymine" title="wikilink">Thymine</a> (T)), nucleotide similarity matrices are much simpler than <a class="uri" href="protein" title="wikilink">protein</a> similarity matrices. For example, a simple matrix will assign identical bases a score of +1 and non-identical bases a score of −1. A more complicated matrix would give a higher score to transitions (changes from a <a class="uri" href="pyrimidine" title="wikilink">pyrimidine</a> such as C or T to another pyrimidine, or from a <a class="uri" href="purine" title="wikilink">purine</a> such as A or G to another purine) than to transversions (from a pyrimidine to a purine or vice versa). The match/mismatch ratio of the matrix sets the target evolutionary distance.<a class="footnoteRef" href="#fn4" id="fnref4"><sup>4</sup></a><a class="footnoteRef" href="#fn5" id="fnref5"><sup>5</sup></a> The +1/−3 DNA matrix used by BLASTN is best suited for finding matches between sequences that are 99% identical; a +1/−1 (or +4/−4) matrix is much more suited to sequences with about 70% similarity. Matrices for lower similarity sequences require longer sequence alignments.</p>

<p><a href="Amino_acid" title="wikilink">Amino acid</a> similarity matrices are more complicated, because there are 20 amino acids coded for by the <a href="genetic_code" title="wikilink">genetic code</a>, and so a larger number of possible substitutions. Therefore, the similarity matrix for amino acids contains 400 entries (although it is usually <a href="symmetric_matrix" title="wikilink">symmetric</a>). The first approach scored all amino acid changes equally. A later refinement was to determine amino acid similarities based on how many base changes were required to change a codon to code for that amino acid. This model is better, but it doesn't take into account the selective pressure of amino acid changes. Better models took into account the chemical properties of amino acids.</p>

<p>One approach has been to empirically generate the similarity matrices. The <a href="Margaret_Oakley_Dayhoff" title="wikilink">Dayhoff</a> method used phylogenetic trees and sequences taken from species on the tree. This approach has given rise to the <a href="point_accepted_mutation" title="wikilink">PAM</a> series of matrices. PAM matrices are labelled based on how many nucleotide changes have occurred, per 100 amino acids. While the PAM matrices benefit from having a well understood evolutionary model, they are most useful at short evolutionary distances (PAM10 - PAM120). At long evolutionary distances, for example PAM250 or 20% identity, it has been shown that the <a class="uri" href="BLOSUM" title="wikilink">BLOSUM</a> matrices are much more effective.</p>

<p>The BLOSUM series were generated by comparing a number of divergent sequences. The BLOSUM series are labeled based on how much entropy remains unmutated between all sequences, so a lower BLOSUM number corresponds to a higher PAM number.</p>
<h2 id="see-also">See also</h2>
<ul>
<li><a href="Recurrence_plot" title="wikilink">Recurrence plot</a>, a visualisation tool of recurrences in dynamical (and other) systems</li>
<li><a href="Self-similarity_matrix" title="wikilink">Self-similarity matrix</a></li>
<li><a href="Similarity_testing" title="wikilink">Similarity testing</a></li>
</ul>
<h2 id="notes-and-references">Notes and references</h2>
<references>
</references>

<p>"</p>

<p><a class="uri" href="Category:Bioinformatics" title="wikilink">Category:Bioinformatics</a> <a class="uri" href="Category:DNA" title="wikilink">Category:DNA</a> <a class="uri" href="Category:Matrices" title="wikilink">Category:Matrices</a> <a href="Category:Computational_phylogenetics" title="wikilink">Category:Computational phylogenetics</a> <a href="Category:Statistical_distance_measures" title="wikilink">Category:Statistical distance measures</a> <a href="Category:Multivariate_statistics" title="wikilink">Category:Multivariate statistics</a></p>
<section class="footnotes">
<hr/>
<ol>
<li id="fn1"><a href="#fnref1">↩</a></li>
<li id="fn2"></li>
<li id="fn3"><a href="#fnref3">↩</a></li>
<li id="fn4"><a href="#fnref4">↩</a></li>
<li id="fn5"><a href="#fnref5">↩</a></li>
</ol>
</section>
</body>
</html>
