<html lang="en">
<head>
<meta charset="utf-8"/>
<title offset="1782">Substitution model</title>
<script src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_SVG.js" type="text/javascript">
</script>
</head>
<body>
<h1>Substitution model</h1>
<hr>In biology, a '''substitution
<p><code>model''' describes the process from which a sequence of characters changes into another set of traits. For example, in </code><a href="cladistics" title="wikilink"><code>cladistics</code></a><code>, each position in the sequence might correspond to a property of a </code><a href="species" title="wikilink"><code>species</code></a><code> which can either be present or absent. The </code><a href="Alphabet_(computer_science)" title="wikilink"><code>alphabet</code></a><code> could then consist of "0" for absence and "1" for presence. Then the sequence 00110 could mean, for example, that a species does not have feathers or lay eggs, does have fur, is warm-blooded, and cannot breathe underwater. Another sequence 11010 would mean that a species has feathers, lays eggs, does not have fur, is warm-blooded, and cannot breathe underwater. In </code><a href="phylogenetics" title="wikilink"><code>phylogenetics</code></a><code>, sequences are often obtained by firstly obtaining a </code><a href="nucleotide_sequence" title="wikilink"><code>nucleotide</code></a><code> or </code><a href="protein_sequence" title="wikilink"><code>protein</code> <code>sequence</code></a><code> </code><a href="sequence_alignment" title="wikilink"><code>alignment</code></a><code>, and then taking the </code><a href="nucleotide" title="wikilink"><code>bases</code></a><code> or </code><a href="amino_acid" title="wikilink"><code>amino</code> <code>acids</code></a><code> at corresponding positions in the alignment as the characters. Sequences achieved by this might look like AGCGGAGCTTA and GCCGTAGACGC.</code></p>

<p>Substitution models are used for a number of things:</p>
<ol>
<li>Constructing <a href="evolutionary_tree" title="wikilink">evolutionary trees</a> in phylogenetics or cladistics.</li>
<li>Simulating sequences to test other methods and algorithms.</li>
</ol>
<h2 id="neutral-independent-finite-sites-models">Neutral, independent, finite sites models</h2>

<p>Most substitution models used to date are neutral, independent, finite sites models.</p>
<dl>
<dt><a href="Neutral_theory_of_molecular_evolution" title="wikilink">Neutral</a> : <a href="Natural_selection" title="wikilink">Selection</a> does not operate on the substitutions, and so they are unconstrained.<br/>
<a href="Statistical_independence" title="wikilink">Independent</a> : Changes in one site do not affect the probability of changes in another site.<br/>
Finite Sites : There are finitely many sites, and so over evolution, a single site can be changed multiple times. This means that, for example, if a character has value 0 at time 0 and at time <em>t</em>, it could be that no changes occurred, or that it changed to a 1 and back to a 0, or that it changed to a 1 and back to a 0 and then to a 1 and then back to a 0, and so on.</dt>
</dl>
<h2 id="the-molecular-clock-and-the-units-of-time">The molecular clock and the units of time</h2>

<p>Typically, a branch length of a phylogenetic tree is expressed as the expected number of substitutions per site; if the evolutionary model indicates that each site within an ancestral sequence will typically experience <em>x</em> substitutions by the time it evolves to a particular descendant's sequence then the ancestor and descendant are considered to be separated by branch length <em>x</em>.</p>

<p>Sometimes a branch length is measured in terms of geological years. For example, a fossil record may make it possible to determine the number of years between an ancestral species and a descendant species. Because some species evolve at faster rates than others, these two measures of branch length are not always in direct proportion. The expected number of substitutions per site per year is often indicated with the Greek letter mu (μ).</p>

<p>A model is said to have a strict <a href="molecular_clock" title="wikilink">molecular clock</a> if the expected number of substitutions per year μ is constant regardless of which species' evolution is being examined. An important implication of a strict molecular clock is that the number of expected substitutions between an ancestral species and any of its present-day descendants must be independent of which descendant species is examined.</p>

<p>Note that the assumption of a strict molecular clock is often unrealistic, especially across long periods of evolution. For example, even though <a href="rodent" title="wikilink">rodents</a> are genetically very similar to <a href="primate" title="wikilink">primates</a>, they have undergone a much higher number of substitutions in the estimated time since divergence in some regions of the <a class="uri" href="genome" title="wikilink">genome</a>.<a class="footnoteRef" href="#fn1" id="fnref1"><sup>1</sup></a> This could be due to their shorter <a href="generation_time" title="wikilink">generation time</a>,<a class="footnoteRef" href="#fn2" id="fnref2"><sup>2</sup></a> higher <a href="metabolic_rate" title="wikilink">metabolic rate</a>, increased population structuring, increased rate of <a class="uri" href="speciation" title="wikilink">speciation</a>, or smaller <a href="body_size" title="wikilink">body size</a>.<a class="footnoteRef" href="#fn3" id="fnref3"><sup>3</sup></a><a class="footnoteRef" href="#fn4" id="fnref4"><sup>4</sup></a> When studying ancient events like the <a href="Cambrian_explosion" title="wikilink">Cambrian explosion</a> under a molecular clock assumption, poor concurrence between <a class="uri" href="cladistic" title="wikilink">cladistic</a> and phylogenetic data is often observed. There has been some work on models allowing variable rate of evolution (see for example <a class="footnoteRef" href="#fn5" id="fnref5"><sup>5</sup></a> and <a class="footnoteRef" href="#fn6" id="fnref6"><sup>6</sup></a>).</p>

<p>Models that can take into account variability of the rate of the molecular clock between different evolutionary lineages in the phylogeny are called “relaxed” in opposition to “strict”. In such models the rate can be assumed to be correlated or not between ancestors and descendants and rate variation among lineages can be drawn from many distributions but usually exponential and lognormal distributions are applied. There is a special case, called “local molecular clock” when a phylogeny is divided into at least two partitions (sets of lineages) and in each strict molecular clock is applied but with different rate.</p>
<h2 id="time-reversible-and-stationary-models">Time-reversible and stationary models</h2>

<p>Many useful substitution models are <a href="time_reversibility" title="wikilink">time-reversible</a>; in terms of the mathematics, the model does not care which sequence is the ancestor and which is the descendant so long as all other parameters (such as the number of substitutions per site that is expected between the two sequences) are held constant.</p>

<p>When an analysis of real biological data is performed, there is generally no access to the sequences of ancestral species, only to the present-day species. However, when a model is time-reversible, which species was the ancestral species is irrelevant. Instead, the phylogenetic tree can be rooted using any of the species, re-rooted later based on new knowledge, or left unrooted. This is because there is no 'special' species, all species will eventually derive from one another with the same probability.</p>

<p>A model is time reversible if and only if it satisfies the property (the notation is explained below)</p>

<p>

<math display="block" id="Substitution_model:0">
 <semantics>
  <mrow>
   <mrow>
    <msub>
     <mi>π</mi>
     <mi>i</mi>
    </msub>
    <msub>
     <mi>Q</mi>
     <mrow>
      <mi>i</mi>
      <mi>j</mi>
     </mrow>
    </msub>
   </mrow>
   <mo>=</mo>
   <mrow>
    <msub>
     <mi>π</mi>
     <mi>j</mi>
    </msub>
    <msub>
     <mi>Q</mi>
     <mrow>
      <mi>j</mi>
      <mi>i</mi>
     </mrow>
    </msub>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <eq></eq>
    <apply>
     <times></times>
     <apply>
      <csymbol cd="ambiguous">subscript</csymbol>
      <ci>π</ci>
      <ci>i</ci>
     </apply>
     <apply>
      <csymbol cd="ambiguous">subscript</csymbol>
      <ci>Q</ci>
      <apply>
       <times></times>
       <ci>i</ci>
       <ci>j</ci>
      </apply>
     </apply>
    </apply>
    <apply>
     <times></times>
     <apply>
      <csymbol cd="ambiguous">subscript</csymbol>
      <ci>π</ci>
      <ci>j</ci>
     </apply>
     <apply>
      <csymbol cd="ambiguous">subscript</csymbol>
      <ci>Q</ci>
      <apply>
       <times></times>
       <ci>j</ci>
       <ci>i</ci>
      </apply>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \pi_{i}Q_{ij}=\pi_{j}Q_{ji}
  </annotation>
 </semantics>
</math>

 or, equivalently, the <a href="detailed_balance" title="wikilink">detailed balance</a> property,</p>

<p>

<math display="block" id="Substitution_model:1">
 <semantics>
  <mrow>
   <mrow>
    <msub>
     <mi>π</mi>
     <mi>i</mi>
    </msub>
    <mi>P</mi>
    <msub>
     <mrow>
      <mo stretchy="false">(</mo>
      <mi>t</mi>
      <mo stretchy="false">)</mo>
     </mrow>
     <mrow>
      <mi>i</mi>
      <mi>j</mi>
     </mrow>
    </msub>
   </mrow>
   <mo>=</mo>
   <mrow>
    <msub>
     <mi>π</mi>
     <mi>j</mi>
    </msub>
    <mi>P</mi>
    <msub>
     <mrow>
      <mo stretchy="false">(</mo>
      <mi>t</mi>
      <mo stretchy="false">)</mo>
     </mrow>
     <mrow>
      <mi>j</mi>
      <mi>i</mi>
     </mrow>
    </msub>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <eq></eq>
    <apply>
     <times></times>
     <apply>
      <csymbol cd="ambiguous">subscript</csymbol>
      <ci>π</ci>
      <ci>i</ci>
     </apply>
     <ci>P</ci>
     <apply>
      <csymbol cd="ambiguous">subscript</csymbol>
      <ci>t</ci>
      <apply>
       <times></times>
       <ci>i</ci>
       <ci>j</ci>
      </apply>
     </apply>
    </apply>
    <apply>
     <times></times>
     <apply>
      <csymbol cd="ambiguous">subscript</csymbol>
      <ci>π</ci>
      <ci>j</ci>
     </apply>
     <ci>P</ci>
     <apply>
      <csymbol cd="ambiguous">subscript</csymbol>
      <ci>t</ci>
      <apply>
       <times></times>
       <ci>j</ci>
       <ci>i</ci>
      </apply>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \pi_{i}P(t)_{ij}=\pi_{j}P(t)_{ji}
  </annotation>
 </semantics>
</math>

 for every <em>i</em>, <em>j</em>, and <em>t</em>.</p>

<p>Time-reversibility should not be confused with stationarity. A model is <strong>stationary</strong> if <em>Q</em> does not change with time. The analysis below assumes a stationary model.</p>
<h2 id="the-mathematics-of-substitution-models">The mathematics of substitution models</h2>

<p>Stationary, neutral, independent, finite sites models (assuming a constant rate of evolution) have two parameters, <em>π</em>, an <strong>equilibrium vector</strong> of base (or character) frequencies and a <strong>rate matrix</strong>, <em>Q</em>, which describes the rate at which bases of one type change into bases of another type; element 

<math display="inline" id="Substitution_model:2">
 <semantics>
  <msub>
   <mi>Q</mi>
   <mrow>
    <mi>i</mi>
    <mi>j</mi>
   </mrow>
  </msub>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <csymbol cd="ambiguous">subscript</csymbol>
    <ci>Q</ci>
    <apply>
     <times></times>
     <ci>i</ci>
     <ci>j</ci>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   Q_{ij}
  </annotation>
 </semantics>
</math>

 for <em>i</em> ≠ <em>j</em> is the rate at which base <em>i</em> goes to base <em>j</em>. The diagonals of the <em>Q</em> matrix are chosen so that the rows sum to zero:</p>

<p>

<math display="block" id="Substitution_model:3">
 <semantics>
  <mrow>
   <mrow>
    <msub>
     <mi>Q</mi>
     <mrow>
      <mi>i</mi>
      <mi>i</mi>
     </mrow>
    </msub>
    <mo>=</mo>
    <mrow>
     <mo>-</mo>
     <mrow>
      <munder>
       <mo largeop="true" movablelimits="false" symmetric="true">∑</mo>
       <mrow>
        <mo stretchy="false">{</mo>
        <mi>j</mi>
        <mo>∣</mo>
        <mrow>
         <mi>j</mi>
         <mo>≠</mo>
         <mi>i</mi>
        </mrow>
        <mo stretchy="false">}</mo>
       </mrow>
      </munder>
      <mpadded width="+1.7pt">
       <msub>
        <mi>Q</mi>
        <mrow>
         <mi>i</mi>
         <mi>j</mi>
        </mrow>
       </msub>
      </mpadded>
     </mrow>
    </mrow>
   </mrow>
   <mo>,</mo>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <eq></eq>
    <apply>
     <csymbol cd="ambiguous">subscript</csymbol>
     <ci>Q</ci>
     <apply>
      <times></times>
      <ci>i</ci>
      <ci>i</ci>
     </apply>
    </apply>
    <apply>
     <minus></minus>
     <apply>
      <apply>
       <csymbol cd="ambiguous">subscript</csymbol>
       <sum></sum>
       <apply>
        <csymbol cd="latexml">conditional-set</csymbol>
        <ci>j</ci>
        <apply>
         <neq></neq>
         <ci>j</ci>
         <ci>i</ci>
        </apply>
       </apply>
      </apply>
      <apply>
       <csymbol cd="ambiguous">subscript</csymbol>
       <ci>Q</ci>
       <apply>
        <times></times>
        <ci>i</ci>
        <ci>j</ci>
       </apply>
      </apply>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   Q_{ii}=-{\sum_{\{j\mid j\neq i\}}Q_{ij}}\,,
  </annotation>
 </semantics>
</math>

 The equilibrium row vector <em>π</em> must be annihilated by the rate matrix <em>Q</em>:</p>

<p>

<math display="block" id="Substitution_model:4">
 <semantics>
  <mrow>
   <mrow>
    <mpadded width="+1.7pt">
     <mi>π</mi>
    </mpadded>
    <mi>Q</mi>
   </mrow>
   <mo>=</mo>
   <mn>0 .</mn>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <eq></eq>
    <apply>
     <times></times>
     <ci>π</ci>
     <ci>Q</ci>
    </apply>
    <cn type="float">0 .</cn>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \pi\,Q=0\,.
  </annotation>
 </semantics>
</math>

</p>

<p>The transition matrix function is a function from the branch lengths (in some units of time, possibly in substitutions), to a <a href="matrix_(mathematics)" title="wikilink">matrix</a> of conditional probabilities. It is denoted 

<math display="inline" id="Substitution_model:5">
 <semantics>
  <mrow>
   <mi>P</mi>
   <mrow>
    <mo stretchy="false">(</mo>
    <mi>t</mi>
    <mo stretchy="false">)</mo>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <times></times>
    <ci>P</ci>
    <ci>t</ci>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   P(t)
  </annotation>
 </semantics>
</math>

. The entry in the <em>i</em><sup>th</sup> column and the <em>j</em><sup>th</sup> row, 

<math display="inline" id="Substitution_model:6">
 <semantics>
  <mrow>
   <msub>
    <mi>P</mi>
    <mrow>
     <mi>i</mi>
     <mi>j</mi>
    </mrow>
   </msub>
   <mrow>
    <mo stretchy="false">(</mo>
    <mi>t</mi>
    <mo stretchy="false">)</mo>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <times></times>
    <apply>
     <csymbol cd="ambiguous">subscript</csymbol>
     <ci>P</ci>
     <apply>
      <times></times>
      <ci>i</ci>
      <ci>j</ci>
     </apply>
    </apply>
    <ci>t</ci>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   P_{ij}(t)
  </annotation>
 </semantics>
</math>

, is the probability, after time <em>t</em>, that there is a base <em>j</em> at a given position, conditional on there being a base <em>i</em> in that position at time 0. When the model is time reversible, this can be performed between any two sequences, even if one is not the ancestor of the other, if you know the total branch length between them.</p>

<p>The asymptotic properties of <em>P</em><sub><em>ij</em></sub>(t) are such that <em>P</em><sub><em>ij</em></sub>(0) = δ<sub><em>ij</em></sub>, where δ<sub><em>ij</em></sub> is the <a href="Kronecker_delta" title="wikilink">Kronecker delta</a> function. That is, there is no change in base composition between a sequence and itself. At the other extreme, 

<math display="inline" id="Substitution_model:7">
 <semantics>
  <mrow>
   <mrow>
    <mrow>
     <msub>
      <mo>lim</mo>
      <mrow>
       <mi>t</mi>
       <mo>→</mo>
       <mi mathvariant="normal">∞</mi>
      </mrow>
     </msub>
     <mrow>
      <msub>
       <mi>P</mi>
       <mrow>
        <mi>i</mi>
        <mi>j</mi>
       </mrow>
      </msub>
      <mrow>
       <mo stretchy="false">(</mo>
       <mi>t</mi>
       <mo stretchy="false">)</mo>
      </mrow>
     </mrow>
    </mrow>
    <mo>=</mo>
    <mpadded width="+1.7pt">
     <msub>
      <mi>π</mi>
      <mi>j</mi>
     </msub>
    </mpadded>
   </mrow>
   <mo>,</mo>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <eq></eq>
    <apply>
     <apply>
      <csymbol cd="ambiguous">subscript</csymbol>
      <limit></limit>
      <apply>
       <ci>normal-→</ci>
       <ci>t</ci>
       <infinity></infinity>
      </apply>
     </apply>
     <apply>
      <times></times>
      <apply>
       <csymbol cd="ambiguous">subscript</csymbol>
       <ci>P</ci>
       <apply>
        <times></times>
        <ci>i</ci>
        <ci>j</ci>
       </apply>
      </apply>
      <ci>t</ci>
     </apply>
    </apply>
    <apply>
     <csymbol cd="ambiguous">subscript</csymbol>
     <ci>π</ci>
     <ci>j</ci>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \lim_{t\rightarrow\infty}P_{ij}(t)=\pi_{j}\,,
  </annotation>
 </semantics>
</math>

 or, in other words, as time goes to infinity the probability of finding base <em>j</em> at a position given there was a base <em>i</em> at that position originally goes to the equilibrium probability that there is base <em>j</em> at that position, regardless of the original base. Furthermore, it follows that 

<math display="inline" id="Substitution_model:8">
 <semantics>
  <mrow>
   <mrow>
    <mi>π</mi>
    <mi>P</mi>
    <mrow>
     <mo stretchy="false">(</mo>
     <mi>t</mi>
     <mo stretchy="false">)</mo>
    </mrow>
   </mrow>
   <mo>=</mo>
   <mi>π</mi>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <eq></eq>
    <apply>
     <times></times>
     <ci>π</ci>
     <ci>P</ci>
     <ci>t</ci>
    </apply>
    <ci>π</ci>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \pi P(t)=\pi
  </annotation>
 </semantics>
</math>

 for all <em>t</em>.</p>

<p>The transition matrix can be computed from the rate matrix via <a href="matrix_exponential" title="wikilink">matrix exponentiation</a>:</p>

<p>

<math display="block" id="Substitution_model:9">
 <semantics>
  <mrow>
   <mrow>
    <mrow>
     <mi>P</mi>
     <mrow>
      <mo stretchy="false">(</mo>
      <mi>t</mi>
      <mo stretchy="false">)</mo>
     </mrow>
    </mrow>
    <mo>=</mo>
    <msup>
     <mi>e</mi>
     <mrow>
      <mi>Q</mi>
      <mi>t</mi>
     </mrow>
    </msup>
    <mo>=</mo>
    <mrow>
     <munderover>
      <mo largeop="true" movablelimits="false" symmetric="true">∑</mo>
      <mrow>
       <mi>n</mi>
       <mo>=</mo>
       <mn>0</mn>
      </mrow>
      <mi mathvariant="normal">∞</mi>
     </munderover>
     <mrow>
      <msup>
       <mi>Q</mi>
       <mi>n</mi>
      </msup>
      <mpadded width="+1.7pt">
       <mfrac>
        <msup>
         <mi>t</mi>
         <mi>n</mi>
        </msup>
        <mrow>
         <mi>n</mi>
         <mo lspace="0pt" rspace="3.5pt">!</mo>
        </mrow>
       </mfrac>
      </mpadded>
     </mrow>
    </mrow>
   </mrow>
   <mo>,</mo>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <and></and>
    <apply>
     <eq></eq>
     <apply>
      <times></times>
      <ci>P</ci>
      <ci>t</ci>
     </apply>
     <apply>
      <csymbol cd="ambiguous">superscript</csymbol>
      <ci>e</ci>
      <apply>
       <times></times>
       <ci>Q</ci>
       <ci>t</ci>
      </apply>
     </apply>
    </apply>
    <apply>
     <eq></eq>
     <share href="#.cmml">
     </share>
     <apply>
      <apply>
       <csymbol cd="ambiguous">superscript</csymbol>
       <apply>
        <csymbol cd="ambiguous">subscript</csymbol>
        <sum></sum>
        <apply>
         <eq></eq>
         <ci>n</ci>
         <cn type="integer">0</cn>
        </apply>
       </apply>
       <infinity></infinity>
      </apply>
      <apply>
       <times></times>
       <apply>
        <csymbol cd="ambiguous">superscript</csymbol>
        <ci>Q</ci>
        <ci>n</ci>
       </apply>
       <apply>
        <divide></divide>
        <apply>
         <csymbol cd="ambiguous">superscript</csymbol>
         <ci>t</ci>
         <ci>n</ci>
        </apply>
        <apply>
         <factorial></factorial>
         <ci>n</ci>
        </apply>
       </apply>
      </apply>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   P(t)=e^{Qt}=\sum_{n=0}^{\infty}Q^{n}\frac{t^{n}}{n!}\,,
  </annotation>
 </semantics>
</math>

 where <em>Q</em><sup><em>n</em></sup> is the matrix <em>Q</em> multiplied by itself enough times to give its <em>n</em><sup>th</sup> power.</p>

<p>If <em>Q</em> is <a href="Diagonalizable_matrix" title="wikilink">diagonalizable</a>, the matrix exponential can be <a href="Matrix_exponential#Diagonalizable_case" title="wikilink">computed</a> directly: let <em>Q</em> = <em>U</em><sup>−1</sup> Λ <em>U</em> be a diagonalization of <em>Q</em>, with</p>

<p>

<math display="block" id="Substitution_model:10">
 <semantics>
  <mrow>
   <mrow>
    <mi mathvariant="normal">Λ</mi>
    <mo>=</mo>
    <mpadded width="+1.7pt">
     <mrow>
      <mo>(</mo>
      <mtable displaystyle="true">
       <mtr>
        <mtd columnalign="center">
         <msub>
          <mi>λ</mi>
          <mn>1</mn>
         </msub>
        </mtd>
        <mtd columnalign="center">
         <mi mathvariant="normal">…</mi>
        </mtd>
        <mtd columnalign="center">
         <mn>0</mn>
        </mtd>
       </mtr>
       <mtr>
        <mtd columnalign="center">
         <mi mathvariant="normal">⋮</mi>
        </mtd>
        <mtd columnalign="center">
         <mi mathvariant="normal">⋱</mi>
        </mtd>
        <mtd columnalign="center">
         <mi mathvariant="normal">⋮</mi>
        </mtd>
       </mtr>
       <mtr>
        <mtd columnalign="center">
         <mn>0</mn>
        </mtd>
        <mtd columnalign="center">
         <mi mathvariant="normal">…</mi>
        </mtd>
        <mtd columnalign="center">
         <msub>
          <mi>λ</mi>
          <mn>4</mn>
         </msub>
        </mtd>
       </mtr>
      </mtable>
      <mo>)</mo>
     </mrow>
    </mpadded>
   </mrow>
   <mo>,</mo>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <eq></eq>
    <ci>normal-Λ</ci>
    <matrix>
     <matrixrow>
      <apply>
       <csymbol cd="ambiguous">subscript</csymbol>
       <ci>λ</ci>
       <cn type="integer">1</cn>
      </apply>
      <ci>normal-…</ci>
      <cn type="integer">0</cn>
     </matrixrow>
     <matrixrow>
      <ci>normal-⋮</ci>
      <ci>normal-⋱</ci>
      <ci>normal-⋮</ci>
     </matrixrow>
     <matrixrow>
      <cn type="integer">0</cn>
      <ci>normal-…</ci>
      <apply>
       <csymbol cd="ambiguous">subscript</csymbol>
       <ci>λ</ci>
       <cn type="integer">4</cn>
      </apply>
     </matrixrow>
    </matrix>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \Lambda=\begin{pmatrix}\lambda_{1}&\ldots&0\\
\vdots&\ddots&\vdots\\
0&\ldots&\lambda_{4}\end{pmatrix}\,,
  </annotation>
 </semantics>
</math>

 where Λ is a diagonal matrix and where 

<math display="inline" id="Substitution_model:11">
 <semantics>
  <mrow>
   <mo stretchy="false">{</mo>
   <msub>
    <mi>λ</mi>
    <mi>i</mi>
   </msub>
   <mo stretchy="false">}</mo>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <set>
    <apply>
     <csymbol cd="ambiguous">subscript</csymbol>
     <ci>λ</ci>
     <ci>i</ci>
    </apply>
   </set>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \{\lambda_{i}\}
  </annotation>
 </semantics>
</math>


 are the eigenvalues of <em>Q</em>, each repeated according to its multiplicity. Then</p>

<p>

<math display="block" id="Substitution_model:12">
 <semantics>
  <mrow>
   <mrow>
    <mrow>
     <mi>P</mi>
     <mrow>
      <mo stretchy="false">(</mo>
      <mi>t</mi>
      <mo stretchy="false">)</mo>
     </mrow>
    </mrow>
    <mo>=</mo>
    <msup>
     <mi>e</mi>
     <mrow>
      <mi>Q</mi>
      <mi>t</mi>
     </mrow>
    </msup>
    <mo>=</mo>
    <msup>
     <mi>e</mi>
     <mrow>
      <msup>
       <mi>U</mi>
       <mrow>
        <mo>-</mo>
        <mn>1</mn>
       </mrow>
      </msup>
      <mrow>
       <mo stretchy="false">(</mo>
       <mrow>
        <mi mathvariant="normal">Λ</mi>
        <mi>t</mi>
       </mrow>
       <mo stretchy="false">)</mo>
      </mrow>
      <mi>U</mi>
     </mrow>
    </msup>
    <mo>=</mo>
    <mrow>
     <msup>
      <mi>U</mi>
      <mrow>
       <mo>-</mo>
       <mn>1</mn>
      </mrow>
     </msup>
     <mpadded width="+1.7pt">
      <msup>
       <mi>e</mi>
       <mrow>
        <mi mathvariant="normal">Λ</mi>
        <mi>t</mi>
       </mrow>
      </msup>
     </mpadded>
     <mpadded width="+1.7pt">
      <mi>U</mi>
     </mpadded>
    </mrow>
   </mrow>
   <mo>,</mo>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <and></and>
    <apply>
     <eq></eq>
     <apply>
      <times></times>
      <ci>P</ci>
      <ci>t</ci>
     </apply>
     <apply>
      <csymbol cd="ambiguous">superscript</csymbol>
      <ci>e</ci>
      <apply>
       <times></times>
       <ci>Q</ci>
       <ci>t</ci>
      </apply>
     </apply>
    </apply>
    <apply>
     <eq></eq>
     <share href="#.cmml">
     </share>
     <apply>
      <csymbol cd="ambiguous">superscript</csymbol>
      <ci>e</ci>
      <apply>
       <times></times>
       <apply>
        <csymbol cd="ambiguous">superscript</csymbol>
        <ci>U</ci>
        <apply>
         <minus></minus>
         <cn type="integer">1</cn>
        </apply>
       </apply>
       <apply>
        <times></times>
        <ci>normal-Λ</ci>
        <ci>t</ci>
       </apply>
       <ci>U</ci>
      </apply>
     </apply>
    </apply>
    <apply>
     <eq></eq>
     <share href="#.cmml">
     </share>
     <apply>
      <times></times>
      <apply>
       <csymbol cd="ambiguous">superscript</csymbol>
       <ci>U</ci>
       <apply>
        <minus></minus>
        <cn type="integer">1</cn>
       </apply>
      </apply>
      <apply>
       <csymbol cd="ambiguous">superscript</csymbol>
       <ci>e</ci>
       <apply>
        <times></times>
        <ci>normal-Λ</ci>
        <ci>t</ci>
       </apply>
      </apply>
      <ci>U</ci>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   P(t)=e^{Qt}=e^{U^{-1}(\Lambda t)U}=U^{-1}e^{\Lambda t}\,U\,,
  </annotation>
 </semantics>
</math>

 where the diagonal matrix <em>e</em><sup>Λt</sup> is given by</p>

<p>

<math display="block" id="Substitution_model:13">
 <semantics>
  <mrow>
   <mrow>
    <msup>
     <mi>e</mi>
     <mrow>
      <mi mathvariant="normal">Λ</mi>
      <mi>t</mi>
     </mrow>
    </msup>
    <mo>=</mo>
    <mpadded width="+1.7pt">
     <mrow>
      <mo>(</mo>
      <mtable displaystyle="true">
       <mtr>
        <mtd columnalign="center">
         <msup>
          <mi>e</mi>
          <mrow>
           <msub>
            <mi>λ</mi>
            <mn>1</mn>
           </msub>
           <mi>t</mi>
          </mrow>
         </msup>
        </mtd>
        <mtd columnalign="center">
         <mi mathvariant="normal">…</mi>
        </mtd>
        <mtd columnalign="center">
         <mn>0</mn>
        </mtd>
       </mtr>
       <mtr>
        <mtd columnalign="center">
         <mi mathvariant="normal">⋮</mi>
        </mtd>
        <mtd columnalign="center">
         <mi mathvariant="normal">⋱</mi>
        </mtd>
        <mtd columnalign="center">
         <mi mathvariant="normal">⋮</mi>
        </mtd>
       </mtr>
       <mtr>
        <mtd columnalign="center">
         <mn>0</mn>
        </mtd>
        <mtd columnalign="center">
         <mi mathvariant="normal">…</mi>
        </mtd>
        <mtd columnalign="center">
         <msup>
          <mi>e</mi>
          <mrow>
           <msub>
            <mi>λ</mi>
            <mn>4</mn>
           </msub>
           <mi>t</mi>
          </mrow>
         </msup>
        </mtd>
       </mtr>
      </mtable>
      <mo>)</mo>
     </mrow>
    </mpadded>
   </mrow>
   <mo>.</mo>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <eq></eq>
    <apply>
     <csymbol cd="ambiguous">superscript</csymbol>
     <ci>e</ci>
     <apply>
      <times></times>
      <ci>normal-Λ</ci>
      <ci>t</ci>
     </apply>
    </apply>
    <matrix>
     <matrixrow>
      <apply>
       <csymbol cd="ambiguous">superscript</csymbol>
       <ci>e</ci>
       <apply>
        <times></times>
        <apply>
         <csymbol cd="ambiguous">subscript</csymbol>
         <ci>λ</ci>
         <cn type="integer">1</cn>
        </apply>
        <ci>t</ci>
       </apply>
      </apply>
      <ci>normal-…</ci>
      <cn type="integer">0</cn>
     </matrixrow>
     <matrixrow>
      <ci>normal-⋮</ci>
      <ci>normal-⋱</ci>
      <ci>normal-⋮</ci>
     </matrixrow>
     <matrixrow>
      <cn type="integer">0</cn>
      <ci>normal-…</ci>
      <apply>
       <csymbol cd="ambiguous">superscript</csymbol>
       <ci>e</ci>
       <apply>
        <times></times>
        <apply>
         <csymbol cd="ambiguous">subscript</csymbol>
         <ci>λ</ci>
         <cn type="integer">4</cn>
        </apply>
        <ci>t</ci>
       </apply>
      </apply>
     </matrixrow>
    </matrix>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   e^{\Lambda t}=\begin{pmatrix}e^{\lambda_{1}t}&\ldots&0\\
\vdots&\ddots&\vdots\\
0&\ldots&e^{\lambda_{4}t}\end{pmatrix}\,.
  </annotation>
 </semantics>
</math>

</p>
<h2 id="gtr-generalised-time-reversible">GTR: Generalised time reversible</h2>

<p>GTR is the most general neutral, independent, finite-sites, time-reversible model possible. It was first described in a general form by <a href="Simon_Tavaré" title="wikilink">Simon Tavaré</a> in 1986.<a class="footnoteRef" href="#fn7" id="fnref7"><sup>7</sup></a></p>

<p>The GTR parameters for nucleotides consist of an equilibrium base frequency vector, 

<math display="inline" id="Substitution_model:14">
 <semantics>
  <mrow>
   <mover accent="true">
    <mi>π</mi>
    <mo stretchy="false">→</mo>
   </mover>
   <mo>=</mo>
   <mrow>
    <mo stretchy="false">(</mo>
    <msub>
     <mi>π</mi>
     <mn>1</mn>
    </msub>
    <mo>,</mo>
    <msub>
     <mi>π</mi>
     <mn>2</mn>
    </msub>
    <mo>,</mo>
    <msub>
     <mi>π</mi>
     <mn>3</mn>
    </msub>
    <mo>,</mo>
    <msub>
     <mi>π</mi>
     <mn>4</mn>
    </msub>
    <mo stretchy="false">)</mo>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <eq></eq>
    <apply>
     <ci>normal-→</ci>
     <ci>π</ci>
    </apply>
    <vector>
     <apply>
      <csymbol cd="ambiguous">subscript</csymbol>
      <ci>π</ci>
      <cn type="integer">1</cn>
     </apply>
     <apply>
      <csymbol cd="ambiguous">subscript</csymbol>
      <ci>π</ci>
      <cn type="integer">2</cn>
     </apply>
     <apply>
      <csymbol cd="ambiguous">subscript</csymbol>
      <ci>π</ci>
      <cn type="integer">3</cn>
     </apply>
     <apply>
      <csymbol cd="ambiguous">subscript</csymbol>
      <ci>π</ci>
      <cn type="integer">4</cn>
     </apply>
    </vector>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \vec{\pi}=(\pi_{1},\pi_{2},\pi_{3},\pi_{4})
  </annotation>
 </semantics>
</math>

, giving the frequency at which each base occurs at each site, and the rate matrix</p>

<p>

<math display="block" id="Substitution_model:15">
 <semantics>
  <mrow>
   <mi>Q</mi>
   <mo>=</mo>
   <mrow>
    <mo>(</mo>
    <mtable displaystyle="true">
     <mtr>
      <mtd columnalign="center">
       <mrow>
        <mo>-</mo>
        <mrow>
         <mo stretchy="false">(</mo>
         <mrow>
          <msub>
           <mi>x</mi>
           <mn>1</mn>
          </msub>
          <mo>+</mo>
          <msub>
           <mi>x</mi>
           <mn>2</mn>
          </msub>
          <mo>+</mo>
          <msub>
           <mi>x</mi>
           <mn>3</mn>
          </msub>
         </mrow>
         <mo stretchy="false">)</mo>
        </mrow>
       </mrow>
      </mtd>
      <mtd columnalign="center">
       <msub>
        <mi>x</mi>
        <mn>1</mn>
       </msub>
      </mtd>
      <mtd columnalign="center">
       <msub>
        <mi>x</mi>
        <mn>2</mn>
       </msub>
      </mtd>
      <mtd columnalign="center">
       <msub>
        <mi>x</mi>
        <mn>3</mn>
       </msub>
      </mtd>
     </mtr>
     <mtr>
      <mtd columnalign="center">
       <mstyle displaystyle="false">
        <mfrac>
         <mrow>
          <msub>
           <mi>π</mi>
           <mn>1</mn>
          </msub>
          <msub>
           <mi>x</mi>
           <mn>1</mn>
          </msub>
         </mrow>
         <msub>
          <mi>π</mi>
          <mn>2</mn>
         </msub>
        </mfrac>
       </mstyle>
      </mtd>
      <mtd columnalign="center">
       <mrow>
        <mo>-</mo>
        <mrow>
         <mo stretchy="false">(</mo>
         <mrow>
          <mstyle displaystyle="false">
           <mfrac>
            <mrow>
             <msub>
              <mi>π</mi>
              <mn>1</mn>
             </msub>
             <msub>
              <mi>x</mi>
              <mn>1</mn>
             </msub>
            </mrow>
            <msub>
             <mi>π</mi>
             <mn>2</mn>
            </msub>
           </mfrac>
          </mstyle>
          <mo>+</mo>
          <msub>
           <mi>x</mi>
           <mn>4</mn>
          </msub>
          <mo>+</mo>
          <msub>
           <mi>x</mi>
           <mn>5</mn>
          </msub>
         </mrow>
         <mo stretchy="false">)</mo>
        </mrow>
       </mrow>
      </mtd>
      <mtd columnalign="center">
       <msub>
        <mi>x</mi>
        <mn>4</mn>
       </msub>
      </mtd>
      <mtd columnalign="center">
       <msub>
        <mi>x</mi>
        <mn>5</mn>
       </msub>
      </mtd>
     </mtr>
     <mtr>
      <mtd columnalign="center">
       <mstyle displaystyle="false">
        <mfrac>
         <mrow>
          <msub>
           <mi>π</mi>
           <mn>1</mn>
          </msub>
          <msub>
           <mi>x</mi>
           <mn>2</mn>
          </msub>
         </mrow>
         <msub>
          <mi>π</mi>
          <mn>3</mn>
         </msub>
        </mfrac>
       </mstyle>
      </mtd>
      <mtd columnalign="center">
       <mstyle displaystyle="false">
        <mfrac>
         <mrow>
          <msub>
           <mi>π</mi>
           <mn>2</mn>
          </msub>
          <msub>
           <mi>x</mi>
           <mn>4</mn>
          </msub>
         </mrow>
         <msub>
          <mi>π</mi>
          <mn>3</mn>
         </msub>
        </mfrac>
       </mstyle>
      </mtd>
      <mtd columnalign="center">
       <mrow>
        <mo>-</mo>
        <mrow>
         <mo stretchy="false">(</mo>
         <mrow>
          <mstyle displaystyle="false">
           <mfrac>
            <mrow>
             <msub>
              <mi>π</mi>
              <mn>1</mn>
             </msub>
             <msub>
              <mi>x</mi>
              <mn>2</mn>
             </msub>
            </mrow>
            <msub>
             <mi>π</mi>
             <mn>3</mn>
            </msub>
           </mfrac>
          </mstyle>
          <mo>+</mo>
          <mstyle displaystyle="false">
           <mfrac>
            <mrow>
             <msub>
              <mi>π</mi>
              <mn>2</mn>
             </msub>
             <msub>
              <mi>x</mi>
              <mn>4</mn>
             </msub>
            </mrow>
            <msub>
             <mi>π</mi>
             <mn>3</mn>
            </msub>
           </mfrac>
          </mstyle>
          <mo>+</mo>
          <msub>
           <mi>x</mi>
           <mn>6</mn>
          </msub>
         </mrow>
         <mo stretchy="false">)</mo>
        </mrow>
       </mrow>
      </mtd>
      <mtd columnalign="center">
       <msub>
        <mi>x</mi>
        <mn>6</mn>
       </msub>
      </mtd>
     </mtr>
     <mtr>
      <mtd columnalign="center">
       <mstyle displaystyle="false">
        <mfrac>
         <mrow>
          <msub>
           <mi>π</mi>
           <mn>1</mn>
          </msub>
          <msub>
           <mi>x</mi>
           <mn>3</mn>
          </msub>
         </mrow>
         <msub>
          <mi>π</mi>
          <mn>4</mn>
         </msub>
        </mfrac>
       </mstyle>
      </mtd>
      <mtd columnalign="center">
       <mstyle displaystyle="false">
        <mfrac>
         <mrow>
          <msub>
           <mi>π</mi>
           <mn>2</mn>
          </msub>
          <msub>
           <mi>x</mi>
           <mn>5</mn>
          </msub>
         </mrow>
         <msub>
          <mi>π</mi>
          <mn>4</mn>
         </msub>
        </mfrac>
       </mstyle>
      </mtd>
      <mtd columnalign="center">
       <mstyle displaystyle="false">
        <mfrac>
         <mrow>
          <msub>
           <mi>π</mi>
           <mn>3</mn>
          </msub>
          <msub>
           <mi>x</mi>
           <mn>6</mn>
          </msub>
         </mrow>
         <msub>
          <mi>π</mi>
          <mn>4</mn>
         </msub>
        </mfrac>
       </mstyle>
      </mtd>
      <mtd columnalign="center">
       <mrow>
        <mo>-</mo>
        <mrow>
         <mo stretchy="false">(</mo>
         <mrow>
          <mstyle displaystyle="false">
           <mfrac>
            <mrow>
             <msub>
              <mi>π</mi>
              <mn>1</mn>
             </msub>
             <msub>
              <mi>x</mi>
              <mn>3</mn>
             </msub>
            </mrow>
            <msub>
             <mi>π</mi>
             <mn>4</mn>
            </msub>
           </mfrac>
          </mstyle>
          <mo>+</mo>
          <mstyle displaystyle="false">
           <mfrac>
            <mrow>
             <msub>
              <mi>π</mi>
              <mn>2</mn>
             </msub>
             <msub>
              <mi>x</mi>
              <mn>5</mn>
             </msub>
            </mrow>
            <msub>
             <mi>π</mi>
             <mn>4</mn>
            </msub>
           </mfrac>
          </mstyle>
          <mo>+</mo>
          <mstyle displaystyle="false">
           <mfrac>
            <mrow>
             <msub>
              <mi>π</mi>
              <mn>3</mn>
             </msub>
             <msub>
              <mi>x</mi>
              <mn>6</mn>
             </msub>
            </mrow>
            <msub>
             <mi>π</mi>
             <mn>4</mn>
            </msub>
           </mfrac>
          </mstyle>
         </mrow>
         <mo stretchy="false">)</mo>
        </mrow>
       </mrow>
      </mtd>
     </mtr>
    </mtable>
    <mo>)</mo>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <eq></eq>
    <ci>Q</ci>
    <matrix>
     <matrixrow>
      <apply>
       <minus></minus>
       <apply>
        <plus></plus>
        <apply>
         <csymbol cd="ambiguous">subscript</csymbol>
         <ci>x</ci>
         <cn type="integer">1</cn>
        </apply>
        <apply>
         <csymbol cd="ambiguous">subscript</csymbol>
         <ci>x</ci>
         <cn type="integer">2</cn>
        </apply>
        <apply>
         <csymbol cd="ambiguous">subscript</csymbol>
         <ci>x</ci>
         <cn type="integer">3</cn>
        </apply>
       </apply>
      </apply>
      <apply>
       <csymbol cd="ambiguous">subscript</csymbol>
       <ci>x</ci>
       <cn type="integer">1</cn>
      </apply>
      <apply>
       <csymbol cd="ambiguous">subscript</csymbol>
       <ci>x</ci>
       <cn type="integer">2</cn>
      </apply>
      <apply>
       <csymbol cd="ambiguous">subscript</csymbol>
       <ci>x</ci>
       <cn type="integer">3</cn>
      </apply>
     </matrixrow>
     <matrixrow>
      <apply>
       <divide></divide>
       <apply>
        <times></times>
        <apply>
         <csymbol cd="ambiguous">subscript</csymbol>
         <ci>π</ci>
         <cn type="integer">1</cn>
        </apply>
        <apply>
         <csymbol cd="ambiguous">subscript</csymbol>
         <ci>x</ci>
         <cn type="integer">1</cn>
        </apply>
       </apply>
       <apply>
        <csymbol cd="ambiguous">subscript</csymbol>
        <ci>π</ci>
        <cn type="integer">2</cn>
       </apply>
      </apply>
      <apply>
       <minus></minus>
       <apply>
        <plus></plus>
        <apply>
         <divide></divide>
         <apply>
          <times></times>
          <apply>
           <csymbol cd="ambiguous">subscript</csymbol>
           <ci>π</ci>
           <cn type="integer">1</cn>
          </apply>
          <apply>
           <csymbol cd="ambiguous">subscript</csymbol>
           <ci>x</ci>
           <cn type="integer">1</cn>
          </apply>
         </apply>
         <apply>
          <csymbol cd="ambiguous">subscript</csymbol>
          <ci>π</ci>
          <cn type="integer">2</cn>
         </apply>
        </apply>
        <apply>
         <csymbol cd="ambiguous">subscript</csymbol>
         <ci>x</ci>
         <cn type="integer">4</cn>
        </apply>
        <apply>
         <csymbol cd="ambiguous">subscript</csymbol>
         <ci>x</ci>
         <cn type="integer">5</cn>
        </apply>
       </apply>
      </apply>
      <apply>
       <csymbol cd="ambiguous">subscript</csymbol>
       <ci>x</ci>
       <cn type="integer">4</cn>
      </apply>
      <apply>
       <csymbol cd="ambiguous">subscript</csymbol>
       <ci>x</ci>
       <cn type="integer">5</cn>
      </apply>
     </matrixrow>
     <matrixrow>
      <apply>
       <divide></divide>
       <apply>
        <times></times>
        <apply>
         <csymbol cd="ambiguous">subscript</csymbol>
         <ci>π</ci>
         <cn type="integer">1</cn>
        </apply>
        <apply>
         <csymbol cd="ambiguous">subscript</csymbol>
         <ci>x</ci>
         <cn type="integer">2</cn>
        </apply>
       </apply>
       <apply>
        <csymbol cd="ambiguous">subscript</csymbol>
        <ci>π</ci>
        <cn type="integer">3</cn>
       </apply>
      </apply>
      <apply>
       <divide></divide>
       <apply>
        <times></times>
        <apply>
         <csymbol cd="ambiguous">subscript</csymbol>
         <ci>π</ci>
         <cn type="integer">2</cn>
        </apply>
        <apply>
         <csymbol cd="ambiguous">subscript</csymbol>
         <ci>x</ci>
         <cn type="integer">4</cn>
        </apply>
       </apply>
       <apply>
        <csymbol cd="ambiguous">subscript</csymbol>
        <ci>π</ci>
        <cn type="integer">3</cn>
       </apply>
      </apply>
      <apply>
       <minus></minus>
       <apply>
        <plus></plus>
        <apply>
         <divide></divide>
         <apply>
          <times></times>
          <apply>
           <csymbol cd="ambiguous">subscript</csymbol>
           <ci>π</ci>
           <cn type="integer">1</cn>
          </apply>
          <apply>
           <csymbol cd="ambiguous">subscript</csymbol>
           <ci>x</ci>
           <cn type="integer">2</cn>
          </apply>
         </apply>
         <apply>
          <csymbol cd="ambiguous">subscript</csymbol>
          <ci>π</ci>
          <cn type="integer">3</cn>
         </apply>
        </apply>
        <apply>
         <divide></divide>
         <apply>
          <times></times>
          <apply>
           <csymbol cd="ambiguous">subscript</csymbol>
           <ci>π</ci>
           <cn type="integer">2</cn>
          </apply>
          <apply>
           <csymbol cd="ambiguous">subscript</csymbol>
           <ci>x</ci>
           <cn type="integer">4</cn>
          </apply>
         </apply>
         <apply>
          <csymbol cd="ambiguous">subscript</csymbol>
          <ci>π</ci>
          <cn type="integer">3</cn>
         </apply>
        </apply>
        <apply>
         <csymbol cd="ambiguous">subscript</csymbol>
         <ci>x</ci>
         <cn type="integer">6</cn>
        </apply>
       </apply>
      </apply>
      <apply>
       <csymbol cd="ambiguous">subscript</csymbol>
       <ci>x</ci>
       <cn type="integer">6</cn>
      </apply>
     </matrixrow>
     <matrixrow>
      <apply>
       <divide></divide>
       <apply>
        <times></times>
        <apply>
         <csymbol cd="ambiguous">subscript</csymbol>
         <ci>π</ci>
         <cn type="integer">1</cn>
        </apply>
        <apply>
         <csymbol cd="ambiguous">subscript</csymbol>
         <ci>x</ci>
         <cn type="integer">3</cn>
        </apply>
       </apply>
       <apply>
        <csymbol cd="ambiguous">subscript</csymbol>
        <ci>π</ci>
        <cn type="integer">4</cn>
       </apply>
      </apply>
      <apply>
       <divide></divide>
       <apply>
        <times></times>
        <apply>
         <csymbol cd="ambiguous">subscript</csymbol>
         <ci>π</ci>
         <cn type="integer">2</cn>
        </apply>
        <apply>
         <csymbol cd="ambiguous">subscript</csymbol>
         <ci>x</ci>
         <cn type="integer">5</cn>
        </apply>
       </apply>
       <apply>
        <csymbol cd="ambiguous">subscript</csymbol>
        <ci>π</ci>
        <cn type="integer">4</cn>
       </apply>
      </apply>
      <apply>
       <divide></divide>
       <apply>
        <times></times>
        <apply>
         <csymbol cd="ambiguous">subscript</csymbol>
         <ci>π</ci>
         <cn type="integer">3</cn>
        </apply>
        <apply>
         <csymbol cd="ambiguous">subscript</csymbol>
         <ci>x</ci>
         <cn type="integer">6</cn>
        </apply>
       </apply>
       <apply>
        <csymbol cd="ambiguous">subscript</csymbol>
        <ci>π</ci>
        <cn type="integer">4</cn>
       </apply>
      </apply>
      <apply>
       <minus></minus>
       <apply>
        <plus></plus>
        <apply>
         <divide></divide>
         <apply>
          <times></times>
          <apply>
           <csymbol cd="ambiguous">subscript</csymbol>
           <ci>π</ci>
           <cn type="integer">1</cn>
          </apply>
          <apply>
           <csymbol cd="ambiguous">subscript</csymbol>
           <ci>x</ci>
           <cn type="integer">3</cn>
          </apply>
         </apply>
         <apply>
          <csymbol cd="ambiguous">subscript</csymbol>
          <ci>π</ci>
          <cn type="integer">4</cn>
         </apply>
        </apply>
        <apply>
         <divide></divide>
         <apply>
          <times></times>
          <apply>
           <csymbol cd="ambiguous">subscript</csymbol>
           <ci>π</ci>
           <cn type="integer">2</cn>
          </apply>
          <apply>
           <csymbol cd="ambiguous">subscript</csymbol>
           <ci>x</ci>
           <cn type="integer">5</cn>
          </apply>
         </apply>
         <apply>
          <csymbol cd="ambiguous">subscript</csymbol>
          <ci>π</ci>
          <cn type="integer">4</cn>
         </apply>
        </apply>
        <apply>
         <divide></divide>
         <apply>
          <times></times>
          <apply>
           <csymbol cd="ambiguous">subscript</csymbol>
           <ci>π</ci>
           <cn type="integer">3</cn>
          </apply>
          <apply>
           <csymbol cd="ambiguous">subscript</csymbol>
           <ci>x</ci>
           <cn type="integer">6</cn>
          </apply>
         </apply>
         <apply>
          <csymbol cd="ambiguous">subscript</csymbol>
          <ci>π</ci>
          <cn type="integer">4</cn>
         </apply>
        </apply>
       </apply>
      </apply>
     </matrixrow>
    </matrix>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   Q=\begin{pmatrix}{-(x_{1}+x_{2}+x_{3})}&x_{1}&x_{2}&x_{3}\\
{\pi_{1}x_{1}\over\pi_{2}}&{-({\pi_{1}x_{1}\over\pi_{2}}+x_{4}+x_{5})}&x_{4}&x%
_{5}\\
{\pi_{1}x_{2}\over\pi_{3}}&{\pi_{2}x_{4}\over\pi_{3}}&{-({\pi_{1}x_{2}\over\pi%
_{3}}+{\pi_{2}x_{4}\over\pi_{3}}+x_{6})}&x_{6}\\
{\pi_{1}x_{3}\over\pi_{4}}&{\pi_{2}x_{5}\over\pi_{4}}&{\pi_{3}x_{6}\over\pi_{4%
}}&{-({\pi_{1}x_{3}\over\pi_{4}}+{\pi_{2}x_{5}\over\pi_{4}}+{\pi_{3}x_{6}\over%
\pi_{4}})}\end{pmatrix}
  </annotation>
 </semantics>
</math>

</p>

<p>Because the model must be time reversible and must approach the equilibrium nucleotide (base) frequencies at long times, each rate below the diagonal equals the reciprocal rate above the diagonal multiplied by the equilibrium ratio of the two bases. As such, the nucleotide GTR requires 6 substitution rate parameters and 4 equilibrium base frequency parameters. Since the 4 frequency parameters must sum to 1, there are only 3 free frequency parameters. The total of 9 free parameters is often further reduced to 8 parameters plus 

<math display="inline" id="Substitution_model:16">
 <semantics>
  <mi>μ</mi>
  <annotation-xml encoding="MathML-Content">
   <ci>μ</ci>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \mu
  </annotation>
 </semantics>
</math>

, the overall number of substitutions per unit time. When measuring time in substitutions (

<math display="inline" id="Substitution_model:17">
 <semantics>
  <mi>μ</mi>
  <annotation-xml encoding="MathML-Content">
   <ci>μ</ci>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \mu
  </annotation>
 </semantics>
</math>

=1) only 8 free parameters remain.</p>

<p>In general, to compute the number of parameters, you count the number of entries above the diagonal in the matrix, i.e. for n trait values per site 

<math display="inline" id="Substitution_model:18">
 <semantics>
  <mfrac>
   <mrow>
    <msup>
     <mi>n</mi>
     <mn>2</mn>
    </msup>
    <mo>-</mo>
    <mi>n</mi>
   </mrow>
   <mn>2</mn>
  </mfrac>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <divide></divide>
    <apply>
     <minus></minus>
     <apply>
      <csymbol cd="ambiguous">superscript</csymbol>
      <ci>n</ci>
      <cn type="integer">2</cn>
     </apply>
     <ci>n</ci>
    </apply>
    <cn type="integer">2</cn>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   {{n^{2}-n}\over 2}
  </annotation>
 </semantics>
</math>

, and then add <em>n-1</em> for the equilibrium frequencies, and subtract 1 because 

<math display="inline" id="Substitution_model:19">
 <semantics>
  <mi>μ</mi>
  <annotation-xml encoding="MathML-Content">
   <ci>μ</ci>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \mu
  </annotation>
 </semantics>
</math>

 is fixed. You get</p>

<p>

<math display="block" id="Substitution_model:20">
 <semantics>
  <mrow>
   <mrow>
    <mrow>
     <mfrac>
      <mrow>
       <msup>
        <mi>n</mi>
        <mn>2</mn>
       </msup>
       <mo>-</mo>
       <mi>n</mi>
      </mrow>
      <mn>2</mn>
     </mfrac>
     <mo>+</mo>
     <mrow>
      <mo stretchy="false">(</mo>
      <mrow>
       <mi>n</mi>
       <mo>-</mo>
       <mn>1</mn>
      </mrow>
      <mo stretchy="false">)</mo>
     </mrow>
    </mrow>
    <mo>-</mo>
    <mn>1</mn>
   </mrow>
   <mo>=</mo>
   <mrow>
    <mrow>
     <mrow>
      <mfrac>
       <mn>1</mn>
       <mn>2</mn>
      </mfrac>
      <msup>
       <mi>n</mi>
       <mn>2</mn>
      </msup>
     </mrow>
     <mo>+</mo>
     <mrow>
      <mfrac>
       <mn>1</mn>
       <mn>2</mn>
      </mfrac>
      <mi>n</mi>
     </mrow>
    </mrow>
    <mo>-</mo>
    <mn>2.</mn>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <eq></eq>
    <apply>
     <minus></minus>
     <apply>
      <plus></plus>
      <apply>
       <divide></divide>
       <apply>
        <minus></minus>
        <apply>
         <csymbol cd="ambiguous">superscript</csymbol>
         <ci>n</ci>
         <cn type="integer">2</cn>
        </apply>
        <ci>n</ci>
       </apply>
       <cn type="integer">2</cn>
      </apply>
      <apply>
       <minus></minus>
       <ci>n</ci>
       <cn type="integer">1</cn>
      </apply>
     </apply>
     <cn type="integer">1</cn>
    </apply>
    <apply>
     <minus></minus>
     <apply>
      <plus></plus>
      <apply>
       <times></times>
       <apply>
        <divide></divide>
        <cn type="integer">1</cn>
        <cn type="integer">2</cn>
       </apply>
       <apply>
        <csymbol cd="ambiguous">superscript</csymbol>
        <ci>n</ci>
        <cn type="integer">2</cn>
       </apply>
      </apply>
      <apply>
       <times></times>
       <apply>
        <divide></divide>
        <cn type="integer">1</cn>
        <cn type="integer">2</cn>
       </apply>
       <ci>n</ci>
      </apply>
     </apply>
     <cn type="float">2.</cn>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   {{n^{2}-n}\over 2}+(n-1)-1={1\over 2}n^{2}+{1\over 2}n-2.
  </annotation>
 </semantics>
</math>

</p>

<p>For example, for an amino acid sequence (there are 20 "standard" amino acids that make up <a class="uri" href="proteins" title="wikilink">proteins</a>), you would find there are 208 parameters. However, when studying coding regions of the genome, it is more common to work with a <a class="uri" href="codon" title="wikilink">codon</a> substitution model (a codon is three bases and codes for one amino acid in a protein). There are 

<math display="inline" id="Substitution_model:21">
 <semantics>
  <mrow>
   <msup>
    <mn>4</mn>
    <mn>3</mn>
   </msup>
   <mo>=</mo>
   <mn>64</mn>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <eq></eq>
    <apply>
     <csymbol cd="ambiguous">superscript</csymbol>
     <cn type="integer">4</cn>
     <cn type="integer">3</cn>
    </apply>
    <cn type="integer">64</cn>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   4^{3}=64
  </annotation>
 </semantics>
</math>

 codons, resulting in 2078 free parameters, but when the rates for transitions between codons which differ by more than one base are assumed to be zero, then there are only 

<math display="inline" id="Substitution_model:22">
 <semantics>
  <mrow>
   <mrow>
    <mrow>
     <mfrac>
      <mrow>
       <mn>20</mn>
       <mo>×</mo>
       <mn>19</mn>
       <mo>×</mo>
       <mn>3</mn>
      </mrow>
      <mn>2</mn>
     </mfrac>
     <mo>+</mo>
     <mn>63</mn>
    </mrow>
    <mo>-</mo>
    <mn>1</mn>
   </mrow>
   <mo>=</mo>
   <mn>632</mn>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <eq></eq>
    <apply>
     <minus></minus>
     <apply>
      <plus></plus>
      <apply>
       <divide></divide>
       <apply>
        <times></times>
        <cn type="integer">20</cn>
        <cn type="integer">19</cn>
        <cn type="integer">3</cn>
       </apply>
       <cn type="integer">2</cn>
      </apply>
      <cn type="integer">63</cn>
     </apply>
     <cn type="integer">1</cn>
    </apply>
    <cn type="integer">632</cn>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   {{20\times 19\times 3}\over 2}+63-1=632
  </annotation>
 </semantics>
</math>

 parameters.</p>
<h2 id="mechanistic-vs.-empirical-models">Mechanistic vs. empirical models</h2>

<p>A main difference in evolutionary models is how many parameters are estimated every time for the data set under consideration and how many of them are estimated once on a large data set. Mechanistic models describe all substitutions as a function of a number of parameters which are estimated for every data set analyzed, preferably using <a href="maximum_likelihood" title="wikilink">maximum likelihood</a>. This has the advantage that the model can be adjusted to the particularities of a specific data set (e.g. different composition biases in DNA). Problems can arise when too many parameters are used, particularly if they can compensate for each other. Then it is often the case that the data set is too small to yield enough information to estimate all parameters accurately.</p>

<p>Empirical models are created by estimating many parameters (typically all entries of the rate matrix and the character frequencies, see the GTR model above) from a large data set. These parameters are then fixed and will be reused for every data set. This has the advantage that those parameters can be estimated more accurately. Normally, it is not possible to estimate all entries of the substitution matrix from the current data set only. On the downside, the estimated parameters might be too generic and do not fit a particular data set well enough.</p>

<p>With the large-scale genome sequencing still producing very large amounts of DNA and protein sequences, there is enough data available to create empirical models with any number of parameters. Because of the problems mentioned above, the two approaches are often combined, by estimating most of the parameters once on large-scale data, while a few remaining parameters are then adjusted to the data set under consideration. The following sections give an overview of the different approaches taken for DNA, protein or codon-based models.</p>
<h2 id="models-of-dna-substitution">Models of DNA substitution</h2>

<p>See main article: <strong><a href="Models_of_DNA_evolution" title="wikilink">Models of DNA evolution</a></strong> for more formal descriptions of the DNA models.</p>

<p>Models of DNA evolution were first proposed in 1969 by Jukes and Cantor,<a class="footnoteRef" href="#fn8" id="fnref8"><sup>8</sup></a> assuming equal transition rates as well as equal equilibrium frequencies for all bases. In 1980 <a href="Motoo_Kimura" title="wikilink">Kimura</a><a class="footnoteRef" href="#fn9" id="fnref9"><sup>9</sup></a> introduced a model with two parameters: one for the transition and one for the transversion rate and in 1981, Felsenstein<a class="footnoteRef" href="#fn10" id="fnref10"><sup>10</sup></a> proposed a four-parameter model in which the substitution rate corresponds to the equilibrium frequency of the target nucleotide. Hasegawa, Kishino and Yano (HKY)<a class="footnoteRef" href="#fn11" id="fnref11"><sup>11</sup></a> unified the two last models to a five-parameter model. In the 1990s, models similar to HKY were developed and refined by several researchers.<a class="footnoteRef" href="#fn12" id="fnref12"><sup>12</sup></a><a class="footnoteRef" href="#fn13" id="fnref13"><sup>13</sup></a><a class="footnoteRef" href="#fn14" id="fnref14"><sup>14</sup></a></p>

<p>For DNA substitution models, mainly mechanistic models (as described above) are employed. The small number of parameters to estimate makes this feasible, but also DNA is often highly optimized for specific purposes (e.g. fast <a href="Gene_expression" title="wikilink">expression</a> or stability) depending on the organism and the type of gene, making it necessary to adjust the model to these circumstances.</p>
<h2 id="models-of-amino-acid-substitutions">Models of amino acid substitutions</h2>

<p>For many analyses, particularly for longer evolutionary distances, the evolution is modeled on the amino acid level. Since not all DNA substitution also alter the encoded amino acid, information is lost when looking at amino acids instead of nucleotide bases. However, several advantages speak in favor of using the amino acid information: DNA is much more inclined to show <a href="compositional_bias" title="wikilink">compositional bias</a> than amino acids, not all positions in the DNA evolve at the same speed (<a href="Missense_mutation" title="wikilink">non-synonymous</a> mutations are more likely to become fixed in the population than synonymous ones), but probably most important, because of those fast evolving positions and the limited alphabet size (only four possible states), the DNA suffers much more from back substitutions, making it difficult to accurately estimate longer distances.</p>

<p>Unlike the DNA models, amino acid models traditionally are empirical models. They were pioneered in the 1970s by Dayhoff and co-workers ,<a class="footnoteRef" href="#fn15" id="fnref15"><sup>15</sup></a> by estimating replacement rates from protein alignments with at least 85% identity. This minimized the chances of observing multiple substitutions at a site. From the estimated rate matrix, a series of replacement probability matrices were derived, known under names such as <a href="Point_accepted_mutation" title="wikilink">PAM</a>250. The Dayhoff model was used to assess the significance of homology search results, but also for phylogenetic analyses. The Dayhoff PAM matrices were based on relatively few alignments (since not more were available at that time), but in the 1990s, new matrices were estimated using almost the same methodology, but based on the large protein databases available then (,<a class="footnoteRef" href="#fn16" id="fnref16"><sup>16</sup></a><a class="footnoteRef" href="#fn17" id="fnref17"><sup>17</sup></a> the latter being known as "JTT" matrices).</p>
<h2 id="references">References</h2>

<p>"</p>

<p><a class="uri" href="Category:Bioinformatics" title="wikilink">Category:Bioinformatics</a> <a href="Category:Stochastic_models" title="wikilink">Category:Stochastic models</a> <a href="Category:Computational_phylogenetics" title="wikilink">Category:Computational phylogenetics</a> <a href="Category:Statistical_genetics" title="wikilink">Category:Statistical genetics</a></p>
<section class="footnotes">
<hr/>
<ol>
<li id="fn1"><a href="#fnref1">↩</a></li>
<li id="fn2"><a href="#fnref2">↩</a></li>
<li id="fn3"><a href="#fnref3">↩</a></li>
<li id="fn4"><a href="#fnref4">↩</a></li>
<li id="fn5"><a href="#fnref5">↩</a></li>
<li id="fn6"><a href="#fnref6">↩</a></li>
<li id="fn7"><a href="#fnref7">↩</a></li>
<li id="fn8"><a href="#fnref8">↩</a></li>
<li id="fn9"><a href="#fnref9">↩</a></li>
<li id="fn10"><a href="#fnref10">↩</a></li>
<li id="fn11"><a href="#fnref11">↩</a></li>
<li id="fn12"><a href="#fnref12">↩</a></li>
<li id="fn13"><a href="#fnref13">↩</a></li>
<li id="fn14"><a href="#fnref14">↩</a></li>
<li id="fn15"><a href="#fnref15">↩</a></li>
<li id="fn16"><a href="#fnref16">↩</a></li>
<li id="fn17"><a href="#fnref17">↩</a></li>
</ol>
</section>
</hr></body>
</html>
