<html lang="en">
<head>
<meta charset="utf-8"/>
<title offset="1589">Transferase</title>
<script src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_SVG.js" type="text/javascript">
</script>
</head>
<body>
<h1>Transferase</h1>
<hr/>
<figure><b>(Figure)</b>
<figcaption>RNA Polymerase from <em><a href="Saccharomyces_cerevisiae" title="wikilink">Saccharomyces cerevisiae</a></em> complexed with α-amanitin (in red). Despite the use of the term "polymerase," RNA Polymerases are classified as a form of nucleotidyl transferase.<a class="footnoteRef" href="#fn1" id="fnref1"><sup>1</sup></a></figcaption>
</figure>

<p>In <a class="uri" href="biochemistry" title="wikilink">biochemistry</a>, <strong>transferase</strong> is the general name for the class of enzymes that enact the transfer of specific <a href="functional_group" title="wikilink">functional groups</a> (e.g. a methyl or glycosyl group) from one <a class="uri" href="molecule" title="wikilink">molecule</a> (called the donor) to another (called the acceptor).<a class="footnoteRef" href="#fn2" id="fnref2"><sup>2</sup></a> They are involved in hundreds of different <a href="Molecular_pathway" title="wikilink">biochemical pathways</a> throughout <a class="uri" href="biology" title="wikilink">biology</a>, and are integral to some of life’s most important processes.</p>

<p>Transferases are involved in myriad reactions in the cell. Some examples of these reactions include the activity of <a href="Coenzyme_A" title="wikilink">CoA</a> transferase, which transfers thiol esters,<a class="footnoteRef" href="#fn3" id="fnref3"><sup>3</sup></a> the action of <a class="uri" href="N-acetyltransferase" title="wikilink">N-acetyltransferase</a> is part of the pathway that metabolizes tryptophan,<a class="footnoteRef" href="#fn4" id="fnref4"><sup>4</sup></a> and also includes the regulation of <a href="pyruvate_dehydrogenase" title="wikilink">PDH</a>, which converts <a class="uri" href="pyruvate" title="wikilink">pyruvate</a> to <a href="Acetyl_CoA" title="wikilink">Acetyl CoA</a>.<a class="footnoteRef" href="#fn5" id="fnref5"><sup>5</sup></a> Transferases are also utilized during translation. In this case, an amino acid chain is the functional group transferred by a <a href="Peptidyl_transferase" title="wikilink">Peptidyl transferase</a>. The transfer involves the removal of the growing <a href="amino_acid" title="wikilink">amino acid</a> chain from the <a href="transfer_RNA" title="wikilink">tRNA</a> molecule in the <a class="uri" href="A-site" title="wikilink">A-site</a> of the <a class="uri" href="ribosome" title="wikilink">ribosome</a> and its subsequent addition to the amino acid attached to the tRNA in the <a class="uri" href="P-site" title="wikilink">P-site</a>.<a class="footnoteRef" href="#fn6" id="fnref6"><sup>6</sup></a></p>

<p>Mechanistically, an enzyme that catalyzed the following reaction would be a transferase:</p>

<p>

<math display="inline" id="Transferase:0">
 <semantics>
  <mrow>
   <mrow>
    <mrow>
     <mi>X</mi>
     <mi>g</mi>
     <mi>r</mi>
     <mi>o</mi>
     <mi>u</mi>
     <mi>p</mi>
    </mrow>
    <mo>+</mo>
    <mi>Y</mi>
   </mrow>
   <munderover accent="true" accentunder="true">
    <mo>→</mo>
    <mrow>
     <mi>t</mi>
     <mi>r</mi>
     <mi>a</mi>
     <mi>n</mi>
     <mi>s</mi>
     <mi>f</mi>
     <mi>e</mi>
     <mi>r</mi>
     <mi>a</mi>
     <mi>s</mi>
     <mi>e</mi>
    </mrow>
    <mi></mi>
   </munderover>
   <mrow>
    <mi>X</mi>
    <mo>+</mo>
    <mrow>
     <mi>Y</mi>
     <mi>g</mi>
     <mi>r</mi>
     <mi>o</mi>
     <mi>u</mi>
     <mi>p</mi>
    </mrow>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <apply>
     <apply>
      <times></times>
      <ci>t</ci>
      <ci>r</ci>
      <ci>a</ci>
      <ci>n</ci>
      <ci>s</ci>
      <ci>f</ci>
      <ci>e</ci>
      <ci>r</ci>
      <ci>a</ci>
      <ci>s</ci>
      <ci>e</ci>
     </apply>
     <apply>
      <csymbol cd="latexml">absent</csymbol>
      <ci>normal-→</ci>
     </apply>
    </apply>
    <apply>
     <plus></plus>
     <apply>
      <times></times>
      <ci>X</ci>
      <ci>g</ci>
      <ci>r</ci>
      <ci>o</ci>
      <ci>u</ci>
      <ci>p</ci>
     </apply>
     <ci>Y</ci>
    </apply>
    <apply>
     <plus></plus>
     <ci>X</ci>
     <apply>
      <times></times>
      <ci>Y</ci>
      <ci>g</ci>
      <ci>r</ci>
      <ci>o</ci>
      <ci>u</ci>
      <ci>p</ci>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   Xgroup+Y\xrightarrow[transferase]{}X+Ygroup
  </annotation>
 </semantics>
</math>

</p>

<p>In the above reaction, X would be the donor, and Y would be the acceptor.<a class="footnoteRef" href="#fn7" id="fnref7"><sup>7</sup></a> "Group" would be the functional group transferred as a result of transferase activity. The donor is often a <a class="uri" href="coenzyme" title="wikilink">coenzyme</a>.</p>
<h2 id="history">History</h2>

<p>Some of the most important discoveries relating to transferases occurred as early as the 1930s. Earliest discoveries of transferase activity occurred in other classifications of <a href="enzyme" title="wikilink">enzymes</a>, including <a class="uri" href="Beta-galactosidase" title="wikilink">Beta-galactosidase</a>, <a class="uri" href="protease" title="wikilink">protease</a>, and acid/base <a class="uri" href="phosphatase" title="wikilink">phosphatase</a>. Prior to the realization that individual enzymes were capable of such a task, it was believed that two or more enzymes enacted functional group transfers.<a class="footnoteRef" href="#fn8" id="fnref8"><sup>8</sup></a> </p>

<p><a class="uri" href="Transamination" title="wikilink">Transamination</a>, or the transfer of an <a class="uri" href="amine" title="wikilink">amine</a> (or NH<sub>2</sub>) group from an amino acid to a <a href="keto_acid" title="wikilink">keto acid</a> by an <a class="uri" href="aminotransferase" title="wikilink">aminotransferase</a> (also known as a "transaminase"), was first noted in 1930 by D. M. Needham, after observing the disappearance of <a href="glutamic_acid" title="wikilink">glutamic acid</a> added to pigeon breast muscle.<a class="footnoteRef" href="#fn9" id="fnref9"><sup>9</sup></a> This observance was later verified by the discovery of its reaction mechanism by Braunstein and Kritzmann in 1937.<a class="footnoteRef" href="#fn10" id="fnref10"><sup>10</sup></a> Their analysis showed that this reversible reaction could be applied to other tissues.<a class="footnoteRef" href="#fn11" id="fnref11"><sup>11</sup></a> This assertion was validated by <a href="Rudolf_Schoenheimer" title="wikilink">Rudolf Schoenheimer</a>'s work with <a href="radioisotope" title="wikilink">radioisotopes</a> as <a href="Isotopic_labelling" title="wikilink">tracers</a> in 1937.<a class="footnoteRef" href="#fn12" id="fnref12"><sup>12</sup></a><a class="footnoteRef" href="#fn13" id="fnref13"><sup>13</sup></a> This in turn would pave the way for the possibility that similar transfers were a primary means of producing most amino acids via amino transfer.<a class="footnoteRef" href="#fn14" id="fnref14"><sup>14</sup></a></p>

<p>Another such example of early transferase research and later reclassification involved the discovery of uridyl transferase. In 1953, the enzyme <a href="UDP-glucose_pyrophosphorylase" title="wikilink">UDP-glucose pyrophosphorylase</a> was shown to be a transferase, when it was found that it could reversibly produce <a href="uridine_triphosphate" title="wikilink">UTP</a> and <a href="glucose_1-phosphate" title="wikilink">G1P</a> from <a href="Uridine_diphosphate_glucose" title="wikilink">UDP-glucose</a> and an organic <a class="uri" href="pyrophosphate" title="wikilink">pyrophosphate</a>.<a class="footnoteRef" href="#fn15" id="fnref15"><sup>15</sup></a></p>

<p>Another example of historical significance relating to transferase is the discovery of the mechanism of <a class="uri" href="catecholamine" title="wikilink">catecholamine</a> breakdown by <a class="uri" href="catechol-O-methyltransferase" title="wikilink">catechol-O-methyltransferase</a>. This discovery was a large part of the reason for <a href="Julius_Axelrod" title="wikilink">Julius Axelrod’s</a> 1970 <a href="Nobel_Prize_in_Physiology_or_Medicine" title="wikilink">Nobel Prize in Physiology or Medicine</a> (shared with <a href="Bernard_Katz" title="wikilink">Sir Bernard Katz</a> and <a href="Ulf_von_Euler" title="wikilink">Ulf von Euler</a>).<a class="footnoteRef" href="#fn16" id="fnref16"><sup>16</sup></a></p>

<p>Classification of transferases continues to this day, with new ones being discovered frequently.<a class="footnoteRef" href="#fn17" id="fnref17"><sup>17</sup></a><a class="footnoteRef" href="#fn18" id="fnref18"><sup>18</sup></a> An example of this is Pipe, a sulfotransferase involved in the dorsal-ventral patterning of <em><a class="uri" href="Drosophilia" title="wikilink">Drosophilia</a></em>.<a class="footnoteRef" href="#fn19" id="fnref19"><sup>19</sup></a> Initially, the exact mechanism of Pipe was unknown, due to a lack of information on its substrate.<a class="footnoteRef" href="#fn20" id="fnref20"><sup>20</sup></a> Research into Pipe's catalytic activity eliminated the likelihood of it being a heparan sulfate glycosaminoglycan.<a class="footnoteRef" href="#fn21" id="fnref21"><sup>21</sup></a> Further research has shown that Pipe targets the ovarian structures for sulfation.<a class="footnoteRef" href="#fn22" id="fnref22"><sup>22</sup></a> Pipe is currently classified as a <em>Drosophilia</em> <a href="heparan_sulfate_2-O-sulfotransferase" title="wikilink">heparan sulfate 2-O-sulfotransferase</a>.<a class="footnoteRef" href="#fn23" id="fnref23"><sup>23</sup></a></p>
<h2 id="nomenclature">Nomenclature</h2>

<p><a href="Systematic_name" title="wikilink">Systematic names</a> of transferases are constructed in the form of "donor:acceptor grouptransferase."<a class="footnoteRef" href="#fn24" id="fnref24"><sup>24</sup></a> For example, methylamine:L-glutamate N-methyltransferase would be the standard naming convention for the transferase <a href="methylamine-glutamate_N-methyltransferase" title="wikilink">methylamine-glutamate N-methyltransferase</a>, where <a class="uri" href="methylamine" title="wikilink">methylamine</a> is the donor, <a class="uri" href="L-glutamate" title="wikilink">L-glutamate</a> is the acceptor, and <a class="uri" href="methyltransferase" title="wikilink">methyltransferase</a> is the EC category grouping. This same action by the transferase can be illustrated as follows:</p>
<dl>
<dd>methylamine + L-glutamate 

<math display="inline" id="Transferase:1">
 <semantics>
  <mo>⇌</mo>
  <annotation-xml encoding="MathML-Content">
   <ci>normal-⇌</ci>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \rightleftharpoons
  </annotation>
 </semantics>
</math>

 [[Ammonia|NH<sub>3]]</sub> + <a href="gamma-Glutamylmethylamide" title="wikilink">N-methyl-L-glutamate</a><a class="footnoteRef" href="#fn25" id="fnref25"><sup>25</sup></a>
</dd>
</dl>

<p>However, other accepted names are more frequently used for transferases, and are often formed as "acceptor grouptransferase" or "donor grouptransferase." For example, a <a href="DNA_methyltransferase" title="wikilink">DNA methyltransferase</a> is a transferase that catalyzes the transfer of a <a class="uri" href="methyl" title="wikilink">methyl</a> group to a <a class="uri" href="DNA" title="wikilink">DNA</a> acceptor. In practice, many molecules are not referred to using this terminology due to more prevalent common names.<a class="footnoteRef" href="#fn26" id="fnref26"><sup>26</sup></a> For example, <a href="RNA_Polymerase" title="wikilink">RNA Polymerase</a> is the modern common name for what was formerly known as RNA nucleotidyltransferase, a kind of <a href="nucleotidyl_transferase" title="wikilink">nucleotidyl transferase</a> that transfers <a class="uri" href="nucleotides" title="wikilink">nucleotides</a> to the 3’ end of a growing <a class="uri" href="RNA" title="wikilink">RNA</a> strand.<a class="footnoteRef" href="#fn27" id="fnref27"><sup>27</sup></a> In the EC system of classification, the accepted name for RNA Polymerase is DNA-directed RNA polymerase.<a class="footnoteRef" href="#fn28" id="fnref28"><sup>28</sup></a></p>
<h2 id="classification">Classification</h2>

<p>Described primarily based on the type of biochemical group transferred, transferases can be divided into ten categories (based on the <a href="Enzyme_Commission_number" title="wikilink">EC Number</a> classification).<a class="footnoteRef" href="#fn29" id="fnref29"><sup>29</sup></a> These categories comprise over 450 different unique enzymes.<a class="footnoteRef" href="#fn30" id="fnref30"><sup>30</sup></a> In the EC numbering system, transferases have been given a classification of <strong>EC2</strong>. It is important to note, that <a class="uri" href="hydrogen" title="wikilink">hydrogen</a> is not considered a functional group when it comes to transferase targets; instead, hydrogen transfer is included under <a href="oxidoreductase" title="wikilink">oxidoreductases</a>, due to electron transfer considerations.</p>
<table>
<tbody>
<tr class="odd">
<td style="text-align: left;">
<p>Classification of transferases into subclasses:</p></td>
<td style="text-align: left;">
<p><strong>EC number</strong></p></td>
<td style="text-align: left;">
<p><strong>Examples</strong></p></td>
<td style="text-align: left;">
<p><strong>Group(s) transferred</strong></p></td>
</tr>
<tr class="even">
<td style="text-align: left;">
<p><strong><a href=":Category:EC_2.1" title="wikilink">EC 2.1</a></strong></p></td>
<td style="text-align: left;">
<p><a class="uri" href="methyltransferase" title="wikilink">methyltransferase</a> and <a class="uri" href="formyltransferase" title="wikilink">formyltransferase</a></p></td>
<td style="text-align: left;">
<p>single-<a class="uri" href="carbon" title="wikilink">carbon</a> groups</p></td>
</tr>
<tr class="odd">
<td style="text-align: left;">
<p><strong><a href=":Category:EC_2.2" title="wikilink">EC 2.2</a></strong></p></td>
<td style="text-align: left;">
<p><a class="uri" href="transketolase" title="wikilink">transketolase</a> and <a class="uri" href="transaldolase" title="wikilink">transaldolase</a></p></td>
<td style="text-align: left;">
<p><a class="uri" href="aldehyde" title="wikilink">aldehyde</a> or <a class="uri" href="ketone" title="wikilink">ketone</a> groups</p></td>
</tr>
<tr class="even">
<td style="text-align: left;">
<p><strong><a href=":Category:EC_2.3" title="wikilink">EC 2.3</a></strong></p></td>
<td style="text-align: left;">
<p><a class="uri" href="acyltransferase" title="wikilink">acyltransferase</a></p></td>
<td style="text-align: left;">
<p><a class="uri" href="acyl" title="wikilink">acyl</a> groups or groups that become <a class="uri" href="alkyl" title="wikilink">alkyl</a> groups during transfer</p></td>
</tr>
<tr class="odd">
<td style="text-align: left;">
<p><strong><a href=":Category:EC_2.4" title="wikilink">EC 2.4</a></strong></p></td>
<td style="text-align: left;">
<p><a class="uri" href="glycosyltransferase" title="wikilink">glycosyltransferase</a>, <a class="uri" href="hexosyltransferase" title="wikilink">hexosyltransferase</a>, and <a class="uri" href="pentosyltransferase" title="wikilink">pentosyltransferase</a></p></td>
<td style="text-align: left;">
<p><a class="uri" href="glycosyl" title="wikilink">glycosyl</a> groups, as well as <a class="uri" href="hexoses" title="wikilink">hexoses</a> and <a href="pentose" title="wikilink">pentoses</a></p></td>
</tr>
<tr class="even">
<td style="text-align: left;">
<p><strong><a href=":Category:EC_2.5" title="wikilink">EC 2.5</a></strong></p></td>
<td style="text-align: left;">
<p><a href="riboflavin_synthase" title="wikilink">riboflavin synthase</a> and <a href="chlorophyll_synthase" title="wikilink">chlorophyll synthase</a></p></td>
<td style="text-align: left;">
<p><a class="uri" href="alkyl" title="wikilink">alkyl</a> or <a class="uri" href="aryl" title="wikilink">aryl</a> groups, other than methyl groups</p></td>
</tr>
<tr class="odd">
<td style="text-align: left;">
<p><strong><a href=":Category:EC_2.6" title="wikilink">EC 2.6</a></strong></p></td>
<td style="text-align: left;">
<p><a class="uri" href="transaminase" title="wikilink">transaminase</a>, and <a class="uri" href="oximinotransferase" title="wikilink">oximinotransferase</a></p></td>
<td style="text-align: left;">
<p><a href="nitrogen" title="wikilink">nitrogenous</a> groups</p></td>
</tr>
<tr class="even">
<td style="text-align: left;">
<p><strong><a href=":Category:EC_2.7" title="wikilink">EC 2.7</a></strong></p></td>
<td style="text-align: left;">
<p><a class="uri" href="phosphotransferase" title="wikilink">phosphotransferase</a>, <a class="uri" href="polymerase" title="wikilink">polymerase</a>, and <a class="uri" href="kinase" title="wikilink">kinase</a></p></td>
<td style="text-align: left;">
<p><a class="uri" href="phosphorus" title="wikilink">phosphorus</a>-containing groups; subclasses are based on the acceptor (e.g. <a class="uri" href="alcohol" title="wikilink">alcohol</a>, <a class="uri" href="carboxyl" title="wikilink">carboxyl</a>, etc.)</p></td>
</tr>
<tr class="odd">
<td style="text-align: left;">
<p><strong><a href=":Category:EC_2.8" title="wikilink">EC 2.8</a></strong></p></td>
<td style="text-align: left;">
<p><a class="uri" href="sulfurtransferase" title="wikilink">sulfurtransferase</a> and <a class="uri" href="sulfotransferase" title="wikilink">sulfotransferase</a></p></td>
<td style="text-align: left;">
<p><a class="uri" href="sulfur" title="wikilink">sulfur</a>-containing groups</p></td>
</tr>
<tr class="even">
<td style="text-align: left;">
<p><strong><a href=":Category:EC_2.9" title="wikilink">EC 2.9</a></strong></p></td>
<td style="text-align: left;">
<p><a class="uri" href="selenotransferase" title="wikilink">selenotransferase</a></p></td>
<td style="text-align: left;">
<p><a class="uri" href="selenium" title="wikilink">selenium</a>-containing groups</p></td>
</tr>
<tr class="odd">
<td style="text-align: left;">
<p><strong><a href=":Category:EC_2.10" title="wikilink">EC 2.10</a></strong></p></td>
<td style="text-align: left;">
<p><a class="uri" href="molybdenumtransferase" title="wikilink">molybdenumtransferase</a> and <a class="uri" href="tungstentransferase" title="wikilink">tungstentransferase</a></p></td>
<td style="text-align: left;">
<p><a class="uri" href="molybdenum" title="wikilink">molybdenum</a> or <a class="uri" href="tungsten" title="wikilink">tungsten</a></p></td>
</tr>
</tbody>
</table>
<h2 id="reactions">Reactions</h2>
<h3 id="ec-2.1-single-carbon-transferases">EC 2.1: single carbon transferases</h3>

<p> EC 2.1 includes enzymes that transfer single-carbon groups. This category consists of transfers of <a class="uri" href="methyl" title="wikilink">methyl</a>, hydroxymethyl, formyl, carboxy, carbamoyl, and amido groups.<a class="footnoteRef" href="#fn31" id="fnref31"><sup>31</sup></a> Carbamoyltransferases, as an example, transfer a carbamoyl group from one molecule to another.<a class="footnoteRef" href="#fn32" id="fnref32"><sup>32</sup></a> Carbamoyl groups follow the formula NH<sub>2</sub>CO.<a class="footnoteRef" href="#fn33" id="fnref33"><sup>33</sup></a> In <a href="Aspartate_carbamoyltransferase" title="wikilink">ATCase</a> such a transfer is written as Carbamyl phosphate + L-aspertate 

<math display="inline" id="Transferase:2">
 <semantics>
  <mo>→</mo>
  <annotation-xml encoding="MathML-Content">
   <ci>normal-→</ci>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \rightarrow
  </annotation>
 </semantics>
</math>

 L-carbamyl aspartate + <a class="uri" href="phosphate" title="wikilink">phosphate</a>.<a class="footnoteRef" href="#fn34" id="fnref34"><sup>34</sup></a></p>
<h3 id="ec-2.2-aldehyde-and-ketone-transferases">EC 2.2: aldehyde and ketone transferases</h3>

<p> Enzymes that transfer aldehyde or ketone groups and included in EC 2.2. This category consists of various transketolases and transaldolases.<a class="footnoteRef" href="#fn35" id="fnref35"><sup>35</sup></a> Transaldolase, the namesake of aldehyde transferases, is an important part of the pentose phosphate pathway.<a class="footnoteRef" href="#fn36" id="fnref36"><sup>36</sup></a> The reaction it catalyzes consists of a transfer of a dihydroxyacetone functional group to <a href="Glyceraldehyde_3-phosphate" title="wikilink">Glyceraldehyde 3-phosphate</a> (also known as G3P). The reaction is as follows: <a href="sedoheptulose_7-phosphate" title="wikilink">sedoheptulose 7-phosphate</a> + glyceraldehyde 3-phosphate 

<math display="inline" id="Transferase:3">
 <semantics>
  <mo>⇌</mo>
  <annotation-xml encoding="MathML-Content">
   <ci>normal-⇌</ci>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \rightleftharpoons
  </annotation>
 </semantics>
</math>

 <a href="erythrose_4-phosphate" title="wikilink">erythrose 4-phosphate</a> + <a href="fructose_6-phosphate" title="wikilink">fructose 6-phosphate</a>.<a class="footnoteRef" href="#fn37" id="fnref37"><sup>37</sup></a></p>
<h3 id="ec-2.3-acyl-transferases">EC 2.3: acyl transferases</h3>

<p>Transfer of acyl groups or acyl groups that become alkyl groups during the process of being transferred are key aspects of EC 2.3. Further, this category also differentiates between amino-acyl and non-amino-acyl groups. <a href="Peptidyl_transferase" title="wikilink">Peptidyl transferase</a> is a <a class="uri" href="ribozyme" title="wikilink">ribozyme</a> that facilitates formation of <a href="peptide_bonds" title="wikilink">peptide bonds</a> during <a href="Translation_(biology)" title="wikilink">translation</a>.<a class="footnoteRef" href="#fn38" id="fnref38"><sup>38</sup></a> As an aminoacyltransferase, it catalyzes the transfer of a peptide to an <a class="uri" href="aminoacyl-tRNA" title="wikilink">aminoacyl-tRNA</a>, following this reaction: peptidyl-tRNA<sub>A</sub> + aminoacyl-tRNA<sub>B</sub> 

<math display="inline" id="Transferase:4">
 <semantics>
  <mo>⇌</mo>
  <annotation-xml encoding="MathML-Content">
   <ci>normal-⇌</ci>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \rightleftharpoons
  </annotation>
 </semantics>
</math>

 tRNA<sub>A</sub> + peptidyl aminoacyl-tRNA<sub>B</sub>.<a class="footnoteRef" href="#fn39" id="fnref39"><sup>39</sup></a></p>
<h3 id="ec-2.4-glycosyl-hexosyl-and-pentosyl-transferases">EC 2.4: glycosyl, hexosyl, and pentosyl transferases</h3>

<p>EC 2.4 includes enzymes that transfer <a class="uri" href="glycosyl" title="wikilink">glycosyl</a> groups, as well as those that transfer hexose and pentose. <a class="uri" href="Glycosyltransferase" title="wikilink">Glycosyltransferase</a> is a subcategory of EC 2.4 transferases that is involved in <a class="uri" href="biosynthesis" title="wikilink">biosynthesis</a> of <a class="uri" href="disaccharides" title="wikilink">disaccharides</a> and <a class="uri" href="polysaccharides" title="wikilink">polysaccharides</a> through transfer of <a class="uri" href="monosaccharides" title="wikilink">monosaccharides</a> to other molecules.<a class="footnoteRef" href="#fn40" id="fnref40"><sup>40</sup></a> An example of a prominent glycosyltransferase is <a href="lactose_synthase" title="wikilink">lactose synthase</a> which is a dimer possessing two <a href="protein_subunit" title="wikilink">protein subunits</a>. Its primary action is to produce <a class="uri" href="lactose" title="wikilink">lactose</a> from <a class="uri" href="glucose" title="wikilink">glucose</a> and UDP-galactose.<a class="footnoteRef" href="#fn41" id="fnref41"><sup>41</sup></a> This occurs via the following pathway: UDP-β-D-galactose + D-glucose 

<math display="inline" id="Transferase:5">
 <semantics>
  <mo>⇌</mo>
  <annotation-xml encoding="MathML-Content">
   <ci>normal-⇌</ci>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \rightleftharpoons
  </annotation>
 </semantics>
</math>

 <a href="Uridine_diphosphate" title="wikilink">UDP</a> + lactose.<a class="footnoteRef" href="#fn42" id="fnref42"><sup>42</sup></a></p>
<h3 id="ec-2.5-alkyl-and-aryl-transferases">EC 2.5: alkyl and aryl transferases</h3>

<p>EC 2.5 relates to enzymes that transfer alkyl or aryl groups, but does not include methyl groups. This is in contrast to functional groups that become alkyl groups when transferred, as those are included in EC 2.3. EC 2.5 currently only possesses one sub-class: Alkyl and aryl transferases.<a class="footnoteRef" href="#fn43" id="fnref43"><sup>43</sup></a> <a href="Cysteine_synthase" title="wikilink">Cysteine synthase</a>, for example, catalyzes the formation of acetic acids and <a class="uri" href="cysteine" title="wikilink">cysteine</a> from O<sub>3</sub>-acetyl-L-serine and hydrogen sulfide: O<sub>3</sub>-acetyl-L-serine + H<sub>2</sub>S 

<math display="inline" id="Transferase:6">
 <semantics>
  <mo>⇌</mo>
  <annotation-xml encoding="MathML-Content">
   <ci>normal-⇌</ci>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \rightleftharpoons
  </annotation>
 </semantics>
</math>

 L-cysteine + acetate.<a class="footnoteRef" href="#fn44" id="fnref44"><sup>44</sup></a></p>
<h3 id="ec-2.6-nitrogenous-transferases">EC 2.6: nitrogenous transferases</h3>

<p><a href="image:Aspartate_aminotransferase_reaction.png" title="wikilink">thumb|450px|Aspartate aminotransferase can act on several different amino acids</a> The grouping consistent with transfer of <a class="uri" href="nitrogenous" title="wikilink">nitrogenous</a> groups is EC 2.6. This includes enzymes like <a class="uri" href="transaminase" title="wikilink">transaminase</a> (also known as "aminotransferase"), and a very small number of <a href="oximinotransferase" title="wikilink">oximinotransferases</a> and other nitrogen group transferring enzymes. EC 2.6 previously included <a class="uri" href="amidinotransferase" title="wikilink">amidinotransferase</a> but it has since been reclassified as a subcategory of EC 2.1 (single-carbon transferring enzymes).<a class="footnoteRef" href="#fn45" id="fnref45"><sup>45</sup></a> In the case of <a href="aspartate_transaminase" title="wikilink">aspartate transaminase</a>, which can act on <a class="uri" href="tyrosine" title="wikilink">tyrosine</a>, <a class="uri" href="phenylalanine" title="wikilink">phenylalanine</a>, and <a class="uri" href="tryptophan" title="wikilink">tryptophan</a>, it reversibly transfers an <a class="uri" href="amino" title="wikilink">amino</a> group from one molecule to the other.<a class="footnoteRef" href="#fn46" id="fnref46"><sup>46</sup></a></p>

<p>The reaction, for example, follows the following order: L-aspartate +2-oxoglutarate 

<math display="inline" id="Transferase:7">
 <semantics>
  <mo>⇌</mo>
  <annotation-xml encoding="MathML-Content">
   <ci>normal-⇌</ci>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \rightleftharpoons
  </annotation>
 </semantics>
</math>

 oxaloacetate + L-glutamate.<a class="footnoteRef" href="#fn47" id="fnref47"><sup>47</sup></a></p>
<h3 id="ec-2.7-phosphorus-transferases">EC 2.7: phosphorus transferases</h3>

<p>While EC 2.7 includes enzymes that transfer <a class="uri" href="phosphorus" title="wikilink">phosphorus</a>-containing groups, it also includes nuclotidyl transferases as well.<a class="footnoteRef" href="#fn48" id="fnref48"><sup>48</sup></a> Sub-category <a class="uri" href="phosphotransferase" title="wikilink">phosphotransferase</a> is divided up in categories based on the type of group that accepts the transfer.<a class="footnoteRef" href="#fn49" id="fnref49"><sup>49</sup></a> Groups that are classified as phosphate acceptors include: alcohols, carboxy groups, nitrogenous groups, and phosphate groups.<a class="footnoteRef" href="#fn50" id="fnref50"><sup>50</sup></a> Further constituents of this subclass of transferases are various kinases. A prominent kinase is <a href="cyclin-dependent_kinase" title="wikilink">cyclin-dependent kinase</a> (or CDK), which comprises a sub-family of <a href="protein_kinase" title="wikilink">protein kinases</a>. As their name implies, CDKs are heavily dependent on specific <a class="uri" href="cyclin" title="wikilink">cyclin</a> molecules for <a href="Activation#Biochemistry" title="wikilink">activation</a>.<a class="footnoteRef" href="#fn51" id="fnref51"><sup>51</sup></a> Once combined, the CDK-cyclin complex is capable of enacting its function within the cell cycle.<a class="footnoteRef" href="#fn52" id="fnref52"><sup>52</sup></a></p>

<p>The reaction catalyzed by CDK is as follows: ATP + a target protein 

<math display="inline" id="Transferase:8">
 <semantics>
  <mo>→</mo>
  <annotation-xml encoding="MathML-Content">
   <ci>normal-→</ci>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \rightarrow
  </annotation>
 </semantics>
</math>

 ADP + a phosphoprotein.<a class="footnoteRef" href="#fn53" id="fnref53"><sup>53</sup></a></p>
<h3 id="ec-2.8-sulfur-transferases">EC 2.8: sulfur transferases</h3>
<figure><b>(Figure)</b>
<figcaption>Ribbon diagram of a variant structure of estrogen sulfotransferase (PDB 1aqy EBI)<a class="footnoteRef" href="#fn54" id="fnref54"><sup>54</sup></a></figcaption>
</figure>

<p>Transfer of sulfur-containing groups is covered by EC 2.8 and is subdivided into the subcategories of sulfurtransferases, sulfotransferases, and CoA-transferases, as well as enzymes that transfer alkylthio groups.<a class="footnoteRef" href="#fn55" id="fnref55"><sup>55</sup></a> A specific group of sulfotransferases are those that use <a href="3'-Phosphoadenosine-5'-phosphosulfate" title="wikilink">PAPS</a> as a sulfate group donor.<a class="footnoteRef" href="#fn56" id="fnref56"><sup>56</sup></a> Within this group is <a href="alcohol_sulfotransferase" title="wikilink">alcohol sulfotransferase</a> which has a broad targeting capacity.<a class="footnoteRef" href="#fn57" id="fnref57"><sup>57</sup></a> Due to this, alcohol sulfotransferase is also known by several other names including "hydroxysteroid sulfotransferase," "steroid sulfokinase," and "estrogen sulfotransferase."<a class="footnoteRef" href="#fn58" id="fnref58"><sup>58</sup></a> Decreases in its activity has been linked to human liver disease.<a class="footnoteRef" href="#fn59" id="fnref59"><sup>59</sup></a> This transferase acts via the following reaction: 3'-phosphoadenylyl sulfate + an alcohol 

<math display="inline" id="Transferase:9">
 <semantics>
  <mo>⇌</mo>
  <annotation-xml encoding="MathML-Content">
   <ci>normal-⇌</ci>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \rightleftharpoons
  </annotation>
 </semantics>
</math>

 adenosine 3',5'bisphosphate + an alkyl sulfate.<a class="footnoteRef" href="#fn60" id="fnref60"><sup>60</sup></a></p>
<h3 id="ec-2.9-selenium-transferases">EC 2.9: selenium transferases</h3>

<p>EC 2.9 includes enzymes that transfer <a class="uri" href="selenium" title="wikilink">selenium</a>-containing groups.<a class="footnoteRef" href="#fn61" id="fnref61"><sup>61</sup></a> This category only contains two transferases, and thus is one of the smallest categories of transferase. Selenocysteine synthase, which was first added to the classification system in 1999, converts seryl-tRNA(Sec UCA) into selenocysteyl-tRNA(Sec UCA).<a class="footnoteRef" href="#fn62" id="fnref62"><sup>62</sup></a></p>
<h3 id="ec-2.10-metal-transferases">EC 2.10: metal transferases</h3>

<p>The category of EC 2.10 includes enzymes that transfer <a class="uri" href="molybdenum" title="wikilink">molybdenum</a> or <a class="uri" href="tungsten" title="wikilink">tungsten</a>-containing groups. However, as of 2011, only one enzyme has been added: <a href="molybdopterin_molybdotransferase" title="wikilink">molybdopterin molybdotransferase</a>.<a class="footnoteRef" href="#fn63" id="fnref63"><sup>63</sup></a> This enzyme is a component of MoCo biosynthesis in <em>Escherichia coli</em>.<a class="footnoteRef" href="#fn64" id="fnref64"><sup>64</sup></a> The reaction it catalyzes is as follows: adenylyl-molybdopterin + <a class="uri" href="molybdate" title="wikilink">molybdate</a> 

<math display="inline" id="Transferase:10">
 <semantics>
  <mo>→</mo>
  <annotation-xml encoding="MathML-Content">
   <ci>normal-→</ci>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \rightarrow
  </annotation>
 </semantics>
</math>

 <a href="molybdenum_cofactor" title="wikilink">molybdenum cofactor</a> + AMP.<a class="footnoteRef" href="#fn65" id="fnref65"><sup>65</sup></a></p>
<h2 id="role-in-histo-blood-group">Role in histo-blood group</h2>

<p>The A and B transferases are the foundation of the human <a href="ABO_blood_group" title="wikilink">ABO blood group</a> system. Both A and B transferases are glycosyltransferases, meaning they transfer a sugar molecule onto an H-antigen.<a class="footnoteRef" href="#fn66" id="fnref66"><sup>66</sup></a> This allows H-antigen to synthesize the <a class="uri" href="glycoprotein" title="wikilink">glycoprotein</a> and <a class="uri" href="glycolipid" title="wikilink">glycolipid</a> conjugates that are known as the A/B <a class="uri" href="antigens" title="wikilink">antigens</a>.<a class="footnoteRef" href="#fn67" id="fnref67"><sup>67</sup></a> The full name of A transferase is alpha 1-3-N-acetylgalactosaminyltransferase<a class="footnoteRef" href="#fn68" id="fnref68"><sup>68</sup></a> and its function in the cell is to add N-acetylgalactosamine to H-antigen, creating A-antigen.<a class="footnoteRef" href="#fn69" id="fnref69"><sup>69</sup></a> The full name of B transferase is alpha 1-3-galactosyltransferase,<a class="footnoteRef" href="#fn70" id="fnref70"><sup>70</sup></a> and its function in the cell is to add a <a class="uri" href="galactose" title="wikilink">galactose</a> molecule to H-antigen, creating B-antigen.<a class="footnoteRef" href="#fn71" id="fnref71"><sup>71</sup></a></p>

<p>It is possible for <em><a href="Homo_sapiens" title="wikilink">Homo sapiens</a></em> to have any of four different <a href="blood_type" title="wikilink">blood types</a>: Type A (express A antigens), Type B (express B antigens), Type AB (express both A and B antigens) and Type O (express neither A nor B antigens).<a class="footnoteRef" href="#fn72" id="fnref72"><sup>72</sup></a> The gene for A and B transferases is located on <a class="uri" href="chromosome" title="wikilink">chromosome</a> nine.<a class="footnoteRef" href="#fn73" id="fnref73"><sup>73</sup></a> The gene contains seven <a href="exon" title="wikilink">exons</a> and six <a href="intron" title="wikilink">introns</a><a class="footnoteRef" href="#fn74" id="fnref74"><sup>74</sup></a> and the gene itself is over 18kb long.<a class="footnoteRef" href="#fn75" id="fnref75"><sup>75</sup></a> The alleles for A and B transferases are extremely similar. The resulting enzymes only differ in 4 amino acid residues.<a class="footnoteRef" href="#fn76" id="fnref76"><sup>76</sup></a> The differing residues are located at positions 176, 235, 266, and 268 in the enzymes.<a class="footnoteRef" href="#fn77" id="fnref77"><sup>77</sup></a></p>
<h2 id="deficiencies">Deficiencies</h2>

<p> Transferase <a href="Deficiency_(medicine)" title="wikilink">deficiencies</a> are at the root of many common <a href="illness" title="wikilink">illnesses</a>. The most common result of a transferase deficiency is a buildup of a <a href="Cellular_waste_product" title="wikilink">cellular product</a>.</p>
<h3 id="scot-deficiency">SCOT deficiency</h3>

<p><a href="OXCT1" title="wikilink">Succinyl-CoA:3-ketoacid CoA</a> transferase deficiency (or <a href="SCOT_deficiency" title="wikilink">SCOT deficiency</a>) leads to a buildup of <a href="ketone" title="wikilink">ketones</a>.<a class="footnoteRef" href="#fn78" id="fnref78"><sup>78</sup></a> <a class="uri" href="Ketones" title="wikilink">Ketones</a> are created upon the breakdown of fats in the body and are an important energy source.<a class="footnoteRef" href="#fn79" id="fnref79"><sup>79</sup></a> Inability to utilize <a class="uri" href="ketones" title="wikilink">ketones</a> leads to intermittent <a class="uri" href="ketoacidosis" title="wikilink">ketoacidosis</a>, which usually first manifests during infancy.<a class="footnoteRef" href="#fn80" id="fnref80"><sup>80</sup></a> Disease sufferers experience nausea, vomiting, inability to feed, and breathing difficulties.<a class="footnoteRef" href="#fn81" id="fnref81"><sup>81</sup></a> In extreme cases, ketoacidosis can lead to coma and death.<a class="footnoteRef" href="#fn82" id="fnref82"><sup>82</sup></a> The deficiency is caused by <a class="uri" href="mutation" title="wikilink">mutation</a> in the gene OXTC1.<a class="footnoteRef" href="#fn83" id="fnref83"><sup>83</sup></a> Treatments mostly rely on controlling the diet of the patient.<a class="footnoteRef" href="#fn84" id="fnref84"><sup>84</sup></a></p>
<h3 id="cpt-ii-deficiency">CPT-II deficiency</h3>

<p><a href="Carnitine_palmitoyltransferase_II" title="wikilink">Carnitine palmitoyltransferase II</a> deficiency (also known as <a href="Carnitine_palmitoyltransferase_II_deficiency" title="wikilink">CPT-II deficiency</a>) leads to an excess long chain <a href="fatty_acids" title="wikilink">fatty acids</a>, as the <a href="Human_body" title="wikilink">body</a> lacks the ability to transport fatty acids into the <a class="uri" href="mitochondria" title="wikilink">mitochondria</a> to be processed as a fuel source.<a class="footnoteRef" href="#fn85" id="fnref85"><sup>85</sup></a> The disease is caused by a defect in the gene CPT2.<a class="footnoteRef" href="#fn86" id="fnref86"><sup>86</sup></a> This deficiency will present in patients in one of three ways: lethal neonatal, severe infantile hepatocardiomuscular, and myopathic form.<a class="footnoteRef" href="#fn87" id="fnref87"><sup>87</sup></a> The myopathic is the least severe form of the deficiency and can manifest at any point in the lifespan of the patient.<a class="footnoteRef" href="#fn88" id="fnref88"><sup>88</sup></a> The other two forms appear in infancy.<a class="footnoteRef" href="#fn89" id="fnref89"><sup>89</sup></a> Common symptoms of the lethal neonatal form and the severe infantile forms are liver failure, heart problems, seizures and death.<a class="footnoteRef" href="#fn90" id="fnref90"><sup>90</sup></a> The myopathic form is characterized by muscle pain and weakness following vigorous exercise.<a class="footnoteRef" href="#fn91" id="fnref91"><sup>91</sup></a> Treatment generally includes dietary modifications and carnitine supplements.<a class="footnoteRef" href="#fn92" id="fnref92"><sup>92</sup></a></p>
<h3 id="galactosemia">Galactosemia</h3>

<p><a class="uri" href="Galactosemia" title="wikilink">Galactosemia</a> results from an inability to process galactose, a <a href="Monosaccharide" title="wikilink">simple sugar</a>.<a class="footnoteRef" href="#fn93" id="fnref93"><sup>93</sup></a> This deficiency occurs when the gene for <a href="galactose-1-phosphate_uridylyltransferase" title="wikilink">galactose-1-phosphate uridylyltransferase</a> (GALT) has any number of mutations, leading to a deficiency in the amount of GALT produced.<a class="footnoteRef" href="#fn94" id="fnref94"><sup>94</sup></a><a class="footnoteRef" href="#fn95" id="fnref95"><sup>95</sup></a> There are two forms of Galactosemia: classic and Duarte.<a class="footnoteRef" href="#fn96" id="fnref96"><sup>96</sup></a> Duarte galactosemia is generally less severe than classic galactosemia and is caused by a deficiency of <a class="uri" href="galactokinase" title="wikilink">galactokinase</a>.<a class="footnoteRef" href="#fn97" id="fnref97"><sup>97</sup></a> Galactosemia renders infants unable to process the sugars in breast milk, which leads to vomiting and <a href="Anorexia_(symptom)" title="wikilink">anorexia</a> within days of birth.<a class="footnoteRef" href="#fn98" id="fnref98"><sup>98</sup></a> Most symptoms of the disease are caused by a buildup of <a class="uri" href="galactose-1-phosphate" title="wikilink">galactose-1-phosphate</a> in the body.<a class="footnoteRef" href="#fn99" id="fnref99"><sup>99</sup></a> Common symptoms include liver failure, <a class="uri" href="sepsis" title="wikilink">sepsis</a>, failure to grow, and mental impairment, among others.<a class="footnoteRef" href="#fn100" id="fnref100"><sup>100</sup></a> Buildup of a second toxic substance, <a class="uri" href="galactitol" title="wikilink">galactitol</a>, occurs in the lenses of the eyes, causing <a href="cataract" title="wikilink">cataracts</a>.<a class="footnoteRef" href="#fn101" id="fnref101"><sup>101</sup></a> Currently, the only available treatment is early diagnosis followed by adherence to a diet devoid of lactose, and prescription of antibiotics for infections that may develop.<a class="footnoteRef" href="#fn102" id="fnref102"><sup>102</sup></a></p>
<h3 id="choline-acetyltransferase-deficiencies">Choline acetyltransferase deficiencies</h3>

<p><a href="Choline_acetyltransferase" title="wikilink">Choline acetyltransferase</a> (also known as ChAT or CAT) is an important enzyme which produces the <a class="uri" href="neurotransmitter" title="wikilink">neurotransmitter</a> <a class="uri" href="acetylcholine" title="wikilink">acetylcholine</a>.<a class="footnoteRef" href="#fn103" id="fnref103"><sup>103</sup></a> Acetylcholine is involved in many neuropsychic functions such as memory, attention, sleep and arousal.<a class="footnoteRef" href="#fn104" id="fnref104"><sup>104</sup></a><a class="footnoteRef" href="#fn105" id="fnref105"><sup>105</sup></a><a class="footnoteRef" href="#fn106" id="fnref106"><sup>106</sup></a> The enzyme is globular in shape and consists of a single amino acid chain.<a class="footnoteRef" href="#fn107" id="fnref107"><sup>107</sup></a> ChAT functions to transfer an <a href="acetyl_group" title="wikilink">acetyl group</a> from acetyl co-enzyme A to <a class="uri" href="choline" title="wikilink">choline</a> in the <a href="synapse" title="wikilink">synapses</a> of <a class="uri" href="nerve" title="wikilink">nerve</a> cells and exists in two forms: soluble and membrane bound.<a class="footnoteRef" href="#fn108" id="fnref108"><sup>108</sup></a> The ChAT gene is located on <a class="uri" href="chromosome" title="wikilink">chromosome</a> 10.<a class="footnoteRef" href="#fn109" id="fnref109"><sup>109</sup></a></p>
<h4 id="alzheimers-disease">Alzheimer's disease</h4>

<p>Decreased expression of ChAT is one of the hallmarks of <a href="Alzheimer's_disease" title="wikilink">Alzheimer's disease</a>.<a class="footnoteRef" href="#fn110" id="fnref110"><sup>110</sup></a> Patients with Alzheimer's disease show a 30 to 90% reduction in activity in several regions of the brain, including the <a href="temporal_lobe" title="wikilink">temporal lobe</a>, the <a href="parietal_lobe" title="wikilink">parietal lobe</a> and the <a href="frontal_lobe" title="wikilink">frontal lobe</a>.<a class="footnoteRef" href="#fn111" id="fnref111"><sup>111</sup></a> However, ChAT deficiency is not believed to be the main cause of this disease.<a class="footnoteRef" href="#fn112" id="fnref112"><sup>112</sup></a></p>
<h4 id="amyotrophic-lateral-sclerosis-als-or-lou-gehrigs-disease">Amyotrophic lateral sclerosis (ALS or Lou Gehrig's disease)</h4>

<p>Patients with <a href="Amyotrophic_lateral_sclerosis" title="wikilink">ALS</a> show a marked decrease in ChAT activity in motor neurons in the <a href="spinal_cord" title="wikilink">spinal cord</a> and <a class="uri" href="brain" title="wikilink">brain</a>.<a class="footnoteRef" href="#fn113" id="fnref113"><sup>113</sup></a> Low levels of ChAT activity are an early indication of the disease and are detectable long before motor neurons begin to die. This can even be detected before the patient is <a class="uri" href="symptomatic" title="wikilink">symptomatic</a>.<a class="footnoteRef" href="#fn114" id="fnref114"><sup>114</sup></a></p>
<h4 id="huntingtons-disease">Huntington's disease</h4>

<p>Patients with <a href="Huntington's_disease" title="wikilink">Huntington's</a> also show a marked decrease in ChAT production.<a class="footnoteRef" href="#fn115" id="fnref115"><sup>115</sup></a> Though the specific cause of the reduced production is not clear, it is believed that the death of medium-sized motor neurons with spiny <a href="dendrite" title="wikilink">dendrites</a> leads to the lower levels of ChAT production.<a class="footnoteRef" href="#fn116" id="fnref116"><sup>116</sup></a></p>
<h4 id="schizophrenia">Schizophrenia</h4>

<p>Patients with Schizophrenia also exhibit decreased levels of ChAT, localized to the <a href="Mesopontine" title="wikilink">mesopontine tegment</a> of the brain<a class="footnoteRef" href="#fn117" id="fnref117"><sup>117</sup></a> and the <a href="nucleus_accumbens" title="wikilink">nucleus accumbens</a>,<a class="footnoteRef" href="#fn118" id="fnref118"><sup>118</sup></a> which is believed to correlate with the decreased cognitive functioning experienced by these patients.<a class="footnoteRef" href="#fn119" id="fnref119"><sup>119</sup></a></p>
<h4 id="sudden-infant-death-syndrome-sids">Sudden infant death syndrome (SIDS)</h4>

<p>Recent studies have shown that <a href="Sudden_infant_death_syndrome" title="wikilink">SIDS</a> infants show decreased levels of ChAT in both the <a class="uri" href="hypothalamus" title="wikilink">hypothalamus</a> and the <a class="uri" href="striatum" title="wikilink">striatum</a>.<a class="footnoteRef" href="#fn120" id="fnref120"><sup>120</sup></a> SIDS infants also display fewer neurons capable of producing ChAT in the vagus system.<a class="footnoteRef" href="#fn121" id="fnref121"><sup>121</sup></a> These defects in the medulla could lead to an inability to control essential <a href="Autonomic_function" title="wikilink">autonomic</a> functions such as the <a class="uri" href="cardiovascular" title="wikilink">cardiovascular</a> and <a class="uri" href="respiratory" title="wikilink">respiratory</a> systems.<a class="footnoteRef" href="#fn122" id="fnref122"><sup>122</sup></a></p>
<h4 id="congenital-myasthenic-syndrome-cms">Congenital myasthenic syndrome (CMS)</h4>

<p><a href="Congenital_myasthenic_syndrome" title="wikilink">CMS</a> is a family of diseases that are characterized by defects in <a href="neuromuscular_transmission" title="wikilink">neuromuscular transmission</a> which leads to recurrent bouts of <a class="uri" href="apnea" title="wikilink">apnea</a> (inability to breathe) that can be fatal.<a class="footnoteRef" href="#fn123" id="fnref123"><sup>123</sup></a> ChAT deficiency is implicated in myasthenia syndromes where the transition problem occurs <a href="presynaptic" title="wikilink">presynaptically</a>.<a class="footnoteRef" href="#fn124" id="fnref124"><sup>124</sup></a> These syndromes are characterized by the patients’ inability to resynthesize <a class="uri" href="acetylcholine" title="wikilink">acetylcholine</a>.<a class="footnoteRef" href="#fn125" id="fnref125"><sup>125</sup></a></p>
<h2 id="uses-in-biotechnology">Uses in biotechnology</h2>
<h3 id="terminal-transferases">Terminal transferases</h3>

<p><a href="Terminal_deoxynucleotidyl_transferase" title="wikilink">Terminal transferases</a> are transferases that can be used to label DNA or to produce <a href="plasmid_vector" title="wikilink">plasmid vectors</a>.<a class="footnoteRef" href="#fn126" id="fnref126"><sup>126</sup></a> It accomplishes both of these tasks by adding <a class="uri" href="deoxynucleotides" title="wikilink">deoxynucleotides</a> in the form of a template to the <a href="Directionality_(molecular_biology)" title="wikilink">downstream</a> end or <a href="Sticky_and_blunt_ends" title="wikilink">3'</a> end of an existing DNA molecule. Terminal transferase is one of the few DNA polymerases that can function without an RNA primer.<a class="footnoteRef" href="#fn127" id="fnref127"><sup>127</sup></a></p>
<h3 id="glutathione-transferases">Glutathione transferases</h3>

<p>The family of glutathione transferases (GST) is extremely diverse, and therefore can be used for a number of biotechnological purposes. Plants use glutathione transferases as a means to segregate toxic metals from the rest of the cell.<a class="footnoteRef" href="#fn128" id="fnref128"><sup>128</sup></a> These glutathione transferases can be used to create <a href="biosensor" title="wikilink">biosensors</a> to detect contaminants such as herbicides and insecticides.<a class="footnoteRef" href="#fn129" id="fnref129"><sup>129</sup></a> Glutathione transferases are also used in transgenic plants to increase resistance to both biotic and abiotic stress.<a class="footnoteRef" href="#fn130" id="fnref130"><sup>130</sup></a> <a href="Glutathione_S-transferase" title="wikilink">Glutathione transferases</a> are currently being explored as targets for <a href="Chemotherapy" title="wikilink">anti-cancer medications</a> due to their role in <a href="drug_resistance" title="wikilink">drug resistance</a>.<a class="footnoteRef" href="#fn131" id="fnref131"><sup>131</sup></a> Further, glutathione transferase genes have been investigated due to their ability to prevent <a href="Oxidative_stress" title="wikilink">oxidative damage</a> and have shown improved resistance in <a href="Transgenesis" title="wikilink">transgenic</a> <a href="cultigen" title="wikilink">cultigens</a>.<a class="footnoteRef" href="#fn132" id="fnref132"><sup>132</sup></a></p>
<h3 id="rubber-transferases">Rubber transferases</h3>

<p>Currently the only available commercial source of natural <a class="uri" href="rubber" title="wikilink">rubber</a> is the <a href="Hevea_brasiliensis" title="wikilink">Hevea</a> plant (Hevea brasiliensis). Natural rubber is superior to synthetic rubber in a number of commercial uses.<a class="footnoteRef" href="#fn133" id="fnref133"><sup>133</sup></a> Efforts are being made to produce transgenic plants capable of synthesizing natural rubber, including <a class="uri" href="tobacco" title="wikilink">tobacco</a> and <a class="uri" href="sunflower" title="wikilink">sunflower</a>.<a class="footnoteRef" href="#fn134" id="fnref134"><sup>134</sup></a> These efforts are focused on sequencing the subunits of the rubber transferase enzyme complex in order to transfect these genes into other plants.<a class="footnoteRef" href="#fn135" id="fnref135"><sup>135</sup></a></p>
<h2 id="see-also">See also</h2>
<ul>
<li><a href="http://www.chem.qmul.ac.uk/iubmb/enzyme/EC2/intro.html">EC 2 Introduction</a> from the Department of Chemistry at <a href="Queen_Mary,_University_of_London" title="wikilink">Queen Mary, University of London</a></li>
</ul>
<h2 id="references">References</h2>

<p>"</p>

<p><a href="Category:Transferases" title="wikilink"> </a></p>
<section class="footnotes">
<hr/>
<ol>
<li id="fn1"><a href="#fnref1">↩</a></li>
<li id="fn2"><a href="#fnref2">↩</a></li>
<li id="fn3"><a href="#fnref3">↩</a></li>
<li id="fn4"><a href="#fnref4">↩</a></li>
<li id="fn5"><a href="#fnref5">↩</a></li>
<li id="fn6">Watson, James D. Molecular Biology of the Gene. Upper Saddle River, NJ: Pearson, 2013. Print.<a href="#fnref6">↩</a></li>
<li id="fn7"><a href="#fnref7">↩</a></li>
<li id="fn8"><a href="#fnref8">↩</a></li>
<li id="fn9"><a href="#fnref9">↩</a></li>
<li id="fn10"><a href="#fnref10">↩</a></li>
<li id="fn11"><a href="#fnref11">↩</a></li>
<li id="fn12"><a href="#fnref12">↩</a></li>
<li id="fn13"><a href="#fnref13">↩</a></li>
<li id="fn14"><a href="#fnref14">↩</a></li>
<li id="fn15"><a href="#fnref15">↩</a></li>
<li id="fn16"><a href="#fnref16">↩</a></li>
<li id="fn17"><a href="#fnref17">↩</a></li>
<li id="fn18"><a href="#fnref18">↩</a></li>
<li id="fn19"><a href="#fnref19">↩</a></li>
<li id="fn20"><a href="#fnref20">↩</a></li>
<li id="fn21"><a href="#fnref21">↩</a></li>
<li id="fn22"><a href="#fnref22">↩</a></li>
<li id="fn23"><a href="#fnref23">↩</a></li>
<li id="fn24"><a href="#fnref24">↩</a></li>
<li id="fn25"><a href="#fnref25">↩</a></li>
<li id="fn26"><a href="#fnref26">↩</a></li>
<li id="fn27"><a href="#fnref27">↩</a></li>
<li id="fn28"><a href="#fnref28">↩</a></li>
<li id="fn29"><a href="#fnref29">↩</a></li>
<li id="fn30"><a href="#fnref30">↩</a></li>
<li id="fn31"><a href="#fnref31">↩</a></li>
<li id="fn32"><a href="#fnref32">↩</a></li>
<li id="fn33"><a href="#fnref33">↩</a></li>
<li id="fn34"><a href="#fnref34">↩</a></li>
<li id="fn35"><a href="#fnref35">↩</a></li>
<li id="fn36"><a href="#fnref36">↩</a></li>
<li id="fn37"><a href="#fnref37">↩</a></li>
<li id="fn38"><a href="#fnref38">↩</a></li>
<li id="fn39"><a href="#fnref39">↩</a></li>
<li id="fn40"><a href="#fnref40">↩</a></li>
<li id="fn41"><a href="#fnref41">↩</a></li>
<li id="fn42"><a href="#fnref42">↩</a></li>
<li id="fn43"><a href="#fnref43">↩</a></li>
<li id="fn44"><a href="#fnref44">↩</a></li>
<li id="fn45"><a href="#fnref45">↩</a></li>
<li id="fn46"><a href="#fnref46">↩</a></li>
<li id="fn47"><a href="#fnref47">↩</a></li>
<li id="fn48"><a href="#fnref48">↩</a></li>
<li id="fn49"></li>
<li id="fn50"></li>
<li id="fn51"><a href="#fnref51">↩</a></li>
<li id="fn52"><a href="#fnref52">↩</a></li>
<li id="fn53"><a href="#fnref53">↩</a></li>
<li id="fn54"><a href="#fnref54">↩</a></li>
<li id="fn55"><a href="#fnref55">↩</a></li>
<li id="fn56"><a href="#fnref56">↩</a></li>
<li id="fn57"><a href="#fnref57">↩</a></li>
<li id="fn58"><a href="#fnref58">↩</a></li>
<li id="fn59"><a href="#fnref59">↩</a></li>
<li id="fn60"><a href="#fnref60">↩</a></li>
<li id="fn61"><a href="#fnref61">↩</a></li>
<li id="fn62"><a href="#fnref62">↩</a></li>
<li id="fn63"><a href="#fnref63">↩</a></li>
<li id="fn64"><a href="#fnref64">↩</a></li>
<li id="fn65"><a href="#fnref65">↩</a></li>
<li id="fn66"><a href="#fnref66">↩</a></li>
<li id="fn67"></li>
<li id="fn68"><a href="#fnref68">↩</a></li>
<li id="fn69"><a href="#fnref69">↩</a></li>
<li id="fn70"></li>
<li id="fn71"></li>
<li id="fn72"><a href="#fnref72">↩</a></li>
<li id="fn73"><a href="#fnref73">↩</a></li>
<li id="fn74"><a href="#fnref74">↩</a></li>
<li id="fn75"><a href="#fnref75">↩</a></li>
<li id="fn76"></li>
<li id="fn77"><a href="#fnref77">↩</a></li>
<li id="fn78"><a href="#fnref78">↩</a></li>
<li id="fn79"><a href="#fnref79">↩</a></li>
<li id="fn80"></li>
<li id="fn81"></li>
<li id="fn82"></li>
<li id="fn83"><a href="#fnref83">↩</a></li>
<li id="fn84"><a href="#fnref84">↩</a></li>
<li id="fn85"><a href="#fnref85">↩</a></li>
<li id="fn86"><a href="#fnref86">↩</a></li>
<li id="fn87"></li>
<li id="fn88"></li>
<li id="fn89"></li>
<li id="fn90"></li>
<li id="fn91"></li>
<li id="fn92"></li>
<li id="fn93"><a href="#fnref93">↩</a></li>
<li id="fn94"><a href="#fnref94">↩</a></li>
<li id="fn95"><a href="#fnref95">↩</a></li>
<li id="fn96"><a href="#fnref96">↩</a></li>
<li id="fn97"><a href="#fnref97">↩</a></li>
<li id="fn98"></li>
<li id="fn99"></li>
<li id="fn100"><a href="#fnref100">↩</a></li>
<li id="fn101"><a href="#fnref101">↩</a></li>
<li id="fn102"><a href="#fnref102">↩</a></li>
<li id="fn103"><a href="#fnref103">↩</a></li>
<li id="fn104"><a href="#fnref104">↩</a></li>
<li id="fn105"><a href="#fnref105">↩</a></li>
<li id="fn106"><a href="#fnref106">↩</a></li>
<li id="fn107"></li>
<li id="fn108"><a href="#fnref108">↩</a></li>
<li id="fn109"><a href="#fnref109">↩</a></li>
<li id="fn110"><a href="#fnref110">↩</a></li>
<li id="fn111"><a href="#fnref111">↩</a></li>
<li id="fn112"></li>
<li id="fn113"><a href="#fnref113">↩</a></li>
<li id="fn114"><a href="#fnref114">↩</a></li>
<li id="fn115"><a href="#fnref115">↩</a></li>
<li id="fn116"></li>
<li id="fn117"><a href="#fnref117">↩</a></li>
<li id="fn118"><a href="#fnref118">↩</a></li>
<li id="fn119"></li>
<li id="fn120"></li>
<li id="fn121"><a href="#fnref121">↩</a></li>
<li id="fn122"></li>
<li id="fn123"><a href="#fnref123">↩</a></li>
<li id="fn124"><a href="#fnref124">↩</a></li>
<li id="fn125"></li>
<li id="fn126"><a href="#fnref126">↩</a></li>
<li id="fn127"></li>
<li id="fn128"><a href="#fnref128">↩</a></li>
<li id="fn129"><a href="#fnref129">↩</a></li>
<li id="fn130"></li>
<li id="fn131"></li>
<li id="fn132"><a href="#fnref132">↩</a></li>
<li id="fn133"><a href="#fnref133">↩</a></li>
<li id="fn134"><a href="#fnref134">↩</a></li>
<li id="fn135"></li>
</ol>
</section>
</body>
</html>
