<html lang="en">
<head>
<meta charset="utf-8"/>
<title offset="39">Allele</title>
<script src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_SVG.js" type="text/javascript">
</script>
</head>
<body>
<h1>Allele</h1>
<hr/>

<p>An <strong>allele</strong> ( or ), or <strong>allel</strong>, is one of a number of alternative forms of the same <a class="uri" href="gene" title="wikilink">gene</a> or same <a href="Locus_(genetics)" title="wikilink">genetic locus</a>.<a class="footnoteRef" href="#fn1" id="fnref1"><sup>1</sup></a><a class="footnoteRef" href="#fn2" id="fnref2"><sup>2</sup></a> Sometimes, different alleles can result in different observable <a href="phenotypic_trait" title="wikilink">phenotypic traits</a>, such as different <a class="uri" href="pigmentation" title="wikilink">pigmentation</a>. However, most genetic variations result in little or no observable variation.</p>

<p>Most <a href="multicellular_organism" title="wikilink">multicellular organisms</a> have two sets of <a href="chromosome" title="wikilink">chromosomes</a>; that is, they are <a class="uri" href="diploid" title="wikilink">diploid</a>. These chromosomes are referred to as <a href="homologous_chromosome" title="wikilink">homologous chromosomes</a>. Diploid organisms have one copy of each <a class="uri" href="gene" title="wikilink">gene</a> (and, therefore, one allele) on each chromosome. If both alleles are the same, they and the organism are <a class="uri" href="homozygous" title="wikilink">homozygous</a> with respect to that gene. If the alleles are different, they and the organism are <a class="uri" href="heterozygous" title="wikilink">heterozygous</a> with respect to that gene.</p>

<p>The word "allele" is a short form of <strong>allelomorph</strong> ("other form"), which was used in the early days of <a class="uri" href="genetics" title="wikilink">genetics</a> to describe variant forms of a <a class="uri" href="gene" title="wikilink">gene</a> detected as different <a class="uri" href="phenotypes" title="wikilink">phenotypes</a>. It derives from the Greek prefix <em>ἀλλήλ</em>, <em>allel</em>, meaning "reciprocal" or "each other", which itself is related to the Greek adjective ἄλλος (allos; cognate with Latin "alius"), meaning "other".</p>
<h2 id="dominant-and-recessive-alleles">Dominant and recessive alleles</h2>

<p>In many cases, genotypic interactions between the two alleles at a locus can be described as <a href="Dominance_(genetics)" title="wikilink">dominant</a> or <a class="uri" href="recessive" title="wikilink">recessive</a>, according to which of the two homozygous phenotypes the heterozygote most resembles. Where the heterozygote is indistinguishable from one of the homozygotes, the allele involved is said to be <a href="Dominance_(genetics)" title="wikilink">dominant</a> to the other, which is said to be <a class="uri" href="recessive" title="wikilink">recessive</a> to the former.<a class="footnoteRef" href="#fn3" id="fnref3"><sup>3</sup></a> The degree and pattern of dominance varies among loci. This type of interaction was first formally described by <a href="Gregor_Mendel" title="wikilink">Gregor Mendel</a>. However, many traits defy this simple categorization and the phenotypes are modeled by <a class="uri" href="co-dominance" title="wikilink">co-dominance</a> and <a href="Quantitative_trait_locus" title="wikilink">polygenic inheritance</a>.</p>

<p>The term "<a href="wild_type" title="wikilink">wild type</a>" allele is sometimes used to describe an allele that is thought to contribute to the typical phenotypic character as seen in "wild" populations of organisms, such as fruit flies (<em><a href="Drosophila_melanogaster" title="wikilink">Drosophila melanogaster</a></em>). Such a "wild type" allele was historically regarded as dominant, common, and normal, in contrast to "<a class="uri" href="mutant" title="wikilink">mutant</a>" alleles regarded as recessive, rare, and frequently deleterious. It was formerly thought that most individuals were homozygous for the "wild type" allele at most gene loci, and that any alternative "mutant" allele was found in homozygous form in a small minority of "affected" individuals, often as <a href="genetic_diseases" title="wikilink">genetic diseases</a>, and more frequently in heterozygous form in "<a href="Genetic_carrier" title="wikilink">carriers</a>" for the mutant allele. It is now appreciated that most or all gene loci are highly polymorphic, with multiple alleles, whose frequencies vary from population to population, and that a great deal of genetic variation is hidden in the form of alleles that do not produce obvious phenotypic differences.</p>
<h2 id="multiple-alleles">Multiple alleles</h2>
<figure><b>(Figure)</b>
<figcaption>In the <a href="ABO_blood_group_system" title="wikilink">ABO blood group system</a>, a person with Type A blood displays A-antigens and may have a genotype I<sup>A</sup>I<sup>A</sup> or I<sup>A</sup>i. A person with Type B blood displays B-antigens and may have the genotype I<sup>B</sup>I<sup>B</sup> or I<sup>B</sup>i. A person with Type AB blood displays both A- and B-antigens and has the genotype I<sup>A</sup>I<sup>B</sup> and a person with Type O blood, displaying neither antigen, has the genotype ii.</figcaption>
</figure>

<p>A population or <a class="uri" href="species" title="wikilink">species</a> of organisms typically includes multiple alleles at each <a href="Locus_(genetics)" title="wikilink">locus</a> among various individuals. Allelic variation at a locus is measurable as the number of alleles (<a href="Polymorphism_(biology)" title="wikilink">polymorphism</a>) present, or the proportion of heterozygotes in the population. A <a href="null_allele" title="wikilink">null allele</a> is a gene variant that lacks the gene's normal function because it either is not expressed, or the expressed protein is inactive.</p>

<p>For example, at the gene locus for the <a class="uri" href="ABO" title="wikilink">ABO</a> <a href="blood_type" title="wikilink">blood type</a> <a class="uri" href="carbohydrate" title="wikilink">carbohydrate</a> <a href="antigen" title="wikilink">antigens</a> in humans,<ref name="&lt;ref name=" onlinemendelianinheritanceinman"=""> classical genetics recognizes three alleles, I<sup>A</sup>, I<sup>B</sup>, and i, that determine compatibility of <a href="blood_transfusion" title="wikilink">blood transfusions</a>. Any individual has one of six possible <a href="genotype" title="wikilink">genotypes</a> (I<sup>A</sup>I<sup>A</sup>, I<sup>A</sup>i, I<sup>B</sup>I<sup>B</sup>, I<sup>B</sup>i, I<sup>A</sup>I<sup>B</sup>, and ii) that produce one of four possible <a href="phenotype" title="wikilink">phenotypes</a>: "Type A" (produced by I<sup>A</sup>I<sup>A</sup> homozygous and I<sup>A</sup>i heterozygous genotypes), "Type B" (produced by I<sup>B</sup>I<sup>B</sup> homozygous and I<sup>B</sup>i heterozygous genotypes), "Type AB" produced by I<sup>A</sup>I<sup>B</sup> heterozygous genotype, and "Type O" produced by ii homozygous genotype. It is now known that each of the A, B, and O alleles is actually a class of multiple alleles with different DNA sequences that produce proteins with identical properties: more than 70 alleles are known at the ABO locus.<a class="footnoteRef" href="#fn4" id="fnref4"><sup>4</sup></a> An individual with "Type A" blood may be an AO heterozygote, an AA homozygote, or an AA heterozygote with two different "A" alleles.</ref></p>
<h2 id="allele-and-genotype-frequencies">Allele and genotype frequencies</h2>

<p>The frequency of alleles in a diploid population can be used to predict the frequencies of the corresponding genotypes (see <a href="Hardy-Weinberg_principle" title="wikilink">Hardy-Weinberg principle</a>). For a simple model, with two alleles:</p>

<p>

<math display="block" id="Allele:0">
 <semantics>
  <mrow>
   <mrow>
    <mi>p</mi>
    <mo>+</mo>
    <mi>q</mi>
   </mrow>
   <mo>=</mo>
   <mpadded width="+1.7pt">
    <mn>1</mn>
   </mpadded>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <eq></eq>
    <apply>
     <plus></plus>
     <ci>p</ci>
     <ci>q</ci>
    </apply>
    <cn type="integer">1</cn>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   p+q=1\,
  </annotation>
 </semantics>
</math>

</p>

<p>

<math display="block" id="Allele:1">
 <semantics>
  <mrow>
   <mrow>
    <msup>
     <mi>p</mi>
     <mn>2</mn>
    </msup>
    <mo>+</mo>
    <mrow>
     <mn>2</mn>
     <mi>p</mi>
     <mi>q</mi>
    </mrow>
    <mo>+</mo>
    <msup>
     <mi>q</mi>
     <mn>2</mn>
    </msup>
   </mrow>
   <mo>=</mo>
   <mpadded width="+1.7pt">
    <mn>1</mn>
   </mpadded>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <eq></eq>
    <apply>
     <plus></plus>
     <apply>
      <csymbol cd="ambiguous">superscript</csymbol>
      <ci>p</ci>
      <cn type="integer">2</cn>
     </apply>
     <apply>
      <times></times>
      <cn type="integer">2</cn>
      <ci>p</ci>
      <ci>q</ci>
     </apply>
     <apply>
      <csymbol cd="ambiguous">superscript</csymbol>
      <ci>q</ci>
      <cn type="integer">2</cn>
     </apply>
    </apply>
    <cn type="integer">1</cn>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   p^{2}+2pq+q^{2}=1\,
  </annotation>
 </semantics>
</math>

</p>

<p>where <em>p</em> is the frequency of one allele and <em>q</em> is the frequency of the alternative allele, which necessarily sum to unity. Then, <em>p</em><sup>2</sup> is the fraction of the population homozygous for the first allele, 2<em>pq</em> is the fraction of heterozygotes, and <em>q</em><sup>2</sup> is the fraction homozygous for the alternative allele. If the first allele is dominant to the second then the fraction of the population that will show the dominant phenotype is <em>p</em><sup>2</sup> + 2<em>pq</em>, and the fraction with the recessive phenotype is <em>q</em><sup>2</sup>.</p>

<p>With three alleles:</p>

<p>

<math display="block" id="Allele:2">
 <semantics>
  <mrow>
   <mrow>
    <mi>p</mi>
    <mo>+</mo>
    <mi>q</mi>
    <mo>+</mo>
    <mi>r</mi>
   </mrow>
   <mo>=</mo>
   <mpadded width="+1.7pt">
    <mn>1</mn>
   </mpadded>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <eq></eq>
    <apply>
     <plus></plus>
     <ci>p</ci>
     <ci>q</ci>
     <ci>r</ci>
    </apply>
    <cn type="integer">1</cn>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   p+q+r=1\,
  </annotation>
 </semantics>
</math>

 and</p>

<p>

<math display="block" id="Allele:3">
 <semantics>
  <mrow>
   <mrow>
    <msup>
     <mi>p</mi>
     <mn>2</mn>
    </msup>
    <mo>+</mo>
    <mrow>
     <mn>2</mn>
     <mi>p</mi>
     <mi>q</mi>
    </mrow>
    <mo>+</mo>
    <mrow>
     <mn>2</mn>
     <mi>p</mi>
     <mi>r</mi>
    </mrow>
    <mo>+</mo>
    <msup>
     <mi>q</mi>
     <mn>2</mn>
    </msup>
    <mo>+</mo>
    <mrow>
     <mn>2</mn>
     <mi>q</mi>
     <mi>r</mi>
    </mrow>
    <mo>+</mo>
    <msup>
     <mi>r</mi>
     <mn>2</mn>
    </msup>
   </mrow>
   <mo>=</mo>
   <mpadded width="+1.7pt">
    <mn>1.</mn>
   </mpadded>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <eq></eq>
    <apply>
     <plus></plus>
     <apply>
      <csymbol cd="ambiguous">superscript</csymbol>
      <ci>p</ci>
      <cn type="integer">2</cn>
     </apply>
     <apply>
      <times></times>
      <cn type="integer">2</cn>
      <ci>p</ci>
      <ci>q</ci>
     </apply>
     <apply>
      <times></times>
      <cn type="integer">2</cn>
      <ci>p</ci>
      <ci>r</ci>
     </apply>
     <apply>
      <csymbol cd="ambiguous">superscript</csymbol>
      <ci>q</ci>
      <cn type="integer">2</cn>
     </apply>
     <apply>
      <times></times>
      <cn type="integer">2</cn>
      <ci>q</ci>
      <ci>r</ci>
     </apply>
     <apply>
      <csymbol cd="ambiguous">superscript</csymbol>
      <ci>r</ci>
      <cn type="integer">2</cn>
     </apply>
    </apply>
    <cn type="float">1.</cn>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   p^{2}+2pq+2pr+q^{2}+2qr+r^{2}=1.\,
  </annotation>
 </semantics>
</math>

</p>

<p>In the case of multiple alleles at a diploid locus, the number of possible genotypes (G) with a number of alleles (a) is given by the expression:</p>

<p>

<math display="block" id="Allele:4">
 <semantics>
  <mrow>
   <mrow>
    <mi>G</mi>
    <mo>=</mo>
    <mfrac>
     <mrow>
      <mi>a</mi>
      <mrow>
       <mo stretchy="false">(</mo>
       <mrow>
        <mi>a</mi>
        <mo>+</mo>
        <mn>1</mn>
       </mrow>
       <mo stretchy="false">)</mo>
      </mrow>
     </mrow>
     <mn>2</mn>
    </mfrac>
   </mrow>
   <mo>.</mo>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <eq></eq>
    <ci>G</ci>
    <apply>
     <divide></divide>
     <apply>
      <times></times>
      <ci>a</ci>
      <apply>
       <plus></plus>
       <ci>a</ci>
       <cn type="integer">1</cn>
      </apply>
     </apply>
     <cn type="integer">2</cn>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   G=\frac{a(a+1)}{2}.
  </annotation>
 </semantics>
</math>

</p>
<h2 id="allelic-dominance-in-genetic-disorders">Allelic dominance in genetic disorders</h2>

<p>A number of <a href="genetic_disorders" title="wikilink">genetic disorders</a> are caused when an individual inherits two recessive alleles for a single-gene trait. Recessive genetic disorders include <a class="uri" href="Albinism" title="wikilink">Albinism</a>, <a href="Cystic_Fibrosis" title="wikilink">Cystic Fibrosis</a>, <a class="uri" href="Galactosemia" title="wikilink">Galactosemia</a>, <a class="uri" href="Phenylketonuria" title="wikilink">Phenylketonuria</a> (PKU), and <a href="Tay-Sachs_Disease" title="wikilink">Tay-Sachs Disease</a>. Other disorders are also due to recessive alleles, but because the gene locus is located on the X chromosome, so that males have only one copy (that is, they are <a href="hemizygosity" title="wikilink">hemizygous</a>), they are more frequent in males than in females. Examples include red-green <a href="color_blindness" title="wikilink">color blindness</a> and <a href="Fragile_X_syndrome" title="wikilink">Fragile X syndrome</a>.</p>

<p>Other disorders, such as <a href="Huntington_disease" title="wikilink">Huntington disease</a>, occur when an individual inherits only one dominant allele.</p>
<h2 id="see-also">See also</h2>
<ul>
<li><a class="uri" href="Allozyme" title="wikilink">Allozyme</a></li>
<li><a class="uri" href="Evolution" title="wikilink">Evolution</a></li>
<li><a href="Genealogical_DNA_test" title="wikilink">Genealogical DNA test</a></li>
<li><a class="uri" href="Haplo-sufficiency" title="wikilink">Haplo-sufficiency</a></li>
<li><a class="uri" href="Meiosis" title="wikilink">Meiosis</a></li>
<li><a href="Mendelian_error" title="wikilink">Mendelian error</a></li>
<li><a href="Mendelian_inheritance" title="wikilink">Mendelian inheritance</a></li>
<li><a class="uri" href="Mitosis" title="wikilink">Mitosis</a></li>
<li><a class="uri" href="Penetrance" title="wikilink">Penetrance</a></li>
<li><a href="Polymorphism_(biology)" title="wikilink">Polymorphism</a></li>
<li><a href="Punnett_square" title="wikilink">Punnett square</a></li>
</ul>
<h2 id="references-and-notes">References and notes</h2>
<ul>
<li>National Geographic Society, Alton Biggs, Lucy Daniel, Edward Ortleb, Peter Rillero, Dinah Zike. "Life Science". New York, Ohio, California, Illinois: Glencoe McGraw-Hill. 2002</li>
</ul>
<h2 id="external-links">External links</h2>
<ul>
<li><a href="http://alfred.med.yale.edu/alfred/index.asp">ALFRED: The ALlele FREquency Database</a></li>
</ul>

<p>"</p>

<p><a href="Category:Classical_genetics" title="wikilink">Category:Classical genetics</a> <a href="Category:Genetic_genealogy" title="wikilink">Category:Genetic genealogy</a></p>
<section class="footnotes">
<hr/>
<ol>
<li id="fn1"></li>
<li id="fn2"></li>
<li id="fn3"></li>
<li id="fn4"></li>
</ol>
</section>
</body>
</html>
