<html lang="en">
<head>
<meta charset="utf-8"/>
<title offset="1423">Disulfide bond</title>
<script src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_SVG.js" type="text/javascript">
</script>
</head>
<body>
<h1>Disulfide bond</h1>
<hr/>

<p>In <a class="uri" href="chemistry" title="wikilink">chemistry</a>, a <strong>disulfide bond</strong> (Br.E. <strong>disulphide bond</strong>) is a <a href="covalent_bond" title="wikilink">covalent bond</a>, usually derived by the coupling of two <a class="uri" href="thiol" title="wikilink">thiol</a> <a href="functional_group" title="wikilink">groups</a>. The linkage is also called an <strong>SS-bond</strong> or <strong>disulfide bridge</strong>. The overall connectivity is therefore <em>R<strong>–S–S–</strong>R</em>. The terminology is widely used in biochemistry. In formal terms, the connection is a <strong>persulfide</strong>, in analogy to its <a href="Congener_(chemistry)" title="wikilink">congener</a>, <a href="organic_peroxide" title="wikilink">peroxide</a> (<em>R–O–O–R</em>), but this terminology is obscure and is no longer used (except in reference to <em>R–S–S–H</em> or <em>H–S–S–H</em> compounds).</p>
<figure><b>(Figure)</b>
<figcaption>Formal depiction of disulfide bond formation as an oxidation.</figcaption>
</figure>
<h2 id="properties">Properties</h2>

<p>The <a class="uri" href="disulfide" title="wikilink">disulfide</a> bonds are strong, with a typical <a href="bond_dissociation_energy" title="wikilink">bond dissociation energy</a> of 60 kcal/mole (251 kJ mol<sup>−1</sup>). However, being about 40% weaker than <a href="C–C_bond" title="wikilink">C–C</a> and <a href="C–H_bond" title="wikilink">C–H bonds</a>, the disulfide bond is often the "weak link" in many molecules. Furthermore, reflecting the <a class="uri" href="polarizability" title="wikilink">polarizability</a> of divalent sulfur, the S–S bond is susceptible to scission by polar reagents, both <a href="electrophile" title="wikilink">electrophiles</a> and especially <a href="nucleophile" title="wikilink">nucleophiles</a>:<a class="footnoteRef" href="#fn1" id="fnref1"><sup>1</sup></a></p>
<dl>
<dd>RS–SR + Nu<sup>−</sup> → RS–Nu + RS<sup>−</sup>
</dd>
</dl>

<p>The disulfide bond is about 2.05 Å in length, about 0.5 Å longer than a C–C bond. Rotation about the S–S axis is subject to a low barrier. Disulfides show a distinct preference for <a href="dihedral_angle" title="wikilink">dihedral angles</a> approaching 90°. When the angle approaches 0° or 180°, then the disulfide is a significantly better oxidant.</p>

<p>Disulfides where the two R groups are the same are called symmetric, examples being <a href="diphenyl_disulfide" title="wikilink">diphenyl disulfide</a> and <a href="dimethyl_disulfide" title="wikilink">dimethyl disulfide</a>. When the two R groups are not identical, the compound is said to be an unsymmetric or mixed disulfide.<a class="footnoteRef" href="#fn2" id="fnref2"><sup>2</sup></a></p>

<p>Although the <a class="uri" href="hydrogenation" title="wikilink">hydrogenation</a> of disulfides is usually not practical, the equilibrium constant for the reaction provides a measure of the standard redox potential for disulfides:</p>
<dl>
<dd>RSSR + H<sub>2</sub> → 2 RSH
</dd>
</dl>

<p>This value is about −250 mV vs NHE (pH = 7). By comparison, the standard reduction potential for <a href="ferrodoxin" title="wikilink">ferrodoxins</a> is about −430 mV.</p>
<h2 id="preparation-and-reactions">Preparation and reactions</h2>
<h3 id="formation-of-disulfides">Formation of disulfides</h3>

<p>Disulfide bonds are usually formed from the <a class="uri" href="oxidation" title="wikilink">oxidation</a> of <a href="thiol" title="wikilink">sulfhydryl (–SH) groups</a>, especially in biological contexts.<a class="footnoteRef" href="#fn3" id="fnref3"><sup>3</sup></a> The transformation is depicted as follows:</p>
<dl>
<dd>2 RSH  RS–SR + 2 H<sup>+</sup> + 2 e<sup>−</sup>
</dd>
</dl>

<p>A variety of oxidants promote this reaction including air and <a href="hydrogen_peroxide" title="wikilink">hydrogen peroxide</a>. Such reactions are thought to proceed via <a href="sulfenic_acid" title="wikilink">sulfenic acid</a> intermediates. In the laboratory, <a class="uri" href="iodine" title="wikilink">iodine</a> in the presence of base is commonly employed to oxidize thiols to disulfides. Several metals, such as copper(II) and iron(III) <a href="metal_complex" title="wikilink">complexes</a> affect this reaction. Alternatively, disulfide bonds in proteins often formed by <a href="thiol-disulfide_exchange" title="wikilink">thiol-disulfide exchange</a>:</p>
<dl>
<dd>RS–SR + R'SH  R'S–SR + RSH
</dd>
</dl>

<p>Such reactions are mediated by enzymes in some cases and in other cases are under equilibrium control, especially in the presence of a catalytic amount of base.</p>

<p>The <a class="uri" href="alkylation" title="wikilink">alkylation</a> of alkali metal di- and <a href="polysulfide" title="wikilink">polysulfides</a> gives disulfides. "Thiokol" polymers arise when <a href="sodium_polysulfide" title="wikilink">sodium polysulfide</a> is treated with an alkyl dihalide. In the converse reaction, carbanionic reagents react with elemental sulfur to afford mixtures of the thioether, disulfide, and higher polysulfides. These reactions are often unselective but can be optimized for specific applications.</p>

<p>Many specialized methods have been developed for forming disulfides, usually for applications in <a href="organic_synthesis" title="wikilink">organic synthesis</a>. Reagents that deliver the equivalent of "RS<sup>+</sup>" react with thiols to give asymmetrical disulfides:<a class="footnoteRef" href="#fn4" id="fnref4"><sup>4</sup></a></p>
<dl>
<dd>RSH + R'SNR"<sub>2</sub> → RS-SR' + HNR"<sub>2</sub>, where R"<sub>2</sub>N = <a href="phthalimide" title="wikilink">phthalimido</a>
</dd>
</dl>
<h3 id="cleavage-of-disulfides">Cleavage of disulfides</h3>

<p>The most important aspect of disulfide bonds is their cleavage, which occurs via reduction. A variety of reductants can be used. In biochemistry, thiols such as <a href="mercaptoethanol" title="wikilink">mercaptoethanol (b-ME)</a> or <a href="Dithiothreitol" title="wikilink">dithiothreitol (DTT)</a> serve as reductants, the thiol reagents are used in excess to drive the equilibrium to the right:</p>
<dl>
<dd>RS–SR + 2 HOCH<sub>2</sub>CH<sub>2</sub>SH  HOCH<sub>2</sub>CH<sub>2</sub>S–SCH<sub>2</sub>CH<sub>2</sub>OH + 2 RSH
</dd>
</dl>

<p>The reductant <a href="TCEP" title="wikilink">Tris(2-carboxyethyl)phosphine (TCEP)</a> is useful, beside being odorless compared to b-ME and DTT, because it is selective, working at both alkaline and acidic conditions (unlike DTT), is more hydrophilic and more resistant to oxidation in air. Furthermore, it is often not needed to remove TCEP before modification of protein thiols .<a class="footnoteRef" href="#fn5" id="fnref5"><sup>5</sup></a></p>

<p>In organic synthesis, hydride agents are typically employed for scission of disulfides, such as <a href="sodium_borohydride" title="wikilink">sodium borohydride</a>. More aggressive, alkali metals will effect this reaction:</p>
<dl>
<dd>RS–SR + 2 Na → 2 NaSR
</dd>
</dl>

<p>These reactions are often followed by protonation of the resulting metal thiolate:</p>
<dl>
<dd>NaSR + HCl → HSR + NaCl
</dd>
</dl>

<p>Thiol–disulfide exchange is a <a href="chemical_reaction" title="wikilink">chemical reaction</a> in which a <a href="thiol" title="wikilink">thiolate</a> group  attacks a <a class="uri" href="sulfur" title="wikilink">sulfur</a> <a class="uri" href="atom" title="wikilink">atom</a> of a disulfide bond –S–S–. The original disulfide bond is broken, and its other sulfur atom (green atom in Figure 1) is released as a new thiolate, carrying away the negative charge. Meanwhile, a new disulfide bond forms between the attacking thiolate (red atom in Figure 1) and the original sulfur atom (blue atom in Figure 1).<a class="footnoteRef" href="#fn6" id="fnref6"><sup>6</sup></a><a class="footnoteRef" href="#fn7" id="fnref7"><sup>7</sup></a></p>
<figure><b>(Figure)</b>
<figcaption>Thiol–disulfide exchange showing the linear intermediate in which the charge is shared among the three sulfur atoms. The thiolate group (shown in red) attacks a sulfur atom (shown in blue) of the disulfide bond, displacing the other sulfur atom (shown in green) and forming a new disulfide bond.</figcaption>
</figure>

<p>Thiolates, not thiols, attack disulfide bonds. Hence, thiol–disulfide exchange is inhibited at low pH (typically, below 8) where the protonated thiol form is favored relative to the deprotonated thiolate form. (The <a class="uri" href="pKa" title="wikilink">pKa</a> of a typical thiol group is roughly 8.3, but can vary due to its environment.)</p>

<p>Thiol–disulfide exchange is the principal reaction by which disulfide bonds are formed and rearranged in a <a class="uri" href="protein" title="wikilink">protein</a>. The rearrangement of disulfide bonds within a protein generally occurs via intra-protein thiol–disulfide exchange reactions; a <a href="thiol" title="wikilink">thiolate</a> group of a <a class="uri" href="cysteine" title="wikilink">cysteine</a> residue attacks one of the protein's own disulfide bonds. This process of disulfide rearrangement (known as <strong>disulfide shuffling</strong>) does not change the number of disulfide bonds within a protein, merely their location (i.e., which cysteines are bonded). Disulfide reshuffling is generally much faster than oxidation/reduction reactions, which change the number of disulfide bonds within a protein. The oxidation and reduction of protein disulfide bonds <em>in vitro</em> also generally occurs via thiol–disulfide exchange reactions. Typically, the thiolate of a redox reagent such as <a class="uri" href="glutathione" title="wikilink">glutathione</a> or <a class="uri" href="dithiothreitol" title="wikilink">dithiothreitol</a> attacks the disulfide bond on a protein forming a <strong>mixed disulfide bond</strong> between the protein and the reagent. This mixed disulfide bond when attacked by another thiolate from the reagent, leaves the cysteine oxidized. In effect, the disulfide bond is transferred from the protein to the reagent in two steps, both thiol–disulfide exchange reactions.</p>

<p>The <em>in vivo</em> oxidation and reduction of protein disulfide bonds by thiol–disulfide exchange is facilitated by a protein called <a class="uri" href="thioredoxin" title="wikilink">thioredoxin</a>. This small protein, essential in all known organisms, contains two cysteine amino acid residues in a <a href="vicinal_(chemistry)" title="wikilink">vicinal</a> arrangement (i.e., next to each other), which allows it to form an internal disulfide bond, or disulfide bonds with other proteins. As such, it can be used as a repository of reduced or oxidized disulfide bond moieties.</p>
<h3 id="other-reactions-of-disulfides">Other reactions of disulfides</h3>

<p>Many specialized <a href="organic_reaction" title="wikilink">organic reactions</a> have been developed for disulfides, again mainly associated with the scission of the S–S bond, which is usually the weakest bond in a molecule. In the Zincke disulfide cleavage reactions, disulfides are cleaved to give the to a <a href="sulfenyl_halide" title="wikilink">sulfenyl halide</a> by reaction with <a class="uri" href="bromine" title="wikilink">bromine</a> or <a class="uri" href="chlorine" title="wikilink">chlorine</a>.<a class="footnoteRef" href="#fn8" id="fnref8"><sup>8</sup></a><a class="footnoteRef" href="#fn9" id="fnref9"><sup>9</sup></a></p>
<figure><b>(Figure)</b>
<figcaption>Disulfide bonds (schematic presentation) in a protein.</figcaption>
</figure>
<h2 id="occurrence-in-proteins">Occurrence in proteins</h2>

<p>Disulfide bonds play an important role in the folding and stability of some proteins, usually proteins secreted to the extracellular medium.<a class="footnoteRef" href="#fn10" id="fnref10"><sup>10</sup></a> Since most cellular compartments are <a href="reducing_environment" title="wikilink">reducing environments</a>, in general, disulfide bonds are unstable in the <a class="uri" href="cytosol" title="wikilink">cytosol</a>, with some exceptions as noted below, unless a sulfhydryl oxidase is present.<a class="footnoteRef" href="#fn11" id="fnref11"><sup>11</sup></a></p>
<figure><b>(Figure)</b>
<figcaption><a class="uri" href="Cystine" title="wikilink">Cystine</a> is composed of two <a href="cysteine" title="wikilink">cysteines</a> linked by a disulfide bond (shown here in its neutral form).</figcaption>
</figure>

<p>Disulfide bonds in proteins are formed between the <a class="uri" href="thiol" title="wikilink">thiol</a> groups of <a class="uri" href="cysteine" title="wikilink">cysteine</a> residues by the process of <a href="oxidative_folding" title="wikilink">oxidative folding</a>. The other sulfur-containing amino acid, <a class="uri" href="methionine" title="wikilink">methionine</a>, cannot form disulfide bonds. A disulfide bond is typically denoted by hyphenating the abbreviations for cysteine, e.g., when referring to <a href="Ribonuclease_A" title="wikilink">Ribonuclease A</a> the "Cys26–Cys84 disulfide bond", or the "26–84 disulfide bond", or most simply as "C26–C84"<a class="footnoteRef" href="#fn12" id="fnref12"><sup>12</sup></a> where the disulfide bond is understood and does not need to be mentioned. The prototype of a protein disulfide bond is the two-amino-acid peptide <a class="uri" href="cystine" title="wikilink">cystine</a>, which is composed of two <a class="uri" href="cysteine" title="wikilink">cysteine</a> amino acids joined by a disulfide bond (shown in Figure 2 in its unionized form). The structure of a disulfide bond can be described by its 

<math display="inline" id="Disulfide_bond:0">
 <semantics>
  <msub>
   <mi>χ</mi>
   <mrow>
    <mi>s</mi>
    <mi>s</mi>
   </mrow>
  </msub>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <csymbol cd="ambiguous">subscript</csymbol>
    <ci>χ</ci>
    <apply>
     <times></times>
     <ci>s</ci>
     <ci>s</ci>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \chi_{ss}
  </annotation>
 </semantics>
</math>

 <a href="dihedral_angle" title="wikilink">dihedral angle</a> between the 

<math display="inline" id="Disulfide_bond:1">
 <semantics>
  <mrow>
   <msup>
    <mi>C</mi>
    <mi>β</mi>
   </msup>
   <mo>-</mo>
   <msup>
    <mi>S</mi>
    <mi>γ</mi>
   </msup>
   <mo>-</mo>
   <msup>
    <mi>S</mi>
    <mi>γ</mi>
   </msup>
   <mo>-</mo>
   <msup>
    <mi>C</mi>
    <mi>β</mi>
   </msup>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <minus></minus>
    <apply>
     <csymbol cd="ambiguous">superscript</csymbol>
     <ci>C</ci>
     <ci>β</ci>
    </apply>
    <apply>
     <csymbol cd="ambiguous">superscript</csymbol>
     <ci>S</ci>
     <ci>γ</ci>
    </apply>
    <apply>
     <csymbol cd="ambiguous">superscript</csymbol>
     <ci>S</ci>
     <ci>γ</ci>
    </apply>
    <apply>
     <csymbol cd="ambiguous">superscript</csymbol>
     <ci>C</ci>
     <ci>β</ci>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   C^{\beta}-S^{\gamma}-S^{\gamma}-C^{\beta}
  </annotation>
 </semantics>
</math>

 atoms, which is usually close to ±90°.</p>

<p>The disulfide bond stabilizes the folded form of a protein in several ways:</p>

<p>1) It holds two portions of the protein together, biasing the protein towards the folded topology. That is, the disulfide bond <em>destabilizes the unfolded form</em> of the protein by lowering its <a href="loop_entropy" title="wikilink">entropy</a>.</p>

<p>2) The disulfide bond may form the nucleus of a <a href="hydrophobic_core" title="wikilink">hydrophobic core</a> of the folded protein, i.e., local hydrophobic residues may condense around the disulfide bond and onto each other through <a href="hydrophobic_interaction" title="wikilink">hydrophobic interactions</a>.</p>

<p>3) Related to #1 and #2, the disulfide bond link <em>two</em> segments of the protein chain, the disulfide bond <em>increases</em> the effective local concentration of protein residues and <em>lowers</em> the effective local concentration of water molecules. Since water molecules attack amide-amide <a href="hydrogen_bond" title="wikilink">hydrogen bonds</a> and break up <a href="secondary_structure" title="wikilink">secondary structure</a>, a disulfide bond stabilizes secondary structure in its vicinity. For example, researchers have identified several pairs of peptides that are unstructured in isolation, but adopt stable secondary and tertiary structure upon forming a disulfide bond between them.</p>

<p>A <em>disulfide species</em> is a particular pairing of cysteines in a disulfide-bonded protein and is usually depicted by listing the disulfide bonds in parentheses, e.g., the "(26–84, 58–110) disulfide species". A <em>disulfide ensemble</em> is a grouping of all disulfide species with the same number of disulfide bonds, and is usually denoted as the 1S ensemble, the 2S ensemble, etc. for disulfide species having one, two, etc. disulfide bonds. Thus, the (26–84) disulfide species belongs to the 1S ensemble, whereas the (26–84, 58–110) species belongs to the 2S ensemble. The single species with no disulfide bonds is usually denoted as R for "fully reduced". Under typical conditions, <a href="thiol-disulfide_exchange" title="wikilink">disulfide reshuffling</a> is much faster than the formation of new disulfide bonds or their reduction; hence, the disulfide species within an ensemble equilibrate more quickly than between ensembles.</p>

<p>The native form of a protein is usually a single disulfide species, although some proteins may cycle between a few disulfide states as part of their function, e.g., <a class="uri" href="thioredoxin" title="wikilink">thioredoxin</a>. In proteins with more than two cysteines, non-native disulfide species may be formed, which are almost always misfolded. As the number of cysteines increases, the number of nonnative species increases factorially.</p>
<h3 id="formula-to-calculate-the-possible-number-of-distinct-disulfide-bond-isomers-or-connectivities">Formula to calculate the possible number of distinct disulfide bond isomers or connectivities</h3>

<p>The number of ways <em>i</em> in which <em>p</em> disulfide bonds can be formed from <em>n</em> cysteine residues present in a protein is given by the formula</p>

<p>

<math display="block" id="Disulfide_bond:2">
 <semantics>
  <mrow>
   <mi>i</mi>
   <mo>=</mo>
   <mfrac>
    <mrow>
     <mi>n</mi>
     <mo lspace="0pt" rspace="3.5pt">!</mo>
    </mrow>
    <mrow>
     <mrow>
      <mrow>
       <mo stretchy="false">(</mo>
       <mrow>
        <mi>n</mi>
        <mo>-</mo>
        <mrow>
         <mn>2</mn>
         <mi>p</mi>
        </mrow>
       </mrow>
       <mo stretchy="false">)</mo>
      </mrow>
      <mo lspace="0pt" rspace="3.5pt">!</mo>
     </mrow>
     <mrow>
      <mi>p</mi>
      <mo lspace="0pt" rspace="3.5pt">!</mo>
     </mrow>
     <msup>
      <mn>2</mn>
      <mi>p</mi>
     </msup>
    </mrow>
   </mfrac>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <eq></eq>
    <ci>i</ci>
    <apply>
     <divide></divide>
     <apply>
      <factorial></factorial>
      <ci>n</ci>
     </apply>
     <apply>
      <times></times>
      <apply>
       <factorial></factorial>
       <apply>
        <minus></minus>
        <ci>n</ci>
        <apply>
         <times></times>
         <cn type="integer">2</cn>
         <ci>p</ci>
        </apply>
       </apply>
      </apply>
      <apply>
       <factorial></factorial>
       <ci>p</ci>
      </apply>
      <apply>
       <csymbol cd="ambiguous">superscript</csymbol>
       <cn type="float">2</cn>
       <ci>p</ci>
      </apply>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   i=\frac{n!}{(n-2p)!\ p!\ 2^{p}}
  </annotation>
 </semantics>
</math>

</p>

<p>Here,</p>
<dl>
<dd><em>i</em> is the number of different disulfide bond isomers or connectivities possible.
</dd>
<dd><em>n</em> is the number of cysteines present in the protein molecule.
</dd>
<dd><em>p</em> is the number of disulphide bonds that are formed (hence <em>2p</em> is less than or equal to n).
</dd>
</dl>

<p>The above formula is the most general relation which can be used to calculate the number of possible disulfide bond isomers (or connectivities) when <em>n</em> is either even or odd, and when all or only some of the cysteines are involved in the formation of disulfide bonds.</p>

<p>However, many of the naturally occurring proteins that have disulfide bonds possess an even number of cysteines with all of the cysteines participating in the formation of disulfide bonds. For this specific case, <em>n</em> is an even number and <em>p</em> is equal to <em>n/2</em>. Substituting the value of <em>p</em>, the above formula for the possible number of disulfide bond connectivities simplifies to:</p>

<p>

<math display="block" id="Disulfide_bond:3">
 <semantics>
  <mrow>
   <mi>i</mi>
   <mo>=</mo>
   <mfrac>
    <mrow>
     <mi>n</mi>
     <mo lspace="0pt" rspace="3.5pt">!</mo>
    </mrow>
    <mrow>
     <mrow>
      <mrow>
       <mo stretchy="false">(</mo>
       <mrow>
        <mi>n</mi>
        <mo>-</mo>
        <mrow>
         <mn>2</mn>
         <mi>p</mi>
        </mrow>
       </mrow>
       <mo stretchy="false">)</mo>
      </mrow>
      <mo lspace="0pt" rspace="3.5pt">!</mo>
     </mrow>
     <mrow>
      <mi>p</mi>
      <mo lspace="0pt" rspace="3.5pt">!</mo>
     </mrow>
     <msup>
      <mn>2</mn>
      <mi>p</mi>
     </msup>
    </mrow>
   </mfrac>
   <mo>=</mo>
   <mfrac>
    <mrow>
     <mi>n</mi>
     <mo lspace="0pt" rspace="3.5pt">!</mo>
    </mrow>
    <mrow>
     <mrow>
      <mrow>
       <mo stretchy="false">(</mo>
       <mrow>
        <mi>n</mi>
        <mo>-</mo>
        <mrow>
         <mn>2</mn>
         <mrow>
          <mo stretchy="false">(</mo>
          <mfrac>
           <mi>n</mi>
           <mn>2</mn>
          </mfrac>
          <mo stretchy="false">)</mo>
         </mrow>
        </mrow>
       </mrow>
       <mo stretchy="false">)</mo>
      </mrow>
      <mo lspace="0pt" rspace="3.5pt">!</mo>
     </mrow>
     <mrow>
      <mrow>
       <mo stretchy="false">(</mo>
       <mfrac>
        <mi>n</mi>
        <mn>2</mn>
       </mfrac>
       <mo stretchy="false">)</mo>
      </mrow>
      <mo lspace="0pt" rspace="3.5pt">!</mo>
     </mrow>
     <msup>
      <mn>2</mn>
      <mrow>
       <mo stretchy="false">(</mo>
       <mfrac>
        <mi>n</mi>
        <mn>2</mn>
       </mfrac>
       <mo stretchy="false">)</mo>
      </mrow>
     </msup>
    </mrow>
   </mfrac>
   <mo>=</mo>
   <mfrac>
    <mrow>
     <mi>n</mi>
     <mo lspace="0pt" rspace="3.5pt">!</mo>
    </mrow>
    <mrow>
     <mrow>
      <mrow>
       <mo stretchy="false">(</mo>
       <mfrac>
        <mi>n</mi>
        <mn>2</mn>
       </mfrac>
       <mo stretchy="false">)</mo>
      </mrow>
      <mo lspace="0pt" rspace="3.5pt">!</mo>
     </mrow>
     <msup>
      <mn>2</mn>
      <mrow>
       <mo stretchy="false">(</mo>
       <mfrac>
        <mi>n</mi>
        <mn>2</mn>
       </mfrac>
       <mo stretchy="false">)</mo>
      </mrow>
     </msup>
    </mrow>
   </mfrac>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <and></and>
    <apply>
     <eq></eq>
     <ci>i</ci>
     <apply>
      <divide></divide>
      <apply>
       <factorial></factorial>
       <ci>n</ci>
      </apply>
      <apply>
       <times></times>
       <apply>
        <factorial></factorial>
        <apply>
         <minus></minus>
         <ci>n</ci>
         <apply>
          <times></times>
          <cn type="integer">2</cn>
          <ci>p</ci>
         </apply>
        </apply>
       </apply>
       <apply>
        <factorial></factorial>
        <ci>p</ci>
       </apply>
       <apply>
        <csymbol cd="ambiguous">superscript</csymbol>
        <cn type="float">2</cn>
        <ci>p</ci>
       </apply>
      </apply>
     </apply>
    </apply>
    <apply>
     <eq></eq>
     <share href="#.cmml">
     </share>
     <apply>
      <divide></divide>
      <apply>
       <factorial></factorial>
       <ci>n</ci>
      </apply>
      <apply>
       <times></times>
       <apply>
        <factorial></factorial>
        <apply>
         <minus></minus>
         <ci>n</ci>
         <apply>
          <times></times>
          <cn type="integer">2</cn>
          <apply>
           <divide></divide>
           <ci>n</ci>
           <cn type="integer">2</cn>
          </apply>
         </apply>
        </apply>
       </apply>
       <apply>
        <factorial></factorial>
        <apply>
         <divide></divide>
         <ci>n</ci>
         <cn type="integer">2</cn>
        </apply>
       </apply>
       <apply>
        <csymbol cd="ambiguous">superscript</csymbol>
        <cn type="float">2</cn>
        <apply>
         <divide></divide>
         <ci>n</ci>
         <cn type="integer">2</cn>
        </apply>
       </apply>
      </apply>
     </apply>
    </apply>
    <apply>
     <eq></eq>
     <share href="#.cmml">
     </share>
     <apply>
      <divide></divide>
      <apply>
       <factorial></factorial>
       <ci>n</ci>
      </apply>
      <apply>
       <times></times>
       <apply>
        <factorial></factorial>
        <apply>
         <divide></divide>
         <ci>n</ci>
         <cn type="integer">2</cn>
        </apply>
       </apply>
       <apply>
        <csymbol cd="ambiguous">superscript</csymbol>
        <cn type="float">2</cn>
        <apply>
         <divide></divide>
         <ci>n</ci>
         <cn type="integer">2</cn>
        </apply>
       </apply>
      </apply>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   i=\frac{n!}{(n-2p)!\ p!\ 2^{p}}=\frac{n!}{(n-2(\frac{n}{2}))!\ {(\frac{n}{2})}%
!\ {2^{(\frac{n}{2})}}}=\frac{n!}{{(\frac{n}{2})}!\ {2^{(\frac{n}{2})}}}
  </annotation>
 </semantics>
</math>

</p>

<p>For this particular case (<em>n</em> is even and all cysteines form disulfide bonds), a formula which is more easier to remember is given by:</p>

<p>

<math display="block" id="Disulfide_bond:4">
 <semantics>
  <mrow>
   <mi>i</mi>
   <mo>=</mo>
   <mrow>
    <mrow>
     <mo stretchy="false">(</mo>
     <mrow>
      <mi>n</mi>
      <mo>-</mo>
      <mn>1</mn>
     </mrow>
     <mo rspace="7.5pt" stretchy="false">)</mo>
    </mrow>
    <mrow>
     <mo stretchy="false">(</mo>
     <mrow>
      <mi>n</mi>
      <mo>-</mo>
      <mn>3</mn>
     </mrow>
     <mo rspace="7.5pt" stretchy="false">)</mo>
    </mrow>
    <mrow>
     <mo stretchy="false">(</mo>
     <mrow>
      <mi>n</mi>
      <mo>-</mo>
      <mn>5</mn>
     </mrow>
     <mo rspace="7.5pt" stretchy="false">)</mo>
    </mrow>
    <mi mathvariant="normal">…</mi>
    <mn>1</mn>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <eq></eq>
    <ci>i</ci>
    <apply>
     <times></times>
     <apply>
      <minus></minus>
      <ci>n</ci>
      <cn type="integer">1</cn>
     </apply>
     <apply>
      <minus></minus>
      <ci>n</ci>
      <cn type="integer">3</cn>
     </apply>
     <apply>
      <minus></minus>
      <ci>n</ci>
      <cn type="integer">5</cn>
     </apply>
     <ci>normal-…</ci>
     <cn type="integer">1</cn>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   i=(n-1)\ (n-3)\ (n-5)\ ...1
  </annotation>
 </semantics>
</math>

</p>

<p>Both of the above relations hold good for the proteins which have an even number of cysteines and all the cysteines are involved in the formation of disulfide bonds. Both of the above formulae are derived using the same logic and essentially represent a simplification of the same starting formula.</p>

<p>As a specific example for the above case, an eight-cysteine protein such as <a href="ribonuclease_A" title="wikilink">ribonuclease A</a> can form 105 different four-disulfide species when all the cysteines are involved in the formation of disulfide bonds. Here <em>n</em>= 8 and <em>p</em>= 4.</p>

<p>So, here</p>

<p>

<math display="block" id="Disulfide_bond:5">
 <semantics>
  <mrow>
   <mi>i</mi>
   <mo>=</mo>
   <mrow>
    <mrow>
     <mo stretchy="false">(</mo>
     <mrow>
      <mn>8</mn>
      <mo>-</mo>
      <mn>1</mn>
     </mrow>
     <mo rspace="7.5pt" stretchy="false">)</mo>
    </mrow>
    <mrow>
     <mo stretchy="false">(</mo>
     <mrow>
      <mn>8</mn>
      <mo>-</mo>
      <mn>3</mn>
     </mrow>
     <mo rspace="7.5pt" stretchy="false">)</mo>
    </mrow>
    <mrow>
     <mo stretchy="false">(</mo>
     <mrow>
      <mn>8</mn>
      <mo>-</mo>
      <mn>5</mn>
     </mrow>
     <mo rspace="7.5pt" stretchy="false">)</mo>
    </mrow>
    <mrow>
     <mo stretchy="false">(</mo>
     <mrow>
      <mn>8</mn>
      <mo>-</mo>
      <mn>7</mn>
     </mrow>
     <mo stretchy="false">)</mo>
    </mrow>
   </mrow>
   <mo>=</mo>
   <mrow>
    <mpadded width="+5pt">
     <mn>7</mn>
    </mpadded>
    <mo>⋅</mo>
    <mpadded width="+5pt">
     <mn>5</mn>
    </mpadded>
    <mo>⋅</mo>
    <mpadded width="+5pt">
     <mn>3</mn>
    </mpadded>
    <mo>⋅</mo>
    <mn>1</mn>
   </mrow>
   <mo>=</mo>
   <mn>105</mn>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <and></and>
    <apply>
     <eq></eq>
     <ci>i</ci>
     <apply>
      <times></times>
      <apply>
       <minus></minus>
       <cn type="integer">8</cn>
       <cn type="integer">1</cn>
      </apply>
      <apply>
       <minus></minus>
       <cn type="integer">8</cn>
       <cn type="integer">3</cn>
      </apply>
      <apply>
       <minus></minus>
       <cn type="integer">8</cn>
       <cn type="integer">5</cn>
      </apply>
      <apply>
       <minus></minus>
       <cn type="integer">8</cn>
       <cn type="integer">7</cn>
      </apply>
     </apply>
    </apply>
    <apply>
     <eq></eq>
     <share href="#.cmml">
     </share>
     <apply>
      <ci>normal-⋅</ci>
      <cn type="integer">7</cn>
      <cn type="float">5</cn>
      <cn type="float">3</cn>
      <cn type="float">1</cn>
     </apply>
    </apply>
    <apply>
     <eq></eq>
     <share href="#.cmml">
     </share>
     <cn type="integer">105</cn>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   i=(8-1)\ (8-3)\ (8-5)\ (8-7)=7\ \cdot\ 5\ \cdot\ 3\ \cdot\ 1=105
  </annotation>
 </semantics>
</math>

</p>

<p>Only one of the 105 possible isomers is the native disulfide species. Isomerases have been identified that catalyze the interconversion of disulfide species, accelerating the formation of the native disulfide species.</p>

<p>Disulfide species that have only native disulfide bonds (but not all of them) are denoted by <em>des</em> followed by the lacking native disulfide bond(s) in square brackets. For example, the des[40–95] disulfide species has all the native disulfide bonds except that between cysteines 40 and 95. Disulfide species that lack one native disulfide bond are frequently folded, in particular, if the missing disulfide bond is exposed to solvent in the folded, native protein.</p>

<p>In order to analyze the structure of proteins, it is often necessary to break disulfide bonds. This reduction of disulfide bonds can be accomplished by treatment with <a class="uri" href="2-mercaptoethanol" title="wikilink">2-mercaptoethanol</a>, <a class="uri" href="dithiothreitol" title="wikilink">dithiothreitol</a>, or <a href="TCEP" title="wikilink">tris(2-carboxyethyl)phosphine</a>.</p>
<h3 id="in-prokaryotes-and-archaea">In prokaryotes and archaea</h3>

<p>Disulfide bonds play an important protective role for <a class="uri" href="bacteria" title="wikilink">bacteria</a> as a reversible switch that turns a protein on or off when bacterial cells are exposed to <a class="uri" href="oxidation" title="wikilink">oxidation</a> reactions. <a href="Hydrogen_peroxide" title="wikilink">Hydrogen peroxide</a> (<a href="hydrogen" title="wikilink">H</a><sub>2</sub><a href="oxygen" title="wikilink">O</a><sub>2</sub>) in particular could severely damage <a class="uri" href="DNA" title="wikilink">DNA</a> and kill the <a href="bacteria" title="wikilink">bacterium</a> at low concentrations if not for the protective action of the SS-bond. Archaea typically have fewer disulfides than higher organisms.<a class="footnoteRef" href="#fn13" id="fnref13"><sup>13</sup></a></p>
<h3 id="in-eukaryotes">In eukaryotes</h3>

<p>In <a href="eukaryote" title="wikilink">eukaryotic</a> cells, in general, stable disulfide bonds are formed in the lumen of the <a href="rough_endoplasmic_reticulum" title="wikilink">RER</a> (rough endoplasmic reticulum) and the <a href="mitochondrial_intermembrane_space" title="wikilink">mitochondrial intermembrane space</a> but not in the <a class="uri" href="cytosol" title="wikilink">cytosol</a>. This is due to the more oxidizing environment of the aforementioned compartments and more reducing environment of the cytosol (see <a class="uri" href="glutathione" title="wikilink">glutathione</a>). Thus disulfide bonds are mostly found in secretory proteins, lysosomal proteins, and the exoplasmic domains of membrane proteins.</p>

<p>Notable exceptions to this rule include a number of cytosolic proteins which have cysteine residues in proximity to each other that function as oxidation sensors or redox catalysts; when the reductive potential of the cell fails, they oxidize and trigger cellular response mechanisms. <a class="uri" href="Vaccinia" title="wikilink">Vaccinia</a> virus also produces cytosolic proteins and peptides that have many disulfide bonds; although the reason for this is unknown presumably they have protective effects against intracellular proteolysis machinery.</p>

<p>Disulfide bonds are also formed within and between <a href="protamine" title="wikilink">protamines</a> in the <a class="uri" href="sperm" title="wikilink">sperm</a> <a class="uri" href="chromatin" title="wikilink">chromatin</a> of many <a href="mammal" title="wikilink">mammalian</a> species.</p>
<h3 id="disulfides-in-regulatory-proteins">Disulfides in regulatory proteins</h3>

<p>As disulfide bonds can be reversibly reduced and re-oxidized, the redox state of these bonds has evolved into a signaling element. In <a class="uri" href="chloroplasts" title="wikilink">chloroplasts</a>, for example, the enzymatic reduction of disulfide bonds has been linked to the control of numerous metabolic pathways as well as gene expression. The reductive signaling activity has been shown, thus far, to be carried by the <a href="ferredoxin_thioredoxin_system" title="wikilink">ferredoxin thioredoxin system</a>, channeling electrons from the light reactions of <a href="photosystem_I" title="wikilink">photosystem I</a> to catalytically reduce disulfides in regulated proteins in a light dependent manner. In this way chloroplasts adjust the activity of key processes such as the <a href="Calvin-Benson_cycle" title="wikilink">Calvin-Benson cycle</a>, <a class="uri" href="starch" title="wikilink">starch</a> degradation, <a href="Adenosine_triphosphate" title="wikilink">ATP</a> production and gene expression according to light intensity.</p>
<h3 id="in-hair-and-feathers">In hair and feathers</h3>

<p>Over 90% of the dry weight of <a class="uri" href="hair" title="wikilink">hair</a> comprises proteins called <a href="keratin" title="wikilink">keratins</a>, which have a high disulfide content, from the amino acid cysteine. The robustness conferred in part by disulfide linkages is illustrated by the recovery of virtually intact hair from ancient Egyptian tombs. <a href="Feather" title="wikilink">Feathers</a> have similar keratins and are extremely resistant to protein digestive enzymes. Different parts of the hair and feather have different cysteine levels, leading to harder or softer material. Manipulating disulfide bonds in hair is the basis for the <a href="permanent_wave" title="wikilink">permanent wave</a> in hairstyling. Reagents that affect the making and breaking of S-S bonds are key, e.g., <a href="ammonium_thioglycolate" title="wikilink">ammonium thioglycolate</a>. The high disulfide content of feathers dictates the high sulfur content of bird eggs. The high sulfur content of hair and feathers contributes to the disagreeable odor that results when they are burned.</p>
<h2 id="in-industry">In industry</h2>

<p>Disulfide and (polysulfide) bonds are the crosslinking groups that result from the <a class="uri" href="vulcanization" title="wikilink">vulcanization</a> of <a class="uri" href="rubber" title="wikilink">rubber</a>. In analogy to the role of disulfides in proteins, the S-S linkages in rubber are crosslinkers, and strongly affect the <a class="uri" href="rheology" title="wikilink">rheology</a> of the material.</p>
<h2 id="related-compounds">Related compounds</h2>

<p><a href="Thiosulfoxide" title="wikilink">Thiosulfoxides</a> are orthogonally isomeric with disulfides, having the second sulfur branching from the first and not partaking in a continuous chain. i.e. –S(=S)– rather than –S–S–.</p>

<p>Disulfide bonds are analogous but more common than related <a href="peroxide" title="wikilink">peroxides</a> and <a class="uri" href="diselenide" title="wikilink">diselenide</a> bonds. Intermediate compounds of these also exist, for example, <a class="uri" href="thioperoxides" title="wikilink">thioperoxides</a>, also known as oxadisulfide bonds, have formula R<sup>1</sup>OSR<sup>2</sup> (equivalently R<sup>2</sup>SOR<sup>1</sup>). These are isomeric to <a href="sulfoxide" title="wikilink">sulfoxides</a> in a similar manner to the above; i.e. –S(=O)– rather than –S–O–.</p>

<p><a href="Thiuram_disulfide" title="wikilink">Thiuram disulfides</a>, with the formula (R<sub>2</sub>NC(S)S)<sub>2</sub>, are disulfides but they behave distinctly because of the <a class="uri" href="thiocarbonyl" title="wikilink">thiocarbonyl</a> group.</p>

<p>Compounds with three sulfur atoms, e.g., CH<sub>3</sub>S–S–SCH<sub>3</sub>, are called trisulfides, or trisulfide bonds.</p>
<h2 id="references">References</h2>
<h2 id="further-reading">Further reading</h2>
<ul>
<li></li>
<li></li>
<li></li>
<li></li>
<li></li>
<li></li>
<li></li>
<li></li>
<li></li>
<li></li>
</ul>
<h2 id="external-links">External links</h2>
<ul>
<li><a href="http://www.organic-chemistry.org/synthesis/S1S/disulfides.shtm">Synthesis of Disulfides</a></li>
<li><a href="http://www.madsci.org/posts/archives/may98/892764277.Ch.r.html">Disulfide bonds and hair</a></li>
</ul>

<p>"</p>

<p><a href="Category:Organic_disulfides" title="wikilink"> </a> <a href="Category:Protein_structure" title="wikilink">Category:Protein structure</a> <a href="Category:Posttranslational_modification" title="wikilink">Category:Posttranslational modification</a> <a class="uri" href="Category:Sulfur" title="wikilink">Category:Sulfur</a></p>
<section class="footnotes">
<hr/>
<ol>
<li id="fn1">R. J. Cremlyn “An Introduction to Organosulfur Chemistry” John Wiley and Sons: Chichester (1996). ISBN 0-471-95512-4.<a href="#fnref1">↩</a></li>
<li id="fn2"><a href="#fnref2">↩</a></li>
<li id="fn3"><a href="#fnref3">↩</a></li>
<li id="fn4"></li>
<li id="fn5"><a href="http://www.interchim.fr/ft/2/242214.pdf">TCEP technical information</a>, from <a class="uri" href="Interchim" title="wikilink">Interchim</a><a href="#fnref5">↩</a></li>
<li id="fn6"><a href="#fnref6">↩</a></li>
<li id="fn7"><a href="#fnref7">↩</a></li>
<li id="fn8">For example the conversion of <em>di-o-nitrophenyl disulfide</em> to <em>o-nitrophenylsulfur chloride</em> <a href="#fnref8">↩</a></li>
<li id="fn9">Methyl sulfenyl chloride: <a href="#fnref9">↩</a></li>
<li id="fn10"></li>
<li id="fn11"><a href="#fnref11">↩</a></li>
<li id="fn12"><a href="#fnref12">↩</a></li>
<li id="fn13"><a href="#fnref13">↩</a></li>
</ol>
</section>
</body>
</html>
