<html lang="en">
<head>
<meta charset="utf-8"/>
<title offset="1981">Dominance (genetics)</title>
<script src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_SVG.js" type="text/javascript">
</script>
</head>
<body>
<h1>Dominance (genetics)</h1>
<hr/>

<p><strong>Dominance</strong> in <a class="uri" href="genetics" title="wikilink">genetics</a> is a relationship between <a href="allele" title="wikilink">alleles</a> of one <a class="uri" href="gene" title="wikilink">gene</a>, in which the effect on <a class="uri" href="phenotype" title="wikilink">phenotype</a> of one allele masks the contribution of a second allele at the same <a href="locus_(genetics)" title="wikilink">locus</a>.<a class="footnoteRef" href="#fn1" id="fnref1"><sup>1</sup></a><a class="footnoteRef" href="#fn2" id="fnref2"><sup>2</sup></a> The first allele is <strong>dominant</strong> and the second allele is <strong>recessive</strong>. For genes on an <a class="uri" href="autosome" title="wikilink">autosome</a> (any <a class="uri" href="chromosome" title="wikilink">chromosome</a> other than a <a href="sex_chromosome" title="wikilink">sex chromosome</a>), the alleles and their associated traits are <strong>autosomal dominant</strong> or <strong>autosomal recessive</strong>. Dominance is a key concept in <a href="Mendelian_inheritance" title="wikilink">Mendelian inheritance</a> and <a href="classical_genetics" title="wikilink">classical genetics</a>. Often the dominant allele codes for a functional <a class="uri" href="protein" title="wikilink">protein</a> whereas the recessive allele does not.</p>

<p>A classic example of dominance is the inheritance of <a class="uri" href="seed" title="wikilink">seed</a> shape, for example a <a class="uri" href="pea" title="wikilink">pea</a> shape in peas. Peas may be round, associated with allele <em>R</em> or wrinkled, associated with allele <em>r</em>. In this case, three combinations of alleles (<a class="uri" href="genotypes" title="wikilink">genotypes</a>) are possible: <em>RR</em>, <em>Rr</em>, and <em>rr</em>. The <em>RR</em> individuals have round peas and the <em>rr</em> individuals have wrinkled peas. In <em>Rr</em> individuals the <em>R</em> allele masks the presence of the <em>r</em> allele, so these individuals also have round peas. Thus, allele <em>R</em> is dominant to allele <em>r</em>, and allele <em>r</em> is recessive to allele <em>R</em>. This use of upper case letters for dominant alleles and lower caseones for recessive alleles is a widely followed convention.</p>

<p>More generally, where a gene exists in two allelic versions (designated <em>A</em> and <em>a</em>), three combinations of alleles are possible: <em>AA</em>, <em>Aa</em>, and <em>aa</em>. If <em>AA</em> and <em>aa</em> individuals (<a href="zygosity" title="wikilink">homozygotes</a>) show different forms of some trait (phenotypes), and <em>Aa</em> individuals (heterozygotes) show the same phenotype as <em>AA</em> individuals, then allele <em>A</em> is said to <em>dominate</em> or <em>be dominant to</em> or <em>show dominance to</em> allele <em>a</em>, and <em>a</em> is said to <em>be recessive to</em> <em>A</em>.</p>

<p>Dominance is not inherent to an allele. It is a relationship between alleles; one allele can be dominant over a second allele, recessive to a third allele, and codominant to a fourth. Also, an allele may be dominant for a particular aspect of phenotype but not for other aspects influenced by the same gene. Dominance differs from <a class="uri" href="epistasis" title="wikilink">epistasis</a>, a relationship in which an allele of one gene affects the expression of another allele at a different gene.<a class="footnoteRef" href="#fn3" id="fnref3"><sup>3</sup></a></p>
<h2 id="background">Background</h2>

<p>The concept of dominance was introduced by <a href="Gregor_Mendel" title="wikilink">Gregor Mendel</a>. Though Mendel, "The Father of Genetics", first used the term in the 1860s, it was not widely known until the early twentieth century. Mendel observed that, for a variety of traits of garden peas having to do with the appearance of seeds, seed pods, and plants, there were two discrete phenotypes, such as round versus wrinkled seeds, yellow versus green seeds, red versus white flowers or tall versus short plants. When bred separately, the plants always produced the same phenotypes, generation after generation. However, when lines with different phenotypes were crossed (interbred), one and only one of the parental phenotypes showed up in the offspring (green, or round, or red, or tall). However, when these <a href="Hybrid_(biology)" title="wikilink">hybrid</a> plants were crossed, the offspring plants showed the two original phenotypes, in a characteristic 3:1 ratio, the more common phenotype being that of the parental hybrid plants. Mendel reasoned that each parent in the first cross was a homozygote for different alleles (one parent AA and the other parent aa), that each contributed one allele to the offspring, with the result that all of these hybrids were heterozygotes (A), and that one of the two alleles in the hybrid cross dominated expression of the other: A masked a. The final cross between two heterozygotes (Aa X Aa) would produce AA, Aa, and aa offspring in a 1:2:1 genotype ratio with the first two classes showing the (A) phenotype, and the last showing the (a) phenotype, thereby producing the 3:1 phenotype ratio.</p>

<p>Mendel did not use the terms gene, allele, phenotype, genotype, homozygote, and heterozygote, all of which were introduced later. He did introduce the notation of capital and lowercase letters for dominant and recessive alleles, respectively, still in use today.</p>
<h3 id="chromosomes-genes-and-alleles">Chromosomes, genes, and alleles</h3>

<p> </p>

<p>Most animals and some plants have paired <a href="chromosome" title="wikilink">chromosomes</a>, and are described as diploid. They have two versions of each chromosome, one contributed by the mother's <a class="uri" href="ovum" title="wikilink">ovum</a>, and the other by the father's <a class="uri" href="sperm" title="wikilink">sperm</a>, known as <a href="gamete" title="wikilink">gametes</a>, described as haploid, and created through <a class="uri" href="meiosis" title="wikilink">meiosis</a>. These gametes then fuse during <a class="uri" href="fertilization" title="wikilink">fertilization</a> during <a href="sexual_reproduction" title="wikilink">sexual reproduction</a>, into a new single cell <a class="uri" href="zygote" title="wikilink">zygote</a>, which divide twice, resulting in a new organism with the same number of pairs of chromosomes in each (non-gamete) cell as its parents.</p>

<p>Each chromosome of a matching (homologous) pair is structurally similar to the other, and has a very similar <a class="uri" href="DNA" title="wikilink">DNA</a> <a href="sequencing" title="wikilink">sequence</a> (<a href="Locus_(genetics)" title="wikilink">loci</a>, singular locus). The DNA in each chromosome functions as a series of discrete <a class="uri" href="genes" title="wikilink">genes</a> that influence various traits. Thus, each gene also has a corresponding homologue, which may exist in different versions called <a href="allele" title="wikilink">alleles</a>. The alleles at the same locus on the two homologous chromosomes may be identical or different.</p>

<p><a href="Blood_type" title="wikilink">Blood types</a> in humans is determined by a gene that creates either type A; B or O and is located in the long arm of chromosome nine. There are three different alleles that could be present at this locus, but only two can be present in any individual, one inherited from their mother and one from their father.<a class="footnoteRef" href="#fn4" id="fnref4"><sup>4</sup></a></p>

<p>If two alleles of a given gene are identical, the organism is called a homozygote and is said to be homozygous with respect to that gene; if instead the two alleles are different, the organism is a heterozygote and is heterozygous. The genetic makeup of an organism, either at a single locus or over all its genes collectively, is called its <a class="uri" href="genotype" title="wikilink">genotype</a>. The genotype of an organism directly and indirectly affects its molecular, physical, and other traits, which individually or collectively are called its <a class="uri" href="phenotype" title="wikilink">phenotype</a>. At heterozygous gene loci, the two alleles interact to produce the phenotype.</p>
<h2 id="dominance">Dominance</h2>
<h3 id="complete-dominance">Complete dominance</h3>

<p>In complete dominance, the effect of one allele in a heterozygous genotype completely masks the effect of the other. The allele that masks the other is said to be <em>dominant</em> to the latter, and the allele that is masked is said to be <em>recessive</em> to the former.<a class="footnoteRef" href="#fn5" id="fnref5"><sup>5</sup></a> Complete dominance therefore means that the phenotype of the heterozygote is indistinguishable from that of the dominant homozygote.</p>

<p>A classic example of dominance is the inheritance of seed shape (pea shape) in peas. Peas may be round (associated with allele <em>R</em>) or wrinkled (associated with allele <em>r</em>). In this case, three combinations of alleles (<a class="uri" href="genotypes" title="wikilink">genotypes</a>) are possible: <em>RR</em> and <em>rr</em> are homozygous and <em>Rr</em> is heterozygous. The <em>RR</em> individuals have round peas and the <em>rr</em> individuals have wrinkled peas. In <em>Rr</em> individuals the <em>R</em> allele masks the presence of the <em>r</em> allele, so these individuals also have round peas. Thus, allele <em>R</em> is dominant to allele <em>r</em>, and allele <em>r</em> is recessive to allele <em>R</em>.</p>
<h3 id="incomplete-dominance">Incomplete dominance</h3>
<figure><b>(Figure)</b>
<figcaption>This <a href="Punnett_square" title="wikilink">Punnett square</a> illustrates incomplete dominance. In this example, the red petal trait associated with the R <a class="uri" href="allele" title="wikilink">allele</a> blends with the white petal trait of the r allele so that plants with the Rr genotype have pink flowers.</figcaption>
</figure>

<p>Incomplete dominance (also called partial dominance) occurs when the phenotype of the heterozygous phenotype is distinct from and often intermediate to the phenotypes of the homozygous phenotypes. For example, the <a class="uri" href="snapdragon" title="wikilink">snapdragon</a> flower color is either homozygous for red or white. When the red homozygous flower is paired with the white homozygous flower, the result yields a pink snapdragon flower. The pink snapdragon is the result of incomplete dominance. A similar type of incomplete dominance is found in the <a href="four_o'clock_flower" title="wikilink">four o'clock plant</a> wherein pink color is produced when true-bred parents of white and red flowers are crossed. In <a href="quantitative_genetics" title="wikilink">quantitative genetics</a>, where phenotypes are measured and treated numerically, if a heterozygote's phenotype is exactly between (numerically) that of the two homozygotes, the phenotype is said to exhibit <em>no dominance</em> at all, i.e. dominance exists only when the heterozygote's phenotype measure lies closer to one homozygote than the other.</p>

<p>When plants of the F<sub>1</sub> generation are self-pollinated, the phenotypic and genotypic ratio of the F<sub>2</sub> generation will be 1:2:1 (Red:Pink:White) for both generations.<a class="footnoteRef" href="#fn6" id="fnref6"><sup>6</sup></a></p>

<p>See <a href="partial_dominance_hypothesis_theory" title="wikilink">partial dominance hypothesis theory</a>.</p>
<h3 id="co-dominance">Co-dominance</h3>

<p> </p>

<p><strong>Co-dominance</strong> occurs when the contributions of both alleles are visible in the phenotype.</p>

<p>For example, in the <a href="ABO_blood_group_system" title="wikilink">ABO blood group system</a>, chemical modifications to a <a class="uri" href="glycoprotein" title="wikilink">glycoprotein</a> (the H antigen) on the surfaces of blood cells are controlled by three alleles, two of which are co-dominant to each other (<em>I<sup>A</sup></em>, <em>I<sup>B</sup></em>) and dominant over the recessive <em>i</em> at the <a href="ABO_(gene)" title="wikilink">ABO locus</a>. The <em>I<sup>A</sup></em> and <em>I<sup>B</sup></em> alleles produce different modifications. The enzyme coded for by <em>I<sup>A</sup></em> adds an N-acetylgalactosamine to the membrane-bound H antigen. The <em>I<sup>B</sup></em> enzyme adds a galactose. The <em>i</em> allele produces no modification. Thus <em>I<sup>A</sup></em> and <em>I<sup>B</sup></em> alleles are each dominant to <em>i</em> (<em>I<sup>A</sup>I<sup>A</sup></em> and <em>I<sup>A</sup>i</em> individuals both have type A blood, and <em>I<sup>B</sup>I<sup>B</sup></em> and <em>I<sup>B</sup>i</em> individuals both have type B blood. But <em>I<sup>A</sup>I<sup>B</sup></em> individuals have both modifications on their blood cells and thus have type AB blood, so the <em>I<sup>A</sup></em> and <em>I<sup>B</sup></em> alleles are said to be co-dominant.)</p>

<p>Another example occurs at the locus for the <a class="uri" href="Beta-globin" title="wikilink">Beta-globin</a> component of <a class="uri" href="hemoglobin" title="wikilink">hemoglobin</a>, where the three molecular phenotypes of <em>Hb<sup>A</sup>/Hb<sup>A</sup></em>, <em>Hb<sup>A</sup>/Hb<sup>S</sup></em>, and <em>Hb<sup>S</sup>/Hb<sup>S</sup></em> are all distinguishable by <a href="protein_electrophoresis" title="wikilink">protein electrophoresis</a>. (The medical condition produced by the heterozygous genotype is called <em><a href="sickle-cell_trait" title="wikilink">sickle-cell trait</a></em> and is a milder condition distinguishable from <em><a href="sickle-cell_anemia" title="wikilink">sickle-cell anemia</a></em>, thus the alleles show <em>incomplete dominance</em> with respect to anemia, see above). For most gene loci at the molecular level, both alleles are expressed co-dominantly, because both are <a href="Transcription_(genetics)" title="wikilink">transcribed</a> into <a class="uri" href="RNA" title="wikilink">RNA</a>.</p>

<p>Co-dominance, where allelic products co-exist in the phenotype, is different from incomplete dominance, where the quantitative interaction of allele products produces an intermediate phenotype. For example, in Co-dominance, a red homozygous flower and a white homozygous flower will produce offspring that have red and white spots. When plants of the F1 generation are self-pollinated, the phenotypic and genotypic ratio of the F2 generation will be 1:2:1 (Red:Spotted:White). These ratios are the same as those for incomplete dominance. Again, note that this classical terminology is inappropriate – in reality such cases should not be said to exhibit dominance at all.</p>
<h3 id="which-trait-is-dominant">Which trait is <em>dominant</em>?</h3>

<p>By definition, the terms <em>dominant</em> and <em>recessive</em> refer to the genotypic interaction of alleles in producing the phenotype of the heterozygote. The key concept is genetic: Which of the two alleles present in the heterozygote is expressed, such that the organism is phenotypically identical to one of the two homozygotes. It is sometimes convenient to talk about the trait corresponding to the dominant allele as the <em>dominant trait</em>, and the trait corresponding to the hidden allele as the <em>recessive trait</em>. However, this can easily lead to confusion in understanding the concept as phenotypic. For example, to say that "green peas" dominate "yellow peas" confuses inherited genotypes and expressed phenotypes, and will subsequently confuse discussion of the molecular basis of the phenotypic difference.</p>
<h3 id="addressing-common-misconceptions">Addressing common misconceptions</h3>

<p>Dominance is not inherent. One allele can be dominant to a second allele, recessive to a third allele, and codominant to a fourth.</p>

<p>Dominance is unrelated to the nature of the phenotype itself, that is, whether it is regarded as "normal" or "abnormal," "standard" or "nonstandard," "healthy" or "diseased," "stronger" or "weaker," or more or less extreme. A dominant allele may account for any of these trait types.</p>

<p>Dominance does not determine whether an allele is deleterious, neutral or advantageous. However, <a href="Natural_selection" title="wikilink"> selection</a> works through differential reproduction of phenotypes, and dominance affects the exposure of alleles in phenotypes, and hence the rate of change in allele frequencies under selection. Deleterious recessive alleles may persist in a population at low frequencies, with most copies carried in heterozygotes, at no cost to those individuals. These rare recessives are the basis for many hereditary <a href="genetic_disorder" title="wikilink">genetic disorders</a>.</p>

<p>Dominance is also unrelated to the distribution of alleles in the population. Some dominant alleles are extremely common, while others are extremely rare. The most common allele in a population may be recessive when combined with some rare variants.</p>
<h2 id="nomenclature">Nomenclature</h2>

<p>In genetics, symbols began as algebraic placeholders. When one allele is dominant to another, the oldest convention is to symbolize the dominant allele with a capital letter. The recessive allele is assigned the same letter in lower case. In the pea example, once the dominance relationship between the two alleles is known, it is possible to designate the dominant allele that produces a round shape by a capital-letter symbol <strong>R</strong>, and the recessive allele that produces a wrinkled shape by a lower-case symbol <strong>r</strong>. The homozygous dominant, heterozygous, and homozygous recessive genotypes are then written <strong>RR</strong>, <strong>Rr</strong>, and <strong>rr</strong>, respectively. It would also be possible to designate the two alleles as <strong>W</strong> and <strong>w</strong>, and the three genotypes <strong>WW</strong>, <strong>Ww</strong>, and <strong>ww</strong>, the first two of which produced round peas and the third wrinkled peas. Note that the choice of "<strong>R</strong>" or "<strong>W</strong>" as the symbol for the dominant allele does not pre-judge whether the allele causing the "round" or "wrinkled" phenotype when homozygous is the dominant one.</p>

<p>In most cases, genes have several alleles. If multiple dominant alleles exist, they may be designated with different <a href="superscript" title="wikilink">superscripts</a> over the upper case letter, for example <em>A<sup>x</sup></em>. If multiple recessive alleles exist, they are designated with different superscripts on the lower case <em>a</em>, for example <em>a<sup>x</sup></em>.</p>

<p>To indicate that two alleles are co-dominant (and that neither is dominant over the other), they are both written in upper-case, with a superscript to indicate the different alleles.</p>

<p>Rules of genetic nomenclature have evolved as genetics has become more complex. Committees have standardized the rules for some species, but not for all. Rules for one species may differ somewhat from the rules for a different species.<a class="footnoteRef" href="#fn7" id="fnref7"><sup>7</sup></a><a class="footnoteRef" href="#fn8" id="fnref8"><sup>8</sup></a></p>
<h2 id="relationship-to-other-genetic-concepts">Relationship to other genetic concepts</h2>
<h3 id="multiple-alleles">Multiple alleles</h3>

<p>Although any individual of a diploid organism has at most two different alleles at any one locus (barring aneuploidies), most genes exist in a large number of allelic versions in the population as a whole. If the alleles have different effects on the phenotype, sometimes their dominance interactions with each other can be described as a series.</p>

<p>For example, coat color in domestic cats is affected by a series of alleles of the <em>TYR</em> gene (which encodes the enzyme <a class="uri" href="tyrosinase" title="wikilink">tyrosinase</a>). The alleles <em>C</em>, <em>c<sup>b</sup></em>, <em>c<sup>s</sup></em>, and <em>c<sup>a</sup></em> (full colour, <a href="Burmese_(cat)#Genetics" title="wikilink">Burmese</a>, <a href="Siamese_(cat)" title="wikilink">Siamese</a>, and <a class="uri" href="albino" title="wikilink">albino</a>, respectively) produce different levels of pigment and hence different levels of colour dilution. The <em>C</em> allele (full colour) is completely dominant over the last three and the <em>c<sup>a</sup></em> allele (albino) is completely recessive to the first three. <a class="footnoteRef" href="#fn9" id="fnref9"><sup>9</sup></a> <a class="footnoteRef" href="#fn10" id="fnref10"><sup>10</sup></a> <a class="footnoteRef" href="#fn11" id="fnref11"><sup>11</sup></a></p>
<h3 id="autosomal-versus-sex-linked-dominance">Autosomal <em>versus</em> sex-linked dominance</h3>

<p>In humans and other <a class="uri" href="mammal" title="wikilink">mammal</a> species, <a href="sex-determination_system" title="wikilink">sex is determined</a> by two sex chromosomes called the <a href="X_chromosome" title="wikilink">X chromosome</a> and the <a href="Y_chromosome" title="wikilink">Y chromosome</a>. Human females are typically <strong>XX</strong>; males are typically <strong>XY</strong>. The remaining pairs of chromosome are found in both sexes and are called <a href="autosome" title="wikilink">autosomes</a>; genetic traits due to loci on these chromosomes are described as autosomal, and may be dominant or recessive. Genetic traits on the <strong>X</strong> and <strong>Y</strong> chromosomes are called sex-linked, because they are linked to sex chromosomes, not because they are characteristic of one sex or the other. In practice, the term almost always refers to <strong>X</strong>-linked traits and a great many such traits (such as red-green colour vision deficiency) are not affected by sex. Females have two copies of every gene locus found on the X chromosome, just as for the autosomes, and the same dominance relationships apply. Males however have only one copy of each X chromosome gene locus, and are described as <a href="zygosity" title="wikilink">hemizygous</a> for these genes. The Y chromosome is much smaller than the <strong>X</strong>, and contains a much smaller set of genes, including, but not limited to, those that influence 'maleness', such as the <strong><a href="http://www.ncbi.nlm.nih.gov/omim/480000">SRY</a></strong> gene for <a href="testis_determining_factor" title="wikilink">testis determining factor</a>. Dominance rules for sex-linked gene loci are determined by their behavior in the female: because the male has only one allele (except in the case of certain types of Y chromosome <a class="uri" href="aneuploidy" title="wikilink">aneuploidy</a>), that allele is always expressed regardless of whether it is dominant or recessive.</p>
<h3 id="epistasis">Epistasis</h3>

<p><a class="uri" href="Epistasis" title="wikilink">Epistasis</a> ["<em>epi</em> + <em>stasis</em> = to sit on top"] is an interaction between alleles at two <em>different</em> gene loci that affect a single trait, which may sometimes resemble a dominance interaction between two <em>different</em> alleles at the <em>same</em> locus. Epistasis modifies the characteristic <a href="Dihybrid_cross" title="wikilink">9:3:3:1</a> ratio expected for two non-epistatic genes. For two loci, 14 classes of epistatic interactions are recognized. As an example of <strong>''recessive epistasis</strong>'', one gene locus may determine whether a flower pigment is yellow (<strong>AA</strong> or <strong>Aa</strong>) or green (<strong>aa</strong>), while another locus determines whether the pigment is produced (<strong>BB</strong> or <strong>Bb</strong>) or not (<strong>bb</strong>). In a <strong>bb</strong> plant, the flowers will be white, irrespective of the genotype of the other locus as <strong>AA</strong>, <strong>Aa</strong>, or <strong>aa</strong>. The <strong>bb</strong> combination is <em>not</em> dominant to the <strong>A</strong> allele: rather, the <strong>B</strong> gene shows <strong><em>recessive epistasis</em></strong> to the <strong>A</strong> gene, because the <strong>B</strong> locus when homozygous for the <em>recessive</em> allele (<strong>bb</strong>) suppresses phenotypic expression of the <strong>A</strong> locus. In a cross between two <strong>AaBb</strong> plants, this produces a characteristic <strong>9:3:4</strong> ratio, in this case of yellow : green : white flowers.</p>

<p>In <strong><em>dominant epistasis</em></strong>, one gene locus may determine yellow or green pigment as in the previous example: <strong>AA</strong> and <strong>Aa</strong> are yellow, and <strong>aa</strong> are green. A second locus determines whether a pigment precursor is produced (<strong>dd</strong>) or not (<strong>DD</strong> or <strong>Dd</strong>). Here, in a <strong>DD</strong> or <strong>Dd</strong> plant, the flowers will be colorless irrespective of the genotype at the <strong><em>A</em></strong> locus, because of the epistatic effect of the dominant <strong>D</strong> allele. Thus, in a cross between two <strong>AaDd</strong> plants, 3/4 of the plants will be colorless, and the yellow and green phenotypes are expressed only in <strong>dd</strong> plants. This produces a characteristic <strong>12:3:1</strong> ratio of white : yellow : green plants.</p>

<p><strong><em>Supplementary epistasis</em></strong> occurs when two loci affect the same phenotype. For example, if pigment color is produced by <strong>CC</strong> or <strong>Cc</strong> but not <strong>cc</strong>, and by <strong>DD</strong> or <strong>Dd</strong> but not <strong>dd</strong>, then pigment is not produced in any genotypic combination with either <strong>cc</strong> <em>or</em> <strong>dd</strong>. That is, <em>both</em> loci must have at least one dominant allele to produce the phenotype. This produces a characteristic <strong>9:7</strong> ratio of pigmented to unpigmented plants. <strong><em>Complementary epistasis</em></strong> in contrast produces an unpigmented plant if and only if the genotype is <strong>cc</strong> <em>and</em> <strong>dd</strong>, and the characteristic ratio is <strong>15:1</strong> between pigmented and unpigmented plants.<a class="footnoteRef" href="#fn12" id="fnref12"><sup>12</sup></a></p>

<p>Classical genetics considered epistatic interactions between two genes at a time. It is now evident from molecular genetics that all gene loci are involved in complex interactions with many other genes (e.g., metabolic pathways may involve scores of genes), and that this creates epistatic interactions that are much more complex than the classic two-locus models.</p>
<h3 id="hardy-weinberg-principle-estimation-of-carrier-frequency">Hardy-Weinberg principle (Estimation of carrier frequency)</h3>

<p>The frequency of the homozygous state (which is the carrier state for a recessive trait) can be estimated using the <a href="Hardy-Weinberg_formula" title="wikilink">Hardy-Weinberg formula</a>

<math display="block" id="Dominance_(genetics):0">
 <semantics>
  <mrow>
   <mrow>
    <msup>
     <mi>p</mi>
     <mn>2</mn>
    </msup>
    <mo>+</mo>
    <mrow>
     <mn>2</mn>
     <mi>p</mi>
     <mi>q</mi>
    </mrow>
    <mo>+</mo>
    <msup>
     <mi>q</mi>
     <mn>2</mn>
    </msup>
   </mrow>
   <mo>=</mo>
   <mn>1</mn>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <eq></eq>
    <apply>
     <plus></plus>
     <apply>
      <csymbol cd="ambiguous">superscript</csymbol>
      <ci>p</ci>
      <cn type="integer">2</cn>
     </apply>
     <apply>
      <times></times>
      <cn type="integer">2</cn>
      <ci>p</ci>
      <ci>q</ci>
     </apply>
     <apply>
      <csymbol cd="ambiguous">superscript</csymbol>
      <ci>q</ci>
      <cn type="integer">2</cn>
     </apply>
    </apply>
    <cn type="integer">1</cn>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   p^{2}+2pq+q^{2}=1
  </annotation>
 </semantics>
</math>

</p>

<p>This formula applies to a gene with exactly two alleles and relates the frequencies of those alleles in a large population to the frequencies of their three genotypes in that population.</p>

<p>For example, if <em>p</em> is the frequency of allele <strong>A</strong>, and <em>q</em> is the frequency of allele <strong>a</strong> then the terms <em>p</em><sup>2</sup>, 2<em>pq</em>, and <em>q</em><sup>2</sup> are the frequencies of the genotypes <strong>AA</strong>, <strong>Aa</strong> and <strong>aa</strong> respectively. Since the gene has only two alleles, all alleles must be either <strong>A</strong> or <strong>a</strong> and 

<math display="inline" id="Dominance_(genetics):1">
 <semantics>
  <mrow>
   <mrow>
    <mi>p</mi>
    <mo>+</mo>
    <mi>q</mi>
   </mrow>
   <mo>=</mo>
   <mn>1</mn>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <eq></eq>
    <apply>
     <plus></plus>
     <ci>p</ci>
     <ci>q</ci>
    </apply>
    <cn type="integer">1</cn>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   p+q=1
  </annotation>
 </semantics>
</math>

. Now, if <strong>A</strong> is completely dominant to <strong>a</strong> then the frequency of the carrier genotype <strong>Aa</strong> cannot be directly observed (since it has the same traits as the homozygous genotype <strong>AA</strong>), however it can be estimated from the frequency of the recessive trait in the population, since this is the same as that of the homozygous genotype <strong>aa</strong>. i.e. the individual allele frequencies can be estimated: <mtpl> }}</mtpl>, 

<math display="inline" id="Dominance_(genetics):2">
 <semantics>
  <mrow>
   <mi>p</mi>
   <mo>=</mo>
   <mrow>
    <mn>1</mn>
    <mi mathvariant="normal">–</mi>
    <mi>q</mi>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <eq></eq>
    <ci>p</ci>
    <apply>
     <times></times>
     <cn type="integer">1</cn>
     <ci>normal-–</ci>
     <ci>q</ci>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   p=1–q
  </annotation>
 </semantics>
</math>

, and from those the frequency of the carrier genotype can be derived: <mtpl> 2<em>pq</em>}}</mtpl>.</p>

<p>This formula relies on a number of <a href="Hardy-Weinberg_formula#Deviations_from_Hardy–Weinberg_equilibrium" title="wikilink">assumptions</a> and an accurate estimate of the frequency of the recessive trait. In general, any real-world situation will deviate from these assumptions to some degree, introducing corresponding inaccuracies into the estimate. If the recessive trait is rare, then it will be hard to estimate its frequency accurately, as a very large sample size will be needed.</p>
<h2 id="molecular-mechanisms">Molecular mechanisms</h2>

<p>The molecular basis of dominance was unknown to Mendel. It is now understood that a gene locus includes a long series (hundreds to thousands) of <a href="nucleobase" title="wikilink">bases</a> or <a href="nucleotide" title="wikilink">nucleotides</a> of <a href="dNA" title="wikilink">deoxyribonucleic acid</a> (DNA) at a particular point on a chromosome. The <a href="central_dogma_of_molecular_biology" title="wikilink">central dogma of molecular biology</a> states that "<em><a class="uri" href="DNA" title="wikilink">DNA</a> makes <a class="uri" href="RNA" title="wikilink">RNA</a> makes <a class="uri" href="protein" title="wikilink">protein</a></em>", that is, that DNA is <a href="Transcription_(genetics)" title="wikilink">transcribed</a> to make an RNA copy, and RNA is <a href="translation" title="wikilink">translated</a> to make a protein. In this process, different alleles at a locus may or may not be transcribed, and if transcribed may be translated to slightly different versions of the same protein (called <a href="protein_isoform" title="wikilink">isoforms</a>). Proteins often function as <a href="enzyme" title="wikilink">enzymes</a> that catalyze chemical reactions in the cell, which directly or indirectly produce phenotypes. In any diploid organism, the DNA sequences of the two alleles present at any gene locus may be identical (homozygous) or different (heterozygous). Even if the gene locus is heterozygous at the level of the DNA sequence, the proteins made by each allele may be identical. In the absence of any difference between the protein products, neither allele can be said to be dominant (see <em>co-dominance</em>, above). Even if the two protein products are slightly different (<a href="allozyme" title="wikilink">allozymes</a>), it is likely that they produce the same phenotype with respect to enzyme action, and again neither allele can be said to be dominant.</p>
<h3 id="loss-of-function-and-haplosufficiency">Loss of function and haplosufficiency</h3>

<p>Dominance typically occurs when one of the two alleles is non-functional at the molecular level, that is, it is not transcribed or else does not produce a functional protein product. This can be the result of a <a class="uri" href="mutation" title="wikilink">mutation</a> that alters the DNA sequence of the allele. An organism homozygous for the non-functional allele will generally show a distinctive phenotype, due to the absence of the protein product. For example, in humans and other organisms, the unpigmented skin of the <a href="albinism" title="wikilink">albino</a> phenotype<a class="footnoteRef" href="#fn13" id="fnref13"><sup>13</sup></a> results when an individual is homozygous for an allele that prevents synthesis of the skin pigment protein <a class="uri" href="melanin" title="wikilink">melanin</a>. It is important to understand that it is not the lack of function that allows the allele to be described as recessive: this is the interaction with the alternative allele in the heterozygote. Three general types of interaction are possible:</p>
<ol>
<li>In the typical case, the single functional allele makes sufficient protein to produce a phenotype identical to that of the homozygote: this is called <em><a href="haplo-sufficiency" title="wikilink">haplosufficiency</a></em>. For example, suppose the standard amount of enzyme produced in the functional homozygote is 100%, with the two functional alleles contributing 50% each. The single functional allele in the heterozygote produces 50% of the standard amount of enzyme, which is sufficient to produce the standard phenotype. If the heterozygote and the functional-allele homozygote have identical phenotypes, the functional allele is dominant to the non-functional allele. This occurs at the albino gene locus: the heterozygote produces sufficient enzyme to convert the pigment precursor to melanin, and the individual has standard pigmentation.</li>
<li>Less commonly, the presence of a single functional allele gives a phenotype that is not normal but less severe than that of the non-functional homozygote. This occurs when the functional allele is not haplo-sufficient. The terms haplo-insufficiency and incomplete dominance are typically applied to these cases. The intermediate interaction occurs where the heterozygous genotype produces a phenotype intermediate between the two homozygotes. Depending on which of the two homozygotes the heterozygote most resembles, one allele is said to show <em>incomplete dominance</em> over the other. For example, in humans the <em>Hb</em> gene locus is responsible for the Beta-chain protein (<a class="uri" href="HBB" title="wikilink">HBB</a>) that is one of the two <a class="uri" href="globin" title="wikilink">globin</a> proteins that make up the blood pigment <a class="uri" href="hemoglobin" title="wikilink">hemoglobin</a><a class="footnoteRef" href="#fn14" id="fnref14"><sup>14</sup></a> Many people are homozygous for an allele called <em>Hb<sup>A</sup></em>; some persons carry an alternative allele called <em>Hb<sup>S</sup></em>, either as homozygotes or heterozygotes. The hemoglobin molecules of <em>Hb<sup>S</sup></em>/<em>Hb<sup>S</sup></em> homozygotes undergo a change in shape that distorts the morphology of the <a href="red_blood_cells" title="wikilink">red blood cells</a>, and causes a severe, life-threatening form of <a class="uri" href="anemia" title="wikilink">anemia</a> called <a href="sickle-cell_anemia" title="wikilink">sickle-cell anemia</a>. Persons heterozygous <em>Hb<sup>A</sup></em>/<em>Hb<sup>S</sup></em> for this allele have a much less severe form of anemia called <a href="sickle-cell_trait" title="wikilink">sickle-cell trait</a>. Because the disease phenotype of <em>Hb<sup>A</sup></em>/<em>Hb<sup>S</sup></em> heterozygotes is more similar to but not identical to the <em>Hb<sup>A</sup></em>/<em>Hb<sup>A</sup></em> homozygote, the <em>Hb<sup>A</sup></em> allele is said to be <em>incompletely dominant</em> to the <em>Hb<sup>S</sup></em> allele.</li>
<li>Rarely, a single functional allele in the heterozygote may produce insufficient gene product for any function of the gene, and the phenotype resembles that of the homozygote for the non-functional allele. This complete <em><a class="uri" href="haploinsufficiency" title="wikilink">haploinsufficiency</a></em> is very unusual. In these cases the non-functional allele would be said to be dominant to the functional allele. This situation may occur when the non-functional allele produces a defective protein that interferes with the proper function of the protein produced by the standard allele. The presence of the defective protein "dominates" the standard protein, and the disease phenotype of the heterozygote more closely resembles that of the homozygote for two defective alleles. Note that the term dominant is often incorrectly applied to defective alleles whose homozygous phenotype has not been examined, but which cause a distinct phenotype when heterozygous with the normal allele. This phenomenon occurs in a number of <a href="trinucleotide_repeat" title="wikilink">trinucleotide repeat</a> diseases: for an example and more details see <a href="Huntington_Disease" title="wikilink">Huntington Disease</a>.<a class="footnoteRef" href="#fn15" id="fnref15"><sup>15</sup></a></li>
</ol>
<h3 id="dominant-negative-mutations">Dominant-negative mutations</h3>

<p>Many proteins are normally active in the form of a multimer, an aggregate of multiple copies of the same protein, otherwise known as a <a href="Multiprotein_complex#Homomultimeric_and_heteromultimeric_proteins" title="wikilink">homomultimeric protein or homooligomeric protein</a>. In fact, a majority of the 83,000 different enzymes from 9800 different organisms in the BRENDA Enzyme Database<a class="footnoteRef" href="#fn16" id="fnref16"><sup>16</sup></a> represent homooligomers.<a class="footnoteRef" href="#fn17" id="fnref17"><sup>17</sup></a> When the wild-type version of the protein is present along with a mutant version, a mixed multimer can be formed. A mutation that leads to a mutant protein that disrupts the activity of the wild-type protein in the multimer is a dominant-negative mutation.</p>

<p>A dominant-negative mutation may arise in a human somatic cell and provide a proliferative advantage to the mutant cell, leading to its clonal expansion. For instance, a dominant-negative mutation in a gene necessary for the normal process of programmed cell death (<a class="uri" href="Apoptosis" title="wikilink">Apoptosis</a>) in response to DNA damage can make the cell resistant to apoptosis. This will allow proliferation of the clone even when excessive DNA damage is present. Such dominant-negative mutations occur in the tumor suppressor gene <em><a class="uri" href="p53" title="wikilink">p53</a></em>.<a class="footnoteRef" href="#fn18" id="fnref18"><sup>18</sup></a><a class="footnoteRef" href="#fn19" id="fnref19"><sup>19</sup></a> The P53 wild-type protein is normally present as a four-protein multimer (oligotetramer). Dominant-negative <em>p53</em> mutations occur in a number of different types of cancer and pre-cancerous lesions (e.g. brain tumors, breast cancer, oral pre-cancerous lesions and oral cancer).<a class="footnoteRef" href="#fn20" id="fnref20"><sup>20</sup></a></p>

<p>Dominant-negative mutations also occur in other tumor suppressor genes. For instance two dominant-negative germ line mutations were identified in the <a href="Ataxia_telangiectasia_mutated" title="wikilink">Ataxia telangiectasia mutated</a> (ATM) gene which increases susceptibility to breast cancer.<a class="footnoteRef" href="#fn21" id="fnref21"><sup>21</sup></a> Dominant negative mutations of the transcription factor <a href="CEBPA" title="wikilink">C/EBPα</a> can cause acute myeloid leukemia.<a class="footnoteRef" href="#fn22" id="fnref22"><sup>22</sup></a> Inherited dominant negative mutations can also increase the risk of diseases other than cancer. Dominant-negative mutations in <a href="Peroxisome_proliferator-activated_receptor_gamma" title="wikilink">Peroxisome proliferator-activated receptor gamma</a> (PPARγ) are associated with severe insulin resistance, diabetes mellitus and hypertension.<a class="footnoteRef" href="#fn23" id="fnref23"><sup>23</sup></a></p>

<p>Dominant-negative mutations have also been described in organisms other than humans. In fact, the first study reporting a <strong>mutant protein</strong> inhibiting the normal function of a wild-type protein in a mixed multimer was with the bacteriophage T4 tail fiber protein GP37.<a class="footnoteRef" href="#fn24" id="fnref24"><sup>24</sup></a> Mutations that produce a truncated protein rather than a full-length mutant protein seem to have the strongest dominant-negative effect in the studies of P53, ATM, C/EBPα, and bacteriophage T4 GP37.</p>
<h2 id="dominant-and-recessive-genetic-diseases-in-humans">Dominant and recessive genetic diseases in humans</h2>

<p>In humans, many genetic traits or diseases are classified simply as "dominant" or "recessive". Especially with respect to so-called recessive diseases, this can oversimplify the underlying molecular basis and lead to misunderstanding of the nature of dominance. For example, the genetic disease <a class="uri" href="phenylketonuria" title="wikilink">phenylketonuria</a> (PKU)<a class="footnoteRef" href="#fn25" id="fnref25"><sup>25</sup></a> results from any of a large number (&gt;60) of alleles at the gene locus for the enzyme <a href="phenylalanine_hydroxylase" title="wikilink">phenylalanine hydroxylase</a> (<strong>PAH</strong>).<a class="footnoteRef" href="#fn26" id="fnref26"><sup>26</sup></a> Many of these alleles produce little or no <strong>PAH</strong>, as a result of which the substrate <a class="uri" href="phenylalanine" title="wikilink">phenylalanine</a> (Phe) and its metabolic byproducts accumulate in the <a href="central_nervous_system" title="wikilink">central nervous system</a> and can cause severe <a href="intellectual_disability" title="wikilink">intellectual disability</a> if untreated.</p>

<p>The genotypes and phenotypic consequences of interactions among three alleles are shown in the following table:<a class="footnoteRef" href="#fn27" id="fnref27"><sup>27</sup></a></p>
<table>
<thead>
<tr class="header">
<th style="text-align: left;">
<p>Genotype</p></th>
<th style="text-align: left;">
<p><strong>PAH</strong> activity</p></th>
<th style="text-align: left;">
<p>[<strong>Phe</strong>] conc</p></th>
<th style="text-align: left;">
<p>PKU ?</p></th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;">
<p><strong>AA</strong></p></td>
<td style="text-align: left;">
<p>100%</p></td>
<td style="text-align: left;">
<p>60 uM</p></td>
<td style="text-align: left;">
<p>No</p></td>
</tr>
<tr class="even">
<td style="text-align: left;">
<p><strong>AB</strong></p></td>
<td style="text-align: left;">
<p>30%</p></td>
<td style="text-align: left;">
<p>120 uM</p></td>
<td style="text-align: left;">
<p>No</p></td>
</tr>
<tr class="odd">
<td style="text-align: left;">
<p><strong>CC</strong></p></td>
<td style="text-align: left;">
<p>5%</p></td>
<td style="text-align: left;">
<p>200 ~ 300 uM</p></td>
<td style="text-align: left;">
<p>Hyperphenylalaninemia</p></td>
</tr>
<tr class="even">
<td style="text-align: left;">
<p><strong>BB</strong></p></td>
<td style="text-align: left;">
<p>0.3%</p></td>
<td style="text-align: left;">
<p>600 ~ 2400 uM</p></td>
<td style="text-align: left;">
<p>Yes</p></td>
</tr>
</tbody>
</table>

<p>In unaffected persons homozygous for a standard functional allele (<strong>AA</strong>), <strong>PAH</strong> activity is standard (100%), and the concentration of phenylalanine in the blood [<strong>Phe</strong>] is about 60 uM. In untreated persons homozygous for one of the PKU alleles (<strong>BB</strong>), <strong>PAH</strong> activity is close to zero, [Phe] ten to forty times standard, and the individual manifests PKU.</p>

<p>In the <strong>AB</strong> heterozygote, <strong>PAH</strong> activity is only 30% (not 50%) of standard, blood [<strong>Phe</strong>] is elevated two-fold, and the person does not manifest PKU. Thus, the <strong>A</strong> allele is dominant to the <strong>B</strong> allele with respect to PKU, but the <strong>B</strong> allele is incompletely dominant to the <strong>A</strong> allele with respect to its molecular effect, determination of <strong>PAH</strong> activity level (0.3% </p>
<section class="footnotes">
<hr/>
<ol>
<li id="fn1"><a href="#fnref1">↩</a></li>
<li id="fn2"><a href="#fnref2">↩</a></li>
<li id="fn3"><a href="#fnref3">↩</a></li>
<li id="fn4"><a href="#fnref4">↩</a></li>
<li id="fn5"><a href="#fnref5">↩</a></li>
<li id="fn6"><a href="#fnref6">↩</a></li>
<li id="fn7"><a href="http://www.informatics.jax.org/mgihome/nomen/gene.shtml">1</a>, Online 'Guidelines for nomenclature of genes, genetic markers, alleles, and mutations in mouse and rat'<a href="#fnref7">↩</a></li>
<li id="fn8"><a href="http://www.maizegdb.org/maize_nomenclature.php">2</a>, Online 'A standard for maize genetic nomenclature'<a href="#fnref8">↩</a></li>
<li id="fn9"><a href="#fnref9">↩</a></li>
<li id="fn10"><a href="#fnref10">↩</a></li>
<li id="fn11"><a href="#fnref11">↩</a></li>
<li id="fn12"><a href="#fnref12">↩</a></li>
<li id="fn13"><a href="#fnref13">↩</a></li>
<li id="fn14"></li>
<li id="fn15"><a href="#fnref15">↩</a></li>
<li id="fn16"><a href="#fnref16">↩</a></li>
<li id="fn17"><a href="#fnref17">↩</a></li>
<li id="fn18"><a href="#fnref18">↩</a></li>
<li id="fn19"><a href="#fnref19">↩</a></li>
<li id="fn20"></li>
<li id="fn21"><a href="#fnref21">↩</a></li>
<li id="fn22"><a href="#fnref22">↩</a></li>
<li id="fn23"><a href="#fnref23">↩</a></li>
<li id="fn24"><a href="#fnref24">↩</a></li>
<li id="fn25"><a href="#fnref25">↩</a></li>
<li id="fn26"><a href="#fnref26">↩</a></li>
<li id="fn27"><a href="#fnref27">↩</a></li>
</ol>
</section>
</body>
</html>
