<html lang="en">
<head>
<meta charset="utf-8"/>
<title offset="754">Oxidative phosphorylation</title>
<script src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_SVG.js" type="text/javascript">
</script>
</head>
<body>
<h1>Oxidative phosphorylation</h1>
<hr/>
<figure><b>(Figure)</b>
<figcaption>The <a href="electron_transport_chain" title="wikilink">electron transport chain</a> in the <a href="cell_(biology)" title="wikilink">cell</a> is the site of oxidative phosphorylation in <a href="prokaryote" title="wikilink">prokaryotes</a>. The NADH and succinate generated in the <a href="citric_acid_cycle" title="wikilink">citric acid cycle</a> are oxidized, releasing energy to power the <a href="ATP_synthase" title="wikilink">ATP synthase</a>.</figcaption>
</figure>

<p><strong>Oxidative phosphorylation</strong> (or OXPHOS in short) is the <a href="metabolic_pathway" title="wikilink">metabolic pathway</a> in which the <a class="uri" href="mitochondria" title="wikilink">mitochondria</a> in <a href="Cell_(biology)" title="wikilink">cells</a> use their structure, <a href="enzyme" title="wikilink">enzymes</a>, and <a class="uri" href="energy" title="wikilink">energy</a> released by the <a href="redox" title="wikilink">oxidation</a> of <a href="nutrient" title="wikilink">nutrients</a> to reform <a href="Adenosine_triphosphate" title="wikilink">ATP</a>. Although the many forms of life on earth use a range of different nutrients, ATP is the molecule that supplies energy to <a class="uri" href="metabolism" title="wikilink">metabolism</a>. Almost all <a href="aerobic_organism" title="wikilink">aerobic organisms</a> carry out oxidative phosphorylation. This pathway is probably so pervasive because it is a highly efficient way of releasing energy, compared to alternative <a href="fermentation_(biochemistry)" title="wikilink">fermentation</a> processes such as anaerobic <a class="uri" href="glycolysis" title="wikilink">glycolysis</a>.</p>

<p>During oxidative phosphorylation, electrons are transferred from <a href="reducing_agent" title="wikilink">electron donors</a> to <a href="oxidizing_agent" title="wikilink">electron acceptors</a> such as <a class="uri" href="oxygen" title="wikilink">oxygen</a>, in <a href="redox_reaction" title="wikilink">redox reactions</a>. These redox reactions release energy, which is used to form ATP. In <a href="eukaryote" title="wikilink">eukaryotes</a>, these redox reactions are carried out by a series of <a href="protein_complex" title="wikilink">protein complexes</a> within the cell's intermembrane wall <a href="mitochondrion" title="wikilink">mitochondria</a>, whereas, in <a href="prokaryote" title="wikilink">prokaryotes</a>, these proteins are located in the cells' intermembrane space. These linked sets of proteins are called <a href="electron_transport_chain" title="wikilink">electron transport chains</a>. In eukaryotes, five main protein complexes are involved, whereas in prokaryotes many different enzymes are present, using a variety of electron donors and acceptors.</p>

<p>The energy released by electrons flowing through this electron transport chain is used to transport <a href="proton" title="wikilink">protons</a> across the <a href="inner_mitochondrial_membrane" title="wikilink">inner mitochondrial membrane</a>, in a process called <em><a href="electron_transport" title="wikilink">electron transport</a></em>. This generates <a href="potential_energy" title="wikilink">potential energy</a> in the form of a <a class="uri" href="pH" title="wikilink">pH</a> gradient and an <a href="Membrane_potential" title="wikilink">electrical potential</a> across this membrane. This store of energy is tapped by allowing protons to flow back across the membrane and down this gradient, through a large <a class="uri" href="enzyme" title="wikilink">enzyme</a> called <a href="ATP_synthase" title="wikilink">ATP synthase</a>; this process is known as <a class="uri" href="chemiosmosis" title="wikilink">chemiosmosis</a>. This enzyme uses this energy to generate ATP from <a href="adenosine_diphosphate" title="wikilink">adenosine diphosphate</a> (ADP), in a <a class="uri" href="phosphorylation" title="wikilink">phosphorylation</a> reaction. This reaction is driven by the proton flow, which forces the <a class="uri" href="rotation" title="wikilink">rotation</a> of a part of the enzyme; the ATP synthase is a rotary mechanical motor.</p>

<p>Although oxidative phosphorylation is a vital part of <a class="uri" href="metabolism" title="wikilink">metabolism</a>, it produces <a href="reactive_oxygen_species" title="wikilink">reactive oxygen species</a> such as <a class="uri" href="superoxide" title="wikilink">superoxide</a> and <a href="hydrogen_peroxide" title="wikilink">hydrogen peroxide</a>, which lead to propagation of <a href="Radical_(chemistry)" title="wikilink">free radicals</a>, damaging cells and contributing to <a class="uri" href="disease" title="wikilink">disease</a> and, possibly, <a class="uri" href="aging" title="wikilink">aging</a> (<a class="uri" href="senescence" title="wikilink">senescence</a>). The enzymes carrying out this metabolic pathway are also the target of many drugs and poisons that <a href="enzyme_inhibitor" title="wikilink">inhibit</a> their activities.</p>
<h2 id="overview-of-energy-transfer-by-chemiosmosis">Overview of energy transfer by chemiosmosis</h2>

<p>Oxidative phosphorylation works by using <a class="uri" href="energy" title="wikilink">energy</a>-releasing chemical reactions to drive energy-requiring reactions: The two sets of reactions are said to be <em>coupled</em>. This means one cannot occur without the other. The flow of electrons through the electron transport chain, from electron donors such as <a class="uri" href="NADH" title="wikilink">NADH</a> to <a href="electron_acceptor" title="wikilink">electron acceptors</a> such as <a class="uri" href="oxygen" title="wikilink">oxygen</a>, is an <a class="uri" href="exergonic" title="wikilink">exergonic</a> process – it releases energy, whereas the synthesis of ATP is an <a class="uri" href="endergonic" title="wikilink">endergonic</a> process, which requires an input of energy. Both the electron transport chain and the ATP synthase are embedded in a membrane, and energy is transferred from electron transport chain to the ATP synthase by movements of protons across this membrane, in a process called <em><a class="uri" href="chemiosmosis" title="wikilink">chemiosmosis</a></em>.<a class="footnoteRef" href="#fn1" id="fnref1"><sup>1</sup></a> In practice, this is like a simple <a href="Electrical_network" title="wikilink">electric circuit</a>, with a current of protons being driven from the negative N-side of the membrane to the positive P-side by the proton-pumping enzymes of the electron transport chain. These enzymes are like a <a href="battery_(electricity)" title="wikilink">battery</a>, as they perform <a href="Work_(thermodynamics)" title="wikilink">work</a> to drive current through the circuit. The movement of protons creates an <a href="electrochemical_gradient" title="wikilink">electrochemical gradient</a> across the membrane, which is often called the <em>proton-motive force</em>. It has two components: a difference in proton concentration (a H<sup>+</sup> gradient, Δ<a class="uri" href="pH" title="wikilink">pH</a>) and a difference in <a href="electric_potential" title="wikilink">electric potential</a>, with the N-side having a negative charge.<a class="footnoteRef" href="#fn2" id="fnref2"><sup>2</sup></a></p>

<p>ATP synthase releases this stored energy by completing the circuit and allowing protons to flow down the electrochemical gradient, back to the N-side of the membrane.<a class="footnoteRef" href="#fn3" id="fnref3"><sup>3</sup></a> This kinetic energy drives the rotation of part of the enzymes structure and couples this motion to the synthesis of ATP.</p>

<p>The two components of the proton-motive force are <a href="thermodynamic" title="wikilink">thermodynamically</a> equivalent: In mitochondria, the largest part of energy is provided by the potential; in <a class="uri" href="alkaliphile" title="wikilink">alkaliphile</a> bacteria the electrical energy even has to compensate for a counteracting inverse pH difference. Inversely, <a href="chloroplast" title="wikilink">chloroplasts</a> operate mainly on ΔpH. However, they also require a small membrane potential for the kinetics of ATP synthesis. At least in the case of the <a href="Fusobacteria" title="wikilink">fusobacterium</a> <em>P. modestum</em> it drives the counter-rotation of subunits a and c of the F<sub>O</sub> motor of ATP synthase.<a class="footnoteRef" href="#fn4" id="fnref4"><sup>4</sup></a></p>

<p>The amount of energy released by oxidative phosphorylation is high, compared with the amount produced by <a href="anaerobic_fermentation" title="wikilink">anaerobic fermentation</a>. <a class="uri" href="Glycolysis" title="wikilink">Glycolysis</a> produces only 2 ATP molecules, but somewhere between 30 and 36 ATPs are produced by the oxidative phosphorylation of the 10 NADH and 2 succinate molecules made by converting one molecule of <a class="uri" href="glucose" title="wikilink">glucose</a> to carbon dioxide and water,<a class="footnoteRef" href="#fn5" id="fnref5"><sup>5</sup></a> while each cycle of <a href="beta_oxidation" title="wikilink">beta oxidation</a> of a <a href="fatty_acid" title="wikilink">fatty acid</a> yields about 14 ATPs. These ATP yields are theoretical maximum values; in practice, some protons leak across the membrane, lowering the yield of ATP.<a class="footnoteRef" href="#fn6" id="fnref6"><sup>6</sup></a></p>
<h2 id="electron-and-proton-transfer-molecules">Electron and proton transfer molecules</h2>

<p> The electron transport chain carries both protons and electrons, passing electrons from donors to acceptors, and transporting protons across a membrane. These processes use both soluble and protein-bound transfer molecules. In mitochondria, electrons are transferred within the intermembrane space by the water-<a class="uri" href="soluble" title="wikilink">soluble</a> electron transfer protein <a href="cytochrome_c" title="wikilink">cytochrome c</a>.<a class="footnoteRef" href="#fn7" id="fnref7"><sup>7</sup></a> This carries only electrons, and these are transferred by the reduction and oxidation of an <a class="uri" href="iron" title="wikilink">iron</a> atom that the protein holds within a <a class="uri" href="heme" title="wikilink">heme</a> group in its structure. Cytochrome c is also found in some bacteria, where it is located within the <a href="periplasmic_space" title="wikilink">periplasmic space</a>.<a class="footnoteRef" href="#fn8" id="fnref8"><sup>8</sup></a></p>

<p>Within the inner mitochondrial membrane, the <a class="uri" href="lipid" title="wikilink">lipid</a>-soluble electron carrier <a href="coenzyme_Q10" title="wikilink">coenzyme Q10</a> (Q) carries both electrons and protons by a <a class="uri" href="redox" title="wikilink">redox</a> cycle.<a class="footnoteRef" href="#fn9" id="fnref9"><sup>9</sup></a> This small <a href="1,4-Benzoquinone" title="wikilink">benzoquinone</a> molecule is very <a href="hydrophobe" title="wikilink">hydrophobic</a>, so it diffuses freely within the membrane. When Q accepts two electrons and two protons, it becomes reduced to the <em><a href="Hydroquinone" title="wikilink">ubiquinol</a></em> form (QH<sub>2</sub>); when QH<sub>2</sub> releases two electrons and two protons, it becomes oxidized back to the <em>ubiquinone</em> (Q) form. As a result, if two enzymes are arranged so that Q is reduced on one side of the membrane and QH<sub>2</sub> oxidized on the other, ubiquinone will couple these reactions and shuttle protons across the membrane.<a class="footnoteRef" href="#fn10" id="fnref10"><sup>10</sup></a> Some bacterial electron transport chains use different quinones, such as <a href="vitamin_K" title="wikilink">menaquinone</a>, in addition to ubiquinone.<a class="footnoteRef" href="#fn11" id="fnref11"><sup>11</sup></a></p>

<p>Within proteins, electrons are transferred between <a href="Flavin_group" title="wikilink">flavin</a> cofactors,<a class="footnoteRef" href="#fn12" id="fnref12"><sup>12</sup></a><a class="footnoteRef" href="#fn13" id="fnref13"><sup>13</sup></a> iron–sulfur clusters, and cytochromes. There are several types of iron–sulfur cluster. The simplest kind found in the electron transfer chain consists of two iron atoms joined by two atoms of inorganic <a class="uri" href="sulfur" title="wikilink">sulfur</a>; these are called [2Fe–2S] clusters. The second kind, called [4Fe–4S], contains a cube of four iron atoms and four sulfur atoms. Each iron atom in these clusters is coordinated by an additional <a href="amino_acid" title="wikilink">amino acid</a>, usually by the sulfur atom of <a class="uri" href="cysteine" title="wikilink">cysteine</a>. Metal ion cofactors undergo redox reactions without binding or releasing protons, so in the electron transport chain they serve solely to transport electrons through proteins. Electrons move quite long distances through proteins by hopping along chains of these cofactors.<a class="footnoteRef" href="#fn14" id="fnref14"><sup>14</sup></a> This occurs by <a href="quantum_tunnelling" title="wikilink">quantum tunnelling</a>, which is rapid over distances of less than 1.4

<math display="inline" id="Oxidative_phosphorylation:0">
 <semantics>
  <mrow>
   <mi></mi>
   <mo>×</mo>
   <mrow>
    <msup>
     <mn>10</mn>
     <mi mathvariant="normal">−</mi>
    </msup>
    <mn>9</mn>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <times></times>
    <csymbol cd="latexml">absent</csymbol>
    <apply>
     <times></times>
     <apply>
      <csymbol cd="ambiguous">superscript</csymbol>
      <cn type="integer">10</cn>
      <ci>normal-−</ci>
     </apply>
     <cn type="integer">9</cn>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \times 10^{−}9
  </annotation>
 </semantics>
</math>

 m.<a class="footnoteRef" href="#fn15" id="fnref15"><sup>15</sup></a></p>
<h2 id="eukaryotic-electron-transport-chains">Eukaryotic electron transport chains</h2>

<p>Many <a class="uri" href="catabolic" title="wikilink">catabolic</a> biochemical processes, such as <a class="uri" href="glycolysis" title="wikilink">glycolysis</a>, the <a href="citric_acid_cycle" title="wikilink">citric acid cycle</a>, and <a href="beta_oxidation" title="wikilink">beta oxidation</a>, produce the reduced <a class="uri" href="coenzyme" title="wikilink">coenzyme</a> <a class="uri" href="NADH" title="wikilink">NADH</a>. This coenzyme contains electrons that have a high <a href="Standard_electrode_potential" title="wikilink">transfer potential</a>; in other words, they will release a large amount of energy upon oxidation. However, the cell does not release this energy all at once, as this would be an uncontrollable reaction. Instead, the electrons are removed from NADH and passed to oxygen through a series of enzymes that each release a small amount of the energy. This set of enzymes, consisting of complexes I through IV, is called the electron transport chain and is found in the inner membrane of the mitochondrion. <a href="succinic_acid" title="wikilink">Succinate</a> is also oxidized by the electron transport chain, but feeds into the pathway at a different point.</p>

<p>In <a href="eukaryote" title="wikilink">eukaryotes</a>, the enzymes in this electron transport system use the energy released from the oxidation of NADH to pump <a href="proton" title="wikilink">protons</a> across the inner membrane of the mitochondrion. This causes protons to build up in the <a href="intermembrane_space" title="wikilink">intermembrane space</a>, and generates an <a href="electrochemical_gradient" title="wikilink">electrochemical gradient</a> across the membrane. The energy stored in this potential is then used by ATP synthase to produce ATP. Oxidative phosphorylation in the eukaryotic mitochondrion is the best-understood example of this process. The mitochondrion is present in almost all eukaryotes, with the exception of anaerobic protozoa such as <em><a href="Trichomonas_vaginalis" title="wikilink">Trichomonas vaginalis</a></em> that instead reduce protons to hydrogen in a remnant mitochondrion called a <a class="uri" href="hydrogenosome" title="wikilink">hydrogenosome</a>.<a class="footnoteRef" href="#fn16" id="fnref16"><sup>16</sup></a></p>
<table>
<tbody>
<tr class="odd">
<td style="text-align: left;">
<p>Typical respiratory enzymes and substrates in eukaryotes.</p></td>
</tr>
<tr class="even">
<td style="text-align: left;">
<p>Respiratory enzyme</p></td>
</tr>
<tr class="odd">
<td style="text-align: left;">
<p><a href="NADH_dehydrogenase" title="wikilink">NADH dehydrogenase</a></p></td>
</tr>
<tr class="even">
<td style="text-align: left;">
<p><a href="Succinate_dehydrogenase" title="wikilink">Succinate dehydrogenase</a></p></td>
</tr>
<tr class="odd">
<td style="text-align: left;">
<p><a href="Coenzyme_Q_-_cytochrome_c_reductase" title="wikilink">Cytochrome bc<sub>1</sub> complex</a></p></td>
</tr>
<tr class="even">
<td style="text-align: left;">
<p>Cytochrome bc<sub>1</sub> complex</p></td>
</tr>
<tr class="odd">
<td style="text-align: left;">
<p><a href="Cytochrome_c_oxidase" title="wikilink">Complex IV</a></p></td>
</tr>
<tr class="even">
<td style="text-align: left;">
<p>Complex IV</p></td>
</tr>
<tr class="odd">
<td style="text-align: left;">
<p>Complex IV</p></td>
</tr>
<tr class="even">
<td style="text-align: left;">
<p>align=left <span style="font-size:87%;"> Conditions: pH = 7<a class="footnoteRef" href="#fn17" id="fnref17"><sup>17</sup></a></span></p></td>
</tr>
</tbody>
</table>
<h3 id="nadh-coenzyme-q-oxidoreductase-complex-i">NADH-coenzyme Q oxidoreductase (complex I)</h3>
<figure><b>(Figure)</b>
<figcaption>Complex I or <a href="NADH_dehydrogenase" title="wikilink">NADH-Q oxidoreductase</a>. The abbreviations are discussed in the text. In all diagrams of respiratory complexes in this article, the matrix is at the bottom, with the intermembrane space above.</figcaption>
</figure>

<p><a href="NADH_dehydrogenase" title="wikilink">NADH-coenzyme Q oxidoreductase</a>, also known as <em>NADH dehydrogenase</em> or <em>complex I</em>, is the first protein in the electron transport chain.<a class="footnoteRef" href="#fn18" id="fnref18"><sup>18</sup></a> Complex I is a giant <a class="uri" href="enzyme" title="wikilink">enzyme</a> with the mammalian complex I having 46 subunits and a molecular mass of about 1,000 <a href="atomic_mass_unit" title="wikilink">kilodaltons</a> (kDa).<a class="footnoteRef" href="#fn19" id="fnref19"><sup>19</sup></a> The structure is known in detail only from a bacterium;<a class="footnoteRef" href="#fn20" id="fnref20"><sup>20</sup></a><a class="footnoteRef" href="#fn21" id="fnref21"><sup>21</sup></a> in most organisms the complex resembles a boot with a large "ball" poking out from the membrane into the mitochondrion.<a class="footnoteRef" href="#fn22" id="fnref22"><sup>22</sup></a><a class="footnoteRef" href="#fn23" id="fnref23"><sup>23</sup></a> The genes that encode the individual proteins are contained in both the <a href="cell_nucleus" title="wikilink">cell nucleus</a> and the <a href="mitochondrial_genome" title="wikilink">mitochondrial genome</a>, as is the case for many enzymes present in the mitochondrion.</p>

<p>The reaction that is catalyzed by this enzyme is the two electron oxidation of <a href="Nicotinamide_adenine_dinucleotide" title="wikilink">NADH</a> by <a href="coenzyme_Q10" title="wikilink">coenzyme Q10</a> or <em>ubiquinone</em> (represented as Q in the equation below), a lipid-soluble <a class="uri" href="quinone" title="wikilink">quinone</a> that is found in the mitochondrion membrane:</p>

<p>

<math display="inline" id="Oxidative_phosphorylation:1">
 <semantics>
  <mrow>
   <mrow>
    <mi>NADH</mi>
    <mo>+</mo>
    <mi mathvariant="normal">Q</mi>
    <mo>+</mo>
    <mrow>
     <mpadded width="+2.8pt">
      <mn>5</mn>
     </mpadded>
     <msubsup>
      <mi mathvariant="normal">H</mi>
      <mi>matrix</mi>
      <mo>+</mo>
     </msubsup>
    </mrow>
   </mrow>
   <mo>→</mo>
   <mrow>
    <msup>
     <mi>NAD</mi>
     <mo>+</mo>
    </msup>
    <mo>+</mo>
    <msub>
     <mi>QH</mi>
     <mn>2</mn>
    </msub>
    <mo>+</mo>
    <mrow>
     <mpadded width="+2.8pt">
      <mn>4</mn>
     </mpadded>
     <mpadded width="-1.7pt">
      <msubsup>
       <mi mathvariant="normal">H</mi>
       <mi>intermembrane</mi>
       <mo>+</mo>
      </msubsup>
     </mpadded>
    </mrow>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <ci>normal-→</ci>
    <apply>
     <plus></plus>
     <ci>NADH</ci>
     <ci>normal-Q</ci>
     <apply>
      <times></times>
      <cn type="integer">5</cn>
      <apply>
       <csymbol cd="ambiguous">subscript</csymbol>
       <apply>
        <csymbol cd="ambiguous">superscript</csymbol>
        <ci>normal-H</ci>
        <plus></plus>
       </apply>
       <ci>matrix</ci>
      </apply>
     </apply>
    </apply>
    <apply>
     <plus></plus>
     <apply>
      <csymbol cd="ambiguous">superscript</csymbol>
      <ci>NAD</ci>
      <plus></plus>
     </apply>
     <apply>
      <csymbol cd="ambiguous">subscript</csymbol>
      <ci>QH</ci>
      <cn type="integer">2</cn>
     </apply>
     <apply>
      <times></times>
      <cn type="integer">4</cn>
      <apply>
       <csymbol cd="ambiguous">subscript</csymbol>
       <apply>
        <csymbol cd="ambiguous">superscript</csymbol>
        <ci>normal-H</ci>
        <plus></plus>
       </apply>
       <ci>intermembrane</ci>
      </apply>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \rm NADH+Q+5\;H^{+}_{matrix}\rightarrow NAD^{+}+QH_{2}+4\;H^{+}_{intermembrane}\!
  </annotation>
 </semantics>
</math>

</p>

<p>The start of the reaction, and indeed of the entire electron chain, is the binding of a NADH molecule to complex I and the donation of two electrons. The electrons enter complex I via a <a href="prosthetic_group" title="wikilink">prosthetic group</a> attached to the complex, <a href="flavin_mononucleotide" title="wikilink">flavin mononucleotide</a> (FMN). The addition of electrons to FMN converts it to its reduced form, FMNH<sub>2</sub>. The electrons are then transferred through a series of <a href="iron-sulfur_cluster" title="wikilink">iron–sulfur clusters</a>: the second kind of prosthetic group present in the complex.<a class="footnoteRef" href="#fn24" id="fnref24"><sup>24</sup></a> There are both [2Fe–2S] and [4Fe–4S] iron–sulfur clusters in complex I.</p>

<p>As the electrons pass through this complex, four protons are pumped from the matrix into the intermembrane space. Exactly how this occurs is unclear, but it seems to involve <a href="conformational_change" title="wikilink">conformational changes</a> in complex I that cause the protein to bind protons on the N-side of the membrane and release them on the P-side of the membrane.<a class="footnoteRef" href="#fn25" id="fnref25"><sup>25</sup></a> Finally, the electrons are transferred from the chain of iron–sulfur clusters to a ubiquinone molecule in the membrane.<a class="footnoteRef" href="#fn26" id="fnref26"><sup>26</sup></a> Reduction of ubiquinone also contributes to the generation of a proton gradient, as two protons are taken up from the matrix as it is reduced to <a class="uri" href="ubiquinol" title="wikilink">ubiquinol</a> (QH<sub>2</sub>).</p>
<h3 id="succinate-q-oxidoreductase-complex-ii">Succinate-Q oxidoreductase (complex II)</h3>
<figure><b>(Figure)</b>
<figcaption>Complex II: <a href="Succinate_-_coenzyme_Q_reductase" title="wikilink">Succinate-Q oxidoreductase</a>.</figcaption>
</figure>

<p><a href="Succinate_-_coenzyme_Q_reductase" title="wikilink">Succinate-Q oxidoreductase</a>, also known as <em>complex II</em> or <em>succinate dehydrogenase</em>, is a second entry point to the electron transport chain.<a class="footnoteRef" href="#fn27" id="fnref27"><sup>27</sup></a> It is unusual because it is the only enzyme that is part of both the citric acid cycle and the electron transport chain. Complex II consists of four protein subunits and contains a bound <a href="FAD" title="wikilink">flavin adenine dinucleotide</a> (FAD) cofactor, iron–sulfur clusters, and a <a class="uri" href="heme" title="wikilink">heme</a> group that does not participate in electron transfer to coenzyme Q, but is believed to be important in decreasing production of reactive oxygen species.<a class="footnoteRef" href="#fn28" id="fnref28"><sup>28</sup></a><a class="footnoteRef" href="#fn29" id="fnref29"><sup>29</sup></a> It oxidizes <a href="succinic_acid" title="wikilink">succinate</a> to <a href="Fumaric_acid" title="wikilink">fumarate</a> and reduces ubiquinone. As this reaction releases less energy than the oxidation of NADH, complex II does not transport protons across the membrane and does not contribute to the proton gradient.</p>

<p>

<math display="block" id="Oxidative_phosphorylation:2">
 <semantics>
  <mrow>
   <mrow>
    <mi>Succinate</mi>
    <mo>+</mo>
    <mi mathvariant="normal">Q</mi>
   </mrow>
   <mo>→</mo>
   <mrow>
    <mi>Fumarate</mi>
    <mo>+</mo>
    <mpadded width="-1.7pt">
     <msub>
      <mi>QH</mi>
      <mn>2</mn>
     </msub>
    </mpadded>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <ci>normal-→</ci>
    <apply>
     <plus></plus>
     <ci>Succinate</ci>
     <ci>normal-Q</ci>
    </apply>
    <apply>
     <plus></plus>
     <ci>Fumarate</ci>
     <apply>
      <csymbol cd="ambiguous">subscript</csymbol>
      <ci>QH</ci>
      <cn type="integer">2</cn>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \rm Succinate+Q\rightarrow Fumarate+QH_{2}\!
  </annotation>
 </semantics>
</math>

</p>

<p>In some eukaryotes, such as the <a href="parasitic_worm" title="wikilink">parasitic worm</a> <em><a href="Large_roundworm_of_pigs" title="wikilink">Ascaris suum</a></em>, an enzyme similar to complex II, fumarate reductase (menaquinol:fumarate oxidoreductase, or QFR), operates in reverse to oxidize ubiquinol and reduce fumarate. This allows the worm to survive in the anaerobic environment of the <a href="large_intestine" title="wikilink">large intestine</a>, carrying out anaerobic oxidative phosphorylation with fumarate as the electron acceptor.<a class="footnoteRef" href="#fn30" id="fnref30"><sup>30</sup></a> Another unconventional function of complex II is seen in the <a class="uri" href="malaria" title="wikilink">malaria</a> parasite <em><a href="Plasmodium_falciparum" title="wikilink">Plasmodium falciparum</a></em>. Here, the reversed action of complex II as an oxidase is important in regenerating ubiquinol, which the parasite uses in an unusual form of <a class="uri" href="pyrimidine" title="wikilink">pyrimidine</a> biosynthesis.<a class="footnoteRef" href="#fn31" id="fnref31"><sup>31</sup></a></p>
<h3 id="electron-transfer-flavoprotein-q-oxidoreductase">Electron transfer flavoprotein-Q oxidoreductase</h3>

<p><a href="Electron-transferring-flavoprotein_dehydrogenase" title="wikilink">Electron transfer flavoprotein-ubiquinone oxidoreductase</a> (ETF-Q oxidoreductase), also known as <em>electron transferring-flavoprotein dehydrogenase</em>, is a third entry point to the electron transport chain. It is an enzyme that accepts electrons from <a href="electron-transferring_flavoprotein" title="wikilink">electron-transferring flavoprotein</a> in the mitochondrial matrix, and uses these electrons to reduce ubiquinone.<a class="footnoteRef" href="#fn32" id="fnref32"><sup>32</sup></a> This enzyme contains a <a href="Flavin_group" title="wikilink">flavin</a> and a [4Fe–4S] cluster, but, unlike the other respiratory complexes, it attaches to the surface of the membrane and does not cross the lipid bilayer.<a class="footnoteRef" href="#fn33" id="fnref33"><sup>33</sup></a></p>

<p>

<math display="block" id="Oxidative_phosphorylation:3">
 <semantics>
  <mrow>
   <mrow>
    <msub>
     <mi>ETF</mi>
     <mi>red</mi>
    </msub>
    <mo>+</mo>
    <mi mathvariant="normal">Q</mi>
   </mrow>
   <mo>→</mo>
   <mrow>
    <msub>
     <mi>ETF</mi>
     <mi>ox</mi>
    </msub>
    <mo>+</mo>
    <mpadded width="-1.7pt">
     <msub>
      <mi>QH</mi>
      <mn>2</mn>
     </msub>
    </mpadded>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <ci>normal-→</ci>
    <apply>
     <plus></plus>
     <apply>
      <csymbol cd="ambiguous">subscript</csymbol>
      <ci>ETF</ci>
      <ci>red</ci>
     </apply>
     <ci>normal-Q</ci>
    </apply>
    <apply>
     <plus></plus>
     <apply>
      <csymbol cd="ambiguous">subscript</csymbol>
      <ci>ETF</ci>
      <ci>ox</ci>
     </apply>
     <apply>
      <csymbol cd="ambiguous">subscript</csymbol>
      <ci>QH</ci>
      <cn type="integer">2</cn>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \rm ETF_{red}+Q\rightarrow ETF_{ox}+QH_{2}\!
  </annotation>
 </semantics>
</math>

</p>

<p>In mammals, this metabolic pathway is important in <a href="beta_oxidation" title="wikilink">beta oxidation</a> of <a href="fatty_acid" title="wikilink">fatty acids</a> and catabolism of <a href="amino_acid" title="wikilink">amino acids</a> and <a class="uri" href="choline" title="wikilink">choline</a>, as it accepts electrons from multiple <a class="uri" href="acetyl-CoA" title="wikilink">acetyl-CoA</a> dehydrogenases.<a class="footnoteRef" href="#fn34" id="fnref34"><sup>34</sup></a><a class="footnoteRef" href="#fn35" id="fnref35"><sup>35</sup></a> In plants, ETF-Q oxidoreductase is also important in the metabolic responses that allow survival in extended periods of darkness.<a class="footnoteRef" href="#fn36" id="fnref36"><sup>36</sup></a></p>
<h3 id="q-cytochrome-c-oxidoreductase-complex-iii">Q-cytochrome c oxidoreductase (complex III)</h3>
<figure><b>(Figure)</b>
<figcaption>The two electron transfer steps in complex III: <a href="Coenzyme_Q_-_cytochrome_c_reductase" title="wikilink">Q-cytochrome c oxidoreductase</a>. After each step, Q (in the upper part of the figure) leaves the enzyme.</figcaption>
</figure>

<p><a href="Coenzyme_Q_-_cytochrome_c_reductase" title="wikilink">Q-cytochrome c oxidoreductase</a> is also known as <em>cytochrome c reductase</em>, <em>cytochrome bc<sub>1</sub> complex</em>, or simply <em>complex III</em>.<a class="footnoteRef" href="#fn37" id="fnref37"><sup>37</sup></a><a class="footnoteRef" href="#fn38" id="fnref38"><sup>38</sup></a> In mammals, this enzyme is a <a href="protein_dimer" title="wikilink">dimer</a>, with each subunit complex containing 11 protein subunits, an [2Fe-2S] iron–sulfur cluster and three <a href="cytochrome" title="wikilink">cytochromes</a>: one <a class="uri" href="cytochrome" title="wikilink">cytochrome</a> c<sub>1</sub> and two b <a class="uri" href="cytochromes" title="wikilink">cytochromes</a>.<a class="footnoteRef" href="#fn39" id="fnref39"><sup>39</sup></a> A cytochrome is a kind of electron-transferring protein that contains at least one <a class="uri" href="heme" title="wikilink">heme</a> group. The iron atoms inside complex III’s heme groups alternate between a reduced ferrous (+2) and oxidized ferric (+3) state as the electrons are transferred through the protein.</p>

<p>The reaction catalyzed by complex III is the oxidation of one molecule of <a class="uri" href="ubiquinol" title="wikilink">ubiquinol</a> and the reduction of two molecules of <a href="cytochrome_c" title="wikilink">cytochrome c</a>, a heme protein loosely associated with the mitochondrion. Unlike coenzyme Q, which carries two electrons, cytochrome c carries only one electron.</p>

<p>

<math display="block" id="Oxidative_phosphorylation:4">
 <semantics>
  <mrow>
   <mrow>
    <msub>
     <mi>QH</mi>
     <mn>2</mn>
    </msub>
    <mo>+</mo>
    <mrow>
     <mpadded width="+2.8pt">
      <mn>2</mn>
     </mpadded>
     <mpadded width="+1.7pt">
      <mi>Cyt</mi>
     </mpadded>
     <msub>
      <mi mathvariant="normal">c</mi>
      <mi>ox</mi>
     </msub>
    </mrow>
    <mo>+</mo>
    <mrow>
     <mpadded width="+2.8pt">
      <mn>2</mn>
     </mpadded>
     <msubsup>
      <mi mathvariant="normal">H</mi>
      <mi>matrix</mi>
      <mo>+</mo>
     </msubsup>
    </mrow>
   </mrow>
   <mo>→</mo>
   <mrow>
    <mi mathvariant="normal">Q</mi>
    <mo>+</mo>
    <mrow>
     <mpadded width="+2.8pt">
      <mn>2</mn>
     </mpadded>
     <mpadded width="+1.7pt">
      <mi>Cyt</mi>
     </mpadded>
     <msub>
      <mi mathvariant="normal">c</mi>
      <mi>red</mi>
     </msub>
    </mrow>
    <mo>+</mo>
    <mrow>
     <mpadded width="+2.8pt">
      <mn>4</mn>
     </mpadded>
     <mpadded width="-1.7pt">
      <msubsup>
       <mi mathvariant="normal">H</mi>
       <mi>intermembrane</mi>
       <mo>+</mo>
      </msubsup>
     </mpadded>
    </mrow>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <ci>normal-→</ci>
    <apply>
     <plus></plus>
     <apply>
      <csymbol cd="ambiguous">subscript</csymbol>
      <ci>QH</ci>
      <cn type="integer">2</cn>
     </apply>
     <apply>
      <times></times>
      <cn type="integer">2</cn>
      <ci>Cyt</ci>
      <apply>
       <csymbol cd="ambiguous">subscript</csymbol>
       <ci>normal-c</ci>
       <ci>ox</ci>
      </apply>
     </apply>
     <apply>
      <times></times>
      <cn type="integer">2</cn>
      <apply>
       <csymbol cd="ambiguous">subscript</csymbol>
       <apply>
        <csymbol cd="ambiguous">superscript</csymbol>
        <ci>normal-H</ci>
        <plus></plus>
       </apply>
       <ci>matrix</ci>
      </apply>
     </apply>
    </apply>
    <apply>
     <plus></plus>
     <ci>normal-Q</ci>
     <apply>
      <times></times>
      <cn type="integer">2</cn>
      <ci>Cyt</ci>
      <apply>
       <csymbol cd="ambiguous">subscript</csymbol>
       <ci>normal-c</ci>
       <ci>red</ci>
      </apply>
     </apply>
     <apply>
      <times></times>
      <cn type="integer">4</cn>
      <apply>
       <csymbol cd="ambiguous">subscript</csymbol>
       <apply>
        <csymbol cd="ambiguous">superscript</csymbol>
        <ci>normal-H</ci>
        <plus></plus>
       </apply>
       <ci>intermembrane</ci>
      </apply>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \rm QH_{2}+2\;Cyt\,c_{ox}+2\;H^{+}_{matrix}\rightarrow Q+2\;Cyt\,c_{red}+4\;H^%
{+}_{intermembrane}\!
  </annotation>
 </semantics>
</math>

</p>

<p>As only one of the electrons can be transferred from the QH<sub>2</sub> donor to a cytochrome c acceptor at a time, the reaction mechanism of complex III is more elaborate than those of the other respiratory complexes, and occurs in two steps called the <a href="Q_cycle" title="wikilink">Q cycle</a>.<a class="footnoteRef" href="#fn40" id="fnref40"><sup>40</sup></a> In the first step, the enzyme binds three substrates, first, QH<sub>2</sub>, which is then oxidized, with one electron being passed to the second substrate, cytochrome c. The two protons released from QH<sub>2</sub> pass into the intermembrane space. The third substrate is Q, which accepts the second electron from the QH<sub>2</sub> and is reduced to Q<sup>.−</sup>, which is the <a href="semiquinone" title="wikilink">ubisemiquinone</a> <a href="free_radical" title="wikilink">free radical</a>. The first two substrates are released, but this ubisemiquinone intermediate remains bound. In the second step, a second molecule of QH<sub>2</sub> is bound and again passes its first electron to a cytochrome c acceptor. The second electron is passed to the bound ubisemiquinone, reducing it to QH<sub>2</sub> as it gains two protons from the mitochondrial matrix. This QH<sub>2</sub> is then released from the enzyme.<a class="footnoteRef" href="#fn41" id="fnref41"><sup>41</sup></a></p>

<p>As coenzyme Q is reduced to ubiquinol on the inner side of the membrane and oxidized to ubiquinone on the other, a net transfer of protons across the membrane occurs, adding to the proton gradient.<a class="footnoteRef" href="#fn42" id="fnref42"><sup>42</sup></a> The rather complex two-step mechanism by which this occurs is important, as it increases the efficiency of proton transfer. If, instead of the Q cycle, one molecule of QH<sub>2</sub> were used to directly reduce two molecules of cytochrome c, the efficiency would be halved, with only one proton transferred per cytochrome c reduced.<a class="footnoteRef" href="#fn43" id="fnref43"><sup>43</sup></a></p>
<h3 id="cytochrome-c-oxidase-complex-iv">Cytochrome c oxidase (complex IV)</h3>
<figure><b>(Figure)</b>
<figcaption>Complex IV: <a href="cytochrome_c_oxidase" title="wikilink">cytochrome c oxidase</a>.</figcaption>
</figure>

<p><a href="Cytochrome_c_oxidase" title="wikilink">Cytochrome c oxidase</a>, also known as <em>complex IV</em>, is the final protein complex in the electron transport chain.<a class="footnoteRef" href="#fn44" id="fnref44"><sup>44</sup></a> The mammalian enzyme has an extremely complicated structure and contains 13 subunits, two heme groups, as well as multiple metal ion cofactors – in all, three atoms of <a class="uri" href="copper" title="wikilink">copper</a>, one of <a class="uri" href="magnesium" title="wikilink">magnesium</a> and one of <a class="uri" href="zinc" title="wikilink">zinc</a>.<a class="footnoteRef" href="#fn45" id="fnref45"><sup>45</sup></a></p>

<p>This enzyme mediates the final reaction in the electron transport chain and transfers electrons to oxygen, while pumping protons across the membrane.<a class="footnoteRef" href="#fn46" id="fnref46"><sup>46</sup></a> The final <a href="electron_acceptor" title="wikilink">electron acceptor</a> oxygen, which is also called the <em>terminal electron acceptor</em>, is reduced to water in this step. Both the direct pumping of protons and the consumption of matrix protons in the reduction of oxygen contribute to the proton gradient. The reaction catalyzed is the oxidation of cytochrome c and the reduction of oxygen:</p>

<p>

<math display="inline" id="Oxidative_phosphorylation:5">
 <semantics>
  <mrow>
   <mrow>
    <mrow>
     <mpadded width="+2.8pt">
      <mn>4</mn>
     </mpadded>
     <mpadded width="+1.7pt">
      <mi>Cyt</mi>
     </mpadded>
     <msub>
      <mi mathvariant="normal">c</mi>
      <mi>red</mi>
     </msub>
    </mrow>
    <mo>+</mo>
    <msub>
     <mi mathvariant="normal">O</mi>
     <mn>2</mn>
    </msub>
    <mo>+</mo>
    <mrow>
     <mpadded width="+2.8pt">
      <mn>8</mn>
     </mpadded>
     <msubsup>
      <mi mathvariant="normal">H</mi>
      <mi>matrix</mi>
      <mo>+</mo>
     </msubsup>
    </mrow>
   </mrow>
   <mo>→</mo>
   <mrow>
    <mrow>
     <mpadded width="+2.8pt">
      <mn>4</mn>
     </mpadded>
     <mpadded width="+1.7pt">
      <mi>Cyt</mi>
     </mpadded>
     <msub>
      <mi mathvariant="normal">c</mi>
      <mi>ox</mi>
     </msub>
    </mrow>
    <mo>+</mo>
    <mrow>
     <mpadded width="+2.8pt">
      <mn>2</mn>
     </mpadded>
     <msub>
      <mi mathvariant="normal">H</mi>
      <mn>2</mn>
     </msub>
     <mi mathvariant="normal">O</mi>
    </mrow>
    <mo>+</mo>
    <mrow>
     <mpadded width="+2.8pt">
      <mn>4</mn>
     </mpadded>
     <mpadded width="-1.7pt">
      <msubsup>
       <mi mathvariant="normal">H</mi>
       <mi>intermembrane</mi>
       <mo>+</mo>
      </msubsup>
     </mpadded>
    </mrow>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <ci>normal-→</ci>
    <apply>
     <plus></plus>
     <apply>
      <times></times>
      <cn type="integer">4</cn>
      <ci>Cyt</ci>
      <apply>
       <csymbol cd="ambiguous">subscript</csymbol>
       <ci>normal-c</ci>
       <ci>red</ci>
      </apply>
     </apply>
     <apply>
      <csymbol cd="ambiguous">subscript</csymbol>
      <ci>normal-O</ci>
      <cn type="integer">2</cn>
     </apply>
     <apply>
      <times></times>
      <cn type="integer">8</cn>
      <apply>
       <csymbol cd="ambiguous">subscript</csymbol>
       <apply>
        <csymbol cd="ambiguous">superscript</csymbol>
        <ci>normal-H</ci>
        <plus></plus>
       </apply>
       <ci>matrix</ci>
      </apply>
     </apply>
    </apply>
    <apply>
     <plus></plus>
     <apply>
      <times></times>
      <cn type="integer">4</cn>
      <ci>Cyt</ci>
      <apply>
       <csymbol cd="ambiguous">subscript</csymbol>
       <ci>normal-c</ci>
       <ci>ox</ci>
      </apply>
     </apply>
     <apply>
      <times></times>
      <cn type="integer">2</cn>
      <apply>
       <csymbol cd="ambiguous">subscript</csymbol>
       <ci>normal-H</ci>
       <cn type="integer">2</cn>
      </apply>
      <ci>normal-O</ci>
     </apply>
     <apply>
      <times></times>
      <cn type="integer">4</cn>
      <apply>
       <csymbol cd="ambiguous">subscript</csymbol>
       <apply>
        <csymbol cd="ambiguous">superscript</csymbol>
        <ci>normal-H</ci>
        <plus></plus>
       </apply>
       <ci>intermembrane</ci>
      </apply>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \rm 4\;Cyt\,c_{red}+O_{2}+8\;H^{+}_{matrix}\rightarrow 4\;Cyt\,c_{ox}+2\;H_{2}%
O+4\;H^{+}_{intermembrane}\!
  </annotation>
 </semantics>
</math>

</p>
<h3 id="alternative-reductases-and-oxidases">Alternative reductases and oxidases</h3>

<p>Many eukaryotic organisms have electron transport chains that differ from the much-studied mammalian enzymes described above. For example, <a href="plant" title="wikilink">plants</a> have alternative NADH oxidases, which oxidize NADH in the cytosol rather than in the mitochondrial matrix, and pass these electrons to the ubiquinone pool.<a class="footnoteRef" href="#fn47" id="fnref47"><sup>47</sup></a> These enzymes do not transport protons, and, therefore, reduce ubiquinone without altering the electrochemical gradient across the inner membrane.<a class="footnoteRef" href="#fn48" id="fnref48"><sup>48</sup></a></p>

<p>Another example of a divergent electron transport chain is the <em><a href="alternative_oxidase" title="wikilink">alternative oxidase</a></em>, which is found in <a href="plant" title="wikilink">plants</a>, as well as some <a href="fungus" title="wikilink">fungi</a>, <a href="protist" title="wikilink">protists</a>, and possibly some animals.<a class="footnoteRef" href="#fn49" id="fnref49"><sup>49</sup></a><a class="footnoteRef" href="#fn50" id="fnref50"><sup>50</sup></a> This enzyme transfers electrons directly from ubiquinol to oxygen.<a class="footnoteRef" href="#fn51" id="fnref51"><sup>51</sup></a></p>

<p>The electron transport pathways produced by these alternative NADH and ubiquinone oxidases have lower <a href="adenosine_triphosphate" title="wikilink">ATP</a> yields than the full pathway. The advantages produced by a shortened pathway are not entirely clear. However, the alternative oxidase is produced in response to stresses such as cold, <a href="reactive_oxygen_species" title="wikilink">reactive oxygen species</a>, and infection by pathogens, as well as other factors that inhibit the full electron transport chain.<a class="footnoteRef" href="#fn52" id="fnref52"><sup>52</sup></a><a class="footnoteRef" href="#fn53" id="fnref53"><sup>53</sup></a> Alternative pathways might, therefore, enhance an organisms' resistance to injury, by reducing <a href="oxidative_stress" title="wikilink">oxidative stress</a>.<a class="footnoteRef" href="#fn54" id="fnref54"><sup>54</sup></a></p>
<h3 id="organization-of-complexes">Organization of complexes</h3>

<p>The original model for how the respiratory chain complexes are organized was that they diffuse freely and independently in the mitochondrial membrane.<a class="footnoteRef" href="#fn55" id="fnref55"><sup>55</sup></a> However, recent data suggest that the complexes might form higher-order structures called supercomplexes or "<a href="respirasome" title="wikilink">respirasomes</a>."<a class="footnoteRef" href="#fn56" id="fnref56"><sup>56</sup></a> In this model, the various complexes exist as organized sets of interacting enzymes.<a class="footnoteRef" href="#fn57" id="fnref57"><sup>57</sup></a> These associations might allow channeling of substrates between the various enzyme complexes, increasing the rate and efficiency of electron transfer.<a class="footnoteRef" href="#fn58" id="fnref58"><sup>58</sup></a> Within such mammalian supercomplexes, some components would be present in higher amounts than others, with some data suggesting a ratio between complexes I/II/III/IV and the ATP synthase of approximately 1:1:3:7:4.<a class="footnoteRef" href="#fn59" id="fnref59"><sup>59</sup></a> However, the debate over this supercomplex hypothesis is not completely resolved, as some data do not appear to fit with this model.<a class="footnoteRef" href="#fn60" id="fnref60"><sup>60</sup></a><a class="footnoteRef" href="#fn61" id="fnref61"><sup>61</sup></a></p>
<h2 id="prokaryotic-electron-transport-chains">Prokaryotic electron transport chains</h2>

<p>In contrast to the general similarity in structure and function of the electron transport chains in eukaryotes, <a class="uri" href="bacteria" title="wikilink">bacteria</a> and <a class="uri" href="archaea" title="wikilink">archaea</a> possess a large variety of electron-transfer enzymes. These use an equally wide set of chemicals as substrates.<a class="footnoteRef" href="#fn62" id="fnref62"><sup>62</sup></a> In common with eukaryotes, prokaryotic electron transport uses the energy released from the oxidation of a substrate to pump ions across a membrane and generate an electrochemical gradient. In the bacteria, oxidative phosphorylation in <em><a href="Escherichia_coli" title="wikilink">Escherichia coli</a></em> is understood in most detail, while archaeal systems are at present poorly understood.<a class="footnoteRef" href="#fn63" id="fnref63"><sup>63</sup></a></p>

<p>The main difference between eukaryotic and prokaryotic oxidative phosphorylation is that bacteria and archaea use many different substances to donate or accept electrons. This allows prokaryotes to grow under a wide variety of environmental conditions.<a class="footnoteRef" href="#fn64" id="fnref64"><sup>64</sup></a> In <em>E. coli</em>, for example, oxidative phosphorylation can be driven by a large number of pairs of reducing agents and oxidizing agents, which are listed below. The <a href="Standard_electrode_potential#Non-standard_condition" title="wikilink">midpoint potential</a> of a chemical measures how much energy is released when it is oxidized or reduced, with reducing agents having negative potentials and oxidizing agents positive potentials.</p>
<table>
<tbody>
<tr class="odd">
<td style="text-align: left;">
<p>Respiratory enzymes and substrates in <em>E. coli</em>.<a class="footnoteRef" href="#fn65" id="fnref65"><sup>65</sup></a></p></td>
</tr>
<tr class="even">
<td style="text-align: left;">
<p>Respiratory enzyme</p></td>
</tr>
<tr class="odd">
<td style="text-align: left;">
<p><a href="Formate_dehydrogenase" title="wikilink">Formate dehydrogenase</a></p></td>
</tr>
<tr class="even">
<td style="text-align: left;">
<p><a class="uri" href="Hydrogenase" title="wikilink">Hydrogenase</a></p></td>
</tr>
<tr class="odd">
<td style="text-align: left;">
<p><a href="NADH_dehydrogenase" title="wikilink">NADH dehydrogenase</a></p></td>
</tr>
<tr class="even">
<td style="text-align: left;">
<p><a href="Glycerol-3-phosphate_dehydrogenase" title="wikilink">Glycerol-3-phosphate dehydrogenase</a></p></td>
</tr>
<tr class="odd">
<td style="text-align: left;">
<p><a href="Pyruvate_dehydrogenase#Other_forms" title="wikilink">Pyruvate oxidase</a></p></td>
</tr>
<tr class="even">
<td style="text-align: left;">
<p><a href="Lactate_dehydrogenase" title="wikilink">Lactate dehydrogenase</a></p></td>
</tr>
<tr class="odd">
<td style="text-align: left;">
<p><a href="D-amino_acid_dehydrogenase" title="wikilink"><small>D</small>-amino acid dehydrogenase</a></p></td>
</tr>
<tr class="even">
<td style="text-align: left;">
<p><a href="Quinoprotein_glucose_dehydrogenase" title="wikilink">Glucose dehydrogenase</a></p></td>
</tr>
<tr class="odd">
<td style="text-align: left;">
<p><a href="Succinate_-_coenzyme_Q_reductase" title="wikilink">Succinate dehydrogenase</a></p></td>
</tr>
<tr class="even">
<td style="text-align: left;">
<p><a href="Ubiquinol_oxidase" title="wikilink">Ubiquinol oxidase</a></p></td>
</tr>
<tr class="odd">
<td style="text-align: left;">
<p><a href="Nitrate_reductase" title="wikilink">Nitrate reductase</a></p></td>
</tr>
<tr class="even">
<td style="text-align: left;">
<p><a href="Nitrite_reductase" title="wikilink">Nitrite reductase</a></p></td>
</tr>
<tr class="odd">
<td style="text-align: left;">
<p><a href="DMSO_reductase" title="wikilink">Dimethyl sulfoxide reductase</a></p></td>
</tr>
<tr class="even">
<td style="text-align: left;">
<p><a href="Trimethylamine_N-oxide_reductase" title="wikilink">Trimethylamine <em>N</em>-oxide reductase</a></p></td>
</tr>
<tr class="odd">
<td style="text-align: left;">
<p><a href="Fumarate_reductase" title="wikilink">Fumarate reductase</a></p></td>
</tr>
</tbody>
</table>

<p>As shown above, <em>E. coli</em> can grow with reducing agents such as formate, hydrogen, or lactate as electron donors, and nitrate, DMSO, or oxygen as acceptors.<a class="footnoteRef" href="#fn66" id="fnref66"><sup>66</sup></a> The larger the difference in midpoint potential between an oxidizing and reducing agent, the more energy is released when they react. Out of these compounds, the succinate/fumarate pair is unusual, as its midpoint potential is close to zero. Succinate can therefore be oxidized to fumarate if a strong oxidizing agent such as oxygen is available, or fumarate can be reduced to succinate using a strong reducing agent such as formate. These alternative reactions are catalyzed by <a href="Succinate_-_coenzyme_Q_reductase" title="wikilink">succinate dehydrogenase</a> and <a href="fumarate_reductase" title="wikilink">fumarate reductase</a>, respectively.<a class="footnoteRef" href="#fn67" id="fnref67"><sup>67</sup></a></p>

<p>Some prokaryotes use redox pairs that have only a small difference in midpoint potential. For example, <a href="nitrification" title="wikilink">nitrifying</a> bacteria such as <em><a class="uri" href="Nitrobacter" title="wikilink">Nitrobacter</a></em> oxidize nitrite to nitrate, donating the electrons to oxygen. The small amount of energy released in this reaction is enough to pump protons and generate ATP, but not enough to produce NADH or NADPH directly for use in <a class="uri" href="anabolism" title="wikilink">anabolism</a>.<a class="footnoteRef" href="#fn68" id="fnref68"><sup>68</sup></a> This problem is solved by using a <a href="nitrite_oxidoreductase" title="wikilink">nitrite oxidoreductase</a> to produce enough proton-motive force to run part of the electron transport chain in reverse, causing complex I to generate NADH.<a class="footnoteRef" href="#fn69" id="fnref69"><sup>69</sup></a><a class="footnoteRef" href="#fn70" id="fnref70"><sup>70</sup></a></p>

<p>Prokaryotes control their use of these electron donors and acceptors by varying which enzymes are produced, in response to environmental conditions.<a class="footnoteRef" href="#fn71" id="fnref71"><sup>71</sup></a> This flexibility is possible because different oxidases and reductases use the same ubiquinone pool. This allows many combinations of enzymes to function together, linked by the common ubiquinol intermediate.<a class="footnoteRef" href="#fn72" id="fnref72"><sup>72</sup></a> These respiratory chains therefore have a <a href="modular_design" title="wikilink">modular design</a>, with easily interchangeable sets of enzyme systems.</p>

<p>In addition to this metabolic diversity, prokaryotes also possess a range of <a href="isozyme" title="wikilink">isozymes</a> – different enzymes that catalyze the same reaction. For example, in <em>E. coli</em>, there are two different types of ubiquinol oxidase using oxygen as an electron acceptor. Under highly aerobic conditions, the cell uses an oxidase with a low affinity for oxygen that can transport two protons per electron. However, if levels of oxygen fall, they switch to an oxidase that transfers only one proton per electron, but has a high affinity for oxygen.<a class="footnoteRef" href="#fn73" id="fnref73"><sup>73</sup></a></p>
<h2 id="atp-synthase-complex-v">ATP synthase (complex V)</h2>

<p>ATP synthase, also called <em>complex V</em>, is the final enzyme in the oxidative phosphorylation pathway. This enzyme is found in all forms of life and functions in the same way in both prokaryotes and eukaryotes.<a class="footnoteRef" href="#fn74" id="fnref74"><sup>74</sup></a> The enzyme uses the energy stored in a proton gradient across a membrane to drive the synthesis of ATP from ADP and <a class="uri" href="phosphate" title="wikilink">phosphate</a> (P<sub>i</sub>). Estimates of the number of protons required to synthesize one ATP have ranged from three to four,<a class="footnoteRef" href="#fn75" id="fnref75"><sup>75</sup></a><a class="footnoteRef" href="#fn76" id="fnref76"><sup>76</sup></a> with some suggesting cells can vary this ratio, to suit different conditions.<a class="footnoteRef" href="#fn77" id="fnref77"><sup>77</sup></a></p>

<p>

<math display="block" id="Oxidative_phosphorylation:6">
 <semantics>
  <mrow>
   <mrow>
    <mi>ADP</mi>
    <mo>+</mo>
    <msub>
     <mi mathvariant="normal">P</mi>
     <mi mathvariant="normal">i</mi>
    </msub>
    <mo>+</mo>
    <mrow>
     <mpadded width="+2.8pt">
      <mn>4</mn>
     </mpadded>
     <msubsup>
      <mi mathvariant="normal">H</mi>
      <mi>intermembrane</mi>
      <mo>+</mo>
     </msubsup>
    </mrow>
   </mrow>
   <mo>⇌</mo>
   <mrow>
    <mi>ATP</mi>
    <mo>+</mo>
    <mrow>
     <msub>
      <mi mathvariant="normal">H</mi>
      <mn>2</mn>
     </msub>
     <mi mathvariant="normal">O</mi>
    </mrow>
    <mo>+</mo>
    <mrow>
     <mpadded width="+2.8pt">
      <mn>4</mn>
     </mpadded>
     <mpadded width="-1.7pt">
      <msubsup>
       <mi mathvariant="normal">H</mi>
       <mi>matrix</mi>
       <mo>+</mo>
      </msubsup>
     </mpadded>
    </mrow>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <ci>normal-⇌</ci>
    <apply>
     <plus></plus>
     <ci>ADP</ci>
     <apply>
      <csymbol cd="ambiguous">subscript</csymbol>
      <ci>normal-P</ci>
      <ci>normal-i</ci>
     </apply>
     <apply>
      <times></times>
      <cn type="integer">4</cn>
      <apply>
       <csymbol cd="ambiguous">subscript</csymbol>
       <apply>
        <csymbol cd="ambiguous">superscript</csymbol>
        <ci>normal-H</ci>
        <plus></plus>
       </apply>
       <ci>intermembrane</ci>
      </apply>
     </apply>
    </apply>
    <apply>
     <plus></plus>
     <ci>ATP</ci>
     <apply>
      <times></times>
      <apply>
       <csymbol cd="ambiguous">subscript</csymbol>
       <ci>normal-H</ci>
       <cn type="integer">2</cn>
      </apply>
      <ci>normal-O</ci>
     </apply>
     <apply>
      <times></times>
      <cn type="integer">4</cn>
      <apply>
       <csymbol cd="ambiguous">subscript</csymbol>
       <apply>
        <csymbol cd="ambiguous">superscript</csymbol>
        <ci>normal-H</ci>
        <plus></plus>
       </apply>
       <ci>matrix</ci>
      </apply>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \rm ADP+P_{i}+4\;H^{+}_{intermembrane}\rightleftharpoons ATP+H_{2}O+4\;H^{+}_{%
matrix}\!
  </annotation>
 </semantics>
</math>

</p>

<p>This <a class="uri" href="phosphorylation" title="wikilink">phosphorylation</a> reaction is an <a href="chemical_equilibrium" title="wikilink">equilibrium</a>, which can be shifted by altering the proton-motive force. In the absence of a proton-motive force, the ATP synthase reaction will run from right to left, hydrolyzing ATP and pumping protons out of the matrix across the membrane. However, when the proton-motive force is high, the reaction is forced to run in the opposite direction; it proceeds from left to right, allowing protons to flow down their concentration gradient and turning ADP into ATP.<a class="footnoteRef" href="#fn78" id="fnref78"><sup>78</sup></a> Indeed, in the closely related <a href="V-ATPase" title="wikilink">vacuolar type H+-ATPases</a>, the hydrolysis reaction is used to acidify cellular compartments, by pumping protons and hydrolysing ATP.<a class="footnoteRef" href="#fn79" id="fnref79"><sup>79</sup></a></p>

<p>ATP synthase is a massive protein complex with a mushroom-like shape. The mammalian enzyme complex contains 16 subunits and has a mass of approximately 600 <a href="kilodalton" title="wikilink">kilodaltons</a>.<a class="footnoteRef" href="#fn80" id="fnref80"><sup>80</sup></a> The portion embedded within the membrane is called F<sub>O</sub> and contains a ring of c subunits and the proton channel. The stalk and the ball-shaped headpiece is called F<sub>1</sub> and is the site of ATP synthesis. The ball-shaped complex at the end of the F<sub>1</sub> portion contains six proteins of two different kinds (three α subunits and three β subunits), whereas the "stalk" consists of one protein: the γ subunit, with the tip of the stalk extending into the ball of α and β subunits.<a class="footnoteRef" href="#fn81" id="fnref81"><sup>81</sup></a> Both the α and β subunits bind nucleotides, but only the β subunits catalyze the ATP synthesis reaction. Reaching along the side of the F<sub>1</sub> portion and back into the membrane is a long rod-like subunit that anchors the α and β subunits into the base of the enzyme.</p>

<p>As protons cross the membrane through the channel in the base of ATP synthase, the F<sub>O</sub> proton-driven motor rotates.<a class="footnoteRef" href="#fn82" id="fnref82"><sup>82</sup></a> Rotation might be caused by changes in the <a class="uri" href="ionization" title="wikilink">ionization</a> of amino acids in the ring of c subunits causing <a class="uri" href="electrostatic" title="wikilink">electrostatic</a> interactions that propel the ring of c subunits past the proton channel.<a class="footnoteRef" href="#fn83" id="fnref83"><sup>83</sup></a> This rotating ring in turn drives the rotation of the central <a class="uri" href="axle" title="wikilink">axle</a> (the γ subunit stalk) within the α and β subunits. The α and β subunits are prevented from rotating themselves by the side-arm, which acts as a <a class="uri" href="stator" title="wikilink">stator</a>. This movement of the tip of the γ subunit within the ball of α and β subunits provides the energy for the active sites in the β subunits to undergo a cycle of movements that produces and then releases ATP.<a class="footnoteRef" href="#fn84" id="fnref84"><sup>84</sup></a>  This ATP synthesis reaction is called the <em>binding change mechanism</em> and involves the active site of a β subunit cycling between three states.<a class="footnoteRef" href="#fn85" id="fnref85"><sup>85</sup></a> In the "open" state, ADP and phosphate enter the active site (shown in brown in the diagram). The protein then closes up around the molecules and binds them loosely – the "loose" state (shown in red). The enzyme then changes shape again and forces these molecules together, with the active site in the resulting "tight" state (shown in pink) binding the newly produced ATP molecule with very high <a href="Dissociation_constant" title="wikilink">affinity</a>. Finally, the active site cycles back to the open state, releasing ATP and binding more ADP and phosphate, ready for the next cycle.</p>

<p>In some bacteria and archaea, ATP synthesis is driven by the movement of sodium ions through the cell membrane, rather than the movement of protons.<a class="footnoteRef" href="#fn86" id="fnref86"><sup>86</sup></a><a class="footnoteRef" href="#fn87" id="fnref87"><sup>87</sup></a> Archaea such as <em><a class="uri" href="Methanococcus" title="wikilink">Methanococcus</a></em> also contain the A<sub>1</sub>A<sub>o</sub> synthase, a form of the enzyme that contains additional proteins with little similarity in sequence to other bacterial and eukaryotic ATP synthase subunits. It is possible that, in some species, the A<sub>1</sub>A<sub>o</sub> form of the enzyme is a specialized sodium-driven ATP synthase,<a class="footnoteRef" href="#fn88" id="fnref88"><sup>88</sup></a> but this might not be true in all cases.<a class="footnoteRef" href="#fn89" id="fnref89"><sup>89</sup></a></p>
<h2 id="reactive-oxygen-species">Reactive oxygen species</h2>

<p>Molecular oxygen is an ideal terminal <a href="electron_acceptor" title="wikilink">electron acceptor</a> because it is a strong oxidizing agent. The reduction of oxygen does involve potentially harmful intermediates.<a class="footnoteRef" href="#fn90" id="fnref90"><sup>90</sup></a> Although the transfer of four electrons and four protons reduces oxygen to water, which is harmless, transfer of one or two electrons produces <a class="uri" href="superoxide" title="wikilink">superoxide</a> or <a class="uri" href="peroxide" title="wikilink">peroxide</a> anions, which are dangerously reactive.</p>

<p>

<math display="block" id="Oxidative_phosphorylation:7">
 <semantics>
  <mtable displaystyle="true">
   <mtr>
    <mtd columnalign="center">
     <mi></mi>
    </mtd>
    <mtd columnalign="center">
     <msup>
      <mi mathvariant="normal">e</mi>
      <mo>-</mo>
     </msup>
    </mtd>
    <mtd columnalign="center">
     <mi></mi>
    </mtd>
    <mtd columnalign="center">
     <msup>
      <mi mathvariant="normal">e</mi>
      <mo>-</mo>
     </msup>
    </mtd>
   </mtr>
   <mtr>
    <mtd columnalign="center">
     <msub>
      <mtext>O</mtext>
      <mn>2</mn>
     </msub>
    </mtd>
    <mtd columnalign="center">
     <mo>⟶</mo>
    </mtd>
    <mtd columnalign="center">
     <msubsup>
      <mtext>O</mtext>
      <mn>2</mn>
      <munder accentunder="true">
       <mo>∙</mo>
       <mo>¯</mo>
      </munder>
     </msubsup>
    </mtd>
    <mtd columnalign="center">
     <mo>⟶</mo>
    </mtd>
    <mtd columnalign="center">
     <msubsup>
      <mtext>O</mtext>
      <mn>2</mn>
      <mrow>
       <mn>2</mn>
       <mo>-</mo>
      </mrow>
     </msubsup>
    </mtd>
   </mtr>
   <mtr>
    <mtd columnalign="center">
     <mi></mi>
    </mtd>
    <mtd columnalign="center">
     <mi></mi>
    </mtd>
    <mtd columnalign="center">
     <mtext>Superoxide</mtext>
    </mtd>
    <mtd columnalign="center">
     <mi></mi>
    </mtd>
    <mtd columnalign="center">
     <mtext>Peroxide</mtext>
    </mtd>
   </mtr>
   <mtr>
    <mtd columnalign="center">
     <mi></mi>
    </mtd>
    <mtd columnalign="center">
     <mi></mi>
    </mtd>
   </mtr>
  </mtable>
  <annotation-xml encoding="MathML-Content">
   <matrix>
    <matrixrow>
     <csymbol cd="latexml">absent</csymbol>
     <apply>
      <csymbol cd="ambiguous">superscript</csymbol>
      <ci>normal-e</ci>
      <minus></minus>
     </apply>
     <csymbol cd="latexml">absent</csymbol>
     <apply>
      <csymbol cd="ambiguous">superscript</csymbol>
      <ci>normal-e</ci>
      <minus></minus>
     </apply>
    </matrixrow>
    <matrixrow>
     <apply>
      <csymbol cd="ambiguous">subscript</csymbol>
      <mtext>O</mtext>
      <cn type="integer">2</cn>
     </apply>
     <ci>normal-⟶</ci>
     <apply>
      <csymbol cd="ambiguous">superscript</csymbol>
      <apply>
       <csymbol cd="ambiguous">subscript</csymbol>
       <mtext>O</mtext>
       <cn type="integer">2</cn>
      </apply>
      <apply>
       <ci>normal-¯</ci>
       <ci>normal-∙</ci>
      </apply>
     </apply>
     <ci>normal-⟶</ci>
     <apply>
      <csymbol cd="ambiguous">superscript</csymbol>
      <apply>
       <csymbol cd="ambiguous">subscript</csymbol>
       <mtext>O</mtext>
       <cn type="integer">2</cn>
      </apply>
      <apply>
       <csymbol cd="latexml">limit-from</csymbol>
       <cn type="integer">2</cn>
       <minus></minus>
      </apply>
     </apply>
    </matrixrow>
    <matrixrow>
     <csymbol cd="latexml">absent</csymbol>
     <csymbol cd="latexml">absent</csymbol>
     <mtext>Superoxide</mtext>
     <csymbol cd="latexml">absent</csymbol>
     <mtext>Peroxide</mtext>
    </matrixrow>
    <matrixrow>
     <csymbol cd="latexml">absent</csymbol>
     <csymbol cd="latexml">absent</csymbol>
    </matrixrow>
   </matrix>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \begin{matrix}&{\mathrm{e}^{-}}&&{\mathrm{e}^{-}}\\
{\mbox{O}_{2}}&\longrightarrow&\mbox{O}_{2}^{\underline{\bullet}}&%
\longrightarrow&\mbox{O}_{2}^{2-}\\
&&\mbox{Superoxide}&&\mbox{Peroxide}\\
&\end{matrix}
  </annotation>
 </semantics>
</math>

</p>

<p>These <a href="reactive_oxygen_species" title="wikilink">reactive oxygen species</a> and their reaction products, such as the <a class="uri" href="hydroxyl" title="wikilink">hydroxyl</a> radical, are very harmful to cells, as they oxidize proteins and cause <a href="mutation" title="wikilink">mutations</a> in <a class="uri" href="DNA" title="wikilink">DNA</a>. This cellular damage might contribute to <a class="uri" href="disease" title="wikilink">disease</a> and is proposed as one cause of <a href="free-radical_theory_of_aging" title="wikilink">aging</a>.<a class="footnoteRef" href="#fn91" id="fnref91"><sup>91</sup></a><a class="footnoteRef" href="#fn92" id="fnref92"><sup>92</sup></a></p>

<p>The cytochrome c oxidase complex is highly efficient at reducing oxygen to water, and it releases very few partly reduced intermediates; however small amounts of superoxide anion and peroxide are produced by the electron transport chain.<a class="footnoteRef" href="#fn93" id="fnref93"><sup>93</sup></a> Particularly important is the reduction of <a href="coenzyme_Q" title="wikilink">coenzyme Q</a> in complex III, as a highly reactive ubisemiquinone free radical is formed as an intermediate in the Q cycle. This unstable species can lead to electron "leakage" when electrons transfer directly to oxygen, forming superoxide.<a class="footnoteRef" href="#fn94" id="fnref94"><sup>94</sup></a> As the production of reactive oxygen species by these proton-pumping complexes is greatest at high membrane potentials, it has been proposed that mitochondria regulate their activity to maintain the membrane potential within a narrow range that balances ATP production against oxidant generation.<a class="footnoteRef" href="#fn95" id="fnref95"><sup>95</sup></a> For instance, oxidants can activate <a href="uncoupling_protein" title="wikilink">uncoupling proteins</a> that reduce membrane potential.<a class="footnoteRef" href="#fn96" id="fnref96"><sup>96</sup></a></p>

<p>To counteract these reactive oxygen species, cells contain numerous <a class="uri" href="antioxidant" title="wikilink">antioxidant</a> systems, including antioxidant <a href="vitamin" title="wikilink">vitamins</a> such as <a href="vitamin_C" title="wikilink">vitamin C</a> and <a href="vitamin_E" title="wikilink">vitamin E</a>, and antioxidant enzymes such as <a href="superoxide_dismutase" title="wikilink">superoxide dismutase</a>, <a class="uri" href="catalase" title="wikilink">catalase</a>, and <a class="uri" href="peroxidases" title="wikilink">peroxidases</a>,<a class="footnoteRef" href="#fn97" id="fnref97"><sup>97</sup></a> which detoxify the reactive species, limiting damage to the cell.</p>
<h2 id="inhibitors">Inhibitors</h2>

<p>There are several well-known <a href="drug" title="wikilink">drugs</a> and <a href="toxin" title="wikilink">toxins</a> that inhibit oxidative phosphorylation. Although any one of these toxins inhibits only one enzyme in the electron transport chain, inhibition of any step in this process will halt the rest of the process. For example, if <a class="uri" href="oligomycin" title="wikilink">oligomycin</a> inhibits ATP synthase, protons cannot pass back into the mitochondrion.<a class="footnoteRef" href="#fn98" id="fnref98"><sup>98</sup></a> As a result, the proton pumps are unable to operate, as the gradient becomes too strong for them to overcome. NADH is then no longer oxidized and the citric acid cycle ceases to operate because the concentration of NAD<sup>+</sup> falls below the concentration that these enzymes can use.</p>
<table>
<thead>
<tr class="header">
<th style="text-align: left;">
<p>Compounds</p></th>
<th style="text-align: left;">
<p>Use</p></th>
<th style="text-align: left;">
<p>Site of action</p></th>
<th style="text-align: left;">
<p>Effect on oxidative phosphorylation</p></th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;">
<p><a class="uri" href="Cyanide" title="wikilink">Cyanide</a><br/>
<a href="carbon_monoxide" title="wikilink">Carbon monoxide</a><br/>
<a class="uri" href="Azide" title="wikilink">Azide</a><br/>
<a href="Hydrogen_sulfide" title="wikilink">Hydrogen sulfide</a></p></td>
<td style="text-align: left;">
<p>Poisons</p></td>
<td style="text-align: left;">
<p>Complex IV</p></td>
<td style="text-align: left;">
<p>Inhibit the electron transport chain by binding more strongly than oxygen to the <a href="iron" title="wikilink">Fe</a>–<a href="copper" title="wikilink">Cu</a> center in cytochrome c oxidase, preventing the reduction of oxygen.<a class="footnoteRef" href="#fn99" id="fnref99"><sup>99</sup></a></p></td>
</tr>
<tr class="even">
<td style="text-align: left;">
<p><a class="uri" href="Oligomycin" title="wikilink">Oligomycin</a></p></td>
<td style="text-align: left;">
<p><a class="uri" href="Antibiotic" title="wikilink">Antibiotic</a></p></td>
<td style="text-align: left;">
<p>Complex V</p></td>
<td style="text-align: left;">
<p>|Inhibits ATP synthase by blocking the flow of protons through the F<sub>o</sub> subunit.<a class="footnoteRef" href="#fn100" id="fnref100"><sup>100</sup></a></p></td>
</tr>
<tr class="odd">
<td style="text-align: left;">
<p><a href="Carbonyl_cyanide_m-chlorophenyl_hydrazone" title="wikilink">CCCP</a><br/>
<a class="uri" href="2,4-Dinitrophenol" title="wikilink">2,4-Dinitrophenol</a></p></td>
<td style="text-align: left;">
<p>Poisons, weight-loss<a class="footnoteRef" href="#fn101" id="fnref101"><sup>101</sup></a></p></td>
<td style="text-align: left;">
<p>Inner membrane</p></td>
<td style="text-align: left;">
<p>|<a href="Ionophore" title="wikilink">Ionophores</a> that disrupt the proton gradient by carrying protons across a membrane. This ionophore <a class="uri" href="uncouples" title="wikilink">uncouples</a> proton pumping from ATP synthesis because it carries protons across the inner mitochondrial membrane.<a class="footnoteRef" href="#fn102" id="fnref102"><sup>102</sup></a></p></td>
</tr>
<tr class="even">
<td style="text-align: left;">
<p><a class="uri" href="Rotenone" title="wikilink">Rotenone</a></p></td>
<td style="text-align: left;">
<p><a class="uri" href="Pesticide" title="wikilink">Pesticide</a></p></td>
<td style="text-align: left;">
<p>Complex I</p></td>
<td style="text-align: left;">
<p>|Prevents the transfer of electrons from complex I to ubiquinone by blocking the ubiquinone-binding site.<a class="footnoteRef" href="#fn103" id="fnref103"><sup>103</sup></a></p></td>
</tr>
<tr class="odd">
<td style="text-align: left;">
<p><a class="uri" href="Malonate" title="wikilink">Malonate</a> and <a class="uri" href="oxaloacetate" title="wikilink">oxaloacetate</a></p></td>
<td style="text-align: left;">
<p>Poisons</p></td>
<td style="text-align: left;">
<p>Complex II</p></td>
<td style="text-align: left;">
<p>|Competitive inhibitors of succinate dehydrogenase (complex II).<a class="footnoteRef" href="#fn104" id="fnref104"><sup>104</sup></a></p></td>
</tr>
<tr class="even">
<td style="text-align: left;">
<p><a href="Antimycin_A" title="wikilink">Antimycin A</a></p></td>
<td style="text-align: left;">
<p><a class="uri" href="Piscicide" title="wikilink">Piscicide</a></p></td>
<td style="text-align: left;">
<p>Complex III</p></td>
<td style="text-align: left;">
<p>|Binds to the Qi site of <a href="Coenzyme_Q_–_cytochrome_c_reductase" title="wikilink">cytochrome c reductase</a>, thereby inhibiting the <a class="uri" href="oxidation" title="wikilink">oxidation</a> of <a class="uri" href="ubiquinol" title="wikilink">ubiquinol</a>.</p></td>
</tr>
<tr class="odd">
</tr>
</tbody>
</table>

<p>Not all inhibitors of oxidative phosphorylation are toxins. In <a href="brown_adipose_tissue" title="wikilink">brown adipose tissue</a>, regulated proton channels called <a href="uncoupling_protein" title="wikilink">uncoupling proteins</a> can uncouple respiration from ATP synthesis.<a class="footnoteRef" href="#fn105" id="fnref105"><sup>105</sup></a> This rapid respiration produces heat, and is particularly important as a way of maintaining <a href="body_temperature" title="wikilink">body temperature</a> for <a href="hibernation" title="wikilink">hibernating</a> animals, although these proteins may also have a more general function in cells' responses to stress.<a class="footnoteRef" href="#fn106" id="fnref106"><sup>106</sup></a></p>
<h2 id="history">History</h2>

<p>The field of oxidative phosphorylation began with the report in 1906 by <a href="Arthur_Harden" title="wikilink">Arthur Harden</a> of a vital role for phosphate in cellular <a href="fermentation_(biochemistry)" title="wikilink">fermentation</a>, but initially only <a href="sugar_phosphates" title="wikilink">sugar phosphates</a> were known to be involved.<a class="footnoteRef" href="#fn107" id="fnref107"><sup>107</sup></a> However, in the early 1940s, the link between the oxidation of sugars and the generation of ATP was firmly established by <a href="Herman_Kalckar" title="wikilink">Herman Kalckar</a>,<a class="footnoteRef" href="#fn108" id="fnref108"><sup>108</sup></a> confirming the central role of ATP in energy transfer that had been proposed by <a href="Fritz_Albert_Lipmann" title="wikilink">Fritz Albert Lipmann</a> in 1941.<a class="footnoteRef" href="#fn109" id="fnref109"><sup>109</sup></a> Later, in 1949, Morris Friedkin and <a href="Albert_L._Lehninger" title="wikilink">Albert L. Lehninger</a> proved that the coenzyme NADH linked metabolic pathways such as the citric acid cycle and the synthesis of ATP.<a class="footnoteRef" href="#fn110" id="fnref110"><sup>110</sup></a> The term <em>oxidative phosphorylation</em> was coined by  in 1939.<a class="footnoteRef" href="#fn111" id="fnref111"><sup>111</sup></a><a class="footnoteRef" href="#fn112" id="fnref112"><sup>112</sup></a></p>

<p>For another twenty years, the mechanism by which ATP is generated remained mysterious, with scientists searching for an elusive "high-energy intermediate" that would link oxidation and phosphorylation reactions.<a class="footnoteRef" href="#fn113" id="fnref113"><sup>113</sup></a> This puzzle was solved by <a href="Peter_D._Mitchell" title="wikilink">Peter D. Mitchell</a> with the publication of the <a href="chemiosmotic_theory" title="wikilink">chemiosmotic theory</a> in 1961.<a class="footnoteRef" href="#fn114" id="fnref114"><sup>114</sup></a> At first, this proposal was highly controversial, but it was slowly accepted and Mitchell was awarded a <a href="Nobel_Prize" title="wikilink">Nobel prize</a> in 1978.<a class="footnoteRef" href="#fn115" id="fnref115"><sup>115</sup></a><a class="footnoteRef" href="#fn116" id="fnref116"><sup>116</sup></a> Subsequent research concentrated on purifying and characterizing the enzymes involved, with major contributions being made by <a href="David_E._Green" title="wikilink">David E. Green</a> on the complexes of the electron-transport chain, as well as <a href="Efraim_Racker" title="wikilink">Efraim Racker</a> on the ATP synthase.<a class="footnoteRef" href="#fn117" id="fnref117"><sup>117</sup></a> A critical step towards solving the mechanism of the ATP synthase was provided by <a href="Paul_D._Boyer" title="wikilink">Paul D. Boyer</a>, by his development in 1973 of the "binding change" mechanism, followed by his radical proposal of rotational catalysis in 1982.<a class="footnoteRef" href="#fn118" id="fnref118"><sup>118</sup></a><a class="footnoteRef" href="#fn119" id="fnref119"><sup>119</sup></a> More recent work has included <a href="x-ray_crystallography" title="wikilink">structural studies</a> on the enzymes involved in oxidative phosphorylation by <a href="John_E._Walker" title="wikilink">John E. Walker</a>, with Walker and Boyer being awarded a Nobel Prize in 1997.<a class="footnoteRef" href="#fn120" id="fnref120"><sup>120</sup></a></p>
<h2 id="see-also">See also</h2>
<ul>
<li><a class="uri" href="Respirometry" title="wikilink">Respirometry</a></li>
<li><a href="TIM/TOM_Complex" title="wikilink">TIM/TOM Complex</a></li>
</ul>
<h2 id="notes">Notes</h2>
<h2 id="references">References</h2>
<h2 id="further-reading">Further reading</h2>
<h3 id="introductory">Introductory</h3>
<ul>
<li></li>
<li></li>
<li></li>
</ul>
<h3 id="advanced">Advanced</h3>
<ul>
<li></li>
<li></li>
<li></li>
<li></li>
</ul>
<h2 id="external-links">External links</h2>
<h3 id="general-resources">General resources</h3>
<ul>
<li><a href="http://www.wiley.com/legacy/college/boyer/0470003790/animations/electron_transport/electron_transport.htm">Animated diagrams illustrating oxidative phosphorylation</a> <a href="John_Wiley_&amp;_Sons" title="wikilink">Wiley and Co</a> <em>Concepts in Biochemistry</em></li>
<li><a href="http://www.life.uiuc.edu/crofts/bioph354/">On-line biophysics lectures</a> Antony Crofts, <a href="University_of_Illinois_at_Urbana-Champaign" title="wikilink">University of Illinois at Urbana-Champaign</a></li>
<li><a href="http://www.youtube.com/watch?v=PjdPTY1wHdQ">ATP Synthase</a> Graham Johnson</li>
</ul>
<h3 id="structural-resources">Structural resources</h3>
<ul>
<li><a href="Protein_Data_Bank" title="wikilink">PDB</a> molecule of the month:
<ul>
<li><a href="http://www.rcsb.org/pdb/static.do?p=education_discussion/molecule_of_the_month/pdb72_1.html">ATP synthase</a></li>
<li><a href="http://www.rcsb.org/pdb/static.do?p=education_discussion/molecule_of_the_month/pdb36_1.html">Cytochrome c</a></li>
<li><a href="http://www.rcsb.org/pdb/static.do?p=education_discussion/molecule_of_the_month/pdb5_1.html">Cytochrome c oxidase</a></li>
</ul></li>
<li>Interactive molecular models at <a href="Universidade_Fernando_Pessoa" title="wikilink">Universidade Fernando Pessoa</a>:
<ul>
<li><a href="http://www2.ufp.pt/~pedros/anim/2frame-ien.htm">NADH dehydrogenase</a></li>
<li><a href="http://www2.ufp.pt/~pedros/anim/2frame-iien.htm">succinate dehydrogenase</a></li>
<li><a href="http://www2.ufp.pt/~pedros/anim/2frame-iiien.htm">Coenzyme Q - cytochrome c reductase</a></li>
<li><a href="http://www2.ufp.pt/~pedros/anim/2frame-iven.htm">cytochrome c oxidase</a></li>
</ul></li>
</ul>

<p>"</p>

<p></p>

<p><a href="Category:Cellular_respiration" title="wikilink">Category:Cellular respiration</a> <a href="Category:Integral_membrane_proteins" title="wikilink">Category:Integral membrane proteins</a> <a class="uri" href="Category:Metabolism" title="wikilink">Category:Metabolism</a></p>
<section class="footnotes">
<hr/>
<ol>
<li id="fn1"><a href="#fnref1">↩</a></li>
<li id="fn2"><a href="#fnref2">↩</a></li>
<li id="fn3"><a href="#fnref3">↩</a></li>
<li id="fn4"></li>
<li id="fn5"><a href="#fnref5">↩</a></li>
<li id="fn6"><a href="#fnref6">↩</a></li>
<li id="fn7"><a href="#fnref7">↩</a></li>
<li id="fn8"><a href="#fnref8">↩</a></li>
<li id="fn9"><a href="#fnref9">↩</a></li>
<li id="fn10"><a href="#fnref10">↩</a></li>
<li id="fn11"><a href="#fnref11">↩</a></li>
<li id="fn12"></li>
<li id="fn13"><a href="#fnref13">↩</a></li>
<li id="fn14"><a href="#fnref14">↩</a></li>
<li id="fn15"><a href="#fnref15">↩</a></li>
<li id="fn16"><a href="#fnref16">↩</a></li>
<li id="fn17"></li>
<li id="fn18"><a href="#fnref18">↩</a></li>
<li id="fn19"><a href="#fnref19">↩</a></li>
<li id="fn20"><a href="#fnref20">↩</a></li>
<li id="fn21">Efremov R.G., Baradaran R., &amp; Sazanov L.A., (2010) The arcdhitecture of respiratory complex I, Nature 465, 441-445<a href="#fnref21">↩</a></li>
<li id="fn22"><a href="#fnref22">↩</a></li>
<li id="fn23"><a href="#fnref23">↩</a></li>
<li id="fn24"></li>
<li id="fn25"><a href="#fnref25">↩</a></li>
<li id="fn26"></li>
<li id="fn27"><a href="#fnref27">↩</a></li>
<li id="fn28"><a href="#fnref28">↩</a></li>
<li id="fn29"><a href="#fnref29">↩</a></li>
<li id="fn30"><a href="#fnref30">↩</a></li>
<li id="fn31"><a href="#fnref31">↩</a></li>
<li id="fn32"><a href="#fnref32">↩</a></li>
<li id="fn33"><a href="#fnref33">↩</a></li>
<li id="fn34"><a href="#fnref34">↩</a></li>
<li id="fn35"><a href="#fnref35">↩</a></li>
<li id="fn36"><a href="#fnref36">↩</a></li>
<li id="fn37"><a href="#fnref37">↩</a></li>
<li id="fn38"><a href="#fnref38">↩</a></li>
<li id="fn39"><a href="#fnref39">↩</a></li>
<li id="fn40"><a href="#fnref40">↩</a></li>
<li id="fn41"><a href="#fnref41">↩</a></li>
<li id="fn42"></li>
<li id="fn43"></li>
<li id="fn44"><a href="#fnref44">↩</a></li>
<li id="fn45"><a href="#fnref45">↩</a></li>
<li id="fn46"><a href="#fnref46">↩</a></li>
<li id="fn47"><a href="#fnref47">↩</a></li>
<li id="fn48"><a href="#fnref48">↩</a></li>
<li id="fn49"><a href="#fnref49">↩</a></li>
<li id="fn50"><a href="#fnref50">↩</a></li>
<li id="fn51"><a href="#fnref51">↩</a></li>
<li id="fn52"><a href="#fnref52">↩</a></li>
<li id="fn53"><a href="#fnref53">↩</a></li>
<li id="fn54"><a href="#fnref54">↩</a></li>
<li id="fn55"><a href="#fnref55">↩</a></li>
<li id="fn56"><a href="#fnref56">↩</a></li>
<li id="fn57"><a href="#fnref57">↩</a></li>
<li id="fn58"><a href="#fnref58">↩</a></li>
<li id="fn59"><a href="#fnref59">↩</a></li>
<li id="fn60"></li>
<li id="fn61"><a href="#fnref61">↩</a></li>
<li id="fn62"><a href="#fnref62">↩</a></li>
<li id="fn63"><a href="#fnref63">↩</a></li>
<li id="fn64"><a href="#fnref64">↩</a></li>
<li id="fn65"><a href="#fnref65">↩</a></li>
<li id="fn66"></li>
<li id="fn67"><a href="#fnref67">↩</a></li>
<li id="fn68"><a href="#fnref68">↩</a></li>
<li id="fn69"><a href="#fnref69">↩</a></li>
<li id="fn70"><a href="#fnref70">↩</a></li>
<li id="fn71"><a href="#fnref71">↩</a></li>
<li id="fn72"></li>
<li id="fn73"><a href="#fnref73">↩</a></li>
<li id="fn74"><a href="#fnref74">↩</a></li>
<li id="fn75"><a href="#fnref75">↩</a></li>
<li id="fn76"><a href="#fnref76">↩</a></li>
<li id="fn77"><a href="#fnref77">↩</a></li>
<li id="fn78"></li>
<li id="fn79"><a href="#fnref79">↩</a></li>
<li id="fn80"><a href="#fnref80">↩</a></li>
<li id="fn81"><a href="#fnref81">↩</a></li>
<li id="fn82"><a href="#fnref82">↩</a></li>
<li id="fn83"><a href="#fnref83">↩</a></li>
<li id="fn84"><a href="#fnref84">↩</a></li>
<li id="fn85"><a href="#fnref85">↩</a></li>
<li id="fn86"><a href="#fnref86">↩</a></li>
<li id="fn87"><a href="#fnref87">↩</a></li>
<li id="fn88"><a href="#fnref88">↩</a></li>
<li id="fn89"></li>
<li id="fn90"><a href="#fnref90">↩</a></li>
<li id="fn91"><a href="#fnref91">↩</a></li>
<li id="fn92"><a href="#fnref92">↩</a></li>
<li id="fn93"><a href="#fnref93">↩</a></li>
<li id="fn94"><a href="#fnref94">↩</a></li>
<li id="fn95"><a href="#fnref95">↩</a></li>
<li id="fn96"><a href="#fnref96">↩</a></li>
<li id="fn97"></li>
<li id="fn98"></li>
<li id="fn99"><a href="#fnref99">↩</a></li>
<li id="fn100"><a href="#fnref100">↩</a></li>
<li id="fn101">DNP was extensively used as an <a href="anti-obesity_medication" title="wikilink">anti-obesity medication</a> in the 1930s but was ultimately discontinued due to its dangerous side effects. However, illicit use of the drug for this purpose continues today. See <a href="2,4-Dinitrophenol#Dieting_aid" title="wikilink">2,4-Dinitrophenol#Dieting aid</a> for more information.<a href="#fnref101">↩</a></li>
<li id="fn102"><a href="#fnref102">↩</a></li>
<li id="fn103"><a href="#fnref103">↩</a></li>
<li id="fn104"><a href="#fnref104">↩</a></li>
<li id="fn105"><a href="#fnref105">↩</a></li>
<li id="fn106"><a href="#fnref106">↩</a></li>
<li id="fn107"><a href="#fnref107">↩</a></li>
<li id="fn108"><a href="#fnref108">↩</a></li>
<li id="fn109"><a href="#fnref109">↩</a></li>
<li id="fn110"><a href="#fnref110">↩</a></li>
<li id="fn111"><a href="#fnref111">↩</a></li>
<li id="fn112"><a href="#fnref112">↩</a></li>
<li id="fn113"><a href="#fnref113">↩</a></li>
<li id="fn114"><a href="#fnref114">↩</a></li>
<li id="fn115"><a href="#fnref115">↩</a></li>
<li id="fn116"><a href="#fnref116">↩</a></li>
<li id="fn117"><a href="#fnref117">↩</a></li>
<li id="fn118"></li>
<li id="fn119"><a href="#fnref119">↩</a></li>
<li id="fn120"><a href="#fnref120">↩</a></li>
</ol>
</section>
</body>
</html>
