<html lang="en">
<head>
<meta charset="utf-8"/>
<title offset="812">Peroxisome</title>
<script src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_SVG.js" type="text/javascript">
</script>
</head>
<body>
<h1>Peroxisome</h1>
<hr/>

<p> <embed src="Distribution of peroxisomes labelled with a monomeric eqFP611 variant in HEK293 cells during mitosis - pone.0004391.s005.ogv" title="fig:Distribution of peroxisomes (white) in HEK 293 cells during mitosis."></embed></p>

<p><strong>Peroxisomes</strong> <a class="footnoteRef" href="#fn1" id="fnref1"><sup>1</sup></a> (also called <strong>microbodies</strong>) are <a href="organelle" title="wikilink">organelles</a> found in virtually all <a class="uri" href="eukaryotic" title="wikilink">eukaryotic</a> cells.<a class="footnoteRef" href="#fn2" id="fnref2"><sup>2</sup></a> They are involved in the <a class="uri" href="catabolism" title="wikilink">catabolism</a> of <a href="very_long_chain_fatty_acid" title="wikilink">very long chain fatty acids</a>, <a href="branched_chain_fatty_acids" title="wikilink">branched chain fatty acids</a>, <a href="D-amino_acids" title="wikilink">D-amino acids</a>, and <a href="polyamine" title="wikilink">polyamines</a>, <a href="redox" title="wikilink">reduction</a> of <a href="reactive_oxygen_species" title="wikilink">reactive oxygen species</a> - specifically <a href="hydrogen_peroxide" title="wikilink">hydrogen peroxide</a><a class="footnoteRef" href="#fn3" id="fnref3"><sup>3</sup></a> - and biosynthesis of <a class="uri" href="plasmalogens" title="wikilink">plasmalogens</a>, i.e. <a href="ether_phospholipid" title="wikilink">ether phospholipids</a> critical for the normal function of mammalian brains and lungs.<a class="footnoteRef" href="#fn4" id="fnref4"><sup>4</sup></a> They also contain approximately 10% of the total activity of two enzymes in the <a href="pentose_phosphate_pathway" title="wikilink">pentose phosphate pathway</a>, which is important for energy metabolism.<a class="footnoteRef" href="#fn5" id="fnref5"><sup>5</sup></a> It is vigorously debated if peroxisomes are involved in <a class="uri" href="isoprenoid" title="wikilink">isoprenoid</a> and <a class="uri" href="cholesterol" title="wikilink">cholesterol</a> synthesis in animals.<a class="footnoteRef" href="#fn6" id="fnref6"><sup>6</sup></a> Other known peroxisomal functions include the <a href="glyoxylate_cycle" title="wikilink">glyoxylate cycle</a> in germinating seeds ("<a class="uri" href="glyoxysomes" title="wikilink">glyoxysomes</a>"), <a class="uri" href="photorespiration" title="wikilink">photorespiration</a> in leaves,<a class="footnoteRef" href="#fn7" id="fnref7"><sup>7</sup></a> <a class="uri" href="glycolysis" title="wikilink">glycolysis</a> in <a class="uri" href="trypanosomes" title="wikilink">trypanosomes</a> ("<a href="glycosome" title="wikilink">glycosomes</a>"), and <a class="uri" href="methanol" title="wikilink">methanol</a> and/or amine oxidation and assimilation in some yeasts.</p>

<p>Peroxisomes were identified as organelles by the Belgian cytologist <a href="Christian_de_Duve" title="wikilink">Christian de Duve</a> in 1967<a class="footnoteRef" href="#fn8" id="fnref8"><sup>8</sup></a> after they had been first described by a Swedish doctoral student, J. Rhodin in 1954.<a class="footnoteRef" href="#fn9" id="fnref9"><sup>9</sup></a></p>
<h2 id="metabolic-functions">Metabolic functions</h2>

<p>A major function of the peroxisome is the breakdown of very long chain <a href="fatty_acids" title="wikilink">fatty acids</a> through <a class="uri" href="beta-oxidation" title="wikilink">beta-oxidation</a>. In animal cells, the long fatty acids are converted to medium chain fatty acids, which are subsequently shuttled to <a class="uri" href="mitochondria" title="wikilink">mitochondria</a> where they are eventually broken down to carbon dioxide and water. In yeast and plant cells, this process is exclusive for the peroxisomes.<a class="footnoteRef" href="#fn10" id="fnref10"><sup>10</sup></a></p>

<p>The first reactions in the formation of <a class="uri" href="plasmalogen" title="wikilink">plasmalogen</a> in animal cells also occur in peroxisomes. Plasmalogen is the most abundant phospholipid in <a class="uri" href="myelin" title="wikilink">myelin</a>. Deficiency of plasmalogens causes profound abnormalities in the myelination of <a href="neuron" title="wikilink">nerve cells</a>, which is one reason why many <a href="peroxisomal_disorders" title="wikilink">peroxisomal disorders</a> affect the nervous system.<a class="footnoteRef" href="#fn11" id="fnref11"><sup>11</sup></a> Peroxisomes also play a role in the production of <a class="uri" href="bile" title="wikilink">bile</a> acids important for the absorption of fats and fat-soluble vitamins, such as vitamins A and K. Skin disorders are features of genetic disorders affecting peroxisome function as a result.</p>

<p>Peroxisomes contain oxidative <a href="enzyme" title="wikilink">enzymes</a>, such as <a class="uri" href="catalase" title="wikilink">catalase</a>, <a href="D-amino_acid_oxidase" title="wikilink">D-amino acid oxidase</a>, and <a href="uric_acid_oxidase" title="wikilink">uric acid oxidase</a>.<a class="footnoteRef" href="#fn12" id="fnref12"><sup>12</sup></a> However the last enzyme is absent in humans, explaining the disease known as <a class="uri" href="gout" title="wikilink">gout</a>, caused by the accumulation of uric acid. Certain enzymes within the peroxisome, by using molecular oxygen, remove hydrogen atoms from specific organic substrates (labeled as R), in an oxidative reaction, producing <a href="hydrogen_peroxide" title="wikilink">hydrogen peroxide</a> (H<sub>2</sub>O<sub>2</sub>, itself toxic):</p>

<p>

<math display="block" id="Peroxisome:0">
 <semantics>
  <mrow>
   <mrow>
    <msub>
     <mi>RH</mi>
     <mn>2</mn>
    </msub>
    <mo>+</mo>
    <msub>
     <mi mathvariant="normal">O</mi>
     <mn>2</mn>
    </msub>
   </mrow>
   <mo>→</mo>
   <mrow>
    <mi mathvariant="normal">R</mi>
    <mo>+</mo>
    <mrow>
     <msub>
      <mi mathvariant="normal">H</mi>
      <mn>2</mn>
     </msub>
     <msub>
      <mi mathvariant="normal">O</mi>
      <mn>2</mn>
     </msub>
    </mrow>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <ci>normal-→</ci>
    <apply>
     <plus></plus>
     <apply>
      <csymbol cd="ambiguous">subscript</csymbol>
      <ci>RH</ci>
      <cn type="integer">2</cn>
     </apply>
     <apply>
      <csymbol cd="ambiguous">subscript</csymbol>
      <ci>normal-O</ci>
      <cn type="integer">2</cn>
     </apply>
    </apply>
    <apply>
     <plus></plus>
     <ci>normal-R</ci>
     <apply>
      <times></times>
      <apply>
       <csymbol cd="ambiguous">subscript</csymbol>
       <ci>normal-H</ci>
       <cn type="integer">2</cn>
      </apply>
      <apply>
       <csymbol cd="ambiguous">subscript</csymbol>
       <ci>normal-O</ci>
       <cn type="integer">2</cn>
      </apply>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \mathrm{RH}_{\mathrm{2}}+\mathrm{O}_{\mathrm{2}}\rightarrow\mathrm{R}+\mathrm{%
H}_{2}\mathrm{O}_{2}
  </annotation>
 </semantics>
</math>

</p>

<p>Catalase, another peroxisomal enzyme, uses this H<sub>2</sub>O<sub>2</sub> to oxidize other substrates, including <a class="uri" href="phenols" title="wikilink">phenols</a>, <a href="formic_acid" title="wikilink">formic acid</a>, <a class="uri" href="formaldehyde" title="wikilink">formaldehyde</a>, and <a class="uri" href="alcohol" title="wikilink">alcohol</a>, by means of the peroxidation reaction:</p>

<p>

<math display="block" id="Peroxisome:1">
 <semantics>
  <mrow>
   <mrow>
    <mrow>
     <msub>
      <mi mathvariant="normal">H</mi>
      <mn>2</mn>
     </msub>
     <msub>
      <mi mathvariant="normal">O</mi>
      <mn>2</mn>
     </msub>
    </mrow>
    <mo>+</mo>
    <mrow>
     <msup>
      <mi mathvariant="normal">R</mi>
      <mo>′</mo>
     </msup>
     <msub>
      <mi mathvariant="normal">H</mi>
      <mn>2</mn>
     </msub>
    </mrow>
   </mrow>
   <mo>→</mo>
   <mrow>
    <msup>
     <mi mathvariant="normal">R</mi>
     <mo>′</mo>
    </msup>
    <mo>+</mo>
    <mrow>
     <mn>2</mn>
     <msub>
      <mi mathvariant="normal">H</mi>
      <mn>2</mn>
     </msub>
     <mi mathvariant="normal">O</mi>
    </mrow>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <ci>normal-→</ci>
    <apply>
     <plus></plus>
     <apply>
      <times></times>
      <apply>
       <csymbol cd="ambiguous">subscript</csymbol>
       <ci>normal-H</ci>
       <cn type="integer">2</cn>
      </apply>
      <apply>
       <csymbol cd="ambiguous">subscript</csymbol>
       <ci>normal-O</ci>
       <cn type="integer">2</cn>
      </apply>
     </apply>
     <apply>
      <times></times>
      <apply>
       <csymbol cd="ambiguous">superscript</csymbol>
       <ci>normal-R</ci>
       <ci>normal-′</ci>
      </apply>
      <apply>
       <csymbol cd="ambiguous">subscript</csymbol>
       <ci>normal-H</ci>
       <cn type="integer">2</cn>
      </apply>
     </apply>
    </apply>
    <apply>
     <plus></plus>
     <apply>
      <csymbol cd="ambiguous">superscript</csymbol>
      <ci>normal-R</ci>
      <ci>normal-′</ci>
     </apply>
     <apply>
      <times></times>
      <cn type="integer">2</cn>
      <apply>
       <csymbol cd="ambiguous">subscript</csymbol>
       <ci>normal-H</ci>
       <cn type="integer">2</cn>
      </apply>
      <ci>normal-O</ci>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \mathrm{H}_{2}\mathrm{O}_{2}+\mathrm{R^{\prime}H}_{2}\rightarrow\mathrm{R^{%
\prime}}+2\mathrm{H}_{2}\mathrm{O}
  </annotation>
 </semantics>
</math>

, thus eliminating the poisonous hydrogen peroxide in the process.</p>

<p>This reaction is important in liver and kidney cells, where the peroxisomes detoxify various toxic substances that enter the blood. About 25% of the <a class="uri" href="ethanol" title="wikilink">ethanol</a> alcohol humans drink is oxidized to <a class="uri" href="acetaldehyde" title="wikilink">acetaldehyde</a> in this way.<a class="footnoteRef" href="#fn13" id="fnref13"><sup>13</sup></a> In addition, when excess H<sub>2</sub>O<sub>2</sub> accumulates in the cell, catalase converts it to H<sub>2</sub>O through this reaction:</p>

<p>

<math display="block" id="Peroxisome:2">
 <semantics>
  <mrow>
   <mrow>
    <mn>2</mn>
    <msub>
     <mi mathvariant="normal">H</mi>
     <mn>2</mn>
    </msub>
    <msub>
     <mi mathvariant="normal">O</mi>
     <mn>2</mn>
    </msub>
   </mrow>
   <mo>→</mo>
   <mrow>
    <mrow>
     <mn>2</mn>
     <msub>
      <mi mathvariant="normal">H</mi>
      <mn>2</mn>
     </msub>
     <mi mathvariant="normal">O</mi>
    </mrow>
    <mo>+</mo>
    <msub>
     <mi mathvariant="normal">O</mi>
     <mn>2</mn>
    </msub>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <ci>normal-→</ci>
    <apply>
     <times></times>
     <cn type="integer">2</cn>
     <apply>
      <csymbol cd="ambiguous">subscript</csymbol>
      <ci>normal-H</ci>
      <cn type="integer">2</cn>
     </apply>
     <apply>
      <csymbol cd="ambiguous">subscript</csymbol>
      <ci>normal-O</ci>
      <cn type="integer">2</cn>
     </apply>
    </apply>
    <apply>
     <plus></plus>
     <apply>
      <times></times>
      <cn type="integer">2</cn>
      <apply>
       <csymbol cd="ambiguous">subscript</csymbol>
       <ci>normal-H</ci>
       <cn type="integer">2</cn>
      </apply>
      <ci>normal-O</ci>
     </apply>
     <apply>
      <csymbol cd="ambiguous">subscript</csymbol>
      <ci>normal-O</ci>
      <cn type="integer">2</cn>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   2\mathrm{H}_{2}\mathrm{O}_{2}\rightarrow 2\mathrm{H}_{2}\mathrm{O}+\mathrm{O}_%
{2}
  </annotation>
 </semantics>
</math>

</p>

<p>In higher plants, peroxisomes contain also a complex battery of antioxidative enzymes such as superoxide . dismutase, the components of the <a href="ascorbate-glutathione_cycle" title="wikilink">ascorbate-glutathione cycle</a>, and the NADP-dehydrogenases of the pentose-phosphate pathway. It has been demonstrated that peroxisomes generate <a class="uri" href="superoxide" title="wikilink">superoxide</a> (O<sub>2</sub><sup>•−</sup>) and <a href="nitric_oxide" title="wikilink">nitric oxide</a> (<sup>•</sup>NO) radicals.<a class="footnoteRef" href="#fn14" id="fnref14"><sup>14</sup></a><a class="footnoteRef" href="#fn15" id="fnref15"><sup>15</sup></a></p>

<p>The peroxisome of plant cells is polarised when fighting fungal penetration. Infection causes a <a class="uri" href="glucosinolate" title="wikilink">glucosinolate</a> molecule to play an antifungal role to be made and delivered to the outside of the cell through the action of the peroxisomal proteins (PEN2 and PEN3).<a class="footnoteRef" href="#fn16" id="fnref16"><sup>16</sup></a></p>
<h2 id="peroxisome-assembly">Peroxisome assembly</h2>

<p>Peroxisomes can be derived from the <a href="endoplasmic_reticulum" title="wikilink">endoplasmic reticulum</a> and replicate by fission.<a class="footnoteRef" href="#fn17" id="fnref17"><sup>17</sup></a> Peroxisome matrix proteins are translated in the cytoplasm prior to import. Specific amino acid sequences (PTS or <a href="peroxisomal_targeting_signal" title="wikilink">peroxisomal targeting signal</a>) at the <em><a class="uri" href="C-terminus" title="wikilink">C-terminus</a></em> (PTS1) or <em><a class="uri" href="N-terminus" title="wikilink">N-terminus</a></em> (PTS2) of peroxisomal matrix proteins signals them to be imported into the organelle. There are at least 32 known peroxisomal proteins, called <a href="peroxin" title="wikilink">peroxins</a>,<a class="footnoteRef" href="#fn18" id="fnref18"><sup>18</sup></a> which participate in the process of peroxisome assembly. Proteins do not have to unfold to be imported into the peroxisome. The protein receptors, the peroxins <em><a class="uri" href="PEX5" title="wikilink">PEX5</a></em> and <em><a class="uri" href="PEX7" title="wikilink">PEX7</a></em>, accompany their cargoes (containing a PTS1 or a PTS2 amino acid sequence, respectively) all the way into the peroxisome where they release the cargo and then return to the <a class="uri" href="cytosol" title="wikilink">cytosol</a> - a step named <em>recycling</em>. A model describing the import cycle is referred to as the <em>extended shuttle mechanism</em>.<a class="footnoteRef" href="#fn19" id="fnref19"><sup>19</sup></a> There is now evidence that ATP hydrolysis is required for the recycling of receptors to the <a class="uri" href="cytosol" title="wikilink">cytosol</a>. Also, <a class="uri" href="ubiquitination" title="wikilink">ubiquitination</a> appears to be crucial for the export of PEX5 from the peroxisome, to the cytosol.</p>
<h2 id="associated-medical-conditions">Associated medical conditions</h2>

<p><a href="Peroxisomal_disorders" title="wikilink">Peroxisomal disorders</a> are a class of medical conditions that typically affect the human nervous system as well as many other organ systems. Two common examples are <a class="uri" href="X-linked" title="wikilink">X-linked</a> <a class="uri" href="adrenoleukodystrophy" title="wikilink">adrenoleukodystrophy</a> and <a href="peroxisome_biogenesis_disorders" title="wikilink">peroxisome biogenesis disorders</a>.<a class="footnoteRef" href="#fn20" id="fnref20"><sup>20</sup></a><a class="footnoteRef" href="#fn21" id="fnref21"><sup>21</sup></a></p>
<h2 id="genes">Genes</h2>

<p><em>PEX</em> genes encode the protein machinery ("peroxins") required for proper peroxisome assembly, as described above. Membrane assembly and maintenance requires three of these (peroxins 3, 16, and 19) and may occur without the import of the matrix (lumen) enzymes. Proliferation of the organelle is regulated by Pex11p.</p>

<p>Genes that encode peroxin proteins include: <a class="uri" href="PEX1" title="wikilink">PEX1</a>, <a class="uri" href="PEX2" title="wikilink">PEX2</a> - <a class="uri" href="PXMP3" title="wikilink">PXMP3</a>, <a class="uri" href="PEX3" title="wikilink">PEX3</a>, <a class="uri" href="PEX5" title="wikilink">PEX5</a>, <a class="uri" href="PEX6" title="wikilink">PEX6</a>, <a class="uri" href="PEX7" title="wikilink">PEX7</a>, <a class="uri" href="PEX10" title="wikilink">PEX10</a>, <a class="uri" href="PEX11A" title="wikilink">PEX11A</a>, <a class="uri" href="PEX11B" title="wikilink">PEX11B</a>, <a class="uri" href="PEX11G" title="wikilink">PEX11G</a>, <a class="uri" href="PEX12" title="wikilink">PEX12</a>, <a class="uri" href="PEX13" title="wikilink">PEX13</a>, <a class="uri" href="PEX14" title="wikilink">PEX14</a>, <a class="uri" href="PEX16" title="wikilink">PEX16</a>, <a class="uri" href="PEX19" title="wikilink">PEX19</a>, <a class="uri" href="PEX26" title="wikilink">PEX26</a>, <a class="uri" href="PEX28" title="wikilink">PEX28</a>, <a class="uri" href="PEX30" title="wikilink">PEX30</a>, and <a class="uri" href="PEX31" title="wikilink">PEX31</a>.</p>
<h2 id="evolutionary-origins">Evolutionary origins</h2>

<p>The protein content of peroxisomes varies across species, but the presence of proteins common to many species has been used to suggest an <a href="Endosymbiotic_theory" title="wikilink">endosymbiotic</a> origin; that is, peroxisomes evolved from bacteria that invaded larger cells as parasites, and very gradually evolved a symbiotic relationship.<a class="footnoteRef" href="#fn22" id="fnref22"><sup>22</sup></a> However, this view has been challenged by recent discoveries.<a class="footnoteRef" href="#fn23" id="fnref23"><sup>23</sup></a> For example, peroxisome-less mutants can restore peroxisomes upon introduction of the wild-type gene.</p>

<p>Two independent evolutionary analyses of the peroxisomal <a class="uri" href="proteome" title="wikilink">proteome</a> found homologies between the peroxisomal import machinery and the <a href="Endoplasmic_Reticulum_Associated_Protein_Degradation" title="wikilink">ERAD</a> pathway in the <a href="endoplasmic_reticulum" title="wikilink">endoplasmic reticulum</a>,<a class="footnoteRef" href="#fn24" id="fnref24"><sup>24</sup></a><a class="footnoteRef" href="#fn25" id="fnref25"><sup>25</sup></a> along with a number of metabolic enzymes that were likely recruited from the <a class="uri" href="mitochondria" title="wikilink">mitochondria</a>.<a class="footnoteRef" href="#fn26" id="fnref26"><sup>26</sup></a> Recently, it has been suggested that the peroxisome may have had an <a href="actinobacteria" title="wikilink">actinobacterial</a> origin,<a class="footnoteRef" href="#fn27" id="fnref27"><sup>27</sup></a> however, this is controversial.<a class="footnoteRef" href="#fn28" id="fnref28"><sup>28</sup></a></p>
<h2 id="other-related-organelles">Other related organelles</h2>

<p>Other organelles of the <a class="uri" href="microbody" title="wikilink">microbody</a> family related to peroxisomes include <a href="glyoxysome" title="wikilink">glyoxysomes</a> of <a href="plant" title="wikilink">plants</a> and <a href="filamentous_fungi" title="wikilink">filamentous fungi</a>, <a href="glycosome" title="wikilink">glycosomes</a> of <a href="Kinetoplastida" title="wikilink">kinetoplastids</a><a class="footnoteRef" href="#fn29" id="fnref29"><sup>29</sup></a> and <a href="Woronin_body" title="wikilink">Woronin bodies</a> of <a href="filamentous_fungi" title="wikilink">filamentous fungi</a>.</p>
<h2 id="references">References</h2>
<h2 id="further-reading">Further reading</h2>
<ul>
<li></li>
</ul>
<ul>
<li></li>
</ul>
<h2 id="external-links">External links</h2>
<ul>
<li><a href="http://www.peroxisomeDB.org">PeroxisomeDB: Peroxisome-Database</a></li>
<li><a href="http://www.peroxisomekb.nl">PeroxisomeKB: Peroxisome Knowledge Base</a></li>
</ul>

<p>"</p>

<p><a class="uri" href="Category:Organelles" title="wikilink">Category:Organelles</a> <a class="uri" href="Category:Metabolism" title="wikilink">Category:Metabolism</a> <a href="Category:Articles_containing_video_clips" title="wikilink">Category:Articles containing video clips</a></p>
<section class="footnotes">
<hr/>
<ol>
<li id="fn1"><a href="#fnref1">↩</a></li>
<li id="fn2"><a href="#fnref2">↩</a></li>
<li id="fn3"><a href="#fnref3">↩</a></li>
<li id="fn4"><a href="#fnref4">↩</a></li>
<li id="fn5"></li>
<li id="fn6"></li>
<li id="fn7"><a href="#fnref7">↩</a></li>
<li id="fn8"><a href="#fnref8">↩</a></li>
<li id="fn9"><a href="#fnref9">↩</a></li>
<li id="fn10"><a href="#fnref10">↩</a></li>
<li id="fn11"></li>
<li id="fn12"><a href="#fnref12">↩</a></li>
<li id="fn13"></li>
<li id="fn14"><a href="#fnref14">↩</a></li>
<li id="fn15"><a href="#fnref15">↩</a></li>
<li id="fn16"><a href="#fnref16">↩</a></li>
<li id="fn17"><a href="#fnref17">↩</a></li>
<li id="fn18"><a href="#fnref18">↩</a></li>
<li id="fn19"><a href="#fnref19">↩</a></li>
<li id="fn20"><a href="#fnref20">↩</a></li>
<li id="fn21"><a href="#fnref21">↩</a></li>
<li id="fn22"><a href="#fnref22">↩</a></li>
<li id="fn23"><a href="#fnref23">↩</a></li>
<li id="fn24"><a href="#fnref24">↩</a></li>
<li id="fn25"><a href="#fnref25">↩</a></li>
<li id="fn26"></li>
<li id="fn27"><a href="#fnref27">↩</a></li>
<li id="fn28"><a href="#fnref28">↩</a></li>
<li id="fn29"><a href="#fnref29">↩</a></li>
</ol>
</section>
</body>
</html>
