<html lang="en">
<head>
<meta charset="utf-8"/>
<title offset="877">Quasispecies model</title>
<script src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_SVG.js" type="text/javascript">
</script>
</head>
<body>
<h1>Quasispecies model</h1>
<hr/>

<p>The <strong>quasispecies model</strong> is a description of the process of the Darwinian <a class="uri" href="evolution" title="wikilink">evolution</a> of certain <a href="self-replication" title="wikilink">self-replicating</a> entities within the framework of physical chemistry. Put simply, a quasispecies is a large group or <strong>cloud</strong> of related <a href="genotype" title="wikilink">genotypes</a> that exist in an environment of high mutation rate, where a large fraction of offspring are expected to contain one or more mutations relative to the parent. This is in contrast to a <a class="uri" href="species" title="wikilink">species</a>, which from an evolutionary perspective is a more-or-less stable single genotype, most of the offspring of which will be genetically accurate copies.<a class="footnoteRef" href="#fn1" id="fnref1"><sup>1</sup></a></p>

<p>It is useful mainly in providing a qualitative understanding of the evolutionary processes of self-replicating macromolecules such as <a class="uri" href="RNA" title="wikilink">RNA</a> or <a class="uri" href="DNA" title="wikilink">DNA</a> or simple asexual organisms such as <a href="bacterium" title="wikilink">bacteria</a> or <a href="virus" title="wikilink">viruses</a> (see also <a href="viral_quasispecies" title="wikilink">viral quasispecies</a>), and is helpful in explaining something of the early stages of the <a href="origin_of_life" title="wikilink">origin of life</a>. Quantitative predictions based on this model are difficult because the parameters that serve as its input are impossible to obtain from actual biological systems. The quasispecies model was put forward by <a href="Manfred_Eigen" title="wikilink">Manfred Eigen</a> and <a href="Peter_Schuster" title="wikilink">Peter Schuster</a><a class="footnoteRef" href="#fn2" id="fnref2"><sup>2</sup></a> based on initial work done by Eigen.<a class="footnoteRef" href="#fn3" id="fnref3"><sup>3</sup></a></p>
<h2 id="simplified-explanation">Simplified explanation</h2>

<p>When evolutionary biologists describe competition between species, they generally assume that each species is a single genotype whose descendants are mostly accurate copies. (Such genotypes are said to have a high reproductive <em>fidelity</em>.) Evolutionarily, we are interested in the behavior and fitness of that one species or genotype over time.</p>

<p>Some organisms or genotypes, however, may exist in circumstances of low fidelity, where most descendants contain one or more mutations. A group of such genotypes is constantly changing, so discussions of which single genotype is the most fit become meaningless. Importantly, if many closely related genotypes are only one mutation away from each other, then genotypes in the group can mutate back and forth into each other. For example, with one mutation per generation, a child of the sequence AGGT could be AGTT, and a grandchild could be AGGT again. Thus we can envision a <strong>cloud</strong> of related genotypes that is rapidly mutating, with sequences going back and forth among different points in the cloud. Though the proper definition is mathematical, that cloud, roughly speaking, is a quasispecies.</p>

<p>Quasispecies behavior exists for large numbers of individuals existing at a certain (high) range of mutation rates.</p>
<h3 id="quasispecies-fitness-and-evolutionary-selection">Quasispecies, fitness, and evolutionary selection</h3>

<p>In a species, though reproduction may be mostly accurate, periodic mutations will give rise to one or more competing genotypes. If a mutation results in greater replication and survival, the mutant genotype may out-compete the parent genotype and come to dominate the species. Thus, the individual genotypes (or species) may be seen as the units on which selection acts and biologists will often speak of a single genotype's <a href="fitness_(biology)" title="wikilink">fitness</a>.</p>

<p>In a quasispecies, however, mutations are ubiquitous and so the fitness of an individual genotype becomes meaningless: if one particular mutation generates a boost in reproductive success, it can't amount to much because that genotype's offspring are unlikely to be accurate copies with the same properties. Instead, what matters is the <em>connectedness</em> of the cloud.<a class="footnoteRef" href="#fn4" id="fnref4"><sup>4</sup></a> For example, the sequence AGGT has 12 (3+3+3+3) possible single point mutants AGGA, AGGG, and so on. If 10 of those mutants are viable genotypes that may reproduce (and some of whose offspring or grandchildren may mutate back into AGGT again), we would consider that sequence a well-connected node in the cloud. If instead only two of those mutants are viable, the rest being lethal mutations, then that sequence is poorly connected and most of its descendants will not reproduce. The analog of fitness for a quasispecies is the tendency of nearby relatives within the cloud to be well-connected, meaning that more of the mutant descendants will be viable and give rise to further descendants within the cloud.</p>

<p>When the fitness of a single genotype becomes meaningless because of the high rate of mutations, the cloud as a whole or quasispecies becomes the natural unit of selection.</p>
<h3 id="application-to-biological-research">Application to biological research</h3>

<p>Quasispecies represents the evolution of high-mutation-rate viruses such as <a class="uri" href="HIV" title="wikilink">HIV</a> and sometimes single genes or molecules within the genomes of other organisms.<a class="footnoteRef" href="#fn5" id="fnref5"><sup>5</sup></a><a class="footnoteRef" href="#fn6" id="fnref6"><sup>6</sup></a><a class="footnoteRef" href="#fn7" id="fnref7"><sup>7</sup></a> Quasispecies models have also been proposed by Jose Fontanari and <a href="Emmanuel_David_Tannenbaum" title="wikilink">Emmanuel David Tannenbaum</a> to model the evolution of sexual reproduction.<a class="footnoteRef" href="#fn8" id="fnref8"><sup>8</sup></a> Quasispecies was also shown in compositional replicators (based on the <a href="Gard_model" title="wikilink">Gard model</a> for <a class="uri" href="abiogenesis" title="wikilink">abiogenesis</a>) <a class="footnoteRef" href="#fn9" id="fnref9"><sup>9</sup></a> and was also suggested to be applicable to describe cell's replication, which amongst other things requires the maintenance and evolution of the internal composition of the parent and bud.</p>
<h2 id="formal-background">Formal background</h2>

<p>The model rests on four assumptions:</p>
<ol>
<li>The self-replicating entities can be represented as sequences composed of a small number of building blocks‚Äîfor example, sequences of RNA consisting of the four bases <a class="uri" href="adenine" title="wikilink">adenine</a>, <a class="uri" href="guanine" title="wikilink">guanine</a>, <a class="uri" href="cytosine" title="wikilink">cytosine</a>, and <a class="uri" href="uracil" title="wikilink">uracil</a>.</li>
<li>New sequences enter the system solely as the result of a copy process, either correct or erroneous, of other sequences that are already present.</li>
<li>The substrates, or raw materials, necessary for ongoing replication are always present in sufficient quantity. Excess sequences are washed away in an outgoing flux.</li>
<li>Sequences may decay into their building blocks. The probability of decay does not depend on the sequences' age; old sequences are just as likely to decay as young sequences.</li>
</ol>

<p>In the quasispecies model, <a href="mutation" title="wikilink">mutations</a> occur through errors made in the process of copying already existing sequences. Further, <a href="natural_selection" title="wikilink">selection</a> arises because different types of sequences tend to replicate at different rates, which leads to the suppression of sequences that replicate more slowly in favor of sequences that replicate faster. However, the quasispecies model does not predict the ultimate extinction of all but the fastest replicating sequence. Although the sequences that replicate more slowly cannot sustain their abundance level by themselves, they are constantly replenished as sequences that replicate faster mutate into them. At equilibrium, removal of slowly replicating sequences due to decay or outflow is balanced by replenishing, so that even relatively slowly replicating sequences can remain present in finite abundance.</p>

<p>Due to the ongoing production of mutant sequences, selection does not act on single sequences, but on mutational "clouds" of closely related sequences, referred to as <em>quasispecies</em>. In other words, the evolutionary success of a particular sequence depends not only on its own replication rate, but also on the replication rates of the mutant sequences it produces, and on the replication rates of the sequences of which it is a mutant. As a consequence, the sequence that replicates fastest may even disappear completely in selection-mutation equilibrium, in favor of more slowly replicating sequences that are part of a quasispecies with a higher average growth rate.<a class="footnoteRef" href="#fn10" id="fnref10"><sup>10</sup></a> Mutational clouds as predicted by the quasispecies model have been observed in RNA viruses and in <em>in vitro</em> RNA replication.<a class="footnoteRef" href="#fn11" id="fnref11"><sup>11</sup></a><a class="footnoteRef" href="#fn12" id="fnref12"><sup>12</sup></a></p>

<p>The mutation rate and the general fitness of the molecular sequences and their neighbors is crucial to the formation of a quasispecies. If the mutation rate is zero, there is no exchange by mutation, and each sequence is its own species. If the mutation rate is too high, exceeding what is known as the <a href="Error_threshold_(evolution)" title="wikilink">error threshold</a>, the quasispecies will break down and be dispersed over the entire range of available sequences.</p>
<h2 id="mathematical-description">Mathematical description</h2>

<p>A simple mathematical model for a quasispecies is as follows: let there be 

<math display="inline" id="Quasispecies_model:0">
 <semantics>
  <mi>S</mi>
  <annotation-xml encoding="MathML-Content">
   <ci>S</ci>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   S
  </annotation>
 </semantics>
</math>

 possible sequences and let there be 

<math display="inline" id="Quasispecies_model:1">
 <semantics>
  <msub>
   <mi>n</mi>
   <mi>i</mi>
  </msub>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <csymbol cd="ambiguous">subscript</csymbol>
    <ci>n</ci>
    <ci>i</ci>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   n_{i}
  </annotation>
 </semantics>
</math>

 organisms with sequence <em>i</em>. Let's say that each of these organisms asexually gives rise to 

<math display="inline" id="Quasispecies_model:2">
 <semantics>
  <msub>
   <mi>A</mi>
   <mi>i</mi>
  </msub>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <csymbol cd="ambiguous">subscript</csymbol>
    <ci>A</ci>
    <ci>i</ci>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   A_{i}
  </annotation>
 </semantics>
</math>

 offspring. Some are duplicates of their parent, having sequence <em>i</em>, but some are mutant and have some other sequence. Let the mutation rate 

<math display="inline" id="Quasispecies_model:3">
 <semantics>
  <msub>
   <mi>q</mi>
   <mrow>
    <mi>i</mi>
    <mi>j</mi>
   </mrow>
  </msub>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <csymbol cd="ambiguous">subscript</csymbol>
    <ci>q</ci>
    <apply>
     <times></times>
     <ci>i</ci>
     <ci>j</ci>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   q_{ij}
  </annotation>
 </semantics>
</math>


 correspond to the <a class="uri" href="probability" title="wikilink">probability</a> that a <em>j</em> type parent will produce an <em>i</em> type organism. Then the <a href="Expected_value" title="wikilink">expected</a> number of <em>i</em> type organisms produced by any <em>j</em> type parent is 

<math display="inline" id="Quasispecies_model:4">
 <semantics>
  <mrow>
   <msub>
    <mi>w</mi>
    <mrow>
     <mi>i</mi>
     <mi>j</mi>
    </mrow>
   </msub>
   <mo>=</mo>
   <mrow>
    <msub>
     <mi>A</mi>
     <mi>j</mi>
    </msub>
    <msub>
     <mi>q</mi>
     <mrow>
      <mi>i</mi>
      <mi>j</mi>
     </mrow>
    </msub>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <eq></eq>
    <apply>
     <csymbol cd="ambiguous">subscript</csymbol>
     <ci>w</ci>
     <apply>
      <times></times>
      <ci>i</ci>
      <ci>j</ci>
     </apply>
    </apply>
    <apply>
     <times></times>
     <apply>
      <csymbol cd="ambiguous">subscript</csymbol>
      <ci>A</ci>
      <ci>j</ci>
     </apply>
     <apply>
      <csymbol cd="ambiguous">subscript</csymbol>
      <ci>q</ci>
      <apply>
       <times></times>
       <ci>i</ci>
       <ci>j</ci>
      </apply>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   w_{ij}=A_{j}q_{ij}
  </annotation>
 </semantics>
</math>

,</p>

<p>where 

<math display="inline" id="Quasispecies_model:5">
 <semantics>
  <mrow>
   <mrow>
    <msub>
     <mo largeop="true" symmetric="true">‚àë</mo>
     <mi>i</mi>
    </msub>
    <msub>
     <mi>q</mi>
     <mrow>
      <mi>i</mi>
      <mi>j</mi>
     </mrow>
    </msub>
   </mrow>
   <mo>=</mo>
   <mpadded width="+1.7pt">
    <mn>1</mn>
   </mpadded>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <eq></eq>
    <apply>
     <apply>
      <csymbol cd="ambiguous">subscript</csymbol>
      <sum></sum>
      <ci>i</ci>
     </apply>
     <apply>
      <csymbol cd="ambiguous">subscript</csymbol>
      <ci>q</ci>
      <apply>
       <times></times>
       <ci>i</ci>
       <ci>j</ci>
      </apply>
     </apply>
    </apply>
    <cn type="integer">1</cn>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \sum_{i}q_{ij}=1\,
  </annotation>
 </semantics>
</math>

.</p>

<p>Then the total number of <em>i</em>-type organisms after the first round of reproduction, given as 

<math display="inline" id="Quasispecies_model:6">
 <semantics>
  <msubsup>
   <mi>n</mi>
   <mi>i</mi>
   <mo>‚Ä≤</mo>
  </msubsup>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <csymbol cd="ambiguous">subscript</csymbol>
    <apply>
     <csymbol cd="ambiguous">superscript</csymbol>
     <ci>n</ci>
     <ci>normal-‚Ä≤</ci>
    </apply>
    <ci>i</ci>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   n^{\prime}_{i}
  </annotation>
 </semantics>
</math>

, is</p>

<p>

<math display="block" id="Quasispecies_model:7">
 <semantics>
  <mrow>
   <msubsup>
    <mi>n</mi>
    <mi>i</mi>
    <mo>‚Ä≤</mo>
   </msubsup>
   <mo>=</mo>
   <mrow>
    <munder>
     <mo largeop="true" movablelimits="false" symmetric="true">‚àë</mo>
     <mi>j</mi>
    </munder>
    <mrow>
     <msub>
      <mi>w</mi>
      <mrow>
       <mi>i</mi>
       <mi>j</mi>
      </mrow>
     </msub>
     <mpadded width="+1.7pt">
      <msub>
       <mi>n</mi>
       <mi>j</mi>
      </msub>
     </mpadded>
    </mrow>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <eq></eq>
    <apply>
     <csymbol cd="ambiguous">subscript</csymbol>
     <apply>
      <csymbol cd="ambiguous">superscript</csymbol>
      <ci>n</ci>
      <ci>normal-‚Ä≤</ci>
     </apply>
     <ci>i</ci>
    </apply>
    <apply>
     <apply>
      <csymbol cd="ambiguous">subscript</csymbol>
      <sum></sum>
      <ci>j</ci>
     </apply>
     <apply>
      <times></times>
      <apply>
       <csymbol cd="ambiguous">subscript</csymbol>
       <ci>w</ci>
       <apply>
        <times></times>
        <ci>i</ci>
        <ci>j</ci>
       </apply>
      </apply>
      <apply>
       <csymbol cd="ambiguous">subscript</csymbol>
       <ci>n</ci>
       <ci>j</ci>
      </apply>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   n^{\prime}_{i}=\sum_{j}w_{ij}n_{j}\,
  </annotation>
 </semantics>
</math>

</p>

<p>Sometimes a death rate term 

<math display="inline" id="Quasispecies_model:8">
 <semantics>
  <msub>
   <mi>D</mi>
   <mi>i</mi>
  </msub>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <csymbol cd="ambiguous">subscript</csymbol>
    <ci>D</ci>
    <ci>i</ci>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   D_{i}
  </annotation>
 </semantics>
</math>


 is included so that:</p>

<p>

<math display="block" id="Quasispecies_model:9">
 <semantics>
  <mrow>
   <msub>
    <mi>w</mi>
    <mrow>
     <mi>i</mi>
     <mi>j</mi>
    </mrow>
   </msub>
   <mo>=</mo>
   <mrow>
    <mrow>
     <msub>
      <mi>A</mi>
      <mi>j</mi>
     </msub>
     <msub>
      <mi>q</mi>
      <mrow>
       <mi>i</mi>
       <mi>j</mi>
      </mrow>
     </msub>
    </mrow>
    <mo>-</mo>
    <mrow>
     <msub>
      <mi>D</mi>
      <mi>i</mi>
     </msub>
     <mpadded width="+1.7pt">
      <msub>
       <mi>Œ¥</mi>
       <mrow>
        <mi>i</mi>
        <mi>j</mi>
       </mrow>
      </msub>
     </mpadded>
    </mrow>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <eq></eq>
    <apply>
     <csymbol cd="ambiguous">subscript</csymbol>
     <ci>w</ci>
     <apply>
      <times></times>
      <ci>i</ci>
      <ci>j</ci>
     </apply>
    </apply>
    <apply>
     <minus></minus>
     <apply>
      <times></times>
      <apply>
       <csymbol cd="ambiguous">subscript</csymbol>
       <ci>A</ci>
       <ci>j</ci>
      </apply>
      <apply>
       <csymbol cd="ambiguous">subscript</csymbol>
       <ci>q</ci>
       <apply>
        <times></times>
        <ci>i</ci>
        <ci>j</ci>
       </apply>
      </apply>
     </apply>
     <apply>
      <times></times>
      <apply>
       <csymbol cd="ambiguous">subscript</csymbol>
       <ci>D</ci>
       <ci>i</ci>
      </apply>
      <apply>
       <csymbol cd="ambiguous">subscript</csymbol>
       <ci>Œ¥</ci>
       <apply>
        <times></times>
        <ci>i</ci>
        <ci>j</ci>
       </apply>
      </apply>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   w_{ij}=A_{j}q_{ij}-D_{i}\delta_{ij}\,
  </annotation>
 </semantics>
</math>

</p>

<p>where 

<math display="inline" id="Quasispecies_model:10">
 <semantics>
  <msub>
   <mi>Œ¥</mi>
   <mrow>
    <mi>i</mi>
    <mi>j</mi>
   </mrow>
  </msub>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <csymbol cd="ambiguous">subscript</csymbol>
    <ci>Œ¥</ci>
    <apply>
     <times></times>
     <ci>i</ci>
     <ci>j</ci>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \delta_{ij}
  </annotation>
 </semantics>
</math>

 is equal to 1 when i=j and is zero otherwise. Note that the <em>n-th</em> generation can be found by just taking the <em>n-th</em> power of <strong>W</strong> substituting it in place of <strong>W</strong> in the above formula.</p>

<p>This is just a <a href="system_of_linear_equations" title="wikilink">system of linear equations</a>. The usual way to solve such a system is to first <a href="Diagonalizable_matrix" title="wikilink">diagonalize</a> the <strong>W</strong> matrix. Its diagonal entries will be <a class="uri" href="eigenvalues" title="wikilink">eigenvalues</a> corresponding to certain linear combinations of certain subsets of sequences which will be <a class="uri" href="eigenvectors" title="wikilink">eigenvectors</a> of the <strong>W</strong> matrix. These subsets of sequences are the quasispecies. Assuming that the matrix <strong>W</strong> is a <a href="primitive_matrix" title="wikilink">primitive matrix</a> (<a href="Irreducibility_(mathematics)" title="wikilink">irreducible</a> and <a href="Perron‚ÄìFrobenius_theorem" title="wikilink">aperiodic</a>), then after very many generations only the eigenvector with the largest eigenvalue will prevail, and it is this quasispecies that will eventually dominate. The components of this eigenvector give the relative abundance of each sequence at equilibrium.</p>
<h3 id="note-about-primitive-matrices">Note about primitive matrices</h3>

<p><strong>W</strong> being primitive means that for some integer 

<math display="inline" id="Quasispecies_model:11">
 <semantics>
  <mrow>
   <mi>n</mi>
   <mo>></mo>
   <mn>0</mn>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <gt></gt>
    <ci>n</ci>
    <cn type="integer">0</cn>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   n>0
  </annotation>
 </semantics>
</math>

, that the 

<math display="inline" id="Quasispecies_model:12">
 <semantics>
  <msup>
   <mi>n</mi>
   <mrow>
    <mi>t</mi>
    <mi>h</mi>
   </mrow>
  </msup>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <csymbol cd="ambiguous">superscript</csymbol>
    <ci>n</ci>
    <apply>
     <times></times>
     <ci>t</ci>
     <ci>h</ci>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   n^{th}
  </annotation>
 </semantics>
</math>

 power of <strong>W</strong> is &gt; 0, i.e. all the entries are positive. If <strong>W</strong> is primitive then each type can, through a sequence of mutations (i.e. powers of <strong>W</strong>) mutate into all the other types after some number of generations. <strong>W</strong> is not primitive if it is periodic, where the population can perpetually cycle through different disjoint sets of compositions, or if it is reducible, where the dominant species (or quasispecies) that develops can depend on the initial population, as is the case in the simple example given below.</p>
<h3 id="alternative-formulations">Alternative formulations</h3>

<p>The quasispecies formulae may be expressed as a set of linear differential equations. If we consider the difference between the new state 

<math display="inline" id="Quasispecies_model:13">
 <semantics>
  <msubsup>
   <mi>n</mi>
   <mi>i</mi>
   <mo>‚Ä≤</mo>
  </msubsup>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <csymbol cd="ambiguous">subscript</csymbol>
    <apply>
     <csymbol cd="ambiguous">superscript</csymbol>
     <ci>n</ci>
     <ci>normal-‚Ä≤</ci>
    </apply>
    <ci>i</ci>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   n^{\prime}_{i}
  </annotation>
 </semantics>
</math>


 and the old state 

<math display="inline" id="Quasispecies_model:14">
 <semantics>
  <msub>
   <mi>n</mi>
   <mi>i</mi>
  </msub>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <csymbol cd="ambiguous">subscript</csymbol>
    <ci>n</ci>
    <ci>i</ci>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   n_{i}
  </annotation>
 </semantics>
</math>

 to be the state change over one moment of time, then we can state that the time derivative of 

<math display="inline" id="Quasispecies_model:15">
 <semantics>
  <msub>
   <mi>n</mi>
   <mi>i</mi>
  </msub>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <csymbol cd="ambiguous">subscript</csymbol>
    <ci>n</ci>
    <ci>i</ci>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   n_{i}
  </annotation>
 </semantics>
</math>

 is given by this difference, 

<math display="inline" id="Quasispecies_model:16">
 <semantics>
  <mrow>
   <msub>
    <mover accent="true">
     <mi>n</mi>
     <mo>Àô</mo>
    </mover>
    <mi>i</mi>
   </msub>
   <mo>=</mo>
   <mrow>
    <msubsup>
     <mi>n</mi>
     <mi>i</mi>
     <mo>‚Ä≤</mo>
    </msubsup>
    <mo>-</mo>
    <msub>
     <mi>n</mi>
     <mi>i</mi>
    </msub>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <eq></eq>
    <apply>
     <csymbol cd="ambiguous">subscript</csymbol>
     <apply>
      <ci>normal-Àô</ci>
      <ci>n</ci>
     </apply>
     <ci>i</ci>
    </apply>
    <apply>
     <minus></minus>
     <apply>
      <csymbol cd="ambiguous">subscript</csymbol>
      <apply>
       <csymbol cd="ambiguous">superscript</csymbol>
       <ci>n</ci>
       <ci>normal-‚Ä≤</ci>
      </apply>
      <ci>i</ci>
     </apply>
     <apply>
      <csymbol cd="ambiguous">subscript</csymbol>
      <ci>n</ci>
      <ci>i</ci>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \dot{n}_{i}=n^{\prime}_{i}-n_{i}
  </annotation>
 </semantics>
</math>

 we can write:</p>

<p>

<math display="block" id="Quasispecies_model:17">
 <semantics>
  <mrow>
   <msub>
    <mover accent="true">
     <mi>n</mi>
     <mo>Àô</mo>
    </mover>
    <mi>i</mi>
   </msub>
   <mo>=</mo>
   <mrow>
    <mrow>
     <munder>
      <mo largeop="true" movablelimits="false" symmetric="true">‚àë</mo>
      <mi>j</mi>
     </munder>
     <mrow>
      <msub>
       <mi>w</mi>
       <mrow>
        <mi>i</mi>
        <mi>j</mi>
       </mrow>
      </msub>
      <msub>
       <mi>n</mi>
       <mi>j</mi>
      </msub>
     </mrow>
    </mrow>
    <mo>-</mo>
    <mpadded width="+1.7pt">
     <msub>
      <mi>n</mi>
      <mi>i</mi>
     </msub>
    </mpadded>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <eq></eq>
    <apply>
     <csymbol cd="ambiguous">subscript</csymbol>
     <apply>
      <ci>normal-Àô</ci>
      <ci>n</ci>
     </apply>
     <ci>i</ci>
    </apply>
    <apply>
     <minus></minus>
     <apply>
      <apply>
       <csymbol cd="ambiguous">subscript</csymbol>
       <sum></sum>
       <ci>j</ci>
      </apply>
      <apply>
       <times></times>
       <apply>
        <csymbol cd="ambiguous">subscript</csymbol>
        <ci>w</ci>
        <apply>
         <times></times>
         <ci>i</ci>
         <ci>j</ci>
        </apply>
       </apply>
       <apply>
        <csymbol cd="ambiguous">subscript</csymbol>
        <ci>n</ci>
        <ci>j</ci>
       </apply>
      </apply>
     </apply>
     <apply>
      <csymbol cd="ambiguous">subscript</csymbol>
      <ci>n</ci>
      <ci>i</ci>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \dot{n}_{i}=\sum_{j}w_{ij}n_{j}-n_{i}\,
  </annotation>
 </semantics>
</math>

</p>

<p>The quasispecies equations are usually expressed in terms of concentrations 

<math display="inline" id="Quasispecies_model:18">
 <semantics>
  <msub>
   <mi>x</mi>
   <mi>i</mi>
  </msub>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <csymbol cd="ambiguous">subscript</csymbol>
    <ci>x</ci>
    <ci>i</ci>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   x_{i}
  </annotation>
 </semantics>
</math>


 where</p>

<p>

<math display="block" id="Quasispecies_model:19">
 <semantics>
  <mrow>
   <mpadded width="+5pt">
    <msub>
     <mi>x</mi>
     <mi>i</mi>
    </msub>
   </mpadded>
   <mpadded width="+5pt">
    <mover>
     <mo movablelimits="false">=</mo>
     <mi>def</mi>
    </mover>
   </mpadded>
   <mfrac>
    <msub>
     <mi>n</mi>
     <mi>i</mi>
    </msub>
    <mrow>
     <msub>
      <mo largeop="true" symmetric="true">‚àë</mo>
      <mi>j</mi>
     </msub>
     <msub>
      <mi>n</mi>
      <mi>j</mi>
     </msub>
    </mrow>
   </mfrac>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <apply>
     <csymbol cd="ambiguous">superscript</csymbol>
     <eq></eq>
     <ci>def</ci>
    </apply>
    <apply>
     <csymbol cd="ambiguous">subscript</csymbol>
     <ci>x</ci>
     <ci>i</ci>
    </apply>
    <apply>
     <divide></divide>
     <apply>
      <csymbol cd="ambiguous">subscript</csymbol>
      <ci>n</ci>
      <ci>i</ci>
     </apply>
     <apply>
      <apply>
       <csymbol cd="ambiguous">subscript</csymbol>
       <sum></sum>
       <ci>j</ci>
      </apply>
      <apply>
       <csymbol cd="ambiguous">subscript</csymbol>
       <ci>n</ci>
       <ci>j</ci>
      </apply>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   x_{i}\ \stackrel{\mathrm{def}}{=}\ \frac{n_{i}}{\sum_{j}n_{j}}
  </annotation>
 </semantics>
</math>

.</p>

<p>

<math display="block" id="Quasispecies_model:20">
 <semantics>
  <mrow>
   <mpadded width="+5pt">
    <msubsup>
     <mi>x</mi>
     <mi>i</mi>
     <mo>‚Ä≤</mo>
    </msubsup>
   </mpadded>
   <mpadded width="+5pt">
    <mover>
     <mo movablelimits="false">=</mo>
     <mi>def</mi>
    </mover>
   </mpadded>
   <mfrac>
    <msubsup>
     <mi>n</mi>
     <mi>i</mi>
     <mo>‚Ä≤</mo>
    </msubsup>
    <mrow>
     <msub>
      <mo largeop="true" symmetric="true">‚àë</mo>
      <mi>j</mi>
     </msub>
     <msubsup>
      <mi>n</mi>
      <mi>j</mi>
      <mo>‚Ä≤</mo>
     </msubsup>
    </mrow>
   </mfrac>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <apply>
     <csymbol cd="ambiguous">superscript</csymbol>
     <eq></eq>
     <ci>def</ci>
    </apply>
    <apply>
     <csymbol cd="ambiguous">subscript</csymbol>
     <apply>
      <csymbol cd="ambiguous">superscript</csymbol>
      <ci>x</ci>
      <ci>normal-‚Ä≤</ci>
     </apply>
     <ci>i</ci>
    </apply>
    <apply>
     <divide></divide>
     <apply>
      <csymbol cd="ambiguous">subscript</csymbol>
      <apply>
       <csymbol cd="ambiguous">superscript</csymbol>
       <ci>n</ci>
       <ci>normal-‚Ä≤</ci>
      </apply>
      <ci>i</ci>
     </apply>
     <apply>
      <apply>
       <csymbol cd="ambiguous">subscript</csymbol>
       <sum></sum>
       <ci>j</ci>
      </apply>
      <apply>
       <csymbol cd="ambiguous">subscript</csymbol>
       <apply>
        <csymbol cd="ambiguous">superscript</csymbol>
        <ci>n</ci>
        <ci>normal-‚Ä≤</ci>
       </apply>
       <ci>j</ci>
      </apply>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   x^{\prime}_{i}\ \stackrel{\mathrm{def}}{=}\ \frac{n^{\prime}_{i}}{\sum_{j}n^{%
\prime}_{j}}
  </annotation>
 </semantics>
</math>

.</p>

<p>The above equations for the quasispecies then become for the discrete version:</p>

<p>

<math display="block" id="Quasispecies_model:21">
 <semantics>
  <mrow>
   <msubsup>
    <mi>x</mi>
    <mi>i</mi>
    <mo>‚Ä≤</mo>
   </msubsup>
   <mo>=</mo>
   <mfrac>
    <mrow>
     <msub>
      <mo largeop="true" symmetric="true">‚àë</mo>
      <mi>j</mi>
     </msub>
     <mrow>
      <msub>
       <mi>w</mi>
       <mrow>
        <mi>i</mi>
        <mi>j</mi>
       </mrow>
      </msub>
      <msub>
       <mi>x</mi>
       <mi>j</mi>
      </msub>
     </mrow>
    </mrow>
    <mrow>
     <msub>
      <mo largeop="true" symmetric="true">‚àë</mo>
      <mrow>
       <mi>i</mi>
       <mi>j</mi>
      </mrow>
     </msub>
     <mrow>
      <msub>
       <mi>w</mi>
       <mrow>
        <mi>i</mi>
        <mi>j</mi>
       </mrow>
      </msub>
      <msub>
       <mi>x</mi>
       <mi>j</mi>
      </msub>
     </mrow>
    </mrow>
   </mfrac>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <eq></eq>
    <apply>
     <csymbol cd="ambiguous">subscript</csymbol>
     <apply>
      <csymbol cd="ambiguous">superscript</csymbol>
      <ci>x</ci>
      <ci>normal-‚Ä≤</ci>
     </apply>
     <ci>i</ci>
    </apply>
    <apply>
     <divide></divide>
     <apply>
      <apply>
       <csymbol cd="ambiguous">subscript</csymbol>
       <sum></sum>
       <ci>j</ci>
      </apply>
      <apply>
       <times></times>
       <apply>
        <csymbol cd="ambiguous">subscript</csymbol>
        <ci>w</ci>
        <apply>
         <times></times>
         <ci>i</ci>
         <ci>j</ci>
        </apply>
       </apply>
       <apply>
        <csymbol cd="ambiguous">subscript</csymbol>
        <ci>x</ci>
        <ci>j</ci>
       </apply>
      </apply>
     </apply>
     <apply>
      <apply>
       <csymbol cd="ambiguous">subscript</csymbol>
       <sum></sum>
       <apply>
        <times></times>
        <ci>i</ci>
        <ci>j</ci>
       </apply>
      </apply>
      <apply>
       <times></times>
       <apply>
        <csymbol cd="ambiguous">subscript</csymbol>
        <ci>w</ci>
        <apply>
         <times></times>
         <ci>i</ci>
         <ci>j</ci>
        </apply>
       </apply>
       <apply>
        <csymbol cd="ambiguous">subscript</csymbol>
        <ci>x</ci>
        <ci>j</ci>
       </apply>
      </apply>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   x^{\prime}_{i}=\frac{\sum_{j}w_{ij}x_{j}}{\sum_{ij}w_{ij}x_{j}}
  </annotation>
 </semantics>
</math>

</p>

<p>or, for the continuum version:</p>

<p>

<math display="block" id="Quasispecies_model:22">
 <semantics>
  <mrow>
   <mrow>
    <msub>
     <mover accent="true">
      <mi>x</mi>
      <mo>Àô</mo>
     </mover>
     <mi>i</mi>
    </msub>
    <mo>=</mo>
    <mrow>
     <mrow>
      <munder>
       <mo largeop="true" movablelimits="false" symmetric="true">‚àë</mo>
       <mi>j</mi>
      </munder>
      <mrow>
       <msub>
        <mi>w</mi>
        <mrow>
         <mi>i</mi>
         <mi>j</mi>
        </mrow>
       </msub>
       <msub>
        <mi>x</mi>
        <mi>j</mi>
       </msub>
      </mrow>
     </mrow>
     <mo>-</mo>
     <mrow>
      <msub>
       <mi>x</mi>
       <mi>i</mi>
      </msub>
      <mrow>
       <munder>
        <mo largeop="true" movablelimits="false" symmetric="true">‚àë</mo>
        <mrow>
         <mi>i</mi>
         <mi>j</mi>
        </mrow>
       </munder>
       <mrow>
        <msub>
         <mi>w</mi>
         <mrow>
          <mi>i</mi>
          <mi>j</mi>
         </mrow>
        </msub>
        <msub>
         <mi>x</mi>
         <mi>j</mi>
        </msub>
       </mrow>
      </mrow>
     </mrow>
    </mrow>
   </mrow>
   <mo>.</mo>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <eq></eq>
    <apply>
     <csymbol cd="ambiguous">subscript</csymbol>
     <apply>
      <ci>normal-Àô</ci>
      <ci>x</ci>
     </apply>
     <ci>i</ci>
    </apply>
    <apply>
     <minus></minus>
     <apply>
      <apply>
       <csymbol cd="ambiguous">subscript</csymbol>
       <sum></sum>
       <ci>j</ci>
      </apply>
      <apply>
       <times></times>
       <apply>
        <csymbol cd="ambiguous">subscript</csymbol>
        <ci>w</ci>
        <apply>
         <times></times>
         <ci>i</ci>
         <ci>j</ci>
        </apply>
       </apply>
       <apply>
        <csymbol cd="ambiguous">subscript</csymbol>
        <ci>x</ci>
        <ci>j</ci>
       </apply>
      </apply>
     </apply>
     <apply>
      <times></times>
      <apply>
       <csymbol cd="ambiguous">subscript</csymbol>
       <ci>x</ci>
       <ci>i</ci>
      </apply>
      <apply>
       <apply>
        <csymbol cd="ambiguous">subscript</csymbol>
        <sum></sum>
        <apply>
         <times></times>
         <ci>i</ci>
         <ci>j</ci>
        </apply>
       </apply>
       <apply>
        <times></times>
        <apply>
         <csymbol cd="ambiguous">subscript</csymbol>
         <ci>w</ci>
         <apply>
          <times></times>
          <ci>i</ci>
          <ci>j</ci>
         </apply>
        </apply>
        <apply>
         <csymbol cd="ambiguous">subscript</csymbol>
         <ci>x</ci>
         <ci>j</ci>
        </apply>
       </apply>
      </apply>
     </apply>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \dot{x}_{i}=\sum_{j}w_{ij}x_{j}-x_{i}\sum_{ij}w_{ij}x_{j}.
  </annotation>
 </semantics>
</math>

</p>
<h3 id="simple-example">Simple example</h3>

<p>The quasispecies concept can be illustrated by a simple system consisting of 4 sequences. Sequences [0,0], [0,1], [1,0], and [1,1] are numbered 1, 2, 3, and 4, respectively. Let's say the [0,0] sequence never mutates and always produces a single offspring. Let's say the other 3 sequences all produce, on average, 

<math display="inline" id="Quasispecies_model:23">
 <semantics>
  <mrow>
   <mn>1</mn>
   <mo>-</mo>
   <mi>k</mi>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <minus></minus>
    <cn type="integer">1</cn>
    <ci>k</ci>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   1-k
  </annotation>
 </semantics>
</math>


 replicas of themselves, and 

<math display="inline" id="Quasispecies_model:24">
 <semantics>
  <mi>k</mi>
  <annotation-xml encoding="MathML-Content">
   <ci>k</ci>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   k
  </annotation>
 </semantics>
</math>

 of each of the other two types, where 

<math display="inline" id="Quasispecies_model:25">
 <semantics>
  <mrow>
   <mn>0</mn>
   <mo>‚â§</mo>
   <mi>k</mi>
   <mo>‚â§</mo>
   <mn>1</mn>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <and></and>
    <apply>
     <leq></leq>
     <cn type="integer">0</cn>
     <ci>k</ci>
    </apply>
    <apply>
     <leq></leq>
     <share href="#.cmml">
     </share>
     <cn type="integer">1</cn>
    </apply>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   0\leq k\leq 1
  </annotation>
 </semantics>
</math>

. The <strong>W</strong> matrix is then:</p>

<p>

<math display="block" id="Quasispecies_model:26">
 <semantics>
  <mrow>
   <mi>ùêñ</mi>
   <mo>=</mo>
   <mrow>
    <mo>[</mo>
    <mtable displaystyle="true">
     <mtr>
      <mtd columnalign="center">
       <mn>1</mn>
      </mtd>
      <mtd columnalign="center">
       <mn>0</mn>
      </mtd>
      <mtd columnalign="center">
       <mn>0</mn>
      </mtd>
      <mtd columnalign="center">
       <mn>0</mn>
      </mtd>
     </mtr>
     <mtr>
      <mtd columnalign="center">
       <mn>0</mn>
      </mtd>
      <mtd columnalign="center">
       <mrow>
        <mn>1</mn>
        <mo>-</mo>
        <mi>k</mi>
       </mrow>
      </mtd>
      <mtd columnalign="center">
       <mrow>
        <mi>k</mi>
        <mo>;</mo>
       </mrow>
      </mtd>
      <mtd columnalign="center">
       <mrow>
        <mi>k</mi>
        <mo>;</mo>
       </mrow>
      </mtd>
     </mtr>
     <mtr>
      <mtd columnalign="center">
       <mn>0</mn>
      </mtd>
      <mtd columnalign="center">
       <mrow>
        <mi>k</mi>
        <mo>;</mo>
       </mrow>
      </mtd>
      <mtd columnalign="center">
       <mrow>
        <mn>1</mn>
        <mo>-</mo>
        <mi>k</mi>
       </mrow>
      </mtd>
      <mtd columnalign="center">
       <mrow>
        <mi>k</mi>
        <mo>;</mo>
       </mrow>
      </mtd>
     </mtr>
     <mtr>
      <mtd columnalign="center">
       <mn>0</mn>
      </mtd>
      <mtd columnalign="center">
       <mrow>
        <mi>k</mi>
        <mo>;</mo>
       </mrow>
      </mtd>
      <mtd columnalign="center">
       <mrow>
        <mi>k</mi>
        <mo>;</mo>
       </mrow>
      </mtd>
      <mtd columnalign="center">
       <mrow>
        <mn>1</mn>
        <mo>-</mo>
        <mi>k</mi>
       </mrow>
      </mtd>
     </mtr>
    </mtable>
    <mo>]</mo>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <eq></eq>
    <ci>ùêñ</ci>
    <matrix>
     <matrixrow>
      <cn type="integer">1</cn>
      <cn type="integer">0</cn>
      <cn type="integer">0</cn>
      <cn type="integer">0</cn>
     </matrixrow>
     <matrixrow>
      <cn type="integer">0</cn>
      <apply>
       <minus></minus>
       <cn type="integer">1</cn>
       <ci>k</ci>
      </apply>
      <ci>k</ci>
      <ci>k</ci>
     </matrixrow>
     <matrixrow>
      <cn type="integer">0</cn>
      <ci>k</ci>
      <apply>
       <minus></minus>
       <cn type="integer">1</cn>
       <ci>k</ci>
      </apply>
      <ci>k</ci>
     </matrixrow>
     <matrixrow>
      <cn type="integer">0</cn>
      <ci>k</ci>
      <ci>k</ci>
      <apply>
       <minus></minus>
       <cn type="integer">1</cn>
       <ci>k</ci>
      </apply>
     </matrixrow>
    </matrix>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \mathbf{W}=\begin{bmatrix}1&0&0&0\\
0&1-k&k;&k;\\
0&k;&1-k&k;\\
0&k;&k;&1-k\end{bmatrix}
  </annotation>
 </semantics>
</math>

.</p>

<p>The diagonalized matrix is:</p>

<p>

<math display="block" id="Quasispecies_model:27">
 <semantics>
  <mrow>
   <msup>
    <mi>ùêñ</mi>
    <mo>‚Ä≤</mo>
   </msup>
   <mo>=</mo>
   <mrow>
    <mo>[</mo>
    <mtable displaystyle="true">
     <mtr>
      <mtd columnalign="center">
       <mrow>
        <mn>1</mn>
        <mo>-</mo>
        <mrow>
         <mn>2</mn>
         <mi>k</mi>
        </mrow>
       </mrow>
      </mtd>
      <mtd columnalign="center">
       <mn>0</mn>
      </mtd>
      <mtd columnalign="center">
       <mn>0</mn>
      </mtd>
      <mtd columnalign="center">
       <mn>0</mn>
      </mtd>
     </mtr>
     <mtr>
      <mtd columnalign="center">
       <mn>0</mn>
      </mtd>
      <mtd columnalign="center">
       <mrow>
        <mn>1</mn>
        <mo>-</mo>
        <mrow>
         <mn>2</mn>
         <mi>k</mi>
        </mrow>
       </mrow>
      </mtd>
      <mtd columnalign="center">
       <mn>0</mn>
      </mtd>
      <mtd columnalign="center">
       <mn>0</mn>
      </mtd>
     </mtr>
     <mtr>
      <mtd columnalign="center">
       <mn>0</mn>
      </mtd>
      <mtd columnalign="center">
       <mn>0</mn>
      </mtd>
      <mtd columnalign="center">
       <mn>1</mn>
      </mtd>
      <mtd columnalign="center">
       <mn>0</mn>
      </mtd>
     </mtr>
     <mtr>
      <mtd columnalign="center">
       <mn>0</mn>
      </mtd>
      <mtd columnalign="center">
       <mn>0</mn>
      </mtd>
      <mtd columnalign="center">
       <mn>0</mn>
      </mtd>
      <mtd columnalign="center">
       <mrow>
        <mn>1</mn>
        <mo>+</mo>
        <mi>k</mi>
       </mrow>
      </mtd>
     </mtr>
    </mtable>
    <mo>]</mo>
   </mrow>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <eq></eq>
    <apply>
     <csymbol cd="ambiguous">superscript</csymbol>
     <ci>ùêñ</ci>
     <ci>normal-‚Ä≤</ci>
    </apply>
    <matrix>
     <matrixrow>
      <apply>
       <minus></minus>
       <cn type="integer">1</cn>
       <apply>
        <times></times>
        <cn type="integer">2</cn>
        <ci>k</ci>
       </apply>
      </apply>
      <cn type="integer">0</cn>
      <cn type="integer">0</cn>
      <cn type="integer">0</cn>
     </matrixrow>
     <matrixrow>
      <cn type="integer">0</cn>
      <apply>
       <minus></minus>
       <cn type="integer">1</cn>
       <apply>
        <times></times>
        <cn type="integer">2</cn>
        <ci>k</ci>
       </apply>
      </apply>
      <cn type="integer">0</cn>
      <cn type="integer">0</cn>
     </matrixrow>
     <matrixrow>
      <cn type="integer">0</cn>
      <cn type="integer">0</cn>
      <cn type="integer">1</cn>
      <cn type="integer">0</cn>
     </matrixrow>
     <matrixrow>
      <cn type="integer">0</cn>
      <cn type="integer">0</cn>
      <cn type="integer">0</cn>
      <apply>
       <plus></plus>
       <cn type="integer">1</cn>
       <ci>k</ci>
      </apply>
     </matrixrow>
    </matrix>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   \mathbf{W^{\prime}}=\begin{bmatrix}1-2k&0&0&0\\
0&1-2k&0&0\\
0&0&1&0\\
0&0&0&1+k\end{bmatrix}
  </annotation>
 </semantics>
</math>

.</p>

<p>And the eigenvectors corresponding to these eigenvalues are:</p>
<dl>
<dd>
<table>
<thead>
<tr class="header">
<th style="text-align: left;">
<p><a class="uri" href="Eigenvalue" title="wikilink">Eigenvalue</a></p></th>
<th style="text-align: left;">
<p><a class="uri" href="Eigenvector" title="wikilink">Eigenvector</a></p></th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;">
<p>1-2k</p></td>
<td style="text-align: left;">
<p>[0,-1,0,1]</p></td>
</tr>
<tr class="even">
<td style="text-align: left;">
<p>1-2k</p></td>
<td style="text-align: left;">
<p>[0,-1,1,0]</p></td>
</tr>
<tr class="odd">
<td style="text-align: left;">
<p>1</p></td>
<td style="text-align: left;">
<p>[1,0,0,0]</p></td>
</tr>
<tr class="even">
<td style="text-align: left;">
<p>1+k</p></td>
<td style="text-align: left;">
<p>[0,1,1,1]</p></td>
</tr>
</tbody>
</table>
</dd>
</dl>

<p>Only the eigenvalue 

<math display="inline" id="Quasispecies_model:28">
 <semantics>
  <mrow>
   <mn>1</mn>
   <mo>+</mo>
   <mi>k</mi>
  </mrow>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <plus></plus>
    <cn type="integer">1</cn>
    <ci>k</ci>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   1+k
  </annotation>
 </semantics>
</math>

 is more than unity. For the n-th generation, the corresponding eigenvalue will be 

<math display="inline" id="Quasispecies_model:29">
 <semantics>
  <msup>
   <mrow>
    <mo stretchy="false">(</mo>
    <mrow>
     <mn>1</mn>
     <mo>+</mo>
     <mi>k</mi>
    </mrow>
    <mo stretchy="false">)</mo>
   </mrow>
   <mi>n</mi>
  </msup>
  <annotation-xml encoding="MathML-Content">
   <apply>
    <csymbol cd="ambiguous">superscript</csymbol>
    <apply>
     <plus></plus>
     <cn type="integer">1</cn>
     <ci>k</ci>
    </apply>
    <ci>n</ci>
   </apply>
  </annotation-xml>
  <annotation encoding="application/x-tex">
   (1+k)^{n}
  </annotation>
 </semantics>
</math>

 and so will increase without bound as time goes by. This eigenvalue corresponds to the eigenvector [0,1,1,1], which represents the quasispecies consisting of sequences 2, 3, and 4, which will be present in equal numbers after a very long time. Since all population numbers must be positive, the first two quasispecies are not legitimate. The third quasispecies consists of only the non-mutating sequence 1. It's seen that even though sequence 1 is the most fit in the sense that it reproduces more of itself than any other sequence, the quasispecies consisting of the other three sequences will eventually dominate (assuming that the initial population was not homogeneous of the sequence 1 type).</p>
<h2 id="references">References</h2>
<h2 id="further-reading">Further reading</h2>
<ul>
<li></li>
<li></li>
<li></li>
</ul>

<p>{{-}} </p>

<p>"</p>

<p><a class="uri" href="Category:Virology" title="wikilink">Category:Virology</a> <a href="Category:Evolutionary_biology" title="wikilink">Category:Evolutionary biology</a> <a href="Category:Microbial_population_biology" title="wikilink">Category:Microbial population biology</a> <a href="Category:Evolutionary_dynamics" title="wikilink">Category:Evolutionary dynamics</a> <a href="Category:Mathematical_modeling" title="wikilink">Category:Mathematical modeling</a></p>
<section class="footnotes">
<hr/>
<ol>
<li id="fn1"><a href="#fnref1">‚Ü©</a></li>
<li id="fn2"><a href="#fnref2">‚Ü©</a></li>
<li id="fn3"><a href="#fnref3">‚Ü©</a></li>
<li id="fn4">Villarreal, L.P. &amp; Witzany, G. (2013). ‚ÄûRethinking quasispecies theory: From fittest type to cooperative consortia.‚Äú World Journal of Biological Chemistry 4: 70-79. PMID 24340131 <a class="uri" href="doi:10.4331/wjbc.v4.i4.79">doi:10.4331/wjbc.v4.i4.79</a>. PMID 24340131<a href="#fnref4">‚Ü©</a></li>
<li id="fn5"><a href="#fnref5">‚Ü©</a></li>
<li id="fn6"><a href="#fnref6">‚Ü©</a></li>
<li id="fn7"><a href="#fnref7">‚Ü©</a></li>
<li id="fn8"><a href="#fnref8">‚Ü©</a></li>
<li id="fn9"><a href="#fnref9">‚Ü©</a></li>
<li id="fn10"><a href="#fnref10">‚Ü©</a></li>
<li id="fn11"><a href="#fnref11">‚Ü©</a></li>
<li id="fn12"><a href="#fnref12">‚Ü©</a></li>
</ol>
</section>
</body>
</html>
